<?xml version="1.0" encoding="iso-8859-1" ?> 
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" 
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">  
<!--http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd-->  
<html xmlns="http://www.w3.org/1999/xhtml"  
> 
<head><title></title> 
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" /> 
<meta name="generator" content="TeX4ht (http://www.cse.ohio-state.edu/~gurari/TeX4ht/)" /> 
<meta name="originator" content="TeX4ht (http://www.cse.ohio-state.edu/~gurari/TeX4ht/)" /> 
<!-- html,xhtml --> 
<meta name="src" content="20170511_thesis_introduction.tex" /> 
<meta name="date" content="2017-05-11 19:55:00" /> 
<link rel="stylesheet" type="text/css" href="20170511_thesis_introduction.css" /> 
</head><body 
>
                                                                                    
                                                                                    
<h2 class="likechapterHead"><a 
 id="x1-1000"></a>Contents</h2> <div class="tableofcontents">
<span class="chapterToc" >1 <a 
href="#x1-40001" id="QQ2-1-4">Introduction</a></span>
<br />&#x00A0;<span class="sectionToc" >1.1 <a 
href="#x1-50001" id="QQ2-1-5">Cancer Research in the Post-Genomic Era</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.1.1 <a 
href="#x1-60001" id="QQ2-1-6">Cancer as a Global Health Concern</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.1.1.1 <a 
href="#x1-70001" id="QQ2-1-7">The Genetics and Molecular Biology of Cancers</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.1.2 <a 
href="#x1-80002" id="QQ2-1-8">The Human Genome Revolution</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.1.2.1 <a 
href="#x1-90001" id="QQ2-1-9">The First Human Genome Sequence</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.1.2.2 <a 
href="#x1-100002" id="QQ2-1-10">Impact of Genomics</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.1.3 <a 
href="#x1-110003" id="QQ2-1-11">Technologies to Enable Genetics Research</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.1.3.1 <a 
href="#x1-120001" id="QQ2-1-12">DNA Sequencing and Genotyping Technologies</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.1.3.2 <a 
href="#x1-130002" id="QQ2-1-13">Microarrays and Quantitative Technologies</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.1.3.3 <a 
href="#x1-140003" id="QQ2-1-14">Massively Parallel &#x201C;Next Generation&#x201D; Sequencing</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.1.3.4 <a 
href="#x1-170004" id="QQ2-1-17">Bioinformatics as Interdisciplinary Genomic Analysis</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.1.4 <a 
href="#x1-180004" id="QQ2-1-18">Follow-up Large-Scale Genomics Projects</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.1.5 <a 
href="#x1-190005" id="QQ2-1-19">Cancer Genomes</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.1.5.1 <a 
href="#x1-200001" id="QQ2-1-20">The Cancer Genome Atlas Project</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.1.6 <a 
href="#x1-240006" id="QQ2-1-24">Genomic Cancer Medicine</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.1.6.1 <a 
href="#x1-250001" id="QQ2-1-25">Cancer Genes and Driver Mutations</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.1.6.2 <a 
href="#x1-260002" id="QQ2-1-26">Personalised or Precision Cancer Medicine</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.1.6.3 <a 
href="#x1-280003" id="QQ2-1-28">Targeted Therapeutics and Pharmacogenomics</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.1.6.4 <a 
href="#x1-300004" id="QQ2-1-30">Systems and Network Biology</a></span>
<br />&#x00A0;<span class="sectionToc" >1.2 <a 
href="#x1-320002" id="QQ2-1-32">A Synthetic Lethal Approach to Cancer Medicine</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.1 <a 
href="#x1-330001" id="QQ2-1-33">Synthetic Lethal Genetic Interactions</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.2 <a 
href="#x1-340002" id="QQ2-1-35">Synthetic Lethal Concepts in Genetics</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.3 <a 
href="#x1-350003" id="QQ2-1-36">Studies of Synthetic Lethality</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.3.1 <a 
href="#x1-360001" id="QQ2-1-37">Synthetic Lethal Pathways and Networks</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.4 <a 
href="#x1-380004" id="QQ2-1-39">Synthetic Lethal Concepts in Cancer</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.5 <a 
href="#x1-390005" id="QQ2-1-41">Clinical Impact of Synthetic Lethality in Cancer</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.6 <a 
href="#x1-400006" id="QQ2-1-42">High-throughput Screening for Synthetic Lethality</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.6.1 <a 
href="#x1-410001" id="QQ2-1-43">Synthetic Lethal Screens</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.7 <a 
href="#x1-420007" id="QQ2-1-44">Computational Prediction of Synthetic Lethality</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.7.1 <a 
href="#x1-430001" id="QQ2-1-45">Bioinformatics Approaches to Genetic Interactions</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.7.2 <a 
href="#x1-440002" id="QQ2-1-48">Comparative Genomics</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.7.3 <a 
href="#x1-450003" id="QQ2-1-50">Analysis and Modelling of Protein Data</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.7.4 <a 
href="#x1-460004" id="QQ2-1-51">Differential Gene Expression</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.7.5 <a 
href="#x1-470005" id="QQ2-1-52">Data Mining and Machine Learning</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.7.6 <a 
href="#x1-480006" id="QQ2-1-53">Bimodality</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.2.7.7 <a 
href="#x1-490007" id="QQ2-1-54">Rationale for Further Development</a></span>
                                                                                    
                                                                                    
<br />&#x00A0;<span class="sectionToc" >1.3 <a 
href="#x1-500003" id="QQ2-1-55">E-cadherin as a Synthetic Lethal Target</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.1 <a 
href="#x1-510001" id="QQ2-1-56">The <span 
class="cmti-12">CDH1 </span>gene and it&#x2019;s Biological Functions</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.3.1.1 <a 
href="#x1-520001" id="QQ2-1-57">Cytoskeleton</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.3.1.2 <a 
href="#x1-530002" id="QQ2-1-58">Extracellular and Tumour Micro-Environment</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.3.1.3 <a 
href="#x1-540003" id="QQ2-1-59">Cell-Cell Adhesion and Signalling</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.2 <a 
href="#x1-550002" id="QQ2-1-60"><span 
class="cmti-12">CDH1 </span>as a Tumour (and Invasion) Suppressor</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.3.2.1 <a 
href="#x1-560001" id="QQ2-1-61">Breast Cancers and Invasion</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.3 <a 
href="#x1-570003" id="QQ2-1-62">Hereditary Diffuse Gastric Cancer and Lobular Breast Cancer</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.4 <a 
href="#x1-580004" id="QQ2-1-63">Somatic Mutations</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.3.4.1 <a 
href="#x1-590001" id="QQ2-1-64">Mutation Rate</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.3.4.2 <a 
href="#x1-600002" id="QQ2-1-65">Co-occurring Mutations</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.5 <a 
href="#x1-610005" id="QQ2-1-66">Models of <span 
class="cmti-12">CDH1 </span>loss in cell lines</a></span>
<br />&#x00A0;<span class="sectionToc" >1.4 <a 
href="#x1-620004" id="QQ2-1-67">Summary and Research Direction of Thesis</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-68">                                           References</a></span>
</div>
                                                                                    
                                                                                    
<h2 class="likechapterHead"><a 
 id="x1-2000"></a>List of Figures</h2><div class="tableofcontents"><span class="lofToc" >1.1&#x00A0;<a 
href="#x1-33001r1">Synthetic genetic interactions</a></span><br /><span class="lofToc" >1.2&#x00A0;<a 
href="#x1-38001r2">Synthetic
lethality in cancer</a></span><br />
</div>
                                                                                    
                                                                                    
<h2 class="likechapterHead"><a 
 id="x1-3000"></a>List of Tables</h2><div class="tableofcontents"><span class="lotToc" >1.1&#x00A0;<a 
href="#x1-43001r1">Methods for Predicting Genetic Interactions</a></span><br /><span class="lotToc" >1.2&#x00A0;<a 
href="#x1-43002r2">Methods
for Predicting Synthetic Lethality in Cancer</a></span><br /><span class="lotToc" >1.3&#x00A0;<a 
href="#x1-44001r3">Methods used by
Wu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span>&#x00A0;(2014)</a></span><br />
</div>
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
<h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;1</span><br /><a 
 id="x1-40001"></a>Introduction</h2>
<!--l. 11--><p class="noindent" >The thesis presents research into genetic interactions based on <a 
href="#glo:genomics">genomics</a> data and
<a 
href="#glo:bioinformatics">bioinformatics</a> approaches. This Chapter introduces the recent developments in <a 
href="#glo:genomics">genomics</a> and
<a 
href="#glo:bioinformatics">bioinformatics</a>, particularly in their application to cancer research. <a 
href="#glo:synthetic lethal">Synthetic lethal</a>
interactions are a long standing area of research in genetics in both model organisms and
cancer biology. Various reasons why these interactions are of interest in fundamental and
translational biology will be outlined but first these and similar interactions will be defined.
A <a 
href="#glo:bioinformatics">bioinformatics</a> approach to <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions enables much wider exploration of
the inter-connected nature of genes and proteins within a cancer cell than previous
candidate-based approaches. An alternative approach is experimental screening which will be
presented and contrasted with <a 
href="#glo:bioinformatics">bioinformatics</a> approaches in more detail. An emerging
application of <a 
href="#glo:synthetic lethal">synthetic lethality</a> is the design of treatments with specificity against
loss of function mutations in tumour suppressor genes. <a 
href="#glo:E-cadherin">E-cadherin</a> (encoded by
<span 
class="cmti-12">CDH1 </span>) is a prime example of this which will be the focus of the analysis in this
thesis and as such the role of this gene in cellular and cancer biology will be briefly
reviewed.
<a 
 id="x1-4001r1"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">1.1 </span> <a 
 id="x1-50001"></a>Cancer Research in the Post-Genomic Era</h3>
<!--l. 19--><p class="noindent" ><a 
href="#glo:genomics">Genomics</a> technologies have the potential to vastly impact upon various areas
including health and cancer medicine. Considering the progress in recent <a 
href="#glo:genomics">genomics</a>
research, this technology and the findings from it have considerable potential for
significant impacts in the clinic and wider applications of genetics either directly or by
enabling more focused genetics research on candidates selected from <a 
href="#glo:genomics">genomics</a> or
<a 
href="#glo:bioinformatics">bioinformatics</a> analysis. The completion of the draft Human Genome (<a 
href="#XLander2001">Lander <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XLander2001">2001</a>)
marks a major accomplishment in genetics research and raises new challenges to
utilise this genomic scale information effectively. Technologies in this area have
rapidly developed since completion of the human genome project and many global
large-scale projects have expanded upon the human genome, to populations (<a 
href="#X1000Genomes2010">1000
Genomes</a>,&#x00A0;<a 
href="#X1000Genomes2010">2010</a>), to cancers (<a 
href="#XDickson1999">Dickson</a>,&#x00A0;<a 
href="#XDickson1999">1999</a>;&#x00A0;<a 
href="#XICGC2011">Zhang <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XICGC2011">2011</a>), and to deeper
functional understanding (<a 
href="#XFANTOM2001">Kawai <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XFANTOM2001">2001</a>;&#x00A0;<a 
href="#XENCODE2004">ENCODE</a>,&#x00A0;<a 
href="#XENCODE2004">2004</a>). However, impact on
the clinic has been slower than initially anticipated following the completion of
the &#x201C;draft&#x201D; genome with <a 
href="#glo:genomics">genomics</a> technologies yet to become widely adopted in
healthcare and oncology. Here we outline the <a 
href="#glo:genomics">genomics</a> technologies and <a 
href="#glo:bioinformatics">bioinformatics</a>
approaches which have led to availability of <a 
href="#glo:genomics">genomics</a> data and techiques used in
                                                                                    
                                                                                    
this thesis and potential for applications in cancer research or the clinic in the
future.
<a 
 id="x1-5001r1"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.1.1 </span> <a 
 id="x1-60001"></a>Cancer as a Global Health Concern</h4>
<!--l. 23--><p class="noindent" >Cancer is a class of diseases involving malignant cellular growth, invasion of tissues, and
spread to other organs. While there are also environmental factors, most cancers occur more
frequently with age and family history. Accordingly, genetics has become widely
acknowledged as having an important role in cancer risk (in addition to environmental
factors). Cancers arise from dysregulated cellular growth or differentiation from stem cells.
These can occur through genetic mutations or alterations in gene regulation or
expression.
</p><!--l. 25--><p class="indent" >Cancers are a major global health concern, being the second leading cause of death
globally (<a 
href="#XWorldHealthOrg2017">WHO</a>,&#x00A0;<a 
href="#XWorldHealthOrg2017">2017</a>), with an estimated annual incidence of 14.1 million cases and annual
mortality of 8.2 million people (<a 
href="#XFerlay2015">Ferlay <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XFerlay2015">2015</a>). Breast and stomach cancers are among
the 5 most frequent cancers globally, with breast cancer affecting women more than other
cancer tissue types. Breast cancer has an estimated annual incidence of 1.6 million cases and
mortality of 520 thousand people. Stomach cancer has an estimated annual incidence of 950
thousand cases and a mortality of 723 thousand people. Cancer is also a major health
concern here in New Zealand, with 19.1 thousand people (including 2.5 thousand cases of
breast cancer and 370 cases of stomach cancer) diagnosed annually (<a 
href="#XCIX2013">Hanna</a>,&#x00A0;<a 
href="#XCIX2013">2003</a>), among
the highest incidence (age-standardised per capita) of cancer in the world (<a 
href="#XFerlay2015">Ferlay
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XFerlay2015">2015</a>).
</p><!--l. 27--><p class="indent" >While the genetic contribution to cancer risk and many of the molecular changes
occurring in cancers are widely acknowledged (<a 
href="#XASCO2017">ASCO</a>,&#x00A0;<a 
href="#XASCO2017">2017</a>;&#x00A0;<a 
href="#XCancerResearchUK2017">Cancer Research
UK</a>,&#x00A0;<a 
href="#XCancerResearchUK2017">2017</a>;&#x00A0;<a 
href="#XCSNZ2017">Cancer Society of NZ</a>,&#x00A0;<a 
href="#XCSNZ2017">2017</a>), the majority of these findings have yet to impact
on clinical practice. Diagnostics are traditionally based on pathological examination of tissue
samples where histological staining for cell type, biomolecules and biomarkers continue to be
widely used, although genetic and biochemical markers are being adopted for some cancer
types. In general, the current standard of care includes surgery, radiation, and cytotoxic
chemotherapy, depending on whether the cancer is localised or has become systemic (via
metastasis) and spread to other organ systems. These approaches can be effective against
certain cancers, particularly in early stage cancer or in patients with particular
subtypes (such as acute myeloid leukaemia) which respond well to modern treatment
regimens. Thus early diagnosis is important to patient survival and quality of life.
                                                                                    
                                                                                    
National screening programs (which prioritise patients with a high risk of cancer)
therefore aim to diagnose cancers early and subtypes more accurately. Therefore
identification of patients with genetic variants or family histories (for inclusion in national
databases) for high risk of particular cancers is an important health issue, particularly
where effective interventions exist if these cancers are diagnosed early. Thus high
risk individuals being regularly monitored for some cancers are sometimes offered
preventative surgery or treatment for pre-cancerous tissue (<a 
href="#XGuilford2010">Guilford <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XGuilford2010">2010</a>;&#x00A0;<a 
href="#XScheuer2002">Scheuer
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XScheuer2002">2002</a>).
</p><!--l. 29--><p class="indent" >Chemotherapy is a last-resort treatment for many advanced stage (systemic) cancers
which is designed to inhibit the growth and spread of cancer throughout the body by
targeting rapidly growing cells. However, this approach often has severe adverse effects, a
narrow therapeutic window, and is not suitable for chemopreventative application in many
cases (<a 
href="#XKaelin2009">Kaelin, Jr</a>,&#x00A0;<a 
href="#XKaelin2009">2009</a>). Since surveillence preventative surgery (removing the tissue at risk
of cancer) is not completely effective at preventing cancers and may impact on quality of life,
depending on the cancer tissue types they are at risk of, alternative treatment strategies
based on molecular biology and other fields are being investigated. These alternatives include
immunological, endocrine, and targeted therapeutics, with a particular interest in
treatments with specificity against cancer cells and wider applications (i.e., tolerable
effective doses in applciations as a chemopreventative or against advanced stage
cancers).
<a 
 id="x1-6001r1"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.1.1.1 </span> <a 
 id="x1-70001"></a>The Genetics and Molecular Biology of Cancers</h5>
<!--l. 37--><p class="noindent" >Cancers involve dysregulation of genes with both somatic mutations or regulatory
disruptions which accumulate during a patient&#x2019;s lifetime and germline mutations
which predispose individuals to high-risk early onset cancers (<a 
href="#XACS2017">American Cancer
Society</a>,&#x00A0;<a 
href="#XACS2017">2017</a>;&#x00A0;<a 
href="#XGuilford1998">Guilford <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XGuilford1998">1998</a>;&#x00A0;<a 
href="#XNCI2015">NCI</a>,&#x00A0;<a 
href="#XNCI2015">2015</a>). Cancer is widely viewed to be a genetic
disease due to these familial cancer syndromes, hereditary risk factors, and the molecular
changes occurring in cancers, including numerous cancer genes which have been
identified <a 
href="#XStratton2009">Stratton <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XStratton2009">2009</a>);&#x00A0;<a 
href="#XVogelstein2013">Vogelstein <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XVogelstein2013">2013</a>). Cancer genes are generally
classified into two classes: &#x201C;oncogenes&#x201D; which are activated in cancers, driving
tumour growth and invasion, or &#x201C;tumour suppressors&#x201D; which are inactivated in
cancers, removing cellular regulation and genomic maintenance functions. The
mutations which cause cancers accumulate with age and have been suggested to be
inevitably coupled with aging due to the association of cancer incidence with the stem
                                                                                    
                                                                                    
cell divisions in which mutations could occur across tissue types (<a 
href="#XTomasetti2015">Tomasetti and
Vogelstein</a>,&#x00A0;<a 
href="#XTomasetti2015">2015</a>).
</p><!--l. 47--><p class="indent" ><a 
href="#XHanahan2000">Hanahan and Weinberg</a>&#x00A0;(<a 
href="#XHanahan2000">2000</a>) identified several key molecular and cellular traits shared
across most cancers as a rational approach to the complex changes that occur in cancer
initiation and progression due to common molecular machinery underlying all cells. A cancer
cell must possess limitless replication potential, modulate growth signals to grow indefinitely,
and gain invasive or metastatic capabilities. In addition, cancers must evade apoptosis, the
immune system, and sustain angiogenesis and energy metabolism in order to survive
(<a 
href="#XHanahan2000">Hanahan and Weinberg</a>,&#x00A0;<a 
href="#XHanahan2000">2000</a>,&#x00A0;<a 
href="#XHanahan2011">2011</a>). In order to achieve this, cancer cells undergo changes
to their genomes and the surrounding cells to create a tumour microenvironment. Thus
genomic instability has a key role in the survival and proliferation of cancer cells and the
progression of further disease, as these malignant characteristics are acquired. Identifying
the mechanisms of these acquired traits and the underlying genetic mutation or
dysregulation behind them, such as <a 
href="#glo:E-cadherin">E-cadherin</a> mutation in metastasis or p53 mutation in
genomic instability (<a 
href="#XHanahan2000">Hanahan and Weinberg</a>,&#x00A0;<a 
href="#XHanahan2000">2000</a>), will be an important step in
understanding and inhibiting cancer with the next generation of genomically-informed
treatments.
</p><!--l. 49--><p class="indent" >Molecular biological processes have particular importance in characterising breast
cancers. Gene expression and regulatory signals confer cell identity and response to the
environment. Therefore gene expression has been investigated with microarray
technologies <a 
href="#XPerou2000">Perou <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XPerou2000">2000</a>), with &#x201C;intrinsic subtypes&#x201D; identified characterised by
estrogen receptor, <span 
class="cmti-12">HER2</span>, and basal, epithelial signalling. The expression profiles were
similar across independent samples of the same tumour and between primary and
metastatic tumours of the same patient. Thus expression profiles represent the
molecular state of a tumour rather than the sample and the molecular configuration of
the cells regulation is carried through the cellular lineage of during metastasis
preserving the molecular subtype. These molecular intrinsic subtypes &#x201C;luminal
A&#x201D;, &#x201C;luminal B&#x201D;, &#x201C;HER2-enriched&#x201D;, &#x201C;basal-like&#x201D;, and &#x201C;normal-like&#x201D; have been
replicated across microarray studies (<a 
href="#XHu2006">Hu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XHu2006">2006</a>), with their relevance to prognosis
(including predicting survival and response to neoadjuvant chemotherapy) demonstrated
and a 50-gene subtype predictor from microarray and <a 
href="#glo:qPCR">quantitative PCR (qPCR)</a>
analysis has been provided (<a 
href="#XParker2009">Parker <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XParker2009">2009</a>;&#x00A0;<a 
href="#XSorlie2001">S&#x00F8;rlie <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XSorlie2001">2001</a>). This has been
further updated with the &#x201C;claudin-low&#x201D; subtype (<a 
href="#XHerschkowitz2007">Herschkowitz <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XHerschkowitz2007">2007</a>) and
stimulated further investigations into subtyping of breast cancers by molecular
properties.
                                                                                    
                                                                                    
</p><!--l. 51--><p class="indent" >Despite differences in subtyping performed by different research groups and companies,
there is widespread agreement that distinguishing luminal, HER2-enriched, and triple
negative tumours can be performed with expression profiles and have value in our
understanding of cancer progression and prognostic importance for patients <a 
href="#XDai2015">Dai
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XDai2015">2015</a>). High-throughput technologies have the potential to enable such subtyping on a
vast scale in discovery of further subtypes in breast cancer or other diseases and in
identification of these subtypes along with mutations in routine clinical diagnostic and
prognostic testing. The &#x201C;Pan cancer&#x201D; approaches by the cancer genome atlas project (as
discussed in more detail in Section&#x00A0;<a 
href="#x1-210001">1.1.5.1.1<!--tex4ht:ref: TCGA_Findings --></a>) expand on the importance of molecular
differences between cancers by examining molecular profiles across cancer tissue types
(<a 
href="#XTCGA2013PAN">Weinstein <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2013PAN">2013</a>).
</p><!--l. 53--><p class="indent" >The molecular variability of cancer has also been approached rationally at a pathway
level with patients subgroups activating different molecular pathways reflecting
differences in disease mechanisms (<a 
href="#XGatza2010">Gatza <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XGatza2010">2010</a>). A robust approach to measuring
pathway activation in cancer is with a&#x201C;metagene&#x201D; which gives a consistent signal as a
consensus of expression across genes even if they are inversely correlated (<a 
href="#XAnjomshoaa2008">Anjomshoaa
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XAnjomshoaa2008">2008</a>;&#x00A0;<a 
href="#XHuang2003">Huang <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XHuang2003">2003</a>;&#x00A0;<a 
href="#XNagalla2013">Nagalla <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XNagalla2013">2013</a>). These are derived from
the first principal component or eigenvector of a singular matrix decomposition,
capturing the the most consistent variation across genes in a pathway or gene
signature. <a 
href="#XGatza2010">Gatza <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XGatza2010">2010</a>) used gene signatures for 18 cellular pathways in
breast cancer to define subtypes with distinct molecular pathway activity. In a
meta-analysis of Affymetrix microarray expression data for 1,143 samples and
50 cell lines, unsupervised hierarchical clustering robustly defined subtypes with
common homogeneous pathway activity despite variation in the specific mutations
giving rise to them. These subtypes with shared pathway activity have similar
molecular characteristics (such as DNA copy number), clinical properties and prognosis,
building upon the intrinsic subypes (<a 
href="#XParker2009">Parker <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XParker2009">2009</a>) and providing a functional
interpretation for molecular stratification (<a 
href="#XGatza2010">Gatza <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XGatza2010">2010</a>). The pathway-based subtypes
often correspond to intrinsic subtypes (<a 
href="#XGatza2010">Gatza <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XGatza2010">2010</a>,&#x00A0;<a 
href="#XGatza2014">2014</a>) and provide finer
molecular stratification such as environmental stress response (to hypoxia within
HER2-enriched cancers) (<a 
href="#XGatza2011">Gatza <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XGatza2011">2011</a>) and have pathway-specfic DNA copy
number variation or essential genes (<a 
href="#XGatza2014">Gatza <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XGatza2014">2014</a>). <a 
href="#XGatza2014">Gatza <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XGatza2014">2014</a>) extend
these analyses include 52 pathway signatures from previous publications in breast
cancer, replicating known characteristics (such as hormone re) of each subtype and
identifying novel subtype-specific driver genes of proliferation by analysis of microarray
                                                                                    
                                                                                    
expression from 476 from <a 
href="#glo:TCGA">The Cancer Genome Atlas (TCGA)</a>. In addition to distinct
biological functions driving growth of breast cancer subtypes, these molecular subtypes
provide a rational approach based on molecular properties to cancer treatment
with combination and targeted therapies (<a 
href="#XGatza2010">Gatza <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XGatza2010">2010</a>,&#x00A0;<a 
href="#XGatza2014">2014</a>;&#x00A0;<a 
href="#XHanahan2000">Hanahan and
Weinberg</a>,&#x00A0;<a 
href="#XHanahan2000">2000</a>).
</p><!--l. 55--><p class="indent" >Cancer is a major health concern with a well-established genetic contribution, in risk and
in the molecular changes occurring during progression (<a 
href="#XStratton2009">Stratton <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XStratton2009">2009</a>). Many genes
have been discovered to be important in different cancers with molecular differences between
cancers, including alterations across the genome, being of clinical importance. As such
cancers were among the first samples investigated with <a 
href="#glo:genomics">genomics</a> following the sequencing of
the human genome <a 
href="#XDickson1999">Dickson</a>&#x00A0;(<a 
href="#XDickson1999">1999</a>) and continue to be the subject of <a 
href="#glo:genomics">genomics</a> and
<a 
href="#glo:bioinformatics">bioinformatics</a> investigations.
<a 
 id="x1-7001r6"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.1.2 </span> <a 
 id="x1-80002"></a>The Human Genome Revolution</h4>
<!--l. 58--><p class="noindent" >The advent of the Human Genome sequence (<a 
href="#XLander2001">Lander <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XLander2001">2001</a>) has transformed genetics
research including the study of health and disease (<a 
href="#XLander2011">Lander</a>,&#x00A0;<a 
href="#XLander2011">2011</a>;&#x00A0;<a 
href="#XPeltonen2001">Peltonen and
McKusick</a>,&#x00A0;<a 
href="#XPeltonen2001">2001</a>). Systematic, unbiased studies across all of the genes in the genome are
viable in unprecedented ways. The successful undertaking of such an international scientific
megaproject has set an example for numerous initiatives to follow, including many <a 
href="#glo:genomics">genomics</a>
investigations expanding to species, to the functional, or to the population level (<a 
href="#XCollins2003">Collins
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XCollins2003">2003</a>). These projects serve as an excellent resource for genetics research globally,
particularly for cancers where <a 
href="#glo:genomics">genomics</a> investigation have been widely applied to different
tissues across molecular profiles <a 
href="#XBamford2004">Bamford <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XBamford2004">2004</a>);&#x00A0;<a 
href="#XTCGA2013PAN">Weinstein <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XTCGA2013PAN">2013</a>);&#x00A0;<a 
href="#XICGC2011">Zhang
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XICGC2011">2011</a>) . Genome sequencing technologies continue to improve, decrease in
price, and become increasingly feasible in a wider range of research and clinical
applications.
<a 
 id="x1-8001r7"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.1.2.1 </span> <a 
 id="x1-90001"></a>The First Human Genome Sequence</h5>
<!--l. 61--><p class="noindent" >The first human genome is a good example of a large-scale <a 
href="#glo:genomics">genomics</a> project for it&#x2019;s success
as an international collaboration and releasing their data as a resource for the wider scientific
community (<a 
href="#XCollins2003">Collins <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XCollins2003">2003</a>;&#x00A0;<a 
href="#XLander2001">Lander <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XLander2001">2001</a>). This particular project generated
significant public interest due to it being a landmark achievement, the first of it&#x2019;s scale, and
some controversial findings. Namely, the number of genes discovered (particularly those
                                                                                    
                                                                                    
specific to vertebrates) was much lower than most estimates for a genome of it&#x2019;s size and the
number of repetitious transposon elements was very high. Even the figure of 30&#x2013;40,000 genes
given by the original publication is now regarded to be an overestimate (<a 
href="#XEzkurdia2014">Ezkurdia
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XEzkurdia2014">2014</a>;&#x00A0;<a 
href="#XIHGSC2004">IHGSC</a>,&#x00A0;<a 
href="#XIHGSC2004">2004</a>).
</p><!--l. 63--><p class="indent" >Accounting for the &#x201C;complexity&#x201D; encoded by the human genome with so few genes has
led to investigations into molecular function, expression profiling, and population variation.
When announcing the draft genome, <a 
href="#XLander2001">Lander <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XLander2001">2001</a>) conceded that genomic
information alone was not sufficient for biological understanding and that many
investigations remained to carried out, with their objective being to share the raw genome
data so that it was available for further inquiry rather than interpreting it themselves. While
<a 
href="#glo:genomics">genomics</a> technologies and <a 
href="#glo:genomics">genomics</a> projects have flourished since then, the need in turn for
systematic means of interpreting data of such scale and for the interdisciplinary expertise to
do so has only grown.
<a 
 id="x1-9001r9"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.1.2.2 </span> <a 
 id="x1-100002"></a>Impact of Genomics</h5>
<!--l. 77--><p class="noindent" ><a 
href="#glo:genomics">Genomics</a> has stimulated investigations into many of these previously largely explored areas
of functional genetics and thus been of immense value in genetics research, attracting high
expectations for further applications. <a 
href="#glo:genomics">Genomics</a> research created widespread anticipation for
potential applications in healthcare, agriculture, ecology, conservation, and evolutionary
biology despite few of these being realised yet.
</p><!--l. 79--><p class="indent" >Cancer research is an area of particularly high expectations for the clinical impact of
<a 
href="#glo:genomics">genomics</a> in oncology. <a 
href="#glo:genomics">Genomics</a> technologies have potential applications across cancer
diagnostics, prognosis, management, and developing treatment. Cancers often involve
genetic mutation or dysregulated gene expression which can be detected in a genome
or transcriptome with potential to improve patient care. While direct impact of
<a 
href="#glo:genomics">genomics</a> on the clinic has been limited, compared to initial expectations following the
publication of the human genome, diagnostic cancer genes and therapeutic targets
identified with <a 
href="#glo:genomics">genomics</a> research have begun to be introduced in the clinic (<a 
href="#XStratton2009">Stratton
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XStratton2009">2009</a>).
<a 
 id="x1-10001r8"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.1.3 </span> <a 
 id="x1-110003"></a>Technologies to Enable Genetics Research</h4>
<a 
 id="x1-11001r10"></a>
                                                                                    
                                                                                    
<h5 class="subsubsectionHead"><span class="titlemark">1.1.3.1 </span> <a 
 id="x1-120001"></a>DNA Sequencing and Genotyping Technologies</h5>
<!--l. 88--><p class="noindent" >Genotyping was once commonly performed on variable regions of the genome with
<a 
href="#glo:RFLP">restriction fragment length polymorphism (RFLP)</a> or repetitious microsatellite
regions. These exploited sequence variation at target sites of restriction enzymes
or measured the length of repetitious regions, using <a 
href="#glo:PCR">polymerase chain reaction
(PCR)</a>, restriction enzymes, and gel electrophoresis to measure <a 
href="#glo:DNA">deoxyribonucleic
acid (DNA)</a> genotypes at particular sites. This is laborious and limited to well
characterised variable regions of the genome, generally genes or nearby marker
regions.
</p><!--l. 90--><p class="indent" >The <a 
href="#glo:Sanger">Sanger</a> (dideoxy) chain termination method (<a 
href="#XSanger1975">Sanger and Coulson</a>,&#x00A0;<a 
href="#XSanger1975">1975</a>)
enabled DNA sequencing and genotyping at a widespread scale, being less technically
difficult than the Maxam-Gilbert sequencing by degradation method (<a 
href="#XGilbert1973">Gilbert and
Maxam</a>,&#x00A0;<a 
href="#XGilbert1973">1973</a>;&#x00A0;<a 
href="#XMaxam1977">Maxam and Gilbert</a>,&#x00A0;<a 
href="#XMaxam1977">1977</a>), which required more radioactive and toxic
reactants. The <a 
href="#glo:Sanger">Sanger</a> methodology has relatively long read length (particularly compared to
early versions of more recent technologies), with read lengths of 500&#x2013;700 base pairs
accurately sequenced in most applications, usually following targeted amplification with
<a 
href="#glo:PCR">PCR</a>. <a 
href="#glo:Sanger">Sanger</a> sequencing by gel electrophoresis takes around 6-8 hours and has
been further refined with the &#x201C;capillary&#x201D; approach to 1&#x2013;3 hours and requiring
less input DNA and reactants. The capillary approach has been scaled up to run
in parallel from a 96 well plate, at 166 kilobases per hour. The 96 well parallel
capillary method was one of the main innovations which made the first Human
Genome Project feasible and was used throughout (<a 
href="#XLander2001">Lander <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XLander2001">2001</a>). Due to
the quality of the <a 
href="#glo:Sanger">Sanger</a> sequence reads and low cost, it is still widely used in
smaller scale applications, clinical testing, and to validate the findings of newer
approaches.
<a 
 id="x1-12001r12"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.1.3.2 </span> <a 
 id="x1-130002"></a>Microarrays and Quantitative Technologies</h5>
<!--l. 94--><p class="noindent" >Real-time or <a 
href="#glo:qPCR">qPCR</a> is another adaptation of genetic technologies to quantitatively
study nucleic acids, often reverse transcribed &#x201C;<a 
href="#glo:cDNA">deoxyribonucleic acid (cDNA)</a>&#x201D;
or messenger &#x201C;<a 
href="#glo:mRNA">messenger ribonucleic acid (mRNA)</a>&#x201D; to measure (relative) gene
expression or transcript abundance. While numerous quality control measures are
required to correctly interpret a qPCR experiment, these have similarly become
widely adopted and are still used for smaller scale experiments and as a &#x201C;gold
standard&#x201D; for measuring gene expression (<a 
href="#XAdamski2014">Adamski <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XAdamski2014">2014</a>). This also represents a
                                                                                    
                                                                                    
shift in the application of <a 
href="#glo:qPCR">qPCR</a> and sequencing technology, where the primary
interest is quantifying the amount of input material (by the rate of amplification
to a certain level) rather than the qualitative nature of the sequence itself. The
more recent technologies of microarrays and <a 
href="#glo:RNA-Seq">RNA-Seq</a> have similarly embraced this
application to quantify DNA copy number, <a 
href="#glo:RNA">ribonucleic acid (RNA)</a> expression,
and DNA methylation levels. Due to results of comparable or arguably better
quality from these newer technologies (<a 
href="#XBeck2016">Beck <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBeck2016">2016</a>;&#x00A0;<a 
href="#XGit2010">Git <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XGit2010">2010</a>;&#x00A0;<a 
href="#XMcCourt2013">McCourt
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XMcCourt2013">2013</a>;&#x00A0;<a 
href="#XRobin2016">Robin <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XRobin2016">2016</a>), this &#x201C;gold standard&#x201D; status has begun to come under
scrutiny.
</p><!--l. 96--><p class="indent" >Microarrays represent a truly high-throughput molecular technique, reducing the cost,
time, and labour required to study molecular factors such as genotype, expression, or
methylation across many genes, making it feasible to do so over a statistically meaningful
number of samples. Microarrays are manufactured with probes which measure binding of
particular nucleotide sequences to either quantitatively detect the presence of a sequence
such as a <a 
href="#glo:SNP">single nucleotide polymorphism (SNP)</a> or quantify DNA copy number, gene
expression, or DNA CpG methylation. Microarray technologies have popularised &#x201C;genome
scale&#x201D; studies of genetic variation and expression.
</p><!--l. 98--><p class="indent" >In addition to being more versatile and higher-throughput than <a 
href="#glo:PCR">PCR</a> based techniques,
microarrays are considered cost-effective, particularly when scaled up to a large
number of probes. They are also available with established gene panels or customised
probes from a number of commercial manufacturers. These remained popular during
the introduction of newer technologies due to reliability and relatively lower cost,
especially in large-scale projects involving many samples. However, microarrays have
issues with signal-to-noise ratio, with both sensitivity to low nucleic acid abundance
and &#x201C;saturation&#x201D; of probes at high abundance, edge effects, and requiring more
starting material than <a 
href="#glo:qPCR">qPCR</a>. Thus <a 
href="#glo:qPCR">qPCR</a> is still used for many small gene panel
studies.
<a 
 id="x1-13001r13"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.1.3.3 </span> <a 
 id="x1-140003"></a>Massively Parallel &#x201C;Next Generation&#x201D; Sequencing</h5>
<!--l. 103--><p class="noindent" >Similar to microarrays, the introduction of massively parallel sequencing technologies have
further expanded the availability of high-throughput molecular studies to researchers, with
corresponding availability of <a 
href="#glo:genomics">genomics</a> data from these studies. This <a 
href="#glo:NGS">&#x201C;Next-Generation
Sequencing&#x201D; (NGS)</a> expands not only gene expression studies (compared to microarrays) but
extends to genome sequencing <span 
class="cmti-12">de novo </span>for previously unknown genome and transcriptome
                                                                                    
                                                                                    
sequences at an unprecedented scale. This has been a particularly important technological
revolution in <a 
href="#glo:genomics">genomics</a>, as the cost and time of genome sequencing has dropped
dramatically and enabled sequencing projects of far more samples and applications
beyond the Human Genome Project. Particularly, when dealing with variants in a
species with an existing reference sequence such as humans, where there is a low
computational cost of mapping to a reference compared to a genome assembly.
However, the cost of sequencing (<a 
href="#glo:RNA-Seq">RNA-Seq</a>) for gene expression or DNA methylation
studies is still considerably higher than a microarray study (limiting feasible sample
sizes).
</p><!--l. 105--><p class="indent" >Compared with arrays, <a 
href="#glo:NGS">NGS</a> studies have additional challenges, particularly regarding
large data and compute requirements to handle the raw output data. Compared with the the
established methods to analyse microarray data, handling <a 
href="#glo:NGS">NGS</a> data can be more technically
difficult. While methods developed for analysing microarray data can be repurposed for
sequence analysis in many cases, more <a 
href="#glo:bioinformatics">bioinformatics</a> expertise is required particularly to
handle the raw read data and changing approaches for various developments in sequencing
technologies. One of the main computational challenges is the assembly of reads or mapping
to a reference genome due to the inherently small reads of most <a 
href="#glo:NGS">NGS</a> technologies compared
to the Sanger methodology. Furthermore, there are fewer software releases and best practices
established specifically for <a 
href="#glo:RNA-Seq">RNA-Seq</a> data, thus many analyses are still conducted with
customised analysis approaches and command-line tools. Compared to existing
graphical tools or pipelines for microarray analysis, this is a more active technology for
<a 
href="#glo:bioinformatics">bioinformatics</a> research where many applications of <a 
href="#glo:genomics">genomics</a> data have yet to be
explored.
</p><!--l. 107--><p class="indent" >However, the methodology itself has challenges with the sample preparation, requiring a
relatively high quantity of input material and &#x201C;contamination&#x201D; with over abundant
ribosomal rRNA taking up the majority of the sequencing if not purified correctly. This
abundance of rRNA is a particularly important issue in microarray and <a 
href="#glo:RNA">RNA</a> experiments in
Eukaryotes where it is commonplace to target the <a 
href="#glo:mRNA">mRNA</a> by binding to the poly-A tail
(<a 
href="#glo:RNA-Seq">RNA-Seq</a>) or 5&#x2019; cap (<a 
href="#glo:CAGE-Seq">CAGE-Seq</a>). However, this has the potential to exclude <a 
href="#glo:miRNA">miRNA
(miRNA)</a> and <a 
href="#glo:lncRNA">long non-coding ribonucleic acid (lncRNA)</a> of interest unless the sample is
prepared specifically to study these. Similarly capturing a subsection of the genome for
<a 
href="#glo:exome">exome</a> analysis or <a 
href="#glo:RRBS">reduced representation bisulfite sequencing (RRBS)</a>, focuses on sequencing
DNA sequences and methylation levels of CpG sites near known genes to reduce cost, noise,
and incidental findings.
                                                                                    
                                                                                    
</p><!--l. 110--><p class="indent" >In many cases, the benefits of <a 
href="#glo:NGS">NGS</a> technologies over microarrays still outweigh the
additional cost. <a 
href="#glo:NGS">NGS</a> technologies have the advantage of greater potential accuracy and
sensitivity than microarrays, depending on the sequencing depth or &#x201C;coverage&#x201D;, theoretically
sensitive down to the exact number of molecules for each transcript. <a 
href="#glo:NGS">NGS</a> experiments are
regarded as &#x201C;reproducible&#x201D; with no need for technical replicates, although these are still
performed for a subset of samples in many projects for quality assurance purposes. <a 
href="#glo:NGS">NGS</a> has
a wider dynamic range than microarrays and is able to detect <a 
href="#glo:SNP">SNPs</a>, <a 
href="#glo:InDel">insertions or deletions</a>,
and splice variants in addition to quantifying DNA copy number or transcript abundance.
<a 
href="#glo:NGS">NGS</a> scales to all genes and beyond for these molecular applications without having to design
new probes as required for a microarray. Thus <a 
href="#glo:NGS">NGS</a> technologies are not limited to genes
with already characterised sequence or functions, do not need to be updated with
new probes for each genome annotation release, and do not require a reference
genome at all for new species. A &#x201C;transcriptome&#x201D; can be assembled <span 
class="cmti-12">de novo </span>for
an expression study in any organism by sequencing the mRNA extracted from a
cell.
<a 
 id="x1-14001r1"></a>
</p>
<a 
 id="x1-150001"></a>Molecular Profiling with Genomics Technology
<!--l. 117--><p class="indent" ><a 
href="#glo:NGS">NGS</a> is highly adaptable to different applications: DNA sequencing (whole <a 
href="#glo:genomics">genome</a> or
<a 
href="#glo:exome">exome</a>), DNA methylation (<a 
href="#glo:bisulfite-Seq">bisulfite-Seq</a>), <a 
href="#glo:RNA-Seq">RNA-Seq</a>, <a 
href="#glo:miRNA">miRNA</a>, <a 
href="#glo:lncRNA">lncRNA</a>, or <a 
href="#glo:ChIP-Seq">chromatin
immunopreciptation sequencing (ChIP-Seq)</a>. <a 
href="#glo:RNA-Seq">RNA-Seq</a> of the <a 
href="#glo:transcriptome">transcriptome</a> is a common
adaptation where <a 
href="#glo:RNA">RNA</a> is reverse transcribed and sequenced from the resulting <a 
href="#glo:cDNA">cDNA</a>. This
is utilised to quantify the levels of <a 
href="#glo:RNA">RNA</a> and identify which regions of DNA are expressed.
Similar bisulfite treatment converts cytosine residues to uracil (sequenced as thymidine),
sparing methylated cytosine enabling it to be distinguished with bisulfite-Seq for
high-throughput detection of the notable epigenetic mark and is a common procedure to
generate an <a 
href="#glo:epigenome">epigenome</a>. Subsets of the nucleic acid may be extracted for sequencing such as
the coding regions of DNA (for the &#x201C;<a 
href="#glo:exome">exome</a>&#x201D;), the mRNA 5<span 
class="cmsy-10x-x-120">&#x2032; </span>cap (<a 
href="#glo:CAGE-Seq">CAGE-Seq</a>),
mRNA 3<span 
class="cmsy-10x-x-120">&#x2032; </span>poly-A tail (<a 
href="#glo:RNA-Seq">RNA-Seq</a>), <a 
href="#glo:microRNA">microRNA</a>, or an enriched subset of variable
regions for DNA sequencing (&#x201C;genotyping by sequencing&#x201D;) and methylation studies
(&#x201C;reduced-representation bisulfite sequencing). High-throughput gel and mass spectrometry
techniques have been applied to proteins and metabolites to generate the <a 
href="#glo:proteome">proteome</a> and
<a 
href="#glo:metabolome">metabolome</a> respectively. These &#x201C;<a 
href="#glo:omics">omics</a>&#x201D; technologies are applicable across a wide range of
biomolecules in a cell and these &#x201C;molecular profiles&#x201D; are produced in many experimental
laboratories.
                                                                                    
                                                                                    
<a 
 id="x1-15001r15"></a>
</p>
<a 
 id="x1-160002"></a>Sequencing Technologies
<!--l. 128--><p class="indent" >Illumina sequencing (developed by Solexa and later acquired by Illumina) was
released in 2006. It utilises reversible terminating dyes to sequence by synthesis
with a lower accuracy (98%) and read lengths of 150&#x2013;250bp. Illumina more than
makes up for relatively short reads (along with improving the read length of the
technology) and low accuracy with high-throughput and cost effectiveness, with a
Hi-Seq 4000 platform producing up to 10 billion paired-end reads (1500Gbp) in
a run of appropriately 3 days, capable of sequencing 12 human genomes (30<span 
class="cmsy-10x-x-120">&#x00D7;</span>
coverage) or 100 human transcriptomes simultaneously (<a 
href="#XIllumina">Illumina</a>,&#x00A0;<a 
href="#XIllumina">2017</a>). Illumina
has further reduced the cost of sequencing with the economies of scale with the
Hi-Seq X 10 claiming to produce a human genome (with 30<span 
class="cmsy-10x-x-120">&#x00D7; </span>coverage) for less than
US<span 
class="tcrm-1200">$</span>1000, the first platform to achieve this long-standing goal in <a 
href="#glo:genomics">genomics</a>. The
high-throughput of Illumina sequencing also makes deep sequencing for high coverage, high
quality consensus reads, and sensitive <a 
href="#glo:RNA-Seq">RNA-Seq</a> experiments feasible. Illumina
sequencing now has a dominating market share of the <a 
href="#glo:NGS">NGS</a> technologies. Their Hi-Seq
platforms were used in large-scale gemomics projects (such as the cancer genome atlas
discussed in Section&#x00A0;<a 
href="#x1-200001">1.1.5.1<!--tex4ht:ref: TCGA_Intro --></a>) to generate the sequence data used throughout this
thesis.
</p><!--l. 143--><p class="indent" ><a 
href="#glo:NGS">NGS</a> technologies continue to be refined with Illumina (and competitors) continuing to
release refined productions, offer additional genomics-based services, and decreases overhead
and operating costs to enable a wider range of research projects. As such <a 
href="#glo:RNA-Seq">RNA-Seq</a> for
examining the transcriptome of an organisms and for expression studies in diseases is a
growing field of research and expression data will continue to be generated for a range of
samples in many healthy and diseased tissues. The technology be also be further improved
developments in speed and accuracy (such as Ion Torrent platforms) and towards long reads
of single molecules (such as Pacific Biosciences, Oxford Nanopore, and Quantum Biosystems
Japan).
</p><!--l. 149--><p class="indent" >Due to such benefits of sequencing over previous technologies (and their continued
refinement), this thesis has focused on gene expression data generated by <a 
href="#glo:RNA-Seq">RNA-Seq</a> rather by
microarrays. <a 
href="#glo:RNA-Seq">RNA-Seq</a> data is widely available as a resource from large-scale cancer
<a 
href="#glo:genomics">genomics</a> projects and methods to make inferences from <a 
href="#glo:RNA-Seq">RNA-Seq</a> experiments could
feasibly be applied to many other studies based on these current (or similar future)
technologies.
                                                                                    
                                                                                    
<a 
 id="x1-16001r14"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.1.3.4 </span> <a 
 id="x1-170004"></a>Bioinformatics as Interdisciplinary Genomic Analysis</h5>
<!--l. 154--><p class="noindent" ><a 
href="#glo:genomics">Genomics</a> technologies have given rise to data at a scale previously rarely encountered in
molecular biology, making inference with conventional techniques difficult. Computational,
Mathematical, and Statistical skills are required to handle this data effectively, in addition to
biological background to frame and interpret research questions. Drawing upon
these disciplines to handle biological data has become the field of &#x201C;<a 
href="#glo:bioinformatics">Bioinformatics</a>&#x201D;,
focusing specifically on making inferences from <a 
href="#glo:genomics">genomics</a> and high-throughput
molecular data or developing the tools to do so. This contrasts with the existing
fields of &#x201C;theoretical&#x201D; or &#x201C;computational biology&#x201D; which existed prior to <a 
href="#glo:genomics">genomics</a>
data, focusing on modelling and simulating aspects of biology without necessarily
addressing the <a 
href="#glo:genomics">genomics</a> data or detecting the phenomena in nature, extending
beyond genetics to cell modelling, neuroscience, cancer development, ecology, and
evolution.
</p><!--l. 156--><p class="indent" >In practice, many researchers identify with both <a 
href="#glo:bioinformatics">bioinformatics</a> and computational
biology, or draw upon the findings and methods of the other field. This thesis uses many
approaches in <a 
href="#glo:bioinformatics">bioinformatics</a> to biological research questions and established mathematical
or <a 
href="#glo:bioinformatics">bioinformatics</a> resources.
</p><!--l. 158--><p class="indent" >Gene expression analysis is the focus of many <a 
href="#glo:bioinformatics">bioinformatics</a> research groups, drawing
upon statistical approaches to appropriately handle microarray and <a 
href="#glo:RNA-Seq">RNA-Seq</a> data along
with making biological inferences from a large number of statistical tests. This presents
various challenges from normalising sample data and accounting for batch effects to
developing or applying statistical tests tailored to biological hypotheses and testing them at
a genome-wide scale, generally across thousands of genes. There are numerous
approaches for dealing with these challenges, some of which will be described in
Chapter&#x00A0;<a 
href="#x1-650002">2<!--tex4ht:ref: chap:methods --></a>.
<a 
 id="x1-17001r11"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.1.4 </span> <a 
 id="x1-180004"></a>Follow-up Large-Scale Genomics Projects</h4>
<!--l. 164--><p class="noindent" >A number of projects have attempted to follow up on the human genome project to varying
degrees of success. The genomes have since been sequenced for a variety of model organisms,
organisms of importance in health, agriculture, <a 
href="#glo:metagenomics">metagenomics</a> of microorganisms
(microbiome), ecology and conservation. <a 
href="#glo:genomics">Genomics</a> projects have also been applied functional
genetics (<a 
href="#XFANTOM2001">Kawai <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XFANTOM2001">2001</a>;&#x00A0;<a 
href="#XENCODE2004">ENCODE</a>,&#x00A0;<a 
href="#XENCODE2004">2004</a>) and to human populations with an interest
                                                                                    
                                                                                    
variability between individuals and health or disease risk (<a 
href="#XHapMap2003">HapMap</a>,&#x00A0;<a 
href="#XHapMap2003">2003</a>;&#x00A0;<a 
href="#X1000Genomes2010">1000
Genomes</a>,&#x00A0;<a 
href="#X1000Genomes2010">2010</a>).
</p><!--l. 168--><p class="indent" ><a 
href="#glo:genomics">Genomics</a> databases have also focused on faciltating distribution of genomic data
generated by researchers, rather than generating it themselves. GenBank hosted by
the <a 
href="#glo:NCBI">National Center for Biotechnology Information (NCBI)</a> in the US, the <a 
href="#glo:ENA">The
European Nucleotide Archive (ENA)</a> hosted by the <a 
href="#glo:EMBL">European Molecular Biology
Laboratory (EMBL)</a>, and the <a 
href="#glo:DDBJ">DNA Data Bank of Japan (DDBJ)</a> hosted by the
<a 
href="#glo:NIG">National Institute of Genetics (NIG)</a> do so by serving as public repositories of DNA
sequence data. <a 
href="#glo:GEO">Gene Expression Omnibus (GEO)</a> (<a 
href="#XGEO2016">Clough and Barrett</a>,&#x00A0;<a 
href="#XGEO2016">2016</a>),
arrayExpress (<a 
href="#XArrayExpress2013">Rustici <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XArrayExpress2013">2013</a>), and caArray (<a 
href="#XcaArray2014">Heiskanen <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XcaArray2014">2014</a>) serve a
similar purpose as a resource for gene expression datasets, originally developed for
microarray data but <a 
href="#glo:RNA-Seq">RNA-Seq</a> data is now supported by some platforms. They are
repositories for researchers to deposit, share, and access gene expression data, serving as
a resource to support ongoing research where larger datasets than would were
previously accessible for many individual laboratories are available (<a 
href="#XRung2013">Rung and
Brazma</a>,&#x00A0;<a 
href="#XRung2013">2013</a>). These resources cover not only DNA sequence across the genome but also
molecular profiles of other factors by adapting genomic sequencing or other high
throughput technologies for quantifying gene expression or DNA methylation.
Sharing the expression datasets generated in a publication is now required by some
journals.
</p><!--l. 170--><p class="indent" >Similarly, international projects and consortiums have begun to release data gathered
using common agreed upon protocols in laboratories across the world, often hosting public
databases of these themselves, publishing their own investigations into the datasets as they
are released, or offering basic searches and analytics of the data via a web portal. These
databases include many of the <a 
href="#glo:genomics">genomics</a> projects discussed above and the cancer-specific
projects discussed below. In many ways, the quality, consistency, and accessibility of these
international projects has become more appealing than accessing smaller studies, particularly
for gene expression datasets where the more recent, larger projects have switched
from microarray to <a 
href="#glo:RNA-Seq">RNA-Seq</a> technologies. This distinction will also be discussed
later.
<a 
 id="x1-18001r18"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.1.5 </span> <a 
 id="x1-190005"></a>Cancer Genomes</h4>
<!--l. 173--><p class="noindent" >The importance of genomics technologies in the future of cancer research was noticed, even
in the early days of <a 
href="#glo:genomics">genomics</a> (<a 
href="#XDickson1999">Dickson</a>,&#x00A0;<a 
href="#XDickson1999">1999</a>). The Cancer Genome Project (CGP) based at
                                                                                    
                                                                                    
Wellcome Trust Sanger Institute in the UK were among the first to launch investigations into
cancer after the publication of the Human Genome, using this genome sequence, consensus
across the cancer research literature, and sequencing the genes of cancers themselves.
Initially, the Sanger Institute set out to sequence 20 genes across 378 samples while the
Human Genome project was still ongoing (<a 
href="#XCollins2007">Collins and Barker</a>,&#x00A0;<a 
href="#XCollins2007">2007</a>), optimising
sequencing and computation infrastructure for a larger project while doing so. The
main aim of the Cancer Genome Project was to discover &#x201C;cancer genes&#x201D;, those
frequently mutated in cancers by comparing the genes of cancer and normal tissue
samples, both &#x201C;oncogenes&#x201D; and &#x201C;tumour suppressors&#x201D; which are activated and
inactivated respectively in cancers. This project is ongoing and the UK continues to be
involved in international sequencing initiatives and those focused on particular tissue
types.
</p><!--l. 175--><p class="indent" >The Sanger Institute also hosts the Catalogue of Somatic Mutations in Cancer
(<a 
href="#XCOSMICdb">COSMIC</a>,&#x00A0;<a 
href="#XCOSMICdb">2016</a>), a database and website of cancer genes. This launched with 66,634
samples and 10,647 mutations from initial investigations into <span 
class="cmti-12">BRAF</span>, HRAS, <span 
class="cmti-12">KRAS</span>,
and NRAS (<a 
href="#XBamford2004">Bamford <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBamford2004">2004</a>). It has since expanded to include 1,257,487
samples with 4,175,8787 gene mutations curated from 23,870 publications, including
29,112 whole genomes (<a 
href="#XCOSMICdb">COSMIC</a>,&#x00A0;<a 
href="#XCOSMICdb">2016</a>). This database now also identifies cancer
genes from DNA copy number, differential gene expression and differential DNA
methylation.
<a 
 id="x1-19001r17"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.1.5.1 </span> <a 
 id="x1-200001"></a>The Cancer Genome Atlas Project</h5>
<!--l. 178--><p class="noindent" >Based in the US, <a 
href="#glo:TCGA">The Cancer Genome Atlas</a> (<a 
href="#glo:TCGA">TCGA</a>) project was established in 2005, a
combined effort of the <a 
href="#glo:NCI">National Cancer Institute (NCI)</a> and the <a 
href="#glo:NHGRI">National Human
Genome Research Institute (NHGRI)</a> of the <a 
href="#glo:NIH">National Institutes of Health (NIH)</a>
(<a 
href="#XTCGA2017web">TCGA</a>,&#x00A0;<a 
href="#XTCGA2017web">2017a</a>). They first set out to demonstrate the pilot project on brain (<a 
href="#XTCGA2008GBM">McLendon
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2008GBM">2008</a>), ovarian (<a 
href="#XTCGA2011OV">Bell <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2011OV">2011</a>), and squamous cell lung (<a 
href="#XTCGA2012LUSC">Hammerman
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2012LUSC">2012</a>) cancers. In 2009, the project expanded aiming to analyse 500 samples
each for 20-25 tumour tissue types. They have since exceeded that goal, with data
available for 33 cancer types including 10 &#x201C;rare&#x201D; cancers, a total of over 10,000
samples.
</p><!--l. 180--><p class="indent" >The <a 
href="#glo:TCGA">TCGA</a> projects set out to generate a molecular &#x201C;profile&#x201D; of the tumour (and some
matched normal tissue) samples: the genotype, somatic mutations, gene expression, DNA
copy number, and <a 
href="#glo:RNA">RNA</a> methylation levels. While these were originally performed
                                                                                    
                                                                                    
largely with microarray technologies, exome and <a 
href="#glo:RNA-Seq">RNA-Seq</a> has been since adopted
and performed for many <a 
href="#glo:TCGA">TCGA</a> samples, with whole genomes being performed
for some samples. Data which cannot be used to identify the patients (such as
somatic mutation, expression, methylation, and various clincal factors) are publicly
available.
<a 
 id="x1-20001r16"></a>
</p>
<a 
 id="x1-210001"></a>Findings from Cancer Genomes
<!--l. 189--><p class="indent" >The Cancer Genome Atlas pilot projects (<a 
href="#XTCGA2011OV">Bell <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2011OV">2011</a>;&#x00A0;<a 
href="#XTCGA2012LUSC">Hammerman
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2012LUSC">2012</a>;&#x00A0;<a 
href="#XTCGA2008GBM">McLendon <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2008GBM">2008</a>) serve to demonstrate the power of applying <a 
href="#glo:genomics">genomics</a>
technologies to cancer research at such as scale. In addition to sequencing the whole
genome or a subset (exome), DNA copy number, gene expression, DNA methylation,
and somatic mutations were also analysed. The initial projects used microarray
technologies for expression and methylation data but these have since been replaced by
<a 
href="#glo:RNA-Seq">RNA-Seq</a> for expression. <a 
href="#glo:TCGA">TCGA</a> demonstrated the potential discovery of the molecular
basis of cancer by analysing 206 glioblastoma brain cancer samples (<a 
href="#XTCGA2008GBM">McLendon
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2008GBM">2008</a>), highlighting the roles of <span 
class="cmti-12">ERBB2</span>, <span 
class="cmti-12">NF1</span>, <span 
class="cmti-12">TP53</span>, and <span 
class="cmti-12">PIK3R1 </span>mutations, along
with altered methylation of <span 
class="cmti-12">MGMT</span>, and the core pathways of RTK, p53, and
RB signalling in brain cancer. An analysis of 489 serious ovarian cancers (<a 
href="#XTCGA2011OV">Bell
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2011OV">2011</a>) similarly reported <span 
class="cmti-12">TP53 </span>mutations specifically over-represented in high grade
tumours and reported 133 copy number variants, 168 differentially methylated
regions, and recurrently somatic mutations in 9 genes in low grade tumours including
<span 
class="cmti-12">NF1</span>, <span 
class="cmti-12">BRCA1</span>, <span 
class="cmti-12">BRCA2</span>, <span 
class="cmti-12">RB1</span>, and <span 
class="cmti-12">CDK12</span>. Four transcriptional subtypes of ovarian
cancers were identified, alterations in <span 
class="cmti-12">BRCA1</span>, <span 
class="cmti-12">BRCA2</span>, and <span 
class="cmti-12">CCLE </span>had an impact
on patient survival, and the homologous recombination, NOTCH and FOXM1
signalling pathways were involved in ovarian cancer growth. The <a 
href="#glo:genomics">genomics</a> of 178
squamous cell lung cancers (<a 
href="#XTCGA2012LUSC">Hammerman <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2012LUSC">2012</a>) were highly complex, averaging
at 360 muations in coding regions. While no targeted therapies existed for this
cancer subtype, 11 recurrently mutated genes were identified including <span 
class="cmti-12">TP53 </span>and
<span 
class="cmti-12">HLA-A</span>. The pathways altered in various squamous cell lung cancers were NFE2L2,
KEAP1, differentiation genes, PI3K, CDKN2A and RB1. These aberrant genes and
pathways represent potential therapeutic targets which could be identified for most
samples.
</p><!--l. 191--><p class="indent" >The <a 
href="#glo:TCGA">TCGA</a> breast cancer analysis (<a 
href="#XTCGA2012">TCGA</a>,&#x00A0;<a 
href="#XTCGA2012">2012</a>) consisted of 802 samples with exomes,
copy number variants, RPPA protein quantification, and DNA methylation, mRNA, and
                                                                                    
                                                                                    
microRNA arrays with 97 whole genomes sequenced. Four main molecular classes were
identified to subtype the samples, despite considerable heterogeneity between samples.
Recurrent mutations across more than 10% of samples were identified in <span 
class="cmti-12">TP53</span>,<span 
class="cmti-12">PIK3CA</span>, and
<span 
class="cmti-12">GATA</span>. <a 
href="#glo:TCGA">TCGA</a> further suggests subtypes by HER2 and EGFR protein levels. In a further
analysis of 817 breast cancer samples including 127 invasive lobular breast and 88 mixed
type samples (<a 
href="#XTCGA2015LBC">Ciriello <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2015LBC">2015</a>), 3 molecular subtypes of lobular breast cancer were
identified. Lobular breast cancer was also characterised by recurrent mutations in <span 
class="cmti-12">CDH1</span>,
<span 
class="cmti-12">PTEN</span>, <span 
class="cmti-12">TBX2</span>, and <span 
class="cmti-12">FOXA1</span>.
</p><!--l. 193--><p class="indent" ><a 
href="#glo:TCGA">TCGA</a> reported results of colon and rectal cancers in a combined analysis of 267 samples
(<a 
href="#XTCGA2012CRC">Muzny <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2012CRC">2012</a>), finding no genomic distinction between colorectal cancers.
Apart from 16% of hypermutated colorectal cancers, the remaining samples were
very similar at the molecular level with 24 significantly recurrently mutated genes
identified. These include the expected <span 
class="cmti-12">APC</span>, <span 
class="cmti-12">TP53</span>, <span 
class="cmti-12">SMAD4</span>, <span 
class="cmti-12">PIK3CA</span>, and <span 
class="cmti-12">KRAS</span>
genes. Additionally, novel recurrent mutations were identified in <span 
class="cmti-12">ARID1A</span>, <span 
class="cmti-12">SOX9</span>,
and <span 
class="cmti-12">FAM123 </span>along with recurrent copy number alterations in <span 
class="cmti-12">ERBB2 </span>and <span 
class="cmti-12">IFG2</span>.
Thus the molecular findings of colon and rectal tumours can be applicable across
colorectal cancers, including the known characteristics of microsatellite instability
(MSI) and CpG island methylator phenotype (CIMP) found in some colorectal
tumours.
</p><!--l. 195--><p class="indent" >The <a 
href="#glo:TCGA">TCGA</a> stomach cancer analysis of 295 samples (<a 
href="#XTCGA2014GC">Bass <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2014GC">2014</a>) identified 4
molecular subtypes of stomach cancers characterised by: the Epstein-Barr virus, MSI,
<a 
href="#glo:genomics">genomics</a> instability, and chromosomal instability. Abberrations in <span 
class="cmti-12">PD-L1</span>, <span 
class="cmti-12">PIK3CA</span>,
and <span 
class="cmti-12">JAK2 </span>were also identified in stomach cancers which may present therapeutic
targets.
<a 
 id="x1-21001r21"></a>
</p>
<a 
 id="x1-220002"></a>Genomic Comparisons Across Cancer Tissues
<!--l. 199--><p class="indent" ><a 
href="#glo:TCGA">TCGA</a> have identified various genes as recurrent, driver mutations across cancer types
which are likely to have a role in driving the proliferation of these cancers and present a
molecular target that could be applied across tissue types. These include <span 
class="cmti-12">TP53 </span>(in brain,
lung/head/neck squamous cell, breast, colorectal, uterine, and endometrial cancers),
<span 
class="cmti-12">ERBB2 </span>/HER2/NEU (in brain, breast, colorectal, bladder, and lung cancers), <span 
class="cmti-12">PIK3CA,</span>
<span 
class="cmti-12">PIK3R1 </span>(in brain, breast, colorectal, endometrial, bladder, clear cell renal, and
lung cancers), <span 
class="cmti-12">BRCA1</span>/<span 
class="cmti-12">BRCA2 </span>(in breast and ovarian cancers), <span 
class="cmti-12">NF1 </span>(in brain,
ovarian, and skin cancers), <span 
class="cmti-12">ARID1A </span>(in colorectal, endometrial, and clear cell
                                                                                    
                                                                                    
renal cancers), <span 
class="cmti-12">KRAS </span>(in colorectal, endometrial, and skin cancers), <span 
class="cmti-12">BRAF </span>(in
colorectal, thyroid, and skin cancers), <span 
class="cmti-12">EGFR </span>(in brain, breast, and lung cancers), and
<span 
class="cmti-12">PTEN </span>(in breast, endometrial, and uterine cancers) (<a 
href="#XTCGA2014TH">Agrawal <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2014TH">2014</a>;&#x00A0;<a 
href="#XTCGA2015SK">Akbani
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2015SK">2015</a>;&#x00A0;<a 
href="#XTCGA2014GC">Bass <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2014GC">2014</a>;&#x00A0;<a 
href="#XTCGA2011OV">Bell <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2011OV">2011</a>;&#x00A0;<a 
href="#XTCGA2017CERV">Burk <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2017CERV">2017</a>;&#x00A0;<a 
href="#XTCGA2017UT">Cherniack
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2017UT">2017</a>;&#x00A0;<a 
href="#XTCGA2015LBC">Ciriello <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2015LBC">2015</a>;&#x00A0;<a 
href="#XTCGA2014LU">Collisson <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2014LU">2014</a>;&#x00A0;<a 
href="#XTCGA2013RCC">Creighton <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2013RCC">2013</a>;&#x00A0;<a 
href="#XTCGA2012LUSC">Hammerman
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2012LUSC">2012</a>;&#x00A0;<a 
href="#XTCGA2013ENDO">Kandoth <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2013ENDO">2013</a>;&#x00A0;<a 
href="#XTCGA2015HNSC">Lawrence <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2015HNSC">2015</a>;&#x00A0;<a 
href="#XTCGA2008GBM">McLendon <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2008GBM">2008</a>;&#x00A0;<a 
href="#XTCGA2012CRC">Muzny
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2012CRC">2012</a>;&#x00A0;<a 
href="#XTCGA2012">TCGA</a>,&#x00A0;<a 
href="#XTCGA2012">2012</a>;&#x00A0;<a 
href="#XTCGA2014BL">Weinstein <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2014BL">2014</a>). In addition to disregarding the
tissue-based distinction between colon and rectal cancers based on molecular similarlity
(<a 
href="#XTCGA2012CRC">Muzny <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2012CRC">2012</a>), the <a 
href="#glo:TCGA">TCGA</a> project have observed differences within tumour
types and proposed molecular subtyping for breast, clear cell renal, papillary renal,
stomach, skin, bladder, and prostate cancers (<a 
href="#XTCGA2015PR">Abeshouse <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2015PR">2015</a>;&#x00A0;<a 
href="#XTCGA2015SK">Akbani
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2015SK">2015</a>;&#x00A0;<a 
href="#XTCGA2014GC">Bass <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2014GC">2014</a>;&#x00A0;<a 
href="#XTCGA2015LBC">Ciriello <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2015LBC">2015</a>;&#x00A0;<a 
href="#XTCGA2013RCC">Creighton <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2013RCC">2013</a>;&#x00A0;<a 
href="#XTCGA2012LUSC">Hammerman
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2012LUSC">2012</a>;&#x00A0;<a 
href="#XTCGA2016PRC">Linehan <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2016PRC">2016</a>;&#x00A0;<a 
href="#XTCGA2012CRC">Muzny <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2012CRC">2012</a>;&#x00A0;<a 
href="#XTCGA2012">TCGA</a>,&#x00A0;<a 
href="#XTCGA2012">2012</a>;&#x00A0;<a 
href="#XTCGA2014BL">Weinstein
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2014BL">2014</a>).
</p><!--l. 201--><p class="indent" >The &#x201C;<a 
href="#glo:Pan cancer">Pan cancer</a>&#x201D; project (<a 
href="#XTCGA2014PAN">Hoadley <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2014PAN">2014</a>;&#x00A0;<a 
href="#XTCGA2013PAN">Weinstein <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2013PAN">2013</a>) analysed 3527
samples across 12 tissue types for DNA, RNA, protein, and epigenetic molecular profiles.
This project was initated in 2012 to perform a comprehensive analysis of molecular data
across cancer types to identify molecular simliarities and differences. Recurrent <span 
class="cmti-12">TP53</span>
mutations characterised high grade tumours across breast, ovarian, and endometrial cancers.
HER2 was identified in brain, endometrial, bladder, and lung cancers, in addition
to the known role of HER2 in breast cancers. <span 
class="cmti-12">BRCA1 </span>and <span 
class="cmti-12">BRCA2 </span>mutations
were also detected across cancers, mainly breast and ovarian cancers as expected.
Microsatellite instability characterised both endometrial and colorectal cancers. The
Pan cancer project (<a 
href="#XTCGA2014PAN">Hoadley <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2014PAN">2014</a>) has identified 11 molecular subtypes
across these tissues, 5 of corresponding to tissue cancer types and the remainder
reassigned due to molecular similarities shared across cancer types. Squamous
cell lung, head, and neck and a subset bladder cancers were grouped together by
molecular similarities, characteried by a high frequency of <span 
class="cmti-12">TP53 </span>mutations. Conversely,
bladder cancers were divided into 3 of these molecular subtypes with distinct profiles.
This project further supports the genomic stratification of patients, demonstrated
in breast cancer (<a 
href="#XParker2009">Parker <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XParker2009">2009</a>;&#x00A0;<a 
href="#XMETABRIC2016">Pereira <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XMETABRIC2016">2016</a>;&#x00A0;<a 
href="#XPerou2000">Perou <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XPerou2000">2000</a>),
which may apply to other cancer types and to molecular characteristics across them
targeting recurrent mechanisms of cancer growth and progression (<a 
href="#XHanahan2000">Hanahan and
Weinberg</a>,&#x00A0;<a 
href="#XHanahan2000">2000</a>,&#x00A0;<a 
href="#XHanahan2011">2011</a>).
<a 
 id="x1-22001r22"></a>
                                                                                    
                                                                                    
</p>
<a 
 id="x1-230003"></a>Cancer Genomic Data Resources
<!--l. 204--><p class="indent" >While the findings from the <a 
href="#glo:TCGA">TCGA</a> projects themselves are a considerable contribution to
understanding cancer biology within and across tissue types, the main eventual
benefit of such projects will be the avaialability of the data for the research
community to analyse further and use to inform future investigations (<a 
href="#XTCGA2008GBM">McLendon
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2008GBM">2008</a>;&#x00A0;<a 
href="#XTCGA2017web">TCGA</a>,&#x00A0;<a 
href="#XTCGA2017web">2017a</a>;&#x00A0;<a 
href="#XTCGA2013PAN">Weinstein <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2013PAN">2013</a>). These serve as a vast resource
of common and rare cancer types and are publicly available for further analysis
(<a 
href="#XcBioPortal">cBioPortal</a>,&#x00A0;<a 
href="#XcBioPortal">2017</a>;&#x00A0;<a 
href="#XTCGA2017web">TCGA</a>,&#x00A0;<a 
href="#XTCGA2017web">2017a</a>;&#x00A0;<a 
href="#XICGC2011">Zhang <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XICGC2011">2011</a>). This also applies to the Molecular
Taxonomy of Breast Cancer International Consortium (METABRIC) project which focuses
on breast cancer which also aimed to identify novel molecular subtypes (<a 
href="#XMETABRIC2012">Curtis
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XMETABRIC2012">2012</a>). They performed an analysis of 2433 breast cancer samples with long-term
clincal data, gene expression, copy number variants, and 173 genes sequenced which
identified 40 driver mutations in breast cancer in addition to further support for
molecular subtyping to identify patient groups with different clinical outcomes (<a 
href="#XMETABRIC2016">Pereira
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XMETABRIC2016">2016</a>).
<a 
 id="x1-23001r19"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.1.6 </span> <a 
 id="x1-240006"></a>Genomic Cancer Medicine</h4>
<!--l. 216--><p class="noindent" >There is much anticipation in cancer research for <a 
href="#glo:genomics">genomics</a> technologies to have a clinical
impact in cancer medicine: from diagnosis and prognosis to treatment developments and
strategies. These may result either from direct use of genome or <a 
href="#glo:RNA-Seq">RNA-Seq</a> in clinical
laboratories or indirectly from biomarkers and treatments developed with research faciltated
by <a 
href="#glo:genomics">genomics</a>. This second strategy is likely to have a more immediate patient benefit due to
the cost of genome sequencing, particularly considering adoption in public healthcare
systems with a limited budget.
<a 
 id="x1-24001r20"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.1.6.1 </span> <a 
 id="x1-250001"></a>Cancer Genes and Driver Mutations</h5>
<!--l. 219--><p class="noindent" >There are two main categories of &#x201C;cancer genes&#x201D; (<a 
href="#XFutreal2001">Futreal <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XFutreal2001">2001</a>). Oncogenes are those
activated in cancers either by gain of function mutations in proto-oncogenes, amplification of
DNA copies, or evelated gene expression. Their normal functions are typically to regulate
stem cells or to promote cellular growth and recurrent mutations are typically concentrated
to particular gene regions. Conversely, tumour suppressor genes are those inactivated in
                                                                                    
                                                                                    
cancer either by loss of function mutations, deletion of DNA copies, repression of gene
expression, or hypermethylation. Their normal functions are typically to regulate cell
division, DNA repair, and cell signalling.
</p><!--l. 221--><p class="indent" >Detecting these cancer genes is a major challenge in cancer biology and has been
revolutionised by genomic technologies. Recurrent mutations, or DNA copy number variants
and differential gene expression or DNA methylation are all indicative of cancer
genes (<a 
href="#XMattison2009">Mattison <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XMattison2009">2009</a>), which can be detected in <a 
href="#glo:genomics">genomics</a> data (<a 
href="#XMETABRIC2016">Pereira
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XMETABRIC2016">2016</a>;&#x00A0;<a 
href="#XTCGA2013PAN">Weinstein <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2013PAN">2013</a>). Important &#x201C;driver&#x201D; cancer genes (<a 
href="#XStratton2009">Stratton
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XStratton2009">2009</a>) are difficult to detect from &#x201C;passenger&#x201D; mutations due to patient
variation, tumour heterogeneity, and genomic instability. However, many cancer genes
have been replicated from previous studies or well supported from <a 
href="#glo:genomics">genomics</a> data.
There remains the challenge of translating the identification of cancer genes to
patient benefit with characterisation of variants of unknown significance, which
mutation or gene expression markers can be used to monitor tumour progression or
treament response, and design of therapeutic intervention against many molecular
target for which they have yet to be developed or repurposed from other disease to
cancers.
</p><!--l. 223--><p class="indent" >Driver mutations can be identified by whether they co-occur or are mutually exclusive
with mutations in other genes in cancers, are recurrently mutated across a significant
proportion of samples for a specific tissue type, or if mutations are recurrent across different
cancer tissue types (<a 
href="#XcBioPortal">cBioPortal</a>,&#x00A0;<a 
href="#XcBioPortal">2017</a>;&#x00A0;<a 
href="#XMETABRIC2016">Pereira <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XMETABRIC2016">2016</a>;&#x00A0;<a 
href="#XCOSMICdb">COSMIC</a>,&#x00A0;<a 
href="#XCOSMICdb">2016</a>;&#x00A0;<a 
href="#XTCGA2013PAN">Weinstein
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2013PAN">2013</a>;&#x00A0;<a 
href="#XICGC2011">Zhang <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XICGC2011">2011</a>). Approximately 140 driver mutations have been identified,
including many novel genes in particular cancers from <a 
href="#glo:genomics">genomics</a> studies, with 2&#x2013;8 in typically
occurring in each tumour usually affecting cell fate, survival, or genome maintenance
(<a 
href="#XVogelstein2013">Vogelstein <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XVogelstein2013">2013</a>).
<a 
 id="x1-25001r25"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.1.6.2 </span> <a 
 id="x1-260002"></a>Personalised or Precision Cancer Medicine</h5>
<!--l. 226--><p class="noindent" >The notion of using a patient&#x2019;s genome to tailor healthcare to an individual has been
appealing since the advent of <a 
href="#glo:genomics">genomics</a>, popularised with the term &#x201C;personalised medicine&#x201D;.
This approach was expected to span from preventative lifestyle advice to effective
treatments. Personalised medicine was described in contrast with current strategies of
health advice, screening, prognostics, and treatments based on what works well
for the majority of the population. Adverse effects of these treatments occur in a
                                                                                    
                                                                                    
significant subpopulation, particularly demographics under-represented in clinical
studies.
</p><!--l. 228--><p class="indent" >The importance of <a 
href="#glo:genomics">genomics</a> is still recognised in translational cancer research.
Applications are particularly emphasised in molecular diagnosis, prognosis, and treatments of
patients already presenting with cancers in the clinic rather than preventative
medicine. This is in part due to the vast number of variants of unknown clinical
significance, the ethical issue of reporting on incidental findings, and the regulatory
issues direct-to-consumer genetics companies have encountered offering health risk
assessment.
</p><!--l. 230--><p class="indent" >More recently the term &#x201C;genomic medicine&#x201D; has been preffered the describe the paradigm
of treating cancers by their genomic features, particularly grouping patients by the mutation,
expression, or DNA methylation profiles of their cancers. Radical proponents advocate
for these molecular subtypes to supercede tissue or cell type specific diagnosis of
cancers. However, in practice they are often used in combination, with clinical and
pathological factors being informative of prognosis and the medical expertise required for
treatment. The related term of &#x201C;precision medicine&#x201D; also stems from this trend with the
rationale to target these molecular subtypes with separate treatment strategies,
particularly in developing and applying treatments targeted against a particular
mutation specific to cancers. To this end much research in this field is focused on
identifying mutations and gene expression signatures amenable to distinguishing
cancers, particularly oncogenic driver mutations, and developing treatments against
them.
<a 
 id="x1-26001r23"></a>
</p>
<a 
 id="x1-270001"></a>Molecular Diagnostics and Pan-Cancer Medicine
<!--l. 233--><p class="indent" >There is growing support for the use of molecular tools such as mutations or gene
expression signatures to diagnose tumour subtypes addition to (or in lieu of) tissue of origin
or histology. This is particularly important in breast cancer where analysis of molecular data
detected several distinct &#x201C;intrinsic subtypes&#x201D; with differences in malignancy and patient
outcome which were distinguished by molecular mechanisms rather than tissue or cellular
phenotype (<a 
href="#XParker2009">Parker <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XParker2009">2009</a>;&#x00A0;<a 
href="#XPerou2000">Perou <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XPerou2000">2000</a>). Conversely, common molecular
mechanisms may be shared between cancers across tissue types as discovered by the &#x201C;Pan
cancer&#x201D; studies, such as those conducted by the <a 
href="#glo:TCGA">TCGA</a> and <a 
href="#glo:ICGC">International Cancer Genome
Consortium (ICGC)</a> projects, which combined molecular profiles across tissue types
<a 
href="#XTCGA2013PAN">Weinstein <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XTCGA2013PAN">2013</a>). The molecular subtypes could feasibly be included in clinical
                                                                                    
                                                                                    
testing as a panel of biomarkers for diagnostics and prognosis. Such biomarkers
also have the potential to monitor drug response or risk of recurrence. This is
also raises the need for development of treatments that target these molecular
subtypes.
<a 
 id="x1-27001r26"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.1.6.3 </span> <a 
 id="x1-280003"></a>Targeted Therapeutics and Pharmacogenomics</h5>
<!--l. 236--><p class="noindent" >Targeted therapies with specificity against a molecular target are emerging as precision
cancer medicine. Molecular targets can be tested in laboratory conditions with <a 
href="#glo:RNA">RNA</a>
interference or pharmacological agents. Identification of molecular targets is important for
developing novel anti-cancer treatments along with validation and drug testing. For
oncogenic mutations, the recurrent mutant variant or overexpressed gene is directly inhibited
using structure-aided drug design or compound screening. However, oncogenes with high
homology to other genes or tumour suppressor genes (where lost in cancers) are not
amenable to direct targeting (<a 
href="#XKaelin2009">Kaelin, Jr</a>,&#x00A0;<a 
href="#XKaelin2009">2009</a>).
</p><!--l. 241--><p class="indent" >Despite controversy over their prohibitively high cost (<a 
href="#XPharmac2016">PHARMAC</a>,&#x00A0;<a 
href="#XPharmac2016">2016</a>), targeted
therapeutics have been applied as monoclonal antibodies against oncogenes (such as <span 
class="cmti-12">HER2</span>)
with relative sucess in clinical trials (<a 
href="#XMiles2001">Miles</a>,&#x00A0;<a 
href="#XMiles2001">2001</a>), generating considerable interest in wider
application of this approach. Targeted therapeutics have potential to have applications
across cancer tissue types, specificity against tumour cells, wide therapeutic windows, and
combination therapies (even in advanced disease or as a chemopreventative in high-risk
individuals).
<a 
 id="x1-28001r27"></a>
</p>
<a 
 id="x1-290001"></a>Targeting Oncogenic Driver Mutations
<!--l. 247--><p class="indent" >Oncogene targeted therapies have also been developed with some examples of
effective clinical application against cancers. However, they have already begun
to manifest problems with resistance, recurrence, tissue specificity, and design of
inhibitors specific to oncogenic variants rather than proto-oncogene precusors.
Targeted anticancer therapeutics can exploit complex interactions to distinguish
normal and cancerous cells which may benefit from studies of gene regulation or
interaction networks. The unexpected synergy between inhibitors of the oncogenes
<span 
class="cmti-12">BRAF </span><sup class="textsuperscript"><span 
class="cmr-10x-x-109">V600E</span> </sup> and <span 
class="cmti-12">EGFR </span>in colorectal cancer is an example of such a system <a 
href="#XPrahallad2012">Prahallad
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XPrahallad2012">2012</a>).
                                                                                    
                                                                                    
</p><!--l. 253--><p class="indent" >Despite successful application of vemurafenib against <span 
class="cmti-12">BRAF</span><sup class="textsuperscript"><span 
class="cmr-10x-x-109">V600E</span></sup> in melanomas
<a 
href="#XDienstmann2011">Dienstmann and Tabernero</a>&#x00A0;(<a 
href="#XDienstmann2011">2011</a>);&#x00A0;<a 
href="#XRavnan2012">Ravnan and Matalka</a>&#x00A0;(<a 
href="#XRavnan2012">2012</a>), colorectal cancers with
<span 
class="cmti-12">BRAF </span><sup class="textsuperscript"><span 
class="cmr-10x-x-109">V600E</span> </sup> mutations have poor prognosis and lack drug response. <a 
href="#XPrahallad2012">Prahallad <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XPrahallad2012">2012</a>)
used an <a 
href="#glo:RNAi">RNA interference (RNAi)</a> screen and found that <span 
class="cmti-12">EGFR </span>inhibition is synergistic with
vemurafenib aginst <span 
class="cmti-12">BRAF</span><sup class="textsuperscript"><span 
class="cmr-10x-x-109">V600E</span></sup> in colon cell lines and xenografts due to feedback activation
of <span 
class="cmti-12">EGFR</span>. Vemurafenib induced rapid reactivation of MAPK/ERK signalling via <span 
class="cmti-12">EGFR </span>in
colorectal cell lines in a tissue-specific manner <a 
href="#XCorcoran2012">Corcoran <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XCorcoran2012">2012</a>) and may be be
relevant to acquired resistance in melanoma <a 
href="#XSun2014">Sun <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XSun2014">2014</a>). Thus combination therapies
against several molecular pathways may be necessary to anticipate acquired resistance
<a 
href="#XRavnan2012">Ravnan and Matalka</a>&#x00A0;(<a 
href="#XRavnan2012">2012</a>) and targeted therapeutics may be further refined from
understanding the pathway structure and functional interactions across cancer
cells.
<a 
 id="x1-29001r28"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.1.6.4 </span> <a 
 id="x1-300004"></a>Systems and Network Biology</h5>
<!--l. 257--><p class="noindent" >It is also important to consider that driver mutations in oncogenes and tumour
suppressor genes do not occur in isolation. The genetic interaction, regulatory
and cellular signalling, and metabolic reactions of are all inter-related and may
each be perturbed by aberrations in gene function occurring in cancers. These
relationships can be represented by biological networks by mapping pairs of genes
with a particular relationship. Due to the complexity of a cell, these molecular
networks are very large consisting of thousands of nodes comprised by genes or
proteins.
</p><!--l. 259--><p class="indent" >The properties of large networks were first studied by constructing random networks by
randomly linking a fixed number of nodes (<a 
href="#XErdos1959">Erd<img 
src="20170511_thesis_introduction0x.png" alt="&#x02DD;o  "  class="Huml"  />s and R&#x00E9;nyi</a>,&#x00A0;<a 
href="#XErdos1959">1959</a>,&#x00A0;<a 
href="#XErdos1960">1960</a>). Despite the
random nature of these networks, properties such as their connectivity were well
characterised. The vertex degree (number of partners for each node) of random network
follows a Poisson distribution, however this property does not hold in nature, suggesting
that natural networks are non-random or not formed in this way <a 
href="#XBarabasi2004">Barab&#x00E1;si and
Oltvai</a>&#x00A0;(<a 
href="#XBarabasi2004">2004</a>).
</p><!--l. 261--><p class="indent" >This work formed the foundation for studying complex networks (<a 
href="#XvanSteen2010">van Steen</a>,&#x00A0;<a 
href="#XvanSteen2010">2010</a>),
which model features of real world networks not found in Erd<img 
src="20170511_thesis_introduction1x.png" alt="&#x02DD;o  "  class="Huml"  />s and R&#x00E9;nyi&#x2018;s random
networks (<a 
href="#XErdos1959">Erd<img 
src="20170511_thesis_introduction2x.png" alt="&#x02DD;o  "  class="Huml"  />s and R&#x00E9;nyi</a>,&#x00A0;<a 
href="#XErdos1959">1959</a>,&#x00A0;<a 
href="#XErdos1960">1960</a>). The small world property, made popular by
findings in social networks (<a 
href="#XTravers1969">Travers and Milgram</a>,&#x00A0;<a 
href="#XTravers1969">1969</a>), is the remarkably short path
lengths between any nodes in a small world network. A small world network is
                                                                                    
                                                                                    
well-connected with a characteristic path length (the average length of shortest paths
between all pairs of nodes) proportional to the logarithm of the number of nodes. <a 
href="#XWatts1998">Watts and
Strogatz</a>&#x00A0;(<a 
href="#XWatts1998">1998</a>) developed a model of random rewiring of a regular network to construct
random networks with the small world property and a high clustering coefficient. While these
properties are more representative of networks occurring in nature, their model is limited by
the degree distribution which converges to a Poisson distribution as it is rewired <a 
href="#XBarrat2000">Barrat and
Weigt</a>&#x00A0;(<a 
href="#XBarrat2000">2000</a>).
</p><!--l. 263--><p class="indent" >The vertex degree distribution of naturally occurring networks often follows a power law
distribution with the majority of nodes having far fewer connections than average and a
small subset of highly connected network &#x2018;hubs&#x2019; <a 
href="#XBarabasi1999">Barab&#x00E1;si and Albert</a>&#x00A0;(<a 
href="#XBarabasi1999">1999</a>). Hubs further
differentiate into &#x2018;party&#x2019; hubs (which interact simultaneously with many partners) and &#x2018;date&#x2019;
hubs (which interact with different partners in different conditions) <a 
href="#XHan2004">Han <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XHan2004">2004</a>).
Network hubs can also be classed as associative or dissociative depending on whether
they tend toward or away from connecting directly to other network hubs (<a 
href="#XvanSteen2010">van
Steen</a>,&#x00A0;<a 
href="#XvanSteen2010">2010</a>). The associative and dissociative properties can also be used to test
whether nodes of a particular subgroup (e.g., gene function) associate with each
other.
</p><!--l. 265--><p class="indent" ><a 
href="#XBarabasi1999">Barab&#x00E1;si and Albert</a>&#x00A0;(<a 
href="#XBarabasi1999">1999</a>) constructed a network model in an entirely different way to
randomly generate scale-free networks which have a power law degree distribution. They
constructed random networks by preferential attachment, modelling growth of a network by
sequentially adding nodes with links to existing nodes. The scale-free nature of the random
networks was ensured by adding new nodes with an increasing probability of attachment to
an existing node if it has higher degree. These networks successfully capture the scale-free
nature of many real world networks with short characteristic path length and low
eccentricity resulting in super small worlds <a 
href="#XBarabasi1999">Barab&#x00E1;si and Albert</a>&#x00A0;(<a 
href="#XBarabasi1999">1999</a>). Scale-free
networks are limited by a low clustering coefficient and lack of modular structure;
however, they have enabled the study of scale-free network topology and served as a
basis for modified scale-free models (<a 
href="#XDorogovtsev2003">Dorogovtsev and Mendes</a>,&#x00A0;<a 
href="#XDorogovtsev2003">2003</a>;&#x00A0;<a 
href="#XHolme2002">Holme and
Kim</a>,&#x00A0;<a 
href="#XHolme2002">2002</a>).
</p><!--l. 267--><p class="indent" ><a 
href="#XHan2004">Han <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XHan2004">2004</a>) observed dynamic modularity in biological networks and suggested the
network structure may underpin genetic robustness and plasticity. They focus on
network hubs which are more likely to be essential genes and define the subgroups
of hubs based on correlation of gene expression with protein-protein interaction
partners: &#x2018;party&#x2019; hubs (which interact simultaneously with many partners) and
&#x2018;date&#x2019; hubs (which interact with different partners in different conditions). Party
                                                                                    
                                                                                    
and date hubs occurred most frequently within and between network modules
respectively. Party hubs were considered local regulators, whereas date hubs were
considered important to network connectivity as global regulators. This distinction
between classes of network hubs was supported by differences in tissue specificity
and clinical relevance as a proposed predictor of clinical outcome in breast cancer
with an <a 
href="#glo:AUROC">area under the reciever operating characteristic (AUROC)</a> of 0.784 <a 
href="#XTaylor2009">Taylor
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XTaylor2009">2009</a>). However, correlation between expression and protein interactions
were not robustly reproduced. The importance of date hubs has been criticised for
assuming a bimodal distribution and basing the global importance of data hubs on a
small subset <a 
href="#XAgarwal2010">Agarwal <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XAgarwal2010">2010</a>). As an alternative interpretation, (<a 
href="#XAgarwal2010">Agarwal
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XAgarwal2010">2010</a>) suggest the importance of interactions rather than network hubs as
interactions important to the network were between functionally similar proteins.
Network hubs can also be classed as associative or dissociative depending on whether
they tend toward or away from connecting directly to other network hubs (van
Steen 2010). The associative and dissociative properties can also be used to test
whether nodes of a particular subgroup (e.g., gene function) associate with each
other.
</p><!--l. 269--><p class="indent" >Applications of network theory are diverse, including uses in social sciences, engineering,
and computer science. Due to their complexity and difficulty of gathering sufficient empirical
data, biological applications of network theory are relatively unexplored. High-throughput
technologies such as siRNA screens, two-hybrid screens, microarrays and massively
parallel sequencing have made generating genome-scale molecular data feasible
and enabled analysis of biological networks at the molecular level. Many types
of inter-related molecular networks can be constructed and analysed, depending
on the biological application. Genetic interaction networks will be the focus of
this project because they are relatively unexplored compared to other molecular
networks, have potential for applications in drug discovery (particularly cancer
treatment), and may lead to better understanding of the role of genetics in cellular
function and disease. Genetic interactions are usually studied at a high-throughput
scale in simple model organisms such as bacteria, yeasts or the nematode worm;
studies in humans, mammals, and non-model organisms (where applications would
have the most societal impact) are limited by cost, time and labour constraints.
Computational approaches with effective predictive models are the only feasible approach to
study the connectivity of a biological network in a complex metazoan cell at the
genome-scale.
                                                                                    
                                                                                    
<a 
 id="x1-30001r29"></a>
</p>
<a 
 id="x1-310001"></a>Network Medicine, and Polypharmacology
<!--l. 273--><p class="indent" >Molecular networks are biological networks consisting of biological molecules including
genes, transcripts (with non-coding and microRNAs), or proteins related by known
interactions and gene regulatory or metabolic pathways. Targeted therapeutics have had
some success for drug discovery, particularly in anticancer applications, including exploiting
these molecular networks by designing combination therapies and applying a network
pharmacology framework <a 
href="#XHopkins2008">Hopkins</a>&#x00A0;(<a 
href="#XHopkins2008">2008</a>). Rational design of drugs selective to a single
target has often failed to deliver clinical efficacy. Many existing effective drugs modulate
multiple proteins, having been selected for biological effects or clinical outcome rather than
molecular targets. Proponents of network biology and polypharmacology (specific binding to
multiple targets) recommend developing drugs with a desired target profile designed for the
target topology <a 
href="#XBarabasi2004">Barab&#x00E1;si and Oltvai</a>&#x00A0;(<a 
href="#XBarabasi2004">2004</a>);&#x00A0;<a 
href="#XHopkins2008">Hopkins</a>&#x00A0;(<a 
href="#XHopkins2008">2008</a>). Multi-target treatments
aim to achieve a clinical outcome through modulation of molecular networks since
the genetic robustness of a cell often compensates for loss of a single molecular
target.
</p><!--l. 275--><p class="indent" >While multi-target drugs may be more difficult to design, they are faster to test clinically
than drug combinations which are usually required to be tested separately first
<a 
href="#XHopkins2008">Hopkins</a>&#x00A0;(<a 
href="#XHopkins2008">2008</a>). <a 
href="#glo:synthetic lethal">Synthetic lethal</a> treatments for cancer, drug combinations and multi-target
drugs to combat resistance to chemotherapy and antibiotics can be informed by biological
networks <a 
href="#XBarabasi2004">Barab&#x00E1;si and Oltvai</a>&#x00A0;(<a 
href="#XBarabasi2004">2004</a>);&#x00A0;<a 
href="#XHopkins2008">Hopkins</a>&#x00A0;(<a 
href="#XHopkins2008">2008</a>). Further optimisation of timing and
dosing of drug combinations may increase efficacy of treatments with low efficacy applied
separately. Low doses and drug holidays are other counter intuitive approaches which may
increase clinical efficacy, reduce adverse effects, and reduce drug resistance (<a 
href="#XSun2014">Sun
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XSun2014">2014</a>;&#x00A0;<a 
href="#XTsai2012">Tsai <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTsai2012">2012</a>).
</p><!--l. 277--><p class="indent" >A molecular map of the interactions and pathways in the mammalian cellular network
has the potential to impact upon drug design and clinical practice, particularly in treatment
of cancer and infectious disease. Characterisation of the target system and impact of existing
treatments, such as <span 
class="cmti-12">BRAF</span><sup class="textsuperscript"><span 
class="cmr-10x-x-109">V600E</span></sup> and <span 
class="cmti-12">EGFR </span>inhibitors, enable wider application of the
mechanisms for such interventions exploiting genetic interactions or pathways. This could
lead to development of more effective treatment interventions for these systems and
prediction of similar molecular systems for development of novel drug targets and
combinations.
<a 
 id="x1-31001r5"></a>
                                                                                    
                                                                                    
</p>
<h3 class="sectionHead"><span class="titlemark">1.2 </span> <a 
 id="x1-320002"></a>A Synthetic Lethal Approach to Cancer Medicine</h3>
<!--l. 285--><p class="noindent" ><a 
href="#glo:synthetic lethal">Synthetic lethality</a> has vast potential to improve cancer medicine by expanding application of
targeted therapeutics to include inactivation of tumour suppressors and genes that are
difficult to target directly. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> interactions are also studied for gene
function and drug mode-of-action in model organisms. This section introduces the
concept of <a 
href="#glo:synthetic lethal">synthetic lethality</a> as it was originally conceived and how it has been
adopted conceptually in cancer research. Detecting these interactions at scale and
interpreting them is the focus of this thesis, hence we start with an overview of the
concepts involved, initial work on the interaction, and the rationale for applications
to cancer. Specific investigations into <a 
href="#glo:synthetic lethal">synthetic lethality</a> in cancer, detection by
experimental screening, and prediction by computational analysis will then be
reviewed.
<a 
 id="x1-32001r24"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.2.1 </span> <a 
 id="x1-330001"></a>Synthetic Lethal Genetic Interactions</h4>
<!--l. 294--><p class="noindent" >Genetic interactions are a core concept of molecular biology, discovered among earliest
investigations of Mendelian genetics, and receiving revived interest with new technologies
and potential applications. Biological epistasis is the effect of an allele at one locus
&#x201C;masking&#x201D; the phenotype of another locus (<a 
href="#XBateson1909">Bateson and Mendel</a>,&#x00A0;<a 
href="#XBateson1909">1909</a>). Statistical epistasis
is where there is significant disparity between the observed and expected phenotype of a
double mutant, compared to the respective phenotypes of single mutants and the wild-type
(<a 
href="#XFisher1919">Fisher</a>,&#x00A0;<a 
href="#XFisher1919">1919</a>). Fisher&#x2019;s definition lends itself to quantitative traits and more broadly
encompasses synthetic genetic interactions (<a 
href="#glo:SGI">synthetic genetic interactions</a>). These
have become popular for studies in yeast genetics and cancer drug design (<a 
href="#XBoone2007">Boone
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBoone2007">2007</a>;&#x00A0;<a 
href="#XKaelin2005">Kaelin, Jr</a>,&#x00A0;<a 
href="#XKaelin2005">2005</a>).
</p><!--l. 297--><p class="indent" >Synthetic genetic interactions are substantial deviations of growth or viability from the
expected null mutant phenotype (of an organism or cell) assuming additive (deleterious)
effects of the single mutants. The double mutant does not necessarily have either single
mutant phenotype (as shown for cellular growth phenotypes in Figure&#x00A0;<a 
href="#x1-33001r1">1.1<!--tex4ht:ref: fig:Costanzo2011 --></a>). Most <a 
href="#glo:SGI">SGIs</a>
are more viable than either single mutant or less viable than the expected double
mutant. Mutations are &#x201C;synergistic&#x201D; in negative <a 
href="#glo:SGI">SGIs</a> with more deviation from the
wild-type than expected. Formally, &#x201C;synthetic sick&#x201D; (SSL) and &#x201C;synthetic lethal&#x201D; (SL)
interactions are negative <a 
href="#glo:SGI">SGIs</a> giving growth inhibition and inviability respectively.
<a 
href="#glo:synthetic lethal">Synthetic lethality</a> in cancer research more broadly describes any negative <a 
href="#glo:SGI">SGI</a>
                                                                                    
                                                                                    
with specific inhibition of a mutant cell, including SSL interactions. Mutations are
&#x201C;alleviating&#x201D; in positive <a 
href="#glo:SGI">SGIs</a> with less deviation from the wild-type than expected.
For viability, &#x201C;suppression&#x201D; and &#x201C;rescue&#x201D; are positive <a 
href="#glo:SGI">SGIs</a> giving at least partial
restoration of wild-type growth from single mutants with growth impairment and lethal
phenotypes respectively. Negative <a 
href="#glo:SGI">SGIs</a> were markedly more common than positive
<a 
href="#glo:SGI">SGIs</a> in a number of studies in model systems <a 
href="#XBoucher2013">Boucher and Jenna</a>&#x00A0;(<a 
href="#XBoucher2013">2013</a>);&#x00A0;<a 
href="#XTong2004">Tong
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XTong2004">2004</a>).
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                    
                                                                                    
<a 
 id="x1-33001r1"></a>
                                                                                    
                                                                                    
____________________________________________________________________________________
____________________________________________________________________________________&#x00A0;&#x00A0;&#x00A0;&#x00A0;&#x00A0;        <!--l. 301--><p class="noindent" ><img 
src="20170511_thesis_introduction0y.png" alt="pict"  
 />
<br /> </p><div class="caption" 
><span class="id">Figure&#x00A0;1.1: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Synthetic genetic interactions. </span><span 
class="cmr-10x-x-109">Impact of various negative and positive</span>
<a 
href="#glo:SGI"><span 
class="cmr-10x-x-109">SGIs</span></a><span 
class="cmr-10x-x-109">: negative interactions involve deleterious (sick) or inviable (lethal) phenotypess</span>
<span 
class="cmr-10x-x-109">whereas positive interactions involve restoring viability by masking or suppressing the</span>
<span 
class="cmr-10x-x-109">other  mutation  or  complete  rescue  of  the  wildtype  phenotype.  Figure  adapted  from</span>
<span 
class="cmr-10x-x-109">(</span><a 
href="#XCostanzo2011"><span 
class="cmr-10x-x-109">Costanzo </span><span 
class="cmti-10x-x-109">et</span><span 
class="cmti-10x-x-109">&#x00A0;al.</span></a><span 
class="cmr-10x-x-109">,</span><span 
class="cmr-10x-x-109">&#x00A0;</span><a 
href="#XCostanzo2011"><span 
class="cmr-10x-x-109">2011</span></a><span 
class="cmr-10x-x-109">) concerning growth viability fitness in yeast.</span></span></div><!--tex4ht:label?: x1-33001r1 -->
                                                                                    
                                                                                    
</div><hr class="endfigure" />
<a 
 id="x1-33002r33"></a>
<h4 class="subsectionHead"><span class="titlemark">1.2.2 </span> <a 
 id="x1-340002"></a>Synthetic Lethal Concepts in Genetics</h4>
<!--l. 312--><p class="noindent" ><a 
href="#glo:synthetic lethal">Synthetic lethal</a> genes are generally regarded to arise due to functional redundancy. Due to
the functional level of <a 
href="#glo:SGI">SGIs</a>, <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes do not need directly interact, nor be
expressed in the same cell or at the same developmental stage: serving related functions is
sufficient to affect cell (or organism) viability and be relevant to drug-mode-of-action cancer
biology. Combined loss of genes performing an essential or important function in a cell are
therefore deleterious. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> gene pairs are therefore pairwise essential with
&#x201C;induced essentiality&#x201D;: each <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene becomes essential to the cell upon loss of
the other.
</p><!--l. 314--><p class="indent" >Since <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene partners can be affected by extracellular stimuli such as
chemicals, essentiality of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes can be induced by the environment of a cell.
An environmental stress condition may inhibit one or the other <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene, such as
exposure to chemicals, in which case the <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner gene is &#x201C;conditionally
essential&#x201D; (<a 
href="#XHillenmeyer2008">Hillenmeyer</a>,&#x00A0;<a 
href="#XHillenmeyer2008">2008</a>). Thus the evolutionary rationale for the abundance of <a 
href="#glo:SGI">SGIs</a>
(compared to the surprisingly low number of essential genes) in a Eukaryotic genome
attributed to genetic functional redundancy and network robustness of a cell which are
advantageous to survival.
</p><!--l. 316--><p class="indent" >Biological functions are typically performed by a pathway of genes (or their products).
Many genes of the same pathway may be fucntionally interchangable as <a 
href="#glo:synthetic lethal">synthetic lethal</a>
partners of a particular gene since loss of the pathway is deleterious without the <a 
href="#glo:synthetic lethal">synthetic
lethal</a> partner gene. Therefore biological pathways can be subject to induced essentiality
under loss of a gene and <a 
href="#glo:synthetic lethal">synthetic lethality</a> may occur at pathway level or in a gene
regulation network.
<a 
 id="x1-34001r35"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.2.3 </span> <a 
 id="x1-350003"></a>Studies of Synthetic Lethality</h4>
<!--l. 319--><p class="noindent" >Genetic high-throughput screens have identified unexpected, functionally informative, and
clinically relevant <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions; including <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of genes
recurrently mutated in cancer or attributed to familial early-onset cancers. While screening
presents an appealing strategy for <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery, computational approaches are
becoming popular as an alternative or complement to experimental methods to
overcome inherent bias and limitations of experimental screens. An array of recently
                                                                                    
                                                                                    
developed computational methods (<a 
href="#XJerby2014">Jerby-Arnon <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XJerby2014">2014</a>;&#x00A0;<a 
href="#XLu2015">Lu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XLu2015">2015</a>;&#x00A0;<a 
href="#XTiong2014">Tiong
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTiong2014">2014</a>;&#x00A0;<a 
href="#XWang2013">Wang and Simon</a>,&#x00A0;<a 
href="#XWang2013">2013</a>;&#x00A0;<a 
href="#XWappett2014">Wappett</a>,&#x00A0;<a 
href="#XWappett2014">2014</a>) show the need for <a 
href="#glo:synthetic lethal">synthetic lethal</a>
discovery in the fundamental genetics and translational cancer research community. However,
existing computational methods are not suitable for queries of genomic data for
interacting partners of a particular gene: they have been applied pairwise across the
genome, do not have software released to apply the methodology, or lack statistical
measures of error for further analysis. A robust prediction of gene interactions
is an effective and practical approach at a scale of the entire genome for ideal
translational applications, analysis of biological systems, and constructing functional gene
networks.
<a 
 id="x1-35001r30"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.2.3.1 </span> <a 
 id="x1-360001"></a>Synthetic Lethal Pathways and Networks</h5>
<!--l. 322--><p class="noindent" ><a 
href="#glo:SGI">SGIs</a> are very common in genomes, with 4<span 
class="cmsy-10x-x-120">&#x00D7; </span>more interactions detected with synthetic gene
array mating screens than protein-protein interactions yeast-2-hybrid studies (<a 
href="#XTong2004">Tong
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTong2004">2004</a>). The <a 
href="#glo:SGI">SGI</a> network is scale-free with power-law vertex degree distribution and
low average shortest path length (3.3) as expected for a complex biological network
(<a 
href="#XBarabasi2004">Barab&#x00E1;si and Oltvai</a>,&#x00A0;<a 
href="#XBarabasi2004">2004</a>). Highly connected &#x201C;hub&#x201D; genes with the highest number of links
(vertex degree) are functionally important with many negative <a 
href="#glo:SGI">SGI</a> hubs involved in cell
cycle regulation and many positive <a 
href="#glo:SGI">SGI</a> hubs involved in translation (<a 
href="#XBaryshnikova2010b">Baryshnikova
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBaryshnikova2010b">2010b</a>;&#x00A0;<a 
href="#XCostanzo2010">Costanzo <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XCostanzo2010">2010</a>). Negative <a 
href="#glo:SGI">SGIs</a> were far more common than positive
<a 
href="#glo:SGI">SGIs</a>, with synthetic gene loss being more likely to be deleterious to cell than advantageous
which indicates than <a 
href="#glo:synthetic lethal">synthetic lethality</a> may be comparably easier to detect than other
<a 
href="#glo:SGI">SGIs</a>.
</p><!--l. 324--><p class="indent" >Essential pathways are highly buffered with 5<span 
class="cmsy-10x-x-120">&#x00D7; </span>more interactions than other
<a 
href="#glo:SGI">SGIs</a>, consistent with strong selection for survival, as found with conditional and
partial mutations in essential genes (<a 
href="#XDavierwala2005">Davierwala <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XDavierwala2005">2005</a>). This <a 
href="#glo:SGI">SGI</a> network had
scale-free topology and rarely shared interactions with the protein-protein interaction
network. These networks are related by an &#x201C;orthogonal&#x201D; relationship: shared partners
in one network tend to be themselves connected directly in the other network.
Essential genes were likely to have closely related functions, whereas non-essential
networks were relatively more inclined to have <a 
href="#glo:SGI">SGIs</a> between distinct biological
pathways.
<a 
 id="x1-36001r31"></a>
</p>
                                                                                    
                                                                                    
<a 
 id="x1-370001"></a>Evolution of Synthetic Lethality
<!--l. 327--><p class="indent" >There is poor conservation of specific <a 
href="#glo:SGI">SGIs</a> between <span 
class="cmti-12">S. cerevisiae </span>and <span 
class="cmti-12">S. pombe </span>with 29%
of the interactions tested in both distantly related species being conserved between them
(<a 
href="#XDixon2008">Dixon <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XDixon2008">2008</a>). The remaining interactions show high species-specific differences;
however, many of the species-specific interactions were still conserved between
biological pathways, protein complexes, or protein-protein interaction modules.
Similarly, conservation of pathway redundancy was also found between Eukaryotes (<span 
class="cmti-12">S.</span>
<span 
class="cmti-12">cerevisiae</span>) and prokaryotes (<span 
class="cmti-12">E. coli</span>) (<a 
href="#XButland2008">Butland <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XButland2008">2008</a>). Negative <a 
href="#glo:SGI">SGIs</a> were
more likely to be conserved between biological pathways, whereas positive <a 
href="#glo:SGI">SGIs</a>
were more likely to be conserved within a pathway or protein complex (<a 
href="#XRoguev2008">Roguev
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XRoguev2008">2008</a>).
</p><!--l. 329--><p class="indent" >A modest 5% of interactions were conserved between unicellular (<span 
class="cmti-12">S. cerevisiae</span>) and
multicellular (<span 
class="cmti-12">C. elegans</span>) organisms. However, the nematode <a 
href="#glo:SGI">SGI</a> network had similar
scale-free topology and modularity despite differences in methodology: metazoan <a 
href="#glo:RNAi">RNAi</a>
screens are incomplete knockouts whereas screening null mutations is feasible in yeast
(<a 
href="#XBussey2006">Bussey <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBussey2006">2006</a>). The nematode <a 
href="#glo:SGI">SGI</a> screen identified network hubs with important
interactions to orthologues of known human disease genes (<a 
href="#XLehner2006">Lehner <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XLehner2006">2006</a>).
Despite the lack of direct conservation of <a 
href="#glo:SGI">SGIs</a> between yeasts and nematode worms,
genetic redundancy at the gene or pathway level may yet be consistent with an
induced essentiality model of <a 
href="#glo:SGI">SGIs</a> where gene functions are conserved with network
restructuring over evolutionary change (<a 
href="#XTischler2008">Tischler <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTischler2008">2008</a>). While nematode
models are more closely related to human cells, cancer cells can present growth and
viability phenotypes more comparable to yeast models. Therefore findings from both
SGA and <a 
href="#glo:RNAi">RNAi</a> models are relevant to understanding cellular network structure
and in healthy and cancerous human cells. <a 
href="#glo:RNAi">RNAi</a> has also been applied to human
and mouse cancer cells in cell culture and genetic screening experiments. These
findings suggest that <a 
href="#glo:SGI">SGI</a> network &#x201C;rewiring&#x201D; is a concern for identifying specific
<a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions in cancer and a pathway approach may be more robust
in the context of evolution, patient variation, tumour heterogeneity, and disease
progression.
<a 
 id="x1-37001r36"></a>
</p>
                                                                                    
                                                                                    
<h4 class="subsectionHead"><span class="titlemark">1.2.4 </span> <a 
 id="x1-380004"></a>Synthetic Lethal Concepts in Cancer</h4>
<!--l. 333--><p class="noindent" >Loss of function occurs in many genes in cancers including tumour suppressors and yet few
interventions target such mutations compared to targeted therapies for gain of function
mutation in oncogenes (<a 
href="#XKaelin2005">Kaelin, Jr</a>,&#x00A0;<a 
href="#XKaelin2005">2005</a>). <a 
href="#glo:synthetic lethal">Synthetic lethality</a> is a powerful design
strategy for therapies selective against loss of gene function with potential for
application against a range of genes and diseases (<a 
href="#XFece2015">Fece de&#x00A0;la Cruz <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XFece2015">2015</a>;&#x00A0;<a 
href="#XKaelin2009">Kaelin,
Jr</a>,&#x00A0;<a 
href="#XKaelin2009">2009</a>). Since <a 
href="#glo:synthetic lethal">synthetic lethality</a> affects cellular viability by indirect functional
relationships between genes, it is suitable for indirectly targeting mutations in
cancers. Once <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of cancer genes are identified, targeted
therapeutics can be applied against them. When genes are disrupted in cancers, the
induced essentiality of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners is a vulnerability that may be
exploited for anti-cancer therapy. This has the potential to be very specific against
cancer cells (with the target mutation) over non-cancer cells (with a functional
compensating gene). Analogous to &#x201C;oncogene addiction&#x201D;, where cancer cells adapt to
particular oncogenic growth signals and become reliant on them to remain viable (<a 
href="#XLuo2009">Luo
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XLuo2009">2009</a>;&#x00A0;<a 
href="#XWeinstein2000">Weinstein</a>,&#x00A0;<a 
href="#XWeinstein2000">2000</a>), <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of inactivated tumour
suppressors are required to maintain cancer cell viability and proliferation. As such
cancers are subject to &#x201C;non-oncogene addiction&#x201D; and are feasible anti-cancer drug
targets.
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                    
                                                                                    
<a 
 id="x1-38001r2"></a>
                                                                                    
                                                                                    
____________________________________________________________________________________
____________________________________________________________________________________&#x00A0;&#x00A0;&#x00A0;&#x00A0;&#x00A0;        <!--l. 338--><p class="noindent" ><img 
src="20170511_thesis_introduction1y.png" alt="pict"  
 />
<br /> </p><div class="caption" 
><span class="id">Figure&#x00A0;1.2: </span><span  
class="content"><a 
href="#glo:synthetic lethal"><span 
class="cmbx-10x-x-109">Synthetic lethality</span></a> <span 
class="cmbx-10x-x-109">in cancer. </span><span 
class="cmr-10x-x-109">Rationale of exploiting </span><a 
href="#glo:synthetic lethal"><span 
class="cmr-10x-x-109">synthetic lethality</span></a>
<span 
class="cmr-10x-x-109">for specifity against a tumour suppressor gene (e.g., </span><span 
class="cmti-10x-x-109">CDH1</span><span 
class="cmr-10x-x-109">) while other cells are spared</span>
<span 
class="cmr-10x-x-109">under the inhibition of a partner gene.</span></span></div><!--tex4ht:label?: x1-38001r2 -->
                                                                                    
                                                                                    
</div><hr class="endfigure" />
<!--l. 347--><p class="indent" >The <a 
href="#glo:synthetic lethal">synthetic lethal</a> approach to cancer medicine is most amenable to loss of function
mutations in tumour suppressor genes, where it would feasibly be effective against any loss of
function mutation across the tumour suppressor with a viable <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner gene
(as shown in Figure&#x00A0;<a 
href="#x1-38001r2">1.2<!--tex4ht:ref: fig:SL_Concept --></a>). However, the approach may also be suitable for cases where cancer
cells have mutations where the normal function of the gene is disrupted such as if it were
overexpression (&#x201C;synthetic dosage lethality&#x201D;) or if an oncogene interfered with the
function of the proto-oncogenic variant such as competitive inhibition. Thus <a 
href="#glo:synthetic lethal">synthetic
lethality</a> expands the range of cancer-specific mutations feasible to target with
targeted therapeutics to absence of tumour suppressor genes and distinguishing highly
homologous oncogenes by functional differences by targeting their <a 
href="#glo:synthetic lethal">synthetic lethal</a>
partners.
<a 
 id="x1-38002r39"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.2.5 </span> <a 
 id="x1-390005"></a>Clinical Impact of Synthetic Lethality in Cancer</h4>
<!--l. 355--><p class="noindent" >The <a 
href="#glo:synthetic lethal">synthetic lethal</a> interaction of <span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2 </span>with <span 
class="cmti-12">PARP1 </span>in breast cancer is an
example of how gene interactions are important in cancer, including translation to the clinic.
These genetic interactions enable specific targeting of mutations in <span 
class="cmti-12">BRCA1 </span>or
<span 
class="cmti-12">BRCA2 </span>tumour suppressor genes with PARP inhibitors by inducing <a 
href="#glo:synthetic lethal">synthetic
lethality</a> in breast cancer (<a 
href="#XFarmer2005">Farmer <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XFarmer2005">2005</a>). PARP inhibitors are one of the first
targeted therapeutics against a tumour suppressor mutation with success in clinical
trials.
</p><!--l. 357--><p class="indent" ><span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2 </span>and <span 
class="cmti-12">PARP1 </span>genes demonstrate the application of the <a 
href="#glo:synthetic lethal">synthetic lethal</a>
approach to cancer therapy <a 
href="#XAshworth2008">Ashworth</a>&#x00A0;(<a 
href="#XAshworth2008">2008</a>);&#x00A0;<a 
href="#XKaelin2005">Kaelin, Jr</a>&#x00A0;(<a 
href="#XKaelin2005">2005</a>). <span 
class="cmti-12">BRCA1 </span>and <span 
class="cmti-12">BRCA2 </span>are
homologous DNA repair genes, widely known as tumour suppressors; mutation carriers have
substantially increased risk of breast (risk by age 70 of 57% for <span 
class="cmti-12">BRCA1 </span>and 59% for
<span 
class="cmti-12">BRCA2 </span>) and ovarian cancers (risk by age 70 of 40% for <span 
class="cmti-12">BRCA1 </span>and 18% for <span 
class="cmti-12">BRCA2</span>)
(<a 
href="#XChen2007">Chen and Parmigiani</a>,&#x00A0;<a 
href="#XChen2007">2007</a>). The <span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2 </span>genes, which usually repair DNA or
destroy the cell if it cannot be repaired, have inactivating somatic mutations in some
familial and sporadic cancers. Poly-ADP-ribose polymerase (PARP) genes are
tumour suppressor genes involved in base excision DNA repair. Loss of PARP
activity results in single-stranded DNA breaks. However, <span 
class="cmti-12">PARP1</span><sup><span 
class="cmsy-8">&#x2212;</span><span 
class="cmmi-8">&#x2215;</span><span 
class="cmsy-8">&#x2212;</span></sup> knockout
mice are viable and healthy indicating low toxicity from PARP inhibition (<a 
href="#XBryant2005">Bryant
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBryant2005">2005</a>).
                                                                                    
                                                                                    
</p><!--l. 359--><p class="indent" ><a 
href="#XBryant2005">Bryant <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XBryant2005">2005</a>) showed that <span 
class="cmti-12">BRCA2 </span>cells were sensitive to PARP inhibition by
siRNA of <span 
class="cmti-12">PARP1 </span>or drug inhibition (which targets <span 
class="cmti-12">PARP1 </span>and <span 
class="cmti-12">PARP2</span>) using Chinese
hamster ovary cells, MCF7 and MDA-MB-231 breast cell lines. This effect was sufficient to
kill mouse tumour xenografts and showed high specificity to <span 
class="cmti-12">BRCA2 </span>deficient cells in culture
and xenografts. <a 
href="#XFarmer2005">Farmer <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XFarmer2005">2005</a>) replicated these results in embryonic stem cells and
showed that <span 
class="cmti-12">BRCA1 </span>cells were also sensitive to PARP inhibition relative to the wild-type
with siRNA and drug experiments in cell culture and drug activity against <span 
class="cmti-12">BRCA1 </span>or
<span 
class="cmti-12">BRCA2 </span>deficient embryonic stem cell mouse xenografts. They found evidence that
PARP inhibition causes DNA lesions, usually repaired in wild-type cells, which
lead to chromosomal instability, cell cycle arrest, and induction of apoptosis in
<span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2 </span>deficient cells. Therefore, the pathways cooperate to repair
DNA giving a plausible mechanism for combined loss as an effective anti-cancer
treatment.
</p><!--l. 361--><p class="indent" >Thus PARP inhibitors have potential for clinical use against <span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2</span>
mutations in hereditary and sporadic cancers (Ashworth 2008; Kaelin2005). PARP inhibition
has been found to be effective in cancer patients carrying <span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2 </span>mutations and
some other ovarian cancers, suggesting <a 
href="#glo:synthetic lethal">synthetic lethality</a> between PARP and other DNA
repair pathways (<a 
href="#XStrom2012">Str&#x00F6;m and Helleday</a>,&#x00A0;<a 
href="#XStrom2012">2012</a>). This supports the potential for PARP
inhibition as a chemo-preventative alternative to prophylactic surgery for high
risk individuals with <span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2 </span>mutations (<a 
href="#XStrom2012">Str&#x00F6;m and Helleday</a>,&#x00A0;<a 
href="#XStrom2012">2012</a>).
Hormone-based therapy has also been suggested as a chemo-preventative in such
high risk individuals and aromatase inhibitors have completed phase I clinical
trials for this purpose (Bozovic-Spasojevic2012). <a 
href="#XStrom2012">Str&#x00F6;m and Helleday</a>&#x00A0;(<a 
href="#XStrom2012">2012</a>) also
postulate increased efficacy of PARP inhibitors in the hypoxic DNA-damaging tumour
micro-environment.
</p><!--l. 363--><p class="indent" >A PARP inhibitor, olaparib, showed fewer adverse effects than cytotoxic chemotherapy
and anti-tumour activity in phase I trials against <span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2 </span>deficient familial
breast, ovarian, and prostate cancers (<a 
href="#XFong2009">Fong <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XFong2009">2009</a>) and sporadic ovarian cancer (<a 
href="#XFong2010">Fong
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XFong2010">2010</a>). AstraZeneca has reported phase II trials showing the treatment is effective in
<span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2 </span>deficient breast (<a 
href="#XTutt2010">Tutt <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTutt2010">2010</a>) and ovarian cancers (<a 
href="#XAudeh2010">Audeh
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XAudeh2010">2010</a>) with a favourable therapeutic window and similar toxicity between carriers of
<span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2 </span>mutations and sporadic cases. AstraZeneca announced that olaparib has
begun phase III trials for breast and ovarian cancers in 2013. Mixed results in phase II trials
in ovarian cancer are behind the delays addressed by retrospective analysis of the cohort
subgroup with confirmed mutation of <span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2 </span>genes in the tumour;
                                                                                    
                                                                                    
unsurprisingly these patients, benefit most from the PARP inhibitor treatment
and have increased platinum sensitivity in combination treatment. These PARP
inhibitors are FDA approved for some cancers <a 
href="#XMcLachlan2016">McLachlan <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XMcLachlan2016">2016</a>), are effective
against germline and sporadic <span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2 </span>mutations, and are a potential
prevention alternative to prophylactic surgery for high risk mutation carriers <a 
href="#XStrom2012">Str&#x00F6;m and
Helleday</a>&#x00A0;(<a 
href="#XStrom2012">2012</a>).
</p><!--l. 365--><p class="indent" >This demonstrates the clinical impact of a well characterised system of <a 
href="#glo:synthetic lethal">synthetic lethality</a>
with known cancer risk genes. <a 
href="#glo:synthetic lethal">Synthetic lethality</a> has the benefit of being effective against
inactivation of tumour suppressor genes by any means, broader than targeting a particular
oncogenic mutation (<a 
href="#XKaelin2005">Kaelin, Jr</a>,&#x00A0;<a 
href="#XKaelin2005">2005</a>). The targeted therapy is effective in both sporadic
and hereditary <span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2 </span>deficient tumours acting against an oncogenic molecular
aberration across several tissues.
<a 
 id="x1-39001r41"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.2.6 </span> <a 
 id="x1-400006"></a>High-throughput Screening for Synthetic Lethality</h4>
<!--l. 382--><p class="noindent" >The function of signalling pathways and combinations of interacting genes are important in
cancer research but classical genetics approaches have been limited to non-redundant
pathways (<a 
href="#XFraser2004">Fraser</a>,&#x00A0;<a 
href="#XFraser2004">2004</a>). The emerging <a 
href="#glo:RNAi">RNAi</a> technologies have vastly expanded the
potential for studying genetic redundancy in mammalian experimental models including
testing experimentally for <a 
href="#glo:synthetic lethal">synthetic lethality</a> (<a 
href="#XFraser2004">Fraser</a>,&#x00A0;<a 
href="#XFraser2004">2004</a>). Identifying <a 
href="#glo:synthetic lethal">synthetic lethality</a> is
crucial to study gene function, drug mechanisms, and design novel therapies (<a 
href="#XLum2004">Lum
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XLum2004">2004</a>). Candidate selection of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs relevant to cancer has shown
some success but is limited because interactions are difficult to predict; they can occur
between seemingly unrelated pathways in model organisms (<a 
href="#XCostanzo2011">Costanzo <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XCostanzo2011">2011</a>). While
biologically informed hypotheses have had some success in <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery (<a 
href="#XBitler2015">Bitler
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBitler2015">2015</a>;&#x00A0;<a 
href="#XBryant2005">Bryant <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBryant2005">2005</a>;&#x00A0;<a 
href="#XFarmer2005">Farmer <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XFarmer2005">2005</a>), interactions occurring indirectly
between distinct pathways would be missed (<a 
href="#XBoone2007">Boone <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBoone2007">2007</a>;&#x00A0;<a 
href="#XCostanzo2011">Costanzo <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XCostanzo2011">2011</a>).
Scanning the entire genome for interactions against a clinically relevant gene is
an emerging strategy being explored with high-throughput screens (<a 
href="#XFece2015">Fece de&#x00A0;la
Cruz <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XFece2015">2015</a>) and computational approaches (<a 
href="#XBoucher2013">Boucher and Jenna</a>,&#x00A0;<a 
href="#XBoucher2013">2013</a>;&#x00A0;<a 
href="#XvanSteen2011">van
Steen</a>,&#x00A0;<a 
href="#XvanSteen2011">2012</a>).
</p><!--l. 384--><p class="indent" >Experimental screening for <a 
href="#glo:synthetic lethal">synthetic lethality</a> is an appealing strategy for wider discovery
of functional interactions <span 
class="cmti-12">in vivo </span>despite many potential sources of error which must be
considered. The <a 
href="#glo:synthetic lethal">synthetic lethal</a> concept has both genetic and pharmacological screening
applications to cancer research. Genetic screens, with <a 
href="#glo:RNAi">RNAi</a> to discover the specific genes
                                                                                    
                                                                                    
involved, inform development of targeted therapies with a known mode of action,
anticipated mechanisms of resistance, and biomarkers for treatment response. <a 
href="#glo:RNAi">RNAi</a> is a
transient knockdown of gene expression more similar to the effect of drugs than
complete gene loss and makes comparison to screens in model organisms difficult
(<a 
href="#XBussey2006">Bussey <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBussey2006">2006</a>). The <a 
href="#glo:RNAi">RNAi</a> gene knockdown process has inherent toxicity to
some cells, potential off-target effects, and issues with a high false positive rate.
Therefore, it is important to validate any candidates in a secondary screen and
replicate knockdown experiments with a number of independent shRNAs. Alternative
gene knockout procedures have also been proposed for <a 
href="#glo:synthetic lethal">synthetic lethal</a> screening
including a genome-wide application of the CRISPR/Cas9/sgRNA genome editing
technology (<a 
href="#XSander2014">Sander and Joung</a>,&#x00A0;<a 
href="#XSander2014">2014</a>), episomal gene transfer (<a 
href="#XVargas2004">Vargas <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XVargas2004">2004</a>), or
<a 
href="#glo:RNAi">RNAi</a> with lentiviral transfection for delivery of shRNA (<a 
href="#XTelford2015">Telford <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTelford2015">2015</a>).
Genetic screens have potential for quantitative gene disruption experiments to
selectively target overexpressed genes in cancer via synthetic dosage lethality. While
powerful for understanding fundamental cellular function, analysis of isogenic cell lines
is inherently limited by assuming only a single mutation differs between them
despite susceptibility to &#x201C;genetic drift&#x201D; and cannot account for diverse genetic
backgrounds or tumour heterogeneity (<a 
href="#XFece2015">Fece de&#x00A0;la Cruz <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XFece2015">2015</a>). Genetic screens thus
identify targets to develop or repurpose targeted therapies for disease but alone
will not directly identify a lead compound to develop for the market or clinical
translation.
</p><!--l. 386--><p class="indent" >Chemical screens are immediately applicable to the clinic by directly screening for
selective lead compounds with suitable pharmacological properties. However chemical screens
lack a known mode of action, may affect many targets, and screen a narrow range of genes
with existing drugs. With either approach there are many challenges translating candidates
into the clinic such as finding targets relevant to a range of patients, validation of
targets, accounting for a range of genetic (and epigenetic) contexts or tumour
micro-environment, identifying effective synergistic combinations, enhancers of existing
radiation or cytotoxic treatments, avoiding inherent or acquired drug resistance, and
developing biomarkers for patients which will respond to <a 
href="#glo:synthetic lethal">synthetic lethal</a> treatment,
including integrating these into clinical trials and clinical practice. Identifying
specific target genes is an effective way to anticipate such challenges, which can
be approached with genetic screens, so we will focus on these and computational
alternatives. Screening methods have proven a fruitful area of research, despite being
costly, laborious, and having many different sources of error. These limitations
                                                                                    
                                                                                    
suggest a need for complementary computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a>
discovery.
<a 
 id="x1-40001r37"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.2.6.1 </span> <a 
 id="x1-410001"></a>Synthetic Lethal Screens</h5>
<!--l. 390--><p class="noindent" >Overexpression of genes is another suitable application for <a 
href="#glo:synthetic lethal">synthetic lethality</a> since
overexpressed genes cannot be distinguished from the wild-type by direct sequence specific
targeted therapy. Overexpression of oncogenes, such as <span 
class="cmti-12">EGFR</span>, <span 
class="cmti-12">MYC</span>, and <span 
class="cmti-12">PIM1</span>,
has been found to drive many cancers. <span 
class="cmti-12">PIM1 </span>is a candidate for <a 
href="#glo:synthetic lethal">synthetic lethal</a>
drug design in lymphomas and prostate cancers, where it interacts with <span 
class="cmti-12">MYC </span>to
drive cancer growth. <a 
href="#XvanderMeer2014">van&#x00A0;der Meer <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XvanderMeer2014">2014</a>) performed an <a 
href="#glo:RNAi">RNAi</a> screen for
<a 
href="#glo:synthetic lethal">synthetic lethality</a> between <span 
class="cmti-12">PIM1 </span>overexpression and gene knockdown in RWPE
prostate cancer cell lines. <span 
class="cmti-12">PLK1 </span>gene knockdown and drug inhibition was effective
as a specific inhibitor of <span 
class="cmti-12">PIM1 </span>overexpressing prostate cells in cell culture and
mouse tumour xenografts. <span 
class="cmti-12">PLK1 </span>inhibition reduced <span 
class="cmti-12">MYC </span>expression in pre-clinical
models, consistent with expression in human tumours in which <span 
class="cmti-12">PIM1 </span>and <span 
class="cmti-12">PLK1</span>
are co-expressed and correlated with tumour grade. Thus <a 
href="#glo:RNAi">RNAi</a> screening was
valuable to identify therapeutic targets and biomarkers for patient response as
demonstrated with the finding of <span 
class="cmti-12">PLK1 </span>as a candidate drug target against prostate cancer
progression.
</p><!--l. 392--><p class="indent" ><a 
href="#glo:HLRCC">Hereditary leiomyomatosis and renal cell carcinoma (HLRCC)</a> is a cancer syndrome of
predisposition to benign tumours in the uterus and risk of malignant cancer of the kidney
attributed to inherited mutations in fumarate hydratase (<span 
class="cmti-12">FH</span>). <a 
href="#XBoettcher2014">Boettcher <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XBoettcher2014">2014</a>)
performed an <a 
href="#glo:RNAi">RNAi</a> screen on HEK293T renal cells for <a 
href="#glo:synthetic lethal">synthetic lethality</a> with <span 
class="cmti-12">FH</span>. They
found enrichment of haem metabolism (consistent with the literature) and adenylate cyclase
pathways (consistent with cAMP dysregulation in <span 
class="cmti-12">FH </span>mutant cells). <a 
href="#glo:synthetic lethal">Synthetic lethality</a>
between <span 
class="cmti-12">FH </span>mutation and adenylate cyclases was validated with gene knockdown, drug
experiments, and replicated across both HEK293T renal cells and VOK262 cells
derived from a HLRCC patient, suggesting new potential treatments against the
disease.
</p><!--l. 394--><p class="indent" >Similarly, <a 
href="#glo:HDGC">hereditary diffuse gastric cancer (HDGC)</a> is a cancer syndrome of
predisposition to early-onset malignant stomach and breast cancers attributed to mutations
in <a 
href="#glo:E-cadherin">E-cadherin</a> (<span 
class="cmti-12">CDH1</span>). <a 
href="#XTelford2015">Telford <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XTelford2015">2015</a>) performed an <a 
href="#glo:RNAi">RNAi</a> screen on MCF10A breast
cells for <a 
href="#glo:synthetic lethal">synthetic lethality</a> with <span 
class="cmti-12">CDH1</span>. They found enrichment of G-protein coupled
receptors (GPCRs) and cytoskeletal gene functions. The results were consistent with a
                                                                                    
                                                                                    
concurrent drug compound screen with a number of candidates validated by lentiviral
shRNA gene knockdown and drug testing including inhibitors of Janus kinase, histone
deacetylases, phosphoinositide 3-kinase, aurora kinase, and tyrosine kinases. Therefore the
<a 
href="#glo:synthetic lethal">synthetic lethal</a> strategy has potential for clinical impact against <a 
href="#glo:HDGC">HDGC</a>, with
particular interest in interventions with low adverse effects for chemo-prevention,
including repurposing existing approved drugs for activity against <span 
class="cmti-12">CDH1 </span>deficient
cancers.
</p><!--l. 396--><p class="indent" ><a 
href="#glo:RNAi">RNAi</a> screening for <a 
href="#glo:synthetic lethal">synthetic lethality</a> is also useful for functional genetics to
understand drug sensitivity. <a 
href="#XAarts2015">Aarts <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XAarts2015">2015</a>) screened WiDr colorectal cells for
<a 
href="#glo:synthetic lethal">synthetic lethality</a> between <span 
class="cmti-12">WEE1 </span>inhibitor treatment and an <a 
href="#glo:RNAi">RNAi</a> library of
1206 genes with functions known to be amenable to drug treatment or important
in cancer such as kinases, phosphatases, tumour suppressors, and DNA repair
(a pathway <span 
class="cmti-12">WEE1 </span>regulates). Screening identified a number of <a 
href="#glo:synthetic lethal">synthetic lethal</a>
candidates including genes involved in cell cycle regulation, DNA replication, repair,
homologous recombination, and Fanconi anaemia. <a 
href="#glo:synthetic lethal">Synthetic lethality</a> with cell-cycle and
DNA repair genes was consistent with the literature and validation in a panel of
breast and colorectal cell lines supported checkpoint kinases, Fanconi anaemia, and
homologous recombination as <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span 
class="cmti-12">WEE1</span>. These results show
that <a 
href="#glo:synthetic lethal">synthetic lethality</a> can be used to improve drug sensitivity as a combination
treatment, especially to exploit genomic instability and DNA repair, which are
known to be clinically applicable from previous results with <span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2</span>
genes and PARP inhibitors (<a 
href="#XLord2014">Lord <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XLord2014">2015</a>). Therefore, <span 
class="cmti-12">WEE1 </span>inhibitors are an
example of treatment which could be repurposed with the <a 
href="#glo:synthetic lethal">synthetic lethal</a> strategy.
Similar findings would be valuable to clinicians as a source of biomarkers and novel
treatments. While using a panel of cell lines to replicate findings across genetic
background is a promising approach to ensure wide clinical application of validated
<a 
href="#glo:synthetic lethal">synthetic lethal</a> partners, a computational approach may be more effective as it could
account for wider patient variation than scaling up intensive experiments on a
wide array of cell lines and could screen beyond limited candidates from an <a 
href="#glo:RNAi">RNAi</a>
library.
</p><!--l. 398--><p class="indent" >Chemical genetic screens are also a viable strategy to identify therapeutically
relevant <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions. <a 
href="#XBitler2015">Bitler <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XBitler2015">2015</a>) investigated <span 
class="cmti-12">ARID1A</span>
mutations, aberrations in chromatin remodelling known to be common in ovarian
cancers, for drug response. Ovarian RMG1 cells were screened for drug response
specific to <span 
class="cmti-12">ARID1A </span>knockdown cells. They used <span 
class="cmti-12">ARID1A </span>gene knockdown for
                                                                                    
                                                                                    
consistent genetic background, with control experiments and 3D cell culture to ensure
relevance to drug activity in the tumour micro-environment. Screening a panel of
commercially available drugs targeting epigenetic regulators found <span 
class="cmti-12">ESH2 </span>methyltransferase
inhibitors effective and specific against <span 
class="cmti-12">ARID1A </span>mutation with validation in a panel of
ovarian cell lines. <a 
href="#glo:synthetic lethal">Synthetic lethality</a> between <span 
class="cmti-12">ARID1A </span>and <span 
class="cmti-12">ESH2 </span>was supported by
decreases in H3K27me3 epigenetic marks and markers of apoptosis in response to
<span 
class="cmti-12">ESH2 </span>inhibitors. This was mechanistically supported with differential expression of
<span 
class="cmti-12">PIK3IP1 </span>and association of both <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes with the <span 
class="cmti-12">PIK3IP1 </span>promoter
identifying the PI3K-AKT signalling pathway as disrupted when both genes were
inhibited.
</p><!--l. 401--><p class="indent" >This successfully demonstrates the importance of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in epigenetic
regulators, identifies a therapeutically relevant <a 
href="#glo:synthetic lethal">synthetic lethal</a> interaction, and shows that
chemical genetic screens could model drug response and combination therapy in cancer cells.
However this approach is limited to finding <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions between genes with
known similar function, which may not be the most suitable for treatment. Further limiting
experiments to genes with existing targeted drugs reduces the number of <a 
href="#glo:synthetic lethal">synthetic lethal</a>
interactions detected, assumes the specificity of drugs to a particular target, and
many of these drugs are not yet clinically available yet anyway, although they are
clinical trials for other diseases or limited to access by patients from a particular
countries.
</p><!--l. 403--><p class="indent" >The examples above show that high-throughput screens are an effective approach to
discover <a 
href="#glo:synthetic lethal">synthetic lethality</a> in cancer with a wide range of applications. Screens
are more comprehensive than hypothesis-driven candidate gene approaches and
successfully find known and novel <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions with potential for rapid
clinical application. They have the power to test mode of action of drugs, find
unexpected <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions between pathways, or identify effective treatment
strategies without needing a clear mechanism. However, <a 
href="#glo:synthetic lethal">synthetic lethal</a> screens are
costly, labour-intensive, error-prone, and biased towards genes with effective <a 
href="#glo:RNAi">RNAi</a>
knockdown libraries. Limited genetic background, lethality to wild-type cell during
gene knockdown, off-target effects, and difficultly replicating <a 
href="#glo:synthetic lethal">synthetic lethality</a>
across different cell lines, tissues, laboratories, or conditions stems from a high false
positive rate and a lack of standardised thresholds to identify <a 
href="#glo:synthetic lethal">synthetic lethality</a> in a
high-throughput screen. Therefore there is a need for replication, validation, and
alternative approaches to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates. Varied conditions between
                                                                                    
                                                                                    
experimental screens and differences between <a 
href="#glo:RNAi">RNAi</a> and drug screens renders meta-analysis
ineffective.
</p><!--l. 405--><p class="indent" >Genome-scale <a 
href="#glo:synthetic lethal">synthetic lethal</a> experiments (across gene pairs) are not feasible, even in
model organisms, and they typically focus on specific gene candidates or the partners of a
gene of interest (such as importance in health). Therefore a computational approach is more
suitable for this task and may further augment experimental screening to replicate screen
candidates beyond experimental models.
<a 
 id="x1-41001r42"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.2.7 </span> <a 
 id="x1-420007"></a>Computational Prediction of Synthetic Lethality</h4>
<a 
 id="x1-42001r43"></a>
<h5 class="subsubsectionHead"><span class="titlemark">1.2.7.1 </span> <a 
 id="x1-430001"></a>Bioinformatics Approaches to Genetic Interactions</h5>
<!--l. 431--><p class="noindent" >Prediction of gene interaction networks is a feasible alternative to high-throughput screening
with biological importance and clinical relevance. There are many existing methods to
predict gene networks, as reviewed by <a 
href="#XvanSteen2011">van Steen</a>&#x00A0;(<a 
href="#XvanSteen2011">2012</a>) and <a 
href="#XBoucher2013">Boucher and Jenna</a>&#x00A0;(<a 
href="#XBoucher2013">2013</a>) and
summarised in Table&#x00A0;<a 
href="#x1-43001r1">1.1<!--tex4ht:ref: tab:methods_model --></a>. However, many of these methods have limitations including
the requirement for existing <a 
href="#glo:SGI">SGI</a> data, several data inputs, and reliability of gene
function annotation. Many of the existing methods also assume conservation of
individual interactions between species, which has been found not to hold in yeast
studies (<a 
href="#XDixon2008">Dixon <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XDixon2008">2008</a>). Tissue specificity is important in gene regulation
and gene expression, which are used as predictors of genetic interaction. However,
tissue specificity of genetic interactions cannot be explored in yeast studies and has
not been considered in many studies of multicellular model organisms, human
networks, or cancers. Similarly, investigation into tissue specificity of protein-protein
interactions (PPIs) , an important predictor of genetic interactions, is difficult given that
high-throughput two-hybrid screens occur out of cellular context for multicellular
organisms.
</p>
<div class="table">
                                                                                    
                                                                                    
<!--l. 433--><p class="indent" ><a 
 id="x1-43001r1"></a></p><hr class="float" /><div class="float" 
>
                                                                                    
                                                                                    
<div class="caption" 
><span class="id">Table&#x00A0;1.1: </span><span  
class="content">Methods for Predicting Genetic Interactions</span></div><!--tex4ht:label?: x1-43001r1 -->
<!--tex4ht:inline--><div class="tabular"><table id="TBL-1" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-1-1g"><col 
id="TBL-1-1" /><col 
id="TBL-1-2" /><col 
id="TBL-1-3" /><col 
id="TBL-1-4" /><col 
id="TBL-1-5" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-1-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-1-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"><span 
class="cmbx-12">Method</span></div>                                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-1-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"><span 
class="cmbx-12">Input Data</span></div>                               </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-1-3"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"><span 
class="cmbx-12">Species</span></div>                                                                              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-1-4"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"><span 
class="cmbx-12">Source</span></div>                                                                              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-1-5"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"><span 
class="cmbx-12">Tool Offered</span></div>
</td></tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-1-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-2-1"  
class="td11"> <span id="textcolor1">Between Pathways Model</span>                                                                                  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-2-2"  
class="td11"> PPI, <a 
href="#glo:SGI">SGI</a>                                               </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-2-3"  
class="td11"> <span id="textcolor2"><span 
class="cmti-12">S. cerevisiae</span></span>                                                                                            </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-2-4"  
class="td11"> <a 
href="#XKelley2005">Kelley and Ideker</a>&#x00A0;(<a 
href="#XKelley2005">2005</a>)                                                                                  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-2-5"  
class="td11"> &#x00A0;                    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-3-1"  
class="td11"> Within Pathways Model                                                                                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-3-2"  
class="td11"> PPI, <a 
href="#glo:SGI">SGI</a>                                               </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-3-3"  
class="td11"> <span 
class="cmti-12">S. cerevisiae                                                                                            </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-3-4"  
class="td11"> <a 
href="#XKelley2005">Kelley and Ideker</a>&#x00A0;(<a 
href="#XKelley2005">2005</a>)                                                                                  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-3-5"  
class="td11"> &#x00A0;                    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-4-1"  
class="td11"> <span id="textcolor3">Decision Tree</span>                                                                                                 </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-4-2"  
class="td11"> PPI, expression, phenotype                       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-4-3"  
class="td11"> <span id="textcolor4"><span 
class="cmti-12">S. cerevisiae</span></span>                                                                                            </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-4-4"  
class="td11"> <a 
href="#XWong2004">Wong <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XWong2004">2004</a>)                                                                                          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-4-5"  
class="td11"> 2 Hop              </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-5-1"  
class="td11"> Logistic Regression                                                                                          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-5-2"  
class="td11"> <a 
href="#glo:SGI">SGI</a>, PPI, co-expression, phenotype            </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-5-3"  
class="td11"> <span 
class="cmti-12">C. elegans                                                                                               </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-5-4"  
class="td11"> <a 
href="#XZhong2006">Zhong and Sternberg</a>&#x00A0;(<a 
href="#XZhong2006">2006</a>)                                                                              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-5-5"  
class="td11"> Gene Orienteer  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-6-1"  
class="td11"> <span id="textcolor5">Network Sampling</span>                                                                                           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-6-2"  
class="td11"> <a 
href="#glo:SGI">SGI</a>, PPI, GO                                        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-6-3"  
class="td11"> <span id="textcolor6"><span 
class="cmti-12">S. cerevisiae</span></span>                                                                                            </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-6-4"  
class="td11">  <!--tex4ht:inline--><div class="tabular"> <table id="TBL-2" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-2-1g"><col 
id="TBL-2-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-2-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-1-1"  
class="td00"><a 
href="#XLeMeur2008">Le&#x00A0;Meur and Gentleman</a>&#x00A0;(<a 
href="#XLeMeur2008">2008</a>)</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-2-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-2-1"  
class="td00"><a 
href="#XLeMeur2014">Le&#x00A0;Meur <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XLeMeur2014">2014</a>)</td>
</tr></table>                                                                                                                 </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-6-5"  
class="td11"> SLGI(R)          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-7-1"  
class="td11"> Random Walk                                                                                                </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-7-2"  
class="td11"> GO, PPI, expression                                </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-7-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-3" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-3-1g"><col 
id="TBL-3-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-3-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-1-1"  
class="td00"><span 
class="cmti-12">S. cerevisiae</span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-2-1"  
class="td00"><span 
class="cmti-12">C. elegans</span></td></tr></table>                                                                                       </div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-7-4"  
class="td11"> <a 
href="#XChipman2009">Chipman and Singh</a>&#x00A0;(<a 
href="#XChipman2009">2009</a>)                                                                               </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-7-5"  
class="td11"> &#x00A0;                    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-8-1"  
class="td11"> <span id="textcolor7">Shared Function</span>                                                                                              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-8-2"  
class="td11"> Co-expression, PPI, text mining, phylogeny  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-8-3"  
class="td11"> <span id="textcolor8"><span 
class="cmti-12">C. elegans</span></span>                                                                                               </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-8-4"  
class="td11"> <a 
href="#XLee2010b">Lee <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XLee2010b">2010b</a>)                                                                                           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-8-5"  
class="td11"> WormNet         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-9-1"  
class="td11"> Logistic Regression                                                                                          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-9-2"  
class="td11"> Co-expression, PPI, phenotype                  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-9-3"  
class="td11"> <span 
class="cmti-12">C. elegans                                                                                               </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-9-4"  
class="td11"> <a 
href="#XLee2010a">Lee <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XLee2010a">2010a</a>)                                                                                           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-9-5"  
class="td11"> GI Finder        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-10-1"  
class="td11"> <span id="textcolor9">Jaccard Index</span>                                                                                                 </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-10-2"  
class="td11"> GO, <a 
href="#glo:SGI">SGI</a>, PPI, phenotype                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-10-3"  
class="td11"> Eukarya                                                                                                        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-10-4"  
class="td11"> <a 
href="#XHoehndorf2013">Hoehndorf <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XHoehndorf2013">2013</a>)                                                                                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-10-5"  
class="td11"> &#x00A0;                    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-11-1"  
class="td11"> <span id="textcolor10">Machine Learning</span>                                                                                            </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-11-2"  
class="td11"> &#x00A0;                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-11-3"  
class="td11"> &#x00A0;                                                                                                                  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-11-4"  
class="td11"> <a 
href="#XPandey2010">Pandey <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XPandey2010">2010</a>)                                                                                        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-11-5"  
class="td11"> MNMC            </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-12-1"  
class="td11"> Machine Learning Meta-Analysis                                                                        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-12-2"  
class="td11"> &#x00A0;                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-12-3"  
class="td11"> &#x00A0;                                                                                                                  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-12-4"  
class="td11"> <a 
href="#XWu2014">Wu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XWu2014">2014</a>)                                                                                             </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-12-5"  
class="td11"> MetaSL           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-13-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-4" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-4-1g"><col 
id="TBL-4-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-4-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-1-1"  
class="td00"><span id="textcolor11">Flux Variability Analysis</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-2-1"  
class="td00"><span id="textcolor12">Flux Balance Analysis</span>     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-3-1"  
class="td00"><span id="textcolor13">Network Simulation</span>        </td></tr></table>                                                                            </div>   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-13-2"  
class="td11"> Metabolism                                           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-13-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-5" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-5-1g"><col 
id="TBL-5-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-5-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-1-1"  
class="td00"><span id="textcolor14"><span 
class="cmti-12">E. coli</span></span>          </td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-2-1"  
class="td00"><span id="textcolor15"><span 
class="cmti-12">M. pneumoniae</span></span></td></tr></table>                                                                                   </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-13-4"  
class="td11"> <a 
href="#XGuell2014">G&#x00FC;ell <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XGuell2014">2014</a>)                                                                                          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-13-5"  
class="td11"> &#x00A0;                    </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-1-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-14-1"  
class="td11">                                                                          </td></tr></table>                                                                                                                                                                                                                                                      </div>
                                                                                    
                                                                                    
</div><hr class="endfloat" />
</div>
                                                                                    
                                                                                    
<!--l. 597--><p class="indent" ><a 
 id="x1-43002r2"></a></p><hr class="float" /><div class="float" 
>
                                                                                    
                                                                                    
<div class="caption" 
><span class="id">Table&#x00A0;1.2: </span><span  
class="content">Methods for Predicting Synthetic Lethality in Cancer</span></div><!--tex4ht:label?: x1-43002r2 -->
<!--tex4ht:inline--><div class="tabular"><table id="TBL-6" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-6-1g"><col 
id="TBL-6-1" /><col 
id="TBL-6-2" /><col 
id="TBL-6-3" /><col 
id="TBL-6-4" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-6-1-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-6-1-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left; vertical-align:middle;"><span 
class="cmbx-12">Method</span></div>                                                                               </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-6-1-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left; vertical-align:middle;"><span id="textcolor16"><span 
class="cmbx-12">Input Data</span></span></div>                                                                          </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-6-1-3"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left; vertical-align:middle;"><span 
class="cmbx-12">Source</span></div>                                                                                </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-1-4"  
class="td11"> <span 
class="cmbx-12">Tool Offered</span>                               </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-6-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-2-1"  
class="td11"> <span id="textcolor17">Network Centrality</span>                                                                                          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-2-2"  
class="td11"> protein-protein interactions                                                                                </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-2-3"  
class="td11"> <a 
href="#XKranthi2013">Kranthi <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XKranthi2013">2013</a>)                                                                                      </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-2-4"  
class="td11"> &#x00A0;                                                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-3-1"  
class="td11"> Differential Expression                                                                                     </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-3-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-7" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-7-1g"><col 
id="TBL-7-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-7-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-7-1-1"  
class="td00">Expression</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-7-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-7-2-1"  
class="td00">Mutation</td></tr></table>                                                                                                </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-3-3"  
class="td11"> <a 
href="#XWang2013">Wang and Simon</a>&#x00A0;(<a 
href="#XWang2013">2013</a>)                                                                                  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-3-4"  
class="td11"> &#x00A0;                                                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-4-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-8" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-8-1g"><col 
id="TBL-8-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-8-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-8-1-1"  
class="td00"><span id="textcolor18">Comparative <a 
href="#glo:genomics">Genomics</a></span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-8-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-8-2-1"  
class="td00"><span id="textcolor19">Chemical-<a 
href="#glo:genomics">Genomics</a></span></td></tr></table>                                                                              </div>   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-4-2"  
class="td11">  <!--tex4ht:inline--><div class="tabular"> <table id="TBL-9" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-9-1g"><col 
id="TBL-9-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-9-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-9-1-1"  
class="td00">Yeast synthetic gene interactions</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-9-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-9-2-1"  
class="td00">Homology</td></tr></table>                                                                  </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-4-3"  
class="td11"> <a 
href="#XHeiskanen2012">Heiskanen and Aittokallio</a>&#x00A0;(<a 
href="#XHeiskanen2012">2012</a>)                                                                      </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-4-4"  
class="td11"> &#x00A0;                                                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-5-1"  
class="td11"> Comparative <a 
href="#glo:genomics">Genomics</a>                                                                                     </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-5-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-10" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-10-1g"><col 
id="TBL-10-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-10-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-1-1"  
class="td00">Yeast synthetic gene interactions</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-10-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-2-1"  
class="td00">Homology</td></tr></table>                                                                   </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-5-3"  
class="td11"> <a 
href="#XDeshpande2013">Deshpande <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XDeshpande2013">2013</a>)                                                                                  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-5-4"  
class="td11"> &#x00A0;                                                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-6-1"  
class="td11"> Machine Learning                                                                                            </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-6-2"  
class="td11"> &#x00A0;                                                                                                                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-6-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-11" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-11-1g"><col 
id="TBL-11-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-11-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-11-1-1"  
class="td00">Discussed by <a 
href="#XBabyak2004">Babyak</a>&#x00A0;(<a 
href="#XBabyak2004">2004</a>)</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-11-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-11-2-1"  
class="td00">and <a 
href="#XLee2009">Lee and Marcotte</a>&#x00A0;(<a 
href="#XLee2009">2009</a>)</td>
</tr></table>                                                                                                                 </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-6-4"  
class="td11"> &#x00A0;                                                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-7-1"  
class="td11"> <span id="textcolor20">Differential Expression</span>                                                                                     </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-7-2"  
class="td11"> Expression                                                                                                      </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-7-3"  
class="td11"> <a 
href="#XTiong2014">Tiong <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XTiong2014">2014</a>)                                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-7-4"  
class="td11"> &#x00A0;                                                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-8-1"  
class="td11"> Literature Database                                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-8-2"  
class="td11"> &#x00A0;                                                                                                                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-8-3"  
class="td11"> <a 
href="#XLi2014">Li <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XLi2014">2014</a>)                                                                                              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-8-4"  
class="td11"> Syn-Lethality                                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-9-1"  
class="td11"> <span id="textcolor21">Meta-Analysis</span>                                                                                                </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-9-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-12" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-12-1g"><col 
id="TBL-12-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-12-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-12-1-1"  
class="td00">Meta-Analysis    </td></tr><tr  
 style="vertical-align:baseline;" id="TBL-12-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-12-2-1"  
class="td00">Machine Learning</td></tr></table>                                                                                        </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-9-3"  
class="td11"> <a 
href="#XWu2014">Wu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XWu2014">2014</a>)                                                                                            </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-9-4"  
class="td11"> MetaSL                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-10-1"  
class="td11"> Pathway Analysis                                                                                            </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-10-2"  
class="td11"> &#x00A0;                                                                                                                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-10-3"  
class="td11"> <a 
href="#XZhang2015">Zhang <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XZhang2015">2015</a>)                                                                                        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-10-4"  
class="td11"> &#x00A0;                                                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-11-1"  
class="td11"> <span id="textcolor22">Protein Domains</span>                                                                                             </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-11-2"  
class="td11"> Homology                                                                                                       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-11-3"  
class="td11"> <a 
href="#XKozlov2015">Kozlov <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XKozlov2015">2015</a>)                                                                                       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-11-4"  
class="td11"> &#x00A0;                                                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-12-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-13" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-13-1g"><col 
id="TBL-13-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-13-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-13-1-1"  
class="td00">Data-Mining      </td></tr><tr  
 style="vertical-align:baseline;" id="TBL-13-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-13-2-1"  
class="td00">Machine Learning</td></tr></table>                                                                                       </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-12-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-14" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-14-1g"><col 
id="TBL-14-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-14-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-14-1-1"  
class="td00">Expression                              </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-14-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-14-2-1"  
class="td00">Somatic mutation and DNA CNV</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-14-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-14-3-1"  
class="td00">siRNA in cell lines</td></tr></table>                                                                   </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-12-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-15" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-15-1g"><col 
id="TBL-15-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-15-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-15-1-1"  
class="td00"><a 
href="#XJerby2014">Jerby-Arnon <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XJerby2014">2014</a>)</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-15-2-1"  
class="td00"><a 
href="#XRyan2014">Ryan <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XRyan2014">2014</a>)           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-15-3-1"  
class="td00"><a 
href="#XCrunkhorn2014">Crunkhorn</a>&#x00A0;(<a 
href="#XCrunkhorn2014">2014</a>)           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-15-4-1"  
class="td00"><a 
href="#XLokody2014">Lokody</a>&#x00A0;(<a 
href="#XLokody2014">2014</a>)                </td></tr></table>                                                                          </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-12-4"  
class="td11"> <a 
href="#glo:DAISY">DAISY</a> (method)                                </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-13-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-16" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-16-1g"><col 
id="TBL-16-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-16-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-16-1-1"  
class="td00"><span id="textcolor23">Genome Evolution</span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-16-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-16-2-1"  
class="td00"><span id="textcolor24">Hypothesis Test</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-16-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-16-3-1"  
class="td00"><span id="textcolor25">Machine Learning</span>  </td></tr></table>                                                                                    </div>   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-13-2"  
class="td11">  <!--tex4ht:inline--><div class="tabular"> <table id="TBL-17" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-17-1g"><col 
id="TBL-17-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-17-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-17-1-1"  
class="td00">Expression</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-17-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-17-2-1"  
class="td00">DNA CNV </td></tr><tr  
 style="vertical-align:baseline;" id="TBL-17-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-17-3-1"  
class="td00">Known SL</td></tr></table>                                                                                              </div>   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-13-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-18" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-18-1g"><col 
id="TBL-18-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-18-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-18-1-1"  
class="td00"><a 
href="#XLu2013">Lu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XLu2013">2013</a>)</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-18-2-1"  
class="td00"><a 
href="#XLu2015">Lu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XLu2015">2015</a>) </td></tr></table>                                                                                       </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-13-4"  
class="td11"> &#x00A0;                                                     </td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-14-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-19" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-19-1g"><col 
id="TBL-19-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-19-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-19-1-1"  
class="td00"><span id="textcolor26">Bimodality</span></td></tr></table>                                                                                               </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-14-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-20" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-20-1g"><col 
id="TBL-20-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-20-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-1-1"  
class="td00">Expression         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-20-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-2-1"  
class="td00">DNA CNV         </td></tr><tr  
 style="vertical-align:baseline;" id="TBL-20-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-3-1"  
class="td00">Somatic Mutation</td></tr></table>                                                                                        </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-14-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-21" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-21-1g"><col 
id="TBL-21-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-21-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-21-1-1"  
class="td00"><a 
href="#XWappett2014">Wappett</a>&#x00A0;(<a 
href="#XWappett2014">2014</a>)       </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-21-2-1"  
class="td00"><a 
href="#XWappett2016">Wappett <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XWappett2016">2016</a>)</td></tr></table>                                                                                </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-14-4"  
class="td11"> <a 
href="#glo:BiSEp">BImodal Subsetting ExPression (BiSEp)</a>  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-15-1"  
class="td11"> Directional Chi-Square                                                                                     </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-15-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-22" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-22-1g"><col 
id="TBL-22-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-22-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-22-1-1"  
class="td00">Expression (microarray)</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-22-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-22-2-1"  
class="td00">Somatic mutation</td></tr></table>                                                                               </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-15-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-23" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-23-1g"><col 
id="TBL-23-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-23-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-1-1"  
class="td00">Kelly, S. T., Guilford, P. J., and Black, M. A.</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-23-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-2-1"  
class="td00">Dissertation (Kelly, 2013) and developed here </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-23-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-3-1"  
class="td00">                                       </td></tr></table>                               </div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-6-15-4"  
class="td11"> SLIPT                                              </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-6-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-16-1"  
class="td11">                                                                          </td></tr></table>                                                                                                                                                                                                                               </div>
                                                                                    
                                                                                    
</div><hr class="endfloat" />
<!--l. 795--><p class="indent" >There are a number of existing computational methods for predicting <a 
href="#glo:synthetic lethal">synthetic lethal</a>
gene pairs in humans with a specfic interest in cancer (as summarised in&#x00A0;<a 
href="#x1-43002r2">1.2<!--tex4ht:ref: tab:methods_SL --></a>). While these
demonstrate the power and need for predictions of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in human and cancer
contexts, limitations of previous methods could be met with a different approach. Existing
computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> prediction are often difficult to interpret,
replicate for new genes, or are reliant on data types not available for a wider range of genes
to test.
<a 
 id="x1-43003r45"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.2.7.2 </span> <a 
 id="x1-440002"></a>Comparative Genomics</h5>
<!--l. 799--><p class="noindent" >A comparative <a 
href="#glo:genomics">genomics</a> approach by <a 
href="#XDeshpande2013">Deshpande <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XDeshpande2013">2013</a>) used the results of well
characterised high-throughput mutation screens in <span 
class="cmti-12">S. cerevisiae </span>as candidates for <a 
href="#glo:synthetic lethal">synthetic
lethality</a> in humans (<a 
href="#XBaryshnikova2010a">Baryshnikova <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBaryshnikova2010a">2010a</a>;&#x00A0;<a 
href="#XCostanzo2010">Costanzo <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XCostanzo2010">2010</a>,&#x00A0;<a 
href="#XCostanzo2011">2011</a>;&#x00A0;<a 
href="#XTong2001">Tong
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTong2001">2001</a>,&#x00A0;<a 
href="#XTong2004">2004</a>). Yeast <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners were compared to human orthologues to
find cancer relevant <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate pairs with direct therapeutic potential.
Proposed as a complementary approach to siRNA screens, approximately 24,000 of the
116,000 negative <a 
href="#glo:SGI">SGIs</a> in yeast (<a 
href="#XCostanzo2011">Costanzo <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XCostanzo2011">2011</a>) were matched to human
orthologues, with over 500 involving a cancer gene (<a 
href="#XFutreal2004">Futreal <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XFutreal2004">2004</a>). Under strict
criteria of one-to-one orthologues, large effect size and significant interaction in yeast
data (<span 
class="cmmi-12">&#x03F5; &#x003C; </span><span 
class="cmsy-10x-x-120">&#x2212;</span>0<span 
class="cmmi-12">.</span>2, <span 
class="cmmi-12">p &#x003C; </span>0<span 
class="cmmi-12">.</span>05), 1,522 interactions were identified with 70 involving
cancer genes. Of the 21 gene interactions tested with pairs of siRNA in IMR1
fibroblast cells, 6 exhibited <a 
href="#glo:synthetic lethal">synthetic lethal</a> effects. The two strongest interactions
(<span 
class="cmti-12">SMARCB1 </span>with <span 
class="cmti-12">PSMA4 </span>and <span 
class="cmti-12">ASPSCR1 </span>with <span 
class="cmti-12">PSMC2</span>) were successfully validated
by protein analysis of human cells and replication with tetrad analysis for yeast
orthologues.
</p><!--l. 801--><p class="indent" >Another approach to systematic <a 
href="#glo:synthetic lethal">synthetic lethality</a> discovery specific to human cancer (in
contrast to the plethora of yeast <a 
href="#glo:synthetic lethal">synthetic lethality</a> data) was to build a database as done by
<a 
href="#XLi2014">Li <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XLi2014">2014</a>). In their relational database, called &#x201C;Syn-lethality&#x201D;, they have curated both
known experimentally discovered <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs in humans (113 pairs) from
the literature and those predicted from <a 
href="#glo:synthetic lethal">synthetic lethality</a> between orthologous
genes in <span 
class="cmti-12">S. cerevisiae </span>yeast (1114 pairs). This knowledge-based database is the first
dedicated to human cancer <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions and integrates gene function
annotation, pathway and molecular mechanism data with experimental and predicted
<a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs. This combination of data sources is intended to tackle
                                                                                    
                                                                                    
the trade-off between more conclusive <a 
href="#glo:synthetic lethal">synthetic lethal</a> experiments in yeast and
more clinically relevant <a 
href="#glo:synthetic lethal">synthetic lethal</a> experiments in human cancer models, such
as <a 
href="#glo:RNAi">RNAi</a>, especially when high-throughput screens are costly and prone to false
positives in either system and difficult to replicate across gene backgrounds. This
database centralises a wealth of knowledge scattered in the literature including cancer
relevant genes (<span 
class="cmti-12">BRCA1</span>, <span 
class="cmti-12">BRCA2</span>, <span 
class="cmti-12">PARP1</span>, <span 
class="cmti-12">PTEN</span>, <span 
class="cmti-12">VHL</span>, <span 
class="cmti-12">MYC</span>, <span 
class="cmti-12">EGFR</span>, <span 
class="cmti-12">MSH2</span>,
<span 
class="cmti-12">KRAS</span>, and <span 
class="cmti-12">TP53</span>) and is publicly available as a Java App. These included the
previously mentioned interactions of <span 
class="cmti-12">BRCA1 </span>and <span 
class="cmti-12">BRCA2 </span>with <span 
class="cmti-12">PARP1 </span>and <span 
class="cmti-12">TP53 </span>with
<span 
class="cmti-12">WEE1 </span>and <span 
class="cmti-12">PLK1</span>. However, the computational methodology was not released, so
it is not possible to replicate their results, nor to add to the findings with new
datasets, which are limited to 647 human genes. Suggested future directions were
promising, such as constructing networks of known <a 
href="#glo:synthetic lethal">synthetic lethality</a>, applying known
<a 
href="#glo:synthetic lethal">synthetic lethality</a> to cancer treatment, data mining, replicating the approach for
<a 
href="#glo:synthetic lethal">synthetic lethality</a> in model organisms, signalling pathways, and developing a complete
global network in human cancer or yeast (both of which are still incomplete with
experimental data), some of which has been implemented in &#x201C;SynLethDB&#x201D; (<a 
href="#XGuo2016">Guo
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XGuo2016">2016</a>).
                                                                                    
                                                                                    
</p><!--l. 804--><p class="indent" ><a 
 id="x1-44001r3"></a></p><hr class="float" /><div class="float" 
>
                                                                                    
                                                                                    
<div class="caption" 
><span class="id">Table&#x00A0;1.3: </span><span  
class="content">Machine Learning Methods used by <a 
href="#XWu2014">Wu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XWu2014">2014</a>)</span></div><!--tex4ht:label?: x1-44001r3 -->
<!--tex4ht:inline--><div class="tabular"><table id="TBL-24" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-24-1g"><col 
id="TBL-24-1" /><col 
id="TBL-24-2" /><col 
id="TBL-24-3" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-24-1-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-24-1-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left; vertical-align:middle;"><span 
class="cmbx-12">Method</span></div>                                                                              </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-24-1-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left; vertical-align:middle;"><span 
class="cmbx-12">Source</span></div>                     </td><td  style="white-space:nowrap; text-align:left;" id="TBL-24-1-3"  
class="td11"> <span 
class="cmbx-12">Tool Offered</span> </td></tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-24-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-24-2-1"  
class="td
11"> <span id="textcolor27">Random Forest</span>                                                                                              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-24-2-2"  
class="td11"> <a 
href="#XBreiman2001">Breiman</a>&#x00A0;(<a 
href="#XBreiman2001">2001</a>)              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-24-2-3"  
class="td11"> &#x00A0;                   </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-24-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-24-3-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-25" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-25-1g"><col 
id="TBL-25-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-25-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-25-1-1"  
class="td00">Random Forest                </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-25-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-25-2-1"  
class="td00">J48 (decision tree)             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-25-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-25-3-1"  
class="td00">Bayes (Log Regression)       </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-25-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-25-4-1"  
class="td00">Bayes (Network)               </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-25-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-25-5-1"  
class="td00">PART (Rule-based)           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-25-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-25-6-1"  
class="td00">RBF Network                   </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-25-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-25-7-1"  
class="td00">Bagging / Bootstrap          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-25-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-25-8-1"  
class="td00">Classification via Regression</td></tr></table>                                                                        </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-24-3-2"  
class="td11"> <a 
href="#XHall2009">Hall <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XHall2009">2009</a>) &#x00A0;         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-24-3-3"  
class="td11"> WEKA           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-24-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-24-4-1"  
class="td11"> <span id="textcolor28">Support Vector Machine (Linear)</span>                                                                      </td><td  style="white-space:nowrap; text-align:left;" id="TBL-24-4-2"  
class="td11"> <a 
href="#XVapnik1995">Vapnik</a>&#x00A0;(<a 
href="#XVapnik1995">1995</a>)               </td><td  style="white-space:nowrap; text-align:left;" id="TBL-24-4-3"  
class="td11"> &#x00A0;                   </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-24-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-24-5-1"  
class="td11"> Support Vector Machine (RBF &#x2013; Gaussian)                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-24-5-2"  
class="td11"> <a 
href="#XJoachims1999">Joachims</a>&#x00A0;(<a 
href="#XJoachims1999">1999</a>)             </td><td  style="white-space:nowrap; text-align:left;" id="TBL-24-5-3"  
class="td11"> &#x00A0;                   </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-24-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-24-6-1"  
class="td11"> <span id="textcolor29">Multi-Network Multi-Class (MNMC)</span>                                                                  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-24-6-2"  
class="td11"> <a 
href="#XPandey2010">Pandey <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XPandey2010">2010</a>)       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-24-6-3"  
class="td11"> &#x00A0;                   </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-24-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-24-7-1"  
class="td11"> MetaSL (Meta-Analysis)                                                                                  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-24-7-2"  
class="td11"> <a 
href="#XWu2014">Wu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XWu2014">2014</a>)            </td><td  style="white-space:nowrap; text-align:left;" id="TBL-24-7-3"  
class="td11"> MetaSL          </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-24-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-24-8-1"  
class="td11">                                                                         </td></tr></table>                                                </div>
                                                                                    
                                                                                    
</div><hr class="endfloat" />
<!--l. 862--><p class="indent" >Machine learning approaches have also been proposed for <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery
(<a 
href="#XBabyak2004">Babyak</a>,&#x00A0;<a 
href="#XBabyak2004">2004</a>;&#x00A0;<a 
href="#XLee2009">Lee and Marcotte</a>,&#x00A0;<a 
href="#XLee2009">2009</a>). Due to concerns that these may be
subject to overfitting or noise, <a 
href="#XWu2014">Wu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XWu2014">2014</a>) developed a meta-analysis method
(based on the machine learning methods in Table&#x00A0;<a 
href="#x1-44001r3">1.3<!--tex4ht:ref: tab:methods_meta --></a>) for <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs
relevant to developing selective drugs against human cancer, building upon their
previous database (<a 
href="#XLi2014">Li <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XLi2014">2014</a>). They used training data of 10,885 <a 
href="#glo:synthetic lethal">synthetic
lethal</a> interactions from yeast experiments of which 7347 occurred between the
5,504 non-essential genes. Their &#x201C;metaSL&#x201D; approach utilises genomic, proteomic
and annotation data (including GO terms <a 
href="#XAshburner2000">Ashburner <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XAshburner2000">2000</a>), PPI, protein
complexes, and biological pathway) with strong statistical performance in yeast data
(<a 
href="#glo:AUROC">AUROC</a> of 0.871). The predicted orthologous <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in human
data were not experimentally validated but several would be relevant to cancer
such as <span 
class="cmti-12">EGFR </span>with <span 
class="cmti-12">PRKCZ</span>. They note that computational approaches scale-up
across the genome at lower cost than experimental screen and share their most
supported interactions online. However, the method is not available for analysis of
other genes studied by the cancer research community. While machine learning has
great potential as a predictor, the results vary greatly depending on the predictive
features selected and it is not clear which threshold should be used to report reliably
detected genes. Syn-Lethality (<a 
href="#XLi2014">Li <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XLi2014">2014</a>) and MetaSL (<a 
href="#XWu2014">Wu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XWu2014">2014</a>)
demonstrate the value of computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethality</a> but omit
many genes of importance in cancer, such as <span 
class="cmti-12">CDH1</span>, and there remains a need to
enable biological researchers to query such genes in a particular tissue or genetic
background.
</p><!--l. 864--><p class="indent" >There is also concern for analyses based on yeast data that many <a 
href="#glo:synthetic lethal">synthetic lethal</a>
interactions may not be conserved between species <a 
href="#XDixon2009a">Dixon <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XDixon2009a">2009</a>), although interactions
between pathways may be more comparable. It is unsurprising that many of the interactions
identified were not experimentally validated. There have been many gene duplications in the
separate evolutionary histories of humans and yeast which may lead to differences in
genetic redundancy. Yeast are not an ideal human cancer model because they are
do not have tissue specificity, multicellular gene regulation, or orthologues to a
number of known cancer genes such as p53. Although these studies have tried to
anticipate these issues with stringent criteria such as requiring one-to-one orthologues,
there remains the possibility that changes in gene function may affect whether
these are solely redundant such as if functions had coevolved without sequence
                                                                                    
                                                                                    
homology. Many genes will also be excluded since they lack homologues in yeast, the
corresponding experimental data, or having paralogues in either species. Thus
conservation of yeast interactions is not an ideal strategy and analysis of human data
directly for comparison with human experimental data will be the focus of this
thesis.
<a 
 id="x1-44002r48"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.2.7.3 </span> <a 
 id="x1-450003"></a>Analysis and Modelling of Protein Data</h5>
<!--l. 868--><p class="noindent" ><a 
href="#XKranthi2013">Kranthi <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XKranthi2013">2013</a>) took a network approach to discovery of <a 
href="#glo:synthetic lethal">synthetic lethal</a>
candidate selection applying the concept to &#x201C;centrality&#x201D; to a human PPI network
involving interacting partners of known cancer genes. The effect of removing pairs of
genes on connectivity of the network was used as a surrogate for viability which is
supported by observations that the PPI and <a 
href="#glo:synthetic lethal">synthetic lethal</a> networks are orthogonal in
<span 
class="cmti-12">S. cerevisiae </span>studies (<a 
href="#XTong2004">Tong <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTong2004">2004</a>). They showed that the human cancer
protein interaction network (of 1539 proteins and 6471 interactions) exhibits the
power law distribution expected of a scale-free <a 
href="#glo:synthetic lethal">synthetic lethal</a> network with high
connectivity (average vertex degree of 23.67 and network efficiency of 0.2952). Their top
100 candidate interactions included interactions of the tumour suppressor <span 
class="cmti-12">TP53</span>
with <span 
class="cmti-12">BRCA1</span>, <span 
class="cmti-12">CDKNA1</span>, <span 
class="cmti-12">CDKNA2</span>, <span 
class="cmti-12">MET</span>, and <span 
class="cmti-12">RB1 </span>which have been detected by
prior studies. The gene pairs were often observed to be in the same or a plausible
compensatory pathway. Thus the network structure is important in the biological
functions of cancers and could be exploited for targeting <span 
class="cmti-12">TP53 </span>loss of function
mutations.
</p><!--l. 870--><p class="indent" >However, their approach was limited to known cancer genes and is not applicable to
genes that do not have PPI data. Other nucleotide sequencing data types are more
commonly available for cancer studies at a genomic scale. Of further concern is that
the results were enriched for p53 <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners which is relevant to
many cancer researchers but this genome-wide approach did not detect many other
cancer genes due to multiple testing. This enrichment may be due to the known
drastic effect of removing p53 itself from the network as a master regulator, cancer
driving tumour suppressor gene, and highly connected network &#x201C;hub&#x201D;. The focus on
cancer genes is useful for translation into therapeutics but does not account for
variable genetic backgrounds or effect of protein removal on the whole cellular
network.
                                                                                    
                                                                                    
</p><!--l. 872--><p class="indent" >Focusing on the potential for <a 
href="#glo:synthetic lethal">synthetic lethality</a> to be an effective anti-cancer drug target,
<a 
href="#XZhang2015">Zhang <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XZhang2015">2015</a>) used modelling of signalling pathways to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a>
interactions between known drug targets and cancer genes by simulating gene knockdowns.
A computational approach was applied to avoid the limitations of experimental <a 
href="#glo:RNAi">RNAi</a>
screens such as scale, instability of knockdown, and off-target effects. This &#x2018;hybrid&#x2019; method of
a data-driven model and known signalling pathways showed potential as a means to predict
cell death in single and combination gene knockouts. They used time series protein
phosphorylation data (<a 
href="#XLee2012">Lee <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XLee2012">2012</a>) for 28 signalling proteins and <a 
href="#glo:GO">Gene Ontology (GO)</a>
(GO) pathways <a 
href="#XAshburner2000">Ashburner <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XAshburner2000">2000</a>);&#x00A0;<a 
href="#XBlake2015">Blake <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XBlake2015">2015</a>). This approach successfully
detected many known essential genes in the human gene essentiality database, known
<a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in the Syn-Lethality database (<a 
href="#XLi2014">Li <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XLi2014">2014</a>), and predicted novel
<a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs. The strongest essential genes in single knockdowns were
<span 
class="cmti-12">AKT</span>, <span 
class="cmti-12">TP53</span>, <span 
class="cmti-12">CHK1</span>, <span 
class="cmti-12">S6K1</span>, and <span 
class="cmti-12">CYCLIND1</span>. Pairwise knockdowns identified 252
candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions including <span 
class="cmti-12">TP53 </span>with <span 
class="cmti-12">CHK1</span>, <span 
class="cmti-12">S6K1</span>, <span 
class="cmti-12">WEE1</span>,
<span 
class="cmti-12">CYCLIND1</span>, and <span 
class="cmti-12">CASP9</span>; <span 
class="cmti-12">AKT </span>with <span 
class="cmti-12">WEE1</span>; and <span 
class="cmti-12">CDK1 </span>with <span 
class="cmti-12">CYCLIND1</span>. These novel
results contained many <span 
class="cmti-12">TP53 </span>and AKT <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners, genes known to
be important in many cancers, however these also have a severe impact on the
signalling pathways in their essentiality analysis of single gene disruptions and large
phenotypic changes in cancer. This approach is amenable to detect functionally
related pathways and protein complexes across the molecular function, cellular
component, and biological process annotations provided by GO. The results were
consistent with the experimental results in the literature but the novel <a 
href="#glo:synthetic lethal">synthetic
lethal</a> interactions have yet to be validated. While the mathematical reasoning and
algorithms are given, the code was not released to replicate the findings or apply the
methodology beyond the signalling pathways analysed by <a 
href="#XZhang2015">Zhang <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XZhang2015">2015</a>).
While this is an interesting approach, the analysis of this thesis will focus on gene
expression and <a 
href="#glo:RNAi">RNAi</a> data which is available to test a wider range of candidate gene
pairs.
<a 
 id="x1-45001r50"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.2.7.4 </span> <a 
 id="x1-460004"></a>Differential Gene Expression</h5>
<!--l. 878--><p class="noindent" >Differential gene expression has been explored to predict <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs in cancer
which would be widely applicable due to the availability of public gene expression data
for a large number of samples and cancer types. <a 
href="#XWang2013">Wang and Simon</a>&#x00A0;(<a 
href="#XWang2013">2013</a>) found
differentially expressed genes (by the t-test, adjusted by FDR) between tumours with or
                                                                                    
                                                                                    
without functional p53 mutations in <a 
href="#glo:TCGA">TCGA</a> (<a 
href="#XTCGA2008GBM">McLendon <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2008GBM">2008</a>) and <a 
href="#glo:CCLE">Cancer Cell
Line Encyclopaedia (CCLE)</a> (<a 
href="#XBarretina2012">Barretina <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBarretina2012">2012</a>) <a 
href="#glo:RNA-Seq">RNA-Seq</a> gene expression
data as candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner pathways of p53. They identified 2, 8,
and 21 candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes in 3 microarray datasets from
the NCI60 cell lines, 31 partner genes from the <a 
href="#glo:CCLE">CCLE</a> <a 
href="#glo:RNA-Seq">RNA-Seq</a> data, and 50 in
<a 
href="#glo:TCGA">TCGA</a> <a 
href="#glo:RNA-Seq">RNA-Seq</a> data. <span 
class="cmti-12">PLK1 </span>was replicated across 4 of these analyses and 17
other genes were replicated across 2 analyses (including <span 
class="cmti-12">MTOR</span>, <span 
class="cmti-12">PLK4</span>, <span 
class="cmti-12">MAST2</span>,
<span 
class="cmti-12">MAP3K4</span>, <span 
class="cmti-12">AURKA</span>, <span 
class="cmti-12">BUB1 </span>and 6 CDK genes) with many playing a role in cell cycle
regulation. This was supported by a drug sensitivity experiment on the NCI60 cell lines
which found that cells which lacked functional p53 were more sensitive to paclitaxel
(which targets <span 
class="cmti-12">PLK1</span>, <span 
class="cmti-12">AURKA</span>, and <span 
class="cmti-12">BUB1</span>). This demonstrated the potential of
gene expression as a surrogate for gene function and use of public genomic data to
predict <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs in cancer. <a 
href="#XWang2013">Wang and Simon</a>&#x00A0;(<a 
href="#XWang2013">2013</a>) advocated for
pre-screening of expression profiles to augment future <a 
href="#glo:RNAi">RNAi</a> screens. However, the
analyses were limited to kinase genes and focused on currently druggable genes,
lacking wider application of <a 
href="#glo:synthetic lethal">synthetic lethal</a> prediction methodology. This approach
may not be feasible or applicable in cancer genes with a lower mutation rate than
p53.
</p><!--l. 880--><p class="indent" ><a 
href="#XTiong2014">Tiong <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XTiong2014">2014</a>) also investigated gene expression as a predictor of synthetic
lescale-freethal pairs with colorectal cancer microarrays from a Han Chinese population with
a sample size of 70 tumour and 12 normal tissue samples. Simultaneously differentially
expression of &#x201C;tumour dependent&#x201D; gene pairs (which includes co-expression) between cancer
and normal tissue was used to rank 663 candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions identified in
cell line siRNA experiments. Of the top 20 genes, 17 were tested for testing differential
expression at the protein level with immunohistochemistry staining and correlation with
clinical characteristics, with 11 pairs exhibiting synergistic effects. Some of the
predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs were consistent with the literature (including <span 
class="cmti-12">TP53</span>
with <span 
class="cmti-12">S6K1 </span>and partners of <span 
class="cmti-12">KRAS</span>, <span 
class="cmti-12">PTEN</span>, <span 
class="cmti-12">BRCA1</span>, and <span 
class="cmti-12">BRCA2</span>) and two novel
<a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions (<span 
class="cmti-12">TP53 </span>with <span 
class="cmti-12">CSNK1E </span>and <span 
class="cmti-12">CTNNB1</span>) were validated in
pre-clinical models. This serves a valuable proof-of-concept for integration of <span 
class="cmti-12">in silico</span>
approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery in cancer demonstrating it&#x2019;s utility to
triage and identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of p53 applicable to colorectal tissues.
Although the experimental work was the focus of the paper, these findings show
that <a 
href="#glo:bioinformatics">bioinformatics</a> <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates can be validated in patient tissue
                                                                                    
                                                                                    
samples (from a non-caucasian population) to find those applicable to colorectal
cancers.
<a 
 id="x1-46001r51"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.2.7.5 </span> <a 
 id="x1-470005"></a>Data Mining and Machine Learning</h5>
<!--l. 886--><p class="noindent" >Recognising the utility of <a 
href="#glo:synthetic lethal">synthetic lethality</a> to drug inhibition and specificity of anti-cancer
treatments, <a 
href="#XJerby2014">Jerby-Arnon <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XJerby2014">2014</a>) also saw the need for effective prediction of gene
essentiality and <a 
href="#glo:synthetic lethal">synthetic lethality</a> to augment experimental studies of SL. They developed
the <a 
href="#glo:DAISY">DAta mIning SYnthetic lethal identification pipeline (DAISY)</a>, a data-driven approach
for genome-wide analysis of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in public cancer <a 
href="#glo:genomics">genomics</a> data from <a 
href="#glo:TCGA">TCGA</a>
and <a 
href="#glo:CCLE">CCLE</a> (<a 
href="#XBarretina2012">Barretina <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBarretina2012">2012</a>). <a 
href="#glo:DAISY">DAISY</a> is intended to predict the candidate
<a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of a query gene such as genes recurrently mutated in
cancer.
</p><!--l. 888--><p class="indent" ><a 
href="#XJerby2014">Jerby-Arnon <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XJerby2014">2014</a>) combined a computational approach to triage candidates with
a conventional <a 
href="#glo:RNAi">RNAi</a> screen to validate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners. They screened a
selection of computationally predicted candidates and randomly selected genes
with <a 
href="#glo:RNAi">RNAi</a> against <span 
class="cmti-12">VHL </span>loss of function mutation in RCC4 renal cell lines. The
computational method had a high <a 
href="#glo:AUROC">AUROC</a> of 0.779 and predictions were enriched 4<span 
class="cmsy-10x-x-120">&#x00D7; </span>for
validated <a 
href="#glo:RNAi">RNAi</a> hits over randomly selected genes. This approach detected known
<a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs such as <span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2 </span>genes with <span 
class="cmti-12">PARP1 </span>and <span 
class="cmti-12">MSH2 </span>with
<span 
class="cmti-12">DHFR</span>. The <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates identified with both <a 
href="#glo:RNAi">RNAi</a> screening and
computational prediction formed an extensive network of 2077 genes with 2816 <a 
href="#glo:synthetic lethal">synthetic
lethal</a> interactions and similar network of 3158 genes with 3635 synthetic dosage
lethal interactions (for <a 
href="#glo:synthetic lethal">synthetic lethality</a> with over-expression). Each network was
scale-free as expected of a biological network and was enriched for known cancer
genes, essential genes in mice, and could be harnessed for predicting prognosis and
drug response. While demonstrating the feasibility of combining experimental and
computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethality</a> in cancer, there remain challenges
in predicting <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes, novel drug targets, and translation into the
clinic.
</p><!--l. 890--><p class="indent" >The <a 
href="#glo:DAISY">DAISY</a> methodology (<a 
href="#XJerby2014">Jerby-Arnon <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XJerby2014">2014</a>) compares the results of analysis of
several data types to predict <a 
href="#glo:synthetic lethal">synthetic lethality</a>, namely: DNA copy number and
somatic mutation for <a 
href="#glo:TCGA">TCGA</a> patient samples and <a 
href="#glo:CCLE">CCLE</a> cell lines. The cell lines
were also analysed with gene expression and gene essentiality (shRNA screening)
profiles. Genes were classed as inactivated by copy number deletion, somatic loss of
                                                                                    
                                                                                    
function mutation, or low expression and tested for <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene partners
which are either essential in screens or not deleted with copy number variants.
Co-expression is also used for <a 
href="#glo:synthetic lethal">synthetic lethality</a> prediction based on studies in yeast
(<a 
href="#XCostanzo2010">Costanzo <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XCostanzo2010">2010</a>;&#x00A0;<a 
href="#XKelley2005">Kelley and Ideker</a>,&#x00A0;<a 
href="#XKelley2005">2005</a>). Copy number, gene expression
and, essentiality analyses are stringently compared by adjusting each for multiple
tests with Bonferroni correction and only taking hits which occur in all analyses.
This methodology was also adapted for synthetic dosage lethality by testing for
partner genes where genes are overactive with high copy number or expression.
As discussed above, the predictions performed well and an <a 
href="#glo:RNAi">RNAi</a> screen for the
example of <span 
class="cmti-12">VHL </span>in renal cancer validated predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span 
class="cmti-12">VHL</span>
demonstrating the feasibility of combining approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery in
cancer and using computational predictions to enable more efficient high-throughput
screening. <a 
href="#glo:DAISY">DAISY</a> performs well statistically with a <a 
href="#glo:AUROC">AUROC</a> of 0.779 on a set of gene
pairs with experimental screen data, although co-expression and shRNA functional
examination contributes much less of this than the mutation and copy number analysis
(<a 
href="#glo:AUROC">AUROC</a> 0.683 alone). However, this methodology is very stringent, missing potentially
valuable <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates, may not be applicable to genes of interest to
other groups and the software for the procedure has not been publicly released for
replication.
</p><!--l. 892--><p class="indent" >Although the <a 
href="#glo:DAISY">DAISY</a> procedure performs well and has been well received by the scientific
community (<a 
href="#XCrunkhorn2014">Crunkhorn</a>,&#x00A0;<a 
href="#XCrunkhorn2014">2014</a>;&#x00A0;<a 
href="#XLokody2014">Lokody</a>,&#x00A0;<a 
href="#XLokody2014">2014</a>;&#x00A0;<a 
href="#XRyan2014">Ryan <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XRyan2014">2014</a>), showing a need for such
methodology, there is no indication of adoption of the methodology in the community
yet. The co-expression analysis may not be the most effective way to test gene
expression for directional <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions (where inverse correlation
would be expected). In the interests of a large sample size, tissue types were not
tested separately despite tissue-specific <a 
href="#glo:synthetic lethal">synthetic lethality</a> being likely since gene
function (and by extension expression, isoforms, and clinical characteristics) in cancers
may often be tissue-dependent. Some data forms and analyses used, such as gene
essentiality, may not be available for all cancers, genes, or tissues, and may not be
reproduced.
</p><!--l. 894--><p class="indent" ><a 
href="#XLu2015">Lu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XLu2015">2015</a>) critique the assumption of co-expression in the <a 
href="#glo:DAISY">DAISY</a> methodology and
propose an alternative computational prediction of <a 
href="#glo:synthetic lethal">synthetic lethality</a> based on
machine learning methods and a &#x201C;cancer genome evolution&#x201D; hypothesis. Using DNA
copy number and gene expression data from <a 
href="#glo:TCGA">TCGA</a> patient samples, a cancer
genome evolution model assumes that <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs behave in two
                                                                                    
                                                                                    
distinct ways in response to an inactive <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner gene, either a
&#x201C;compensation&#x201D; pattern where the other <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner is overactive or a
&#x201C;co-loss underrepresentation&#x201D; pattern where the other <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner
is less likely to be lost, since loss of both genes would cause death of the cancer
cell. During the genome evolution of cancers, the cell becomes addicted to the
remaining <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner due to induced gene essentiality. These patterns
would explain why <a 
href="#glo:DAISY">DAISY</a> detects only a small number of <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs,
compared to the large number expected based on model organism studies (<a 
href="#XBoone2007">Boone
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBoone2007">2007</a>), and the disparity between screening and computationally predicted
<a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates due to testing different classes of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene
pairs.
</p><!--l. 896--><p class="indent" ><a 
href="#XLu2015">Lu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XLu2015">2015</a>) compared a genome-wide computational model of genome evolution and
gene expression patterns to the experimental data of <a 
href="#XVizeacoumar2013">Vizeacoumar <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XVizeacoumar2013">2013</a>) and <a 
href="#XLaufer2013">Laufer
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XLaufer2013">2013</a>). This simpler model performing well with an <a 
href="#glo:AUROC">AUROC</a> of 0.751 but was less
than <a 
href="#glo:DAISY">DAISY</a>, although it did not rely on data from cell lines which may not represent patient
disease. They predict a larger comprehensive list of 591,000 human <a 
href="#glo:synthetic lethal">synthetic lethal</a>
partners with a probability score threshold of 0.81, giving a precision of 67% and
14<span 
class="cmsy-10x-x-120">&#x00D7; </span>enrichment of <a 
href="#glo:synthetic lethal">synthetic lethal</a> true positives compared to randomly selected
gene pairs. Discovery of such a vast number of cancer-relevant <a 
href="#glo:synthetic lethal">synthetic lethal</a>
interactions in humans would not be feasible experimentally and is a valuable
resource for research and clinical applications. These predictions are not limited by
assuming co-expression of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners or evolutionary conservation
with model organisms enabling wider <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery. However, there
remains a lack of basis for an expectation of how many <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners a
particular gene will have, how many pairs there are in the human genome, and
whether pathways or correlation structure would influence predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a>
partners.
</p><!--l. 898--><p class="indent" >Large scale, computational approaches have yet to determine whether <a 
href="#glo:synthetic lethal">synthetic lethal</a>
interactions are tissue-specific since <a 
href="#XLu2015">Lu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XLu2015">2015</a>) used pan-cancer data for
14136 patients with 31 cancer types. Experimental data used for comparison was
a small training dataset specific to colorectal cancer, and based on screens for
other phenotypes, which may limit performance of the model or application to
other cancers. Proposed expansion of the computational approach to mutation,
microRNA, or epigenetic modulation of gene function and tumour micro-environment or
heterogeneity suggests that <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery could be widely applied to the
                                                                                    
                                                                                    
current challenges in cancer <a 
href="#glo:genomics">genomics</a>. This approach was also based on machine
learning methodology and not supported by a software release for the community to
develop, contribute to, or reproduce beyond the gene pairs given in the supplementary
results.
<a 
 id="x1-47001r52"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.2.7.6 </span> <a 
 id="x1-480006"></a>Bimodality</h5>
<!--l. 902--><p class="noindent" ><a 
href="#XWappett2016">Wappett <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XWappett2016">2016</a>) demonstrate a multi-omic approach to identification of <a 
href="#glo:synthetic lethal">synthetic
lethality</a> in cancer with a strategy to detect bimodal patterns in molecular profiles. They
released this solution as the <a 
href="#glo:BiSEp">BiSEp</a> R package <a 
href="#XWappett2014">Wappett</a>&#x00A0;(<a 
href="#XWappett2014">2014</a>) which aims to detect subtle
bimodal and non-normal patterns in expression data. Since loss of gene function is not
consistently genetic, <a 
href="#XWappett2016">Wappett <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XWappett2016">2016</a>) advocate the use of gene expression (loss of
<a 
href="#glo:mRNA">mRNA</a>) and deletion (loss of copy number) data in addition to mutation. The <a 
href="#glo:BiSEp">BiSEp</a>
procedure was demonstrated on an analysis of 881 cell lines from <a 
href="#glo:CCLE">CCLE</a> (<a 
href="#XBarretina2012">Barretina
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBarretina2012">2012</a>), 442 cell lines from <a 
href="#glo:COSMIC">Catalogue Of Somatic Mutations In Cancer (COSMIC)</a>
(<a 
href="#XForbes2015">Forbes <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XForbes2015">2015</a>), and RSEM normalised <a 
href="#glo:RNA-Seq">RNA-Seq</a> data for 178 <a 
href="#glo:TCGA">TCGA</a> lung patient
samples (<a 
href="#XTCGA2014LU">Collisson <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2014LU">2014</a>). <a 
href="#glo:BiSEp">BiSEp</a> was demonstrated to have significant enrichment of
validated tumour suppressor, <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs (detecting 76 experimentally
supported gene pairs) and was improved (detecting 420) with expression data
rather than relying on detecting loss of gene function by mutation or deletion. They
identified interactions with genes relevant to cancer with support in experimental
screens including <span 
class="cmti-12">ERCC4 </span>with <span 
class="cmti-12">XRCC1</span>, <span 
class="cmti-12">BRCA1 </span>with <span 
class="cmti-12">PARP3</span>, and <span 
class="cmti-12">SMARCA1 </span>with
<span 
class="cmti-12">SMARCA4</span>.
</p><!--l. 904--><p class="indent" ><a 
href="#XWappett2016">Wappett <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XWappett2016">2016</a>) demonstrated that analysis of <a 
href="#glo:genomics">genomics</a> data, particularly
expression data, is relevant to augment the identification of <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions with
screening experiments. They further show that this is applicable in both genetically
homogeneous cell lines and heterogeneous cell population from patient samples. This
approach is limited however to genes which exhibit bimodal expression patterns which do not
commonly occur, particularly in normalised gene expression data, and other approaches
may need to be considered for gene such as <span 
class="cmti-12">CDH1 </span>which were not identified by
<a 
href="#glo:BiSEp">BiSEp</a>.
<a 
 id="x1-48001r53"></a>
</p>
                                                                                    
                                                                                    
<h5 class="subsubsectionHead"><span class="titlemark">1.2.7.7 </span> <a 
 id="x1-490007"></a>Rationale for Further Development</h5>
<!--l. 908--><p class="noindent" >Many of the approaches discussed here aimed to identify the strongest <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs
across the yeast or human genomes (<a 
href="#XDeshpande2013">Deshpande <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XDeshpande2013">2013</a>;&#x00A0;<a 
href="#XLu2015">Lu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XLu2015">2015</a>;&#x00A0;<a 
href="#XWappett2016">Wappett
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XWappett2016">2016</a>;&#x00A0;<a 
href="#XWu2014">Wu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XWu2014">2014</a>), which may not be an ideal strategy to identify interactions in
particular functions or relevance to particular cancers. These demonstrate a need for
computational approaches to prioritise candidate gene pairs for validation but this thesis
will focus on the interactions with <span 
class="cmti-12">CDH1 </span>with particular importance in breast
and stomach cancers, although these partners may be applicable in other cancers.
As such, this thesis presents a query-based method, amenable to identification of
candidate partners for a selected gene of functional or translational importance such as
<span 
class="cmti-12">CDH1</span>.
<a 
 id="x1-49001r32"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">1.3 </span> <a 
 id="x1-500003"></a>E-cadherin as a Synthetic Lethal Target</h3>
<!--l. 915--><p class="noindent" ><a 
href="#glo:E-cadherin">E-cadherin</a> is a transmembrane protein (encoded by <span 
class="cmti-12">CDH1</span>) with several characterised
functions in the cytoskeleton and cell-to-cell signalling. Here we outline the key known
functions of <a 
href="#glo:E-cadherin">E-cadherin</a> and it&#x2019;s importance in cancer biology. <span 
class="cmti-12">CDH1 </span>is a tumour supressor
gene, with loss of function occurring in both familial (germline mutations) and sporadic
(somatic mutations) cancers. As such, <span 
class="cmti-12">CDH1 </span>inactivation is a prime example of a genetic
event that could be targeted by <a 
href="#glo:synthetic lethal">synthetic lethality</a> for anti-cancer treatments. Most notably
this includes patients at risk of developing hereditary breast and stomach cancers for which
conventional surgical or cytotoxic chemotherapy is not ideal (due to impact of quality of life)
and who have a known genetic aberration in their familial syndromic cancers.
Effective treatments against <span 
class="cmti-12">CDH1 </span>inactivation would also benefit patients with
sporadic diffuse gastric cancers since they often present with symptoms at a late
stage.
<a 
 id="x1-50001r44"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.3.1 </span> <a 
 id="x1-510001"></a>The <span 
class="cmti-12">CDH1 </span>gene and it&#x2019;s Biological Functions</h4>
<!--l. 918--><p class="noindent" >The tumour suppressor gene <span 
class="cmti-12">CDH1 </span>is implicated in hereditary and sporadic lobular breast
cancers (<a 
href="#XBerx1996">Berx <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBerx1996">1996</a>;&#x00A0;<a 
href="#XBerx2009">Berx and van Roy</a>,&#x00A0;<a 
href="#XBerx2009">2009</a>;&#x00A0;<a 
href="#XDeLeeuw1997">De&#x00A0;Leeuw <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XDeLeeuw1997">1997</a>;&#x00A0;<a 
href="#XMasciari2007">Masciari
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XMasciari2007">2007</a>;&#x00A0;<a 
href="#XSemb1998">Semb and Christofori</a>,&#x00A0;<a 
href="#XSemb1998">1998</a>;&#x00A0;<a 
href="#XVos1997">Vos <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XVos1997">1997</a>). The <span 
class="cmti-12">CDH1 </span>gene encodes the
<a 
href="#glo:E-cadherin">E-cadherin</a> protein and is normally expressed in epithelial tissues, where it has also been
identified as an invasion suppressor and loss of <span 
class="cmti-12">CDH1 </span>function has been implicated in breast
                                                                                    
                                                                                    
cancer progression and metastasis (<a 
href="#XBecker1994">Becker <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBecker1994">1994</a>;&#x00A0;<a 
href="#XBerx1995">Berx <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBerx1995">1995</a>;&#x00A0;<a 
href="#XChristofori1999">Christofori and
Semb</a>,&#x00A0;<a 
href="#XChristofori1999">1999</a>).
<a 
 id="x1-51001r54"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.3.1.1 </span> <a 
 id="x1-520001"></a>Cytoskeleton</h5>
<!--l. 921--><p class="noindent" >The primary function of <span 
class="cmti-12">CDH1 </span>is cell-cell adhesion forming the adherens junction,
maintaining the cytoskeleton and mediating molecular signals between cells. The function of
the adherens complex is particularly important for cell structure and regulation because
it interacts with cytoskeletal actins and microtubules. The cytoskeletal role of
<a 
href="#glo:E-cadherin">E-cadherin</a> maintains healthy cellular viability and growth in epithelial tissues
including cellular polarity. <a 
href="#glo:E-cadherin">E-cadherin</a> is not essential to cellular viability but loss
in epithelial cells does lead to defects in cytoskeletal structure and proliferation.
In addition to a central role in the adherens complex, <a 
href="#glo:E-cadherin">E-cadherin</a> is involved in
many other cellular functions and thus <span 
class="cmti-12">CDH1 </span>is regarded as a highly <a 
href="#glo:pleiotropy">pleiotropic</a>
gene.
<a 
 id="x1-52001r57"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.3.1.2 </span> <a 
 id="x1-530002"></a>Extracellular and Tumour Micro-Environment</h5>
<!--l. 924--><p class="noindent" >As a transmembrane signalling protein <a 
href="#glo:E-cadherin">E-cadherin</a> also interacts with the extracellular
environment and other cells, most notably forming tight junctions between cells. These
junctions serve to both regulate movement of ion signals between cells and separate
membrane proteins on the apical and basal surfaces of a cell, maintaining cell polarity. Thus
<a 
href="#glo:E-cadherin">E-cadherin</a> is an important regulator of epithelial tissues by intercellular communication. It
also has important roles in the extracellular matrix, including fibrin clot formation. The
role of intercellular interactions and the tissue micro-environment are important
themes in cancer research, being a potential mechanism for cancer progression
and malignancy in a addition to it&#x2019;s potential for specifically targeting tumour
cells.
<a 
 id="x1-53001r58"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.3.1.3 </span> <a 
 id="x1-540003"></a>Cell-Cell Adhesion and Signalling</h5>
<!--l. 927--><p class="noindent" >The signals mediated by tight junctions are also passed on to intracellular signalling
pathways and thus <a 
href="#glo:E-cadherin">E-cadherin</a> also has a role in maintaining cellular function and growth.
One such example is the regulation of <span 
class="cmmi-12">&#x03B2;</span>-catenin which interacts with both the actin
                                                                                    
                                                                                    
cytoskeleton and acts as a transcription factor via the WNT pathway. Similarly, the Hippo
and PI3K/AKT pathways are implicated in being mediated by <a 
href="#glo:E-cadherin">E-cadherin</a>, having roles in
promoting cell survival, proliferation, and repressing apoptosis. <a 
href="#glo:E-cadherin">E-cadherin</a> shares several
downstream pathways with signalling pathways such as integrins and thus indirectly
interacts with them, particularly since feedback loops may occur in such pathways.
Conversely, the multifaceted roles of <a 
href="#glo:E-cadherin">E-cadherin</a> have been shown with differing
overexpression in ovarian cells promoting tumour growth, while it maintains healthy cellular
functions in other cells.
<a 
 id="x1-54001r56"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.3.2 </span> <a 
 id="x1-550002"></a><span 
class="cmti-12">CDH1 </span>as a Tumour (and Invasion) Suppressor</h4>
<!--l. 930--><p class="noindent" ><a 
href="#glo:E-cadherin">E-cadherin</a> has key roles in maintaining cellular structure and regulating growth, consistent
with <span 
class="cmti-12">CDH1 </span>being a tumour suppressor gene. Loss of <span 
class="cmti-12">CDH1 </span>in epithelial tissues leads to
disrupted cell polarity, differentiation, and migration. <a 
href="#glo:E-cadherin">E-cadherin</a> loss has been identified as a
recurrent driver tumour suppressor mutation in sporadic cancers of many tissues including
breast, stomach, lung, colon, and pancreas tissue.
<a 
 id="x1-55001r59"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.3.2.1 </span> <a 
 id="x1-560001"></a>Breast Cancers and Invasion</h5>
<!--l. 933--><p class="noindent" ><a 
href="#glo:E-cadherin">E-cadherin</a> loss in breast cancers has been shown to cause increased proliferation, lymph
node invasion, and metastasis with poor cell-cell contact. Thus <span 
class="cmti-12">CDH1 </span>gene has also been
implicated as an invasion suppressor, with a key role in the <a 
href="#glo:EMT">epithelial-mesenchymal transition
(EMT)</a>, an established mechanism of cancer progression (<a 
href="#XHanahan2011">Hanahan and Weinberg</a>,&#x00A0;<a 
href="#XHanahan2011">2011</a>).
The epithelial-mesenchymal transition is important during development and wound healing
but such changes in cellular differentiation also occur in cancers. If <span 
class="cmti-12">CDH1 </span>is inactivated by
mutation or DNA methylation (<a 
href="#XBerx1996">Berx <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XBerx1996">1996</a>;&#x00A0;<a 
href="#XGuilford1999">Guilford</a>,&#x00A0;<a 
href="#XGuilford1999">1999</a>;&#x00A0;<a 
href="#XMachado2001">Machado <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XMachado2001">2001</a>), it
is likely that <a 
href="#glo:EMT">EMT</a> will drive growth of <a 
href="#glo:E-cadherin">E-cadherin</a> deficient cancers (<a 
href="#XBerx2009">Berx and van
Roy</a>,&#x00A0;<a 
href="#XBerx2009">2009</a>;&#x00A0;<a 
href="#XGraziano2003">Graziano <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XGraziano2003">2003</a>;&#x00A0;<a 
href="#XPolyak2009">Polyak and Weinberg</a>,&#x00A0;<a 
href="#XPolyak2009">2009</a>). While loss of <a 
href="#glo:E-cadherin">E-cadherin</a>
is not sufficient to cause <a 
href="#glo:EMT">EMT</a> or tumourigenesis, it is an important step in this
mechanism of tumour progression and a potential therapeutic intervention may
therefore also impede cancer progression and have activity against advanced stage
cancers.
<a 
 id="x1-56001r60"></a>
</p>
                                                                                    
                                                                                    
<h4 class="subsectionHead"><span class="titlemark">1.3.3 </span> <a 
 id="x1-570003"></a>Hereditary Diffuse Gastric Cancer and Lobular Breast Cancer</h4>
<!--l. 939--><p class="noindent" ><span 
class="cmti-12">CDH1 </span>loss of function mutations also causes familial cancers, including diffuse gastric cancer
and lobular breast cancer (<a 
href="#XGraziano2003">Graziano <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XGraziano2003">2003</a>;&#x00A0;<a 
href="#XGuilford2010">Guilford <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XGuilford2010">2010</a>,&#x00A0;<a 
href="#XHDGC">1999</a>;&#x00A0;<a 
href="#XOliveira2009">Oliveira
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XOliveira2009">2009</a>). Individuals carrying a null mutation in <span 
class="cmti-12">CDH1 </span>have a syndromic predisposition
to early-onset these cancers, known as <a 
href="#glo:HDGC">hereditary diffuse gastric cancer (HDGC)</a>
(<a 
href="#XGuilford1998">Guilford <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XGuilford1998">1998</a>). Due to the loss of an allele, these individuals are prone to
carcinogenic lesion in the breast and stomach when the other allele is inactivated,
occurring much more frequently and thus younger than in individuals without
a second functional allele of <span 
class="cmti-12">CDH1</span>. The loss of the second allele is most often
hypermethylation suppressing expression rather than mutation, although loss of
heterozygousity may also occur. Therefore <a 
href="#glo:HDGC">HDGC</a> is an autosomal dominant cancer
syndrome with incomplete penetrance. The &#x201C;lifetime&#x201D; (until age 80 years) risk for
mutation carriers of diffuse gastric cancer is 70% in males and 56% in females.
In addition, the lifetime risk of lobular breast cancer is 42% in female mutation
carriers.
</p><!--l. 941--><p class="indent" ><a 
href="#glo:HDGC">HDGC</a> affects less than 1 in a million people globally (<a 
href="#XFerlay2015">Ferlay <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XFerlay2015">2015</a>) and
less than 1% of gastric cancers. However, <a 
href="#glo:HDGC">HDGC</a> is documented to affect several
hundred families globally. <a 
href="#glo:E-cadherin">E-cadherin</a> mutations in the germline is implicated in
1-3% of gastric cancers presenting with a family history, varing between high and
low incidence populations. <a 
href="#glo:E-cadherin">E-cadherin</a> is also mutated in 13% of sporadic gastric
cancers.
</p><!--l. 943--><p class="indent" >While diagnostic testing for <span 
class="cmti-12">CDH1 </span>genotype has enabled more effective management of
<a 
href="#glo:HDGC">HDGC</a> and improved patient outcomes, there are still limited options for clinical
interventions (<a 
href="#XGuilford2010">Guilford <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XGuilford2010">2010</a>). Individuals with a family history of <a 
href="#glo:HDGC">HDGC</a> are
recommended to be tested for <span 
class="cmti-12">CDH1 </span>mutations in late adolescence and are offered
prophylactic stomach surgery before the risk of developing cancers increases with age.
Another option is annual endoscopic screening to diagnose early stage stomach
cancers with surgical intervention once they are detected (<a 
href="#XOliveira2013">Oliveira <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XOliveira2013">2013</a>).
However, these early stage cancers are difficult to detect and may be missed in
regular screening. Thus patients carrying <span 
class="cmti-12">CDH1 </span>mutations either have surgical
interventions with a significant impact on quality of life and risk of complications or
remain at risk of developing advanced stage stomach cancers. Due to the lower
mortality rate due to stomach cancers, there is increasing concerns among these
<a 
href="#glo:HDGC">HDGC</a> families on the elevated risk of lobular breast cancers for women later in
life.
                                                                                    
                                                                                    
</p><!--l. 945--><p class="indent" >The current clinical management of <a 
href="#glo:HDGC">HDGC</a> still has significant risks for patients and
therefore a greater understanding of the molecular and cellular function of <span 
class="cmti-12">CDH1 </span>is
important for its role in these cancers. Such studies may lead to alternative treament
strategies such as pharmacological treatments with specificity against <span 
class="cmti-12">CDH1 </span>null cells, once
they lose the second allele. While a loss of gene function cannot be targeted directly,
designing a treatment with specifity against <span 
class="cmti-12">CDH1 </span>may also have activity in sporadic
cancers in a range of epithelial cancers. Thus an effective treatment against <span 
class="cmti-12">CDH1 </span>mutant
cancers would potentially have significant therapeutic and preventative applications in a
large number of patients.
<a 
 id="x1-57001r62"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.3.4 </span> <a 
 id="x1-580004"></a>Somatic Mutations</h4>
<a 
 id="x1-58001r61"></a>
<h5 class="subsubsectionHead"><span class="titlemark">1.3.4.1 </span> <a 
 id="x1-590001"></a>Mutation Rate</h5>
<!--l. 950--><p class="noindent" >Estimates for the prevalence of <span 
class="cmti-12">CDH1 </span>somatic mutations in sporadic cancers varies. The
Cancer Gene Census (<a 
href="#XFutreal2004">Futreal <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XFutreal2004">2004</a>;&#x00A0;<a 
href="#XPleasance2010">Pleasance <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XPleasance2010">2010</a>) detected 994 distinct
mutations in 10,143 tumour samples (at a rate of 7.52%), <a 
href="#XCOSMICdb">COSMIC</a>&#x00A0;(<a 
href="#XCOSMICdb">2016</a>) detected 632
distinct mutations in 43,865 tumour samples (at a rate of 1.71%), and detected mutations in
13.2% of 53 of the NCI60 cancer cell lines. While there is no consensus on the prevalence of
<span 
class="cmti-12">CDH1 </span>mutations, the vast variability of mutations is consistent with it&#x2019;s role as a tumour
supressor and it has been found to be recurrently mutated in a wide range of cancers of
epithelial tissues.
</p><!--l. 952--><p class="indent" ><a 
href="#XCOSMICdb">COSMIC</a>&#x00A0;(<a 
href="#XCOSMICdb">2016</a>) reports <span 
class="cmti-12">CDH1 </span>mutations in 40 cancer tissue types including stomach
(11.40% in 1342 samples), breast (10.29% in 3343 samples), large colon (2.87%), skin
(2.83%), endometrial (2.81%), and bladder (1.9%) cancer. <a 
href="#XICGC2017web">ICGC</a>&#x00A0;(<a 
href="#XICGC2017web">2017</a>) reports <span 
class="cmti-12">CDH1</span>
mutations in 29 cancer tissue types including skin (23.41% in 598 samples), breast (14.50%
in 1696 samples), ovary (13.98% in 93 samples), and stomach (11.41% in 289 samples) cancer
samples. <span 
class="cmti-12">CDH1 </span>mutations are reported at similar rates in breast and stomach
cancer in other cancer <a 
href="#glo:genomics">genomics</a> projects and studies across distinct populations.
<a 
href="#XcBioPortal">cBioPortal</a>&#x00A0;(<a 
href="#XcBioPortal">2017</a>) reports <span 
class="cmti-12">CDH1 </span>mutation prevalence in stomach cancer at 16.7% (<a 
href="#XKakiuchi2014">Kakiuchi
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XKakiuchi2014">2014</a>, 30 samples), 15% (<a 
href="#XWang2014">Wang <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XWang2014">2014</a>, 100 samples), 14.1% (<a 
href="#XChen2015">Chen
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XChen2015">2015</a>, 78 samples), and 9.4% (<a 
href="#XTCGA2017prov">TCGA</a>,&#x00A0;<a 
href="#XTCGA2017prov">2017b</a>, 393 samples). <a 
href="#XcBioPortal">cBioPortal</a>&#x00A0;(<a 
href="#XcBioPortal">2017</a>) also
reports <span 
class="cmti-12">CDH1 </span>mutation prevalence in breast cancer at 12.7% (<a 
href="#XTCGA2017prov">TCGA</a>,&#x00A0;<a 
href="#XTCGA2017prov">2017b</a>, 963
samples) and 10.8% (<a 
href="#XMETABRIC2012">Curtis <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XMETABRIC2012">2012</a>;&#x00A0;<a 
href="#XMETABRIC2016">Pereira <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XMETABRIC2016">2016</a>, 2051 samples). The
rare plasmacytoid bladder cancer subtype also has a high prevalence of <span 
class="cmti-12">CDH1</span>
                                                                                    
                                                                                    
mutations in <a 
href="#XCOSMICdb">COSMIC</a>&#x00A0;(<a 
href="#XCOSMICdb">2016</a>) at a rate of 81.8% (N=33). These demonstrate that
<span 
class="cmti-12">CDH1 </span>is important in many cancers and targeting <span 
class="cmti-12">CDH1 </span>may be widely applied
against sporadic cancers in addition to hereditary cancers. However, some of these
studies have focused on disease subgroups (such as lobular subtype or estogren
receptor negative breast cancers) with poor patient outcomes which may have
inflated the prevalence of <span 
class="cmti-12">CDH1 </span>mutations which are more common in some of these
subtypes.
<a 
 id="x1-59001r64"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">1.3.4.2 </span> <a 
 id="x1-600002"></a>Co-occurring Mutations</h5>
<!--l. 956--><p class="noindent" >Another concern is that <span 
class="cmti-12">CDH1 </span>mutations may co-occur with other known cancer driver
genes such as highly prevalent tumour suppressor gene <span 
class="cmti-12">TP53 </span>or the proto-oncogene
<span 
class="cmti-12">PIK3CA</span>. <a 
href="#XcBioPortal">cBioPortal</a>&#x00A0;(<a 
href="#XcBioPortal">2017</a>) reports the prevalence of the mutations in these genes at 10%
for <span 
class="cmti-12">CDH1</span>, 49% for <span 
class="cmti-12">TP53</span>, 22% for <span 
class="cmti-12">PIK3CA </span>in stomach cancer (<a 
href="#XTCGA2017prov">TCGA</a>,&#x00A0;<a 
href="#XTCGA2017prov">2017b</a>, 393 samples).
There is no evidence of significant co-occurring mutations between <span 
class="cmti-12">CDH1 </span>and
<span 
class="cmti-12">PIK3CA </span>(mutex <span 
class="cmmi-12">p </span>= 0<span 
class="cmmi-12">.</span>231) but there is evidence for significant mutually exclusive
mutations for <span 
class="cmti-12">CDH1 </span>(mutex <span 
class="cmmi-12">p </span>= 0<span 
class="cmmi-12">.</span>002) and <span 
class="cmti-12">PIK3CA </span>(mutex <span 
class="cmmi-12">p </span>= 0<span 
class="cmmi-12">.</span>004) with <span 
class="cmti-12">TP53</span>.
<a 
href="#XcBioPortal">cBioPortal</a>&#x00A0;(<a 
href="#XcBioPortal">2017</a>) also reports the prevalence of the mutations in these genes at
13% for <span 
class="cmti-12">CDH1</span>, 32% for <span 
class="cmti-12">TP53</span>, 36% for <span 
class="cmti-12">PIK3CA </span>in breast cancer (<a 
href="#XTCGA2017prov">TCGA</a>,&#x00A0;<a 
href="#XTCGA2017prov">2017b</a>,
963 samples). There is evidence of significant co-occurring mutations with <span 
class="cmti-12">CDH1</span>
and <span 
class="cmti-12">PIK3CA </span>(mutex <span 
class="cmmi-12">p &#x003C; </span>0<span 
class="cmmi-12">.</span>0001) and evidence for significant mutually exclusive
mutations for <span 
class="cmti-12">CDH1 </span>(mutex <span 
class="cmmi-12">p </span>= 0<span 
class="cmmi-12">.</span>003) and <span 
class="cmti-12">PIK3CA </span>(mutex <span 
class="cmmi-12">p </span>= 0<span 
class="cmmi-12">.</span>032) with
<span 
class="cmti-12">TP53</span>.
</p><!--l. 958--><p class="indent" >These cancer driver mutations have distinct molecular features, leading to disease
progression in distinct ways which is a concern for drug resistance when several
mutations may accumulate, particularly for sporadic cancers where this is common.
Targeting <span 
class="cmti-12">CDH1 </span>specifically is most suitable for hereditary cancers and combination
therapies may be required for sporadic cancers. However, <span 
class="cmti-12">CDH1 </span>and <span 
class="cmti-12">TP53 </span>mutant
cancers appear to be distinct pathways of tumour progression so the high impact of
<span 
class="cmti-12">TP53 </span>mutation on cancer cells need not be considered for the purposes of studying
<span 
class="cmti-12">CDH1</span>.
<a 
 id="x1-60001r63"></a>
</p>
                                                                                    
                                                                                    
<h4 class="subsectionHead"><span class="titlemark">1.3.5 </span> <a 
 id="x1-610005"></a>Models of <span 
class="cmti-12">CDH1 </span>loss in cell lines</h4>
<!--l. 961--><p class="noindent" >Previous work our research group has published used a model of homozygous <span 
class="cmti-12">CDH1</span><sup><span 
class="cmsy-8">&#x2212;</span><span 
class="cmmi-8">&#x2215;</span><span 
class="cmsy-8">&#x2212;</span></sup> null
mutation in non-malignant MCF10A breast cells to show that loss of <span 
class="cmti-12">CDH1 </span>alone was not
sufficient to induce <a 
href="#glo:EMT">EMT</a> with compensatory changes in the expression of other
cell adhesion genes occurring (<a 
href="#XChen2014">Chen <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XChen2014">2014</a>). However, <span 
class="cmti-12">CDH1 </span>deficient cells
did manifest changes in morphology, migration, and weaker cell adhesion (<a 
href="#XChen2014">Chen
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XChen2014">2014</a>).
</p><!--l. 963--><p class="indent" >This <span 
class="cmti-12">CDH1</span><sup><span 
class="cmsy-8">&#x2212;</span><span 
class="cmmi-8">&#x2215;</span><span 
class="cmsy-8">&#x2212;</span></sup> MCF10A model has been used in a genome-wide screen of 18,120 genes
using <a 
href="#glo:siRNA">short interfering ribonucleic acid (siRNA)</a> and a complementary drug screen using
4,057 compounds to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners to <a 
href="#glo:E-cadherin">E-cadherin</a> (<a 
href="#XTelford2015">Telford <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTelford2015">2015</a>).
One of the strongest candidate pathways identified by <a 
href="#XTelford2015">Telford <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XTelford2015">2015</a>) were the GPCR
signalling cascades, which were highly enriched by <a 
href="#glo:GO">GO</a> analysis of the candidate
<a 
href="#glo:synthetic lethal">synthetic lethal</a> partners the primary <a 
href="#glo:siRNA">siRNA</a> screen. This was supported by validation
with Pertussis toxin, known to target G<sub><span 
class="cmmi-8">&#x03B1;i</span></sub> signalling (<a 
href="#XClark2004">Clark</a>,&#x00A0;<a 
href="#XClark2004">2004</a>), as were various
candidate cytoskeletal pathways by inhibition of <a 
href="#glo:JAK">Janus kinase (JAK)</a> and aurora kinase.
The drug screen also produced candidates in <a 
href="#glo:HDAC">histone deacetylase (HDAC)</a> and
<a 
href="#glo:PI3K">phosphoinositide 3-kinase (PI3K)</a> which were supported by validation and time course
experiments.
<a 
 id="x1-61001r55"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">1.4 </span> <a 
 id="x1-620004"></a>Summary and Research Direction of Thesis</h3>
<!--l. 985--><p class="noindent" ><a 
href="#glo:genomics">Genomics</a> technologies and the data available from them have immense potential for
understanding of genetics and improving healthcare, including identification of genes altered
in cancer for molecular diagnosis, prognostic biomarkers, and therapeutic targets. This has
been demonstrated with the identification of cancer genes in many cancers, distinguishing
tumour subtypes by expression profiles, and the development of targeted therapies against
oncogenes (such as <span 
class="cmti-12">BRAF </span>and tumour suppressors (such as <span 
class="cmti-12">BRCA1</span>). <a 
href="#glo:synthetic lethal">Synthetic lethality</a>
is an important genetic interaction to study fundamental cellular functions and
exploit them for biomarkers and cancer treatment. They present a means to target
loss of function mutations and genetic dysregulation in tumour suppressor genes
by identifying interacting partners with redundant or compensating molecular
functions.
</p><!--l. 987--><p class="indent" ><span 
class="cmti-12">CDH1 </span>(encoding <a 
href="#glo:E-cadherin">E-cadherin</a>) is an example of a tumour suppressor gene implicated in
sporadic breast and stomach cancers. Germline mutations in <span 
class="cmti-12">CDH1 </span>are also found in many
patients with familial early onset cancers (<a 
href="#glo:HDGC">HDGC</a>). Discovery of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners
                                                                                    
                                                                                    
would be contribute to an understanding on the molecular mechanisms driving the growth
of <span 
class="cmti-12">CDH1 </span>deficient tumours and identification of potential therapeutic targets or
chemopreventative agents for management of <a 
href="#glo:HDGC">HDGC</a>. The clinical potential of the <a 
href="#glo:synthetic lethal">synthetic
lethal</a> approach has been demonstrated with the application of olaparib against <span 
class="cmti-12">BRCA1 </span>and
<span 
class="cmti-12">BRCA2 </span>mutations <a 
href="#XLord2014">Lord <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XLord2014">2015</a>) but there remains the need to systematically identify
<a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes for other tumour suppressors such as <span 
class="cmti-12">CDH1</span>. A <a 
href="#glo:synthetic lethal">synthetic lethal</a>
screen has been conducted on breast cell lines <a 
href="#XTelford2015">Telford <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XTelford2015">2015</a>) but computational
approaches to identification of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span 
class="cmti-12">CDH1 </span>remains to be
done.
</p><!--l. 992--><p class="indent" >While there are a wide range of experimental and computational approaches to <a 
href="#glo:synthetic lethal">synthetic
lethal</a> discovery, many are limited to particular applications, prone to false positives,
inconsistent across independent approaches, or enriched for particular genes of interest.
Therefore <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions are difficult to replicate or apply in the clinic.
Computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethality</a> are not widely adopted by the cancer
research community and experimental approaches cannot be combined to study <a 
href="#glo:synthetic lethal">synthetic
lethality</a> at a genome-wide scale. However, these show interest in <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery in
the community and the need for robust predictions of <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions in cancer
and human tissues.
</p><!--l. 994--><p class="indent" >Effective screening, prediction, and analysis of <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions are a crucial
part of developing next generation anti-cancer strategies. Therefore, we propose developing a
computational statistical procedure to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions and construct
gene networks. This will enable the development of personalised medicine targeted to
particular molecular aberrations. Genetic tests and <a 
href="#glo:genomics">genomics</a> have the potential to
revolutionise cancer screening, diagnosis, and prognostics; targeted therapeutics, similarly,
have applications in prevention and therapy of sporadic or hereditary cancers with known
molecular properties.
</p><!--l. 996--><p class="indent" >To address the concerns raised by recent computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a>
discovery in cancer (<a 
href="#XJerby2014">Jerby-Arnon <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XJerby2014">2014</a>;&#x00A0;<a 
href="#XLu2015">Lu <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XLu2015">2015</a>;&#x00A0;<a 
href="#XWappett2016">Wappett <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XWappett2016">2016</a>), I
present similar analysis using solely gene expression data which is widely available for a large
number of samples in many different cancers. This uses a statistical methodology the
<a 
href="#glo:SLIPT">Synthetic Lethal Interaction Prediction Tool (SLIPT)</a> developed for this purpose. To further
determine the limitations and implications of <a 
href="#glo:synthetic lethal">synthetic lethal</a> predictions, modelling and
simulation was performed upon the statistical behaviour of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs in
<a 
href="#glo:genomics">genomics</a> data. Comparison of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene candidates from public data analysis and
experimental candidates, pathway analysis, and networks structure will also be
                                                                                    
                                                                                    
presented to investigate the relationships between <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates. Release
of R codes used for simulation, prediction, and analysis will enable adoption of
the methodology in the cancer research community and comparison to existing
methods.
</p><!--l. 998--><p class="indent" >My thesis aims to develop such predictions for <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes with a focus
on the example of <a 
href="#glo:E-cadherin">E-cadherin</a> to compare to the findings of <a 
href="#XTelford2015">Telford <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XTelford2015">2015</a>), develop
of network approaches for pathway structure, and simulate gene expression on
pathway structre with the following <a 
href="#glo:bioinformatics">bioinformatics</a> and computational biology
investigations:
</p>
<ul class="itemize1">
<li class="itemize">
<!--l. 1004--><p class="noindent" >Developed a query-based <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection methodology (<a 
href="#glo:SLIPT">SLIPT</a>) for use
on gene expression data
</p></li>
<li class="itemize">
<!--l. 1008--><p class="noindent" >Adapt this methodology to utilise somatic mutation for query genes or candidate
pathway metagenes
</p></li>
<li class="itemize">
<!--l. 1012--><p class="noindent" >Apply <a 
href="#glo:synthetic lethal">Synthetic lethal</a> prediction to public breast cancer <a 
href="#glo:genomics">genomics</a> data from
<a 
href="#glo:TCGA">TCGA</a> (<a 
href="#XTCGA2012">TCGA</a>,&#x00A0;<a 
href="#XTCGA2012">2012</a>)
</p></li>
<li class="itemize">
<!--l. 1016--><p class="noindent" >Identify over-represented biological pathways using Reactome (<a 
href="#XReactome">Croft <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XReactome">2014</a>)
among <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate partner genes
</p></li>
<li class="itemize">
<!--l. 1020--><p class="noindent" >Compare these at the gene and pathway level to experimental screen data in
breast cell lines from <a 
href="#XTelford2015">Telford <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>&#x00A0;(<a 
href="#XTelford2015">2015</a>)
</p></li>
<li class="itemize">
                                                                                    
                                                                                    
<!--l. 1024--><p class="noindent" >Replicate these analyses in stomach cancer <a 
href="#glo:genomics">genomics</a> data from <a 
href="#glo:TCGA">TCGA</a> (<a 
href="#XTCGA2014GC">Bass
<span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span></a>,&#x00A0;<a 
href="#XTCGA2014GC">2014</a>)
</p></li>
<li class="itemize">
<!--l. 1028--><p class="noindent" >Determine  whether  <a 
href="#glo:synthetic lethal">synthetic  lethal</a>  candidates  have  importance  in  biological
networks of candidate partner pathways
</p></li>
<li class="itemize">
<!--l. 1032--><p class="noindent" >Determine whether there are relationships within biological network structures
between experimental and predicted gene candidate partners
</p></li>
<li class="itemize">
<!--l. 1036--><p class="noindent" >Develop a statistical model of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene expression
</p></li>
<li class="itemize">
<!--l. 1040--><p class="noindent" >Simulate gene expression with <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes and pathway structures
</p></li>
<li class="itemize">
<!--l. 1044--><p class="noindent" >Evaluate the effects of modification to the <a 
href="#glo:SLIPT">SLIPT</a> procedure on it&#x2019;s statistical
performance
</p></li>
<li class="itemize">
<!--l. 1048--><p class="noindent" >Compare  the  statistical  performance  of  the  <a 
href="#glo:SLIPT">SLIPT</a>  procedure  to  alternative
statistical methods
</p></li>
<li class="itemize">
<!--l. 1052--><p class="noindent" >Release  a  <a 
href="#glo:synthetic lethal">synthetic  lethal</a>  prediction  methodology  (<a 
href="#glo:SLIPT">SLIPT</a>)  to  the  research
community for wider application
                                                                                    
                                                                                    
</p>
<div class="center" 
>
<!--l. 1057--><p class="noindent" >
</p><!--l. 1058--><p class="noindent" ><span 
class="cmbx-12">Thesis Aims</span></p></div>
</li></ul>
<ul class="itemize1">
<li class="itemize">To develop a statistical approach to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs in cancer
from expression data
</li>
<li class="itemize">To apply this methodology to public cancer gene expression data against <span 
class="cmti-12">CDH1</span>
and analyse pathway structure with comparisons to experimental screen data
</li>
<li class="itemize">To construct a statistical model of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in multivariate normal
expression data
</li>
<li class="itemize">To develop  a  simulation  pipeline  of  expression  with  pathway  structure  on  a
high-performance computing cluster
</li>
<li class="itemize">To examine  the  statistical  performance  of  the  methodology  with  simulated
expression including pathways and compare it to other approaches
</li>
<li class="itemize">To release the <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection methodology and pathway simulation
procedure as R software packages
</li></ul>
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
                                                                                    
<a 
 id="Q1-1-68"></a>
                                                                                    
                                                                                    
<h2 class="likechapterHead"><a 
 id="x1-630000"></a>References</h2>
<div class="thebibliography">
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAarts2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Aarts, M., Bajrami, I., Herrera-Abreu, M.T., Elliott, R., Brough, R., Ashworth, A.,
Lord, C.J., and Turner, N.C. (2015) Functional genetic screen identifies increased
sensitivity to wee1 inhibition in cells with defects in fanconi anemia and hr pathways.
<span 
class="cmti-12">Mol Cancer Ther</span>, <span 
class="cmbx-12">14</span>(4): 865&#x2013;76.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2015PR"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Abeshouse, A., Ahn, J., Akbani, R., Ally, A., Amin, S., Andry, C.D., Annala, M.,
Aprikian, A., Armenia, J., Arora, A., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2015) The Molecular Taxonomy of
Primary Prostate Cancer. <span 
class="cmti-12">Cell</span>, <span 
class="cmbx-12">163</span>(4): 1011&#x2013;1025.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAdamski2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Adamski, M.G., Gumann, P., and Baird, A.E. (2014) A method for quantitative
analysis of standard and high-throughput qPCR expression data based on input
sample quantity. <span 
class="cmti-12">PLoS ONE</span>, <span 
class="cmbx-12">9</span>(8): e103917.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xvioplot"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Adler, D. (2005) <span 
class="cmti-12">vioplot: Violin plot</span>. R package version 0.2.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAgarwal2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Agarwal, S., Deane, C.M., Porter, M.A., and Jones, N.S. (2010) Revisiting date
and party hubs: Novel approaches to role assignment in protein interaction networks.
<span 
class="cmti-12">PLoS Comput Biol</span>, <span 
class="cmbx-12">6</span>(6): e1000817.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2014TH"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Agrawal, N., Akbani, R., Aksoy, B.A., Ally, A., Arachchi, H., Asa, S.L., Auman,
J.T., Balasundaram, M., Balu, S., Baylin, S.B., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2014) Integrated genomic
characterization of papillary thyroid carcinoma. <span 
class="cmti-12">Cell</span>, <span 
class="cmbx-12">159</span>(3): 676&#x2013;690.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2015SK"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Akbani, R., Akdemir, K.C., Aksoy, B.A., Albert, M., Ally, A., Amin, S.B., Arachchi,
H., Arora,  A.,  Auman,  J.T.,  Ayala,  B.,  <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.  </span>(2015)  Genomic  Classification  of
Cutaneous Melanoma. <span 
class="cmti-12">Cell</span>, <span 
class="cmbx-12">161</span>(7): 1681&#x2013;1696.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAkobeng2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Akobeng,  A.K.  (2007)  Understanding  diagnostic  tests  3:  receiver  operating
characteristic curves. <span 
class="cmti-12">Acta Pdiatrica</span>, <span 
class="cmbx-12">96</span>(5): 644&#x2013;647.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XACS2017"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>American              Cancer              Society              (2017)              Genetics
and cancer.  <a 
href="https://www.cancer.org/cancer/cancer-causes/genetics.html" class="url" ><span 
class="cmtt-12">https://www.cancer.org/cancer/cancer-_causes/genetics.html</span></a>.
Accessed: 22/03/2017.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XASCO2017"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>American Society for Clinical Oncology (ASCO) (2017) The genetics of cancer.
<a 
href="http://www.cancer.net/navigating-cancer-care/cancer-basics/genetics/genetics-cancer" class="url" ><span 
class="cmtt-12">http://www.cancer.net/navigating-_cancer-_care/cancer-_basics/genetics/genetics-_cancer</span></a>.
Accessed: 22/03/2017.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAnjomshoaa2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Anjomshoaa,  A.,  Lin,  Y.H.,  Black,  M.A.,  McCall,  J.L.,  Humar,  B.,  Song,  S.,
Fukuzawa, R.,  Yoon,  H.S.,  Holzmann,  B.,  Friederichs,  J.,  <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2008)  Reduced
expression of a gene proliferation signature is associated with enhanced malignancy
in colon cancer. <span 
class="cmti-12">Br J Cancer</span>, <span 
class="cmbx-12">99</span>(6): 966&#x2013;973.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xgenesetdb"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Araki, H., Knapp, C., Tsai, P., and Print, C. (2012) GeneSetDB: A comprehensive
                                                                                    
                                                                                    
meta-database, statistical and visualisation framework for gene set analysis.  <span 
class="cmti-12">FEBS</span>
<span 
class="cmti-12">Open Bio</span>, <span 
class="cmbx-12">2</span>: 76&#x2013;82.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAshburner2000"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Ashburner, M., Ball, C.A., Blake, J.A., Botstein, D., Butler, H., Cherry, J.M.,
Davis, A.P., Dolinski, K., Dwight, S.S., Eppig, J.T., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2000) Gene ontology: tool
for the unification of biology. The Gene Ontology Consortium.  <span 
class="cmti-12">Nat Genet</span>, <span 
class="cmbx-12">25</span>(1):
25&#x2013;29.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAshworth2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Ashworth, A. (2008) A synthetic lethal therapeutic approach: poly(adp) ribose
polymerase inhibitors for the treatment of cancers deficient in dna double-strand
break repair. <span 
class="cmti-12">J Clin Oncol</span>, <span 
class="cmbx-12">26</span>(22): 3785&#x2013;90.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAudeh2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Audeh, M.W.,  Carmichael,  J.,  Penson,  R.T.,  Friedlander,  M.,  Powell,  B.,
Bell-McGuinn, K.M., Scott, C., Weitzel, J.N., Oaknin, A., Loman, N., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2010)
Oral poly(adp-ribose)  polymerase  inhibitor  olaparib  in  patients  with  <span 
class="cmti-12">BRCA1  </span>or
<span 
class="cmti-12">BRCA2 </span>mutations and recurrent ovarian cancer: a proof-of-concept trial.  <span 
class="cmti-12">Lancet</span>,
<span 
class="cmbx-12">376</span>(9737): 245&#x2013;51.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBabyak2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Babyak, M.A. (2004) What you see may not be what you get: a brief, nontechnical
introduction  to  overfitting  in  regression-type  models.    <span 
class="cmti-12">Psychosom  Med</span>,  <span 
class="cmbx-12">66</span>(3):
411&#x2013;21.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBamford2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Bamford, S., Dawson, E., Forbes, S., Clements, J., Pettett, R., Dogan, A., Flanagan,
A., Teague, J., Futreal, P.A., Stratton, M.R., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2004) The COSMIC (Catalogue
of Somatic Mutations in Cancer) database and website. <span 
class="cmti-12">Br J Cancer</span>, <span 
class="cmbx-12">91</span>(2): 355&#x2013;358.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBarabasi1999"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Barab&#x00E1;si, A.L. and Albert, R. (1999) Emergence of scaling in random networks.
<span 
class="cmti-12">Science</span>, <span 
class="cmbx-12">286</span>(5439): 509&#x2013;12.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBarabasi2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Barab&#x00E1;si, A.L. and Oltvai, Z.N. (2004) Network biology: understanding the cell&#x2019;s
functional organization. <span 
class="cmti-12">Nat Rev Genet</span>, <span 
class="cmbx-12">5</span>(2): 101&#x2013;13.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBarrat2000"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Barrat, A. and Weigt, M. (2000) On the properties of small-world network models.
<span 
class="cmti-12">The European Physical Journal B - Condensed Matter and Complex Systems</span>, <span 
class="cmbx-12">13</span>(3):
547&#x2013;560.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBarretina2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Barretina, J., Caponigro, G., Stransky, N., Venkatesan, K., Margolin, A.A., Kim,
S., Wilson, C.J., Lehar, J., Kryukov, G.V., Sonkin, D., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2012) The Cancer
Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity.
<span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">483</span>(7391): 603&#x2013;607.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xsafe"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Barry, W.T. (2016) <span 
class="cmti-12">safe: Significance Analysis of Function and Expression</span>.   R
package version 3.14.0.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBaryshnikova2010a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Baryshnikova,  A.,  Costanzo,  M.,  Dixon,  S.,  Vizeacoumar,  F.J.,  Myers,  C.L.,
Andrews, B.,  and  Boone,  C.  (2010a)  Synthetic  genetic  array  (sga)  analysis  in
saccharomyces cerevisiae and schizosaccharomyces pombe.  <span 
class="cmti-12">Methods Enzymol</span>, <span 
class="cmbx-12">470</span>:
145&#x2013;79.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBaryshnikova2010b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Baryshnikova, A., Costanzo, M., Kim, Y., Ding, H., Koh, J., Toufighi, K., Youn,
J.Y., Ou, J., San&#x00A0;Luis, B.J., Bandyopadhyay, S., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2010b) Quantitative analysis
                                                                                    
                                                                                    
of fitness and genetic interactions in yeast on a genome scale.  <span 
class="cmti-12">Nat Meth</span>, <span 
class="cmbx-12">7</span>(12):
1017&#x2013;1024.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2014GC"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Bass, A.J., Thorsson, V., Shmulevich, I., Reynolds, S.M., Miller, M., Bernard, B.,
Hinoue, T., Laird, P.W., Curtis, C., Shen, H., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2014) Comprehensive molecular
characterization of gastric adenocarcinoma. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">513</span>(7517): 202&#x2013;209.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMatrix"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Bates, D. and Maechler, M. (2016) <span 
class="cmti-12">Matrix: Sparse and Dense Matrix Classes and</span>
<span 
class="cmti-12">Methods</span>. R package version 1.2-7.1.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBateson1909"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Bateson, W. and Mendel, G. (1909) <span 
class="cmti-12">Mendel&#x2019;s principles of heredity, by W. Bateson</span>.
University Press, Cambridge [Eng.].
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBeck2016"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Beck, T.F., Mullikin, J.C., and Biesecker, L.G. (2016) Systematic Evaluation of
Sanger Validation  of  Next-Generation  Sequencing  Variants.   <span 
class="cmti-12">Clin  Chem</span>,  <span 
class="cmbx-12">62</span>(4):
647&#x2013;654.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBecker1994"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Becker, K.F., Atkinson, M.J., Reich, U., Becker, I., Nekarda, H., Siewert, J.R.,
and Hfler, H. (1994) E-cadherin gene mutations provide clues to diffuse type gastric
carcinomas. <span 
class="cmti-12">Cancer Research</span>, <span 
class="cmbx-12">54</span>(14): 3845&#x2013;3852.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2011OV"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Bell, D., Berchuck,  A.,  Birrer,  M.,  Chien,  J.,  Cramer,  D.,  Dao,  F.,  Dhir,  R.,
DiSaia, P., Gabra, H., Glenn, P., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2011) Integrated genomic analyses of ovarian
carcinoma. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">474</span>(7353): 609&#x2013;615.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xfdr1995"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Benjamini, Y. and Hochberg, Y. (1995) Controlling the false discovery rate: A
practical and powerful approach to multiple testing. <span 
class="cmti-12">Journal of the Royal Statistical</span>
<span 
class="cmti-12">Society Series B (Methodological)</span>, <span 
class="cmbx-12">57</span>(1): 289&#x2013;300.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBerx1995"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Berx, G., Cleton-Jansen, A.M., Nollet, F., de&#x00A0;Leeuw, W.J., van&#x00A0;de Vijver, M.,
Cornelisse, C., and van Roy, F. (1995) E-cadherin is a tumour/invasion suppressor
gene mutated in human lobular breast cancers. <span 
class="cmti-12">EMBO J</span>, <span 
class="cmbx-12">14</span>(24): 6107&#x2013;15.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBerx1996"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Berx, G., Cleton-Jansen, A.M., Strumane, K., de&#x00A0;Leeuw, W.J., Nollet, F., van Roy,
F., and Cornelisse, C. (1996) E-cadherin is inactivated in a majority of invasive
human lobular breast cancers by truncation mutations throughout its extracellular
domain. <span 
class="cmti-12">Oncogene</span>, <span 
class="cmbx-12">13</span>(9): 1919&#x2013;25.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBerx2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Berx, G.  and  van  Roy,  F.  (2009)  Involvement  of  members  of  the  cadherin
superfamily in cancer. <span 
class="cmti-12">Cold Spring Harb Perspect Biol</span>, <span 
class="cmbx-12">1</span>: a003129.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBitler2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Bitler, B.G., Aird, K.M., Garipov, A., Li, H., Amatangelo, M., Kossenkov, A.V.,
Schultz, D.C., Liu, Q., Shih&#x00A0;Ie, M., Conejo-Garcia, J.R., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2015) Synthetic
lethality by targeting ezh2 methyltransferase activity in arid1a-mutated cancers. <span 
class="cmti-12">Nat</span>
<span 
class="cmti-12">Med</span>, <span 
class="cmbx-12">21</span>(3): 231&#x2013;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBlake2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Blake, J.A., Christie, K.R., Dolan, M.E., Drabkin, H.J., Hill, D.P., Ni, L., Sitnikov,
D., Burgess, S., Buza, T., Gresham, C., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2015) Gene Ontology Consortium:
going forward. <span 
class="cmti-12">Nucleic Acids Res</span>, <span 
class="cmbx-12">43</span>(Database issue): D1049&#x2013;1056.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBoettcher2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Boettcher, M., Lawson, A., Ladenburger, V., Fredebohm, J., Wolf, J., Hoheisel,
J.D., Frezza, C., and Shlomi, T. (2014) High throughput synthetic lethality screen
reveals a tumorigenic role of adenylate cyclase in fumarate hydratase-deficient cancer
cells. <span 
class="cmti-12">BMC Genomics</span>, <span 
class="cmbx-12">15</span>: 158.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBoone2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Boone, C., Bussey, H., and Andrews, B.J. (2007) Exploring genetic interactions and
networks with yeast. <span 
class="cmti-12">Nat Rev Genet</span>, <span 
class="cmbx-12">8</span>(6): 437&#x2013;49.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBorgatti2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Borgatti, S.P. (2005) Centrality and network flow. <span 
class="cmti-12">Social Networks</span>, <span 
class="cmbx-12">27</span>(1): 55 &#x2013; 71.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBoucher2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Boucher, B. and Jenna, S. (2013) Genetic interaction networks: better understand
to better predict. <span 
class="cmti-12">Front Genet</span>, <span 
class="cmbx-12">4</span>: 290.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBreiman2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Breiman, L. (2001) Random forests. <span 
class="cmti-12">Machine Learning</span>, <span 
class="cmbx-12">45</span>(1): 5&#x2013;32.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBrin1998"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Brin, S. and Page, L. (1998) The anatomy of a large-scale hypertextual web search
engine. <span 
class="cmti-12">Computer Networks and ISDN Systems</span>, <span 
class="cmbx-12">30</span>(1): 107 &#x2013; 117.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBryant2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Bryant, H.E., Schultz, N., Thomas, H.D., Parker, K.M., Flower, D., Lopez, E.,
Kyle, S., Meuth,  M.,  Curtin,  N.J.,  and  Helleday,  T.  (2005)  Specific  killing  of
<span 
class="cmti-12">BRCA2 </span>deficient tumours with inhibitors of poly<img 
src="20170511_thesis_introduction3x.png" alt="adpribose  "  class="math"  /> polymerase.  <span 
class="cmti-12">Nature</span>,
<span 
class="cmbx-12">434</span>(7035): 913&#x2013;7.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2017CERV"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Burk, R.D., Chen, Z., Saller, C., Tarvin, K., Carvalho, A.L., Scapulatempo-Neto,
                                                                                    
                                                                                    
C., Silveira, H.C., Fregnani, J.H., Creighton, C.J., Anderson, M.L., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2017)
Integrated genomic  and  molecular  characterization  of  cervical  cancer.    <span 
class="cmti-12">Nature</span>,
<span 
class="cmbx-12">543</span>(7645): 378&#x2013;384.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBussey2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Bussey, H., Andrews, B., and Boone, C. (2006) From worm genetic networks to
complex human diseases. <span 
class="cmti-12">Nat Genet</span>, <span 
class="cmbx-12">38</span>(8): 862&#x2013;3.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XButland2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Butland, G., Babu, M., Diaz-Mejia, J.J., Bohdana, F., Phanse, S., Gold, B., Yang,
W., Li, J., Gagarinova, A.G., Pogoutse, O., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2008) esga: E. coli synthetic
genetic array analysis. <span 
class="cmti-12">Nat Methods</span>, <span 
class="cmbx-12">5</span>(9): 789&#x2013;95.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCancerResearchUK2017"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Cancer Research    UK    (2017)    Family    history    and    cancer    genes.
<a 
href="http://www.cancerresearchuk.org/about-cancer/causes-of-cancer/inherited-cancer-genes-and-increased-cancer-risk/family-history-and-inherited-cancer-genes" class="url" ><span 
class="cmtt-12">http://www.cancerresearchuk.org/about-_cancer/causes-_of-_cancer/inherited-_cancer-_genes-_and-_increased-_cancer-_risk/family-_history-_and-_inherited-_cancer-_genes</span></a>.
Accessed: 22/03/2017.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCCLE"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Cancer Cell Line Encyclopedia (CCLE) (2014) Broad-Novartis Cancer Cell Line
Encyclopedia. <a 
href="http://www.broadinstitute.org/ccle" class="url" ><span 
class="cmtt-12">http://www.broadinstitute.org/ccle</span></a>. Accessed: 07/11/2014.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XcBioPortal"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>cBioPortal  for  Cancer  Genomics  (cBioPortal)  (2017)  cBioPortal  for  Cancer
Genomics. <a 
href="http://www.cbioportal.org/" class="url" ><span 
class="cmtt-12">http://www.cbioportal.org/</span></a>. Accessed: 26/03/2017.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPathwayCommons"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Cerami, E.G.,  Gross,  B.E.,  Demir,  E.,  Rodchenkov,  I.,  Babur,  O.,  Anwar,  N.,
Schultz, N., Bader, G.D., and Sander, C. (2011) Pathway Commons, a web resource
for biological pathway data. <span 
class="cmti-12">Nucleic Acids Res</span>, <span 
class="cmbx-12">39</span>(Database issue): D685&#x2013;690.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XChen2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Chen, A., Beetham, H., Black, M.A., Priya, R., Telford, B.J., Guest, J., Wiggins,
G.A.R., Godwin,  T.D.,  Yap,  A.S.,  and  Guilford,  P.J.  (2014)  E-cadherin  loss
alters cytoskeletal organization and adhesion in non-malignant breast cells but is
insufficient to induce an epithelial-mesenchymal transition.  <span 
class="cmti-12">BMC Cancer</span>, <span 
class="cmbx-12">14</span>(1):
552.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XChen2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Chen, K., Yang, D., Li, X., Sun, B., Song, F., Cao, W., Brat, D.J., Gao, Z., Li, H.,
Liang, H., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2015) Mutational landscape of gastric adenocarcinoma in Chinese:
implications for prognosis and therapy. <span 
class="cmti-12">Proc Natl Acad Sci USA</span>, <span 
class="cmbx-12">112</span>(4): 1107&#x2013;1112.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XChen2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Chen, S.  and  Parmigiani,  G.  (2007)  Meta-analysis  of  BRCA1  and  BRCA2
penetrance. <span 
class="cmti-12">J Clin Oncol</span>, <span 
class="cmbx-12">25</span>(11): 1329&#x2013;1333.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XChen2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Chen, X. and Tompa, M. (2010) Comparative assessment of methods for aligning
multiple genome sequences. <span 
class="cmti-12">Nat Biotechnol</span>, <span 
class="cmbx-12">28</span>(6): 567&#x2013;572.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2017UT"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Cherniack, A.D., Shen, H., Walter, V., Stewart, C., Murray, B.A., Bowlby, R.,
Hu, X., Ling, S., Soslow, R.A., Broaddus, R.R., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2017) Integrated Molecular
Characterization of Uterine Carcinosarcoma. <span 
class="cmti-12">Cancer Cell</span>, <span 
class="cmbx-12">31</span>(3): 411&#x2013;423.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XChipman2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Chipman, K. and Singh, A. (2009) Predicting genetic interactions with random
walks on biological networks. <span 
class="cmti-12">BMC Bioinformatics</span>, <span 
class="cmbx-12">10</span>(1): 17.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XChristofori1999"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Christofori,  G.  and  Semb,  H.  (1999)  The  role  of  the  cell-adhesion  molecule
E-cadherin as a tumour-suppressor gene. <span 
class="cmti-12">Trends in Biochemical Sciences</span>, <span 
class="cmbx-12">24</span>(2): 73
&#x2013; 76.
                                                                                    
                                                                                    
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2015LBC"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Ciriello, G.,  Gatza,  M.L.,  Beck,  A.H.,  Wilkerson,  M.D.,  Rhie,  S.K.,  Pastore,
A., Zhang, H., McLellan, M., Yau, C., Kandoth, C., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2015) Comprehensive
Molecular Portraits of Invasive Lobular Breast Cancer. <span 
class="cmti-12">Cell</span>, <span 
class="cmbx-12">163</span>(2): 506&#x2013;519.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XClark2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Clark, M.J. (2004) Endogenous Regulator of G Protein Signaling Proteins Suppress
G&#x00A0;o-Dependent  &#x00A0;-Opioid  Agonist-Mediated  Adenylyl  Cyclase  Supersensitization.
<span 
class="cmti-12">Journal of Pharmacology and Experimental Therapeutics</span>, <span 
class="cmbx-12">310</span>(1): 215&#x2013;222.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGEO2016"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Clough, E.  and  Barrett,  T.  (2016)  The  Gene  Expression  Omnibus  Database.
<span 
class="cmti-12">Methods Mol Biol</span>, <span 
class="cmbx-12">1418</span>: 93&#x2013;110.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCollingridge2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Collingridge, D.S. (2013) A primer on quantitized data analysis and permutation
testing. <span 
class="cmti-12">Journal of Mixed Methods Research</span>, <span 
class="cmbx-12">7</span>(1): 81&#x2013;97.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCollins2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Collins, F.S. and Barker, A.D. (2007) Mapping the cancer genome. Pinpointing the
genes involved in cancer will help chart a new course across the complex landscape
of human malignancies. <span 
class="cmti-12">Sci Am</span>, <span 
class="cmbx-12">296</span>(3): 50&#x2013;57.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCollins2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Collins, F.S., Morgan, M., and Patrinos, A. (2003) The Human Genome Project:
lessons from large-scale biology. <span 
class="cmti-12">Science</span>, <span 
class="cmbx-12">300</span>(5617): 286&#x2013;290.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2014LU"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Collisson, E., Campbell, J., Brooks, A., Berger, A., Lee, W., Chmielecki, J., Beer,
D., Cope, L., Creighton, C., Danilova, L., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2014) Comprehensive molecular
profiling of lung adenocarcinoma. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">511</span>(7511): 543&#x2013;550.
                                                                                    
                                                                                    
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCorcoran2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Corcoran, R.B., Ebi, H., Turke, A.B., Coffee, E.M., Nishino, M., Cogdill, A.P.,
Brown, R.D.,  Della&#x00A0;Pelle,  P.,  Dias-Santagata,  D.,  Hung,  K.E.,  <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.  </span>(2012)
Egfr-mediated  reactivation  of  mapk  signaling  contributes  to  insensitivity  of
<span 
class="cmti-12">BRAF </span>-mutant  colorectal  cancers  to  raf  inhibition  with  vemurafenib.    <span 
class="cmti-12">Cancer</span>
<span 
class="cmti-12">Discovery</span>, <span 
class="cmbx-12">2</span>(3): 227&#x2013;235.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCostanzo2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Costanzo, M., Baryshnikova, A., Bellay, J., Kim, Y., Spear, E.D., Sevier, C.S., Ding,
H., Koh, J.L., Toufighi, K., Mostafavi, S., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2010) The genetic landscape of a
cell. <span 
class="cmti-12">Science</span>, <span 
class="cmbx-12">327</span>(5964): 425&#x2013;31.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCostanzo2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Costanzo, M., Baryshnikova, A., Myers, C.L., Andrews, B., and Boone, C. (2011)
Charting the genetic interaction map of a cell. <span 
class="cmti-12">Curr Opin Biotechnol</span>, <span 
class="cmbx-12">22</span>(1): 66&#x2013;74.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2013RCC"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Creighton,  C.J.,  Morgan,  M.,  Gunaratne,  P.H.,  Wheeler,  D.A.,  Gibbs,  R.A.,
Robertson, A., Chu, A., Beroukhim, R., Cibulskis, K., Signoretti, S., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2013)
Comprehensive molecular characterization of clear cell renal cell carcinoma. <span 
class="cmti-12">Nature</span>,
<span 
class="cmbx-12">499</span>(7456): 43&#x2013;49.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XReactome"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Croft, D., Mundo, A.F., Haw, R., Milacic, M., Weiser, J., Wu, G., Caudy, M.,
Garapati, P., Gillespie, M., Kamdar, M.R., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2014) The Reactome pathway
knowledgebase. <span 
class="cmti-12">Nucleic Acids Res</span>, <span 
class="cmbx-12">42</span>(database issue): D472D477.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCrunkhorn2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Crunkhorn, S. (2014) Cancer: Predicting synthetic lethal interactions.  <span 
class="cmti-12">Nat Rev</span>
<span 
class="cmti-12">Drug Discov</span>, <span 
class="cmbx-12">13</span>(11): 812.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xigraph"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Csardi, G. and Nepusz, T. (2006) The igraph software package for complex network
research. <span 
class="cmti-12">InterJournal</span>, <span 
class="cmbx-12">Complex Systems</span>: 1695.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMETABRIC2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Curtis, C., Shah, S.P., Chin, S.F., Turashvili, G., Rueda, O.M., Dunning, M.J.,
Speed, D., Lynch, A.G., Samarajiwa, S., Yuan, Y., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2012) The genomic and
transcriptomic architecture of 2,000 breast tumours reveals novel subgroups. <span 
class="cmti-12">Nature</span>,
<span 
class="cmbx-12">486</span>(7403): 346&#x2013;352.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDai2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Dai, X., Li, T., Bai, Z., Yang, Y., Liu, X., Zhan, J., and Shi, B. (2015) Breast
cancer intrinsic subtype classification, clinical use and future trends.  <span 
class="cmti-12">Am J Cancer</span>
<span 
class="cmti-12">Res</span>, <span 
class="cmbx-12">5</span>(10): 2929&#x2013;2943.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDavierwala2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Davierwala,  A.P.,  Haynes,  J.,  Li,  Z.,  Brost,  R.L.,  Robinson,  M.D.,  Yu,  L.,
Mnaimneh, S., Ding, H., Zhu, H., Chen, Y., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2005) The synthetic genetic
interaction spectrum of essential genes. <span 
class="cmti-12">Nat Genet</span>, <span 
class="cmbx-12">37</span>(10): 1147&#x2013;1152.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDeLeeuw1997"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>De&#x00A0;Leeuw, W.J., Berx, G., Vos, C.B., Peterse, J.L., Van&#x00A0;de Vijver, M.J., Litvinov,
S., Van&#x00A0;Roy, F., Cornelisse, C.J., and Cleton-Jansen, A.M. (1997) Simultaneous loss
of E-cadherin and catenins in invasive lobular breast cancer and lobular carcinoma
in situ. <span 
class="cmti-12">J Pathol</span>, <span 
class="cmbx-12">183</span>(4): 404&#x2013;11.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xpaxtools"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Demir, E.,  Babur,  O.,  Rodchenkov,  I.,  Aksoy,  B.A.,  Fukuda,  K.I.,  Gross,  B.,
Sumer, O.S., Bader, G.D., and Sander, C. (2013) Using biological pathway data with
Paxtools. <span 
class="cmti-12">PLoS Comput Biol</span>, <span 
class="cmbx-12">9</span>(9): e1003194.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDeshpande2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Deshpande,  R.,  Asiedu,  M.K.,  Klebig,  M.,  Sutor,  S.,  Kuzmin,  E.,  Nelson,  J.,
Piotrowski, J., Shin, S.H., Yoshida, M., Costanzo, M., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2013) A comparative
genomic approach  for  identifying  synthetic  lethal  interactions  in  human  cancer.
<span 
class="cmti-12">Cancer Res</span>, <span 
class="cmbx-12">73</span>(20): 6128&#x2013;36.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDickson1999"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Dickson, D. (1999) Wellcome funds cancer database. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">401</span>(6755): 729.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDienstmann2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Dienstmann, R. and Tabernero, J. (2011) <span 
class="cmti-12">BRAF  </span>as a target for cancer therapy.
<span 
class="cmti-12">Anticancer Agents Med Chem</span>, <span 
class="cmbx-12">11</span>(3): 285&#x2013;95.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDijkstra1959"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Dijkstra, E.W.  (1959)  A  note  on  two  problems  in  connexion  with  graphs.
<span 
class="cmti-12">Numerische Mathematik</span>, <span 
class="cmbx-12">1</span>(1): 269&#x2013;271.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDixon2009a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Dixon, S.J., Andrews, B.J., and Boone, C. (2009) Exploring the conservation of
synthetic lethal genetic interaction networks. <span 
class="cmti-12">Commun Integr Biol</span>, <span 
class="cmbx-12">2</span>(2): 78&#x2013;81.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDixon2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Dixon, S.J.,  Fedyshyn,  Y.,  Koh,  J.L.,  Prasad,  T.S.,  Chahwan,  C.,  Chua,  G.,
Toufighi, K.,  Baryshnikova,  A.,  Hayles,  J.,  Hoe,  K.L.,  <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.  </span>(2008)  Significant
conservation  of  synthetic  lethal  genetic  interaction  networks  between  distantly
related eukaryotes. <span 
class="cmti-12">Proc Natl Acad Sci U S A</span>, <span 
class="cmbx-12">105</span>(43): 16653&#x2013;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDorogovtsev2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Dorogovtsev, S.N. and Mendes, J.F. (2003) <span 
class="cmti-12">Evolution of networks: From biological</span>
<span 
class="cmti-12">nets to the Internet and WWW</span>. Oxford University Press, USA.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XErdos1959"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Erd<img 
src="20170511_thesis_introduction4x.png" alt="&#x02DD;o "  class="Huml"  />s, P. and R&#x00E9;nyi, A. (1959) On random graphs I.  <span 
class="cmti-12">Publ Math Debrecen</span>, <span 
class="cmbx-12">6</span>:
290&#x2013;297.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XErdos1960"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Erd<img 
src="20170511_thesis_introduction5x.png" alt="&#x02DD;o "  class="Huml"  />s, P. and R&#x00E9;nyi, A. (1960) On the evolution of random graphs.  In <span 
class="cmti-12">Publ.</span>
<span 
class="cmti-12">Math. Inst. Hung. Acad. Sci</span>, volume&#x00A0;5, 17&#x2013;61.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XEroles2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Eroles, P., Bosch, A., Perez-Fidalgo, J.A., and Lluch, A. (2012) Molecular biology
in breast cancer: intrinsic subtypes and signaling pathways. <span 
class="cmti-12">Cancer Treat Rev</span>, <span 
class="cmbx-12">38</span>(6):
698&#x2013;707.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XEzkurdia2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Ezkurdia, I., Juan, D., Rodriguez, J.M., Frankish, A., Diekhans, M., Harrow, J.,
Vazquez, J., Valencia, A., and Tress, M.L. (2014) Multiple evidence strands suggest
that there may be as few as 19 000 human protein-coding genes. <span 
class="cmti-12">Human Molecular</span>
<span 
class="cmti-12">Genetics</span>, <span 
class="cmbx-12">23</span>(22): 5866.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFarmer2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Farmer, H., McCabe, N., Lord, C.J., Tutt, A.N., Johnson, D.A., Richardson, T.B.,
Santarosa, M., Dillon, K.J., Hickson, I., Knights, C., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2005) Targeting the dna
repair defect in BRCA mutant cells as a therapeutic strategy.  <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">434</span>(7035):
917&#x2013;21.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFawcett2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Fawcett, T. (2006) An introduction to ROC analysis. <span 
class="cmti-12">Pattern Recognition Letters</span>,
<span 
class="cmbx-12">27</span>(8): 861 &#x2013; 874. <span 
class="cmsy-10x-x-120">{</span>ROC<span 
class="cmsy-10x-x-120">} </span>Analysis in Pattern Recognition.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFece2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Fece de&#x00A0;la  Cruz,  F.,  Gapp,  B.V.,  and  Nijman,  S.M.  (2015)  Synthetic  lethal
vulnerabilities of cancer. <span 
class="cmti-12">Annu Rev Pharmacol Toxicol</span>, <span 
class="cmbx-12">55</span>: 513&#x2013;531.
                                                                                    
                                                                                    
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFerlay2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Ferlay, J., Soerjomataram, I., Dikshit, R., Eser, S., Mathers, C., Rebelo, M., Parkin,
D.M., Forman, D., and Bray, F. (2015) Cancer incidence and mortality worldwide:
sources, methods and major patterns in GLOBOCAN 2012.  <span 
class="cmti-12">Int J Cancer</span>, <span 
class="cmbx-12">136</span>(5):
E359&#x2013;386.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFisher1919"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Fisher, R.A.  (1919)  Xv.the  correlation  between  relatives  on  the  supposition  of
mendelian inheritance. <span 
class="cmti-12">Earth and Environmental Science Transactions of the Royal</span>
<span 
class="cmti-12">Society of Edinburgh</span>, <span 
class="cmbx-12">52</span>(02): 399&#x2013;433.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFong2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Fong, P.C.,  Boss,  D.S.,  Yap,  T.A.,  Tutt,  A.,  Wu,  P.,  Mergui-Roelvink,  M.,
Mortimer, P., Swaisland, H., Lau, A., O&#x2019;Connor, M.J., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2009) Inhibition of
poly(adp-ribose) polymerase in tumors from BRCA mutation carriers.  <span 
class="cmti-12">N Engl J</span>
<span 
class="cmti-12">Med</span>, <span 
class="cmbx-12">361</span>(2): 123&#x2013;34.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFong2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Fong, P.C., Yap, T.A., Boss, D.S., Carden, C.P., Mergui-Roelvink, M., Gourley, C.,
De&#x00A0;Greve, J., Lubinski, J., Shanley, S., Messiou, C., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2010) Poly(adp)-ribose
polymerase inhibition: frequent durable responses in BRCA carrier ovarian cancer
correlating with platinum-free interval. <span 
class="cmti-12">J Clin Oncol</span>, <span 
class="cmbx-12">28</span>(15): 2512&#x2013;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XForbes2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Forbes, S.A.,  Beare,  D.,  Gunasekaran,  P.,  Leung,  K.,  Bindal,  N.,  Boutselakis,
H., Ding, M., Bamford, S., Cole, C., Ward, S., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2015) COSMIC: exploring
the world&#x2019;s knowledge of somatic mutations in human cancer.  <span 
class="cmti-12">Nucleic Acids Res</span>,
<span 
class="cmbx-12">43</span>(Database issue): D805&#x2013;811.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFraser2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Fraser, A.  (2004)  Towards  full  employment:  using  RNAi  to  find  roles  for  the
redundant. <span 
class="cmti-12">Oncogene</span>, <span 
class="cmbx-12">23</span>(51): 8346&#x2013;52.
                                                                                    
                                                                                    
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFutreal2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Futreal, P.A., Coin, L., Marshall, M., Down, T., Hubbard, T., Wooster, R., Rahman,
N., and Stratton, M.R. (2004) A census of human cancer genes.  <span 
class="cmti-12">Nat Rev Cancer</span>,
<span 
class="cmbx-12">4</span>(3): 177&#x2013;183.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFutreal2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Futreal, P.A., Kasprzyk, A., Birney, E., Mullikin, J.C., Wooster, R., and Stratton,
M.R. (2001) Cancer and genomics. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">409</span>(6822): 850&#x2013;852.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGao2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Gao, B. and  Roux,  P.P.  (2015)  Translational  control  by  oncogenic  signaling
pathways. <span 
class="cmti-12">Biochimica et Biophysica Acta</span>, <span 
class="cmbx-12">1849</span>(7): 753&#x2013;65.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGatza2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Gatza, M.L.,  Kung,  H.N.,  Blackwell,  K.L.,  Dewhirst,  M.W.,  Marks,  J.R.,  and
Chi, J.T. (2011) Analysis of tumor environmental response and oncogenic pathway
activation identifies distinct basal and luminal features in HER2-related breast tumor
subtypes. <span 
class="cmti-12">Breast Cancer Res</span>, <span 
class="cmbx-12">13</span>(3): R62.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGatza2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Gatza, M.L., Lucas, J.E., Barry, W.T., Kim, J.W., Wang, Q., Crawford, M.D.,
Datto, M.B., Kelley, M., Mathey-Prevot, B., Potti, A., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2010) A pathway-based
classification of human breast cancer. <span 
class="cmti-12">Proc Natl Acad Sci USA</span>, <span 
class="cmbx-12">107</span>(15): 6994&#x2013;6999.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGatza2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Gatza, M.L.,  Silva,  G.O.,  Parker,  J.S.,  Fan,  C.,  and  Perou,  C.M.  (2014)  An
integrated genomics approach identifies drivers of proliferation in luminal-subtype
human breast cancer. <span 
class="cmti-12">Nat Genet</span>, <span 
class="cmbx-12">46</span>(10): 1051&#x2013;1059.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGentleman2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Gentleman, R.C., Carey, V.J., Bates, D.M., Bolstad, B., Dettling, M., Dudoit, S.,
                                                                                    
                                                                                    
Ellis, B., Gautier, L., Ge, Y., Gentry, J., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2004) Bioconductor: open software
development for computational biology and bioinformatics.   <span 
class="cmti-12">Genome Biol</span>, <span 
class="cmbx-12">5</span>(10):
R80.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGenz2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Genz, A.  and  Bretz,  F.  (2009)  Computation  of  multivariate  normal  and
t probabilities.    In  <span 
class="cmti-12">Lecture  Notes  in  Statistics</span>,  volume  195.  Springer-Verlag,
Heidelberg.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xmvtnorm"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Genz, A., Bretz, F., Miwa, T., Mi, X., Leisch, F., Scheipl, F., and Hothorn, T.
(2016) <span 
class="cmti-12">mvtnorm: Multivariate Normal and t Distributions</span>. R package version 1.0-5.
URL.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGilbert1973"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Gilbert, W. and Maxam, A. (1973) The nucleotide sequence of the lac operator.
<span 
class="cmti-12">Proceedings of the National Academy of Sciences</span>, <span 
class="cmbx-12">70</span>(12): 3581&#x2013;3584.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGit2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Git, A., Dvinge,  H.,  Salmon-Divon,  M.,  Osborne,  M.,  Kutter,  C.,  Hadfield,
J., Bertone,  P.,  and  Caldas,  C.  (2010)  Systematic  comparison  of  microarray
profiling, real-time PCR, and next-generation sequencing technologies for measuring
differential microRNA expression. <span 
class="cmti-12">RNA</span>, <span 
class="cmbx-12">16</span>(5): 991&#x2013;1006.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGlobus"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Globus     (Globus)     (2017)     Research     data     management     simplified.
<a 
href="https://www.globus.org/" class="url" ><span 
class="cmtt-12">https://www.globus.org/</span></a>. Accessed: 25/03/2017.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGraziano2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Graziano,  F.,  Humar,  B.,  and  Guilford,  P.  (2003)  The  role  of  the  E-cadherin
gene (<span 
class="cmti-12">CDH1</span>) in diffuse gastric cancer susceptibility: from the laboratory to clinical
practice. <span 
class="cmti-12">Annals of Oncology</span>, <span 
class="cmbx-12">14</span>(12): 1705&#x2013;1713.
                                                                                    
                                                                                    
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGuell2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>G&#x00FC;ell, O., Sagus, F., and Serrano, M. (2014) Essential plasticity and redundancy
of metabolism unveiled by synthetic lethality analysis.  <span 
class="cmti-12">PLoS Comput Biol</span>, <span 
class="cmbx-12">10</span>(5):
e1003637.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGuilford1999"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Guilford, P. (1999) E-cadherin downregulation in cancer: fuel on the fire? <span 
class="cmti-12">Molecular</span>
<span 
class="cmti-12">Medicine Today</span>, <span 
class="cmbx-12">5</span>(4): 172 &#x2013; 177.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGuilford1998"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Guilford, P., Hopkins, J., Harraway, J., McLeod, M., McLeod, N., Harawira, P.,
Taite, H., Scoular,  R.,  Miller,  A.,  and  Reeve,  A.E.  (1998)  E-cadherin  germline
mutations in familial gastric cancer. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">392</span>(6674): 402&#x2013;5.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGuilford2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Guilford, P., Humar, B., and Blair, V. (2010) Hereditary diffuse gastric cancer:
translation of <span 
class="cmti-12">CDH1 </span>germline mutations into clinical practice. <span 
class="cmti-12">Gastric Cancer</span>, <span 
class="cmbx-12">13</span>(1):
1&#x2013;10.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHDGC"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Guilford, P.J., Hopkins, J.B., Grady, W.M., Markowitz, S.D., Willis, J., Lynch,
H., Rajput, A., Wiesner, G.L., Lindor, N.M., Burgart, L.J., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(1999) E-cadherin
germline mutations define an inherited cancer syndrome dominated by diffuse gastric
cancer. <span 
class="cmti-12">Hum Mutat</span>, <span 
class="cmbx-12">14</span>(3): 249&#x2013;55.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGuo2016"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Guo, J., Liu, H., and Zheng, J. (2016) SynLethDB: synthetic lethality database
toward discovery of selective and sensitive anticancer drug targets.  <span 
class="cmti-12">Nucleic Acids</span>
<span 
class="cmti-12">Res</span>, <span 
class="cmbx-12">44</span>(D1): D1011&#x2013;1017.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHajian-Tilaki2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Hajian-Tilaki, K. (2013) Receiver Operating Characteristic (ROC) Curve Analysis
for Medical Diagnostic Test Evaluation. <span 
class="cmti-12">Caspian J Intern Med</span>, <span 
class="cmbx-12">4</span>(2): 627&#x2013;635.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHall2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Hall, M., Frank, E., Holmes, G., Pfahringer, B., Reutemann, P., and Witten, I.H.
(2009) The weka data mining software: an update.  <span 
class="cmti-12">SIGKDD Explor Newsl</span>, <span 
class="cmbx-12">11</span>(1):
10&#x2013;18.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2012LUSC"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Hammerman, P.S., Lawrence, M.S., Voet, D., Jing, R., Cibulskis, K., Sivachenko,
A., Stojanov,   P.,   McKenna,   A.,   Lander,   E.S.,   Gabriel,   S.,   <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.   </span>(2012)
Comprehensive genomic characterization of squamous cell lung cancers.   <span 
class="cmti-12">Nature</span>,
<span 
class="cmbx-12">489</span>(7417): 519&#x2013;525.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHan2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Han, J.D.J.,  Bertin,  N.,  Hao,  T.,  Goldberg,  D.S.,  Berriz,  G.F.,  Zhang,  L.V.,
Dupuy, D., Walhout, A.J.M., Cusick, M.E., Roth, F.P., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2004) Evidence for
dynamically organized modularity in the yeast protein-protein interaction network.
<span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">430</span>(6995): 88&#x2013;93.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHanahan2000"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Hanahan, D. and Weinberg, R.A. (2000) The hallmarks of cancer.  <span 
class="cmti-12">Cell</span>, <span 
class="cmbx-12">100</span>(1):
57&#x2013;70.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHanahan2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Hanahan, D. and Weinberg, R.A. (2011) Hallmarks of cancer: the next generation.
<span 
class="cmti-12">Cell</span>, <span 
class="cmbx-12">144</span>(5): 646&#x2013;674.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCIX2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Hanna, S.   (2003)   Cancer   incidence   in   new   zealand   (2003-2007).       In
D.&#x00A0;Forman,  D.&#x00A0;Bray  F&#x00A0;Brewster,  C.&#x00A0;Gombe&#x00A0;Mbalawa,  B.&#x00A0;Kohler,  M.&#x00A0;Pi&#x00F1;eros,
                                                                                    
                                                                                    
E.&#x00A0;Steliarova-Foucher, R.&#x00A0;Swaminathan, and J.&#x00A0;Ferlay (editors), <span 
class="cmti-12">Cancer Incidence</span>
<span 
class="cmti-12">in Five Continents</span>,  volume&#x00A0;X,  902&#x2013;907.  International  Agency  for  Research
on Cancer,  Lyon,  France.     Electronic  version  <a 
href="http://ci5.iarc.fr" class="url" ><span 
class="cmtt-12">http://ci5.iarc.fr</span></a> Accessed
22/03/2017.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XcaArray2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Heiskanen,  M.,  Bian,  X.,  Swan,  D.,  and  Basu,  A.  (2014)  caArray  microarray
database in the cancer biomedical informatics grid<sup class="textsuperscript"><span 
class="cmr-10x-x-109">TM</span></sup> (caBIG<sup class="textsuperscript"><span 
class="cmr-10x-x-109">TM</span></sup>). <span 
class="cmti-12">Cancer Research</span>,
<span 
class="cmbx-12">67</span>(9 Supplement): 3712&#x2013;3712.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHeiskanen2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Heiskanen, M.A. and Aittokallio, T. (2012) Mining high-throughput screens for
cancer drug  targets-lessons  from  yeast  chemical-genomic  profiling  and  synthetic
lethality. <span 
class="cmti-12">Wiley Interdisciplinary Reviews: Data Mining and Knowledge Discovery</span>,
<span 
class="cmbx-12">2</span>(3): 263&#x2013;272.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHell1976"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Hell, P. (1976) Graphs with given neighbourhoods i. probl&#x00E9;mes combinatorics at
theorie des graphes. <span 
class="cmti-12">Proc Coil Int CNRS, Orsay</span>, <span 
class="cmbx-12">260</span>: 219&#x2013;223.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHerschkowitz2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Herschkowitz,  J.I.,  Simin,  K.,  Weigman,  V.J.,  Mikaelian,  I.,  Usary,  J.,  Hu,  Z.,
Rasmussen,  K.E.,  Jones,  L.P.,  Assefnia,  S.,  Chandrasekharan,  S.,  <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.  </span>(2007)
Identification  of  conserved  gene  expression  features  between  murine  mammary
carcinoma models and human breast tumors. <span 
class="cmti-12">Genome Biol</span>, <span 
class="cmbx-12">8</span>(5): R76.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHillenmeyer2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Hillenmeyer, M.E. (2008) The chemical genomic portrait of yeast: uncovering a
phenotype for all genes. <span 
class="cmti-12">Science</span>, <span 
class="cmbx-12">320</span>: 362&#x2013;365.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2014PAN"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Hoadley, K.A.,  Yau,  C.,  Wolf,  D.M.,  Cherniack,  A.D.,  Tamborero,  D.,  Ng,
                                                                                    
                                                                                    
S., Leiserson,  M.D.,  Niu,  B.,  McLellan,  M.D.,  Uzunangelov,  V.,  <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.  </span>(2014)
Multiplatform analysis of 12 cancer types reveals molecular classification within and
across tissues of origin. <span 
class="cmti-12">Cell</span>, <span 
class="cmbx-12">158</span>(4): 929&#x2013;944.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHoehndorf2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Hoehndorf, R., Hardy, N.W., Osumi-Sutherland, D., Tweedie, S., Schofield, P.N.,
and Gkoutos,  G.V.  (2013)  Systematic  analysis  of  experimental  phenotype  data
reveals gene functions. <span 
class="cmti-12">PLoS ONE</span>, <span 
class="cmbx-12">8</span>(4): e60847.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHolm1979"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Holm, S.   (1979)   A   simple   sequentially   rejective   multiple   test   procedure.
<span 
class="cmti-12">Scandinavian Journal of Statistics</span>, <span 
class="cmbx-12">6</span>(2): 65&#x2013;70.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHolme2002"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Holme, P.  and  Kim,  B.J.  (2002)  Growing  scale-free  networks  with  tunable
clustering. <span 
class="cmti-12">Physical Review E</span>, <span 
class="cmbx-12">65</span>(2): 026107.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHopkins2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Hopkins, A.L. (2008) Network pharmacology: the next paradigm in drug discovery.
<span 
class="cmti-12">Nat Chem Biol</span>, <span 
class="cmbx-12">4</span>(11): 682&#x2013;690.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHu2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Hu, Z., Fan, C., Oh, D.S., Marron, J.S., He, X., Qaqish, B.F., Livasy, C., Carey,
L.A., Reynolds,  E.,  Dressler,  L.,  <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2006)  The  molecular  portraits  of  breast
tumors are conserved across microarray platforms. <span 
class="cmti-12">BMC Genomics</span>, <span 
class="cmbx-12">7</span>: 96.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHuang2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Huang, E., Cheng, S., Dressman, H., Pittman, J., Tsou, M., Horng, C., Bild, A.,
Iversen, E., Liao, M., Chen, C., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2003) Gene expression predictors of breast
cancer outcomes. <span 
class="cmti-12">Lancet</span>, <span 
class="cmbx-12">361</span>: 1590&#x2013;1596.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XIllumina"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Illumina, Inc (Illumina) (2017) Sequencing and array-based solutions for genetic
research. <a 
href="https://www.illumina.com/" class="url" ><span 
class="cmtt-12">https://www.illumina.com/</span></a>. Accessed: 26/03/2017.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHapMap2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>International HapMap 3 Consortium (HapMap) (2003) The International HapMap
Project. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">426</span>(6968): 789&#x2013;796.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XIHGSC2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Internationl Human Genome Sequencing Consortium (IHGSC) (2004) Finishing
the euchromatic sequence of the human genome. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">431</span>(7011): 931&#x2013;945.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XJerby2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Jerby-Arnon, L., Pfetzer, N., Waldman, Y., McGarry, L., James, D., Shanks, E.,
Seashore-Ludlow, B., Weinstock, A., Geiger, T., Clemons, P., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2014) Predicting
cancer-specific vulnerability via data-driven detection of synthetic lethality.  <span 
class="cmti-12">Cell</span>,
<span 
class="cmbx-12">158</span>(5): 1199&#x2013;1209.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XJoachims1999"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Joachims, T. (1999) Making large-scale support vector machine learning practical.
In S.&#x00A0;Bernhard, lkopf, J.C.B. Christopher, and J.S. Alexander (editors), <span 
class="cmti-12">Advances</span>
<span 
class="cmti-12">in kernel methods</span>, 169&#x2013;184. MIT Press.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XJu2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Ju, Z., Liu, W., Roebuck, P.L., Siwak, D.R., Zhang, N., Lu, Y., Davies, M.A.,
Akbani, R.,  Weinstein,  J.N.,  Mills,  G.B.,  <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2015)  Development  of  a  robust
classifier for quality control of reverse-phase protein arrays.  <span 
class="cmti-12">Bioinformatics</span>, <span 
class="cmbx-12">31</span>(6):
912.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKaelin2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Kaelin, Jr, W. (2005) The concept of synthetic lethality in the context of anticancer
therapy. <span 
class="cmti-12">Nat Rev Cancer</span>, <span 
class="cmbx-12">5</span>(9): 689&#x2013;98.
                                                                                    
                                                                                    
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKaelin2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Kaelin, Jr, W. (2009) Synthetic lethality: a framework for the development of wiser
cancer therapeutics. <span 
class="cmti-12">Genome Med</span>, <span 
class="cmbx-12">1</span>: 99.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKakiuchi2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Kakiuchi,  M.,  Nishizawa,  T.,  Ueda,  H.,  Gotoh,  K.,  Tanaka,  A.,  Hayashi,  A.,
Yamamoto,  S.,  Tatsuno,  K.,  Katoh,  H.,  Watanabe,  Y.,  <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.  </span>(2014)  Recurrent
gain-of-function mutations of RHOA in diffuse-type gastric carcinoma.  <span 
class="cmti-12">Nat Genet</span>,
<span 
class="cmbx-12">46</span>(6): 583&#x2013;587.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKamada1989"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Kamada, T. and Kawai, S. (1989) An algorithm for drawing general undirected
graphs. <span 
class="cmti-12">Information Processing Letters</span>, <span 
class="cmbx-12">31</span>(1): 7&#x2013;15.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2013ENDO"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Kandoth,  C.,  Schultz,  N.,  Cherniack,  A.D.,  Akbani,  R.,  Liu,  Y.,  Shen,  H.,
Robertson, A.G., Pashtan, I., Shen, R., Benz, C.C., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2013) Integrated genomic
characterization of endometrial carcinoma. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">497</span>(7447): 67&#x2013;73.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFANTOM2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Kawai, J., Shinagawa, A., Shibata, K., Yoshino, M., Itoh, M., Ishii, Y., Arakawa,
T., Hara, A., Fukunishi, Y., Konno, H., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2001) Functional annotation of a
full-length mouse cDNA collection. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">409</span>(6821): 685&#x2013;690.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKelley2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Kelley, R. and Ideker, T. (2005) Systematic interpretation of genetic interactions
using protein networks. <span 
class="cmti-12">Nat Biotech</span>, <span 
class="cmbx-12">23</span>(5): 561&#x2013;566.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKellyBMC"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Kelly, S., Chen, A., Guilford, P., and Black, M. (2017a) Synthetic lethal interaction
prediction of target pathways in E-cadherin deficient breast cancers.  Submitted to
<span 
class="cmti-12">BMC Genomics</span>.
                                                                                    
                                                                                    
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKelly2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Kelly, S.T. (2013) <span 
class="cmti-12">Statistical Predictions of Synthetic Lethal Interactions in Cancer</span>.
Dissertation, University of Otago.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKellyHDGC"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Kelly, S.T.,  Single,  A.B.,  Telford,  B.J.,  Beetham,  H.G.,  Godwin,  T.D.,  Chen,
A., Black,  M.A.,  and  Guilford,  P.J.  (2017b)  Towards  HDGC  chemoprevention:
vulnerabilities in E-cadherin-negative cells identified by genome-wide interrogation
of isogenic cell lines and whole tumors. Submitted to <span 
class="cmti-12">Cancer Prev Res</span>.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKozlov2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Kozlov, K.N.,  Gursky,  V.V.,  Kulakovskiy,  I.V.,  and  Samsonova,  M.G.  (2015)
Sequence-based model of gap gene regulation network.  <span 
class="cmti-12">BMC Genomics</span>, <span 
class="cmbx-12">15</span>(Suppl
12): S6.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKranthi2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Kranthi, S., Rao, S., and Manimaran, P. (2013) Identification of synthetic lethal
pairs in biological systems through network information centrality.  <span 
class="cmti-12">Mol BioSyst</span>,
<span 
class="cmbx-12">9</span>(8): 2163&#x2013;2167.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLander2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Lander, E.S. (2011) Initial impact of the sequencing of the human genome. <span 
class="cmti-12">Nature</span>,
<span 
class="cmbx-12">470</span>(7333): 187&#x2013;197.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLander2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Lander, E.S., Linton, L.M., Birren, B., Nusbaum, C., Zody, M.C., Baldwin, J.,
Devon, K., Dewar, K., Doyle, M., FitzHugh, W., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2001) Initial sequencing and
analysis of the human genome. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">409</span>(6822): 860&#x2013;921.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xbowtie"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Langmead,  B.,  Trapnell,  C.,  Pop,  M.,  and  Salzberg,  S.L.  (2009)  Ultrafast  and
                                                                                    
                                                                                    
memory-efficient alignment of short DNA sequences to the human genome. <span 
class="cmti-12">Genome</span>
<span 
class="cmti-12">Biol</span>, <span 
class="cmbx-12">10</span>(3): R25.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLatora2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Latora, V. and Marchiori, M. (2001) Efficient behavior of small-world networks.
<span 
class="cmti-12">Phys Rev Lett</span>, <span 
class="cmbx-12">87</span>: 198701.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLaufer2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Laufer, C.,  Fischer,  B.,  Billmann,  M.,  Huber,  W.,  and  Boutros,  M.  (2013)
Mapping genetic interactions in human cancer cells with RNAi and multiparametric
phenotyping. <span 
class="cmti-12">Nat Methods</span>, <span 
class="cmbx-12">10</span>(5): 427&#x2013;31.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLaw2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Law, C.W., Chen, Y., Shi, W., and Smyth, G.K. (2014) voom: precision weights
unlock linear model analysis tools for RNA-seq read counts.  <span 
class="cmti-12">Genome Biol</span>, <span 
class="cmbx-12">15</span>(2):
R29.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2015HNSC"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Lawrence, M.S., Sougnez, C., Lichtenstein, L., Cibulskis, K., Lander, E., Gabriel,
S.B., Getz, G., Ally, A., Balasundaram, M., Birol, I., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2015) Comprehensive
genomic characterization  of  head  and  neck  squamous  cell  carcinomas.   <span 
class="cmti-12">Nature</span>,
<span 
class="cmbx-12">517</span>(7536): 576&#x2013;582.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLeMeur2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Le&#x00A0;Meur, N. and Gentleman, R. (2008) Modeling synthetic lethality. <span 
class="cmti-12">Genome Biol</span>,
<span 
class="cmbx-12">9</span>(9): R135.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLeMeur2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Le&#x00A0;Meur, N., Jiang, Z., Liu, T., Mar, J., and Gentleman, R.C. (2014) Slgi: Synthetic
lethal genetic interaction. r package version 1.26.0.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLee2010a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Lee, A.Y., Perreault, R., Harel, S., Boulier, E.L., Suderman, M., Hallett, M., and
Jenna, S. (2010a) Searching for signaling balance through the identification of genetic
interactors of the rab guanine-nucleotide dissociation inhibitor gdi-1.  <span 
class="cmti-12">PLoS ONE</span>,
<span 
class="cmbx-12">5</span>(5): e10624.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLee2010b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Lee, I., Lehner, B., Vavouri, T., Shin, J., Fraser, A.G., and Marcotte, E.M. (2010b)
Predicting genetic modifier loci using functional gene networks.  <span 
class="cmti-12">Genome Research</span>,
<span 
class="cmbx-12">20</span>(8): 1143&#x2013;1153.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLee2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Lee, I. and Marcotte, E.M. (2009) Effects of functional bias on supervised learning
of a gene network model. <span 
class="cmti-12">Methods Mol Biol</span>, <span 
class="cmbx-12">541</span>: 463&#x2013;75.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLee2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Lee, M.J., Ye, A.S., Gardino, A.K., Heijink, A.M., Sorger, P.K., MacBeath, G., and
Yaffe, M.B. (2012) Sequential application of anticancer drugs enhances cell death by
rewiring apoptotic signaling networks. <span 
class="cmti-12">Cell</span>, <span 
class="cmbx-12">149</span>(4): 780&#x2013;94.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLehner2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Lehner, B.,  Crombie,  C.,  Tischler,  J.,  Fortunato,  A.,  and  Fraser,  A.G.  (2006)
Systematic  mapping  of  genetic  interactions  in  caenorhabditis  elegans  identifies
common modifiers of diverse signaling pathways. <span 
class="cmti-12">Nat Genet</span>, <span 
class="cmbx-12">38</span>(8): 896&#x2013;903.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLi2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Li, X.J., Mishra, S.K., Wu, M., Zhang, F., and Zheng, J. (2014) Syn-lethality:
An integrative knowledge base of synthetic lethality towards discovery of selective
anticancer therapies. <span 
class="cmti-12">Biomed Res Int</span>, <span 
class="cmbx-12">2014</span>: 196034.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2016PRC"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Linehan, W.M., Spellman, P.T., Ricketts, C.J., Creighton, C.J., Fei, S.S., Davis, C.,
                                                                                    
                                                                                    
Wheeler, D.A., Murray, B.A., Schmidt, L., Vocke, C.D., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2016) Comprehensive
Molecular Characterization  of  Papillary  Renal-Cell  Carcinoma.   <span 
class="cmti-12">N  Engl  J  Med</span>,
<span 
class="cmbx-12">374</span>(2): 135&#x2013;145.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLokody2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Lokody, I. (2014) Computational modelling: A computational crystal ball. <span 
class="cmti-12">Nature</span>
<span 
class="cmti-12">Reviews Cancer</span>, <span 
class="cmbx-12">14</span>(10): 649&#x2013;649.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLord2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Lord, C.J., Tutt, A.N., and Ashworth, A. (2015) Synthetic lethality and cancer
therapy: lessons learned from the development of PARP inhibitors. <span 
class="cmti-12">Annu Rev Med</span>,
<span 
class="cmbx-12">66</span>: 455&#x2013;470.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLu2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Lu, X., Kensche,  P.R.,  Huynen,  M.A.,  and  Notebaart,  R.A.  (2013)  Genome
evolution predicts genetic interactions in protein complexes and reveals cancer drug
targets. <span 
class="cmti-12">Nat Commun</span>, <span 
class="cmbx-12">4</span>: 2124.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLu2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Lu, X., Megchelenbrink, W., Notebaart, R.A., and Huynen, M.A. (2015) Predicting
human genetic  interactions  from  cancer  genome  evolution.    <span 
class="cmti-12">PLoS  One</span>,  <span 
class="cmbx-12">10</span>(5):
e0125795.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLum2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Lum, P.Y.,  Armour,  C.D.,  Stepaniants,  S.B.,  Cavet,  G.,  Wolf,  M.K.,  Butler,
J.S., Hinshaw,  J.C.,  Garnier,  P.,  Prestwich,  G.D.,  Leonardson,  A.,  <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2004)
Discovering modes of action for therapeutic compounds using a genome-wide screen
of yeast heterozygotes. <span 
class="cmti-12">Cell</span>, <span 
class="cmbx-12">116</span>(1): 121&#x2013;137.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLuo2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Luo, J., Solimini, N.L., and Elledge, S.J. (2009) Principles of Cancer Therapy:
Oncogene and Non-oncogene Addiction. <span 
class="cmti-12">Cell</span>, <span 
class="cmbx-12">136</span>(5): 823&#x2013;837.
                                                                                    
                                                                                    
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMachado2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Machado, J., Olivera, C., Carvalh, R., Soares, P., Berx, G., Caldas, C., Sercuca, R.,
Carneiro, F., and Sorbrinho-Simoes, M. (2001) E-cadherin gene (<span 
class="cmti-12">CDH1</span>) promoter
methylation as the second hit in sporadic diffuse gastric carcinoma.  <span 
class="cmti-12">Oncogene</span>, <span 
class="cmbx-12">20</span>:
1525&#x2013;1528.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMasciari2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Masciari, S., Larsson, N., Senz, J., Boyd, N., Kaurah, P., Kandel, M.J., Harris,
L.N., Pinheiro, H.C., Troussard, A., Miron, P., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2007) Germline E-cadherin
mutations in familial lobular breast cancer. <span 
class="cmti-12">J Med Genet</span>, <span 
class="cmbx-12">44</span>(11): 726&#x2013;31.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMattison2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Mattison, J., van&#x00A0;der Weyden, L., Hubbard, T., and Adams, D.J. (2009) Cancer
gene discovery in mouse and man. <span 
class="cmti-12">Biochim Biophys Acta</span>, <span 
class="cmbx-12">1796</span>(2): 140&#x2013;161.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMaxam1977"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Maxam, A.M.  and  Gilbert,  W.  (1977)  A  new  method  for  sequencing  DNA.
<span 
class="cmti-12">Proceedings of the National Academy of Science</span>, <span 
class="cmbx-12">74</span>(2): 560&#x2013;564.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMcCourt2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>McCourt, C.M., McArt, D.G., Mills, K., Catherwood, M.A., Maxwell, P., Waugh,
D.J., Hamilton,  P.,  O&#x2019;Sullivan,  J.M.,  and  Salto-Tellez,  M.  (2013)  Validation  of
next generation sequencing technologies in comparison to current diagnostic gold
standards for BRAF, EGFR and KRAS mutational analysis.   <span 
class="cmti-12">PLoS ONE</span>, <span 
class="cmbx-12">8</span>(7):
e69604.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMcLachlan2016"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>McLachlan, J., George, A., and Banerjee, S. (2016) The current status of parp
inhibitors in ovarian cancer. <span 
class="cmti-12">Tumori</span>, <span 
class="cmbx-12">102</span>(5): 433&#x2013;440.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2008GBM"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>McLendon,   R.,   Friedman,   A.,   Bigner,   D.,   Van&#x00A0;Meir,   E.G.,   Brat,   D.J.,
Mastrogianakis, G.M., Olson, J.J., Mikkelsen, T., Lehman, N., Aldape, K., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al.</span>
(2008) Comprehensive genomic characterization defines human glioblastoma genes
and core pathways. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">455</span>(7216): 1061&#x2013;1068.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMiles2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Miles, D.W. (2001) Update on HER-2 as a target for cancer therapy: herceptin in
the clinical setting. <span 
class="cmti-12">Breast Cancer Res</span>, <span 
class="cmbx-12">3</span>(6): 380&#x2013;384.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMortazavi2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Mortazavi,  A.,  Williams,  B.A.,  McCue,  K.,  Schaeffer,  L.,  and  Wold,  B.  (2008)
Mapping and quantifying mammalian transcriptomes by RNA-Seq.  <span 
class="cmti-12">Nat Methods</span>,
<span 
class="cmbx-12">5</span>(7): 621&#x2013;628.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2012CRC"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Muzny, D.M.,  Bainbridge,  M.N.,  Chang,  K.,  Dinh,  H.H.,  Drummond,  J.A.,
Fowler, G., Kovar, C.L., Lewis, L.R., Morgan, M.B., Newsham, I.F., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2012)
Comprehensive molecular characterization of human colon and rectal cancer. <span 
class="cmti-12">Nature</span>,
<span 
class="cmbx-12">487</span>(7407): 330&#x2013;337.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNagalla2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Nagalla, S., Chou, J.W., Willingham, M.C., Ruiz, J., Vaughn, J.P., Dubey, P.,
Lash, T.L., Hamilton-Dutoit, S.J., Bergh, J., Sotiriou, C., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2013) Interactions
between immunity, proliferation and molecular subtype in breast cancer prognosis.
<span 
class="cmti-12">Genome Biol</span>, <span 
class="cmbx-12">14</span>(4): R34.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNeeley2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Neeley, E.S., Kornblau, S.M., Coombes, K.R., and Baggerly, K.A. (2009) Variable
slope normalization of reverse phase protein arrays. <span 
class="cmti-12">Bioinformatics</span>, <span 
class="cmbx-12">25</span>(11): 1384.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xmatrixcalc"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Novomestky, F. (2012) <span 
class="cmti-12">matrixcalc: Collection of functions for matrix calculations</span>.
R package version 1.0-3.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XOliveira2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Oliveira, C., Senz, J., Kaurah, P., Pinheiro, H., Sanges, R., Haegert, A., Corso, G.,
Schouten, J., Fitzgerald, R., Vogelsang, H., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2009) Germline <span 
class="cmti-12">CDH1 </span>deletions
in hereditary  diffuse  gastric  cancer  families.   <span 
class="cmti-12">Human Molecular Genetics</span>,  <span 
class="cmbx-12">18</span>(9):
1545&#x2013;1555.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XOliveira2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Oliveira, C., Seruca, R., Hoogerbrugge, N., Ligtenberg, M., and Carneiro, F. (2013)
Clinical utility gene card for: Hereditary diffuse gastric cancer (HDGC). <span 
class="cmti-12">Eur J Hum</span>
<span 
class="cmti-12">Genet</span>, <span 
class="cmbx-12">21</span>(8).
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPandey2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Pandey, G., Zhang, B., Chang, A.N., Myers, C.L., Zhu, J., Kumar, V., and Schadt,
E.E. (2010) An integrative multi-network and multi-classifier approach to predict
genetic interactions. <span 
class="cmti-12">PLoS Comput Biol</span>, <span 
class="cmbx-12">6</span>(9).
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XParker2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Parker, J., Mullins, M., Cheung, M., Leung, S., Voduc, D., Vickery, T., Davies, S.,
Fauron, C., He, X., Hu, Z., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2009) Supervised risk predictor of breast cancer
based on intrinsic subtypes. <span 
class="cmti-12">Journal of Clinical Oncology</span>, <span 
class="cmbx-12">27</span>(8): 1160&#x2013;1167.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPeltonen2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Peltonen, L. and McKusick, V.A. (2001) Genomics and medicine. Dissecting human
disease in the postgenomic era. <span 
class="cmti-12">Science</span>, <span 
class="cmbx-12">291</span>(5507): 1224&#x2013;1229.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMETABRIC2016"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Pereira, B., Chin, S.F., Rueda, O.M., Vollan, H.K., Provenzano, E., Bardwell, H.A.,
Pugh, M., Jones, L., Russell, R., Sammut, S.J., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2016) Erratum: The somatic
                                                                                    
                                                                                    
mutation profiles of 2,433 breast cancers refine their genomic and transcriptomic
landscapes. <span 
class="cmti-12">Nat Commun</span>, <span 
class="cmbx-12">7</span>: 11908.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPerou2000"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Perou, C.M., S&#x00F8;rlie, T., Eisen, M.B., van&#x00A0;de Rijn, M., Jeffrey, S.S., Rees, C.A.,
Pollack, J.R., Ross, D.T., Johnsen, H., Akslen, L.A., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2000) Molecular portraits
of human breast tumours. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">406</span>(6797): 747&#x2013;752.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPleasance2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Pleasance, E.D., Cheetham, R.K., Stephens, P.J., McBride, D.J., Humphray, S.J.,
Greenman, C.D., Varela, I., Lin, M.L., Ordonez, G.R., Bignell, G.R., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2010) A
comprehensive catalogue of somatic mutations from a human cancer genome. <span 
class="cmti-12">Nature</span>,
<span 
class="cmbx-12">463</span>(7278): 191&#x2013;196.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPolyak2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Polyak, K.  and  Weinberg,  R.A.  (2009)  Transitions  between  epithelial  and
mesenchymal states: acquisition of malignant and stem cell traits. <span 
class="cmti-12">Nat Rev Cancer</span>,
<span 
class="cmbx-12">9</span>(4): 265&#x2013;73.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPrahallad2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Prahallad, A., Sun, C., Huang, S., Di&#x00A0;Nicolantonio, F., Salazar, R., Zecchin, D.,
Beijersbergen, R.L., Bardelli, A., and Bernards, R. (2012) Unresponsiveness of colon
cancer to <span 
class="cmti-12">BRAF</span>(v600e)  inhibition  through  feedback  activation  of  egfr.   <span 
class="cmti-12">Nature</span>,
<span 
class="cmbx-12">483</span>(7387): 100&#x2013;3.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XR_core"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>R Core Team (2016) <span 
class="cmti-12">R: A Language and Environment for Statistical Computing</span>.
R Foundation for Statistical Computing, Vienna, Austria. R version 3.3.2.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRavnan2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Ravnan, M.C. and Matalka, M.S. (2012) Vemurafenib in patients with <span 
class="cmti-12">BRAF </span>v600e
mutation-positive advanced melanoma. <span 
class="cmti-12">Clin Ther</span>, <span 
class="cmbx-12">34</span>(7): 1474&#x2013;86.
                                                                                    
                                                                                    
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xlimma"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Ritchie, M.E., Phipson, B., Wu, D., Hu, Y., Law, C.W., Shi, W., and Smyth,
G.K. (2015) limma powers differential expression analyses for RNA-sequencing and
microarray studies. <span 
class="cmti-12">Nucleic Acids Research</span>, <span 
class="cmbx-12">43</span>(7): e47.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRobin2016"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Robin, J.D., Ludlow, A.T., LaRanger, R., Wright, W.E., and Shay, J.W. (2016)
Comparison of DNA Quantification Methods for Next Generation Sequencing.  <span 
class="cmti-12">Sci</span>
<span 
class="cmti-12">Rep</span>, <span 
class="cmbx-12">6</span>: 24067.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRobinson2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Robinson,  M.D.  and  Oshlack,  A.  (2010)  A  scaling  normalization  method  for
differential expression analysis of RNA-seq data. <span 
class="cmti-12">Genome Biol</span>, <span 
class="cmbx-12">11</span>(3): R25.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRoguev2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Roguev, A., Bandyopadhyay, S., Zofall, M., Zhang, K., Fischer, T., Collins, S.R.,
Qu, H., Shales, M., Park, H.O., Hayles, J., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2008) Conservation and rewiring of
functional modules revealed by an epistasis map in fission yeast. <span 
class="cmti-12">Science</span>, <span 
class="cmbx-12">322</span>(5900):
405&#x2013;10.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRung2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Rung, J. and Brazma, A. (2013) Reuse of public genome-wide gene expression data.
<span 
class="cmti-12">Nat Rev Genet</span>, <span 
class="cmbx-12">14</span>(2): 89&#x2013;99.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XArrayExpress2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Rustici, G., Kolesnikov, N., Brandizi, M., Burdett, T., Dylag, M., Emam, I., Farne,
A., Hastings, E., Ison, J., Keays, M., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2013) ArrayExpress update&#x2013;trends in
database growth and links to data analysis tools.  <span 
class="cmti-12">Nucleic Acids Res</span>, <span 
class="cmbx-12">41</span>(Database
issue): D987&#x2013;990.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRyan2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Ryan, C., Lord, C., and Ashworth, A. (2014) Daisy: Picking synthetic lethals from
cancer genomes. <span 
class="cmti-12">Cancer Cell</span>, <span 
class="cmbx-12">26</span>(3): 306&#x2013;308.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSander2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Sander, J.D. and Joung, J.K. (2014) Crispr-cas systems for editing, regulating and
targeting genomes. <span 
class="cmti-12">Nat Biotechnol</span>, <span 
class="cmbx-12">32</span>(4): 347&#x2013;55.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSanger1975"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Sanger, F. and Coulson, A. (1975) A rapid method for determining sequences in
dna by primed synthesis with dna polymerase. <span 
class="cmti-12">Journal of Molecular Biology</span>, <span 
class="cmbx-12">94</span>(3):
441 &#x2013; 448.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XScheuer2002"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Scheuer, L., Kauff, N., Robson, M., Kelly, B., Barakat, R., Satagopan, J., Ellis, N.,
Hensley, M., Boyd, J., Borgen, P., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2002) Outcome of preventive surgery and
screening for breast and ovarian cancer in BRCA mutation carriers.  <span 
class="cmti-12">J Clin Oncol</span>,
<span 
class="cmbx-12">20</span>(5): 1260&#x2013;1268.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSemb1998"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Semb, H. and Christofori, G. (1998) The tumor-suppressor function of E-cadherin.
<span 
class="cmti-12">Am J Hum Genet</span>, <span 
class="cmbx-12">63</span>(6): 1588&#x2013;93.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XROCR"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Sing, T., Sander, O., Beerenwinkel, N., and Lengauer, T. (2005) Rocr: visualizing
classifier performance in r. <span 
class="cmti-12">Bioinformatics</span>, <span 
class="cmbx-12">21</span>(20): 7881.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xslurm"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Slurm development    team    (Slurm)    (2017)    Slurm    workload    manager.
<a 
href="https://slurm.schedmd.com/" class="url" ><span 
class="cmtt-12">https://slurm.schedmd.com/</span></a>. Accessed: 25/03/2017.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSorlie2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S&#x00F8;rlie, T., Perou, C.M., Tibshirani, R., Aas, T., Geisler, S., Johnsen, H., Hastie, T.,
Eisen, M.B., van&#x00A0;de Rijn, M., Jeffrey, S.S., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2001) Gene expression patterns of
breast carcinomas distinguish tumor subclasses with clinical implications. <span 
class="cmti-12">Proc Natl</span>
<span 
class="cmti-12">Acad Sci USA</span>, <span 
class="cmbx-12">98</span>(19): 10869&#x2013;10874.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XStajich2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Stajich, J.E. and Lapp, H. (2006) Open source tools and toolkits for bioinformatics:
significance, and where are we? <span 
class="cmti-12">Brief Bioinformatics</span>, <span 
class="cmbx-12">7</span>(3): 287&#x2013;296.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XStratton2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Stratton, M.R.,  Campbell,  P.J.,  and  Futreal,  P.A.  (2009)  The  cancer  genome.
<span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">458</span>(7239): 719&#x2013;724.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XStrom2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Str&#x00F6;m, C.  and  Helleday,  T.  (2012)  Strategies  for  the  use  of  poly(adenosine
diphosphate ribose) polymerase (parp) inhibitors in cancer therapy.  <span 
class="cmti-12">Biomolecules</span>,
<span 
class="cmbx-12">2</span>(4): 635&#x2013;649.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSun2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Sun, C., Wang, L., Huang, S., Heynen, G.J.J.E., Prahallad, A., Robert, C., Haanen,
J., Blank, C., Wesseling, J., Willems, S.M., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2014) Reversible and adaptive
resistance to <span 
class="cmti-12">BRAF</span>(v600e) inhibition in melanoma. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">508</span>(7494): 118&#x2013;122.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTaylor2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Taylor, I.W.,  Linding,  R.,  Warde-Farley,  D.,  Liu,  Y.,  Pesquita,  C.,  Faria,  D.,
Bull, S., Pawson, T., Morris, Q., and Wrana, J.L. (2009) Dynamic modularity in
protein interaction networks predicts breast cancer outcome. <span 
class="cmti-12">Nat Biotechnol</span>, <span 
class="cmbx-12">27</span>(2):
199&#x2013;204.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTelford2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Telford, B.J., Chen, A., Beetham, H., Frick, J., Brew, T.P., Gould, C.M., Single, A.,
                                                                                    
                                                                                    
Godwin, T., Simpson, K.J., and Guilford, P. (2015) Synthetic lethal screens identify
vulnerabilities in gpcr signalling and cytoskeletal organization in E-cadherin-deficient
cells. <span 
class="cmti-12">Mol Cancer Ther</span>, <span 
class="cmbx-12">14</span>(5): 1213&#x2013;1223.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="X1000Genomes2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>The 1000 Genomes Project Consortium (1000 Genomes) (2010) A map of human
genome variation from population-scale sequencing. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">467</span>(7319): 1061&#x2013;1073.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>The Cancer  Genome  Atlas  Research  Network  (TCGA)  (2012)  Comprehensive
molecular portraits of human breast tumours. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">490</span>(7418): 61&#x2013;70.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2017web"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>The Cancer Genome Atlas Research Network (TCGA) (2017a) The Cancer Genome
Atlas Project. <a 
href="https://cancergenome.nih.gov/" class="url" ><span 
class="cmtt-12">https://cancergenome.nih.gov/</span></a>. Accessed: 26/03/2017.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2017prov"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>The Cancer  Genome  Atlas  Research  Network  (TCGA)  (2017b)  The  Cancer
Genome Atlas Project Data Portal. <a 
href="https://tcga-data.nci.nih.gov/" class="url" ><span 
class="cmtt-12">https://tcga-_data.nci.nih.gov/</span></a>. Accessed:
06/02/2017 (via cBioPortal.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCSNZ2017"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>The Cancer Society of New Zealand (Cancer Society of NZ) (2017) What is cancer?
<a 
href="https://otago-southland.cancernz.org.nz/en/cancer-information/other-links/what-is-cancer-3/" class="url" ><span 
class="cmtt-12">https://otago-_southland.cancernz.org.nz/en/cancer-_information/other-_links/what-_is-_cancer-_3/</span></a>.
Accessed: 22/03/2017.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCOSMICdb"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>The Catalogue Of Somatic Mutations In Cancer (COSMIC) (2016) Cosmic: The
catalogue of somatic mutations in cancer.  <a 
href="http://cancer.sanger.ac.uk/cosmic" class="url" ><span 
class="cmtt-12">http://cancer.sanger.ac.uk/cosmic</span></a>.
Release 79 (23/08/2016), Accessed: 05/02/2017.
</p>
                                                                                    
                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCRAN"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>The Comprehensive     R     Archive     Network     (CRAN)     (2017)     Cran.
<a 
href="https://cran.r-project.org/" class="url" ><span 
class="cmtt-12">https://cran.r-_project.org/</span></a>. Accessed: 24/03/2017.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XENCODE2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>The ENCODE   Project   Consortium   (ENCODE)   (2004)   The   ENCODE
(ENCyclopedia Of DNA Elements) Project. <span 
class="cmti-12">Science</span>, <span 
class="cmbx-12">306</span>(5696): 636&#x2013;640.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XICGC2017web"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>The Internation Cancer Genome Consortium (ICGC) (2017) ICGC Data Portal.
<a 
href="https://dcc.icgc.org/" class="url" ><span 
class="cmtt-12">https://dcc.icgc.org/</span></a>. Accessed: 06/02/2017.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNCI2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>The National   Cancer   Institute   (NCI)   (2015)   The   genetics   of   cancer.
<a 
href="https://www.cancer.gov/about-cancer/causes-prevention/genetics" class="url" ><span 
class="cmtt-12">https://www.cancer.gov/about-_cancer/causes-_prevention/genetics</span></a>.
Published: 22/04/2015, Accessed: 22/03/2017.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNeSI"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>The New     Zealand     eScience     Infrastructure     (NeSI)     (2017)     NeSI.
<a 
href="https://www.nesi.org.nz/" class="url" ><span 
class="cmtt-12">https://www.nesi.org.nz/</span></a>. Accessed: 25/03/2017.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPharmac2016"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>The Pharmaceutical  Management  Agency  (PHARMAC)  (2016)  Approval  of
multi-product funding proposal with roche.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xsnow"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Tierney, L., Rossini, A.J., Li, N., and Sevcikova, H. (2015) <span 
class="cmti-12">snow: Simple Network</span>
<span 
class="cmti-12">of Workstations</span>. R package version 0.4-2.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTiong2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Tiong, K.L., Chang, K.C., Yeh, K.T., Liu, T.Y., Wu, J.H., Hsieh, P.H., Lin, S.H.,
                                                                                    
                                                                                    
Lai, W.Y., Hsu, Y.C., Chen, J.Y., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2014) Csnk1e/ctnnb1 are synthetic lethal
to tp53 in colorectal cancer and are markers for prognosis. <span 
class="cmti-12">Neoplasia</span>, <span 
class="cmbx-12">16</span>(5): 441&#x2013;50.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTischler2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Tischler, J., Lehner, B., and Fraser, A.G. (2008) Evolutionary plasticity of genetic
interaction networks. <span 
class="cmti-12">Nat Genet</span>, <span 
class="cmbx-12">40</span>(4): 390&#x2013;391.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTomasetti2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Tomasetti, C. and Vogelstein, B. (2015) Cancer etiology. Variation in cancer risk
among tissues  can  be  explained  by  the  number  of  stem  cell  divisions.   <span 
class="cmti-12">Science</span>,
<span 
class="cmbx-12">347</span>(6217): 78&#x2013;81.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTong2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Tong, A.H.,  Evangelista,  M.,  Parsons,  A.B.,  Xu,  H.,  Bader,  G.D.,  Page,  N.,
Robinson, M., Raghibizadeh, S., Hogue, C.W., Bussey, H., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2001) Systematic
genetic analysis with ordered arrays of yeast deletion mutants. <span 
class="cmti-12">Science</span>, <span 
class="cmbx-12">294</span>(5550):
2364&#x2013;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTong2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Tong, A.H., Lesage, G., Bader, G.D., Ding, H., Xu, H., Xin, X., Young, J., Berriz,
G.F., Brost, R.L., Chang, M., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2004) Global mapping of the yeast genetic
interaction network. <span 
class="cmti-12">Science</span>, <span 
class="cmbx-12">303</span>(5659): 808&#x2013;13.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTravers1969"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Travers, J.  and  Milgram,  S.  (1969)  An  experimental  study  of  the  small  world
problem. <span 
class="cmti-12">Sociometry</span>, <span 
class="cmbx-12">32</span>(4): 425&#x2013;443.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTsai2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Tsai, H.C.,  Li,  H.,  Van&#x00A0;Neste,  L.,  Cai,  Y.,  Robert,  C.,  Rassool,  F.V.,  Shin,
J.J., Harbom, K.M., Beaty, R., Pappou, E., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2012) Transient low doses of
dna-demethylating  agents  exert  durable  antitumor  effects  on  hematological  and
epithelial tumor cells. <span 
class="cmti-12">Cancer Cell</span>, <span 
class="cmbx-12">21</span>(3): 430&#x2013;46.
                                                                                    
                                                                                    
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTutt2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Tutt, A.,  Robson,  M.,  Garber,  J.E.,  Domchek,  S.M.,  Audeh,  M.W.,  Weitzel,
J.N., Friedlander, M., Arun, B., Loman, N., Schmutzler, R.K., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2010) Oral
poly(adp-ribose) polymerase inhibitor olaparib in patients with <span 
class="cmti-12">BRCA1 </span>or <span 
class="cmti-12">BRCA2</span>
mutations and advanced breast cancer: a proof-of-concept trial. <span 
class="cmti-12">Lancet</span>, <span 
class="cmbx-12">376</span>(9737):
235&#x2013;44.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XvanderMeer2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>van&#x00A0;der Meer, R., Song, H.Y., Park, S.H., Abdulkadir, S.A., and Roh, M. (2014)
RNAi screen identifies a synthetic lethal interaction between PIM1 overexpression
and PLK1 inhibition. <span 
class="cmti-12">Clinical Cancer Research</span>, <span 
class="cmbx-12">20</span>(12): 3211&#x2013;3221.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XvanSteen2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>van Steen, K. (2012) Travelling the world of genegene interactions.  <span 
class="cmti-12">Briefings in</span>
<span 
class="cmti-12">Bioinformatics</span>, <span 
class="cmbx-12">13</span>(1): 1&#x2013;19.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XvanSteen2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>van Steen,  M.  (2010)  <span 
class="cmti-12">Graph  Theory  and  Complex  Networks:  An  Introduction</span>.
Maarten van Steen, VU Amsterdam.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVapnik1995"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Vapnik, V.N. (1995) <span 
class="cmti-12">The nature of statistical learning theory</span>. Springer-Verlag New
York, Inc.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVargas2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Vargas, J.J., Gusella, G., Najfeld, V., Klotman, M., and Cara, A. (2004) Novel
integrase-defective lentiviral episomal vectors for gene transfer. <span 
class="cmti-12">Hum Gene Ther</span>, <span 
class="cmbx-12">15</span>:
361&#x2013;372.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVizeacoumar2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Vizeacoumar, F.J., Arnold, R., Vizeacoumar, F.S., Chandrashekhar, M., Buzina,
                                                                                    
                                                                                    
A., Young, J.T., Kwan, J.H., Sayad, A., Mero, P., Lawo, S., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2013) A negative
genetic interaction map in isogenic cancer cell lines reveals cancer cell vulnerabilities.
<span 
class="cmti-12">Mol Syst Biol</span>, <span 
class="cmbx-12">9</span>: 696.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVogelstein2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Vogelstein,  B.,  Papadopoulos,  N.,  Velculescu,  V.E.,  Zhou,  S.,  Diaz,  L.A.,  and
Kinzler, K.W. (2013) Cancer genome landscapes. <span 
class="cmti-12">Science</span>, <span 
class="cmbx-12">339</span>(6127): 1546&#x2013;1558.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVos1997"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Vos, C.B., Cleton-Jansen, A.M., Berx, G., de&#x00A0;Leeuw, W.J., ter Haar, N.T., van
Roy, F., Cornelisse, C.J., Peterse, J.L., and van&#x00A0;de Vijver, M.J. (1997) E-cadherin
inactivation in lobular carcinoma in situ of the breast: an early event in tumorigenesis.
<span 
class="cmti-12">Br J Cancer</span>, <span 
class="cmbx-12">76</span>(9): 1131&#x2013;3.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xmapsplice"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Wang, K.,  Singh,  D.,  Zeng,  Z.,  Coleman,  S.J.,  Huang,  Y.,  Savich,  G.L.,  He,
X., Mieczkowski, P., Grimm, S.A., Perou, C.M., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2010) MapSplice: accurate
mapping of RNA-seq reads for splice junction discovery. <span 
class="cmti-12">Nucleic Acids Res</span>, <span 
class="cmbx-12">38</span>(18):
e178.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWang2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Wang, K., Yuen, S.T., Xu, J., Lee, S.P., Yan, H.H., Shi, S.T., Siu, H.C., Deng,
S., Chu, K.M., Law, S., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2014) Whole-genome sequencing and comprehensive
molecular profiling identify new driver mutations in gastric cancer. <span 
class="cmti-12">Nat Genet</span>, <span 
class="cmbx-12">46</span>(6):
573&#x2013;582.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWang2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Wang, X. and Simon, R. (2013) Identification of potential synthetic lethal genes to
p53 using a computational biology approach. <span 
class="cmti-12">BMC Medical Genomics</span>, <span 
class="cmbx-12">6</span>(1): 30.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWappett2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Wappett,  M.  (2014)  Bisep:  Toolkit  to  identify  candidate  synthetic  lethality.  r
package version 2.0.
                                                                                    
                                                                                    
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWappett2016"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Wappett, M., Dulak, A., Yang, Z.R., Al-Watban, A., Bradford, J.R., and Dry, J.R.
(2016) Multi-omic measurement of mutually exclusive loss-of-function enriches for
candidate synthetic lethal gene pairs. <span 
class="cmti-12">BMC Genomics</span>, <span 
class="cmbx-12">17</span>: 65.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xgplots"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Warnes, G.R., Bolker, B., Bonebakker, L., Gentleman, R., Liaw, W.H.A., Lumley,
T., Maechler, M., Magnusson, A., Moeller, S., Schwartz, M., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2015) <span 
class="cmti-12">gplots:</span>
<span 
class="cmti-12">Various R Programming Tools for Plotting Data</span>. R package version 2.17.0.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWatts1998"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Watts, D.J.  and  Strogatz,  S.H.  (1998)  Collective  dynamics  of  &#x2019;small-world&#x2019;
networks. <span 
class="cmti-12">Nature</span>, <span 
class="cmbx-12">393</span>(6684): 440&#x2013;2.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWeinstein2000"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Weinstein, I.B. (2000) Disorders in cell circuitry during multistage carcinogenesis:
the role of homeostasis. <span 
class="cmti-12">Carcinogenesis</span>, <span 
class="cmbx-12">21</span>(5): 857&#x2013;864.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2014BL"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Weinstein, J.N., Akbani, R., Broom, B.M., Wang, W., Verhaak, R.G., McConkey,
D., Lerner, S., Morgan, M., Creighton, C.J., Smith, C., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2014) Comprehensive
molecular characterization  of  urothelial  bladder  carcinoma.   <span 
class="cmti-12">Nature</span>,  <span 
class="cmbx-12">507</span>(7492):
315&#x2013;322.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTCGA2013PAN"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Weinstein, J.N., Collisson, E.A., Mills, G.B., Shaw, K.R., Ozenberger, B.A., Ellrott,
K., Shmulevich, I., Sander, C., Stuart, J.M., Chang, K., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2013) The Cancer
Genome Atlas Pan-Cancer analysis project. <span 
class="cmti-12">Nat Genet</span>, <span 
class="cmbx-12">45</span>(10): 1113&#x2013;1120.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xdevtools"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Wickham, H. and Chang, W. (2016) <span 
class="cmti-12">devtools: Tools to Make Developing R Packages</span>
<span 
class="cmti-12">Easier</span>. R package version 1.12.0.
                                                                                    
                                                                                    
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="Xroxygen"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Wickham,   H.,   Danenberg,   P.,   and   Eugster,   M.   (2017)   <span 
class="cmti-12">roxygen2:  In-Line</span>
<span 
class="cmti-12">Documentation for R</span>. R package version 6.0.1.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWong2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Wong, S.L., Zhang, L.V., Tong, A.H.Y., Li, Z., Goldberg, D.S., King, O.D., Lesage,
G., Vidal, M., Andrews, B., Bussey, H., <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2004) Combining biological networks
to predict genetic interactions.  <span 
class="cmti-12">Proceedings of the National Academy of Sciences of</span>
<span 
class="cmti-12">the United States of America</span>, <span 
class="cmbx-12">101</span>(44): 15682&#x2013;15687.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWorldHealthOrg2017"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>World              Health              Organization              (WHO)              (2017)
Fact sheet: Cancer.  <a 
href="http://www.who.int/mediacentre/factsheets/fs297/en/" class="url" ><span 
class="cmtt-12">http://www.who.int/mediacentre/factsheets/fs297/en/</span></a>.
Updated February 2017, Accessed: 22/03/2017.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWu2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Wu, M., Li, X., Zhang, F., Li, X., Kwoh, C.K., and Zheng, J. (2014) In silico
prediction of synthetic lethality by meta-analysis of genetic interactions, functions,
and pathways in yeast and human cancer. <span 
class="cmti-12">Cancer Inform</span>, <span 
class="cmbx-12">13</span>(Suppl 3): 71&#x2013;80.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRmpi"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Yu, H. (2002) Rmpi: Parallel statistical computing in r. <span 
class="cmti-12">R News</span>, <span 
class="cmbx-12">2</span>(2): 10&#x2013;14.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XZhang2015"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Zhang, F., Wu, M., Li, X.J., Li, X.L., Kwoh, C.K., and Zheng, J. (2015) Predicting
essential genes and synthetic lethality via influence propagation in signaling pathways
of cancer cell fates. <span 
class="cmti-12">J Bioinform Comput Biol</span>, <span 
class="cmbx-12">13</span>(3): 1541002.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XICGC2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Zhang, J.,  Baran,  J.,  Cros,  A.,  Guberman,  J.M.,  Haider,  S.,  Hsu,  J.,  Liang,
Y., Rivkin,  E.,  Wang,  J.,  Whitty,  B.,  <span 
class="cmti-12">et</span><span 
class="cmti-12">&#x00A0;al. </span>(2011)  International  cancer  genome
                                                                                    
                                                                                    
consortium data portala one-stop shop for cancer genomics data.  <span 
class="cmti-12">Database: The</span>
<span 
class="cmti-12">Journal of Biological Databases and Curation</span>, <span 
class="cmbx-12">2011</span>: bar026.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XZhong2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Zhong, W. and Sternberg, P.W. (2006) Genome-wide prediction of c. elegans genetic
interactions. <span 
class="cmti-12">Science</span>, <span 
class="cmbx-12">311</span>(5766): 1481&#x2013;1484.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XZweig1993"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Zweig, M.H. and Campbell, G. (1993) Receiver-operating characteristic (roc) plots:
a fundamental  evaluation  tool  in  clinical  medicine.   <span 
class="cmti-12">Clinical  Chemistry</span>,  <span 
class="cmbx-12">39</span>(4):
561&#x2013;577.
</p>
</div>
                                                                                    
                                                                                    
 
</body></html> 

                                                                                    
                                                                                    
                                                                                    


