%Background
\textbf{Background}

Synthetic lethal interactions are re-emerging in genetics research in the genomics era driven by potential applications in precision medicine against cancers. This approach aims to exploit functional redundancy at the genetic level against mutations in cancers for developing specific treatments against them, including loss of function events in tumour suppressors. Of particular interest is the targeting loss of function of E-cadherin, encoded by \textit{CDH1}, a tumour supressor gene involved in Breast and Stomach cancers. Experimental screens have been used to identify candidate synthetic lethal interactions and here bioinformatics analysis used to augment the triage drug target triage process. Furthermore the pathway composition of synthetic lethal candidates and the effect of pathway structure on their detection in genomics data. 

%Methods \ Approach
\textbf{Approach}

A computational statistics methodology, the Synthetic Lethal Prediction Tool (SLIPT) has been developed to detect synthetic lethal interactions in gene expression datasets. The methodology has been demonstrated on Breast and Stomach cancer datasets from The Cancer Genome Atlas (TCGA) database, testing for interactions with \textit{CDH1}. Various analyses have been applied to further euclidate these candidates, including differential gene expression, correlation co-expression, unsupervised clustering, gene set over-representation analysis, singular-value decomposition ``metagenes'', and permutation re-sampling analysis. A particular challenge of performing these analyses was to compare SLIPT gene candidates to the results of an experimental synthetic lethal siRNA screen of E-cadherin Telford \textit{et al.} (2015) at the pathway level. Graph theory methods including information centrality and shortest paths were applied to the most supported pathways from both the computational and experimental synthetic lethal candidates to test for graph structure among hits from each approach. Simulation and modelling was performed to test the statistical performance of the SLIPT methodology and further applied to datasets with simulated correlation structures, including those derived from known graph stuctures.

%Results \ Findings
\textbf{Findings}

%chapt 3
A vast number of genes having expression consistent with being synthetic lethal partners of \textit{CDH1} were detected in both TCGA Breast and Stomach cancer genes. For breast cancers, these genes clustered into several distinct groups, with distinct enriched biological functions and elevated expression in different clinical subclasses such as normal-like, basal, or estrogen receptor negative samples. While the number of genes detected by both computational and experimental approaches were not significant, there was significant pathway composition in the overlapping genes. In particular $G_{\alpha i}$ signalling, cytoplasmic microfibres, and extracelluar fibrin clotting were supported by both approaches even after permutation testing. These findings are consistent with the known roles of E-cadherin in cytoskeletal or cell signalling roles and the proposed downstream targets of GCPR singalling of Telford \textit{et al}. (2015). Many of these and related pathways were replicated in the separate stomach cancer dataset. Furthermore other candidate pathways uniquely supported by the computational predictions included regulation of immune signaling and translational elongation, both unlikely to have been detected with high dose siRNA in an isogenic cell line and these are still candidates for further testing in mouse xenograft models. 

%chapt 4
A number of approaches were adapted or developed to test whether there was a connection between synthetic lethal candidates in the graph structures of the pathways most supported by prior analyses. Network centrality measures were used to compare the importance or connectivity of genes in the pathway subnetworks but no significant difference was found between synthetic candidates and other genes within the same pathway. Another hypothesis was that computational synthetic lethal candidates would be downstream of experimental candidates within a pathway but no evidence of directionality between the candidates was detected. 

%chapt 5

A model of synthetic lethality was developed and was sucessfully implemented to simulate gene expression datasets with known underlying synthetic lethal partners of a query gene. For small numbers of known synthetic lethal partners, the SLIPT methdology performed well respect to reciever operator characteristic curves. As the number of true partners to detect increases, the power to detect them diminishes. Increasing sample sizes, however, was able to mitigate this effect somewhat as expected. This finding was replicated in simulations up to a feasible number of human genes (20,000) with more true negatives and correlations structures. The SLIPT methdology performs similarly across these conditions and performs better than Pearson's correlation (for co-expression) of the $\chi^2$-test without a directional criterion.  However, correlation structure of the dataset does impact on synthetic lethal predictions, genes correlated with (or in a pathway structure near to) true synthetic lethal partners having elevated test statistic values over other true negatives. A quadratic (second order polynomial) least squares linear regression methodology has been developed as a comparable alternative with the added benefit of conditioning against known partners (or strongest candidates prior analyses).

%Conclusions

Thus my thesis has developed, evaluated and refined a bioinformatics approach to discovery of synthetic lethal genes solely from gene expression data.  

\textit{Publications during Candidature}

Several publications have been prepared during the candidature of this thesis, including some findings related to the thesis topic on which we elaborate in more depth here. Please see the original article for the results which have been accepted for publication by peer-review:

Kelly, S. T. and Spencer, H. G. (2017) Population-Genetics Models of Sex-Limited Genomic Imprinting. \textit{Theoretical Population Biology}

Kelly, S. T., Chen, A., Guilford, P. J., and Black, M. A. (2017) Synthetic lethal interaction prediction of target pathways in E-cadherin deficient breast cancers. (Manuscript in Preparation for \textit{BMC Genomics})

\textit{Software Packages during Candidature}

Several software packages in the R language have been released on GitHub while preparing this thesis. Please see the appropriate GitHub repository for more information on installing and running these packages, on the following account: \url{https://github.com/TomKellyGenetics}

\href{https://github.com/TomKellyGenetics/slipt}{\texttt{slipt}} is the Synthetic Lethal interaction Prediction Tool, released to accompany the synthetic lethal publication above. \href{https://github.com/TomKellyGenetics/slipt-app}{\texttt{slipt-app}} contains an application developed in the R \texttt{shiny} environment as part of a related project.

Several plotting functions were customised for the Figures in this thesis (and the above publications), notably \href{https://github.com/TomKellyGenetics/heatmap.2x}{\texttt{heatmap.2x}} and \href{https://github.com/TomKellyGenetics/vioplotx}{\texttt{vioplotx}} have been prepared largely for use during this project but are also documented and available to other R users. These are enhancements to the CRAN \texttt{gplots} and \texttt{vioplot} packages respectively and are intended be user-friendly for those familiar with \texttt{heatmap.2} or \texttt{vioplot} and \texttt{boxplot} (base R) functions. These are backwards compatible with these functions, taking similar imputs as demonstrated in the appropriate vignettes.

The use of iGraph (the R \texttt{igraph} package) operations of graph-network structure in the analysis and simulations of pathways involved several original or customised functions to manipuate or plot \texttt{igraph} objects and adjacency matrices. These can be install separately from their respective repositories of with the metapackage: \href{https://github.com/TomKellyGenetics/igraph.extensions}{\texttt{igraph.extensions}}. \href{https://github.com/TomKellyGenetics/plot.igraph}{\texttt{plot.igraph}} enables plotting graph networks with customised inhibitor arrow and node colours.\href{https://github.com/TomKellyGenetics/info.centrality}{\texttt{info.centrality}} enables the calculation of additional node and network centrality metrics not available in the \texttt{igraph} package. \href{https://github.com/TomKellyGenetics/pathway.structure.permutation}{\texttt{pathway\-.structure\-.permutation}} enables testing of related states or node groups in a network by directionality of shortest paths. The \href{https://github.com/TomKellyGenetics/graphsim}{\texttt{graphsim}} package has been set up to simulate a multi-variate normal gene expression dataset with \texttt{mvtnorm} while deriving the correlation structure, $\Sigma$, from a graph structure. Note that these require various packages for graph theory, statistics and matrix operations and these will be installed as dependencies.

