\chapter{Simulation and Modeling of Synthetic Lethal Pathways}
\label{chap:simulation}

Simulation and modelling of synthetic lethality in gene expression will be revisited in greater detail in this chapter, building upon the results provided to support the use of \gls{SLIPT} in Section~\ref{chapt2:simulation_2015}. A simulation procedure for generating simulated data with underlying (known) synthetic lethal partners of a query gene, such as \textit{CDH1}, was developed (as described in Section~\ref{methods:simulating_SL}) by sampling from a Multivariate normal distribution based on a statistical model of synthetic lethality in expression data (as described in Section~\ref{methods:SL_Model}). This simulation framework was applied to simulated data (in Section~\ref{chapt2:simulation_2015}), including simple correlation structures to assess the statistical performance of the \gls{SLIPT} methodology and support it's use a computational approach for detecting synthetic lethal candidates from expression data throughout this thesis (in Chapters~\ref{chap:SLIPT} and~\ref{chap:Pathways}). 

While this basic framework was sufficent to support the use of \gls{SLIPT} in prior Chapters, further investigations with simulations were conducted to assess the strengths and limitations of the \gls{SLIPT} methodology, compare it to alternative statistical approaches to synthetic lethal detection, and assess it's performance upon more complex correlation structures. Together these simulation investigations assess the performance of the \gls{SLIPT} methdology, including on pathway graph structures (such as those discussed in Chapter~\ref{chap:Pathways}) and determine whether the \gls{SLIPT} methdology (or similar refined bioinformatics strategies) are statistically rigourous or suitable for wider genomics applications.

These simulation investigations continue to utilise the Multivariate Normal procedure (as applied in Section~\ref{chapt2:simulation_2015}) with further refinements. The \gls{SLIPT} methodology (and it's equivalent $\chi^2$ test) were applied across a range of parameters (including altering the quantiles for detecting synthetic lethal direction and compared correlation. This was also applied to with query correlated genes (as performed in Section~\ref{chapt2:simulation_2015}).

A refined simulation procedure was developed specifically to extend the simulation procedure (described in Section~\ref{methods:simulation_SL_expression}) to utilise pathway graph structures for the correlation structures of simulated datasets (as described in Section~\ref{methods:graphsim}). This methdology can be applied to simulated correlation structures across simple graph structures to test specfic network modules or use pathway structures based on biological pathways (as discussed in Chapter~\ref{chap:Pathways}). Thus graph structure and simulation approaches were combined to test whether a gene locus in a pathway affects detection by \gls{SLIPT} and whether \gls{SLIPT} performance is affected by pathway structure. The simulation procedure based on graph structures were applied in a computational pipeline across many parameters with high-performance computing (as discussed in Section~\ref{methods:HPC}) and the core simulation functions have been released as a software package for wider use to test bioinformatics and statistical methods on graph structures (as described in Section~\ref{methods:igraph_extensions}).

%\section{Simulations and Modelling Synthetic Lethality in Expression Data}
%%committee
\iffalse
Synthetic lethality was modelled for effects on expression levels and whether these are detectable in known interacting and non-interacting genes in simulated data. These were conducted for expression data but the nature of these simulations would be relevant to how synthetic lethality would manifest in other factors, particularly DNA copy number variation and DNA methylation. These simulations were discussed at length in the previous meeting and showed that synthetic lethality was detectable with our approach in simple cases. While it was less effective, the methods were able to detect synthetic lethal genes in expression data with correlation structure (generated with the multi-variate normal distribution) and were distinguishable from correlated genes. Therefore the strongest (most significant) synthetic lethal genes are more likely to be true synthetic lethal partners and a high number of hits are expected from correlated genes and co-regulated pathways.

The power of the method to detect interactions depleted with increasing multiple tests, interactions, and cryptic (third party) interacting partners. Increased sample size counteracted these effects as expected. This led the idea that pathways would be more suitable as the focus of this project. Biological pathways led to fewer multiple tests, more relevant to understanding cancer biology, and are often drug targets in practice.
\fi

\section{Comparing methods} \label{chap5:compare_ methods}

Methods were compared \ldots

\subsection{Performance of SLIPT and $\chi^2$  across Quantiles}

Text

\subsubsection{Correlated Query Genes affects Specificity}

Text

\subsection{Correlation as a Synthetic Lethal Detection Strategy}

Text

\subsection{Testing for Bimodality with BiSEp}

Text


%\subsection{Linear models}

\iffalse
\section{Developing a linear model predictor of synthetic lethality}
\subsection{Linear models}
\subsection{Polynomial models}
\subsection{Conditioning}
\subsection{SLIPTv2}
\fi

\section{Simulations with Graph Structures}

Text

\begin{figure*}[!htb]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subfigure[Activating pathway structure]{%
            \label{fig:simple_graph1:first}
            \fbox{
            \includegraphics[width=0.45\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1".pdf}}
            }
        }%
        \subfigure[Pathway structure including inhibitions]{%
           \label{fig:simple_graph1:second}
            \fbox{
            \includegraphics[width=0.45\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1i".pdf}}
           }
        }%
%
    \end{center}
   \caption[Simulated graph structures]{\small \textbf{\textbf{Simulated graph structures.}} A constructed graph structure used as an example to demonstrate the simulation procedure. Activating links are denoted by blue arrows and inhibiting links by red edges.}
%}
\label{fig:simple_graph1}
\end{mdframed}
\end{figure*}

\begin{figure*}[!htb]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subfigure[Activating pathway structure]{%
            \label{fig:simple_graph2:first}
            \fbox{
            \includegraphics[width=0.45\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph2".pdf}}
            }
        }%
        \subfigure[Pathway structure including inhibitions]{%
           \label{fig:simple_graph2:second}
            \fbox{
            \includegraphics[width=0.45\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph2i".pdf}}
           }
        }%
%
    \end{center}
   \caption[Simulated graph structures]{\small \textbf{\textbf{Simulated graph structures.}} A constructed graph structure used as an example to demonstrate the simulation procedure. Activating links are denoted by blue arrows and inhibiting links by red edges.}
%}
\label{fig:simple_graph2}
\end{mdframed}
\end{figure*}

\begin{figure*}[!htb]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subfigure[Activating pathway structure]{%
            \label{fig:simple_graph3:first}
            \fbox{
            \includegraphics[width=0.45\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph3".pdf}}
            }
        }%
        \subfigure[Pathway structure including inhibitions]{%
           \label{fig:simple_graph3:second}
            \fbox{
            \includegraphics[width=0.45\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph3i".pdf}}
           }
        }%
%
    \end{center}
   \caption[Simulated graph structures]{\small \textbf{\textbf{Simulated graph structures.}} A constructed graph structure used as an example to demonstrate the simulation procedure. Activating links are denoted by blue arrows and inhibiting links by red edges.}
%}
\label{fig:simple_graph3}
\end{mdframed}
\end{figure*}

\begin{figure*}[!htb]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subfigure[Activating pathway structure]{%
            \label{fig:simple_graph4:first}
            \fbox{
            \includegraphics[width=0.45\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4".pdf}}
            }
        }%
        \subfigure[Pathway structure including inhibitions]{%
           \label{fig:simple_graph4:second}
            \fbox{
            \includegraphics[width=0.45\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4i".pdf}}
           }
        }%

        \subfigure[Pathway structure including inhibitions]{%
           \label{fig:simple_graph4:third}
            \fbox{
            \includegraphics[width=0.45\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4i2".pdf}}
           }
        }%
        %
    \end{center}
   \caption[Simulated graph structures]{\small \textbf{\textbf{Simulated graph structures.}} A constructed graph structure used as an example to demonstrate the simulation procedure. Activating links are denoted by blue arrows and inhibiting links by red edges.}
%}
\label{fig:simple_graph4}
\end{mdframed}
\end{figure*}

\begin{figure*}[!htb]
\begin{mdframed}
\begin{center}
  \resizebox{0.9 \textwidth}{!}{
    \fbox{\includegraphics{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph5".pdf}}
   }
   \end{center}
   \caption[Simulated graph structures]{\small \textbf{\textbf{Simulated graph structures.}} A constructed graph structure used as an example to demonstrate the simulation procedure. Activating links are denoted by blue arrows and inhibiting links by red edges.}
\label{fig:simple_graph5}
\end{mdframed}
\end{figure*}

\begin{figure*}[!htb]
\begin{mdframed}
\begin{center}
  \resizebox{0.9 \textwidth}{!}{
    \fbox{\includegraphics{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph7".pdf}}
   }
   \end{center}
   \caption[Simulated graph structures]{\small \textbf{\textbf{Simulated graph structures.}} A constructed graph structure used as an example to demonstrate the simulation procedure. Activating links are denoted by blue arrows and inhibiting links by red edges.}
\label{fig:simple_graph7}
\end{mdframed}
\end{figure*}

\begin{figure*}[!htb]
\begin{mdframed}
\begin{center}
  \resizebox{0.9 \textwidth}{!}{
    \fbox{\includegraphics{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph7i".pdf}}
   }
   \end{center}
   \caption[Simulated graph structures]{\small \textbf{\textbf{Simulated graph structures.}} A constructed graph structure used as an example to demonstrate the simulation procedure. Activating links are denoted by blue arrows and inhibiting links by red edges.}
\label{fig:simple_graph7i}
\end{mdframed}
\end{figure*}

\begin{figure*}[!htb]
\begin{mdframed}
\begin{center}
  \resizebox{0.9 \textwidth}{!}{
    \fbox{\includegraphics{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph7i2".pdf}}
   }
   \end{center}
   \caption[Simulated graph structures]{\small \textbf{\textbf{Simulated graph structures.}} A constructed graph structure used as an example to demonstrate the simulation procedure. Activating links are denoted by blue arrows and inhibiting links by red edges.}
\label{fig:simple_graph7i2}
\end{mdframed}
\end{figure*}

\begin{figure*}[!htb]
\begin{mdframed}
\begin{center}
  \resizebox{0.9 \textwidth}{!}{
    \fbox{\includegraphics{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph_pi3k".pdf}}
   }
   \end{center}
   \caption[Simulated graph structures]{\small \textbf{\textbf{Simulated graph structures.}} A constructed graph structure used as an example to demonstrate the simulation procedure. Activating links are denoted by blue arrows and inhibiting links by red edges.}
\label{fig:simple_graph_pi3k1}
\end{mdframed}
\end{figure*}

\begin{figure*}[!htb]
\begin{mdframed}
\begin{center}
  \resizebox{0.9 \textwidth}{!}{
    \fbox{\includegraphics{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph_Gai".pdf}}
   }
   \end{center}
   \caption[Simulated graph structures]{\small \textbf{\textbf{Simulated graph structures.}} A constructed graph structure used as an example to demonstrate the simulation procedure. Activating links are denoted by blue arrows and inhibiting links by red edges.}
\label{fig:simple_graph_Gai1}
\end{mdframed}
\end{figure*}

\subsection{Performance over a Graph Structure}

%Graph1 cf Graph 2
%Graph 4 - RUN0921

Text

\subsection{Synthetic Lethality across Graph Stuctures}

%Graph 4 - RUN1109 STR

Text

\subsection{Performance with inhibition links}

%Graph4 v 4i v 4i2 RUN0921

Text

\subsection{Performance with 20,000 genes}

%Graph4 & 4i RUN0921 v RUN0914

Text

\section{Simulations over pathway-based graphs}

%PI3K and GAI

Text


\section{Discussion}

Text


\section{Summary}

Text


\clearpage

\paragraph{Aims}

  \begin{itemize}
   \item A Model of Synthetic Lethal Genes in Gene Expression Data
   
   \bigskip
   
   \item Comparison of SLIPT to Alternative Approaches
   
   \bigskip
   
   \item Simulations of Known Synthetic Lethal Genes within Pathway Networks
      
  \end{itemize}

\paragraph{Summary}

    \begin{itemize}
      \item We have designed a straight-forward rational query-based synthetic lethal detection method with the example of application to \textit{CDH1} in cancer gene expression
      
      \bigskip
      
      \item I have developed a simulation pipeline to generate continuous gene expression with pathway structure including a procedure to simulate synthetic lethality 
      
      \bigskip
      
      \item The simulation procedure shows that SLIPT is robust across pathway structures and has desirable performance compared to other statistical techniques 
      \end{itemize}