\chapter{Synthetic Lethal Pathway Structure}
\label{chap:Pathways}
  
Having identified key \glspl{pathway} implicated in \gls{synthetic lethal} genetic interactions with \textit{CDH1} (in Chapter~\ref{chap:SLIPT}), these were investigated in this Chapter for the \gls{synthetic lethal} genes within them, and for their relationships to \glslink{graph}{pathway} structure. This chapter will focus on the Reactome biological \glspl{pathway} detected across analyses in Chapter~\ref{chap:SLIPT}. 
%
%The \gls{synthetic lethal} genes identified were further examined within the context of biological \glspl{pathway}. 
Specifically, investigations were performed to determine whether \gls{synthetic lethal} candidates, detected by \gls{SLIPT} or \gls{siRNA}, exhibited differences with respect to metrics of \glslink{graph}{pathway} structure of network connectivity and importance (as described in Sections~\ref{methods:network_metrics} and~\ref{methods:igraph_extensions}). The relationships between \gls{synthetic lethal} candidates, detected by either approach, were also examined to determine whether \gls{SLIPT} candidate genes were upstream or downstream \gls{siRNA} candidate genes. These directional relationships were tested by resampling (as described in Sections~\ref{methods:pathway_str} and~\ref{methods:network_permutation}) and comparisons to the \gls{pathway} hierarchical score based on biological context (as derived in Section~\ref{methods:pathway_rank}). 
%
%The \gls{pathway} relationships between \gls{SLIPT} and \gls{siRNA} \gls{synthetic lethal} gene candidate partners for \textit{CDH1} were examined within the biological \glspl{pathway} identified previously (in Chapter~\ref{chap:SLIPT}).
Together these investigations into structural relationships demonstrate how a combination of network biology and statistical techniques can be performed with genes identified by a \gls{bioinformatics} analysis.

\FloatBarrier

\section{Synthetic Lethal Genes in Reactome Pathways} \label{chapt4:SL_Genes}

\FloatBarrier

The \glslink{graph}{graph} structure for Reactome \glspl{pathway} was obtained from Pathway Commons via \gls{BioPAX} (as described in Section~\ref{methods:graph_data}). The \glspl{pathway} describe the (directional) relationships between biomolecules, including genes that encode proteins in biological \glspl{pathway}. These relationships include cell signalling (e.g., kinase phosphorylation cascades), gene regulation (e.g., transcription factors, chromatin modifiers, \acrshort{RNA} binding proteins), and metabolism (e.g., the product of an enzyme being the substrate of another). Together these relationships describe the known functional \glspl{pathway} in a human cell with a reasonable resolution, from a curated database supported by publications documenting \gls{pathway} relationships. 

Pathway structures from the Reactome network (as described in Section~\ref{methods:subgraphs}) were used to derive the \glslink{graph}{graph} structure of each biological \gls{pathway}. The \gls{synthetic lethal} candidate genes for notable \glspl{pathway} discussed in Chapter~\ref{chap:SLIPT}, including candidate \gls{synthetic lethal} \glspl{pathway} of \textit{CDH1}, were examined to show the \gls{SLIPT} and \gls{siRNA} candidates within these \glspl{pathway}. The \gls{synthetic lethal} genes considered here are those candidates detected by \gls{SLIPT} (as described in Section~\ref{methods:SLIPT}) in \gls{TCGA} breast cancer \glslink{gene expression}{expression} and \gls{mutation} data \citep{TCGA2012} in comparison to the candidate gene partners from the \gls{siRNA} screening in breast cell lines \citep{Telford2015}. 

\FloatBarrier

\subsection{The PI3K/AKT Pathway}  \label{chapt4:SL_Genes_PI3K}

\FloatBarrier

The \acrfull{PI3K} cascade signalling \gls{pathway} is important in cancer because it is involved in mediating signals between the \glspl{GPCR} and regulation of protein translation, which have both been strongly implicated to be \gls{synthetic lethal} \glspl{pathway} with loss of \textit{CDH1} function (Chapter~\ref{chap:SLIPT}). These \glspl{pathway} are all subject to dysregulation in cancer \citep{Dorsam2007, Courtney2010, Gao2015}. Thus the PI3K cascade will be examined along with the most supported \gls{synthetic lethal} \glspl{pathway} (as identified in Chapter~\ref{chap:SLIPT}). %It also exhibited a relationship with \textit{CDH1} mutations in \gls{metagene} analyses (in Appendix~\ref{chapt3:metagene_results}).

The \gls{PI3K} \gls{pathway} is well characterised and has an established direction of signal transduction from extracellular stimuli (and membrane bound receptors) to the inner mechanisms of the cell, namely, the regulation of protein translation. The production of proteins is necessary for the growth of the cell so it is reasonable to suggest that these processes may be subject to (non-\gls{oncogene}) addiction in some cancer cells which rely upon them for sustained protein production and cell growth. This is also supported by the \glspl{oncogene} \textit{PIK3CA} and \textit{AKT1} being involved with the PI3K cascade and the related PI3K/AKT \gls{pathway}, which may be subject to \gls{oncogene addiction} when these proto-oncogenes are activated.

\begin{figure*}[!tb]
%\begin{mdframed}
  \begin{center}
  \resizebox{1 \textwidth}{!}{
    %\input{{{"SL_Model.pdf_tex"}}
    %\fbox{
    \includegraphics{{"/home/tomkelly/Downloads/Pathway_Structure/graph_plot_Pi3k_exprSL2".pdf}}
   %}
   }
   \end{center}
   \caption[Synthetic lethality in the PI3K cascade]{\small \textbf{Synthetic lethality in the PI3K cascade.} The Reactome \gls{PI3K} Cascade \gls{pathway} with \gls{synthetic lethal} candidates coloured as shown in the legend.
}
\label{fig:SL_Pathway_Pi3K}
%\end{mdframed}
\end{figure*}

The \gls{PI3K} cascade was not supported across \gls{SLIPT} in \gls{TCGA} breast \glslink{gene expression}{expression} data and the \gls{siRNA} primary screen by over-representation (in Section~\ref{chapt3:compare_pathway}) or resampling (in Section~\ref{chapt3:compare_pathway_perm}) but genes within this \gls{pathway} were detectable by either approach (as shown in Figure~\ref{fig:SL_Pathway_Pi3K}).  While few genes were identified by both approaches, they include genes that are highly connected in the PI3K cascade and are hubs to information transmission such as \textit{FGF9},\textit{PDE3B}, and \textit{PDE4A}. The key upstream genes \textit{PIK3CA} and \textit{PIK3CG} were detected by \gls{siRNA} whereas the downstream \textit{PIK3R1} and \textit{AKT2} genes were detected by \gls{SLIPT}. Genes detected by either method were also prevalent in the \gls{PI3K}, \gls{PDE}, and \gls{AMPK} modules, in addition to the downstream translation factors and ribosomal genes (\textit{EIF4B}, \textit{EEF2K}, and \textit{RPS6}). Together these suggest that there may be further structure between the \gls{SLIPT} and \gls{siRNA} candidate partners of \textit{CDH1} in \glspl{pathway} as illustrated by \gls{PI3K}. As such, \glslink{graph}{pathway} structure will be investigated to detect differences in the upstream and downstream gene candidates detected by either method. Pathway structure may account for the disparity between \gls{SLIPT} and \gls{siRNA} genes, even in \glspl{pathway} such as PI3K where they did not significantly intersect. For instance, \gls{SLIPT} gene partners may be downstream of \gls{siRNA} candidates rather than replicating them directly.

This disparity between \gls{SLIPT} and \gls{siRNA} gene candidates \gls{synthetic lethal} partners of \text{CDH1} (i.e., a high number of genes detected by either approach with few detected by both) was replicated in the related PI3K/AKT \gls{pathway} and the ``PI3K/AKT in cancer'' \gls{pathway} (shown in Appendix Figures~\ref{fig:SL_Pathway_Pi3KAkt} and~\ref{fig:SL_Pathway_Pi3KAktCancer}). Many \gls{synthetic lethal} candidates were at the upstream core of these \gls{pathway} networks and the downstream extremities. It is particularly notable that many genes important in cell signalling and gene regulation were detected by either \gls{synthetic lethal} detection approach. These include \textit{AKT1}, \textit{AKT2}, and \textit{AKT3}, the Calmodulin signalling genes \textit{CALM1} and \textit{CAMK4}, and the forkhead family transcription factors \textit{FOXO1} (a \gls{tumour suppressor}) and \textit{FOXO4} (an inhibitor of \acrshort{EMT}).

\FloatBarrier


\subsection{The Extracellular Matrix}  \label{chapt4:SL_Genes_ECM}

The extracellular \glspl{pathway} ``elastic fibre formation'' and ``fibrin clot formation'' (shown in Figures~\ref{fig:SL_Pathway_ElasticFibre} and~\ref{fig:SL_Pathway_FibrinFormation} respectively) were both supported across analyses (in Chapter~\ref{chap:SLIPT}). These \glspl{pathway} were identified by both \gls{SLIPT} (for \gls{TCGA} breast cancer) and \gls{siRNA} gene candidates as they had significant over-representation and resampling analyses.

\begin{figure*}[!tb]
%\begin{mdframed}
  \begin{center}
  \resizebox{1 \textwidth}{!}{
    %\input{{{"SL_Model.pdf_tex"}}
    %\fbox{
    \includegraphics{{"/home/tomkelly/Downloads/Pathway_Structure/graph_plot_ElasticFibre_exprSL2".pdf}}
   %}
   }
   \end{center}
   \caption[Synthetic lethality in Elastic Fibre Formation]{\small \textbf{Synthetic lethality in Elastic Fibre Formation.} The Reactome Elastic Fibre Formation \gls{pathway} with \gls{synthetic lethal} candidates coloured as shown in the legend.
}
\label{fig:SL_Pathway_ElasticFibre}
%\end{mdframed}
\end{figure*}

Particularly for elastic fibres (Figure~\ref{fig:SL_Pathway_ElasticFibre}), the vast majority of genes were detected by either approach, in addition to a significant proportion of genes detected by both approaches (as determined in Section~\ref{chapt3:compare_pathway}). The genes detected by both approaches also appeared to have a non-random distribution in the network, with \textit{TFGB1}, \textit{ITGB8}, and \textit{MFAP2} exhibiting high connectivity, and having a central role in their respective \gls{pathway} modules. In addition to a structural role in the extracellular matrix and connective tissue (including the tumour microenvironment), these proteins including Furin, \gls{TGFB}, and the \glspl{BMP}, are also involved in responses to endocrine signals and interact with the cellular receptors for signalling \glspl{pathway}. Therefore it is plausible that \textit{CDH1} deficient tumours will be subject to \gls{non-oncogene addiction} to the extracellular environment and growth signals arising from this \gls{pathway}. The \glslink{graph}{pathway} structure also indicated that the genes detected by \gls{siRNA} (or by both approaches) may be be downstream of those detected by \gls{SLIPT}, in addition to whether connectivity or \gls{centrality} is higher for \gls{synthetic lethal} candidates than other genes in the \gls{pathway}.

Genes detected as \gls{synthetic lethal} partners of \textit{CDH1} by \gls{SLIPT} or \gls{siRNA} screening were also common in the Fibrin clot formation \gls{pathway} (shown in Figure~\ref{fig:SL_Pathway_FibrinFormation}). This is consistent with the established pleiotropic role of \textit{CDH1} in regulating fibrin clotting. It is also notable that the genes detected by either method appear to be highly connected such as \textit{C1QBP} \textit{KNG1}, \textit{F8}, \textit{F10}, \textit{F12}, \textit{F13A}, and \textit{PROC} (including many of the coagulation factors). \Gls{synthetic lethal} candidates also include \textit{SERPINE2} and \textit{PRCP}, which only affect downstream genes, in addition to \textit{PROCR} and \textit{VWF}, which are only affected by upstream genes. 

\begin{figure*}[!tb]
%\begin{mdframed}
  \begin{center}
  \resizebox{1 \textwidth}{!}{
    %\input{{{"SL_Model.pdf_tex"}}
    %\fbox{
    \includegraphics{{"/home/tomkelly/Downloads/Pathway_Structure/graph_plot_FibrinFormation_exprSL2".pdf}}
   %}
   }
   \end{center}
   \caption[Synthetic lethality in Fibrin Clot Formation]{\small \textbf{Synthetic lethality in Fibrin Clot Formation.} The Reactome Fibrin Clot Formation \gls{pathway} with \gls{synthetic lethal} candidates coloured as shown in the legend.
}
\label{fig:SL_Pathway_FibrinFormation}
%\end{mdframed}
\end{figure*}


Many of these genes are involved in the larger Extracellular Matrix \gls{pathway} (shown in Appendix Figure~\ref{fig:SL_Pathway_ExtracellularMatrix}), including many of the \gls{synthetic lethal} candidates discussed for elastic fibres. The number of \gls{SLIPT} candidate genes outnumbers those identified by \gls{siRNA}, as expected from an isolated cell model. However, the endocrine response genes (e.g., \textit{TGFB1} and \textit{LTBP4}) which are potentially artifacts of the cell line growth process were replicated with \gls{SLIPT} analysis in patient tumours (TCGA breast cancer data). There is also additional support for \gls{synthetic lethal} genes (e.g., \textit{ITGB2}, \textit{MFAP2}, and \textit{SPARC}) being highly connected networks hubs of the \gls{pathway}. The complexity of the extracellular matrix \gls{pathway} lends credence to the need for formal network analysis approaches to interpret the \glslink{graph}{pathway} structure of \gls{synthetic lethal} candidates. Furthermore, statistical approaches are needed to determine whether the apparent structural relationships between \gls{synthetic lethal} candidates could have occurred by chance 

\FloatBarrier

\subsection{G Protein Coupled Receptors}  \label{chapt4:SL_Genes_GPCR}

\acrfull{GPCR} \glspl{pathway} are highly complex (as shown in Figure~\ref{fig:SL_Pathway_GPCR} and Appendix Figure~\ref{fig:SL_Pathway_GPCR_Downstream}). Many of genes in these \glspl{pathway} were \gls{synthetic lethal} candidates, detected by either \gls{SLIPT} or \gls{siRNA} screening, including genes frequently detected by both approaches, consistent with these \glspl{pathway} being supported by prior analyses (in Sections~\ref{chapt3:compare_pathway} and~\ref{chapt3:compare_pathway_perm}). \Gls{synthetic lethal} candidates include the \gls{PDE} and Calmodulin genes (as discussed in Section~\ref{chapt4:SL_Genes_GPCR}) in addition to others such as the regulators of \gls{RGS}, \gls{CXCR}, \acrfull{JAK}, and the \gls{RHO} genes. These are important regulatory signalling \glspl{pathway} necessary for cellular growth and cancer proliferation. Thus the \gls{GPCR} \glspl{pathway} (and downstream PI3K/AKT signals) are a potentially actionable vulnerability against \textit{CDH1} deficient cancers, particularly since many existing drug targets are in these signalling \glspl{pathway}, some of which have been experimentally validated \citep{Telford2015}. While statistically significant numbers of genes in GCPR \glspl{pathway} were detected by both approaches (in Sections~\ref{chapt3:compare_pathway} and~\ref{chapt3:compare_pathway_perm}), the complexity of \gls{GPCR} networks (containing hundreds of genes) further support the needs for a rational network-based approach to the relationships between \gls{SLIPT} and experimental candidates.

\begin{figure*}[!htbp]
%\begin{mdframed}
  \begin{center}
  \resizebox{1 \textwidth}{!}{
    %\input{{{"SL_Model.pdf_tex"}}
    %\fbox{
    \includegraphics{{"/home/tomkelly/Downloads/Pathway_Structure/graph_plot_GPCR_exprSL2".png}}
   %}
   }
   \end{center}
   \caption[Synthetic lethality in the GPCRs]{\small \textbf{Synthetic lethality in the GPCRs.} The Reactome G$_{\alpha i}$ \gls{pathway} with synthetic lethal candidates, coloured as shown in the legend.
}
\label{fig:SL_Pathway_GPCR}
%\end{mdframed}
\end{figure*}



%\FloatBarrier

\subsection{Gene Regulation and Translation}  \label{chapt4:SL_Genes_Translation}

While very few \gls{synthetic lethal} genes were detected in translational \glspl{pathway} in an experimental screen against \textit{CDH1} \citep{Telford2015}, these were highly over-represented in translational elongation (as shown in Appendix Figure~\ref{fig:SL_Pathway_TranslationElongation}). These \gls{SLIPT} genes include many ribosomal proteins and the regulatory ``elongation factors'' which may be subject to responses in the upstream signalling \glspl{pathway}. This observation further indicates that \glslink{graph}{pathway} structure may be used to identify relationships between \gls{synthetic lethal} candidates detected by \gls{SLIPT} and \gls{siRNA}. The computational approach with \gls{SLIPT} may exhibit the ability to detect downstream genes in the core translational processes, which experimental screening did not identify. The experimental screening may similarly detect upstream regulatory genes less sensitive to inactivation, that is, genes that are less likely to be indiscriminately lethal to both genotypes at high doses of inactivation.

Many of these \gls{SLIPT} candidate genes are also among the \gls{NMD} \gls{pathway} (shown in Appendix Figure~\ref{fig:SL_Pathway_NMD}) or 3$^\prime$ \gls{UTR} mediated translational regulation (shown in Appendix Figure~\ref{fig:SL_Pathway_Three_prime_UTR}). While genes in these \glspl{pathway} were also supported by experimental screening with \gls{siRNA}, there were differences in which genes were detected within the \glslink{graph}{pathway} structures. In particular, \textit{UPF1} was detected in the \gls{siRNA} screen and is the focal downstream gene for the entire \gls{NMD} \gls{pathway} showing that (in this case) \gls{siRNA} genes are downstream effectors of those detected by \gls{SLIPT}.  3$^\prime$ \gls{UTR} mediated translational regulation has a similar structure with two modules connected solely by \textit{RPL13A}, giving an example of \gls{SLIPT} candidate genes with high connectivity, although there were many ribosomal proteins detected by \gls{SLIPT}. However, the detection of \textit{EIF3K}, a regulatory elongation factor (not \gls{essential} to ribosomal function) was replicated across \gls{SLIPT} and \gls{siRNA} screening, while the majority of the elongation factors were not detected by either approach. Regulatory genes, being more amenable to experimental validation, also support further investigation into \glslink{graph}{pathway} structure. The \gls{SLIPT} candidates may support experimental candidates in biological \glspl{pathway} by detecting downstream genes, which may not be detectable by experimental screening with high dose inhibitors. This difference between the approaches may explain the greater number of \gls{SLIPT} candidate partners of \textit{CDH1} than those experimentally identified.


\FloatBarrier

\section{Network Analysis of Synthetic Lethal Genes}   \label{chapt4:Network_Test}

\glsreset{ANOVA}

To demonstrate the network properties of \gls{synthetic lethal} candidates in a pathway, a network analsis was performed on the genes detected as \gls{synthetic lethal} partners of \textit{CDH1} with the \gls{SLIPT} computational approach and the \gls{siRNA} screen \citep{Telford2015} %were compared across network metrics 
in %the example of 
G$_{\alpha i}$ signalling, a \gls{GPCR} \gls{pathway}. This \gls{pathway} was used to demonstrate deeper network analysis approaches to \gls{synthetic lethal} candiates within complex \glspl{pathway}, as it was supported across analyses (in Chapter~\ref{chap:SLIPT}), with significant over-representation in both \gls{SLIPT} and \gls{siRNA} screening, and the genes differed considerably between \gls{synthetic lethal} detection methods (shown in Appendix Figures~\ref{fig:SL_Pathway_GPCR}).  These network metrics were used to measure whether the network properties differed between groups of genes detected by either or both approaches. These analyses serve to test both whether \gls{synthetic lethal} gene candidates had higher connectivity or importance in a network and whether either detection approach is biased towards genes with different network properties.  

%(where the genes differed considerably between \gls{synthetic lethal} detection methods)

\FloatBarrier


\subsection{Gene Connectivity and Vertex Degree}  \label{chapt4:Network_Vertex_Degree}

Vertex degree (the number of connections) for each gene is a fundamental property of a network. The vast majority of genes had a relatively modest number of connections, each with only a few genes in the G$_{\alpha i}$ \gls{pathway} (shown in Figure~\ref{fig:SL_Pathway_GPCR_Vertex_Degree}) having \gls{pathway} relationships with a high number of genes, consistent with the \gls{scale-free} property of biological networks \citep{Barabasi2004}. The number of connections was similar between gene groups (by \gls{synthetic lethal} detection). Genes detected by \gls{siRNA} included those with the fewest connections, despite there being fewer genes that were detected by either approach. 
%The median connectivity of genes detected by both approaches was marginally higher. 
There was no statistically significant effect of either computational or experimental \gls{synthetic lethal} detection method on \glslink{vertex}{vertex} degree, as determined by \gls{ANOVA} (shown by Table~\ref{tab:SL_Pathway_GPCR_Vertex_Degree}).

\begin{figure*}[!htb]
%\begin{mdframed}
  \begin{center}
  \resizebox{0.95 \textwidth}{!}{
    %\input{{{"SL_Model.pdf_tex"}}
    %\fbox{
    \includegraphics{{"/home/tomkelly/Downloads/Pathway_Structure/Centrality_exprSL/GPCR_network_vertex_degree_stripchart2".pdf}}
   %}
   }
   \end{center}
   \caption[Synthetic lethality and vertex degree]{\small \textbf{Synthetic lethality and vertex degree.} The number of connected genes (\gls{vertex degree}) was compared (on a log-scale) across genes detected by \gls{SLIPT} and \gls{siRNA} screening in the Reactome G$_{\alpha i}$ cascade \gls{pathway}. There were no differences in \glslink{vertex}{vertex} degree between the groups (shown in Table~\ref{tab:SL_Pathway_GPCR_Vertex_Degree}), although genes detected by \gls{siRNA} included those with the fewest connections. 
}
\label{fig:SL_Pathway_GPCR_Vertex_Degree}
%\end{mdframed}
\end{figure*} \filbreak

\begin{table*}[!htb]
\caption{\acrshort{ANOVA} for synthetic lethality and vertex degree}
\label{tab:SL_Pathway_GPCR_Vertex_Degree}
\noindent\makebox[\textwidth][c]{%               %centering
\resizebox{0.8 \textwidth}{!}{
\begin{threeparttable}
\begin{tabular}{lccccc}
\hline
                 & DF & Sum Squares & Mean Squares & F-value & p-value \\
\hline
\rowcolor{black!10}
siRNA              &     1    &    21     &    20.8     &    0.0030    &    0.9561 \\
\rowcolor{black!5}
SLIPT              &     1    &    16215    &    16215    &    2.3722    &    0.1246 \\
\rowcolor{black!10}
siRNA$\times$SLIPT     &     1    &    17      &    17      &   0.0025     &    0.9603 \\
\hline
\end{tabular}
\begin{tablenotes}
\raggedright \small
Analysis of variance for \glslink{vertex}{vertex} degree against \gls{synthetic lethal} detection approaches (with an interaction term)
\end{tablenotes}
\end{threeparttable}
}
}
\end{table*} \filbreak

The results for the G$_{\alpha i}$ \gls{pathway} were very similar when testing \glspl{synthetic lethal} against \textit{CDH1} \gls{mutation} (\acrshort{mtSLIPT}). In either case, there was no significant evidence that \gls{SLIPT} or \acrshort{mtSLIPT}-specific genes had higher connectivity than those detected by \gls{siRNA} screening (shown in Appendix Figure~\ref{fig:mtSL_Pathway_GPCR_Vertex_Degree} and Appendix Table~\ref{tab:mtSL_Pathway_GPCR_Vertex_Degree}). Thus \gls{synthetic lethal} detection does not discriminate among genes by their connectivity in this \gls{pathway} network, nor is either approach constrained to detecting highly connected genes. Both approaches have been demonstrated to detect genes with many and very few connections in the G$_{\alpha i}$ signalling \gls{pathway}.

\FloatBarrier

\subsection{Gene Importance and Centrality}  \label{chapt4:Network_Centrality}

%\subsection{Gene Importance and Information Centrality}  \label{chapt4:Network_InfoCent}
\subsubsection{Information Centrality}  \label{chapt4:Network_InfoCent}

\Gls{information centrality} is a measure of the importance of \glslink{vertex}{nodes} in a network in terms of how vital they are to the transmission of information throughout the network. This applies well to biological \glspl{pathway}, partcularly gene regulation and cell signalling. The \glslink{vertex}{nodes} with the highest \gls{information centrality} are not necessarily the most connected, as they may also include \glslink{vertex}{nodes} that pass signals between highly connected network hubs. \Gls{information centrality} therefore provides a distinct metric for the connectivity of a gene in a \gls{pathway}, which has the added benefit of being directly related to the disruption of \gls{pathway} function were it to be inactivated or removed.
%
\Gls{information centrality} has also been suggested to be indicative of the essentiality of genes or proteins \citep{Kranthi2013}.% The \gls{information centrality} for each gene was computed across the entire Reactome network (as discussed in Appendix~\ref{appendix:infocent_essential}). Reactome contains substrates and cofactors in addition to genes and proteins. In support of \gls{centrality} as a measure of essentiality or importance to the network, a number of \glslink{vertex}{nodes} with the highest \gls{centrality} (shown in and Appendix Table~\ref{tab:infocent_reactome}) were \gls{essential} nutrients, including Mg\textsuperscript{2$+$}, Ca\textsuperscript{2$+$}, Zn\textsuperscript{2$+$}, and Fe.%\textsuperscript{3$+$}

%Genes important in development of epithelial tissues and breast cancer were also detected with relatively high \gls{information centrality} (as shown by the distribution across the Reactome network in Appendix Figure~\ref{fig:infocent_reactome}). Interleukin 8 (encoded by \textit{IL8}) is a chemokine important in epithelial cells, the innate immune system, and binding \glspl{GPCR}. \textit{GATA4} is an embryonic transcription factor involved in heart development, \gls{EMT}, and has been shown to be recurrently mutated in breast cancer \citep{TCGA2012}. $\beta$-catenin (encoded by the proto-oncogene \textit{CTNNB1}) is a regulatory protein which binds to \gls{E-cadherin}, being involved in cell-cell adhesion and \gls{WNT} signalling. Together these show that \gls{information centrality} identifies \glslink{vertex}{nodes} of importance to biological functions in \gls{pathway} networks, including those relevant to \textit{CDH1} deficient breast cancers. 

Within the G$_{\alpha i}$ \gls{pathway} (shown in Figure~\ref{fig:SL_Pathway_GPCR_InfoCent}), the \gls{information centrality} across gene groups detected by either \gls{synthetic lethal} approach did not differ significantly (shown by Table~\ref{tab:SL_Pathway_GPCR_InfoCent}). %The genes with the highest \gls{information centrality} included the synthetic candidates \textit{PDE3B} (detected by \gls{SLIPT} and \gls{siRNA}) and \textit{AKT2} (detected by \gls{SLIPT}) which were markedly higher than most other genes in the \gls{pathway}. The higher \gls{centrality} of these genes is consistent with their known biological role in PI3K/AKT signalling and the \glslink{graph}{pathway} structure (shown in Figure~\ref{fig:SL_Pathway_Pi3K}). Other biomolecules with high \gls{centrality} included the \textit{RPS6KB1} and \textit{RPTOR} genes, \gls{AMP}, \gls{PIP2}, and \gls{PIP3}.  %The \gls{information centrality} of the G$_{\alpha i}$ \gls{pathway} was 1.338.
Genes detected by \gls{SLIPT} span the complete range of \gls{PageRank centrality} values for this \gls{pathway}. %, which was replicated when testing \glspl{synthetic lethal} against \textit{CDH1} \gls{mutation} (shown in Appendix Figure~\ref{fig:mtSL_Pathway_GPCR_InfoCent})
%
These findings were replicated (shown in Appendix Figure~\ref{fig:mtSL_Pathway_GPCR_InfoCent} %) when testing \glspl{synthetic lethal} against \textit{CDH1} \gls{mutation} (\acrshort{mtSLIPT}). The differences in network \gls{centrality} between gene groups detected by either method were not statistically significant as determined by \gls{ANOVA} (shown by Table~\ref{tab:SL_Pathway_GPCR_InfoCent}
and Appendix Table~\ref{tab:mtSL_Pathway_GPCR_InfoCent}). Thus neither method was unable to detect \gls{synthetic lethal} genes in the G$_{\alpha i}$ \gls{pathway} with particular \gls{centrality} constraints but they were also not detecting genes with higher \gls{centrality} than expected by chance. 

\begin{figure*}[!htb]
%\begin{mdframed}
  \begin{center}
  \resizebox{0.95 \textwidth}{!}{
    %\input{{{"SL_Model.pdf_tex"}}
    %\fbox{
    \includegraphics{{"/home/tomkelly/Downloads/Pathway_Structure/Centrality_exprSL/GPCR_network_Info_Centrality(Log)_stripchart2".pdf}}
   %}
   }
   \end{center}
   \caption[Synthetic lethality and centrality]{\small \textbf{Synthetic lethality and centrality.} The \gls{information centrality} was compared (on a log-scale) across genes detected by \gls{SLIPT} and \gls{siRNA} screening in the Reactome G$_{\alpha i}$ \gls{pathway}. Genes detected by \gls{SLIPT} or \gls{siRNA} did not have higher \gls{centrality} than other genes (shown in Table~\ref{tab:SL_Pathway_GPCR_InfoCent}). %The gene with the highest \gls{centrality} was detected by both approaches.
   Genes detected by \gls{SLIPT} spanned the range of \gls{centrality} values.
}
\label{fig:SL_Pathway_GPCR_InfoCent}
%\end{mdframed}
\end{figure*} \filbreak

\begin{table*}[!htb]
\caption{\acrshort{ANOVA} for synthetic lethality and information centrality}
\label{tab:SL_Pathway_GPCR_InfoCent}
\noindent\makebox[\textwidth][c]{%               %centering
\resizebox{0.8 \textwidth}{!}{
\begin{threeparttable}
\begin{tabular}{lccccc}
\hline
                 & DF & Sum Squares & Mean Squares & F-value & p-value \\
\hline
\rowcolor{black!10}
siRNA              &     1    &    0.00000000 & $2.7000 \times 10^{-9}$ & 0.0016 & 0.96783 \\
\rowcolor{black!5}
SLIPT              &     1    &    0.00000548 & $5.4831 \times 10^{-6}$ & 3.3253 & 0.06926 \\
\rowcolor{black!10}
siRNA$\times$SLIPT &      1   &    0.00000002 & $1.8800 \times 10^{-8}$ & 0.0114 & 0.91511 \\
\hline
\end{tabular}
\begin{tablenotes}
\raggedright \small
Analysis of variance for \gls{information centrality} against \gls{synthetic lethal} detection approaches (with an interaction term)
\end{tablenotes}
\end{threeparttable}
}
}
\end{table*} \filbreak

\FloatBarrier

%\subsection{Gene Importance and PageRank Centrality}  \label{chapt4:Network_PageRank}
\subsubsection{PageRank Centrality}  \label{chapt4:Network_PageRank}

\FloatBarrier

\gls{PageRank centrality} is another network analysis procedure to infer a hierarchy of gene importance from a network using connections and structure \citep{Brin1998}. In contrast to the \gls{information centrality} approach of removing \glslink{vertex}{nodes}, PageRank uses the eigenvalue properties of the adjacency matrix to rank genes according to the number of connections and paths they are involved in. 

This distinction is immediately clear within the G$_{\alpha i}$ \gls{pathway} (shown in Figure~\ref{fig:SL_Pathway_GPCR_PageRank}), which differs considerably from the \gls{information centrality} scores (as shown in Figure~\ref{fig:SL_Pathway_GPCR_InfoCent}). 
%Genes detected by \gls{SLIPT} span the complete range of \gls{PageRank centrality} values for this \gls{pathway}. 
Genes detected by either \gls{synthetic lethal} approach did not include those with the highest PageRank centrality. There was a significant association between genes detected by \gls{SLIPT} (which had a lower median) with PageRank \gls{centrality} (shown by Table~\ref{tab:SL_Pathway_GPCR_PageRank}). The genes detected by \gls{SLIPT} span the range of \gls{centrality} values of \gls{siRNA} showing that both approaches were capable of detecting genes of moderately high \gls{centrality} (as shown for information centrality) and that the lower \gls{centrality} of \gls{SLIPT} candidates in the G$_{\alpha i}$ \gls{pathway} may be due to \gls{synthetic lethal} partners being less critical to the \gls{pathway}, rather than a limitation of the methodology. While it is expected that some \gls{synthetic lethal} genes will be important to the function of the \gls{pathway}, it is possible that with with high \gls{centrality} were avoided if they are essential to cellular viability.


\begin{figure*}[!htb]
%\begin{mdframed}
  \begin{center}
  \resizebox{0.95 \textwidth}{!}{
    %\input{{{"SL_Model.pdf_tex"}}
    %\fbox{
    \includegraphics{{"/home/tomkelly/Downloads/Pathway_Structure/Centrality_exprSL/GPCR_network_pagerank_stripchart2".pdf}}
   %}
   }
   \end{center}
   \caption[Synthetic lethality and PageRank]{\small \textbf{Synthetic lethality and PageRank.} The \gls{PageRank centrality} was compared (on a log-scale) across genes detected by \acrshort{mtSLIPT} and \gls{siRNA} screening in the Reactome G$_{\alpha i}$ \gls{pathway}. Genes detected by with either \gls{synthetic lethal} detection approach had a more restricted range of \gls{centrality} values but only \gls{SLIPT} genes had a significant association with \gls{centrality} (shown in Table~\ref{tab:SL_Pathway_GPCR_PageRank}).
}
\label{fig:SL_Pathway_GPCR_PageRank}
%\end{mdframed}
\end{figure*} %\filbreak


There was not a significant association between \gls{siRNA} candidates and PageRank centrality. The significant result for \gls{SLIPT} was not replicated when testing \glspl{synthetic lethal} against \textit{CDH1} \gls{mutation} (shown in Appendix Figure~\ref{fig:mtSL_Pathway_GPCR_PageRank} and Appendix Table~\ref{tab:mtSL_Pathway_GPCR_PageRank}). However, this may be due to fewer genes being detected by \acrshort{mtSLIPT} and \gls{siRNA}.

\begin{table*}[!htb]
\caption{\acrshort{ANOVA} for synthetic lethality and PageRank centrality}
\label{tab:SL_Pathway_GPCR_PageRank}
\noindent\makebox[\textwidth][c]{%               %centering
\resizebox{0.8 \textwidth}{!}{
\begin{threeparttable}
\begin{tabular}{lccccc}
\hline
                 & DF & Sum Squares & Mean Squares & F-value & p-value \\
\hline
\rowcolor{black!10}
siRNA              &     1    &    0.0001059 & $1.0589 \times 10^{-4}$ & 2.1021 & 0.14818 \\
\rowcolor{black!5}
SLIPT              &     1    &    0.0002881 & $2.8808 \times 10^{-4}$ & 5.7188 & 0.01743 \\
\rowcolor{black!10}
siRNA$\times$SLIPT     &     1    &    0.0000477 & $4.7704 \times 10^{-5}$ & 0.9470 & 0.33131 \\
\hline
\end{tabular}
\begin{tablenotes}
\raggedright \small
Analysis of variance for \gls{PageRank centrality} against \gls{synthetic lethal} detection approaches (with an interaction term)
\end{tablenotes}
\end{threeparttable}
}
}
\end{table*} %\filbreak

\FloatBarrier

\iffalse
\section{Relationships between Synthetic Lethal Genes}

\FloatBarrier

\subsection{Hierarchical Pathway Structure}
%closer to membrane or nucleus

\subsubsection{Contextual Hierarchy of PI3K}  \label{chapt4:Network_Hierachy}

\FloatBarrier

A contextual hierarchy of genes in the G$_{\alpha i}$ \gls{pathway} was derived (as described in in Section~\ref{methods:pathway_rank}) to assign scores for their relative order in the \gls{pathway}. In the case of \gls{PI3K} (shown in Figure~\ref{fig:SL_Pathway_PI3K_Ranking}), this orders genes from the upstream genes, which respond to signals from extracellular stimuli, to the downstream genes which transmit these to the \gls{gene expression} (translation) responses of the cell. The directionality of this \gls{pathway} is evident in transmitting signals from the \gls{PI3K} complex, via AKT, \gls{PDE}, and mTOR to the ribosomal regulatory proteins. This hierarchical procedure enables testing whether the biological context of a gene in a \gls{pathway} is relevant to detection as a \gls{synthetic lethal} candidate by either computational \gls{SLIPT} analysis or experimental \gls{siRNA} screening.

\begin{figure*}[!htb]
%\begin{mdframed}
  \begin{center}
  \resizebox{1 \textwidth}{!}{
    %\input{{{"SL_Model.pdf_tex"}}
    %\fbox{
    \includegraphics{{"/home/tomkelly/Downloads/Pathway_Structure/Discrete_Pi3k/graph_distance".pdf}}
   %}
   }
   \end{center}
   \caption[Hierarchical structure of PI3K]{\small \textbf{Hierarchical structure of PI3K.} A contextual score was used for ranking genes within the \gls{PI3K} Cascade to demonstrate a \glslink{graph}{pathway} structure analysis to examine whether genes detected by either \gls{SLIPT} or \gls{siRNA} were more frequently upstream or downstream in the G$_{\alpha i}$ \gls{pathway}.
}
\label{fig:SL_Pathway_PI3K_Ranking}
%\end{mdframed}
\end{figure*}


%\FloatBarrier

\subsubsection{Testing Contextual Hierarchy of Synthetic Lethal Genes}  \label{chapt4:Network_Hierachy_Test}

%\begin{itemize}
% \item Are there more SL genes of a particular rank? %%
% \item Are there more SL genes up/downstream of a particular rank? %%
% \item Is there an association with SLIPT (Chi-sq) or \gls{siRNA} (viability) score? x
%\end{itemize}

\begin{figure*}[!b]
%\begin{mdframed}
 \begin{center}
%
        \subcaptionbox{Hierarchical Distance Score \label{fig:SL_Pathway_PI3K_Distance_Vioplot_Counts}}{
	  %\fbox{
	  \includegraphics[width=0.75 \textwidth]{{"/home/tomkelly/Downloads/Pathway_Structure/Discrete_Pi3k/SL_distance_counts_vioplot".pdf}}
	%}
        }%

        \subcaptionbox{Proportion of Genes \label{fig:SL_Pathway_PI3K_Distance_Barplot_Counts}}{%
	  %\fbox{
	  \includegraphics[width=0.75 \textwidth]{{"/home/tomkelly/Downloads/Pathway_Structure/Discrete_Pi3k/SL_distance_counts_barplot_prop".pdf}}
	%}
        }%
      \end{center}
   \caption[Hierarchy score in PI3K against synthetic lethality in PI3K]{\small \textbf{Hierarchy score in \gls{PI3K} against synthetic lethality in \gls{PI3K}.} The hierarchical distance scores were similarly distributed across \gls{SLIPT} and \gls{siRNA} genes. The number of \gls{SLIPT} and \gls{siRNA} genes against the hierarchical distance scores showing no significant tendency for either method to either of the \gls{pathway} upstream or downstream extremities.
}
%\end{mdframed}
\end{figure*}
This \gls{pathway} hierarchy in the \gls{PI3K} cascade was tested for differences between genes detected across \gls{SLIPT} and \gls{siRNA} screening. The \gls{synthetic lethal} candidates for \textit{CDH1} detected by either method (as shown by Figure~\ref{fig:SL_Pathway_PI3K_Distance_Vioplot_Counts}) did not differ, each being distributed throughout the \gls{pathway}. When adjusted for being more numerous, there was little indication that \gls{SLIPT} candidate genes are more frequently upstream or downstream of \gls{siRNA} candidate genes (as shown by Figure~\ref{fig:SL_Pathway_PI3K_Distance_Barplot_Counts}) and were more frequent at moderate hierarchies which contained more genes. \Gls{synthetic lethal} candidates from both methods were less frequently detected in the downstream effectors of the \gls{pathway} (e.g., the mTOR complex), although core \gls{pathway} genes (e.g., \textit{AKT2} and \textit{PDE3B}) were detectable as \gls{synthetic lethal} candidates (as discussed for Figures~\ref{fig:SL_Pathway_Pi3K} and~\ref{fig:SL_Pathway_PI3K_PageRank}).

Similarly, when testing \glspl{synthetic lethal} against \textit{CDH1} \gls{mutation} (\acrshort{mtSLIPT}), the hierarchical score for the G$_{\alpha i}$ \gls{pathway} did not differ between \acrshort{mtSLIPT}-specific and \gls{siRNA}-specific gene candidates (as shown by Appendix Figure~\ref{fig:mtSL_Pathway_PI3K_Distance_Vioplot_Counts}). The median among genes detected by both approaches was marginally elevated such that these genes may be further downstream in the \gls{pathway} that other \gls{synthetic lethal} candidate partners of \textit{CDH1}. There were fewer genes overall with higher scores (shown in Appendix Figure~\ref{fig:mtSL_Pathway_PI3K_Distance_Barplot_Counts}). While these were more frequently detected by both \gls{SLIPT} and \gls{siRNA}, there was no significant variation in \gls{pathway} hierarchy (shown by \gls{ANOVA} in Table~\ref{tab:SL_Pathway_PI3K_Distance_Counts} and Appendix Table~\ref{tab:mtSL_Pathway_PI3K_Distance_Counts}) accounted for by \gls{SLIPT} or \gls{siRNA} detection in the G$_{\alpha i}$ \gls{pathway} (as shown in Figure~\ref{fig:SL_Pathway_Pi3K}). Thus these hierarchical scores may be observed by sampling variation and there was no indication that \gls{SLIPT} or \gls{siRNA} detection differs along the direction of the \gls{pathway}. Genes detected by either method were no more or less common among upstream or downstream of the \gls{pathway}.

%\FloatBarrier

%see earlier Section
%Figure~\ref{fig:SL_Pathway_Pi3K}


\begin{table*}[!h]
\caption{\acrshort{ANOVA} for synthetic lethality and PI3K hierarchy}
\label{tab:SL_Pathway_PI3K_Distance_Counts}
\noindent\makebox[\textwidth][c]{%               %centering
\resizebox{0.8 \textwidth}{!}{
\begin{threeparttable}
\begin{tabular}{lccccc}
\hline
                 & DF & Sum Squares & Mean Squares & F-value & p-value \\
\hline
\rowcolor{black!10}
siRNA              &     1    &     0.001 & 0.00066 & 0.0004 & 0.9842 \\
\rowcolor{black!5}
SLIPT              &     1    &    0.456 & 0.45605 & 0.2740 & 0.6016 \\
\rowcolor{black!10}
siRNA$\times$SLIPT     &     1    &    0.019 & 0.01878 & 0.0113 & 0.9156 \\
\hline
\end{tabular}
\begin{tablenotes}
\raggedright \small
Analysis of variance for \gls{PI3K} hierarchy score against \gls{synthetic lethal} detection approaches (with an interaction term)
\end{tablenotes}
\end{threeparttable}
}
}
\end{table*}

[remove this paragraph and Figures~\ref{fig:SL_Pathway_PI3K_Distance_Vioplot} and~\ref{fig:mtSL_Pathway_PI3K_Distance_Vioplot}?]

Furthermore the \gls{pathway} hierarchical scores did not exhibit different more or less \gls{SLIPT} than \gls{siRNA} genes above or below the given threshold. Since the ideal threshold to detect \glslink{graph}{pathway} structure is unclear, an exploratory analysis was performed, with $\chi^2$-test for the \gls{SLIPT} or \gls{siRNA} candidate genes upstream or downstream of each gene. It is unsurprising that these $\chi^2$ tests were highest when the gene used as a threshold was in the middle of the \gls{pathway} (as shown in Figure~\ref{fig:SL_Pathway_PI3K_Distance_Vioplot}). However, there was no statistically significant support for \glslink{graph}{pathway} structure by this approach, as none of the $\chi^2$ values were high enough to detect \glslink{graph}{pathway} structure between \gls{SLIPT} and \gls{siRNA} gene candidates. Nor was structure detectable for \acrshort{mtSLIPT} testing \glspl{synthetic lethal} against \textit{CDH1} \gls{mutation} (as shown in Appendix Figure~\ref{fig:mtSL_Pathway_PI3K_Distance_Vioplot}).

%\FloatBarrier

\begin{figure*}[!htb]
%\begin{mdframed}
  \begin{center}
  \resizebox{0.75 \textwidth}{!}{
    %\fbox{
    %\includegraphics{{"/home/tomkelly/Downloads/Pathway_Structure/Discrete_Pi3k/SL_distance_vioplot_exprSL".png}}
    %\includegraphics{{"/home/tomkelly/Downloads/Pathway_Structure/Discrete_Pi3k/SL_distance_vioplot".pdf}}
    \includegraphics{{"/home/tomkelly/Downloads/Pathway_Structure/Discrete_Pi3k/SL_distance_stripchart".pdf}}
   %}
   }
   \end{center}
   \caption[Structure of synthetic lethality in PI3K]{\small \textbf{Structure of synthetic lethality in \gls{PI3K}.} The number of \gls{SLIPT} and \gls{siRNA} genes upstream or downstream of each gene in the Reactome PI3K \gls{pathway} were tested (by the $\chi^2$-test). These are plotted as a split jitter stripchart against the hierarchical distance scores showing no significant tendency for either method to either of the \gls{pathway} upstream or downstream extremities.
}
\label{fig:SL_Pathway_PI3K_Distance_Vioplot}
%\end{mdframed}
\end{figure*}

\FloatBarrier

\subsection{Upstream or Downstream Synthetic Lethality}
\fi

\FloatBarrier

\section{Relationships between Synthetic Lethal Genes}
%\subsection{Upstream or Downstream Synthetic Lethality}

This network analyses so far have tested whether \gls{synthetic lethal} candidate genes were more connected or important within a \glslink{graph}{pathway} structure, such as the G$_{\alpha i}$ \gls{pathway}. However these metrics do not ascertain whether there were relationships between \gls{SLIPT} and \gls{siRNA} candidate partners of \textit{CDH1}. In particular, it is plausible that they may be upstream or downstream of one and other within a \gls{pathway}.

The direction of a biological \gls{pathway} is important, particularly those involved in cell signalling which respond to extracellular stimuli and transmit these signals via intermediary proteins to regulate core functions and responses of the cell. These \glspl{pathway} regulate process such as \gls{gene expression} and protein translation, which are important in the proliferation of cancers \citep{Gao2015}. Therefore it is important to determine which \gls{synthetic lethal} candidates were upstream or downstream in the context of a biological \gls{pathway}. In particular, \glslink{graph}{pathway} structure may be used to identify relationships between \gls{SLIPT} and \gls{siRNA} gene candidates.

%The hierarchical approach is designed to detect differences in \gls{pathway} location between gene groups.
A \glslink{graph}{pathway} structure method has been devised to use \glslink{graph}{network} structures to identify directional relationships between individual \gls{SLIPT} and \gls{siRNA} genes. This \glslink{graph}{pathway} structure methodology will be applied (as described in Section~\ref{methods:pathway_str}) to detect the direction of \glspl{shortest path} between \gls{SLIPT} and \gls{siRNA} gene candidates. This will be used to demonstrate the methodology on the \gls{PI3K} and G$_{\alpha i}$ \glspl{pathway}, to develop a statistical test for \glslink{graph}{pathway} structure between between \gls{SLIPT} and \gls{siRNA} gene candidates using resampling  (as described in Section~\ref{methods:network_permutation}), and to apply this test for \glslink{graph}{pathway} structure among \gls{synthetic lethal} gene candidates to the \glspl{pathway} identified in Chapter~\ref{chap:SLIPT} and discussed in Section~\ref{chapt4:SL_Genes}.

%A contextual hierarchy of genes in the G$_{\alpha i}$ \gls{pathway} was derived (as described in in Section~\ref{methods:pathway_rank}) to assign scores for their relative order in the \gls{pathway}. In the case of \gls{PI3K} (shown in Figure~\ref{fig:SL_Pathway_PI3K_Ranking}), this orders genes from the upstream genes,  The directionality of this \gls{pathway} is evident in transmitting signals from the \gls{PI3K} complex, via AKT, \gls{PDE}, and mTOR to the ribosomal regulatory proteins. This hierarchical procedure enables testing whether the biological context of a gene in a \gls{pathway} is relevant to detection as a \gls{synthetic lethal} candidate by either computational \gls{SLIPT} analysis or experimental \gls{siRNA} screening.

\FloatBarrier

\subsection{Detecting Upstream or Downstream Synthetic Lethality}  \label{chapt4:Structure_GPCR}
%\subsubsection{Measuring Structure of Candidates within PI3K}  \label{chapt4:Structure_GPCR}

Shortest paths in a \gls{pathway} network were used to devise a strategy to detect \glslink{graph}{pathway} structure between \gls{SLIPT} and \gls{siRNA} gene candidates partners of \textit{CDH1} (as described in Section~\ref{methods:pathway_str}). Thus we can determine whether individual \gls{SLIPT} genes have upstream or downstream \gls{siRNA} candidates (scored as ``up'' or ``down'' events respectively). This procedure enables the detection of directional relationships between \gls{SLIPT} and \gls{siRNA} gene candidates. %(in contrast to the hierarchical approach).


\begin{figure*}[!htb]
%\begin{mdframed}
    \begin{center}
%
        \subcaptionbox{Resampling in G$_{\alpha i}$ signalling \label{fig:SL_Pathway_GPCR_Perm}}{%
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Downloads/Pathway_Structure/test_GPCR_exprSL".pdf}}
        }%
        \subcaptionbox{Resampling in the PI3K cascade  \label{fig:SL_Pathway_PI3K_Perm}}{%
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Downloads/Pathway_Structure/test_PI3K_exprSL".pdf}}
        }%
    \end{center}
   \caption[Structure of synthetic lethality resampling]{\small \textbf{Structure of synthetic lethality resampling.} A null distribution with 10,000 iterations of the number of \gls{siRNA} genes upstream or downstream of \gls{SLIPT} genes (depicted as the difference of these) in each \gls{pathway}. To assess significance, the observed events (with \glspl{shortest path}) were compared to the 90\% and 95\% intervals for the null distribution (shown in blue). Genes detected by both methods were not fixed to the same number as observed for the alternative null distribution (shown in violet), although the significance of the observed number of events (red) was changed in either case. The genes detected by both approaches were included in computing the number of \glspl{shortest path} (in either direction) between \gls{SLIPT} and \gls{siRNA} genes. The permutations show (a) a significant \gls{pathway} relationship for G$_{\alpha i}$ signalling and (b) and non-significant relationship for the \gls{PI3K} cascade. 
}
\label{fig:SL_Pathway_Perm}
%\end{mdframed}
\end{figure*}

The total number of gene candidate pairs in either direction can be compared within a \gls{pathway} network to assess the overall directional relationships in a \gls{pathway}. This directionality is detectable by the difference between the number of \gls{SLIPT} candidate genes with upstream and downstream \gls{siRNA} gene partners. However, this measure alone is not sufficient to determine whether there is evidence of \glslink{graph}{pathway} structure between \gls{SLIPT} and \gls{siRNA} gene candidates partners of \textit{CDH1} in a \gls{pathway} network. Nevertheless, it does serve to measure the magnitude (and direction) of the consensus of directional relationships (upstream and downstream) between \gls{SLIPT} and \gls{siRNA} gene candidates partners. This measure of \glslink{graph}{pathway} structure can be used for testing for statistical significance of \glslink{graph}{pathway} structure by resampling, using a permutation procedure to test whether these relationships are detectable among randomly selected gene groups rather than the detected \gls{SLIPT} and \gls{siRNA} gene candidates partners (as described in Sections~\ref{methods:permutation} and~\ref{methods:network_permutation}).

This resampling procedure was performed for the G$_{\alpha i}$ and \gls{PI3K} \glspl{pathway} to generate a null distribution for the difference in the number of ``up events'' and ``down events'' for these \gls{pathway} structures (as shown in Figures~\ref{fig:SL_Pathway_Pi3K} and~\ref{fig:SL_Pathway_GPCR}). The resulting null distributions (as shown in Figure~\ref{fig:SL_Pathway_Perm}) were used to detect whether genes detected by \gls{SLIPT} had significantly more upstream or downstream \gls{siRNA} candidates in either \gls{pathway}. Therefore it can be shown that \gls{siRNA} genes were significantly downstream of \gls{SLIPT} candidate genes by resampling for the G$_{\alpha i}$ signalling \gls{pathway} (as shown in Figure~\ref{fig:SL_Pathway_GPCR_Perm}). This demonstrates that \gls{pathway} relationships can be detected between \gls{synthetic lethal} candidates by this procedure and that \gls{siRNA} genes were downstream of gene detected by \gls{SLIPT} in an example of GPCR signalling expanding on support for \glspl{synthetic lethal} in this \gls{pathway} (as shown in Chapter~\ref{chap:SLIPT}). These structural relationships may also account for why each the computational and experimental approaches did not detect many of the same specific genes because they are detecting different parts of the \gls{pathway}.

In contrast, there was not significant evidence of such \gls{pathway} structure between \gls{siRNA} and \gls{SLIPT} candidate genes when resampling within the \gls{PI3K} cascade \gls{pathway} (as shown in Figure~\ref{fig:SL_Pathway_PI3K_Perm}). This indicates that these relationships may be \gls{pathway} specific rather than a general property of these \gls{synthetic lethal} detection methods. These results were robustly reproducible, with similar findings (as shown in Appendix Figure~\ref{fig:mtSL_Pathway_Perm}) for each \gls{pathway} when testing for \glspl{synthetic lethal} against \textit{CDH1} \gls{mutation} (\acrshort{mtSLIPT}). Neither relaxing the fixing the number of genes detected by both approaches (as shown in Figure~\ref{fig:SL_Pathway_Perm} and Appendix Figure~\ref{fig:mtSL_Pathway_Perm}) nor excluding these jointly detected genes altered the findings of this approach. These genes were included in the analysis because they can disproportionately count towards \gls{siRNA} genes being upstream (or downstream) of \gls{SLIPT} genes as they may still have different proportions of gene detected by either approach upstream (or downstream) of them. Furthermore, expanding the range of \glspl{shortest path} to consider \glslink{edge}{links} in related \glspl{pathway} (using the ``meta-\glspl{pathway}'' constructed in Section~\ref{methods:subgraphs}) also had little effect on the null distribution generated, despite increasing the computational complexity of the procedure.

\FloatBarrier

\subsection{Resampling for Synthetic Lethal Pathway Structure}  \label{chapt4:Structure_Perm}
%\subsubsection{Resampling for Synthetic Lethal Pathway Structure}  \label{chapt4:Structure_Perm}

The permutation procedure (as described in Section~\ref{methods:network_permutation}) that was performed in Section~\ref{chapt4:Structure_GPCR} for the G$_{\alpha i}$ and \gls{PI3K} \glspl{pathway} was also applied to other \glspl{pathway} identified in Chapter~\ref{chap:SLIPT} and discussed in Section~\ref{chapt4:SL_Genes}. In addition to the cell signalling \glspl{pathway} (PI3K/AKT and GCPRs demonstrated in Section~\ref{chapt4:Structure_GPCR}), the \glspl{pathway} tested include extracellular matrix (with constituent elastic fibre and fibrin \glspl{pathway}), and translational \glspl{pathway} (with \gls{NMD} and 3$^\prime$\gls{UTR} regulation).

The resampling results across these \glspl{pathway} (as shown in Table~\ref{tab:pathway_str_exprSL}) had limited support for association between \glslink{graph}{pathway} structure and detection of \gls{synthetic lethal} genes, with the majority of these being non-significant (as shown for \gls{PI3K} in Figure~\ref{fig:SL_Pathway_PI3K_Perm}), with the exception of G$_{\alpha i}$ signalling (as shown in Figure~\ref{fig:SL_Pathway_GPCR_Perm}). However, the exact distribution for these \glspl{pathway} will differ depending on their structure, the number of genes they consist of, and the proportion of \gls{synthetic lethal} candidates among them (including a higher frequency of genes detected by both methods for the \glspl{pathway} identified in Section~\ref{chapt3:compare_pathway_perm}). This resampling is therefore an appropriate procedure to use to detect structural relationships across \glspl{pathway} as it does not assume an underlying test statistic distribution.

Pathway structure was supported for the \gls{NMD} \gls{pathway} (which is consistent with \gls{siRNA} being downstream in Appendix Figure~\ref{fig:SL_Pathway_NMD}). However, this observation rests upon a single gene and was not replicated when testing \glspl{synthetic lethal} (\acrshort{mtSLIPT}) against \textit{CDH1} \gls{mutation} (as shown in Appendix Table~\ref{tab:pathway_str_mtSL}) nor was it supported by the related 3$^\prime$\gls{UTR} regulation and translational elongation \glspl{pathway}.

\begin{table*}[!htb]
\caption{Resampling for \glslink{graph}{pathway} structure of \gls{synthetic lethal} detection methods}
\label{tab:pathway_str_exprSL}
\noindent\makebox[\textwidth][c]{%               %centering
\resizebox{1.2 \textwidth}{!}{
\begin{threeparttable}
\begin{tabular}{l|cc|cc|cccc|cc|c}
\cline{2-12}
                                          & \multicolumn{2}{c|}{\textbf{Graph}} & \multicolumn{2}{c|}{\textbf{States}} & \multicolumn{4}{c|}{\textbf{Observed}}        & \multicolumn{2}{c|}{\textbf{Permutation p-value}} & \multicolumn{1}{c}{\textbf{p-value (FDR)}} \\ %& \multicolumn{1}{c}{\textbf{Adj. p-value}\tnote{*}}
\hline
\textbf{Pathway}                                   & \textbf{Nodes} & \textbf{Edges}  & \textbf{SLIPT} & \textbf{siRNA} & \textbf{Up}   & \textbf{Down} & \textbf{Up$-$Down} & \textbf{Up$/$Down}           & \textbf{Up$-$Down} & \textbf{Down$-$Up} & \textbf{Down$-$Up} \\
\hline
\rowcolor{black!10}
PI3K Cascade                              & 138         & 1495         & 38            & 25          & 122  & 128  & -6      & 0.953        & 0.5326             & 0.4606  & 0.6734            \\%fdr:  0.6734 0.6734 0.0145 0.6734 0.6734 0.6734 0.6734 0.0010 0.6734 0.6734
\rowcolor{black!5}
PI3K/AKT Signalling in Cancer              & 275         & 12882        & 98            & 44          & 779  & 679  & 100     & 1.147        & 0.3255             & 0.6734  & 0.6734              \\
\rowcolor{black!10}
\textbf{G$_{\alpha i}$ Signalling}                  & 292         & 22003        & 95            & 58          & 836  & 1546 & -710    & 0.541        & 0.9971             & 0.0029   & 0.0145             \\
\rowcolor{black!5}
GPCR downstream                           & 1270        & 142071       & 312           & 160         & 9755 & 9261 & 494     & 1.053        & 0.3692             & 0.6305  & 0.6734              \\
\rowcolor{black!10}
Elastic fibre formation                   & 42          & 175          & 24            & 7           & 1    & 2    & -1      & 0.500        & 0.5461             & 0.3865   & 0.6734             \\
\rowcolor{black!5}
Extracellular matrix                      & 299         & 3677         & 127           & 29          & 547  & 455  & 92      & 1.202        & 0.3351             & 0.6636   & 0.6734             \\
\rowcolor{black!10}
Formation of Fibrin                       & 52          & 243          & 18            & 5           & 12   & 17   & -5      & 0.706        & 0.6198             & 0.3564    & 0.6734            \\
\rowcolor{black!5}
\textbf{Nonsense-Mediated Decay}                   & 103         & 102          & 74            & 2           & 0    & 74   & -74     & 0            & 1.0000             & $<0.0001$  & $<0.0010$                \\
\rowcolor{black!10}
3' -UTR-mediated translational regulation & 107         & 2860         & 77            & 1           & 0    & 0    & 0       &              & 0.4902             & 0.5027   & 0.6734             \\
\rowcolor{black!5}
Eukaryotic Translation Elongation         & 92          & 3746         & 76            & 0           & 0    & 0    & 0       &              & 0.4943             & 0.4933   & 0.6734             \\
\hline
%fdr:  0.6734 0.6734 0.0145 0.6734 0.6734 0.6734 0.6734 0.0010 0.6734 0.6734
%holm: 0.4606 0.6734 0.0029 0.6305 0.3865 0.6636 0.3564 0.0001 0.5027 0.4933
\end{tabular}
\begin{tablenotes}
\raggedright \small
Pathways in the Reactome network tested for structural relationships between \gls{SLIPT} and \gls{siRNA} genes by resampling. The raw p-value (computed without adjusting for multiple comparisons over \glspl{pathway}) is given for the difference in upstream and downstream paths from \gls{SLIPT} to \gls{siRNA} gene candidate partners of \textit{CDH1} with significant \glspl{pathway} highlighted in bold. Sampling was performed only in the target \gls{pathway} and \glspl{shortest path} were computed within it. Loops or paths in either direction that could not be resolved were excluded from the analysis. The gene detected by both \gls{SLIPT} and \gls{siRNA} (or resampling for them) were includued in the analysis and the number of these were fixed to the number observed.
%\item[*] Adjusted p-values from the \gls{FDR} multiple comparison procedure
\end{tablenotes}
\end{threeparttable}
}
}
\end{table*}

There does not appear to be a consensus on the directionality of \gls{SLIPT} and \gls{siRNA} candidates across \glspl{pathway} as distinct \glspl{pathway} showed stronger tendency for \gls{siRNA} genes to be either upstream or downstream. Even related \glspl{pathway} such as \gls{PI3K} and PI3K/AKT signalling showed directional events in opposite directions. The strongest \gls{pathway} (among those tested) with support for directional \glspl{pathway} structure is G$_{\alpha i}$ signalling (as shown in Figure~\ref{fig:SL_Pathway_GPCR_Perm}). In contrast to the other \glspl{pathway} G$_{\alpha i}$ signalling showed significant downstream \gls{siRNA} genes for \gls{SLIPT} from a large number of \glspl{shortest path} (in Table~\ref{tab:pathway_str_exprSL}). This would indicate that \gls{SLIPT} detects upstream regulators of genes experimentally validated by \gls{siRNA} in this \gls{pathway}. This result was \gls{pathway} was also the strongest result in \acrshort{mtSLIPT} results (Appendix Table~\ref{tab:pathway_str_mtSL}), although it was not significant after adjusting for multiple testing in this case. %However, these results are borderline significant (with raw permutation p-values) and are unlikely to be detected after adjusting for multiple comparisons across the 10 \glspl{pathway} presented here (nor in the 1652 Reactome \glspl{pathway} used previously in Chapter~\ref{chap:SLIPT}).

%adj. by 10: Gai = 0.145, NMD < 0.01

There is insufficient evidence to determine whether there is \glslink{graph}{pathway} structure, that genes were detected upstream or downstream by either method, between the \gls{SLIPT} and \gls{siRNA} candidates in many of the \gls{synthetic lethal} \glspl{pathway} (identified in Chapter~\ref{chap:SLIPT}). In particular, directional structure among \gls{synthetic lethal} candidates for \textit{CDH1} was not strongly supported in most of the signalling \glspl{pathway} (with the exception of G$_{\alpha i}$ signalling) upon which the rationale for \glslink{graph}{pathway} structure hypotheses were based. While there is statistically significant over-representation of many of these \glspl{pathway} in genes detected by both \gls{SLIPT} and \gls{siRNA} (as described in Chapter~\ref{chap:SLIPT}), many of these did not show relationships with respect to \glslink{graph}{pathway} structure. Despite the design of a robust resampling approach to test relationships between gene groups, the detection of structural relationships between \gls{SLIPT} and \gls{siRNA} gene candidates did not generalise across \glspl{pathway} (and was specific to a few). Such structural relationships may apply more broadly to gene networks as different biological \glspl{pathway} were more over-represented among \gls{SLIPT} and \gls{siRNA} gene candidates. Furthermore, \glslink{graph}{pathway} structure did not account for the discrepancy between \gls{SLIPT} and \gls{siRNA} gene candidates which did not significantly intersect, such as the \gls{PI3K} cascade. 
%Furthermore, the \gls{pathway} relationships are unlikely to be statistically supported by resampling when testing across the search space of Reactome \glspl{pathway} and adjusting for multiple comparisons. 

\FloatBarrier

\section{Discussion}

These investigations used a functional \gls{pathway} network that encapsulates protein complexes and functional modules. The Reactome network uses curated, experimentally identified \glspl{pathway} to determine relationships between genes and does not have the limitation of relying solely on protein binding or text-mining which are prone to false positives \citep{Reactome} . While it is not documented whether these relationships are activating or inhibitory, the Reactome network \citep{Reactome} is sufficient to test \gls{pathway} relationships with directional information.

Synthetic lethal genes and \glspl{pathway} (for \textit{CDH1} loss in cancer) were identified across \gls{gene expression} and \gls{mutation} datasets in Chapter~\ref{chap:SLIPT}.
%replace with sentence in next paragraph?
These \glslink{graph}{pathway} structure investigations extend those investigations into \gls{synthetic lethal} gene candidates including exploring the discrepancy between \gls{SLIPT} and \gls{siRNA} candidate genes in a \gls{pathway} such as \gls{PI3K} in which they did not significantly intersect.
%
Pathways with replicated \gls{synthetic lethal} genes across these detection methods, breast and stomach cancer data, and patient and cell line data were also investigated including \glspl{pathway} from the extracellular microenvironment to core translational \glspl{pathway} and the signalling \glspl{pathway} between them.

Synthetic lethal gene candidates in the context of \glslink{graph}{pathway} structures can also be interpreted to provide additional mechanisms and support for belonging to a \gls{synthetic lethal} \gls{pathway}. Gene candidates with known mechanisms are ideal for triage of targets specific to \textit{CDH1} deficient tumours and for further experimental validation in preclinical models.
%compare sentence below to paragraph above 
This chapter presents computational methods to use \glslink{graph}{pathway} structure in an attempt to detect genes with importance in a \gls{pathway} and reconcile the differences between \gls{SLIPT} and \gls{siRNA} candidate genes with \gls{pathway} relationships (e.g., one group being downstream of the other).

Many genes were detected by either \gls{SLIPT} and \gls{siRNA}.  The differences between these computational and experimental screening approaches could feasibly lead to differences in which genes within a \gls{synthetic lethal} \gls{pathway} are identified. Genes detected by \gls{synthetic lethal} detection strategies included those of biological importance within \gls{synthetic lethal} \glspl{pathway}, those which are actionable drug targets, and those with functional implications for the biological growth mechanisms or vulnerabilities of \textit{CDH1} deficient tumours. It appeared that genes detected by both approaches were highly connected (or of importance) in the \glslink{graph}{network} structure or some \glspl{pathway} and that there may be some structure with \gls{SLIPT} and \gls{siRNA} upstream or downstream of each other. 

The complexity of biological \glspl{pathway} meant that relationships between gene candidates were difficult to discern without formal mathematical and computational approaches and thus these were used to analyse large biological networks. Network analysis techniques were applied to formalise and quantify the connectivity and importance (centrality) of genes within \glspl{pathway} (using \gls{PI3K} as an example). However, these network techniques were unable to identify distinct differences in the network properties of genes detected as \gls{synthetic lethal} candidates by computational or experimental methods. These network metrics support the application of synthetic detection across \glspl{pathway} (and the findings using \glspl{pathway} as gene sets in Chapter~\ref{chap:SLIPT}) as neither \gls{synthetic lethal} detection approach was biased towards genes of higher importance or connectivity and neither approach was insensitive to genes of lower  importance or connectivity. \gls{SLIPT} is therefore not biased towards genes with  more crucial role in the \gls{pathway} as inferred by \gls{pathway} connectivity and \gls{centrality} measures and detects genes irrespective of \glslink{graph}{pathway} structure. 

%Similarly, a network hierarchy based on biological context (ordered from receiving extracellular stimuli to affecting downstream \gls{gene expression} and cell growth) was devised to test whether \gls{PI3K} genes of a particular upstream or downstream level were more frequently detected as \gls{synthetic lethal} candidates. However, this approach was unable to ascertain whether genes detected by either method were further upstream or downstream in the \gls{pathway} and there was no statistical evidence that either method differed in which levels of this structure were detected.

A measure of \glslink{graph}{pathway} structure between individual \gls{SLIPT} and \gls{siRNA} candidate genes within a \gls{pathway} was devised using the direction of \glspl{shortest path} in a directed \glslink{graph}{graph} structure. This is amenable to detecting the consensus directionality of the \gls{pathway} across pairs of genes detected by either method. The \glslink{graph}{pathway} structure methodology developed here is generally applicable to comparison of \glslink{vertex}{node} groups (allowing overlapping) including genes in biological \glspl{pathway} and their detection by different methodologies. While the \glslink{graph}{pathway} structure measure alone is not able to detect structural relationships between gene groups (e.g., \gls{SLIPT} and \gls{siRNA} gene candidates), it is amenable to resampling to determine whether these relationships are statistically significant. This approach successfully detected a statistically robust relationship between \gls{SLIPT} and \gls{siRNA} candidate genes on the G$_{\alpha i}$ signalling \gls{pathway}, despite there being few differences between these genes with respect to network metrics of connectivity or centrality.

\section{Summary}

Together these analyses of biological \glspl{pathway}, network metrics, and statistical procedures devised specifically for this purpose were applied to Reactome \glslink{graph}{pathway} structures to test whether structural relationships existed between \gls{synthetic lethal} candidates. Of particular interest was whether these relationships relate to the differences between the computational (\gls{SLIPT}) and experimental (\gls{siRNA}) \gls{synthetic lethal} candidate partners of \textit{CDH1} (in the \glspl{pathway} discussed in Chapter~\ref{chap:SLIPT}).

While biologically relevant relationships were observed in specific \glspl{pathway}, there were few detectable structural relationships between \gls{SLIPT} and \gls{siRNA} gene candidates, apart from structural relationships specific to G$_{\alpha i}$ signalling. In this \gls{pathway}, \gls{synthetic lethal} candidates did not exhibit significant differences in network connectivity or \gls{centrality} measures. These network analyses were also unable to ascertain whether the candidates detected by either method stratified into upstream and downstream genes on the \gls{pathway}. % and they likely do not.

A statistical resampling procedure was applied to \gls{shortest path} analysis to test whether pairs of \gls{SLIPT} and \gls{siRNA} gene candidates were more likely to be upstream or downstream of each other. This approach did not detect many structural relationships in the \gls{synthetic lethal} \glspl{pathway} identified in Chapter~\ref{chap:SLIPT}. Overall, support for \glslink{graph}{pathway} structure between \gls{SLIPT} and \gls{siRNA} gene candidates was weak and the direction was inconsistent between \glspl{pathway}. Therefore \glslink{graph}{pathway} structure does not appear to generally account for the differences between the \gls{SLIPT} and \gls{siRNA} gene candidates, although it may apply in specific \glspl{pathway} as demonstrated with G$_{\alpha i}$ signalling. It was possible to detect some \gls{pathway} relationships between \gls{synthetic lethal} candidates in \gls{synthetic lethal} \glspl{pathway}, in addition to the significantly over-represented genes shared between \gls{SLIPT} and \gls{siRNA} (as identified in  Chapter~\ref{chap:SLIPT}).

Furthermore, the resampling procedure demonstrated in this chapter is more widely applicable to gene states in \glslink{graph}{network} structures and may be of further utility in the analysis of biological \glspl{pathway} or networks. This approach was able to quantify structural relationships that were otherwise difficult to interpret and to conclusively exclude many potential relationships. In this respect, the network resampling methodology may also be applicable to triage of experimental validation.

\clearpage

\iffalse
\paragraph{Aims}

  \begin{itemize}
   \item Synthetic Lethal Genes within a Biological Pathway Structure
   
   \bigskip
   
   \item Importance and Connectivity of Synthetic Lethal Genes within Pathway Networks
   
   \bigskip
   
   \item Upstream and Downstream Relationships between SLIPT and \gls{siRNA} Candidates
  \end{itemize}

\paragraph{Summary}

  \begin{itemize}
   \item Synthetic Lethal genes were explored within a \glslink{graph}{graph} structures for key \glspl{pathway} identified previously 
   
   \bigskip
   
   \item In some cases these \glslink{graph}{graph} structures appeared to have relationships between \gls{synthetic lethal} genes  
   
   \bigskip
   
   \item However, no existing network metrics of importance and connectivity with the networks were elevated significantly for Synthetic Lethal genes
   
   \bigskip
   
   \item Nor was there significant evidence of upstream and downstream relationships between SLIPT and \gls{siRNA} Candidates in a \gls{shortest path} permutation analysis
  \end{itemize}
  
\clearpage
\fi