\chapter{Discussion}
\label{chap:discussion}

This thesis combines analysis of \gls{gene expression} data from \gls{TCGA} with experimental screening results \citep{Telford2015} to demonstrate \gls{synthetic lethal} discovery for partners of \textit{CDH1}. % in \glslink{gene expression}{expression} data generated by \glspl{genomic} technologies with comparisons to existing experimental candidates.
Together these findings further elucidate known and novel functions of \textit{CDH1} in the cell, \gls{functional redundancy} in cancer, and represent potential \glslink{targeted therapy}{therapeutic targets} against loss of \textit{CDH1} function. \Gls{synthetic lethal} candidate genes were further investigated for relationships within \gls{synthetic lethal} pathways, and in the process a network-based approach to compare gene sets was developed.

The \gls{SLIPT} \gls{synthetic lethal} detection methodology was applied to \gls{gene expression} data throughout this thesis and was evaluated with simulated data. A procedure was developed to stringently generate \gls{gene expression} data from known \gls{synthetic lethal} partners in simulated data. These simulations included simple and complex correlation structures, and modelling \gls{synthetic lethal} genes within pathways. Together, these results demonstrate \gls{SLIPT} as a robust widely applicable \gls{gene expression} analysis procedure for discovery of \gls{synthetic lethal} partner genes (for which an R package has been made available). The performance of \gls{SLIPT} on simulated data also highlights the strengths and limitations of the procedure.

\section{Synthetic Lethality and \textit{CDH1} Biology}
\label{chapt6:implications}

The \textit{CDH1} \gls{tumour suppressor} gene was the focus to demonstrate the novel \gls{SLIPT} methodology for identifying \gls{synthetic lethal} partners. This gene is important in \glslink{sporadic}{sporadic} breast and stomach cancers, in addition to \gls{familial} syndromes, such as \gls{HDGC}. The analysis of \gls{synthetic lethal} partners of \textit{CDH1} in breast and stomach cancers was enabled by the availability of molecular data \citep{TCGA2012, TCGA2014GC} and a \gls{synthetic lethal} screen conducted in MCF10A breast cells \citep{Chen2014, Telford2015}.

Synthetic lethal interactions arise due to \gls{functional redundancy} \citep{Boone2007, Kaelin2005, Fece2015} and so the \gls{synthetic lethal} partners of \textit{CDH1} indicate the wide-ranging biological functions that \gls{E-cadherin} is involved in. The diverse \gls{synthetic lethal} pathways identified support the known pleiotropic nature of the \textit{CDH1} gene \citep{Kroepil2012}, across established functions of \textit{CDH1}, candidates from an experimental screen \citep{Telford2015}, and novel interactions identified for further investigation. The highly pleiotropic functions of \gls{E-cadherin} was also consistent with \textit{CDH1} being a \gls{tumour suppressor} gene. % for which epithelial cells are significantly disrupted at the molecular level and prone to becoming cancerous.

\subsection{Established Functions of \textit{CDH1}}
\label{chapt6:function}

\textit{CDH1} has established functions in cell-cell communication and maintaining the cytoskeleton, specifically with cell-cell adhesion by forming tight junctions and the adherens complex \citep{Jeanes2008}. More recently, functions of \textit{CDH1} in the extracellular matrix and fibrin clotting have also been identified \citep{Tunggal2005, Cardiff2011, Wojtukiewicz2016}. \Gls{synthetic lethal} interactions within the same biological pathway were expected \citep{Kelley2005, Boone2007}. \Gls{synthetic lethal} interactions identified in these pathways are consistent with the known functions of \textit{CDH1}, which are also potentially actionable targets against cancers.

%extracellular / fibrin clotting

\subsection{The Molecular Role of \textit{CDH1} in Cancer}
\label{chapt6:cancer}

The involvement of \textit{CDH1} in the extracellular matrix is important in cancers as it indicates that \textit{CDH1} loss may affect the tumour microenvironment, contributing to its role as a tumour and invasion suppressor. Furthermore, perturbations in the extracellular matrix and tumour microenvironment present a means by which to specifically inhibit (cancerous) \textit{CDH1}-deficient cells, in addition to those identified by \gls{siRNA} screening \citep{Telford2015}. 
%Few genes in extracellular pathways were detected in an experimental screen \citep{Telford2015} conducted in an isolated cell model \citep{Chen2014} but these are not expected to be detected in such as system. 
These may be further supported in further investigations with 3D cell culture, ``organoid'', or mouse xenograft cancer models.

Many of the pathways involved in cell signalling, including \glspl{GPCR}, were identified by \gls{SLIPT} in addition to the experimental screen, supporting findings in cell line models \citep{Telford2015}. These pathways are \gls{essential} to the growth of \textit{CDH1}-deficient cancers and present a potential vulnerability specific to these (cancerous) cells. Furthermore, the replication of \glspl{synthetic lethal} of \textit{CDH1} with cell signalling pathways in \gls{TCGA} data, across cancer types and genetic backgrounds, robustly supports these pathways being clinically applicable beyond the genetic background of the model system of \textit{CDH1}\textsuperscript{-/-} MCF10A cells \citep{Chen2014}. While specific \gls{synthetic lethal} genes were not consistently detected between the \gls{SLIPT} analyses and \gls{siRNA} screen \citep{Telford2015}, they were sufficient to identify \gls{synthetic lethal} pathways for further experimental investigation, which are more likely to be replicated between genetic backgrounds \citep{Dixon2008}. Together these results demonstrate \textcolor{black}{that} computational methods, such as \gls{SLIPT}, could be integrated with an experimental screen to triage potential therapeutic targets  for further pre-clinical investigation.

The analysis of \glslink{gene expression}{expression} data with \gls{SLIPT} is also indicative of biological mechanisms of \glspl{synthetic lethal} in pathways beyond those identified in screening experiments \citep{Telford2015}. In particular, translation and regulatory pathways, involving 3$^\prime$ \glspl{UTR} and \gls{NMD}, were candidate \gls{synthetic lethal} pathways with \textit{CDH1} \textcolor{black}{identified} by \gls{SLIPT}. These pathways represent downstream targets, regulated by the putative \gls{synthetic lethal} signalling pathways, which cancer cells are dependent on to proliferate and evade host defense processes, such as apoptosis and immune responses \citep{Gao2015}.

\textcolor{black}{
It is possible that either the \gls{SLIPT} methodology or the \gls{siRNA} screen \citep{Telford2015} are biased towards detecting particular pathways. In particular, the primary \gls{siRNA} screen (conducted in a cell line model) detected a considerable number of kinase signalling pathways. The \gls{SLIPT} methodology could similarly be biased towards pathways regulating or involved in translation. The pathways detected by both approaches (as verified by resampling analysis) with distinct limitations are therefore the strongest candidate synthetic lethal pathways of \textit{CDH1}. It is still necessary to perform further experimental validation of the candidate genes and pathways identified by both of these approaches. Nevertheless, \gls{SLIPT} is a useful triage utility to identify synthetic lethal genes and was shown to be effective at predicting genes that would be validated in the secondary siRNA screen \citep{Telford2015} and simulation results (presented in Section~\ref{chapt2:simulation_2015} and Chapter~\ref{chap:simulation}).
}
%translation

%\section{Synthetic Lethal Discovery in Expression Data}
%\label{chapt6:SLIPT_applications}

\section{Significance}
\label{chapt6:significance}

\subsection{Synthetic Lethality in the Genomic Era}
\label{chapt6:significance_genetics}

An effective \gls{synthetic lethal} discovery tool for \glspl{bioinformatics} analysis could have a wide range of applications in genetics research, including functional \glspl{genomic}, medical and agricultural applications.  The \gls{SLIPT} approach demonstrated in this thesis is widely applicable to other genes, cell types, and biological research questions. In addition to further query of cancer genes,  \gls{synthetic lethal} gene functions \textcolor{black}{also have} implications for \glslink{functional redundancy}{genetic redundancy}. Highly redundant genes, and the genetically robust systems they give rise to, are also relevant to evolutionary, developmental, and systems biology to understand how these change over time and play \textcolor{black}{a fundamental role in fundamental development} \citep{Nowak1997, Boone2007, Tischler2008}.

Developmental genes in particular, are highly evolutionarily conserved and subject to high rates of \glslink{functional redundancy}{redundancy} \citep{Nowak1997, Kockel1997,Fromental-Ramain1996}. These genes are often difficult to study with conventional functional genetics techniques since individual knockouts of redundant genes may not have a \gls{mutant} phenotype. Identifying genes with a common function is therefore also important to characterise developmental genes. \Gls{synthetic lethal} discovery methods, such as \gls{SLIPT}, represent a \gls{genomic} approach to further systematic characterisation of gene function including highly redundant developmental genes.

Variants of unknown significance and modifier loci are a major concerns in human genetics, including ``monogenic'' and ``rare'' diseases. Many of these variants could be difficult to characterise individually due to \gls{synthetic lethal} interactions, where additional loci contribute to the disease (or only compensate for some variants). Systematic identification of \gls{synthetic lethal} interactions therefore also has applications in the study of these ``oligogenic'' diseases. There could also be similar applications in the study of heritability for traits including agricultural \glslink{genome}{genomic} selection.

%\glslink{functional redundancy}{Genetic redundancy} is also a concern in pharmacology. 
Polypharmacology and network medicine are rationales that account for \glslink{functional redundancy}{genetic redundancy} by using drugs with multiple (known and specific) targets \citep{Hopkins2008, Barabasi2011}. Further characterisation of \gls{synthetic lethal} genes could asist with the design of effective multi-target drugs or combination therapies in a range of therapeutic applications, including molecular targeted therapies against cancer for which combination therapies may alleviate acquired resistance against individual targeted therapies. Characterisation of genetic interactions and combination therapies also has the potential to expand pharmacogenomic investigations. \Glspl{synthetic lethal} may elucidate the impact of genotypes at multiple loci, which lead to adverse effects due to variants in \gls{synthetic lethal} genes.

Redundant functions and \gls{synthetic lethal} interactions also present a means to expand upon the concept of the ``minimal'' \gls{genome} \citep{Hutchison2016}. It is important to account for \gls{essential} gene functions that are performed by redundant genes (or in combination with \glslink{pleiotropy}{pleiotropic} genes), rather than simply those that are perturbed by individual genes. An \gls{essential} gene approach therefore is likely to produce an underestimate that does not account for \gls{synthetic lethal} interactions. 

\Gls{synthetic lethal} interactions are fundamentally important throughout genetics. Further understanding of them in a \gls{genomic} context, facilitated by methods such as \gls{SLIPT}, would contribute towards deeper understanding of gene functions and their role in traits or diseases in the post-genomic era. Genes do not function in isolation and understanding them in the context of the complexity of a cell and across genetic backgrounds is \gls{essential} to further characterise their functions and ensure that findings can be validated or applied beyond experimental systems.


\subsection{Clinical Interventions based on Synthetic Lethality}
\label{chapt6:significance_clinic}

Synthetic lethal discovery with \gls{SLIPT} is of particular relevance to cancer for the discovery \textcolor{black}{of} \gls{synthetic lethal} drug targets. The cancer research community relies on cell line and mouse models for screening and validation experiments \citep{Fece2015} which would benefit from integration with \gls{gene expression} analysis, as demonstrated for \textit{CDH1} and the screen conducted by \citet{Telford2015}. \Gls{synthetic lethal} drug design against cancer \glspl{mutation}, including gene loss or over-expression, could lead to a revolution in cancer \glslink{treatment}{therapy} and \gls{chemoprevention}. These \glslink{targeted therapy}{therapeutics} would enable personalised treatment for cancer patients and high risk individuals.  The \gls{synthetic lethal} strategy for cancer treatment has been shown to be clinically effective \citep{McLachlan2016,Farmer2005, Bryant2005}. Many large-scale \gls{RNAi} screens have been conducted recently, aiming to discover gene function and drug targets for similar application with other cancer genes, including cancers in other tissues \citep{Fece2015}.

%However, there are limitations to both experimental screens and computational approaches, both known to be prone to false-positives.  Modelling and simulation of \gls{synthetic lethal} discovery in \gls{genomic} data has been explored to address these concerns and ensure the validity of candidate \gls{synthetic lethal} interactions, particularly given the recent emergence of a number of conflicting \gls{synthetic lethal} screening and prediction approaches.  Exploring \glspl{synthetic lethal} in simulated data will ensure the optimal performance of our prediction method with comparison to the distribution of test statistic distribution in empirical \gls{gene expression} data, informed selection of thresholds for prediction, and estimated error rates.  The model of \gls{gene expression} with known \gls{synthetic lethal} genes is limited by the assumption that it represents the distribution of \gls{gene expression} when it may not.  Having shown \glspl{synthetic lethal} is detectable in simple models and added correlation structure, the model still needs to be developed to better represent real data.  However, the behaviour of \gls{synthetic lethal} genes and effects of parameters explored so far remains important to inform future model design and interpretation of empirical data analysis.


%Network analysis enables properties of the network and itâ€™s connectivity to be measured and compared across datasets \citep{Barabasi2004}.  Tissue specificity is an important consideration, largely unexplored with \gls{synthetic lethal} studies, since it has clinical importance to ensure targeted drug \glspl{treatment} are effective, predict adverse effects in other tissues, determine whether targeted \glspl{treatment} could be repurposed for other cancer types or diseases, and whether drug resistance mechanisms could emerge.  Comparison of tissues, populations, and species can all ensure that \gls{synthetic lethal} predictions are robust, that experimental candidates are clinically relevant, and \glspl{treatment} designed to exploit them would be specific to the disease in large patient cohort (with known biomarkers).

While \gls{SLIPT} analysis and \gls{RNAi} screens represent a significant step towards anti-cancer medicines, further validation is required to ensure that the \gls{synthetic lethal} candidate genes and pathways identified for \textit{CDH1} in breast and stomach cancer are applicable against \textit{CDH1}-deficient cancers in the clinic.  Validation with \gls{RNAi} or pharmacological inhibitors is needed because false positives may occur in \gls{SLIPT} analysis or \gls{siRNA} screens. These candidates still need to be tested in pre-clinical models (cell lines and mouse xenografts) before proceeding to clinical trials. A therapeutic intervention will also require a \glspl{targeted therapy} \textcolor{black}{to be developed} or repurposed against a \gls{synthetic lethal} partner. 
%Drug targets must be feasible to have effective anti-cancer interventions designed against them, which raises the need for targets with existing drugs in the clinic, trials, or feasible to development with structural analysis or screening. 
Drug targets could be triaged from \gls{synthetic lethal} genes by functions known to be amenable to drugs, having protein structures with conserved specific sites that are not homologous to other genes, or \textcolor{black}{those} with existing drugs. Both structure-aided drug design and compound screening are viable ways to target \gls{synthetic lethal} partners. % accompany genetic screens and computational analysis with pharmacological investigations.

\glslink{targeted therapy}{Targeted therapeutics} designed based on \gls{synthetic lethal} interactions could expand the applications of ``precision medicine'' against molecular targets. %, particularly in cancer where many have been cancer genes have been identified.
\Glspl{synthetic lethal} expands the range of cancer genes which can be (indirectly) targeted to include \gls{tumour suppressor} genes with loss of function, such as \textit{CDH1}. \Glspl{oncogene} with disrupted functions that are over-expressed or highly homologous to non-cancerous proto-oncogenes, such as \textit{MYC}, \textit{EGFR} or \textit{KRAS}, may also be targeted by \glspl{synthetic lethal}. Applications against \gls{tumour suppressor} genes is particularly important, as these cannot be approached by careful dosing. \Gls{synthetic lethal} drug design has the benefit of being highly specific against a particular genotype (e.g., \text{CDH1}\textsuperscript{-/-}) with the potential for \glslink{targeted therapy}{targeted therapies} with a wide therapeutic index and few adverse effects, in contrast to many cancer treatment regimens \citep{Hopkins2008, Kaelin2009}. These properties are highly desirable for \gls{chemoprevention} applications, such as treatment against \textit{CDH1}-deficient in \gls{HDGC} patients \citep{Guilford2010}, as an alternative to monitoring or surgery.

\iffalse
\section{Evaluating the Synthetic Lethality Prediction Tool}
\label{chapt6:slipt}


\subsection{Strength of the Synthetic Lethality Prediction Tool}
\label{chapt6:slipt_strengths}

\subsection{Limitations of the Synthetic Lethality Prediction Tool}
\label{chapt6:slipt_limitations}

\subsection{Comparisons to Alternative Methods}
\label{chapt6:slipt_compare}

\subsubsection{Combined with Experimental Screening}
\label{chapt6:slipt_compare_experimental}

\subsubsection{Differences to Computational Methods}
\label{chapt6:slipt_compare_computational}
\fi

\section{Future Directions}
\label{chapt6:future}

While further validation and pre-clinical testing is required to translate the findings for \textit{CDH1} to cancer therapy or prevention, there are also further avenues for research into the detection of \glspl{synthetic lethal} in \gls{gene expression} and other \glspl{genomic} data. The \gls{SLIPT} methodology is amenable to wider application against a range of genes for which loss of function is deleterious, including other cancer genes in breast cancer or other tissues. \Gls{synthetic lethal} interactions are functionally informative, particularly for mode-of-action of known drug targets, and are also relevant for identifying functions of newly characterised genes in \glspl{genomic} studies and designing specific interventions against cells with loss of function in cancer and other diseases. Thus \gls{synthetic lethal} detection using \gls{SLIPT} in \glslink{gene expression}{expression} data could be further used for many other genes, including others relevant to human health and disease.

These investigations do not need to be limited to \glslink{gene expression}{expression} data. While \glslink{gene expression}{expression} as a measure of gene function has been the focus of this thesis, other \glspl{genomic} data could be used for a similar purpose for \gls{SLIPT} analysis. These include \acrshort{DNA} copy number, \acrshort{DNA} methylation, histone activation, \gls{mutation} status, protein abundance, and protein activation state. In particular, \acrshort{DNA} copy number and \glspl{mutation} have been demonstrated by other approaches to \gls{synthetic lethal} analysis \citep{Jerby2014, Srihari2015, Lu2015, Wappett2016}, although some of these have not been released for wider application. \textcolor{black}{This approach lends itself to any molecular data that can be partitioned in to lowly and highly active states such as deletion and ampliflications for DNA copy number or hypermethylation and hypomethylation for DNA methylation.}

For some applications or genes, these molecular profiles may be more informative of gene function and \gls{synthetic lethal} relationships. However, \glslink{gene expression}{expression} was the focus of the investigations thus far as a widely accepted measure of gene function which has widely available \glspl{genomic} data.  \gls{SLIPT} is compatible with each of these data types (if the thresholds are selected appropriately) and may perform better for some applications with these molecular profiles or a weighted combination of these. As demonstrated, \gls{SLIPT} is also suitable for future investigations with pathway \glspl{metagene} and other summary data as well.

It may also be possible to improve the performance of \gls{SLIPT} with refinements to the statistical or computational approach. This thesis has focused on rational query-based approach which computes relatively quickly in R \citep{R_core}, and is relatively intuitive to interpret. These computations are compatible with parallel computing and the computational resources may be further reduced by using a different computing language. The \texttt{slipt} R package has been documented and released as open-source software (as described in Section~\ref{methods:r_packages}) to facilitate further development, wider adoption, or comparison with other scientific software for similar purposes. 

Alternative methods \textcolor{black}{may also} improve on the statistical performance of \gls{SLIPT}. In particular, the sensitivity was generally \textcolor{black}{an issue} with higher numbers of \gls{synthetic lethal} partners in simulated data. While approaches using continuous data such as Pearson correlation and linear regression did not perform as well as \gls{SLIPT}, they could be improved. A least squares regression approach in particular, enables multiple measures of relationships such as the coefficients of the fitted curve and significance of the fit (computed from the residuals). A linear modelling approach using regression is also amenable to refinement such as extending from fitting a linear relationship to a polynomial or logistic regression. Another benefit to fitting linear models is that these would enable the conditioning of known \gls{synthetic lethal} partners to identify subtle signatures of further interacting partners.

This approach could also be applied iteratively on the strongest candidates from previous \gls{synthetic lethal} analyses in further rounds of prediction conditioned upon them. Similarly, \gls{synthetic lethal} prediction could also be approached with a Bayesian framework \citep{Friedman2000, Jansen2003, Imoto2004} which is also amenable to Bayesian priors on known or previously predicted \gls{synthetic lethal} partners. Either of these approaches has the potential to improve upon the \gls{synthetic lethal} predictions which have been demonstrated as possible and biologically relevant by \gls{SLIPT}. \textcolor{black}{Another approach to overcome the issue for loss of information from categorisation of expression data to estimate gene function would be \textcolor{red}{to} estimate the underlying hidden state with a likelihood model instead. The model of synthetic lethality (in Section~\ref{methods:SL_Model}) would serve as a good foundation for this approach.}

\textcolor{black}{
Using another data types, such as DNA methylation, may provide another perspective on synthetic lethal candidates from molecular profiles but this wealth of information becomes difficult to interpret stringently as \gls{DAISY} has been critiqued for \citep{Lu2015}. These analysis could be combined to improve the results of the SLIPT approach: either by using a combination of molecular data to estimate gene function and partition samples or to perform SLIPT separately for each and weight the results. These are likely to be subject to diminishing returns for additional computations required. For instance, DNA methylation influences gene expression and may not add much information about gene function if expression is already known. However, a weighting of these results could improve the accuracy of the SLIPT procedure. In particular, a combination of SLIPT results for different molecular data across different partitioning thresholds could be optimised with a machine learning approach.   
}
%\chapter{Conclusion}
%\label{chap:conclusion}
%\clearpage
\section{Conclusions}
\label{chap:conclusion}

Synthetic lethal interactions are important for understanding gene function and the development of highly specific \glslink{targeted therapy}{targeted} cancer \glspl{treatment}. In particular, \glspl{synthetic lethal} could expand the repertoire of applications for precision cancer medicine by indirectly targeting loss of function in \gls{tumour suppressor} genes.  \Gls{synthetic lethal} discovery with experimental screening is error-prone and limited by the model systems in which it is performed. Thus there is a need for a \gls{bioinformatics} tool to predict \gls{synthetic lethal} interactions from \gls{gene expression} data, which would facilitate the rapid identification of \gls{synthetic lethal} candidates, and augment functional genetic screens and triage of cancer drug targets. This thesis develops the \acrfull{SLIPT} methodology as a statistically robust procedure to perform this analysis.

The \gls{SLIPT} methodology has been demonstrated to identify biologically relevant genes and pathways. A comprehensive analysis of \gls{synthetic lethal} partners of the \textit{CDH1} gene was performed in \gls{TCGA} breast cancer data \citep{TCGA2012}, with many of these findings replicated in stomach cancer data \citep{TCGA2014GC}. These genes clustered into several distinct groups, with distinct biological functions and elevated \glslink{gene expression}{expression} in different clinical subtypes.  These analyses identified \gls{synthetic lethal} candidates in the $G_{\alpha i}$ signalling, cytoplasmic microfibres, and extracellular fibrin clotting pathways. These \glspl{pathway} were supported by an \gls{siRNA} screen performed by \citet{Telford2015} and were consistent with the known cytoskeletal and cell signalling roles of \gls{E-cadherin}. 
%These findings support interventions against these pathways being applicable to specific cancer therapeutics beyond the pre-clinical cell line models in which they were validated. 
\gls{SLIPT} also identified \gls{synthetic lethal} partners in novel pathways for \textit{CDH1}, including the regulation of immune signalling and translational elongation, which extend the range of established functions of \textit{CDH1} and present further biological mechanisms %to investigate the malignancy and
that can be investigated to exploit the
vulnerabilities of \textit{CDH1}-deficient cancers.

While some of these pathways are not expected to be detected in an isolated experimental cell line model, \glslink{graph}{pathway} structure may have accounted for this disparity. Thus \gls{synthetic lethal} candidates detected by \gls{SLIPT} and \gls{siRNA} were compared within \glslink{graph}{graph} structures of the candidate \gls{synthetic lethal} pathways. However, this did not generally account for differences between these approaches. Neither \gls{synthetic lethal} detection methodology preferentially detected genes of more importance or connectivity in \glslink{graph}{pathway} structures using established network metrics, nor could it be generally established that \gls{SLIPT} gene candidates were upstream or downstream of \gls{siRNA} gene candidates in \glslink{graph}{pathway} structures across biological pathways. However, it could be shown that \gls{SLIPT} genes had lower centrality and were upstream of \gls{siRNA} candidates, specifically in the $G_{\alpha i}$ signalling pathway.

Pathway \glslink{graph}{graph} structures were also included in investigations with simulated data to ascertain whether the \gls{SLIPT} procedure performed well in data with complex correlation structures derived based on biological pathways. A simulation procedure was developed based on a statistical model of \glspl{synthetic lethal} which generates multivariate normal data with known \gls{synthetic lethal} partners and correlation structures. The \gls{SLIPT} methodology \textcolor{black}{performed well at detecting synthetic lethality in simulated expression data}, particularly when detecting few known \gls{synthetic lethal} genes, with large sample sizes, and a background of many non-synthetic lethal genes to distinguish true partners from. This method had high specificity, performed better than Pearson correlation or the $\chi^2$-test, and \textcolor{black}{had} optimal performance across simulation parameter combinations for the thresholds used throughout this thesis. These findings were robust across correlation structures, including those derived from complex \glslink{graph}{pathway} structures containing strong positive and negative correlations between genes. 
Together, these findings support the release of the \gls{SLIPT} software R packages and the application of the method to identify \gls{synthetic lethal} genes within pathways and use candidate \gls{synthetic lethal} genes to identify \gls{synthetic lethal} pathways, as demonstrated in this thesis.

%In summary, I present a widely applicable \gls{synthetic lethal} procedure using \gls{gene expression} data for wider use in \glspl{genomic} research, including the development of precision cancer medicine. This methodology is supported by the release of a software package in R, simulation results based on a statistical model of \glspl{synthetic lethal}, the demonstration of \gls{bioinformatics} and network biology investigations into interactions with the \textit{CDH1} gene in breast and stomach cancers. 


\iffalse
\clearpage
\paragraph{Aims}

  \begin{itemize}
   \item To develop a statistical approach to detect \gls{synthetic lethal} gene pairs in cancer from \glslink{gene expression}{expression} data

   \bigskip
   
   \item To apply this methodology to public cancer \gls{gene expression} data against \textit{CDH1} and analyse \glslink{graph}{pathway} structure with comparisons to experimental screen data

   \bigskip
   
   \item To construct a statistical model of \glspl{synthetic lethal} in multivariate normal \glslink{gene expression}{expression} data
 
   \bigskip
   
   \item To develop a simulation pipeline of \glslink{gene expression}{expression} with \glslink{graph}{pathway} structure on a high-performance computing cluster 

   \bigskip
   
   \item To examine the statistical performance of the methodology with simulated \glslink{gene expression}{expression} including pathways and compare it to other approaches

   \bigskip
   
   \item To release the \gls{synthetic lethal} detection methodology and pathway simulation procedure as R software packages
   
  \end{itemize}
  

\clearpage
  
 \paragraph{Summary}
 
   \begin{itemize}
   \item I have developed a Synthetic Lethal detection method that generates a high number of \gls{synthetic lethal} candidates
   
   \bigskip
   
   \item Pathways in cell signalling, extracellular matrix, and cytoskeletal functions were supported with experimental candidates and the known functions of \gls{E-cadherin}
   
   \bigskip
   
   \item Several candidate pathways were supported by \gls{mutation} analysis and replicated across breast and stomach cancer
   
   \bigskip
   
   \item Translation and immune functions were uniquely detected by the computational approach which may be explained by differences between patient samples and cell line models
   
   \bigskip
   
   \item There remains the need to identify actionable genes within these pathways, relationships with experimental candidates, and how these pathways may affect viability when lost
  \end{itemize}
  
    \begin{itemize}
   \item Synthetic Lethal genes were explored within a \glslink{graph}{graph} structures for key pathways identified previously 
   
   \bigskip
   
   \item In some cases these \glslink{graph}{graph} structures appeared to have relationships between \gls{synthetic lethal} genes  
   
   \bigskip
   
   \item However, no existing network metrics of importance and connectivity with the networks were elevated significantly for Synthetic Lethal genes
   
   \bigskip
   
   \item Nor was there significant evidence of upstream and downstream relationships between SLIPT and \gls{siRNA} Candidates in a \gls{shortest path} permutation analysis
  \end{itemize}
  
      \begin{itemize}
      \item I have designed a straight-forward rational query-based \gls{synthetic lethal} detection method with the example of application to \textit{CDH1} in cancer \gls{gene expression}
      
      \bigskip
      
      \item I have developed a simulation pipeline to generate continuous \gls{gene expression} with \glslink{graph}{pathway} structure including a procedure to simulate \glspl{synthetic lethal} 
      
      \bigskip
      
      \item Our simulation procedure is robust across \glslink{graph}{pathway} structures and has desirable performance compared to other statistical techniques 
      \end{itemize}
 \fi