\chapter{Simulation and Modeling of Synthetic Lethal Pathways}
\label{chap:simulation}

Simulation and modelling of \glspl{synthetic lethal} in \gls{gene expression} is revisited in greater detail in this chapter, building upon the results which supported the use of \gls{SLIPT} (in Section~\ref{chapt2:simulation_2015}) . In Chapter~\ref{chap:methods_dev}, a simulation procedure for generating simulated data with underlying (known) \gls{synthetic lethal} partners of a query gene, such as \textit{CDH1}, was developed (as described in Section~\ref{methods:simulating_SL}) by sampling from a Multivariate normal distribution based on a statistical model of \glspl{synthetic lethal} in \glslink{gene expression}{expression} data (as described in Section~\ref{methods:SL_Model}). This simulation framework was applied to simulated data (in Section~\ref{chapt2:simulation_2015}), including simple correlation structures to assess the statistical performance of the \gls{SLIPT} methodology and support its use as a computational approach for detecting \gls{synthetic lethal} candidates from \glslink{gene expression}{expression} data throughout this thesis (Chapters~\ref{chap:SLIPT} and~\ref{chap:Pathways}). 

While this basic framework was provided some support for the use of \gls{SLIPT}, further investigations with simulations were conducted to assess the strengths and limitations of the \gls{SLIPT} methodology, compare it to alternative statistical approaches to \gls{synthetic lethal} detection, and assess its performance under more complex correlation structures. Together these simulation investigations assess the performance of the \gls{SLIPT} methodology, including on pathway \gls{graph} structures (e.g., those discussed in Chapter~\ref{chap:Pathways}) and indicate whether the \gls{SLIPT} methodology (or similar refined \gls{bioinformatics} strategies) are statistically rigourous or suitable for wider \glspl{genomic} applications.

These simulation investigations continue to utilise the Multivariate Normal simulation procedure (as applied in Section~\ref{chapt2:simulation_2015}) with further refinements. The \gls{SLIPT} methodology (and the $\chi^2$ test) were applied across a range of parameters (including altering the quantiles for detecting \gls{synthetic lethal} direction and compared to correlation). This was also applied to query correlated genes (as performed in Section~\ref{chapt2:simulation_2015}).

A refined simulation procedure was developed specifically to extend the methodology described in Section~\ref{methods:simulation_SL_expression} to utilise pathway \gls{graph} structures for the correlation structures of simulated datasets (as described in Section~\ref{methods:graphsim}). This methodology can be applied to simulated correlation structures across simple \gls{graph} structures to test specific network modules or use \glslink{graph}{pathway} structures based on biological pathways. Thus \gls{graph} structure and simulation approaches were combined to test whether a gene locus in a pathway affects detection by \gls{SLIPT} and whether \gls{SLIPT} performance is affected by \glslink{graph}{pathway} structure. The simulation procedure based on \gls{graph} structures was applied in a computational pipeline across many parameter combinations using high-performance computing resources (as discussed in Section~\ref{methods:HPC}) and the core simulation functions have been released as a software package for wider use to test \gls{bioinformatics} and statistical methods on \gls{graph} structures (as described in Section~\ref{methods:igraph_extensions}).

%\section{Simulations and Modelling Synthetic Lethality in Expression Data}
%%committee
\iffalse
Synthetic lethality was modelled for effects on \glslink{gene expression}{expression} levels and whether these are detectable in known interacting and non-interacting genes in simulated data. These were conducted for \glslink{gene expression}{expression} data but the nature of these simulations would be relevant to how \glspl{synthetic lethal} would manifest in other factors, particularly \acrshort{DNA} \gls{copy number} variation and \acrshort{DNA} methylation. These simulations were discussed at length in the previous meeting and showed that \glspl{synthetic lethal} was detectable with our approach in simple cases. While it was less effective, the methods were able to detect \gls{synthetic lethal} genes in \glslink{gene expression}{expression} data with correlation structure (generated with the multi-variate normal distribution) and were distinguishable from correlated genes. Therefore the strongest (most significant) \gls{synthetic lethal} genes are more likely to be true \gls{synthetic lethal} partners and a high number of hits are expected from correlated genes and co-regulated pathways.

The power of the method to detect interactions depleted with increasing multiple tests, interactions, and cryptic (third party) interacting partners. Increased sample size counteracted these effects as expected. This led the idea that pathways would be more suitable as the focus of this project. Biological pathways led to fewer multiple tests, more relevant to understanding cancer biology, and are often drug targets in practice.
\fi

\section{Synthetic Lethal Detection Methods} \label{chapt5:compare_methods}

The \gls{SLIPT} methodology (as it has been applied throughout Chapters~\ref{chap:SLIPT} and~\ref{chap:Pathways}) was compared for alternative computational approaches to detecting \glspl{synthetic lethal} in simulated \gls{gene expression} data. As discussed in Section~\ref{chapt2:simulation_2015}, this procedure enabled testing ability of \gls{SLIPT} to detect known \gls{synthetic lethal} partner genes by sampling from a statistical model of \glspl{synthetic lethal}. While comprehensive benchmarking has not been performed, several approaches to \gls{synthetic lethal} detection are considered (e.g., Pearson correlation, the $\chi^2$ test, and testing for bimodality) to evaluate the strengths of the \gls{SLIPT} methodology, including modifications to the parameters of \gls{SLIPT}.

%Further testing of the performance of the SLIPT software R package (which is publicly released on GitHub as described in Section~\ref{methods:r_packages}) has been left to third party researchers to impartially compare it to other software for \gls{synthetic lethal} detection which is outside the scope of this thesis.
The following comparisons of simulations of computational detection of \glspl{synthetic lethal} with different statistical rationales suffice to discuss the strengths of \gls{SLIPT}, evaluate whether it is appropriate for further application in \glspl{genomic} research, and identify limitations which may be addressed with further developments. Some potential avenues for further development of computational \gls{synthetic lethal} discovery will be discussed in Section~\ref{chapt6:future}.

\subsection{Performance of SLIPT and $\chi^2$ across Quantiles}
\label{chapt5:compare_chisq}

Simulated datasets with \gls{synthetic lethal} partner genes were generated using the multivariate normal simulation procedure (as described in Section~\ref{methods:simulating_SL}) with performance assessed using \gls{AUROC} analysis (as described in Section~\ref{methods:simulation}). \Gls{synthetic lethal} detection was compared for modifications to the \gls{SLIPT} methodology (as described in Section~\ref{methods:SLIPT}), namely that the quantiles used to define low and high \glslink{gene expression}{expression} was varied. Rather, than $\sfrac{1}{3}$ (as used throughout this thesis) the samples below the lowest $\sfrac{1}{n}$ quantile and above the highest $\sfrac{1}{n}$ quantile were used for \gls{SLIPT} (and the $\chi^2$-test) to detect lowly and highly expressing samples respectively. The quantiles tested range from 2, splitting at the $\sfrac{1}{2}$ quantile (the median), to 100, using the lowest (1\%) and highest (99\%) percentiles.

This enabled testing of the threshold for lowly expressing genes which is most able to distinguish \gls{synthetic lethal} genes, even with higher-order \gls{synthetic lethal} interactions (as discussed in Section~\ref{methods:SL_Model}). Both \gls{SLIPT} with the directional criteria for \glspl{synthetic lethal} and significance of the equivalent $\chi^2$ test were performed for each quantile. Pearson correlation was also tested on simulated continuous \glslink{gene expression}{expression} data for \gls{synthetic lethal} detection in simulated data, considering both positive and negative correlations separately as predictors of \glspl{synthetic lethal} for comparison with $\chi^2$ based approaches, using discete categories fo gene function deriving from quantiles. 

The results presented throughout this section use the example of 5 \gls{synthetic lethal} partners to illustrate the differences in performance between the standard \gls{SLIPT} procedure (slipt-3) to $n$ quantiles (slipt-$n$), the $\chi^2$-test on the same quantiles, and positive or negative correlation. However, similar results across different numbers of known \gls{synthetic lethal} genes are shown in Appendix~\ref{appendix:compare_chisq}. The \gls{synthetic lethal} detection procedures were compared with 10,000 simulations of a small dataset of 100 genes and 1000 samples without correlation structure between genes, as performed in Section~\ref{chapt2:simulation_mvtnorm}) . As shown in Figure~\ref{fig:simulation1108_Graph5All5SL}, the 3-quantiles previously used have optimal performance and \gls{SLIPT} has a comparable or higher performance than the $\chi^2$-test alone across quantiles.

    \begin{figure*}[!htbp]
    %\begin{mdframed}
    \begin{center}
%
        \subcaptionbox{Barplot of $\chi^2$, SLIPT, and correlation.}{%
            \label{fig:simulation1108_Graph5All5SL:barplot}
            \includegraphics[width=0.5\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(5)".png}}
        }%

        \subcaptionbox{Lineplot of $\chi^2$, SLIPT, and correlation.}{%
            \label{fig:simulation1108_Graph5All5SL:lineplot}
            \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png}}
        }%
      \end{center}
      \caption[Performance of $\chi^2$ and SLIPT across quantiles]{\textbf{Performance of $\chi^2$ and SLIPT across quantiles}. \Gls{synthetic lethal} detection (of 5 genes) with quantiles as on the axes. The barplot uses the same hues for each quantile (grey for correlation) and darker for $\chi^2$ (and positive correlation). The line plot (with log-scale quantiles) is coloured according to the legend. \gls{SLIPT} and  $\chi^2$ perform similarly, peaking at $\sfrac{1}{3}$-quantiles and converging to random (0.5). Negative correlation was higher than positive but not optimal quantiles for \gls{SLIPT} or $\chi^2$.}
    \label{fig:simulation1108_Graph5All5SL}
    %\end{mdframed}
    \end{figure*}

Pearson correlation was also tested as a predictor of \glspl{synthetic lethal} (i.e., whether highly positive or negative correlations with the query gene detected \gls{synthetic lethal} partners). Positive correlation performed worse than random (with an \gls{AUROC} lower than 0.5) as thus coexpression of genes is not predictive of \glspl{synthetic lethal} in simulated data. Conversely, negative correlation is predictive of \glspl{synthetic lethal}, consistent with \gls{synthetic lethal} gene activity being mutually exclusive. However, neither correlation approach performed as well as the optimal quantiles for the \gls{SLIPT}  procedure or $\chi^2$-test.

These results are shown in both a bargraph and lineplot to show the individual results of each parameter, and to compare \gls{SLIPT} with the $\chi^2$-test side-by-side across quantiles. Similarly, these plots are given for detecting a range of known \gls{synthetic lethal} partners in the simulations in Figures~\ref{fig:simulation1108_Graph5Allbarplot} and~\ref{fig:simulation1108_Graph5Alllineplot}. These demonstrate that the findings shown for 5 \gls{synthetic lethal} genes are robust across different numbers of underlying \gls{synthetic lethal} genes.

    \begin{figure*}[!tb]
    %\begin{mdframed}
    \begin{center}
%
        %\subcaptionbox{Performance of $\chi^2$ and SLIPT}{%
        %    \label{fig:simulation1107_Graph5All5SL:barplot}
        %    \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_Cor_v_nCor_Allxy(5)".png}}
        %}%

        %\subcaptionbox{Comparison to Correlation}{%
        %    \label{fig:simulation1107_Graph5All5SL:lineplot}
            \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png}}
        %}%
      \end{center}
      \caption[Performance of $\chi^2$ and SLIPT across quantiles with more genes]{\textbf{Performance of $\chi^2$ and SLIPT across quantiles with more genes}. \Gls{synthetic lethal} detection (of 5 genes in 20,000) with quantiles as in axis labels. The line plot (with log-scale quantiles) is coloured according to the legend. As for simulations with fewer genes, \gls{SLIPT} and  $\chi^2$ perform similarly, peaking at $\sfrac{1}{3}$-quantiles and converging to random (0.5). Negative correlation was higher than positive but not optimal quantiles for \gls{SLIPT} or $\chi^2$.}
    \label{fig:simulation1107_Graph5All5SL}
    %\end{mdframed}
    \end{figure*}

The \gls{synthetic lethal} detection procedures were also tested with 1000 simulations of a larger dataset of 20,000 genes and 1000 samples. While fewer simulations gives a less accurate \gls{ROC} result, this is sufficient to replicate the above findings with a feasible number of genes in a human \gls{gene expression} dataset and assess the impact of a higher proportion of non \gls{synthetic lethal} genes (potential false positives). Simulated datasets of this size were also used in Section~ \ref{chapt2:simulation_mvtnorm} to test the specificity in a number of genes similar to that in experimental datasets for cancer \glspl{genome}. As shown in Figure~\ref{fig:simulation1107_Graph5All5SL}, the above findings were replicated in  simulations of a larger dataset with 20,000 genes. These were also robustly replicated across varying numbers of underlying \gls{synthetic lethal} genes (as shown in Figure~\ref{fig:simulation1107_Graph5Alllineplot}).

\FloatBarrier
    
\subsubsection{Correlated Query Genes affects Specificity}
\label{chapt5:compare_chisq_query_cor}

\FloatBarrier

    \begin{figure*}[!tb]
    %\begin{mdframed}
    \begin{center}
%
        %\subcaptionbox{Performance of $\chi^2$ and SLIPT}{%
        %    \label{fig:simulation1107_Graph5All5SL:barplot}
        %    \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_Cor_v_nCor_Allxy(5)".png}}
        %}%

        %\subcaptionbox{Comparison to Correlation}{%
        %    \label{fig:simulation1107_Graph5All5SL:lineplot}
            \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png}}
        %}%
      \end{center}
       \caption[Performance of $\chi^2$ and SLIPT across quantiles with query correlation]{\textbf{Performance of $\chi^2$ and SLIPT across quantiles with query correlation}. \Gls{synthetic lethal} detection (of 5 genes in 100 including 5 query correlated) with quantiles as in axis labels. The line plot (with log-scale quantiles) is coloured according to the legend. \gls{SLIPT} performs consistently higher than $\chi^2$ due to higher specificity. Negative correlation performed modestly.}
    \label{fig:simulation1108_query_cor_Graph5All5SL}
    %\end{mdframed}
    \end{figure*}

    
As discussed in Section~\ref{chapt2:simulation_mvtnorm_query_cor}, positively correlated genes (with the query gene) have an impact of on the performance of \gls{synthetic lethal} detection. \gls{SLIPT} was able to distinguish these from \gls{synthetic lethal} partners and hence is likely to have a higher specificity in datasets which include positively correlated genes with the query gene (as expected in \gls{gene expression} data). The \gls{synthetic lethal} detection procedures were compared with 10,000 simulations of a small dataset of 100 genes (with 5 correlated with the query gene) and 1000 samples otherwise without correlation structure between genes. As shown in Figure~\ref{fig:simulation1108_query_cor_Graph5All5SL}, this specificity is reflected in the increased \gls{AUROC} performance values for \gls{SLIPT} (in contrast to Figure \ref{fig:simulation1108_Graph5All5SL}). This specificity can be attributed to the directional criteria (as described in Section~\ref{methods:SLIPT}) since the $\chi^2$-test alone performs comparatively poorly with positively correlated genes.
    
The \gls{synthetic lethal} detection procedures were also compared with 1000 simulations of a larger dataset of 20,000 genes (with 1000 correlated with the query gene) and 1000 samples otherwise without correlation structure between genes. This simulation increases the number of genes (and proportion of negative genes) to those comparable with a human \gls{gene expression} dataset while maintaining a comparable 5\% of positively correlated genes. As shown in Figure~\ref{fig:simulation1107_query_cor_Graph5All5SL}, \gls{SLIPT} still outperforms $\chi^2$ or negative correlation and is optimal at the 3-quantile. The difference between \gls{SLIPT} and $\chi^2$ was less pronounced in a larger dataset with many weakly correlated genes. The greater specificity of \gls{SLIPT} than $\chi^2$-test to distinguish positively correlated non \gls{synthetic lethal} genes is not as evident with a large number of negative genes (as potential false positives). However, specificity is an important consideration in large-scale \glspl{genomic} analysis where there are potentially many false positives.

   
    \begin{figure*}[!tb]
    %\begin{mdframed}
    \begin{center}
%
        %\subcaptionbox{Performance of $\chi^2$ and SLIPT}{%
        %    \label{fig:simulation1107_Graph5All5SL:barplot}
        %    \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_Cor_v_nCor_Allxy(5)".png}}
        %}%

        %\subcaptionbox{Comparison to Correlation}{%
        %    \label{fig:simulation1107_Graph5All5SL:lineplot}
            \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png}}
        %}%
      \end{center}
      \caption[Performance of $\chi^2$ and SLIPT across quantiles with query correlation and more genes]{\textbf{Performance of $\chi^2$ and SLIPT across quantiles with query correlation and more genes}. \Gls{synthetic lethal} detection (of 5 genes in 20,000 including 1000 query correlated) with quantiles as in axis labels. The line plot (with log-scale quantiles) is coloured according to the legend. \gls{SLIPT} performs consistently higher than $\chi^2$ due to higher specificity. Negative correlation performed modestly.}
    \label{fig:simulation1107_query_cor_Graph5All5SL}
    %\end{mdframed}
    \end{figure*}

Nevertheless, \gls{SLIPT} with 3-quantiles (as performed throughout Chapters~\ref{chap:SLIPT} and~\ref{chap:Pathways}), had higher performance than  when other quantile thresholds were used, particularly when positive correlations were present (replicating the Section~\ref{chapt2:simulation_mvtnorm_query_cor}). These findings hold across different numbers of underlying \gls{synthetic lethal} genes (as shown in Figures~\ref{fig:simulation1108_query_cor_Graph5Alllineplot} and ~\ref{fig:simulation1107_query_cor_Graph5Alllineplot}).

Together these results support the use of \gls{SLIPT}, particularly the use of quantiles as thresholds for gene function and specific use of 3-quantiles which perform well compared to other quantiles. A particular concern in the design of \gls{SLIPT} for \glslink{gene expression}{expression} data whether the  samples sizes are sufficient when the data is divided into quantiles. The \gls{SLIPT} methodology further performed better for 3-quantiles (and other moderate values) than $\chi^2$ or correlation as a predictor of \glspl{synthetic lethal}. These results are irrespective of sample size or p-value threshold since the results replicated across sample sizes and the \gls{AUROC} values were independent significance thresholds. Using a moderate number of quantiles for \gls{SLIPT} ensures that there are a sufficient number of samples expected below and above them so that deviations from these are statistically detectable. These quantiles were also optimal for the $\chi^2$ test which uses the same expected values asthe \gls{SLIPT} directional conditions.

\FloatBarrier

\subsection{Alternative Synthetic Lethal Detection Strategies}

The \gls{SLIPT} approach (and $\chi^2$) to detect \glspl{synthetic lethal} from binning \glslink{gene expression}{expression} to estimate gene function also outperforms correlations which use continuous data directly. Correlation performing poorly as a \gls{synthetic lethal} detection strategy consistent with there not necessarily being a relationship between \gls{synthetic lethal} partners which can be in distinct biological pathways, expressed at different times or in different cell types. Nevertheless, correlation is among the alternative detection methods considered in further detail.

The \gls{BiSEp} R package \citep{Wappett2014} for using bimodality to detect \glspl{synthetic lethal} \citep{Wappett2016} were also considered, along with a linear regression approach. These statistical methods span a range of computational approaches to detecting \glspl{synthetic lethal} and serve to compare alternatives to \gls{SLIPT}, supporting its design and application.
%These investigations are intended not intended to be a comprehensive benchmarking of existing \gls{synthetic lethal} tools, implementing other \gls{synthetic lethal} detection software is out of the scope of this project.
However, these comparisons are able provide supporting data from statistical modelling and simulations for the viability of the \gls{SLIPT} methodology for \gls{synthetic lethal} discovery in cancer (as demonstrated in Chapter~\ref{chap:SLIPT}) and further applications.

\subsubsection{Correlation for Synthetic Lethal Detection}
\label{chapt5:compare_correlation}

\FloatBarrier

As shown in Section~\ref{chapt5:compare_chisq}, negative (Pearson) correlation performed better than positive correlation, indicating the inverse relationships were more predictive of \glspl{synthetic lethal}. However, neither correlation approach performed as well as \gls{SLIPT} or the $\chi^2$ test as a predictor of \gls{synthetic lethal} gene partners. It is notable that negative correlation still often performed considerably better than random chance.

    \begin{figure*}[!htb]
    %\begin{mdframed}
    \begin{center}
      \resizebox{1 \textwidth}{!}{
	\includegraphics{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_Cor_randx/SL_Model_Test_Graph_10K_Graph4_ROC_Compare(1)_SLIPT_v_Cor".png}}
      }
      \end{center}
      \caption[Performance of negative correlation and SLIPT]{\textbf{Performance of negative correlation and SLIPT}. \Gls{synthetic lethal} detection with \gls{SLIPT} was compared to negative (Pearson) correlation across parameters. SLIPT consistently outperformed correlation. Both approaches had lower performance for more \gls{synthetic lethal} partners and lower sample sizes. 10,000 simulations were performed with correlation structure as used in Section~\ref{chapt5:graphsim_performance_constructed}}
    \label{fig:simulation1205_randx_Graph4cfnCor}
    %\end{mdframed}
    \end{figure*}
    
    
Negative correlation was compared directly to the \gls{SLIPT} methodology (as described in Section~\ref{methods:SLIPT}) across numbers of known \gls{synthetic lethal} partners and sample size (ranging from 500 to 5000). This comparison used 1000 simulations of a dataset with 20,000 genes and \gls{synthetic lethal} genes from within a network (sampled as in Section~\ref{methods:graphsim})) with a 0.8 correlation between adjacent genes. In a direct comparison of \gls{SLIPT} and negative correlation (shown in Figure~\ref{fig:simulation1205_randx_Graph4cfnCor}), \gls{SLIPT} consistently has higher performance in simulated data across parameter values and (inverse) correlation-based approaches perform modestly in comparison. Thus using thresholds to categorise \glslink{gene expression}{expression} data (as performed by \gls{SLIPT} and $\chi^2$) does not compromise the performance of these methods by losing continuous data that would be used for calculating correlations. Similarly, the slope of a linear regression did not perform as well at \gls{synthetic lethal} detection than \gls{SLIPT}.
%[Add Other Graphs to Appendix?]


Both \gls{SLIPT} and correlation had poorer performance with increasing numbers of the \gls{synthetic lethal} genes to detect, while they had higher performance in higher sample sizes, as expected (as previously observed for \gls{SLIPT} in Section~\ref{chapt2:simulation_2015}). Thus the issue with detection of greater numbers of \gls{synthetic lethal} genes is not specific to \gls{SLIPT} but occurs across computational methods of \gls{synthetic lethal} discovery in (simulated) \glslink{gene expression}{expression} data and likely stems from cryptic higher-order \gls{synthetic lethal} interactions (as conservatively assumed in Section~\ref{methods:SL_Model}). 
    
\FloatBarrier

\subsubsection{Testing for Bimodality with BiSEp}
\label{chapt5:compare_bisep}

Extensive attempts were also made to compare \gls{SLIPT} to the \gls{BiSEp} methodology \citep{Wappett2016}, a statistical approach to identify \gls{synthetic lethal} gene pairs from mutually exclusive relationships using bimodal distributions. This \gls{synthetic lethal} detection methodology is also designed for \glslink{gene expression}{expression} analysis in cancer and is readily available as an (open-source) R package \citep{Wappett2014}, a practice which facilitates adoption and testing of the methodology on the same datasets and simulations procedures as previously used for \gls{SLIPT}.

The \gls{BiSEp} package is designed for global testing of all potential gene pairs in the \glspl{genome} for \glspl{synthetic lethal} rather than focusing on the search space of  potential partners of the query gene. This approach was unable to detect \gls{synthetic lethal} gene pairs in the \gls{TCGA} breast cancer \glslink{gene expression}{expression} dataset \citep{TCGA2012}. However, this may be due to stringent thresholds under the multiple testing of millions of potential gene pairs.

For a direct comparison with the query-based \gls{SLIPT} approach, the source code of the \gls{BiSEp} R functions was modified to test solely for the partners of a specific gene. This approach was still unable to detect \gls{synthetic lethal} partners of \textit{CDH1} in \gls{TCGA} breast cancer \glslink{gene expression}{expression} data \citep{TCGA2012}, even with the detection thresholds for bimodality and significance greatly relaxed from those which the package defaults to.

To circumvent multiple testing issues, \gls{BiSEp} only tests gene pairs for \glspl{synthetic lethal} between genes with a detectable bimodal distribution. However, even with relaxed thresholds, bimodal distributions were not detectable in the normalised \gls{TCGA} data \citep{TCGA2012}. Such normalisation \cite{limma} is standard practice for \glslink{gene expression}{expression} datasets generated from \glspl{microarray} or \gls{RNA-Seq} and therefore \gls{BiSEp} may not be appropriate to apply to this data. However, it is noted that \gls{BiSEp} may also use other data types such as \acrshort{DNA} \gls{copy number} or cell line data for which it may be more applicable \citep{Wappett2016}.

Nevertheless, attempts were made to test \gls{BiSEp} on simulated datasets with underlying \gls{synthetic lethal} genes (using the procedures described in Sections~\ref{methods:simulating_SL} and~\ref{methods:graphsim}). However, \gls{BiSEp} was also unable to detect genes with bimodal distributions of genes (and thus unable to detect \glspl{synthetic lethal}) in a limited number of computationally intensive simulations. Therefore investigations on a wider range of parameters were not performed.

%%move to discussion

\iffalse

\begin{itemize}
 \item Designed for global SL
 \item Unable to detect SL partners in \gls{TCGA} data
 \item Source code modified to test partners of query gene (R package)
 \item Still unable to identify SL genes for CDH1 in TCGA
 \item Computationally-intensive, longer to run than SLIPT, more difficult to evaluate many iterations
 \item Unable to identify SL candidates in a limited number of simulations
 \item Assumes Bi-modal distribution detectable: not appropriate for normalised \glslink{gene expression}{expression} data (standard in the \acrshort{RNA}-Seq analysis) or ranked (metagenes) -- may be applicable to other datasets
 \item Comparing software is non-trivial (even those released as R packages), the above results are sufficient to evaluate \gls{SLIPT}, and further benchmarking out of scope.
\end{itemize}

\subsubsubsection{Implementation and Computation Time}
\label{chapt5:compare_compute_time}
\fi

%[Compare runtime?]

\iffalse
\begin{itemize}
 \item ChiSQ
 \item SLIPT
 \item Correlation
 \item BiSEp
 \item LM/GLM
\end{itemize}


\subsubsection{Testing Synthetic Lethal Genes with Linear Models}
\label{chapt5:compare_linear_model}
[Move to future Dir??]

\begin{itemize}
 \item Strategy to detect SL with linear models by fit to curve (significance) and slope (direction)
 \item Amenable to conditioning on known SL or iterative conditioning on strongest SL to detect other partners of higher-order SL
 \item All attempts: linear, GLM, and linear polynomial (quadratic, cubic, or quintic) underperform \gls{SLIPT}, similar to Pearson correlation results
 \item Linear models and regression may still be an avenue for further detection of SL (e.g., with Bayes)
\end{itemize}
\fi

%[Discuss linear models?]

%\subsection{Linear models}

\iffalse
\section{Developing a linear model predictor of \glspl{synthetic lethal}}
\subsection{Linear models}
\subsection{Polynomial models}
\subsection{Conditioning}
\subsection{SLIPTv2}
\fi

\section{Simulations with Graph Structures}
\label{chapt5:graphsim}

The simulations of \glspl{synthetic lethal} performed in Section~\ref{chapt2:simulation_2015} included correlated blocks of genes as a rudimentary representation of \glslink{graph}{pathway} structure and co-regulated genes. The simulation procedure has been enhanced to account for more complex \glslink{graph}{pathway} structures by sampling from multivariate normal distributions with correlation structure derived from \gls{graph} structures (as described in Section~\ref{methods:graphsim}). This approach enabled the simulation of \gls{synthetic lethal} pathways with known correlation structure and synthetic lethal partners (of a gene not in the pathway).  Using this procedure, the performance of \gls{SLIPT} was evaluated under simple controlled correlation structures and complex correlations, such as those derived from biological networks (e.g., those described in Chapter~\ref{chap:Pathways}). The \gls{SLIPT} methodology waw tested in artificial constructed networks to evaluate the effect of \glslink{graph}{pathway} structure on \gls{synthetic lethal} detection These included large biologically feasible pathways to ensure that the \gls{SLIPT} methodology is robust under complex correlation structures and applicable to such complex \glspl{genomic} data.

These simulations combine the approach of prior simulation analyses (in Sections~\ref{chapt2:simulation_2015} and~\ref{chapt5:compare_methods}) with the \gls{graph} structures for biological pathways (as used in Chapter~\ref{chap:Pathways}). This enabled testing whether subtle or large differences in \glslink{graph}{pathway} structure affect \gls{synthetic lethal} detection, whether inhibiting relationships (or inverse correlations) between genes affect \gls{synthetic lethal} detection, and whether \gls{synthetic lethal} detection varies by which gene is \gls{synthetic lethal} and which genes are closely linked within the \glslink{graph}{pathway} structure. In addition, large numbers of \gls{synthetic lethal} genes and biologically feasible numbers of genes (with many non-synthetic lethal genes) was tested to replicate the findings of Sections~\ref{chapt2:simulation_2015} and~\ref{chapt5:compare_methods} in correlated structures derived from pathway graphs, including examples of biological pathways from Reactome \citep{Reactome}.

Simple and more complex constructed \gls{graph} structures were used to demonstrate the impact of \glslink{graph}{pathway} structure of the performance of \gls{SLIPT} for \gls{synthetic lethal} detection in simulations. In addition, more complex constructed \gls{graph} structures were compared to the \gls{PI3K} and G$_{\alpha i}$ signalling pathways derived from Reactome which were used for simulation of \glslink{graph}{pathway} structures of biological complexity (as shown in Figures~\ref{fig:SL_Pathway_Pi3K} and~\ref{fig:SL_Pathway_GPCR}).

\FloatBarrier

\subsection{Performance over Graph Structures}
\label{chapt5:graphsim_performance}

\subsubsection{Simple Graph Structures}
\label{chapt5:graphsim_performance_simple}

\FloatBarrier

Simple pathway modules were used to test the effect of \glslink{graph}{pathway} structure on the performance of detecting \gls{synthetic lethal} partners within \gls{graph} structures. For an intial comparison, the \gls{graph} structures (shown by Figure~\ref{fig:simple_graph1}) were used where a gene has one upstream regulator and two downstream (Figure~\ref{fig:simple_graph2:first}) or a gene has two upstream regulators and one downstream gene (Figure~\ref{fig:simple_graph2:first}). \gls{SLIPT} has a high performance in these simulations, detecting randomly selected \gls{synthetic lethal} partners in both of these small simple networks (in Figures~\ref{fig:simulation1205_randx_Graph1} and~\ref{fig:simulation1205_randx_Graph2}). 

\begin{figure*}[!htb]
%\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Diverging Module \label{fig:simple_graph1:first}}{%
            \fbox{
            \includegraphics[width=0.25\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1".pdf}}
            }
        }%
%        \subcaptionbox{Inhibiting Graph1}{%
%           \label{fig:simple_graph1:second}
%            \fbox{
%            \includegraphics[width=0.25\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1i".pdf}}
%           }
%        }%
%
%   \end{center}
%   \caption[Simulated \gls{graph} structures]{\small \textbf{Simulated \gls{graph} structures.} A constructed \gls{graph} structure used for the simulation procedure. Activating \glslink{edge}{links} are denoted by blue arrows and inhibiting \glslink{edge}{links} by red \glspl{edge}.}
%%}
%\label{fig:simple_graph1}
%%\end{mdframed}
%\end{figure*}
%\begin{figure*}[!htb]
%%\begin{mdframed}
%%  \resizebox{\textwidth}{!}{
%         \begin{center}
%
        \subcaptionbox{Converging Module \label{fig:simple_graph2:first}}{%
            \fbox{
            \includegraphics[width=0.25\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph2".pdf}}
            }
        }%
%        \subcaptionbox{Inhibiting Graph2}{%
%           \label{fig:simple_graph2:second}
%            \fbox{
%            \includegraphics[width=0.25\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph2i".pdf}}
%           }
%        }%
%
    \end{center}
   \caption[Simple \gls{graph} structures]{\small \textbf{Simple \gls{graph} structures.} A simple \gls{graph} structures used to demonstrate the simulation procedure. These are examples of a pathway diverging or converging respectively which enabled testing the importance of direction in \glslink{graph}{pathway} structures. These are used with both activating and inhibiting relationships as shown.}
%}
\label{fig:simple_graph1}
%\end{mdframed}
\end{figure*}

\begin{figure*}[!htbp]
%\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation \label{fig:simulation1205_randx_Graph1ROC:Perf}}{%
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic \label{fig:simulation1205_randx_Graph1ROC:ROC}}{%
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure \label{fig:simulation1205_randx_Graph1}}{%
           \raisebox{0.1875\textwidth}{
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1".pdf}}
           }
        }%
        \subcaptionbox{Statistical performance \label{fig:simulation1205_randx_Graph1ROC:AUC}}{%
           \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1_AUC_samplesx".png}}
        }%
    \end{center}
   \caption[Performance of simulations on a simple graph]{\small \textbf{Performance of simulations on a simple graph.} Simulation of \glspl{synthetic lethal} was performed sampling from a multivariate normal distribution generated from a diverging \gls{graph} structure. Performance of \gls{SLIPT} declines for more synthetic partners but this is mitigated by increased sample sizes (in darker colours). This manifests as a decline in specificity and the false positive rate. For each parameter value, 10,000 simulations were used. Colours of the \gls{ROC} curves in Figure~\ref{fig:simulation1205_randx_Graph1ROC:ROC} correspond to the parameters in Figure~\ref{fig:simulation1205_randx_Graph1ROC:AUC}.}
%}
\label{fig:simulation1205_randx_Graph1}
%\end{mdframed}
\end{figure*}

As previously observed (in Section~\ref{chapt2:simulation_2015}), performance declined with higher numbers of \gls{synthetic lethal} genes and lower sample sizes. However, the sensitivity of \gls{SLIPT} is high with conventional p-value thresholds (adjusted by \{gls{FDR}). \Gls{synthetic lethal} partners are often distinguishable for non \gls{synthetic lethal} genes, even in simple highly correlated networks. The small number of genes and their high correlation has an impact on the \gls{ROC} curves for higher numbers of \gls{synthetic lethal} partners which are skewed compared to those observed previously. Specificity cannot be tested if all potential partner genes are \gls{synthetic lethal} which limits the number of \gls{synthetic lethal} genes that can be tested. 

These results were consistent between the pathway modules of diverging (in Figure~\ref{fig:simulation1205_randx_Graph1cf2:Graph1}) and converging signals (in Figure~\ref{fig:simulation1205_randx_Graph1cf2:Graph2}). The \gls{AUROC} performance and underlying curves were strikingly similar between these \gls{graph} structures (as shown in Figures~\ref{fig:simulation1205_randx_Graph1} and~\ref{fig:simulation1205_randx_Graph2}). Thus the performance of \gls{SLIPT} was not perturbed by \glslink{graph}{pathway} structure, specifically the direction of pathway relationships since these \gls{graph} structures also demonstrate pathways in opposite direction. In a direct comparison (shown in Figure~\ref{fig:simulation1205_randx_Graph1cf2:Compare}), the performance of simulations did not differ across parameter values in these simple \glspl{graph} and therefore \gls{SLIPT} is robust to pathway direction.


\begin{figure*}[!htb]
%\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
       \subcaptionbox{Diverging Module \label{fig:simulation1205_randx_Graph1cf2:Graph1}}{%
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1".pdf}}
        }%
       \subcaptionbox{Converging Module \label{fig:simulation1205_randx_Graph1cf2:Graph2}}{%
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph2".pdf}}
        }%

%
        \subcaptionbox{Performance Between Graph Structures \label{fig:simulation1205_randx_Graph1cf2:Compare}}{%
            \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph1_ROC_Compare_Graph2(1)".pdf}}
        }%
        %\subcaptionbox{Corresponding $\chi^2$ values}{%
        %    \label{fig:simulation_May4SLreps:second}
        %    \includegraphics[width=0.35\textwidth]{{"SL_Model_May15mvnorm_heatmap_10XSL_cor_comp2(2)".png}}
        %}%

    \end{center}
   \caption[Performance of simulations is similar in simple graphs]{\small \textbf{Performance of simulations is similar in simple graphs.} The \gls{AUROC} values for simulations of multivariate normal distributions based on each \gls{graph} structure yielded indistinguishable performance across parameter values in 10,000 simulations.}
%}
\label{fig:simulation1205_randx_Graph1cf2}
%\end{mdframed}
\end{figure*}

%\FloatBarrier

\subsubsection{Constructed Graph Structures}
\label{chapt5:graphsim_performance_constructed}

%\FloatBarrier

A more complex \gls{graph} structure was used to examine the performance of detecting \gls{synthetic lethal} partners with \gls{SLIPT} in simulated \glslink{gene expression}{expression} data with pathway correlation structures. For a simple chain of gene representing a pathway (shown in Figure~\ref{fig:simulation1205_randx_Graph4}), the above findings were generally replicated. Performance was high across parameter values in small networks, with similar decreases in higher numbers of \gls{synthetic lethal} genes to detect and lower sample size.

\begin{figure*}[!htbp]
%\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation \label{fig:simulation1205_randx_Graph4ROC:Perf}}{%
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic \label{fig:simulation1205_randx_Graph4ROC:ROC}}{%
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure \label{fig:simulation1205_randx_Graph4}}{%
           \raisebox{0.1875\textwidth}{
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4".pdf}}
           }
        }%
        \subcaptionbox{Statistical performance \label{fig:simulation1205_randx_Graph4ROC:AUC}}{%
           \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4_AUC_samplesx".png}}
        }%
    \end{center}
   \caption[Performance of simulations on a pathway]{\small \textbf{Performance of simulations on a pathway.} Simulation of \glspl{synthetic lethal} was performed sampling from a multivariate normal distribution generated from a \glslink{graph}{pathway} structure. Performance of \gls{SLIPT} declines for more synthetic partners and lower sample sizes (in darker colours). For each parameter value, 10,000 simulations were used. Colours of the \gls{ROC} curves in Figure~\ref{fig:simulation1205_randx_Graph4ROC:ROC} correspond to the parameters in Figure~\ref{fig:simulation1205_randx_Graph4ROC:AUC}.}
%}
\label{fig:simulation1205_randx_Graph4}
%\end{mdframed}
\end{figure*}

When detecting \gls{synthetic lethal} genes with \gls{SLIPT} using adjusted (\{gls{FDR}) p-value thresholds, the performance differences were primarily due to changes in specificity, as the small numbers of \gls{synthetic lethal} genes still had highly significant p-values. Despite lower specificity and performance in \gls{ROC} curves, the accuracy increases and false positive rate decreases desirably with higher numbers of \gls{synthetic lethal} genes due to the high sensitivity and proportion of \gls{synthetic lethal} genes detected. Therefore the thresholds imposed by adjusted p-values appear to be appropriate for detecting \gls{synthetic lethal} partners, even in strongly correlated pathways, at least in these small-scale test cases.

An artifact of these small test cases led to the skewed \gls{ROC} curves (as discussed in Section~\ref{chapt5:graphsim_performance_simple}), which may be the result of the low number of non-synthetic lethal genes to identify as true negatives, affecting the accuracy of specificity. This issue does not occur in larger, more complex \gls{graph} structures, even with modest total numbers of genes and high correlations (as shown in Section~\ref{chapt5:complex_graphs}). This issue is unlikely to occur in large \glslink{gene expression}{expression} datasets with many non \gls{synthetic lethal} genes, as shown previously (in Section~\ref{chapt2:simulation_2015} and~\ref{chapt5:graphsim_performance_simple}) with \gls{graph} structures in larger datasets (in Section~\ref{chapt5:graphsim_performance_20K}). 

\FloatBarrier


\subsection{Performance with Inhibitions}
\label{chapt5:graphsim_performance_inhib}

\FloatBarrier

Simulations of \glspl{synthetic lethal} in \glslink{gene expression}{expression} data were also performed with correlation structures derived from \glspl{graph} containing inhibiting relationships (as are commonplace in biological pathways) which produce negative correlations. As shown in Figure~\ref{fig:simulation1205_randx_Graph1i}, these are not an issue for detection by \gls{SLIPT}. Rather, the \gls{SLIPT} procedure performs well on simple graph modules with highly negative correlations. With \gls{synthetic lethal} detection based on p-value (adjusted by \{gls{FDR}), there was higher specificity, higher accuracy, and lower false positive rate in an inhibitory graph than the same graph with activating relationships (as shown by Figure~\ref{fig:simulation1205_randx_Graph1}).

\begin{figure*}[!htbp]
%\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation \label{fig:simulation1205_randx_Graph1iROC:Perf}}{%
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1i_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic \label{fig:simulation1205_randx_Graph1iROC:ROC}}{%
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1i_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure \label{fig:simulation1205_randx_Graph1i}}{%
           \raisebox{0.1875\textwidth}{
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1i".pdf}}
           }
        }%
        \subcaptionbox{Statistical performance \label{fig:simulation1205_randx_Graph1iROC:AUC}}{%
           \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1i_AUC_samplesx".png}}
        }%
    \end{center}
   \caption[Performance of simulations on a simple graph with inhibition]{\small \textbf{Performance of simulations on a simple graph with inhibition.} Simulation of \glspl{synthetic lethal} was performed sampling from a multivariate normal distribution generated from an inhibiting graph. Performance of \gls{SLIPT} declines for more synthetic partners and lower sample sizes. For each parameter value, 10,000 simulations were used. Colours of the \gls{ROC} curves in Figure~\ref{fig:simulation1205_randx_Graph1iROC:ROC} correspond to the parameters in Figure~\ref{fig:simulation1205_randx_Graph1iROC:AUC}.}
%}
\label{fig:simulation1205_randx_Graph1i}
%\end{mdframed}
\end{figure*}


The \gls{ROC} curves for an inhibiting graph also show consistently high specificity irrespective of detection threshold with only the upper extreme of the curve exhibiting a skew below random performance (in Figure~\ref{fig:simulation1205_randx_Graph1i}). Nevertheless, the \gls{AUROC} values show a high performance across parameter values, particularly avoiding issues with higher numbers of \gls{synthetic lethal} partners (as observed in Section~\ref{chapt5:graphsim_performance_simple}). However,  performance was marginally lower for higher numbers of \gls{synthetic lethal} genes to detect and lower sample sizes, consistent with previously observations.

Negatively correlated simulated datasets are also unperturbed by minor differences in \gls{graph} structure, such as changing in the direction of the graph module. As observed for activating relationships in these graph modules, the performance was highly concordant between the graph modules (shown by similar results in Figures~\ref{fig:simulation1205_randx_Graph1i} and~\ref{fig:simulation1205_randx_Graph2i}).

Detection of \glspl{synthetic lethal} by \gls{SLIPT} in simulated data with inhibiting relationships outperforms simulations with activating relationships in the same \gls{graph} structure (as shown in Figure~\ref{fig:simulation1205_randx_Graph1cf1i}). Thus \gls{SLIPT} is robust in \gls{gene expression} datasets with inverse correlations and performs well in them, at least in simple test cases. This is important because such relationships occur frequently in biological pathways and therefore the findings inferred from \gls{graph} structures without inhibiting relationships are a conservative estimate.

The \gls{SLIPT} methodology likely performs better in biological pathways (which contain negative correlations) than the \gls{graph} structures discussed previously (in Section~\ref{chapt5:graphsim_performance}). This is likely since negative correlations lead to \gls{synthetic lethal} partners and inversely correlated genes which are positively correlated with the query gene. As previously shown, the \gls{SLIPT} methodology performs well with specificity against positively correlated query genes (in Sections~\ref{chapt2:simulation_mvtnorm_query_cor} and~\ref{chapt5:compare_correlation}).% and negative correlation with the query gene is a better predictor of \glspl{synthetic lethal} than positive correlation (Sections~\ref{chapt5:compare_chisq_query_cor} and~\ref{chapt5:compare_chisq}), although it still performs worse than \gls{SLIPT}.

\begin{figure*}[!t]
%\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
       \subcaptionbox{Activating Graph}{%
           \label{fig:simulation1205_randx_Graph1cf1i:Graph1}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1".pdf}}
        }%
       \subcaptionbox{Inhibiting Graph}{%
           \label{fig:simulation1205_randx_Graph1cf1i:Graph1i}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1i".pdf}}
        }%

%
        \subcaptionbox{Performance Between Graph Structures}{%
            \label{fig:simulation1205_randx_Graph1cf1i:Compare}
            \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph1_ROC_Compare(1)".pdf}}
        }%
        %\subcaptionbox{Corresponding $\chi^2$ values}{%
        %    \label{fig:simulation_May4SLreps:second}
        %    \includegraphics[width=0.35\textwidth]{{"SL_Model_May15mvnorm_heatmap_10XSL_cor_comp2(2)".png}}
        %}%

    \end{center}
   \caption[Performance is higher on a simple inhibiting graph]{\small \textbf{Performance is higher on a simple inhibiting graph.} The \gls{AUROC} values for simulations of multivariate normal distributions based on inhibitions in the Graph structure yielded consistently higher performance across parameter values in 10,000 simulations.}
%}
\label{fig:simulation1205_randx_Graph1cf1i}
%\end{mdframed}
\end{figure*}

Similarly, more complex \gls{graph} structures with entirely inhibiting relationships (negative correlations) also perform desirably on p-value thresholds (adjusted by \{gls{FDR}) and have high performance across increasing numbers of \gls{synthetic lethal} genes, particularly for sufficiently high sample sizes (as shown by Figure~\ref{fig:simulation1205_randx_Graph4i}). However, this is not necessarily the case for \gls{graph} structures with a combination of activating and inhibiting relationships (i.e., containing positive and negative correlations) As shown by Figure~\ref{fig:simulation1205_randx_Graph4i2}, such a mixed \glslink{graph}{network} structure does not necessarily have high performance across parameters as observed for purely inhibiting networks.

\begin{figure*}[!htbp]
%\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1205_randx_Graph4i2ROC:Perf}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4i2_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1205_randx_Graph4i2ROC:ROC}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4i2_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph4i2}
           \raisebox{0.1875\textwidth}{
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4i2".pdf}}
           }
        }%
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1205_randx_Graph4i2ROC:AUC}
           \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4i2_AUC_samplesx".png}}
        }%
    \end{center}
   \caption[Performance of simulations on a constructed graph with inhibition]{\small \textbf{Performance of simulations on a constructed graph with inhibition.} Simulation of \glspl{synthetic lethal} was performed sampling from a multivariate normal distribution generated from  \glslink{graph}{pathway} structure with a combination of inhibitions. Performance of \gls{SLIPT} declines for more synthetic partners and lower sample sizes. For each parameter value, 10,000 simulations were used.}
%}
\label{fig:simulation1205_randx_Graph4i2}
%\end{mdframed}
\end{figure*}

These still appear to have desirably high sensitivity, high accuracy, and low false positive rate for detecting more \gls{synthetic lethal} genes, despite poor specificity. The \gls{ROC} curves are particularly skewed for high proportions of the network being \gls{synthetic lethal} and may stem from low numbers of true negative genes to detect (as discussed in Section~\ref{chapt5:graphsim_performance_simple}). In a direct comparison of performance (shown in Figure~\ref{fig:simulation1205_randx_Graph4cf4i}), the purely inhibiting graph had consistently higher performance than the activating one as observed for simpler \glspl{graph} (in Figure~\ref{fig:simulation1205_randx_Graph1cf1i}).

In contrast, the combination of activating and inhibiting relationships had slightly lower performance across parameters compared to the same \gls{graph} structure with activating relationships. Therefore correlation structure can impact on the performance of \gls{SLIPT} in a graph network, in either direction, specifically the addition of negative correlations. However,  this may be an artifact of the simulation procedure as \gls{synthetic lethal} genes from the correlation structure were randomly selected (without regard to their relationships), with the query gene added to ensure that conditions for \gls{synthetic lethal} relationships were met. 

\begin{figure*}[!t]
%\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
       \subcaptionbox{Activating Graph}{%
           \label{fig:simulation1205_randx_Graph4cf4i:Graph4}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4".pdf}}
        }%
       \subcaptionbox{Inhibiting Graph}{%
           \label{fig:simulation1205_randx_Graph4cf4i:Graph4i}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4i".pdf}}
        }%
       \subcaptionbox{Mixed Graph}{%
           \label{fig:simulation1205_randx_Graph4cf4i:Graph4i2}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4i2".pdf}}
        }%

%
        \subcaptionbox{Performance Between Graphs (a) and (b)}{%
            \label{fig:simulation1205_randx_Graph4cf4i:Compare4i}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph4_ROC_Compare(1)".pdf}}
        }%
        \subcaptionbox{Performance Between Graphs (a) and (c)}{%
            \label{fig:simulation1205_randx_Graph4cf4i:Compare4i2}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph4_ROC_Compare(4)".pdf}}
        }%
        %\subcaptionbox{Corresponding $\chi^2$ values}{%
        %    \label{fig:simulation_May4SLreps:second}
        %    \includegraphics[width=0.35\textwidth]{{"SL_Model_May15mvnorm_heatmap_10XSL_cor_comp2(2)".png}}
        %}%

    \end{center}
   \caption[Performance is affected by inhibition in graphs]{\small \textbf{Performance is affected by inhibition in graphs.} The \gls{AUROC} values for simulations of multivariate normal distributions based on \gls{graph} structure containing only inhibitions in the Graph structure yielded consistently higher performance across parameter values in 10,000 simulations. A combination of activating and inhibiting relationships had lower performance but was more similar to the activating graph.}
%}
\label{fig:simulation1205_randx_Graph4cf4i}
%\end{mdframed}
\end{figure*}

This system for simulating inhibitory pathways is not ideal since it lead to \gls{synthetic lethal} gene combinations, by randomly selecting them, which are unlikely to occur in biological pathways. These randomly selected \gls{synthetic lethal} genes may account for the detection results being suboptimal (i.e., difficult to detect \gls{synthetic lethal} partners) compared to previous investigations. It is expected that inversely correlated synthetic partner genes will be highly expressed in a mutually exclusive manner such that at least one of them will be compensating for loss of the query gene in most samples, leading to a weak \gls{synthetic lethal} signature in \glslink{gene expression}{expression} data in this case. Furthermore, this case may not be representative of empirical biological data with \gls{synthetic lethal} partners of \gls{tumour suppressor} genes which are commonly inversely correlated to the query gene (to some extent) and therefore it is unlikely that they are strongly negative correlated with each other, unless they are \gls{synthetic lethal} partners of each other as well. It is plausible that many \gls{synthetic lethal} partner genes will serve to separately compensate for the loss of query gene function and be positively correlated with each other. Nonetheless, these simulations are sufficient to demonstrate that correlation structure (particularly negative correlations) have an impact on the detection of \glspl{synthetic lethal}. However, \gls{SLIPT} is still able to perform well across \glspl{graph} with different activating and inhibiting relationships and the perturbations in performance are marginal, particularly those reducing performance compared to an activating network.  

\FloatBarrier

\subsection{Synthetic Lethality across Graph Structures}
\label{chapt5:graphsim_str}

\FloatBarrier

While \gls{synthetic lethal} genes are distinguishable in principle from those highly positively correlated with them (as shown by \gls{ROC} analysis), they are not necessarily distinguished as reflected by low specificity and high false positive rates in poorly performing simulations throughout this section. The negative correlations are not subject to the same issue, they sometimes perturb the correlation structure between \gls{synthetic lethal} partner genes making it difficult to detect many of them. Thus far, \gls{synthetic lethal} genes have been selected randomly which is a limited approach. To examine the impact of pathway relationships in more more detail, specific genes will be selected to be \gls{synthetic lethal} within a network. Replicate simulations were performed for \gls{synthetic lethal} detection with a fixed \gls{synthetic lethal} gene, in contrast to previous investigations (randomly selecting \gls{synthetic lethal} genes). This investigation was performed to demonstrate the impact of these genes being \gls{synthetic lethal} in the detection of neighbouring genes in the pathway network, under \gls{graph} structure activating and inhibiting relationships.

For instance, detection of a \gls{synthetic lethal} gene in an activating \gls{graph} structure (as shown in Figure~\ref{fig:simulation1206_Str_randx_Graph4D:Graph4}) is straightforward: the $\chi^2$ values across simulations are clearly distinguishable from non \gls{synthetic lethal} genes (shown in Figure~\ref{fig:simulation1206_Str_randx_Graph4D:vioplot}). A small number of simulations were performed for each gene being designated as \gls{synthetic lethal}. In each case (of each gene being the \gls{synthetic lethal} partner), the \gls{synthetic lethal} gene was detectable with highest $\chi^2$ value, being distinguishable amongst 20,000 genes including the highly correlated graph network (as shown in Figure~\ref{fig:simulation1206_Str_randx_Graph4}).

\begin{figure*}[!htb]
%\begin{mdframed}
     \begin{center}
       \subcaptionbox{Activating Graph \label{fig:simulation1206_Str_randx_Graph4D:Graph4}}{%
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4D".pdf}}
        }%
        \subcaptionbox{Inhibiting Graph \label{fig:simulation1206_Str_randx_Graph4iD:Graph4i}}{%
           \label{fig:simulation1206_Str_randx_Graph4iD:Graph4i}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4iD".pdf}}
        }%
        
       \subcaptionbox{$\chi^2$ distribution for Activating Graph  \label{fig:simulation1206_Str_randx_Graph4D:vioplot}}{%
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(4)".pdf}}
        }%
       \subcaptionbox{$\chi^2$ distribution for Inhibiting Graph \label{fig:simulation1206_Str_randx_Graph4iD:vioplot}}{%
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i_ROC_samples_SLstry_vioplot(4)".pdf}}
        }%
       \end{center}
      \caption[Detection of Synthetic Lethality within a Graph Structure with Inhibitions]{\small \textbf{Detection of Synthetic Lethality within Graph Structure with Inhibitions.} The gene ``D'' was designated to be \gls{synthetic lethal} and the $\chi^2$ value from \gls{SLIPT} was computed for each gene across each \gls{graph} structure. The $\chi^2$ values were computed in 100 simulations of datasets of 20,000 genes including the \gls{graph} structure and 1000 samples. Adjacent genes exhibited lower $\chi^2$ values with inhibiting relationships.} 
%\end{mdframed}
\addtocounter{figure}{-1}
\phantomcaption
\label{fig:simulation1206_Str_randx_Graph4D2}
\end{figure*}

\iffalse
\begin{figure*}[!htb]
%\begin{mdframed}
     \begin{center}
      \resizebox{0.8 \textwidth}{!}{
         \includegraphics{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(4)".pdf}}
      \end{center}
      \caption[Performance is affected by inhibition in graphs]{\small \textbf{Performance is affected by inhibition in graphs.} The gene category (blue for query, cyan for query-correlated, red for SL, orange for SL-correlated, forest green for non-SL-correlated, and green for non-SL) ordered by $\chi^2$ signed by the \gls{SLIPT} directional condition is shown across simulations. For each of 1--10 SL partners, 10 simulations demonstrate that the increasing numbers of SL partners become harder detect. The $\chi^2$ values show a clear threshold for SL and correlated genes when there are fewer of them, distinguishable from correlated genes in this case.}
%\end{mdframed}
%\label{fig:simulation1206_Str_randx_Graph4D}
\end{figure*}
\fi

\iffalse
\begin{figure*}[!htb]
%\begin{mdframed}
     \begin{center}
       \subcaptionbox{Graph Structure \label{fig:simulation1206_Str_randx_Graph4D:Graph4}}{%
           \raisebox{0.15\textwidth}{
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4D".pdf}}
           }
        }%
       \subcaptionbox{$\chi^2$ distribution for each gene \label{fig:simulation1206_Str_randx_Graph4D:vioplot}}{%
           \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(4)".pdf}}
        }%
       \end{center}
      \caption[Detection of Synthetic Lethality within a Graph Structure]{\small \textbf{Detection of Synthetic Lethality within a Graph Structure.} The gene ``D'' was designated to be \gls{synthetic lethal} and the $\chi^2$ value from \gls{SLIPT} was computed for each gene across the \gls{graph} structure. The $\chi^2$ values were computed in 100 simulations of datasets of 20,000 genes including the \gls{graph} structure and 1000 samples.}
%\end{mdframed}
\addtocounter{figure}{-1}
\phantomcaption
\label{fig:simulation1206_Str_randx_Graph4D}
\end{figure*}
\fi

This is consistent with previous observations that \gls{SLIPT} performed optimally for a single \gls{synthetic lethal} partner in this network (in Figure~\ref{fig:simulation1205_randx_Graph4}). Despite optimal performance in a \gls{ROC} curve irrespective of detection threshold, many of the highly correlated genes would be detected as false positives using a conventional p-value threshold (even if adjusted by \{gls{FDR}) from a $\chi^2$ test with 4 degrees of freedom as performed by \gls{SLIPT} (as described in Section~\ref{methods:SLIPT}). In particular, the genes that are adjacent to the \gls{synthetic lethal} gene ``D'' within the \gls{graph} structure exhibited high test statistics across simulations which would often be reported as false positives (as shown in Figure~\ref{fig:simulation1206_Str_randx_Graph4D:vioplot}). This is not specific to example of gene ``D'', with the neighbouring genes exhibiting higher $\chi^2$ test statistics for each gene in the network when it is designated as the \gls{synthetic lethal} partner (as shown in Figure~\ref{fig:simulation1206_Str_randx_Graph4}).

Thus the \gls{synthetic lethal} signal propagates from the true \gls{synthetic lethal} gene throughout the network such genes nearer to the true \gls{synthetic lethal} gene (more highly correlated) have higher test statistics and are more likely to be detected by \gls{SLIPT} as false positives. This tendency for adjacent genes to be detected as \gls{synthetic lethal} false positives is consistent with the \gls{synthetic lethal} pathways being more concordant between \gls{SLIPT} in \gls{TCGA} data \citep{TCGA2012} and the \gls{siRNA} screen \citep{Telford2015} than individual gene results (in Chapter~\ref{chap:SLIPT}). False positive genes are therefore still more likely to be involved in a \gls{synthetic lethal} pathway by being correlated with a true \gls{synthetic lethal} gene and \gls{synthetic lethal} pathways are likely to have many genes detected by \gls{SLIPT} giving a consensus of evidence, supporting the pathway over-representation approach in particular which may account for how it differs from pathway \glspl{metagene}. Furthermore, \gls{SLIPT} is still viable to detect true \gls{synthetic lethal} partners or prioritise those most likely to be experimentally validated since those with the strongest support (i.e, higher $\chi^2$ values and more significant p-values) are more likely to be the underlying \gls{synthetic lethal} gene.

In contrast to an activating graph (Figure~\ref{fig:simulation1206_Str_randx_Graph4D:Graph4}), the immediately adjacent genes  in an inhibiting graph (Figure~\ref{fig:simulation1206_Str_randx_Graph4iD:Graph4i}) had neither an elevated $\chi^2$ test statistics indicating \glspl{synthetic lethal} nor a significant inverse effect (as shown in Figure~\ref{fig:simulation1206_Str_randx_Graph4iD:vioplot}). Similar simulations were performed a \gls{graph} structure with inhibiting relationships within a dataset of 20,000 genes. The adjacent genes to the \gls{synthetic lethal} gene ``D'' did not have elevated $\chi^2$ values and therefore true \gls{synthetic lethal} partners were highly distinguishable from non \gls{synthetic lethal} genes with inhibiting relationships. This was not specific to ``D'' and was shown across any gene in the \gls{graph} structure if it were designated to be the \gls{synthetic lethal} partner of the query gene (shown in Figure~\ref{fig:simulation1206_Str_randx_Graph4i}). This is consistent with the detection of many genes involved in kinase signalling, gene regulation, and other known cancer pathways (in Chapter~\ref{chap:SLIPT}) which frequently have inhibitory steps. These results support \gls{SLIPT} as an appropriate approach to distinguish \gls{synthetic lethal} partners in biological pathways, including those relevant to cancer growth and inhibition.


\iffalse
\begin{figure*}[!ht]
%\begin{mdframed}
     \begin{center}
       \subcaptionbox{Graph Structure}{%
           \raisebox{0.15\textwidth}{
           \label{fig:simulation1206_Str_randx_Graph4iD:Graph4i}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4iD".pdf}}
           }
        }%
       \subcaptionbox{$\chi^2$ distribution for each gene}{%
           \label{fig:simulation1206_Str_randx_Graph4iD:vioplot}
           \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i_ROC_samples_SLstry_vioplot(4)".pdf}}
        }%
       \end{center}
      \caption[Detection of Synthetic Lethality within an Inhibiting Graph Structure]{\small \textbf{Detection of Synthetic Lethality within an Inhibiting  Graph Structure.} The gene ``D'' was designated to be \gls{synthetic lethal} and the $\chi^2$ value from \gls{SLIPT} was computed for each gene across the \gls{graph} structure. The $\chi^2$ values were computed in 100 simulations of datasets of 20,000 genes including the \gls{graph} structure and 1000 samples.}
%\end{mdframed}
%\label{fig:simulation1206_Str_randx_Graph4iD}
\end{figure*}
\fi

%%move fig fig:simulation1206_Str_randx_Graph4D2 (to replace fig:simulation1206_Str_randx_Graph4D)

However, it should be noted that the 2\textsuperscript{nd} degree neighbours of the \gls{synthetic lethal} gene still exhibited moderate $\chi^2$ values (and are moderately correlated with the \gls{synthetic lethal} gene). It is still possible for these to be detected as false positives as previously described for an activating \gls{graph} structure although the presence of inhibitory relationships (and negative correlations) further increases the differences in test statistics for correlated genes and underlying \gls{synthetic lethal} partners as shown by the extreme example (in Figure~\ref{fig:simulation1206_Str_randx_Graph4i}).

These findings are consistent with simulations in a graph containing a combination of activating and inhibiting relationships which exhibits a either of these $\chi^2$ profiles depending on which gene is \gls{synthetic lethal} and the relationships to adjacent genes (as shown in Figure~\ref{fig:simulation1206_Str_randx_Graph4i2}). Note that in this case, the \gls{synthetic lethal} gene is distinguishable and inhibitory relationships within this \gls{graph} structure make it easier to detect underlying \gls{synthetic lethal} genes with \gls{SLIPT} by a more highly significant $\chi^2$ test. This contrasts with randomly selecting multiple \gls{synthetic lethal} genes (in Figure~\ref{fig:simulation1205_randx_Graph4cf4i}) where the performance of \gls{SLIPT} was impeded by the inhibitory relationships between \gls{synthetic lethal} partners in this \gls{graph} structure. Therefore the random \gls{synthetic lethal} genes selected previously with negative correlations between them which had poor performance are likely to have created an artifact in the simulation results as they are biologically implausible and constrain the \gls{synthetic lethal} simulation procedure

The results with one \gls{synthetic lethal} partner were sufficient to infer the impact of \gls{synthetic lethal} partners within pathways on neighbouring (correlated) genes. However,  it is plausible that the \gls{synthetic lethal} signatures in \glslink{gene expression}{expression} data would propagate through a network with multiple \gls{synthetic lethal} partners as sources, provided that the correlations between \gls{synthetic lethal} partners is biological feasible. These simulations were performed on a correlated \gls{graph} structure within a larger \gls{gene expression} dataset of 20,000 genes (as performed in Sections~\ref{chapt2:simulation_2015} and~\ref{chapt5:graphsim_performance_20K}), a feasible number for a full human \gls{gene expression} dataset, and as such are comparable to the findings below.

\FloatBarrier

\subsection{Performance within a Simulated Human Genome}
\label{chapt5:graphsim_performance_20K}

\FloatBarrier


As noted in Section~\ref{chapt5:graphsim_performance_simple}, the high proportion of \gls{synthetic lethal} partners in small networks made accurately assessing the performance of \gls{SLIPT} with higher numbers of true partners to detect (and fewer true negatives) difficult. Similarly, adding more true negative genes in previous simulations increased the performance of \gls{SLIPT}, particularly the specificity to reduce the number of false positives (as shown in Sections~\ref{chapt2:simulation_2015} and~\ref{chapt5:compare_methods}). Building on these findings, here the \gls{graph} structures (as used in Section~\ref{chapt5:graphsim_performance}) of genes with correlations from sampling a multivariate normal distribution were included in a larger simulated dataset of 20,000 genes. This simulation procedure serves to test the performance of \gls{SLIPT} at detecting \gls{synthetic lethal} partners within correlated \gls{graph} structures (of a \gls{synthetic lethal} pathway) in the context of biologically feasible numbers of genes. 

%\subsubsection{Simple Graph Structures in a \Glspl{genome}}
%\label{chapt5:graphsim_performance_20K_simple}

%\FloatBarrier

The simulations performed in Section~\ref{chapt5:graphsim_performance_simple} were replicated within a dataset of 20,000 genes with the rest being composed on non \gls{synthetic lethal} genes without correlation structure. The aforementioned issue with specificity in a higher number of underlying \gls{synthetic lethal} genes did not occur in a simple \gls{graph} structure (as shown in Figure~\ref{fig:simulation1207_randx_Graph1}). For such a small graph module of highly correlated genes within a \gls{gene expression} dataset, detection of \gls{synthetic lethal} genes within the network by \gls{SLIPT} and distinguishing these from the larger dataset performed well across parameter values. In this case, a reduction in sensitivity was the cause of poorer performance as a higher number of non \gls{synthetic lethal} genes were detected as true negative with a low false positive rate and high accuracy. This further supports the use of stringent $\chi^2$ p-value thresholds (adjusted by \{gls{FDR}) for testing for \glspl{synthetic lethal} in \gls{gene expression} data across the number of genes in human and cancer data.

\begin{figure*}[!htbp]
%\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1207_randx_Graph1ROC:Perf}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph1_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1207_randx_Graph1ROC:ROC}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph1_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1207_randx_Graph1}
           \raisebox{0.1875\textwidth}{
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1".pdf}}
           }
        }%
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1207_randx_Graph1ROC:AUC}
           \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph1_AUC_samplesx".png}}
        }%
    \end{center}
   \caption[Performance of simulations including a simple graph]{\small \textbf{Performance of simulations including a simple graph.} Simulation of \glspl{synthetic lethal} was performed sampling from a multivariate normal distribution (without correlation structure apart from the graph shown). Performance of \gls{SLIPT} was high across parameters for detecting \glspl{synthetic lethal} in the \gls{graph} structure within a larger dataset. The sensitivity decreases for a greater number of true positives to detect but the specificity remains high with a low false positive rate.}
%}
\label{fig:simulation1207_randx_Graph1}
%\end{mdframed}
\end{figure*}

In a direct comparison with simulations in the \gls{graph} structure alone (as performed in Section~\ref{chapt5:graphsim_performance_simple}), detection of \glspl{synthetic lethal} with \gls{SLIPT} performs consistently better in a larger dataset with many true negative genes to detect (as shown in Figure~\ref{fig:simulation1205_randx_Graph1cf20K}). This is a desirable property of the \gls{SLIPT} methodology as it has a high specificity and low false positive rate. \gls{SLIPT} is therefore applicable to large \gls{gene expression} datasets where these are important considerations since the number of negative genes to correctly identify often vastly outnumbers the number of positive genes to detect.

%This finding was replicated across simple graph modules with similar results between \glspl{graph} with different directions (as shown in Figure~\ref{fig:simulation1205_randx_Graph1cf20K}). Higher performance of \gls{synthetic lethal} detection in an activating \gls{graph} structure within the context of a larger \gls{gene expression} dataset by further replicated across the \gls{graph} structures presented earlier (as shown in the Appendix by Figures~\ref{fig:simulation1207_randx_Graph2}\nobreakdash--\ref{fig:simulation1207_randx_Graph7}) and is not specific to the modules shown here.

\begin{figure*}[!tb]
%\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
       \subcaptionbox{Diverging Module}{%
           \label{fig:simulation1205_randx_Graph1cf20K:Graph1}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1".pdf}}
        }%
       \subcaptionbox{Converging Module}{%
           \label{fig:simulation1205_randx_Graph1cf20K:Graph2}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph2".pdf}}
        }%
       %\subcaptionbox{Graph Structure}{%
       %    \label{fig:simulation1205_randx_Graph1cf20K:Graph3i2}
       %    \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph3i2".pdf}}
       %}%

%
        \subcaptionbox{Performance in Diverging Module}{%
            \label{fig:simulation1205_randx_Graph1cf20K:Compare1}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph1_ROC_Compare(2)".pdf}}
        }%
        \subcaptionbox{Performance in Converging Module}{%
            \label{fig:simulation1205_randx_Graph1cf20K:Compare2}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph2_ROC_Compare(2)".pdf}}
        }%

    \end{center}
%}
   \caption[Performance on a simple graph improves with more genes]{\small \textbf{Performance on a simple graph improves with more genes.} Simulations were performed with each of the \gls{graph} structures to detect \gls{synthetic lethal} partners within them. In either structure, performance of detection in a dataset containing on the \gls{graph} structure (in colour) was lower than testing the \gls{graph} structure within a larger dataset of non \gls{synthetic lethal} genes (without correlations).}
\label{fig:simulation1205_randx_Graph1cf20K}
%\end{mdframed}
\end{figure*}

%\FloatBarrier
%\subsubsection{Constructed Graph Structures in a \Glspl{genome}}
%\label{chapt5:graphsim_performance_20K_constructed}
%\label{fig:simulation1207_randx_Graph3}
%\label{fig:simulation1207_randx_Graph3i}

This increase in performance with more negative genes to detect does not necessarily apply in an inhibiting \gls{graph} structure. While an increased performance for an activating graph was replicated in this case, the performance of simulations of an entirely inhibiting \gls{graph} structure did not improve within a larger dataset (as shown in Figure~\ref{fig:simulation1205_randx_Graph3cf20K}). There is cause for concern since the biological pathways commonly contain inhibiting relationships (and inverse correlations), however, they are rarely as frequent as modelled here. It is reassuring that the performance in the inhibiting \gls{graph} structure is comparable to simulations of the \gls{graph} structure in isolation rather than diminished.
\iffalse
\begin{figure*}[!tb]
%\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
       \subcaptionbox{Graph3 Structure}{%
           \label{fig:simulation1205_randx_Graph3cf20K:Graph3}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph3".pdf}}
        }%
       \subcaptionbox{Inhibiting Structure}{%
           \label{fig:simulation1205_randx_Graph3cf20K:Graph3i}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph3i".pdf}}
        }%
       %\subcaptionbox{Graph Structure}{%
       %    \label{fig:simulation1205_randx_Graph3cf20K:Graph3i2}
       %    \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph3i2".pdf}}
       %}%

%
        \subcaptionbox{Performance in Graph3}{%
            \label{fig:simulation1205_randx_Graph3cf20K:Compare3}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph3_ROC_Compare(2)".pdf}}
        }%
        \subcaptionbox{Performance with Inhibition}{%
            \label{fig:simulation1205_randx_Graph3cf20K:Compare3i}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph3_ROC_Compare(3)".pdf}}
        }%

    \end{center}
   \caption[Performance on an inhibiting graph with more genes]{\small \textbf{Performance on an inhibiting graph with more genes.} Simulations were performed in a \gls{graph} structure with activating and inhibiting relationships to detect \gls{synthetic lethal} partners within them. In contrast to an activating graph, performance of detection in a dataset containing on the \gls{graph} structure (in colour) was not lower than testing the \gls{graph} structure within a larger dataset of non \gls{synthetic lethal} genes (without correlations) in an inhibiting \gls{graph} structure with negative correlations.}
%}
\label{fig:simulation1205_randx_Graph3cf20K}
%\end{mdframed}
\end{figure*}
\fi
It is expected that the findings based on simulations of genes with \glslink{graph}{pathway} structures in smaller datasets (as described in Section~\ref{chapt5:graphsim_performance}) will be relevant to larger datasets since the simulation results in these perform comparably or higher with more non \gls{synthetic lethal} genes to distinguish from them even with inhibitory relationships within the \gls{graph} structure 
%Therefore the findings based on simulations of genes with \gls{graph} structures in smaller datasets (as described in Section~\ref{chapt5:graphsim_performance}) will be relevant to larger datasets since the simulation results in these perform comparably or higher with more non \gls{synthetic lethal} genes to distinguish from them even with inhibitory relationships within the \gls{graph} structure (as shown with supporting results Appendix Figures~\ref{fig:simulation1207_randx_Graph1i}\nobreakdash--\ref{fig:simulation1207_randx_Graph7i2}). Hence these findings will be relevant in the context of empirical \gls{gene expression} datasets with thousands of genes such as those in human and cancers.


%\label{fig:simulation1207_randx_Graph4}
%\label{fig:simulation1207_randx_Graph4i2}

Performance of \gls{synthetic lethal} detection of \gls{SLIPT} in \gls{graph} structures with inhibitions included in a larger dataset of non \gls{synthetic lethal} genes did not necessarily diminish to the level of the \gls{graph} structure simulated alone. In some cases (as shown in Figure~\ref{fig:simulation1205_randx_Graph4cf20K}), the performance of an inhibitory \gls{graph} structure was consistently elevated when included within a larger data. However, these did not perform as well as the equivalent activating \gls{graph} structures within a similar dataset.
\begin{figure*}[!tb]
%\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph4cf20K:Graph4}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4".pdf}}
        }%
       \subcaptionbox{Inhibiting Structure}{%
           \label{fig:simulation1205_randx_Graph4cf20K:Graph4i2}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4i2".pdf}}
        }%
       %\subcaptionbox{Graph Structure}{%
       %    \label{fig:simulation1205_randx_Graph4cf20K:Graph4i2}
       %    \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4i2".pdf}}
       %}%

%
        \subcaptionbox{Performance in Graph}{%
            \label{fig:simulation1205_randx_Graph4cf20K:Compare4}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph4_ROC_Compare(2)".pdf}}
        }%
        \subcaptionbox{Performance with Inhibitions}{%
            \label{fig:simulation1205_randx_Graph4cf20K:Compare4i2}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph4_ROC_Compare(5)".pdf}}
        }%

    \end{center}
   \caption[Performance on an inhibiting graph improves with more genes]{\small \textbf{Performance on an inhibiting graph improves with more genes.} Simulations were performed in a \gls{graph} structure with activating and inhibiting relationships to detect \gls{synthetic lethal} partners within them. In contrast to an activating graph, performance of detection in a dataset containing only the \gls{graph} structure (in colour) was as much lower than testing the \gls{graph} structure within a larger dataset of non \gls{synthetic lethal} genes (without correlations) in an inhibiting \gls{graph} structure with negative correlations.}
%}
\label{fig:simulation1205_randx_Graph4cf20K}
%\end{mdframed}
\end{figure*}

This poorer performance is unlikely to occur due to highly negatively correlated genes being false positives as they will be positively correlated with the query gene if they are negatively correlated with a \gls{synthetic lethal} partner (i.e., within a \gls{synthetic lethal} pathway). The \gls{SLIPT} procedure performs well at distinguishing these, as previously shown (in Sections~\ref{chapt2:simulation_mvtnorm_query_cor} and~\ref{chapt5:compare_chisq_query_cor}). These false positives will also be a minority amongst a larger dataset of non \gls{synthetic lethal} genes without correlation to the query or \gls{synthetic lethal} genes.

It more likely that the poorer performance stems from negative correlations between \gls{synthetic lethal} genes which makes them more difficult to individually detect (as observed in Section~\ref{chapt5:graphsim_performance_inhib}). As discussed in Section~\ref{chapt5:graphsim_str}, this is likely an artifact of the simulation procedure selecting random \gls{synthetic lethal} genes which may be biologically implausible (e.g., strong inhibitory relationships between them). Therefore the poorer performing inhibiting \glspl{graph} within larger datasets are not cause for concern as the cases where \gls{SLIPT} performs poorly are combinations of simulated \gls{synthetic lethal} genes which are unlikely to occur within biological pathways. Furthermore the simulation procedure has used included higher-order \gls{synthetic lethal} to produce the weakest signal of \glspl{synthetic lethal} for individual partner genes and these are still detectable by \gls{SLIPT}.

%This interpretation is consistent with the poorly performing simulations in inhibiting \glspl{graph} having a low sensitivity (in Figures~\ref{fig:simulation1207_randx_Graph1i}\nobreakdash--\ref{fig:simulation1207_randx_Graph7i2}). These simulations still show high specificity and accuracy with a low false positive rate for \gls{synthetic lethal} detection with p-value thresholds (adjusted by \{gls{FDR}) for \gls{SLIPT}. Such results support \gls{SLIPT} as a stringent methodology to detect \gls{synthetic lethal} interactions in large \gls{gene expression} data, excluding the majority of false positives with many of those remaining belonging to \gls{synthetic lethal} pathways.


\FloatBarrier

\section{Simulations in More Complex Graph Structures}
\label{chapt5:complex_graphs}
%%intro blurb

As shown in Figure~\ref{fig:simulation1205_randx_Graph5}, sensitivity declines over a greater range for the number of \gls{synthetic lethal} partners in a larger network with a tradeoff with specificity. However,  the accuracy declines for greater numbers of \gls{synthetic lethal} partners and the false positive rate peaks at intermediate values. In this range, difference between simulations varies with greater sample size. The \gls{AUROC} results were similar for other more complex \gls{graph} structures (as shown in Figures~\ref{fig:simulation1205_randx_Graph6} and~\ref{fig:simulation1205_randx_Graph7}). These \glspl{graph} performed similarly to each other, although they had differences from Figure~\ref{fig:simulation1205_randx_Graph5} in their sensitivity and specificity at an adjusted (\{gls{FDR}) p-value threshold. This difference may stem from different ratios of \gls{synthetic lethal} and non-synthetic lethal genes to detect, since the latter \glspl{graph} (in Figures~\ref{fig:simulation1205_randx_Graph6} and~\ref{fig:simulation1205_randx_Graph7}) had half the total genes to that shown in Figure~\ref{fig:simulation1205_randx_Graph5}.

However, the \gls{graph} structures (of similar size) were highly distinct and yet had similar performance profiles across parameters. Therefore \gls{SLIPT} is robust across \glslink{graph}{pathway} structures and is more affected by the number of genes to detect and the proportion of them out of those tested. As such findings from previous simulations in similar correlation structures (in Section~\ref{chapt2:simulation_2015}) should be applicable to \glslink{gene expression}{expression} data with more complex correlation structures such as those occurring in biological pathways. Specifically, \gls{synthetic lethal} partners are distinguishable from closely correlated genes in the context of a biological pathway network both irrespective of thresholds (shown by \gls{ROC}) and with the sensitivity and specificity of p-value thresholds (adjusted by \{gls{FDR}) as used for \gls{SLIPT} (in Chapters~\ref{chap:SLIPT} and~\ref{chap:Pathways}).

%%example figure?

%%discuss inhib / 20K?
%inhib

The findings for inhibitory \gls{graph} structures were replicated with larger more complex \gls{graph} structures with inhibiting relationships and more \gls{synthetic lethal} genes to detect (shown in Figures~\ref{fig:simulation1205_randx_Graph5i}\nobreakdash--\ref{fig:simulation1205_randx_Graph7i2}). In each \gls{graph} structure, simulations entirely with inhibiting relationships (Figures~\ref{fig:simulation1205_randx_Graph5i},~\ref{fig:simulation1205_randx_Graph6i}, and~\ref{fig:simulation1205_randx_Graph7i}) had higher performance than the equivalent graph with entirely activating relationships (Figures~\ref{fig:simulation1205_randx_Graph5},~\ref{fig:simulation1205_randx_Graph6}, and~\ref{fig:simulation1205_randx_Graph7}) or a combination of activating and inhibiting relationships (Figures~\ref{fig:simulation1205_randx_Graph5i2},~\ref{fig:simulation1205_randx_Graph6i2}, and~\ref{fig:simulation1205_randx_Graph7i2}). As previously observed (in Figures~\ref{fig:simulation1205_randx_Graph6} and~\ref{fig:simulation1205_randx_Graph7}), the proportion of underlying \gls{synthetic lethal} genes to detect had a greater impact on performance of detection with \gls{SLIPT} than the specific structure of the genes which was replicated with inhibiting states (in Figures~\ref{fig:simulation1205_randx_Graph6i} and~\ref{fig:simulation1205_randx_Graph7i}) and combinations with a similar proportion of negative inhibitions (in Figures~\ref{fig:simulation1205_randx_Graph6i2} and~\ref{fig:simulation1205_randx_Graph7i2}). While the presence of negative correlations subtly affects the performance of \gls{SLIPT}, the methodology is robust across the exact structures of genes and is therefore applicable to detecting \gls{synthetic lethal} genes in a range of (synthetic lethal) biological pathways with different structural relationships.

%%20K?

%Therefore the findings based on simulations of genes with \gls{graph} structures in smaller datasets (as described in Section~\ref{chapt5:graphsim_performance}) will be relevant to larger datasets since the simulation results in these perform comparably or higher with more non \gls{synthetic lethal} genes to distinguish from them even with inhibitory relationships within the \gls{graph} structure (as shown with supporting results Appendix Figures~\ref{fig:simulation1207_randx_Graph1i}\nobreakdash--\ref{fig:simulation1207_randx_Graph7i2}). Hence these findings will be relevant in the context of empirical \gls{gene expression} datasets with thousands of genes such as those in human and cancers.

%20K inhib?

%This interpretation is consistent with the poorly performing simulations in inhibiting \glspl{graph} having a low sensitivity (in Figures~\ref{fig:simulation1207_randx_Graph1i}\nobreakdash--\ref{fig:simulation1207_randx_Graph7i2}). These simulations still show high specificity and accuracy with a low false positive rate for \gls{synthetic lethal} detection with p-value thresholds (adjusted by \{gls{FDR}) for \gls{SLIPT}. Such results support \gls{SLIPT} as a stringent methodology to detect \gls{synthetic lethal} interactions in large \gls{gene expression} data, excluding the majority of false positives with many of those remaining belonging to \gls{synthetic lethal} pathways.

\subsection{Simulations over Pathway-based Graphs}

\FloatBarrier

Thus far simulations of \glspl{synthetic lethal} in \gls{gene expression} with correlation structures have used simple blocks of correlated genes (as used in Section~\ref{chapt2:simulation_2015}) or derived from artifically constructed \gls{graph} structures (as used in Section~\ref{chapt5:graphsim}). While these are sufficient to make inferences on the impact of correlation structure, it remains to be shown whether these findings are reproducible in the complexity of the biological \glslink{graph}{network} structure. Specifically, \gls{SLIPT} was tested on simulated data with known underlying simulated \gls{synthetic lethal} partners (as described in Section~\ref{methods:simulating_SL}) with multivariate normal correlation structure derived from biological pathways (as described in Section~\ref{methods:graphsim}).

The Reactome \glslink{graph}{pathway} structure for the \gls{PI3K} cascade (as used extensively in Chapter~\ref{chap:Pathways}) was used to demonstrate the simulation procedure for detecting \glspl{synthetic lethal} in the \gls{graph} structure of a biological pathway. It is appropriate to do so since this pathway has clear directionality and signalling pathways were among those identified to be \gls{synthetic lethal} candidates (in Chapter~\ref{chap:SLIPT}). The \gls{PI3K} pathway having 138 genes is also of a moderate size and complexity compared to other biological pathways which is therefore suitable for comparison to previous \gls{graph} structures of a similar scale (50--100 genes) with the complexity of a characteristic of a biological pathway.

The performance of \gls{synthetic lethal} detection with \gls{SLIPT} in simulated \glslink{gene expression}{expression} data based on the Reactome \gls{PI3K} pathway (as shown in Figure~\ref{fig:simulation1205_randx_Graph_pi3k}) was concordant with previous findings. \gls{SLIPT} had high performance at detecting a low number of \gls{synthetic lethal} genes with poorer performance for high numbers of \gls{synthetic lethal} genes or lower sample sizes. In particular, the performance of simulations in the \gls{PI3K} pathway was highly resembled the simulation results for constructed \glspl{graph} of similar scale and complexity (as shown in Figures~\ref{fig:simulation1205_randx_Graph6} and~\ref{fig:simulation1205_randx_Graph7}). Using thresholds based on the $\chi^2$ p-value (adjusted by \{gls{FDR}), simulations in the biological \gls{PI3K} pathway had a higher sensitivity and lower specificity. While the performance decreases for more \gls{synthetic lethal} genes to detect within the simulated \gls{PI3K} pathway, this primarily involves a reduction in sensitivity to detecting underlying \gls{synthetic lethal} genes rather than false positives as the false positive rate decreases, the accuracy increases, and the specificity is relatively unperturbed (being more dependent on sample size). Thus \gls{SLIPT} is stringent in biological \gls{graph} structures and appropriate for detection of \gls{synthetic lethal} genes in complex correlation structures in \gls{gene expression} data involving biological pathways. 

\begin{figure*}[!htb]
%\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1205_randx_Graph_pi3kROC:Perf}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_pi3k_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1205_randx_Graph_pi3kROC:ROC}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_pi3k_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph_pi3k}
           %\raisebox{0.475\textwidth}{
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph_pi3k".pdf}}
           %}
        }%
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1205_randx_Graph_pi3kROC:AUC}
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_pi3k_AUC_samplesx_prop".png}}
        }%
    \end{center}
   \caption[Performance of simulations on the PI3K cascade]{\small \textbf{Performance of simulations on the \gls{PI3K} cascade.} Simulation of \glspl{synthetic lethal} was performed sampling from a multivariate normal distribution based on the Reactome \gls{PI3K} cascade. Performance of \gls{SLIPT} was high across parameters for detecting \glspl{synthetic lethal} in the \gls{graph} structure within a larger dataset. The performance decreases for a greater number of true positives to detect but the accuracy increases with a low false positive rate.}
%}
\label{fig:simulation1205_randx_Graph_pi3k}
%\end{mdframed}
\end{figure*}

These simulations were replicated in the larger and more complex , one of the most well supported \gls{synthetic lethal} pathways with loss of \textit{CDH1} in cancer (in Chapters~\ref{chap:SLIPT} and~\ref{chap:Pathways}). This pathway showed similar relationships between sensitivity, specificity, and false positive rate with number of \gls{synthetic lethal} partners and sample size (as shown in Figure~\ref{fig:simulation1205_randx_Graph_Gai}). While the overall performance was lower than for smaller networks structures, many of the findings from previous networks were replicated in a larger more complex biological network. In the G$_{\alpha i}$ signalling pathway, \gls{SLIPT} performed well for detecting low numbers of \gls{synthetic lethal} genes and was highly stringent against false positives for higher numbers of \gls{synthetic lethal} genes. 

\FloatBarrier

\subsection{Pathway Structures in a Simulated Human Genome}

\FloatBarrier

Simulations were also performed with \gls{graph} structures from biological pathways included in a larger dataset to simulate \gls{gene expression} data of the scale typical for human and cancer studies. These simulations (as discussed in Section~\ref{chapt5:graphsim_performance_20K}) have a higher specificity and therefore performance of \gls{SLIPT} for detecting \gls{synthetic lethal} genes was higher. The simulated \gls{PI3K} pathway (as shown in Figure~\ref{fig:simulation1207_randx_Graph_pi3k}), is no exception with high performance across parameter values, remaining high up to many genes. While the sensitivity decreases for high numbers of \gls{synthetic lethal} genes to detect within the \gls{PI3K} pathway, the \gls{SLIPT} methodology remains accurate with high specificity in a large simulated \gls{gene expression} dataset. 

\begin{figure*}[!htb]
%\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1207_randx_Graph_pi3kROC:Perf}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_pi3k_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1207_randx_Graph_pi3kROC:ROC}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_pi3k_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1207_randx_Graph_pi3k}
           %\raisebox{0.475\textwidth}{
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph_pi3k".pdf}}
           %}
        }%
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1207_randx_Graph_pi3kROC:AUC}
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_pi3k_AUC_samplesx_prop".png}}
        }%
    \end{center}
   \caption[Performance of simulations including the PI3K cascade]{\small \textbf{Performance of simulations including the \gls{PI3K} cascade.} Simulation of \glspl{synthetic lethal} was performed sampling from a multivariate normal distribution (without correlation structure apart from the Reactome \gls{PI3K} cascade). Performance of \gls{SLIPT} was high across parameters for detecting \glspl{synthetic lethal} in the \gls{graph} structure within a larger dataset. The sensitivity decreases for a greater number of true positives to detect but the specificity remains high with a low false positive rate.}
%}
\label{fig:simulation1207_randx_Graph_pi3k}
%\end{mdframed}
\end{figure*}

Therefore the \gls{SLIPT} is a highly stringent approach suitable for application to detecting \gls{synthetic lethal} genes and pathways within highly complex \glslink{gene expression}{expression} data with biological \glslink{graph}{pathway} structure.  In particular, the poorer performing simulations were highly stringent with low false positive rates which are an important consideration given the number of non \gls{synthetic lethal} genes to distinguish in a \gls{gene expression} dataset. The enrichment of true \gls{synthetic lethal} partners makes \gls{SLIPT} valuable for triage of candidates interacting \gls{synthetic lethal} partners for further validation and for pathway analysis.

The performance of simulation of \glspl{synthetic lethal} within a biological pathway (e.g., the example of the \gls{PI3K} cascade) was markedly higher in the context of a larger dataset of thousands of genes. As shown in a direct comparison with the \gls{graph} structure alone (in Figure~\ref{fig:simulation1205_randx_Graphpwaycf20K:Compare1}), performance was consistently higher across parameters in pathways of biological complexity from the Reactome database \citep{Reactome} such as \gls{PI3K} cascade). These findings were also replicated in the larger G$_{\alpha i}$ signalling pathway (shown in Figures~\ref{fig:simulation1207_randx_Graph_Gai} and~\ref{fig:simulation1205_randx_Graphpwaycf20K:Compare2}). 

\begin{figure*}[!thb]
%\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
       \subcaptionbox{The PI3K cascade \label{fig:simulation1205_randx_Graphpwaycf20K:Graph_pi3k}}{%
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph_pi3k".pdf}}
        }%
       \subcaptionbox{G signalling \label{fig:simulation1205_randx_Graphpwaycf20K:Graph_Gai}}{%
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph_Gai".pdf}}
        }%
       %\subcaptionbox{Graph Structure}{%
       %    \label{fig:simulation1205_randx_Graphpwaycf20K:Graph3i2}
       %    \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph3i2".pdf}}
       %}%

%
        \subcaptionbox{Performance in the PI3K cascade \label{fig:simulation1205_randx_Graphpwaycf20K:Compare1}}{%
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_pi3k_ROC_Compare(2)".pdf}}
        }%
        \subcaptionbox{Performance in G signalling \label{fig:simulation1205_randx_Graphpwaycf20K:Compare2}}{%
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Gai_ROC_Compare(2)".pdf}}
        }%

    \end{center}
   \caption[Performance on pathways improves with more genes]{\small \textbf{Performance on pathways improves with more genes.}  Simulations were performed in a \gls{graph} structures for the \gls{PI3K} cascade and G$_{\alpha i}$ signalling pathways structures to detect \gls{synthetic lethal} partners within them. As for constructed graphs, performance of detection in a dataset containing only the \gls{graph} structure (in colour) was as much lower than testing the \gls{graph} structure within a larger dataset of non \gls{synthetic lethal} genes (without correlations) for both \glspl{graph} of biological complexity.}
%}
\label{fig:simulation1205_randx_Graphpwaycf20K}
%\end{mdframed}
\end{figure*}

The biologically complex \gls{graph} structures based on the Reactome pathway use activating relationships to test \gls{synthetic lethal} detection with \gls{SLIPT} in the context of complex correlation structures. Inhibiting relationships were not used, these annotations are not provided in the Reactome database \citep{Reactome}. However, these investigations with pathway based \gls{graph} structures are informative of the findings in constructed \glspl{graph} (as used in Section~\ref{chapt5:graphsim}) being relevant to \gls{gene expression} data containing real correlated pathways. Furthermore previously comparisons between simulations with inhibiting relationships indicate that the performance of \gls{synthetic lethal} detection in an equivalent \gls{graph} structure with inhibitory relationships will likely be higher.

Negative genes (non \gls{synthetic lethal}) inversely correlated with the underlying \gls{synthetic lethal} partners will be distinguishable by \gls{SLIPT} with high specificity. Since \gls{synthetic lethal} genes are detectable will reasonable performance in large scale simulated \gls{gene expression} data and highly (positively) correlated genes in \glslink{graph}{pathway} structures, these findings serve as a conservative lower estimate for \gls{SLIPT} detecting \gls{synthetic lethal} genes within a \gls{synthetic lethal} biological pathway in empirical data. While \gls{synthetic lethal} genes are distinguishable from correlated genes to varying extents in simulations, false positives are also more likely to be within the same (synthetic lethal pathways). Therefore \gls{SLIPT} is both effective at triage of \gls{synthetic lethal} candidates within a biological pathway and at identifying \gls{synthetic lethal} pathways in high dimensional \gls{gene expression} data.


\FloatBarrier

\section{Discussion}

\subsection{Simulation Procedure}

Simulations have been performed to assess the performance of the \gls{SLIPT} methodology (as described in Section~\ref{methods:SLIPT} and with modifications) for detecting known underlying \gls{synthetic lethal} partners of a query gene. These simulations support the the findings in empirical data (in Chapters~\ref{chap:SLIPT} and~\ref{chap:Pathways}) by addressing whether the methodology used to generate them is accurate or has desirable statistical performance in controlled simulated conditions. These investigations include adjusting parameters such as the numbers of \gls{synthetic lethal} genes which were known in empirical data to assess the performance of the \gls{SLIPT} methodology across simulation parameters and characterise the datasets for which \gls{SLIPT} performs well. Simulation and statistically modelling also enabled comparison of the \gls{SLIPT} methodology to other statistical approaches to \gls{synthetic lethal} detection in \glslink{gene expression}{expression} data.

These simulations are based on a statistical model of \glspl{synthetic lethal} (as described in Section~\ref{methods:SL_Model}) which was designed stringently to ensure that if \glspl{synthetic lethal} is detectable in the simulated datasets it would also be detectable by the same methodology in empirical \glslink{gene expression}{expression} data. The model of \glspl{synthetic lethal} made conservative assumptions such as the low threshold of \glslink{gene expression}{expression} for gene function or the inclusion of cryptic higher-order \glspl{synthetic lethal} (when testing pairwise). These assumptions decrease the likelihood that \gls{synthetic lethal} signatures would be detectable in \glslink{gene expression}{expression} data. Thus it is reassuring that \glspl{synthetic lethal} is still detectable in under many simulation parameters as the performance of \gls{SLIPT} would be expected to be higher were these assumptions to be violated in empirical data.

\glsreset{HPC}
\glsreset{NeSI}
\glsreset{Slurm}

The simulation procedure (as described in Section~\ref{methods:simulating_SL}) is designed as a computational pipeline with arguments passes to scripts. The \gls{SLIPT} methodology and simulation of \glslink{gene expression}{expression} from \gls{graph} structures were both used as R \citep{R_core} software packages developed and released for this project (as described in Section~\ref{methods:r_packages}). This design ensures that the simulations can be robustly applied across parameters with consistency between simulations apart from the differences discussed. The simulation procedure is also flexible to simulating other datasets, including \gls{synthetic lethal} relationships and pathway correlation structures, should these be relevant to future investigations or \gls{bioinformatics} tool development. The computational pipeline is also compatible with parallel computing and made use of \gls{HPC} infrastructure provided by \gls{NeSI} using the \gls{Slurm} submission system (as described in Section~\ref{methods:HPC}). This parallel computing pipeline enabled extensive investigations into \glspl{synthetic lethal} in simulated data, including approximately 2 million cpu-hours on \gls{NeSI}. 


\subsection{Comparing Methods with Simulated Data}

Attempts were made to implement alternative \gls{synthetic lethal} detection approaches such as linear regression and the \gls{BiSEp} R package (discussed in Section~\ref{chapt5:compare_methods}). However, those tested were ineffective at detecting \glspl{synthetic lethal} in multivariate normal simulated data in comparison to \gls{SLIPT}. While some of the published \gls{synthetic lethal} detection methods \citep{Jerby2014, Lu2015} did not provide reproducible software releases for direct comparison, some of the central assumptions used in their design were tested by the statistical methods considered for \gls{synthetic lethal} detection in \glslink{gene expression}{expression} data.  

Another consideration is that \gls{BiSEp} takes considerably more time to compute predictions than \gls{SLIPT} or $\chi^2$ which limited the number of simulations that were feasible and made it difficult to apply across parameters in the simulation pipeline (even when using supercomputing infrastructure as discussed in Section~\ref{methods:HPC}). The computationally intensive nature of the \gls{BiSEp} procedure does not appear to be the issue for detecting \gls{synthetic lethal} genes in \gls{TCGA} data or simulations, although it has made more extensive simulations challenging. Rather, \gls{BiSEp} is not suitable in either case since the \gls{TCGA} data is normalised with \texttt{voom} \citep{limma} and simulated data is generated by sampling from a multivariate normal distribution. In either case, even subtle bimodal signatures in \glslink{gene expression}{expression} data were not consistently detectable or sufficient to detect \glspl{synthetic lethal}.

The \gls{BiSEp} methodology may perform better on other data types but it cannot be directly compared with the results for \gls{SLIPT} throughout this thesis which have used normalised or (multivariate) normally distributed data. Since it requires bimodal distributions, \gls{BiSEp} is not suitable for stringently normalised \glslink{gene expression}{expression} data nor would it be expected to perform on (ranked) pathway \glspl{metagene}. Thus \gls{SLIPT} represents a distinct approach more suitable for these data types whereas \gls{BiSEp} may be applicable to other applications in which bimodal distributions are more frequent.

This investigation also demonstrates that implementing scientific software from other research groups is not a trivial exercise, even when released as an open-source R package. Therefore, the above results were used to evaluate \gls{SLIPT} and compare it to other statistical rationales. An comprehensive comparison to contemporary \gls{synthetic lethal} detection approaches (and those released in the future) or further benchmarking is left to an impartial researcher to evaluate. The above findings show that the \gls{SLIPT} approach is able to detect \gls{synthetic lethal} genes in simulated data with comparable or better performance than a range of distinct statistical techniques and was appropriate for use throughout this thesis.  

\subsection{Design and Performance of SLIPT}

The simulation procedure using sampling from a multivariate normal distribution was used throughout the majority of the simulation investigations in this thesis. This approach has the advantages of emulating the continuous normalised \glslink{gene expression}{expression} data used for \gls{gene expression} analysis and enabled the simulation of correlation structures (as discussed in Section~\ref{chapt2:simulation_2015}). These simulations scaled to datasets of comparable scale to those used in \gls{gene expression} anlaysis with thousands of genes. The \gls{SLIPT} methodology was shown to perform robustly across large numbers of genes and simple correlation structures. This includes high specificity against genes positively correlated with the query gene for which the directional \gls{SLIPT} methodology more suited to distinguishing \gls{synthetic lethal} genes from than the $\chi^2$ test without directional criteria on the number of samples observed.

These findings were expanded upon in this chapter. Specifically, different quantiles were compared for \gls{SLIPT} and the $\chi^2$ test. These approaches using threshold based discrete gene function were compared to the Pearson correlation without loss of the continuous \glslink{gene expression}{expression} data. The 3-quantiles for \gls{SLIPT} (as described in Section~\ref{methods:SLIPT}) were optimal for both \gls{SLIPT} and the $\chi^2$ alone. In addition to being optimal for estimating the significance of \gls{synthetic lethal} interactions, these quantiles were also optimal for the directional criteria of \gls{SLIPT} since this method outperformed the $\chi^2$ test and was the most different at the 3-quantile. As previously, noted this difference was more pronounced with positively correlated genes (with the query gene) for which the specificity of \gls{SLIPT} improves and was replicated in large datasets with thousands of genes as occur in human \glslink{gene expression}{expression} data. These results were not simply due to sufficient samples for significant p-values since the performance as determined by \gls{AUROC} analysis is independent from significance thresholds. This indicates that the \gls{SLIPT} methodology (as it has been used in Chapters~\ref{chap:SLIPT} and~\ref{chap:Pathways}) is optimal and the parameters used to design it were appropriate.

Both discrete functional approaches (\gls{SLIPT} and $\chi^2$) were able to outperform negative correlation which supports their use. In particular, this result addresses the concern that arbitary thresholds of low and high gene function (as used by \gls{SLIPT}) lose useful data by compressing the spectrum of \gls{gene expression} into categorical data. However, this does not impede the performance of \gls{SLIPT} and can reduce statistical if the quantiles used are optimal. The poorer performance of correlation-based detection of \glspl{synthetic lethal} also indicates affirms the concept of gene function for \glspl{synthetic lethal} being qualitative, that is \glslink{gene expression}{expression} must be sufficient for cell viability and higher \glslink{gene expression}{expression} is not necessarily higher function (as this is not the case for all genes). Furthermore, the finding that negative correlation outperforms positive correlation is also consistent with co-expression being a poor predictor of \glspl{synthetic lethal} compared to other approaches \citep{Jerby2014}, supporting the claims of \citet{Lu2015}.

Compared with \gls{SLIPT}, neither correlation approaches nor bimodality signatures were suitable for detecting \glspl{synthetic lethal} in \glslink{gene expression}{expression} data. The correlation-based approaches make assumptions about the relationship between \gls{gene expression} and function which do not necessarily hold for all genes. Similarly, the bimodal approach is not appropriate for normalised data since deviations from a normal distribution have already been used for ensuring data quality, as is common practice for \gls{RNA-Seq} data. Other approaches were continuous data such as fitting linear models are likely to be prone to similar issues and not perform as well as \gls{SLIPT}. However, it is possible that these may be improved with conditioning on known \gls{synthetic lethal} partners with multivariate regression or Bayesian priors. Similarly, \gls{synthetic lethal} detection could be performed by iteratively conditioning upon the strong candidate from previous analysis. These approaches may be able to better circumvent the issues of high-order \glspl{synthetic lethal} and multiple testing. 

Nevertheless, the above findings are sufficient to assess the performance of \gls{SLIPT} and present an effective straightforward approach to \gls{synthetic lethal} detection in \gls{gene expression} data. Further development of linear models, Bayesian inference approaches, or comparison to existing \gls{synthetic lethal} approaches (e.g., machine learning) remain as future directions. Developing and testing more sophisticated statistical approach to \gls{synthetic lethal} detection may benefit from the concepts discussed with regard to the relatively simple \gls{SLIPT} methodology. Similarly, further comparisons and benchmarking of \gls{SLIPT} against other computational approaches to \gls{synthetic lethal} detection in \gls{gene expression} data is more suitable for an independent researcher and the \texttt{slipt} R package has been released (as described in Section~\ref{methods:r_packages}) for this purpose, in addition to further application in research.

%\ref{chapt5:compare_methods}

\subsection{Simulations from Graph Structures}

The simple correlation structures (as used in Section~\ref{chapt2:simulation_2015}) were expanded upon to simulate correlated genes based on \gls{graph} structures using the multivariate normal simulation procedure on correlation structures generated from grahp structures (as described in Section~\ref{chapt5:graphsim}). These simulations enable further investigations into te performance of \gls{SLIPT} in the context of more complex correlation structures. The simulation of \glslink{gene expression}{expression} from \glslink{graph}{network} structures is widely applicable to simulating pathway \glslink{gene expression}{expression} data and as such the \texttt{graphsim} R package has been released (as described in Section~\ref{methods:r_packages}).

These investigations show that \gls{SLIPT} performs robustly across datasets with different correlation structures, including those derived from \glspl{graph} with the complexity of biological pathways. The \gls{SLIPT} methodology was able to detect \gls{synthetic lethal} genes within \gls{synthetic lethal} pathways across many \gls{graph} structures. This methodology performed particularly well with \gls{synthetic lethal} pathways in the context of a larger dataset with a high specificity which supports \gls{SLIPT} as a stringent approach to \gls{synthetic lethal} detection in highly dimensional \gls{gene expression} data. Together these results support the use of \gls{SLIPT} in biological gene expresssion data since it is able to detect \gls{synthetic lethal} genes in highly complex correlation structures.

Similarly, the inclusion of inhibitory relationships in \gls{graph} structures was shown to increase the performance in simple networks supporting \gls{SLIPT} being applicable to biological data in which these relationships are common. While these results were not replicated in more complex inhibitory \gls{graph} structures, this is likely an artifact of the simulation procedure (which randomly selects \gls{synthetic lethal} genes) generating biologically implausible combinations of \gls{synthetic lethal} genes which are difficult to detect. When the test statistics in simulations with a \gls{synthetic lethal} gene were examined in more detail, the test statistics of the \gls{synthetic lethal} gene were consistently higher and distinguishable from nearby genes in the \gls{graph} structure. In contrast to previous concerns with inhibiting relationships, these differences were more pronounced with genes which had inhibitory relationships with \gls{synthetic lethal} genes. While distinguishable from nearby genes in a \glslink{graph}{pathway} structure, the genes correlated with \gls{synthetic lethal} still had higher test statistics than more distant genes (similar to observations with correlated genes in Section~\ref{chapt2:simulation_2015}).

In addition to being able to detect \gls{synthetic lethal} genes in a pathway, the proximal genes in a pathway are most likely to be false positives and therefore \gls{SLIPT} is also able to detect \gls{synthetic lethal} pathways. Therefore \gls{SLIPT} identifies genes which are likely to be constituent of a \gls{synthetic lethal} pathway and is more likely to rank underlying \gls{synthetic lethal} genes with greater significance. Together these findings support the use of \gls{SLIPT} throughout this thesis, further application of \gls{SLIPT}, and further development of such strategies for \gls{synthetic lethal} detection. Similarly, the simulation procedures developed and demonstrated for examining \gls{synthetic lethal} detection in \glslink{gene expression}{expression} data using \gls{graph} structures is amenable to further development and investigations into \glslink{graph}{pathway} structure in \glslink{gene expression}{expression} data such as predicting biological pathways from \glslink{gene expression}{expression} data or the impact of pathways on differential \glslink{gene expression}{expression} analyses.


\section{Summary}

A statistical model and simulation procedure has been developed to test the performance of the \gls{SLIPT} methodology in controlled conditions, using multivariate normal distributions. This simulation procedure has been developed into a computational pipeline which was able to test the statistical performance (using stringent assumptions) of \gls{SLIPT} across many parameters and compare it to alternative \gls{synthetic lethal} detection strategies. The \gls{SLIPT} methodology performs well at detecting small numbers of \gls{synthetic lethal} genes in simple systems. It does not perform as well in more complex systems but neither do alternative strategies. The \gls{SLIPT} methodology performs well compared to Pearson correlation and similar methods based on the $\chi^2$ test. Thus \gls{SLIPT} is an effective detection method for \gls{synthetic lethal} relationships in \glslink{gene expression}{expression} data despite its relatively simple design.

Simulations of more complex datasets, including large numbers of genes, complex correlation structure derived from \gls{graph} structures, and correlations with the query gene. \gls{SLIPT} performs robustly across these, including correlation structures based on complex biological pathways. The performance of \gls{SLIPT} improves in larger datasets, datasets with positive correlations with the query genes, and some \gls{graph} structures which include inhibiting relationships, namely those datasets more representative of \gls{gene expression} in biological data. \gls{SLIPT} was both capable of recurrently detecting genes within a \gls{synthetic lethal} pathway and distinguishing \gls{synthetic lethal} genes from correlated with them, even in highly complex correlation structures. Therefore \gls{SLIPT} is a stringent \gls{synthetic lethal} detection strategy and is applicable to \gls{gene expression} as previously demonstrated for the partners of \textit{CDH1} in breast and stomach cancer in this thesis.

\clearpage

\iffalse
\paragraph{Aims}

  \begin{itemize}
   \item A Model of Synthetic Lethal Genes in Gene Expression Data
   
   \bigskip
   
   \item Comparison of SLIPT to Alternative Approaches
   
   \bigskip
   
   \item Simulations of Known Synthetic Lethal Genes within Pathway Networks
      
  \end{itemize}

\paragraph{Summary}

    \begin{itemize}
      \item We have designed a straight-forward rational query-based \gls{synthetic lethal} detection method with the example of application to \textit{CDH1} in cancer \gls{gene expression}
      
      \bigskip
      
      \item I have developed a simulation pipeline to generate continuous \gls{gene expression} with \glslink{graph}{pathway} structure including a procedure to simulate \glspl{synthetic lethal} 
      
      \bigskip
      
      \item The simulation procedure shows that SLIPT is robust across \glslink{graph}{pathway} structures and has desirable performance compared to other statistical techniques 
      \end{itemize}
\fi