\chapter{Simulation and Modeling of Synthetic Lethal Pathways}
\label{chap:simulation}

Simulation and modelling of synthetic lethality in gene expression will be revisited in greater detail in this chapter, building upon the results provided to support the use of \gls{SLIPT} in Section~\ref{chapt2:simulation_2015}. A simulation procedure for generating simulated data with underlying (known) synthetic lethal partners of a query gene, such as \textit{CDH1}, was developed (as described in Section~\ref{methods:simulating_SL}) by sampling from a Multivariate normal distribution based on a statistical model of synthetic lethality in expression data (as described in Section~\ref{methods:SL_Model}). This simulation framework was applied to simulated data (in Section~\ref{chapt2:simulation_2015}), including simple correlation structures to assess the statistical performance of the \gls{SLIPT} methodology and support it's use a computational approach for detecting synthetic lethal candidates from expression data throughout this thesis (in Chapters~\ref{chap:SLIPT} and~\ref{chap:Pathways}). 

While this basic framework was sufficent to support the use of \gls{SLIPT} in prior Chapters, further investigations with simulations were conducted to assess the strengths and limitations of the \gls{SLIPT} methodology, compare it to alternative statistical approaches to synthetic lethal detection, and assess it's performance upon more complex correlation structures. Together these simulation investigations assess the performance of the \gls{SLIPT} methdology, including on pathway graph structures (such as those discussed in Chapter~\ref{chap:Pathways}) and determine whether the \gls{SLIPT} methdology (or similar refined bioinformatics strategies) are statistically rigourous or suitable for wider genomics applications.

These simulation investigations continue to utilise the Multivariate Normal simulation procedure (as applied in Section~\ref{chapt2:simulation_2015}) with further refinements. The \gls{SLIPT} methodology (and the equivalent $\chi^2$ test alone) were applied across a range of parameters (including altering the quantiles for detecting synthetic lethal direction and compared correlation. This was also applied to with query correlated genes (as performed in Section~\ref{chapt2:simulation_2015}).

A refined simulation procedure was developed specifically to extend the simulation procedure (described in Section~\ref{methods:simulation_SL_expression}) to utilise pathway graph structures for the correlation structures of simulated datasets (as described in Section~\ref{methods:graphsim}). This methdology can be applied to simulated correlation structures across simple graph structures to test specfic network modules or use pathway structures based on biological pathways (as discussed in Chapter~\ref{chap:Pathways}). Thus graph structure and simulation approaches were combined to test whether a gene locus in a pathway affects detection by \gls{SLIPT} and whether \gls{SLIPT} performance is affected by pathway structure. The simulation procedure based on graph structures were applied in a computational pipeline across many parameters with high-performance computing (as discussed in Section~\ref{methods:HPC}) and the core simulation functions have been released as a software package for wider use to test bioinformatics and statistical methods on graph structures (as described in Section~\ref{methods:igraph_extensions}).

%\section{Simulations and Modelling Synthetic Lethality in Expression Data}
%%committee
\iffalse
Synthetic lethality was modelled for effects on expression levels and whether these are detectable in known interacting and non-interacting genes in simulated data. These were conducted for expression data but the nature of these simulations would be relevant to how synthetic lethality would manifest in other factors, particularly DNA copy number variation and DNA methylation. These simulations were discussed at length in the previous meeting and showed that synthetic lethality was detectable with our approach in simple cases. While it was less effective, the methods were able to detect synthetic lethal genes in expression data with correlation structure (generated with the multi-variate normal distribution) and were distinguishable from correlated genes. Therefore the strongest (most significant) synthetic lethal genes are more likely to be true synthetic lethal partners and a high number of hits are expected from correlated genes and co-regulated pathways.

The power of the method to detect interactions depleted with increasing multiple tests, interactions, and cryptic (third party) interacting partners. Increased sample size counteracted these effects as expected. This led the idea that pathways would be more suitable as the focus of this project. Biological pathways led to fewer multiple tests, more relevant to understanding cancer biology, and are often drug targets in practice.
\fi

\section{Comparing methods} \label{chap5:compare_ methods}

The \gls{SLIPT} methodology (as it has been applied throughout Chaptersr~\ref{chap:SLIPT}and ~\ref{chap:Pathways}) was compared to alternative computational approaches to detecting synthetic lethality in simulated gene expression data. As discussed in Section~\ref{chapt2:simulation_2015}, this procedure enables testing the performance of detecting known synthetic lethal partner genes by sampling from a statistical model of synthetic lethality. While comprehensive benchmarking has not been performed, several approaches to synthetic lethal detection are considered (e.g., Pearson's correlation, the $\chi^2$ test, and testing for bimodality) to evaluate the strengths of the SLIPT methodology, including modifications to the parameters of \gls{SLIPT}.

Further testing of the performance of the SLIPT software R package (which is publicly released on GitHub as described in Section~\ref{methods:r_packages}) has been left to third party researchers to impartially compare it to other software for synthetic lethal detection which is outside the scope of this thesis. The following comparisons of simulations of computational detection of synthetic lethality with different statistical rationales suffice to discuss the strengths of \gls{SLIPT}, evaluate whether it is appropriate for further application in genomics research, and identify limitations which may be addressed with further developments. Some potential avenues for further development of computational synthetic lethal discovery will be discussed in Section~\ref{chap6:future}.

\subsection{Performance of SLIPT and $\chi^2$ across Quantiles}
\label{chap5:compare_chisq}

Simulated datasets with synthetic lethal partner genes were generated using the multivariate normal simulation procedure (as described in Section~\ref{methods:simulating_SL}) with performance assessed using \gls{AUROC} analysis (as described in Section~\ref{methods:simulation}). Synthetic lethal detection was compared for modifications to the \gls{SLIPT} methdology (as described in Section~\ref{methods:SLIPT}), namely that the quantiles used to define low and high expression was varied. Rather than $\frac{1}{3}$ (as used throughout this thesis) the samples below the lowest $\frac{1}{n}$ quantile and above the highest $\frac{1}{n}$ quantile were used for \gls{SLIPT} (and the $\chi^2$-test) to detect lowly and highly expressing samples respectively. The quantiles tested range from 2, splitting at the $\frac{1}{2}$ quantile (the median), to 100, using the lowest (1\%) and highest (99\%) percentiles.

This enables testing of the threshold for lowly expressing genes which is most able to distinguish synthetic lethal genes, even with higher-order synthetic lethal interactions (as discussed in Section~\ref{methods:SL_Model}). Both \gls{SLIPT} with the directional criteria for synthetic lethality and significance of the equivalent $\chi^2$ test were performed for each quantile. Pearson's correlation was also tested on simulated continuous expression data for synthetic lethal detection in simulated data, considering both positive and negative correlations separately as predictors of synthetic lethality for comparison with $\chi^2$ based approaches, using discete categories fo gene function deriving from quantiles. 

The results presented throughout this section use the example of 5 synthetic lethal partners to illustrate the differences in performance between the standard \gls{SLIPT} procedure (slipt-3) to $n$ quantiles (slipt-$n$), the $\chi^2$-test on the same quantiles, and positive or negative correlation. However, similar results across different numbers of known synthetic lethal genes are shown in Appendix~\ref{appendix:compare_chisq}. The synthetic lethal detection procedures were compared with 10,000 simulations of a small dataset of 100 genes and 1000 samples without correlation structure between genes as performed in Section~\ref{chapt2:simulation_mvtnorm}) . As shown in Figure~\ref{fig:simulation1108_Graph5All5SL}, the 3-quantiles previously used have optimal performance and \gls{SLIPT} has a comparable or higher performance than the $\chi^2$-test alone across quantiles.

    \begin{figure*}[!htbp]
    \begin{mdframed}
    \begin{center}
%
        \subcaptionbox{Barplot of $\chi^2$, SLIPT, and correlation.}{%
            \label{fig:simulation1108_Graph5All5SL:barplot}
            \includegraphics[width=0.55\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(5)".png}}
        }%

        \subcaptionbox{Lineplot of $\chi^2$, SLIPT, and correlation.}{%
            \label{fig:simulation1108_Graph5All5SL:lineplot}
            \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png}}
        }%
      \end{center}
      \caption[Performance of $\chi^2$ and SLIPT across quantiles]{\textbf{Performance of $\chi^2$ and SLIPT across quantiles}. Synthetic lethal detection (of 5 genes) with quantiles as in axis labels. The barplot uses the same hues for each quantile (grey for correlation) and darker for $\chi^2$ (and positive correlation). The line plot is coloured according to the legend. \gls{SLIPT} and  $\chi^2$ perform similarly, peaking at $\frac{1}{3}$-quantiles and converging to random (0.5). Negative correlation was higher than positive but not optimal quantiles for \gls{SLIPT} or $\chi^2$.}
    \label{fig:simulation1108_Graph5All5SL}
    \end{mdframed}
    \end{figure*}

Pearson's correlation performed worse than random (with an \gls{AUROC} lower than 0.5) as thus coexpression of genes is not predictive of synthetic lethality in simulated data. Conversely, negative correlation is predictive of synthetic lethality, consistent with synthetic lethal gene activity being mutually exclusive. However, neither correlation approach performed as well as the optimal quantiles for the \gls{SLIPT}  procedure or $\chi^2$-test.

These results are shown in both a bargraph and lineplot to show the individual results of each parameter and compare \gls{SLIPT} with the $\chi^2$-test side-by-side across quantiles. Similarly, these plots are given for detecting a range of known synthetic lethal partners in the simulations in Figures~\ref{fig:simulation1108_Graph5Allbarplot} and~\ref{fig:simulation1108_Graph5Alllineplot}. These demonstrate that the findings shown for 5 synthetic lethal genes is robust across different numbers of underlying synthetic lethal genes.

    \begin{figure*}[!tb]
    \begin{mdframed}
    \begin{center}
%
        %\subcaptionbox{Performance of $\chi^2$ and SLIPT}{%
        %    \label{fig:simulation1107_Graph5All5SL:barplot}
        %    \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_Cor_v_nCor_Allxy(5)".png}}
        %}%

        %\subcaptionbox{Comparison to Correlation}{%
        %    \label{fig:simulation1107_Graph5All5SL:lineplot}
            \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png}}
        %}%
      \end{center}
      \caption[Performance of $\chi^2$ and SLIPT across quantiles with more genes]{\textbf{Performance of $\chi^2$ and SLIPT across quantiles with more genes}. Synthetic lethal detection (of 5 genes in 20,000) with quantiles as in axis labels. The line plot is coloured according to the legend. As for simulations with fewer genes, \gls{SLIPT} and  $\chi^2$ perform similarly, peaking at $\frac{1}{3}$-quantiles and converging to random (0.5). Negative correlation was higher than positive but not optimal quantiles for \gls{SLIPT} or $\chi^2$.}
    \label{fig:simulation1107_Graph5All5SL}
    \end{mdframed}
    \end{figure*}

The synthetic lethal detection procedures were also tested with 1000 simulations of a larger dataset of 20,000 genes and 1000 samples. While fewer simulations gives a less accurate \gls{ROC} result, this is sufficient to replicate the above findings with a feasible human of genes in a human gene expression dataset and assess the impact of a higher proportion of non synthetic lethal genes (potential false positives). Simulated datasets of this size were also used in Section~ \ref{chapt2:simulation_mvtnorm} to test the specificity in a number of genes similar to that in experimental datasets for cancer genomes. As shown in Figure~\ref{fig:simulation1107_Graph5All5SL}, the above findings were replicated in  simulations of a larger dataset with 20,000 genes. These were also robustly replicated across varying numbers of underlying synthetic lethal genes (as shown in Figure~\ref{fig:simulation1107_Graph5Alllineplot}).

\FloatBarrier
    
\subsubsection{Correlated Query Genes affects Specificity}
\label{chap5:compare_chisq_query_cor}

\FloatBarrier

    \begin{figure*}[!tb]
    \begin{mdframed}
    \begin{center}
%
        %\subcaptionbox{Performance of $\chi^2$ and SLIPT}{%
        %    \label{fig:simulation1107_Graph5All5SL:barplot}
        %    \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_Cor_v_nCor_Allxy(5)".png}}
        %}%

        %\subcaptionbox{Comparison to Correlation}{%
        %    \label{fig:simulation1107_Graph5All5SL:lineplot}
            \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png}}
        %}%
      \end{center}
       \caption[Performance of $\chi^2$ and SLIPT across quantiles with query correlation]{\textbf{Performance of $\chi^2$ and SLIPT across quantiles with query correlation}. Synthetic lethal detection (of 5 genes in 100 including 5 query correlated) with quantiles as in axis labels. The line plot is coloured according to the legend. \gls{SLIPT} performs consistently higher than $\chi^2$ due to higher specificity. Negative correlation performed modestly.}
    \label{fig:simulation1108_query_cor_Graph5All5SL}
    \end{mdframed}
    \end{figure*}

    
As discussed in Section~\ref{chapt2:simulation_mvtnorm_query_cor}, postively correlated genes (with the query gene) have an impact of on the performance of synthetic lethal detection. \gls{SLIPT} able to distinguish these from synthetic lethal partners and hence has a higher specificity in datasets which include postively correlated genes with the query gene (as expected in gene expression data). The synthetic lethal detection procedures were compared with 10,000 simulations of a small dataset of 100 genes (with 5 correlated with the query gene) and 1000 samples otherwise without correlation structure between genes. As shown in Figure~\ref{fig:simulation1108_query_cor_Graph5All5SL}, this specificity is reflected in the increased \gls{AUROC} performance values for \gls{SLIPT} (in contrast to Figure \ref{fig:simulation1108_Graph5All5SL}). This specificity can be attributed to the directional criteria (as described in Section~\ref{methods:SLIPT}) since the $\chi^2$-test alone performs comparatively poorly with positively correlated genes.
    
The synthetic lethal detection procedures were also compared with 1000 simulations of a larger dataset of 20,000 genes (with 1000 correlated with the query gene) and 1000 samples otherwise without correlation structure between genes. This simulation increases the number of genes (and proportion of negative genes) to those feasible in a human gene expression dataset while maintaining a comparable 5\% of postively correlated genes. As shown in Figure~\ref{fig:simulation1107_query_cor_Graph5All5SL}, \gls{SLIPT} still outperforms $\chi^2$ or negative correlation and is optimal at the 3-quantile. However, the difference between \gls{SLIPT} and $\chi^2$ were less pronounced in a larger dataset since the sheer number of negative genes (as potential false postives) affects the specificity of \gls{SLIPT} which distinguishes it from $\chi^2$-test alone and is an important consideration in large-scale genomics analysis.

   
    \begin{figure*}[!tb]
    \begin{mdframed}
    \begin{center}
%
        %\subcaptionbox{Performance of $\chi^2$ and SLIPT}{%
        %    \label{fig:simulation1107_Graph5All5SL:barplot}
        %    \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_Cor_v_nCor_Allxy(5)".png}}
        %}%

        %\subcaptionbox{Comparison to Correlation}{%
        %    \label{fig:simulation1107_Graph5All5SL:lineplot}
            \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png}}
        %}%
      \end{center}
      \caption[Performance of $\chi^2$ and SLIPT across quantiles with query correlation and more genes]{\textbf{Performance of $\chi^2$ and SLIPT across quantiles with query correlation and more}. Synthetic lethal detection (of 5 genes in 20,000 including 1000 query correlated) with quantiles as in axis labels. The line plot is coloured according to the legend. \gls{SLIPT} performs consistently higher than $\chi^2$ due to higher specificity. Negative correlation performed modestly.}
    \label{fig:simulation1107_query_cor_Graph5All5SL}
    \end{mdframed}
    \end{figure*}

Nevertheless, \gls{SLIPT} with 3-quantiles (as performed throughout Chapters~\ref{chap:SLIPT} and~\ref{chap:Pathways}), has higher performance than other quantiles, particularly with postive correlations (replicating the Section~\ref{chapt2:simulation_mvtnorm_query_cor}). These findings hold across different numbers of underlying synthetic lethal genes (as shown in Figures~\ref{fig:simulation1108_query_cor_Graph5Alllineplot} and ~\ref{fig:simulation1107_query_cor_Graph5Alllineplot}).

Together these results support the use of \gls{SLIPT}, particularly the use of quantiles as thresholds for gene function and specific use of 3-quantiles which perform well compared to other quantiles. A particular concern in the design of \gls{SLIPT} for expression data was sufficient samples size when the data is divided into quantiles. The \gls{SLIPT} methdology further peforms better for 3-quantiles (and other moderate values), irrespective of sample size or p-value threshold as \gls{AUROC} values are independent from them. Such quantiles ensure that there are a sufficent number of samples expected below and above them so that deviations from these are statistically detectable. Thhese quantiles were also optimal for the $\chi^2$ as both significance and the \gls{SLIPT} directional conditions rely use the same expected values.

\FloatBarrier

\subsection{Alternative Synthetic Lethal Detection Strategies}

The categorical approach for gene function to detect synthetic lethality also outperforms correlations which use continuous data directly. Correlation performing poorly as a synthetic lethal detection strategy consistent with there not necessarily being a relationship between synthetic lethal partners which can be in distinct biological pathways, expressed at different times or in different cell types. Nevertheless, correlation is among the alternative detection methods considered in further detail.

The \gls{BiSEp} R package \citep{Wappett2014} for using bimodality to detect synthetic lethality \citep{Wappett2016} and linear models were also considered. These statistical methods span a range of computational approaches to detecting synthetic lethality and serve to compare alternatives to \gls{SLIPT}, supporting it's design (see Section~\ref{methods:SLIPT}) and application (in Chapters~\ref{chap:SLIPT} and~\ref{chap:Pathways}). Although these are intended not intended to be a comprehensive benchmarking of existing synthetic lethal tools, implementing other synthetic lethal detection software is out of the scope of this project. However, these comparisons are able provide supporting data from statistical modelling and simulations for the viability of the \gls{SLIPT} methodology for syntetic lethal discovery in cancer (as demonstrated in Chapter~\ref{chap:SLIPT}) and further applications.

\subsubsection{Correlation for Synthetic Lethal Detection}
\label{chap5:compare_correlation}

\FloatBarrier

As shown in Section~\ref{chap5:compare_chisq}, negative (Pearson's) correlation performed better than positive correlation, indicating the inverse relationships were more predictive of synthetic lethality. However, neither correlation approach performed as well as \gls{SLIPT} or the $\chi^2$ test as a predictor of synthetic lethal gene partners. Although negative correlation still often performed considerably better than random chance.

    \begin{figure*}[!htb]
    \begin{mdframed}
    \begin{center}
      \resizebox{1 \textwidth}{!}{
	\includegraphics{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_Cor_randx/SL_Model_Test_Graph_10K_Graph4_ROC_Compare(1)_SLIPT_v_Cor".png}}
      }
      \end{center}
      \caption[Performance of negative correlation and SLIPT]{\textbf{Performance of negative correlation and SLIPT}. Synthetic lethal detection with SLIPT was compared to negative (Pearson's) correlation across parameters. SLIPT consistently outperformed correlation, although both approaches had lower performance for more synthetic lethal partners and lower sample sizes.}
    \label{fig:simulation1205_randx_Graph4cfnCor}
    \end{mdframed}
    \end{figure*}
    
    
Negative correlation was compared directly to the \gls{SLIPT} methdology (as described in Section~\ref{methods:SLIPT}) across numbers of known synthetic lethal partners and sample size (ranging from 500 to 5000). This comparison used 1000 simulations of a dataset with 20,000 genes and synthetic lethal genes from within a network (sampled as in Section~\ref{methods:graphsim})) with a 0.8 correlation between adjacent genes (as explored in more detail in Section~\ref{chap5:graphsim}). In a direct comparison of \gls{SLIPT} and negative correlation (shown in Figure~\ref{fig:simulation1205_randx_Graph4cfnCor}), \gls{SLIPT} consistently has higher performance insimulated data across parameter values and (inverse) correlation-based approaches perform modestly in comparison. 

[Add Other Graphs to Appendix?]


Both \gls{SLIPT} and correlation had poorer performance with increasing numbers of the synthetic lethal genes to detect while they had higher performance in higher sample sizes as expected (as previously observed for \gls{SLIPT} in Section~\ref{chapt2:simulation_2015}). Thus the issue with detection of greater numbers of synthetic lethal genes is not specfic to \gls{SLIPT} but occurs across computational methods of synthetic lethal discovery in (simulated) expression data and likely stems from cryptic higher-order synthetic lethal interactions (as conservatively assumed in Section~\ref{methods:SL_Model}). 
    
\FloatBarrier

\subsubsection{Testing for Bimodality with BiSEp}
\label{chap5:compare_bisep}

Exhaustive attempts were also made to compare \gls{SLIPT} to the \gls{BiSEp} methodology \citep{Wappett2016}, a statistical approach to identify synthetic lethal gene pairs from mutually exclusive relationships using bimodal distributions. This synthetic lethal detection methodology is also designed for expression analysis in cancer and is readily available as an (open-source) R package \citep{Wappett2014}, a practice which facilitates adoption and testing of the methdology on the same datasets and simulations procedures as previously used for \gls{SLIPT}.

The \gls{BiSEp} package is designed for global testing of all potential gene pairs in the genome for synthetic lethality rather than focusing on the search space of  potential partners of the query gene. This approach was unable to detect synthetic lethal gene pairs in the TCGA breast cancer expression dataset \citep{TCGA2012}. However, this may be due to stringent thresholds under the multiple testing of millions of potential gene pairs.

For a direct comparison with the query-based \gls{SLIPT} approach, the source code of the \gls{BiSEp} R functions were modified to test solely for the partners of a specific gene. This approach was still unable to detect synthetic lethal partners of \textit{CDH1} in TCGA breast cancer expression data \citep{TCGA2012}, even with the detection thresholds for bimodality and significance greatly relaxed from those which the package defaults to.

To circumvent multiple testing issues, \gls{BiSEp} only tests gene pairs for synthetic lethality between genes with a detectable bimodal distribution. However, even with relaxed thresholds bimodal distributions were not detectable in the normalised TCGA data \citep{TCGA2012}. Such normalisation \cite{limma} is standard practice for expression datasets generated from microarrays or \gls{RNA-Seq} and therefore \gls{BiSEp} may not be appropriate to apply to this data. However, it is noted that \gls{BiSEp} may also use other data types such as DNA copy number or cell line data for which it may be more applicable \cite{Wappett2016}.

Nevertheless, attempts were made to test \gls{BiSEp} on simulated datasets with underlying synthetic lethal genes (using the procedures described in Sections~\ref{methods:simulating_SL} and~\ref{methods:graphsim}). However, \gls{BiSEp} was also unable to detect genes with bimodal distributions of genes (and thus unable to detect synthetic lethality) in a limited number of simulations. Another consideration is that \gls{BiSEp} takes considerably more time to compute predictions than \gls{SLIPT} or $\chi^2$ which limited the number of simulations that were feasible and made it difficult to apply across parameters in the simulation pipeline (even when using supercomputing infrastructure as discussed in Section~\ref{methods:HPC}).

The computationally intensive nature of the \gls{BiSEp} procedure does not appear to be the issue for detecting synthetic lethal genes in TCGA data or simulations, although it has made more extensive simulations challenging. Rather \gls{BiSEp} is not suitable in either case since the TCGA data is normalised with \texttt{voom} \citep{limma} and simulated data is generated by sampling from a multivariate normal distribution. In either case, even subtle bimodal signatures in expression data were not consistently detectable or sufficient to detect synthetic lethality. The \gls{BiSEp} methdology may perform better on other data types but it cannot be directly compared with the results for \gls{SLIPT} throughout this thesis which have used normalised or (multivariate) normally distributed data. Since it requires bimodal distributions, \gls{BiSEp} is not suitable for stringently normalised expression data nor would it be expected to perform on (ranked) pathway metagenes. Thus \gls{SLIPT} represents a distinct approach more suitable for these data types whereas \gls{BiSEp} may be applicable to other applications in which bimodal distributions are more frequent.

This investigation also demonstrates that implementing scientific software from other research groups is not a trivial exercise, even when released as an open-source R package. Therefore, the above results are sufficient to evaluate \gls{SLIPT} and compare it to other statistical rationales. An comprehensive comparison to contemporary synthetic lethal detection approaches (and those released in the future) or further benchmarking is left to an impartial researcher to evaluate and is outside the scope of this thesis. The above findings show that the \gls{SLIPT} approach is able to detect synthetic lethal genes in simulated data with comparable or better performance than a range of distinct statistical techniques and was appropriate for use throughout this thesis.  

\iffalse

\begin{itemize}
 \item Designed for global SL
 \item Unable to detect SL partners in TCGA data
 \item Source code modified to test partners of query gene (R package)
 \item Still unable to identify SL genes for CDH1 in TCGA
 \item Computationally-intensive, longer to run than SLIPT, more difficult to evaluate many iterations
 \item Unable to identify SL candidates in a limited number of simulations
 \item Assumes Bi-modal distribution detectable: not appropriate for normalised expression data (standard in the RNA-Seq analysis) or ranked (metagenes) -- may be applicable to other datasets
 \item Comparing software is non-trivial (even those released as R packages), the above results are sufficent to evaluate \gls{SLIPT}, and further benchmarking out of scope.
\end{itemize}

\subsubsubsection{Implementation and Computation Time}
\label{chap5:compare_compute_time}
\fi

[Compare runtime?]

\iffalse
\begin{itemize}
 \item ChiSQ
 \item SLIPT
 \item Correlation
 \item BiSEp
 \item LM/GLM
\end{itemize}


\subsubsection{Testing Synthetic Lethal Genes with Linear Models}
\label{chap5:compare_linear_model}
[Move to future Dir??]

\begin{itemize}
 \item Strategy to detect SL with linear models by fit to curve (significance) and slope (direction)
 \item Amenable to conditioning on known SL or iterative conditioning on strongest SL to detect other partners of higher-order SL
 \item All attempts: linear, GLM, and linear polynomial (quadratic, cubic, or quintic) underperform \gls{SLIPT}, similar to Pearson's correlation results
 \item Linear models and regression may still be an avenue for further detection of SL (e.g., with Bayes)
\end{itemize}
\fi

[Discuss linear models?]

%\subsection{Linear models}

\iffalse
\section{Developing a linear model predictor of synthetic lethality}
\subsection{Linear models}
\subsection{Polynomial models}
\subsection{Conditioning}
\subsection{SLIPTv2}
\fi

\section{Simulations with Graph Structures}
\label{chap5:graphsim}

Simulations of synthetic lethality in Section~\ref{chapt2:simulation_2015} included correlated blocks of genes as a rudimentary model of pathway structure and co-regulated genes. Here the simulation procedure was expanded to account for more complex graph structures by sampling from multivariate normal distributions with correlation structure derived from graph structures (as described in Section~\ref{methods:graphsim}). This approach enables simulation of synthetic lethal pathways with known correlation structure and known partners (of a gene not in the pathway) and evaluation of the performance of \gls{SLIPT} under simple controlled correlation structures and complex correlations such as those derived from biological networks (such as those described in Chapter~\ref{chap:Pathways}). The \gls{SLIPT} methdology will be tested both in artificial constructed networks to evaluate the effect of pathway structure on synthetic lethal detection and on large biologically feasible pathways to test whether \gls{SLIPT} is robust under complex correlation structures and applicable to such complex genomics data.

These simulations combine the approach of prior simulation analyses (in Sections~\ref{chapt2:simulation_2015} and~\ref{chap5:compare_ methods}) with the graph structures for biological pathways (as used in Chapter~\ref{chap:Pathways}). This enables testing whether subtle or large differences in pathway structure affect synthetic lethal detection, whether inhibiting relationships (or inverse correlations) between genes affects synthetic lethal detection, and whether synthetic lethal detection varies across which gene is synthetic lethal or affects proximal genes in the pathway structure. In addition, large numbers of synthetic lethal genes and biologically feasible numbers of genes (with many non-synthetic lethal genes) will be tested to replicate the findings of Sections~\ref{chapt2:simulation_2015} and~\ref{chap5:compare_ methods} in correlated structures derived from pathway graphs, including examples of biological pathways from Reactome.

To demonstrate the impact of pathway structure of the performance of \gls{SLIPT} for synthetic lethal detection in simulations, simple and more complex constructed graph structures will be used (as depicted in Figures~\ref{fig:simple_graph1}\nobreakdash--\ref{fig:simple_graph7}). In addition, the \gls{PI3K} and G$_{\alpha i}$ signalling pathways derived from Reactome will be used for simulation of pathway structures of biological complexity (as shown in Figures~\ref{fig:SL_Pathway_Pi3K} and~\ref{fig:SL_Pathway_GPCR}).

\FloatBarrier

\subsection{Performance over a Graph Structure}
\label{chap5:graphsim_performance}

\subsubsection{Simple Graph Structures}
\label{chap5:graphsim_performance_simple}

\FloatBarrier

Simple pathway modules were used to test the effect of pathway structure on the performance of detecting synthetic lethal partners within graph structures. To start with, the cases (shown by Figure~\ref{fig:simple_graph1}) where a gene has one upstream regulator and two downstream (Graph1) or a gene has two upstream regulators and one downstream gene (Graph2) were used. In these simulations, \gls{SLIPT} has a high performance detecting randomly selected synthetic lethal partners in small simple networks (as shown in Figures~\ref{fig:simulation1205_randx_Graph1} and~\ref{fig:simulation1205_randx_Graph2}). 

\begin{figure*}[!htbp]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1205_randx_Graph1ROC:Perf}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1205_randx_Graph1ROC:ROC}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph1}
           \raisebox{0.1875\textwidth}{
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1".pdf}}
           }
        }%
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1205_randx_Graph1ROC:AUC}
           \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1_AUC_samplesx".png}}
        }%
    \end{center}
   \caption[Performance of simulations on a simple graph]{\small \textbf{Performance of simulations on a simple graph.} Simulation of synthetic lethality was performed sampling from a multivariate normal distribution generated from Graph1. Performance of \gls{SLIPT} declines for more synthetic partners but this is mitigated by increased sample sizes (in darker colours). This manifests as a decline in specificity and the false discovery rate. For each parameter value, 10,000 simulations were used.}
%}
\label{fig:simulation1205_randx_Graph1}
\end{mdframed}
\end{figure*}

As previously observed (in Section~\ref{chapt2:simulation_2015}), performance declines with higher numbers of syntethic lethal genes to detect and lower sample sizes. Although the sensitivity of \gls{SLIPT} is high with conventional p-value thresholds (adjusted by FDR). Thus synthetic lethal partners are often distinguishable for non synthetic lethal genes, even in simple highly correlated networks. The small number of genes and their high correlation has an impact on the \gls{ROC} curves for higher numbers of synthetic lethal partners which are skewed compared to those observed previously. Note that specificity cannot be tested if all genes potential partner genes are synthetic lethal which limits the number of synthetic lethal genes which can be tested. 

These results are particularly consistent between the pathway modules of diverging (Graph1) and converging (Graph2) signals, with the \gls{AUROC} performance and underlying curves being strikingly similar between these graph structures (as shown in Figures~\ref{fig:simulation1205_randx_Graph1} and~\ref{fig:simulation1205_randx_Graph2}). This indicates that the performance of \gls{SLIPT} is not perturbed by pathway structure, in particular the direction of pathway relationships as these graph structures also demonstrate pathways in opposite direction. In a direct comparison (shown in Figure~\ref{fig:simulation1205_randx_Graph1cf2}), the performance of simulations in these simple graphs does not differ across parameter values and therefore \gls{SLIPT} is robust to pathway direction.


\begin{figure*}[!htb]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
       \subcaptionbox{Graph1 Structure}{%
           \label{fig:simulation1205_randx_Graph1cf2:Graph1}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1".pdf}}
        }%
       \subcaptionbox{Graph2 Structure}{%
           \label{fig:simulation1205_randx_Graph1cf2:Graph2}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph2".pdf}}
        }%

%
        \subcaptionbox{Gene category in simulations}{%
            \label{fig:simulation1205_randx_Graph1cf2:Compare}
            \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph1_ROC_Compare_Graph2(1)".pdf}}
        }%
        %\subcaptionbox{Corresponding $\chi^2$ values}{%
        %    \label{fig:simulation_May4SLreps:second}
        %    \includegraphics[width=0.35\textwidth]{{"SL_Model_May15mvnorm_heatmap_10XSL_cor_comp2(2)".png}}
        %}%

    \end{center}
   \caption[Performance of simulations is similar in simple graphs]{\small \textbf{Performance of simulations is similar in simple graphs.} The \gls{AUROC} values for simulations of multivariate normal distributions based on each Graph structure yielded indistinguishable performance across parameter values in 10,000 simulations.}
%}
\label{fig:simulation1205_randx_Graph1cf2}
\end{mdframed}
\end{figure*}

\FloatBarrier

\subsubsection{Constructed Graph Structures}
\label{chap5:graphsim_performance_constructed}

\FloatBarrier

Progressively more complex graph structures were used to test the performance detecting synthetic lethal partners with \gls{SLIPT} in simulated expression data with pathway correlation structures. For simple chains of gene representing pathways (shown in Figures~\ref{fig:simulation1205_randx_Graph3} and~\ref{fig:simulation1205_randx_Graph4}), the above findings were generally replicated. Performance was high across parameter values in these small networks, with similar decreases in higher numbers of synthetic lethal genes to detect and lower sample size.

\begin{figure*}[!htbp]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1205_randx_Graph4ROC:Perf}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1205_randx_Graph4ROC:ROC}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph4}
           \raisebox{0.1875\textwidth}{
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4".pdf}}
           }
        }%
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1205_randx_Graph4ROC:AUC}
           \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4_AUC_samplesx".png}}
        }%
    \end{center}
   \caption[Performance of simulations on a constructed graph]{\small \textbf{Performance of simulations on a constructed graph.} Simulation of synthetic lethality was performed sampling from a multivariate normal distribution generated from Graph4. Performance of \gls{SLIPT} declines for more synthetic partners and lower sample sizes. For each parameter value, 10,000 simulations were used.}
%}
\label{fig:simulation1205_randx_Graph4}
\end{mdframed}
\end{figure*}

When detecting synthetic lethal genes with \gls{SLIPT} using adjusted (FDR) p-value thresholds, the performance differences can be largely attributed to changes in specificity. Although the accuracy increases and false discovery rate decreases desirably with higher numbers of synthetic lethal  genes despite a lower performance in \gls{ROC} curves. Therefore the thresholds imposed by adjusted p-values are appropriate for detecting synthetic lethal partners, even in strongly correlated pathways, at least in these small-scale test cases.

However, an artifact of these small test cases is the skewed \gls{ROC} curves (as discussed in Section~\ref{chap5:graphsim_performance_simple}) which may be related to the low number of non-synthetic lethal genes to identify as true negatives, affecting the accuracy of specificity. This is unlikely to occur in large expression datasets with many negative genes, as shown previously (in Section~\ref{chapt2:simulation_2015}) and~\ref{chap5:graphsim_performance_simple}) in simulations of graphs structures in larger datasets (in Section~\ref{chap5:graphsim_performance_20K}). This does not occur in larger, more complex graphs structures, even with modest total numbers of genes and high correlations.

As shown in Figure~\ref{fig:simulation1205_randx_Graph5}, sensitivity declines over a greater range for the number of synthetic lethal partners in a larger network with a tradeoff with specificity. Although the accuracy declines for greater numbers of synthetic lethal partners and the false discovery rate peaks at intermediate values. In this range difference between simulations with greater sample size. The \gls{AUROC} results were similar for other more complex graph structures (as shown in Figures~\ref{fig:simulation1205_randx_Graph6} and~\ref{fig:simulation1205_randx_Graph7}), these graphs performed similarly to each other, although they had differences from Figure~\ref{fig:simulation1205_randx_Graph5} in their sensitivity and specificity at an adjusted (FDR) p-value threshold. This difference may stem from different ratios of synthetic lethal and non-synthetic lethal genes to detect, since the latter graphs (in Figures~\ref{fig:simulation1205_randx_Graph6} and~\ref{fig:simulation1205_randx_Graph7}) had half the total genes to that shown in Figure~\ref{fig:simulation1205_randx_Graph5}.

However, the graph structures (of similar size) were highly distinct and yet had similar performance profiles across parameters. Therefore \gls{SLIPT} is robust across pathway structures and is more affected by the number of genes to detect and their proportion in those tested and the findings from previous simulations in similar correlation structures (in Section~\ref{chapt2:simulation_2015}) should be applicable to expression data with more complex correlation structures such as biological data containing biological pathways. Specifically, synthetic lethal partners are distinguishable from closely correlated genes in te context of a biological pathway network both irrespective of thresholds (shown by \gls{ROC}) and with the sensitivity and specificity of p-value thresholds (adjusted by FDR) as used for \gls{SLIPT} (in Chapters~\ref{chap:SLIPT} and~\ref{chap:Pathways}).


\begin{figure*}[!htbp]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1205_randx_Graph5ROC:Perf}
            \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph5_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1205_randx_Graph5ROC:ROC}
            \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph5_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph5}
           %\raisebox{0.1875\textwidth}{
           \includegraphics[width=0.5\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph5".pdf}}
           %}
        }%
        
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1205_randx_Graph5ROC:AUC}
           \includegraphics[width=0.5\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph5_AUC_samplesx_prop".png}}
        }%
    \end{center}
   \caption[Performance of simulations on a large graph]{\small \textbf{Performance of simulations on a large graph.} Simulation of synthetic lethality was performed sampling from a multivariate normal distribution generated from Graph5. Performance of \gls{SLIPT} declines for more synthetic partners and lower sample sizes. For each parameter value, 10,000 simulations were used.}
%}
\label{fig:simulation1205_randx_Graph5}
\end{mdframed}
\end{figure*}


\FloatBarrier


\subsection{Performance with Inhibitions}
\label{chap5:graphsim_performance_inhib}

\FloatBarrier

\begin{figure*}[!htbp]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1205_randx_Graph1iROC:Perf}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1i_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1205_randx_Graph1iROC:ROC}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1i_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph1i}
           \raisebox{0.1875\textwidth}{
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1i".pdf}}
           }
        }%
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1205_randx_Graph1iROC:AUC}
           \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1i_AUC_samplesx".png}}
        }%
    \end{center}
   \caption[Performance of simulations on a simple inhibiting graph]{\small \textbf{Performance of simulations on a simple inhibiting graph.} Simulation of synthetic lethality was performed sampling from a multivariate normal distribution (without correlation structure). Performance of \gls{SLIPT} declines for more synthetic partners but this is mitigated by increased sample sizes (in darker colours). This generally occurs as the sensitivity decreases for a greater number of true positives to detect, leading to a trade off in accuracy as seen in a trough for false discovery rate and the ROC curves.}
%}
\label{fig:simulation1205_randx_Graph1i}
\end{mdframed}
\end{figure*}

\begin{figure*}[!t]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph1cf1i:Graph1}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1".pdf}}
        }%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph1cf1i:Graph1i}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1i".pdf}}
        }%

%
        \subcaptionbox{Gene category in simulations}{%
            \label{fig:simulation1205_randx_Graph1cf1i:Compare}
            \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph1_ROC_Compare(1)".pdf}}
        }%
        %\subcaptionbox{Corresponding $\chi^2$ values}{%
        %    \label{fig:simulation_May4SLreps:second}
        %    \includegraphics[width=0.35\textwidth]{{"SL_Model_May15mvnorm_heatmap_10XSL_cor_comp2(2)".png}}
        %}%

    \end{center}
   \caption[Performance is higher on a simple inhibiting graph]{\small \textbf{Performance is higher on a simple inhibiting graph.} The gene category (blue for query, cyan for query-correlated, red for SL, orange for SL-correlated, forest green for non-SL-correlated, and green for non-SL) ordered by $\chi^2$ signed by the \gls{SLIPT} directional condition is shown across simulations. For each of 1--10 SL partners, 10 simulations demonstrate that the increasing numbers of SL partners become harder detect. The $\chi^2$ values show a clear threshold for SL and correlated genes when there are fewer of them, distinguishable from correlated genes in this case.}
%}
\label{fig:simulation1205_randx_Graph1cf1i}
\end{mdframed}
\end{figure*}

\begin{figure*}[!htbp]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1205_randx_Graph4i2ROC:Perf}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4i2_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1205_randx_Graph4i2ROC:ROC}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4i2_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph4i2}
           \raisebox{0.1875\textwidth}{
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4i2".pdf}}
           }
        }%
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1205_randx_Graph4i2ROC:AUC}
           \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4i2_AUC_samplesx".png}}
        }%
    \end{center}
   \caption[Performance of simulations on an inhibiting graph]{\small \textbf{Performance of simulations on an inhibiting graph.} Simulation of synthetic lethality was performed sampling from a multivariate normal distribution (without correlation structure). Performance of \gls{SLIPT} declines for more synthetic partners but this is mitigated by increased sample sizes (in darker colours). This generally occurs as the sensitivity decreases for a greater number of true positives to detect, leading to a trade off in accuracy as seen in a trough for false discovery rate and the ROC curves.}
%}
\label{fig:simulation1205_randx_Graph4i2}
\end{mdframed}
\end{figure*}

\begin{figure*}[!t]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph4cf4i:Graph4}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4".pdf}}
        }%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph4cf4i:Graph4i}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4i".pdf}}
        }%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph4cf4i:Graph4i2}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4i2".pdf}}
        }%

%
        \subcaptionbox{Gene category in simulations}{%
            \label{fig:simulation1205_randx_Graph4cf4i:Compare4i}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph4_ROC_Compare(1)".pdf}}
        }%
        \subcaptionbox{Gene category in simulations}{%
            \label{fig:simulation1205_randx_Graph4cf4i:Compare4i2}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph4_ROC_Compare(4)".pdf}}
        }%
        %\subcaptionbox{Corresponding $\chi^2$ values}{%
        %    \label{fig:simulation_May4SLreps:second}
        %    \includegraphics[width=0.35\textwidth]{{"SL_Model_May15mvnorm_heatmap_10XSL_cor_comp2(2)".png}}
        %}%

    \end{center}
   \caption[Performance is affected by inhibition in graphs]{\small \textbf{Performance is affected by inhibition in graphs.} The gene category (blue for query, cyan for query-correlated, red for SL, orange for SL-correlated, forest green for non-SL-correlated, and green for non-SL) ordered by $\chi^2$ signed by the \gls{SLIPT} directional condition is shown across simulations. For each of 1--10 SL partners, 10 simulations demonstrate that the increasing numbers of SL partners become harder detect. The $\chi^2$ values show a clear threshold for SL and correlated genes when there are fewer of them, distinguishable from correlated genes in this case.}
%}
\label{fig:simulation1205_randx_Graph4cf4i}
\end{mdframed}
\end{figure*}

\FloatBarrier

\subsection{Synthetic Lethality across Graph Structures}
\label{chap5:graphsim_str}

\FloatBarrier

%Graph 4 - RUN1109 / 1206 STR
%Graph4i?

\begin{figure*}[!ht]
\begin{mdframed}
     \begin{center}
      \resizebox{0.8 \textwidth}{!}{
         \includegraphics{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(4)".pdf}}
      \end{center}
      \caption[Performance is affected by inhibition in graphs]{\small \textbf{Performance is affected by inhibition in graphs.} The gene category (blue for query, cyan for query-correlated, red for SL, orange for SL-correlated, forest green for non-SL-correlated, and green for non-SL) ordered by $\chi^2$ signed by the \gls{SLIPT} directional condition is shown across simulations. For each of 1--10 SL partners, 10 simulations demonstrate that the increasing numbers of SL partners become harder detect. The $\chi^2$ values show a clear threshold for SL and correlated genes when there are fewer of them, distinguishable from correlated genes in this case.}
\end{mdframed}
\label{fig:simulation1206_Str_randx_Graph4D}
\end{figure*}

\begin{figure*}[!ht]
\begin{mdframed}
     \begin{center}
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1206_Str_randx_Graph4D:Graph4}
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4D".pdf}}
        }%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1206_Str_randx_Graph4D:vioplot}
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(4)".pdf}}
        }%
       \end{center}
      \caption[Performance is affected by inhibition in graphs]{\small \textbf{Performance is affected by inhibition in graphs.} The gene category (blue for query, cyan for query-correlated, red for SL, orange for SL-correlated, forest green for non-SL-correlated, and green for non-SL) ordered by $\chi^2$ signed by the \gls{SLIPT} directional condition is shown across simulations. For each of 1--10 SL partners, 10 simulations demonstrate that the increasing numbers of SL partners become harder detect. The $\chi^2$ values show a clear threshold for SL and correlated genes when there are fewer of them, distinguishable from correlated genes in this case.}
\end{mdframed}
\label{fig:simulation1206_Str_randx_Graph4D}
\end{figure*}

\begin{figure*}[!ht]
\begin{mdframed}
     \begin{center}
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1206_Str_randx_Graph4iD:Graph4i}
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4iD".pdf}}
        }%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1206_Str_randx_Graph4iD:vioplot}
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i_ROC_samples_SLstry_vioplot(4)".pdf}}
        }%
       \end{center}
      \caption[Performance is affected by inhibition in graphs]{\small \textbf{Performance is affected by inhibition in graphs.} The gene category (blue for query, cyan for query-correlated, red for SL, orange for SL-correlated, forest green for non-SL-correlated, and green for non-SL) ordered by $\chi^2$ signed by the \gls{SLIPT} directional condition is shown across simulations. For each of 1--10 SL partners, 10 simulations demonstrate that the increasing numbers of SL partners become harder detect. The $\chi^2$ values show a clear threshold for SL and correlated genes when there are fewer of them, distinguishable from correlated genes in this case.}
\end{mdframed}
\label{fig:simulation1206_Str_randx_Graph4iD}
\end{figure*}

\begin{figure*}[!ht]
\begin{mdframed}
     \begin{center}
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1206_Str_randx_Graph4iD:Graph4i}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4D".pdf}}
        }%
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1206_Str_randx_Graph4iD:Graph4i}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4iD".pdf}}
        }%
        
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1206_Str_randx_Graph4iD:vioplot}
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(4)".pdf}}
        }%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1206_Str_randx_Graph4iD:vioplot}
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i_ROC_samples_SLstry_vioplot(4)".pdf}}
        }%
       \end{center}
      \caption[Performance is affected by inhibition in graphs]{\small \textbf{Performance is affected by inhibition in graphs.} The gene category (blue for query, cyan for query-correlated, red for SL, orange for SL-correlated, forest green for non-SL-correlated, and green for non-SL) ordered by $\chi^2$ signed by the \gls{SLIPT} directional condition is shown across simulations. For each of 1--10 SL partners, 10 simulations demonstrate that the increasing numbers of SL partners become harder detect. The $\chi^2$ values show a clear threshold for SL and correlated genes when there are fewer of them, distinguishable from correlated genes in this case.}
\end{mdframed}
\label{fig:simulation1206_Str_randx_Graph4D2}
\end{figure*}

\FloatBarrier

\subsection{Performance with Feasible Gene Numbers (20,000)}
\label{chap5:graphsim_performance_20K}

\subsubsection{Simple Graph Structures in a Genome}
\label{chap5:graphsim_performance_20K_simple}

\FloatBarrier

\begin{figure*}[!htbp]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1207_randx_Graph1ROC:Perf}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph1_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1207_randx_Graph1ROC:ROC}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph1_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1207_randx_Graph1}
           \raisebox{0.1875\textwidth}{
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1".pdf}}
           }
        }%
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1207_randx_Graph1ROC:AUC}
           \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph1_AUC_samplesx".png}}
        }%
    \end{center}
   \caption[Performance of simulations including a simple graph]{\small \textbf{Performance of simulations including a simple graph.} Simulation of synthetic lethality was performed sampling from a multivariate normal distribution (without correlation structure). Performance of \gls{SLIPT} declines for more synthetic partners but this is mitigated by increased sample sizes (in darker colours). This generally occurs as the sensitivity decreases for a greater number of true positives to detect, leading to a trade off in accuracy as seen in a trough for false discovery rate and the ROC curves.}
%}
\label{fig:simulation1207_randx_Graph1}
\end{mdframed}
\end{figure*}

\begin{figure*}[!t]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph1cf20K:Graph1}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph1".pdf}}
        }%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph1cf20K:Graph2}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph2".pdf}}
        }%
       %\subcaptionbox{Graph Structure}{%
       %    \label{fig:simulation1205_randx_Graph1cf20K:Graph3i2}
       %    \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph3i2".pdf}}
       %}%

%
        \subcaptionbox{Gene category in simulations}{%
            \label{fig:simulation1205_randx_Graph1cf20K:Compare1}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph1_ROC_Compare(2)".pdf}}
        }%
        \subcaptionbox{Gene category in simulations}{%
            \label{fig:simulation1205_randx_Graph1cf20K:Compare2}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph2_ROC_Compare(2)".pdf}}
        }%

    \end{center}
   \caption[Performance on a simple graph improves with more genes]{\small \textbf{Performance on a simple graph improves with more genes.} The gene category (blue for query, cyan for query-correlated, red for SL, orange for SL-correlated, forest green for non-SL-correlated, and green for non-SL) ordered by $\chi^2$ signed by the \gls{SLIPT} directional condition is shown across simulations. For each of 1--10 SL partners, 10 simulations demonstrate that the increasing numbers of SL partners become harder detect. The $\chi^2$ values show a clear threshold for SL and correlated genes when there are fewer of them, distinguishable from correlated genes in this case.}
%}
\label{fig:simulation1205_randx_Graph1cf20K}
\end{mdframed}
\end{figure*}

\subsubsection{Constructed Graph Structures in a Genome}
\label{chap5:graphsim_performance_20K_constructed}

\begin{figure*}[!htbp]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1207_randx_Graph3ROC:Perf}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph3_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1207_randx_Graph3ROC:ROC}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph3_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1207_randx_Graph3}
           \raisebox{0.1875\textwidth}{
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph3".pdf}}
           }
        }%
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1207_randx_Graph3ROC:AUC}
           \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph3_AUC_samplesx".png}}
        }%
    \end{center}
   \caption[Performance of simulations including a graph structure]{\small \textbf{Performance of simulations including a graph structure.} Simulation of synthetic lethality was performed sampling from a multivariate normal distribution (without correlation structure). Performance of \gls{SLIPT} declines for more synthetic partners but this is mitigated by increased sample sizes (in darker colours). This generally occurs as the sensitivity decreases for a greater number of true positives to detect, leading to a trade off in accuracy as seen in a trough for false discovery rate and the ROC curves.}
%}
\label{fig:simulation1207_randx_Graph3}
\end{mdframed}
\end{figure*}

\begin{figure*}[!htbp]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1207_randx_Graph3iROC:Perf}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph3i_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1207_randx_Graph3iROC:ROC}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph3i_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1207_randx_Graph3i}
           \raisebox{0.1875\textwidth}{
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph3i".pdf}}
           }
        }%
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1207_randx_Graph3iROC:AUC}
           \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph3i_AUC_samplesx".png}}
        }%
    \end{center}
   \caption[Performance of simulations including an inhbiting graph]{\small \textbf{Performance of simulations including an inhbiting graph.} Simulation of synthetic lethality was performed sampling from a multivariate normal distribution (without correlation structure). Performance of \gls{SLIPT} declines for more synthetic partners but this is mitigated by increased sample sizes (in darker colours). This generally occurs as the sensitivity decreases for a greater number of true positives to detect, leading to a trade off in accuracy as seen in a trough for false discovery rate and the ROC curves.}
%}
\label{fig:simulation1207_randx_Graph3i}
\end{mdframed}
\end{figure*}

\begin{figure*}[!t]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph3cf20K:Graph3}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph3".pdf}}
        }%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph3cf20K:Graph3i}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph3i".pdf}}
        }%
       %\subcaptionbox{Graph Structure}{%
       %    \label{fig:simulation1205_randx_Graph3cf20K:Graph3i2}
       %    \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph3i2".pdf}}
       %}%

%
        \subcaptionbox{Gene category in simulations}{%
            \label{fig:simulation1205_randx_Graph3cf20K:Compare3}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph3_ROC_Compare(2)".pdf}}
        }%
        \subcaptionbox{Gene category in simulations}{%
            \label{fig:simulation1205_randx_Graph3cf20K:Compare3i}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph3_ROC_Compare(3)".pdf}}
        }%

    \end{center}
   \caption[Performance on an inhibiting graph improves with more genes]{\small \textbf{Performance on an inhibiting graph improves with more genes.} The gene category (blue for query, cyan for query-correlated, red for SL, orange for SL-correlated, forest green for non-SL-correlated, and green for non-SL) ordered by $\chi^2$ signed by the \gls{SLIPT} directional condition is shown across simulations. For each of 1--10 SL partners, 10 simulations demonstrate that the increasing numbers of SL partners become harder detect. The $\chi^2$ values show a clear threshold for SL and correlated genes when there are fewer of them, distinguishable from correlated genes in this case.}
%}
\label{fig:simulation1205_randx_Graph3cf20K}
\end{mdframed}
\end{figure*}

\begin{figure*}[!htbp]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1207_randx_Graph4ROC:Perf}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph4_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1207_randx_Graph4ROC:ROC}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph4_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1207_randx_Graph4}
           \raisebox{0.1875\textwidth}{
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4".pdf}}
           }
        }%
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1207_randx_Graph4ROC:AUC}
           \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph4_AUC_samplesx".png}}
        }%
    \end{center}
   \caption[Performance of simulations including a graph structure]{\small \textbf{Performance of simulations including a graph structure.} Simulation of synthetic lethality was performed sampling from a multivariate normal distribution (without correlation structure). Performance of \gls{SLIPT} declines for more synthetic partners but this is mitigated by increased sample sizes (in darker colours). This generally occurs as the sensitivity decreases for a greater number of true positives to detect, leading to a trade off in accuracy as seen in a trough for false discovery rate and the ROC curves.}
%}
\label{fig:simulation1207_randx_Graph4}
\end{mdframed}
\end{figure*}

\begin{figure*}[!htbp]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1207_randx_Graph4i2ROC:Perf}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph4i2_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1207_randx_Graph4i2ROC:ROC}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph4i2_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1207_randx_Graph4i2}
           \raisebox{0.1875\textwidth}{
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4i2".pdf}}
           }
        }%
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1207_randx_Graph4i2ROC:AUC}
           \includegraphics[width=0.6\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph4i2_AUC_samplesx".png}}
        }%
    \end{center}
   \caption[Performance of simulations including an inhibiting graph]{\small \textbf{Performance of simulations including an inhibiting graph.} Simulation of synthetic lethality was performed sampling from a multivariate normal distribution (without correlation structure). Performance of \gls{SLIPT} declines for more synthetic partners but this is mitigated by increased sample sizes (in darker colours). This generally occurs as the sensitivity decreases for a greater number of true positives to detect, leading to a trade off in accuracy as seen in a trough for false discovery rate and the ROC curves.}
%}
\label{fig:simulation1207_randx_Graph4i2}
\end{mdframed}
\end{figure*}

\begin{figure*}[!t]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph4cf20K:Graph4}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4".pdf}}
        }%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph4cf20K:Graph4i2}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4i2".pdf}}
        }%
       %\subcaptionbox{Graph Structure}{%
       %    \label{fig:simulation1205_randx_Graph4cf20K:Graph4i2}
       %    \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph4i2".pdf}}
       %}%

%
        \subcaptionbox{Gene category in simulations}{%
            \label{fig:simulation1205_randx_Graph4cf20K:Compare4}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph4_ROC_Compare(2)".pdf}}
        }%
        \subcaptionbox{Gene category in simulations}{%
            \label{fig:simulation1205_randx_Graph4cf20K:Compare4i2}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph4_ROC_Compare(5)".pdf}}
        }%

    \end{center}
   \caption[Performance on an inhibiting graph improves with more genes]{\small \textbf{Performance on an inhibiting graph improves with more genes.} The gene category (blue for query, cyan for query-correlated, red for SL, orange for SL-correlated, forest green for non-SL-correlated, and green for non-SL) ordered by $\chi^2$ signed by the \gls{SLIPT} directional condition is shown across simulations. For each of 1--10 SL partners, 10 simulations demonstrate that the increasing numbers of SL partners become harder detect. The $\chi^2$ values show a clear threshold for SL and correlated genes when there are fewer of them, distinguishable from correlated genes in this case.}
%}
\label{fig:simulation1205_randx_Graph4cf20K}
\end{mdframed}
\end{figure*}

\FloatBarrier

\section{Simulations over pathway-based graphs}

\FloatBarrier

%PI3K and GAI

Text

\begin{figure*}[!htbp]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1205_randx_Graph_pi3kROC:Perf}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_pi3k_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1205_randx_Graph_pi3kROC:ROC}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_pi3k_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graph_pi3k}
           %\raisebox{0.475\textwidth}{
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph_pi3k".pdf}}
           %}
        }%
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1205_randx_Graph_pi3kROC:AUC}
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_pi3k_AUC_samplesx_prop".png}}
        }%
    \end{center}
   \caption[Performance of simulations on the PI3K cascade]{\small \textbf{Performance of simulations on the PI3K cascade.} Simulation of synthetic lethality was performed sampling from a multivariate normal distribution (without correlation structure). Performance of \gls{SLIPT} declines for more synthetic partners but this is mitigated by increased sample sizes (in darker colours). This generally occurs as the sensitivity decreases for a greater number of true positives to detect, leading to a trade off in accuracy as seen in a trough for false discovery rate and the ROC curves.}
%}
\label{fig:simulation1205_randx_Graph_pi3k}
\end{mdframed}
\end{figure*}

\begin{figure*}[!htbp]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
        \subcaptionbox{Statistical evaluation}{%
            \label{fig:simulation1207_randx_Graph_pi3kROC:Perf}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_pi3k_ROC1_samplesx".png}}
        }%
        \subcaptionbox{Receiver operating characteristic}{%
            \label{fig:simulation1207_randx_Graph_pi3kROC:ROC}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_pi3k_ROC2_samplesx".png}}
        }%
        
        \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1207_randx_Graph_pi3k}
           %\raisebox{0.475\textwidth}{
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph_pi3k".pdf}}
           %}
        }%
        \subcaptionbox{Statistical performance}{%
           \label{fig:simulation1207_randx_Graph_pi3kROC:AUC}
           \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_pi3k_AUC_samplesx_prop".png}}
        }%
    \end{center}
   \caption[Performance of simulations including the PI3K cascade]{\small \textbf{Performance of simulations including the PI3K cascade.} Simulation of synthetic lethality was performed sampling from a multivariate normal distribution (without correlation structure). Performance of \gls{SLIPT} declines for more synthetic partners but this is mitigated by increased sample sizes (in darker colours). This generally occurs as the sensitivity decreases for a greater number of true positives to detect, leading to a trade off in accuracy as seen in a trough for false discovery rate and the ROC curves.}
%}
\label{fig:simulation1207_randx_Graph_pi3k}
\end{mdframed}
\end{figure*}

\subsection{Pathway Structures in a Genome}

\FloatBarrier

\begin{figure*}[!thb]
\begin{mdframed}
%  \resizebox{\textwidth}{!}{
         \begin{center}
%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graphpwaycf20K:Graph_pi3k}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph_pi3k".pdf}}
        }%
       \subcaptionbox{Graph Structure}{%
           \label{fig:simulation1205_randx_Graphpwaycf20K:Graph_Gai}
           \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph_Gai".pdf}}
        }%
       %\subcaptionbox{Graph Structure}{%
       %    \label{fig:simulation1205_randx_Graphpwaycf20K:Graph3i2}
       %    \includegraphics[width=0.3\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/Graph3i2".pdf}}
       %}%

%
        \subcaptionbox{Gene category in simulations}{%
            \label{fig:simulation1205_randx_Graphpwaycf20K:Compare1}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_pi3k_ROC_Compare(2)".pdf}}
        }%
        \subcaptionbox{Gene category in simulations}{%
            \label{fig:simulation1205_randx_Graphpwaycf20K:Compare2}
            \includegraphics[width=0.475\textwidth]{{"/home/tomkelly/Documents/PhD Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Gai_ROC_Compare(2)".pdf}}
        }%

    \end{center}
   \caption[Performance on pathways improves with more genes]{\small \textbf{Performance on pathways improves with more genes.} The gene category (blue for query, cyan for query-correlated, red for SL, orange for SL-correlated, forest green for non-SL-correlated, and green for non-SL) ordered by $\chi^2$ signed by the \gls{SLIPT} directional condition is shown across simulations. For each of 1--10 SL partners, 10 simulations demonstrate that the increasing numbers of SL partners become harder detect. The $\chi^2$ values show a clear threshold for SL and correlated genes when there are fewer of them, distinguishable from correlated genes in this case.}
%}
\label{fig:simulation1205_randx_Graphpwaycf20K}
\end{mdframed}
\end{figure*}

\FloatBarrier

\section{Discussion}

Text


\section{Summary}

Text


\clearpage

\paragraph{Aims}

  \begin{itemize}
   \item A Model of Synthetic Lethal Genes in Gene Expression Data
   
   \bigskip
   
   \item Comparison of SLIPT to Alternative Approaches
   
   \bigskip
   
   \item Simulations of Known Synthetic Lethal Genes within Pathway Networks
      
  \end{itemize}

\paragraph{Summary}

    \begin{itemize}
      \item We have designed a straight-forward rational query-based synthetic lethal detection method with the example of application to \textit{CDH1} in cancer gene expression
      
      \bigskip
      
      \item I have developed a simulation pipeline to generate continuous gene expression with pathway structure including a procedure to simulate synthetic lethality 
      
      \bigskip
      
      \item The simulation procedure shows that SLIPT is robust across pathway structures and has desirable performance compared to other statistical techniques 
      \end{itemize}