%\chapter{Conclusion}
%\label{chap:conclusion}
\clearpage
\section{Conclusions}
\label{chap:conclusion}

Synthetic lethal interactions are important for understanding gene function and development of highly specific, \glslink{targeted therapy}{targeted} cancer \glspl{treatment}. In particular, \glspl{synthetic lethal} could expand the repertoire of applications for precision cancer medicine by indirectly targeting loss of function in \gls{tumour suppressor} genes.  However, \gls{synthetic lethal} discovery with experimental screening is error prone and limited by the model systems in which it is performed. Thus there is a need for \gls{bioinformatics} tool to predict \gls{synthetic lethal} interactions from \gls{gene expression} data, which would facilitate the rapid identification of \gls{synthetic lethal} candidates, and augment functional genetic screens and triage of cancer drug targets. This thesis develops the \acrfull{SLIPT} methodology as a statically robust procedure perform this analysis.

The \gls{SLIPT} methodology has been demonstrated to identify biologically relevant genes and pathways. An comprehensive analysis of \gls{synthetic lethal} partners of the \textit{CDH1} gene was performed in \gls{TCGA} breast cancer data \citep{TCGA2012} with many of these findings replicated in stomach cancer data \citep{TCGA2014GC}. These genes clustered into several distinct groups, with distinct biological functions and elevated \glslink{gene expression}{expression} in different clinical subtypes.  These analyses identified \gls{synthetic lethal} candidates in the $G_{\alpha i}$ signalling, cytoplasmic microfibres, and extracellular fibrin clotting pathways which were supported by an \gls{siRNA} screen performed by \citet{Telford2015} and were consistent with the known cytoskeletal and cell signalling roles of \gls{E-cadherin}. 
%These findings support interventions against these pathways being applicable to specific cancer therapeutics beyond the pre-clinical cell line models in which they were validated. 
\gls{SLIPT} also identified \gls{synthetic lethal} partners in novel pathways for \textit{CDH1}, including the regulation of immune signalling and translational elongation, which extend the range of established functions of \textit{CDH1} and present further biological mechanisms %to investigate the malignancy and
taht can be investigated to exploit the
vulnerabilities of \textit{CDH1}-deficient cancers.

While some of these pathways are not expected to be detected in an isolated experimental cell line model, \glslink{graph}{pathway} structure may have accounted for this disparity. Thus \gls{synthetic lethal} candidates detected by \gls{SLIPT} and \gls{siRNA} were compared within \glslink{graph}{graph} structures of the candidate \gls{synthetic lethal} pathways. However, this did not generally account for differences between these approaches. Neither \gls{synthetic lethal} detection methodology preferentially detected genes of more importance or connectivity in \glslink{graph}{pathway} structures using established network metrics, nor could it be generally established that \gls{SLIPT} gene candidates were upstream or downstream of \gls{siRNA} gene candidates in \glslink{graph}{pathway} structures across biological pathways.

Pathway \glslink{graph}{graph} structures were also included in investigations with simulated data to ascertain whether the \gls{SLIPT} procedure performed desirably in data with complex correlation structures derived based on biological pathways. A simulation procedure was developed based on a statistical model of \glspl{synthetic lethal} which generates multivariate normal data with known \gls{synthetic lethal} partners and correlation structures. The \gls{SLIPT} methodology had high statistical performance, particularly when detecting few known \gls{synthetic lethal} genes, with large sample sizes, and a background of many non \gls{synthetic lethal} genes to distinguish true partners from. This method had high specificity, performed better than Pearson correlation or the $\chi^2$-test, and had had optimal performance across simulation parameter combinations for the thresholds used throughout this thesis. These findings were robust across correlation structures, including those derived from complex \glslink{graph}{pathway} structures containing strong positive and negative correlations between genes. 
Together, these findings support the release of the \gls{SLIPT} software R packages and the application of the method to identify \gls{synthetic lethal} genes within pathways and use candidate \gls{synthetic lethal} genes to identify \gls{synthetic lethal} pathways, as demonstrated in this thesis.

%In summary, I present a widely applicable \gls{synthetic lethal} procedure using \gls{gene expression} data for wider use in \glspl{genomic} research, including the development of precision cancer medicine. This methodology is supported by the release of a software package in R, simulation results based on a statistical model of \glspl{synthetic lethal}, the demonstration of \gls{bioinformatics} and network biology investigations into interactions with the \textit{CDH1} gene in breast and stomach cancers. 


\iffalse
\clearpage
\paragraph{Aims}

  \begin{itemize}
   \item To develop a statistical approach to detect \gls{synthetic lethal} gene pairs in cancer from \glslink{gene expression}{expression} data

   \bigskip
   
   \item To apply this methodology to public cancer \gls{gene expression} data against \textit{CDH1} and analyse \glslink{graph}{pathway} structure with comparisons to experimental screen data

   \bigskip
   
   \item To construct a statistical model of \glspl{synthetic lethal} in multivariate normal \glslink{gene expression}{expression} data
 
   \bigskip
   
   \item To develop a simulation pipeline of \glslink{gene expression}{expression} with \glslink{graph}{pathway} structure on a high-performance computing cluster 

   \bigskip
   
   \item To examine the statistical performance of the methodology with simulated \glslink{gene expression}{expression} including pathways and compare it to other approaches

   \bigskip
   
   \item To release the \gls{synthetic lethal} detection methodology and pathway simulation procedure as R software packages
   
  \end{itemize}
  

\clearpage
  
 \paragraph{Summary}
 
   \begin{itemize}
   \item We have developed a Synthetic Lethal detection method that generates a high number of \gls{synthetic lethal} candidates
   
   \bigskip
   
   \item Pathways in cell signalling, extracellular matrix, and cytoskeletal functions were supported with experimental candidates and the known functions of \gls{E-cadherin}
   
   \bigskip
   
   \item Several candidate pathways were supported by \gls{mutation} analysis and replicated across breast and stomach cancer
   
   \bigskip
   
   \item Translation and immune functions were uniquely detected by the computational approach which may be explained by differences between patient samples and cell line models
   
   \bigskip
   
   \item There remains the need to identify actionable genes within these pathways, relationships with experimental candidates, and how these pathways may affect viability when lost
  \end{itemize}
  
    \begin{itemize}
   \item Synthetic Lethal genes were explored within a \glslink{graph}{graph} structures for key pathways identified previously 
   
   \bigskip
   
   \item In some cases these \glslink{graph}{graph} structures appeared to have relationships between \gls{synthetic lethal} genes  
   
   \bigskip
   
   \item However, no existing network metrics of importance and connectivity with the networks were elevated significantly for Synthetic Lethal genes
   
   \bigskip
   
   \item Nor was there significant evidence of upstream and downstream relationships between SLIPT and \gls{siRNA} Candidates in a \gls{shortest path} permutation analysis
  \end{itemize}
  
      \begin{itemize}
      \item We have designed a straight-forward rational query-based \gls{synthetic lethal} detection method with the example of application to \textit{CDH1} in cancer \gls{gene expression}
      
      \bigskip
      
      \item We have developed a simulation pipeline to generate continuous \gls{gene expression} with \glslink{graph}{pathway} structure including a procedure to simulate \glspl{synthetic lethal} 
      
      \bigskip
      
      \item Our simulation procedure is robust across \glslink{graph}{pathway} structures and has desirable performance compared to other statistical techniques 
      \end{itemize}
 \fi