<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
</head>
<body>
<?xml version="1.0" encoding="utf-8" ?> <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" 
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><br /><!--http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd--><br />
<html xmlns="http://www.w3.org/1999/xhtml"  
> 
<head><title>
A Bioinformatics Approach to Synthetic Lethal Interactions in Breast Cancer with Gene Expression Data
</title> 
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" /> <meta name="generator" content="TeX4ht (http://www.cse.ohio-state.edu/~gurari/TeX4ht/)" /> <meta name="originator" content="TeX4ht (http://www.cse.ohio-state.edu/~gurari/TeX4ht/)" /> <!-- html,xhtml --> <meta name="src" content="thesis.tex" /> <meta name="date" content="2017-06-27 13:44:00" /> <link rel="stylesheet" type="text/css" href="thesis.css" />
</head><body 
>
<div class="center" 
>
<!--l. 170--><p class="noindent" >
</p><!--l. 170--><p class="noindent" >
<span class="cmbx-12">Library Declaration Form</span><br /> <img 
src="logo.png" alt="pict"  
 /> <br /> University of Otago Library
</p></div>
<table  
cellpadding="0" border="0" cellspacing="0" 
class="tabbing"><tr  
style="vertical-align:baseline;" class="tabbing"><td  
class="tabbing">
</td></tr></table>
<!--l. 170--><p class="noindent" >                          </p><table  
cellpadding="0" border="0" cellspacing="0" 
class="tabbing"><tr  
style="vertical-align:baseline;" class="tabbing"><td  
class="tabbing">
Author’s full name and year of birth: Simon Thomas Kelly,
</td></tr></table>
<!--l. 170--><p class="noindent" >                              </p><table  
cellpadding="0" border="0" cellspacing="0" 
class="tabbing"><tr  
style="vertical-align:baseline;" class="tabbing"><td style="width:292;" 
class="tabbing">
   (for cataloguing purposes)
</td><td  
class="tabbing">
24 February 1992
</td></tr></table>
<!--l. 170--><p class="noindent" >                                                                                    </p><table  
cellpadding="0" border="0" cellspacing="0" 
class="tabbing"><tr  
style="vertical-align:baseline;" class="tabbing"><td  
class="tabbing"></td></tr></table>
<!--l. 170--><p class="noindent" >
</p><!--l. 170--><p class="noindent" >
Title of thesis: A Bioinformatics Approach to Synthetic Lethal Interactions in Breast Cancer with Gene Expression Data<br 
class="newline" />
</p><!--l. 170--><p class="noindent" >
Degree: Doctor of Philosophy<br 
class="newline" />
</p><!--l. 170--><p class="noindent" >
Department: Department of Biochemistry<br 
class="newline" />
</p><!--l. 170--><p class="noindent" >
Permanent Address: 710 Cumberland Street, Dunedin, NZ<br 
class="newline" />
</p><!--l. 170--><p class="noindent" >
I agree that this thesis may be consulted for research and study purposes and that reasonable quotation may be made from it, provided that proper acknowledgement of its use is made.<br 
class="newline" />
</p><!--l. 170--><p class="noindent" >
I consent to this thesis being copied in part or in whole for
</p><dl class="list1"><dt class="list">
i)
</dt><dd 
class="list">
<!--l. 170--><p class="noindent" >
a library
</p></dd><dt class="list">
ii)
</dt><dd 
class="list">
<!--l. 170--><p class="noindent" >
an individual
</p></dd></dl>
<!--l. 170--><p class="noindent" >
at the discretion of the University of Otago.<br 
class="newline" />
</p><!--l. 170--><p class="noindent" >
      Signature:<br 
class="newline" />
</p><!--l. 170--><p class="noindent" >
<p>      Date:</p>
</p>
<div class="maketitle">
                                                                                    


<h2 class="titleHead">
A Bioinformatics Approach to Synthetic Lethal Interactions in Breast Cancer with Gene Expression Data
</h2>
<div class="author" >
<span class="cmr-12x-x-120">S. Thomas Kelly</span>
</div>
a thesis submitted for the degree of <span class="cmr-12x-x-120">Doctor of Philosophy</span> at the University of Otago, Dunedin, New Zealand.
<div class="date">
<p>27 June 2017</p>
</div>
</div><div 
class="abstract" 
>
<div class="center" 
>
<!--l. 170--><p class="noindent" >
</p><!--l. 170--><p class="noindent" >
<span class="cmbx-12">Abstract</span>
</p></div>
<div class="quote">
<!--l. 2--><p class="noindent" >
<strong><span class="cmbx-12">Background</span></strong>
</p><!--l. 4--><p class="noindent" >
Synthetic lethal genetic interactions are re-emerging in the post-genomics era due to their potential for use in precision medicine against cancers. Synthetic lethal drug design exploits the functional redundancy of genes disrupted in cancers (including tumour suppressors) to develop specific treatments against them. <span class="cmti-12">CDH1</span>, which encodes <a 
href="#glo:E-cadherin">E-cadherin</a>, is a tumour supressor gene with loss of function in breast and stomach cancers. Experimental screens have identified candidate synthetic lethal interactions, which can be further supported with bioinformatics analysis. Furthermore, gene expression data enables investigation of synthetic lethal pathways and graph structure of synthetic lethal genes within them.
</p><!--l. 7--><p class="noindent" >
<strong><span class="cmbx-12">Approach</span></strong>
</p><!--l. 9--><p class="noindent" >
<p>A computational methodology, the Synthetic Lethal Prediction Tool (<a 
href="#glo:SLIPT">SLIPT</a>) has been developed to detect synthetic lethal interactions in gene expression data. This methodology was demonstrated on interactions with <span class="cmti-12">CDH1 </span>in breast and stomach cancer data from The Cancer Genome Atlas (<a 
href="#glo:TCGA">TCGA</a>) project. Synthetic lethal genes and pathways were further investigated with unsupervised clustering, gene set over-representation analysis, metagenes, and permutation resampling. In particular, analyses focused on comparing <a 
href="#glo:SLIPT">SLIPT</a> gene candidates to an experimental <a 
href="#glo:siRNA">siRNA</a> screen <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>). Network analysis methods were applied to</p>
the most supported pathways to test for pathway structure between synthetic lethal candidates. Simulation and modelling was used to assess the statistical performance of <a 
href="#glo:SLIPT">SLIPT</a>, including simulated data with correlation structures derived from graph structures.
</p><!--l. 12--><p class="noindent" >
<strong><span class="cmbx-12">Findings</span></strong>
</p><!--l. 15--><p class="noindent" >
Many candidate synthetic lethal partners of <span class="cmti-12">CDH1 </span>were detected in <a 
href="#glo:TCGA">TCGA</a> breast cancer. These genes clustered into several distinct groups, with distinct biological functions and elevated expression in different clinical subtypes. While the number of genes detected by both <a 
href="#glo:SLIPT">Synthetic Lethal Interaction Prediction Tool (SLIPT)</a> and <a 
href="#glo:siRNA">short interfering RNA (siRNA)</a> was not significant, these contained significantly enriched pathways. In particular, <span class="cmmi-12">G</span><sub><span class="cmmi-8">αi</span></sub> signalling, cytoplasmic microfibres, and extracellular fibrin clotting were robustly supported by both approaches, which is consistent with the known cytoskeletal and cell signalling roles of <a 
href="#glo:E-cadherin">E-cadherin</a>. Many of these pathways were replicated in stomach cancer data. The pathways supported only by <a 
href="#glo:SLIPT">SLIPT</a> included regulation of immune signalling and translation, which were not expected to be detected in an isogenic cell line model but are still candidates for further investigation.
</p><!--l. 18--><p class="noindent" >
Synthetic lethal candidates detected by <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> were compared within the graph structures of the candidate synthetic lethal pathways. There was little support, across pathways, that <a 
href="#glo:SLIPT">SLIPT</a> gene candidates were consistently upstream or downstream of <a 
href="#glo:siRNA">siRNA</a> gene candidates within pathways. However, <a 
href="#glo:SLIPT">SLIPT</a> genes had lower centrality and were upstream of <a 
href="#glo:siRNA">siRNA</a> candidates, specifically in the <span class="cmmi-12">G</span><sub><span class="cmmi-8">αi</span></sub> signalling pathway.
</p><!--l. 24--><p class="noindent" >
<p>A statistical model of synthetic lethality was used to simulate gene expression data with known synthetic lethal partners for a gene. The <a 
href="#glo:SLIPT">SLIPT</a> methodology had high statistical performance when detecting few synthetic lethal partners, which diminished with more synthetic lethal partners or lower sample size. The <a 
href="#glo:SLIPT">SLIPT</a> methodology performed better than Pearson correlation or the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>-test. In particular, it performed well with high specificity for datasets containing thousands of genes or genes positively correlated with the query gene (as expected to occur in expression data). <a 
href="#glo:SLIPT">SLIPT</a> was robust across correlation structures, including those derived from complex pathway structures and often distinguished synthetic lethal</p>
<p>genes from those positively or negatively correlated with them. Therefore <a 
href="#glo:SLIPT">SLIPT</a> is appropriate to identify synthetic lethal genes within pathways and use candidate synthetic lethal genes (and their correlates) to identify synthetic lethal pathways.</p>
</p><!--l. 28--><p class="noindent" >
<strong><span class="cmbx-12">Summary</span></strong>
</p><!--l. 30--><p class="noindent" >
Thus this thesis has developed, evaluated, and refined a bioinformatics approach to discovery of synthetic lethal genes solely from gene expression data. This approach has been demonstrated to detect biologically informative and clinically relevant candidate synthetic lethal partners for <span class="cmti-12">CDH1 </span>in breast and stomach cancers. These investigations have also involved the development of network analysis and simulation procedures which may be more widely applicable.
</p>
</div>
</div>
                                                                                    

<div class="center" 
>
<!--l. 170--><p class="noindent" >
</p><!--l. 170--><p class="noindent" >
<span class="cmbx-12">Acknowledgements</span>
</p></div>
<div class="quote">
<!--l. 1--><p class="noindent" >
I thank my supervisors A/Prof. Mik Black and Prof. Parry Guilford for their support and guidance throughout this my postgraduate studies. It has been a great experience, I look forward to seeing what your research groups produce in the future, may this not be the end for us.
</p><!--l. 3--><p class="noindent" >
I am also thankful for the guidance and mentorship of Prof. Hamish Spencer for career advice throughout my studies and time in his research group.
</p><!--l. 5--><p class="noindent" >
I am also grateful to the past and current members of these research groups, and my peers at the laboratory benches and computers across campus. The peer support, camaraderie, and guidance of newer students has been an incredible part of my time at Otago and has made my thesis studies not just easier but possible at all. The postgraduate community is very special here and I have truly made some lifelong friends from all over the world. You are talented researchers and amazing people. May we meet again some day. Where-ever you may end up, its small world and there’s always time to catch up. I’d be delighted to host some visits while working abroad.
</p><!--l. 7--><p class="noindent" >
<p>I cannot thank my friends, flatmates, family, and diligent proofreaders enough for their patience and support during such as massive, challenging, and (I’m sure you’ve heard too often) stressful undertaking during both my PhD and the study leading up to it. There are too many of you to name everyone here without leaving someone out, so thank you all for everything you’ve done, both the good times and the tough. Thank you</p>
for at least pretending to understand complex math oft brought up at the wrong moment. Thank you for checking my writing or slides, even when sprung on you last minute. Thank for your time when what I really needed was a chat, a walk, a drink with “the guys”, or a moment to think clearly.
</p><!--l. 9--><p class="noindent" >
I thank the various organisations that supported this research project:
</p>
<ul class="itemize1">
<li class="itemize">
<span class="cmr-10x-x-109">This thesis was supported by the Postgraduate Tassell Scholarship in</span> <span class="cmr-10x-x-109">Cancer Research, a University of Otago Doctoral Scholarship.</span>
</li>
<li class="itemize">
<span class="cmr-10x-x-109">The New Zealand eScience Infrastucture (NeSI) provided access to the</span> <span class="cmr-10x-x-109">Intel Pan high-performance computing cluster, support, and training to use</span> <span class="cmr-10x-x-109">it effectively. Various aspects of this thesis would not have been possible</span> <span class="cmr-10x-x-109">without access to such an incredible national resource.</span>
</li>
<li class="itemize">
<span class="cmr-10x-x-109">The Health Research Council (HRC) of New Zealand provided funding for</span> <span class="cmr-10x-x-109">experimental research in the Cancer Genetics Laboratory. Some aspects of</span> <span class="cmr-10x-x-109">this project would not have been possible without access to the data and</span> <span class="cmr-10x-x-109">findings funded by this grant.</span>
</li>
<li class="itemize">
<span class="cmr-10x-x-109">The Allan Wilson Centre and Otago School of Biomedical Sciences provided</span> <span class="cmr-10x-x-109">funding for summer research placements which was a valuable opportunity</span> <span class="cmr-10x-x-109">to gain experience and training used in this thesis project.</span>
</li></ul>
<!--l. 31--><p class="noindent" >
I thank the following organisations for support towards presenting findings in this thesis at conference and seminars:
</p>
<ul class="itemize1">
<li class="itemize">
<span class="cmr-10x-x-109">Google (eResearch 2014, Hamilton)</span>
</li>
<li class="itemize">
<span class="cmr-10x-x-109">NeSI (Software Carpentry training and Research Bazaar 2015, Melbourne)</span>
</li>
<li class="itemize">
<p><span class="cmr-10x-x-109">REANNZ, NZGL, and NeSI (eResearch 2016, Queenstown)</span></p>
</li>
<li class="itemize">
<span class="cmr-10x-x-109">Otago Division of Health Sciences, Oxford Global, and Maurice and Phyllis</span> <span class="cmr-10x-x-109">Paykel Trust (NGS Asia 2016, Singapore)</span>
</li>
<li class="itemize">
<span class="cmr-10x-x-109">RIKEN Division of Genomic Technologies and the Okinawa Institute of</span> <span class="cmr-10x-x-109">Science and Technology (seminar visits in Japan)</span>
</li></ul>
<!--l. 56--><p class="noindent" >
Thanks most of all to my fianceé, Dr Yui Kawagishi, you’ve been an inspiration. Thank you for your support and encouragement, every day, even from afar: it has always made a difference. It’s been incredible to see you flourish in your career and I look forward to joining you again soon. May the next chapter of our adventures involve a bit less Skype across timezones.
</p><!--l. 58--><p class="noindent" >
<span class="udgj-30x-x-120">jG C MiG</span><span class="udgj-75x-x-120">/</span><span class="udgj-30x-x-120">b   </span><span class="udgj-98x-x-120"> </span><span class="udgj5f-x-x-120">5</span><span class="udgj-30x-x-120">d<code>&lt;/span&gt;!&lt;span  class=&quot;udgj-30x-x-120&quot;&gt; GZQ&lt;/span&gt;&lt;span  class=&quot;udgj-88x-x-120&quot;&gt;H&lt;/span&gt;&lt;span  class=&quot;udgj-30x-x-120&quot;&gt;NZ &lt;/span&gt;!&lt;span  class=&quot;udgj-30x-x-120&quot;&gt;</code></span><span class="udgj-67x-x-120">f</span> <span class="udgj-67x-x-120"> </span><span class="udgjff-x-x-120"> </span>
</p></div>
                                                                                    

<h2 class="likechapterHead">
<a 
 id="x1-1000"></a>Contents
</h2> <div class="tableofcontents">
<p><span class="chapterToc"><a 
href="#Q1-1-4">Glossary</a></span> <br /><span class="chapterToc"><a 
href="#Q1-1-6">Acronyms</a></span> <br /><span class="chapterToc">1 <a 
href="#x1-60001" id="QQ2-1-8">Introduction and Literature Review</a></span> <br /> <span class="sectionToc">1.1 <a 
href="#x1-70001" id="QQ2-1-9">Cancer Research in the Post-Genomic Era</a></span> <br />  <span class="subsectionToc">1.1.1 <a 
href="#x1-80001" id="QQ2-1-10">Cancer is a Global Health Issue</a></span> <br />   <span class="subsubsectionToc">1.1.1.1 <a 
href="#x1-90001" id="QQ2-1-11">The Genetics and Molecular Biology of Cancers</a></span> <br />  <span class="subsectionToc">1.1.2 <a 
href="#x1-100002" id="QQ2-1-12">The Genomics Revolution in Cancer Research</a></span> <br />   <span class="subsubsectionToc">1.1.2.1 <a 
href="#x1-110001" id="QQ2-1-13">High-Throughput Technologies</a></span> <br />   <span class="subsubsectionToc">1.1.2.2 <a 
href="#x1-120002" id="QQ2-1-14">Bioinformatics and Genomic Data</a></span> <br />  <span class="subsectionToc">1.1.3 <a 
href="#x1-130003" id="QQ2-1-15">Genomics Projects</a></span> <br />   <span class="subsubsectionToc">1.1.3.1 <a 
href="#x1-140001" id="QQ2-1-16">The Cancer Genome Project</a></span> <br />   <span class="subsubsectionToc">1.1.3.2 <a 
href="#x1-150002" id="QQ2-1-17">The Cancer Genome Atlas Project</a></span> <br />  <span class="subsectionToc">1.1.4 <a 
href="#x1-160004" id="QQ2-1-18">Genomic Cancer Medicine</a></span> <br />   <span class="subsubsectionToc">1.1.4.1 <a 
href="#x1-170001" id="QQ2-1-19">Cancer Genes and Driver Mutations</a></span> <br />   <span class="subsubsectionToc">1.1.4.2 <a 
href="#x1-180002" id="QQ2-1-20">Precision Cancer Medicine</a></span> <br />   <span class="subsubsectionToc">1.1.4.3 <a 
href="#x1-190003" id="QQ2-1-21">Molecular Diagnostics and Pan-Cancer Medicine</a></span> <br />   <span class="subsubsectionToc">1.1.4.4 <a 
href="#x1-200004" id="QQ2-1-22">Targeted Therapeutics and Pharmacogenomics</a></span> <br />  <span class="subsectionToc">1.1.5 <a 
href="#x1-210005" id="QQ2-1-23">Systems and Network Biology</a></span> <br /> <span class="sectionToc">1.2 <a 
href="#x1-220002" id="QQ2-1-24">Synthetic Lethal Cancer Medicine</a></span> <br />  <span class="subsectionToc">1.2.1 <a 
href="#x1-230001" id="QQ2-1-25">Synthetic Lethal Genetic Interactions</a></span> <br />  <span class="subsectionToc">1.2.2 <a 
href="#x1-240002" id="QQ2-1-27">Synthetic Lethal Concepts in Genetics</a></span> <br />  <span class="subsectionToc">1.2.3 <a 
href="#x1-250003" id="QQ2-1-28">Synthetic Lethality in Model Systems</a></span> <br />   <span class="subsubsectionToc">1.2.3.1 <a 
href="#x1-260001" id="QQ2-1-29">Synthetic Lethal Pathways and Networks</a></span> <br />   <span class="subsubsectionToc">1.2.3.2 <a 
href="#x1-270002" id="QQ2-1-30">Evolution of Synthetic Lethality</a></span> <br />  <span class="subsectionToc">1.2.4 <a 
href="#x1-280004" id="QQ2-1-31">Synthetic Lethality in Cancer</a></span> <br />  <span class="subsectionToc">1.2.5 <a 
href="#x1-290005" id="QQ2-1-33">Clinical Impact of Synthetic Lethality in Cancer</a></span> <br />  <span class="subsectionToc">1.2.6 <a 
href="#x1-300006" id="QQ2-1-34">High-throughput Screening for Synthetic Lethality</a></span> <br />   <span class="subsubsectionToc">1.2.6.1 <a 
href="#x1-310001" id="QQ2-1-35">Synthetic Lethal Screens</a></span> <br />  <span class="subsectionToc">1.2.7 <a 
href="#x1-320007" id="QQ2-1-36">Computational Prediction of Synthetic Lethality</a></span> <br />   <span class="subsubsectionToc">1.2.7.1 <a 
href="#x1-330001" id="QQ2-1-37">Bioinformatics Approaches to Genetic Interactions</a></span> <br />   <span class="subsubsectionToc">1.2.7.2 <a 
href="#x1-340002" id="QQ2-1-40">Comparative Genomics</a></span> <br />   <span class="subsubsectionToc">1.2.7.3 <a 
href="#x1-350003" id="QQ2-1-42">Analysis and Modelling of Protein Data</a></span> <br />   <span class="subsubsectionToc">1.2.7.4 <a 
href="#x1-360004" id="QQ2-1-43">Differential Gene Expression</a></span> <br />   <span class="subsubsectionToc">1.2.7.5 <a 
href="#x1-370005" id="QQ2-1-44">Data Mining and Machine Learning</a></span> <br />   <span class="subsubsectionToc">1.2.7.6 <a 
href="#x1-380006" id="QQ2-1-45">Mutual Exclusivity and Bimodality</a></span> <br />   <span class="subsubsectionToc">1.2.7.7 <a 
href="#x1-390007" id="QQ2-1-46">Rationale for Further Development</a></span> <br /> <span class="sectionToc">1.3 <a 
href="#x1-400003" id="QQ2-1-47">E-cadherin as a Synthetic Lethal Target</a></span></p>
<p><br />  <span class="subsectionToc">1.3.1 <a 
href="#x1-410001" id="QQ2-1-48">The <span class="cmti-12">CDH1 </span>gene and its Biological Functions</a></span> <br />  <span class="subsectionToc">1.3.2 <a 
href="#x1-420002" id="QQ2-1-49">Hereditary Diffuse Gastric (and Lobular Breast) Cancer</a></span> <br />  <span class="subsectionToc">1.3.3 <a 
href="#x1-430003" id="QQ2-1-50">Cell Line Models of <span class="cmti-12">CDH1 </span>Null Mutations</a></span> <br /> <span class="sectionToc">1.4 <a 
href="#x1-440004" id="QQ2-1-51">Summary and Research Direction of Thesis</a></span> <br />  <span class="subsectionToc">1.4.1 <a 
href="#x1-450001" id="QQ2-1-52">Thesis Aims</a></span> <br /><span class="chapterToc">2 <a 
href="#x1-460002" id="QQ2-1-53">Methods and Resources</a></span> <br /> <span class="sectionToc">2.1 <a 
href="#x1-470001" id="QQ2-1-54">Bioinformatics Resources for Genomics Research</a></span> <br />  <span class="subsectionToc">2.1.1 <a 
href="#x1-480001" id="QQ2-1-55">Public Data and Software Packages</a></span> <br />   <span class="subsubsectionToc">2.1.1.1 <a 
href="#x1-490001" id="QQ2-1-56">Cancer Genome Atlas Data</a></span> <br />   <span class="subsubsectionToc">2.1.1.2 <a 
href="#x1-500002" id="QQ2-1-57">Reactome and Annotation Data</a></span> <br /> <span class="sectionToc">2.2 <a 
href="#x1-510002" id="QQ2-1-58">Data Handling</a></span> <br />  <span class="subsectionToc">2.2.1 <a 
href="#x1-520001" id="QQ2-1-59">Normalisation</a></span> <br />  <span class="subsectionToc">2.2.2 <a 
href="#x1-530002" id="QQ2-1-60">Sample Triage</a></span> <br />  <span class="subsectionToc">2.2.3 <a 
href="#x1-540003" id="QQ2-1-68">Metagenes and the Singular Value Decomposition</a></span> <br />  <span class="subsectionToc">2.2.4 <a 
href="#x1-550004" id="QQ2-1-69">Candidate Triage and Integration with Screen Data</a></span> <br /> <span class="sectionToc">2.3 <a 
href="#x1-560003" id="QQ2-1-70">Techniques</a></span> <br />  <span class="subsectionToc">2.3.1 <a 
href="#x1-570001" id="QQ2-1-71">Statistical Procedures and Tests</a></span> <br />  <span class="subsectionToc">2.3.2 <a 
href="#x1-580002" id="QQ2-1-72">Gene Set Over-representation Analysis</a></span> <br />  <span class="subsectionToc">2.3.3 <a 
href="#x1-590003" id="QQ2-1-73">Clustering</a></span> <br />  <span class="subsectionToc">2.3.4 <a 
href="#x1-600004" id="QQ2-1-74">Heatmap</a></span> <br />  <span class="subsectionToc">2.3.5 <a 
href="#x1-610005" id="QQ2-1-75">Modelling and Simulations</a></span> <br />   <span class="subsubsectionToc">2.3.5.1 <a 
href="#x1-620001" id="QQ2-1-76">Receiver Operating Characteristic Curves</a></span> <br />  <span class="subsectionToc">2.3.6 <a 
href="#x1-630006" id="QQ2-1-77">Resampling Analysis</a></span> <br /> <span class="sectionToc">2.4 <a 
href="#x1-640004" id="QQ2-1-78">Pathway Structure Methods</a></span> <br />  <span class="subsectionToc">2.4.1 <a 
href="#x1-650001" id="QQ2-1-79">Network and Graph Analysis</a></span> <br />  <span class="subsectionToc">2.4.2 <a 
href="#x1-660002" id="QQ2-1-80">Sourcing Graph Structure Data</a></span> <br />  <span class="subsectionToc">2.4.3 <a 
href="#x1-670003" id="QQ2-1-81">Constructing Pathway Subgraphs</a></span> <br />  <span class="subsectionToc">2.4.4 <a 
href="#x1-680004" id="QQ2-1-82">Network Analysis Metrics</a></span> <br /> <span class="sectionToc">2.5 <a 
href="#x1-690005" id="QQ2-1-83">Implementation</a></span> <br />  <span class="subsectionToc">2.5.1 <a 
href="#x1-700001" id="QQ2-1-84">Computational Resources and Linux Utilities</a></span> <br />  <span class="subsectionToc">2.5.2 <a 
href="#x1-710002" id="QQ2-1-87">R Language and Packages</a></span> <br />  <span class="subsectionToc">2.5.3 <a 
href="#x1-720003" id="QQ2-1-91">High Performance and Parallel Computing</a></span> <br /><span class="chapterToc">3 <a 
href="#x1-730003" id="QQ2-1-92">Methods Developed During Thesis</a></span> <br /> <span class="sectionToc">3.1 <a 
href="#x1-740001" id="QQ2-1-93">A Synthetic Lethal Detection Methodology</a></span> <br /> <span class="sectionToc">3.2 <a 
href="#x1-750002" id="QQ2-1-96">Synthetic Lethal Simulation and Modelling</a></span> <br />  <span class="subsectionToc">3.2.1 <a 
href="#x1-760001" id="QQ2-1-97">A Model of Synthetic Lethality in Expression Data</a></span> <br />  <span class="subsectionToc">3.2.2 <a 
href="#x1-770002" id="QQ2-1-101">Simulation Procedure</a></span> <br /> <span class="sectionToc">3.3 <a 
href="#x1-780003" id="QQ2-1-111">Detecting Simulated Synthetic Lethal Partners</a></span> <br />  <span class="subsectionToc">3.3.1 <a 
href="#x1-790001" id="QQ2-1-112">Binomial Simulation of Synthetic Lethality</a></span> <br />  <span class="subsectionToc">3.3.2 <a 
href="#x1-800002" id="QQ2-1-115">Multivariate Normal Simulation of Synthetic Lethality</a></span> <br />   <span class="subsubsectionToc">3.3.2.1 <a 
href="#x1-810001" id="QQ2-1-120">Multivariate Normal Simulation with Correlated Genes</a></span> <br />   <span class="subsubsectionToc">3.3.2.2 <a 
href="#x1-820002" id="QQ2-1-143">Specificity with Query-Correlated Pathways</a></span> <br /> <span class="sectionToc">3.4 <a 
href="#x1-830004" id="QQ2-1-151">Graph Structure Methods</a></span> <br />  <span class="subsectionToc">3.4.1 <a 
href="#x1-840001" id="QQ2-1-152">Upstream and Downstream Gene Detection</a></span> <br />   <span class="subsubsectionToc">3.4.1.1 <a 
href="#x1-850001" id="QQ2-1-153">Permutation Analysis for Statistical Significance</a></span> <br />  <span class="subsectionToc">3.4.2 <a 
href="#x1-860002" id="QQ2-1-154">Simulating Gene Expression from Graph Structures</a></span></p>
<p><br /> <span class="sectionToc">3.5 <a 
href="#x1-870005" id="QQ2-1-172">Customised Functions and Packages Developed</a></span> <br />  <span class="subsectionToc">3.5.1 <a 
href="#x1-880001" id="QQ2-1-173">Synthetic Lethal Interaction Prediction Tool</a></span> <br />  <span class="subsectionToc">3.5.2 <a 
href="#x1-890002" id="QQ2-1-174">Data Visualisation</a></span> <br />  <span class="subsectionToc">3.5.3 <a 
href="#x1-900003" id="QQ2-1-179">Extensions to the iGraph Package</a></span> <br />   <span class="subsubsectionToc">3.5.3.1 <a 
href="#x1-910001" id="QQ2-1-180">Sampling Simulated Data from Graph Structures</a></span> <br />   <span class="subsubsectionToc">3.5.3.2 <a 
href="#x1-920002" id="QQ2-1-181">Plotting Directed Graph Structures</a></span> <br />   <span class="subsubsectionToc">3.5.3.3 <a 
href="#x1-930003" id="QQ2-1-183">Computing Information Centrality</a></span> <br />   <span class="subsubsectionToc">3.5.3.4 <a 
href="#x1-940004" id="QQ2-1-184">Testing Pathway Structure with Permutation Testing</a></span> <br />   <span class="subsubsectionToc">3.5.3.5 <a 
href="#x1-950005" id="QQ2-1-185">Metapackage to Install iGraph Functions</a></span> <br /><span class="chapterToc">4 <a 
href="#x1-960004" id="QQ2-1-186">Synthetic Lethal Analysis of Gene Expression Data</a></span> <br /> <span class="sectionToc">4.1 <a 
href="#x1-970001" id="QQ2-1-187">Synthetic Lethal Genes in Breast Cancer</a></span> <br />  <span class="subsectionToc">4.1.1 <a 
href="#x1-980001" id="QQ2-1-189">Synthetic Lethal Pathways in Breast Cancer</a></span> <br />  <span class="subsectionToc">4.1.2 <a 
href="#x1-990002" id="QQ2-1-191">Expression Profiles of Synthetic Lethal Partners</a></span> <br />   <span class="subsubsectionToc">4.1.2.1 <a 
href="#x1-1000001" id="QQ2-1-193">Subgroup Pathway Analysis</a></span> <br /> <span class="sectionToc">4.2 <a 
href="#x1-1010002" id="QQ2-1-195">Comparing Synthetic Lethal Gene Candidates</a></span> <br />  <span class="subsectionToc">4.2.1 <a 
href="#x1-1020001" id="QQ2-1-196">Primary siRNA Screen Candidates</a></span> <br />  <span class="subsectionToc">4.2.2 <a 
href="#x1-1030002" id="QQ2-1-199">Comparison with Correlation</a></span> <br />  <span class="subsectionToc">4.2.3 <a 
href="#x1-1040003" id="QQ2-1-202">Comparison with Primary Screen Viability</a></span> <br />  <span class="subsectionToc">4.2.4 <a 
href="#x1-1050004" id="QQ2-1-205">Comparison with Secondary siRNA Screen Validation</a></span> <br />  <span class="subsectionToc">4.2.5 <a 
href="#x1-1060005" id="QQ2-1-207">Comparison to Primary Screen at Pathway Level</a></span> <br />   <span class="subsubsectionToc">4.2.5.1 <a 
href="#x1-1070001" id="QQ2-1-209">Resampling Genes for Pathway Enrichment</a></span> <br />  <span class="subsectionToc">4.2.6 <a 
href="#x1-1080006" id="QQ2-1-213">Integrating Synthetic Lethal Pathways and Screens</a></span> <br /> <span class="sectionToc">4.3 <a 
href="#x1-1090003" id="QQ2-1-214">Synthetic Lethal Pathway Metagenes</a></span> <br /> <span class="sectionToc">4.4 <a 
href="#x1-1100004" id="QQ2-1-216">Replication in Stomach Cancer</a></span> <br /> <span class="sectionToc">4.5 <a 
href="#x1-1110005" id="QQ2-1-217">Discussion</a></span> <br />  <span class="subsectionToc">4.5.1 <a 
href="#x1-1120001" id="QQ2-1-218">Strengths of the SLIPT Methodology</a></span> <br />  <span class="subsectionToc">4.5.2 <a 
href="#x1-1130002" id="QQ2-1-219">Synthetic Lethal Pathways for E-cadherin</a></span> <br />  <span class="subsectionToc">4.5.3 <a 
href="#x1-1140003" id="QQ2-1-220">Replication and Validation</a></span> <br />   <span class="subsubsectionToc">4.5.3.1 <a 
href="#x1-1150001" id="QQ2-1-221">Integration with siRNA Screening</a></span> <br />   <span class="subsubsectionToc">4.5.3.2 <a 
href="#x1-1160002" id="QQ2-1-222">Replication across Tissues</a></span> <br /> <span class="sectionToc">4.6 <a 
href="#x1-1170006" id="QQ2-1-223">Summary</a></span> <br /><span class="chapterToc">5 <a 
href="#x1-1180005" id="QQ2-1-224">Synthetic Lethal Pathway Structure</a></span> <br /> <span class="sectionToc">5.1 <a 
href="#x1-1190001" id="QQ2-1-225">Synthetic Lethal Genes in Reactome Pathways</a></span> <br />  <span class="subsectionToc">5.1.1 <a 
href="#x1-1200001" id="QQ2-1-226">The PI3K/AKT Pathway</a></span> <br />  <span class="subsectionToc">5.1.2 <a 
href="#x1-1210002" id="QQ2-1-228">The Extracellular Matrix</a></span> <br />  <span class="subsectionToc">5.1.3 <a 
href="#x1-1220003" id="QQ2-1-231">G Protein Coupled Receptors</a></span> <br />  <span class="subsectionToc">5.1.4 <a 
href="#x1-1230004" id="QQ2-1-233">Gene Regulation and Translation</a></span> <br /> <span class="sectionToc">5.2 <a 
href="#x1-1240002" id="QQ2-1-234">Network Analysis of Synthetic Lethal Genes</a></span> <br />  <span class="subsectionToc">5.2.1 <a 
href="#x1-1250001" id="QQ2-1-235">Gene Connectivity and Vertex Degree</a></span> <br />  <span class="subsectionToc">5.2.2 <a 
href="#x1-1260002" id="QQ2-1-238">Gene Importance and Centrality</a></span> <br />   <span class="subsubsectionToc">5.2.2.1 <a 
href="#x1-1270001" id="QQ2-1-239">Information Centrality</a></span> <br />   <span class="subsubsectionToc">5.2.2.2 <a 
href="#x1-1280002" id="QQ2-1-242">PageRank Centrality</a></span> <br /> <span class="sectionToc">5.3 <a 
href="#x1-1290003" id="QQ2-1-245">Relationships between Synthetic Lethal Genes</a></span> <br />  <span class="subsectionToc">5.3.1 <a 
href="#x1-1300001" id="QQ2-1-246">Detecting Upstream or Downstream Synthetic Lethality</a></span> <br />  <span class="subsectionToc">5.3.2 <a 
href="#x1-1310002" id="QQ2-1-250">Resampling for Synthetic Lethal Pathway Structure</a></span> <br /> <span class="sectionToc">5.4 <a 
href="#x1-1320004" id="QQ2-1-252">Discussion</a></span></p>
<p><br /> <span class="sectionToc">5.5 <a 
href="#x1-1330005" id="QQ2-1-253">Summary</a></span> <br /><span class="chapterToc">6 <a 
href="#x1-1340006" id="QQ2-1-254">Simulation and Modelling of Synthetic Lethal Pathways</a></span> <br /> <span class="sectionToc">6.1 <a 
href="#x1-1350001" id="QQ2-1-255">Synthetic Lethal Detection Methods</a></span> <br />  <span class="subsectionToc">6.1.1 <a 
href="#x1-1360001" id="QQ2-1-256">Performance of SLIPT and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> across Quantiles</a></span> <br />   <span class="subsubsectionToc">6.1.1.1 <a 
href="#x1-1370001" id="QQ2-1-261">Correlated Query Genes affects Specificity</a></span> <br />  <span class="subsectionToc">6.1.2 <a 
href="#x1-1380002" id="QQ2-1-264">Alternative Synthetic Lethal Detection Strategies</a></span> <br />   <span class="subsubsectionToc">6.1.2.1 <a 
href="#x1-1390001" id="QQ2-1-265">Correlation for Synthetic Lethal Detection</a></span> <br />   <span class="subsubsectionToc">6.1.2.2 <a 
href="#x1-1400002" id="QQ2-1-267">Testing for Bimodality with BiSEp</a></span> <br /> <span class="sectionToc">6.2 <a 
href="#x1-1410002" id="QQ2-1-268">Simulations with Graph Structures</a></span> <br />  <span class="subsectionToc">6.2.1 <a 
href="#x1-1420001" id="QQ2-1-269">Performance over Graph Structures</a></span> <br />   <span class="subsubsectionToc">6.2.1.1 <a 
href="#x1-1430001" id="QQ2-1-270">Simple Graph Structures</a></span> <br />   <span class="subsubsectionToc">6.2.1.2 <a 
href="#x1-1440002" id="QQ2-1-283">Constructed Graph Structures</a></span> <br />  <span class="subsectionToc">6.2.2 <a 
href="#x1-1450002" id="QQ2-1-289">Performance with Inhibitions</a></span> <br />  <span class="subsectionToc">6.2.3 <a 
href="#x1-1460003" id="QQ2-1-310">Synthetic Lethality across Graph Structures</a></span> <br />  <span class="subsectionToc">6.2.4 <a 
href="#x1-1470004" id="QQ2-1-316">Performance within a Large Simulated Datasets</a></span> <br /> <span class="sectionToc">6.3 <a 
href="#x1-1480003" id="QQ2-1-332">Simulations in More Complex Graph Structures</a></span> <br />  <span class="subsectionToc">6.3.1 <a 
href="#x1-1490001" id="QQ2-1-333">Simulations over Pathway-based Graphs</a></span> <br />  <span class="subsectionToc">6.3.2 <a 
href="#x1-1500002" id="QQ2-1-339">Pathway Structures in a Large Simulated Datasets</a></span> <br /> <span class="sectionToc">6.4 <a 
href="#x1-1510004" id="QQ2-1-350">Discussion</a></span> <br />  <span class="subsectionToc">6.4.1 <a 
href="#x1-1520001" id="QQ2-1-351">Simulation Procedure</a></span> <br />  <span class="subsectionToc">6.4.2 <a 
href="#x1-1530002" id="QQ2-1-352">Comparing Methods with Simulated Data</a></span> <br />  <span class="subsectionToc">6.4.3 <a 
href="#x1-1540003" id="QQ2-1-353">Design and Performance of SLIPT</a></span> <br />  <span class="subsectionToc">6.4.4 <a 
href="#x1-1550004" id="QQ2-1-354">Simulations from Graph Structures</a></span> <br /> <span class="sectionToc">6.5 <a 
href="#x1-1560005" id="QQ2-1-355">Summary</a></span> <br /><span class="chapterToc">7 <a 
href="#x1-1570007" id="QQ2-1-356">Discussion</a></span> <br /> <span class="sectionToc">7.1 <a 
href="#x1-1580001" id="QQ2-1-357">Synthetic Lethality and <span class="cmti-12">CDH1 </span>Biology</a></span> <br />  <span class="subsectionToc">7.1.1 <a 
href="#x1-1590001" id="QQ2-1-358">Established Functions of <span class="cmti-12">CDH1</span></a></span> <br />  <span class="subsectionToc">7.1.2 <a 
href="#x1-1600002" id="QQ2-1-359">The Molecular Role of <span class="cmti-12">CDH1 </span>in Cancer</a></span> <br /> <span class="sectionToc">7.2 <a 
href="#x1-1610002" id="QQ2-1-360">Significance</a></span> <br />  <span class="subsectionToc">7.2.1 <a 
href="#x1-1620001" id="QQ2-1-361">Synthetic Lethality in the Genomic Era</a></span> <br />  <span class="subsectionToc">7.2.2 <a 
href="#x1-1630002" id="QQ2-1-362">Clinical Interventions based on Synthetic Lethality</a></span> <br /> <span class="sectionToc">7.3 <a 
href="#x1-1640003" id="QQ2-1-363">Future Directions</a></span> <br /> <span class="sectionToc">7.4 <a 
href="#x1-1650004" id="QQ2-1-364">Conclusions</a></span> <br /><span class="chapterToc"><a 
href="#Q1-1-365"> Bibliography</a></span> <br /><span class="appendixToc">A <a 
href="#x1-167000A" id="QQ2-1-367">Sample Quality</a></span> <br /> <span class="sectionToc">A.1 <a 
href="#x1-1680001" id="QQ2-1-368">Sample Correlation</a></span> <br /> <span class="sectionToc">A.2 <a 
href="#x1-1690002" id="QQ2-1-371">Replicate Samples in TCGA Breast Cancer Data</a></span> <br /><span class="appendixToc">B <a 
href="#x1-170000B" id="QQ2-1-396">Software Used for Thesis</a></span> <br /><span class="appendixToc">C <a 
href="#x1-171000C" id="QQ2-1-398">Mutation Analysis in Breast Cancer</a></span> <br /> <span class="sectionToc">C.1 <a 
href="#x1-1720001" id="QQ2-1-399">Synthetic Lethal Genes and Pathways</a></span> <br /> <span class="sectionToc">C.2 <a 
href="#x1-1730002" id="QQ2-1-402">Synthetic Lethal Expression Profiles</a></span> <br /> <span class="sectionToc">C.3 <a 
href="#x1-1740003" id="QQ2-1-405">Comparison to Primary Screen</a></span> <br />  <span class="subsectionToc">C.3.1 <a 
href="#x1-1750001" id="QQ2-1-408">Resampling Analysis</a></span> <br /> <span class="sectionToc">C.4 <a 
href="#x1-1760004" id="QQ2-1-411">Compare SLIPT genes</a></span> <br /><span class="appendixToc">D <a 
href="#x1-177000D" id="QQ2-1-415">Metagene Analysis</a></span> <br /> <span class="sectionToc">D.1 <a 
href="#x1-1780001" id="QQ2-1-416">Pathway Signature Expression</a></span></p>
<br /> <span class="sectionToc">D.2 <a 
href="#x1-1790002" id="QQ2-1-419">Synthetic Lethal Reactome Metagenes</a></span> <br /><span class="appendixToc">E <a 
href="#x1-180000E" id="QQ2-1-421">Intrinsic Subtyping</a></span> <br /><span class="appendixToc">F <a 
href="#x1-181000F" id="QQ2-1-423">Stomach Expression Analysis</a></span> <br /> <span class="sectionToc">F.1 <a 
href="#x1-1820001" id="QQ2-1-424">Synthetic Lethal Genes and Pathways</a></span> <br /> <span class="sectionToc">F.2 <a 
href="#x1-1830002" id="QQ2-1-429">Comparison to Primary Screen</a></span> <br />  <span class="subsectionToc">F.2.1 <a 
href="#x1-1840001" id="QQ2-1-432">Resampling Analysis</a></span> <br /> <span class="sectionToc">F.3 <a 
href="#x1-1850003" id="QQ2-1-435">Metagene Analysis</a></span> <br /><span class="appendixToc">G <a 
href="#x1-186000G" id="QQ2-1-437">Synthetic Lethal Genes in Pathways</a></span> <br /><span class="appendixToc">H <a 
href="#x1-187000H" id="QQ2-1-445">Network Analysis for Mutation SLIPT</a></span> <br /><span class="appendixToc">I <a 
href="#x1-188000I" id="QQ2-1-452">Pathway Structure for Mutation SLIPT</a></span> <br /><span class="appendixToc">J <a 
href="#x1-189000J" id="QQ2-1-457">Performance of SLIPT and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup></a></span> <br /> <span class="sectionToc">J.1 <a 
href="#x1-1900001" id="QQ2-1-509">Correlated Query Genes affects Specificity</a></span> <br /><span class="appendixToc">K <a 
href="#x1-191000K" id="QQ2-1-561">Simulations on Graph Structures</a></span> <br />  <span class="subsectionToc">K.0.1 <a 
href="#x1-1920001" id="QQ2-1-567">Simulations from Inhibiting Graph Structures</a></span> <br /> <span class="sectionToc">K.1 <a 
href="#x1-1930001" id="QQ2-1-583">Simulation across Graph Structures</a></span> <br /> <span class="sectionToc">K.2 <a 
href="#x1-1940002" id="QQ2-1-618">Simulations from Complex Graph Structures</a></span> <br />  <span class="subsectionToc">K.2.1 <a 
href="#x1-1950001" id="QQ2-1-634">Simulations from Complex Inhibiting Graphs</a></span> <br /> <span class="sectionToc">K.3 <a 
href="#x1-1960003" id="QQ2-1-665">Simulations from Pathway Graph Structures</a></span>
</div>
                                                                                    

<h2 class="likechapterHead">
<a 
 id="x1-2000"></a>List of Tables
</h2><div class="tableofcontents">
<span class="lotToc">1.1 <a 
href="#x1-330011">Methods for predicting genetic interactions</a></span><br /><span class="lotToc">1.2 <a 
href="#x1-330022">Methods for predicting synthetic lethality in cancer</a></span><br /><span class="lotToc">1.3 <a 
href="#x1-340013">Methods used by Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014)</a></span><br /><span class="lotToc">2.1 <a 
href="#x1-530071">Excluded samples by batch and clinical characteristics.</a></span><br /><span class="lotToc">2.2 <a 
href="#x1-700012">Computers used during thesis</a></span><br /><span class="lotToc">2.3 <a 
href="#x1-700023">Linux utilities and applications used during thesis</a></span><br /><span class="lotToc">2.4 <a 
href="#x1-710014">R installations used during thesis</a></span><br /><span class="lotToc"> <a 
href="#x1-71003">2.5 R Packages used during thesis</a></span><br /><span class="lotToc">2.6 <a 
href="#x1-710046">R packages developed during thesis</a></span><br /><span class="lotToc">4.1 <a 
href="#x1-970011">Candidate synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from SLIPT</a></span><br /><span class="lotToc">4.2 <a 
href="#x1-980012">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT</a></span><br /><span class="lotToc">4.3 <a 
href="#x1-1000013">Pathways for clusters of <span class="cmti-12">CDH1 </span>partners from SLIPT</a></span><br /><span class="lotToc">4.4 <a 
href="#x1-1030024">ANOVA for synthetic lethality and correlation with <span class="cmti-12">CDH1</span></a></span><br /><span class="lotToc">4.5 <a 
href="#x1-1050015">Comparison of SLIPT genes against secondary siRNA screen</a></span><br /><span class="lotToc">4.6 <a 
href="#x1-1060016">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT and siRNA</a></span><br /><span class="lotToc">4.7 <a 
href="#x1-1070027">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT</a></span><br /><span class="lotToc">4.8 <a 
href="#x1-1070038">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT and siRNA primary screen</a></span><br /><span class="lotToc">4.9 <a 
href="#x1-1090019">Examples of candidate metagenes synthetic lethal for <span class="cmti-12">CDH1 </span>from SLIPT</a></span><br /><span class="lotToc">5.1 <a 
href="#x1-1250021">ANOVA for synthetic lethality and vertex degree</a></span><br /><span class="lotToc">5.2 <a 
href="#x1-1270022">ANOVA for synthetic lethality and information centrality</a></span><br /><span class="lotToc">5.3 <a 
href="#x1-1280023">ANOVA for synthetic lethality and PageRank centrality</a></span><br /><span class="lotToc">5.4 <a 
href="#x1-1310014">Resampling for pathway structure of synthetic lethal detection methods</a></span><br /><span class="lotToc"> <a 
href="#x1-170002">B.1 Complete list of R packages used during this thesis</a></span><br /><span class="lotToc">C.1 <a 
href="#x1-1720011">Candidate synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from mtSLIPT</a></span><br /><span class="lotToc">C.2 <a 
href="#x1-1720022">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT</a></span><br /><span class="lotToc">C.3 <a 
href="#x1-1730023">Pathways for clusters of <span class="cmti-12">CDH1 </span>partners from mtSLIPT</a></span><br /><span class="lotToc">C.4 <a 
href="#x1-1740024">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT and siRNA</a></span><br /><span class="lotToc">C.5 <a 
href="#x1-1750015">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT</a></span><br /><span class="lotToc">C.6 <a 
href="#x1-1750026">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT and siRNA primary screen</a></span><br /><span class="lotToc">D.1 <a 
href="#x1-1790011">Candidate synthetic lethal metagenes against <span class="cmti-12">CDH1</span> from mtSLIPT</a></span><br /><span class="lotToc">E.1 <a 
href="#x1-1800011">Comparison of intrinsic subtypes</a></span><br /><span class="lotToc">F.1 <a 
href="#x1-1820011">Synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from SLIPT in stomach cancer</a></span><br /><span class="lotToc">F.2 <a 
href="#x1-1820022">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in stomach cancer</a></span><br /><span class="lotToc">F.3 <a 
href="#x1-1820043">Pathways for clusters of <span class="cmti-12">CDH1 </span>partners in stomach SLIPT</a></span><br /><span class="lotToc">F.4 <a 
href="#x1-1830024">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT and siRNA</a></span><br /><span class="lotToc">F.5 <a 
href="#x1-1840015">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in stomach cancer</a></span><br /><span class="lotToc">F.6 <a 
href="#x1-1840026">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in stomach and siRNA</a></span><br /><span class="lotToc">F.7 <a 
href="#x1-1850017">Synthetic lethal metagenes against <span class="cmti-12">CDH1 </span>in stomach cancer</a></span><br /><span class="lotToc">H.1 <a 
href="#x1-1870041">ANOVA for synthetic lethality and vertex degree</a></span><br /><span class="lotToc">H.2 <a 
href="#x1-1870052">ANOVA for synthetic lethality and information centrality</a></span><br /><span class="lotToc">H.3 <a 
href="#x1-1870063">ANOVA for synthetic lethality and PageRank centrality</a></span><br /><span class="lotToc">I.1 <a 
href="#x1-1880041">Resampling for pathway structure of synthetic lethal detection methods</a></span><br />
</div>
                                                                                    

<h2 class="likechapterHead">
<a 
 id="x1-3000"></a>List of Figures
</h2><div class="tableofcontents">
<p><span class="lofToc">1.1 <a 
href="#x1-230011">Synthetic genetic interactions</a></span><br /><span class="lofToc">1.2 <a 
href="#x1-280012">Synthetic lethality in cancer</a></span><br /><span class="lofToc">a <a 
href="#x1-53001r1">Raw counts (log-scale)</a></span><br /><span class="lofToc">b <a 
href="#x1-53002r2">Voom normalised</a></span><br /><span class="lofToc">2.2 <a 
href="#x1-530032">Read count density</a></span><br /><span class="lofToc">a <a 
href="#x1-53004r1">Mean raw counts (log-scale)</a></span><br /><span class="lofToc">b <a 
href="#x1-53005r2">Mean voom normalised</a></span><br /><span class="lofToc">2.3 <a 
href="#x1-530063">Read count sample mean</a></span><br /><span class="lofToc">3.1 <a 
href="#x1-740011">Framework for synthetic lethal prediction</a></span><br /><span class="lofToc">3.2 <a 
href="#x1-740022">Synthetic lethal prediction adapted for mutation</a></span><br /><span class="lofToc">3.3 <a 
href="#x1-760013">A model of synthetic lethal gene expression</a></span><br /><span class="lofToc">3.4 <a 
href="#x1-760024">Modelling synthetic lethal gene expression</a></span><br /><span class="lofToc">3.5 <a 
href="#x1-760035">Synthetic lethality with multiple genes</a></span><br /><span class="lofToc">a <a 
href="#x1-77001r1">Simulated expression matrix</a></span><br /><span class="lofToc">b <a 
href="#x1-77002r2">Corresponding gene function calls</a></span><br /><span class="lofToc">3.7 <a 
href="#x1-770037">Simulating gene function</a></span><br /><span class="lofToc">a <a 
href="#x1-77004r1">Simulated gene function with SL genes</a></span><br /><span class="lofToc">b <a 
href="#x1-77005r2">Query gene added with SL condition</a></span><br /><span class="lofToc">3.8 <a 
href="#x1-770068">Simulating synthetic lethal gene function</a></span><br /><span class="lofToc">a <a 
href="#x1-77007r1">Initial expression matrix</a></span><br /><span class="lofToc">b <a 
href="#x1-77008r2">Simulated synthetic lethal dataset</a></span><br /><span class="lofToc">3.10 <a 
href="#x1-7700910">Simulating synthetic lethal gene expression</a></span><br /><span class="lofToc">3.11 <a 
href="#x1-7900111">Performance of binomial simulations</a></span><br /><span class="lofToc">3.12 <a 
href="#x1-7900212">Comparison of statistical performance</a></span><br /><span class="lofToc">a <a 
href="#x1-80001r1">Statistical evaluation </a></span><br /><span class="lofToc">b <a 
href="#x1-80002r2">receiver operating characteristic (ROC) curves</a></span><br /><span class="lofToc">c <a 
href="#x1-80003r3">Statistical performance (AUROC) </a></span><br /><span class="lofToc">3.14 <a 
href="#x1-8000414">Performance of multivariate normal simulations</a></span><br /><span class="lofToc">a <a 
href="#x1-81001r1">Input Σ matrix parameter </a></span><br /><span class="lofToc">b <a 
href="#x1-81002r2">Simulated correlation matrix </a></span><br /><span class="lofToc">c <a 
href="#x1-81003r3">Simulated gene expression </a></span><br /><span class="lofToc">d <a 
href="#x1-81004r4">Simulated gene function </a></span><br /><span class="lofToc">3.16 <a 
href="#x1-8100516">Simulating expression with correlated gene blocks</a></span><br /><span class="lofToc">a <a 
href="#x1-81006r1">Input Σ matrix parameter</a></span><br /><span class="lofToc">b <a 
href="#x1-81007r2">Simulated correlation matrix</a></span><br /><span class="lofToc">c <a 
href="#x1-81008r3">Simulated gene expression</a></span><br /><span class="lofToc">d <a 
href="#x1-81009r4">Simulated gene function</a></span><br /><span class="lofToc">3.18 <a 
href="#x1-8101018">Simulating expression with correlated gene blocks</a></span><br /><span class="lofToc">a <a 
href="#x1-81011r1">Gene category in simulations</a></span><br /><span class="lofToc">b <a 
href="#x1-81012r2">Corresponding <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values</a></span><br /><span class="lofToc">3.20 <a 
href="#x1-8101320">Synthetic lethal prediction across simulations</a></span><br /><span class="lofToc">a <a 
href="#x1-81014r1">Statistical evaluation</a></span><br /><span class="lofToc">b <a 
href="#x1-81015r2">ROC curves</a></span><br /><span class="lofToc">c <a 
href="#x1-81016r3">Statistical performance (AUROC) </a></span><br /><span class="lofToc">3.22 <a 
href="#x1-8101722">Performance with correlations</a></span><br /><span class="lofToc">3.23 <a 
href="#x1-8101823">Comparison of statistical performance with correlation structure</a></span><br /><span class="lofToc">a <a 
href="#x1-81019r1">Statistical evaluation </a></span><br /><span class="lofToc">b <a 
href="#x1-81020r2">ROC curves</a></span><br /><span class="lofToc">c <a 
href="#x1-81021r3">Statistical performance (AUROC)</a></span><br /><span class="lofToc">3.25 <a 
href="#x1-8102225">Performance with query correlations</a></span><br /><span class="lofToc">a <a 
href="#x1-82001r1"><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> testing without direction</a></span><br /><span class="lofToc">b <a 
href="#x1-82002r2">SLIPT with directional criteria</a></span><br /><span class="lofToc">3.27 <a 
href="#x1-8200327">Statistical evaluation of directional criteria</a></span><br /><span class="lofToc">a <a 
href="#x1-82004r1"><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> testing without direction</a></span><br /><span class="lofToc">b <a 
href="#x1-82005r2">SLIPT with directional criteria</a></span><br /><span class="lofToc">c <a 
href="#x1-82006r3">Statistical performance (AUROC) </a></span><br /><span class="lofToc">3.29 <a 
href="#x1-8200729">Performance of directional criteria</a></span><br /><span class="lofToc">a <a 
href="#x1-86001r1">Activating pathway structure </a></span><br /><span class="lofToc">b <a 
href="#x1-86002r2">Pathway structure with inhibitions </a></span><br /><span class="lofToc">3.31 <a 
href="#x1-8600331">Simulated graph structures</a></span><br /><span class="lofToc">a <a 
href="#x1-86004r1">Activating pathway structure </a></span><br /><span class="lofToc">b <a 
href="#x1-86005r2">Distance matrix </a></span><br /><span class="lofToc">c <a 
href="#x1-86006r3">Σ (expected correlation) </a></span><br /><span class="lofToc">d <a 
href="#x1-86007r4">Simulated correlation</a></span><br /><span class="lofToc">e <a 
href="#x1-86008r5">Simulated expression data </a></span><br /><span class="lofToc">f <a 
href="#x1-86009r6">Simulated gene function calls </a></span><br /><span class="lofToc">3.33 <a 
href="#x1-8601033">Simulating expression from a graph structure</a></span><br /><span class="lofToc">a <a 
href="#x1-86011r1">Inhibiting pathway structure</a></span><br /><span class="lofToc">b <a 
href="#x1-86012r2">Distance matrix </a></span><br /><span class="lofToc">c <a 
href="#x1-86013r3">Σ (expected correlation) </a></span><br /><span class="lofToc">d <a 
href="#x1-86014r4">Simulated correlation</a></span><br /><span class="lofToc">e <a 
href="#x1-86015r5">Simulated expression data </a></span><br /><span class="lofToc">f <a 
href="#x1-86016r6">Simulated gene function calls </a></span><br /><span class="lofToc">3.35 <a 
href="#x1-8601735">Simulating expression from graph structure with inhibitions</a></span><br /><span class="lofToc">a <a 
href="#x1-89001r1">Customised violin plot</a></span><br /><span class="lofToc">b <a 
href="#x1-89002r2">Split violin plot</a></span><br /><span class="lofToc" >3.37 <a 
href="#x1-8900337">Demonstration of violin plots with custom</p>
<p>features</a></span><br /><span class="lofToc">3.38 <a 
href="#x1-8900438">Demonstration of annotated heatmap</a></span><br /><span class="lofToc">3.39 <a 
href="#x1-9200139">Simulating graph structures</a></span><br /><span class="lofToc">4.1 <a 
href="#x1-990011">Synthetic lethal expression profiles of analysed samples</a></span><br /><span class="lofToc">4.2 <a 
href="#x1-1020012">Comparison of SLIPT with siRNA</a></span><br /><span class="lofToc">4.3 <a 
href="#x1-1020023">Comparison of SLIPT and siRNA genes with correlation</a></span><br /><span class="lofToc">4.4 <a 
href="#x1-1030014">Comparison of SLIPT and siRNA genes with correlation</a></span><br /><span class="lofToc">4.5 <a 
href="#x1-1040015">Comparison of SLIPT and siRNA genes with screen viability</a></span><br /><span class="lofToc">4.6 <a 
href="#x1-1040026">Comparison of SLIPT genes with siRNA screen viability</a></span><br /><span class="lofToc">4.7 <a 
href="#x1-1070017">Resampled intersection of SLIPT and siRNA candidate genes</a></span><br /><span class="lofToc">5.1 <a 
href="#x1-1200011">Synthetic lethality in the PI3K cascade</a></span><br /><span class="lofToc">5.2 <a 
href="#x1-1210012">Synthetic lethality in Elastic Fibre Formation</a></span><br /><span class="lofToc">5.3 <a 
href="#x1-1210023">Synthetic lethality in Fibrin Clot Formation</a></span><br /><span class="lofToc">5.4 <a 
href="#x1-1220014">Synthetic lethality in the GPCRs</a></span><br /><span class="lofToc">5.5 <a 
href="#x1-1250015">Synthetic lethality and vertex degree</a></span><br /><span class="lofToc">5.6 <a 
href="#x1-1270016">Synthetic lethality and centrality</a></span><br /><span class="lofToc">5.7 <a 
href="#x1-1280017">Synthetic lethality and PageRank</a></span><br /><span class="lofToc">a <a 
href="#x1-130001r1">Resampling in G<sub><span class="cmmi-8">αi</span></sub> signalling </a></span><br /><span class="lofToc">b <a 
href="#x1-130002r2">Resampling in the PI3K cascade </a></span><br /><span class="lofToc">5.9 <a 
href="#x1-1300039">Structure of synthetic lethality resampling</a></span><br /><span class="lofToc">a <a 
href="#x1-136001r1">Barplot of <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>, SLIPT, and correlation.</a></span><br /><span class="lofToc">b <a 
href="#x1-136002r2">Lineplot of <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>, SLIPT, and correlation.</a></span><br /><span class="lofToc">6.2 <a 
href="#x1-1360032">Performance of <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> and SLIPT across quantiles</a></span><br /><span class="lofToc">6.3 <a 
href="#x1-1360043">Performance of <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> and SLIPT across quantiles with more genes</a></span><br /><span class="lofToc">6.4 <a 
href="#x1-1370014">Performance of <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> and SLIPT across quantiles with query correlation</a></span><br /><span class="lofToc">6.5 <a 
href="#x1-1370025">Performance of <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> and SLIPT across quantiles with query correlation and more genes</a></span><br /><span class="lofToc">6.6 <a 
href="#x1-1390016">Performance of negative correlation and SLIPT</a></span><br /><span class="lofToc">a <a 
href="#x1-143001r1">Diverging Module </a></span><br /><span class="lofToc">b <a 
href="#x1-143002r2">Converging module </a></span><br /><span class="lofToc">6.8 <a 
href="#x1-1430038">Simple graph structures</a></span><br /><span class="lofToc">a <a 
href="#x1-143004r1">Statistical evaluation </a></span><br /><span class="lofToc">b <a 
href="#x1-143005r2">Receiver operating characteristic </a></span><br /><span class="lofToc">c <a 
href="#x1-143006r3">Graph Structure </a></span><br /><span class="lofToc">d <a 
href="#x1-143007r4">Statistical performance </a></span><br /><span class="lofToc">6.10 <a 
href="#x1-14300810">Performance of simulations on a simple graph</a></span><br /><span class="lofToc">a <a 
href="#x1-143009r1">Diverging Module </a></span><br /><span class="lofToc">b <a 
href="#x1-143010r2">Converging module </a></span><br /><span class="lofToc">c <a 
href="#x1-143011r3">Performance between Graph Structures </a></span><br /><span class="lofToc">6.12 <a 
href="#x1-14301212">Performance of simulations is similar in simple graphs</a></span><br /><span class="lofToc">a <a 
href="#x1-144001r1">Statistical evaluation </a></span><br /><span class="lofToc">b <a 
href="#x1-144002r2">Receiver operating characteristic </a></span><br /><span class="lofToc">c <a 
href="#x1-144003r3">Graph Structure </a></span><br /><span class="lofToc">d <a 
href="#x1-144004r4">Statistical performance </a></span><br /><span class="lofToc">6.14 <a 
href="#x1-14400514">Performance of simulations on a pathway</a></span><br /><span class="lofToc">a <a 
href="#x1-145001r1">Statistical evaluation </a></span><br /><span class="lofToc">b <a 
href="#x1-145002r2">Receiver operating characteristic </a></span><br /><span class="lofToc">c <a 
href="#x1-145003r3">Graph Structure </a></span><br /><span class="lofToc">d <a 
href="#x1-145004r4">Statistical performance </a></span><br /><span class="lofToc">6.16 <a 
href="#x1-14500516">Performance of simulations on a simple graph with inhibition</a></span><br /><span class="lofToc">a <a 
href="#x1-145006r1">Activating Graph</a></span><br /><span class="lofToc">b <a 
href="#x1-145007r2">Inhibiting Graph</a></span><br /><span class="lofToc">c <a 
href="#x1-145008r3">Performance between Graph Structures</a></span><br /><span class="lofToc">6.18 <a 
href="#x1-14500918">Performance is higher on a simple inhibiting graph</a></span><br /><span class="lofToc">a <a 
href="#x1-145010r1">Statistical evaluation</a></span><br /><span class="lofToc">b <a 
href="#x1-145011r2">Receiver operating characteristic</a></span><br /><span class="lofToc">c <a 
href="#x1-145012r3">Graph Structure</a></span><br /><span class="lofToc">d <a 
href="#x1-145013r4">Statistical performance</a></span><br /><span class="lofToc">6.20 <a 
href="#x1-14501420">Performance of simulations on a constructed graph with inhibition</a></span><br /><span class="lofToc">a <a 
href="#x1-145015r1">Activating Graph</a></span><br /><span class="lofToc">b <a 
href="#x1-145016r2">Inhibiting Graph</a></span><br /><span class="lofToc">c <a 
href="#x1-145017r3">Mixed Graph</a></span><br /><span class="lofToc">d <a 
href="#x1-145018r4">Performance between Graphs (a) and (b)</a></span><br /><span class="lofToc">e <a 
href="#x1-145019r5">Performance between Graphs (a) and (c)</a></span><br /><span class="lofToc">6.22 <a 
href="#x1-14502022">Performance is affected by inhibition in graphs</a></span><br /><span class="lofToc">a <a 
href="#x1-146001r1">Activating Graph </a></span><br /><span class="lofToc">b <a 
href="#x1-146002r2">Inhibiting Graph </a></span><br /><span class="lofToc">c <a 
href="#x1-146003r3"><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> distribution for Activating Graph </a></span><br /><span class="lofToc">d <a 
href="#x1-146004r4"><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> distribution for Inhibiting Graph </a></span><br /><span class="lofToc">6.24 <a 
href="#x1-14600524">Detection of synthetic lethality within a graph structure</a></span><br /><span class="lofToc">a <a 
href="#x1-147001r1">Statistical evaluation</a></span><br /><span class="lofToc">b <a 
href="#x1-147002r2">Receiver operating characteristic</a></span><br /><span class="lofToc">c <a 
href="#x1-147003r3">Graph Structure</a></span><br /><span class="lofToc">d <a 
href="#x1-147004r4">Statistical performance</a></span><br /><span class="lofToc">6.25 <a 
href="#x1-14700525">Performance of simulations including a simple graph</a></span><br /><span class="lofToc">a <a 
href="#x1-147006r1">Diverging Module</a></span><br /><span class="lofToc">b <a 
href="#x1-147007r2">Converging module</a></span><br /><span class="lofToc">c <a 
href="#x1-147008r3">Performance in Diverging Module</a></span><br /><span class="lofToc">d <a 
href="#x1-147009r4">Performance in Converging module</a></span><br /><span class="lofToc">6.27 <a 
href="#x1-14701027">Performance on a simple graph improves with more genes</a></span><br /><span class="lofToc">a <a 
href="#x1-147011r1">Graph Structure</a></span><br /><span class="lofToc">b <a 
href="#x1-147012r2">Inhibiting Structure</a></span><br /><span class="lofToc">c <a 
href="#x1-147013r3">Performance in Graph</a></span><br /><span class="lofToc">d <a 
href="#x1-147014r4">Performance with Inhibitions</a></span><br /><span class="lofToc">6.29 <a 
href="#x1-14701529">Performance on an inhibiting graph improves with more genes</a></span><br /><span class="lofToc">a <a 
href="#x1-149001r1">Statistical evaluation</a></span><br /><span class="lofToc">b <a 
href="#x1-149002r2">Receiver operating characteristic</a></span><br /><span class="lofToc">c <a 
href="#x1-149003r3">Graph Structure</a></span><br /><span class="lofToc">d <a 
href="#x1-149004r4">Statistical performance</a></span><br /><span class="lofToc" >6.31 <a 
href="#x1-14900531">Performance of simulations</p>
<p>on the PI3K cascade</a></span><br /><span class="lofToc">a <a 
href="#x1-150001r1">Statistical evaluation</a></span><br /><span class="lofToc">b <a 
href="#x1-150002r2">Receiver operating characteristic</a></span><br /><span class="lofToc">c <a 
href="#x1-150003r3">Graph Structure</a></span><br /><span class="lofToc">d <a 
href="#x1-150004r4">Statistical performance</a></span><br /><span class="lofToc">6.33 <a 
href="#x1-15000533">Performance of simulations including the PI3K cascade</a></span><br /><span class="lofToc">a <a 
href="#x1-150006r1">The PI3K cascade </a></span><br /><span class="lofToc">b <a 
href="#x1-150007r2">G<sub><span class="cmmi-8">αi</span></sub> signalling </a></span><br /><span class="lofToc">c <a 
href="#x1-150008r3">Performance in the PI3K cascade </a></span><br /><span class="lofToc">d <a 
href="#x1-150009r4">Performance in G<sub><span class="cmmi-8">αi</span></sub> signalling </a></span><br /><span class="lofToc">6.35 <a 
href="#x1-15001035">Performance on pathways improves with more genes</a></span><br /><span class="lofToc">A.1 <a 
href="#x1-1680011">Correlation profiles of removed samples</a></span><br /><span class="lofToc">A.2 <a 
href="#x1-1680022">Correlation analysis and sample removal</a></span><br /><span class="lofToc">A.3 <a 
href="#x1-1690013">Replicate excluded samples</a></span><br /><span class="lofToc">a <a 
href="#x1-169002r1">Remaining triplet</a></span><br /><span class="lofToc">b <a 
href="#x1-169003r2">Remaining triplet</a></span><br /><span class="lofToc">c <a 
href="#x1-169004r3">Remaining triplet</a></span><br /><span class="lofToc">d <a 
href="#x1-169005r4">Remaining pair</a></span><br /><span class="lofToc">e <a 
href="#x1-169006r5">Remaining pair</a></span><br /><span class="lofToc">f <a 
href="#x1-169007r6">Remaining pair</a></span><br /><span class="lofToc">A.5 <a 
href="#x1-1690085">Replicate samples with all remaining</a></span><br /><span class="lofToc">a <a 
href="#x1-169009r1">Remaining</a></span><br /><span class="lofToc">b <a 
href="#x1-169010r2">Compare with excluded</a></span><br /><span class="lofToc">c <a 
href="#x1-169011r3">Compare with excluded</a></span><br /><span class="lofToc">d <a 
href="#x1-169012r4">Remaining</a></span><br /><span class="lofToc">e <a 
href="#x1-169013r5">Compare with excluded</a></span><br /><span class="lofToc">f <a 
href="#x1-169014r6">Compare with excluded</a></span><br /><span class="lofToc">g <a 
href="#x1-169015r7">Remaining</a></span><br /><span class="lofToc">h <a 
href="#x1-169016r8">Compare with excluded</a></span><br /><span class="lofToc">i <a 
href="#x1-169017r9">Compare with excluded</a></span><br /><span class="lofToc">A.7 <a 
href="#x1-1690187">Replicate samples with some excluded</a></span><br /><span class="lofToc">a <a 
href="#x1-169019r1">Remaining</a></span><br /><span class="lofToc">b <a 
href="#x1-169020r2">Compare with excluded</a></span><br /><span class="lofToc">c <a 
href="#x1-169021r3">Compare with excluded</a></span><br /><span class="lofToc">d <a 
href="#x1-169022r4">Compare with excluded</a></span><br /><span class="lofToc">e <a 
href="#x1-169023r5">Compare with excluded</a></span><br /><span class="lofToc">C.1 <a 
href="#x1-1730011">Synthetic lethal expression profiles of analysed samples</a></span><br /><span class="lofToc">C.2 <a 
href="#x1-1740012">Comparison of mtSLIPT to siRNA</a></span><br /><span class="lofToc">C.3 <a 
href="#x1-1760013">Compare mtSLIPT and siRNA genes with correlation</a></span><br /><span class="lofToc">C.4 <a 
href="#x1-1760024">Compare mtSLIPT and siRNA genes with correlation</a></span><br /><span class="lofToc">C.5 <a 
href="#x1-1760035">Compare mtSLIPT and siRNA genes with siRNA viability</a></span><br /><span class="lofToc">D.1 <a 
href="#x1-1780011">Pathway metagene expression profiles</a></span><br /><span class="lofToc">D.2 <a 
href="#x1-1780022">Expression profiles for estrogen receptor related genes</a></span><br /><span class="lofToc">F.1 <a 
href="#x1-1820031">Synthetic lethal expression profiles of stomach samples</a></span><br /><span class="lofToc">F.2 <a 
href="#x1-1830012">Comparison of SLIPT in stomach to siRNA</a></span><br /><span class="lofToc">G.1 <a 
href="#x1-1860011">Synthetic lethality in the PI3K/AKT pathway</a></span><br /><span class="lofToc">G.2 <a 
href="#x1-1860022">Synthetic lethality in the PI3K/AKT pathway in cancer</a></span><br /><span class="lofToc">G.3 <a 
href="#x1-1860033">Synthetic lethality in the Extracellular Matrix</a></span><br /><span class="lofToc">G.4 <a 
href="#x1-1860044">Synthetic lethality in the GPCR Downstream</a></span><br /><span class="lofToc">G.5 <a 
href="#x1-1860055">Synthetic lethality in the Translation Elongation</a></span><br /><span class="lofToc">G.6 <a 
href="#x1-1860066">Synthetic lethality in the Nonsense-mediated Decay</a></span><br /><span class="lofToc">G.7 <a 
href="#x1-1860077">Synthetic lethality in the 3<sup><span class="cmsy-8">′</span></sup> UTR</a></span><br /><span class="lofToc">H.1 <a 
href="#x1-1870011">Synthetic lethality and vertex degree</a></span><br /><span class="lofToc">H.2 <a 
href="#x1-1870022">Synthetic lethality and centrality</a></span><br /><span class="lofToc">H.3 <a 
href="#x1-1870033">Synthetic lethality and PageRank</a></span><br /><span class="lofToc">a <a 
href="#x1-188001r1">Resampling in G<sub><span class="cmmi-8">αi</span></sub> signalling </a></span><br /><span class="lofToc">b <a 
href="#x1-188002r2">Resampling in the PI3K cascade </a></span><br /><span class="lofToc">I.2 <a 
href="#x1-1880032">Structure of synthetic lethality resampling</a></span><br /><span class="lofToc">a <a 
href="#x1-189001r1">1 SL partner</a></span><br /><span class="lofToc">b <a 
href="#x1-189002r2">2 SL partners</a></span><br /><span class="lofToc">c <a 
href="#x1-189003r3">3 SL partners</a></span><br /><span class="lofToc">d <a 
href="#x1-189004r4">4 SL partners</a></span><br /><span class="lofToc">e <a 
href="#x1-189005r5">5 SL partners</a></span><br /><span class="lofToc">f <a 
href="#x1-189006r6">6 SL partners</a></span><br /><span class="lofToc">J.2 <a 
href="#x1-1890072">Performance of <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> and SLIPT across quantiles</a></span><br /><span class="lofToc">a <a 
href="#x1-189008r1">7 SL partners</a></span><br /><span class="lofToc">b <a 
href="#x1-189009r2">8 SL partners</a></span><br /><span class="lofToc">c <a 
href="#x1-189010r3">9 SL partners</a></span><br /><span class="lofToc">d <a 
href="#x1-189011r4">10 SL partners</a></span><br /><span class="lofToc">e <a 
href="#x1-189012r5">15 SL partners</a></span><br /><span class="lofToc">f <a 
href="#x1-189013r6">20 SL partners</a></span><br /><span class="lofToc">g <a 
href="#x1-189014r7">30 SL partners</a></span><br /><span class="lofToc">h <a 
href="#x1-189015r8">40 SL partners</a></span><br /><span class="lofToc">i <a 
href="#x1-189016r9">50 SL partners</a></span><br /><span class="lofToc">a <a 
href="#x1-189018r1">1 SL partner</a></span><br /><span class="lofToc">b <a 
href="#x1-189019r2">2 SL partners</a></span><br /><span class="lofToc">c <a 
href="#x1-189020r3">3 SL partners</a></span><br /><span class="lofToc">d <a 
href="#x1-189021r4">4 SL partners</a></span><br /><span class="lofToc">e <a 
href="#x1-189022r5">5 SL partners</a></span><br /><span class="lofToc">f <a 
href="#x1-189023r6">6 SL partners</a></span><br /><span class="lofToc">g <a 
href="#x1-189024r7">7 SL partners</a></span><br /><span class="lofToc">h <a 
href="#x1-189025r8">8 SL partners</a></span><br /><span class="lofToc">i <a 
href="#x1-189026r9">9 SL partners</a></span><br /><span class="lofToc">J.6 <a 
href="#x1-1890276">Performance of <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> and SLIPT across quantiles</a></span><br /><span class="lofToc">a <a 
href="#x1-189028r1">10 SL partners</a></span><br /><span class="lofToc">b <a 
href="#x1-189029r2">15 SL partners</a></span><br /><span class="lofToc">c <a 
href="#x1-189030r3">20 SL partners</a></span><br /><span class="lofToc">d <a 
href="#x1-189031r4">30 SL partners</a></span><br /><span class="lofToc">e <a 
href="#x1-189032r5">40 SL partners</a></span><br /><span class="lofToc">f <a 
href="#x1-189033r6">50 SL partners</a></span><br /><span class="lofToc">a <a 
href="#x1-189035r1">1 SL partner</a></span><br /><span class="lofToc">b <a 
href="#x1-189036r2">2 SL partners</a></span><br /><span class="lofToc">c <a 
href="#x1-189037r3">3 SL partners</a></span><br /><span class="lofToc">d <a 
href="#x1-189038r4">4 SL partners</a></span><br /><span class="lofToc">e <a 
href="#x1-189039r5">5 SL partners</a></span><br /><span class="lofToc">f <a 
href="#x1-189040r6">6 SL partners</a></span><br /><span class="lofToc">g <a 
href="#x1-189041r7">7 SL partners</a></span><br /><span class="lofToc">h <a 
href="#x1-189042r8">8 SL partners</a></span><br /><span class="lofToc">i <a 
href="#x1-189043r9">9 SL partners</a></span><br /><span class="lofToc">J.10 <a 
href="#x1-18904410">Performance of <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> and SLIPT across quantiles with more genes</a></span><br /><span class="lofToc">a <a 
href="#x1-189045r1">10 SL partners</a></span><br /><span class="lofToc">b <a 
href="#x1-189046r2">15 SL partners</a></span><br /><span class="lofToc">c <a 
href="#x1-189047r3">20 SL partners</a></span><br /><span class="lofToc">d <a 
href="#x1-189048r4">30 SL partners</a></span><br /><span class="lofToc">e <a 
href="#x1-189049r5">40 SL partners</a></span><br /><span class="lofToc">f <a 
href="#x1-189050r6">50 SL partners</a></span><br /><span class="lofToc">a <a 
href="#x1-190001r1">1 SL partner</a></span><br /><span class="lofToc">b <a 
href="#x1-190002r2">2 SL partners</a></span><br /><span class="lofToc">c <a 
href="#x1-190003r3">3 SL partners</a></span><br /><span class="lofToc">d <a 
href="#x1-190004r4">4 SL partners</a></span><br /><span class="lofToc">e <a 
href="#x1-190005r5">5 SL partners</a></span><br /><span class="lofToc">f <a 
href="#x1-190006r6">6 SL partners</a></span><br /><span class="lofToc">g <a 
href="#x1-190007r7">7 SL partners</a></span><br /><span class="lofToc">h <a 
href="#x1-190008r8">8 SL partners</a></span><br /><span class="lofToc">i <a 
href="#x1-190009r9">9 SL partners</a></span><br /><span class="lofToc">J.14 <a 
href="#x1-19001014">Performance of <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> and SLIPT across quantiles with query correlation</a></span><br /><span class="lofToc">a <a 
href="#x1-190011r1">10 SL partners</a></span><br /><span class="lofToc">b <a 
href="#x1-190012r2">15 SL partners</a></span><br /><span class="lofToc">c <a 
href="#x1-190013r3">20 SL partners</a></span><br /><span class="lofToc">d <a 
href="#x1-190014r4">30 SL partners</a></span><br /><span class="lofToc">e <a 
href="#x1-190015r5">40 SL partners</a></span><br /><span class="lofToc">f <a 
href="#x1-190016r6">50 SL partners</a></span><br /><span class="lofToc">a <a 
href="#x1-190018r1">1 SL partner</a></span><br /><span class="lofToc">b <a 
href="#x1-190019r2">2 SL partners</a></span><br /><span class="lofToc" >c <a 
href="#x1-190020r3">3</p>
<p>SL partners</a></span><br /><span class="lofToc">d <a 
href="#x1-190021r4">4 SL partners</a></span><br /><span class="lofToc">e <a 
href="#x1-190022r5">5 SL partners</a></span><br /><span class="lofToc">f <a 
href="#x1-190023r6">6 SL partners</a></span><br /><span class="lofToc">g <a 
href="#x1-190024r7">7 SL partners</a></span><br /><span class="lofToc">h <a 
href="#x1-190025r8">8 SL partners</a></span><br /><span class="lofToc">i <a 
href="#x1-190026r9">9 SL partners</a></span><br /><span class="lofToc">J.18 <a 
href="#x1-19002718">Performance of <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> and SLIPT across quantiles with query correlation</a></span><br /><span class="lofToc">a <a 
href="#x1-190028r1">10 SL partners</a></span><br /><span class="lofToc">b <a 
href="#x1-190029r2">15 SL partners</a></span><br /><span class="lofToc">c <a 
href="#x1-190030r3">20 SL partners</a></span><br /><span class="lofToc">d <a 
href="#x1-190031r4">30 SL partners</a></span><br /><span class="lofToc">e <a 
href="#x1-190032r5">40 SL partners</a></span><br /><span class="lofToc">f <a 
href="#x1-190033r6">50 SL partners</a></span><br /><span class="lofToc">a <a 
href="#x1-190035r1">1 SL partner</a></span><br /><span class="lofToc">b <a 
href="#x1-190036r2">2 SL partners</a></span><br /><span class="lofToc">c <a 
href="#x1-190037r3">3 SL partners</a></span><br /><span class="lofToc">d <a 
href="#x1-190038r4">4 SL partners</a></span><br /><span class="lofToc">e <a 
href="#x1-190039r5">5 SL partners</a></span><br /><span class="lofToc">f <a 
href="#x1-190040r6">6 SL partners</a></span><br /><span class="lofToc">g <a 
href="#x1-190041r7">7 SL partners</a></span><br /><span class="lofToc">h <a 
href="#x1-190042r8">8 SL partners</a></span><br /><span class="lofToc">i <a 
href="#x1-190043r9">9 SL partners</a></span><br /><span class="lofToc">J.22 <a 
href="#x1-19004422">Performance of <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> and SLIPT across quantiles with query correlation and more genes</a></span><br /><span class="lofToc">a <a 
href="#x1-190045r1">10 SL partners</a></span><br /><span class="lofToc">b <a 
href="#x1-190046r2">15 SL partners</a></span><br /><span class="lofToc">c <a 
href="#x1-190047r3">20 SL partners</a></span><br /><span class="lofToc">d <a 
href="#x1-190048r4">30 SL partners</a></span><br /><span class="lofToc">e <a 
href="#x1-190049r5">40 SL partners</a></span><br /><span class="lofToc">f <a 
href="#x1-190050r6">50 SL partners</a></span><br /><span class="lofToc">a <a 
href="#x1-191001r1">Statistical evaluation </a></span><br /><span class="lofToc">b <a 
href="#x1-191002r2">ROC </a></span><br /><span class="lofToc">c <a 
href="#x1-191003r3">Graph </a></span><br /><span class="lofToc">d <a 
href="#x1-191004r4">Statistical performance </a></span><br /><span class="lofToc">K.2 <a 
href="#x1-1910052">Performance of simulations on a simple graph</a></span><br /><span class="lofToc">a <a 
href="#x1-192001r1">Statistical evaluation </a></span><br /><span class="lofToc">b <a 
href="#x1-192002r2">ROC </a></span><br /><span class="lofToc">c <a 
href="#x1-192003r3">Graph Structure </a></span><br /><span class="lofToc">d <a 
href="#x1-192004r4">Statistical performance </a></span><br /><span class="lofToc">K.4 <a 
href="#x1-1920054">Performance of simulations on an inhibiting graph</a></span><br /><span class="lofToc">a <a 
href="#x1-192006r1">Statistical evaluation</a></span><br /><span class="lofToc">b <a 
href="#x1-192007r2">ROC </a></span><br /><span class="lofToc">c <a 
href="#x1-192008r3">Graph Structure</a></span><br /><span class="lofToc">d <a 
href="#x1-192009r4">Statistical performance </a></span><br /><span class="lofToc">K.6 <a 
href="#x1-1920106">Performance of simulations on a constructed graph with inhibition</a></span><br /><span class="lofToc">a <a 
href="#x1-192011r1">Statistical evaluation</a></span><br /><span class="lofToc">b <a 
href="#x1-192012r2">ROC </a></span><br /><span class="lofToc">c <a 
href="#x1-192013r3">Graph Structure</a></span><br /><span class="lofToc">d <a 
href="#x1-192014r4">Statistical performance </a></span><br /><span class="lofToc">K.8 <a 
href="#x1-1920158">Performance of simulations on a constructed graph with inhibition</a></span><br /><span class="lofToc">a <a 
href="#x1-193001r1">Activating Graph Structure</a></span><br /><span class="lofToc">b <a 
href="#x1-193002r2"><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> distribution for “A” SL</a></span><br /><span class="lofToc">c <a 
href="#x1-193003r3">Gene “B” SL</a></span><br /><span class="lofToc">d <a 
href="#x1-193004r4">Gene “C” SL</a></span><br /><span class="lofToc">e <a 
href="#x1-193005r5">Gene “D” SL</a></span><br /><span class="lofToc">f <a 
href="#x1-193006r6">Gene “E” SL</a></span><br /><span class="lofToc">K.10 <a 
href="#x1-19300710">Detection of synthetic lethality within a graph structure</a></span><br /><span class="lofToc">a <a 
href="#x1-193008r1">Gene “F” SL</a></span><br /><span class="lofToc">b <a 
href="#x1-193009r2">Gene “G” SL</a></span><br /><span class="lofToc">c <a 
href="#x1-193010r3">Gene “H” SL</a></span><br /><span class="lofToc">d <a 
href="#x1-193011r4">Gene “I” SL</a></span><br /><span class="lofToc">a <a 
href="#x1-193013r1">Inhibiting Graph Structure</a></span><br /><span class="lofToc">b <a 
href="#x1-193014r2"><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> distribution for “A” SL</a></span><br /><span class="lofToc">c <a 
href="#x1-193015r3">Gene “B” SL</a></span><br /><span class="lofToc">d <a 
href="#x1-193016r4">Gene “C” SL</a></span><br /><span class="lofToc">e <a 
href="#x1-193017r5">Gene “D” SL</a></span><br /><span class="lofToc">f <a 
href="#x1-193018r6">Gene “E” SL</a></span><br /><span class="lofToc">g <a 
href="#x1-193019r7">Gene “F” SL</a></span><br /><span class="lofToc">h <a 
href="#x1-193020r8">Gene “G” SL</a></span><br /><span class="lofToc">i <a 
href="#x1-193021r9">Gene “H” SL</a></span><br /><span class="lofToc">j <a 
href="#x1-193022r10">Gene “I” SL</a></span><br /><span class="lofToc">K.14 <a 
href="#x1-19302314">Detection of synthetic lethality within an inhibiting graph</a></span><br /><span class="lofToc">a <a 
href="#x1-193025r1">Inhibiting Graph Structure</a></span><br /><span class="lofToc">b <a 
href="#x1-193026r2"><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> distribution for “A” SL</a></span><br /><span class="lofToc">c <a 
href="#x1-193027r3">Gene “B” SL</a></span><br /><span class="lofToc">d <a 
href="#x1-193028r4">Gene “C” SL</a></span><br /><span class="lofToc">e <a 
href="#x1-193029r5">Gene “D” SL</a></span><br /><span class="lofToc">f <a 
href="#x1-193030r6">Gene “E” SL</a></span><br /><span class="lofToc">g <a 
href="#x1-193031r7">Gene “F” SL</a></span><br /><span class="lofToc">h <a 
href="#x1-193032r8">Gene “G” SL</a></span><br /><span class="lofToc">i <a 
href="#x1-193033r9">Gene “H” SL</a></span><br /><span class="lofToc">j <a 
href="#x1-193034r10">Gene “I” SL</a></span><br /><span class="lofToc">K.15 <a 
href="#x1-19303515">Detection of synthetic lethality within an inhibiting graph</a></span><br /><span class="lofToc">a <a 
href="#x1-194001r1">Statistical evaluation </a></span><br /><span class="lofToc">b <a 
href="#x1-194002r2">ROC </a></span><br /><span class="lofToc">c <a 
href="#x1-194003r3">Graph Structure </a></span><br /><span class="lofToc">d <a 
href="#x1-194004r4">Statistical performance </a></span><br /><span class="lofToc">K.16 <a 
href="#x1-19400516">Performance of simulations on a branching graph</a></span><br /><span class="lofToc">a <a 
href="#x1-194006r1">Statistical evaluation </a></span><br /><span class="lofToc">b <a 
href="#x1-194007r2">ROC </a></span><br /><span class="lofToc">c <a 
href="#x1-194008r3">Graph Structure </a></span><br /><span class="lofToc">d <a 
href="#x1-194009r4">Statistical performance </a></span><br /><span class="lofToc">K.18 <a 
href="#x1-19401018">Performance of simulations on a complex graph</a></span><br /><span class="lofToc">a <a 
href="#x1-194011r1">Statistical evaluation </a></span><br /><span class="lofToc">b <a 
href="#x1-194012r2">ROC </a></span><br /><span class="lofToc">c <a 
href="#x1-194013r3">Graph Structure</a></span><br /><span class="lofToc">d <a 
href="#x1-194014r4">Statistical performance </a></span><br /><span class="lofToc">K.20 <a 
href="#x1-19401520">Performance of simulations on a large graph</a></span><br /><span class="lofToc">a <a 
href="#x1-195001r1">Statistical evaluation</a></span><br /><span class="lofToc">b <a 
href="#x1-195002r2">ROC </a></span><br /><span class="lofToc">c <a 
href="#x1-195003r3">Graph Structure</a></span><br /><span class="lofToc">d <a 
href="#x1-195004r4">Statistical performance </a></span><br /><span class="lofToc">K.22 <a 
href="#x1-19500522">Performance of simulations on a branching graph with inhibition</a></span><br /><span class="lofToc">a <a 
href="#x1-195006r1">Statistical evaluation</a></span><br /><span class="lofToc">b <a 
href="#x1-195007r2">ROC </a></span><br /><span class="lofToc">c <a 
href="#x1-195008r3">Graph Structure</a></span><br /><span class="lofToc">d <a 
href="#x1-195009r4">Statistical performance </a></span><br /><span class="lofToc">K.24 <a 
href="#x1-19501024">Performance of simulations on a branching graph with inhibition</a></span><br /><span class="lofToc">a <a 
href="#x1-195011r1">Statistical evaluation</a></span><br /><span class="lofToc">b <a 
href="#x1-195012r2">ROC </a></span><br /><span class="lofToc">c <a 
href="#x1-195013r3">Graph Structure</a></span><br /><span class="lofToc">d <a 
href="#x1-195014r4">Statistical performance </a></span><br /><span class="lofToc">K.26 <a 
href="#x1-19501526">Performance of simulations on a complex graph with inhibition</a></span><br /><span class="lofToc">a <a 
href="#x1-195016r1">Statistical evaluation</a></span><br /><span class="lofToc">b <a 
href="#x1-195017r2">ROC </a></span><br /><span class="lofToc">c <a 
href="#x1-195018r3">Graph Structure</a></span><br /><span class="lofToc">d <a 
href="#x1-195019r4">Statistical performance </a></span><br /><span class="lofToc">K.28 <a 
href="#x1-19502028">Performance of simulations on a complex graph with inhibition</a></span><br /><span class="lofToc">a <a 
href="#x1-195021r1">Statistical evaluation</a></span><br /><span class="lofToc">b <a 
href="#x1-195022r2">ROC </a></span><br /><span class="lofToc">c <a 
href="#x1-195023r3">Graph Structure</a></span><br /><span class="lofToc">d <a 
href="#x1-195024r4">Statistical performance </a></span><br /><span class="lofToc">K.30 <a 
href="#x1-19502530">Performance of simulations on a large constructed graph with inhibition</a></span><br /><span class="lofToc">a <a 
href="#x1-195026r1">Statistical evaluation</a></span><br /><span class="lofToc">b <a 
href="#x1-195027r2">ROC </a></span><br /><span class="lofToc">c <a 
href="#x1-195028r3">Graph Structure</a></span><br /><span class="lofToc">d <a 
href="#x1-195029r4">Statistical performance </a></span><br /><span class="lofToc">K.32 <a 
href="#x1-19503032">Performance of simulations on a large constructed graph with inhibition</a></span><br /><span class="lofToc">a <a 
href="#x1-196001r1">Statistical evaluation</a></span><br /><span class="lofToc">b <a 
href="#x1-196002r2">ROC</a></span><br /><span class="lofToc">c <a 
href="#x1-196003r3">Graph Structure</a></span><br /><span class="lofToc">d <a 
href="#x1-196004r4">Statistical performance</a></span><br /><span class="lofToc">K.34 <a 
href="#x1-19600534">Performance of simulations on the G<sub><span class="cmmi-8">αi</span></sub> signalling pathway</a></span><br /><span class="lofToc">a <a 
href="#x1-196006r1">Statistical evaluation</a></span><br /><span class="lofToc">b <a 
href="#x1-196007r2">ROC</a></span><br /><span class="lofToc">c <a 
href="#x1-196008r3">Graph Structure</a></span><br /><span class="lofToc" >d <a 
href="#x1-196009r4">Statistical</p>
performance</a></span><br /><span class="lofToc">K.36 <a 
href="#x1-19601036">Performance of simulations including the G<sub><span class="cmmi-8">αi</span></sub> signalling pathway</a></span><br />
</div>
                                                                                    

<p><a 
 id="likesection.1"></a><a 
 id="Q1-1-4"></a></p>
<h2 class="likechapterHead">
<a 
 id="x1-4000"></a>Glossary
</h2>
                                                                                    

<div class="supertabular"><table id="TBL-2" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-2-1g"><col 
id="TBL-2-1" /><col 
id="TBL-2-2" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-2-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-1-1"  
class="td11">                     </td></tr><tr  
 style="vertical-align:baseline;" id="TBL-2-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-2-1"  
class="td11"></td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-3-1"  
class="td11">                      </td></tr><tr  
 style="vertical-align:baseline;" id="TBL-2-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-4-1"  
class="td11"></td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-5-1"  
class="td11"> 
<a 
 id="glo:allele"></a>allele
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-5-2"  
class="td11"> 
A gene variant with a specific sequence and phenotype
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-6-1"  
class="td11"> 
<a 
 id="glo:bioinformatics"></a>bioinformatics
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-6-2"  
class="td11"> 
Statistical or computational approaches to biological data or research tools
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-7-1"  
class="td11"> 
<a 
 id="glo:cancer"></a>cancer
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-7-2"  
class="td11"> 
A class of diseases, formally “malignant neoplasm”, of abnormal cellular growth and spread to other organs
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-8-1"  
class="td11"> 
<a 
 id="glo:cancer gene"></a>cancer gene
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-8-2"  
class="td11"> 
A gene which is involved in the malignancy of some cancers, encompassing <a 
href="#glo:oncogene">oncogenes</a> and <a 
href="#glo:tumour suppressor">tumour suppressors</a>, which have molecular aberrations in cancer or variants which predispose individuals to cancer
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-9-1"  
class="td11"> 
<a 
 id="glo:centrality"></a>centrality
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-9-2"  
class="td11"> 
A network metric which identifies important <a 
href="#glo:vertex">vertices</a>
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-10-1"  
class="td11"> 
<a 
 id="glo:chemoprevention"></a>chemoprevention
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-10-2"  
class="td11"> 
The use of drugs to prevent early-stage cancers, generally applied to high-risk mutation carriers
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-11-1"  
class="td11"> 
<a 
 id="glo:chemotherapy"></a>chemotherapy
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-11-2"  
class="td11"> 
The use of cytotoxic drugs to treat cancers, in combinations, generally applied to advanced stage cancers
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-12-1"  
class="td11"> 
<a 
 id="glo:compound screen"></a>compound screen
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-12-2"  
class="td11"> 
A <a 
href="#glo:high-throughput screen">high-throughput screen</a> performed using a library of chemical compounds
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-13-1"  
class="td11"> 
<a 
 id="glo:computational biology"></a>computational biology
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-13-2"  
class="td11"> 
Applying computational or mathematical modelling to understanding biological systems and relationships
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-14-1"  
class="td11"> 
<a 
 id="glo:driver mutation"></a>driver mutation
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-14-2"  
class="td11"> 
A <a 
href="#glo:mutation">mutation</a> which promotes cancer growth
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-15-1"  
class="td11"> 
<a 
 id="glo:E-cadherin"></a>E-cadherin
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-15-2"  
class="td11"> 
Epithelial cadherin (calcium-dependent adhesion), a cell-adhesion protein encoded by <span class="cmti-12">CDH1</span>
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-16-1"  
class="td11"> 
<a 
 id="glo:edge"></a>edge or link
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-16-2"  
class="td11"> 
A relationship connecting a pair of elements of a graph structure or network, may be weighted or directional
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-17-1"  
class="td11"> 
<a 
 id="glo:epistasis_bio"></a>epistasis (biological)
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-17-2"  
class="td11"> 
The effects of a gene modifying or masking the phenotype of another gene
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-18-1"  
class="td11"> 
<a 
 id="glo:epistasis_stat"></a>epistasis (statistical)
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-18-2"  
class="td11"> 
A divergence of the observed double <a 
href="#glo:mutant">mutant</a> phenotype from that expected based on the respective phenotypes of single <a 
href="#glo:mutant">mutant</a> (<a 
href="#XFisher1919">Fisher</a>, <a 
href="#XFisher1919">1919</a>)
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-19-1"  
class="td11"> 
<a 
 id="glo:essential"></a>essential
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-19-2"  
class="td11"> 
A gene which is required to be functional or expressed for a cell or organism to be viable, grow or develop
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-20-1"  
class="td11"> 
<a 
 id="glo:familial"></a>familial
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-20-2"  
class="td11"> 
A trait recurrently occurring in families, not necessarily with a genetic cause
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-21-1"  
class="td11"> 
<a 
 id="glo:functional redundancy"></a>functional redundancy
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-21-2"  
class="td11"> 
Genes which perform a common function, also known as genetic redundancy
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-22-1"  
class="td11"> 
<a 
 id="glo:gene expression"></a>gene expression
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-22-2"  
class="td11"> 
A measure of the relative expression of each gene from the mRNA extracted from (pooled) cells
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-23-1"  
class="td11"> 
<a 
 id="glo:genetic robustness"></a>genetic robustness
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-23-2"  
class="td11"> 
A system of biological pathways which (has evolved to) continue to function as a whole under various conditions, including the inactivation of various individual genes
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-24-1"  
class="td11"> 
<a 
 id="glo:genome"></a>genome
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-24-2"  
class="td11"> 
All of the DNA sequence in the genome
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-25-1"  
class="td11"> 
<a 
 id="glo:genomic"></a>genomic
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-25-2"  
class="td11"> 
The use of data from all genes in the genome
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-26-1"  
class="td11"> 
<a 
 id="glo:germline"></a>germline mutation
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-26-2"  
class="td11"> 
A <a 
href="#glo:mutation">mutation</a> that occurred in germline cells and is passed between generation
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-27-1"  
class="td11"> 
<a 
 id="glo:graph"></a>graph or network
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-27-2"  
class="td11"> 
A mathematical structure modelling or depicting the relationships between elements
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-28-1"  
class="td11"> 
<a 
 id="glo:hereditary"></a>hereditary
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-28-2"  
class="td11"> 
A trait or disease which has a genetic cause and is inherited from family members
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-29-1"  
class="td11"> 
<a 
 id="glo:high-throughput screen"></a>high-throughput screen
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-29-2"  
class="td11"> 
An experimental procedure to perform a large scale series of chemical, genetic, or pharmacological tests
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-30-1"  
class="td11"> 
<a 
 id="glo:hub"></a>hub
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-30-2"  
class="td11"> 
A central or highly connected component of a network
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-31-1"  
class="td11"> 
<a 
 id="glo:induced essentiality"></a>induced essentiality
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-31-2"  
class="td11"> 
A gene becoming <a 
href="#glo:essential">essential</a> to viability under certain conditions, including inactivation of a synthetic lethal partner
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-32-1"  
class="td11"> 
<a 
 id="glo:information centrality"></a>information centrality
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-32-2"  
class="td11"> 
A network <a 
href="#glo:centrality">centrality</a> metric which uses the impact of removing a <a 
href="#glo:vertex">vertex or node</a> on connections in the network
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-33-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-33-1"  
class="td11"> 
<a 
 id="glo:intrinsic subtype"></a>intrinsic subtype
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-33-2"  
class="td11"> 
Distinguishing cancer by molecular and genetic features
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-34-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-34-1"  
class="td11"> 
<a 
 id="glo:metagene"></a>metagene
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-34-2"  
class="td11"> 
A consistent signal of expression for a collection of genes such as a biological pathway, derived from singular value decomposition
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-35-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-35-1"  
class="td11"> 
<a 
 id="glo:methylation"></a>methylation
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-35-2"  
class="td11"> 
A measure of the epigenetic regulation of DNA at <a 
href="#glo:CpG">CpG dinucleotide (CpG)</a> sites
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-36-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-36-1"  
class="td11"> 
<a 
 id="glo:microarray"></a>microarray
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-36-2"  
class="td11"> 
A high-throughput technique to measure presence or abundance of nucleic acid sequences from binding to probes
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-37-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-37-1"  
class="td11"> 
<a 
 id="glo:molecular profile"></a>molecular profile
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-37-2"  
class="td11"> 
A combination of genetic and biochemical measures which identifies characteristic traits of a tumour
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-2-38-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-38-1"  
class="td11"> 
<a 
 id="glo:molecular subtype"></a>molecular subtype
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-38-2"  
class="td11"> 
A classification of cancers based on an identification using molecular properties
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-39-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-39-1"  
class="td11"> 
<a 
 id="glo:mutant"></a>mutant
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-39-2"  
class="td11"> 
A variant or dysfunctional phenotype arising from a <a 
href="#glo:mutation">mutation</a> in a gene
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-40-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-40-1"  
class="td11"> 
<a 
 id="glo:mutation"></a>mutation
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-40-2"  
class="td11"> 
A change in DNA sequence that disrupts gene function
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-41-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-41-1"  
class="td11"> 
<a 
 id="glo:non-oncogene addiction"></a>non-oncogene addiction
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-41-2"  
class="td11"> 
The dependence of a cancer cell on functioning non-mutant genes
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-42-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-42-1"  
class="td11"> 
<a 
 id="glo:omics"></a>’omics
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-42-2"  
class="td11"> 
A combination of approaches to generating biological data with high-throughput procedures such as genomics, proteomics or metabolomics
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-43-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-43-1"  
class="td11"> 
<a 
 id="glo:oncogene"></a>oncogene
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-43-2"  
class="td11"> 
A gene that potentially causes cancer, typically by over-expression or mutant gene variants
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-44-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-44-1"  
class="td11"> 
<a 
 id="glo:oncogene addiction"></a>oncogene addiction
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-44-2"  
class="td11"> 
The dependence of a cancer cell on a specific oncogenic pathway
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-45-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-45-1"  
class="td11"> 
<a 
 id="glo:PageRank centrality"></a>PageRank centrality
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-45-2"  
class="td11"> 
A network <a 
href="#glo:centrality">centrality</a> metric which uses eigenvectors with a scaling factor (<a 
href="#XBrin1998">Brin and Page</a>, <a 
href="#XBrin1998">1998</a>)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-46-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-46-1"  
class="td11"> 
<a 
 id="glo:pan cancer"></a>pan cancer
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-46-2"  
class="td11"> 
A focus on the molecular and genetic features across cancers in different tissues
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-47-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-47-1"  
class="td11"> 
<a 
 id="glo:passenger mutation"></a>passenger mutation
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-47-2"  
class="td11"> 
A <a 
href="#glo:mutation">mutation</a> that occurs in cancers but does not affect the growth of cancers
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-48-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-48-1"  
class="td11"> 
<a 
 id="glo:pathway"></a>pathway
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-48-2"  
class="td11"> 
A series of biomolecules that produces a particular product or biological function
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-49-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-49-1"  
class="td11"> 
<a 
 id="glo:pleiotropy"></a>pleiotropy
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-49-2"  
class="td11"> 
When a gene has multiple biological functions
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-50-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-50-1"  
class="td11"> 
<a 
 id="glo:precision medicine"></a>precision medicine
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-50-2"  
class="td11"> 
The application of prevention and treatment measures to target diseases by molecular and genetic features
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-51-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-51-1"  
class="td11"> 
<a 
 id="glo:recurrent mutation"></a>recurrent mutation
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-51-2"  
class="td11"> 
The repeated occurrence of mutations in a particular gene across cancers
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-52-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-52-1"  
class="td11"> 
<a 
 id="glo:RNAi screen"></a>RNAi screen
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-52-2"  
class="td11"> 
A <a 
href="#glo:high-throughput screen">high-throughput screen</a> performed using a <a 
href="#glo:RNAi">RNA interference</a> (<a 
href="#glo:RNAi">RNAi</a>)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-53-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-53-1"  
class="td11"> 
<a 
 id="glo:RNA-Seq"></a>RNA-Seq
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-53-2"  
class="td11"> 
The generation of transcriptome data from sequencing RNA
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-54-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-54-1"  
class="td11"> 
<a 
 id="glo:scale-free"></a>scale-free
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-54-2"  
class="td11"> 
A property of a network which has a power law <a 
href="#glo:vertex degree">vertex degree</a> distribution, that is several highly connected <a 
href="#glo:hub">hub</a> genes and many with very few connections
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-55-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-55-1"  
class="td11"> 
<a 
 id="glo:shortest path"></a>shortest path
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-55-2"  
class="td11"> 
A path with the fewest possible <a 
href="#glo:edge">edges</a> which connects two particular <a 
href="#glo:vertex">vertices</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-56-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-56-1"  
class="td11"> 
<a 
 id="glo:small world"></a>small world
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-56-2"  
class="td11"> 
A property of a network which is highly connected and has a low characteristic path length, derived from the mean <a 
href="#glo:shortest path">shortest path</a> length across all pairs of nodes
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-57-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-57-1"  
class="td11"> 
<a 
 id="glo:somatic"></a>somatic mutation
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-57-2"  
class="td11"> 
A <a 
href="#glo:mutation">mutation</a> that occurs in somatic cells, during a patient’s lifespan
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-58-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-58-1"  
class="td11"> 
<a 
 id="glo:sporadic"></a>sporadic cancer
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-58-2"  
class="td11"> 
Cancers which do occur in patients with a family history or carry a high-risk genetic variant
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-59-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-59-1"  
class="td11"> 
<a 
 id="glo:synthetic dosage lethal"></a>synthetic dosage lethal
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-59-2"  
class="td11"> 
A <a 
href="#glo:SGI">synthetic genetic interaction</a> (<a 
href="#glo:SGI">SGI</a>) analogous to <a 
href="#glo:synthetic lethal">synthetic lethality</a> where where one gene is inactivated and the other over-expressed
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-60-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-60-1"  
class="td11"> 
<a 
 id="glo:synthetic lethal"></a>synthetic lethal
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-60-2"  
class="td11"> 
Genetic interactions where inactivation of multiple genes is inviable (or deleterious) which are viable if inactivated separately
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-61-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-61-1"  
class="td11"> 
<a 
 id="glo:synthetic lethal screen"></a>synthetic lethal screen
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-61-2"  
class="td11"> 
A <a 
href="#glo:high-throughput screen">high-throughput screen</a> performed on isogenic cell lines to detect genes for which inhibition specifically deleterious to the null <a 
href="#glo:mutant">mutant</a> genotype
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-62-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-62-1"  
class="td11"> 
<a 
 id="glo:synthetic rescue"></a>synthetic rescue
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-62-2"  
class="td11"> 
A <a 
href="#glo:SGI">synthetic genetic interaction</a> when the combined <a 
href="#glo:mutation">mutations</a> restores the <a 
href="#glo:wild-type">wild-type</a> the phenotype of one of the <a 
href="#glo:mutation">mutations</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-63-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-63-1"  
class="td11"> 
<a 
 id="glo:synthetic sick"></a>synthetic sick
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-63-2"  
class="td11"> 
Genetic interactions where inactivation of multiple genes is deleterious which are viable if inactivated separately
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-64-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-64-1"  
class="td11"> 
<a 
 id="glo:synthetic suppression"></a>synthetic suppression
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-64-2"  
class="td11"> 
A <a 
href="#glo:SGI">synthetic genetic interaction</a> when the combined <a 
href="#glo:mutation">mutations</a> (partially) suppresses the <a 
href="#glo:mutant">mutant</a> phenotype of one of the <a 
href="#glo:mutation">mutations</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-65-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-65-1"  
class="td11"> 
<a 
 id="glo:targeted therapy"></a>targeted therapy
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-65-2"  
class="td11"> 
Cancer treatment that specifically acts against a molecular target, in contrast to standard chemotherapy
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-66-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-66-1"  
class="td11"> 
<a 
 id="glo:transcriptome"></a>transcriptome
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-66-2"  
class="td11"> 
All of the genes expressed in the genome
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-67-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-67-1"  
class="td11"> 
<a 
 id="glo:treatment"></a>treatment
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-67-2"  
class="td11"> 
Medical procedures for a disease to improve patient outcomes
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-68-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-68-1"  
class="td11"> 
<a 
 id="glo:tumour"></a>tumour
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-68-2"  
class="td11"> 
An abnormal lump of tissue or growth of cells, may be cancerous
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-69-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-69-1"  
class="td11"> 
<a 
 id="glo:tumour suppressor"></a>tumour suppressor
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-69-2"  
class="td11"> 
A gene potentially causes cancer, typically by disruption of functions which protect the cell from cancer
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-70-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-70-1"  
class="td11"> 
<a 
 id="glo:vertex degree"></a>vertex degree
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-70-2"  
class="td11"> 
A network metric of connectivity of <a 
href="#glo:vertex">vertices</a> which uses the number of edges connected to each <a 
href="#glo:vertex">vertex or node</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-71-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-71-1"  
class="td11"> 
<a 
 id="glo:vertex"></a>vertex or node
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-71-2"  
class="td11"> 
An element of a graph structure or network
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-72-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-72-1"  
class="td11"> 
<a 
 id="glo:wild-type"></a>wild-type
</td><td  style="white-space:wrap; text-align:left;" id="TBL-2-72-2"  
class="td11"> 
A natural phenotype of a trait or the normally functional <a 
href="#glo:allele">allele</a> which encodes it
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-73-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-73-1"  
class="td11">                      </td></tr></table></div>                                                    
                                                                                    


<!--l. 1--><p class="indent" >
<p><a 
 id="likesection.2"></a><a 
 id="Q1-1-6"></a></p>
</p>
<h2 class="likechapterHead">
<a 
 id="x1-5000"></a>Acronyms
</h2>
                                                                                    

<div class="supertabular"><table id="TBL-4" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-4-1g"><col 
id="TBL-4-1" /><col 
id="TBL-4-2" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-4-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-1-1"  
class="td11">          </td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-2-1"  
class="td11"></td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-3-1"  
class="td11">           </td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-4-1"  
class="td11"></td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-5-1"  
class="td11"> 
<a 
 id="glo:ADP"></a>ADP
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-5-2"  
class="td11"> 
Adenosine Diphosphate
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-6-1"  
class="td11"> 
<a 
 id="glo:AMP"></a>AMP
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-6-2"  
class="td11"> 
Adenosine Monophosphate
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-7-1"  
class="td11"> 
<a 
 id="glo:AMPK"></a>AMPK
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-7-2"  
class="td11"> 
<a 
href="#glo:AMP">AMP</a>-activated Protein Kinase
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-8-1"  
class="td11"> 
<a 
 id="glo:ANOVA"></a>ANOVA
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-8-2"  
class="td11"> 
Analysis of Variance
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-9-1"  
class="td11"> 
<a 
 id="glo:AUROC"></a>AUROC
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-9-2"  
class="td11"> 
Area Under the Receiver Operating Characteristic (curve)
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-10-1"  
class="td11"> 
<a 
 id="glo:Bash"></a>Bash
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-10-2"  
class="td11"> 
Bourne Again Shell
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-11-1"  
class="td11"> 
<a 
 id="glo:BioPAX"></a>BioPAX
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-11-2"  
class="td11"> 
Biological Pathway Exchange
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-12-1"  
class="td11"> 
<a 
 id="glo:BiSEp"></a>BiSEp
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-12-2"  
class="td11"> 
Bimodal Subsetting Expression
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-13-1"  
class="td11"> 
<a 
 id="glo:BMP"></a>BMP
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-13-2"  
class="td11"> 
Bone Morphogenic Protein
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-14-1"  
class="td11"> 
<a 
 id="glo:CCLE"></a>CCLE
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-14-2"  
class="td11"> 
Cancer Cell Line Encyclopaedia
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-15-1"  
class="td11"> 
<a 
 id="glo:cDNA"></a>cDNA
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-15-2"  
class="td11"> 
Complementary DNA (from mRNA)
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-16-1"  
class="td11"> 
<a 
 id="glo:CGP"></a>CGP
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-16-2"  
class="td11"> 
Cancer Genome Project
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-17-1"  
class="td11"> 
<a 
 id="glo:CNV"></a>CNV
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-17-2"  
class="td11"> 
Copy Number Variation
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-18-1"  
class="td11"> 
<a 
 id="glo:COSMIC"></a>COSMIC
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-18-2"  
class="td11"> 
Catalogue Of Somatic Mutations In Cancer
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-19-1"  
class="td11"> 
<a 
 id="glo:CpG"></a>CpG
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-19-2"  
class="td11"> 
5<span class="cmsy-8">′</span>–C–phosphate–G–3<span class="cmsy-8">′</span>
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-20-1"  
class="td11"> 
<a 
 id="glo:CPM"></a>CPM
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-20-2"  
class="td11"> 
Counts Per Million mapped reads
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-21-1"  
class="td11"> 
<a 
 id="glo:CPU"></a>CPU
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-21-2"  
class="td11"> 
Central Processing Unit
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-22-1"  
class="td11"> 
<a 
 id="glo:CRAN"></a>CRAN
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-22-2"  
class="td11"> 
comprehensive R archive network
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-23-1"  
class="td11"> 
<a 
 id="glo:CXCR"></a>CXCR
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-23-2"  
class="td11"> 
Chemokine Receptor
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-24-1"  
class="td11"> 
<a 
 id="glo:DAISY"></a>DAISY
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-24-2"  
class="td11"> 
Data Mining Synthetic Lethal Identification Pipeline
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-25-1"  
class="td11"> 
<a 
 id="glo:DNA"></a>DNA
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-25-2"  
class="td11"> 
Deoxyribonucleic Acid
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-26-1"  
class="td11"> 
<a 
 id="glo:EMT"></a>EMT
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-26-2"  
class="td11"> 
Epithelial-Mesenchymal Transition
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-27-1"  
class="td11"> 
<a 
 id="glo:ER"></a>ER
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-27-2"  
class="td11"> 
Estrogen Receptor
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-28-1"  
class="td11"> 
<a 
 id="glo:FDR"></a>FDR
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-28-2"  
class="td11"> 
False Discovery Rate
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-29-1"  
class="td11"> 
<a 
 id="glo:GO"></a>GO
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-29-2"  
class="td11"> 
Gene Ontology
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-30-1"  
class="td11"> 
<a 
 id="glo:GPCR"></a>GPCR
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-30-2"  
class="td11"> 
G Crotein Coupled Receptor
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-31-1"  
class="td11"> 
<a 
 id="glo:HDAC"></a>HDAC
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-31-2"  
class="td11"> 
Histone Deacetylase
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-32-1"  
class="td11"> 
<a 
 id="glo:HDGC"></a>HDGC
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-32-2"  
class="td11"> 
Hereditary Diffuse Gastric Cancer
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-33-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-33-1"  
class="td11"> 
<a 
 id="glo:HLRCC"></a>HLRCC
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-33-2"  
class="td11"> 
Hereditary Leiomyomatosis and Renal Cell Carcinoma
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-34-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-34-1"  
class="td11"> 
<a 
 id="glo:HPC"></a>HPC
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-34-2"  
class="td11"> 
High Performance Computing
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-35-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-35-1"  
class="td11"> 
<a 
 id="glo:ICGC"></a>ICGC
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-35-2"  
class="td11"> 
International Cancer Genome Consortium
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-36-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-36-1"  
class="td11"> 
<a 
 id="glo:JAK"></a>JAK
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-36-2"  
class="td11"> 
Janus Kinase
</td>
 </tr><tr  
 style="vertical-align:baseline;" id="TBL-4-37-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-37-1"  
class="td11"> 
<a 
 id="glo:microRNA"></a>microRNA
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-37-2"  
class="td11"> 
Micro RNA
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-38-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-38-1"  
class="td11"> 
<a 
 id="glo:mRNA"></a>mRNA
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-38-2"  
class="td11"> 
Messenger RNA
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-39-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-39-1"  
class="td11"> 
<a 
 id="glo:MSI"></a>MSI
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-39-2"  
class="td11"> 
Microsatellite Instability
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-40-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-40-1"  
class="td11"> 
<a 
 id="glo:mtSLIPT"></a>mtSLIPT
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-40-2"  
class="td11"> 
Synthetic Lethal Interaction Prediction Tool (against mutation)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-41-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-41-1"  
class="td11"> 
<a 
 id="glo:NeSI"></a>NeSI
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-41-2"  
class="td11"> 
New Zealand eScience Infrastructure
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-42-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-42-1"  
class="td11"> 
<a 
 id="glo:NGS"></a>NGS
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-42-2"  
class="td11"> 
Next-Generation Sequencing
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-43-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-43-1"  
class="td11"> 
<a 
 id="glo:NMD"></a>NMD
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-43-2"  
class="td11"> 
Nonsense-Mediated Decay
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-44-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-44-1"  
class="td11"> 
<a 
 id="glo:PAM50"></a>PAM50
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-44-2"  
class="td11"> 
Prediction Analysis of Microarray 50
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-45-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-45-1"  
class="td11"> 
<a 
 id="glo:PARP"></a>PARP
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-45-2"  
class="td11"> 
Poly-<a 
href="#glo:ADP">ADP</a>-Ribose Polymerase
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-46-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-46-1"  
class="td11"> 
<a 
 id="glo:PCR"></a>PCR
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-46-2"  
class="td11"> 
Polymerase Chain Reaction
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-47-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-47-1"  
class="td11"> 
<a 
 id="glo:PDE"></a>PDE
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-47-2"  
class="td11"> 
Phosphodiesterase
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-48-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-48-1"  
class="td11"> 
<a 
 id="glo:PI3K"></a>PI3K
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-48-2"  
class="td11"> 
Phosphoinositide 3-kinase
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-49-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-49-1"  
class="td11"> 
<a 
 id="glo:PPI"></a>PPI
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-49-2"  
class="td11"> 
Protein-Protein Interaction
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-50-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-50-1"  
class="td11"> 
<a 
 id="glo:PR"></a>PR
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-50-2"  
class="td11"> 
Progesterone Receptor
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-51-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-51-1"  
class="td11"> 
<a 
 id="glo:RGS"></a>RGS
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-51-2"  
class="td11"> 
G-protein Signalling
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-52-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-52-1"  
class="td11"> 
<a 
 id="glo:RHO"></a>RHO
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-52-2"  
class="td11"> 
Ras Homolog Family
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-53-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-53-1"  
class="td11"> 
<a 
 id="glo:RNA"></a>RNA
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-53-2"  
class="td11"> 
Ribonucleic Acid
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-54-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-54-1"  
class="td11"> 
<a 
 id="glo:RNAi"></a>RNAi
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-54-2"  
class="td11"> 
RNA Interference
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-55-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-55-1"  
class="td11"> 
<a 
 id="glo:ROC"></a>ROC
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-55-2"  
class="td11"> 
Reciever Operating Characteristic (curve)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-56-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-56-1"  
class="td11"> 
<a 
 id="glo:RSEM"></a>RSEM
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-56-2"  
class="td11"> 
RNA-Seq by Expectation Maximization (normalisation
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-57-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-57-1"  
class="td11"> 
<a 
 id="glo:SGA"></a>SGA
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-57-2"  
class="td11"> 
Synthetic Gene Array (technique)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-58-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-58-1"  
class="td11"> 
<a 
 id="glo:SGI"></a>SGI
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-58-2"  
class="td11"> 
Synthetic Genetic Interaction
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-59-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-59-1"  
class="td11"> 
<a 
 id="glo:shRNA"></a>shRNA
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-59-2"  
class="td11"> 
Short Hairpin RNA
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-60-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-60-1"  
class="td11"> 
<a 
 id="glo:siRNA"></a>siRNA
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-60-2"  
class="td11"> 
Short Interfering RNA
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-61-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-61-1"  
class="td11"> 
<a 
 id="glo:SL"></a>SL
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-61-2"  
class="td11"> 
Synthetic Lethal
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-62-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-62-1"  
class="td11"> 
<a 
 id="glo:SLIPT"></a>SLIPT
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-62-2"  
class="td11"> 
Synthetic Lethal Interaction Prediction Tool
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-63-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-63-1"  
class="td11"> 
<a 
 id="glo:Slurm"></a>Slurm
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-63-2"  
class="td11"> 
Simple Linux Utility for Resource Management
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-64-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-64-1"  
class="td11"> 
<a 
 id="glo:SNP"></a>SNP
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-64-2"  
class="td11"> 
Single Nucleotide Polymorphism
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-65-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-65-1"  
class="td11"> 
<a 
 id="glo:SOCKS"></a>SOCKS
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-65-2"  
class="td11"> 
Socket Secure
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-66-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-66-1"  
class="td11"> 
<a 
 id="glo:SR"></a>SR
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-66-2"  
class="td11"> 
Synthetic Rescue (or viability)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-67-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-67-1"  
class="td11"> 
<a 
 id="glo:SS"></a>SS
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-67-2"  
class="td11"> 
Synthetic Suppression
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-68-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-68-1"  
class="td11"> 
<a 
 id="glo:SSL"></a>SSL
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-68-2"  
class="td11"> 
Synthetic Sick
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-69-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-69-1"  
class="td11"> 
<a 
 id="glo:TCGA"></a>TCGA
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-69-2"  
class="td11"> 
The Cancer Genome Atlas (genomics project)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-70-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-70-1"  
class="td11"> 
<a 
 id="glo:TGFB"></a>TGF<span class="cmmi-12">β </span>
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-70-2"  
class="td11"> 
Transforming Growth Factor <span class="cmmi-12">β</span>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-71-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-71-1"  
class="td11"> 
<a 
 id="glo:UCSC"></a>UCSC
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-71-2"  
class="td11"> 
University of California, Santa Cruz
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-72-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-72-1"  
class="td11"> 
<a 
 id="glo:UTR"></a>UTR
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-72-2"  
class="td11"> 
Untranslated Region (of mRNA)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-73-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-73-1"  
class="td11"> 
<a 
 id="glo:WNT"></a>WNT
</td><td  style="white-space:wrap; text-align:left;" id="TBL-4-73-2"  
class="td11"> 
Wingless-Related Integration Site
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-74-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-74-1"  
class="td11">           </td></tr></table></div>                                                                 
                                                                                    






<h2 class="chapterHead">
<span class="titlemark">Chapter 1</span><br /><a 
 id="x1-60001"></a>Introduction and Literature Review
</h2>
<!--l. 11--><p class="noindent" >
This thesis presents research into genetic interactions using <a 
href="#glo:genomic">genomics</a> data and <a 
href="#glo:bioinformatics">bioinformatics</a> approaches. Chapter <a 
href="#x1-60001">1<!--tex4ht:ref: chap:intro --></a> introduces recent developments in <a 
href="#glo:genomic">genomics</a> and <a 
href="#glo:bioinformatics">bioinformatics</a>, particularly in their application to <a 
href="#glo:cancer">cancer</a> research. Studies of <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions, which have fundamental importance in genetics in model organisms and renewed relevance in <a 
href="#glo:cancer">cancer</a> biology specifically, will be discussed and reviewed in detail. A bioinformatic approach to <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions enables a wider exploration of the function of genes and proteins in <a 
href="#glo:cancer">cancer</a> cells, in contrast with candidate gene and experimental screening approaches. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> drug design aims to develop <a 
href="#glo:treatment">treatments</a> with specificity against loss of function <a 
href="#glo:mutation">mutations</a> in <a 
href="#glo:tumour suppressor">tumour suppressor</a> genes, such as <span class="cmti-12">CDH1 </span>(which encodes <a 
href="#glo:E-cadherin">E-cadherin</a>) and was the focus of the analysis in this thesis. The role of <span class="cmti-12">CDH1 </span>in cellular and <a 
href="#glo:cancer">cancer</a> biology is therefore also briefly reviewed. <a 
 id="x1-6001r1"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">1.1 </span> <a 
 id="x1-70001"></a>Cancer Research in the Post-Genomic Era
</h3>
<!--l. 21--><p class="noindent" >
<a 
href="#glo:genomic">Genomic</a> technologies are expected to significantly impact on the clinical treatment of <a 
href="#glo:cancer">cancers</a>, along with wider applications of genetics (<a 
href="#XGoodwin2016">Goodwin <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGoodwin2016">2016</a>; <a 
href="#XRoychowdhury2016">Roychowdhury and Chinnaiyan</a>, <a 
href="#XRoychowdhury2016">2016</a>). These technologies enable focused genetic investigations on candidate genes selected from <a 
href="#glo:bioinformatics">bioinformatic</a> analyses of <a 
href="#glo:genomic">genomic</a> data. Facilitated by rapidly developing technologies, large-scale projects have investigated populations (<a 
href="#X1000Genomes2010">1000 Genomes</a>, <a 
href="#X1000Genomes2010">2010</a>), <a 
href="#glo:cancer">cancers</a> (<a 
href="#XDickson1999">Dickson</a>, <a 
href="#XDickson1999">1999</a>; <a 
href="#XICGC2011">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XICGC2011">2011</a>), and functional <a 
href="#glo:genomic">genomics</a> (<a 
href="#XFANTOM2001">Kawai <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFANTOM2001">2001</a>; <a 
href="#XENCODE2004">ENCODE</a>, <a 
href="#XENCODE2004">2004</a>), however, <a 
href="#glo:genomic">genomic</a> technologies have yet to be widely adopted in healthcare or oncology (<a 
href="#XRoychowdhury2016">Roychowdhury and Chinnaiyan</a>, <a 
href="#XRoychowdhury2016">2016</a>; <a 
href="#XWaldron2016">Waldron</a>, <a 
href="#XWaldron2016">2016</a>). <a 
href="#glo:bioinformatics">Bioinformatics</a> analysis for interpretation of <a 
href="#glo:genomic">genomic</a> data is one of the main approaches to address this disparity (<a 
href="#XGoodwin2016">Goodwin <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGoodwin2016">2016</a>). Here, I outline the <a 
href="#glo:cancer">cancer</a> <a 
href="#glo:genomic">genomics</a> projects and findings which have led to the availability of <a 
href="#glo:genomic">genomics</a> data used in this thesis, and recent findings in <a 
href="#glo:cancer">cancer</a> research which demonstrate potential applications of using this data. <a 
 id="x1-7001r1"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.1.1 </span> <a 
 id="x1-80001"></a>Cancer is a Global Health Issue
</h4>
<!--l. 25--><p class="noindent" >
<p>Cancers are the second leading cause of death globally (<a 
href="#XWorldHealthOrg2017">WHO</a>, <a 
href="#XWorldHealthOrg2017">2017</a>), with an estimated annual incidence of 14.1 million cases and annual mortality of 8.2 million people (<a 
href="#XFerlay2015">Ferlay <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFerlay2015">2015</a>). Breast and stomach <a 
href="#glo:cancer">cancers</a> are among the most prevalent <a 
href="#glo:cancer">cancers</a>. Breast</p>
cancer is the most common <a 
href="#glo:cancer">cancer</a> in women and has an estimated annual incidence of 1.6 million cases and mortality of 520,000 people. Stomach cancer has an estimated annual incidence of 950,000 cases and a mortality of 723,000 people. Cancer is also a major health concern here in New Zealand, with 19,100 people (including 2500 cases of breast cancer and 370 cases of stomach cancer) diagnosed annually (<a 
href="#XCIX2013">Hanna</a>, <a 
href="#XCIX2013">2003</a>). New Zealand has among the highest incidence (age-standardised per capita) of <a 
href="#glo:cancer">cancer</a> in the world (<a 
href="#XFerlay2015">Ferlay <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFerlay2015">2015</a>).
</p><!--l. 27--><p class="indent" >
While environmental factors often play a role, genetics is an important contributor to cancer risk. Most <a 
href="#glo:cancer">cancers</a> occur more frequently with age and family history. Cancers arise from dysregulated cellular growth or differentiation. These can occur through genetic <a 
href="#glo:mutation">mutations</a> or alterations in gene regulation or <a 
href="#glo:gene expression">expression</a> which generally accumulate as the disease develops. Therefore, early diagnosis is important to ensure patient survival and quality of life. Identification of patients with genetic variants or family histories at a high-risk of particular cancers is an important health issue. These high-risk individuals are regularly monitored for some cancers and are sometimes offered preventative surgery (<a 
href="#XGuilford2010">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford2010">2010</a>; <a 
href="#XScheuer2002">Scheuer <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XScheuer2002">2002</a>).
</p><!--l. 35--><p class="indent" >
<a 
href="#glo:chemotherapy">Chemotherapy</a> is a treatment for many advanced stage cancers, designed to inhibit rapidly growing cells. However, this approach often has severe adverse effects, a narrow therapeutic window, and is not suitable for <a 
href="#glo:chemoprevention">chemopreventative</a> application in many cases (<a 
href="#XKaelin2009">Kaelin, Jr</a>, <a 
href="#XKaelin2009">2009</a>). Patients at high-risk of cancers are offered surveillence and preventative surgery but these approaches are not completely effective at preventing cancers and can severely impact on quality of life (<a 
href="#XGuilford2010">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford2010">2010</a>). Alternative <a 
href="#glo:chemoprevention">chemoprevention</a> and treatment strategies based on molecular biology and other fields are being investigated, including targeted molecular therapeutics (<a 
href="#XBozovic-Spasojevic2012">Bozovic-Spasojevic <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBozovic-Spasojevic2012">2012</a>). <a 
 id="x1-8001r1"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.1.1 </span> <a 
 id="x1-90001"></a>The Genetics and Molecular Biology of Cancers
</h5>
<!--l. 43--><p class="noindent" >
<p>Cancers involve dysregulation of genes, including <a 
href="#glo:somatic">somatic</a> and <a 
href="#glo:hereditary">hereditary</a> <a 
href="#glo:mutation">mutations</a>, which may predispose individuals to high-risk cancers and <a 
href="#glo:familial">familial</a> cancer syndromes (<a 
href="#XACS2017">American Cancer Society</a>, <a 
href="#XACS2017">2017</a>; <a 
href="#XGuilford1998">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford1998">1998</a>; <a 
href="#XStratton2009">Stratton <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XStratton2009">2009</a>; <a 
href="#XNCI2015">NCI</a>, <a 
href="#XNCI2015">2015</a>; <a 
href="#XVogelstein2013">Vogelstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XVogelstein2013">2013</a>). The occurence of <a 
href="#glo:somatic">somatic mutation</a> <a 
href="#glo:mutation">mutations</a> increases the risk of <a 
href="#glo:cancer">cancer</a> with age. An association of cancer incidence with the stem cell divisions in which <a 
href="#glo:mutation">mutations</a> could</p>
occur across tissue types, suggests that cancers may be inseparably coupled with aging (<a 
href="#XTomasetti2015">Tomasetti and Vogelstein</a>, <a 
href="#XTomasetti2015">2015</a>).
</p><!--l. 56--><p class="indent" >
<a 
href="#XHanahan2000">Hanahan and Weinberg</a> (<a 
href="#XHanahan2000">2000</a>) proposed the “hallmarks of cancer”, molecular and cellular traits shared across cancers. These form the basis of a rational approach to categorising the complex changes that occur in <a 
href="#glo:cancer">cancer</a>. These traits include limitless replication potential, signals for indefinite growth, and invasive or metastatic capabilities. <a 
href="#glo:cancer">Cancers</a> also evade apoptosis and the immune system, and sustain angiogenesis and energy metabolism (<a 
href="#XHanahan2011">Hanahan and Weinberg</a>, <a 
href="#XHanahan2011">2011</a>). To achieve this, <a 
href="#glo:cancer">cancer</a> cells change their <a 
href="#glo:genome">genomes</a> and the <a 
href="#glo:tumour">tumour</a> microenvironment. <a 
href="#glo:genomic">Genomic</a> instability has a role in the survival and proliferation of <a 
href="#glo:cancer">cancer</a> cells and the progression of disease, as these malignant characteristics are acquired. Identifying the genetic mechanisms involved in the acquisition of these traits is important for understanding and effectively inhibiting <a 
href="#glo:cancer">cancer</a>. <a 
 id="x1-9001r10"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.1.2 </span> <a 
 id="x1-100002"></a>The Genomics Revolution in Cancer Research
</h4>
<!--l. 62--><p class="noindent" >
<a 
href="#glo:genomic">Genomic</a> technologies have transformed genetics research, including the study of health and disease (<a 
href="#XGoodwin2016">Goodwin <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGoodwin2016">2016</a>; <a 
href="#XLander2011">Lander</a>, <a 
href="#XLander2011">2011</a>). <a 
href="#glo:genomic">Genomics</a> enables systematic, unbiased studies across all of the genes in the <a 
href="#glo:genome">genomes</a>. Cancer <a 
href="#glo:genomic">genomics</a> investigations have been widely applied to different tissues across <a 
href="#glo:molecular profile">molecular profiles</a> (<a 
href="#XBamford2004">Bamford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBamford2004">2004</a>; <a 
href="#XTCGA2013PAN">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013PAN">2013</a>; <a 
href="#XICGC2011">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XICGC2011">2011</a>). <a 
href="#glo:genome">Genomes</a> sequencing technologies continue to improve and become feasible in a wider range of applications.
</p><!--l. 82--><p class="indent" >
<a 
href="#glo:genomic">Genomics</a> has been used in many investigations (<a 
href="#XGoodwin2016">Goodwin <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGoodwin2016">2016</a>) but relatively few of the potential applications in healthcare have been realised yet (<a 
href="#XRoychowdhury2016">Roychowdhury and Chinnaiyan</a>, <a 
href="#XRoychowdhury2016">2016</a>; <a 
href="#XTran2012">Tran <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTran2012">2012</a>). Cancer <a 
href="#glo:genomic">genomics</a>, in particular, could have numerous benefits across diagnostics, prognosis, management, and treatment (<a 
href="#XRoychowdhury2016">Roychowdhury and Chinnaiyan</a>, <a 
href="#XRoychowdhury2016">2016</a>). While direct impact of <a 
href="#glo:genomic">genomics</a> on the clinic has been limited thus far, the <a 
href="#glo:cancer gene">cancer genes</a> and therapeutic targets identified have begun to be introduced in the clinic (<a 
href="#XStratton2009">Stratton <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XStratton2009">2009</a>). <a 
 id="x1-10001r11"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.2.1 </span> <a 
 id="x1-110001"></a>High-Throughput Technologies
</h5>
<!--l. 109--><p class="noindent" >
<p>These investigations have been enabled by recent developments in <a 
href="#glo:genomic">genomics</a> technologies, including <a 
href="#glo:microarray">microarrays</a> and more recently <a 
href="#glo:NGS">“Next-Generation Sequencing” (NGS)</a>, which can both be used to generate high-throughput <a 
href="#glo:gene expression">expression</a> data. <a 
href="#glo:microarray">Microarray</a> are a</p>
high-throughput molecular technique, reducing the cost, time, and labour required to study genes at the “genome” scale (<a 
href="#XSchena1996">Schena</a>, <a 
href="#XSchena1996">1996</a>). <a 
href="#glo:microarray">Microarray</a> can detect genotype or <a 
href="#glo:gene expression">expression</a> across many genes, making it feasible to perform on a statistically informative number of samples. <a 
href="#glo:microarray">Microarray</a> are manufactured with probes which measure binding of nucleotides which either detect the presence of a sequence such as a <a 
href="#glo:SNP">single nucleotide polymorphism (SNP)</a> or quantify sequences for <a 
href="#glo:DNA">DNA</a> copy number, <a 
href="#glo:gene expression">gene expression</a>, or <a 
href="#glo:DNA">DNA</a> <a 
href="#glo:CpG">CpG</a> methylation. In addition to being more versatile, with higher-throughput than <a 
href="#glo:PCR">polymerase chain reaction (PCR)</a> based techniques, <a 
href="#glo:microarray">microarrays</a> are considered cost-effective, particularly when scaled up to a large number of probes.
</p><!--l. 118--><p class="indent" >
The introduction of massively parallel sequencing technologies has further expanded high-throughput molecular studies and the availability of <a 
href="#glo:genomic">genomics</a> data. <a 
href="#glo:NGS">NGS</a> enables rapid <span class="cmti-12">de novo </span><a 
href="#glo:genome">genomes</a> and <a 
href="#glo:transcriptome">transcriptome</a> sequencing, in addition to <a 
href="#glo:gene expression">gene expression</a> studies (<a 
href="#XGoodwin2016">Goodwin <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGoodwin2016">2016</a>). However, the cost of sequencing for <a 
href="#glo:gene expression">gene expression</a> studies is still considerably higher than a <a 
href="#glo:microarray">microarray</a> study, limiting feasible sample sizes, and <a 
href="#glo:NGS">NGS</a> studies have large compute requirements to handle the raw data. In many cases, the benefits of <a 
href="#glo:NGS">NGS</a> technologies outweigh the additional cost. <a 
href="#glo:NGS">NGS</a> technologies have the advantage of greater potential accuracy and sensitivity than <a 
href="#glo:microarray">microarrays</a>. <a 
href="#glo:NGS">NGS</a> has a wider dynamic range than <a 
href="#glo:microarray">microarrays</a> and are not limited to genes with an already characterised sequence or functions (<a 
href="#XTarazona2011">Tarazona <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTarazona2011">2011</a>).
</p><!--l. 139--><p class="indent" >
<a 
href="#glo:NGS">NGS</a> is highly adaptable to different applications, including <a 
href="#glo:DNA">DNA</a> sequencing (obtaining the base sequence for the exome or whole <a 
href="#glo:genomic">genome</a>) or <a 
href="#glo:RNA-Seq">RNA-Seq</a> (<a 
href="#XGoodwin2016">Goodwin <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGoodwin2016">2016</a>; <a 
href="#XTran2012">Tran <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTran2012">2012</a>; <a 
href="#XWaldron2016">Waldron</a>, <a 
href="#XWaldron2016">2016</a>). <a 
href="#glo:RNA-Seq">RNA-Seq</a> of the <a 
href="#glo:transcriptome">transcriptome</a> is a common adaptation where <a 
href="#glo:RNA">RNA</a> is reverse transcribed and sequenced from the resulting <a 
href="#glo:cDNA">complementary DNA (cDNA)</a>. This is utilised to quantify the levels of <a 
href="#glo:RNA">RNA</a> and identify which regions of <a 
href="#glo:DNA">DNA</a> are expressed. Subsets of the nucleic acid may be extracted for sequencing such as the coding regions of <a 
href="#glo:DNA">DNA</a> (for the “exome”), mRNA, or <a 
href="#glo:microRNA">micro RNA (microRNA)</a>. These “<a 
href="#glo:omics">’omics</a>” technologies (<a 
href="#XRoychowdhury2016">Roychowdhury and Chinnaiyan</a>, <a 
href="#XRoychowdhury2016">2016</a>; <a 
href="#XWaldron2016">Waldron</a>, <a 
href="#XWaldron2016">2016</a>) are applicable across a wide range of biomolecules to generate “<a 
href="#glo:molecular profile">molecular profiles</a>” of a cell or sample (<a 
href="#XPerou2000">Perou <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XPerou2000">2000</a>).
</p><!--l. 175--><p class="indent" >
<p><a 
href="#glo:NGS">NGS</a> technologies continue to be refined (<a 
href="#XGoodwin2016">Goodwin <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGoodwin2016">2016</a>) with Illumina (the platform used to generate data in this project) and competitors continuing to improve products and decrease costs. As such, <a 
href="#glo:RNA-Seq">RNA-Seq</a> for examining <a 
href="#glo:transcriptome">transcriptomes</a> or <a 
href="#glo:gene expression">expression</a> studies is a growing field and will continue to be generated for a</p>
range of samples. The technology may yet improve (<a 
href="#XGoodwin2016">Goodwin <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGoodwin2016">2016</a>) with developments in speed and accuracy (such as semi-conductor platforms) or long reads, single molecule sequences (such as Pacific Biosciences, Oxford Nanopore, and Quantum Biosystems Japan). Due to the benefits of sequencing and the availability of public data, this thesis has focused on <a 
href="#glo:gene expression">gene expression</a> data generated by <a 
href="#glo:RNA-Seq">RNA-Seq</a>. <a 
href="#glo:RNA-Seq">RNA-Seq</a> data is publicly available from large-scale cancer <a 
href="#glo:genomic">genomics</a> projects and the methods anlaysis developed for <a 
href="#glo:RNA-Seq">RNA-Seq</a> data could be applied to future <a 
href="#glo:genomic">genomics</a> technologies. <a 
 id="x1-11001r13"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.2.2 </span> <a 
 id="x1-120002"></a>Bioinformatics and Genomic Data
</h5>
<!--l. 184--><p class="noindent" >
<a 
href="#glo:genomic">Genomic</a> technologies have generated data at a scale which requires computational, mathematical, and statistical expertise to handle this data effectively (<a 
href="#XMarkowetz2017">Markowetz</a>, <a 
href="#XMarkowetz2017">2017</a>; <a 
href="#XTran2012">Tran <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTran2012">2012</a>), in addition to an understanding of the biological context and research questions. The interdisciplinary field of “<a 
href="#glo:bioinformatics">bioinformatics</a>”, which draws upon these skills, focuses specifically on making inferences from <a 
href="#glo:genomic">genomics</a> data or developing the tools to do so. <a 
href="#glo:gene expression">Gene expression</a> analysis is the focus of many <a 
href="#glo:bioinformatics">bioinformatics</a> research groups, drawing upon statistical approaches to appropriately handle <a 
href="#glo:microarray">microarray</a> and <a 
href="#glo:RNA-Seq">RNA-Seq</a> data along with making biological inferences from a large number of statistical tests.
</p><!--l. 186--><p class="indent" >
<a 
href="#glo:bioinformatics">Bioinformatics</a> is often confused with the broader field “<a 
href="#glo:computational biology">computational biology</a>” (<a 
href="#XMarkowetz2017">Markowetz</a>, <a 
href="#XMarkowetz2017">2017</a>), which focuses on modelling and simulating aspects of biology and is not necessarily limited to genetics or data analysis. In practice, many researchers identify with both <a 
href="#glo:bioinformatics">bioinformatics</a> and <a 
href="#glo:computational biology">computational biology</a> or use techniques in both fields. This thesis uses many of these approaches, mainly in <a 
href="#glo:bioinformatics">bioinformatics</a>, to address biological research questions pertaining to <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions. <a 
 id="x1-12001r12"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.1.3 </span> <a 
 id="x1-130003"></a>Genomics Projects
</h4>
<!--l. 196--><p class="noindent" >
<p><a 
href="#glo:genomic">Genomic</a> projects have also been applied to various organisms, functional genetics (<a 
href="#XFANTOM2001">Kawai <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFANTOM2001">2001</a>; <a 
href="#XENCODE2004">ENCODE</a>, <a 
href="#XENCODE2004">2004</a>), and human populations focusing on variability between individuals and health or disease risk (<a 
href="#XHapMap2003">HapMap</a>, <a 
href="#XHapMap2003">2003</a>; <a 
href="#X1000Genomes2010">1000 Genomes</a>, <a 
href="#X1000Genomes2010">2010</a>). International projects and consortiums have begun to release data gathered using common agreed upon protocols across laboratories. These include many <a 
href="#glo:genomic">genomics</a> projects including cancer <a 
href="#glo:genomic">genomics</a> projects discussed below. The quality, consistency, and accessibility of</p>
these international projects is appealing, particularly for <a 
href="#glo:gene expression">gene expression</a> datasets where the more recent, larger projects have switched from <a 
href="#glo:microarray">microarray</a> to <a 
href="#glo:RNA-Seq">RNA-Seq</a> technologies. <a 
 id="x1-13001r14"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.3.1 </span> <a 
 id="x1-140001"></a>The Cancer Genome Project
</h5>
<!--l. 207--><p class="noindent" >
The <a 
href="#glo:CGP">Cancer Genome Project (CGP)</a> was among the first <a 
href="#glo:genomic">genomics</a> investigations into cancer (<a 
href="#XDickson1999">Dickson</a>, <a 
href="#XDickson1999">1999</a>), using the human <a 
href="#glo:genome">genomes</a> sequence (<a 
href="#XCollins2007">Collins and Barker</a>, <a 
href="#XCollins2007">2007</a>; <a 
href="#XLander2001">Lander <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLander2001">2001</a>), the cancer research literature, and sequencing the genes of cancers themselves. The main aim of the Cancer <a 
href="#glo:genome">Genomes</a> Project was to discover “<a 
href="#glo:cancer gene">cancer genes</a>”, which are frequently mutated in cancers by comparing cancer and normal tissue samples. These include both “<a 
href="#glo:oncogene">oncogenes</a>” (which drive cancer growth) and “<a 
href="#glo:tumour suppressor">tumour suppressors</a>” (which protect against cancers) that are functionally activated and inactivated in cancers respectively. This project is ongoing and the continues to maintain the <a 
href="#glo:COSMIC">Catalogue Of Somatic Mutations In Cancer (COSMIC)</a>, a database of <a 
href="#glo:cancer gene">cancer genes</a> (<a 
href="#XCOSMICdb">COSMIC</a>, <a 
href="#XCOSMICdb">2016</a>). It includes 1,257,487 samples with 4,175,8787 gene <a 
href="#glo:mutation">mutations</a> curated from 23,870 publications, including 29,112 whole <a 
href="#glo:genome">genomes</a> (<a 
href="#XCOSMICdb">COSMIC</a>, <a 
href="#XCOSMICdb">2016</a>). <a 
 id="x1-14001r16"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.3.2 </span> <a 
 id="x1-150002"></a>The Cancer Genome Atlas Project
</h5>
<!--l. 220--><p class="noindent" >
<a 
href="#glo:TCGA">The Cancer Genome Atlas (TCGA)</a> network initially set out to demonstrate utility in a pilot project on brain (<a 
href="#XTCGA2008GBM">McLendon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2008GBM">2008</a>), ovarian (<a 
href="#XTCGA2011OV">Bell <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2011OV">2011</a>), and squamous cell lung (<a 
href="#XTCGA2012LUSC">Hammerman <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012LUSC">2012</a>) cancers. The project then expanded, aiming to analyse 500 samples each for 20-25 <a 
href="#glo:tumour">tumour</a> tissue types. <a 
href="#glo:TCGA">TCGA</a> has since exceeded that goal, with data available for 33 cancer types including 10 “rare” cancers, a total of over 10,000 samples (<a 
href="#XTCGA2017web">TCGA</a>, <a 
href="#XTCGA2017web">2017</a>). The <a 
href="#glo:TCGA">TCGA</a> projects set out to generate a molecular “<a 
href="#glo:molecular profile">profile</a>” of the <a 
href="#glo:tumour">tumour</a> (and some matched normal tissue) samples: genotype, <a 
href="#glo:somatic">somatic</a> <a 
href="#glo:mutation">mutations</a>, <a 
href="#glo:gene expression">gene expression</a>, <a 
href="#glo:microRNA">microRNA</a>, <a 
href="#glo:DNA">DNA</a> copy number, <a 
href="#glo:DNA">DNA</a> methylation, and protein levels. Data which cannot be used to identify the patients is are publicly available
</p><!--l. 235--><p class="indent" >
<p><a 
href="#glo:TCGA">The Cancer Genome Atlas</a> pilot projects (<a 
href="#XTCGA2011OV">Bell <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2011OV">2011</a>; <a 
href="#XTCGA2012LUSC">Hammerman <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012LUSC">2012</a>; <a 
href="#XTCGA2008GBM">McLendon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2008GBM">2008</a>) serve to demonstrate the power of applying <a 
href="#glo:genomic">genomic</a> technologies to cancer research at such as scale. <a 
href="#glo:TCGA">TCGA</a> demonstrated the potential discovery of the molecular basis of cancer with these tissues, including the describing recurrently</p>
mutated genes in each cancer, identifying differentially methylated regions, and proposing transcriptional subtypes for ovarian cancers. The molecular aberrations in each cancer represent potential therapeutic targets in some cases and some were shown to have an impact on patient survival.
</p><!--l. 238--><p class="indent" >
The <a 
href="#glo:TCGA">TCGA</a> breast cancer analysis (<a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>) consisted of 802 samples with exomes, copy number variants, RPPA protein quantification, and <a 
href="#glo:DNA">DNA</a> methylation, <a 
href="#glo:mRNA">mRNA</a>, and <a 
href="#glo:microRNA">microRNA</a> arrays, with 97 whole <a 
href="#glo:genome">genomes</a> sequenced. Four main molecular classes were identified to subtype the samples, despite considerable heterogeneity between samples. Recurrent <a 
href="#glo:mutation">mutations</a> across more than 10% of samples were identified in the <span class="cmti-12">TP53</span>, <span class="cmti-12">PIK3CA</span>, and <span class="cmti-12">GATA3 </span>genes. In a further analysis of 817 breast cancer samples including 127 invasive lobular breast and 88 mixed type samples (<a 
href="#XTCGA2015LBC">Ciriello <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2015LBC">2015</a>), 3 <a 
href="#glo:molecular subtype">molecular subtypes</a> of lobular breast cancer were identified. Lobular breast cancer was also characterised by <a 
href="#glo:recurrent mutation">recurrent mutations</a> in the <span class="cmti-12">CDH1</span>, <span class="cmti-12">PTEN</span>, <span class="cmti-12">TBX2</span>, and <span class="cmti-12">FOXA1</span> genes.
</p><!--l. 243--><p class="indent" >
<a 
href="#glo:TCGA">TCGA</a> stomach cancer analysis of 295 samples (<a 
href="#XTCGA2014GC">Bass <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014GC">2014</a>) identified <a 
href="#glo:molecular subtype">molecular subtypes</a> of stomach cancers characterised by: the Epstein-Barr virus, <a 
href="#glo:MSI">microsatellite instability (MSI)</a>, genomic instability, and chromosomal instability. Abberrations in <span class="cmti-12">PD-L1</span>, <span class="cmti-12">PIK3CA</span>, and <span class="cmti-12">JAK2 </span>were also identified in stomach cancers which may present therapeutic targets.
</p><!--l. 247--><p class="indent" >
<a 
href="#glo:TCGA">TCGA</a> has identified various genes as recurrent, <a 
href="#glo:driver mutation">driver mutations</a> across cancer types which are likely to have a role in driving the development of these cancers and present a molecular target that could be applied across tissue types. In addition to disregarding the tissue-based distinction between colon and rectal cancers based on molecular similarlity (<a 
href="#XTCGA2012CRC">Muzny <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012CRC">2012</a>), <a 
href="#glo:TCGA">TCGA</a> has observed differences within tumour types and proposed <a 
href="#glo:molecular subtype">molecular subtyping</a> for breast, clear cell renal, papillary renal, stomach, skin, bladder, and prostate cancers (<a 
href="#XTCGA2015PR">Abeshouse <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2015PR">2015</a>; <a 
href="#XTCGA2015SK">Akbani <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2015SK">2015</a>; <a 
href="#XTCGA2014GC">Bass <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014GC">2014</a>; <a 
href="#XTCGA2015LBC">Ciriello <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2015LBC">2015</a>; <a 
href="#XTCGA2013RCC">Creighton <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013RCC">2013</a>; <a 
href="#XTCGA2012LUSC">Hammerman <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012LUSC">2012</a>; <a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>; <a 
href="#XTCGA2016PRC">Linehan <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2016PRC">2016</a>; <a 
href="#XTCGA2012CRC">Muzny <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012CRC">2012</a>; <a 
href="#XTCGA2014BL">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014BL">2014</a>).
</p><!--l. 251--><p class="indent" >
<p>The “<a 
href="#glo:pan cancer">Pan Cancer</a>” <a 
href="#glo:TCGA">TCGA</a> project (<a 
href="#XTCGA2014PAN">Hoadley <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014PAN">2014</a>; <a 
href="#XTCGA2013PAN">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013PAN">2013</a>) analysed 3527 samples across 12 tissue types. This project performed a comprehensive analysis of molecular data across cancer types to identify molecular simliarities and differences. These included recurrent <span class="cmti-12">TP53</span>, <span class="cmti-12">BRCA1 </span>and <span class="cmti-12">BRCA2 </span><a 
href="#glo:mutation">mutations</a>, HER2 over-expression, and <a 
href="#glo:MSI">MSI</a> across cancer types. The <a 
href="#glo:pan cancer">Pan Cancer</a> project has identified 11 <a 
href="#glo:molecular subtype">molecular subtypes</a> across these tissues, with only 5 of these corresponding to tissue</p>
cancer types due to molecular similarities shared across cancer types (<a 
href="#XTCGA2014PAN">Hoadley <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014PAN">2014</a>) . The project further supports the <a 
href="#glo:genomic">genomic</a> stratification of cancer patients, demonstrated in breast cancer (<a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>; <a 
href="#XMETABRIC2016">Pereira <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMETABRIC2016">2016</a>; <a 
href="#XPerou2000">Perou <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XPerou2000">2000</a>), and there being core molecular characteristics across cancers (<a 
href="#XHanahan2000">Hanahan and Weinberg</a>, <a 
href="#XHanahan2000">2000</a>, <a 
href="#XHanahan2011">2011</a>).
</p><!--l. 259--><p class="indent" >
While these findings contribute to further understanding cancer biology within and across tissue types, the main objective of such projects is to publicly release data to analyse in future investigations (<a 
href="#XTCGA2008GBM">McLendon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2008GBM">2008</a>; <a 
href="#XTCGA2017web">TCGA</a>, <a 
href="#XTCGA2017web">2017</a>; <a 
href="#XTCGA2013PAN">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013PAN">2013</a>). These serve as a vast resource of common and rare cancer types and are publicly available for further analysis (<a 
href="#XcBioPortal">cBioPortal</a>, <a 
href="#XcBioPortal">2017</a>; <a 
href="#XTCGA2017web">TCGA</a>, <a 
href="#XTCGA2017web">2017</a>; <a 
href="#XICGC2011">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XICGC2011">2011</a>). <a 
 id="x1-15001r15"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.1.4 </span> <a 
 id="x1-160004"></a>Genomic Cancer Medicine
</h4>
<!--l. 271--><p class="noindent" >
Cancer <a 
href="#glo:genomic">genomics</a> has substantial potential for impacts in cancer medicine: from diagnosis to treatment (<a 
href="#XRoychowdhury2016">Roychowdhury and Chinnaiyan</a>, <a 
href="#XRoychowdhury2016">2016</a>; <a 
href="#XTran2012">Tran <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTran2012">2012</a>). Beyond direct use of <a 
href="#glo:genome">genomes</a> or <a 
href="#glo:RNA-Seq">RNA-Seq</a> in clinical laboratories, <a 
href="#glo:genomic">genomic</a> studies also generate biomarkers and inform development of <a 
href="#glo:treatment">treatments</a>. These are likely to have a more immediate patient benefit considering the cost of routine <a 
href="#glo:genome">genomes</a> sequencing for diagnostics. <a 
 id="x1-16001r17"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.4.1 </span> <a 
 id="x1-170001"></a>Cancer Genes and Driver Mutations
</h5>
<!--l. 274--><p class="noindent" >
<p>There are two main classes of “<a 
href="#glo:cancer gene">cancer genes</a>” (<a 
href="#XFutreal2001">Futreal <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFutreal2001">2001</a>). Oncogenes are activated in cancers either by gain of function <a 
href="#glo:mutation">mutations</a> in proto-oncogenes, amplification of <a 
href="#glo:DNA">DNA</a>, or elevated <a 
href="#glo:gene expression">gene expression</a>. Their normal functions are typically to regulate stem cells or to promote cellular growth, with <a 
href="#glo:recurrent mutation">recurrent mutations</a> that are typically concentrated to particular gene regions (“hotspots”). Conversely, <a 
href="#glo:tumour suppressor">tumour suppressor</a> genes are those inactivated in cancer either by loss of function <a 
href="#glo:mutation">mutations</a>, deletion of <a 
href="#glo:DNA">DNA</a> copies, or reduced of <a 
href="#glo:gene expression">gene expression</a>, including hypermethylation. Their normal functions are typically to regulate cell division, <a 
href="#glo:DNA">DNA</a> repair, and cell signalling. Detecting these <a 
href="#glo:cancer gene">cancer genes</a> has accelerated with <a 
href="#glo:genomic">genomic</a> technologies, as demonstrated by <a 
href="#glo:COSMIC">COSMIC</a> and <a 
href="#glo:TCGA">TCGA</a> (<a 
href="#XCOSMICdb">COSMIC</a>, <a 
href="#XCOSMICdb">2016</a>; <a 
href="#XTCGA2013PAN">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013PAN">2013</a>). Recurrent <a 
href="#glo:mutation">mutations</a>, <a 
href="#glo:DNA">DNA</a> copy number variants, differential <a 
href="#glo:gene expression">gene expression</a>, or differential <a 
href="#glo:DNA">DNA</a> methylation are all indicative of <a 
href="#glo:cancer gene">cancer</p>
genes</a> (<a 
href="#XMattison2009">Mattison <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMattison2009">2009</a>), which can be detected in <a 
href="#glo:genomic">genomics</a> data (<a 
href="#XMETABRIC2016">Pereira <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMETABRIC2016">2016</a>; <a 
href="#XTCGA2013PAN">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013PAN">2013</a>).
</p><!--l. 278--><p class="indent" >
Distinguishing important “<a 
href="#glo:driver mutation">driver</a>” <a 
href="#glo:mutation">mutations</a> in <a 
href="#glo:cancer gene">cancer genes</a> from “<a 
href="#glo:passenger mutation">passenger mutation</a>” <a 
href="#glo:mutation">mutations</a> is challenging due to patient variation, tumour heterogeneity, and genomic instability producing many variant gene sequences (<a 
href="#XStratton2009">Stratton <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XStratton2009">2009</a>; <a 
href="#XTran2012">Tran <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTran2012">2012</a>). Driver <a 
href="#glo:mutation">mutations</a> can be identified by whether they co-occur or are mutually exclusive with <a 
href="#glo:mutation">mutations</a> in other genes in cancers, are <a 
href="#glo:recurrent mutation">recurrently mutated</a> across a significant proportion of samples for a specific tissue type, or if <a 
href="#glo:mutation">mutations</a> are recurrent across different cancer tissue types (<a 
href="#XcBioPortal">cBioPortal</a>, <a 
href="#XcBioPortal">2017</a>; <a 
href="#XMETABRIC2016">Pereira <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMETABRIC2016">2016</a>; <a 
href="#XCOSMICdb">COSMIC</a>, <a 
href="#XCOSMICdb">2016</a>; <a 
href="#XTCGA2013PAN">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013PAN">2013</a>; <a 
href="#XICGC2011">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XICGC2011">2011</a>). Approximately 140 <a 
href="#glo:driver mutation">driver mutations</a> have been identified, including many novel genes in particular cancers from <a 
href="#glo:genomic">genomic</a> studies, with 2–8 in typically occurring in each tumour usually affecting cell fate, survival, or <a 
href="#glo:genome">genomes</a> maintenance (<a 
href="#XVogelstein2013">Vogelstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XVogelstein2013">2013</a>). There remains a need to translate the identification of many <a 
href="#glo:cancer gene">cancer genes</a> and <a 
href="#glo:driver mutation">driver mutations</a> to patient benefit by repurposing or designing of <a 
href="#glo:targeted therapy">therapeutic interventions</a> against these molecular targets. <a 
 id="x1-17001r19"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.4.2 </span> <a 
 id="x1-180002"></a>Precision Cancer Medicine
</h5>
<!--l. 284--><p class="noindent" >
The importance of genomics is emphasised in translational cancer research in contrast with current strategies of healthcare based on what works well for the most of the population. Cancers could eventually be treated by their genomic features (<a 
href="#XRoychowdhury2016">Roychowdhury and Chinnaiyan</a>, <a 
href="#XRoychowdhury2016">2016</a>), particularly grouping patients by the <a 
href="#glo:mutation">mutation</a>, <a 
href="#glo:gene expression">expression</a>, or <a 
href="#glo:DNA">DNA</a> methylation profiles of their cancers, which is already done in part (<a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>). Identifying actionable molecular targets is a key aspect of “<a 
href="#glo:precision medicine">precision medicine</a>”, the rationale to target <a 
href="#glo:molecular subtype">molecular subtypes</a> with separate treatment strategies (<a 
href="#XGlaire2017">Glaire <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGlaire2017">2017</a>). To this end many <a 
href="#glo:driver mutation">driver mutations</a> and <a 
href="#glo:gene expression">gene expression</a> signatures for distinguishing cancers have been identified. Some oncogenic <a 
href="#glo:driver mutation">driver mutations</a> have effective pharmacological inhibitors designed against them but there remain many <a 
href="#glo:cancer gene">cancer genes</a> and <a 
href="#glo:mutation">mutations</a>, particularly <a 
href="#glo:tumour suppressor">tumour suppressors</a>, for which there is not yet a <a 
href="#glo:targeted therapy">targeted therapy</a>. <a 
 id="x1-18001r20"></a>
</p>
                                                                                    

<h5 class="subsubsectionHead">
<span class="titlemark">1.1.4.3 </span> <a 
 id="x1-190003"></a>Molecular Diagnostics and Pan-Cancer Medicine
</h5>
<!--l. 291--><p class="noindent" >
Molecular features such as <a 
href="#glo:mutation">mutations</a> or <a 
href="#glo:gene expression">gene expression</a> signatures have been proposed to diagnose tumour subtypes. In breast cancer, several distinct “<a 
href="#glo:intrinsic subtype">intrinsic subtypes</a>” have been identified, distinguished by molecular mechanisms, with differences in malignancy and patient outcome (<a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>; <a 
href="#XPerou2000">Perou <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XPerou2000">2000</a>). Conversely, common molecular mechanisms may be shared between cancers across tissue types as discovered by the “<a 
href="#glo:pan cancer">Pan Cancer</a>” <a 
href="#glo:TCGA">TCGA</a> project, which combined <a 
href="#glo:molecular profile">molecular profiles</a> across tissue types (<a 
href="#XTCGA2013PAN">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013PAN">2013</a>). <a 
href="#glo:molecular subtype">Molecular subtypes</a> could feasibly be included in clinical testing as a panel of biomarkers for diagnosis, monitoring drug response, or predicting risk of recurrence. As these <a 
href="#glo:molecular subtype">molecular subtypes</a> and genetic aberrations specific to cancers have been identified, there is an increasingly clear need for further development of <a 
href="#glo:treatment">treatments</a> that target them.
</p><!--l. 293--><p class="indent" >
<a 
href="#glo:gene expression">Gene expression</a> can be used to characterise breast cancers. The “<a 
href="#glo:intrinsic subtype">intrinsic subtypes</a>” identified were characterised by estrogen receptor, <span class="cmti-12">HER2</span>, and basal, epithelial signalling (<a 
href="#XPerou2000">Perou <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XPerou2000">2000</a>). The <a 
href="#glo:gene expression">expression</a> profiles were similar across independent samples of the same tumour or the same patient and therefore represent the molecular state of a <a 
href="#glo:tumour">tumour</a> The molecular <a 
href="#glo:intrinsic subtype">intrinsic subtypes</a> “luminal A”, “luminal B”, “HER2-enriched”, “basal-like”, and “normal-like” have been replicated across <a 
href="#glo:microarray">microarray</a> studies (<a 
href="#XHu2006">Hu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XHu2006">2006</a>), with their relevance to prognosis demonstrated, and a 50-gene subtype predictor developed (<a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>; <a 
href="#XSorlie2001">Sørlie <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XSorlie2001">2001</a>). Despite specific differences in subtyping, there is widespread agreement that distinguishing luminal, HER2-enriched, and triple negative <a 
href="#glo:tumour">tumours</a> has prognostic importance for patients (<a 
href="#XDai2015">Dai <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDai2015">2015</a>). The “<a 
href="#glo:pan cancer">Pan Cancer</a>” <a 
href="#glo:TCGA">The Cancer Genome Atlas</a> project (discussed in Section <a 
href="#x1-150002">1.1.3.2<!--tex4ht:ref: TCGA_Intro --></a>) demonstrates the importance of molecular similarities and differences between <a 
href="#glo:cancer">cancers</a> across cancer tissue types (<a 
href="#XTCGA2013PAN">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013PAN">2013</a>).
</p><!--l. 299--><p class="indent" >
<p><a 
href="#XGatza2010">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XGatza2010">2010</a>) used gene signatures for 18 cellular <a 
href="#glo:pathway">pathways</a> in breast cancer to define subtypes with distinct molecular <a 
href="#glo:pathway">pathway</a> activity. A “<a 
href="#glo:metagene">metagene</a>” is a measure <a 
href="#glo:pathway">pathway</a> activation (derived from eigenvectors or principal components) which gives a consistent signal of <a 
href="#glo:gene expression">gene expression</a> (<a 
href="#XAnjomshoaa2008">Anjomshoaa <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XAnjomshoaa2008">2008</a>; <a 
href="#XHuang2003">Huang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XHuang2003">2003</a>; <a 
href="#XNagalla2013">Nagalla <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XNagalla2013">2013</a>). Unsupervised hierarchical clustering defined subtypes with common <a 
href="#glo:pathway">pathway</a> activity, despite variation in <a 
href="#glo:mutation">mutations</a>. These subtypes <a 
href="#glo:intrinsic subtype">intrinsic subtypes</a> and provide finer <a 
href="#glo:molecular subtype">molecular stratification</a> with a functional basis (<a 
href="#XGatza2014">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2014">2014</a>; <a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>). The <a 
href="#glo:molecular subtype">subtypes</a> with shared <a 
href="#glo:pathway">pathway</a> activity have similar molecular characteristics (such as <a 
href="#glo:DNA">DNA</a> copy number) and clinical properties including prognosis.</p>
<a 
 id="x1-19001r21"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.4.4 </span> <a 
 id="x1-200004"></a>Targeted Therapeutics and Pharmacogenomics
</h5>
<!--l. 309--><p class="noindent" >
<a 
href="#glo:targeted therapy">Targeted therapies</a> with specificity against a molecular target are examples of <a 
href="#glo:precision medicine">precision cancer medicine</a>. Molecular targets can be tested in laboratory conditions with <a 
href="#glo:RNAi">RNA interference (RNAi)</a> or pharmacological agents (<a 
href="#XFece2015">Fece de la Cruz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFece2015">2015</a>). Identification of molecular targets is important for developing novel anti-cancer <a 
href="#glo:treatment">treatments</a> along with validation and drug testing. For oncogenic <a 
href="#glo:mutation">mutations</a>, the recurrent <a 
href="#glo:mutant">mutant</a> variant or over-expressed gene can be directly inhibited, however, <a 
href="#glo:oncogene">oncogenes</a> with high homology to other genes or <a 
href="#glo:tumour suppressor">tumour suppressor</a> genes are not amenable to direct targeting (<a 
href="#XKaelin2009">Kaelin, Jr</a>, <a 
href="#XKaelin2009">2009</a>). Targeted anticancer therapeutics can exploit complex interactions to distinguish normal and cancerous cells which may benefit from studies of gene regulation or interaction networks (<a 
href="#XHopkins2008">Hopkins</a>, <a 
href="#XHopkins2008">2008</a>). Targeted therapeutics have already been successfully applied as monoclonal antibodies against <a 
href="#glo:oncogene">oncogenes</a>, such as HER2 in breast cancer (<a 
href="#XMiles2001">Miles</a>, <a 
href="#XMiles2001">2001</a>). <a 
 id="x1-20001r18"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.1.5 </span> <a 
 id="x1-210005"></a>Systems and Network Biology
</h4>
<!--l. 339--><p class="noindent" >
Driver <a 
href="#glo:mutation">mutations</a> in <a 
href="#glo:oncogene">oncogenes</a> and <a 
href="#glo:tumour suppressor">tumour suppressor</a> genes do not occur in isolation. The genetic interactions, regulatory and cellular signalling, and metabolic reactions are inter-related and may each be perturbed by aberrations in gene function occurring in <a 
href="#glo:cancer">cancers</a>. These relationships can be represented by biological networks of connected pairs of genes with a relationship. Due to the complexity of a cell, these molecular networks are very large, consisting of thousands of <a 
href="#glo:vertex">nodes</a> comprised by genes or proteins.
</p><!--l. 341--><p class="indent" >
The properties of large <a 
href="#glo:graph">networks</a> were first studied by constructing random <a 
href="#glo:graph">networks</a> by randomly linking a fixed number of <a 
href="#glo:vertex">nodes</a> (<a 
href="#XErdos1959">Erd<img 
src="thesis0x.png" alt="&#x02DD;o  "  class="Huml"  />s and Rényi</a>, <a 
href="#XErdos1959">1959</a>, <a 
href="#XErdos1960">1960</a>). Despite the random nature of these <a 
href="#glo:graph">networks</a>, properties such as their connectivity were well characterised. The <a 
href="#glo:vertex">vertex</a> degree (number of partners for each <a 
href="#glo:vertex">node</a>) of their random <a 
href="#glo:graph">networks</a> followed a Poisson distribution, however this property does not hold in nature. Thus natural <a 
href="#glo:graph">networks</a> are non-random or not formed in this way (<a 
href="#XBarabasi2004">Barabási and Oltvai</a>, <a 
href="#XBarabasi2004">2004</a>).
</p><!--l. 343--><p class="indent" >
<p>This work formed the foundation for studying complex <a 
href="#glo:graph">networks</a> (<a 
href="#XvanSteen2010">van Steen</a>, <a 
href="#XvanSteen2010">2010</a>), which model features of observed <a 
href="#glo:graph">networks</a> not found in Erd<img 
src="thesis1x.png" alt="&#x02DD;o  "  class="Huml"  /> and Rényi‘s random <a 
href="#glo:graph">networks</a> (<a 
href="#XErdos1959">Erd<img 
src="thesis2x.png" alt="&#x02DD;o "  class="Huml"  />s and Rényi</a>, <a 
href="#XErdos1959">1959</a>, <a 
href="#XErdos1960">1960</a>). The <a 
href="#glo:small world">small world</a> property, made popular by findings in</p>
social <a 
href="#glo:graph">networks</a> (<a 
href="#XTravers1969">Travers and Milgram</a>, <a 
href="#XTravers1969">1969</a>), is the remarkably short path lengths between any <a 
href="#glo:vertex">nodes</a> in a <a 
href="#glo:small world">small world</a> <a 
href="#glo:graph">network</a>. A <a 
href="#glo:small world">small world</a> <a 
href="#glo:graph">network</a> is well-connected with a characteristic path length (the average length of <a 
href="#glo:shortest path">shortest paths</a> between all pairs of <a 
href="#glo:vertex">nodes</a>) proportional to the logarithm of the number of <a 
href="#glo:vertex">nodes</a>. <a 
href="#XWatts1998">Watts and Strogatz</a> (<a 
href="#XWatts1998">1998</a>) developed a model of random rewiring of a regular <a 
href="#glo:graph">network</a> to construct random <a 
href="#glo:graph">networks</a> with the <a 
href="#glo:small world">small world</a> property and a high clustering coefficient. While these properties are more representative of <a 
href="#glo:graph">networks</a> occurring in nature, their model was limited by the degree distribution which converges to a Poisson distribution as it is rewired (<a 
href="#XBarrat2000">Barrat and Weigt</a>, <a 
href="#XBarrat2000">2000</a>). The <a 
href="#glo:vertex">vertex</a> degree distribution of naturally occurring <a 
href="#glo:graph">networks</a> often follows a power law distribution with most <a 
href="#glo:vertex">nodes</a> having far fewer connections than average and a small subset of highly connected <a 
href="#glo:graph">network</a> ‘hubs’ (<a 
href="#XBarabasi1999">Barabási and Albert</a>, <a 
href="#XBarabasi1999">1999</a>).
</p><!--l. 347--><p class="indent" >
<a 
href="#XBarabasi1999">Barabási and Albert</a> (<a 
href="#XBarabasi1999">1999</a>) constructed a <a 
href="#glo:graph">network</a> model in an entirely different way to randomly generate <a 
href="#glo:scale-free">scale-free</a> <a 
href="#glo:graph">networks</a> which have a power law degree distribution. They constructed random <a 
href="#glo:graph">networks</a> by preferential attachment, modelling growth of a <a 
href="#glo:graph">network</a> by sequentially adding <a 
href="#glo:vertex">nodes</a> with <a 
href="#glo:edge">links</a> to existing <a 
href="#glo:vertex">nodes</a>. The <a 
href="#glo:scale-free">scale-free</a> nature of the random <a 
href="#glo:graph">networks</a> was ensured by adding new <a 
href="#glo:vertex">nodes</a> with an increasing probability of attachment to an existing <a 
href="#glo:vertex">node</a> if it had a higher degree. These <a 
href="#glo:graph">networks</a> successfully captured the <a 
href="#glo:scale-free">scale-free</a> nature of many observed <a 
href="#glo:graph">networks</a> with short characteristic path length and low eccentricity resulting in super <a 
href="#glo:small world">small worlds</a> (<a 
href="#XBarabasi1999">Barabási and Albert</a>, <a 
href="#XBarabasi1999">1999</a>).
</p><!--l. 352--><p class="indent" >
<p>High-throughput technologies such as <a 
href="#glo:siRNA">siRNA</a> screens, two-hybrid screens, <a 
href="#glo:microarray">microarrays</a> and massively parallel sequencing have generated <a 
href="#glo:genome">genomes</a>-scale data and enabled analysis of biological <a 
href="#glo:graph">networks</a> (<a 
href="#XBarabasi2004">Barabási and Oltvai</a>, <a 
href="#XBarabasi2004">2004</a>; <a 
href="#XBoone2007">Boone <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoone2007">2007</a>; <a 
href="#XGoodwin2016">Goodwin <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGoodwin2016">2016</a>). Molecular networks are biological networks consisting of biological molecules including genes, transcripts (with non-coding and <a 
href="#glo:microRNA">microRNAs</a>), or proteins related by known interactions and gene regulatory or metabolic <a 
href="#glo:pathway">pathways</a>. Many types of molecular networks can be constructed, depending on the biological application (). <a 
href="#glo:SGI">Synthetic genetic interactions</a> are relatively unexplored within molecular networks and may lead to better understanding of the role of gene functions in cellular function and disease. <a 
href="#glo:high-throughput screen">High-throughput screens</a> in humans, mammals, and non-model organisms are costly and labour-intensive (<a 
href="#XFece2015">Fece de la Cruz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFece2015">2015</a>). Computational approaches with effective predictive models are therefore a more feasible alternative to study the connectivity of a biological <a 
href="#glo:graph">network</a> in a complex metazoan cell at the <a 
href="#glo:genome">genomes</a>-scale.</p>
<a 
 id="x1-21001r9"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">1.2 </span> <a 
 id="x1-220002"></a>Synthetic Lethal Cancer Medicine
</h3>
<!--l. 371--><p class="noindent" >
<a 
href="#glo:synthetic lethal">Synthetic lethality</a> has vast potential to improve cancer medicine by expanding application of <a 
href="#glo:targeted therapy">targeted therapeutic</a> to include inactivation of <a 
href="#glo:tumour suppressor">tumour suppressors</a> and genes that are difficult to target directly. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> interactions are also studied for gene function and drug mode-of-action in model organisms. This Section introduces the concept of <a 
href="#glo:synthetic lethal">synthetic lethality</a> as it was originally conceived and how it has been adopted conceptually in cancer research. Detecting these interactions at scale and interpreting them is the focus of this thesis, hence we start with an overview of the concepts involved, initial work on the interaction, and the rationale for applications to cancer. Specific investigations into <a 
href="#glo:synthetic lethal">synthetic lethality</a> in cancer, detection by experimental screening, and prediction by computational analysis will then be reviewed. <a 
 id="x1-22001r23"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.2.1 </span> <a 
 id="x1-230001"></a>Synthetic Lethal Genetic Interactions
</h4>
<!--l. 380--><p class="noindent" >
Genetic interactions are a core concept of molecular biology, discovered among earliest investigations of Mendelian genetics, and have received revived interest with new technologies and potential applications. <a 
href="#glo:epistasis_bio">Biological epistasis</a> is the effect of an <a 
href="#glo:allele">allele</a> at one locus “masking” the phenotype of another locus (<a 
href="#XBateson1909">Bateson and Mendel</a>, <a 
href="#XBateson1909">1909</a>). <a 
href="#glo:epistasis_stat">Statistical epistasis</a> is where there is significant disparity between the observed and expected phenotype of a double <a 
href="#glo:mutant">mutant</a>, compared to the respective phenotypes of single <a 
href="#glo:mutant">mutant</a> and the <a 
href="#glo:wild-type">wild-type</a> (<a 
href="#XFisher1919">Fisher</a>, <a 
href="#XFisher1919">1919</a>). Fisher’s <a 
href="#glo:epistasis_stat">definition</a> lends itself to quantitative traits and more broadly encompasses <a 
href="#glo:SGI">synthetic genetic interactions</a>. These have become popular for studies in yeast genetics and cancer drug design (<a 
href="#XBoone2007">Boone <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoone2007">2007</a>; <a 
href="#XKaelin2005">Kaelin, Jr</a>, <a 
href="#XKaelin2005">2005</a>).
</p><!--l. 382--><p class="indent" >
<p><a 
href="#glo:synthetic sick"></a><a 
href="#glo:synthetic suppression"></a><a 
href="#glo:synthetic rescue"></a><a 
href="#glo:SGI">SGIs</a> are substantial deviations of growth or viability from the expected null <a 
href="#glo:mutant">mutant</a> phenotype (of an organism or cell) assuming additive (deleterious) effects of the single <a 
href="#glo:mutant">mutant</a>. The double <a 
href="#glo:mutant">mutant</a> does not necessarily have either of the single <a 
href="#glo:mutant">mutant</a> phenotypes (as shown for cellular growth phenotypes in Figure <a 
href="#x1-230011">1.1<!--tex4ht:ref: fig:Costanzo2011 --></a>). Most <a 
href="#glo:SGI">SGIs</a> are more viable than either single <a 
href="#glo:mutant">mutant</a> or less viable than the expected double <a 
href="#glo:mutant">mutant</a>. Mutations are “synergistic” in negative <a 
href="#glo:SGI">SGI</a> with more deviation from the <a 
href="#glo:wild-type">wild-type</a> than expected. Formally, ‘<a 
href="#glo:SSL">“synthetic sick” (SSL)</a> and <a 
href="#glo:SL">“synthetic lethal” (SL)</a> interactions are negative <a 
href="#glo:SGI">SGIs</a> giving growth inhibition and complete inviability respectively. In cancer</p>
research, <a 
href="#glo:synthetic lethal">synthetic lethality</a> more broadly describes any negative <a 
href="#glo:SGI">SGI</a> with specific inhibition of a <a 
href="#glo:mutant">mutant</a> cell, including <a 
href="#glo:SSL">SSL</a> interactions. Mutations are “alleviating” in positive <a 
href="#glo:SGI">SGI</a> with less deviation from the <a 
href="#glo:wild-type">wild-type</a> than expected. For viability, <a 
href="#glo:SS">“suppression” (SS)</a> and <a 
href="#glo:SR">“rescue” (SR)</a> are positive <a 
href="#glo:SGI">SGIs</a> giving at least partial restoration of <a 
href="#glo:wild-type">wild-type</a> growth from single <a 
href="#glo:mutant">mutant</a> with growth impairment and lethal phenotypes respectively. Negative <a 
href="#glo:SGI">SGIs</a> were markedly more common than positive <a 
href="#glo:SGI">SGIs</a> in a number of studies in model systems (<a 
href="#XBoucher2013">Boucher and Jenna</a>, <a 
href="#XBoucher2013">2013</a>; <a 
href="#XTong2004">Tong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTong2004">2004</a>).
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                    

<p><a 
 id="x1-230011"></a></p>
<!--l. 389--><p class="noindent" >
<img 
src="Costanzo_2011_SL_mega_no_border.png" alt="pict"  
 /> <br />
</p><div class="caption" 
>
<span class="id">Figure 1.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic genetic interactions.</span></strong> <span class="cmr-10x-x-109">Impact of various negative and positive</span> <a 
href="#glo:SGI"><span class="cmr-10x-x-109">SGIs</span></a><span class="cmr-10x-x-109">: negative interactions involve deleterious (sick) or inviable (lethal) phenotypes whereas</span> <span class="cmr-10x-x-109">positive interactions involve restoring viability by masking or suppressing the other </span><a 
href="#glo:mutation"><span class="cmr-10x-x-109">mutation</span></a> <span class="cmr-10x-x-109">or complete rescue of the</span><a 
href="#glo:wild-type"><span class="cmr-10x-x-109">wild-type</span></a> <span class="cmr-10x-x-109">phenotype. Figure adapted from (</span><a 
href="#XCostanzo2011"><span class="cmr-10x-x-109">Costanzo </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109">,</span><span class="cmr-10x-x-109"> </span><a 
href="#XCostanzo2011"><span class="cmr-10x-x-109">2011</span></a><span class="cmr-10x-x-109">)</span> <span class="cmr-10x-x-109">concerning growth viability fitness in yeast.</span></span>
</div><!--tex4ht:label?: x1-230011 -->
                                                                                    

</div><hr class="endfigure" />
<a 
 id="x1-23002r25"></a>
<h4 class="subsectionHead">
<span class="titlemark">1.2.2 </span> <a 
 id="x1-240002"></a>Synthetic Lethal Concepts in Genetics
</h4>
<!--l. 400--><p class="noindent" >
<a 
href="#glo:synthetic lethal">Synthetic lethal</a> genes are generally regarded to arise due to <a 
href="#glo:functional redundancy">functional redundancy</a> (<a 
href="#XBoone2007">Boone <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoone2007">2007</a>). Due to the functional level of <a 
href="#glo:SGI">SGIs</a>, <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes do not need to directly interact, nor be expressed in the same cell or at the same developmental stage: serving related functions is sufficient to affect cell (or organism) viability and be relevant to drug-mode-of-action cancer biology. Combined loss of genes performing an <a 
href="#glo:essential">essential</a> or important function in a cell are therefore deleterious. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> gene pairs are therefore pairwise <a 
href="#glo:essential">essential</a> with “<a 
href="#glo:induced essentiality">induced essentiality</a>”: each <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene becomes <a 
href="#glo:essential">essential</a> to the cell upon loss of the other (<a 
href="#XAshworth2011">Ashworth <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XAshworth2011">2011</a>; <a 
href="#XKaelin2005">Kaelin, Jr</a>, <a 
href="#XKaelin2005">2005</a>).
</p><!--l. 402--><p class="indent" >
Since <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene partners can be affected by extracellular stimuli such as chemicals, essentiality of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes can be induced by the environment of a cell. An environmental stress condition may inhibit one or the other <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene, such as exposure to chemicals, in which case the <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner gene is “conditionally essential” (<a 
href="#XHillenmeyer2008">Hillenmeyer</a>, <a 
href="#XHillenmeyer2008">2008</a>). Thus the evolutionary rationale for the abundance of <a 
href="#glo:SGI">SGIs</a> (compared to the surprisingly low number of <a 
href="#glo:essential">essential</a> genes) in a Eukaryotic <a 
href="#glo:genome">genomes</a> can be attributed to genetic <a 
href="#glo:functional redundancy">functional redundancy</a> and network <a 
href="#glo:genetic robustness">robustness</a> of a cell which are advantageous to survival.
</p><!--l. 404--><p class="indent" >
Biological functions are typically performed by a <a 
href="#glo:pathway">pathway</a> of genes (or their products). <a 
href="#glo:synthetic lethal">Synthetic lethal</a> genes occur within the same biological <a 
href="#glo:pathway">pathway</a> and between them (<a 
href="#XBoone2007">Boone <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoone2007">2007</a>; <a 
href="#XCostanzo2010">Costanzo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCostanzo2010">2010</a>; <a 
href="#XKelley2005">Kelley and Ideker</a>, <a 
href="#XKelley2005">2005</a>). Many genes of the same <a 
href="#glo:pathway">pathway</a> may be functionally interchangable, <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of a particular gene. Therefore biological <a 
href="#glo:pathway">pathways</a> can exhibit <a 
href="#glo:induced essentiality">induced essentiality</a> with loss of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner gene and <a 
href="#glo:synthetic lethal">synthetic lethality</a> may occur at <a 
href="#glo:pathway">pathway</a> level or in a gene regulation network. <a 
 id="x1-24001r27"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.2.3 </span> <a 
 id="x1-250003"></a>Synthetic Lethality in Model Systems
</h4>
<!--l. 407--><p class="noindent" >
<p>Genetic <a 
href="#glo:high-throughput screen">high-throughput screens</a> have identified unexpected, functionally informative, and clinically relevant <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions; including <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of genes recurrently mutated in cancer or attributed to <a 
href="#glo:familial">familial</a> early-onset cancers (<a 
href="#XLord2014">Lord <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLord2014">2015</a>). While screening presents an appealing strategy for <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery,</p>
computational approaches are becoming popular as an alternative or complement to experimental methods to overcome inherent bias and limitations of experimental screens. An array of recently developed computational methods (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>; <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>; <a 
href="#XTiong2014">Tiong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTiong2014">2014</a>; <a 
href="#XWang2013">Wang and Simon</a>, <a 
href="#XWang2013">2013</a>; <a 
href="#XWappett2014">Wappett</a>, <a 
href="#XWappett2014">2014</a>) show the need for <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery in the fundamental genetics and translational cancer research community. However, many existing computational methods are not suitable for queries of <a 
href="#glo:genomic">genomic</a> data for interacting partners of a particular gene, as (1) they have been applied pairwise across the <a 
href="#glo:genome">genomes</a>, (2) they do not have software released to apply the methodology, or (3) they lack statistical measures of error for further analysis. A robust prediction of gene interactions is an effective and practical approach at a scale of the entire <a 
href="#glo:genome">genomes</a> for ideal translational applications, analysis of biological systems, and constructing functional gene networks. <a 
 id="x1-25001r22"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.3.1 </span> <a 
 id="x1-260001"></a>Synthetic Lethal Pathways and Networks
</h5>
<!--l. 410--><p class="noindent" >
<a 
href="#glo:SGI">SGIs</a> are common in <a 
href="#glo:genome">genomes</a>, four-fold more interactions were detected with <a 
href="#glo:SGA">synthetic gene array (SGA)</a> mating screens than <a 
href="#glo:PPI">protein-protein interactions</a> detected with yeast-2-hybrid (<a 
href="#XTong2004">Tong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTong2004">2004</a>). The <a 
href="#glo:SGI">SGI</a> network was <a 
href="#glo:scale-free">scale-free</a> and had a low average <a 
href="#glo:shortest path">shortest path</a> length, as expected for a complex biological network (<a 
href="#XBarabasi2004">Barabási and Oltvai</a>, <a 
href="#XBarabasi2004">2004</a>). Highly connected “hub” genes with the highest number of <a 
href="#glo:edge">links</a> (<a 
href="#glo:vertex degree">vertex degree</a>) are functionally important with many negative <a 
href="#glo:SGI">SGI</a> hubs involved in cell cycle regulation, and many positive <a 
href="#glo:SGI">SGI</a> hubs involved in translation (<a 
href="#XBaryshnikova2010b">Baryshnikova <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBaryshnikova2010b">2010b</a>; <a 
href="#XCostanzo2010">Costanzo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCostanzo2010">2010</a>). Negative <a 
href="#glo:SGI">SGIs</a> were far more common than positive <a 
href="#glo:SGI">SGIs</a>, with synthetic gene loss being more likely to be deleterious to cell than advantageous, which indicates that <a 
href="#glo:synthetic lethal">synthetic lethality</a> may be comparably easier to detect than other <a 
href="#glo:SGI">SGIs</a>.
</p><!--l. 412--><p class="indent" >
<p><a 
href="#glo:essential">Essential</a> <a 
href="#glo:pathway">pathways</a> are highly buffered, with five-fold more interactions than other <a 
href="#glo:SGI">SGIs</a>, consistent with strong selection for survival, as found with conditional and partial <a 
href="#glo:mutation">mutations</a> in <a 
href="#glo:essential">essential</a> genes (<a 
href="#XDavierwala2005">Davierwala <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDavierwala2005">2005</a>). This <a 
href="#glo:SGI">SGI</a> network had <a 
href="#glo:scale-free">scale-free</a> topology and rarely shared interactions with the protein-protein interaction network. These networks are related by an “orthogonal” relationship: shared partners in one network tend to be themselves connected directly in the other network. Essential genes were likely to have closely related functions, whereas non-<a 
href="#glo:essential">essential</a> networks were relatively more inclined to have <a 
href="#glo:SGI">SGIs</a> between distinct biological <a 
href="#glo:pathway">pathways</a>.</p>
<a 
 id="x1-26001r29"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.3.2 </span> <a 
 id="x1-270002"></a>Evolution of Synthetic Lethality
</h5>
<!--l. 415--><p class="noindent" >
There is poor conservation of specific <a 
href="#glo:SGI">SGIs</a> between <span class="cmti-12">S. cerevisiae </span>and <span class="cmti-12">S. pombe </span>with 29% of the interactions tested in both distantly related species being conserved between them (<a 
href="#XDixon2008">Dixon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDixon2008">2008</a>). The remaining interactions show high species-specific differences, however, many of the species-specific interactions were still conserved between biological <a 
href="#glo:pathway">pathways</a>, protein complexes, or protein-protein interaction modules. Similarly, conservation of <a 
href="#glo:pathway">pathway</a> redundancy was also found between Eukaryotes (<span class="cmti-12">S.</span> <span class="cmti-12">cerevisiae</span>) and prokaryotes (<span class="cmti-12">E. coli</span>) (<a 
href="#XButland2008">Butland <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XButland2008">2008</a>). Negative <a 
href="#glo:SGI">SGIs</a> were more likely to be conserved between biological <a 
href="#glo:pathway">pathways</a>, whereas positive <a 
href="#glo:SGI">SGIs</a> were more likely to be conserved within a <a 
href="#glo:pathway">pathway</a> or protein complex (<a 
href="#XRoguev2008">Roguev <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XRoguev2008">2008</a>).
</p><!--l. 417--><p class="indent" >
A modest 5% of interactions were conserved between unicellular (<span class="cmti-12">S. cerevisiae</span>) and multicellular (<span class="cmti-12">C. elegans</span>) organisms. However, the nematode <a 
href="#glo:SGI">SGI</a> network had similar <a 
href="#glo:scale-free">scale-free</a> topology and modularity despite differences in methodology: metazoan <a 
href="#glo:synthetic lethal">synthetic lethal</a> screens with <a 
href="#glo:RNAi">RNA interference</a> (<a 
href="#glo:RNAi">RNAi</a>) are incomplete knockouts, whereas screening null <a 
href="#glo:mutation">mutations</a> is feasible in yeast (<a 
href="#XBussey2006">Bussey <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBussey2006">2006</a>). The nematode <a 
href="#glo:SGI">SGI</a> screen identified network hubs with important interactions to orthologues of known human disease genes (<a 
href="#XLehner2006">Lehner <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLehner2006">2006</a>). Despite the lack of direct conservation of <a 
href="#glo:SGI">SGIs</a> between yeasts and nematode worms, <a 
href="#glo:functional redundancy">genetic redundancy</a> was consistent with an “<a 
href="#glo:induced essentiality">induced essentiality</a>” model of <a 
href="#glo:SGI">SGIs</a> where gene functions are conserved with network restructuring over evolutionary change (<a 
href="#XTischler2008">Tischler <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTischler2008">2008</a>).
</p><!--l. 419--><p class="indent" >
<p>While nematode models are more closely related to human cells which are also screened with <a 
href="#glo:RNAi">RNAi</a>, cancer cells can present growth and viability phenotypes more comparable to yeast models. Therefore findings from both <a 
href="#glo:SGA">SGA</a> and <a 
href="#glo:RNAi">RNAi</a> models are relevant to understanding human and cancer cells. <a 
href="#glo:RNAi">RNAi</a> has also been applied to human and mouse cancer cells with <a 
href="#glo:siRNA">short interfering RNA</a> (<a 
href="#glo:siRNA">siRNA</a>) in cell culture and genetic screening experiments. These findings suggest that <a 
href="#glo:SGI">SGI</a> network “rewiring” is a concern for identifying specific <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions in cancer as specific <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes may vary between genetic backgrounds. Thus it is expected at a <a 
href="#glo:pathway">pathway</a> approach will be more robust in the context of evolution, patient variation, tumour heterogeneity, or disease progression. <a 
 id="x1-27001r28"></a></p>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.2.4 </span> <a 
 id="x1-280004"></a>Synthetic Lethality in Cancer
</h4>
<!--l. 423--><p class="noindent" >
Loss of function occurs in many genes in cancers, including <a 
href="#glo:tumour suppressor">tumour suppressors</a>, yet few interventions target such <a 
href="#glo:mutation">mutations</a> compared to targeted therapies for gain of function <a 
href="#glo:mutation">mutation</a> in <a 
href="#glo:oncogene">oncogenes</a> (<a 
href="#XKaelin2005">Kaelin, Jr</a>, <a 
href="#XKaelin2005">2005</a>). <a 
href="#glo:synthetic lethal">Synthetic lethality</a> is a powerful design strategy for therapies selective against loss of gene function with potential for application against a range of genes and diseases (<a 
href="#XFece2015">Fece de la Cruz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFece2015">2015</a>; <a 
href="#XKaelin2009">Kaelin, Jr</a>, <a 
href="#XKaelin2009">2009</a>). When genes are disrupted in cancers, the <a 
href="#glo:induced essentiality">induced essentiality</a> of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners presents a vulnerability that may be exploited for anti-cancer therapy. Since <a 
href="#glo:synthetic lethal">synthetic lethality</a> affects cellular viability by indirect functional relationships between genes, it is suitable for indirectly targeting <a 
href="#glo:mutation">mutations</a> in cancers via <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners with <a 
href="#glo:targeted therapy">targeted therapeutic</a>. These have could be highly specific against cancer cells (with the target <a 
href="#glo:mutation">mutation</a>) over non-cancer cells (with a functional compensating gene). Analogous to “<a 
href="#glo:oncogene addiction">oncogene addiction</a>”, where cancer cells adapt to particular oncogenic growth signals and become reliant on them to remain viable (<a 
href="#XLuo2009">Luo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLuo2009">2009</a>; <a 
href="#XWeinstein2000">Weinstein</a>, <a 
href="#XWeinstein2000">2000</a>), <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of inactivated <a 
href="#glo:tumour suppressor">tumour suppressors</a> are required to maintain cancer cell viability and proliferation. As such cancers are subject to “<a 
href="#glo:non-oncogene addiction">non-oncogene addiction</a>” and these genes are feasible anti-cancer drug targets.
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                    

<p><a 
 id="x1-280012"></a></p>
<!--l. 428--><p class="noindent" >
<img 
src="SL_Concept_no_border.png" alt="PIC"  
 /> <br />
</p><div class="caption" 
>
<span class="id">Figure 1.2: </span><span class="content"><strong><a 
href="#glo:synthetic lethal"><span class="cmbx-10x-x-109">Synthetic lethality</span></a> <span class="cmbx-10x-x-109">in cancer.</span></strong> <span class="cmr-10x-x-109">Rationale of exploiting </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a> <span class="cmr-10x-x-109">for</span> <span class="cmr-10x-x-109">specificity against a </span><a 
href="#glo:tumour suppressor"><span class="cmr-10x-x-109">tumour suppressor</span></a> <span class="cmr-10x-x-109">gene (e.g., </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">) while other cells are spared under</span> <span class="cmr-10x-x-109">the inhibition of a partner gene.</span></span>
</div><!--tex4ht:label?: x1-280012 -->
                                                                                    

</div><hr class="endfigure" />
<!--l. 437--><p class="indent" >
The <a 
href="#glo:synthetic lethal">synthetic lethal</a> approach to cancer medicine is most amenable to loss of function <a 
href="#glo:mutation">mutations</a> in <a 
href="#glo:tumour suppressor">tumour suppressor</a> genes, where it would feasibly be effective against any loss of function <a 
href="#glo:mutation">mutation</a> across the <a 
href="#glo:tumour suppressor">tumour suppressor</a> with a viable <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner gene (as shown in Figure <a 
href="#x1-280012">1.2<!--tex4ht:ref: fig:SL_Concept --></a>). However, the approach may also be suitable for cases where cancer cells have <a 
href="#glo:mutation">mutations</a> where the normal function of the gene is disrupted such as if it were over-expressed (“<a 
href="#glo:synthetic dosage lethal">synthetic dosage lethality</a>”) or if an oncogenic <a 
href="#glo:mutation">mutation</a> interfered with the function of the proto-oncogene. Thus <a 
href="#glo:synthetic lethal">synthetic lethality</a> makes it feasible to target a range of cancer-specific <a 
href="#glo:mutation">mutations</a> with <a 
href="#glo:targeted therapy">targeted therapeutic</a>, including inactivated <a 
href="#glo:tumour suppressor">tumour suppressor</a> genes. <a 
href="#glo:synthetic lethal">synthetic lethality</a> may also enable distinguishing highly homologous <a 
href="#glo:oncogene">oncogenes</a> by functional differences by targeting their <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners. <a 
 id="x1-28002r31"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.2.5 </span> <a 
 id="x1-290005"></a>Clinical Impact of Synthetic Lethality in Cancer
</h4>
<!--l. 445--><p class="noindent" >
The <a 
href="#glo:synthetic lethal">synthetic lethal</a> interaction of <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>with <span class="cmti-12">PARP1 </span>in breast cancer is an example of how gene interactions are important in cancer and these discovery of these interactions has lead to translation to the clinic. These genetic interactions enable specific targeting of <a 
href="#glo:mutation">mutations</a> in <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span><a 
href="#glo:tumour suppressor">tumour suppressor</a> genes with <a 
href="#glo:PARP">Poly-<a 
href="#glo:ADP">ADP</a>-ribose polymerase (PARP)</a> inhibitors by inducing <a 
href="#glo:synthetic lethal">synthetic lethality</a> in breast cancer (<a 
href="#XFarmer2005">Farmer <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFarmer2005">2005</a>). <a 
href="#glo:PARP">PARP</a> inhibitors were one of the first <a 
href="#glo:targeted therapy">targeted therapeutic</a> against a <a 
href="#glo:tumour suppressor">tumour suppressor</a> <a 
href="#glo:mutation">mutation</a> to exhibit success in clinical trials.
</p><!--l. 447--><p class="indent" >
<p><span class="cmti-12">BRCA1 </span>/<span class="cmti-12">BRCA2 </span>and <span class="cmti-12">PARP1 </span>genes demonstrate the application of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> approach to cancer therapy (<a 
href="#XAshworth2008">Ashworth</a>, <a 
href="#XAshworth2008">2008</a>; <a 
href="#XKaelin2005">Kaelin, Jr</a>, <a 
href="#XKaelin2005">2005</a>). <span class="cmti-12">BRCA1 </span>and <span class="cmti-12">BRCA2 </span>are homologous <a 
href="#glo:DNA">DNA</a> repair genes, widely known as <a 
href="#glo:tumour suppressor">tumour suppressors</a>; <a 
href="#glo:mutation">mutation</a> carriers have substantially increased risk of breast (risk by age 70 of 57% for <span class="cmti-12">BRCA1 </span>and 59% for <span class="cmti-12">BRCA2 </span>) and ovarian cancers (risk by age 70 of 40% for <span class="cmti-12">BRCA1 </span>and 18% for <span class="cmti-12">BRCA2</span>) (<a 
href="#XChen2007">Chen and Parmigiani</a>, <a 
href="#XChen2007">2007</a>). The <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>genes, which usually repair <a 
href="#glo:DNA">DNA</a> or destroy the cell if it cannot be repaired, have inactivating <a 
href="#glo:somatic">somatic</a> <a 
href="#glo:mutation">mutations</a> in some <a 
href="#glo:familial">familial</a> and <a 
href="#glo:sporadic">sporadic</a> cancers. <a 
href="#glo:PARP">Poly-<a 
href="#glo:ADP">ADP</a>-ribose polymerase</a> (<a 
href="#glo:PARP">PARP</a>) genes are <a 
href="#glo:tumour suppressor">tumour suppressor</a> genes involved in base excision <a 
href="#glo:DNA">DNA</a> repair. Loss of <a 
href="#glo:PARP">PARP</a> activity results in single-stranded <a 
href="#glo:DNA">DNA</a> breaks. However, <span class="cmti-12">PARP1</span><sup><span class="cmsy-8">-</span><span class="cmmi-8">∕</span><span class="cmsy-8">-</span></sup> knockout mice are viable and healthy indicating low toxicity from <a 
href="#glo:PARP">PARP</a> inhibition (<a 
href="#XBryant2005">Bryant <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBryant2005">2005</a>).</p>
</p><!--l. 449--><p class="indent" >
<a 
href="#XBryant2005">Bryant <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XBryant2005">2005</a>) showed that <span class="cmti-12">BRCA2 </span>cells were sensitive to <a 
href="#glo:PARP">PARP</a> inhibition by <a 
href="#glo:siRNA">siRNA</a> of <span class="cmti-12">PARP1 </span>or drug inhibition (which targets <span class="cmti-12">PARP1 </span>and <span class="cmti-12">PARP2</span>) using Chinese hamster ovary cells, MCF7 and MDA-MB-231 breast cell lines. This effect was sufficient to kill mouse tumour xenografts and showed high specificity to <span class="cmti-12">BRCA2 </span>deficient cells in culture and xenografts. <a 
href="#XFarmer2005">Farmer <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XFarmer2005">2005</a>) replicated these results in embryonic stem cells and showed that <span class="cmti-12">BRCA1 </span>cells were also sensitive to <a 
href="#glo:PARP">PARP</a> inhibition relative to the <a 
href="#glo:wild-type">wild-type</a> with <a 
href="#glo:siRNA">siRNA</a> and drug experiments in cell culture and drug activity against <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>deficient embryonic stem cell mouse xenografts. They found evidence that <a 
href="#glo:PARP">PARP</a> inhibition causes <a 
href="#glo:DNA">DNA</a> lesions, usually repaired in <a 
href="#glo:wild-type">wild-type</a> cells, which lead to chromosomal instability, cell cycle arrest, and induction of apoptosis in <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>deficient cells. The combined loss of <a 
href="#glo:DNA">DNA</a> repairs <a 
href="#glo:pathway">pathways</a> gives a plausible mechanism for an effective anti-cancer treatment.
</p><!--l. 451--><p class="indent" >
Thus <a 
href="#glo:PARP">PARP</a> inhibitors could be applied with clinical use against <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2</span> <a 
href="#glo:mutation">mutations</a> in both <a 
href="#glo:hereditary">hereditary</a> and <a 
href="#glo:sporadic">sporadic</a> cancers (<a 
href="#XAshworth2008">Ashworth</a>, <a 
href="#XAshworth2008">2008</a>; <a 
href="#XKaelin2005">Kaelin, Jr</a>, <a 
href="#XKaelin2005">2005</a>). <a 
href="#glo:PARP">PARP</a> inhibition has been found to be effective in ovarian cancer patients carrying <span class="cmti-12">BRCA1</span> or <span class="cmti-12">BRCA2 </span><a 
href="#glo:mutation">mutations</a> and some patient without these <a 
href="#glo:mutation">mutations</a>, suggesting <a 
href="#glo:synthetic lethal">synthetic lethality</a> between <a 
href="#glo:PARP">PARP</a> and other <a 
href="#glo:DNA">DNA</a> repair <a 
href="#glo:pathway">pathways</a> (<a 
href="#XStrom2012">Ström and Helleday</a>, <a 
href="#XStrom2012">2012</a>). This supports the potential for <a 
href="#glo:PARP">PARP</a> inhibition as a <a 
href="#glo:chemoprevention">chemopreventative</a> alternative to prophylactic surgery for high-risk individuals with <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span><a 
href="#glo:mutation">mutations</a> (<a 
href="#XStrom2012">Ström and Helleday</a>, <a 
href="#XStrom2012">2012</a>). Hormone-based therapy has also been suggested as a <a 
href="#glo:chemoprevention">chemopreventative</a> in such high-risk individuals and aromatase inhibitors have completed phase I clinical trials for this purpose (<a 
href="#XBozovic-Spasojevic2012">Bozovic-Spasojevic <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBozovic-Spasojevic2012">2012</a>). <a 
href="#XStrom2012">Ström and Helleday</a> (<a 
href="#XStrom2012">2012</a>) also postulate increased efficacy of <a 
href="#glo:PARP">PARP</a> inhibitors in the hypoxic <a 
href="#glo:DNA">DNA</a>-damaging tumour micro-environment.
</p><!--l. 453--><p class="indent" >
<p>A <a 
href="#glo:PARP">PARP</a> inhibitor, olaparib, showed fewer adverse effects than cytotoxic <a 
href="#glo:chemotherapy">chemotherapy</a> and anti-tumour activity in various clinical trials against <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>deficient <a 
href="#glo:familial">familial</a> or <a 
href="#glo:sporadic">sporadic</a> breast, ovarian, and prostate cancers (<a 
href="#XAudeh2010">Audeh <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XAudeh2010">2010</a>; <a 
href="#XFong2009">Fong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFong2009">2009</a>, <a 
href="#XFong2010">2010</a>; <a 
href="#XTutt2010">Tutt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTutt2010">2010</a>). This treatment has a favourable therapeutic window and similarly low toxicity between <a 
href="#glo:mutation">mutation</a> carriers of <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span><a 
href="#glo:mutation">mutations</a> and <a 
href="#glo:sporadic">sporadic</a> cases. These <a 
href="#glo:PARP">PARP</a> inhibitors have been FDA approved for some cancers <a 
href="#XMcLachlan2016">McLachlan <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XMcLachlan2016">2016</a>), are effective against <a 
href="#glo:germline">germline mutation</a> and <a 
href="#glo:sporadic">sporadic</a> <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span><a 
href="#glo:mutation">mutations</a>, and are a potential prevention alternative to prophylactic surgery for high-risk <a 
href="#glo:mutation">mutation</a> carriers <a 
href="#XStrom2012">Ström and Helleday</a> (<a 
href="#XStrom2012">2012</a>).</p>
</p><!--l. 459--><p class="indent" >
This demonstrates the clinical impact of a well characterised system of <a 
href="#glo:synthetic lethal">synthetic lethality</a> with known cancer risk genes. <a 
href="#glo:synthetic lethal">Synthetic lethality</a> has the benefit of being effective against inactivation of <a 
href="#glo:tumour suppressor">tumour suppressor</a> genes by any means, broader than targeting a specific oncogenic <a 
href="#glo:mutation">mutation</a> (<a 
href="#XKaelin2005">Kaelin, Jr</a>, <a 
href="#XKaelin2005">2005</a>). The <a 
href="#glo:targeted therapy">targeted therapy</a> is effective in both <a 
href="#glo:sporadic">sporadic</a> and <a 
href="#glo:hereditary">hereditary</a> <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>deficient <a 
href="#glo:tumour">tumours</a> acting against an oncogenic molecular aberration across several tissues. <a 
 id="x1-29001r33"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.2.6 </span> <a 
 id="x1-300006"></a>High-throughput Screening for Synthetic Lethality
</h4>
<!--l. 476--><p class="noindent" >
<a 
href="#glo:RNAi">RNA interference</a> (<a 
href="#glo:RNAi">RNAi</a>) technologies have enabled extensive investigations of <a 
href="#glo:functional redundancy">genetic redundancy</a> in mammalian experimental models including testing experimentally for <a 
href="#glo:synthetic lethal">synthetic lethality</a> (<a 
href="#XFraser2004">Fraser</a>, <a 
href="#XFraser2004">2004</a>). <a 
href="#glo:synthetic lethal">Synthetic lethal</a> <a 
href="#glo:RNAi">RNAi</a> screens are performed, using <a 
href="#glo:siRNA">short interfering RNA</a> (<a 
href="#glo:siRNA">siRNA</a>) or <a 
href="#glo:shRNA">short hairpin RNA (shRNA)</a> to target specific genes is isogenic cells. Identifying <a 
href="#glo:synthetic lethal">synthetic lethality</a> is crucial for studying gene function, drug mechanisms, and design novel therapies (<a 
href="#XLum2004">Lum <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLum2004">2004</a>). Candidate selection of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs relevant to cancer has shown some success but is limited because interactions are difficult to predict; they can occur between seemingly unrelated <a 
href="#glo:pathway">pathways</a> in model organisms (<a 
href="#XCostanzo2011">Costanzo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCostanzo2011">2011</a>). While biologically informed hypotheses have had some success in <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery (<a 
href="#XBitler2015">Bitler <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBitler2015">2015</a>; <a 
href="#XBryant2005">Bryant <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBryant2005">2005</a>; <a 
href="#XFarmer2005">Farmer <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFarmer2005">2005</a>), interactions occurring indirectly between distinct <a 
href="#glo:pathway">pathways</a> would be missed (<a 
href="#XBoone2007">Boone <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoone2007">2007</a>; <a 
href="#XCostanzo2011">Costanzo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCostanzo2011">2011</a>). Scanning the entire <a 
href="#glo:genome">genomes</a> for interactions against a clinically relevant gene is an emerging strategy being explored with <a 
href="#glo:high-throughput screen">high-throughput screens</a> (<a 
href="#XFece2015">Fece de la Cruz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFece2015">2015</a>) and computational approaches (<a 
href="#XBoucher2013">Boucher and Jenna</a>, <a 
href="#XBoucher2013">2013</a>; <a 
href="#XvanSteen2011">van Steen</a>, <a 
href="#XvanSteen2011">2012</a>). <a 
href="#glo:high-throughput screen"></a> <a 
href="#glo:compound screen"></a> <a 
href="#glo:RNAi screen"></a>
</p><!--l. 478--><p class="indent" >
<p>Experimental <a 
href="#glo:synthetic lethal screen">screening</a> for <a 
href="#glo:synthetic lethal">synthetic lethality</a> is an appealing strategy for wider discovery of functional interactions <span class="cmti-12">in vivo </span>despite many potential sources of error which must be considered. The <a 
href="#glo:synthetic lethal">synthetic lethal</a> concept has both genetic and pharmacological screening applications to cancer research. Genetic screens, with <a 
href="#glo:RNAi">RNAi</a> to discover the specific genes involved, inform development of targeted therapies with a known mode of action, anticipated mechanisms of resistance, and biomarkers for treatment response. <a 
href="#glo:RNAi">RNAi</a> is a transient knockdown of <a 
href="#glo:gene expression">gene expression</a> more similar to the effect of drugs than complete gene loss and is more representative of disease than model organisms (<a 
href="#XBussey2006">Bussey <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBussey2006">2006</a>). The <a 
href="#glo:RNAi">RNAi</a> gene knockdown process has inherent toxicity to some cells, potential off-target effects, and issues with a high false positive rate. Therefore, it is important to validate any</p>
candidates in a secondary screen and replicate knockdown experiments with a number of independent <a 
href="#glo:shRNA">shRNAs</a>. <a 
href="#glo:RNAi screen">Genetic screens</a> have potential for quantitative gene disruption experiments to selectively target over-expressed genes in cancer via <a 
href="#glo:synthetic dosage lethal">synthetic dosage lethality</a>. While powerful for understanding fundamental cellular function, analysis of isogenic cell lines is inherently limited by assuming only a single <a 
href="#glo:mutation">mutation</a> differs between them and cannot account for diverse genetic backgrounds or tumour heterogeneity (<a 
href="#XFece2015">Fece de la Cruz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFece2015">2015</a>). Genetic screens can thus identify targets to develop, or can repurpose targeted therapies for disease, but alone will not directly identify a lead compound to develop for the market or for clinical translation.
</p><!--l. 482--><p class="indent" >
<a 
href="#glo:compound screen">Chemical screens</a> are immediately applicable to the clinic, as they are directly screening for selective lead compounds with suitable pharmacological properties. However, chemical screens lack a known mode of action, may affect many targets, and screen a narrow range of genes with existing drugs. With either approach there are still many challenges to translating candidates into the clinic. Identifying specific target genes may contribute to overcoming such challenges, which can be approached with genetic screens and computational alternatives. Screening methods have proven a fruitful area of research, despite being costly, laborious, and having many different sources of error. These limitations suggest a need for complementary computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery. <a 
 id="x1-30001r30"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.6.1 </span> <a 
 id="x1-310001"></a>Synthetic Lethal Screens
</h5>
<!--l. 493--><p class="noindent" >
<p>Synthetic lethal screens have been conducted for <a 
href="#glo:cancer gene">cancer genes</a> in a variety of <a 
href="#glo:cancer">cancers</a>. These have found <a 
href="#glo:synthetic lethal">synthetic lethality</a> of <span class="cmti-12">PIM1 </span>over-expression with <span class="cmti-12">PLK1 </span>inhibition in prostate cells (<a 
href="#XvanderMeer2014">van der Meer <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XvanderMeer2014">2014</a>), <span class="cmti-12">FH </span>null mutations (involved in <a 
href="#glo:HLRCC">Hereditary leiomyomatosis and renal cell carcinoma (HLRCC)</a>) with inhibition of adenylate cyclases (<a 
href="#XBoettcher2014">Boettcher <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoettcher2014">2014</a>), and <span class="cmti-12">WEE1 </span>inhibitor treatment with knockdown of checkpoint kinases, Fanconi anaemia, and homologous recombination in colorectal cells (<a 
href="#XAarts2015">Aarts <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XAarts2015">2015</a>). These results include genes that have been found to be co-expressed in cancers, are consistent with those identified in the literature, and that were successfully validated with <a 
href="#glo:RNAi">RNAi</a> and drug experiments. These findings demonstrate that <a 
href="#glo:synthetic lethal">synthetic lethal</a> screening can identify partner genes with clinical relevance as biomarkers, therapeutic targets, or conferring sensitivity to existing treatments. These are of</p>
particular importance for familial cancer syndromes (<a 
href="#XBoettcher2014">Boettcher <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoettcher2014">2014</a>; <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>).
</p><!--l. 495--><p class="indent" >
<a 
href="#glo:HDGC">Hereditary diffuse gastric cancer (HDGC)</a> is a cancer syndrome involving predisposition to early-onset malignant stomach and breast cancers that has been attributed to inactivating <a 
href="#glo:mutation">mutations</a> in <a 
href="#glo:E-cadherin">E-cadherin</a>, encoded by <span class="cmti-12">CDH1 </span>(as discussed in Section <a 
href="#x1-400003">1.3<!--tex4ht:ref: CDH1_section --></a>). <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) performed an <a 
href="#glo:RNAi">RNAi</a> screen on MCF10A breast cells for <a 
href="#glo:synthetic lethal">synthetic lethality</a> with <span class="cmti-12">CDH1</span>. In conjunction with a drug compound screen, inhibitors of <a 
href="#glo:JAK">Janus kinase (JAK)</a>, <a 
href="#glo:HDAC">histone deacetylase (HDAC)</a>, <a 
href="#glo:PI3K">phosphoinositide 3-kinase (PI3K)</a>, aurora kinase, and tyrosine kinases were demonstrated to be <a 
href="#glo:synthetic lethal">synthetic lethal</a> with <span class="cmti-12">CDH1</span>. Therefore the <a 
href="#glo:synthetic lethal">synthetic lethal</a> strategy shows potential to achieve clinical impact against <a 
href="#glo:HDGC">HDGC</a> by the identification of compounds suitable for use in <a 
href="#glo:chemoprevention">chemoprevention</a>.
</p><!--l. 510--><p class="indent" >
The examples above show that high-throughput screens are an effective approach to discover <a 
href="#glo:synthetic lethal">synthetic lethality</a> in cancer. Screens have the power to test mode of action of drugs, find unexpected <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions between <a 
href="#glo:pathway">pathways</a>, or identify effective treatment strategies without prior knowledge of a mechanism. However, <a 
href="#glo:synthetic lethal">synthetic lethal</a> screens are costly, labour-intensive, error-prone, and biased towards genes with effective <a 
href="#glo:RNAi">RNAi</a> knockdown libraries. Off-target effects and inconsistent replication <a 
href="#glo:synthetic lethal">synthetic lethality</a> across different cell lines, tissues, or laboratories, are also problematic. Therefore there is a need for replication, validation, and alternative approaches to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates. In addition, varied conditions across experimental screens and differences between <a 
href="#glo:RNAi">RNAi</a> and drug screens makes meta-analysis extremely challenging.
</p><!--l. 518--><p class="indent" >
Genome-scale <a 
href="#glo:synthetic lethal">synthetic lethal</a> experiments (across gene pairs) are not feasible, even in model organisms, and these studies typically focus on specific gene candidates or the partners of a gene of interest. Therefore a computational approach is more suitable for this task and may also augment existing experimental screens. <a 
 id="x1-31001r34"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.2.7 </span> <a 
 id="x1-320007"></a>Computational Prediction of Synthetic Lethality
</h4>
<a 
 id="x1-32001r35"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.7.1 </span> <a 
 id="x1-330001"></a>Bioinformatics Approaches to Genetic Interactions
</h5>
<!--l. 545--><p class="noindent" >
<p>Prediction of gene interaction networks is a feasible alternative to high-throughput screening, and has both biological importance and clinical relevance. There are many existing methods to predict gene networks, as reviewed by <a 
href="#XvanSteen2011">van Steen</a> (<a 
href="#XvanSteen2011">2012</a>) and <a 
href="#XBoucher2013">Boucher and Jenna</a> (<a 
href="#XBoucher2013">2013</a>) and summarised in Table <a 
href="#x1-330011">1.1<!--tex4ht:ref: tab:methods_model --></a>. However, many of these methods have limitations, including</p>
the requirement for existing <a 
href="#glo:SGI">SGI</a> data, several data inputs, and reliability of gene function annotation. Many of the existing methods also assume conservation of individual interactions between species, which has been found not to hold in yeast studies (<a 
href="#XDixon2008">Dixon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDixon2008">2008</a>). Tissue specificity is important in gene regulation and <a 
href="#glo:gene expression">gene expression</a>, which are used as predictors of genetic interaction. However, tissue specificity of genetic interactions cannot be explored in yeast studies and has not been considered in many studies of multicellular model organisms, human networks, or cancers. Similarly, investigation into tissue specificity of <a 
href="#glo:PPI">PPIs</a>, an important predictor of genetic interactions, is difficult given that high-throughput two-hybrid screens occur out of cellular context for multicellular organisms (<a 
href="#XBruckner2009">Brückner <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBruckner2009">2009</a>).
</p>
<div class="table">
                                                                                    

<!--l. 547--><p class="indent" >
<a 
 id="x1-330011"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 1.1: </span><span class="content">Methods for predicting genetic interactions</span>
</div><!--tex4ht:label?: x1-330011 -->
<!--tex4ht:inline--><div class="tabular"><table id="TBL-5" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-5-1g"><col 
id="TBL-5-1" /><col 
id="TBL-5-2" /><col 
id="TBL-5-3" /><col 
id="TBL-5-4" /><col 
id="TBL-5-5" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-5-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-1-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<strong><span class="cmbx-12">Method</span></strong>
</div>                                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-1-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<strong><span class="cmbx-12">Input Data</span></strong>
</div>                               </td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-1-3"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<strong><span class="cmbx-12">Species</span></strong>
</div>                                                                              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-1-4"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<strong><span class="cmbx-12">Source</span></strong>
</div>                                                                              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-1-5"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<strong><span class="cmbx-12">Tool Offered</span></strong>
</div>
</td></tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-2-1"  
class="td11"> 
<span id="textcolor1">Between Pathways Model</span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-2-2"  
class="td11"> 
<a 
href="#glo:PPI">PPI</a>, <a 
href="#glo:SGI">SGI</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-2-3"  
class="td11"> 
<span id="textcolor2"><span class="cmti-12">S. cerevisiae</span></span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-2-4"  
class="td11"> 
<a 
href="#XKelley2005">Kelley and Ideker</a> (<a 
href="#XKelley2005">2005</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-2-5"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-3-1"  
class="td11"> 
Within Pathways Model
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-3-2"  
class="td11"> 
PPI, <a 
href="#glo:SGI">SGI</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-3-3"  
class="td11"> 
<span class="cmti-12">S. cerevisiae </span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-3-4"  
class="td11"> 
<a 
href="#XKelley2005">Kelley and Ideker</a> (<a 
href="#XKelley2005">2005</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-3-5"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-4-1"  
class="td11"> 
<span id="textcolor3">Decision Tree</span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-4-2"  
class="td11"> 
<a 
href="#glo:PPI">PPI</a>, expression, phenotype
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-4-3"  
class="td11"> 
<span id="textcolor4"><span class="cmti-12">S. cerevisiae</span></span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-4-4"  
class="td11"> 
<a 
href="#XWong2004">Wong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWong2004">2004</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-4-5"  
class="td11"> 
2 Hop
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-5-1"  
class="td11"> 
Logistic Regression
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-5-2"  
class="td11"> 
<a 
href="#glo:SGI">SGI</a>, <a 
href="#glo:PPI">PPI</a>, co-expression, phenotype
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-5-3"  
class="td11"> 
<span class="cmti-12">C. elegans </span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-5-4"  
class="td11"> 
<a 
href="#XZhong2006">Zhong and Sternberg</a> (<a 
href="#XZhong2006">2006</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-5-5"  
class="td11"> 
Gene Orienteer
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-6-1"  
class="td11"> 
<span id="textcolor5">Network Sampling</span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-6-2"  
class="td11"> 
<a 
href="#glo:SGI">SGI</a>, <a 
href="#glo:PPI">PPI</a>, GO
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-6-3"  
class="td11"> 
<span id="textcolor6"><span class="cmti-12">S. cerevisiae</span></span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-6-4"  
class="td11">  <!--tex4ht:inline--><div class="tabular"> <table id="TBL-6" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-6-1g"><col 
id="TBL-6-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-6-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-1-1"  
class="td00">
<a 
href="#XLeMeur2008">Le Meur and Gentleman</a> (<a 
href="#XLeMeur2008">2008</a>)
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-6-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-6-2-1"  
class="td00">
<a 
href="#XLeMeur2014">Le Meur <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLeMeur2014">2014</a>)
</td>
</tr></table>                                                                                                                 </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-6-5"  
class="td11"> 
SLGI(R)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-7-1"  
class="td11"> 
Random Walk
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-7-2"  
class="td11"> 
GO, <a 
href="#glo:PPI">PPI</a>, <a 
href="#glo:gene expression">expression</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-7-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-7" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-7-1g"><col 
id="TBL-7-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-7-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-7-1-1"  
class="td00">
<span class="cmti-12">S. cerevisiae</span>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-7-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-7-2-1"  
class="td00">
<span class="cmti-12">C. elegans</span>
</td></tr></table>                                                                                       </div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-7-4"  
class="td11"> 
<a 
href="#XChipman2009">Chipman and Singh</a> (<a 
href="#XChipman2009">2009</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-7-5"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-8-1"  
class="td11"> 
<span id="textcolor7">Shared Function</span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-8-2"  
class="td11"> 
Co-expression, <a 
href="#glo:PPI">PPI</a>, text mining, phylogeny
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-8-3"  
class="td11"> 
<span id="textcolor8"><span class="cmti-12">C. elegans</span></span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-8-4"  
class="td11"> 
<a 
href="#XLee2010b">Lee <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLee2010b">2010b</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-8-5"  
class="td11"> 
WormNet
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-9-1"  
class="td11"> 
Logistic Regression
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-9-2"  
class="td11"> 
Co-expression, <a 
href="#glo:PPI">PPI</a>, phenotype
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-9-3"  
class="td11"> 
<span class="cmti-12">C. elegans </span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-9-4"  
class="td11"> 
<a 
href="#XLee2010a">Lee <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLee2010a">2010a</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-9-5"  
class="td11"> 
GI Finder
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-10-1"  
class="td11"> 
<span id="textcolor9">Jaccard Index</span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-10-2"  
class="td11"> 
GO, <a 
href="#glo:SGI">SGI</a>, PPI, phenotype
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-10-3"  
class="td11"> 
Eukarya
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-10-4"  
class="td11"> 
<a 
href="#XHoehndorf2013">Hoehndorf <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XHoehndorf2013">2013</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-10-5"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-11-1"  
class="td11"> 
<span id="textcolor10">Machine Learning</span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-11-2"  
class="td11"> 
 
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-11-3"  
class="td11"> 
 
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-11-4"  
class="td11"> 
<a 
href="#XPandey2010">Pandey <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XPandey2010">2010</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-11-5"  
class="td11"> 
MNMC
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-12-1"  
class="td11"> 
Machine Learning Meta-Analysis
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-12-2"  
class="td11"> 
 
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-12-3"  
class="td11"> 
 
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-12-4"  
class="td11"> 
<a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWu2014">2014</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-12-5"  
class="td11"> 
MetaSL
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-13-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-8" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-8-1g"><col 
id="TBL-8-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-8-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-8-1-1"  
class="td00">
<span id="textcolor11">Flux Variability Analysis</span>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-8-2-1"  
class="td00">
<span id="textcolor12">Flux Balance Analysis</span>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-8-3-1"  
class="td00">
<span id="textcolor13">Network Simulation</span>
</td></tr></table>                                                                            </div>   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-13-2"  
class="td11"> 
Metabolism
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-13-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-9" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-9-1g"><col 
id="TBL-9-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-9-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-9-1-1"  
class="td00">
<span id="textcolor14"><span class="cmti-12">E. coli</span></span>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-9-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-9-2-1"  
class="td00">
<span id="textcolor15"><span class="cmti-12">M. pneumoniae</span></span>
</td></tr></table>                                                                                   </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-13-4"  
class="td11"> 
<a 
href="#XGuell2014">Güell <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XGuell2014">2014</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-13-5"  
class="td11"> 
 
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-14-1"  
class="td11">                                                                          </td></tr></table>                                                                                                                                                                                                                                                      </div>
                                                                                    

</div><hr class="endfloat" />
</div>
                                                                                    

<!--l. 711--><p class="indent" >
<a 
 id="x1-330022"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 1.2: </span><span class="content">Methods for predicting synthetic lethality in cancer</span>
</div><!--tex4ht:label?: x1-330022 -->
<!--tex4ht:inline--><div class="tabular"><table id="TBL-10" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-10-1g"><col 
id="TBL-10-1" /><col 
id="TBL-10-2" /><col 
id="TBL-10-3" /><col 
id="TBL-10-4" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-10-1-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-10-1-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left; vertical-align:middle;">
<span class="cmbx-12">Method</span>
</div>                                                                               </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-10-1-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left; vertical-align:middle;">
<span id="textcolor16"><span class="cmbx-12">Input Data</span></span>
</div>                                                                          </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-10-1-3"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left; vertical-align:middle;">
<span class="cmbx-12">Source</span>
</div>                                                                                </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-1-4"  
class="td11"> 
<span class="cmbx-12">Tool Offered</span>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-10-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-2-1"  
class="td11"> 
<span id="textcolor17">Network Centrality</span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-2-2"  
class="td11"> 
protein-protein interactions
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-2-3"  
class="td11"> 
<a 
href="#XKranthi2013">Kranthi <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XKranthi2013">2013</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-2-4"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-3-1"  
class="td11"> 
Differential Expression
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-3-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-11" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-11-1g"><col 
id="TBL-11-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-11-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-11-1-1"  
class="td00">
Expression
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-11-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-11-2-1"  
class="td00">
Mutation
</td></tr></table>                                                                                                </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-3-3"  
class="td11"> 
<a 
href="#XWang2013">Wang and Simon</a> (<a 
href="#XWang2013">2013</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-3-4"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-4-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-12" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-12-1g"><col 
id="TBL-12-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-12-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-12-1-1"  
class="td00">
<span id="textcolor18">Comparative <a 
href="#glo:genomic">Genomic</a></span>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-12-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-12-2-1"  
class="td00">
<span id="textcolor19">Chemical-<a 
href="#glo:genomic">Genomic</a></span>
</td></tr></table>                                                                               </div>   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-4-2"  
class="td11">  <!--tex4ht:inline--><div class="tabular"> <table id="TBL-13" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-13-1g"><col 
id="TBL-13-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-13-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-13-1-1"  
class="td00">
Yeast synthetic gene interactions
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-13-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-13-2-1"  
class="td00">
Homology
</td></tr></table>                                                                  </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-4-3"  
class="td11"> 
<a 
href="#XHeiskanen2012">Heiskanen and Aittokallio</a> (<a 
href="#XHeiskanen2012">2012</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-4-4"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-5-1"  
class="td11"> 
Comparative <a 
href="#glo:genomic">Genomic</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-5-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-14" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-14-1g"><col 
id="TBL-14-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-14-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-14-1-1"  
class="td00">
Yeast synthetic gene interactions
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-14-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-14-2-1"  
class="td00">
Homology
</td></tr></table>                                                                   </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-5-3"  
class="td11"> 
<a 
href="#XDeshpande2013">Deshpande <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XDeshpande2013">2013</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-5-4"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-6-1"  
class="td11"> 
Machine Learning
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-6-2"  
class="td11"> 
 
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-6-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-15" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-15-1g"><col 
id="TBL-15-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-15-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-15-1-1"  
class="td00">
Discussed by <a 
href="#XBabyak2004">Babyak</a> (<a 
href="#XBabyak2004">2004</a>)
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-15-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-15-2-1"  
class="td00">
and <a 
href="#XLee2009">Lee and Marcotte</a> (<a 
href="#XLee2009">2009</a>)
</td>
</tr></table>                                                                                                                 </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-6-4"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-7-1"  
class="td11"> 
<span id="textcolor20">Differential Expression</span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-7-2"  
class="td11"> 
Expression
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-7-3"  
class="td11"> 
<a 
href="#XTiong2014">Tiong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTiong2014">2014</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-7-4"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-8-1"  
class="td11"> 
Literature Database
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-8-2"  
class="td11"> 
 
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-8-3"  
class="td11"> 
<a 
href="#XLi2014">Li <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLi2014">2014</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-8-4"  
class="td11"> 
Syn-Lethality
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-9-1"  
class="td11"> 
<span id="textcolor21">Meta-Analysis</span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-9-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-16" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-16-1g"><col 
id="TBL-16-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-16-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-16-1-1"  
class="td00">
Meta-Analysis
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-16-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-16-2-1"  
class="td00">
Machine Learning
</td></tr></table>                                                                                        </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-9-3"  
class="td11"> 
<a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWu2014">2014</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-9-4"  
class="td11"> 
MetaSL
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-10-1"  
class="td11"> 
Pathway Analysis
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-10-2"  
class="td11"> 
 
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-10-3"  
class="td11"> 
<a 
href="#XZhang2015">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XZhang2015">2015</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-10-4"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-11-1"  
class="td11"> 
<span id="textcolor22">Protein Domains</span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-11-2"  
class="td11"> 
Homology
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-11-3"  
class="td11"> 
<a 
href="#XKozlov2015">Kozlov <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XKozlov2015">2015</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-11-4"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-12-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-17" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-17-1g"><col 
id="TBL-17-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-17-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-17-1-1"  
class="td00">
Data-Mining
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-17-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-17-2-1"  
class="td00">
Machine Learning
</td></tr></table>                                                                                       </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-12-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-18" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-18-1g"><col 
id="TBL-18-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-18-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-18-1-1"  
class="td00">
Expression
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-18-2-1"  
class="td00">
Somatic <a 
href="#glo:mutation">mutation</a> and <a 
href="#glo:DNA">DNA</a> <a 
href="#glo:CNV">CNV</a>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-18-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-18-3-1"  
class="td00">
siRNA in cell lines
</td></tr></table>                                                                   </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-12-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-19" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-19-1g"><col 
id="TBL-19-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-19-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-19-1-1"  
class="td00">
<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XJerby2014">2014</a>)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-19-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-19-2-1"  
class="td00">
<a 
href="#XRyan2014">Ryan <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XRyan2014">2014</a>)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-19-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-19-3-1"  
class="td00">
<a 
href="#XCrunkhorn2014">Crunkhorn</a> (<a 
href="#XCrunkhorn2014">2014</a>)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-19-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-19-4-1"  
class="td00">
<a 
href="#XLokody2014">Lokody</a> (<a 
href="#XLokody2014">2014</a>)
</td></tr></table>                                                                          </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-12-4"  
class="td11"> 
<a 
href="#glo:DAISY">DAISY</a> (method)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-13-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-20" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-20-1g"><col 
id="TBL-20-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-20-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-1-1"  
class="td00">
<span id="textcolor23">Genome Evolution</span>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-20-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-2-1"  
class="td00">
<span id="textcolor24">Hypothesis Test</span>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-20-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-3-1"  
class="td00">
<span id="textcolor25">Machine Learning</span>
</td></tr></table>                                                                                    </div>   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-13-2"  
class="td11">  <!--tex4ht:inline--><div class="tabular"> <table id="TBL-21" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-21-1g"><col 
id="TBL-21-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-21-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-21-1-1"  
class="td00">
Expression
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-21-2-1"  
class="td00">
DNA <a 
href="#glo:CNV">CNV</a>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-21-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-21-3-1"  
class="td00">
Known SL
</td></tr></table>                                                                                              </div>   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-13-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-22" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-22-1g"><col 
id="TBL-22-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-22-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-22-1-1"  
class="td00">
<a 
href="#XLu2013">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2013">2013</a>)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-22-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-22-2-1"  
class="td00">
<a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2015">2015</a>)
</td></tr></table>                                                                                       </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-13-4"  
class="td11"> 
 
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-14-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-23" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-23-1g"><col 
id="TBL-23-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-23-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-1-1"  
class="td00">
<span id="textcolor26">Bimodality</span>
</td></tr></table>                                                                                               </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-14-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-24" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-24-1g"><col 
id="TBL-24-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-24-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-24-1-1"  
class="td00">
Expression
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-24-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-24-2-1"  
class="td00">
DNA <a 
href="#glo:CNV">CNV</a>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-24-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-24-3-1"  
class="td00">
Somatic Mutation
</td></tr></table>                                                                                        </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-14-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-25" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-25-1g"><col 
id="TBL-25-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-25-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-25-1-1"  
class="td00">
<a 
href="#XWappett2014">Wappett</a> (<a 
href="#XWappett2014">2014</a>)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-25-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-25-2-1"  
class="td00">
<a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWappett2016">2016</a>)
</td></tr></table>                                                                                </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-14-4"  
class="td11"> 
<a 
href="#glo:BiSEp">BImodal Subsetting ExPression (BiSEp)</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-15-1"  
class="td11"> 
Directional Chi-Square
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-15-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-26" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-26-1g"><col 
id="TBL-26-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-26-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-26-1-1"  
class="td00">
Expression (<a 
href="#glo:microarray">microarray</a>)
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-26-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-26-2-1"  
class="td00">
Somatic <a 
href="#glo:mutation">mutation</a>
</td></tr></table>                                                                               </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-15-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-27" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-27-1g"><col 
id="TBL-27-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-27-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-27-1-1"  
class="td00">
Kelly, S. T., Guilford, P. J., and Black, M. A.
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-27-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-27-2-1"  
class="td00">
Dissertation (Kelly, 2013) and developed here
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-27-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-27-3-1"  
class="td00">                                       </td></tr></table>                               </div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-15-4"  
class="td11"> 
SLIPT
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-10-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-10-16-1"  
class="td11">                                                                          </td></tr></table>                                                                                                                                                                                                                               </div>
                                                                                    

</div><hr class="endfloat" />
<!--l. 909--><p class="indent" >
There are existing computational methods for predicting <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs in humans, with a specific emphasis on cancer (as summarised in <a 
href="#x1-330022">1.2<!--tex4ht:ref: tab:methods_SL --></a>). While these demonstrate the power and need for predictions of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in human and cancer contexts, limitations of previous methods could be met with a different approach. Existing computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> prediction are often difficult to interpret or replicate for new genes, or are reliant on data types not available for a wider range of genes to test. <a 
 id="x1-33003r37"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.7.2 </span> <a 
 id="x1-340002"></a>Comparative Genomics
</h5>
<!--l. 913--><p class="noindent" >
A comparative <a 
href="#glo:genomic">genomic</a> approach by <a 
href="#XDeshpande2013">Deshpande <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XDeshpande2013">2013</a>) used the results of well characterised high-throughput <a 
href="#glo:mutation">mutation</a> screens in <span class="cmti-12">S. cerevisiae </span>as candidates for <a 
href="#glo:synthetic lethal">synthetic lethality</a> in humans (<a 
href="#XBaryshnikova2010a">Baryshnikova <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBaryshnikova2010a">2010a</a>; <a 
href="#XCostanzo2010">Costanzo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCostanzo2010">2010</a>, <a 
href="#XCostanzo2011">2011</a>; <a 
href="#XTong2001">Tong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTong2001">2001</a>, <a 
href="#XTong2004">2004</a>). Yeast <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners were compared to human orthologues to find cancer relevant <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate pairs with direct therapeutic potential. Proposed as a complementary approach to <a 
href="#glo:siRNA">siRNA</a> screens, approximately 24,000 of the 116,000 negative <a 
href="#glo:SGI">SGI</a> in yeast (<a 
href="#XCostanzo2011">Costanzo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCostanzo2011">2011</a>) were matched to human orthologues, with over 500 involving a <a 
href="#glo:cancer gene">cancer gene</a> (<a 
href="#XFutreal2004">Futreal <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFutreal2004">2004</a>). Under strict criteria of one-to-one orthologues, large effect size and significant interaction in yeast data, 1522 interactions were identified with 70 involving <a 
href="#glo:cancer gene">cancer genes</a>. Of the 21 gene interactions tested with pairs of <a 
href="#glo:siRNA">siRNA</a> in IMR1 fibroblast cells, 6 exhibited <a 
href="#glo:synthetic lethal">synthetic lethal</a> effects. The two strongest interactions (<span class="cmti-12">SMARCB1 </span>with <span class="cmti-12">PSMA4 </span>and <span class="cmti-12">ASPSCR1 </span>with <span class="cmti-12">PSMC2</span>) were successfully validated by protein analysis of human cells and replication with tetrad analysis for yeast orthologues.
</p><!--l. 917--><p class="indent" >
<p>Another approach to systematic <a 
href="#glo:synthetic lethal">synthetic lethality</a> discovery specific to human cancer (in contrast to the plethora of yeast <a 
href="#glo:synthetic lethal">synthetic lethality</a> data) was to build a database as done by <a 
href="#XLi2014">Li <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLi2014">2014</a>). In their relational database, called “Syn-lethality”, they have curated both known experimentally discovered <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs in humans (113 pairs) from the literature and those predicted from <a 
href="#glo:synthetic lethal">synthetic lethality</a> between orthologous genes in <span class="cmti-12">S.</span> <span class="cmti-12">cerevisiae </span>yeast (1114 pairs). This knowledge-based database is the first dedicated to human cancer <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions and integrates gene function annotation, <a 
href="#glo:pathway">pathway</a> and molecular mechanism data with experimental and predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs. This combination of data sources is intended to tackle the trade-off between more conclusive <a 
href="#glo:synthetic lethal">synthetic lethal</a> experiments in yeast and more clinically relevant</p>
<p><a 
href="#glo:synthetic lethal">synthetic lethal</a> experiments in human cancer models, such as <a 
href="#glo:RNAi">RNAi</a>, especially when high-throughput screens are costly and prone to false positives in either syste,m and are difficult to replicate across gene backgrounds. This database centralises a wealth of knowledge scattered in the literature including cancer relevant genes, including the previously mentioned interactions of <span class="cmti-12">BRCA1 </span>and <span class="cmti-12">BRCA2 </span>with <span class="cmti-12">PARP1</span>, and <span class="cmti-12">TP53 </span>with <span class="cmti-12">WEE1 </span>and <span class="cmti-12">PLK1</span>, although the computational methodology was not released and was limited to 647 human genes. Their future directions were promising, such as constructing networks of known <a 
href="#glo:synthetic lethal">synthetic lethality</a>, applying known <a 
href="#glo:synthetic lethal">synthetic lethality</a> to cancer treatment, data mining, replicating the approach for <a 
href="#glo:synthetic lethal">synthetic lethality</a> in model organisms, signalling pathways, and developing a complete global network in human cancer or yeast (both of which are still incomplete with experimental data), some of which has been implemented in “SynLethDB” (<a 
href="#XGuo2016">Guo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuo2016">2016</a>).</p>
</p><!--l. 922--><p class="indent" >
<a 
 id="x1-340013"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 1.3: </span><span class="content">Machine Learning Methods used by <a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWu2014">2014</a>)</span>
</div><!--tex4ht:label?: x1-340013 -->
<!--tex4ht:inline--><div class="tabular"><table id="TBL-28" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-28-1g"><col 
id="TBL-28-1" /><col 
id="TBL-28-2" /><col 
id="TBL-28-3" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-28-1-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-28-1-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left; vertical-align:middle;">
<span class="cmbx-12">Method</span>
</div>                                                                              </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-28-1-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left; vertical-align:middle;">
<span class="cmbx-12">Source</span>
</div>                     </td><td  style="white-space:nowrap; text-align:left;" id="TBL-28-1-3"  
class="td11"> 
<span class="cmbx-12">Tool Offered</span>
</td></tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-28-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-28-2-1"  
class="td
11"> 
<span id="textcolor27">Random Forest</span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-28-2-2"  
class="td11"> 
<a 
href="#XBreiman2001">Breiman</a> (<a 
href="#XBreiman2001">2001</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-28-2-3"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-28-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-28-3-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-29" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-29-1g"><col 
id="TBL-29-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-29-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-29-1-1"  
class="td00">
Random Forest
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-29-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-29-2-1"  
class="td00">
J48 (decision tree)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-29-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-29-3-1"  
class="td00">
Bayes (Log Regression)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-29-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-29-4-1"  
class="td00">
Bayes (Network)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-29-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-29-5-1"  
class="td00">
PART (Rule-based)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-29-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-29-6-1"  
class="td00">
RBF Network
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-29-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-29-7-1"  
class="td00">
Bagging / Bootstrap
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-29-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-29-8-1"  
class="td00">
Classification via Regression
</td></tr></table>                                                                        </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-28-3-2"  
class="td11"> 
<a 
href="#XHall2009">Hall <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XHall2009">2009</a>)  
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-28-3-3"  
class="td11"> 
WEKA
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-28-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-28-4-1"  
class="td11"> 
<span id="textcolor28">Support Vector Machine (Linear)</span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-28-4-2"  
class="td11"> 
<a 
href="#XVapnik1995">Vapnik</a> (<a 
href="#XVapnik1995">1995</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-28-4-3"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-28-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-28-5-1"  
class="td11"> 
Support Vector Machine (RBF – Gaussian)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-28-5-2"  
class="td11"> 
<a 
href="#XJoachims1999">Joachims</a> (<a 
href="#XJoachims1999">1999</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-28-5-3"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-28-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-28-6-1"  
class="td11"> 
<span id="textcolor29">Multi-Network Multi-Class (MNMC)</span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-28-6-2"  
class="td11"> 
<a 
href="#XPandey2010">Pandey <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XPandey2010">2010</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-28-6-3"  
class="td11"> 
 
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-28-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-28-7-1"  
class="td11"> 
MetaSL (Meta-Analysis)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-28-7-2"  
class="td11"> 
<a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWu2014">2014</a>)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-28-7-3"  
class="td11"> 
MetaSL
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-28-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-28-8-1"  
class="td11">                                                                         </td></tr></table>                                                </div>
                                                                                    

</div><hr class="endfloat" />
<!--l. 980--><p class="indent" >
Machine learning approaches have also been explored for <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery (<a 
href="#XBabyak2004">Babyak</a>, <a 
href="#XBabyak2004">2004</a>; <a 
href="#XLee2009">Lee and Marcotte</a>, <a 
href="#XLee2009">2009</a>). Due to concerns that these may be subject to overfitting or noise, <a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWu2014">2014</a>) developed a meta-analysis method (based on the machine learning methods in Table <a 
href="#x1-340013">1.3<!--tex4ht:ref: tab:methods_meta --></a>). They focused on <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs relevant to developing selective drugs against human cancer, building upon their previous database (<a 
href="#XLi2014">Li <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLi2014">2014</a>). Their “metaSL” approach utilises <a 
href="#glo:genomic">genomic</a>, proteomic and annotation data and had a high statistical performance in yeast data with an <a 
href="#glo:AUROC">area under <a 
href="#glo:ROC">receiver operating characteristic</a> (AUROC)</a> of 0.871 (as described in Section <a 
href="#x1-620001">2.3.5.1<!--tex4ht:ref: methods:ROC --></a>). They predicted orthologous <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in human data were not experimentally validated but some were relevant to cancer such as <span class="cmti-12">EGFR </span>with <span class="cmti-12">PRKCZ</span>.
</p><!--l. 984--><p class="indent" >
Computational approaches scale-up across the <a 
href="#glo:genome">genomes</a> at lower cost than experimental screen (<a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWu2014">2014</a>). <a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWu2014">2014</a>) provided their most supported interactions online but the method is not available for analysis of other genes. Syn-Lethality (<a 
href="#XLi2014">Li <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLi2014">2014</a>) and MetaSL (<a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWu2014">2014</a>) demonstrate the value of computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethality</a> but omit many genes of importance in cancer, such as <span class="cmti-12">CDH1</span>. Accordingly, there remains a need to enable biological researchers to query further genes and do so in a particular tissue or genetic background.
</p><!--l. 987--><p class="indent" >
<p>There is also concern for analyses based on yeast data that many <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions may not be conserved between species (<a 
href="#XDixon2009a">Dixon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDixon2009a">2009</a>), although interactions between <a 
href="#glo:pathway">pathways</a> may be more comparable. It is unsurprising that many of the interactions identified were not experimentally validated. There have been many gene duplications in the separate evolutionary histories of humans and yeast which may lead to differences in <a 
href="#glo:functional redundancy">genetic redundancy</a>. Yeast cells are not an ideal human cancer model because they do not have tissue specificity, multicellular gene regulation, or orthologues to several known <a 
href="#glo:cancer gene">cancer genes</a> such as p53 (<a 
href="#XGuaragnella2014">Guaragnella <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuaragnella2014">2014</a>). Although these studies have tried to anticipate these issues with stringent criteria such as requiring one-to-one orthologues (<a 
href="#XDeshpande2013">Deshpande <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDeshpande2013">2013</a>; <a 
href="#XHeiskanen2012">Heiskanen and Aittokallio</a>, <a 
href="#XHeiskanen2012">2012</a>; <a 
href="#XKranthi2013">Kranthi <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XKranthi2013">2013</a>), there remains the possibility that changes in gene function may affect whether these are solely redundant such as if functions had co-evolved without sequence homology. Many genes will also be excluded since they lack homologues in yeast, the corresponding experimental data, or having paralogues in either species. Thus conservation of yeast interactions is not an ideal strategy and analysis of human data</p>
directly for comparison with human experimental data will be the focus of this thesis. <a 
 id="x1-34002r40"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.7.3 </span> <a 
 id="x1-350003"></a>Analysis and Modelling of Protein Data
</h5>
<!--l. 991--><p class="noindent" >
<a 
href="#XKranthi2013">Kranthi <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XKranthi2013">2013</a>) took a network approach to discovery of <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate selection applying the concept to “centrality” to a human <a 
href="#glo:PPI">PPI</a> network involving interacting partners of known <a 
href="#glo:cancer gene">cancer genes</a>. The effect of removing pairs of genes on connectivity of the network was used as a surrogate for viability which is supported by observations that the <a 
href="#glo:PPI">PPI</a> and <a 
href="#glo:synthetic lethal">synthetic lethal</a> networks are orthogonal in <span class="cmti-12">S. cerevisiae </span>studies (<a 
href="#XTong2004">Tong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTong2004">2004</a>). They showed that the human cancer protein interaction network derived protein interactions and cancer gene databases (<a 
href="#XFutreal2004">Futreal <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFutreal2004">2004</a>; <a 
href="#XHiggins2007">Higgins <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XHiggins2007">2007</a>; <a 
href="#XKeshavaPrasad2009">Keshava Prasad <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XKeshavaPrasad2009">2009</a>), consisting of 1539 proteins and 6471 interactions, exhibits the power law distribution expected of a <a 
href="#glo:scale-free">scale-free</a> <a 
href="#glo:synthetic lethal">synthetic lethal</a> network with high connectivity (average <a 
href="#glo:vertex">vertex</a> degree of 23.67 and network efficiency of 0.2952). Their top 100 candidate interactions included interactions of the <a 
href="#glo:tumour suppressor">tumour suppressor</a> <span class="cmti-12">TP53 </span>with <span class="cmti-12">BRCA1</span>, <span class="cmti-12">CDKNA1</span>, <span class="cmti-12">CDKNA2</span>, <span class="cmti-12">MET</span>, and <span class="cmti-12">RB1 </span>which have been detected by prior studies. The gene pairs were often observed to be in the same or a plausible compensatory pathway. This demonstrated that <a 
href="#glo:graph">network</a> structure is important in the biological functions of cancers and could be exploited for targeting <span class="cmti-12">TP53 </span>loss of function <a 
href="#glo:mutation">mutations</a>.
</p><!--l. 993--><p class="indent" >
However, the approach of <a 
href="#XKranthi2013">Kranthi <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XKranthi2013">2013</a>) was limited to known <a 
href="#glo:cancer gene">cancer genes</a> and is not applicable to genes that do not have <a 
href="#glo:PPI">PPI</a> data. Other nucleotide sequencing data types are more commonly available for cancer studies at a <a 
href="#glo:genomic">genomic</a> scale. Of further concern is that the results were enriched for p53 <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners, which is relevant to many <a 
href="#glo:cancer">cancers</a> but this <a 
href="#glo:genome">genome</a>-wide approach did not detect many other <a 
href="#glo:cancer gene">cancer genes</a> due to the extent of multiple testing. This enrichment may be due to the known drastic effect of removing p53 itself from the network as a highly connected, master regulator, and cancer driving <a 
href="#glo:tumour suppressor">tumour suppressor</a> gene. The focus on <a 
href="#glo:cancer gene">cancer genes</a> is useful for translation into therapeutics but does not account for variable genetic backgrounds or effect of protein removal on the cellular network.
</p><!--l. 995--><p class="indent" >
<p>Focusing on the potential for <a 
href="#glo:synthetic lethal">synthetic lethality</a> to be an effective anti-cancer drug target, <a 
href="#XZhang2015">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XZhang2015">2015</a>) used modelling of signalling pathways to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions between known drug targets and <a 
href="#glo:cancer gene">cancer genes</a> by simulating gene knockdowns.</p>
A computational approach was applied to avoid the limitations of experimental <a 
href="#glo:RNAi">RNAi</a> screens such as scale, instability of knockdown, and off-target effects. This ‘hybrid’ method of a data-driven model and known signalling pathways showed potential to predict cell death in single and combination gene knockouts. They used time series protein phosphorylation data (<a 
href="#XLee2012">Lee <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLee2012">2012</a>) for 28 signalling proteins and <a 
href="#glo:GO">Gene Ontology (GO)</a> <a 
href="#glo:pathway">pathways</a> (<a 
href="#XAshburner2000">Ashburner <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XAshburner2000">2000</a>; <a 
href="#XBlake2015">Blake <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBlake2015">2015</a>). This approach successfully detected many known <a 
href="#glo:essential">essential</a> genes in the human gene essentiality database, known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in the Syn-Lethality database (<a 
href="#XLi2014">Li <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLi2014">2014</a>), and predicted novel <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs.
</p><!--l. 997--><p class="indent" >
These novel results contained many <span class="cmti-12">TP53 </span>and AKT <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners (<a 
href="#XZhang2015">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XZhang2015">2015</a>), genes known to be important in many cancers. However, these genes also have a severe impact on the signalling <a 
href="#glo:pathway">pathways</a> in an essentiality analysis of single gene disruptions and large phenotypic changes in cancer (<a 
href="#XZhang2015">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XZhang2015">2015</a>). This approach is amenable to detect functionally related <a 
href="#glo:pathway">pathways</a> and protein complexes across the molecular function, cellular component, and biological process annotations provided by <a 
href="#glo:GO">Gene Ontology</a>. The results were consistent with the experimental results in the literature but the novel <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions have yet to be validated. While the mathematical reasoning and algorithms are given, the code was not released to replicate the findings or apply the methodology beyond the signalling pathways analysed by (<a 
href="#XZhang2015">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XZhang2015">2015</a>). While this is an interesting approach, the analysis of this thesis will focus on <a 
href="#glo:gene expression">gene expression</a> and <a 
href="#glo:RNAi">RNAi</a> data, the widespread availability which allows testing ofa broader range of candidate gene pairs. <a 
 id="x1-35001r42"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.7.4 </span> <a 
 id="x1-360004"></a>Differential Gene Expression
</h5>
<!--l. 1003--><p class="noindent" >
<p>Differential <a 
href="#glo:gene expression">gene expression</a> has been explored to predict <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs in cancer which would be widely applicable due to the availability of public <a 
href="#glo:gene expression">gene expression</a> data for many samples and cancer types. <a 
href="#XWang2013">Wang and Simon</a> (<a 
href="#XWang2013">2013</a>) found differentially expressed genes (by the t-test, adjusted by <a 
href="#glo:FDR">False discovery rate (FDR)</a>) between <a 
href="#glo:tumour">tumours</a> with or without functional p53 <a 
href="#glo:mutation">mutations</a> in <a 
href="#glo:TCGA">TCGA</a> (<a 
href="#XTCGA2008GBM">McLendon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2008GBM">2008</a>) and <a 
href="#glo:CCLE">Cancer Cell Line Encyclopaedia (CCLE)</a> (<a 
href="#XBarretina2012">Barretina <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBarretina2012">2012</a>) <a 
href="#glo:RNA-Seq">RNA-Seq</a> <a 
href="#glo:gene expression">gene expression</a> data as candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner pathways of p53. They identified 2, 8, and 21 candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes in 3 <a 
href="#glo:microarray">microarray</a> datasets from the NCI60 cell lines, 31 partner genes from the <a 
href="#glo:CCLE">CCLE</a></p>
<a 
href="#glo:RNA-Seq">RNA-Seq</a> data (<a 
href="#XBarretina2012">Barretina <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBarretina2012">2012</a>), and 50 in <a 
href="#glo:TCGA">TCGA</a> <a 
href="#glo:RNA-Seq">RNA-Seq</a> data (<a 
href="#XTCGA2012CRC">Muzny <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012CRC">2012</a>). <span class="cmti-12">PLK1 </span>was replicated across 4 of these analyses and 17 other genes were replicated across 2 analyses (including <span class="cmti-12">MTOR</span>, <span class="cmti-12">PLK4</span>, <span class="cmti-12">MAST2</span>, <span class="cmti-12">MAP3K4</span>, <span class="cmti-12">AURKA</span>, <span class="cmti-12">BUB1 </span>and 6 CDK genes) with many playing a role in cell cycle regulation. This was supported by a drug sensitivity experiment on the NCI60 cell lines which found that cells lacking functional p53 were more sensitive to paclitaxel (which targets <span class="cmti-12">PLK1</span>, <span class="cmti-12">AURKA</span>, and <span class="cmti-12">BUB1</span>). This demonstrated the potential of <a 
href="#glo:gene expression">gene expression</a> as a surrogate for gene function, and the use of public <a 
href="#glo:genomic">genomic</a> data to predict <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs in cancer. <a 
href="#XWang2013">Wang and Simon</a> (<a 
href="#XWang2013">2013</a>) advocated for pre-screening of <a 
href="#glo:gene expression">expression</a> profiles to augment future <a 
href="#glo:RNAi">RNAi</a> screens, however, their analyses were limited to kinase genes and focused on currently druggable targets, lacking wider application of <a 
href="#glo:synthetic lethal">synthetic lethal</a> prediction methodology. This approach may not be feasible or applicable in <a 
href="#glo:cancer gene">cancer genes</a> with a lower <a 
href="#glo:mutation">mutation</a> rate than p53.
</p><!--l. 1005--><p class="indent" >
<a 
href="#XTiong2014">Tiong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTiong2014">2014</a>) also investigated <a 
href="#glo:gene expression">gene expression</a> as a predictor of <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs with colorectal cancer <a 
href="#glo:microarray">microarrays</a> from a Han Chinese population with a sample size of 70 tumours and 12 normal tissue samples. Simultaneously differential <a 
href="#glo:gene expression">expressed</a> “tumour dependent” gene pairs (which includes co-expression) between cancer and normal tissue were used to rank 663 candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions identified in cell line <a 
href="#glo:siRNA">siRNA</a> experiments. Of the top 20 gene pairs, 17 were tested for differential <a 
href="#glo:gene expression">expression</a> at the protein level with immunohistochemistry staining and correlation with clinical characteristics, with 11 pairs exhibiting synergistic effects. Some of the predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs were consistent with the literature (including <span class="cmti-12">TP53</span> with <span class="cmti-12">S6K1 </span>and partners of <span class="cmti-12">KRAS</span>, <span class="cmti-12">PTEN</span>, <span class="cmti-12">BRCA1</span>, and <span class="cmti-12">BRCA2</span>) and two novel <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions (<span class="cmti-12">TP53 </span>with <span class="cmti-12">CSNK1E </span>and <span class="cmti-12">CTNNB1</span>) were validated in pre-clinical models. This serves as a valuable proof-of-concept for integration of <span class="cmti-12">in silico</span> approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery in cancer, demonstrating its utility to triage and identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of p53 applicable to colorectal tissues. Although the experimental work was the focus of the paper, these findings show that <a 
href="#glo:bioinformatics">bioinformatics</a> <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates can be validated in patient tissue samples to find those applicable to colorectal cancers (including in a non-Caucasian population). <a 
 id="x1-36001r43"></a>
</p>
                                                                                    

<h5 class="subsubsectionHead">
<span class="titlemark">1.2.7.5 </span> <a 
 id="x1-370005"></a>Data Mining and Machine Learning
</h5>
<!--l. 1011--><p class="noindent" >
Recognising the utility of <a 
href="#glo:synthetic lethal">synthetic lethality</a> to drug inhibition and specificity of anti-cancer <a 
href="#glo:treatment">treatments</a>, <a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XJerby2014">2014</a>) also saw the need for effective prediction of gene essentiality and <a 
href="#glo:synthetic lethal">synthetic lethality</a> to augment experimental studies of SL. They developed the <a 
href="#glo:DAISY">“DAta mIning SYnthetic lethal identification pipeline” (DAISY)</a>, a data-driven approach for <a 
href="#glo:genome">genome</a>-wide analysis of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in public cancer <a 
href="#glo:genomic">genomics</a> data from <a 
href="#glo:TCGA">TCGA</a> and <a 
href="#glo:CCLE">CCLE</a> (<a 
href="#XBarretina2012">Barretina <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBarretina2012">2012</a>). <a 
href="#glo:DAISY">DAISY</a> is intended to predict the candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of a query gene such as genes recurrently mutated in cancer.
</p><!--l. 1013--><p class="indent" >
<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XJerby2014">2014</a>) combined a computational approach to triage candidates with a conventional <a 
href="#glo:RNAi">RNAi</a> screen to validate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners. They screened a selection of computationally predicted candidates and randomly selected genes with <a 
href="#glo:RNAi">RNAi</a> against <span class="cmti-12">VHL </span>loss of function <a 
href="#glo:mutation">mutation</a> in RCC4 renal cell lines. The computational method had a high <a 
href="#glo:AUROC">AUROC</a> of 0.779 and predictions were enriched 4-fold for validated <a 
href="#glo:RNAi">RNAi</a> hits over randomly selected genes. This approach detected known <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs such as <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>genes with <span class="cmti-12">PARP1</span>, and <span class="cmti-12">MSH2 </span>with <span class="cmti-12">DHFR</span>. The <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates identified with both <a 
href="#glo:RNAi">RNAi</a> screening and computational prediction formed an extensive network of 2077 genes with 2816 <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions, and a similar network of 3158 genes with 3635 <a 
href="#glo:synthetic dosage lethal">synthetic dosage lethal</a> interactions (for <a 
href="#glo:synthetic lethal">synthetic lethality</a> with over-expression). Each network was <a 
href="#glo:scale-free">scale-free</a>, as expected of a biological network, and was enriched for known <a 
href="#glo:cancer gene">cancer genes</a> and for <a 
href="#glo:essential">essential</a> genes in mice which could be harnessed for predicting prognosis and drug response.
</p><!--l. 1016--><p class="indent" >
<p>The <a 
href="#glo:DAISY">DAISY</a> methodology (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>) compares the results of analysis of several data types to predict <a 
href="#glo:synthetic lethal">synthetic lethality</a>, namely: <a 
href="#glo:DNA">DNA</a> copy number and <a 
href="#glo:somatic">somatic</a> <a 
href="#glo:mutation">mutation</a> for <a 
href="#glo:TCGA">TCGA</a> patient samples and <a 
href="#glo:CCLE">CCLE</a> cell lines. The cell lines were also analysed with <a 
href="#glo:gene expression">gene expression</a> and gene essentiality (<a 
href="#glo:shRNA">shRNA</a> screening) profiles. Genes were classed as inactivated by copy number deletion, <a 
href="#glo:somatic">somatic</a> loss of function <a 
href="#glo:mutation">mutation</a>, or low <a 
href="#glo:gene expression">expression</a> and tested for <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene partners which are either <a 
href="#glo:essential">essential</a> in screens or not deleted with copy number variants. Co-expression is also used for <a 
href="#glo:synthetic lethal">synthetic lethality</a> prediction based on studies in yeast (<a 
href="#XCostanzo2010">Costanzo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCostanzo2010">2010</a>; <a 
href="#XKelley2005">Kelley and Ideker</a>, <a 
href="#XKelley2005">2005</a>). Copy number, <a 
href="#glo:gene expression">gene expression</a>, and essentiality analyses were stringently compared by adjusting each for multiple tests with Bonferroni correction and only taking candidates identified in all analyses. The predictions performed well and an <a 
href="#glo:RNAi">RNAi</a> screen, for the example of <span class="cmti-12">VHL </span>in renal cancer, validated predicted</p>
<a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">VHL </span>demonstrating the feasibility of combining approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery in cancer and using computational predictions to enable more efficient high-throughput screening. While <a 
href="#glo:DAISY">DAISY</a> performed well statistically, co-expression and <a 
href="#glo:shRNA">shRNA</a> functional examination contributed less to this than the <a 
href="#glo:mutation">mutation</a> and copy number analysis (<a 
href="#glo:AUROC">AUROC</a> 0.683 alone). However, this methodology was very stringent, missing potentially valuable <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates. Additionally, the software for the procedure has not been publicly released for replication.
</p><!--l. 1021--><p class="indent" >
Although the <a 
href="#glo:DAISY">DAISY</a> procedure performed well and has been well received by the scientific community (<a 
href="#XCrunkhorn2014">Crunkhorn</a>, <a 
href="#XCrunkhorn2014">2014</a>; <a 
href="#XLokody2014">Lokody</a>, <a 
href="#XLokody2014">2014</a>; <a 
href="#XRyan2014">Ryan <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XRyan2014">2014</a>), showing a need for such methodology, there has not yet been widespread adoption of this approach. Co-expression analysis may exclude some <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions, where inverse correlation could occur (<a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>). In the interests of a large sample size, tissue types were not tested separately despite tissue-specific <a 
href="#glo:synthetic lethal">synthetic lethality</a> being likely since gene function (and by extension <a 
href="#glo:gene expression">expression</a>, isoforms, and clinical characteristics) in cancers may often be tissue-dependent. Some data forms and analyses used, such as gene essentiality, may not be available for all cancers, genes, or tissues, and may not be reproduced.
</p><!--l. 1023--><p class="indent" >
<a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2015">2015</a>) propose an alternative computational prediction of <a 
href="#glo:synthetic lethal">synthetic lethality</a> based on machine learning methods and a “cancer genome evolution” hypothesis. Using <a 
href="#glo:DNA">DNA</a> copy number and <a 
href="#glo:gene expression">gene expression</a> data from <a 
href="#glo:TCGA">TCGA</a> patient samples, a cancer <a 
href="#glo:genome">genomes</a> evolution model assumes that <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs behave in two distinct ways in response to an inactive <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner gene, either a “compensation” pattern where the other <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner is overactive or a “co-loss underrepresentation” pattern where the other <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner is less likely to be lost, since loss of both genes would cause death of the cancer cell. During the <a 
href="#glo:genome">genomes</a> evolution of cancers, the cell becomes addicted to the remaining <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner due to induced gene essentiality. These patterns would explain why <a 
href="#glo:DAISY">DAISY</a> detects only a small number of <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs, compared to the large number expected based on model organism studies (<a 
href="#XBoone2007">Boone <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoone2007">2007</a>), and the disparity between screening and computationally predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates due to testing different classes of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs.
</p><!--l. 1025--><p class="indent" >
<p><a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2015">2015</a>) compared a <a 
href="#glo:genome">genome</a>-wide computational model of <a 
href="#glo:genome">genomes</a> evolution and <a 
href="#glo:gene expression">gene expression</a> patterns to the experimental data (<a 
href="#XLaufer2013">Laufer <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLaufer2013">2013</a>; <a 
href="#XVizeacoumar2013">Vizeacoumar</p>
<em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XVizeacoumar2013">2013</a>). This more simple model performed well, with an <a 
href="#glo:AUROC">AUROC</a> of 0.751 (lower than <a 
href="#glo:DAISY">DAISY</a>), and did not rely on data from cell lines which may not represent patient disease. <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2015">2015</a>) predicted 591,000 human <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners with a probability score threshold of 0.81, giving a precision of 67% and 14-fold enrichment of <a 
href="#glo:synthetic lethal">synthetic lethal</a> true positives compared to randomly selected gene pairs. Discovery of such a vast number of cancer-relevant <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions in humans would not be feasible experimentally and is a valuable resource for research and clinical applications. These predictions are not limited by assuming co-expression of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners or evolutionary conservation with model organisms enabling wider <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery. However, there remains a lack of basis for an expectation of how many <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners a particular gene will have, how many pairs there are in the human <a 
href="#glo:genome">genomes</a>, and whether <a 
href="#glo:pathway">pathways</a> or correlation structure would influence predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners.
</p><!--l. 1027--><p class="indent" >
Large scale, computational approaches have yet to determine whether <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions are tissue-specific, since <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2015">2015</a>) used <a 
href="#glo:pan cancer">pan cancer</a> data for 14136 patients with 31 cancer types. Experimental data used for comparison was a small training dataset specific to colorectal cancer, and based on screens for other phenotypes, which may limit performance of the model or application to other cancers. Proposed expansion of the computational approach to <a 
href="#glo:mutation">mutation</a>, <a 
href="#glo:microRNA">microRNA</a>, or epigenetic modulation of gene function and tumour micro-environment or heterogeneity suggests that <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery could be widely applied to the current challenges in cancer <a 
href="#glo:genomic">genomics</a>. This approach was also based on machine learning methodology and was not supported by a software release for the community to develop, contribute to, or reproduce beyond the gene pairs given in the supplementary results. <a 
 id="x1-37001r44"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.7.6 </span> <a 
 id="x1-380006"></a>Mutual Exclusivity and Bimodality
</h5>
<!--l. 1031--><p class="noindent" >
<p><a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWappett2016">2016</a>) demonstrated a multi-omic approach to identify <a 
href="#glo:synthetic lethal">synthetic lethality</a> in cancer with a strategy to detect bimodal patterns in <a 
href="#glo:molecular profile">molecular profiles</a>. They released this solution as the <a 
href="#glo:BiSEp">BiSEp</a> R package (<a 
href="#XWappett2014">Wappett</a>, <a 
href="#XWappett2014">2014</a>) which aims to detect subtle bimodal and non-normal patterns in <a 
href="#glo:gene expression">expression</a> data. Since loss of gene function is not consistently genetic. <a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWappett2016">2016</a>) advocate the use of <a 
href="#glo:gene expression">gene expression</a> (loss of <a 
href="#glo:mRNA">mRNA</a>) and deletion (loss of copy number) data in addition to <a 
href="#glo:mutation">mutation</a>. The <a 
href="#glo:BiSEp">BiSEp</a> procedure was demonstrated on an analysis of 881 cell</p>
lines from <a 
href="#glo:CCLE">CCLE</a> (<a 
href="#XBarretina2012">Barretina <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBarretina2012">2012</a>), 442 cell lines from <a 
href="#glo:COSMIC">COSMIC</a> (<a 
href="#XForbes2015">Forbes <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XForbes2015">2015</a>), and <a 
href="#glo:RSEM">RNA-Seq by Expectation Maximization (RSEM)</a> normalised <a 
href="#glo:RNA-Seq">RNA-Seq</a> data for 178 <a 
href="#glo:TCGA">TCGA</a> lung patient samples (<a 
href="#XTCGA2014LU">Collisson <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014LU">2014</a>). <a 
href="#glo:BiSEp">BiSEp</a> was demonstrated to have significant enrichment of validated <a 
href="#glo:tumour suppressor">tumour suppressor</a>, <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs (detecting 76 experimentally supported gene pairs) and was improved (detecting 420) with <a 
href="#glo:gene expression">expression</a> data, rather than relying on detecting loss of gene function by <a 
href="#glo:mutation">mutation</a> or deletion. <a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWappett2016">2016</a>) identified interactions with genes relevant to cancer with support in experimental screens including <span class="cmti-12">ERCC4 </span>with <span class="cmti-12">XRCC1</span>, <span class="cmti-12">BRCA1 </span>with <span class="cmti-12">PARP3</span>, and <span class="cmti-12">SMARCA1 </span>with <span class="cmti-12">SMARCA4</span>.
</p><!--l. 1033--><p class="indent" >
<a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWappett2016">2016</a>) demonstrated that analysis of <a 
href="#glo:genomic">genomics</a> data, particularly <a 
href="#glo:gene expression">expression</a> data, is relevant to augment the identification of <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions with screening experiments. They further showed that this is applicable in both genetically homogeneous cell lines and heterogeneous cell population from patient samples. These approach are limited however, to genes that exhibit bimodal <a 
href="#glo:gene expression">expression</a> patterns which do not commonly occur, particularly in normalised <a 
href="#glo:gene expression">gene expression</a> data. Other approaches may need to be considered for gene such as <span class="cmti-12">CDH1 </span>which were not identified by <a 
href="#glo:BiSEp">BiSEp</a>.
</p><!--l. 1036--><p class="indent" >
<p><a 
href="#XSrihari2015">Srihari <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XSrihari2015">2015</a>) used a computational analysis to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate genes from mutually exclusive alterations in cancers. This analysis focused on <a 
href="#glo:synthetic lethal">synthetic lethality</a> with “<a 
href="#glo:DNA">DNA</a> damage response genes” in cancers, including <span class="cmti-12">CDH1</span>, using <a 
href="#glo:TCGA">TCGA</a> <a 
href="#glo:gene expression">expression</a> and copy number data across several cancers. The 718 genes that were identified as frequently altered in cancer were enriched among essential genes in cell lines deficient in <a 
href="#glo:DNA">DNA</a> damage response genes which demonstrates “induced essentiality” in a cancer model. These were tested by <span class="cmtt-12">underMutExSL</span>, a hypergeometric test, for <a 
href="#glo:synthetic lethal">synthetic lethality</a>. Of the <a 
href="#glo:DNA">DNA</a> damage response genes examined, <span class="cmti-12">CDH1 </span>exhibited the most mutually exclusive alterations with other frequently altered genes in breast cancers, including related genes in focal adhesion such as <span class="cmti-12">PTK2</span>. These results indicate that <span class="cmti-12">CDH1 </span>may be particularly subject to “<a 
href="#glo:non-oncogene addiction">non-oncogene addiction</a>” in breast cancers and that computational analysis is valuable to rationally identify these putative <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes. These results were limited to frequently altered genes in cancer and could be improved with an approach that expands to consider <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes that are not themselves altered in cancers or further investigation into <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways. <a 
 id="x1-38001r45"></a></p>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.7.7 </span> <a 
 id="x1-390007"></a>Rationale for Further Development
</h5>
<!--l. 1040--><p class="noindent" >
Many of the approaches discussed here aimed to identify the strongest <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs across the yeast or human <a 
href="#glo:genome">genomes</a> (<a 
href="#XDeshpande2013">Deshpande <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDeshpande2013">2013</a>; <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>; <a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWappett2016">2016</a>; <a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWu2014">2014</a>), which may not be an ideal strategy to identify interactions in particular functions or relevance to particular cancers. These demonstrate a need for computational approaches to prioritise candidate gene pairs for validation but this thesis will focus on the interactions with <span class="cmti-12">CDH1 </span>with importance in breast and stomach cancers, although these partners may be applicable in other cancers. As such, this thesis presents a query-based method, amenable to identification of candidate partners for a selected gene of functional or translational importance such as <span class="cmti-12">CDH1</span>. <a 
 id="x1-39001r24"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">1.3 </span> <a 
 id="x1-400003"></a>E-cadherin as a Synthetic Lethal Target
</h3>
<!--l. 1048--><p class="noindent" >
<a 
href="#glo:E-cadherin">E-cadherin</a> is a transmembrane protein (encoded by <span class="cmti-12">CDH1</span>) with several well-characterised functions in the cytoskeleton and cell-to-cell signalling. Here we outline the characterised functions of <a 
href="#glo:E-cadherin">E-cadherin</a> and its importance in cancer biology. <span class="cmti-12">CDH1 </span>is a <a 
href="#glo:tumour suppressor">tumour suppressor</a> gene, with loss of function occurring in both <a 
href="#glo:familial">familial</a> (<a 
href="#glo:germline">germline mutation</a> <a 
href="#glo:mutation">mutations</a>) and <a 
href="#glo:sporadic">sporadic</a> (<a 
href="#glo:somatic">somatic</a> <a 
href="#glo:mutation">mutations</a>) cancers. As such, <span class="cmti-12">CDH1 </span>inactivation is a prime example of a genetic event that could be targeted by <a 
href="#glo:synthetic lethal">synthetic lethality</a>. Drugs against <span class="cmti-12">CDH1</span>-deficient cancers would be applicable for <a 
href="#glo:chemoprevention">chemoprevention</a> of <a 
href="#glo:HDGC">HDGC</a> and <a 
href="#glo:treatment">treatment</a> of sporadic cancers carrying somatic <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutations</a>. <a 
 id="x1-40001r36"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.3.1 </span> <a 
 id="x1-410001"></a>The <span class="cmti-12">CDH1 </span>gene and its Biological Functions
</h4>
<!--l. 1053--><p class="noindent" >
<span class="cmti-12">CDH1 </span>is implicated in <a 
href="#glo:hereditary">hereditary</a> and <a 
href="#glo:sporadic">sporadic</a> lobular breast cancers and diffuse gastric cancers (<a 
href="#XBerx1996">Berx <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBerx1996">1996</a>; <a 
href="#XBerx2009">Berx and van Roy</a>, <a 
href="#XBerx2009">2009</a>; <a 
href="#XDeLeeuw1997">De Leeuw <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDeLeeuw1997">1997</a>; <a 
href="#XGuilford1998">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford1998">1998</a>; <a 
href="#XMasciari2007">Masciari <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMasciari2007">2007</a>; <a 
href="#XSemb1998">Semb and Christofori</a>, <a 
href="#XSemb1998">1998</a>; <a 
href="#XVos1997">Vos <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XVos1997">1997</a>). <a 
href="#glo:E-cadherin">E-cadherin</a> is normally expressed in epithelial tissues and loss of <span class="cmti-12">CDH1 </span>function has been implicated in breast cancer progression and metastasis (<a 
href="#XBecker1994">Becker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBecker1994">1994</a>; <a 
href="#XBerx1995">Berx <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBerx1995">1995</a>; <a 
href="#XChristofori1999">Christofori and Semb</a>, <a 
href="#XChristofori1999">1999</a>).
</p><!--l. 1058--><p class="indent" >
<p>The primary function of <span class="cmti-12">CDH1 </span>is cell-cell adhesion at the adherens junction, maintaining the cytoskeleton and mediating molecular signals between cells. The function of the adherens</p>
complex is particularly important for cell structure and regulation because it interacts with actin and microtubule cytoskeletons. The cytoskeletal role of <a 
href="#glo:E-cadherin">E-cadherin</a> includes maintaining cellular polarity (<a 
href="#XJeanes2008">Jeanes <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJeanes2008">2008</a>). <a 
href="#glo:E-cadherin">E-cadherin</a> is not <a 
href="#glo:essential">essential</a> to cellular viability but loss in epithelial cells does lead to defects in cytoskeletal structure and proliferation. In addition to a central role in the adherens complex, <a 
href="#glo:E-cadherin">E-cadherin</a> is involved in many other cellular functions and thus <span class="cmti-12">CDH1 </span>is regarded as a highly <a 
href="#glo:pleiotropy">pleiotropic</a> gene (<a 
href="#XKroepil2012">Kroepil <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XKroepil2012">2012</a>).
</p><!--l. 1062--><p class="indent" >
As a transmembrane signalling protein <a 
href="#glo:E-cadherin">E-cadherin</a> also interacts with the extracellular environment and other cells, most notably forming adherens junctions between cells (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>; <a 
href="#XTunggal2005">Tunggal <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTunggal2005">2005</a>) and regulating of epithelial tissues by intercellular communication (<a 
href="#XJeanes2008">Jeanes <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJeanes2008">2008</a>). The signals mediated by adherens junctions are also passed on to intracellular signalling pathways . One such example is the regulation of <span class="cmmi-12">β</span>-catenin which interacts with both the actin cytoskeleton and acts as a transcription factor via the <a 
href="#glo:WNT">Wingless-related integration site (WNT)</a> pathway (<a 
href="#XJeanes2008">Jeanes <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJeanes2008">2008</a>). Similarly, the Hippo and <a 
href="#glo:PI3K">phosphoinositide 3-kinase (PI3K)</a>/AKT pathways are also mediated in part by <a 
href="#glo:E-cadherin">E-cadherin</a> (<a 
href="#XDeSantis2009">De Santis <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDeSantis2009">2009</a>; <a 
href="#XKim2011">Kim <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XKim2011">2011</a>). <a 
href="#glo:E-cadherin">E-cadherin</a> shares several downstream pathways with cell-surfacce proteins, such as integrins, and thus indirectly interacts with them.
</p><!--l. 1072--><p class="indent" >
The key roles of <a 
href="#glo:E-cadherin">E-cadherin</a> in maintaining cellular structure and regulating growth are consistent with <span class="cmti-12">CDH1 </span>being a <a 
href="#glo:tumour suppressor">tumour suppressor</a> gene. <a 
href="#glo:E-cadherin">E-cadherin</a> loss in breast cancers has been shown to cause increased proliferation, lymph <a 
href="#glo:vertex">node</a> invasion, and metastasis with poor cell-cell contact (<a 
href="#XBerx2009">Berx and van Roy</a>, <a 
href="#XBerx2009">2009</a>). Thus the <span class="cmti-12">CDH1 </span>gene is an established invasion suppressor, with a key role in the <a 
href="#glo:EMT">epithelial-mesenchymal transition (EMT)</a>, an established mechanism of cancer progression (<a 
href="#XHanahan2011">Hanahan and Weinberg</a>, <a 
href="#XHanahan2011">2011</a>). <a 
href="#glo:EMT">EMT</a> is important during development and wound healing but such changes in cellular differentiation also occur in cancers. <a 
 id="x1-41001r48"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.3.2 </span> <a 
 id="x1-420002"></a>Hereditary Diffuse Gastric (and Lobular Breast) Cancer
</h4>
<!--l. 1081--><p class="noindent" >
<p><span class="cmti-12">CDH1 </span>loss of function <a 
href="#glo:mutation">mutations</a> also causes <a 
href="#glo:familial">familial</a> cancers, including diffuse gastric cancer and lobular breast cancer (<a 
href="#XGraziano2003">Graziano <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGraziano2003">2003</a>; <a 
href="#XGuilford2010">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford2010">2010</a>, <a 
href="#XHDGC">1999</a>; <a 
href="#XOliveira2009">Oliveira <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XOliveira2009">2009</a>). Individuals carrying a null <a 
href="#glo:mutation">mutation</a> in <span class="cmti-12">CDH1 </span>have a syndromic predisposition to early-onset of these cancers, including <a 
href="#glo:HDGC">hereditary diffuse gastric cancer (HDGC)</a> (<a 
href="#XGuilford1998">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford1998">1998</a>). Due to carrying a dysfunctional <a 
href="#glo:allele">allele</a>, these</p>
individuals are prone to carcinogenic lesions in the breast or stomach if the remaining functional <a 
href="#glo:allele">allele</a> is inactivated, occurring more frequently and at an earlier age than in individuals with two functional <span class="cmti-12">CDH1 </span><a 
href="#glo:allele">alleles</a>. The loss of the second <a 
href="#glo:allele">allele</a> is most often through hypermethylation suppressing <a 
href="#glo:gene expression">expression</a> rather than <a 
href="#glo:mutation">mutation</a> (<a 
href="#XGrady2000">Grady <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGrady2000">2000</a>; <a 
href="#XGraziano2003">Graziano <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGraziano2003">2003</a>; <a 
href="#XMachado2001">Machado <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMachado2001">2001</a>; <a 
href="#XOliveira2009">Oliveira <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XOliveira2009">2009</a>), although loss of heterozygousity may also occur (<a 
href="#XGuilford2010">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford2010">2010</a>).
</p><!--l. 1083--><p class="indent" >
<a 
href="#glo:HDGC">HDGC</a> is an autosomal dominant cancer syndrome with incomplete penetrance. The “lifetime” (until age 80 years) risk for <a 
href="#glo:mutation">mutation</a> carriers of diffuse gastric cancer is 70% in males and 56% in females (<a 
href="#XHansford2015">Hansford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XHansford2015">2015</a>; <a 
href="#XvanderPost2015">van der Post <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XvanderPost2015">2015</a>). In addition, the lifetime risk of lobular breast cancer is 42% in female <a 
href="#glo:mutation">mutation</a> carriers (<a 
href="#XHansford2015">Hansford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XHansford2015">2015</a>). <a 
href="#glo:HDGC">HDGC</a> represents less than 1% of gastric cancers (<a 
href="#XFerlay2015">Ferlay <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFerlay2015">2015</a>), however, it is a serious health issue for several hundred families globally. <a 
href="#glo:E-cadherin">E-cadherin</a> is also mutated in 13% of <a 
href="#glo:sporadic">sporadic</a> gastric cancers (<a 
href="#XFerlay2015">Ferlay <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFerlay2015">2015</a>).
</p><!--l. 1090--><p class="indent" >
While diagnostic testing for <span class="cmti-12">CDH1 </span>genotype has enabled more effective management of <a 
href="#glo:HDGC">HDGC</a> and improved patient outcomes, there are still limited options for clinical interventions (<a 
href="#XGuilford2010">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford2010">2010</a>). Individuals with a family history of <a 
href="#glo:HDGC">HDGC</a> are recommended to be tested for <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutations</a> in late adolescence and are offered prophylactic stomach surgery at the age of 20 years. Another option is annual endoscopic screening to diagnose early stage stomach cancers with surgical intervention once they are detected (<a 
href="#XOliveira2013">Oliveira <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XOliveira2013">2013</a>). However, these early stage cancers are difficult to detect and may be missed in regular screening. Thus patients carrying germline <span class="cmti-12">CDH1</span> <a 
href="#glo:mutation">mutations</a> either have surgery which has a significant impact on quality of life and risk of complications or remain at risk of developing stomach <a 
href="#glo:cancer">cancer</a>. There are similar concerns for female <a 
href="#glo:mutation">mutations</a> carriers for the management of high-risk lobular breast <a 
href="#glo:cancer">cancer</a>. An effective treatment against <span class="cmti-12">CDH1 </span><a 
href="#glo:mutant">mutant</a> cancers would potentially have significant therapeutic and preventative applications for many patients. <a 
 id="x1-42001r49"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.3.3 </span> <a 
 id="x1-430003"></a>Cell Line Models of <span class="cmti-12">CDH1 </span>Null Mutations
</h4>
<!--l. 1113--><p class="noindent" >
<p>Previous work published by members of our research group used a model of homozygous <span class="cmti-12">CDH1</span> <sup><span class="cmsy-8">-</span><span class="cmmi-8">∕</span><span class="cmsy-8">-</span> </sup> null <a 
href="#glo:mutation">mutation</a> in non-malignant MCF10A breast cells to show that loss of <span class="cmti-12">CDH1</span> alone was not sufficient to induce <a 
href="#glo:EMT">EMT</a> (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>). However, <span class="cmti-12">CDH1 </span>deficient cells did manifest changes in morphology, migration, and weaker cell adhesion (<a 
href="#XChen2014">Chen</p>
<em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>). This <span class="cmti-12">CDH1</span><sup><span class="cmsy-8">-</span><span class="cmmi-8">∕</span><span class="cmsy-8">-</span></sup> MCF10A model has been used in a <a 
href="#glo:genome">genome</a>-wide screen of 18,120 genes using <a 
href="#glo:siRNA">siRNA</a> and a complementary drug screen using 4057 known drugs to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners to <a 
href="#glo:E-cadherin">E-cadherin</a> (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). One of the strongest candidate pathways identified by the <a 
href="#glo:siRNA">siRNA</a> screen was the <a 
href="#glo:GPCR">G protein coupled receptor (GPCR)</a> signalling cascades, which were highly enriched in a <a 
href="#glo:GO">Gene Ontology</a> (<a 
href="#glo:GO">GO</a>) analysis. This was supported by validation with Pertussis toxin, known to target G<sub><span class="cmmi-8">αi</span></sub> signalling (<a 
href="#XClark2004">Clark</a>, <a 
href="#XClark2004">2004</a>). <a 
 id="x1-43001r47"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">1.4 </span> <a 
 id="x1-440004"></a>Summary and Research Direction of Thesis
</h3>
<!--l. 1139--><p class="noindent" >
<a 
href="#glo:genomic">Genomic</a> technologies have immense potential for understanding of genetics and improving healthcare, including identification of genes altered in cancer for molecular diagnosis, prognostic biomarkers, and therapeutic targets. This has been demonstrated by the identification of <a 
href="#glo:driver mutation">driver</a> genes in many cancers, distinguishing tumour subtypes by <a 
href="#glo:gene expression">expression</a> profiles, and the development of targeted therapies against <a 
href="#glo:oncogene">oncogenes</a> (such as <span class="cmti-12">BRAF</span>) and <a 
href="#glo:tumour suppressor">tumour suppressors</a> (such as <span class="cmti-12">BRCA1</span>). <a 
href="#glo:synthetic lethal">Synthetic lethality</a> is an important genetic interaction to study fundamental cellular functions and exploit them for biomarker identification and cancer treatment. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> interactions present a means to target loss of function <a 
href="#glo:mutation">mutations</a> and genetic dysregulation in <a 
href="#glo:tumour suppressor">tumour suppressor</a> genes by identifying interacting partners with redundant or compensating molecular functions.
</p><!--l. 1142--><p class="indent" >
Discovery of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">CDH1 </span>would contribute to an understanding of the molecular mechanisms driving the growth of <span class="cmti-12">CDH1 </span>deficient <a 
href="#glo:tumour">tumours</a> and identification of potential therapeutic targets or <a 
href="#glo:chemoprevention">chemopreventative</a> agents for management of <a 
href="#glo:HDGC">HDGC</a>. The clinical potential of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> approach has been demonstrated with the application of olaparib against <span class="cmti-12">BRCA1 </span>and <span class="cmti-12">BRCA2 </span><a 
href="#glo:mutation">mutations</a> (<a 
href="#XLord2014">Lord <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLord2014">2015</a>) but there remains the need to systematically identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes for other <a 
href="#glo:tumour suppressor">tumour suppressors</a> such as <span class="cmti-12">CDH1</span>.
</p><!--l. 1150--><p class="indent" >
Effective screening, prediction, and analysis of <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions are a crucial part of developing next generation anti-cancer strategies. Therefore, we propose developing a computational statistical procedure to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions and construct gene networks. This will enable the development of novel <a 
href="#glo:precision medicine">precision medicine</a> targeted to particular molecular aberrations.
</p><!--l. 1152--><p class="indent" >
<p>To address the concerns raised by recent computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery in cancer (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>; <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>; <a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWappett2016">2016</a>), I</p>
present similar analysis using solely <a 
href="#glo:gene expression">gene expression</a> data which is widely available for a large number of samples in many different cancers. This uses a statistical methodology the <a 
href="#glo:SLIPT">SLIPT</a> developed for this purpose. To further determine the limitations and implications of <a 
href="#glo:synthetic lethal">synthetic lethal</a> predictions, modelling and simulation was performed upon the statistical behaviour of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs in <a 
href="#glo:genomic">genomics</a> data. Comparison of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene candidates from public data analysis and experimental candidates, pathway analysis, and networks structure will also be presented to investigate the relationships between <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates. Release of the R code used for simulation, prediction, and analysis will enable adoption of the methodology by the research community and comparison to existing methods. This thesis aims to develop predictions for <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes, with a focus on the <span class="cmti-12">CDH1</span>, and to compare these to the findings of <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>). In addition, this thesis will develop of network approaches for <a 
href="#glo:graph">pathway</a> structure and simulate <a 
href="#glo:gene expression">gene expression</a> on <a 
href="#glo:graph">pathway</a> structure with the <a 
href="#glo:bioinformatics">bioinformatics</a> and <a 
href="#glo:computational biology">computational biology</a> investigations. <a 
 id="x1-44001r50"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.4.1 </span> <a 
 id="x1-450001"></a>Thesis Aims
</h4>
<!--l. 1157--><p class="noindent" >
This thesis aims to develop and demonstrate a method to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs using public <a 
href="#glo:gene expression">gene expression</a> data. Chapter <a 
href="#x1-730003">3<!--tex4ht:ref: chap:methods_dev --></a> describes this <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection methodology (<a 
href="#glo:SLIPT">SLIPT</a>), the methods developed to evaluate it, and the release of R software packages. This chapter also documents the simulation procedures and network analysis that were developed to support the use of <a 
href="#glo:SLIPT">SLIPT</a> and perform analyses throughout this thesis.
</p><!--l. 1159--><p class="indent" >
To demonstrate the <a 
href="#glo:SLIPT">SLIPT</a> methodology for analysis of <a 
href="#glo:RNA-Seq">RNA-Seq</a> <a 
href="#glo:gene expression">gene expression</a> data, an analysis was performed in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a> to identify candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene partners of <span class="cmti-12">CDH1 </span>in public breast and stomach cancer data (<a 
href="#XTCGA2014GC">Bass <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014GC">2014</a>; <a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>). This analysis demonstrates the biological relevance of candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners by identifying <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways with comparisons to an experimental <a 
href="#glo:siRNA">siRNA</a> screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>).
</p><!--l. 1161--><p class="indent" >
<p>To assess the importance of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes within <a 
href="#glo:pathway">pathways</a>, network analysis was also performed using <a 
href="#glo:graph">graph</a> structures in Chapter <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>. Chapter <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a> also used <a 
href="#glo:graph">pathway</a> structure to identify directional relationships between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates and explore the disparity between them. The <a 
href="#glo:SLIPT">SLIPT</a> methodology was supported by simulation-based investigations in Chapters <a 
href="#x1-730003">3<!--tex4ht:ref: chap:methods_dev --></a> and <a 
href="#x1-1340006">6<!--tex4ht:ref: chap:simulation --></a> to evaluate performance of <a 
href="#glo:SLIPT">SLIPT</a> with</p>
<p>known <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes in simulated data. Graph structures were used in Chapter <a 
href="#x1-1340006">6<!--tex4ht:ref: chap:simulation --></a> to investigate the effect of <a 
href="#glo:graph">pathway</a> correlation structures on <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection with <a 
href="#glo:SLIPT">SLIPT</a> in simulated data, including complex correlation structures within biological pathways.</p>
</p>
<h2 class="chapterHead">
<span class="titlemark">Chapter 2</span><br /><a 
 id="x1-460002"></a>Methods and Resources
</h2>
<!--l. 5--><p class="noindent" >
In this Chapter, I will outline the various existing resources and methods that were used throughout this project. This includes public data repositories, stable and development releases of software packages (primarily using the R programming environment), and custom implementation of <a 
href="#glo:bioinformatics">bioinformatic</a> methods and statistical concepts with Shell or R scripts developed for this purpose. The methods and packages that have been developed specifically for this project will be covered in Chapter <a 
href="#x1-730003">3<!--tex4ht:ref: chap:methods_dev --></a> with supporting data and demonstration of their use . <a 
 id="x1-46001r51"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">2.1 </span> <a 
 id="x1-470001"></a>Bioinformatics Resources for Genomics Research
</h3>
<a 
 id="x1-47001r52"></a>
<h4 class="subsectionHead">
<span class="titlemark">2.1.1 </span> <a 
 id="x1-480001"></a>Public Data and Software Packages
</h4>
<!--l. 9--><p class="noindent" >
Various <a 
href="#glo:bioinformatics">bioinformatics</a> resources, such as databases and methods, have become integral parts of genetics and <a 
href="#glo:genomic">genomics</a> research. Reference <a 
href="#glo:genome">genomes</a>, genotyped variants, <a 
href="#glo:gene expression">gene expression</a>, and epigenetics profiles are among the most commonly used resources. <a 
href="#glo:gene expression">Gene expression</a> data, in particular, is widely available from <a 
href="#glo:microarray">microarray</a> and <a 
href="#glo:RNA-Seq">RNA-Seq</a> projects, driven by data sharing, data mining, and the wider initiatives for publicly available data for enabling the scientific community to further utilise the data generated beyond a single research group or consortium (<a 
href="#XRung2013">Rung and Brazma</a>, <a 
href="#XRung2013">2013</a>) These datasets are a valuable resource to examine the changes in <a 
href="#glo:gene expression">gene expression</a> occurring in cancers and the variation between samples. The potential for integrating findings from publicly available <a 
href="#glo:genomic">genomic</a> data with experimental investigations has expanded with <a 
href="#glo:RNA-Seq">RNA-Seq</a> datasets, including large-scale cancer <a 
href="#glo:genomic">genomics</a> projects (<a 
href="#XICGC2011">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XICGC2011">2011</a>). This thesis presents such an investigation, enabled by the release of these datasets and tools developed to handle them.
</p><!--l. 14--><p class="indent" >
<p>It is now common practice for <a 
href="#glo:bioinformatics">bioinformatics</a> researchers to release open-source code or provide software packages to enable replication of the findings or further applications of the methods (<a 
href="#XStajich2006">Stajich and Lapp</a>, <a 
href="#XStajich2006">2006</a>). This is part of a wider movement in software and data analysis, including the development of Linux and the R programming environment (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>). In addition to the R packages hosted on <a 
href="#glo:CRAN">comprehensive R archive network (CRAN)</a> (<a 
href="#XCRAN">CRAN</a>, <a 
href="#XCRAN">2017</a>), many packages specifically developed for applications in <a 
href="#glo:bioinformatics">bioinformatics</a> are hosted on the Bioconductor repositories (<a 
href="#XGentleman2004">Gentleman <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGentleman2004">2004</a>), and numerous packages in various stages of development are hosted on GitHub</p>
(<a 
href="https://github.com/" class="url" ><span class="cmtt-12">https://github.com/</span></a>). Packages from each of these resources have been used throughout this project and are cited wherever possible. Several R packages have been developed during this thesis project and publicly released on GitHub or will be released in conjunction with a publication. <a 
 id="x1-48001r46"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">2.1.1.1 </span> <a 
 id="x1-490001"></a>Cancer Genome Atlas Data
</h5>
<!--l. 17--><p class="noindent" >
Molecular profile data for normal and tumour samples were downloaded from publicly available sources, using the <a 
href="#glo:TCGA">TCGA</a> (<a 
href="#XTCGA2017web">TCGA</a>, <a 
href="#XTCGA2017web">2017</a>) and the <a 
href="#glo:ICGC">International Cancer Genome Consortium (ICGC)</a> web portals (<a 
href="#XICGC2011">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XICGC2011">2011</a>). These include <a 
href="#glo:gene expression">gene expression</a> (<a 
href="#glo:RNA-Seq">RNA-Seq</a>), <a 
href="#glo:somatic">somatic</a> <a 
href="#glo:mutation">mutations</a>, and clinical data. The versions were downloaded on the 6<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup> of August 2015 (Release 19) and the 2<sup class="textsuperscript"><span class="cmr-10x-x-109">nd</span></sup> of May 2016 (Release 20) for breast and stomach cancer respectively via the <a 
href="#glo:ICGC">ICGC</a> data portal (<a 
href="https://dcc.icgc.org/" class="url" ><span class="cmtt-12">https://dcc.icgc.org/</span></a>).
</p><!--l. 23--><p class="indent" >
The <a 
href="#glo:TCGA">TCGA</a> project (<a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>) used widely adopted tools: “Bowtie” for alignment (<a 
href="#Xbowtie">Langmead <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xbowtie">2009</a>), “mapslice” to detect splice sites (<a 
href="#Xmapsplice">Wang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xmapsplice">2010</a>), and the <a 
href="#glo:RSEM">RSEM</a> approach to quantify reads as a measure of <a 
href="#glo:gene expression">gene expression</a> (<a 
href="#XRSEM">Li <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XRSEM">2010</a>). These are widely acceptable tools for processing <a 
href="#glo:RNA-Seq">RNA-Seq</a> data which were used to produce the raw counts of mapped reads (tier 1) and normalised <a 
href="#glo:gene expression">expression</a> data (tier 3) publicly downloaded from <a 
href="#glo:ICGC">ICGC</a> and <a 
href="#glo:TCGA">TCGA</a> respectively.
</p><!--l. 27--><p class="indent" >
Raw count and <a 
href="#glo:RSEM">RSEM</a> normalised <a 
href="#glo:TCGA">TCGA</a> <a 
href="#glo:gene expression">expression</a> data from Illumina <a 
href="#glo:RNA-Seq">RNA-Seq</a> protocols were downloaded for 1177 breast samples (113 normal, 1057 primary tumour, and 7 metastases) for 20,501 genes. <a 
href="#glo:TCGA">TCGA</a> breast <a 
href="#glo:somatic">somatic</a> <a 
href="#glo:mutation">mutation</a> data for 981 samples (976 primary tumours and 5 metastases) across 25,836 genes were downloaded. These included 969 samples (964 primary tumours and 5 metastases) with corresponding <a 
href="#glo:RNA-Seq">RNA-Seq</a> <a 
href="#glo:gene expression">expression</a> data and 19,166 genes mapped from Ensembl identifiers to gene symbols. Of these genes, 16,156 had corresponding <a 
href="#glo:gene expression">gene expression</a> information. Unless otherwise stated, the raw counts were used for further processing rather than the <a 
href="#glo:RSEM">RSEM</a> normalised data (provided by <a 
href="#glo:TCGA">TCGA</a> tier 3). <a 
href="#glo:somatic">Somatic</a> <a 
href="#glo:mutation">mutations</a> was reported if there were non-synonymous substitutions, frameshifts, or truncations (by premature stop codons) detected which would likely disrupt the <a 
href="#glo:wild-type">wild-type</a> gene function.
</p><!--l. 31--><p class="indent" >
<p>Raw count <a 
href="#glo:TCGA">TCGA</a> <a 
href="#glo:gene expression">expression</a> data (<a 
href="#glo:TCGA">TCGA</a> tier 1) from Illumina <a 
href="#glo:RNA-Seq">RNA-Seq</a> was downloaded for 450 stomach samples (35 normal, 415 primary tumour) for 20,501 genes.</p>
<a 
href="#glo:TCGA">TCGA</a> stomach <a 
href="#glo:mutation">mutation</a> data was also used for 289 samples across 25807 genes, corresponding to 19436 genes with <a 
href="#glo:gene expression">expression</a> data. <a 
 id="x1-49001r56"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">2.1.1.2 </span> <a 
 id="x1-500002"></a>Reactome and Annotation Data
</h5>
<!--l. 44--><p class="noindent" >
<a 
href="#glo:pathway">Pathway</a> analysis was performed for human <a 
href="#glo:pathway">pathway</a> annotation from the Reactome database (version 52) with <a 
href="#glo:pathway">pathway</a> gene sets derived from the <span class="cmtt-12">reactome.db </span>R package. Entrez identifiers were mapped to gene symbols or aliases to match to <a 
href="#glo:TCGA">TCGA</a> <a 
href="#glo:gene expression">expression</a> and <a 
href="#glo:mutation">mutation</a> data using the <span class="cmtt-12">org.Hs.eg.db </span>R package. Gene <a 
href="#glo:gene expression">expression</a> for breast cancer from Gatza and colleagues were also used (Gatza <span class="cmti-12">et al</span>., 2011; Gatza <span class="cmti-12">et</span> <span class="cmti-12">al</span>., 2014). The gene symbols for each pathway were matched to the <a 
href="#glo:gene expression">expression</a> data and to construct a matrix of category membership using the <span class="cmtt-12">safe </span>R package (<a 
href="#Xsafe">Barry</a>, <a 
href="#Xsafe">2016</a>). <a 
 id="x1-50001r54"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">2.2 </span> <a 
 id="x1-510002"></a>Data Handling
</h3>
<a 
 id="x1-51001r55"></a>
<h4 class="subsectionHead">
<span class="titlemark">2.2.1 </span> <a 
 id="x1-520001"></a>Normalisation
</h4>
<!--l. 50--><p class="noindent" >
Apart from the <a 
href="#glo:PAM50">Prediction Analysis of Microarray 50 (PAM50)</a> subtyping procedure (<a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>), which required <a 
href="#glo:RSEM">RSEM</a> normalised data (J.S. Parker personal communication), the analysis of the <a 
href="#glo:RNA-Seq">RNA-Seq</a> data presented here was based on raw read count data. After some samples were removed for consistency (based on a Euclidean distance correlation matrix as described in Section <a 
href="#x1-530002">2.2.2<!--tex4ht:ref: methods:sample_qc --></a>), raw read counts were log-scaled and the final dataset was normalised as <a 
href="#glo:CPM">Counts per Million mapped reads (CPM)</a>, weighted by variance modelling, using the <span class="cmtt-12">voom </span>function (<a 
href="#XLaw2014">Law <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLaw2014">2014</a>) in the <span class="cmtt-12">limma </span>R package (<a 
href="#Xlimma">Ritchie <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xlimma">2015</a>). This procedure adjusts the data to account for differences in read count by sequencing depth between samples and length between genes. <a 
 id="x1-52001r59"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.2.2 </span> <a 
 id="x1-530002"></a>Sample Triage
</h4>
<!--l. 57--><p class="noindent" >
<p>The <a 
href="#glo:TCGA">TCGA</a> breast <a 
href="#glo:RNA-Seq">RNA-Seq</a> data were assessed for batch effects using a correlation matrix of the log-transformed raw counts for which a heatmap (Euclidean distance, complete linkage) is shown in Figure <a 
href="#x1-1680022">A.2<!--tex4ht:ref: fig:corr_map --></a>. While no major batch effects were detectable between</p>
<p>the samples, 9 samples were excluded due to poor correlation with the remaining samples, as detailed in Table <a 
href="#x1-530071">2.1<!--tex4ht:ref: tab:qc --></a>. These samples showed unusual density plots compared to the rest of the dataset, and exhibited low mean read count in Figures <a 
href="#x1-530032">2.2<!--tex4ht:ref: fig:density --></a> and <a 
href="#x1-530063">2.3<!--tex4ht:ref: fig:boxplot --></a>. A heatmap showing key clinical properties of these excluded samples and their correlation with the remainder of the samples is shown in Figure <a 
href="#x1-1680011">A.1<!--tex4ht:ref: fig:corr_map_part --></a>, and a full correlation heatmap (Figure <a 
href="#x1-1680022">A.2<!--tex4ht:ref: fig:corr_map --></a>) shows these samples as relatively poorly correlated outliers in the bottom rows and left columns. In addition to the clustering analysis (in Appendix <a 
href="#x1-1680001">A.1<!--tex4ht:ref: appendix:sample_correlation --></a>), replicate tumour samples were also examined for sample quality in Appendix <a 
href="#x1-1690002">A.2<!--tex4ht:ref: appendix:replicate_samples --></a>. After removal of these samples, the <a 
href="#glo:TCGA">TCGA</a> dataset used for analysis consisted of the remaining 1168 samples (from 1040 patients): 1049 tumour samples, 112 normal tissue for matched samples, and 7 metastases.</p>
</p><!--l. 64--><p class="indent" >
<a 
 id="x1-530032"></a><a 
 id="x1-530063"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 67--><p class="noindent" >
</p><!--l. 72--><p class="noindent" >
<img 
src="fig2a.png" alt="PIC"  
 /><a 
 id="x1-53001r1"></a> <span class="cmr-10x-x-109">Raw</span> <span class="cmr-10x-x-109">counts</span> <span class="cmr-10x-x-109">(log-scale)</span><img 
src="fig2b.png" alt="PIC"  
 /><a 
 id="x1-53002r2"></a> <span class="cmr-10x-x-109">Voom</span> <span class="cmr-10x-x-109">normalised</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 2.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Read count density.</span></strong> <span class="cmr-10x-x-109">Sample density plots of raw counts on log-scale and</span> <span class="cmr-10x-x-109">voom normalised showing samples removed due to quality concerns.</span></span></p>
</div>
<!--tex4ht:label?: x1-530032 -->
<div class="center" 
>
<!--l. 88--><p class="noindent" >
</p><!--l. 93--><p class="noindent" >
<img 
src="fig3a.png" alt="PIC"  
 /><a 
 id="x1-53004r1"></a> <span class="cmr-10x-x-109">Mean</span> <span class="cmr-10x-x-109">raw</span> <span class="cmr-10x-x-109">counts</span> <span class="cmr-10x-x-109">(log-scale)</span><img 
src="fig3b.png" alt="PIC"  
 /><a 
 id="x1-53005r2"></a> <span class="cmr-10x-x-109">Mean</span> <span class="cmr-10x-x-109">voom</span> <span class="cmr-10x-x-109">normalised</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 2.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">Read count sample mean.</span></strong> <span class="cmr-10x-x-109">Boxplots of sample means for raw counts on</span> <span class="cmr-10x-x-109">log-scale and voom normalised show removed tumour samples with low mean read count.</span></span></p>
</div>
<!--tex4ht:label?: x1-530063 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 107--><p class="indent" >
<a 
 id="x1-530071"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 2.1: </span><span class="content">Excluded samples by batch and clinical characteristics.</span>
</div><!--tex4ht:label?: x1-530071 -->
<!--tex4ht:inline--><div class="tabular"><table id="TBL-30" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-30-1g"><col 
id="TBL-30-1" /><col 
id="TBL-30-2" /><col 
id="TBL-30-3" /><col 
id="TBL-30-4" /><col 
id="TBL-30-5" /><col 
id="TBL-30-6" /><col 
id="TBL-30-7" /><col 
id="TBL-30-8" /><col 
id="TBL-30-9" /><col 
id="TBL-30-10" /><col 
id="TBL-30-11" /><col 
id="TBL-30-12" /><col 
id="TBL-30-13" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-30-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-30-1-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<span class="cmbx-12">Tissue Source</span>
</div>  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-1-3"  
class="td11">  <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<span class="cmbx-12">Type</span>
</div>  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-1-4"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<span class="cmbx-12">Batch</span>
</div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-1-5"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<span class="cmbx-12">Plate</span>
</div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-1-6"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<span class="cmbx-12">Patient</span>
</div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-1-7"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<span class="cmbx-12">Samples</span>
</div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-1-8"  
class="td11">   <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<span class="cmbx-12">p53</span>
</div>    </td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-1-9"  
class="td11">  <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<span class="cmbx-12">Subtype</span>
</div>  </td><td colspan="2" style="white-space:nowrap; text-align:center;" id="TBL-30-1-10"  
class="td11">           <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<span class="cmbx-12">Treatment (History)</span>
</div>             </td><td colspan="3" style="white-space:nowrap; text-align:center;" id="TBL-30-1-12"  
class="td11">         <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<span class="cmbx-12">Clinical Subtypes (Stage)</span>
</div>
</td></tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-30-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-30-2-1"  
class="td11">   
A7 Christiana
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-2-2"  
class="td11"> 
Tumour
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-2-3"  
class="td11">   
47
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-2-4"  
class="td11"> 
A227
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-2-5"  
class="td11">  
A0DB
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-2-6"  
class="td11">   
1 of 3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-2-7"  
class="td11">   
NA
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-2-8"  
class="td11"> 
Luminal A
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-2-9"  
class="td11">            
Mastectomy
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-2-10"  
class="td11"> 
(no)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-2-11"  
class="td11"> 
<a 
href="#glo:ER">Estrogen receptor (ER)</a><sup><span class="cmr-8">+</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-2-12"  
class="td11">   
Ductal
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-2-13"  
class="td11"> 
<ol start="2" style="list-style-type: decimal">
<li></td>
    </tr><tr  
     style="vertical-align:baseline;" id="TBL-30-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-30-3-1"  
    class="td11">   
    
A7 Christiana
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-3-2"  
    class="td11"> 
    
Tumour
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-3-3"  
    class="td11">   
    
96
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-3-4"  
    class="td11"> 
    
A220
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-3-5"  
    class="td11">  
    
A13D
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-3-6"  
    class="td11">   
    
1 of 3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-3-7"  
    class="td11"> 
    
Wildtype
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-3-8"  
    class="td11"> 
    
Luminal A
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-3-9"  
    class="td11">            
    
Mastectomy
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-3-10"  
    class="td11"> 
    
(no)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-3-11"  
    class="td11">         
    
<a 
    href="#glo:ER">ER</a><sup><span class="cmr-8">+</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-3-12"  
    class="td11">   
    
Ductal
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-3-13"  
    class="td11"> 
<ol start="2" style="list-style-type: decimal">
<li></td>
    </tr><tr  
     style="vertical-align:baseline;" id="TBL-30-4-"><td  style="white-space:nowrap; text-align:center;" id="TBL-30-4-1"  
    class="td11">   
    
A7 Christiana
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-4-2"  
    class="td11"> 
    
Tumour
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-4-3"  
    class="td11">   
    
96
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-4-4"  
    class="td11"> 
    
A227
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-4-5"  
    class="td11">  
    
A13E
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-4-6"  
    class="td11">   
    
1 of 3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-4-7"  
    class="td11">   
    
NA
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-4-8"  
    class="td11">   
    
Basal
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-4-9"  
    class="td11">           
    
Lumpectomy
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-4-10"  
    class="td11"> 
    
(no)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-4-11"  
    class="td11">         
    
<a 
    href="#glo:ER">ER</a><sup><span class="cmsy-8">-</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-4-12"  
    class="td11">   
    
Ductal
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-4-13"  
    class="td11"> 
<ol start="2" style="list-style-type: decimal">
<li></td>
    </tr><tr  
     style="vertical-align:baseline;" id="TBL-30-5-"><td  style="white-space:nowrap; text-align:center;" id="TBL-30-5-1"  
    class="td11">   
    
A7 Christiana
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-5-2"  
    class="td11"> 
    
Tumour
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-5-3"  
    class="td11">  
    
142
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-5-4"  
    class="td11"> 
    
A277
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-5-5"  
    class="td11">  
    
A26E
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-5-6"  
    class="td11">   
    
1 of 3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-5-7"  
    class="td11">   
    
NA
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-5-8"  
    class="td11">   
    
Basal
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-5-9"  
    class="td11">           
    
Lumpectomy
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-5-10"  
    class="td11"> 
    
(no)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-5-11"  
    class="td11">         
    
<a 
    href="#glo:ER">ER</a><sup><span class="cmr-8">+</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-5-12"  
    class="td11">   
    
Ductal
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-5-13"  
    class="td11"> 
<ol start="2" style="list-style-type: decimal">
<li></td>
    </tr><tr  
     style="vertical-align:baseline;" id="TBL-30-6-"><td  style="white-space:nowrap; text-align:center;" id="TBL-30-6-1"  
    class="td11"> 
    
A7 Christiana
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-6-2"  
    class="td11"> 
    
Tumour
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-6-3"  
    class="td11">   
    
47
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-6-4"  
    class="td11"> 
    
A277
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-6-5"  
    class="td11">  
    
A0DC
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-6-6"  
    class="td11">   
    
1 of 2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-6-7"  
    class="td11">   
    
NA
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-6-8"  
    class="td11"> 
    
Luminal A
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-6-9"  
    class="td11">            
    
Mastectomy
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-6-10"  
    class="td11"> 
    
(yes)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-6-11"  
    class="td11">         
    
<a 
    href="#glo:ER">ER</a><sup><span class="cmr-8">+</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-6-12"  
    class="td11">   
    
Lobular
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-6-13"  
    class="td11"> 
<ol start="3" style="list-style-type: decimal">
<li></td>
</li>
</ol></li>
</ol></li>
</ol></li>
</ol></li>
</ol>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-30-7-"><td  style="white-space:nowrap; text-align:center;" id="TBL-30-7-1"  
class="td11"> 
A7 Christiana
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-7-2"  
class="td11"> 
Tumour
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-7-3"  
class="td11">  
142
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-7-4"  
class="td11"> 
A220
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-7-5"  
class="td11">  
A26I
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-7-6"  
class="td11">   
1 of 2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-7-7"  
class="td11">  
Mutant
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-7-8"  
class="td11">   
Basal
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-7-9"  
class="td11">           
Lumpectomy
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-7-10"  
class="td11"> 
(yes)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-7-11"  
class="td11">         
<a 
href="#glo:ER">ER</a><sup><span class="cmsy-8">-</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-7-12"  
class="td11">   
Ductal
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-7-13"  
class="td11"> 
<ol start="2" style="list-style-type: decimal">
<li></td>
</li>
</ol>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-30-8-"><td  style="white-space:nowrap; text-align:center;" id="TBL-30-8-1"  
class="td11"> 
AC Intl Genomics
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-8-2"  
class="td11"> 
Tumour
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-8-3"  
class="td11">  
177
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-8-4"  
class="td11"> 
A18M
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-8-5"  
class="td11">  
A2QH
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-8-6"  
class="td11">   
2 of 2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-8-7"  
class="td11">  
Mutant
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-8-8"  
class="td11">   
Basal
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-8-9"  
class="td11">        
Radical Mastectomy
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-8-10"  
class="td11"> 
(no)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-8-11"  
class="td11">         
<a 
href="#glo:ER">ER</a><sup><span class="cmsy-8">-</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-8-12"  
class="td11"> 
Metaplastic
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-8-13"  
class="td11"> 
<ol start="2" style="list-style-type: decimal">
<li></td>
</li>
</ol>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-30-9-"><td  style="white-space:nowrap; text-align:center;" id="TBL-30-9-1"  
class="td11"> 
AC Intl Genomics
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-9-2"  
class="td11"> 
Tumour
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-9-3"  
class="td11">  
177
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-9-4"  
class="td11"> 
A220
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-9-5"  
class="td11">  
A2QH
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-9-6"  
class="td11">   
2 of 2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-9-7"  
class="td11">  
Mutant
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-9-8"  
class="td11">   
Basal
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-9-9"  
class="td11">        
Radical Mastectomy
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-9-10"  
class="td11"> 
(no)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-9-11"  
class="td11">         
<a 
href="#glo:ER">ER</a><sup><span class="cmsy-8">-</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-9-12"  
class="td11"> 
Metaplastic
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-9-13"  
class="td11"> 
<ol start="2" style="list-style-type: decimal">
<li></td>
</li>
</ol>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-30-10-"><td  style="white-space:nowrap; text-align:center;" id="TBL-30-10-1"  
class="td11"> 
GI ABS IUPUI
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-10-2"  
class="td11"> 
Normal
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-10-3"  
class="td11">  
177
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-10-4"  
class="td11"> 
A16F
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-10-5"  
class="td11">  
A2C8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-10-6"  
class="td11">   
1 of 1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-10-7"  
class="td11">   
NA
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-10-8"  
class="td11"> 
Luminal A
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-10-9"  
class="td11"> 
Radical Mastectomy and Neoadjuvant
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-10-10"  
class="td11"> 
(no)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-10-11"  
class="td11">         
<a 
href="#glo:ER">ER</a><sup><span class="cmr-8">+</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-10-12"  
class="td11">   
Ductal
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-30-10-13"  
class="td11"> 
<ol start="2" style="list-style-type: decimal">
<li></td>
    </tr><tr 
    class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
     style="vertical-align:baseline;" id="TBL-30-11-"><td  style="white-space:nowrap; text-align:center;" id="TBL-30-11-1"  
    class="td11">                  </td></tr></table>                                                                                                                                                                                                  </div>

</li>
</ol>
</div><hr class="endfloat" />
<!--l. 138--><p class="indent" >
Similarly, a correlation matrix of log-transformed raw counts was used to evaluate sample quality for <a 
href="#glo:TCGA">TCGA</a> stomach <a 
href="#glo:RNA-Seq">RNA-Seq</a>. A tumour sample (patient 4294) was removed due to similar quality concerns leaving a final dataset for 449 samples (from 417 patients): 414 tumour samples and 35 normal tissue samples. <a 
 id="x1-53008r60"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.2.3 </span> <a 
 id="x1-540003"></a>Metagenes and the Singular Value Decomposition
</h4>
<!--l. 141--><p class="noindent" >
A “<a 
href="#glo:metagene">metagene</a>” offers a one-dimensional summary of <a 
href="#glo:pathway">pathway</a> (expression) activation or inactivation by dimension reduction of a matrix, avoiding negatively correlated genes averaging out the signal of a mean-based centroid (<a 
href="#XHuang2003">Huang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XHuang2003">2003</a>). Constructing <a 
href="#glo:pathway">pathway</a> <a 
href="#glo:metagene">metagenes</a> used gene sets for Reactome and the Gatza signatures (<a 
href="#XGatza2011">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2011">2011</a>, <a 
href="#XGatza2014">2014</a>) as specified above (in Section <a 
href="#x1-500002">2.1.1.2<!--tex4ht:ref: methods:gene_set --></a>). The singular-value decomposition was performed (<span class="cmmi-12">X </span>= <span class="cmmi-12">U</span><sup><span class="cmmi-8">T</span> </sup><span class="cmmi-12">DV </span>, where <span class="cmmi-12">X </span>is the data matrix of the gene set, with genes <span class="cmsy-10x-x-120">×</span> samples) and the leading eigenvector (first column of <span class="cmmi-12">V </span>) corresponding to the largest singular value was used as a <a 
href="#glo:metagene">metagene</a> for the <a 
href="#glo:pathway">pathway</a> gene set. To ensure consistent directionality of <a 
href="#glo:metagene">metagene</a> signals, the median of the gene set in each sample was calculated and correlated against the <a 
href="#glo:metagene">metagene</a> with the (arbitrary) <a 
href="#glo:metagene">metagene</a> sign adjusted as needed to conform with the majority of the gene set (i.e., positive correlation between <a 
href="#glo:metagene">metagene</a> and the median-based centroid). To ensure that genes and <a 
href="#glo:pathway">pathways</a> were weighted equally, <a 
href="#glo:metagene">metagenes</a> were derived from a z-transformed (mean 0, standard deviation 1) dataset of <a 
href="#glo:gene expression">gene expression</a> and samples were scaled (by fractional ranking) for each <a 
href="#glo:metagene">metagene</a> so that they were comparable on a [0<span class="cmmi-12">, </span>1] scale. <a 
 id="x1-54001r68"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.2.4 </span> <a 
 id="x1-550004"></a>Candidate Triage and Integration with Screen Data
</h4>
<!--l. 144--><p class="noindent" >
<p>Candidate triage in combination with the experimental data was intended to integrate findings of the <a 
href="#glo:SLIPT">SLIPT</a> analysis with an ongoing experiment project (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>; <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). The first procedure to compare the <a 
href="#glo:SLIPT">SLIPT</a> gene candidates for <span class="cmti-12">CDH1 </span>with an <a 
href="#glo:siRNA">siRNA</a> experimental screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) was a direct comparison of the overlapping candidates, presented in a Venn diagram and tested with the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test. Since these candidates modestly overlapped at the gene level (even when excluding genes not contained in both datasets), further gene set over-representation analysis was</p>
performed for pathways specific to each detection approach and the intersection of the two.
</p><!--l. 146--><p class="indent" >
The <a 
href="#glo:pathway">pathway</a> composition of the intersection was further verified by a permutation resampling analysis (as described in Section <a 
href="#x1-630006">2.3.6<!--tex4ht:ref: methods:permutation --></a>): the same number of genes detected by <a 
href="#glo:SLIPT">SLIPT</a> were sampled randomly from the universe of genes tested by both approaches. These samplings were performed over 1 million iterations and the <a 
href="#glo:pathway">pathway</a> over-representation was compared for each of the 1652 reactome <a 
href="#glo:pathway">pathways</a>. These over-representation scores (<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>) were compared the observed over-representation in the intersection of the <a 
href="#glo:SLIPT">SLIPT</a> candidates, with the proportion of resamplings with higher <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values used for empirical p-values of <a 
href="#glo:pathway">pathway</a> composition. The <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test was used as an appropriation of Fisher’s exact test on a hypergeometric distribution for resampling to computationally scale <a 
href="#glo:pathway">pathway</a> over-representation tests across iterations. Pathways for which no resamplings were occurred as high as the observed were reported as <span class="cmmi-12">p &lt; </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>. These empirical p-values were adjusted for multiple comparisons (<a 
href="#glo:FDR">FDR</a>). Intersection size was not assumed to be constant across resamplings, so similarly with the proportion of resamplings with higher or lower intersection size were used to evaluate significance of enrichment or depletion respectively (of <a 
href="#glo:siRNA">siRNA</a> candidate among <a 
href="#glo:SLIPT">SLIPT</a> candidate genes). <a 
 id="x1-55001r58"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">2.3 </span> <a 
 id="x1-560003"></a>Techniques
</h3>
<!--l. 151--><p class="noindent" >
Various statistical, computational, and <a 
href="#glo:bioinformatics">bioinformatics</a> techniques were performed throughout this thesis. This Section describes these techniques and gives the parameters used unless otherwise specified. Where relevant, the R package implementation which provided the technique will be acknowledged. <a 
 id="x1-56001r69"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.3.1 </span> <a 
 id="x1-570001"></a>Statistical Procedures and Tests
</h4>
<!--l. 155--><p class="noindent" >
<p>As described in Sections <a 
href="#x1-600004">2.3.4<!--tex4ht:ref: methods:heatmap --></a> and <a 
href="#x1-540003">2.2.3<!--tex4ht:ref: methods:metagene --></a>, the z-transform has been used to generate z-scores in various analyses in this thesis. Each row of the dataset (<span class="cmmi-12">x</span><sub><span class="cmmi-8">i</span></sub>) is transformed into a scores (<span class="cmmi-12">z</span><sub><span class="cmmi-8">i</span> </sub> ) using the mean (<span class="bar-css"><span class="cmmi-12">x</span></span><sub><span class="cmmi-8">i</span></sub>) and standard deviation (<span class="cmmi-12">s</span><sub><span class="cmmi-8">i</span></sub>) of the data such that:</p>
</p>
<center class="math-display" >
<img 
src="thesis3x.png" alt="     xij - ¯xi
zi = --------
        si  " class="math-display"  />
</center> 
This generates data where each row (gene) has a mean of 0 and standard deviation of 1. Where plotted as aa heatmap, any data more than 3 standard deviations above or below the mean was plotted as 3 or <span class="cmsy-10x-x-120">-</span>3 respectively. <!--l. 159-->
<p class="indent" >
Where specified, the Fisher’s exact test, <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test, and correlation were used to measure associations between variables, as implemented in the <span class="cmtt-12">stats </span>R package (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>). Unless otherwise specified, Pearson correlation was used for correlation analyses (<span class="cmmi-12">r</span>) and coefficient of determination (<span class="cmmi-12">R</span><sup><span class="cmr-8">2</span></sup>). Where these comparisons are discussed in more detail, Fisher’s exact test and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> tests are supported by a table or Venn diagram, rendered with the <span class="cmtt-12">limma </span>R package (<a 
href="#Xlimma">Ritchie <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xlimma">2015</a>). In some analyses, correlation is further supported by a scatter plot and a line of best fit derived by least squares linear regression.
</p><!--l. 161--><p class="indent" >
The <span class="cmtt-12">t.test </span>function (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>) has also been used to implement the t-test to compare pairs of data. Where relevant, an <a 
href="#glo:ANOVA">analysis of variance</a> (<a 
href="#glo:ANOVA">ANOVA</a>) has been performed to report significance of multivariate predictors of outcomes, or least squares linear regression performed for the adjusted coefficient of determination (<span class="cmmi-12">R</span><sup><span class="cmr-8">2</span></sup>) and F-statistic p-value to evaluate the fit of the predictor variables. For some analyses these are supported by boxplot or violin plot visualisation (<a 
href="#Xvioplot">Adler</a>, <a 
href="#Xvioplot">2005</a>), rendered in R (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>).
</p><!--l. 163--><p class="indent" >
Multiple comparisons were accounted for with the Benjamini-Hochberg procedure to control the <a 
href="#glo:FDR">FDR</a> unless otherwise specified (<a 
href="#Xfdr1995">Benjamini and Hochberg</a>, <a 
href="#Xfdr1995">1995</a>). This procedure adjusts p-values to achieve an average of the proportion of false-positives among significant tests below a threshold, <span class="cmmi-12">α</span>. The more stringent Holm-Bonferroni (Holm) procedure (<a 
href="#XHolm1979">Holm</a>, <a 
href="#XHolm1979">1979</a>) was also applied in some cases to adjust for multiple comparisons and control the family-wise error rate which adjusts p-values so that the probability that any one of the tests is a false-positive (type-1 error) below a threshold, <span class="cmmi-12">α</span>. <a 
 id="x1-57001r71"></a>
</p>
                                                                                    

<h4 class="subsectionHead">
<span class="titlemark">2.3.2 </span> <a 
 id="x1-580002"></a>Gene Set Over-representation Analysis
</h4>
<!--l. 166--><p class="noindent" >
Gene set enrichment over-representation analysis was performed to test whether there was an enrichment of a gene set (e.g., a biological <a 
href="#glo:pathway">pathway</a>) among a group of input genes. Such input genes may be predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates or a subset defined by clustering (in Section <a 
href="#x1-590003">2.3.3<!--tex4ht:ref: methods:clustering --></a>) or comparison with experimental candidates (in Section <a 
href="#x1-550004">2.2.4<!--tex4ht:ref: methods:venn_analysis --></a>). Initially, these tests were performed using the GeneSetDB web tool (<a 
href="#Xgenesetdb">Araki <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xgenesetdb">2012</a>) hosted by the University of Auckland on the Reactome <a 
href="#glo:pathway">pathways</a> (<a 
href="#XReactome">Croft <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XReactome">2014</a>). Since the GeneSetDB tool used an older version of Reactome (version 40), it was difficult to directly compare with the results of other analysis (in Sections <a 
href="#x1-550004">2.2.4<!--tex4ht:ref: methods:venn_analysis --></a> and <a 
href="#x1-630006">2.3.6<!--tex4ht:ref: methods:permutation --></a>) performed on version 52 (as described in Section <a 
href="#x1-500002">2.1.1.2<!--tex4ht:ref: methods:gene_set --></a>). Thus an implementation of the hypergeometric test in R (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>) was used to test for over-representation against Reactome (version 52) <a 
href="#glo:pathway">pathways</a>. Pathways containing less than 10 genes or more than 500 (as performed in GeneSetDB by <a 
href="#Xgenesetdb">Araki <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xgenesetdb">2012</a>) were excluded before adjusting for multiple comparisons. <a 
 id="x1-58001r72"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.3.3 </span> <a 
 id="x1-590003"></a>Clustering
</h4>
<!--l. 169--><p class="noindent" >
The clustering analysis used unsupervised hierarchical clustering with complete linkage (distance calculated from the furthest possible pairing). For correlation matrices or multivariate normal parameters (e.g., Σ), the distance metric used was Euclidean distance. For empirical or simulated gene and <a 
href="#glo:pathway">pathway</a> <a 
href="#glo:gene expression">expression</a> data correlation distance was used, calculated by <span class="cmmi-12">distance </span>= 1 <span class="cmsy-10x-x-120">- </span><span class="cmmi-12">cor</span>(<span class="cmmi-12">t</span>(<span class="cmmi-12">x</span>)) where <span class="cmmi-12">cor </span>is Pearson correlation and <span class="cmmi-12">t</span>(<span class="cmmi-12">x</span>) is the transpose of the <a 
href="#glo:gene expression">expression</a> matrix. <a 
 id="x1-59001r73"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.3.4 </span> <a 
 id="x1-600004"></a>Heatmap
</h4>
<!--l. 172--><p class="noindent" >
<p>Standardised z-scores of the data were used to plot heatmaps on an appropriate scale. Raw (log-scale) read counts or voom normalised counts per gene (as specified) were plotted as normalised z-scores on a [<span class="cmsy-10x-x-120">-</span>3<span class="cmmi-12">, </span>+3] blue-red scale. Similarly, correlations were plotted on a [<span class="cmsy-10x-x-120">-</span>1<span class="cmmi-12">, </span>+1] blue-red scale. Heatmap dendrograms were generated using the linkage method and distance specified for the clustering performed in Section <a 
href="#x1-590003">2.3.3<!--tex4ht:ref: methods:clustering --></a>. The <span class="cmtt-12">gplots </span>R package (<a 
href="#Xgplots">Warnes <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xgplots">2015</a>) was used to generate many of the heatmaps throughout this thesis, along with a customised heatmap function (released as <span class="cmtt-12">heatmap.2x</span>, detailed in Table <a 
href="#x1-710046">2.6<!--tex4ht:ref: tab:computers_r_packages_dev --></a></p>
and Section <a 
href="#x1-890002">3.5.2<!--tex4ht:ref: methods:r_packages_vis --></a>). Where clearly specified, data have been split into subsets with clustering performed separately on each subset with these plotted alongside each other. <a 
 id="x1-60001r74"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.3.5 </span> <a 
 id="x1-610005"></a>Modelling and Simulations
</h4>
<!--l. 175--><p class="noindent" >
Statistical modelling and simulations were used to test various <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection procedures on simulated data. This involved constructing a statistical model of how <a 
href="#glo:synthetic lethal">synthetic lethality</a> would appear in (continuous normally distributed) <a 
href="#glo:gene expression">gene expression</a> data. Where presented (in Section <a 
href="#x1-760001">3.2.1<!--tex4ht:ref: methods:SL_Model --></a>), the assumptions of the model are stated clearly. The model allows sampling from a multivariate normal distribution (using the <span class="cmtt-12">mvtnorm </span>R package (<a 
href="#XGenz2009">Genz and Bretz</a>, <a 
href="#XGenz2009">2009</a>; <a 
href="#Xmvtnorm">Genz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xmvtnorm">2016</a>)) to generate simulated data with known underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners (detailed in Section <a 
href="#x1-770002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a>). We can test whether statistical procedures, including those developed in this thesis (presented in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>), are capable of detecting <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners within the simulated data. This multivariate normal simulation procedure also enables the inclusion of correlation structure which is either given as correlated blocks of genes or derived from <a 
href="#glo:graph">pathway</a> structures (as detailed in Section <a 
href="#x1-860002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>).
</p><!--l. 177--><p class="indent" >
When this multivariate normal distribution is sampled once and the procedure to add known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners is performed, it generates a simulated dataset. Performing this simulation procedure and testing with a <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection procedure iteratively, these simulations can be used to assess the statistical performance of the detection procedure. The number of iterations (<span class="cmtt-12">Reps</span>) will be given for each simulation result. Typically, these are performed 1000 or 10,000 times depending on computational feasibility of doing so on larger datasets.
</p><!--l. 179--><p class="indent" >
<p>Several measures of statistical performance were used to assess the simulations. The following measures used the final classification of the detection procedure, statistical significance for <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>, significance and directional criteria met for <a 
href="#glo:SLIPT">SLIPT</a> (in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>), and an arbitrary threshold: <span class="cmmi-12">&lt; </span><span class="cmsy-10x-x-120">-</span>0<span class="cmmi-12">.</span>2 and <span class="cmmi-12">&gt; </span>+0<span class="cmmi-12">.</span>2 for negative correlation and correlation respectively. Sensitivity (or “true positive rate”) was measured as the proportion of known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners predicted to be <a 
href="#glo:synthetic lethal">synthetic lethal</a>. Specificity (or “true negative rate”) was measured as the proportion of known non-synthetic lethal partners predicted not to be <a 
href="#glo:synthetic lethal">synthetic lethal</a>. The “false positive rate” was measured here as the proportion of known non-synthetic lethal partners out of all putative partners predicted by the detection procedure. Statistical “accuracy” is the proportion of</p>
true predictions for a detection procedure, which is both the correctly predicted known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners and correctly negative known non-synthetic lethal partners. <a 
 id="x1-61001r57"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">2.3.5.1 </span> <a 
 id="x1-620001"></a>Receiver Operating Characteristic Curves
</h5>
<!--l. 182--><p class="noindent" >
A more general procedure to measure the statistical performance of a simulation is the <a 
href="#glo:ROC">receiver operating characteristic (ROC)</a> curve which does not assume a threshold for classification of <a 
href="#glo:synthetic lethal">synthetic lethality</a> but demonstrates the achievable range of sensitivity and specificity for a model (<a 
href="#XAkobeng2007">Akobeng</a>, <a 
href="#XAkobeng2007">2007</a>; <a 
href="#XFawcett2006">Fawcett</a>, <a 
href="#XFawcett2006">2006</a>; <a 
href="#XZweig1993">Zweig and Campbell</a>, <a 
href="#XZweig1993">1993</a>). These curves (implemented with the <span class="cmtt-12">ROCR </span>R package (<a 
href="#XROCR">Sing <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XROCR">2005</a>)) plot the true positive rate (sensitivity) against the false positive rate (1<span class="cmsy-10x-x-120">-</span>specificity) as the prediction threshold is varied. An ideal detection method will have a true positive rate of 1 and a false positive rate of 0, hence the Area Under the <a 
href="#glo:ROC">ROC</a> curve (AUC or <a 
href="#glo:AUROC">area under <a 
href="#glo:ROC">receiver operating characteristic</a> (AUROC)</a>) is a measure of statistical performance for a detection procedure accounting for this trade-off. <a 
href="#glo:AUROC">AUROC</a> values typically range from 0.5 (the value expected by random chance) to 1 for an optimal detection method, however it is possible for an <a 
href="#glo:AUROC">AUROC</a> below 0.5 for a poor detection method that performs worse than random chance. In cancer biology, it has been suggested that an <a 
href="#glo:AUROC">AUROC</a> of approximately 0<span class="cmmi-12">.</span>8 is a predictive biomarker suitable for publication (<a 
href="#XHajian-Tilaki2013">Hajian-Tilaki</a>, <a 
href="#XHajian-Tilaki2013">2013</a>) but predictors with lower <a 
href="#glo:AUROC">AUROC</a> values may still be informative depending on the context. In this thesis, the <a 
href="#glo:AUROC">AUROC</a> values varied widely across simulation parameters and were primarily used for comparisons across these parameters, although they can also be used to refine thresholds for optimal classification. <a 
 id="x1-62001r75"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.3.6 </span> <a 
 id="x1-630006"></a>Resampling Analysis
</h4>
<!--l. 185--><p class="noindent" >
<p>Resampling analyses (e.g., “permutation” analysis) are used to statistically test the significance of an observation without assuming the underlying distribution of expected test statistics (<a 
href="#XCollingridge2013">Collingridge</a>, <a 
href="#XCollingridge2013">2013</a>). Instead these are derived from randomly shuffling test statistics or randomly sampling predicted candidates. For the purposes of this thesis, this involved randomly sampling genes from those tested to be analysed as putative <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates. This was performed both for testing the significance of <a 
href="#glo:pathway">pathway</a> composition in the intersection with experimental gene candidates (Section <a 
href="#x1-550004">2.2.4<!--tex4ht:ref: methods:venn_analysis --></a>) and for</p>
assessing the significance of <a 
href="#glo:graph">pathway</a> structure among <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates (Section <a 
href="#x1-850001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>).
</p><!--l. 187--><p class="indent" >
These were analysed to compare the observed <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes against values derived from randomly sampling the same number of genes as were observed to be <a 
href="#glo:synthetic lethal">synthetic lethal</a> from among the genes tested. Sampling iteratively across many resampling procedures, these resampling-based values form a null distribution that could be expected if the null hypothesis were true. Thus the proportion of resampling-based values across these iterations that are greater than or equal to that observed, forms an empirically derived p-value to test significance.
</p><!--l. 189--><p class="indent" >
Resampling was performed for comparison (in Section <a 
href="#x1-550004">2.2.4<!--tex4ht:ref: methods:venn_analysis --></a>) with fixed experimental screen candidates (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) both resampling the number of genes overlapping with the screen candidates and test statistics for <a 
href="#glo:pathway">pathway</a> enrichment. Resampling analysis was also applied to <a 
href="#glo:shortest path">shortest paths</a> and network metrics (in Section <a 
href="#x1-850001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>) to test significance of directional relationships between <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate genes within <a 
href="#glo:graph">pathway</a> structures.
</p><!--l. 191--><p class="indent" >
The number of iterations determines the accuracy of these p-values. For <a 
href="#glo:pathway">pathway</a> composition (in Section <a 
href="#x1-550004">2.2.4<!--tex4ht:ref: methods:venn_analysis --></a>), a million iterations were performed using high performance computing (as detailed in Section <a 
href="#x1-720003">2.5.3<!--tex4ht:ref: methods:HPC --></a>) to provide sufficient accuracy after adjusting for multiple comparisons across <a 
href="#glo:pathway">pathways</a>. For the purposes of network analysis (in Section <a 
href="#x1-850001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>), a thousand iterations were sufficient to reject the null hypothesis for the majority of <a 
href="#glo:pathway">pathways</a> tested before adjusting for multiple comparisons, and thus further iterations were not performed. <a 
 id="x1-63001r70"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">2.4 </span> <a 
 id="x1-640004"></a>Pathway Structure Methods
</h3>
<a 
 id="x1-64001r77"></a>
<h4 class="subsectionHead">
<span class="titlemark">2.4.1 </span> <a 
 id="x1-650001"></a>Network and Graph Analysis
</h4>
<!--l. 197--><p class="noindent" >
<p>Networks are important in considering the structure of relationships in molecular biology, including gene regulation, kinase cellular signalling, and metabolic <a 
href="#glo:pathway">pathways</a> (<a 
href="#XBarabasi2004">Barabási and Oltvai</a>, <a 
href="#XBarabasi2004">2004</a>). Network theory is an interdisciplinary field which combines the approaches of computer science with the metrics and fundamental principles of graph theory, an area of pure mathematics dealing with relationships between sets of discrete elements. The vast amounts of molecular and cellular data from high-throughput technologies have enabled the application of network-based and <a 
href="#glo:genome">genomes</a>-wide <a 
href="#glo:bioinformatics">bioinformatics</a> analysis to examine the complexity of a cell at the molecular level and understand aberrations in cancer.</p>
This thesis uses various metrics and analysis procedures developed in Graph and Network theory to analyse <a 
href="#glo:graph">graph</a> structure of biological <a 
href="#glo:pathway">pathways</a>. Where feasible, these have been implemented using the <span class="cmtt-12">igraph </span>R package with such procedures described below (<a 
href="#Xigraph">Csardi and Nepusz</a>, <a 
href="#Xigraph">2006</a>). Custom R functions were used to perform more complex analysis and visualisation of iGraph data (as described in Section <a 
href="#x1-900003">3.5.3<!--tex4ht:ref: methods:igraph_extensions --></a>).
</p><!--l. 199--><p class="indent" >
Graph theory is a branch of pure mathematics which deals with the properties of sets of discrete objects (referred to as a ‘node’ or ‘vertex‘) with some pairs are joined (by a ‘link’ or an ‘edge‘). While a seemingly reductionist abstraction to mathematically study relationships, graph theory has applications in a wide range of fields, including the life sciences. Network theory is the sub-discipline of graph theory that deals with networks, which has become popular due to the vast potential for applications of networks (<a 
href="#XvanSteen2010">van Steen</a>, <a 
href="#XvanSteen2010">2010</a>).
</p><!--l. 201--><p class="indent" >
Applications vary depending on the situation modelled, particularly in how the <a 
href="#glo:edge">edges</a> between <a 
href="#glo:vertex">vertices</a> are defined, whether they are directed or weighted, and whether multiple redundant <a 
href="#glo:edge">edges</a> between a pair of <a 
href="#glo:vertex">vertices</a> (referred to as ‘parallel <a 
href="#glo:edge">edges</a>‘) or <a 
href="#glo:edge">edges</a> connecting a <a 
href="#glo:vertex">vertex</a> to itself (referred to as ‘loops‘) are permitted in the model. Networks are defined such that the <a 
href="#glo:edge">edges</a> represent a relationship between the <a 
href="#glo:vertex">vertices</a> and may be directed, weighted, or contain parallel <a 
href="#glo:edge">edges</a> or loops depending on the application (<a 
href="#XvanSteen2010">van Steen</a>, <a 
href="#XvanSteen2010">2010</a>). Unless otherwise stated, <a 
href="#glo:graph">graph</a> structures and networks in this thesis will be unweighted and have no parallel <a 
href="#glo:edge">edges</a> or loops. Where a directional relationship is known or modelled, it will be represented with a directed <a 
href="#glo:edge">edge</a> in a directed graph. <a 
 id="x1-65001r79"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.4.2 </span> <a 
 id="x1-660002"></a>Sourcing Graph Structure Data
</h4>
<!--l. 204--><p class="noindent" >
<p>Pathway Commons interaction data was sourced using the <a 
href="#glo:BioPAX">Biological PAthway eXchange (BioPAX)</a> with the paxtools-4.3.0 Java application on October 6<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup> 2015 (<a 
href="#XPathwayCommons">Cerami <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XPathwayCommons">2011</a>; <a 
href="#Xpaxtools">Demir <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xpaxtools">2013</a>). This utility was used to import‘sif’ format interaction data into R (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>) and extract the human Reactome (version 52) dataset of interactions was imported (<a 
href="#XReactome">Croft <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XReactome">2014</a>), matching those used for <a 
href="#glo:pathway">pathway</a> enrichment analysis. These interactions were used to construct an adjacency matrix for the Reactome network and subnetworks corresponding to each relevant biological <a 
href="#glo:pathway">pathway</a>. <a 
 id="x1-66001r80"></a></p>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.4.3 </span> <a 
 id="x1-670003"></a>Constructing Pathway Subgraphs
</h4>
<!--l. 207--><p class="noindent" >
Subgraphs for each relevant <a 
href="#glo:pathway">pathway</a> were constructed by matching the <a 
href="#glo:vertex">nodes</a> in the complete Reactome network to the <a 
href="#glo:pathway">pathway</a> gene sets (as derived in Section <a 
href="#x1-500002">2.1.1.2<!--tex4ht:ref: methods:gene_set --></a>). A subgraph with adjacent <a 
href="#glo:vertex">nodes</a> was constructed by adding <a 
href="#glo:vertex">nodes</a> which have an <a 
href="#glo:edge">edge</a> with a gene in the <a 
href="#glo:pathway">pathway</a> gene set. The <a 
href="#glo:pathway">pathways</a> these adjacent <a 
href="#glo:vertex">nodes</a> belong to were added to form a “meta-<a 
href="#glo:pathway">pathway</a>” to account for the possibility for <a 
href="#glo:vertex">nodes</a> within the <a 
href="#glo:pathway">pathway</a> being linked by the surrounding <a 
href="#glo:graph">graph</a> structure. <a 
 id="x1-67001r81"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.4.4 </span> <a 
 id="x1-680004"></a>Network Analysis Metrics
</h4>
<!--l. 210--><p class="noindent" >
The existing network analysis measures applied in this thesis (as described below) used an implementation in the <span class="cmtt-12">igraph </span>R package to compute vertex degree, shortest paths, and centrality (<a 
href="#Xigraph">Csardi and Nepusz</a>, <a 
href="#Xigraph">2006</a>). Additionally, custom features were developed for analysis of iGraph objects in R and released as <span class="cmtt-12">igraph.extensions </span>(as described in Section <a 
href="#x1-900003">3.5.3<!--tex4ht:ref: methods:igraph_extensions --></a>).
</p><!--l. 212--><p class="indent" >
Vertex degree is the number of <a 
href="#glo:edge">edges</a> a <a 
href="#glo:vertex">node</a> has and is a fundamental measure of the importance and connectivity of a network (<a 
href="#XvanSteen2010">van Steen</a>, <a 
href="#XvanSteen2010">2010</a>). More connected <a 
href="#glo:vertex">nodes</a>, such as network hubs, will have a higher <a 
href="#glo:vertex">vertex</a> degree relative to other <a 
href="#glo:vertex">nodes</a>. For the purposes of this thesis, <a 
href="#glo:vertex">vertex</a> degree ignored <a 
href="#glo:edge">edge</a> direction with loops (edges with itself) and double <a 
href="#glo:edge">edges</a> to the same <a 
href="#glo:vertex">node</a> excluded.
</p><!--l. 214--><p class="indent" >
A fundamental concept in network analysis is a “<a 
href="#glo:shortest path">shortest path</a>”, that is the shortest route via <a 
href="#glo:edge">edges</a> between any two particular <a 
href="#glo:vertex">nodes</a> in a network. These are computed by Dijstra’s algorithm (<a 
href="#XDijkstra1959">Dijkstra</a>, <a 
href="#XDijkstra1959">1959</a>) in the <span class="cmtt-12">igraph </span>R package (<a 
href="#Xigraph">Csardi and Nepusz</a>, <a 
href="#Xigraph">2006</a>). Where applicable paths will only use directed <a 
href="#glo:edge">edges</a> in a particular direction. Shortests paths are a useful measure of how close <a 
href="#glo:vertex">nodes</a> are in a network. This is used to compute <a 
href="#glo:information centrality">information centrality</a>, and for further analysis of <a 
href="#glo:graph">pathway</a> structure (as described in Section <a 
href="#x1-840001">3.4.1<!--tex4ht:ref: methods:pathway_str --></a>).
</p><!--l. 216--><p class="indent" >
<p>Network <a 
href="#glo:centrality">centrality</a> is an alternative measure of the importance or influence of a <a 
href="#glo:vertex">node</a> to the <a 
href="#glo:graph">graph</a> structure (<a 
href="#XBorgatti2005">Borgatti</a>, <a 
href="#XBorgatti2005">2005</a>). Various strategies are used to derive centrality, typically based on how connected the <a 
href="#glo:vertex">node</a> is or the impact of <a 
href="#glo:vertex">node</a> removal on the connectivity of the network. One of the most notable is the “<a 
href="#glo:PageRank centrality">PageRank</a>” algorithm, a refinement of eigenvector <a 
href="#glo:centrality">centrality</a> based on the eigenvectors of the adjacency matrix (<a 
href="#XBrin1998">Brin and Page</a>, <a 
href="#XBrin1998">1998</a>). This is implemented in the <span class="cmtt-12">igraph </span>R package (<a 
href="#Xigraph">Csardi and Nepusz</a>, <a 
href="#Xigraph">2006</a>).</p>
</p><!--l. 218--><p class="indent" >
Another network <a 
href="#glo:centrality">centrality</a> measure that has been previously applied to biological protein interaction networks (<a 
href="#XKranthi2013">Kranthi <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XKranthi2013">2013</a>) is the “<a 
href="#glo:information centrality">information centrality</a>”. The <a 
href="#glo:information centrality">information centrality</a> of a <a 
href="#glo:vertex">node</a> is the relative impact on efficiency (transmission of information via <a 
href="#glo:shortest path">shortest paths</a>) of the network when the <a 
href="#glo:vertex">node</a> is removed. Te <a 
href="#glo:centrality">centrality</a> (<span class="cmmi-12">C</span>) (<a 
href="#XKranthi2013">Kranthi <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XKranthi2013">2013</a>) for <a 
href="#glo:vertex">node</a> <span class="cmmi-12">n </span>in graph <span class="cmmi-12">G </span>is defined as:
</p>
<center class="math-display" >
<img 
src="thesis4x.png" alt="                   &#x2032;
Cn =  E-(G)---E-(G-)
          E (G )  " class="math-display"  />
</center>
where <span class="cmmi-12">G</span><span class="cmsy-10x-x-120">′ </span>is the subgraph with the <a 
href="#glo:vertex">node</a> removed and <span class="cmmi-12">E </span>is the efficiency (<a 
href="#XLatora2001">Latora and Marchiori</a>, <a 
href="#XLatora2001">2001</a>), derived from <a 
href="#glo:shortest path">shortest paths</a> (<span class="cmmi-12">d</span><sub><span class="cmmi-8">ij</span></sub> between <a 
href="#glo:vertex">nodes</a> <span class="cmmi-12">i </span>and <span class="cmmi-12">j</span>).
<center class="math-display" >
<img 
src="thesis5x.png" alt="            2      &#x2211;    1
E (G) =  ---------     ---
         N (N  - 1 i&#x003C;j&#x2208;Gdij  " class="math-display"  />
</center> 
The efficiency of the network can be derived from <a 
href="#glo:shortest path">shortest paths</a> implemented in the <span class="cmtt-12">igraph </span>R package and the iterative network <a 
href="#glo:centrality">centrality</a> computation of each <a 
href="#glo:vertex">node</a> has been released as an R package (<span class="cmtt-12">info.centrality</span>) and included in the <span class="cmtt-12">igraph.extensions</span> package. <a 
 id="x1-68001r78"></a>
<h3 class="sectionHead">
<span class="titlemark">2.5 </span> <a 
 id="x1-690005"></a>Implementation
</h3>
<p><a 
 id="x1-69001r82"></a></p>
<h4 class="subsectionHead">
<span class="titlemark">2.5.1 </span> <a 
 id="x1-700001"></a>Computational Resources and Linux Utilities
</h4>
<!--l. 229--><p class="noindent" >
Several computers were used to process and store data during this thesis (as summarised in Table <a 
href="#x1-700012">2.2<!--tex4ht:ref: tab:computers --></a>), running different versions of Linux operating systems, including a personal laptop computer, laboratory desktop machine, departmental server, and the New Zealand eScience Infrastructure Intel Pan high-performance computing cluster (a supercomputer based at the University of Auckland). Each of these systems support a 64-bit architecture. Current workflows on local machines use Elementary OS (based on the Ubuntu versions given in Table <a 
href="#x1-700012">2.2<!--tex4ht:ref: tab:computers --></a>) and the ZSH shell. However, Ubuntu OS and the <a 
href="#glo:Bash">Bourne Again SHell (Bash)</a> were used at the inception of this project and <a 
href="#glo:Bash">Bash</a> continues to be used for running scripts. Various Linux applications and command-line utilities were used on these machines (as summarised in Table <a 
href="#x1-700023">2.3<!--tex4ht:ref: tab:computers_linux --></a>). As such, the workflows developed in this project should be backwards-compatible with Ubuntu Linux (and other derivatives). The majority of novel methodology and implementations were performed in R which is a cross-platform language, packages developed in R will be available for users of Linux, Mac, and Windows machines.
</p>
<div class="table">
                                                                                    

<!--l. 232--><p class="indent" >
<a 
 id="x1-700012"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 2.2: </span><span class="content">Computers used during thesis</span>
</div><!--tex4ht:label?: x1-700012 -->
<!--tex4ht:inline--><div class="tabular"><table id="TBL-31" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-31-1g"><col 
id="TBL-31-1" /></colgroup><colgroup id="TBL-31-2g"><col 
id="TBL-31-2" /></colgroup><colgroup id="TBL-31-3g"><col 
id="TBL-31-3" /></colgroup><colgroup id="TBL-31-4g"><col 
id="TBL-31-4" /></colgroup><colgroup id="TBL-31-5g"><col 
id="TBL-31-5" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-31-1-"><td  style="white-space:nowrap; text-align:right;" id="TBL-31-1-1"  
class="td11">                    <div class="multicolumn"  style="white-space:nowrap; text-align:right;"></div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-1-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Viao Laptop</span>
</div>                                                                       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-1-3"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Lab Machine</span>
</div>                                                                       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-1-4"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Biochem Server</span>
</div>                                                                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-1-5"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">NeSI Pan Cluster</span>
</div>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-31-2-"><td  style="white-space:nowrap; text-align:right;" id="TBL-31-2-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-31-3-"><td  style="white-space:nowrap; text-align:right;" id="TBL-31-3-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:right;">
Operating System (OS)
</div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-3-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-32" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-32-1g"><col 
id="TBL-32-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-32-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-32-1-1"  
class="td00">
Elementary OS
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-32-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-32-2-1"  
class="td00">
Freya 0.3.2
</td></tr></table>                                                                                        </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-3-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-33" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-33-1g"><col 
id="TBL-33-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-33-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-33-1-1"  
class="td00">
Elementary OS
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-33-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-33-2-1"  
class="td00">
Loki 0.4
</td></tr></table>                                                                                        </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-3-4"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-34" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-34-1g"><col 
id="TBL-34-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-34-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-34-1-1"  
class="td00">
Red Hat Enterprise
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-34-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-34-2-1"  
class="td00">
Maipo 7.2
</td></tr></table>                                                                                   </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-3-5"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-35" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-35-1g"><col 
id="TBL-35-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-35-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-35-1-1"  
class="td00">
Cent OS
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-35-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-35-2-1"  
class="td00">
Final 6.4
</td></tr></table>                                                                                                 </div>  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-31-4-"><td  style="white-space:nowrap; text-align:right;" id="TBL-31-4-1"  
class="td11">         <div class="multicolumn"  style="white-space:nowrap; text-align:right;">
Upstream OS
</div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-4-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-36" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-36-1g"><col 
id="TBL-36-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-36-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-36-1-1"  
class="td00">
Ubuntu LTS
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-36-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-36-2-1"  
class="td00">
Trusty 14.04
</td></tr></table>                                                                                            </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-4-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-37" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-37-1g"><col 
id="TBL-37-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-37-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-37-1-1"  
class="td00">
Ubuntu LTS
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-37-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-37-2-1"  
class="td00">
Xenial 16.04
</td></tr></table>                                                                                            </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-4-4"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-4-5"  
class="td11">                                                                         </td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-31-5-"><td  style="white-space:nowrap; text-align:right;" id="TBL-31-5-1"  
class="td11">          <div class="multicolumn"  style="white-space:nowrap; text-align:right;">
Linux Kernel
</div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-5-2"  
class="td11"> 
3.19.0-65-generic
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-5-3"  
class="td11"> 
4.4.0-36-generic
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-5-4"  
class="td11"> 
3.10.0-327.36.2.el7.x86_64
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-5-5"  
class="td11"> 
2.6.32-504.16.2.el6.x86_64
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-31-6-"><td  style="white-space:nowrap; text-align:right;" id="TBL-31-6-1"  
class="td11">           <div class="multicolumn"  style="white-space:nowrap; text-align:right;">
Shell: Bash
</div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-6-2"  
class="td11"> 
4.3.11(1)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-6-3"  
class="td11"> 
4.3.46(1)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-6-4"  
class="td11"> 
4.2.46(1)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-6-5"  
class="td11"> 
4.2.1(1)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-31-7-"><td  style="white-space:nowrap; text-align:right;" id="TBL-31-7-1"  
class="td11">             <div class="multicolumn"  style="white-space:nowrap; text-align:right;">
Shell: zsh
</div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-7-2"  
class="td11"> 
5.0.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-7-3"  
class="td11"> 
5.1.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-7-4"  
class="td11"> 
5.0.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-31-7-5"  
class="td11"> 
5.2
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-31-8-"><td  style="white-space:nowrap; text-align:right;" id="TBL-31-8-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-31-9-"><td  style="white-space:nowrap; text-align:right;" id="TBL-31-9-1"  
class="td11">                      </td></tr></table>                                                                                                                                                                                                                                                                                                                                                                   </div>
                                                                                    

</div><hr class="endfloat" />
</div>
<div class="table">
                                                                                    

<!--l. 262--><p class="indent" >
<a 
 id="x1-700023"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 2.3: </span><span class="content">Linux utilities and applications used during thesis</span>
</div><!--tex4ht:label?: x1-700023 -->
<!--tex4ht:inline--><div class="tabular"><table id="TBL-38" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-38-1g"><col 
id="TBL-38-1" /><col 
id="TBL-38-2" /></colgroup><colgroup id="TBL-38-3g"><col 
id="TBL-38-3" /></colgroup><colgroup id="TBL-38-4g"><col 
id="TBL-38-4" /></colgroup><colgroup id="TBL-38-5g"><col 
id="TBL-38-5" /></colgroup><colgroup id="TBL-38-6g"><col 
id="TBL-38-6" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-38-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-1-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"></div>                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-1-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"></div>                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-1-3"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Viao Laptop</span>
</div>                                                                       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-1-4"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Lab Machine</span>
</div>                                                                       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-1-5"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Biochem Server</span>
</div>                                                                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-1-6"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">NeSI Pan Cluster</span>
</div>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-38-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-2-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-38-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-3-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"></div>                     </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-3-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
OS
</div>                      </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-3-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-39" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-39-1g"><col 
id="TBL-39-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-39-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-39-1-1"  
class="td00">
Elementary OS
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-39-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-39-2-1"  
class="td00">
Freya 0.3.2
</td></tr></table>                                                                                        </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-3-4"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-40" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-40-1g"><col 
id="TBL-40-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-40-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-40-1-1"  
class="td00">
Elementary OS
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-40-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-40-2-1"  
class="td00">
Loki 0.4
</td></tr></table>                                                                                        </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-3-5"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-41" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-41-1g"><col 
id="TBL-41-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-41-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-41-1-1"  
class="td00">
Red Hat Enterprise
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-41-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-41-2-1"  
class="td00">
Maipo 7.2
</td></tr></table>                                                                                   </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-3-6"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-42" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-42-1g"><col 
id="TBL-42-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-42-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-42-1-1"  
class="td00">
Cent OS
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-42-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-42-2-1"  
class="td00">
Final 6.4
</td></tr></table>                                                                                                 </div>  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-4-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"></div>                     </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-4-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
Linux Kernel
</div>            </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-4-3"  
class="td11"> 
3.19.0-65-generic
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-4-4"  
class="td11"> 
4.4.0-36-generic
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-4-5"  
class="td11"> 
3.10.0-327.36.2.el7.x86_64
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-4-6"  
class="td11"> 
2.6.32-504.16.2.el6.x86_64
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-38-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-5-1"  
class="td11">  
Scripting
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-5-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
Shell Bash
</div>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-5-3"  
class="td11"> 
4.3.11(1)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-5-4"  
class="td11"> 
4.3.46(1)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-5-5"  
class="td11"> 
4.2.46(1)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-5-6"  
class="td11"> 
4.2.1(1)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-6-1"  
class="td11">                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-6-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
Shell zsh
</div>                </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-6-3"  
class="td11"> 
5.0.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-6-4"  
class="td11"> 
5.1.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-6-5"  
class="td11"> 
5.0.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-6-6"  
class="td11"> 
5.2
</td></tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-38-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-7-1"  
class="td11"> 
Programming
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-7-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
Python
</div></td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-7-3"  
class="td11"> 
2.7.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-7-4"  
class="td11"> 
2.7.12
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-7-5"  
class="td11"> 
2.7.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-7-6"  
class="td11"></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-8-1"  
class="td11">                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-8-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
Java
</div>                    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-8-3"  
class="td11"> 
1.8.0_101
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-8-4"  
class="td11"> 
9-ea
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-8-5"  
class="td11"> 
1.8.0_101
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-8-6"  
class="td11">                                                                         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-9-1"  
class="td11">                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-9-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
C++
</div>                    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-9-3"  
class="td11"> 
4.8.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-9-4"  
class="td11"> 
5.4.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-9-5"  
class="td11"> 
4.8.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-9-6"  
class="td11"> 
4.4.7
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-38-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-10-1"  
class="td11">  
Text Editor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-10-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
nano
</div>                    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-10-3"  
class="td11"> 
2.2.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-10-4"  
class="td11"> 
2.5.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-10-5"  
class="td11"> 
2.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-10-6"  
class="td11"> 
2.0.9
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-11-1"  
class="td11">                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-11-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
kile (<span class="LATEX">L<span class="A">A</span><span class="TEX">T<span class="E">E</span>X</span></span>)
</div>           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-11-3"  
class="td11"> 
2.1.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-11-4"  
class="td11"> 
2.1.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-11-5"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-11-6"  
class="td11">                                                                         </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-38-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-12-1"  
class="td11">  
Version Control
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-12-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
git
</div>                      </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-12-3"  
class="td11"> 
1.9.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-12-4"  
class="td11"> 
2.11.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-12-5"  
class="td11"> 
1.7.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-12-6"  
class="td11"> 
1.8.3.1
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-38-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-13-1"  
class="td11">  
Shell Utilities
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-13-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
sed
</div>                      </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-13-3"  
class="td11"> 
4.4.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-13-4"  
class="td11"> 
4.4.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-13-5"  
class="td11"> 
4.4.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-13-6"  
class="td11"> 
4.4.1
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-38-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-14-1"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-14-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
grep
</div></td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-14-3"  
class="td11"> 
2.16-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-14-4"  
class="td11"> 
2.25-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-14-5"  
class="td11"> 
2.20
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-14-6"  
class="td11"> 
2.6.3
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-15-1"  
class="td11">                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-15-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
nohup
</div>                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-15-3"  
class="td11"> 
8.21
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-15-4"  
class="td11"> 
8.25
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-15-5"  
class="td11"> 
8.22
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-15-6"  
class="td11"> 
8.4
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-38-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-16-1"  
class="td11"> 
Typesetting
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-16-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="TEX">T<span class="E">E</span>X</span>
</div>                    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-16-3"  
class="td11"> 
3.1415926
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-16-4"  
class="td11"> 
3.14159265
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-16-5"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-16-6"  
class="td11">                                                                         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-17-1"  
class="td11">                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-17-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
TexLive (<span class="LATEX">L<span class="A">A</span><span class="TEX">T<span class="E">E</span>X</span></span>)
</div>       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-17-3"  
class="td11"> 
2013
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-17-4"  
class="td11"> 
2015
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-17-5"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-17-6"  
class="td11">                                                                         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-18-1"  
class="td11">                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-18-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
PDF<span class="TEX">T<span class="E">E</span>X</span>
</div>               </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-18-3"  
class="td11"> 
2.5-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-18-4"  
class="td11"> 
2.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-18-5"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-18-6"  
class="td11">                                                                         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-19-1"  
class="td11">                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-19-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
pandoc
</div>                  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-19-3"  
class="td11"> 
1.12.2.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-19-4"  
class="td11"> 
1.16.0.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-19-5"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-19-6"  
class="td11">                                                                         </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-38-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-20-1"  
class="td11"> 
Remote Computing
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-20-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
Slurm scheduler
</div>         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-20-3"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-20-4"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-20-5"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-20-6"  
class="td11"> 
16.05.6
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-21-1"  
class="td11">                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-21-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
OpenSSH
</div>               </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-21-3"  
class="td11"> 
7.2p2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-21-4"  
class="td11"> 
7.2p2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-21-5"  
class="td11"> 
6.6.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-21-6"  
class="td11"> 
5.3p1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-22-1"  
class="td11">                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-22-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
OpenSSL
</div>                </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-22-3"  
class="td11"> 
1.0.2g
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-22-4"  
class="td11"> 
1.0.2g
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-22-5"  
class="td11"> 
1.0.01e-fips
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-22-6"  
class="td11"> 
1.0.01e-fips
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-23-1"  
class="td11">                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-23-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
rsync
</div>                    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-23-3"  
class="td11"> 
3.1.0p31
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-23-4"  
class="td11"> 
3.1.1p31
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-23-5"  
class="td11"> 
3.0.9p30
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-23-6"  
class="td11">                                                                         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-24-1"  
class="td11">                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-24-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
Globus Online Transfer
</div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-24-3"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-24-4"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-24-5"  
class="td11"> 
3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-24-6"  
class="td11"> 
3.1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-25-1"  
class="td11">                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-25-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
Cisco AnyConnect VPN
</div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-25-3"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-25-4"  
class="td11"> 
3.1.05170
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-25-5"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-25-6"  
class="td11">                                                                         </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-38-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-26-1"  
class="td11"> 
Image Processing
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-26-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
Inkscape
</div>                </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-26-3"  
class="td11"> 
0.48.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-26-4"  
class="td11"> 
0.91
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-26-5"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-26-6"  
class="td11">                                                                         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-27-1"  
class="td11">                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-27-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
GIMP
</div>                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-27-3"  
class="td11"> 
2.8.10
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-27-4"  
class="td11"> 
2.8.16
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-27-5"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-27-6"  
class="td11">                                                                         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-38-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-28-1"  
class="td11">                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-28-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
ImageMagick
</div>            </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-28-3"  
class="td11"> 
6.7.7.10-6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-28-4"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-28-5"  
class="td11">                                                                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-38-28-6"  
class="td11">                                                                         </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-38-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-38-29-1"  
class="td11">                   </td></tr></table>                                                                                                                                                                                                                                                                                                                                                                                               </div>
                                                                                    

</div><hr class="endfloat" />
</div>
<a 
 id="x1-70003r84"></a>
<h4 class="subsectionHead">
<span class="titlemark">2.5.2 </span> <a 
 id="x1-710002"></a>R Language and Packages
</h4>
<!--l. 360--><p class="noindent" >
The R programming language has been used for the majority of this thesis. Current R installations across the machines used are given in Table <a 
href="#x1-710014">2.4<!--tex4ht:ref: tab:computers_r --></a>. Local machines currently run the latest version of the R (at the time of writing) and remote machines run the versions and modules as managed by the system administrator.
</p><!--l. 362--><p class="indent" >
Various scripts and packages in this thesis were developed or run in previous versions of RStudio and R but these run without error in the current version of R (and the older versions on remote machines). The R packages which were used throughout this thesis (as detailed in Table <a 
href="#x1-71002r5">2.5<!--tex4ht:ref: tab:computers_r_packages --></a> with versions specified) were installed from the <a 
href="#glo:CRAN">comprehensive R archive network</a> (<a 
href="#glo:CRAN">CRAN</a>) (<a 
href="#XCRAN">CRAN</a>, <a 
href="#XCRAN">2017</a>), Bioconductor (<a 
href="#XGentleman2004">Gentleman <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGentleman2004">2004</a>, version 3.4; BiocInstaller 1.24.0), or GitHub (<a 
href="https://github.com/" class="url" ><span class="cmtt-12">https://github.com/</span></a>). These packages were not updated when they would change the functionality of scripts or functions in packages, in particular imported data from annotation packages (used to define gene sets) have been saved as local files to continue using stable versions of these <a 
href="#glo:pathway">pathway</a> data (across machines).
</p><!--l. 364--><p class="indent" >
This is a summary of the key packages which (in addition to their dependencies) have been used throughout this project. Where a package implementation has been central to the methods applied, they are described in more detail in the relevant Section. A full table of packages used in this thesis can be found in Appendix <a 
href="#x1-170000B">B<!--tex4ht:ref: appendix:software --></a> (Table <a 
href="#x1-170001r1">B.1<!--tex4ht:ref: tab:computers_r_packages_full --></a>). The R packages developed during this thesis are given in Table <a 
href="#x1-710046">2.6<!--tex4ht:ref: tab:computers_r_packages_dev --></a> with the relevant Sections describing their implementation and use where appropriate, in addition to further details on these functions in Section <a 
href="#x1-870005">3.5<!--tex4ht:ref: methods:r_packages --></a>.
</p>
<div class="table">
                                                                                    

<!--l. 366--><p class="indent" >
<a 
 id="x1-710014"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 2.4: </span><span class="content">R installations used during thesis</span>
</div><!--tex4ht:label?: x1-710014 -->
<!--tex4ht:inline--><div class="tabular"><table id="TBL-43" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-43-1g"><col 
id="TBL-43-1" /><col 
id="TBL-43-2" /></colgroup><colgroup id="TBL-43-3g"><col 
id="TBL-43-3" /></colgroup><colgroup id="TBL-43-4g"><col 
id="TBL-43-4" /></colgroup><colgroup id="TBL-43-5g"><col 
id="TBL-43-5" /></colgroup><colgroup id="TBL-43-6g"><col 
id="TBL-43-6" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-43-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-43-1-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"></div>                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-1-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"></div>         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-1-3"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Viao Laptop</span>
</div>                                                                       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-1-4"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Lab Machine</span>
</div>                                                                       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-1-5"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Biochem Server</span>
</div>                                                                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-1-6"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">NeSI Pan Cluster</span>
</div>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-43-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-43-2-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-43-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-43-3-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"></div>               </td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-3-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
OS
</div>      </td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-3-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-44" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-44-1g"><col 
id="TBL-44-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-44-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-44-1-1"  
class="td00">
Elementary OS
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-44-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-44-2-1"  
class="td00">
Freya 0.3.2
</td></tr></table>                                                                                        </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-3-4"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-45" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-45-1g"><col 
id="TBL-45-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-45-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-45-1-1"  
class="td00">
Elementary OS
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-45-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-45-2-1"  
class="td00">
Loki 0.4
</td></tr></table>                                                                                        </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-3-5"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-46" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-46-1g"><col 
id="TBL-46-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-46-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-46-1-1"  
class="td00">
Red Hat Enterprise
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-46-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-46-2-1"  
class="td00">
Maipo 7.2
</td></tr></table>                                                                                   </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-3-6"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-47" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-47-1g"><col 
id="TBL-47-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-47-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-47-1-1"  
class="td00">
Cent OS
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-47-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-47-2-1"  
class="td00">
Final 6.4
</td></tr></table>                                                                                                 </div>  </td>

</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-43-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-43-4-1"  
class="td11"> 
Programming
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-4-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
R
</div>       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-4-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-4-4"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-4-5"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-4-6"  
class="td11"> 
3.3.0-intel (module)
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-43-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-43-5-1"  
class="td11"> 
Development
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-5-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
RStudio
</div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-5-3"  
class="td11"> 
1.0.136
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-5-4"  
class="td11"> 
1.0.136
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-5-5"  
class="td11"> 
1.0.136 (server)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-43-5-6"  
class="td11">                                                                         </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-43-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-43-6-1"  
class="td11">              </td></tr></table>                                                                                                                                                                                                                                                                                                                                                                               </div>
                                                                                    

</div><hr class="endfloat" />
</div>
<a 
 id="x1-71002r5"></a> <!--l. 394-->
<div class="longtable">
<table id="TBL-48" class="longtable" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-48-1g"><col 
id="TBL-48-1" /><col 
id="TBL-48-2" /><col 
id="TBL-48-3" /><col 
id="TBL-48-4" /></colgroup>
<tr  
 style="vertical-align:baseline;" id="TBL-48-1-"><td colspan="3" style="white-space:nowrap; text-align:left;" id="TBL-48-1-1"  
class="td11"></td> <div class="multicolumn"  style="white-space:nowrap; text-align:center;"> <div class="caption" 
>
<p><span class="id">Table 2.5: </span><span class="content">R Packages used during thesis</span></p>
</div>
<!--tex4ht:label?: x1-71002r5 --> </div> 
<a 
 id="x1-71003"></a>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-2-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Package</span>
</div></td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-2-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Version Used</span>
</div></td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-2-3"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Built</span>
</div></td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-2-4"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Repository</span>
</div>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-48-3-"><td colspan="3" style="white-space:nowrap; text-align:left;" id="TBL-48-3-1"  
class="td11"></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-4-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-5-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-6-"><td colspan="3" style="white-space:nowrap; text-align:left;" id="TBL-48-6-1"  
class="td11"></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-7-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-8-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-9-1"  
class="td11"> 
colorspace
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-9-2"  
class="td11"> 
1.3-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-9-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-9-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-10-1"  
class="td11"> 
curl
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-10-2"  
class="td11"> 
2.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-10-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-10-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-11-1"  
class="td11"> 
data.table
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-11-2"  
class="td11"> 
1.9.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-11-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-11-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-12-1"  
class="td11"> 
dendextend
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-12-2"  
class="td11"> 
1.4.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-12-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-12-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-13-1"  
class="td11"> 
DBI
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-13-2"  
class="td11"> 
0.5-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-13-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-13-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-14-1"  
class="td11"> 
devtools
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-14-2"  
class="td11"> 
1.12.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-14-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-14-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-15-1"  
class="td11"> 
dplyr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-15-2"  
class="td11"> 
0.5.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-15-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-15-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-16-1"  
class="td11"> 
ggplot2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-16-2"  
class="td11"> 
2.2.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-16-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-16-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-17-1"  
class="td11"> 
git2r
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-17-2"  
class="td11"> 
0.18.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-17-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-17-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-18-1"  
class="td11"> 
gplots
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-18-2"  
class="td11"> 
3.0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-18-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-18-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-19-1"  
class="td11"> 
gtools
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-19-2"  
class="td11"> 
3.5.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-19-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-19-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-20-1"  
class="td11"> 
igraph
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-20-2"  
class="td11"> 
1.0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-20-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-20-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-21-1"  
class="td11"> 
matrixcalc
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-21-2"  
class="td11"> 
1.0-3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-21-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-21-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-22-1"  
class="td11"> 
mclust
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-22-2"  
class="td11"> 
5.2.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-22-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-22-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-23-1"  
class="td11"> 
mvtnorm
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-23-2"  
class="td11"> 
1.0-6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-23-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-23-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-24-1"  
class="td11"> 
org.Hs.eg.db
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-24-2"  
class="td11"> 
3.1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-24-3"  
class="td11"> 
3.1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-24-4"  
class="td11"> 
Bioconductor
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-25-1"  
class="td11"> 
openssl
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-25-2"  
class="td11"> 
0.9.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-25-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-25-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-26-1"  
class="td11"> 
plyr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-26-2"  
class="td11"> 
1.8.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-26-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-26-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-27-1"  
class="td11"></td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-27-2"  
class="td11"></td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-27-3"  
class="td11"></td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-27-4"  
class="td11"></td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-48-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-28-1"  
class="td11"> 
purrr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-28-2"  
class="td11"> 
0.2.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-28-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-28-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-29-1"  
class="td11"> 
reactome.db
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-29-2"  
class="td11"> 
1.52.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-29-3"  
class="td11"> 
3.2.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-29-4"  
class="td11"> 
Bioconductor
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-30-1"  
class="td11"> 
RColorBrewer
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-30-2"  
class="td11"> 
1.1-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-30-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-30-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-31-1"  
class="td11"> 
Rcpp
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-31-2"  
class="td11"> 
0.12.9
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-31-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-31-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-32-1"  
class="td11"> 
ROCR
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-32-2"  
class="td11"> 
1.0-7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-32-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-32-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-33-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-33-1"  
class="td11"> 
roxygen2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-33-2"  
class="td11"> 
6.0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-33-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-33-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-34-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-34-1"  
class="td11"> 
shiny
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-34-2"  
class="td11"> 
1.0.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-34-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-34-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-35-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-35-1"  
class="td11"> 
snow
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-35-2"  
class="td11"> 
0.4-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-35-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-35-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-36-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-36-1"  
class="td11"> 
testthat
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-36-2"  
class="td11"> 
1.0.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-36-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-36-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-37-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-37-1"  
class="td11"> 
tidyr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-37-2"  
class="td11"> 
0.6.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-37-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-37-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-38-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-38-1"  
class="td11"> 
tidyverse
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-38-2"  
class="td11"> 
1.1.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-38-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-38-4"  
class="td11"> 
GitHub (hadley)
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-39-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-39-1"  
class="td11"> 
sm
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-39-2"  
class="td11"> 
2.2-5.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-39-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-39-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-40-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-40-1"  
class="td11"> 
Unicode
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-40-2"  
class="td11"> 
9.0.0-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-40-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-40-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-41-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-41-1"  
class="td11"> 
vioplot
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-41-2"  
class="td11"> 
0.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-41-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-41-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-42-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-42-1"  
class="td11"> 
viridis
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-42-2"  
class="td11"> 
0.3.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-42-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-42-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-43-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-43-1"  
class="td11"> 
xml2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-43-2"  
class="td11"> 
1.1.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-43-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-43-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-44-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-44-1"  
class="td11"> 
xtable
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-44-2"  
class="td11"> 
1.8-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-44-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-44-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-45-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-45-1"  
class="td11"> 
zoo
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-45-2"  
class="td11"> 
1.7-14
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-45-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-45-4"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-46-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-46-1"  
class="td11"> 
graphics
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-46-2"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-46-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-46-4"  
class="td11"> 
base
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-47-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-47-1"  
class="td11"> 
grDevices
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-47-2"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-47-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-47-4"  
class="td11"> 
base
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-48-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-48-1"  
class="td11"> </td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-48-49-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-49-1"  
class="td11"> 
cluster
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-49-2"  
class="td11"> 
2.0.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-49-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-49-4"  
class="td11"> 
base
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-50-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-50-1"  
class="td11"> 
graphics
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-50-2"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-50-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-50-4"  
class="td11"> 
base
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-51-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-51-1"  
class="td11"> 
grDevices
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-51-2"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-51-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-51-4"  
class="td11"> 
base
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-52-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-52-1"  
class="td11"> 
Matrix
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-52-2"  
class="td11"> 
1.2-8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-52-3"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-52-4"  
class="td11"> 
base
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-53-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-53-1"  
class="td11"> 
stats
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-53-2"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-53-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-48-53-4"  
class="td11"> 
base
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-54-"><td colspan="3" style="white-space:nowrap; text-align:left;" id="TBL-48-54-1"  
class="td11"></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-55-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-55-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-56-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-56-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-57-"><td colspan="3" style="white-space:nowrap; text-align:left;" id="TBL-48-57-1"  
class="td11"></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-58-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-58-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-59-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-59-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-60-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-60-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-61-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-61-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-62-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-62-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-63-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-63-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-48-64-"><td  style="white-space:nowrap; text-align:left;" id="TBL-48-64-1"  
class="td11"> </td></tr>
                                                                                    

</table></div>
<div class="table">
                                                                                    

<!--l. 530--><p class="indent" >
<a 
 id="x1-710046"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 2.6: </span><span class="content">R packages developed during thesis</span>
</div><!--tex4ht:label?: x1-710046 -->
<!--tex4ht:inline--><div class="tabular"><table id="TBL-49" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-49-1g"><col 
id="TBL-49-1" /></colgroup><colgroup id="TBL-49-2g"><col 
id="TBL-49-2" /><col 
id="TBL-49-3" /><col 
id="TBL-49-4" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-49-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-49-1-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"></div>                         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-1-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Package Name</span>
</div>                   </td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-1-3"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Description and GitHub Repository</span>
</div>                                             </td><td  style="white-space:nowrap; text-align:center;" id="TBL-49-1-4"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<span class="cmbx-12">Section</span>
</div>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-49-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-49-2-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-49-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-49-3-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"></div>                      </td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-3-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmtt-12">slipt</span>
</div>                             </td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-3-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-50" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-50-1g"><col 
id="TBL-50-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-50-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-50-1-1"  
class="td00">
Synthetic lethal detection by <a 
href="#glo:SLIPT">SLIPT</a> (to accompany publication)
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-50-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-50-2-1"  
class="td00">
<a 
href="https://github.com/TomKellyGenetics/slipt" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/slipt</span> </a>
</td></tr></table>               </div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-49-3-4"  
class="td11">   
<a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-49-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-49-4-1"  
class="td11">  
visualisation
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-4-2"  
class="td11"> 
<span class="cmtt-12">vioplotx </span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-4-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-51" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-51-1g"><col 
id="TBL-51-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-51-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-51-1-1"  
class="td00">
Customised violin plots (based on <span class="cmtt-12">vioplot</span>)
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-51-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-51-2-1"  
class="td00">
<a 
href="https://github.com/TomKellyGenetics/vioplotx" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/vioplotx</span></a>
</td></tr></table>                         </div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-49-4-4"  
class="td11">         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-49-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-49-5-1"  
class="td11">                    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-5-2"  
class="td11"> 
<span class="cmtt-12">heatmap.2x </span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-5-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-52" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-52-1g"><col 
id="TBL-52-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-52-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-52-1-1"  
class="td00">
Customised heatmaps (based on <span class="cmtt-12">gplots</span>)
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-52-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-52-2-1"  
class="td00">
<a 
href="https://github.com/TomKellyGenetics/heatmap.2x" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/heatmap.2x</span></a>
</td></tr></table>                       </div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-49-5-4"  
class="td11">  
<a 
href="#x1-600004">2.3.4<!--tex4ht:ref: methods:heatmap --></a>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-49-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-49-6-1"  
class="td11">  
<span class="cmtt-12">igraph.extensions </span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-6-2"  
class="td11"> 
<span class="cmtt-12">igraph.extensions </span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-6-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-53" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-53-1g"><col 
id="TBL-53-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-53-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-53-1-1"  
class="td00">
Meta-package to install the follow iGraph functions
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-53-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-53-2-1"  
class="td00">
<a 
href="https://github.com/TomKellyGenetics/igraph.extensions" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/igraph.extensions</span></a>
</td></tr></table>                </div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-49-6-4"  
class="td11">  
<a 
href="#x1-900003">3.5.3<!--tex4ht:ref: methods:igraph_extensions --></a>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-49-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-49-7-1"  
class="td11">                    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-7-2"  
class="td11"> 
<span class="cmtt-12">plot.igraph </span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-7-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-54" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-54-1g"><col 
id="TBL-54-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-54-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-54-1-1"  
class="td00">
Custom plotting of directed <a 
href="#glo:graph">graphs</a>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-54-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-54-2-1"  
class="td00">
<a 
href="https://github.com/TomKellyGenetics/plot.igraph" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/plot.igraph</span></a>
</td></tr></table>                      </div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-49-7-4"  
class="td11">  
<a 
href="#x1-680004">2.4.4<!--tex4ht:ref: methods:network_metrics --></a>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-49-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-49-8-1"  
class="td11">                    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-8-2"  
class="td11"> 
<span class="cmtt-12">info.centrality </span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-8-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-55" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-55-1g"><col 
id="TBL-55-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-55-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-55-1-1"  
class="td00">
Computing <a 
href="#glo:information centrality">information centrality</a> from network efficiency
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-55-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-55-2-1"  
class="td00">
<a 
href="https://github.com/TomKellyGenetics/info.centrality" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/info.centrality</span></a>
</td></tr></table>                  </div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-49-8-4"  
class="td11">  
<a 
href="#x1-860002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-49-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-49-9-1"  
class="td11">                    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-9-2"  
class="td11"> 
<span class="cmtt-12">pathway.structure.permutation </span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-9-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-56" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-56-1g"><col 
id="TBL-56-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-56-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-56-1-1"  
class="td00">
Testing <a 
href="#glo:graph">pathway</a> structure with resampling analysis
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-56-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-56-2-1"  
class="td00">
<a 
href="https://github.com/TomKellyGenetics/pathway.structure.permutation" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/pathway.structure.permutation</span></a>
</td></tr></table>    </div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-49-9-4"  
class="td11">  
<a 
href="#x1-850001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-49-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-49-10-1"  
class="td11">                    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-10-2"  
class="td11"> 
<span class="cmtt-12">graphsim </span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-49-10-3"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-57" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-57-1g"><col 
id="TBL-57-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-57-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-57-1-1"  
class="td00">
Generating simulated <a 
href="#glo:gene expression">expression</a> from <a 
href="#glo:graph">graph</a> structures
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-57-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-57-2-1"  
class="td00">
<a 
href="https://github.com/TomKellyGenetics/graphsim" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/graphsim</span> </a>
</td></tr></table>                       </div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-49-10-4"  
class="td11">  
<a 
href="#x1-860002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-49-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-49-11-1"  
class="td11">                    </td></tr></table>                                                                                                                                            </div>
                                                                                    

</div><hr class="endfloat" />
</div>
<a 
 id="x1-71005r87"></a>
<h4 class="subsectionHead">
<span class="titlemark">2.5.3 </span> <a 
 id="x1-720003"></a>High Performance and Parallel Computing
</h4>
<!--l. 575--><p class="noindent" >
Another enabling technology for <a 
href="#glo:bioinformatics">bioinformatics</a> is parallel computing, performing independent operations using separate <a 
href="#glo:CPU">central processing unit (CPU)</a> cores: this “multithreading” is widely used to increase the time to compute results. <a 
href="#glo:bioinformatics">Bioinformatics</a> is particularly amenable to this since performing multiple iterations of a simulation or testing separate genes is often “embarrassingly parallel”, as <a 
href="#glo:CPU">CPUs</a> completely independent of each other.
</p><!--l. 577--><p class="indent" >
<a 
href="#glo:NeSI">The New Zealand eScience Infrastructure (NeSI)</a> is a <a 
href="#glo:HPC">High Performance Computing (HPC)</a> organisation providing the Intel Pan cluster or “supercomputer”, hosted by the University of Auckland (<a 
href="#XNeSI">NeSI</a>, <a 
href="#XNeSI">2017</a>). The Pan cluster used throughout this thesis project to optimise and perform computations which would have otherwise been infeasible in the timeframe of thesis. Such technological developments and infrastructure initiatives have enabled <a 
href="#glo:bioinformatics">bioinformatics</a> research including this project. High performance computing on the Pan cluster was used extensively in this project including for resampling analysis (in Sections <a 
href="#x1-630006">2.3.6<!--tex4ht:ref: methods:permutation --></a> and <a 
href="#x1-850001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>), calculating <a 
href="#glo:information centrality">information centrality</a> (in Section <a 
href="#x1-680004">2.4.4<!--tex4ht:ref: methods:network_metrics --></a>), and in simulations (in Sections <a 
href="#x1-610005">2.3.5<!--tex4ht:ref: methods:simulation --></a>, <a 
href="#x1-750002">3.2<!--tex4ht:ref: methods:simulation_SL_expression --></a>, and <a 
href="#x1-860002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>)
</p><!--l. 579--><p class="indent" >
<p>Scripts and data were transferred between the Pan cluster and University of Otago computing resources by <span class="cmtt-12">rsync </span>or the Globus file transfer service (<a 
href="#XGlobus">Globus</a>, <a 
href="#XGlobus">2017</a>). R scripts (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>) were run in parallel with the “simple network of workstations” <span class="cmtt-12">snow </span>R package <a 
href="#Xsnow">Tierney <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#Xsnow">2015</a>). This utilised the “message passing interface” (<a 
href="#XRmpi">Yu</a>, <a 
href="#XRmpi">2002</a>) when it was feasible with memory requirements to run in parallel across multiple compute <a 
href="#glo:vertex">nodes</a>, otherwise <a 
href="#glo:SOCKS">Socket Secure (SOCKS)</a> was used to access multiple cores within an instance of R and pass input data to them. R jobs were submitted to queue for available resources and run on the Pan cluster via the <a 
href="#glo:Slurm">Simple Linux Utility for Resource Management (Slurm)</a> workload manager (<a 
href="#Xslurm">Slurm</a>, <a 
href="#Xslurm">2017</a>). <a 
href="#glo:Slurm">Slurm</a> array job submission and independent running of different parameters (with arguments passed to R from the shell) were used to run memory-intensive job or scripts across many parameters simultaneously. In some cases, this submission was automated across a range of parameters with <a 
href="#glo:Bash">Bash</a> scripts.</p>
</p>
<h2 class="chapterHead">
<span class="titlemark">Chapter 3</span><br /><a 
 id="x1-730003"></a>Methods Developed During Thesis
</h2>
<!--l. 5--><p class="noindent" >
In this Chapter, I outline the rationale and development of various methods used throughout this thesis to examine <a 
href="#glo:synthetic lethal">synthetic lethality</a> in <a 
href="#glo:gene expression">gene expression</a> data, <a 
href="#glo:graph">graph</a> structures, models and simulations. Firstly, the <a 
href="#glo:SLIPT">Synthetic Lethal Interaction Prediction Tool</a> (<a 
href="#glo:SLIPT">SLIPT</a>), a <a 
href="#glo:bioinformatics">bioinformatics</a> approach to triage <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate genes, will be described. This is one of the main research outputs of this thesis project and is supported by comparisons to an experimental screen from a related project and evaluation of performance on simulated data. These supporting findings will be covered in further chapters but simulation data is included to support the use and design of <a 
href="#glo:SLIPT">SLIPT</a>. This includes the construction of a statistical model of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in (continuous multivariate Gaussian) <a 
href="#glo:gene expression">gene expression</a> data, which enables testing <a 
href="#glo:SLIPT">SLIPT</a> upon simulated data with known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners. Another key component of this simulation pipeline is the generation of simulated data from a known <a 
href="#glo:graph">graph</a> structure or simulated biological pathway (as applied in Chapter <a 
href="#x1-1340006">6<!--tex4ht:ref: chap:simulation --></a>). The development of this simulation procedure and other statistical treatment of graph and <a 
href="#glo:graph">network</a> structures will also be covered. Various R packages have been developed to support this project, including the <span class="cmtt-12">slipt </span>package to implement the <a 
href="#glo:SLIPT">SLIPT</a> methodology. Additional R packages for handling <a 
href="#glo:graph">graph</a> structures, simulations, and custom plotting features will be described as research outputs of this thesis, methods applied throughout, and contributions of open-source software. <a 
 id="x1-73001r83"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">3.1 </span> <a 
 id="x1-740001"></a>A Synthetic Lethal Detection Methodology
</h3>
<!--l. 11--><p class="noindent" >
<p>The <a 
href="#glo:SLIPT">SLIPT</a> methodology identifies <a 
href="#glo:gene expression">gene expression</a> patterns consistent with <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions, between a query gene and a panel of candidate interacting partners. <a 
href="#glo:gene expression">Gene expression</a> is scored “low”, “medium”, or “high”, sorting samples by tertiles (<span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span>-quantiles) for each gene. Genes with insufficient <a 
href="#glo:gene expression">expression</a> across all samples are excluded by requiring that the first tertile of raw counts is above zero. A <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test is then performed between the query gene and each candidate partner. The p-values for the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> test are corrected for multiple testing using <a 
href="#glo:FDR">False discovery rate</a> (<a 
href="#glo:FDR">FDR</a>) error control to reduce false positives (<a 
href="#Xfdr1995">Benjamini and Hochberg</a>, <a 
href="#Xfdr1995">1995</a>). Significance is called for <a 
href="#glo:FDR">FDR</a> adjusted <span class="cmmi-12">p &lt; </span>0<span class="cmmi-12">.</span>05. A <a 
href="#glo:synthetic lethal">synthetic lethal</a> interaction is predicted (as shown in Figure <a 
href="#x1-740011">3.1<!--tex4ht:ref: fig:SLIPT_Method --></a>) when (i) the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test is significant; (ii) observed low-query, low-candidate samples are less frequent than expected; and (iii) observed low-query,</p>
<p>high-candidate and high-query, low-candidate samples are more frequent than expected.</p>
</p><!--l. 15--><p class="indent" >
<a 
 id="x1-740011"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 17--><p class="noindent" >
</p><!--l. 18--><p class="noindent" >
<span class="fbox"><img 
src="thesis6x.png" alt="PICT"  /></span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Framework for </span><a 
href="#glo:synthetic lethal"><span class="cmbx-10x-x-109">synthetic lethal</span></a> <span class="cmbx-10x-x-109">prediction.</span></strong> <a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">was designed to identify</span> <span class="cmr-10x-x-109">candidate interacting genes from </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">gene expression</span></a> <span class="cmr-10x-x-109">data using the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test against a query gene.</span> <span class="cmr-10x-x-109">Samples are sorted into low, medium, and high </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">quantiles for each gene to test for a</span> <span class="cmr-10x-x-109">directional shift. A sample being low in both genes of a </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">pair is unlikely, since</span> <span class="cmr-10x-x-109">loss of both genes will be deleterious, and is expected to be statistically under-represented in</span> <span class="cmr-10x-x-109">a </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">gene expression</span></a> <span class="cmr-10x-x-109">dataset. We expect a corresponding (symmetric) increase in frequency of</span> <span class="cmr-10x-x-109">sample with low-high gene pairs. </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">Synthetic lethal</span></a> <span class="cmr-10x-x-109">candidate partners of a gene were identified</span> <span class="cmr-10x-x-109">by running this procedure on all possible partner genes, selecting those with an </span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a><span class="cmr-10x-x-109">-adjusted</span> <span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span> </sup> <span class="cmr-10x-x-109">-derived </span><span class="cmmi-10x-x-109">p &lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05, and meeting the directional criteria. Since </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">genes are</span> <span class="cmr-10x-x-109">partners of each other, commutatively, the symmetric direction criteria were defined such</span> <span class="cmr-10x-x-109">that detected </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">genes are partners of each other. </span></span></p>
</div>
<!--tex4ht:label?: x1-740011 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 29--><p class="indent" >
<p>The <a 
href="#glo:synthetic lethal">synthetic lethal</a> prediction procedure has also been performed with <a 
href="#glo:somatic">somatic</a> <a 
href="#glo:mutation">mutation</a> data for the query gene. This is intended for a query gene known which is recurrently mutated, with the majority of <a 
href="#glo:mutation">mutations</a> disrupting gene function (e.g., null or frameshift <a 
href="#glo:mutation">mutations</a>). A <a 
href="#glo:synthetic lethal">synthetic lethal</a> interaction is predicted (as shown in Figure <a 
href="#x1-740022">3.2<!--tex4ht:ref: fig:SLIPT_Method_mtSL --></a>) when (i) the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test is significant; (ii) observed <a 
href="#glo:mutant">mutant</a>-query, low-candidate samples are less frequent than expected; and (iii) observed <a 
href="#glo:mutant">mutant</a>-query, high-candidate and <a 
href="#glo:wild-type">wild-type</a>-query, low-candidate samples are more frequent than expected.</p>
</p><!--l. 31--><p class="indent" >
<a 
 id="x1-740022"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 33--><p class="noindent" >
</p><!--l. 34--><p class="noindent" >
<span class="fbox"><img 
src="thesis7x.png" alt="PICT"  /></span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethal prediction adapted for </span><a 
href="#glo:mutation"><span class="cmbx-10x-x-109">mutation</span></a><span class="cmbx-10x-x-109">.</span></strong> <a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">was also</span> <span class="cmr-10x-x-109">adapted to identify candidate interacting genes using (</span><a 
href="#glo:somatic"><span class="cmr-10x-x-109">somatic</span></a><span class="cmr-10x-x-109">) </span><a 
href="#glo:mutation"><span class="cmr-10x-x-109">mutation</span></a> <span class="cmr-10x-x-109">data of the query</span> <span class="cmr-10x-x-109">gene in the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span> </sup> <span class="cmr-10x-x-109">test. Samples are sorted into low, medium, and high </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">quantiles</span> <span class="cmr-10x-x-109">for each candidate gene and tested for a directional shift against </span><a 
href="#glo:mutation"><span class="cmr-10x-x-109">mutation</span></a> <span class="cmr-10x-x-109">status of the</span> <span class="cmr-10x-x-109">query gene. A sample having low </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">or </span><a 
href="#glo:mutation"><span class="cmr-10x-x-109">mutation</span></a> <span class="cmr-10x-x-109">for the </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">pair is</span> <span class="cmr-10x-x-109">expected to be unlikely with a corresponding increase in frequency of sample with </span><a 
href="#glo:mutant"><span class="cmr-10x-x-109">mutant</span></a><span class="cmr-10x-x-109">-high</span> <span class="cmr-10x-x-109">or </span><a 
href="#glo:wild-type"><span class="cmr-10x-x-109">wild-type</span></a><span class="cmr-10x-x-109">-low gene pairs. </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">Synthetic lethal</span></a> <span class="cmr-10x-x-109">(mtSL) candidate partners of a gene were</span> <span class="cmr-10x-x-109">identified from running this procedure on all possible partner genes, selecting those with an</span> <a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a><span class="cmr-10x-x-109">-adjusted </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup><span class="cmr-10x-x-109">-derived </span><span class="cmmi-10x-x-109">p &lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05, and meeting the directional criteria. </span></span></p>
</div>
<!--tex4ht:label?: x1-740022 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 44--><p class="indent" >
The <a 
href="#glo:SLIPT">SLIPT</a> methodology can be performed on <a 
href="#glo:gene expression">expression</a> data, including pathway <a 
href="#glo:metagene">metagenes</a> (as generated in Section <a 
href="#x1-540003">2.2.3<!--tex4ht:ref: methods:metagene --></a>). The application of the <a 
href="#glo:SLIPT">SLIPT</a> methodology on public <a 
href="#glo:gene expression">gene expression</a> data will be supported with simulation results (in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a> and Chapter <a 
href="#x1-1340006">6<!--tex4ht:ref: chap:simulation --></a>), including comparison to other statistical methods. <a 
href="#glo:SLIPT">SLIPT</a> results for <span class="cmti-12">CDH1</span> were compared experimental screen results in a breast cell line (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). Primary screen results are discussed in Section <a 
href="#x1-1010002">4.2<!--tex4ht:ref: chapt3:compare_SL_genes --></a> and secondary screen results are presented in Section <a 
href="#x1-1050004">4.2.4<!--tex4ht:ref: chapt3:secondary_screen --></a>. <a 
 id="x1-74003r93"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">3.2 </span> <a 
 id="x1-750002"></a>Synthetic Lethal Simulation and Modelling
</h3>
<!--l. 55--><p class="noindent" >
A statistical model of <a 
href="#glo:synthetic lethal">synthetic lethality</a> was developed to generate simulated data and to evaluate the <a 
href="#glo:SLIPT">SLIPT</a> procedure. This Section describes the <a 
href="#glo:synthetic lethal">synthetic lethal</a> model and the simulation procedure for generating <a 
href="#glo:gene expression">gene expression</a> data with known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners. Simulation results, to support usage of the <a 
href="#glo:SLIPT">SLIPT</a> methodology throughout this thesis, will be presented in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>. The simulation procedure will also be applied in Chapter <a 
href="#x1-1340006">6<!--tex4ht:ref: chap:simulation --></a>, including in combination with simulations from <a 
href="#glo:graph">graph</a> structures (as described in Section <a 
href="#x1-860002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>). <a 
 id="x1-75001r91"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.2.1 </span> <a 
 id="x1-760001"></a>A Model of Synthetic Lethality in Expression Data
</h4>
<!--l. 59--><p class="noindent" >
<p>A conceptual model of <a 
href="#glo:synthetic lethal">synthetic lethality</a> was devised (as shown in Figure <a 
href="#x1-760013">3.3<!--tex4ht:ref: fig:SL_Model --></a>), which will be used to build a statistical model of <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:gene expression">gene expression</a> and to simulate <a 
href="#glo:gene expression">expression</a> data for assessing various potential <a 
href="#glo:synthetic lethal">synthetic lethal</a> prediction methods, including <a 
href="#glo:SLIPT">SLIPT</a>. In the model, <a 
href="#glo:synthetic lethal">synthetic lethality</a> occurs between genes with related functions, as a cell death phenotype, when these functions are inactive.</p>
</p><!--l. 61--><p class="indent" >
<a 
 id="x1-760013"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 63--><p class="noindent" >
</p><!--l. 64--><p class="noindent" >
<span class="fbox"><img 
src=""SL_Model".png" alt="pict"  
 /></span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">A model of </span><a 
href="#glo:synthetic lethal"><span class="cmbx-10x-x-109">synthetic lethal</span></a> <span class="cmbx-10x-x-109">gene expresion.</span></strong> <span class="cmr-10x-x-109">A conceptual model of</span> <a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">interactions between a Query gene and partner gene (</span><span class="cmmi-10x-x-109">G</span><sub><span class="cmmi-8">X</span></sub><span class="cmr-10x-x-109">). Genes that are</span> <a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">may not both be non-functional in the same sample without another gene</span> <span class="cmr-10x-x-109">compensating for the loss of function. This is most likely to be detectable as low </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">gene</span> <span class="cmr-10x-x-109">expression</span></a><span class="cmr-10x-x-109">, whether they are lost by </span><a 
href="#glo:mutation"><span class="cmr-10x-x-109">mutation</span></a><span class="cmr-10x-x-109">, deletion, </span><a 
href="#glo:DNA"><span class="cmr-10x-x-109">DNA</span></a> <span class="cmr-10x-x-109">methylation, or suppressing</span> <span class="cmr-10x-x-109">regulatory signals. This could manifest as coexpression, mutual exclusivity, or directional</span> <span class="cmr-10x-x-109">shifts in sample frequency. Thus the alternative hypothesis (</span><span class="cmmi-10x-x-109">H</span><sub><span class="cmmi-8">A</span></sub><span class="cmr-10x-x-109">) is that </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">genes will have a reduced frequency of co-loss samples while the null hypothesis (</span><span class="cmmi-10x-x-109">H</span><sub><span class="cmr-8">0</span></sub><span class="cmr-10x-x-109">) is</span> <span class="cmr-10x-x-109">that non-synthetic lethal gene pairs would show no such relationship, even if they may be</span> <span class="cmr-10x-x-109">correlated for other means such as pathway relationships. In this model </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">genes</span> <span class="cmr-10x-x-109">may compensate for the loss of each other but this is not assumed, only that loss of both is</span> <span class="cmr-10x-x-109">unfavourable to cell viability and probability of detecting samples with combined gene loss.</span> </span></p>
</div>
<!--tex4ht:label?: x1-760013 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 78--><p class="indent" >
This model suggests that <a 
href="#glo:synthetic lethal">synthetic lethality</a> is detectable in measures of gene inactivation across a sample population, namely <a 
href="#glo:mutation">mutation</a>, <a 
href="#glo:DNA">DNA</a> copy number, <a 
href="#glo:DNA">DNA</a> <a 
href="#glo:methylation">methylation</a>, and <a 
href="#glo:gene expression">expression</a> levels. While any of these mechanisms of gene inactivation could lead to <a 
href="#glo:synthetic lethal">synthetic lethality</a>, <a 
href="#glo:gene expression">expression</a> data is readily available and changes in other mechanisms are likely to impact on the amount of expressed <a 
href="#glo:RNA">RNA</a> that is detectable. Functional relationships between genes could manifest in <a 
href="#glo:gene expression">expression</a> data in several ways, including coexpression, mutual exclusivity and directional shifts. Co-expression is overly simplistic (<a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>) and has previously performed poorly as a predictor of <a 
href="#glo:synthetic lethal">synthetic lethality</a> (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>), although this will still be tested with correlation measures in later simulations. The alternative hypothesis is that <a 
href="#glo:synthetic lethal">synthetic lethality</a> will result in a detectable shift in the number of samples which exhibit low or high <a 
href="#glo:gene expression">expression</a> of either gene. This model does not preclude mutual exclusivity, compensating <a 
href="#glo:gene expression">expression</a>, or co-loss under-representation which may occur between <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes (<a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>; <a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWappett2016">2016</a>).
</p><!--l. 80--><p class="indent" >
<p>The first condition of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> model is that if there are only two <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes (e.g., <span class="cmti-12">CDH1 </span>and one SL partner), then they will not both be non-functional in the same sample (in an ideal model). Gene function is thus determined for each sample in a model of <a 
href="#glo:synthetic lethal">synthetic lethality</a> with the proportion of samples which are functional or non-functional for a gene being arbitrary. Whether a gene is functional can similarly be modelled by an arbitrary threshold of continuous and normally distributed <a 
href="#glo:gene expression">gene expression</a> data to define gene function (as shown in Figure <a 
href="#x1-760024">3.4<!--tex4ht:ref: fig:SL_Model_Expression --></a>). For the purposes of modelling <a 
href="#glo:synthetic lethal">synthetic lethality</a> in cancer <a 
href="#glo:gene expression">expression</a> data, a threshold of the 30<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup> percentile of the <a 
href="#glo:gene expression">expression</a> levels was used because approximately 30% of samples analysed had <span class="cmti-12">CDH1 </span>inactivation (mutations) in breast cancer (<a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>). This was generalised for a model of the proportion of samples inactivated for each gene. The threshold of the 0.3 quantile was used in simulations dervied from this model throughout this thesis. In this ideal case, no samples lowly expressing both of these genes are expected to be observed. While this was not the case, that is to be expected as it is unlikely that only two genes will have an exclusive <a 
href="#glo:synthetic lethal">synthetic lethal</a> partnership.</p>
</p><!--l. 82--><p class="indent" >
<a 
 id="x1-760024"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 84--><p class="noindent" >
</p><!--l. 85--><p class="noindent" >
<span class="fbox"><img 
src=""SL_Model_Expression".png" alt="pict"  
 /></span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.4: </span><span class="content"><strong><span class="cmbx-10x-x-109">Modelling </span><a 
href="#glo:synthetic lethal"><span class="cmbx-10x-x-109">synthetic lethal</span></a> <a 
href="#glo:gene expression"><span class="cmbx-10x-x-109">gene expression</span></a><span class="cmbx-10x-x-109">.</span></strong> <span class="cmr-10x-x-109">When modelling </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic</span> <span class="cmr-10x-x-109">lethal</span></a> <span class="cmr-10x-x-109">interactions between a Query gene and partner genes (</span><span class="cmmi-10x-x-109">G</span><sub><span class="cmmi-8">X</span></sub> <span class="cmr-10x-x-109">and </span><span class="cmmi-10x-x-109">G</span><sub><span class="cmmi-8">Y</span> </sub><span class="cmr-10x-x-109">) above, cellular</span> <span class="cmr-10x-x-109">viability requires that at least of genes is not inactivated. As a model of loss of function,</span> <span class="cmr-10x-x-109">genes are regarded as non-functional with expression below a threshold for the purposes of</span> <span class="cmr-10x-x-109">modelling </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a><span class="cmr-10x-x-109">. Tumour suppressor genes with loss of function also have cancer</span> <span class="cmr-10x-x-109">specific phenotypes (although these thresholds are not assumed to be the same). Expression</span> <span class="cmr-10x-x-109">is modelled by normally (Gaussian) distributed continuous data, such as (log-scale) data from</span> <a 
href="#glo:RNA"><span class="cmr-10x-x-109">RNA</span></a> <span class="cmr-10x-x-109">(</span><a 
href="#glo:microarray"><span class="cmr-10x-x-109">microarray</span></a> <span class="cmr-10x-x-109">or </span><a 
href="#glo:RNA-Seq"><span class="cmr-10x-x-109">RNA-Seq</span></a><span class="cmr-10x-x-109">), protein, or pathway </span><a 
href="#glo:metagene"><span class="cmr-10x-x-109">metagenes</span></a><span class="cmr-10x-x-109">. This rationale generalises to</span> <span class="cmr-10x-x-109">several genes on a multivariate normal distribution. </span></span></p>
</div>
<!--tex4ht:label?: x1-760024 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 98--><p class="indent" >
<a 
 id="x1-760035"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 100--><p class="noindent" >
</p><!--l. 101--><p class="noindent" >
<span class="fbox"><img 
src=""SL_Model_Higher".png" alt="pict"  
 /></span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.5: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality with multiple genes.</span></strong> <span class="cmr-10x-x-109">Higher order </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">interactions may occur between 3 or more genes, affecting the simulated </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">(or</span> <a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">predictions) even if undetected when observed pairwise. Consider interactions</span> <span class="cmr-10x-x-109">between a Query gene and two partner genes (</span><span class="cmmi-10x-x-109">G</span><sub><span class="cmmi-8">X</span></sub> <span class="cmr-10x-x-109">and </span><span class="cmmi-10x-x-109">G</span><sub><span class="cmmi-8">Y</span> </sub><span class="cmr-10x-x-109">). They may interact with the</span> <span class="cmr-10x-x-109">Query pairwise (inviable when either gene pair is lost) or form a higher-order interaction such</span> <span class="cmr-10x-x-109">as the “synthetic lethal triplet” if any of the genes provide an </span><a 
href="#glo:essential"><span class="cmr-10x-x-109">essential</span></a> <span class="cmr-10x-x-109">function (inviable</span> <span class="cmr-10x-x-109">only when all are lost). Either is plausible with the potential </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">pathway</span></a> <span class="cmr-10x-x-109">structures. A </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic</span> <span class="cmr-10x-x-109">lethal</span></a> <span class="cmr-10x-x-109">triple has 8 potential combinations of gene function but one is not expected to be</span> <span class="cmr-10x-x-109">observed (due to inviability), however pairwise inactivation may be observed if additional</span> <span class="cmr-10x-x-109">partner genes are functional. The proportion of these combinations varies depending on the</span> <span class="cmr-10x-x-109">functional threshold. </span></span></p>
</div>
<!--tex4ht:label?: x1-760035 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 116--><p class="indent" >
A <a 
href="#glo:synthetic lethal">synthetic lethal</a> pair of genes is unlikely to act in isolation, therefore higher-order <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions (i.e., 3 or more genes) must be considered in the model as shown in Figure <a 
href="#x1-760035">3.5<!--tex4ht:ref: fig:SL_Model_Higher --></a>. Even when testing pairwise interactions, it is important to model higher level interactions that may interfere. If there are additional <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners, there are two possibilities for adding these: 1) that they are independent partners of the query genes interacting pairwise (and not with each other) or 2) that an additional partner gene interacts with both of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes already in the system and any of the three (or more) are required to be functional for the cell to survive.
</p><!--l. 118--><p class="indent" >
The signal (in terms of <a 
href="#glo:gene expression">gene expression</a> data) will be weaker for this latter case and this model has the more stringent assumption that all <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes interact with each other: that only one of these must be expressed to satisfy the model of <a 
href="#glo:synthetic lethal">synthetic lethality</a>. In this model, any of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes in a higher-order interaction are able to perform the essential function of the others, allowing for higher-level <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners to compensate for loss a <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pair. While samples that express low levels of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs will be under-represented, they may not be completely absent from the dataset, due to these higher-level interactions. In the example of three <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes (shown in Figure <a 
href="#x1-760035">3.5<!--tex4ht:ref: fig:SL_Model_Higher --></a>), only one of the genes involved in the higher-order <a 
href="#glo:synthetic lethal">synthetic lethal</a> interaction is required for cell viability. For <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs, only a subset of these samples will be inviable (i.e., removed from simulated data), leading to an under-representation.
</p><!--l. 120--><p class="indent" >
Samples were not actually removed from a simulated dataset, rather the <a 
href="#glo:gene expression">expression</a> and function of the query gene is generated across samples separately from the pool of potential partner genes. The query gene data was matched to simulated samples (as shown in Figure <a 
href="#x1-770068">3.8<!--tex4ht:ref: fig:simulate_add_query --></a>) satisfying the <a 
href="#glo:synthetic lethal">synthetic lethal</a> condition with the procedure described in Section <a 
href="#x1-770002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a>. This was performed to maintain a comparable samples size across simulations and the preserve the (multivariate) normal distribution of the data. <a 
 id="x1-76004r97"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.2.2 </span> <a 
 id="x1-770002"></a>Simulation Procedure
</h4>
<!--l. 126--><p class="noindent" >
<p>Simulations were developed to generate normal distributions of <a 
href="#glo:gene expression">expression</a> data and define gene function with a threshold cut-off. While gene function was used as an intermediary step</p>
<p>in modelling <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes in <a 
href="#glo:gene expression">expression</a> data, the normal distribution was sampled for simulated data to represent normalised empirical <a 
href="#glo:gene expression">gene expression</a> data for which <a 
href="#glo:SLIPT">SLIPT</a> (and other methods) will be applicable.</p>
</p><!--l. 130--><p class="indent" >
<a 
 id="x1-770037"></a><a 
 id="x1-770068"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 133--><p class="noindent" >
</p><!--l. 139--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_mat".png  /><a 
 id="x1-77001r1"></a> <span class="cmr-10x-x-109">Simulated</span> <a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">matrix</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_disc_mat".png  /><a 
 id="x1-77002r2"></a> <span class="cmr-10x-x-109">Corresponding</span> <span class="cmr-10x-x-109">gene</span> <span class="cmr-10x-x-109">function</span> <span class="cmr-10x-x-109">calls</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.7: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulating gene function.</span></strong></strong> <span class="cmr-10x-x-109">A simulated dataset with samples (columns) and</span> <span class="cmr-10x-x-109">genes A–I (rows) was transformed from a continuous (coloured blue–red) scale to a discrete</span> <span class="cmr-10x-x-109">matrix of gene function (black for functional levels and grey for non-functional).</span></span></p>
</div>
<!--tex4ht:label?: x1-770037 -->
<div class="center" 
>
<!--l. 183--><p class="noindent" >
</p><!--l. 190--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_SL_disc_mat".png  /><a 
 id="x1-77004r1"></a> <span class="cmr-10x-x-109">Simulated</span> <span class="cmr-10x-x-109">gene</span> <span class="cmr-10x-x-109">function</span> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">SL</span> <span class="cmr-10x-x-109">genes</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_disc_query_mat_graph".png  /><a 
 id="x1-77005r2"></a> <span class="cmr-10x-x-109">Query</span> <span class="cmr-10x-x-109">gene</span> <span class="cmr-10x-x-109">added</span> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">SL</span> <span class="cmr-10x-x-109">condition</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.8: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulating </span><a 
href="#glo:synthetic lethal"><span class="cmbx-10x-x-109">synthetic lethal</span></a> <span class="cmbx-10x-x-109">gene function.</span></strong></strong> <span class="cmr-10x-x-109">In a discrete simulated gene</span> <span class="cmr-10x-x-109">function dataset (shaded for functional levels and pale otherwise) with samples (columns)</span> <span class="cmr-10x-x-109">and genes (rows), genes A and I are SL partners of a “Query” gene. A partner was selected</span> <span class="cmr-10x-x-109">(highlighted in green) randomly in each sample for simulating </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a><span class="cmr-10x-x-109">, then ordered</span> <span class="cmr-10x-x-109">such that the query gene or an SL partner were functional in each sample. </span></span></p>
</div>
<!--tex4ht:label?: x1-770068 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 208--><p class="indent" >
Sampling a distribution for <a 
href="#glo:gene expression">expression</a> profiles has the advantage of enabling simulating correlation structures with the multivariate normal distribution, using the <span class="cmtt-12">mvtnorm </span>R package (<a 
href="#XGenz2009">Genz and Bretz</a>, <a 
href="#XGenz2009">2009</a>; <a 
href="#Xmvtnorm">Genz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xmvtnorm">2016</a>). The parameter Σ, the covariance matrix, defines the correlation structure between the simulated genes being sampled. With a diagonal of one, this Σ matrix simulates genes with a standard deviation of one and the covariance parameters between them are the correlations between each gene. In Figure <a 
href="#x1-770037">3.7<!--tex4ht:ref: fig:simulate_function --></a>, an example of such a simulated multivariate normal dataset is shown with the functional threshold applied.
</p><!--l. 210--><p class="indent" >
<p>Once a simulated dataset has been generated, the samples were compared by gene function (as derived from a functional threshold). The known underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners were selected within the dataset and a query gene was generated by sampling from the normal distribution. These were matched (as shown for two <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in Figure <a 
href="#x1-770068">3.8<!--tex4ht:ref: fig:simulate_add_query --></a>) such that the <a 
href="#glo:synthetic lethal">synthetic lethal</a> condition was met: at least one of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes and the query gene are functional in any particular cell. The samples are ordered by functional data (without assuming correlation of underyling <a 
href="#glo:gene expression">expression</a> values) with the query gene in one direction and the remaining dataset ordered by the selected <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner.</p>
</p><!--l. 213--><p class="indent" >
<a 
 id="x1-7700910"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 216--><p class="noindent" >
</p><!--l. 222--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_mat".png  /><a 
 id="x1-77007r1"></a> <span class="cmr-10x-x-109">Initial</span> <a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">matrix</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_query_mat_graph".png  /><a 
 id="x1-77008r2"></a> <span class="cmr-10x-x-109">Simulated</span> <a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic</span> <span class="cmr-10x-x-109">lethal</span></a> <span class="cmr-10x-x-109">dataset</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.10: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulating </span><a 
href="#glo:synthetic lethal"><span class="cmbx-10x-x-109">synthetic lethal</span></a> <a 
href="#glo:gene expression"><span class="cmbx-10x-x-109">gene expression</span></a><span class="cmbx-10x-x-109">.</span></strong></strong> <span class="cmr-10x-x-109">A simulated continuous</span> <a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">dataset (blue–red scale) with samples (columns) and genes A–I (rows) was matched</span> <span class="cmr-10x-x-109">to a query gene such that at least one </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partner was above a functional threshold</span> <span class="cmr-10x-x-109">when the query gene was below it which satisfied the </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">model.</span></span></p>
</div>
<!--tex4ht:label?: x1-7700910 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 237--><p class="indent" >
This procedure produces a simulated dataset where samples with a non-functional query gene have at least one functional partner gene. Similarly, the query gene is functional in all samples where all of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes are non-functional. In this procedure, a dataset has been generated with known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners (as shown in Figure <a 
href="#x1-7700910">3.10<!--tex4ht:ref: fig:simulate_SL --></a>) with few assumptions about the relationships between the each <a 
href="#glo:synthetic lethal">synthetic lethal</a> pair (allowing compensating functions from higher-order interactions). This procedure has been designed to have the most stringent (least detectable) <a 
href="#glo:synthetic lethal">synthetic lethal</a> relationships, where higher-order interactions are possible for the purposes of testing pairwise detection procedures such as <a 
href="#glo:SLIPT">SLIPT</a>. <a 
 id="x1-77010r96"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">3.3 </span> <a 
 id="x1-780003"></a>Detecting Simulated Synthetic Lethal Partners
</h3>
<!--l. 244--><p class="noindent" >
The <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection methodology (<a 
href="#glo:SLIPT">SLIPT</a>), as described in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>, was evaluated with simulated data containing known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners, generated using the procedure described in Section <a 
href="#x1-770002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a>. Simulations were performed to demonstrate the methodology and support its use throughout this thesis. These simulations were performed by sampling from statistical distributions, including the multivariate normal distribution with correlated blocks of genes, generated by Σ matrices such as those shown. A more complex multivariate normal sampling procedure based on pathway <a 
href="#glo:graph">graph</a> structures, as described in Section  <a 
href="#x1-860002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>, was used for further investigations in Chapter <a 
href="#x1-1340006">6<!--tex4ht:ref: chap:simulation --></a>. <a 
 id="x1-78001r101"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.3.1 </span> <a 
 id="x1-790001"></a>Binomial Simulation of Synthetic Lethality
</h4>
<!--l. 249--><p class="noindent" >
The <a 
href="#glo:synthetic lethal">synthetic lethal</a> simulation procedure (described in Section <a 
href="#x1-770002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a>) initially used gene function, sampled directly from a binomial distribution using the binomial probability of observing functional gene levels (<span class="cmmi-12">p </span>= 0<span class="cmmi-12">.</span>3) in one observation (<span class="cmmi-12">n </span>= 1) for each samples:
</p>
                                                                                    

<center class="math-display" >
<img 
src="thesis8x.png" alt="X  ~ Bin (n,p)  " class="math-display"  />
</center>
Once a query gene with <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners has been added, these functional levels were passed directly into <a 
href="#glo:SLIPT">SLIPT</a> as “low” and “high” samples. <!--l. 251-->
<p class="indent" >
<p>The simulation procedure was performed with 20,000 total genes (as occurs in <a 
href="#glo:gene expression">expression</a> datasets) with a variable number of true <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners and 500, 1000, 2000, or 5000 samples. Each <a 
href="#glo:ROC">ROC</a> curve was derived from the results of 10,000 replicate simulations. The statistical performance (as shown in Figure <a 
href="#x1-7900111">3.11<!--tex4ht:ref: fig:Binomial_AUC --></a>) of the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>-derived p-value declined towards random predictions (an <a 
href="#glo:AUROC">AUROC</a> of 0.5) with an more underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners to detect. However, increased sample size somewhat mitigated this decline, as expected with a statistical predictor, particularly for moderate numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners.</p>
</p><!--l. 253--><p class="indent" >
<a 
 id="x1-7900111"></a><a 
 id="x1-7900212"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 255--><p class="noindent" >
</p><!--l. 256--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150311/SL_Model_Binomial_1K_AUC_samples_prop".png  />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.11: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of binomial simulations.</span></strong> <span class="cmr-10x-x-109">Gene function was simulated</span> <span class="cmr-10x-x-109">by binomial sampling and tested for </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a> <span class="cmr-10x-x-109">by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a><span class="cmr-10x-x-109">. Statistical performance</span> <span class="cmr-10x-x-109">declined with additional </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners but this was mitigated by increased sample</span> <span class="cmr-10x-x-109">sizes.</span></span></p>
</div>
<!--tex4ht:label?: x1-7900111 -->
<div class="center" 
>
<!--l. 267--><p class="noindent" >
</p><!--l. 268--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150410y/SL_Model_Test_Graph_10K_Graph1_ROC_Compare_Binom(Feb)_v_Mvtn(Aprxy)_Full".png  />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.12: </span><span class="content"><strong><span class="cmbx-10x-x-109">Comparison of statistical performance.</span></strong> <span class="cmr-10x-x-109">Binomial simulation of </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic</span> <span class="cmr-10x-x-109">lethality</span></a> <span class="cmr-10x-x-109">(in colour) in comparison to multivariate normal simulations (in greyscale), in which</span> <a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">consistently had higher performance across parameters.</span></span></p>
</div>
<!--tex4ht:label?: x1-7900212 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 279--><p class="indent" >
Simulations using this binomial model of <a 
href="#glo:synthetic lethal">synthetic lethality</a> were simplistic but informed the development of more complex simulations including <a 
href="#glo:gene expression">expression</a> and correlation structures. It did not represent the data that <a 
href="#glo:SLIPT">SLIPT</a> will be applied to but the binomial simulations demonstrated that <a 
href="#glo:SLIPT">SLIPT</a> is able to distinguish small numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in a simulated system with behaviours expected with respect to sample size. This supported further development of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> model and simulation pipeline (as described in Section <a 
href="#x1-750002">3.2<!--tex4ht:ref: methods:simulation_SL_expression --></a>) using the multivariate normal distribution.
</p><!--l. 281--><p class="indent" >
The multivariate normal simulation procedure more closely recapitulates the (normalised) <a 
href="#glo:gene expression">expression</a> data that <a 
href="#glo:SLIPT">SLIPT</a> was intended for and enables the methodology procedure to be tested without requiring modifications (in Section <a 
href="#x1-800002">3.3.2<!--tex4ht:ref: chapt2:simulation_mvtnorm --></a>). Sampling continuous <a 
href="#glo:gene expression">expression</a> values from a normal distribution enabled the <a 
href="#glo:gene expression">expression</a> threshold for gene function to differ from the categorical “low” and “high” <a 
href="#glo:gene expression">expression</a> binning performed by <a 
href="#glo:SLIPT">SLIPT</a> (as discussed in Section <a 
href="#x1-760001">3.2.1<!--tex4ht:ref: methods:SL_Model --></a>). The <a 
href="#glo:SLIPT">SLIPT</a> procedure does not assume a known threshold for <a 
href="#glo:gene expression">expression</a> and instead uses <a 
href="#glo:gene expression">expression</a> as an estimate of gene function which does not compromise the statistical performance of the <a 
href="#glo:SLIPT">SLIPT</a> in the multivariate normal simulation. The performance was an improvement over the binomial simulation procedure (shown in Figure <a 
href="#x1-7900212">3.12<!--tex4ht:ref: fig:Binomial_Compare --></a>) across simulation parameters in an equivalent simulation (without correlation structure). This multivariate normal model is also more refined since it defines the <a 
href="#glo:synthetic lethal">synthetic lethal</a> condition, to ensure that at least one <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner was active in query-deficient samples, without disrupting the proportion of samples with each gene being functional. <a 
 id="x1-79003r112"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.3.2 </span> <a 
 id="x1-800002"></a>Multivariate Normal Simulation of Synthetic Lethality
</h4>
<!--l. 287--><p class="noindent" >
<p>The multivariate normal simulation procedure was initially performed using the <span class="cmtt-12">mvtnorm </span>R package (<a 
href="#XGenz2009">Genz and Bretz</a>, <a 
href="#XGenz2009">2009</a>; <a 
href="#Xmvtnorm">Genz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xmvtnorm">2016</a>) (as described in Section <a 
href="#x1-750002">3.2<!--tex4ht:ref: methods:simulation_SL_expression --></a>) without correlation structure. Expression was sampled from multivariate normal distribution with a mean (<span class="cmmi-12">μ </span>= 0), standard deviation (<span class="cmmi-12">σ </span>= 1), and no correlation between genes (<span class="cmmi-12">r </span>= 0):</p>
</p>
<center class="math-display" >
<img 
src="thesis9x.png" alt="X  ~ N  (&#x03BC;¯, &#x03A3;).  " class="math-display"  />
</center>
<p>Once a query gene with <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners has been added, the simulated <a 
href="#glo:gene expression">expression</a> values were tested by <a 
href="#glo:SLIPT">SLIPT</a>, as described in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>.</p>
<!--l. 289--><p class="indent" >
<a 
 id="x1-8000414"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 292--><p class="noindent" >
</p><!--l. 296--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150410y/SL_Model_Apr15mvnormCor_1K_ROC1_samplesx_prop".png  /><a 
 id="x1-80001r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150410y/SL_Model_Apr15mvnormCor_1K_ROC2_samplesx_prop".png  /><a 
 id="x1-80002r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a> <span class="cmr-10x-x-109">curves</span>
</p><!--l. 303--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150410y/SL_Model_Apr15mvnormCor_1K_AUC_samplesx_prop".png  /><a 
 id="x1-80003r3"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span> <span class="cmr-10x-x-109">(</span><a 
href="#glo:AUROC"><span class="cmr-10x-x-109">AUROC</span></a><span class="cmr-10x-x-109">)</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.14: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of multivariate normal simulations.</span></strong> <span class="cmr-10x-x-109">Simulation of </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic</span> <span class="cmr-10x-x-109">lethality</span></a> <span class="cmr-10x-x-109">was performed by sampling from a multivariate normal distribution (without</span> <span class="cmr-10x-x-109">correlation structure). Performance of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">declined with increasing numbers of </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic</span> <span class="cmr-10x-x-109">lethal</span></a> <span class="cmr-10x-x-109">partners but this was mitigated by increased sample sizes (in darker colours). This</span> <span class="cmr-10x-x-109">occurred as the sensitivity decreased with a greater number of true positives to detect, which</span> <span class="cmr-10x-x-109">lead to a trade-off in accuracy as seen in a trough for false positive rate and the </span><a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a> <span class="cmr-10x-x-109">curves.</span></span></p>
</div>
<!--tex4ht:label?: x1-8000414 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 311--><p class="indent" >
The statistical accuracy of <a 
href="#glo:SLIPT">SLIPT</a> as a binary classifier was high across simulations of a full dataset of 20,000 genes (shown in Figure <a 
href="#x1-80001r1">3.13a<!--tex4ht:ref: fig:simulation_Apr15ROC:Perf --></a>). Using the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>-derived p-value as a threshold for prediction, this was largely due to high specificity: the majority of non-synthetic lethal genes were distinguished from the underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes. Thus the <a 
href="#glo:SLIPT">SLIPT</a> methodology performed better with larger datasets with more expected negatives and the results of simulations of smaller numbers of genes (e.g., the <a 
href="#glo:graph">graph</a> structures analysed in Section <a 
href="#x1-1420001">6.2.1<!--tex4ht:ref: chapt5:graphsim_performance --></a>) can be applied to larger datasets, where they are expected to perform comparably or better with a lower false negative rate (as shown in Sections <a 
href="#x1-1470004">6.2.4<!--tex4ht:ref: chapt5:graphsim_performance_20K --></a> and <a 
href="#x1-1500002">6.3.2<!--tex4ht:ref: chapt5:graphsim_performance_20K_pway --></a>). Accordingly, key results will be supported by replication with larger numbers of non-synthetic lethal genes added to the simulations.
</p><!--l. 313--><p class="indent" >
The sensitivity of <a 
href="#glo:SLIPT">SLIPT</a> as a binary classifier of <a 
href="#glo:synthetic lethal">synthetic lethality</a> (as shown in Figure <a 
href="#x1-80001r1">3.13a<!--tex4ht:ref: fig:simulation_Apr15ROC:Perf --></a>) declined with higher numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes to detect, although this is somewhat mitigated by higher sample sizes. The minority of true <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners are more difficult to distinguish when there are more of them (with a weaker <a 
href="#glo:gene expression">expression</a> signal from each). While a reduction of the false positive rate could be achieved for moderate numbers of underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners, the number of partners to be detected in analyses of <a 
href="#glo:gene expression">expression</a> data is unknown. However, this simulation procedure is amenable to assessing the performance of <a 
href="#glo:SLIPT">SLIPT</a> across simulation parameters, <a 
href="#glo:graph">graph</a> structures, and comparisons to other approaches (as presented in Chapter <a 
href="#x1-1340006">6<!--tex4ht:ref: chap:simulation --></a>).
</p><!--l. 315--><p class="indent" >
<p>Not all of the genes detected by <a 
href="#glo:SLIPT">SLIPT</a> were true <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes but they were among the strongest candidates and <a 
href="#glo:SLIPT">SLIPT</a> had higher performance with fewer underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes to detect. These results support a focus on pathway analyses, in particular, the selection of pathways for further investigation. Pathway over-representation analysis was performed to detect functional groups recurrently detected by <a 
href="#glo:SLIPT">SLIPT</a> since individually detected gene candidates were not necessarily <a 
href="#glo:synthetic lethal">synthetic lethal</a>. The detection of functionally related genes (in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>) supports the role of a pathway in <a 
href="#glo:synthetic lethal">synthetic lethal</a> relationships. The use of pathway <a 
href="#glo:metagene">metagenes</a> can reduce the number of potential pathways, compared to genes, to help identify <a 
href="#glo:synthetic lethal">synthetic lethality</a>. These approaches were both applied in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a> to identify the <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways of <span class="cmti-12">CDH1</span>. Pathways are also more likely to replicate across experimental models, as demonstrated by <a 
href="#XDixon2008">Dixon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XDixon2008">2008</a>).</p>
</p><!--l. 317--><p class="indent" >
The <a 
href="#glo:ROC">ROC</a> curves showed that <a 
href="#glo:SLIPT">SLIPT</a> is subject to a near equal trade-off between sensitivity and specificity across threshold values (as shown in Figure <a 
href="#x1-80002r2">3.13b<!--tex4ht:ref: fig:simulation_Apr15ROC:ROC --></a>). The lower sensitivity and higher specificity with a binary classification (as shown in Figure <a 
href="#x1-80001r1">3.13a<!--tex4ht:ref: fig:simulation_Apr15ROC:Perf --></a>) results from stringent testing by <a 
href="#glo:SLIPT">SLIPT</a> with <a 
href="#glo:FDR">FDR</a> adjusted p-values. The area under these curves (<a 
href="#glo:AUROC">AUROC</a>) was used to compare statistical performance (as shown in Figure <a 
href="#x1-80003r3">3.13c<!--tex4ht:ref: fig:simulation_Apr15ROC:AUC --></a>), which had lower performance for more underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners, and higher performance for larger sample size in multivariate normal simulations. <a 
 id="x1-80005r76"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">3.3.2.1 </span> <a 
 id="x1-810001"></a>Multivariate Normal Simulation with Correlated Genes
</h5>
                                                                                    

<!--l. 324--><p class="indent" >
<a 
 id="x1-8100516"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 327--><p class="noindent" >
</p><!--l. 331--><p class="noindent" >
<img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_top(1)".png" alt="pict"  
 /><a 
 id="x1-81001r1"></a> <span class="cmr-10x-x-109">Input</span> <span class="cmr-10x-x-109">Σ</span> <span class="cmr-10x-x-109">matrix</span> <span class="cmr-10x-x-109">parameter</span><img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_top(2)".png" alt="pict"  
 /><a 
 id="x1-81002r2"></a> <span class="cmr-10x-x-109">Simulated</span> <span class="cmr-10x-x-109">correlation</span> <span class="cmr-10x-x-109">matrix</span>
</p><!--l. 338--><p class="noindent" >
<img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_top(4)".png" alt="pict"  
 /><a 
 id="x1-81003r3"></a> <span class="cmr-10x-x-109">Simulated</span> <a 
href="#glo:gene expression"><span class="cmr-10x-x-109">gene</span> <span class="cmr-10x-x-109">expression</span></a><img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_top(3)".png" alt="pict"  
 /><a 
 id="x1-81004r4"></a> <span class="cmr-10x-x-109">Simulated</span> <span class="cmr-10x-x-109">gene</span> <span class="cmr-10x-x-109">function</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.16: </span><span class="content"><strong><span class="cmbx-10x-x-109">Simulating </span><a 
href="#glo:gene expression"><span class="cmbx-10x-x-109">expression</span></a> <span class="cmbx-10x-x-109">with correlated gene blocks.</span></strong> <span class="cmr-10x-x-109">A Σ matrix (a) was</span> <span class="cmr-10x-x-109">used generate 100 genes with a multivariate normal distribution, including correlated blocks</span> <span class="cmr-10x-x-109">of genes (</span><span class="cmmi-10x-x-109">r </span><span class="cmr-10x-x-109">= 0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">8) with correlation (b) similar to Σ, on a red–to–green scale. The annotation</span> <span class="cmr-10x-x-109">for genes gives the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">(in blue for in the direction of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">or red otherwise) and the gene</span> <span class="cmr-10x-x-109">category (blue for query, cyan for query-correlated, red for SL, orange for SL-correlated, forest</span> <span class="cmr-10x-x-109">green for non-synthetic lethal-correlated, and green for non-synthetic lethal). The simulated</span> <a 
href="#glo:gene expression"><span class="cmr-10x-x-109">gene expression</span></a> <span class="cmr-10x-x-109">(c) and function (d) generated were ordered by </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">showing the functional</span> <span class="cmr-10x-x-109">structure of </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">genes and that they were among the strongest </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">results.</span></span></p>
</div>
<!--tex4ht:label?: x1-8100516 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 349--><p class="indent" >
<a 
 id="x1-8101018"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 352--><p class="noindent" >
</p><!--l. 357--><p class="noindent" >
<img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_1K_top(1)".png" alt="pict"  
 /><a 
 id="x1-81006r1"></a> <span class="cmr-10x-x-109">Input</span> <span class="cmr-10x-x-109">Σ</span> <span class="cmr-10x-x-109">matrix</span> <span class="cmr-10x-x-109">parameter</span><img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_1K_top(2)".png" alt="pict"  
 /><a 
 id="x1-81007r2"></a> <span class="cmr-10x-x-109">Simulated</span> <span class="cmr-10x-x-109">correlation</span> <span class="cmr-10x-x-109">matrix</span>
</p><!--l. 366--><p class="noindent" >
<img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_1K_top(4)".png" alt="pict"  
 /><a 
 id="x1-81008r3"></a> <span class="cmr-10x-x-109">Simulated</span> <a 
href="#glo:gene expression"><span class="cmr-10x-x-109">gene</span> <span class="cmr-10x-x-109">expression</span></a><img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_1K_top(3)".png" alt="pict"  
 /><a 
 id="x1-81009r4"></a> <span class="cmr-10x-x-109">Simulated</span> <span class="cmr-10x-x-109">gene</span> <span class="cmr-10x-x-109">function</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.18: </span><span class="content"><strong><span class="cmbx-10x-x-109">Simulating </span><a 
href="#glo:gene expression"><span class="cmbx-10x-x-109">expression</span></a> <span class="cmbx-10x-x-109">with correlated gene blocks.</span></strong> <span class="cmr-10x-x-109">Using the Σ</span> <span class="cmr-10x-x-109">matrix</span><span class="cmr-10x-x-109"> (a), sampling 1000 genes from a multivariate normal distribution produced (b)</span> <span class="cmr-10x-x-109">correlated blocks of genes (correlated by 0.8) on a red–to–green scale. The simulated </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">gene</span> <span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">(c) and function (d) generated were ordered by </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">and </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">direction show that</span> <a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">genes are among the strongest </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">results with high specificity against</span> <span class="cmr-10x-x-109">many potential false positives. These are annotated for log-</span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">(on a red–to–green scale) and</span> <span class="cmr-10x-x-109">category (blue for query, cyan for query-correlated, red for SL, orange for SL-correlated, forest</span> <span class="cmr-10x-x-109">green for non-synthetic lethal-correlated, and green for non-synthetic lethal) for each gene.</span></span></p>
</div>
<!--tex4ht:label?: x1-8101018 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 378--><p class="noindent" >
Correlation structures were added to the simulation procedure (with the Σ matrix, as discussed in Section <a 
href="#x1-750002">3.2<!--tex4ht:ref: methods:simulation_SL_expression --></a>), using correlated blocks of genes (as shown in Figure <a 
href="#x1-81001r1">3.15a<!--tex4ht:ref: fig:simulation_May4SL:first --></a>). These correlated blocks represent genes with correlated <a 
href="#glo:gene expression">expression</a>, such as co-regulation or shared membership of biological pathways. The example (as shown in Figure <a 
href="#x1-8100516">3.16<!--tex4ht:ref: fig:simulation_May4SL --></a>) shows four <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes (out of 100), each with five correlated genes that are not themselves <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of the query gene. These simulations address whether <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes are distinguishable from correlated partners. The Σ matrix produced a similar correlation structure (Figure <a 
href="#x1-81002r2">3.15b<!--tex4ht:ref: fig:simulation_May4SL:second --></a>) and <a 
href="#glo:gene expression">expression</a> profiles (Figure <a 
href="#x1-81003r3">3.15c<!--tex4ht:ref: fig:simulation_May4SL:third --></a>). Apart from correlated blocks of genes (<span class="cmmi-12">r </span>= 0<span class="cmmi-12">.</span>8), the remaining genes had small variations due to random sampling. The structure of the dataset, particularly between <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes and the query, was evident in the simulated <a 
href="#glo:gene expression">gene expression</a> (Figure <a 
href="#x1-81003r3">3.15c<!--tex4ht:ref: fig:simulation_May4SL:third --></a>) and function (Figure <a 
href="#x1-81004r4">3.15d<!--tex4ht:ref: fig:simulation_May4SL:fifth --></a>). When these genes were ordered by the <a 
href="#glo:SLIPT">SLIPT</a> results, the <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes were highly ranked, a the majority of them were distinguishable from highly correlated genes.
</p><!--l. 380--><p class="indent" >
<p>The use of correlation structure was applied to larger datasets, such as the 1000 genes shown in Figure <a 
href="#x1-8101018">3.18<!--tex4ht:ref: fig:simulation_May4SL1K --></a>. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> genes were highly ranked by <a 
href="#glo:SLIPT">SLIPT</a> and were often distinguishable from correlated genes. As previously discussed in Section <a 
href="#x1-800002">3.3.2<!--tex4ht:ref: chapt2:simulation_mvtnorm --></a>, these <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes were still detectable among a larger number of non-synthetic lethal genes, and the <a 
href="#glo:SLIPT">SLIPT</a> methodology performed better on large datasets.</p>
</p><!--l. 383--><p class="indent" >
<a 
 id="x1-8101320"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<pre><code>                                                       &lt;img </code></pre>
src=“”SL_Model_May15mvnorm_heatmap_10XSL_cor_comp2(1)“.png” alt=“pict”<br /> /&gt;<a 
 id="x1-81011r1"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">category</span> <span class="cmr-10x-x-109">in</span> <span class="cmr-10x-x-109">simulations</span><img 
src=""SL_Model_May15mvnorm_heatmap_10XSL_cor_comp2(2)".png" alt="pict"  
 /><a 
 id="x1-81012r2"></a> <span class="cmr-10x-x-109">Corresponding</span> <span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">values</span>
<div class="caption">
<p><span class="id">Figure 3.20: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethal prediction across simulations.</span></strong> <span class="cmr-10x-x-109">The gene category (a)</span> <span class="cmr-10x-x-109">ordered by </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span> </sup> <span class="cmr-10x-x-109">and the </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">directional condition is shown across simulations (blue for query,</span> <span class="cmr-10x-x-109">cyan for query-correlated, red for SL, orange for SL-correlated, forest green for non-synthetic</span> <span class="cmr-10x-x-109">lethal-correlated, and green for non-synthetic lethal). For each number (1–10) of </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic</span> <span class="cmr-10x-x-109">lethal</span></a> <span class="cmr-10x-x-109">partners, 10 simulations show that the increasing numbers of </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners</span> <span class="cmr-10x-x-109">became harder detect (i.e., red cells become interspersed in the columns of (a)). The log-</span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">values (b) showed a threshold for </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">and correlated genes when there are fewer</span> <span class="cmr-10x-x-109">of them, distinguishable from correlated genes in this case.</span></span></p>
</div>
<!--tex4ht:label?: x1-8101320 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 406--><p class="indent" >
<a 
 id="x1-8101722"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 409--><p class="noindent" >
</p><!--l. 414--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150410y/SL_Model_Apr15mvnorm_1K_ROC1_samplesx_prop".png  /><a 
 id="x1-81014r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150410y/SL_Model_Apr15mvnorm_1K_ROC2_samplesx_prop".png  /><a 
 id="x1-81015r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a> <span class="cmr-10x-x-109">curves</span>
</p><!--l. 423--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150410y/SL_Model_Apr15mvnorm_1K_AUC_samplesx_prop".png  /><a 
 id="x1-81016r3"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span> <span class="cmr-10x-x-109">(</span><a 
href="#glo:AUROC"><span class="cmr-10x-x-109">AUROC</span></a><span class="cmr-10x-x-109">)</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.22: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance with correlations.</span></strong> <span class="cmr-10x-x-109">Simulation of </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a> <span class="cmr-10x-x-109">was</span> <span class="cmr-10x-x-109">performed by sampling from a multivariate normal distribution (with correlation structure).</span> <span class="cmr-10x-x-109">Performance of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">declines for more </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners but this is mitigated by</span> <span class="cmr-10x-x-109">increased sample sizes (darker colours). This generally occurs as the sensitivity decreases for</span> <span class="cmr-10x-x-109">a greater number of true positives to detect, leading to a trade-off in accuracy as seen in a</span> <span class="cmr-10x-x-109">trough for false positive rate and the </span><a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a> <span class="cmr-10x-x-109">curves.</span></span></p>
</div>
<!--tex4ht:label?: x1-8101722 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 432--><p class="indent" >
These plots (Figures <a 
href="#x1-8100516">3.16<!--tex4ht:ref: fig:simulation_May4SL --></a> and <a 
href="#x1-8101018">3.18<!--tex4ht:ref: fig:simulation_May4SL1K --></a>) used similar correlated blocks with a non-synthetic lethal gene (true negative) and the query gene (which is not <a 
href="#glo:synthetic lethal">synthetic lethal</a> with itself). Neither of these were <a 
href="#glo:synthetic lethal">synthetic lethal</a> but they could potentially affect performance methodology, particularly the specificity, as correlated non-synthetic lethal genes may be distinguishable from <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes. The non-synthetic lethal correlated block of genes had no impact on <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection but the query correlated genes were important (as shown in Sections <a 
href="#x1-820002">3.3.2.2<!--tex4ht:ref: chapt2:simulation_mvtnorm_query_cor --></a> and and <a 
href="#x1-1370001">6.1.1.1<!--tex4ht:ref: chapt5:compare_chisq_query_cor --></a>).
</p><!--l. 435--><p class="indent" >
<p>The simulations of <a 
href="#glo:gene expression">gene expression</a> data (with 100 genes) with correlations structure were used to examine the variation between detection in different samples and varying the number of underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners. A small number of simulations (10 for each) are shown to demonstrate the variation between replicate simulations from iterative sampling from the same multivariate normal distribution (as shown in Figure <a 
href="#x1-8101320">3.20<!--tex4ht:ref: fig:simulation_May4SLreps --></a>). These simulations showed that <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes were highly ranked by <a 
href="#glo:SLIPT">SLIPT</a> when there are few of them and these were relatively consistent across replicate simulations. However, they were less consistent for higher numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners to detect and were more difficult to distinguish from other genes, particularly those correlated with them. Similarly, the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values showed clear thresholds for <a 
href="#glo:synthetic lethal">synthetic lethal</a> and correlated genes in simple simulations but these were more gradual for higher numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners.</p>
</p><!--l. 437--><p class="indent" >
<a 
 id="x1-8101823"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 439--><p class="noindent" >
</p><!--l. 440--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150410y/SL_Model_Test_Graph_10K_Graph1_ROC_Compare_Mvtn(Apry)_v_Cor(Aprxy)_Full".png  />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.23: </span><span class="content"><strong><span class="cmbx-10x-x-109">Comparison of statistical performance with correlation structure.</span></strong> <span class="cmr-10x-x-109">Multivariate simulation of </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a> <span class="cmr-10x-x-109">with correlation structure (in colour) has</span> <span class="cmr-10x-x-109">comparable performance to simulation without correlations (in greyscale) with known</span> <a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners across parameters.</span></span></p>
</div>
<!--tex4ht:label?: x1-8101823 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 453--><p class="indent" >
<a 
 id="x1-8102225"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 456--><p class="noindent" >
</p><!--l. 461--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150507y/SL_Model_May15mvnorm_dir_1K_ROC1_samplesx_prop".png  /><a 
 id="x1-81019r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150507y/SL_Model_May15mvnorm_dir_1K_ROC2_samplesx_prop".png  /><a 
 id="x1-81020r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a> <span class="cmr-10x-x-109">curves</span>
</p><!--l. 470--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150507y/SL_Model_May15mvnorm_dir_1K_AUC_samplesx_prop".png  /><a 
 id="x1-81021r3"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span> <span class="cmr-10x-x-109">(</span><a 
href="#glo:AUROC"><span class="cmr-10x-x-109">AUROC</span></a><span class="cmr-10x-x-109">)</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.25: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance with query correlations.</span></strong> <span class="cmr-10x-x-109">Simulation of </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a> <span class="cmr-10x-x-109">was performed by sampling from a multivariate normal distribution (with correlation</span> <span class="cmr-10x-x-109">structure including correlated genes with non-synthetic lethal and query genes). Performance</span> <span class="cmr-10x-x-109">of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">declined for more </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners and is mitigated by increased sample</span> <span class="cmr-10x-x-109">sizes (darker colours) but the sensitivity remains higher for a greater number of true positives</span> <span class="cmr-10x-x-109">with corresponding improvements in </span><a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a> <span class="cmr-10x-x-109">curves.</span></span></p>
</div>
<!--tex4ht:label?: x1-8102225 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 478--><p class="indent" >
While the <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes were detected in simple simulations (as shown in Figure <a 
href="#x1-8101320">3.20<!--tex4ht:ref: fig:simulation_May4SLreps --></a>), <a 
href="#glo:ROC">ROC</a> analysis was performed to determine whether they were robustly detectable and to make further comparisons. These results (as shown in Figure <a 
href="#x1-8101722">3.22<!--tex4ht:ref: fig:simulation_Apr15ROC2 --></a>) were similar to simulations without correlation structure. As a binary classifier, <a 
href="#glo:SLIPT">SLIPT</a> had low sensitivity for higher numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners to detect and high specificity with the vast majority of non-synthetic lethal genes (for 20,000 genes). This was reflected in a similar reduction in statistical performance for higher numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners and a higher performance with higher sample size. Overall, the statistical performance was no different to simulations without correlation structure (as shown in Figure <a 
href="#x1-8101823">3.23<!--tex4ht:ref: fig:mvtnorm_cor_compare --></a>).
</p><!--l. 480--><p class="indent" >
<a 
href="#glo:SLIPT">SLIPT</a> was robust across correlation structures and is applicable to gene expression data, with <a 
href="#glo:graph">pathway</a> structures and correlations. These correlation structures were not intended to model specific biological pathways or represent them but showed potential impact of correlation structure on the performance of <a 
href="#glo:SLIPT">SLIPT</a> using highly correlated (<span class="cmmi-12">r </span>= 0<span class="cmmi-12">.</span>8) gene blocks. More complex correlation structures, such as genes positively correlated with the query gene and derived from pathway <a 
href="#glo:graph">graph</a> structures (as described in <a 
href="#x1-860002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>) were examined further in Sections <a 
href="#x1-820002">3.3.2.2<!--tex4ht:ref: chapt2:simulation_mvtnorm_query_cor --></a> and <a 
href="#x1-1420001">6.2.1<!--tex4ht:ref: chapt5:graphsim_performance --></a> respectively.
</p><!--l. 482--><p class="indent" >
In particular, genes correlated with true <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes had little impact on the performance of <a 
href="#glo:SLIPT">SLIPT</a> detection: <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes were as distinguishable from correlated genesas as they are from true negative genes. Genes correlated with <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners did not interfere with the detection of true <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes, although they were often ranked next below them and may support <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways by having related gene functions. <a 
 id="x1-81023r120"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">3.3.2.2 </span> <a 
 id="x1-820002"></a>Specificity with Query-Correlated Pathways
</h5>
<!--l. 488--><p class="noindent" >
<p>Correlation structures were also considered for non-synthetic lethal genes that were (positively) correlated genes with the query gene. Specifically, five highly correlated (<span class="cmmi-12">r </span>= 0<span class="cmmi-12">.</span>8) with the query gene were added (as described in Section <a 
href="#x1-810001">3.3.2.1<!--tex4ht:ref: chapt2:simulation_mvtnorm_cor --></a>). These simulations had similar performance (as shown in Figure <a 
href="#x1-8102225">3.25<!--tex4ht:ref: fig:simulation_May15ROC2 --></a>) to those without these correlations with a higher specificity and a lower false positive rate (shown in Figure <a 
href="#x1-81019r1">3.24a<!--tex4ht:ref: fig:simulation_May15ROC:Perf --></a>).</p>
</p><!--l. 511--><p class="indent" >
<p>The directional criteria of the <a 
href="#glo:SLIPT">SLIPT</a> procedure was important in this case, enhancing its performance, particularly in distinguishing positively correlated non-synthetic lethal genes. The multivariate normal simulations were performed, with 20,000 genes, including all of the correlation structures discussed (with synthetic lethal, non-synthetic lethal, and query correlated genes). These simulations were compared for the direction <a 
href="#glo:SLIPT">SLIPT</a> and the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> testing. There was a considerably higher statistical performance with <a 
href="#glo:SLIPT">SLIPT</a>, particularly increased sensitivity and lower false positive rate (as shown in Figure <a 
href="#x1-8200327">3.27<!--tex4ht:ref: fig:mvtnorm_dir_compare --></a>).</p>
</p><!--l. 513--><p class="indent" >
<a 
 id="x1-8200327"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 516--><p class="noindent" >
</p><!--l. 523--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150507y/SL_Model_May15mvnorm_1K_ROC1_samplesx_prop".png  /><a 
 id="x1-82001r1"></a> <span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span> </sup> <span class="cmr-10x-x-109">testing</span> <span class="cmr-10x-x-109">without</span> <span class="cmr-10x-x-109">direction</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150507y/SL_Model_May15mvnorm_dir_1K_ROC1_samplesx_prop".png  /><a 
 id="x1-82002r2"></a> <a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">directional</span> <span class="cmr-10x-x-109">criteria</span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.27: </span><span class="content"><strong><span class="cmbx-10x-x-109">Statistical evaluation of directional criteria.</span></strong> <span class="cmr-10x-x-109">A simulated multivariate</span> <span class="cmr-10x-x-109">normal dataset of 20,000 genes with correlation structures was tested by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">with the</span> <span class="cmr-10x-x-109">directional condition and the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test. </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">exhibited a consistently higher sensitivity and</span> <span class="cmr-10x-x-109">lower false positive rate.</span></span></p>
</div>
<!--tex4ht:label?: x1-8200327 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 538--><p class="indent" >
<a 
 id="x1-8200729"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 541--><p class="noindent" >
</p><!--l. 548--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150507y/SL_Model_May15mvnorm_1K_ROC2_samplesx_prop".png  /><a 
 id="x1-82004r1"></a> <span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span> </sup> <span class="cmr-10x-x-109">testing</span> <span class="cmr-10x-x-109">without</span> <span class="cmr-10x-x-109">direction</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150507y/SL_Model_May15mvnorm_dir_1K_ROC2_samplesx_prop".png  /><a 
 id="x1-82005r2"></a> <a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">directional</span> <span class="cmr-10x-x-109">criteria</span>
</p><!--l. 560--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20150507y/SL_Model_Test_Graph_10K_Graph1_ROC_Compare_Mvtn(Mayy)_v_Cor(Mayxy)_Full".png  /><a 
 id="x1-82006r3"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span> <span class="cmr-10x-x-109">(</span><a 
href="#glo:AUROC"><span class="cmr-10x-x-109">AUROC</span></a><span class="cmr-10x-x-109">)</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.29: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance with directional criteria.</span></strong> <span class="cmr-10x-x-109">A simulated multivariate normal</span> <span class="cmr-10x-x-109">dataset of 20,000 genes with correlation structures was tested by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test. </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">had higher performance across simulation parameters, clearly differing from random (grey</span> <span class="cmr-10x-x-109">diagonal) in </span><a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a> <span class="cmr-10x-x-109">curves up to 100 SL genes (b). The performance (c) of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">(in greyscale)</span> <span class="cmr-10x-x-109">was consistently higher than the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test (in color).</span></span></p>
</div>
<!--tex4ht:label?: x1-8200729 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 569--><p class="indent" >
These results show that the performance of <a 
href="#glo:SLIPT">SLIPT</a> is appropriate for the analysis of <a 
href="#glo:gene expression">expression</a> datasets, where positively correlated genes commonly occur, with the directional condition robustly improving the performance of <a 
href="#glo:SLIPT">SLIPT</a> across simulation parameters (compared to the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test). Without assuming the underlying number of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes, <a 
href="#glo:SLIPT">SLIPT</a> will performed than the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test alone, irrespective of the significance threshold as shown by <a 
href="#glo:ROC">ROC</a> analysis (as shown in Figure <a 
href="#x1-8200729">3.29<!--tex4ht:ref: fig:mvtnorm_dir_compare2 --></a>). The directional <a 
href="#glo:SLIPT">SLIPT</a> methodology outperformed the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test at detecting <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners with even up to 100 <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes.
</p><!--l. 571--><p class="indent" >
Together these simulation results support the application of the <a 
href="#glo:SLIPT">SLIPT</a> methodology as it has been performed throughout Chapters <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a> and <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>. The methodology and simulation procedure were explored further in Chapter <a 
href="#x1-1340006">6<!--tex4ht:ref: chap:simulation --></a>, with comparison to other <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection approaches and the inclusion of <a 
href="#glo:graph">graph</a> structures. <a 
 id="x1-82008r111"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">3.4 </span> <a 
 id="x1-830004"></a>Graph Structure Methods
</h3>
<!--l. 576--><p class="noindent" >
Graph structures have been used in several ways in this project, including novel approaches to analysis and simulations. Procedures were developed for statistical and network analysis of gene states in <a 
href="#glo:graph">pathway</a> structures. Specifically, the relationships between <a 
href="#glo:siRNA">siRNA</a> and <a 
href="#glo:SLIPT">SLIPT</a> genes were tested within biological pathways in Chapter <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>. These <a 
href="#glo:graph">graph</a> structures were also used in Chapter <a 
href="#x1-1340006">6<!--tex4ht:ref: chap:simulation --></a> to derive correlation structure between simulated <a 
href="#glo:gene expression">gene expression</a> profiles to represent biological pathways. <a 
 id="x1-83001r115"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.4.1 </span> <a 
 id="x1-840001"></a>Upstream and Downstream Gene Detection
</h4>
<!--l. 580--><p class="noindent" >
Comparison of experimental and computational candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes within <a 
href="#glo:graph">pathway</a> structures was performed to determine whether these sets of genes were related by <a 
href="#glo:graph">pathway</a> structure. Considering the differences in how these candidates were generated, it was unsurprising that they did not detect some identical genes within the candidate biological pathways. However, they could still be related by being upstream or downstream of each other.
</p><!--l. 582--><p class="indent" >
<p>Using the Reactome version 52 (<a 
href="#XReactome">Croft <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XReactome">2014</a>), as described in Section <a 
href="#x1-660002">2.4.2<!--tex4ht:ref: methods:graph_data --></a>, genes detected by each <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery approach were mapped to the <a 
href="#glo:graph">graph</a> structure for each candidate pathway identified in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a> (with graphs defined as</p>
described in Section <a 
href="#x1-670003">2.4.3<!--tex4ht:ref: methods:subgraphs --></a>). To test whether <a 
href="#glo:siRNA">siRNA</a> candidate genes were upstream of <a 
href="#glo:SLIPT">SLIPT</a> candidate genes, <a 
href="#glo:shortest path">shortest paths</a> were traced between each pair of these genes in a directed network. The paths where the <a 
href="#glo:siRNA">siRNA</a> candidate was upstream (“up”) and downstream (“down”) of a <a 
href="#glo:SLIPT">SLIPT</a> candidate were scored. This procedure yielded the total number of <a 
href="#glo:shortest path">shortest paths</a> which indicated that <a 
href="#glo:siRNA">siRNA</a> genes were upstream or downstream of the <a 
href="#glo:SLIPT">SLIPT</a> genes and measured the difference between these to determine if there was an imbalance in a particular direction. While this difference was indicative of the number of paths between the gene candidate groups in either direction, it was not sufficient to statistically verify structure or relationships between <a 
href="#glo:siRNA">siRNA</a> and <a 
href="#glo:SLIPT">SLIPT</a> genes. It was be combined with a permutation resampling procedure (as described in Section <a 
href="#x1-850001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>) to test for directional relationships in either direction.
</p><!--l. 584--><p class="indent" >
Initially, this procedure excluded genes that were detected by both approaches since they would count in both directions. Upon further consideration, these genes were restored to accounted for since they may contribute unequally to each gene set if there are unequal numbers of genes above or below them in the <a 
href="#glo:graph">pathway</a> structure. <a 
 id="x1-84001r143"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">3.4.1.1 </span> <a 
 id="x1-850001"></a>Permutation Analysis for Statistical Significance
</h5>
<!--l. 587--><p class="noindent" >
A permutation procedure was developed to randomly assign members of the pathway to <a 
href="#glo:siRNA">siRNA</a> and/or <a 
href="#glo:SLIPT">SLIPT</a> groups, with the same number of each candidate partner gene set as observed in the pathway. These permuted genes were measured for <a 
href="#glo:graph">pathway</a> structure between the permuted gene groups as performed for the observed candidates (as performed in Section <a 
href="#x1-840001">3.4.1<!--tex4ht:ref: methods:pathway_str --></a>). A distribution of <a 
href="#glo:graph">pathway</a> structure relationships expected by chance was generated by permuting iteratively over these pathways. The resulting null distribution was compared to the observed counts of relationships (in either direction). This procedure yielded a permutation p-value as the proportion of permutations in which had a value greater than the observed value. The null hypothesis was that there was no relationship between these gene groups compared to genes that had been selected at random. Thus both the alternate hypotheses that the <a 
href="#glo:siRNA">siRNA</a> genes were either upstream of the <a 
href="#glo:SLIPT">SLIPT</a> genes or that they are downstream of them were testable.
</p><!--l. 589--><p class="indent" >
<p>The permutation procedure does not assume the underlying distribution of the data under the null hypothesis and accounts for the total number of <a 
href="#glo:vertex">nodes</a>, <a 
href="#glo:edge">edges</a>, <a 
href="#glo:siRNA">siRNA</a>, and</p>
<a 
href="#glo:SLIPT">SLIPT</a> genes in each <a 
href="#glo:graph">graph or network</a>pathway structure. The number of genes detected by both <a 
href="#glo:siRNA">siRNA</a> and <a 
href="#glo:SLIPT">SLIPT</a> was not accounted for under the initial <a 
href="#glo:shortest path">shortest path</a> counts procedure that excluded them. Once they were included, it was ensured that the number of intersecting genes was equal to the number observed to test for <a 
href="#glo:graph">pathway</a> structure without changing the intersection size, the subject of prior analyses. <a 
 id="x1-85001r152"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.4.2 </span> <a 
 id="x1-860002"></a>Simulating Gene Expression from Graph Structures
</h4>
<!--l. 601--><p class="noindent" >
The simulation procedure was refined to generate <a 
href="#glo:gene expression">expression</a> data with correlation structure from a known <a 
href="#glo:graph">graph</a> structure. This enabled modelling of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners within a biological pathway and the investigation of the impact of <a 
href="#glo:graph">pathway</a> structure on <a 
href="#glo:synthetic lethal">synthetic lethal</a> prediction. Firstly, a simulated pathway was constructed as a <a 
href="#glo:graph">graph</a> structure, with the <span class="cmtt-12">igraph </span>R package <a 
href="#Xigraph">Csardi and Nepusz</a> (<a 
href="#Xigraph">2006</a>), with the state of the <a 
href="#glo:edge">edges</a> (i.e, whether they activate or inhibit downstream pathway members). This simulation procedure was intended for biological pathway members with correlated <a 
href="#glo:gene expression">gene expression</a> (higher than the background of genes in other pathways) but it may also be applicable to modelling protein levels (e.g, in a kinase regulation cascade) or substrates and products (e.g., in a metabolic pathway).
</p><!--l. 604--><p class="indent" >
<p>The <a 
href="#glo:graph">graph</a> structure was constructed from which simulated data will be generated from, sampling a multivariate normal distribution using the <span class="cmtt-12">mvtnorm </span>R package (<a 
href="#XGenz2009">Genz and Bretz</a>, <a 
href="#XGenz2009">2009</a>; <a 
href="#Xmvtnorm">Genz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xmvtnorm">2016</a>). Throughout this Section, the simulation procedure will be demonstrated with the relatively simple constructed <a 
href="#glo:graph">graph</a> structure shown in Figure <a 
href="#x1-8600331">3.31<!--tex4ht:ref: fig:simple_graph --></a>. This <a 
href="#glo:graph">graph</a> structure visualisation was specifically developed for (directed) iGraph objects in R and has been released in the <span class="cmtt-12">plot.igraph </span>package and <span class="cmtt-12">igraph.extensions </span>library (in Table <a 
href="#x1-710046">2.6<!--tex4ht:ref: tab:computers_r_packages_dev --></a> and Section <a 
href="#x1-900003">3.5.3<!--tex4ht:ref: methods:igraph_extensions --></a>). The <span class="cmtt-12">plot</span><span class="cmtt-12">_directed </span>function enabled customisation of plot parameters for each <a 
href="#glo:vertex">node</a> or <a 
href="#glo:edge">edge</a> and mixed (directed) <a 
href="#glo:edge">edge</a> types for indicating activation or inhibition. These inhibition <a 
href="#glo:edge">links</a> (which occur frequently in biological pathways) were demonstrated in Figure <a 
href="#x1-86002r2">3.30b<!--tex4ht:ref: fig:simple_graph:second --></a>.</p>
</p><!--l. 606--><p class="indent" >
<a 
 id="x1-8600331"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 609--><p class="noindent" >
</p><!--l. 615--><p class="noindent" >
<span class="fbox"><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/simple_graph".png  /></span><a 
 id="x1-86001r1"></a> <span class="cmr-10x-x-109">Activating</span> <a 
href="#glo:graph"><span class="cmr-10x-x-109">pathway</span></a> <span class="cmr-10x-x-109">structure</span><span class="fbox"><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/simple_graph_inhibiting".png  /></span><a 
 id="x1-86002r2"></a> <span class="cmr-10x-x-109">Pathway</span> <span class="cmr-10x-x-109">structure</span> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">inhibitions</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.31: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulated </span><a 
href="#glo:graph"><span class="cmbx-10x-x-109">graph</span></a> <span class="cmbx-10x-x-109">structures.</span></strong></strong> <span class="cmr-10x-x-109">A constructed </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure used as an</span> <span class="cmr-10x-x-109">example to demonstrate the simulation procedure. Activating </span><a 
href="#glo:edge"><span class="cmr-10x-x-109">links</span></a> <span class="cmr-10x-x-109">are denoted by blue arrows</span> <span class="cmr-10x-x-109">and inhibiting </span><a 
href="#glo:edge"><span class="cmr-10x-x-109">links</span></a> <span class="cmr-10x-x-109">by red </span><a 
href="#glo:edge"><span class="cmr-10x-x-109">edges</span></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-8600331 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 629--><p class="indent" >
<a 
 id="x1-8601033"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 632--><p class="noindent" >
</p><!--l. 636--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/simple_graph".png  /><a 
 id="x1-86004r1"></a> <span class="cmr-10x-x-109">Activating</span> <a 
href="#glo:graph"><span class="cmr-10x-x-109">pathway</span></a> <span class="cmr-10x-x-109">structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/dist_mat".png  /><a 
 id="x1-86005r2"></a> <span class="cmr-10x-x-109">Distance</span> <span class="cmr-10x-x-109">matrix</span>
</p><!--l. 643--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/sigma_mat".png  /><a 
 id="x1-86006r3"></a> <span class="cmr-10x-x-109">Σ</span> <span class="cmr-10x-x-109">(expected</span> <span class="cmr-10x-x-109">correlation)</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_cor_mat".png  /><a 
 id="x1-86007r4"></a> <span class="cmr-10x-x-109">Simulated</span> <span class="cmr-10x-x-109">correlation</span>
</p><!--l. 650--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_mat".png  /><a 
 id="x1-86008r5"></a> <span class="cmr-10x-x-109">Simulated</span> <a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">data</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_disc_mat".png  /><a 
 id="x1-86009r6"></a> <span class="cmr-10x-x-109">Simulated</span> <span class="cmr-10x-x-109">gene</span> <span class="cmr-10x-x-109">function</span> <span class="cmr-10x-x-109">calls</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.33: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulating </span><a 
href="#glo:gene expression"><span class="cmbx-10x-x-109">expression</span></a> <span class="cmbx-10x-x-109">from a </span><a 
href="#glo:graph"><span class="cmbx-10x-x-109">graph</span></a> <span class="cmbx-10x-x-109">structure.</span></strong></strong> <span class="cmr-10x-x-109">An example </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure that was used to derive a correlation structure from the relative distances between</span> <a 
href="#glo:vertex"><span class="cmr-10x-x-109">nodes</span></a> <span class="cmr-10x-x-109">and simulate continuous </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">gene expression</span></a> <span class="cmr-10x-x-109">with sampling from the multivariate normal</span> <span class="cmr-10x-x-109">distribution.</span></span></p>
</div>
<!--tex4ht:label?: x1-8601033 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 661--><p class="indent" >
<a 
 id="x1-8601735"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 664--><p class="noindent" >
</p><!--l. 668--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/simple_graph_inhibiting".png  /><a 
 id="x1-86011r1"></a> <span class="cmr-10x-x-109">Inhibiting</span> <span class="cmr-10x-x-109">pathway</span> <span class="cmr-10x-x-109">structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/dist_mat".png  /><a 
 id="x1-86012r2"></a> <span class="cmr-10x-x-109">Distance</span> <span class="cmr-10x-x-109">matrix</span>
</p><!--l. 675--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/sigma_mat_inhibiting".png  /><a 
 id="x1-86013r3"></a> <span class="cmr-10x-x-109">Σ</span> <span class="cmr-10x-x-109">(expected</span> <span class="cmr-10x-x-109">correlation)</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_cor_mat_inhibiting".png  /><a 
 id="x1-86014r4"></a> <span class="cmr-10x-x-109">Simulated</span> <span class="cmr-10x-x-109">correlation</span>
</p><!--l. 683--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_mat_inhibiting".png  /><a 
 id="x1-86015r5"></a> <span class="cmr-10x-x-109">Simulated</span> <a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">data</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_inhib_disc_mat".png  /><a 
 id="x1-86016r6"></a> <span class="cmr-10x-x-109">Simulated</span> <span class="cmr-10x-x-109">gene</span> <span class="cmr-10x-x-109">function</span> <span class="cmr-10x-x-109">calls</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.35: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulating </span><a 
href="#glo:gene expression"><span class="cmbx-10x-x-109">expression</span></a> <span class="cmbx-10x-x-109">from </span><a 
href="#glo:graph"><span class="cmbx-10x-x-109">graph</span></a> <span class="cmbx-10x-x-109">structure with inhibitions.</span></strong></strong> <span class="cmr-10x-x-109">An</span> <span class="cmr-10x-x-109">example </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure that was used to derive a correlation structure from the relative</span> <span class="cmr-10x-x-109">distances between </span><a 
href="#glo:vertex"><span class="cmr-10x-x-109">nodes</span></a> <span class="cmr-10x-x-109">and simulate continuous </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">gene expression</span></a> <span class="cmr-10x-x-109">with sampling from the</span> <span class="cmr-10x-x-109">multivariate normal distribution.</span></span></p>
</div>
<!--tex4ht:label?: x1-8601735 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 694--><p class="indent" >
The simulation procedure was designed to use such <a 
href="#glo:graph">graph</a> structures to inform development of a “Sigma” variance-covariance matrix (Σ) for sampling from a multivariate normal distribution (using the <span class="cmtt-12">mvtnorm </span>R package). Given a <a 
href="#glo:graph">graph</a> structure (or adjacency matrix), such as Figure <a 
href="#x1-86004r1">3.32a<!--tex4ht:ref: fig:simulation_activating:first --></a>, a relation matrix was calculated based on distance such that nearer <a 
href="#glo:vertex">nodes</a> are given higher weight than farther <a 
href="#glo:vertex">nodes</a>. Throughout this thesis, a geometrically decreasing (relative) distance weighting was used, with each more distant <a 
href="#glo:vertex">node</a> being related by <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">2</span> compared to the next nearest, as shown in Figure <a 
href="#x1-86005r2">3.32b<!--tex4ht:ref: fig:simulation_activating:second --></a>. An arithmetically decreasing (absolute) distance weighting is also supported in the <span class="cmtt-12">graphsim </span>R package release of this procedure.
</p><!--l. 696--><p class="indent" >
A Σ matrix can be derived from this distance weighting matrix, creating a matrix (with a diagonal of 1) where each <a 
href="#glo:vertex">node</a> has a variance and standard deviation of 1. Thus covariances between adjacent <a 
href="#glo:vertex">nodes</a> were assigned by a correlation parameter and the remaining matrix based on weighting these correlations by the geometrically weighted distance matrix (or the nearest “positive definite” matrix for Σ weighted for negatively correlated inhibitions). Throughout this thesis, the correlation parameter was 0<span class="cmmi-12">.</span>8, unless otherwise specified (as used for the example in Figure <a 
href="#x1-86006r3">3.32c<!--tex4ht:ref: fig:simulation_activating:third --></a>). This Σ matrix was then used to sample from a multivariate normal distribution such that each gene had a mean of 0, standard deviation 1, and covariance within the range [0<span class="cmmi-12">, </span>1] such that they are correlations. This procedure generated a simulated (continuous normally distributed) <a 
href="#glo:gene expression">expression</a> profile for each <a 
href="#glo:vertex">node</a> (as shown in Figure <a 
href="#x1-86008r5">3.32e<!--tex4ht:ref: fig:simulation_activating:fourth --></a>) with corresponding correlation structure (Figure <a 
href="#x1-86007r4">3.32d<!--tex4ht:ref: fig:simulation_activating:fifth --></a>). The simulated correlation structure closely resembled the expected correlation structure (Sigma in <a 
href="#x1-86006r3">3.32c<!--tex4ht:ref: fig:simulation_activating:third --></a>) even for the relatively modest sample size (<span class="cmmi-12">N </span>= 100) illustrated in <a 
href="#x1-8601033">3.33<!--tex4ht:ref: fig:simulation_activating --></a>. Once a simulated <a 
href="#glo:gene expression">gene expression</a> dataset has been generated (as in Figure <a 
href="#x1-86008r5">3.32e<!--tex4ht:ref: fig:simulation_activating:fourth --></a>), then a discrete matrix of gene function was constructed with a functional threshold quantile to simulate functional relationships of <a 
href="#glo:synthetic lethal">synthetic lethality</a> (as shown in Figure <a 
href="#x1-760024">3.4<!--tex4ht:ref: fig:SL_Model_Expression --></a>). Throughout this thesis, this threshold is the 0.3 quantile (as discussed in Section <a 
href="#x1-760001">3.2.1<!--tex4ht:ref: methods:SL_Model --></a>) which generates functional discrete matrices such as those used for <a 
href="#glo:synthetic lethal">synthetic lethal</a> simulation in Section <a 
href="#x1-770002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a> (as shown Figure <a 
href="#x1-86009r6">3.32f<!--tex4ht:ref: fig:simulation_activating:sixth --></a>).
</p><!--l. 698--><p class="indent" >
<p>The simulation procedure (depicted in Figure <a 
href="#x1-8601033">3.33<!--tex4ht:ref: fig:simulation_activating --></a>) can be used for pathways containing inhibition <a 
href="#glo:edge">links</a> (as shown in Figure <a 
href="#x1-8601735">3.35<!--tex4ht:ref: fig:simulation_inhibiting --></a>) with several refinements. With the inhibition <a 
href="#glo:edge">links</a> (as shown in Figure <a 
href="#x1-86011r1">3.34a<!--tex4ht:ref: fig:simulation_inhibiting:first --></a>), distances were calculated in the same manner as before (Figure <a 
href="#x1-86012r2">3.34b<!--tex4ht:ref: fig:simulation_inhibiting:second --></a>) with inhibitions accounted for by iteratively multiplying downstream <a 
href="#glo:vertex">nodes</a> by <span class="cmsy-10x-x-120">-</span>1 to form blocks of negative correlations (as shown in</p>
Figures <a 
href="#x1-86013r3">3.34c<!--tex4ht:ref: fig:simulation_inhibiting:third --></a> and <a 
href="#x1-86014r4">3.34d<!--tex4ht:ref: fig:simulation_inhibiting:fifth --></a>). A multivariate normal distribution with these negative correlations can be sampled to generate simulated data (as shown in Figures <a 
href="#x1-86015r5">3.34e<!--tex4ht:ref: fig:simulation_inhibiting:fourth --></a> and <a 
href="#x1-86016r6">3.34f<!--tex4ht:ref: fig:simulation_inhibiting:sixth --></a>).
</p><!--l. 700--><p class="indent" >
These simulated datasets could then be used for simulating <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of a query gene within a graph network. The query gene was assumed to be separate from the graph network pathway and was added to the dataset using the procedure in Section <a 
href="#x1-770002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a>. Thus we can simulate known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes within a <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner <a 
href="#glo:graph">pathway</a> structure. <a 
 id="x1-86018r151"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">3.5 </span> <a 
 id="x1-870005"></a>Customised Functions and Packages Developed
</h3>
<!--l. 723--><p class="noindent" >
Various R packages (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>) have been developed throughout this thesis using <span class="cmtt-12">devtools </span>(<a 
href="#Xdevtools">Wickham and Chang</a>, <a 
href="#Xdevtools">2016</a>) and <span class="cmtt-12">roxygen </span>(<a 
href="#Xroxygen">Wickham <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xroxygen">2017</a>) to enable reproducibility of customised analysis and visualisation. Many of these have been documented, demonstrated in vignettes, and released on GitHub (<a 
href="https://github.com/TomKellyGenetics" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics</span></a>) to enable the research community to utilise them in their own analysis. These are summarised in Table <a 
href="#x1-710046">2.6<!--tex4ht:ref: tab:computers_r_packages_dev --></a>, with the corresponding urls for their GitHub repository which contains instructions for installation with the <span class="cmtt-12">devtools </span>R package (<a 
href="#Xdevtools">Wickham and Chang</a>, <a 
href="#Xdevtools">2016</a>) and links the relevant vignette(s). <a 
 id="x1-87001r154"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.5.1 </span> <a 
 id="x1-880001"></a>Synthetic Lethal Interaction Prediction Tool
</h4>
<!--l. 728--><p class="noindent" >
The statistical methodology for detection of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in <a 
href="#glo:gene expression">gene expression</a> data (<a 
href="#glo:SLIPT">SLIPT</a>) is one of the main novel procedures developed in this thesis, as described in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>. The <span class="cmtt-12">slipt </span>R package has been prepared for release to accompany a publication demonstrating the applications of the methodology for identifying candidate interacting genes and pathways with <span class="cmti-12">CDH1 </span>in breast cancer (<a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>).
</p><!--l. 731--><p class="indent" >
<p><a 
href="#glo:SLIPT">SLIPT</a> can be used amenable to analysis of any effectively continuous measure of gene activity (e.g., <a 
href="#glo:microarray">microarray</a>, <a 
href="#glo:RNA-Seq">RNA-Seq</a>, protein abundance, or pathway <a 
href="#glo:metagene">metagenes</a>). Executing <span class="cmtt-12">slipt </span>is straightforward: the <span class="cmtt-12">prep</span><span class="cmtt-12"><em>data</span><span 
class="cmtt-12"></em>for</span><span class="cmtt-12"><em>SL </span>function scores samples as “low”, “medium”, or “high” for each gene, then the <span class="cmtt-12">detect</span><span 
class="cmtt-12"></em>SL </span>function tests a given query gene against all potential partners by performing the chi-squared</p>
test and directional conditions. This function returns a table summarising the observed and expected sample numbers used for the directional criteria, the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values, and corresponding p-values including adjusting for multiple comparisons. The <span class="cmtt-12">count</span><span class="cmtt-12"><em>of</span><span 
class="cmtt-12"></em>SL </span>and <span class="cmtt-12">table</span><span class="cmtt-12"><em>of</span><span 
class="cmtt-12"></em>SL </span>functions serve to facilitate summary and extraction of the positive <a 
href="#glo:SLIPT">SLIPT</a> hits, respectively, from the table of predictions of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners.
</p><!--l. 733--><p class="indent" >
The <a 
href="#glo:SLIPT">SLIPT</a> methodology in this package release was used in later analyses rather than the corresponding source R code, including use on remote machines and upon simulated data. In particular, the functions in the package facilitate alterations to parameters, such as the proportion of samples called as exhibiting low or high gene activity (as shown in Section <a 
href="#x1-1360001">6.1.1<!--tex4ht:ref: chapt5:compare_chisq --></a>). This release supports reproducible research and enables wider use of <a 
href="#glo:SLIPT">SLIPT</a> in future investigations into other disease genes. <a 
 id="x1-88001r173"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.5.2 </span> <a 
 id="x1-890002"></a>Data Visualisation
</h4>
<!--l. 736--><p class="noindent" >
Customised data visualisations in R (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>) were developed to present data throughout this thesis. The <span class="cmtt-12">vioplotx </span>and <span class="cmtt-12">heatmap.2x </span>packages are enhancements of the <span class="cmtt-12">vioplot </span>package (<a 
href="#Xvioplot">Adler</a>, <a 
href="#Xvioplot">2005</a>) and <span class="cmtt-12">heatmap.2 </span>provided by the <span class="cmtt-12">gplots </span>package (<a 
href="#Xgplots">Warnes <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xgplots">2015</a>).
</p><!--l. 738--><p class="indent" >
The <span class="cmtt-12">vioplotx </span>package provides an alternative visualisation (of continuous variables against categories) to the more familiar boxplot, showing variability of the data by the width of the plots. As demonstrated in Figure <a 
href="#x1-8900337">3.37<!--tex4ht:ref: fig:vioplot --></a>, this version enables separate plotting parameters for each violin with vector inputs for colour, shape, and size of various elements of the median point, central boxplot, borders, and fill colour for the violin. Scaling violin width to adjust violin area and splitting data by a second categorical variable is also enabled. This function is intended to be backwards compatible with the inputs of <span class="cmtt-12">vioplot </span>(applying scalar inputs across all violins) and <span class="cmtt-12">boxplot</span> (by enabling formula inputs as an S3 method). Each of these features has been demonstrated with examples in respective vignettes on the package <a 
href="https://github.com/TomKellyGenetics/vioplotx" >GitHub repository</a> (<a 
href="https://github.com/TomKellyGenetics/vioplotx" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/vioplotx</span></a>).
</p><!--l. 740--><p class="indent" >
<p>The <span class="cmtt-12">heatmap.2x </span>function provides extensions for annotation colour bars for both the rows and columns (as shown in Figure <a 
href="#x1-8900438">3.38<!--tex4ht:ref: fig:heatmap.2x --></a>). Multiple bars are enabled on both axes with matrix inputs (rather than single vector for <span class="cmtt-12">heatmap.2</span>) which facilitates additional plotting of gene and sample characteristics for comparison with correlation matrices, <a 
href="#glo:gene expression">expression</a></p>
<p>profiles, or pathway <a 
href="#glo:metagene">metagenes</a>. The annotation bar inputs correspond to their orientation on the plot, each colour bar is provided as a column for the row annotation on the left of the heatmap and as a row for the column annotation on top of the heatmap. Row and column annotation bars are labelled with the column or row names respectively. Additional parameters enable resizing of these annotation bar labels and control of reordering columns for when samples have been ordered in advance (e.g., ranked by a <a 
href="#glo:metagene">metagene</a> or split into groups clustered separately). These features were used through this thesis and have been provided in a package <a 
href="https://github.com/TomKellyGenetics/heatmap.2x" >GitHub repository</a> (<a 
href="https://github.com/TomKellyGenetics/heatmap.2x" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/heatmap.2x</span></a>).</p>
</p><!--l. 742--><p class="indent" >
<a 
 id="x1-8900337"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 745--><p class="noindent" >
</p><!--l. 750--><p class="noindent" >
<img 
src=""vioplot1".png" alt="PIC"  
 /><a 
 id="x1-89001r1"></a> <span class="cmr-10x-x-109">Customised</span> <span class="cmr-10x-x-109">violin</span> <span class="cmr-10x-x-109">plot</span><img 
src=""vioplot2".png" alt="PIC"  
 /><a 
 id="x1-89002r2"></a> <span class="cmr-10x-x-109">Split</span> <span class="cmr-10x-x-109">violin</span> <span class="cmr-10x-x-109">plot</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.37: </span><span class="content"><strong><span class="cmbx-10x-x-109">Demonstration of violin plots with custom features.</span></strong> <span class="cmr-10x-x-109">An example of the</span> <span class="cmtt-10x-x-109">iris </span><span class="cmr-10x-x-109">dataset is plotted to show the custom features of the </span><span class="cmtt-10x-x-109">vioplotx </span><span class="cmr-10x-x-109">package including (a)</span> <span class="cmr-10x-x-109">individual colour, shape and size parameters of each violin, scaling violin widths by area, and</span> <span class="cmr-10x-x-109">(b) splitting violins to compare subsets of data.</span></span></p>
</div>
<!--tex4ht:label?: x1-8900337 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 762--><p class="indent" >
<a 
 id="x1-8900438"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 765--><p class="noindent" >

</p><!--l. 766--><p class="noindent" >
<img 
src=""heatmap2x".png" alt="PIC"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 3.38: </span><span class="content"><strong><span class="cmbx-10x-x-109">Demonstration of annotated heatmap</span></strong><span class="cmr-10x-x-109">. The example heatmap depicts the</span> <span class="cmr-10x-x-109">additional row and column annotation bars enabled by </span><span class="cmtt-10x-x-109">heatmap.2x</span><span class="cmr-10x-x-109">, extending the features</span> <span class="cmr-10x-x-109">of </span><span class="cmtt-10x-x-109">gplots </span><span class="cmr-10x-x-109">with backwards compatible inputs.</span></span></p>
</div>
<!--tex4ht:label?: x1-8900438 -->
                                                                                    

</div><hr class="endfloat" />
<a 
 id="x1-89005r174"></a>
<h4 class="subsectionHead">
<span class="titlemark">3.5.3 </span> <a 
 id="x1-900003"></a>Extensions to the iGraph Package
</h4>
<!--l. 777--><p class="noindent" >
The following features were developed during this thesis using “iGraph” data objects, building upon the <span class="cmtt-12">igraph </span>package (<a 
href="#Xigraph">Csardi and Nepusz</a>, <a 
href="#Xigraph">2006</a>). These have been released as separate packages for each respective procedure and can be installed together as a collection of extensions to the <span class="cmtt-12">igraph </span>package (<a 
href="https://github.com/TomKellyGenetics/igraph.extensions" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/igraph.extensions</span></a>). <a 
 id="x1-90001r153"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">3.5.3.1 </span> <a 
 id="x1-910001"></a>Sampling Simulated Data from Graph Structures
</h5>
<!--l. 780--><p class="noindent" >
The <span class="cmtt-12">graphsim </span>package implements the procedure for simulating <a 
href="#glo:gene expression">gene expression</a> from <a 
href="#glo:graph">graph</a> structures (as described in Section <a 
href="#x1-860002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>). By default, this derives a matrix with a geometrically decreasing weighting by distance (by <a 
href="#glo:shortest path">shortest paths</a>) between each pair of <a 
href="#glo:vertex">nodes</a> with. An absolute decreasing weighting is also available with the option of to derive correlation structures from adjacency matrices or the number of <a 
href="#glo:edge">links</a> common partners (i.e., size of the shared “neighbourhood” (<a 
href="#XHell1976">Hell</a>, <a 
href="#XHell1976">1976</a>)) between each pair of <a 
href="#glo:vertex">nodes</a>. Functions to compute these are called directly by passing parameters to them when running the <span class="cmtt-12">generate</span><span class="cmtt-12">_expression </span>or <span class="cmtt-12">make</span><span class="cmtt-12"><em>sigma</span><span 
class="cmtt-12"></em>mat </span>commands. This package enables simulating <a 
href="#glo:gene expression">expression</a> data directly from a <a 
href="#glo:graph">graph</a> structure (with the intermediate steps automated) or generating Σ parameters for <span class="cmtt-12">mvtnorm</span> from <a 
href="#glo:graph">graph</a> structures or matrices derived from them. These functions support assignment of activating or inhibiting relationships to each <a 
href="#glo:edge">edge</a> (with a <span class="cmtt-12">state</span> parameter). <a 
 id="x1-91001r180"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">3.5.3.2 </span> <a 
 id="x1-920002"></a>Plotting Directed Graph Structures
</h5>
<!--l. 783--><p class="noindent" >
<p>The <span class="cmtt-12">plot.igraph </span>package provides the <span class="cmtt-12">plot</span><span class="cmtt-12">_directed </span>function, specifically developed for directed <a 
href="#glo:graph">graph</a> structures, to plot activating or inhibiting for each <a 
href="#glo:edge">edge</a> (as described in Section <a 
href="#x1-860002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>). As shown in Figure <a 
href="#x1-9200139">3.39<!--tex4ht:ref: fig:simple_graph2 --></a>, this function supports separate plotting parameters for each <a 
href="#glo:vertex">node</a>, <a 
href="#glo:vertex">node</a> label, and <a 
href="#glo:edge">edge</a>. This includes colours of <a 
href="#glo:vertex">node</a> fill, border, label text, and <a 
href="#glo:edge">edges</a> and size of <a 
href="#glo:vertex">nodes</a>, <a 
href="#glo:edge">edge</a> widths, arrowhead lengths, and font size of labels. The <span class="cmtt-12">state</span> parameter for assigning activating or inhibiting to each <a 
href="#glo:edge">edge</a> determines whether <a 
href="#glo:edge">edges</a> were depicted with 30<span class="tcrm-1200">°</span> or 90<span class="tcrm-1200">°</span> arrowheads. Colours are assigned separately so they may be customised. Vectorised parameters are applied across each <a 
href="#glo:vertex">node</a> or <a 
href="#glo:edge">edge</a>, whereas scalar</p>
<p>parameters apply the same plotting parameters across them. The default layout function is <span class="cmtt-12">layout.fruchterman.reingold </span>but any layout function supported by <span class="cmtt-12">plot </span>function in <span class="cmtt-12">igraph </span>(<a 
href="#Xigraph">Csardi and Nepusz</a>, <a 
href="#Xigraph">2006</a>) was compatible, such as <span class="cmtt-12">layout.kamada.kawai </span>used to implement the Kamada–Kawai algorithm (<a 
href="#XKamada1989">Kamada and Kawai</a>, <a 
href="#XKamada1989">1989</a>) for graph plots throughout this thesis.</p>
</p><!--l. 786--><p class="indent" >
<a 
 id="x1-9200139"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 789--><p class="noindent" >
</p>
<div class="fbox">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/simple_graph_inhibiting".png  />
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.39: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulating </span><a 
href="#glo:graph"><span class="cmbx-10x-x-109">graph</span></a> <span class="cmbx-10x-x-109">structures.</span></strong></strong> <span class="cmr-10x-x-109">An example </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure which has been</span> <span class="cmr-10x-x-109">used throughout demonstrating the simulation procedure from </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structures. Activating</span> <a 
href="#glo:edge"><span class="cmr-10x-x-109">links</span></a> <span class="cmr-10x-x-109">are denoted by blue arrows and inhibiting </span><a 
href="#glo:edge"><span class="cmr-10x-x-109">links</span></a> <span class="cmr-10x-x-109">by red </span><a 
href="#glo:edge"><span class="cmr-10x-x-109">edges</span></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-9200139 -->
                                                                                    

</div><hr class="endfloat" />
<a 
 id="x1-92002r181"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">3.5.3.3 </span> <a 
 id="x1-930003"></a>Computing Information Centrality
</h5>
<!--l. 803--><p class="noindent" >
The <a 
href="#glo:shortest path">shortest paths</a> of a network were computed by the <span class="cmtt-12">igraph </span>package (<a 
href="#Xigraph">Csardi and Nepusz</a>, <a 
href="#Xigraph">2006</a>) which can be extended to calculate the network efficiency but was not provided by the package itself (ss described in Section <a 
href="#x1-680004">2.4.4<!--tex4ht:ref: methods:network_metrics --></a>). The “<a 
href="#glo:information centrality">information centrality</a>” of a <a 
href="#glo:vertex">vertex</a> is computed as the relative change in the network efficiency when the <a 
href="#glo:vertex">vertex</a> is removed. <a 
href="#glo:information centrality">Information centrality</a> is calculated iteratively for each <a 
href="#glo:vertex">node</a> and the sum of <a 
href="#glo:information centrality">information centrality</a> for each <a 
href="#glo:vertex">vertex</a> is the <a 
href="#glo:information centrality">information centrality</a> for the network. These metrics were released in the <span class="cmtt-12">info.centrality </span>package (<a 
href="https://github.com/TomKellyGenetics/info.centrality" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/info.centrality</span></a>). <a 
 id="x1-93001r183"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">3.5.3.4 </span> <a 
 id="x1-940004"></a>Testing Pathway Structure with Permutation Testing
</h5>
<!--l. 806--><p class="noindent" >
A network-based procedure developed was used to compare of <a 
href="#glo:siRNA">siRNA</a> and <a 
href="#glo:SLIPT">SLIPT</a> candidate genes in a <a 
href="#glo:graph">pathway</a> structure. Such <a 
href="#glo:graph">pathway</a> structure relationships were tested by computing the number of <a 
href="#glo:shortest path">shortest paths</a> between two different groups of <a 
href="#glo:vertex">nodes</a> in either direction within a graph . This pathway relationship metric was implemented in the <span class="cmtt-12">pathway.structure.permutation </span>package (<a 
href="https://github.com/TomKellyGenetics/pathway.structure.permutation" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/pathway.structure.permutation</span></a>) with permutation testing (as described in Sections <a 
href="#x1-840001">3.4.1<!--tex4ht:ref: methods:pathway_str --></a> and <a 
href="#x1-850001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>). <a 
 id="x1-94001r184"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">3.5.3.5 </span> <a 
 id="x1-950005"></a>Metapackage to Install iGraph Functions
</h5>
<!--l. 809--><p class="noindent" >
<p>These features may be installed together with <span class="cmtt-12">igraph.extensions</span>, which can be accessed from a <a 
href="https://github.com/TomKellyGenetics/igraph.extensions" >GitHub repository</a> (<a 
href="https://github.com/TomKellyGenetics/igraph.extensions" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/igraph.extensions</span></a>). This meta-package installs <span class="cmtt-12">igraph </span>(<a 
href="#Xigraph">Csardi and Nepusz</a>, <a 
href="#Xigraph">2006</a>) and the packages described in Section <a 
href="#x1-900003">3.5.3<!--tex4ht:ref: methods:igraph_extensions --></a> including their dependencies for matrix operations and statistical procedures: <span class="cmtt-12">Matrix</span>, <span class="cmtt-12">matrixcalc</span>, and <span class="cmtt-12">mvtnorm </span>(<a 
href="#XMatrix">Bates and Maechler</a>, <a 
href="#XMatrix">2016</a>; <a 
href="#XGenz2009">Genz and Bretz</a>, <a 
href="#XGenz2009">2009</a>; <a 
href="#Xmvtnorm">Genz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xmvtnorm">2016</a>; <a 
href="#Xmatrixcalc">Novomestky</a>, <a 
href="#Xmatrixcalc">2012</a>).</p>
</p>
<h2 class="chapterHead">
<span class="titlemark">Chapter 4</span><br /><a 
 id="x1-960004"></a>Synthetic Lethal Analysis of Gene Expression Data
</h2>
<!--l. 4--><p class="noindent" >
Having developed a statistical <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection methodology, <a 
href="#glo:SLIPT">SLIPT</a>, it was next applied to publicly available cancer <a 
href="#glo:gene expression">gene expression</a> datasets. The analysis presented in this Chapter focuses on breast cancer for which <a 
href="#glo:TCGA">TCGA</a> <a 
href="#glo:gene expression">expression</a> data (<a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>) from a patient cohort and <a 
href="#glo:siRNA">siRNA</a> screen data, from experiments conducted in MCF10A cells (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), were available. Stomach cancer data (<a 
href="#XTCGA2014GC">Bass <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014GC">2014</a>) was used to replicate findings in an independent dataset, with this cancer chosen because it also occurs in syndromic <a 
href="#glo:HDGC">HDGC</a> patients. The <a 
href="#glo:TCGA">TCGA</a> data also has the advantages of having other clinical and molecular profiles, including <a 
href="#glo:somatic">somatic</a> <a 
href="#glo:mutation">mutation</a> across many of the same samples, in addition to a considerable sample size for <a 
href="#glo:RNA-Seq">RNA-Seq</a> <a 
href="#glo:gene expression">expression</a> data generated with common <a 
href="#glo:TCGA">TCGA</a> procedures to minimise batch effects.
</p><!--l. 6--><p class="indent" >
Synthetic lethal candidate partners for <span class="cmti-12">CDH1 </span>were identified at both the gene and <a 
href="#glo:pathway">pathways</a> level. <a 
href="#glo:SLIPT">SLIPT</a> gene candidates were analysed by cluster analysis for common <a 
href="#glo:gene expression">expression</a> profiles across samples and relationships with clinical factors and <a 
href="#glo:mutation">mutations</a> in key breast cancer genes. These genes were also compared to the gene candidates from primary and secondary (validation) screens conducted by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) on isogenic cell lines. For comparison, the <a 
href="#glo:SLIPT">SLIPT</a> methodology was also applied using <a 
href="#glo:mutation">mutation</a> data for <span class="cmti-12">CDH1</span> against <a 
href="#glo:gene expression">expression</a> of candidate partners (as described in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) which may better represent the null <a 
href="#glo:mutation">mutations</a> in <a 
href="#glo:HDGC">HDGC</a> patients and the experimental cell model (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>). Pathways were analysed by over-representation analysis (with resampling for comparisons with <a 
href="#glo:siRNA">siRNA</a> data) and supported by a <a 
href="#glo:metagene">metagene</a> analysis of <a 
href="#glo:pathway">pathways</a> gene signatures. The <a 
href="#glo:pathway">pathways</a> meta<a 
href="#glo:gene expression">gene expression</a> profiles were used to replicate known relationships between clinical and molecular characteristics for breast cancer and to demonstrate application of <a 
href="#glo:SLIPT">SLIPT</a> directly on <a 
href="#glo:metagene">metagenes</a> to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a>. <a 
 id="x1-96001r172"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">4.1 </span> <a 
 id="x1-970001"></a>Synthetic Lethal Genes in Breast Cancer
</h3>
<!--l. 10--><p class="noindent" >
<p>The <a 
href="#glo:SLIPT">SLIPT</a> methodology (as described in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) was applied to the normalised <a 
href="#glo:TCGA">TCGA</a> breast cancer <a 
href="#glo:gene expression">gene expression</a> dataset (<span class="cmmi-12">n </span>= 1168). As shown in Table <a 
href="#x1-970011">4.1<!--tex4ht:ref: tab:gene_SL --></a>, the most significant genes had strong evidence of <a 
href="#glo:gene expression">expression</a>-based association with <span class="cmti-12">CDH1 </span>(high <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values) with fewer samples exhibiting low <a 
href="#glo:gene expression">expression</a> of both genes than expected statistically. Eukaryotic translation genes were among the highest scoring gene candidates,</p>
including initiation factors, elongation factors, and ribosomal proteins. These are clearly necessary for cancer cells to grow and proliferate, with sustained <a 
href="#glo:gene expression">gene expression</a> needed to maintain growth signalling <a 
href="#glo:pathway">pathways</a> and resist apoptosis or immune factors, translation genes may be subject to <a 
href="#glo:non-oncogene addiction">non-oncogene addiction</a> for <span class="cmti-12">CDH1</span>-deficient cells.
</p><!--l. 12--><p class="indent" >
<p>While these are among the strongest <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates, translational genes are cruicial to the viability of healthy cells and dosing for a selective <a 
href="#glo:synthetic lethal">synthetic lethal</a> effect against these may be difficult compared to other biological functions which may also be supported among the <a 
href="#glo:SLIPT">SLIPT</a> candidate genes. Furthermore, few known biological functions of <span class="cmti-12">CDH1 </span>were among the strongest SL candidates, so the remaining candidate genes may also be informative since they are likely to contain these expected functions in addition to novel relationships for <span class="cmti-12">CDH1</span>. Thus further <a 
href="#glo:pathway">pathways</a> level analyses were also conducted to examine biological functions over-represented among synthetic candidate genes and to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a>.</p>
</p><!--l. 14--><p class="indent" >
<a 
 id="x1-970011"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 4.1: </span><span class="content">Candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene partners of <span class="cmti-12">CDH1 </span>from SLIPT</span>
</div><!--tex4ht:label?: x1-970011 -->
<div class="tabular">
<table id="TBL-58" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-58-1g"><col 
id="TBL-58-1" /><col 
id="TBL-58-2" /><col 
id="TBL-58-3" /><col 
id="TBL-58-4" /><col 
id="TBL-58-5" /><col 
id="TBL-58-6" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-58-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-1-1"  
class="td11"> 
<span class="cmbx-12">Gene </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-1-2"  
class="td11"> 
Observed<sup class="textsuperscript"><span class="cmr-10x-x-109">*</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-1-3"  
class="td11"> 
Expected<sup class="textsuperscript"><span class="cmr-10x-x-109">*</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-1-4"  
class="td11"> 
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-1-5"  
class="td11">   
p-value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-1-6"  
class="td11"> 
p-value (<a 
href="#glo:FDR">FDR</a>)
</td>

</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-58-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-2-1"  
class="td11"> 
<span class="cmti-12">TRIP10 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-2-2"  
class="td11">    
62
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-2-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-2-4"  
class="td11">   
162
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-2-5"  
class="td11"> 
5<span class="cmmi-12">.</span>65 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">34</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-2-6"  
class="td11">  
1<span class="cmmi-12">.</span>84 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-3-1"  
class="td11"> 
<span class="cmti-12">EEF1B2 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-3-2"  
class="td11">    
56
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-3-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-3-4"  
class="td11">   
158
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-3-5"  
class="td11"> 
3<span class="cmmi-12">.</span>10 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">33</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-3-6"  
class="td11">  
9<span class="cmmi-12">.</span>45 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-4-1"  
class="td11"> 
<span class="cmti-12">GBGT1 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-4-2"  
class="td11">    
61
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-4-3"  
class="td11">   
131
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-4-4"  
class="td11">   
156
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-4-5"  
class="td11"> 
1<span class="cmmi-12">.</span>08 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">32</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-4-6"  
class="td11">  
3<span class="cmmi-12">.</span>14 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-5-1"  
class="td11"> 
<span class="cmti-12">ELN </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-5-2"  
class="td11">    
81
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-5-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-5-4"  
class="td11">   
149
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-5-5"  
class="td11"> 
3<span class="cmmi-12">.</span>46 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-5-6"  
class="td11">  
8<span class="cmmi-12">.</span>82 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-6-1"  
class="td11"> 
<span class="cmti-12">TSPAN4 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-6-2"  
class="td11">    
78
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-6-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-6-4"  
class="td11">   
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-6-5"  
class="td11"> 
1<span class="cmmi-12">.</span>63 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-6-6"  
class="td11">  
3<span class="cmmi-12">.</span>79 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-7-1"  
class="td11"> 
<span class="cmti-12">GLIPR2 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-7-2"  
class="td11">    
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-7-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-7-4"  
class="td11">   
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-7-5"  
class="td11"> 
1<span class="cmmi-12">.</span>68 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-7-6"  
class="td11">  
3<span class="cmmi-12">.</span>86 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-8-1"  
class="td11"> 
<span class="cmti-12">RPS20 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-8-2"  
class="td11">    
73
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-8-3"  
class="td11">   
131
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-8-4"  
class="td11">   
145
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-8-5"  
class="td11"> 
1<span class="cmmi-12">.</span>89 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-8-6"  
class="td11">  
4<span class="cmmi-12">.</span>28 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-9-1"  
class="td11"> 
<span class="cmti-12">RPS27A </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-9-2"  
class="td11">    
80
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-9-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-9-4"  
class="td11">   
143
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-9-5"  
class="td11"> 
5<span class="cmmi-12">.</span>53 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-9-6"  
class="td11">  
1<span class="cmmi-12">.</span>18 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-10-1"  
class="td11"> 
<span class="cmti-12">EEF1A1P9 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-10-2"  
class="td11">    
63
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-10-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-10-4"  
class="td11">   
141
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-10-5"  
class="td11"> 
1<span class="cmmi-12">.</span>91 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-10-6"  
class="td11">  
3<span class="cmmi-12">.</span>74 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-11-1"  
class="td11"> 
<span class="cmti-12">C1R </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-11-2"  
class="td11">    
73
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-11-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-11-4"  
class="td11">   
141
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-11-5"  
class="td11"> 
2<span class="cmmi-12">.</span>05 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-11-6"  
class="td11">  
3<span class="cmmi-12">.</span>97 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-12-1"  
class="td11"> 
<span class="cmti-12">LYL1 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-12-2"  
class="td11">    
73
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-12-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-12-4"  
class="td11">   
140
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-12-5"  
class="td11"> 
2<span class="cmmi-12">.</span>99 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-12-6"  
class="td11">  
5<span class="cmmi-12">.</span>74 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-13-1"  
class="td11"> 
<span class="cmti-12">RPLP2 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-13-2"  
class="td11">    
71
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-13-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-13-4"  
class="td11">   
139
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-13-5"  
class="td11"> 
4<span class="cmmi-12">.</span>88 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-13-6"  
class="td11">  
9<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-14-1"  
class="td11"> 
<span class="cmti-12">C10orf10 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-14-2"  
class="td11">    
73
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-14-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-14-4"  
class="td11">   
138
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-14-5"  
class="td11"> 
6<span class="cmmi-12">.</span>72 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-14-6"  
class="td11">  
1<span class="cmmi-12">.</span>20 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-15-1"  
class="td11"> 
<span class="cmti-12">DULLARD </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-15-2"  
class="td11">    
74
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-15-3"  
class="td11">   
131
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-15-4"  
class="td11">   
138
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-15-5"  
class="td11"> 
9<span class="cmmi-12">.</span>29 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-15-6"  
class="td11">  
1<span class="cmmi-12">.</span>61 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-16-1"  
class="td11"> 
<span class="cmti-12">PPM1F </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-16-2"  
class="td11">    
64
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-16-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-16-4"  
class="td11">   
136
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-16-5"  
class="td11"> 
1<span class="cmmi-12">.</span>61 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-16-6"  
class="td11">  
2<span class="cmmi-12">.</span>65 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-17-1"  
class="td11"> 
<span class="cmti-12">OBFC2A </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-17-2"  
class="td11">    
69
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-17-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-17-4"  
class="td11">   
136
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-17-5"  
class="td11"> 
2<span class="cmmi-12">.</span>49 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-17-6"  
class="td11">  
3<span class="cmmi-12">.</span>93 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-18-1"  
class="td11"> 
<span class="cmti-12">RPL11 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-18-2"  
class="td11">    
70
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-18-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-18-4"  
class="td11">   
136
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-18-5"  
class="td11"> 
2<span class="cmmi-12">.</span>56 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-18-6"  
class="td11">  
3<span class="cmmi-12">.</span>97 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-19-1"  
class="td11"> 
<span class="cmti-12">RPL18A </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-19-2"  
class="td11">    
70
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-19-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-19-4"  
class="td11">   
135
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-19-5"  
class="td11"> 
3<span class="cmmi-12">.</span>08 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-19-6"  
class="td11">  
4<span class="cmmi-12">.</span>70 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-20-1"  
class="td11"> 
<span class="cmti-12">MFNG </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-20-2"  
class="td11">    
76
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-20-3"  
class="td11">   
131
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-20-4"  
class="td11">   
133
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-20-5"  
class="td11"> 
7<span class="cmmi-12">.</span>73 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-20-6"  
class="td11">  
1<span class="cmmi-12">.</span>12 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-21-1"  
class="td11"> 
<span class="cmti-12">RPS17 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-21-2"  
class="td11">    
77
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-21-3"  
class="td11">   
131
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-21-4"  
class="td11">   
133
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-21-5"  
class="td11"> 
8<span class="cmmi-12">.</span>94 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-21-6"  
class="td11">  
1<span class="cmmi-12">.</span>29 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-22-1"  
class="td11"> 
<span class="cmti-12">MGAT1 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-22-2"  
class="td11">    
73
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-22-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-22-4"  
class="td11">   
132
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-22-5"  
class="td11"> 
1<span class="cmmi-12">.</span>44 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-22-6"  
class="td11">  
2<span class="cmmi-12">.</span>03 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-23-1"  
class="td11"> 
<span class="cmti-12">RPS12 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-23-2"  
class="td11">    
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-23-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-23-4"  
class="td11">   
128
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-23-5"  
class="td11"> 
8<span class="cmmi-12">.</span>57 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-23-6"  
class="td11">  
1<span class="cmmi-12">.</span>12 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-24-1"  
class="td11"> 
<span class="cmti-12">C10orf54 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-24-2"  
class="td11">    
73
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-24-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-24-4"  
class="td11">   
127
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-24-5"  
class="td11"> 
1<span class="cmmi-12">.</span>37 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-24-6"  
class="td11">  
1<span class="cmmi-12">.</span>75 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-25-1"  
class="td11"> 
<span class="cmti-12">LOC286367 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-25-2"  
class="td11">    
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-25-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-25-4"  
class="td11">   
126
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-25-5"  
class="td11"> 
2<span class="cmmi-12">.</span>20 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-25-6"  
class="td11">  
2<span class="cmmi-12">.</span>70 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-58-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-26-1"  
class="td11"> 
<span class="cmti-12">GMFG </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-26-2"  
class="td11">    
70
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-26-3"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-26-4"  
class="td11">   
126
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-26-5"  
class="td11"> 
2<span class="cmmi-12">.</span>20 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-58-26-6"  
class="td11">  
2<span class="cmmi-12">.</span>70 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-58-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-58-27-1"  
class="td11">            </td></tr></table>
<!--l. 83--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Strongest candidate </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">in </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">breast cancer</span> <span class="cmr-10x-x-109">expression data</span> <sup class="textsuperscript"><span class="cmr-10x-x-109">*</span></sup><span class="cmr-10x-x-109">Observed and expected numbers of samples which had low </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">of both genes</span></p>
</p>
</div><hr class="endfloat" />
<!--l. 86--><p class="indent" >
The modified <a 
href="#glo:mtSLIPT">mtSLIPT</a> methodology (as described in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) was also applied to the normalised <a 
href="#glo:TCGA">TCGA</a> breast cancer <a 
href="#glo:gene expression">gene expression</a> dataset, against <a 
href="#glo:somatic">somatic</a> loss of function <a 
href="#glo:mutation">mutations</a> in <span class="cmti-12">CDH1</span>. As shown in Appendix Table <a 
href="#x1-1720011">C.1<!--tex4ht:ref: tab:gene_mtSL --></a>, the most significant genes also had strong evidence of <a 
href="#glo:gene expression">expression</a> associated with <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutations</a> (high <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values) with fewer samples with <span class="cmti-12">CDH1 </span>exhibiting low <a 
href="#glo:gene expression">expression</a> each candidate gene than expected statistically. These genes were not as strongly supported as the <a 
href="#glo:gene expression">expression</a> analysis (in Table <a 
href="#x1-970011">4.1<!--tex4ht:ref: tab:gene_SL --></a>), however, nor were as many genes detected. This is perhaps unsurprising due to the lower sample size with matching <a 
href="#glo:somatic">somatic</a> <a 
href="#glo:mutation">mutation</a> data and the lower frequency of <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutations</a> compared to low <a 
href="#glo:gene expression">expression</a> defined by <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span> quantiles.
</p><!--l. 88--><p class="indent" >
The <a 
href="#glo:mtSLIPT">mtSLIPT</a> candidates had more genes involved in cell and gene regulation, particularly <a 
href="#glo:DNA">DNA</a> and <a 
href="#glo:RNA">RNA</a> binding factors. The strongest candidates also included microtubule (<span class="cmti-12">KIF12</span>), microfibril (<span class="cmti-12">MFAP4</span>), and cell adhesion (<span class="cmti-12">TENC1</span>) genes consistent with the established cytoskeletal role of <span class="cmti-12">CDH1</span>. The elastin gene (<span class="cmti-12">ELN</span>) was notably strongly supported by both <a 
href="#glo:gene expression">expression</a> and <a 
href="#glo:mutation">mutation</a> <a 
href="#glo:SLIPT">SLIPT</a> analysis of CDH1 supporting interactions with extracellular proteins and the tumour microenvironment. <a 
 id="x1-97002r179"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.1.1 </span> <a 
 id="x1-980001"></a>Synthetic Lethal Pathways in Breast Cancer
</h4>
<!--l. 95--><p class="noindent" >
Translational <a 
href="#glo:pathway">pathways</a> were strongly over-represented in <a 
href="#glo:SLIPT">SLIPT</a> partners, as shown in Table <a 
href="#x1-980012">4.2<!--tex4ht:ref: tab:pathway_exprSL --></a>. These include ribosomal subunits, initiation, peptide elongation, and termination. Regulatory processes involving <a 
href="#glo:mRNA">mRNA</a> including 3’ untranslated region (UTR) binding, L13a-mediated translational silencing, and nonsense-mediated decay were also implicated. These are consistent with protein translation being subject to “<a 
href="#glo:non-oncogene addiction">non-oncogene addiction</a>” (<a 
href="#XLuo2009">Luo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLuo2009">2009</a>), as a core process that is dysregulated to sustain cancer proliferation and survival (<a 
href="#XGao2015">Gao and Roux</a>, <a 
href="#XGao2015">2015</a>).
</p><!--l. 97--><p class="indent" >
<p>Immune <a 
href="#glo:pathway">pathways</a>, including the adaptive immune system and responses to infectious diseases were also strongly implicated as <a 
href="#glo:synthetic lethal">synthetic lethal</a> with loss of <a 
href="#glo:E-cadherin">E-cadherin</a>. This is consistent with the alterations of immune response being a hallmark of cancer <a 
href="#XHanahan2000">Hanahan and Weinberg</a> (<a 
href="#XHanahan2000">2000</a>), since evading the immune system is necessary for cancer survival. Either of these systems are potential means to target <span class="cmti-12">CDH1 </span>deficient cells, although these were not detected in an isolated cell line experimental screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) and the</p>
<p>differences between the findings in patient data are described in more detail in Section <a 
href="#x1-1060005">4.2.5<!--tex4ht:ref: chapt3:compare_pathway --></a>.</p>
</p><!--l. 99--><p class="indent" >
<a 
 id="x1-980012"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 4.2: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT</span>
</div><!--tex4ht:label?: x1-980012 -->
<div class="tabular">
<table id="TBL-59" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-59-1g"><col 
id="TBL-59-1" /><col 
id="TBL-59-2" /><col 
id="TBL-59-3" /><col 
id="TBL-59-4" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-59-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-1-1"  
class="td11"> 
<strong><span class="cmbx-12">Pathways Over-represented</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-1-2"  
class="td11"> 
<strong><span class="cmbx-12">Pathway Size</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-1-3"  
class="td11"> 
<strong><span class="cmbx-12">SL Genes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-1-4"  
class="td11"> 
<strong><span class="cmbx-12">p-value (</span><a 
href="#glo:FDR"><span class="cmbx-12">FDR</span></a><span class="cmbx-12">)</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-59-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-2-1"  
class="td11"> 
Eukaryotic Translation Elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-2-2"  
class="td11">      
86
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-2-3"  
class="td11">     
81
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-2-4"  
class="td11">   
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">207</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-3-1"  
class="td11"> 
Peptide chain elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-3-2"  
class="td11">      
83
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-3-3"  
class="td11">     
78
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-3-4"  
class="td11">   
5<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">201</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-4-1"  
class="td11"> 
Eukaryotic Translation Termination
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-4-2"  
class="td11">      
83
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-4-3"  
class="td11">     
77
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-4-4"  
class="td11">   
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">196</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-5-1"  
class="td11"> 
Viral <a 
href="#glo:mRNA">mRNA</a> Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-5-2"  
class="td11">      
81
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-5-3"  
class="td11">     
76
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-5-4"  
class="td11">   
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">196</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-6-1"  
class="td11"> 
Formation of a pool of free 40S subunits
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-6-2"  
class="td11">      
93
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-6-3"  
class="td11">     
81
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-6-4"  
class="td11">   
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">194</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-7-1"  
class="td11"> 
Nonsense Mediated Decay independent of the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-7-2"  
class="td11">      
88
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-7-3"  
class="td11">     
77
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-7-4"  
class="td11">   
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">187</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-8-1"  
class="td11"> 
L13a-mediated translational silencing of Ceruloplasmin <a 
href="#glo:gene expression">expression</a>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-8-2"  
class="td11">      
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-8-3"  
class="td11">     
82
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-8-4"  
class="td11">   
9<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">183</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-9-1"  
class="td11"> 
3’ -UTR-mediated translational regulation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-9-2"  
class="td11">      
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-9-3"  
class="td11">     
82
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-9-4"  
class="td11">   
9<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">183</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-10-1"  
class="td11"> 
GTP hydrolysis and joining of the 60S ribosomal subunit
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-10-2"  
class="td11">      
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-10-3"  
class="td11">     
82
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-10-4"  
class="td11">   
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">181</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-11-1"  
class="td11"> 
Nonsense-Mediated Decay
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-11-2"  
class="td11">      
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-11-3"  
class="td11">     
80
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-11-4"  
class="td11">   
6<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">176</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-12-1"  
class="td11"> 
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-12-2"  
class="td11">      
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-12-3"  
class="td11">     
80
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-12-4"  
class="td11">   
6<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">176</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-13-1"  
class="td11"> 
Adaptive Immune System
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-13-2"  
class="td11">      
412
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-13-3"  
class="td11">    
167
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-13-4"  
class="td11">   
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">174</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-14-1"  
class="td11"> 
Eukaryotic Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-14-2"  
class="td11">      
111
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-14-3"  
class="td11">     
82
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-14-4"  
class="td11">   
5<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">173</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-15-1"  
class="td11"> 
Cap-dependent Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-15-2"  
class="td11">      
111
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-15-3"  
class="td11">     
82
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-15-4"  
class="td11">   
5<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">173</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-16-1"  
class="td11"> 
SRP-dependent cotranslational protein targeting to membrane
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-16-2"  
class="td11">      
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-16-3"  
class="td11">     
79
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-16-4"  
class="td11">   
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">171</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-17-1"  
class="td11"> 
Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-17-2"  
class="td11">      
141
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-17-3"  
class="td11">     
91
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-17-4"  
class="td11">   
6<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">170</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-18-1"  
class="td11"> 
Infectious disease
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-18-2"  
class="td11">      
347
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-18-3"  
class="td11">    
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-18-4"  
class="td11">   
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">166</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-19-1"  
class="td11"> 
Influenza Infection
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-19-2"  
class="td11">      
117
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-19-3"  
class="td11">     
81
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-19-4"  
class="td11">   
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">163</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-20-1"  
class="td11"> 
Influenza Viral <a 
href="#glo:RNA">RNA</a> Transcription and Replication
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-20-2"  
class="td11">      
108
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-20-3"  
class="td11">     
77
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-20-4"  
class="td11">   
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">160</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-59-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-21-1"  
class="td11"> 
Influenza Life Cycle
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-21-2"  
class="td11">      
112
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-21-3"  
class="td11">     
77
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-59-21-4"  
class="td11">   
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">156</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-59-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-59-22-1"  
class="td11">                                                             </td></tr></table>
<!--l. 155--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Gene set over-representation analysis (hypergeometric test) for Reactome </span><a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathways</span></a> <span class="cmr-10x-x-109">in </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">partners for </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">.</span></p>
</p>
</div><hr class="endfloat" />
<!--l. 158--><p class="indent" >
It is also notable that the <a 
href="#glo:pathway">pathways</a> over-represented in <a 
href="#glo:SLIPT">SLIPT</a> candidate genes have strongly significant over-representation of Reactome <a 
href="#glo:pathway">pathways</a> based on the hypergeometric test (as described in Section <a 
href="#x1-580002">2.3.2<!--tex4ht:ref: methods:enrichment --></a>). Even after adjusting stringently for multiple testing, biologically related <a 
href="#glo:pathway">pathways</a> were supported together. These <a 
href="#glo:pathway">pathways</a> are further supported by testing for <a 
href="#glo:synthetic lethal">synthetic lethality</a> against <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutations</a> (<a 
href="#glo:mtSLIPT">mtSLIPT</a>) with many of these <a 
href="#glo:pathway">pathways</a> also among the most strongly supported in this analysis (shown in Appendix Table <a 
href="#x1-1720022">C.2<!--tex4ht:ref: tab:pathway_mtSL --></a>). This <a 
href="#glo:mutation">mutation</a>-based analysis more closely represents the null <span class="cmti-12">CDH1</span> <a 
href="#glo:mutation">mutations</a> in <a 
href="#glo:HDGC">HDGC</a> (<a 
href="#XGuilford1998">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford1998">1998</a>) and the experimental MCF10A cell model (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>). There was still support for translational and immune <a 
href="#glo:pathway">pathways</a> not detected in the isolated experimental system. <a 
href="#glo:GPCR">GPCRs</a> also among the most strongly supported <a 
href="#glo:pathway">pathways</a>, supporting the experimental findings of <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) for these intracellular signalling <a 
href="#glo:pathway">pathways</a> already being targeted for other diseases. <a 
 id="x1-98002r189"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.1.2 </span> <a 
 id="x1-990002"></a>Expression Profiles of Synthetic Lethal Partners
</h4>
<!--l. 168--><p class="noindent" >
<p>Due to the sheer number of gene candidates, investigations proceeded into correlation structure and <a 
href="#glo:pathway">pathways</a> over-representation. These analyses also examined <a 
href="#glo:gene expression">expression</a> patterns of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene candidates. This serves to explore the functional similarity of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">CDH1</span>, with the eventual aim to assess their utility as drug targets. As shown in Figure <a 
href="#x1-990011">4.1<!--tex4ht:ref: fig:slipt_expr --></a> (which clusters <span class="cmti-12">CDH1 </span>lowly expressing samples separately), there were several large clusters of genes among the <a 
href="#glo:gene expression">expression</a> profiles of the <span class="cmti-12">CDH1 </span><a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate partners. The clustering suggests co-regulation of genes or <a 
href="#glo:pathway">pathways</a> correlation between partner gene candidates. A number of candidates from an experimental <a 
href="#glo:RNAi">RNAi</a> screen study performed by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) were also identified by this approach. In addition, novel gene candidates were also identified, which had not been observed affect viability in isogenic cell line experiments.</p>
</p><!--l. 170--><p class="indent" >
<a 
 id="x1-990011"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<img 
src="CDH1_Heatmaps_Genes_Split_By_CDH1_z-trans_exprSL_cordistx_Pub.png" alt="PIC"  
 /> <br />
<div class="caption">
<p><span class="id">Figure 4.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethal </span><a 
href="#glo:gene expression"><span class="cmbx-10x-x-109">expression</span></a> <span class="cmbx-10x-x-109">profiles of analysed samples.</span></strong> <a 
href="#glo:gene expression"><span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">profile heatmap (correlation distance, complete linkage) of all samples (separated</span> <span class="cmr-10x-x-109">by the</span> <span class="cmr-8">1</span> <span class="cmmi-10x-x-109">∕</span> <span class="cmr-8">3</span> <span class="cmr-10x-x-109">quantile of </span><span class="cmti-10x-x-109">CDH1 </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a><span class="cmr-10x-x-109">) analysed in </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">breast cancer dataset for</span> <a 
href="#glo:gene expression"><span class="cmr-10x-x-109">gene expression</span></a> <span class="cmr-10x-x-109">of 5165 candidate partners of </span><a 
href="#glo:E-cadherin"><span class="cmr-10x-x-109">E-cadherin</span></a> <span class="cmr-10x-x-109">(</span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">) from </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">prediction</span> <span class="cmr-10x-x-109">(with </span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a> <span class="cmr-10x-x-109">adjusted </span><span class="cmmi-10x-x-109">p &lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05). Deeply clustered, inter-correlated genes form several main</span> <span class="cmr-10x-x-109">groups, each containing genes that were SL candidates or lethal in an </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screen (</span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">Telford</span> <em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109">,</span><span class="cmr-10x-x-109"> </span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">2015</span></a><span class="cmr-10x-x-109">). Screen results for </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">(SL), the reverse effect (RSL), or lethal cell</span> <span class="cmr-10x-x-109">viability are shown as reported by </span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">Telford </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">2015</span></a><span class="cmr-10x-x-109">). Clusters had different sample groups</span> <span class="cmr-10x-x-109">highly expressing the </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">candidates in </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">low samples, notably ‘normal-like’,</span> <span class="cmr-10x-x-109">‘basal-like’, and estrogen receptor negative samples have elevated </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">in one or more</span> <span class="cmr-10x-x-109">distinct clusters showing complexity and variation among candidate </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners.</span> <span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">low samples also contained most of samples with </span><span class="cmti-10x-x-109">CDH1 </span><a 
href="#glo:mutation"><span class="cmr-10x-x-109">mutations</span></a> <span class="cmr-10x-x-109">(shown in black).</span> <span class="cmr-10x-x-109">Negative values for </span><a 
href="#glo:mutation"><span class="cmr-10x-x-109">mutation</span></a> <span class="cmr-10x-x-109">and screen data are shown in light grey, with missing data in</span> <span class="cmr-10x-x-109">white. </span></span></p>
</div>
<!--tex4ht:label?: x1-990011 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 183--><p class="indent" >
In these <a 
href="#glo:gene expression">expression</a> profiles, a gene with a moderate or high signal across samples exhibiting low <span class="cmti-12">CDH1 </span><a 
href="#glo:gene expression">expression</a> would represent a potential drug target. However, it appears that several molecular subtypes of cancer have elevation of different clusters of <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates in samples with low <span class="cmti-12">CDH1</span>. This clustering suggests that different targets (or combinations) could be effective in different patients, suggesting potential utility for stratification. In particular, estrogen receptor negative, basal-like subtype, and “normal-like” tumours (<a 
href="#XDai2015">Dai <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDai2015">2015</a>; <a 
href="#XEroles2012">Eroles <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XEroles2012">2012</a>; <a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>) have elevation of genes specific to particular clusters, indicative of some <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions being specific to a particular molecular subtype or genetic background. Thus <a 
href="#glo:synthetic lethal">synthetic lethal</a> drug therapy against these subtypes may be ineffective if it were designed against genes in another cluster.
</p><!--l. 185--><p class="indent" >
A similar correlation structure was observed among the candidates tested against <span class="cmti-12">CDH1</span> <a 
href="#glo:mutation">mutation</a> (<a 
href="#glo:mtSLIPT">mtSLIPT</a>), as shown in Appendix Figure <a 
href="#x1-1730011">C.1<!--tex4ht:ref: fig:slipt_expr_mtSL --></a>. This clustering analysis similarly identified several major clusters of putative <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes. In this case, many partner genes had consistently high <a 
href="#glo:gene expression">expression</a> across most of the (predominantly lobular subtype) <span class="cmti-12">CDH1 </span>breast cancer samples. However, a major exception to this in the <span class="cmti-12">CDH1</span> <a 
href="#glo:gene expression">expression</a> analysis were the normal tissue samples which were excluded from the <a 
href="#glo:mutation">mutation</a> data (as they were not tested for tumour-specific genotypes). This supports <a 
href="#glo:synthetic lethal">synthetic lethal</a> interventions being more applicable to <span class="cmti-12">CDH1 </span><a 
href="#glo:mutant">mutant</a> tumours. There was still considerable correlation structure, particularly among <span class="cmti-12">CDH1</span><a 
href="#glo:wild-type">wild-type</a> samples, sufficient to distinguish gene clusters. In contrast to the <a 
href="#glo:gene expression">expression</a> analysis the (predominantly ductal <span class="cmti-12">CDH1</span><a 
href="#glo:wild-type">wild-type</a>) basal-like subtype and estrogen receptor negative samples had depleted <a 
href="#glo:gene expression">expression</a> among most candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners. This is consistent with <a 
href="#glo:synthetic lethal">synthetic lethal</a> interventions only being effective in lobular estrogen receptor positive breast cancers in which they are a more common, as recurrent (<a 
href="#glo:driver mutation">driver</a>) <a 
href="#glo:mutation">mutation</a>. However, the remaining samples are still informative for <a 
href="#glo:synthetic lethal">synthetic lethal</a> analysis (by <a 
href="#glo:SLIPT">SLIPT</a>) as it requires highly expressing <span class="cmti-12">CDH1 </span>samples for comparison.
</p><!--l. 189--><p class="indent" >
<p>The <span class="cmti-12">CDH1 </span><a 
href="#glo:mutant">mutant</a> samples (as shown in Figure <a 
href="#x1-990011">4.1<!--tex4ht:ref: fig:slipt_expr --></a>) were predominantly among the low <span class="cmti-12">CDH1 </span>expressing samples, clustering throughout them with similar expression profiles to other samples exhibiting low <span class="cmti-12">CDH1 </span>expression. Thus the molecular profiles of <span class="cmti-12">CDH1 </span>low samples were indistinguishable from <span class="cmti-12">CDH1 </span><a 
href="#glo:mutant">mutant</a> samples, with the exception of normal samples (that do not have <a 
href="#glo:somatic">somatic</a> <a 
href="#glo:mutation">mutation</a> data available). Conversely, many of the <span class="cmti-12">CDH1</span> <a 
href="#glo:mutant">mutant</a> samples (in Appendix Figure <a 
href="#x1-1730011">C.1<!--tex4ht:ref: fig:slipt_expr_mtSL --></a>) had among the lowest <span class="cmti-12">CDH1 </span><a 
href="#glo:gene expression">expression</a>, and</p>
some of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners were also highly expressed in low expressing <span class="cmti-12">CDH1 </span><a 
href="#glo:wild-type">wild-type</a> samples, despite these not being considered as “inactivated” by <a 
href="#glo:mtSLIPT">mtSLIPT</a> analysis.
</p><!--l. 191--><p class="indent" >
Together these results support the use of low <span class="cmti-12">CDH1 </span><a 
href="#glo:gene expression">expression</a> as a strategy for detecting <span class="cmti-12">CDH1 </span>inactivation. This has the benefit of increasing sample size (including samples such as normal tissue which do not have <a 
href="#glo:somatic">somatic</a> <a 
href="#glo:mutation">mutation</a> data available) and increasing the expected number of mutually inactive (low-low) samples for the directional criteria of (mt)<a 
href="#glo:SLIPT">SLIPT</a> which enables it to better distinguish significant deviations below this (as discussed in Section <a 
href="#x1-1350001">6.1<!--tex4ht:ref: chapt5:compare_methods --></a>). This also circumvents the assumption that all (detected) <a 
href="#glo:mutation">mutations</a> are inactivating (although synonymous <a 
href="#glo:mutation">mutations</a> were excluded from the analysis), which may not be the case for several highly expressing <span class="cmti-12">CDH1</span> <a 
href="#glo:mutant">mutant</a> samples that do not cluster together in Figure <a 
href="#x1-990011">4.1<!--tex4ht:ref: fig:slipt_expr --></a> or Appendix Figure <a 
href="#x1-1730011">C.1<!--tex4ht:ref: fig:slipt_expr_mtSL --></a>. One of these exhibits among the lowest <a 
href="#glo:gene expression">expression</a> for many predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners and would not be vulnerable to inactivation of these genes. As such, correctly genotyping inactivating <a 
href="#glo:mutation">mutations</a> will be <a 
href="#glo:essential">essential</a> in clinical practice for <a 
href="#glo:synthetic lethal">synthetic lethal</a> targeting of <a 
href="#glo:tumour suppressor">tumour suppressor</a> genes, particularly for other genes such as <span class="cmti-12">TP53 </span>where oncogenic and <a 
href="#glo:tumour suppressor">tumour suppressor</a> <a 
href="#glo:mutation">mutations</a> (with different molecular consequences) are both commons. Using <a 
href="#glo:gene expression">expression</a> as a measure of gene function also avoids the assumptions that <a 
href="#glo:mutation">mutations</a> are <a 
href="#glo:somatic">somatic</a>, rather than <a 
href="#glo:germline">germline mutation</a>, and that gene inactivation occurs by detectable <a 
href="#glo:mutation">mutations</a>, rather than other mechanisms such as epigenetic changes. These may also account for some of the lowly expressing <span class="cmti-12">CDH1</span><a 
href="#glo:wild-type">wild-type</a> samples clustering with similar profiles to <a 
href="#glo:mutant">mutant</a> samples. <a 
 id="x1-99002r185"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">4.1.2.1 </span> <a 
 id="x1-1000001"></a>Subgroup Pathway Analysis
</h5>
                                                                                    

<!--l. 205--><p class="indent" >
<a 
 id="x1-1000013"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 4.3: </span><span class="content">Pathways for clusters of <span class="cmti-12">CDH1 </span>partners from SLIPT</span>
</div><!--tex4ht:label?: x1-1000013 -->
<div class="tabular">
<table id="TBL-60" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-60-1g"><col 
id="TBL-60-1" /><col 
id="TBL-60-2" /><col 
id="TBL-60-3" /><col 
id="TBL-60-4" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-60-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-1-1"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 1</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-1-2"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-1-3"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-1-4"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">p-value (</span><a 
href="#glo:FDR"><span class="cmbx-12x-x-120">FDR</span></a><span class="cmbx-12x-x-120">)</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-60-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-2-1"  
class="td11"> 
Collagen formation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-2-2"  
class="td11">        
67
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-2-3"  
class="td11">        
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-2-4"  
class="td11">     
4<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-3-1"  
class="td11"> 
Extracellular matrix organisation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-3-2"  
class="td11">       
238
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-3-3"  
class="td11">        
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-3-4"  
class="td11">     
1<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-4-1"  
class="td11"> 
Collagen biosynthesis and modifying enzymes
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-4-2"  
class="td11">        
56
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-4-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-4-4"  
class="td11">     
1<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-5-1"  
class="td11"> 
Uptake and actions of bacterial toxins
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-5-2"  
class="td11">        
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-5-3"  
class="td11">         
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-5-4"  
class="td11">     
9<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-6-1"  
class="td11"> 
Elastic fibre formation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-6-2"  
class="td11">        
37
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-6-3"  
class="td11">         
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-6-4"  
class="td11">     
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-7-1"  
class="td11"> 
Muscle contraction
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-7-2"  
class="td11">        
62
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-7-3"  
class="td11">         
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-7-4"  
class="td11">     
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-8-1"  
class="td11"> 
Fatty acid, triacylglycerol, and ketone body metabolism
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-8-2"  
class="td11">       
117
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-8-3"  
class="td11">        
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-8-4"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-9-1"  
class="td11"> 
XBP1(S) activates chaperone genes
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-9-2"  
class="td11">        
51
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-9-3"  
class="td11">         
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-9-4"  
class="td11">     
6<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-10-1"  
class="td11"> 
IRE1alpha activates chaperones
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-10-2"  
class="td11">        
54
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-10-3"  
class="td11">         
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-10-4"  
class="td11">     
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-11-1"  
class="td11"> 
Neurotoxicity of clostridium toxins
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-11-2"  
class="td11">        
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-11-3"  
class="td11">         
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-11-4"  
class="td11">     
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-12-1"  
class="td11"> 
Retrograde neurotrophin signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-12-2"  
class="td11">        
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-12-3"  
class="td11">         
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-12-4"  
class="td11">     
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-13-1"  
class="td11"> 
Assembly of collagen fibrils and other multimeric structures
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-13-2"  
class="td11">        
40
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-13-3"  
class="td11">         
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-13-4"  
class="td11">     
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-14-1"  
class="td11"> 
Collagen degradation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-14-2"  
class="td11">        
58
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-14-3"  
class="td11">         
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-14-4"  
class="td11">     
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-15-1"  
class="td11"> 
Arachidonic acid metabolism
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-15-2"  
class="td11">        
41
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-15-3"  
class="td11">         
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-15-4"  
class="td11">     
2<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-16-1"  
class="td11"> 
Synthesis of PA
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-16-2"  
class="td11">        
26
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-16-3"  
class="td11">         
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-16-4"  
class="td11">     
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-17-1"  
class="td11"> 
Signalling by NOTCH
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-17-2"  
class="td11">        
80
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-17-3"  
class="td11">         
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-17-4"  
class="td11">     
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-18-1"  
class="td11"> 
Signalling to RAS
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-18-2"  
class="td11">        
27
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-18-3"  
class="td11">         
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-18-4"  
class="td11">     
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-19-1"  
class="td11"> 
Integrin cell surface interactions
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-19-2"  
class="td11">        
82
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-19-3"  
class="td11">         
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-19-4"  
class="td11">     
4<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-60-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-20-1"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 2</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-20-2"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-20-3"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-20-4"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">p-value (</span><a 
href="#glo:FDR"><span class="cmbx-12x-x-120">FDR</span></a><span class="cmbx-12x-x-120">)</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-60-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-21-1"  
class="td11"> 
Eukaryotic Translation Elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-21-2"  
class="td11">        
86
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-21-3"  
class="td11">        
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-21-4"  
class="td11">    
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">181</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-22-1"  
class="td11"> 
Viral <a 
href="#glo:mRNA">mRNA</a> Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-22-2"  
class="td11">        
81
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-22-3"  
class="td11">        
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-22-4"  
class="td11">    
9<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">179</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-23-1"  
class="td11"> 
Peptide chain elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-23-2"  
class="td11">        
83
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-23-3"  
class="td11">        
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-23-4"  
class="td11">    
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">175</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-24-1"  
class="td11"> 
Eukaryotic Translation Termination
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-24-2"  
class="td11">        
83
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-24-3"  
class="td11">        
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-24-4"  
class="td11">    
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">175</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-25-1"  
class="td11"> 
Formation of a pool of free 40S subunits
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-25-2"  
class="td11">        
93
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-25-3"  
class="td11">        
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-25-4"  
class="td11">    
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">171</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-26-1"  
class="td11"> 
Nonsense Mediated Decay independent of the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-26-2"  
class="td11">        
88
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-26-3"  
class="td11">        
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-26-4"  
class="td11">    
9<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">168</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-27-1"  
class="td11"> 
L13a-mediated translational silencing of Ceruloplasmin <a 
href="#glo:gene expression">expression</a>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-27-2"  
class="td11">       
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-27-3"  
class="td11">        
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-27-4"  
class="td11">    
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">159</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-28-1"  
class="td11"> 
3’ -UTR-mediated translational regulation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-28-2"  
class="td11">       
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-28-3"  
class="td11">        
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-28-4"  
class="td11">    
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">159</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-29-1"  
class="td11"> 
Nonsense-Mediated Decay
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-29-2"  
class="td11">       
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-29-3"  
class="td11">        
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-29-4"  
class="td11">    
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">159</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-30-1"  
class="td11"> 
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-30-2"  
class="td11">       
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-30-3"  
class="td11">        
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-30-4"  
class="td11">    
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">159</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-31-1"  
class="td11"> 
SRP-dependent cotranslational protein targeting to membrane
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-31-2"  
class="td11">       
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-31-3"  
class="td11">        
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-31-4"  
class="td11">    
3<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">158</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-32-1"  
class="td11"> 
GTP hydrolysis and joining of the 60S ribosomal subunit
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-32-2"  
class="td11">       
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-32-3"  
class="td11">        
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-32-4"  
class="td11">    
3<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">158</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-33-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-33-1"  
class="td11"> 
Eukaryotic Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-33-2"  
class="td11">       
111
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-33-3"  
class="td11">        
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-33-4"  
class="td11">    
4<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">151</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-34-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-34-1"  
class="td11"> 
Cap-dependent Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-34-2"  
class="td11">       
111
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-34-3"  
class="td11">        
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-34-4"  
class="td11">    
4<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">151</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-35-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-35-1"  
class="td11"> 
Influenza Infection
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-35-2"  
class="td11">       
117
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-35-3"  
class="td11">        
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-35-4"  
class="td11">    
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">145</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-36-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-36-1"  
class="td11"> 
Influenza Viral <a 
href="#glo:RNA">RNA</a> Transcription and Replication
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-36-2"  
class="td11">       
108
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-36-3"  
class="td11">        
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-36-4"  
class="td11">    
5<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">145</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-37-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-37-1"  
class="td11"> 
Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-37-2"  
class="td11">       
141
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-37-3"  
class="td11">        
81
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-37-4"  
class="td11">    
8<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">143</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-38-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-38-1"  
class="td11"> 
Influenza Life Cycle
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-38-2"  
class="td11">       
112
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-38-3"  
class="td11">        
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-38-4"  
class="td11">    
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">141</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-60-39-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-39-1"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 3</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-39-2"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-39-3"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-39-4"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">p-value (</span><a 
href="#glo:FDR"><span class="cmbx-12x-x-120">FDR</span></a><span class="cmbx-12x-x-120">)</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-60-40-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-40-1"  
class="td11"> 
Adaptive Immune System
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-40-2"  
class="td11">       
412
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-40-3"  
class="td11">        
90
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-40-4"  
class="td11">     
6<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">61</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-41-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-41-1"  
class="td11"> 
Chemokine receptors bind chemokines
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-41-2"  
class="td11">        
52
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-41-3"  
class="td11">        
27
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-41-4"  
class="td11">     
6<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">56</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-42-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-42-1"  
class="td11"> 
Generation of second messenger molecules
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-42-2"  
class="td11">        
29
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-42-3"  
class="td11">        
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-42-4"  
class="td11">     
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">55</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-43-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-43-1"  
class="td11"> 
Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-43-2"  
class="td11">        
64
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-43-3"  
class="td11">        
29
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-43-4"  
class="td11">     
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">55</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-44-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-44-1"  
class="td11"> 
TCR signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-44-2"  
class="td11">        
62
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-44-3"  
class="td11">        
27
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-44-4"  
class="td11">     
8<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">51</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-45-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-45-1"  
class="td11"> 
Peptide ligand-binding receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-45-2"  
class="td11">       
161
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-45-3"  
class="td11">        
40
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-45-4"  
class="td11">     
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">45</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-46-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-46-1"  
class="td11"> 
Translocation of ZAP-70 to Immunological synapse
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-46-2"  
class="td11">        
16
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-46-3"  
class="td11">        
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-46-4"  
class="td11">     
3<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">43</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-47-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-47-1"  
class="td11"> 
Costimulation by the CD28 family
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-47-2"  
class="td11">        
51
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-47-3"  
class="td11">        
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-47-4"  
class="td11">     
4<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">43</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-48-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-48-1"  
class="td11"> 
PD-1 signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-48-2"  
class="td11">        
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-48-3"  
class="td11">        
15
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-48-4"  
class="td11">     
4<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">41</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-49-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-49-1"  
class="td11"> 
Class A/1 (Rhodopsin-like receptors)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-49-2"  
class="td11">       
258
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-49-3"  
class="td11">        
50
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-49-4"  
class="td11">     
6<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">41</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-50-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-50-1"  
class="td11"> 
Phosphorylation of CD3 and TCR zeta chains
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-50-2"  
class="td11">        
18
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-50-3"  
class="td11">        
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-50-4"  
class="td11">     
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">40</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-51-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-51-1"  
class="td11"> 
Interferon gamma signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-51-2"  
class="td11">        
74
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-51-3"  
class="td11">        
24
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-51-4"  
class="td11">     
5<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">39</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-52-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-52-1"  
class="td11"> 
GPCR ligand binding
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-52-2"  
class="td11">       
326
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-52-3"  
class="td11">        
57
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-52-4"  
class="td11">     
1<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">38</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-53-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-53-1"  
class="td11"> 
Cytokine Signalling in Immune system
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-53-2"  
class="td11">       
268
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-53-3"  
class="td11">        
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-53-4"  
class="td11">     
8<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">37</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-54-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-54-1"  
class="td11"> 
Downstream TCR signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-54-2"  
class="td11">        
45
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-54-3"  
class="td11">        
18
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-54-4"  
class="td11">     
1<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">35</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-55-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-55-1"  
class="td11"> 
G<sub><span class="cmmi-8">αi</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-55-2"  
class="td11">       
167
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-55-3"  
class="td11">        
33
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-55-4"  
class="td11">     
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">33</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-56-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-56-1"  
class="td11"> 
Cell surface interactions at the vascular wall
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-56-2"  
class="td11">        
99
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-56-3"  
class="td11">        
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-56-4"  
class="td11">     
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-57-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-57-1"  
class="td11"> 
Interferon Signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-57-2"  
class="td11">       
164
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-57-3"  
class="td11">        
28
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-57-4"  
class="td11">     
1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-60-58-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-58-1"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 4</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-58-2"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-58-3"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-58-4"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">p-value (</span><a 
href="#glo:FDR"><span class="cmbx-12x-x-120">FDR</span></a><span class="cmbx-12x-x-120">)</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-60-59-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-59-1"  
class="td11"> 
Extracellular matrix organisation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-59-2"  
class="td11">       
238
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-59-3"  
class="td11">        
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-59-4"  
class="td11">     
8<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">41</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-60-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-60-1"  
class="td11"> 
Class A/1 (Rhodopsin-like receptors)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-60-2"  
class="td11">       
258
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-60-3"  
class="td11">        
47
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-60-4"  
class="td11">     
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">36</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-61-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-61-1"  
class="td11"> 
GPCR ligand binding
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-61-2"  
class="td11">       
326
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-61-3"  
class="td11">        
54
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-61-4"  
class="td11">     
2<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">34</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-62-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-62-1"  
class="td11"> 
G<sub><span class="cmmi-8">αs</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-62-2"  
class="td11">        
83
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-62-3"  
class="td11">        
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-62-4"  
class="td11">     
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-63-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-63-1"  
class="td11"> 
GPCR downstream signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-63-2"  
class="td11">       
472
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-63-3"  
class="td11">        
68
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-63-4"  
class="td11">     
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-64-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-64-1"  
class="td11"> 
Haemostasis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-64-2"  
class="td11">       
423
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-64-3"  
class="td11">        
61
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-64-4"  
class="td11">     
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-65-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-65-1"  
class="td11"> 
Platelet activation, signalling and aggregation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-65-2"  
class="td11">       
180
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-65-3"  
class="td11">        
31
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-65-4"  
class="td11">     
7<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-66-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-66-1"  
class="td11"> 
Binding and Uptake of Ligands by Scavenger Receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-66-2"  
class="td11">        
40
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-66-3"  
class="td11">        
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-66-4"  
class="td11">     
9<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-67-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-67-1"  
class="td11"> 
RA biosynthesis <a 
href="#glo:pathway">pathways</a>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-67-2"  
class="td11">        
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-67-3"  
class="td11">        
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-67-4"  
class="td11">     
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-68-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-68-1"  
class="td11"> 
Response to elevated platelet cytosolic Ca<sup><span class="cmr-8">2+</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-68-2"  
class="td11">        
82
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-68-3"  
class="td11">        
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-68-4"  
class="td11">     
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-69-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-69-1"  
class="td11"> 
Developmental Biology
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-69-2"  
class="td11">       
420
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-69-3"  
class="td11">        
57
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-69-4"  
class="td11">     
3<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-70-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-70-1"  
class="td11"> 
G<sub><span class="cmmi-8">αi</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-70-2"  
class="td11">       
167
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-70-3"  
class="td11">        
28
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-70-4"  
class="td11">     
7<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-71-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-71-1"  
class="td11"> 
Platelet degranulation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-71-2"  
class="td11">        
77
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-71-3"  
class="td11">        
18
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-71-4"  
class="td11">     
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-72-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-72-1"  
class="td11"> 
Gastrin-CREB signalling <a 
href="#glo:pathway">pathways</a> via PKC and MAPK
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-72-2"  
class="td11">       
171
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-72-3"  
class="td11">        
28
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-72-4"  
class="td11">     
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-73-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-73-1"  
class="td11"> 
Muscle contraction
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-73-2"  
class="td11">        
62
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-73-3"  
class="td11">        
16
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-73-4"  
class="td11">     
4<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-74-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-74-1"  
class="td11"> 
G<sub><span class="cmmi-8">αq</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-74-2"  
class="td11">       
150
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-74-3"  
class="td11">        
25
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-74-4"  
class="td11">     
3<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-75-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-75-1"  
class="td11"> 
Retinoid metabolism and transport
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-75-2"  
class="td11">        
34
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-75-3"  
class="td11">        
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-75-4"  
class="td11">     
5<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-60-76-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-76-1"  
class="td11"> 
Phase 1 - Functionalisation of compounds
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-76-2"  
class="td11">        
67
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-76-3"  
class="td11">        
16
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-60-76-4"  
class="td11">     
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-60-77-"><td  style="white-space:nowrap; text-align:left;" id="TBL-60-77-1"  
class="td11">                                                                   </td></tr></table>
<!--l. 397--><p class="noindent" ></div>
<p>Pathway over-representation analysis for Reactome <a 
href="#glo:pathway">pathways</a> with the number of genes in each <a 
href="#glo:pathway">pathways</a> (Pathway Size), number of genes within the <a 
href="#glo:pathway">pathways</a> identified (Cluster Genes), and the <a 
href="#glo:pathway">pathways</a> over-representation p-value (adjusted by <a 
href="#glo:FDR">FDR</a>) from the hypergeometric test.</p>
</p>
</div><hr class="endfloat" />
<!--l. 401--><p class="noindent" >
Synthetic lethal gene candidates for <span class="cmti-12">CDH1 </span>from <a 
href="#glo:SLIPT">SLIPT</a> analysis of <a 
href="#glo:RNA-Seq">RNA-Seq</a> <a 
href="#glo:gene expression">gene expression</a> data were also used for <a 
href="#glo:pathway">pathways</a> over-representation analyses (as described in Section <a 
href="#x1-580002">2.3.2<!--tex4ht:ref: methods:enrichment --></a>). The correlation structure in the <a 
href="#glo:gene expression">expression</a> of candidates <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes in <span class="cmti-12">CDH1 </span>low tumours (lowest <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span><sup class="textsuperscript"><span class="cmr-10x-x-109">rd</span></sup> quantile of <a 
href="#glo:gene expression">expression</a>) was examined for distinct biological <a 
href="#glo:pathway">pathways</a> in subgroups of genes elevated in different clusters of samples. These genes were highly expressed in different samples with their clinical factors including estrogen receptor status and <a 
href="#glo:intrinsic subtype">intrinsic subtypes</a>, from the <a 
href="#glo:PAM50">PAM50</a> procedure (<a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>) shown in Figure <a 
href="#x1-990011">4.1<!--tex4ht:ref: fig:slipt_expr --></a>.
</p><!--l. 404--><p class="indent" >
As shown by the most over-represented <a 
href="#glo:pathway">pathways</a> in Table <a 
href="#x1-1000013">4.3<!--tex4ht:ref: tab:pathway_clusters --></a>, each correlated cluster of candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">CDH1 </span>contains functionally different genes. Cluster 1 contains genes with less evidence of over-represented <a 
href="#glo:pathway">pathways</a> than other clusters, corresponding to less correlation between genes within the cluster, and to it being a relatively small group. While there is some indication that collagen biosynthesis, microfibril elastic fibres, extracellular matrix, and metabolic <a 
href="#glo:pathway">pathways</a> may be over-represented in Cluster 1, these results are mainly based on small <a 
href="#glo:pathway">pathways</a> containing few <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes. Genes in Cluster 2 exhibited low <a 
href="#glo:gene expression">expression</a> in normal tissue samples compared to tumour samples (as shown in Figure <a 
href="#x1-990011">4.1<!--tex4ht:ref: fig:slipt_expr --></a>) and show compelling evidence of over-representation of post-transcriptional gene regulation and protein translation processes. Similarly, Cluster 3 has over-representation of immune signalling <a 
href="#glo:pathway">pathways</a> (including chemokines, secondary messenger, and TCR signalling) and downstream intracellular signalling cascades such as <a 
href="#glo:GPCR">GPCR</a> and G<sub><span class="cmmi-8">αi</span></sub> signalling events. While <a 
href="#glo:pathway">pathways</a> over-representation was weaker among genes in Cluster 4, they contained intracellular signalling <a 
href="#glo:pathway">pathways</a> and were highly expressed in normal samples (in contrast to Cluster 2). Cluster 4 also involved extracellular factors and stimuli such as extracellular matrix, platelet activation, ligand receptors, and retinoic acid signalling.
</p><!--l. 407--><p class="indent" >
Based on these results, potential <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">CDH1 </span>include processes known to be dysregulated in cancer, such as translational, cytoskeletal, and immune processes. Intracellular signalling cascades such as the <a 
href="#glo:GPCR">GPCRs</a> and extracellular stimuli for these <a 
href="#glo:pathway">pathways</a> were also implicated in potential <a 
href="#glo:synthetic lethal">synthetic lethality</a> with <span class="cmti-12">CDH1</span>.
</p><!--l. 409--><p class="indent" >
<p>Similar translational, cytoskeletal, and immune processes were identified among <a 
href="#glo:SLIPT">SLIPT</a> partners with respect to <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutation</a>, shown in Appendix Table <a 
href="#x1-1730023">C.3<!--tex4ht:ref: tab:pathway_clusters_mtSL --></a>.</p>
While <a 
href="#glo:GPCR">GPCR</a> signalling was replicated in <a 
href="#glo:mtSLIPT">mtSLIPT</a> analysis, there was also stronger over-representation for NOTCH, ERBB2, and PI3K/AKT signalling in <a 
href="#glo:mutation">mutation</a> analysis consistent with these signals being important for proliferation of <span class="cmti-12">CDH1</span> deficient tumours. The GCPR and PI3K/AKT <a 
href="#glo:pathway">pathways</a> are of particular interest as <a 
href="#glo:pathway">pathways</a> with oncogenic <a 
href="#glo:mutation">mutations</a> that can be targeted and downstream effects on translation (a strongly supported process across analyses). Extracellular matrix <a 
href="#glo:pathway">pathways</a> (e.g., elastic fibre formation) were also supported across analyses (in Table <a 
href="#x1-1000013">4.3<!--tex4ht:ref: tab:pathway_clusters --></a> and Appendix Table <a 
href="#x1-1730023">C.3<!--tex4ht:ref: tab:pathway_clusters_mtSL --></a>) consistent with the established cell-cell signalling role of <span class="cmti-12">CDH1 </span>and the importance of the tumour microenvironment for cancer proliferation. <a 
 id="x1-100002r187"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">4.2 </span> <a 
 id="x1-1010002"></a>Comparing Synthetic Lethal Gene Candidates
</h3>
<a 
 id="x1-101001r191"></a>
<h4 class="subsectionHead">
<span class="titlemark">4.2.1 </span> <a 
 id="x1-1020001"></a>Primary siRNA Screen Candidates
</h4>
<!--l. 425--><p class="noindent" >
Gene candidates were compared between computational (<a 
href="#glo:SLIPT">SLIPT</a> in <a 
href="#glo:TCGA">TCGA</a> breast cancer data) and experimental (the primary <a 
href="#glo:siRNA">siRNA</a> screen performed by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>)) approaches in Figure <a 
href="#x1-1020012">4.2<!--tex4ht:ref: fig:Venn_allgenes --></a>. The number of genes detected by both methods did not produce a significant overlap but these may be difficult to compare due to vast differences between the detection methods. There were similar issues in the comparison of <a 
href="#glo:mtSLIPT">mtSLIPT</a> genes tested against <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutations</a> (in Appendix Figure <a 
href="#x1-1740012">C.2<!--tex4ht:ref: fig:Venn_allgenes_mtSL --></a>), despite excluding genes not tested by both methods in either test. However, these intersecting genes may still be functionally informative or amenable to drug triage as they were replicated across both methods and <a 
href="#glo:pathway">pathways</a> over-representation differed between the Sections of the Venn diagram (as shown in Figure <a 
href="#x1-1020012">4.2<!--tex4ht:ref: fig:Venn_allgenes --></a>).
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                    

<p><a 
 id="x1-1020012"></a></p>
<!--l. 430--><p class="noindent" >
<img 
src="Venn_exprSL_siRNA_allgenes_reduced_Pub.png" alt="PIC"  
 /> <br />
</p><div class="caption" 
>
<span class="id">Figure 4.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Comparison of </span><a 
href="#glo:SLIPT"><span class="cmbx-10x-x-109">SLIPT</span></a> <span class="cmbx-10x-x-109">with </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a><span class="cmbx-10x-x-109">.</span></strong> <span class="cmr-10x-x-109">Testing the overlap of gene candidates</span> <span class="cmr-10x-x-109">for </span><a 
href="#glo:E-cadherin"><span class="cmr-10x-x-109">E-cadherin</span></a> <a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners between computational (</span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a><span class="cmr-10x-x-109">) and experimental</span> <span class="cmr-10x-x-109">screening (</span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a><span class="cmr-10x-x-109">) approaches. The </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test suggests that the overlap is no more than would</span> <span class="cmr-10x-x-109">be expected by chance (</span><span class="cmmi-10x-x-109">p </span><span class="cmr-10x-x-109">= 0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">281). Only genes tested by both methods were included. </span></span>
</div><!--tex4ht:label?: x1-1020012 -->
                                                                                    

</div><hr class="endfigure" />
                                                                                    

<!--l. 444--><p class="indent" >
<a 
 id="x1-1020023"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 446--><p class="noindent" >
</p><!--l. 447--><p class="noindent" >
<img 
src="exprSLIPT_siRNA_vs_Correlation_with_CDH1_nlp.png" alt="pict"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 4.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">Comparison of </span><a 
href="#glo:SLIPT"><span class="cmbx-10x-x-109">SLIPT</span></a> <span class="cmbx-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">genes with correlation.</span></strong> <span class="cmr-10x-x-109">The </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">p-values for genes tested by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">(in </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">breast cancer) </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">analysis were</span> <span class="cmr-10x-x-109">compared against Pearson correlation of </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">gene expression</span></a> <span class="cmr-10x-x-109">with </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">. Genes detected by</span> <a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">or </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">are coloured according to the legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-1020023 -->
                                                                                    

</div><hr class="endfloat" />
<a 
 id="x1-102003r196"></a>
<h4 class="subsectionHead">
<span class="titlemark">4.2.2 </span> <a 
 id="x1-1030002"></a>Comparison with Correlation
</h4>
<!--l. 459--><p class="noindent" >
Another potential means to triage drug target candidates is by correlation of <a 
href="#glo:gene expression">expression</a> profiles with <span class="cmti-12">CDH1</span>. Correlation with <span class="cmti-12">CDH1 </span>was compared to <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> results in Figure <a 
href="#x1-1020023">4.3<!--tex4ht:ref: fig:compare_points_correlation_SL --></a>. The genes not detected by <a 
href="#glo:SLIPT">SLIPT</a> (including <a 
href="#glo:siRNA">siRNA</a> candidates) included genes with non-significant <a 
href="#glo:SLIPT">SLIPT</a> p-values. As expected, these genes were distributed around a correlation of zero. Genes with higher correlation with <span class="cmti-12">CDH1</span> (either direction) were more significant, although there were exceptions to this trend and larger positive correlations than negative correlations. The majority of <a 
href="#glo:SLIPT">SLIPT</a> candidates had negative correlations, particularly genes detected by both approaches, although these were typically weak correlations and are unlikely to be sufficient to detect such genes on their own. This is supported by simulation results in Section <a 
href="#x1-1350001">6.1<!--tex4ht:ref: chapt5:compare_methods --></a>.
</p><!--l. 464--><p class="indent" >
There were not strong positive correlations with <span class="cmti-12">CDH1 </span>among <a 
href="#glo:siRNA">siRNA</a> candidates, consistent with previous findings that co-expression was not predictive of <a 
href="#glo:synthetic lethal">synthetic lethality</a> (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>; <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>). Negative correlation may not be indicative of <a 
href="#glo:synthetic lethal">synthetic lethality</a> either as many <a 
href="#glo:siRNA">siRNA</a> candidates also had positive correlations. The <a 
href="#glo:SLIPT">SLIPT</a> methodology has shown to detect genes with both positive and negative correlations, although it does appear to preferentially detect negatively correlated genes to some extent. These findings were replicated with the <a 
href="#glo:mtSLIPT">mtSLIPT</a> approach against <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutation</a> (in Appendix Figure <a 
href="#x1-1760013">C.3<!--tex4ht:ref: fig:compare_points_correlation_mtSL --></a>), although the range of the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> p-values differs due to lower sample size for <a 
href="#glo:mutation">mutation</a> analysis.
</p><!--l. 468--><p class="indent" >
<p>The apparent tendency for genes detected by <a 
href="#glo:SLIPT">SLIPT</a> or <a 
href="#glo:siRNA">siRNA</a> to have negative correlations with <span class="cmti-12">CDH1 </span><a 
href="#glo:gene expression">expression</a> was not due to the smaller number of genes in these groups. The distribution of <span class="cmti-12">CDH1 </span>correlations differed across these gene groups (as shown by Figure <a 
href="#x1-1030014">4.4<!--tex4ht:ref: fig:compare_correlation_SL --></a> and Appendix Figure <a 
href="#x1-1760024">C.4<!--tex4ht:ref: fig:compare_correlation_mtSL --></a>) and tended to be lower in <a 
href="#glo:SLIPT">SLIPT</a> candidates (as supported by <a 
href="#glo:ANOVA">analysis of variance (ANOVA)</a> in Table <a 
href="#x1-1030024">4.4<!--tex4ht:ref: tab:compare_correlation_SL --></a>). However, these are relatively weak correlations and further triage of gene candidates by correlation is not suitable. The genes detected both <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> did not differ from <a 
href="#glo:SLIPT">SLIPT</a> genes and the number of positively correlated <a 
href="#glo:SLIPT">SLIPT</a> genes was very small.The use of correlation itself is also less effective than <a 
href="#glo:SLIPT">SLIPT</a> to predict <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in the first place (as shown in Section <a 
href="#x1-1390001">6.1.2.1<!--tex4ht:ref: chapt5:compare_correlation --></a>).</p>
</p><!--l. 470--><p class="indent" >
<a 
 id="x1-1030014"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 472--><p class="noindent" >
</p><!--l. 473--><p class="noindent" >
<img 
src="vioplotx_exprSLIPT_siRNA_vs_CDH1_Correlation_with_CDH1.png" alt="pict"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 4.4: </span><span class="content"><strong><span class="cmbx-10x-x-109">Comparison of </span><a 
href="#glo:SLIPT"><span class="cmbx-10x-x-109">SLIPT</span></a> <span class="cmbx-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">genes with correlation.</span></strong> <span class="cmr-10x-x-109">Genes</span> <span class="cmr-10x-x-109">detected as candidate </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">(in </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">breast cancer) </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">analysis and experimental screening (with </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a><span class="cmr-10x-x-109">) were compared against Pearson correlation</span> <span class="cmr-10x-x-109">of </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">gene expression</span></a> <span class="cmr-10x-x-109">with </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">. There were significant differences in correlation between gene</span> <span class="cmr-10x-x-109">groups (as shown in Table</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-1030024"><span class="cmr-10x-x-109">4.4</span><!--tex4ht:ref: tab:compare_correlation_SL --></a><span class="cmr-10x-x-109">). </span></span></p>
</div>
<!--tex4ht:label?: x1-1030014 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 483--><p class="indent" >
<a 
 id="x1-1030024"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 4.4: </span><span class="content"><a 
href="#glo:ANOVA">ANOVA</a> for synthetic lethality and correlation with <span class="cmti-12">CDH1</span></span>
</div><!--tex4ht:label?: x1-1030024 -->
<div class="tabular">
<table id="TBL-61" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-61-1g"><col 
id="TBL-61-1" /><col 
id="TBL-61-2" /><col 
id="TBL-61-3" /><col 
id="TBL-61-4" /><col 
id="TBL-61-5" /><col 
id="TBL-61-6" /></colgroup><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-61-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-61-1-1"  
class="td11">              </td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-1-2"  
class="td11"> 
DF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-1-3"  
class="td11"> 
Sum Squares
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-1-4"  
class="td11"> 
Mean Squares
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-1-5"  
class="td11">  
F-value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-1-6"  
class="td11"> 
p-value
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-61-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-61-2-1"  
class="td11"> 
siRNA
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-2-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-2-3"  
class="td11">    
0.027
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-2-4"  
class="td11">     
0.027
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-2-5"  
class="td11">   
2.8209
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-2-6"  
class="td11"> 
0.09306
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-61-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-61-3-1"  
class="td11"> 
SLIPT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-3-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-3-3"  
class="td11">   
134.603
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-3-4"  
class="td11">    
134.603
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-3-5"  
class="td11"> 
14115.9824
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-3-6"  
class="td11"> 
<span class="cmmi-12">&lt;</span>0.0001
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-61-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-61-4-1"  
class="td11"> 
siRNA<span class="cmsy-10x-x-120">×</span>SLIPT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-4-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-4-3"  
class="td11">    
0.000
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-4-4"  
class="td11">     
0.000
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-4-5"  
class="td11">   
0.0073
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-61-4-6"  
class="td11"> 
0.93212
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-61-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-61-5-1"  
class="td11">               </td></tr></table>
<!--l. 507--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Analysis of variance for correlation with </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">against </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">detection approaches (with</span> <span class="cmr-10x-x-109">an interaction term). Only genes tested by both methods were included in this analysis.</span></p>
</p>
</div><hr class="endfloat" />
<a 
 id="x1-103003r199"></a>
<h4 class="subsectionHead">
<span class="titlemark">4.2.3 </span> <a 
 id="x1-1040003"></a>Comparison with Primary Screen Viability
</h4>
<!--l. 514--><p class="noindent" >
<p>A similar comparison of <a 
href="#glo:SLIPT">SLIPT</a> results was made with the viability ratio (<span class="cmti-12">CDH1</span><sup><span class="cmsy-8">-</span><span class="cmmi-8">∕</span><span class="cmsy-8">-</span></sup> <a 
href="#glo:mutant">mutant</a> to <a 
href="#glo:wild-type">wild-type</a>) of MCF10A cells in the primary <a 
href="#glo:siRNA">siRNA</a> screen performed by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>). The significance and viability thresholds used for <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> detection of <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate partners of <span class="cmti-12">CDH1 </span>are shown in Figure <a 
href="#x1-1040015">4.5<!--tex4ht:ref: fig:compare_points_viability_SL --></a>. Not all of the genes below the viability thresholds were necessarily selected to be candidate partners, however, as additional criteria were used in each case: directional criteria as for <a 
href="#glo:SLIPT">SLIPT</a> (in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) and minimum<a 
href="#glo:wild-type">wild-type</a> viability for <a 
href="#glo:siRNA">siRNA</a> (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>).</p>
</p><!--l. 516--><p class="indent" >
<a 
 id="x1-1040015"></a><a 
 id="x1-1040026"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 518--><p class="noindent" >
</p><!--l. 519--><p class="noindent" >
<img 
src="exprSLIPT_siRNA_vs_Viability_Ratio_with_CDH1_nlp.png" alt="pict"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 4.5: </span><span class="content"><strong><span class="cmbx-10x-x-109">Comparison of </span><a 
href="#glo:SLIPT"><span class="cmbx-10x-x-109">SLIPT</span></a> <span class="cmbx-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">genes with screen viability.</span></strong> <span class="cmr-10x-x-109">The</span> <span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span> </sup> <span class="cmr-10x-x-109">p-values (log-scale) for genes tested by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">(in </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">breast cancer) were compared</span> <span class="cmr-10x-x-109">against the viability ratio of </span><span class="cmti-10x-x-109">CDH1 </span><a 
href="#glo:mutant"><span class="cmr-10x-x-109">mutant</span></a> <span class="cmr-10x-x-109">and</span><a 
href="#glo:wild-type"><span class="cmr-10x-x-109">wild-type</span></a> <span class="cmr-10x-x-109">cells in the primary </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screen.</span> <span class="cmr-10x-x-109">Genes detected by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">or </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">are coloured according to the legend. Lines show the</span> <span class="cmr-10x-x-109">thresholds of significance with </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and of viability used by </span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">Telford </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">2015</span></a><span class="cmr-10x-x-109">). </span></span></p>
</div>
<!--tex4ht:label?: x1-1040015 -->
<div class="center" 
>
<!--l. 531--><p class="noindent" >
</p><!--l. 532--><p class="noindent" >
<img 
src="vioplotx_exprSLIPT_siRNA_vs_Viability_Ratio_with_CDH1.png" alt="pict"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 4.6: </span><span class="content"><strong><span class="cmbx-10x-x-109">Comparison of </span><a 
href="#glo:SLIPT"><span class="cmbx-10x-x-109">SLIPT</span></a> <span class="cmbx-10x-x-109">genes with </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">screen viability.</span></strong> <span class="cmr-10x-x-109">Genes</span> <span class="cmr-10x-x-109">detected as candidate </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">(in </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">breast cancer) </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">analysis were compared against the viability ratio of </span><span class="cmti-10x-x-109">CDH1 </span><a 
href="#glo:mutant"><span class="cmr-10x-x-109">mutant</span></a> <span class="cmr-10x-x-109">and</span><a 
href="#glo:wild-type"><span class="cmr-10x-x-109">wild-type</span></a> <span class="cmr-10x-x-109">cells in the</span> <span class="cmr-10x-x-109">primary </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screen. There were clear no differences in viability between genes detected by</span> <a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and those not detected. The genes identified by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">had a higher viability ratio (by</span> <span class="cmr-10x-x-109">t-test: </span><span class="cmmi-10x-x-109">t </span><span class="cmr-10x-x-109">= 2</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">1553, </span><span class="cmmi-10x-x-109">p </span><span class="cmr-10x-x-109">= 0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">03117), although the effect size was relatively small (mean SLIPT</span><span class="cmsy-10x-x-109">-</span> <span class="cmr-10x-x-109">1.029, mean SLIPT+ 1.037). </span></span></p>
</div>
<!--tex4ht:label?: x1-1040026 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 542--><p class="indent" >
There does not appear to be a clear relationship between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidates. The genes detected by one approach but not the other were numerous in Figure <a 
href="#x1-1020012">4.2<!--tex4ht:ref: fig:Venn_allgenes --></a> and Appendix Figure <a 
href="#x1-1740012">C.2<!--tex4ht:ref: fig:Venn_allgenes_mtSL --></a>. These genes detected by one approach are not necessarily near the thresholds for the other. In this respect, the <a 
href="#glo:SLIPT">SLIPT</a> approach with patient data and the <a 
href="#glo:siRNA">siRNA</a> cell line experiments are independent means to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates. While genes detected by both approaches were not necessarily more strongly supported by either, the genes with a viability closer to 1 (no <a 
href="#glo:synthetic lethal">synthetic lethal</a> effect) in <a 
href="#glo:siRNA">siRNA</a> included those with more significant <a 
href="#glo:SLIPT">SLIPT</a> p-values, whereas more extreme viability ratios tended to be less significant (as shown by Figure <a 
href="#x1-1040015">4.5<!--tex4ht:ref: fig:compare_points_viability_SL --></a>). However, it should be noted that genes with more moderate viability ratios were more common and <a 
href="#glo:SLIPT">SLIPT</a> was capable (despite adjusting for multiple testing) of detecting significant genes with extreme viability ratios, particularly those considerably lower than 1. Lower viability ratios were used by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates in the primary screen. However, there was little support for <a 
href="#glo:SLIPT">SLIPT</a> candidates differing with respect to viability ratio (as shown in Figures <a 
href="#x1-1040026">4.6<!--tex4ht:ref: fig:compare_viability_SL --></a> and <a 
href="#x1-1760035">C.5<!--tex4ht:ref: fig:compare_viability_mtSL --></a>) and the vast majority of <a 
href="#glo:SLIPT">SLIPT</a> candidate genes did not have different viability in the primary screen to genes not identified by <a 
href="#glo:SLIPT">SLIPT</a>. <a 
 id="x1-104003r202"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.2.4 </span> <a 
 id="x1-1050004"></a>Comparison with Secondary siRNA Screen Validation
</h4>
<!--l. 553--><p class="noindent" >
<p>It should be noted that genes with a lower viability ratio were not necessarily the most strongly supported by experimental screening. The primary screen (with 4 pooled <a 
href="#glo:siRNA">siRNAs</a> for each gene) has been used for the majority of comparisons in this thesis because the <a 
href="#glo:genome">genomes</a>-wide panel of target genes screened enables a large number of genes to be compared with <a 
href="#glo:SLIPT">SLIPT</a> results from <a 
href="#glo:gene expression">gene expression</a> and <a 
href="#glo:somatic">somatic</a> <a 
href="#glo:mutation">mutation</a> analysis. A secondary screen was also performed by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) on the isogenic MCF10A breast cell lines to validate the individual (i.e., non-pooled) <a 
href="#glo:siRNA">siRNAs</a> separately, with the strongest candidates being those exhibiting <a 
href="#glo:synthetic lethal">synthetic lethal</a> viability ratios replicated across independently targeting <a 
href="#glo:siRNA">siRNAs</a>. The strongest candidates from the primary screen were subject to a further secondary screen for validation by independent replication with 4 gene knockdowns with different targeting <a 
href="#glo:siRNA">siRNAs</a>. This was performed for the top 500 candidates (with the lowest viability ratio)</p>
from the primary screen: 482 of these genes were also tested by <a 
href="#glo:SLIPT">SLIPT</a> in breast cancer.
</p><!--l. 555--><p class="indent" >
<p>The secondary screen results show that <a 
href="#glo:SLIPT">SLIPT</a> candidate genes were more significantly (<span class="cmmi-12">p </span>= 7<span class="cmmi-12">.</span>49 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">3</span></sup> by Fisher’s exact test) more likely to be validated with detection by more independently targeting <a 
href="#glo:siRNA">siRNAs</a> in the secondary screen. Gene detected by <a 
href="#glo:SLIPT">SLIPT</a> are thus informative of more robust partner genes, in addition to providing support that these interactions are consistent with <a 
href="#glo:gene expression">expression</a> profiles from heterogeneous patient samples across genetic backgrounds. As shown in Table <a 
href="#x1-1050015">4.5<!--tex4ht:ref: tab:secondary_screen --></a>, there is significant association between <a 
href="#glo:SLIPT">SLIPT</a> candidates and stronger validations of <a 
href="#glo:siRNA">siRNA</a> candidates. Since there were more SLIPT<span class="cmsy-10x-x-120">-</span> genes among those not validated and more SLIPT+ genes among those validated with several <a 
href="#glo:siRNA">siRNAs</a>, this supports the use of SLIPT as a <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery procedure which may augment such screening experiments.</p>
</p><!--l. 558--><p class="indent" >
<a 
 id="x1-1050015"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 4.5: </span><span class="content">Comparison of <a 
href="#glo:SLIPT">SLIPT</a> genes against secondary <a 
href="#glo:siRNA">siRNA</a> screen</span>
</div><!--tex4ht:label?: x1-1050015 -->
<div class="center" 
>
<!--l. 561--><p class="noindent" >
</p><!--l. 564--><p class="noindent" ></p>
<div class="tabular">
<table id="TBL-62" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-62-1g"><col 
id="TBL-62-1" /><col 
id="TBL-62-2" /><col 
id="TBL-62-3" /><col 
id="TBL-62-4" /><col 
id="TBL-62-5" /><col 
id="TBL-62-6" /><col 
id="TBL-62-7" /><col 
id="TBL-62-8" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-62-1-"><td  style="white-space:nowrap; text-align:right;" id="TBL-62-1-1"  
class="td11">          </td><td  style="white-space:nowrap; text-align:right;" id="TBL-62-1-2"  
class="td11">          </td><td colspan="3" style="white-space:nowrap; text-align:center;" id="TBL-62-1-3"  
class="td11"></td>        <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<span class="cmbx-12">Secondary Screen</span>
</div>         <td  style="white-space:nowrap; text-align:left;" id="TBL-62-1-8"  
class="td11">        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-62-2-"><td  style="white-space:nowrap; text-align:right;" id="TBL-62-2-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-62-3-"><td  style="white-space:nowrap; text-align:right;" id="TBL-62-3-1"  
class="td11">          </td><td  style="white-space:nowrap; text-align:right;" id="TBL-62-3-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:right;"> 
<strong><span class="cmbx-12">siRNAs</span></strong><sup class="textsuperscript"><span class="cmr-10x-x-109">*</span></sup>
</div></td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-3-3"  
class="td11"> 
0/4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-3-4"  
class="td11"> 
1/4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-3-5"  
class="td11"> 
2/4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-3-6"  
class="td11"> 
3/4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-3-7"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
4/4
</div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-62-3-8"  
class="td11">  
<strong><span class="cmbx-12">Total</span></strong>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-62-4-"><td  style="white-space:nowrap; text-align:right;" id="TBL-62-4-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-62-5-"><td  style="white-space:nowrap; text-align:right;" id="TBL-62-5-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:right;"></div>               </td><td  style="white-space:nowrap; text-align:right;" id="TBL-62-5-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:right;">
Observed
</div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-5-3"  
class="td11"> 
70
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-5-4"  
class="td11"> 
46
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-5-5"  
class="td11"> 
31
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-5-6"  
class="td11">  
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-5-7"  
class="td11">  <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
2
</div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-62-5-8"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"></div>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-62-6-"><td  style="white-space:nowrap; text-align:right;" id="TBL-62-6-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:right;"> <div class="multirow"><!-- rows=-2 -->
<span class="cmbx-12">SLIPT</span>+
</div></div> </td><td  style="white-space:nowrap; text-align:right;" id="TBL-62-6-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:right;">
Expected
</div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-6-3"  
class="td11"> 
85
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-6-4"  
class="td11"> 
44
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-6-5"  
class="td11"> 
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-6-6"  
class="td11">  
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-6-7"  
class="td11">  <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
2
</div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-62-6-8"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"><div class="multirow"><!-- rows=-2 -->
157
</div></div>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-62-7-"><td  style="white-space:nowrap; text-align:right;" id="TBL-62-7-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-62-8-"><td  style="white-space:nowrap; text-align:right;" id="TBL-62-8-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:right;"></div>               </td><td  style="white-space:nowrap; text-align:right;" id="TBL-62-8-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:right;">
Observed
</div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-8-3"  
class="td11"> 
190
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-8-4"  
class="td11"> 
90
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-8-5"  
class="td11"> 
31
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-8-6"  
class="td11"> 
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-8-7"  
class="td11">  <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
4
</div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-62-8-8"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"></div>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-62-9-"><td  style="white-space:nowrap; text-align:right;" id="TBL-62-9-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:right;"><div class="multirow"><!-- rows=-2 -->
<span class="cmbx-12">SLIPT</span><span class="cmsy-10x-x-120">-</span>
</div></div> </td><td  style="white-space:nowrap; text-align:right;" id="TBL-62-9-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:right;">
Expected
</div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-9-3"  
class="td11"> 
175
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-9-4"  
class="td11"> 
91
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-9-5"  
class="td11"> 
42
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-9-6"  
class="td11"> 
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-9-7"  
class="td11">  <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
4
</div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-62-9-8"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"><div class="multirow"><!-- rows=-2 -->
325
</div></div>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-62-10-"><td  style="white-space:nowrap; text-align:right;" id="TBL-62-10-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-62-11-"><td  style="white-space:nowrap; text-align:right;" id="TBL-62-11-1"  
class="td11">          </td><td  style="white-space:nowrap; text-align:right;" id="TBL-62-11-2"  
class="td11">   <div class="multicolumn"  style="white-space:nowrap; text-align:right;"> 
<span class="cmbx-12">Total</span>
</div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-11-3"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
280
</div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-11-4"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
136
</div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-11-5"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
52
</div>  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-11-6"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
18
</div>  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-62-11-7"  
class="td11">  <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
6
</div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-62-11-8"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
482
</div>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-62-12-"><td  style="white-space:nowrap; text-align:right;" id="TBL-62-12-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-62-13-"><td  style="white-space:nowrap; text-align:right;" id="TBL-62-13-1"  
class="td11">          </td></tr></table>
<!--l. 578--><p class="noindent" ></div>
<sup class="textsuperscript"><span class="cmr-10x-x-109">*</span></sup><span class="cmr-10x-x-109">Number of </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNAs</span></a> <span class="cmr-10x-x-109">(targeting the same gene) to successfully reproduce </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a> <span class="cmr-10x-x-109">in MCF10A cells (</span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">Telford </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109">,</span><span class="cmr-10x-x-109"> </span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">2015</span></a><span class="cmr-10x-x-109">)</span>
</p></div>
                                                                                    

</div><hr class="endfloat" />
<!--l. 589--><p class="indent" >
While the individual genes detected by either approach do not necessarily match (and are potentially false-positives), the biological functions important in <span class="cmti-12">CDH1 </span>deficient cancers and potential mechanisms for specific targeting of them can be further supported by <a 
href="#glo:pathway">pathways</a> analysis of the gene detected by either method. The genes detected by both approaches may therefore be more informative at the <a 
href="#glo:pathway">pathways</a> level, where it is less likely for a <a 
href="#glo:pathway">pathways</a> to be consistently detected by chance. As the <a 
href="#glo:SLIPT">SLIPT</a> candidates differ from the <a 
href="#glo:siRNA">siRNA</a> candidates (in addition to those detected by both approaches which were more likely to be validated), they can provide information about additional mechanisms by which <span class="cmti-12">CDH1 </span>deficient cancers proliferate, and vulnerabilities that may be exploited against them by using the <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a>. <a 
 id="x1-105002r205"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.2.5 </span> <a 
 id="x1-1060005"></a>Comparison to Primary Screen at Pathway Level
</h4>
                                                                                    

<!--l. 595--><p class="indent" >
<a 
 id="x1-1060016"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 4.6: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT and <a 
href="#glo:siRNA">siRNA</a></span>
</div><!--tex4ht:label?: x1-1060016 -->
<!--tex4ht:inline--><div class="tabular"><table id="TBL-63" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-63-1g"><col 
id="TBL-63-1" /><col 
id="TBL-63-2" /><col 
id="TBL-63-3" /><col 
id="TBL-63-4" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-63-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-1-1"  
class="td11"> 
<span class="cmbx-12">Predicted only by </span><a 
href="#glo:SLIPT"><span class="cmbx-12">SLIPT</span></a> <span class="cmbx-12">(4025 genes) </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-1-2"  
class="td11"> 
Pathway Size
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-1-3"  
class="td11"> 
Genes Identified
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-1-4"  
class="td11"> 
p-value (<a 
href="#glo:FDR">FDR</a>)
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-63-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-2-1"  
class="td11"> 
Eukaryotic Translation Elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-2-2"  
class="td11">      
80
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-2-3"  
class="td11">       
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-2-4"  
class="td11">  
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">182</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-3-1"  
class="td11"> 
Peptide chain elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-3-2"  
class="td11">      
77
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-3-3"  
class="td11">       
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-3-4"  
class="td11">  
2<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">176</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-4-1"  
class="td11"> 
Viral <a 
href="#glo:mRNA">mRNA</a> Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-4-2"  
class="td11">      
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-4-3"  
class="td11">       
70
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-4-4"  
class="td11">  
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">172</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-5-1"  
class="td11"> 
Eukaryotic Translation Termination
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-5-2"  
class="td11">      
76
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-5-3"  
class="td11">       
70
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-5-4"  
class="td11">  
5<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">170</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-6-1"  
class="td11"> 
Formation of a pool of free 40S subunits
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-6-2"  
class="td11">      
87
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-6-3"  
class="td11">       
74
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-6-4"  
class="td11">  
9<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">166</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-7-1"  
class="td11"> 
Nonsense Mediated Decay independent of the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-7-2"  
class="td11">      
81
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-7-3"  
class="td11">       
70
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-7-4"  
class="td11">  
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">160</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-8-1"  
class="td11"> 
L13a-mediated translational silencing of Ceruloplasmin <a 
href="#glo:gene expression">expression</a>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-8-2"  
class="td11">      
97
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-8-3"  
class="td11">       
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-8-4"  
class="td11">  
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">155</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-9-1"  
class="td11"> 
3’ -UTR-mediated translational regulation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-9-2"  
class="td11">      
97
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-9-3"  
class="td11">       
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-9-4"  
class="td11">  
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">155</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-10-1"  
class="td11"> 
GTP hydrolysis and joining of the 60S ribosomal subunit
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-10-2"  
class="td11">      
98
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-10-3"  
class="td11">       
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-10-4"  
class="td11">  
6<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">154</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-11-1"  
class="td11"> 
Nonsense-Mediated Decay
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-11-2"  
class="td11">      
96
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-11-3"  
class="td11">       
73
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-11-4"  
class="td11">  
5<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">150</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-12-1"  
class="td11"> 
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-12-2"  
class="td11">      
96
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-12-3"  
class="td11">       
73
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-12-4"  
class="td11">  
5<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">150</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-13-1"  
class="td11"> 
SRP-dependent cotranslational protein targeting to membrane
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-13-2"  
class="td11">      
97
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-13-3"  
class="td11">       
73
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-13-4"  
class="td11">  
7<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">149</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-14-1"  
class="td11"> 
Eukaryotic Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-14-2"  
class="td11">     
105
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-14-3"  
class="td11">       
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-14-4"  
class="td11">  
4<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">146</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-15-1"  
class="td11"> 
Cap-dependent Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-15-2"  
class="td11">     
105
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-15-3"  
class="td11">       
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-15-4"  
class="td11">  
4<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">146</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-16-1"  
class="td11"> 
Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-16-2"  
class="td11">     
133
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-16-3"  
class="td11">       
83
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-16-4"  
class="td11">  
4<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">142</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-17-1"  
class="td11"> 
Influenza Viral <a 
href="#glo:RNA">RNA</a> Transcription and Replication
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-17-2"  
class="td11">     
102
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-17-3"  
class="td11">       
71
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-17-4"  
class="td11">  
2<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">137</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-18-1"  
class="td11"> 
Influenza Infection
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-18-2"  
class="td11">     
111
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-18-3"  
class="td11">       
74
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-18-4"  
class="td11">  
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">137</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-19-1"  
class="td11"> 
Influenza Life Cycle
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-19-2"  
class="td11">     
106
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-19-3"  
class="td11">       
71
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-19-4"  
class="td11">  
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">133</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-20-1"  
class="td11"> 
Infectious disease
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-20-2"  
class="td11">     
326
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-20-3"  
class="td11">      
125
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-20-4"  
class="td11">  
4<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">120</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-21-1"  
class="td11"> 
Extracellular matrix organisation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-21-2"  
class="td11">     
189
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-21-3"  
class="td11">       
77
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-21-4"  
class="td11">  
5<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">95</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-63-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-22-1"  
class="td11">                                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-23-1"  
class="td11"> 
<span class="cmbx-12">Detected only by </span><a 
href="#glo:siRNA"><span class="cmbx-12">siRNA</span></a> <span class="cmbx-12">screen (1599 genes) </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-23-2"  
class="td11"> 
Pathway Size
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-23-3"  
class="td11"> 
Genes Identified
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-23-4"  
class="td11"> 
p-value (<a 
href="#glo:FDR">FDR</a>)
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-63-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-24-1"  
class="td11"> 
Class A/1 (Rhodopsin-like receptors)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-24-2"  
class="td11">     
282
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-24-3"  
class="td11">       
44
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-24-4"  
class="td11">  
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-25-1"  
class="td11"> 
GPCR ligand binding
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-25-2"  
class="td11">     
363
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-25-3"  
class="td11">       
52
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-25-4"  
class="td11">  
5<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-26-1"  
class="td11"> 
G<sub><span class="cmmi-8">αq</span></sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-26-2"  
class="td11">     
159
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-26-3"  
class="td11">       
26
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-26-4"  
class="td11">  
6<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">23</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-27-1"  
class="td11"> 
Gastrin-CREB signalling <a 
href="#glo:pathway">pathways</a> via PKC and MAPK
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-27-2"  
class="td11">     
180
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-27-3"  
class="td11">       
27
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-27-4"  
class="td11">  
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-28-1"  
class="td11"> 
G<sub><span class="cmmi-8">αi</span></sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-28-2"  
class="td11">     
184
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-28-3"  
class="td11">       
27
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-28-4"  
class="td11">  
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-29-1"  
class="td11"> 
Downstream signal transduction
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-29-2"  
class="td11">     
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-29-3"  
class="td11">       
23
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-29-4"  
class="td11">  
7<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-30-1"  
class="td11"> 
Signalling by PDGF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-30-2"  
class="td11">     
172
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-30-3"  
class="td11">       
25
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-30-4"  
class="td11">  
4<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-31-1"  
class="td11"> 
Peptide ligand-binding receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-31-2"  
class="td11">     
175
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-31-3"  
class="td11">       
25
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-31-4"  
class="td11">  
8<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-32-1"  
class="td11"> 
Signalling by ERBB2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-32-2"  
class="td11">     
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-32-3"  
class="td11">       
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-32-4"  
class="td11">  
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-33-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-33-1"  
class="td11"> 
DAP12 interactions
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-33-2"  
class="td11">     
159
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-33-3"  
class="td11">       
23
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-33-4"  
class="td11">  
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-34-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-34-1"  
class="td11"> 
DAP12 signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-34-2"  
class="td11">     
149
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-34-3"  
class="td11">       
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-34-4"  
class="td11">  
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-35-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-35-1"  
class="td11"> 
Organelle biogenesis and maintenance
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-35-2"  
class="td11">     
264
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-35-3"  
class="td11">       
33
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-35-4"  
class="td11">  
5<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-36-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-36-1"  
class="td11"> 
Signalling by NGF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-36-2"  
class="td11">     
266
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-36-3"  
class="td11">       
33
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-36-4"  
class="td11">  
8<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-37-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-37-1"  
class="td11"> 
Downstream signalling of activated FGFR1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-37-2"  
class="td11">     
134
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-37-3"  
class="td11">       
20
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-37-4"  
class="td11">  
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-38-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-38-1"  
class="td11"> 
Downstream signalling of activated FGFR2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-38-2"  
class="td11">     
134
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-38-3"  
class="td11">       
20
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-38-4"  
class="td11">  
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-39-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-39-1"  
class="td11"> 
Downstream signalling of activated FGFR3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-39-2"  
class="td11">     
134
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-39-3"  
class="td11">       
20
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-39-4"  
class="td11">  
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-40-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-40-1"  
class="td11"> 
Downstream signalling of activated FGFR4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-40-2"  
class="td11">     
134
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-40-3"  
class="td11">       
20
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-40-4"  
class="td11">  
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-41-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-41-1"  
class="td11"> 
Signalling by FGFR
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-41-2"  
class="td11">     
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-41-3"  
class="td11">       
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-41-4"  
class="td11">  
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-42-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-42-1"  
class="td11"> 
Signalling by FGFR1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-42-2"  
class="td11">     
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-42-3"  
class="td11">       
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-42-4"  
class="td11">  
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-43-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-43-1"  
class="td11"> 
Signalling by FGFR2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-43-2"  
class="td11">     
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-43-3"  
class="td11">       
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-43-4"  
class="td11">  
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-63-44-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-44-1"  
class="td11">                                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-45-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-45-1"  
class="td11"> 
<span class="cmbx-12">Intersection of </span><a 
href="#glo:SLIPT"><span class="cmbx-12">SLIPT</span></a> <span class="cmbx-12">and </span><a 
href="#glo:siRNA"><span class="cmbx-12">siRNA</span></a> <span class="cmbx-12">screen (604 genes) </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-45-2"  
class="td11"> 
Pathway Size
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-45-3"  
class="td11"> 
Genes Identified
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-45-4"  
class="td11"> 
p-value (<a 
href="#glo:FDR">FDR</a>)
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-63-46-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-46-1"  
class="td11"> 
Visual phototransduction
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-46-2"  
class="td11">      
54
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-46-3"  
class="td11">       
9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-46-4"  
class="td11">  
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-47-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-47-1"  
class="td11"> 
G<sub><span class="cmmi-8">αs</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-47-2"  
class="td11">      
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-47-3"  
class="td11">       
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-47-4"  
class="td11">   
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-48-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-48-1"  
class="td11"> 
Retinoid metabolism and transport
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-48-2"  
class="td11">      
24
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-48-3"  
class="td11">       
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-48-4"  
class="td11">   
1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-49-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-49-1"  
class="td11"> 
Acyl chain remodelling of PS
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-49-2"  
class="td11">      
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-49-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-49-4"  
class="td11">   
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-50-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-50-1"  
class="td11"> 
Transcriptional regulation of white adipocyte differentiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-50-2"  
class="td11">      
51
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-50-3"  
class="td11">       
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-50-4"  
class="td11">   
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-51-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-51-1"  
class="td11"> 
Chemokine receptors bind chemokines
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-51-2"  
class="td11">      
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-51-3"  
class="td11">       
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-51-4"  
class="td11">   
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-52-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-52-1"  
class="td11"> 
Signalling by NOTCH4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-52-2"  
class="td11">      
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-52-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-52-4"  
class="td11">   
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-53-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-53-1"  
class="td11"> 
Defective EXT2 causes exostoses 2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-53-2"  
class="td11">      
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-53-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-53-4"  
class="td11">   
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-54-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-54-1"  
class="td11"> 
Defective EXT1 causes exostoses 1, TRPS2 and CHDS
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-54-2"  
class="td11">      
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-54-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-54-4"  
class="td11">   
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-55-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-55-1"  
class="td11"> 
Platelet activation, signalling and aggregation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-55-2"  
class="td11">     
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-55-3"  
class="td11">       
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-55-4"  
class="td11">   
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-56-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-56-1"  
class="td11"> 
Phase 1 - Functionalisation of compounds
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-56-2"  
class="td11">      
41
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-56-3"  
class="td11">       
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-56-4"  
class="td11">   
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-57-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-57-1"  
class="td11"> 
Amine ligand-binding receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-57-2"  
class="td11">      
13
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-57-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-57-4"  
class="td11">   
1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-58-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-58-1"  
class="td11"> 
Acyl chain remodelling of PE
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-58-2"  
class="td11">      
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-58-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-58-4"  
class="td11">   
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-59-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-59-1"  
class="td11"> 
Signalling by GPCR
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-59-2"  
class="td11">     
300
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-59-3"  
class="td11">       
23
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-59-4"  
class="td11">   
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-60-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-60-1"  
class="td11"> 
Molecules associated with elastic fibres
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-60-2"  
class="td11">      
29
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-60-3"  
class="td11">       
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-60-4"  
class="td11">   
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-61-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-61-1"  
class="td11"> 
DAP12 interactions
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-61-2"  
class="td11">     
128
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-61-3"  
class="td11">       
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-61-4"  
class="td11">   
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-62-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-62-1"  
class="td11"> 
Cytochrome P<sub><span class="cmr-8">450</span></sub> - arranged by substrate type
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-62-2"  
class="td11">      
30
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-62-3"  
class="td11">       
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-62-4"  
class="td11">   
3<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-63-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-63-1"  
class="td11"> 
GPCR ligand binding
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-63-2"  
class="td11">     
147
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-63-3"  
class="td11">       
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-63-4"  
class="td11">   
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-64-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-64-1"  
class="td11"> 
Acyl chain remodelling of PC
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-64-2"  
class="td11">      
16
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-64-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-64-4"  
class="td11">   
4<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-63-65-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-65-1"  
class="td11"> 
Response to elevated platelet cytosolic Ca<sup><span class="cmr-8">2+</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-65-2"  
class="td11">      
66
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-65-3"  
class="td11">       
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-63-65-4"  
class="td11">   
4<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-63-66-"><td  style="white-space:nowrap; text-align:left;" id="TBL-63-66-1"  
class="td11">                                                             </td></tr></table>                                                         </div>
                                                                                    

</div><hr class="endfloat" />
<!--l. 739--><p class="noindent" >
These <a 
href="#glo:pathway">pathways</a> over-representation analyses (performed as described in Section <a 
href="#x1-580002">2.3.2<!--tex4ht:ref: methods:enrichment --></a>) correspond to genes separated into <a 
href="#glo:SLIPT">SLIPT</a> or <a 
href="#glo:siRNA">siRNA</a> screen candidates unique to either method, or detected by both (Table <a 
href="#x1-1060016">4.6<!--tex4ht:ref: tab:Venn_over-representation --></a>). The <a 
href="#glo:SLIPT">SLIPT</a>-specific gene candidates were involved most strongly with translational and immune regulatory <a 
href="#glo:pathway">pathways</a>, although extracellular matrix <a 
href="#glo:pathway">pathways</a> were also supported. These <a 
href="#glo:pathway">pathways</a> were largely consistent with those identified in Table <a 
href="#x1-980012">4.2<!--tex4ht:ref: tab:pathway_exprSL --></a> and in the clustering analysis (Table <a 
href="#x1-1000013">4.3<!--tex4ht:ref: tab:pathway_clusters --></a>). The genes detected only by the <a 
href="#glo:siRNA">siRNA</a> screen had over-representation of cell signalling <a 
href="#glo:pathway">pathways</a>, including many containing genes known to be involved in cancer (e.g., MAPK, PDGF, ERBB2, and FGFR), with the detection of Class A GPCRs supporting the independent analyses by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>). The intersection of computational and experimental <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">CDH1 </span>had stronger evidence for over-representation of <a 
href="#glo:GPCR">GPCR</a> <a 
href="#glo:pathway">pathways</a> and more specific subclasses, such as visual phototransduction (<span class="cmmi-12">p </span>= 6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>) and G<sub> <span class="cmmi-8">αs</span></sub> signalling events (<span class="cmmi-12">p </span>= 1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>), than other signalling <a 
href="#glo:pathway">pathways</a>.
</p><!--l. 741--><p class="indent" >
The <a 
href="#glo:pathway">pathways</a> analysis for <a 
href="#glo:mtSLIPT">mtSLIPT</a> against <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutations</a> (in Table <a 
href="#x1-1740024">C.4<!--tex4ht:ref: tab:Venn_over-representation_mtSL --></a>) had similar results to <a 
href="#glo:SLIPT">SLIPT</a>, particularly for <a 
href="#glo:mtSLIPT">mtSLIPT</a>-specific <a 
href="#glo:pathway">pathways</a>. The specific <a 
href="#glo:pathway">pathways</a> composition of the intersection of these analyses differed from <a 
href="#glo:SLIPT">SLIPT</a> against low <span class="cmti-12">CDH1 </span><a 
href="#glo:gene expression">expression</a>. However, signalling <a 
href="#glo:pathway">pathways</a> were also detected, including <a 
href="#glo:GPCR">GPCRs</a>, NOTCH, EERB2, PDGF, and SCF-KIT. These findings indicate the signalling <a 
href="#glo:pathway">pathways</a> are among the most suitable vulnerability to exploit in targeting <span class="cmti-12">CDH1 </span>deficient tumours as they can be detected in both a patient cohort (with <a 
href="#glo:TCGA">TCGA</a> <a 
href="#glo:gene expression">expression</a> data) and experimentally tested by inhibition with <a 
href="#glo:siRNA">siRNA</a> or drugs (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). However, it is possible that the <a 
href="#glo:siRNA">siRNA</a> screen, that was conducted in an isolated experimental system of MCF10A cell lines, was pre-disposed to preferentially detect kinase signalling <a 
href="#glo:pathway">pathways</a> (which are amenable to pharmacological inhibition and clinical application). Nevertheless, the other <a 
href="#glo:pathway">pathways</a> identified by <a 
href="#glo:SLIPT">SLIPT</a> may still be informative of the role of <span class="cmti-12">CDH1 </span>loss of function in cancers or mechanisms by which further gene loss leads to specific inviability. <a 
 id="x1-106002r193"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">4.2.5.1 </span> <a 
 id="x1-1070001"></a>Resampling Genes for Pathway Enrichment
</h5>
<!--l. 751--><p class="noindent" >
<p>A high number of significantly over-represented <a 
href="#glo:pathway">pathways</a> were detected between <a 
href="#glo:SLIPT">SLIPT</a> in <a 
href="#glo:TCGA">TCGA</a> <a 
href="#glo:gene expression">expression</a> data and <a 
href="#glo:siRNA">siRNA</a> genes despite relatively few genes being detected by</p>
both approaches. These strongly supported <a 
href="#glo:pathway">pathways</a> are not unexpected, since <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> are more robustly conserved (<a 
href="#XDixon2008">Dixon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDixon2008">2008</a>) and the computational approach using patient samples from complex tumour micro-environment has considerably different strengths to an experimental screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) based on genetically homogenous cell line models in an isolated laboratory environment. For instance, it is unlikely for immune signalling to be detected in an isolated cell culture system.
</p><!--l. 753--><p class="indent" >
While many <a 
href="#glo:pathway">pathways</a> were highly over-represented in the genes detected by both <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a>, some of these <a 
href="#glo:pathway">pathways</a> were also highly over-represented in the <a 
href="#glo:siRNA">siRNA</a> candidate genes and these may not reflect the results of <a 
href="#glo:SLIPT">SLIPT</a> in expression data. A resampling approach (as described in Section <a 
href="#x1-630006">2.3.6<!--tex4ht:ref: methods:permutation --></a>) was used to assess whether <a 
href="#glo:SLIPT">SLIPT</a> and the <a 
href="#glo:siRNA">siRNA</a> candidate genes had more frequently over-represented <a 
href="#glo:pathway">pathways</a> than expected by chance. This resampling procedure tests whether <a 
href="#glo:pathway">pathways</a> were over-represented in the genes detected by both approaches beyond that expected from any subset of <a 
href="#glo:siRNA">siRNA</a> candidates. Thus resampling can determine whether <a 
href="#glo:SLIPT">SLIPT</a> independently supports these putative <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> (without assuming an underlying test statistic distribution).
</p><!--l. 755--><p class="indent" >
A resampling approach is also applicable to testing whether the number of genes detected by each approach significantly intersected. As shown in Figure <a 
href="#x1-1070017">4.7<!--tex4ht:ref: fig:perm_sample --></a>, resampling did not find evidence of significant depletion or over-representation for experimental <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate genes in the computationally predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">CDH1</span>, and thus the observed overlap may be due to chance. This is consistent with previous findings (as shown in Figure <a 
href="#x1-1020012">4.2<!--tex4ht:ref: fig:Venn_allgenes --></a>) and does not preclude <a 
href="#glo:pathway">pathways</a> relationships being supported by resampling.
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                    

<p><a 
 id="x1-1070017"></a></p>
<!--l. 760--><p class="noindent" >
<img 
src="sample_size_dist_exprSL_1M_Pub.png" alt="PIC"  
 /> <br />
</p><div class="caption" 
>
<span class="id">Figure 4.7: </span><span class="content"><strong><span class="cmbx-10x-x-109">Resampled intersection of </span><a 
href="#glo:SLIPT"><span class="cmbx-10x-x-109">SLIPT</span></a> <span class="cmbx-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">candidate genes.</span></strong> <span class="cmr-10x-x-109">Resampling analysis of intersect size from genes detected by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screening</span> <span class="cmr-10x-x-109">approaches over 1 million replicates. The proportion of expected intersection sizes for</span> <span class="cmr-10x-x-109">random samples below or above the observed intersection size respectively, lacking significant</span> <span class="cmr-10x-x-109">over-representation or depletion of </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screen candidates within the </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">predictions</span> <span class="cmr-10x-x-109">for </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">. </span></span>
</div><!--tex4ht:label?: x1-1070017 -->
                                                                                    

</div><hr class="endfigure" />
<!--l. 770--><p class="indent" >
<p>A permutation analysis was performed to resample the genes tested by both approaches to investigate whether the observed <a 
href="#glo:pathway">pathways</a> over-representation could have occurred in a randomly selected sample of genes from the experimental candidates, that is, whether the <a 
href="#glo:pathway">pathways</a> predictions from <a 
href="#glo:SLIPT">SLIPT</a> could be expected by chance (as described in Sections <a 
href="#x1-550004">2.2.4<!--tex4ht:ref: methods:venn_analysis --></a> and <a 
href="#x1-630006">2.3.6<!--tex4ht:ref: methods:permutation --></a>). While the number of <a 
href="#glo:siRNA">siRNA</a> candidate genes also detected by <a 
href="#glo:SLIPT">SLIPT</a> was not statistically significant (<span class="cmmi-12">p </span>= 0<span class="cmmi-12">.</span>281), this may be due to the vastly different limitations of the approaches and the correlation structure of <a 
href="#glo:gene expression">gene expression</a> not being independent (as assumed for multiple testing procedures). The intersection may still be functionally relevant to <span class="cmti-12">CDH1</span>-deficient cancers, such as the <a 
href="#glo:pathway">pathways</a> data in Table <a 
href="#x1-1060016">4.6<!--tex4ht:ref: tab:Venn_over-representation --></a>. The resampling analysis for <a 
href="#glo:pathway">pathways</a> was compared to the <a 
href="#glo:pathway">pathways</a> over-representation for <a 
href="#glo:SLIPT">SLIPT</a> predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in Table <a 
href="#x1-1070027">4.7<!--tex4ht:ref: tab:pathway_perm --></a>. Similarly, the <a 
href="#glo:pathway">pathways</a> resampling for intersection between <a 
href="#glo:SLIPT">SLIPT</a> predictions and experimental screen candidates was compared to <a 
href="#glo:pathway">pathways</a> over-representation in Table <a 
href="#x1-1070038">4.8<!--tex4ht:ref: tab:pathway_perm_overlap --></a> for intersection with <a 
href="#glo:siRNA">siRNA</a> data.</p>
</p><!--l. 774--><p class="indent" >
<a 
 id="x1-1070027"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 4.7: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT</span>
</div><!--tex4ht:label?: x1-1070027 -->
<div class="tabular">
<table id="TBL-64" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-64-1g"><col 
id="TBL-64-1" /><col 
id="TBL-64-2" /><col 
id="TBL-64-3" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-64-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-1-1"  
class="td11"> 
<span class="cmbx-12">Reactome Pathway </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-1-2"  
class="td11"> 
Over-representation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-1-3"  
class="td11">  
Permutation
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-64-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-2-1"  
class="td11"> 
<strong><span class="cmbx-12">Eukaryotic Translation Elongation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-2-2"  
class="td11">    
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">207</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-2-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-3-1"  
class="td11"> 
<strong><span class="cmbx-12">Peptide chain elongation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-3-2"  
class="td11">    
5<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">201</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-3-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-4-1"  
class="td11"> 
<strong><span class="cmbx-12">Viral </span><a 
href="#glo:mRNA"><span class="cmbx-12">mRNA</span></a> <span class="cmbx-12">Translation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-4-2"  
class="td11">    
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">196</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-4-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-5-1"  
class="td11"> 
<strong><span class="cmbx-12">Eukaryotic Translation Termination</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-5-2"  
class="td11">    
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">196</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-5-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-6-1"  
class="td11"> 
<strong><span class="cmbx-12">Formation of a pool of free 40S subunits</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-6-2"  
class="td11">    
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">194</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-6-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-7-1"  
class="td11"> 
<strong><span class="cmbx-12">Nonsense Mediated Decay independent of the Exon Junction Complex</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-7-2"  
class="td11">    
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">187</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-7-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-8-1"  
class="td11"> 
<strong><span class="cmbx-12">L13a-mediated translational silencing of Ceruloplasmin </span><a 
href="#glo:gene expression"><span class="cmbx-12">expression</span></a></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-8-2"  
class="td11">    
9<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">183</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-8-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-9-1"  
class="td11"> 
<strong><span class="cmbx-12">3’ -UTR-mediated translational regulation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-9-2"  
class="td11">    
9<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">183</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-9-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-10-1"  
class="td11"> 
<strong><span class="cmbx-12">GTP hydrolysis and joining of the 60S ribosomal subunit</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-10-2"  
class="td11">    
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">181</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-10-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-11-1"  
class="td11"> 
<strong><span class="cmbx-12">Nonsense-Mediated Decay</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-11-2"  
class="td11">    
6<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">176</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-11-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-12-1"  
class="td11"> 
<strong><span class="cmbx-12">Nonsense Mediated Decay enhanced by the Exon Junction Complex</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-12-2"  
class="td11">    
6<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">176</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-12-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-13-1"  
class="td11"> 
Adaptive Immune System
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-13-2"  
class="td11">    
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">174</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-13-3"  
class="td11">    
0<span class="cmmi-12">.</span>15753
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-14-1"  
class="td11"> 
<strong><span class="cmbx-12">Eukaryotic Translation Initiation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-14-2"  
class="td11">    
5<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">173</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-14-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-15-1"  
class="td11"> 
<strong><span class="cmbx-12">Cap-dependent Translation Initiation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-15-2"  
class="td11">    
5<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">173</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-15-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-16-1"  
class="td11"> 
<strong><span class="cmbx-12">SRP-dependent cotranslational protein targeting to membrane</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-16-2"  
class="td11">    
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">171</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-16-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-17-1"  
class="td11"> 
<strong><span class="cmbx-12">Translation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-17-2"  
class="td11">    
6<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">170</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-17-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-18-1"  
class="td11"> 
Infectious disease
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-18-2"  
class="td11">    
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">166</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-18-3"  
class="td11">    
0<span class="cmmi-12">.</span>23231
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-19-1"  
class="td11"> 
<strong><span class="cmbx-12">Influenza Infection</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-19-2"  
class="td11">    
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">163</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-19-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-20-1"  
class="td11"> 
<strong><span class="cmbx-12">Influenza Viral </span><a 
href="#glo:RNA"><span class="cmbx-12">RNA</span></a> <span class="cmbx-12">Transcription and Replication</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-20-2"  
class="td11">    
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">160</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-20-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-21-1"  
class="td11"> 
<strong><span class="cmbx-12">Influenza Life Cycle</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-21-2"  
class="td11">    
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">156</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-21-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-22-1"  
class="td11"> 
<span class="cmti-12">Extracellular matrix organisation </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-22-2"  
class="td11">    
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">152</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-22-3"  
class="td11">    
0<span class="cmmi-12">.</span>071761
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-23-1"  
class="td11"> 
GPCR ligand binding
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-23-2"  
class="td11">    
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">143</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-23-3"  
class="td11">    
0<span class="cmmi-12">.</span>55801
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-24-1"  
class="td11"> 
Class A/1 (Rhodopsin-like receptors)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-24-2"  
class="td11">    
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">142</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-24-3"  
class="td11">    
0<span class="cmmi-12">.</span>58901
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-25-1"  
class="td11"> 
<span class="cmti-12">GPCR downstream signalling </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-25-2"  
class="td11">    
7<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">140</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-25-3"  
class="td11">    
0<span class="cmmi-12">.</span>098357
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-26-1"  
class="td11"> 
Haemostasis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-26-2"  
class="td11">    
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">134</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-26-3"  
class="td11">    
0<span class="cmmi-12">.</span>27059
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-27-1"  
class="td11"> 
Developmental Biology
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-27-2"  
class="td11">    
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">123</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-27-3"  
class="td11">    
0<span class="cmmi-12">.</span>52737
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-28-1"  
class="td11"> 
Metabolism of lipids and lipoproteins
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-28-2"  
class="td11">    
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">120</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-28-3"  
class="td11">     
0<span class="cmmi-12">.</span>724
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-29-1"  
class="td11"> 
Cytokine Signalling in Immune system
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-29-2"  
class="td11">    
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">119</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-29-3"  
class="td11">    
0<span class="cmmi-12">.</span>39661
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-30-1"  
class="td11"> 
Peptide ligand-binding receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-30-2"  
class="td11">    
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">109</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-30-3"  
class="td11">    
0<span class="cmmi-12">.</span>61102
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-64-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-31-1"  
class="td11"> 
<strong><span class="cmbx-12">G</span><sub><span class="cmmi-8">αi</span> </sub> <span class="cmbx-12">signalling events</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-31-2"  
class="td11">    
8<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">100</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-64-31-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-64-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-64-32-1"  
class="td11">                                                                      </td></tr></table>
<!--l. 893--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Over-representation (hypergeometric test) and Permutation p-values adjusted for multiple tests across </span><a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathways</span></a> <span class="cmr-10x-x-109">(</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a><span class="cmr-10x-x-109">). Significant</span> <a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathways</span></a> <span class="cmr-10x-x-109">are marked in bold (</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a> <span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05) and italics (</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a> <span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">1).</span></p>
</p>
</div><hr class="endfloat" />
                                                                                    

<!--l. 896--><p class="indent" >
<a 
 id="x1-1070038"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 4.8: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT and <a 
href="#glo:siRNA">siRNA</a> primary screen</span>
</div><!--tex4ht:label?: x1-1070038 -->
<div class="tabular">
<table id="TBL-65" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-65-1g"><col 
id="TBL-65-1" /><col 
id="TBL-65-2" /><col 
id="TBL-65-3" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-65-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-1-1"  
class="td11"> 
<span class="cmbx-12">Reactome Pathway </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-1-2"  
class="td11"> 
Over-representation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-1-3"  
class="td11"> 
Permutation
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-65-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-2-1"  
class="td11"> 
Visual phototransduction
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-2-2"  
class="td11">     
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-2-3"  
class="td11">   
0<span class="cmmi-12">.</span>91116
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-3-1"  
class="td11"> 
<strong><span class="cmbx-12">G</span><sub><span class="cmmi-8">αs</span> </sub> <span class="cmbx-12">signalling events</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-3-2"  
class="td11">     
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-3-3"  
class="td11">   
0<span class="cmmi-12">.</span>012988
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-4-1"  
class="td11"> 
Retinoid metabolism and transport
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-4-2"  
class="td11">     
1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-4-3"  
class="td11">   
0<span class="cmmi-12">.</span>20487
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-5-1"  
class="td11"> 
Transcriptional regulation of white adipocyte differentiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-5-2"  
class="td11">     
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-5-3"  
class="td11">   
0<span class="cmmi-12">.</span>38197
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-6-1"  
class="td11"> 
Acyl chain remodelling of PS
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-6-2"  
class="td11">     
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-6-3"  
class="td11">   
0<span class="cmmi-12">.</span>58485
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-7-1"  
class="td11"> 
Chemokine receptors bind chemokines
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-7-2"  
class="td11">     
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-7-3"  
class="td11">   
0<span class="cmmi-12">.</span>97255
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-8-1"  
class="td11"> 
<span class="cmti-12">Defective EXT2 causes exostoses 2 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-8-2"  
class="td11">     
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-8-3"  
class="td11">   
0<span class="cmmi-12">.</span>056437
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-9-1"  
class="td11"> 
<span class="cmti-12">Defective EXT1 causes exostoses 1, TRPS2 and CHDS </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-9-2"  
class="td11">     
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-9-3"  
class="td11">   
0<span class="cmmi-12">.</span>056437
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-10-1"  
class="td11"> 
Signalling by NOTCH4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-10-2"  
class="td11">     
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-10-3"  
class="td11">   
0<span class="cmmi-12">.</span>15497
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-11-1"  
class="td11"> 
Platelet activation, signalling and aggregation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-11-2"  
class="td11">     
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-11-3"  
class="td11">   
0<span class="cmmi-12">.</span>53358
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-12-1"  
class="td11"> 
Phase 1 - Functionalisation of compounds
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-12-2"  
class="td11">     
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-12-3"  
class="td11">   
0<span class="cmmi-12">.</span>24836
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-13-1"  
class="td11"> 
Amine ligand-binding receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-13-2"  
class="td11">     
1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-13-3"  
class="td11">    
0<span class="cmmi-12">.</span>3195
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-14-1"  
class="td11"> 
Acyl chain remodelling of PE
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-14-2"  
class="td11">     
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-14-3"  
class="td11">    
0<span class="cmmi-12">.</span>7307
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-15-1"  
class="td11"> 
Signalling by GPCR
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-15-2"  
class="td11">     
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-15-3"  
class="td11">    
0<span class="cmmi-12">.</span>9939
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-16-1"  
class="td11"> 
<strong><span class="cmbx-12">Molecules associated with elastic fibres</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-16-2"  
class="td11">     
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-16-3"  
class="td11">  
0<span class="cmmi-12">.</span>0072929
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-17-1"  
class="td11"> 
DAP12 interactions
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-17-2"  
class="td11">     
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-17-3"  
class="td11">   
0<span class="cmmi-12">.</span>78273
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-18-1"  
class="td11"> 
Cytochrome P<sub><span class="cmr-8">450</span></sub> - arranged by substrate type
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-18-2"  
class="td11">     
3<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-18-3"  
class="td11">   
0<span class="cmmi-12">.</span>87019
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-19-1"  
class="td11"> 
GPCR ligand binding
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-19-2"  
class="td11">     
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-19-3"  
class="td11">   
0<span class="cmmi-12">.</span>99417
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-20-1"  
class="td11"> 
Acyl chain remodelling of PC
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-20-2"  
class="td11">     
4<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-20-3"  
class="td11">   
0<span class="cmmi-12">.</span>65415
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-21-1"  
class="td11"> 
Response to elevated platelet cytosolic Ca<sup><span class="cmr-8">2+</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-21-2"  
class="td11">     
4<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-21-3"  
class="td11">   
0<span class="cmmi-12">.</span>55461
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-22-1"  
class="td11"> 
<span class="cmti-12">Arachidonic acid metabolism </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-22-2"  
class="td11">     
4<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-22-3"  
class="td11">   
0<span class="cmmi-12">.</span>060298
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-23-1"  
class="td11"> 
Defective B4GALT7 causes EDS, progeroid type
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-23-2"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-23-3"  
class="td11">   
0<span class="cmmi-12">.</span>15497
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-24-1"  
class="td11"> 
Defective B3GAT3 causes JDSSDHD
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-24-2"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-24-3"  
class="td11">   
0<span class="cmmi-12">.</span>15497
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-25-1"  
class="td11"> 
<strong><span class="cmbx-12">Elastic fibre formation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-25-2"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-25-3"  
class="td11">  
0<span class="cmmi-12">.</span>0019227
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-26-1"  
class="td11"> 
<strong><span class="cmbx-12">HS-GAG degradation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-26-2"  
class="td11">     
6<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-26-3"  
class="td11">   
0<span class="cmmi-12">.</span>017747
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-27-1"  
class="td11"> 
Bile acid and bile salt metabolism
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-27-2"  
class="td11">     
6<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-27-3"  
class="td11">   
0<span class="cmmi-12">.</span>15497
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-28-1"  
class="td11"> 
Netrin-1 signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-28-2"  
class="td11">     
7<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-28-3"  
class="td11">   
0<span class="cmmi-12">.</span>95056
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-29-1"  
class="td11"> 
<strong><span class="cmbx-12">Integration of energy metabolism</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-29-2"  
class="td11">     
7<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-29-3"  
class="td11">  
0<span class="cmmi-12">.</span>0019287
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-30-1"  
class="td11"> 
DAP12 signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-30-2"  
class="td11">     
7<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-30-3"  
class="td11">   
0<span class="cmmi-12">.</span>67835
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-31-1"  
class="td11"> 
GPCR downstream signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-31-2"  
class="td11">     
8<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-31-3"  
class="td11">   
0<span class="cmmi-12">.</span>88678
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-32-1"  
class="td11"> 
<strong><span class="cmbx-12">Diseases associated with glycosaminoglycan metabolism</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-32-2"  
class="td11">     
8<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-32-3"  
class="td11">   
0<span class="cmmi-12">.</span>017747
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-33-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-33-1"  
class="td11"> 
<strong><span class="cmbx-12">Diseases of glycosylation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-33-2"  
class="td11">     
8<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-33-3"  
class="td11">   
0<span class="cmmi-12">.</span>017747
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-34-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-34-1"  
class="td11"> 
Signalling by Retinoic Acid
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-34-2"  
class="td11">     
8<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-34-3"  
class="td11">   
0<span class="cmmi-12">.</span>13592
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-35-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-35-1"  
class="td11"> 
Signalling by Leptin
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-35-2"  
class="td11">     
8<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-35-3"  
class="td11">   
0<span class="cmmi-12">.</span>15497
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-36-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-36-1"  
class="td11"> 
Signalling by SCF-KIT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-36-2"  
class="td11">     
8<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-36-3"  
class="td11">   
0<span class="cmmi-12">.</span>73399
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-37-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-37-1"  
class="td11"> 
Opioid Signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-37-2"  
class="td11">     
8<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-37-3"  
class="td11">   
0<span class="cmmi-12">.</span>99417
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-38-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-38-1"  
class="td11"> 
Signalling by NOTCH
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-38-2"  
class="td11">       
0<span class="cmmi-12">.</span>0001
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-38-3"  
class="td11">   
0<span class="cmmi-12">.</span>26453
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-39-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-39-1"  
class="td11"> 
Platelet homeostasis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-39-2"  
class="td11">       
0<span class="cmmi-12">.</span>0001
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-39-3"  
class="td11">   
0<span class="cmmi-12">.</span>55912
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-40-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-40-1"  
class="td11"> 
Signalling by NOTCH1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-40-2"  
class="td11">      
0<span class="cmmi-12">.</span>00011
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-40-3"  
class="td11">   
0<span class="cmmi-12">.</span>13797
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-41-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-41-1"  
class="td11"> 
Class B/2 (Secretin family receptors)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-41-2"  
class="td11">      
0<span class="cmmi-12">.</span>00011
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-41-3"  
class="td11">    
0<span class="cmmi-12">.</span>4659
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-42-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-42-1"  
class="td11"> 
Diseases of Immune System
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-42-2"  
class="td11">      
0<span class="cmmi-12">.</span>00013
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-42-3"  
class="td11">   
0<span class="cmmi-12">.</span>15497
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-43-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-43-1"  
class="td11"> 
Diseases associated with the TLR signalling cascade
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-43-2"  
class="td11">      
0<span class="cmmi-12">.</span>00013
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-43-3"  
class="td11">   
0<span class="cmmi-12">.</span>15497
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-44-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-44-1"  
class="td11"> 
A tetrasaccharide linker sequence is required for GAG synthesis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-44-2"  
class="td11">      
0<span class="cmmi-12">.</span>00013
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-44-3"  
class="td11">   
0<span class="cmmi-12">.</span>33566
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-45-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-45-1"  
class="td11"> 
Nuclear Receptor transcription <a 
href="#glo:pathway">pathways</a>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-45-2"  
class="td11">      
0<span class="cmmi-12">.</span>00016
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-45-3"  
class="td11">   
0<span class="cmmi-12">.</span>22735
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-46-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-46-1"  
class="td11"> 
<strong><span class="cmbx-12">Formation of Fibrin Clot (Clotting Cascade)</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-46-2"  
class="td11">      
0<span class="cmmi-12">.</span>00016
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-46-3"  
class="td11">  
0<span class="cmmi-12">.</span>0054639
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-47-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-47-1"  
class="td11"> 
Syndecan interactions
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-47-2"  
class="td11">      
0<span class="cmmi-12">.</span>00016
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-47-3"  
class="td11">    
0<span class="cmmi-12">.</span>3974
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-48-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-48-1"  
class="td11"> 
Class A/1 (Rhodopsin-like receptors)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-48-2"  
class="td11">      
0<span class="cmmi-12">.</span>00016
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-48-3"  
class="td11">   
0<span class="cmmi-12">.</span>99454
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-49-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-49-1"  
class="td11"> 
HS-GAG biosynthesis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-49-2"  
class="td11">       
0<span class="cmmi-12">.</span>0002
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-49-3"  
class="td11">   
0<span class="cmmi-12">.</span>37199
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-50-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-50-1"  
class="td11"> 
Platelet degranulation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-50-2"  
class="td11">       
0<span class="cmmi-12">.</span>0002
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-50-3"  
class="td11">   
0<span class="cmmi-12">.</span>39003
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-65-51-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-51-1"  
class="td11"> 
EPH-ephrin mediated repulsion of cells
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-51-2"  
class="td11">      
0<span class="cmmi-12">.</span>00021
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-65-51-3"  
class="td11">    
0<span class="cmmi-12">.</span>6193
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-65-52-"><td  style="white-space:nowrap; text-align:left;" id="TBL-65-52-1"  
class="td11">                                                       </td></tr></table>
<!--l. 1013--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Over-representation (hypergeometric test) and Permutation p-values adjusted for multiple tests across </span><a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathways</span></a> <span class="cmr-10x-x-109">(</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a><span class="cmr-10x-x-109">). Significant </span><a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathways</span></a> <span class="cmr-10x-x-109">are marked in bold (</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a> <span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05) and italics (</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a> <span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">1).</span></p>
</p>
</div><hr class="endfloat" />
<!--l. 1016--><p class="indent" >
The <a 
href="#glo:pathway">pathways</a> resampling approach for <a 
href="#glo:SLIPT">SLIPT</a>-specific gene candidates (Table <a 
href="#x1-1070027">4.7<!--tex4ht:ref: tab:pathway_perm --></a>) largely recapitulates the parametric gene set over-representation analysis for all <a 
href="#glo:SLIPT">SLIPT</a> genes, detecting evidence of <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> for <span class="cmti-12">CDH1 </span>in translational, immune, and cell signalling <a 
href="#glo:pathway">pathways</a> including G<sub><span class="cmmi-8">αi</span></sub> signalling, <a 
href="#glo:GPCR">GPCR</a> downstream signalling, and chemokine receptor binding. While the immune and signal transduction <a 
href="#glo:pathway">pathways</a> were not significantly over-represented in the resampling analysis, the results for the two approaches were largely consistent for translation and post-transcriptional gene regulation, supporting gene set over-representation of the <a 
href="#glo:SLIPT">SLIPT</a>-specific <a 
href="#glo:pathway">pathways</a> in Table <a 
href="#x1-1070027">4.7<!--tex4ht:ref: tab:pathway_perm --></a>. In particular, some of the most significantly over-represented <a 
href="#glo:pathway">pathways</a> had higher observed <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values than any of the 1 million random permutations. Similar <a 
href="#glo:pathway">pathways</a> were also replicated by permutation analysis for mt<a 
href="#glo:SLIPT">SLIPT</a> candidate partners against <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutation</a> (shown in Appendix Table <a 
href="#x1-1750015">C.5<!--tex4ht:ref: tab:pathway_perm_mtSL --></a>). For the genes detected by <a 
href="#glo:SLIPT">SLIPT</a>, the permutation approach detected many of the most strongly over-represented <a 
href="#glo:pathway">pathways</a>.
</p><!--l. 1018--><p class="indent" >
The permutation approach was also applied to the intersection between computational and experimental candidates. This permutation analysis tested whether for consistent detection of <a 
href="#glo:pathway">pathways</a> was independent of their pre-existing status as experimental candidates. In contrast to the <a 
href="#glo:SLIPT">SLIPT</a> pathways (in Table <a 
href="#x1-1070027">4.7<!--tex4ht:ref: tab:pathway_perm --></a>), the <a 
href="#glo:pathway">pathways</a> results for these candidate partners (in Table <a 
href="#x1-1070038">4.8<!--tex4ht:ref: tab:pathway_perm_overlap --></a>) differed considerably between over-representation and resampling analyses.
</p><!--l. 1020--><p class="indent" >
Namely, many of the over-represented <a 
href="#glo:pathway">pathways</a> were not significant in the resampling analysis, including visual phototransduction and retinoic acid signalling, and were likely over-represented in the intersection due to over-representation in the <a 
href="#glo:siRNA">siRNA</a> candidates rather than additional support from <a 
href="#glo:SLIPT">SLIPT</a>. Of the highest over-represented <a 
href="#glo:pathway">pathways</a> in the intersection, only G<sub><span class="cmmi-8">αs</span></sub> signalling events were supported by both over-representation and resampling analyses. Other <a 
href="#glo:pathway">pathways</a> supported by both analyses were cytoplasmic elastic fibre formation, associated HS-GAG protein modification <a 
href="#glo:pathway">pathways</a>, energy metabolism, and the fibrin clotting cascade.
</p><!--l. 1024--><p class="indent" >
<p>Many of the <a 
href="#glo:pathway">pathways</a> supported in the intersection by permutation analysis were also replicated in the <a 
href="#glo:mtSLIPT">mtSLIPT</a> analysis of partners tested with <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutation</a> (in Table <a 
href="#x1-1750026">C.6<!--tex4ht:ref: tab:pathway_perm_overlap_mtSL --></a>), including G<sub><span class="cmmi-8">αs</span></sub>, elastic fibres, HS-GAG, and energy metabolism. While there were differences between the <a 
href="#glo:pathway">pathways</a> identified by over-representation analysis, those replicated by permutation were highly concordant, supporting the combined</p>
use of these <a 
href="#glo:pathway">pathways</a> approaches to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene functions and targets.
</p><!--l. 1026--><p class="indent" >
While this indicates that G<sub><span class="cmmi-8">αs</span></sub> and <a 
href="#glo:GPCR">GPCR</a> class A/1 signalling events were significantly detected by both approaches, <a 
href="#glo:GPCR">GPCR</a> signalling <a 
href="#glo:pathway">pathways</a> overall were not. It is likely that <a 
href="#glo:GPCR">GPCRs</a> were primarily over-represented in the intersection with the experimental candidates due to strong over-representation of these <a 
href="#glo:pathway">pathways</a> in experimental candidates, rather than detection by <a 
href="#glo:SLIPT">SLIPT</a>, which may be driven by these more specific constituent <a 
href="#glo:pathway">pathways</a>.
</p><!--l. 1029--><p class="indent" >
Several <a 
href="#glo:pathway">pathways</a>, including some immune functions and neurotransmitters, were supported by the resampling analysis (in Table <a 
href="#x1-1070038">4.8<!--tex4ht:ref: tab:pathway_perm_overlap --></a> and Appendix Table <a 
href="#x1-1750026">C.6<!--tex4ht:ref: tab:pathway_perm_overlap_mtSL --></a>) when the initial <a 
href="#glo:pathway">pathways</a> over-representation test was not significant. These functions appear to have been detected by both approaches more than expected by chance but must be interpreted with caution since they were still not common enough to be detected in <a 
href="#glo:pathway">pathways</a> over-representation analysis. <a 
 id="x1-107004r207"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.2.6 </span> <a 
 id="x1-1080006"></a>Integrating Synthetic Lethal Pathways and Screens
</h4>
<!--l. 1033--><p class="noindent" >
Based on these results, it appears that computational and experimental approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> screening for <span class="cmti-12">CDH1 </span>lead to a broader functional characterisation, and many candidate partners, when combined, despite different strengths and limitations. Compared to candidate gene approaches, experimental <a 
href="#glo:genome">genomes</a>-wide screens are an appealing unbiased strategy for identifying <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions. Since these screens are costly, laborious, and specific to genetic background, computational analysis can augment candidate triage to either reduce the initial panel of screened genes or prioritise validation.
</p><!--l. 1035--><p class="indent" >
<p><a 
href="#glo:GPCR">GPCR</a> <a 
href="#glo:pathway">pathways</a> were detected among both computational and experimental <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates, with more support in the experimental screen (Table <a 
href="#x1-1070038">4.8<!--tex4ht:ref: tab:pathway_perm_overlap --></a>). The homogeneous cell line model may be more likely to detect particular <a 
href="#glo:pathway">pathways</a>. For instance, <a 
href="#glo:SLIPT">SLIPT</a> identified immune <a 
href="#glo:pathway">pathways</a>, not expected to be detected in isolated cell culture. <a 
href="#glo:GPCR">GPCR</a> signalling was supported in experimental models <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) with some of these <a 
href="#glo:pathway">pathways</a> replicated in varied genetic backgrounds of patient samples. These <a 
href="#glo:pathway">pathways</a> require further investigation such as identification of more specific <a 
href="#glo:pathway">pathways</a>, higher order interactions, and modes of resistance.</p>
</p><!--l. 1037--><p class="indent" >
The <a 
href="#glo:pathway">pathways</a> composition across computational and experimental <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates was informative with over-representation (Table <a 
href="#x1-1060016">4.6<!--tex4ht:ref: tab:Venn_over-representation --></a>) and was supported by resampling analysis (Table <a 
href="#x1-1070038">4.8<!--tex4ht:ref: tab:pathway_perm_overlap --></a>), despite a modest intersection of genes between them (Figure <a 
href="#x1-1020012">4.2<!--tex4ht:ref: fig:Venn_allgenes --></a>). Either approach may be significant for a <a 
href="#glo:pathway">pathways</a> in this intersection without being supported by the other: resampling analysis may support <a 
href="#glo:pathway">pathways</a> that were not over-represented due to small effect sizes, thus both tests are required to identify candidate <a 
href="#glo:pathway">pathways</a>. The <a 
href="#glo:pathway">pathways</a> detected by both over-representation and resampling are the strongest candidates for further investigation, such as G<sub><span class="cmmi-8">αs</span></sub> signalling, a strong candidate in prior analyses with a role in the regulation of translation in cancer <a 
href="#XGao2015">Gao and Roux</a> (<a 
href="#XGao2015">2015</a>), another function supported by <a 
href="#glo:SLIPT">SLIPT</a> analysis.
</p><!--l. 1042--><p class="indent" >
The predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners occurred across functionally distinct <a 
href="#glo:pathway">pathways</a>, including characterised functions of <span class="cmti-12">CDH1</span>. This diversity is consistent with the wide ranging role of <span class="cmti-12">CDH1 </span>in cell-cell adhesion, cell signalling, and the cytoskeletal structure of epithelial tissues. Pathway structure may be relevant to identifying potential drug targets from <a 
href="#glo:gene expression">gene expression</a> signatures, indicating downstream effector genes and mechanisms leading to cell inviability. These distinct <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene clusters and <a 
href="#glo:pathway">pathways</a> may further lead to the elucidation of drug resistance mechanisms. <a 
 id="x1-108001r195"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">4.3 </span> <a 
 id="x1-1090003"></a>Synthetic Lethal Pathway Metagenes
</h3>
<!--l. 1073--><p class="noindent" >
<a 
href="#glo:metagene">Metagenes</a> are a one-dimensional summary of the activity for each biological <a 
href="#glo:pathway">pathway</a>. The direction of <a 
href="#glo:metagene">metagenes</a> (derived by the singular value matrix decomposition as described in Section <a 
href="#x1-540003">2.2.3<!--tex4ht:ref: methods:metagene --></a>) reflects overall activation of the <a 
href="#glo:pathway">pathways</a>. This has been verified by examining the expression patterns of previously published gene signatures (<a 
href="#XGatza2011">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2011">2011</a>, <a 
href="#XGatza2014">2014</a>) in Appendix <a 
href="#x1-177000D">D<!--tex4ht:ref: chapt3:metagene_results --></a>. Pathway <a 
href="#glo:metagene">metagenes</a> for Reactome <a 
href="#glo:pathway">pathways</a> were used for testing <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a>. Since the <a 
href="#glo:metagene">metagenes</a> values are higher when the <a 
href="#glo:pathway">pathways</a> as a whole is activated, they are suitable for <a 
href="#glo:SLIPT">SLIPT</a> analysis using low <a 
href="#glo:metagene">metagene</a> levels to represent less activated <a 
href="#glo:pathway">pathways</a>.
</p><!--l. 1078--><p class="indent" >
<p>The <a 
href="#glo:TCGA">TCGA</a> breast cancer <a 
href="#glo:gene expression">expression</a> data was used to generate <a 
href="#glo:pathway">pathway</a> <a 
href="#glo:metagene">metagenes</a> for each collection of genes in a <a 
href="#glo:pathway">pathway</a> from the Reactome database (<a 
href="#XReactome">Croft <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XReactome">2014</a>). These metagenes were tested against the expression of <span class="cmti-12">CDH1 </span>by <a 
href="#glo:SLIPT">SLIPT</a> to directly detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a>. These <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:metagene">metagenes</a> differed to the</p>
<p>over-represented <a 
href="#glo:pathway">pathways</a> among <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene candidates. However, there were some similarities to previous findings, as shown in Table <a 
href="#x1-1090019">4.9<!--tex4ht:ref: tab:metagene_SL --></a>. In particular, translational <a 
href="#glo:pathway">pathways</a> were replicated as observed in Table <a 
href="#x1-980012">4.2<!--tex4ht:ref: tab:pathway_exprSL --></a>. While the specific <a 
href="#glo:pathway">pathways</a> differ, immune <a 
href="#glo:pathway">pathways</a> (e.g., NF-<span class="cmmi-12">κ</span>B) were also supported by <a 
href="#glo:metagene">metagene</a> <a 
href="#glo:synthetic lethal">synthetic lethal</a> analysis.</p>
</p><!--l. 1081--><p class="indent" >
<a 
 id="x1-1090019"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 4.9: </span><span class="content">Examples of candidate <a 
href="#glo:metagene">metagenes</a> <a 
href="#glo:synthetic lethal">synthetic lethal</a> for <span class="cmti-12">CDH1 </span>from SLIPT</span>
</div><!--tex4ht:label?: x1-1090019 -->
<div class="tabular">
<table id="TBL-66" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-66-1g"><col 
id="TBL-66-1" /><col 
id="TBL-66-2" /><col 
id="TBL-66-3" /><col 
id="TBL-66-4" /><col 
id="TBL-66-5" /><col 
id="TBL-66-6" /><col 
id="TBL-66-7" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-66-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-1-1"  
class="td11"> 
<span class="cmbx-12">Pathway </span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-1-2"  
class="td11">   
ID
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-1-3"  
class="td11"> 
Observed
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-1-4"  
class="td11"> 
Expected
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-1-5"  
class="td11"> 
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-1-6"  
class="td11">   
p-value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-1-7"  
class="td11"> 
p-value (<a 
href="#glo:FDR">FDR</a>)
</td>

</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-66-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-2-1"  
class="td11"> 
Glycogen storage diseases
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-2-2"  
class="td11"> 
3229121
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-2-3"  
class="td11"> 
68
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-2-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-2-5"  
class="td11">   
176
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-2-6"  
class="td11"> 
6<span class="cmmi-12">.</span>62 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">37</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-2-7"  
class="td11">  
1<span class="cmmi-12">.</span>82 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">34</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-3-1"  
class="td11"> 
Myoclonic epilepsy of Lafora
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-3-2"  
class="td11"> 
3785653
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-3-3"  
class="td11"> 
68
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-3-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-3-5"  
class="td11">   
176
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-3-6"  
class="td11"> 
6<span class="cmmi-12">.</span>62 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">37</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-3-7"  
class="td11">  
1<span class="cmmi-12">.</span>82 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">34</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-4-1"  
class="td11"> 
Diseases of carbohydrate metabolism
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-4-2"  
class="td11"> 
5663084
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-4-3"  
class="td11"> 
68
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-4-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-4-5"  
class="td11">   
176
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-4-6"  
class="td11"> 
6<span class="cmmi-12">.</span>62 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">37</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-4-7"  
class="td11">  
1<span class="cmmi-12">.</span>82 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">34</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-5-1"  
class="td11"> 
Arachidonic acid metabolism
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-5-2"  
class="td11"> 
2142753
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-5-3"  
class="td11"> 
81
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-5-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-5-5"  
class="td11">   
157
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-5-6"  
class="td11"> 
8<span class="cmmi-12">.</span>13 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">33</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-5-7"  
class="td11">  
1<span class="cmmi-12">.</span>49 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-6-1"  
class="td11"> 
Translation initiation complex formation
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-6-2"  
class="td11">  
72649
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-6-3"  
class="td11"> 
70
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-6-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-6-5"  
class="td11">   
152
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-6-6"  
class="td11"> 
7<span class="cmmi-12">.</span>08 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">32</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-6-7"  
class="td11">  
1<span class="cmmi-12">.</span>17 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-7-1"  
class="td11"> 
Synthesis of 5-eicosatetraenoic acids
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-7-2"  
class="td11"> 
2142688
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-7-3"  
class="td11"> 
68
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-7-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-7-5"  
class="td11">   
151
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-7-6"  
class="td11"> 
1<span class="cmmi-12">.</span>25 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-7-7"  
class="td11">  
1<span class="cmmi-12">.</span>88 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-8-1"  
class="td11"> 
SRP-dependent cotranslational protein targeting to membrane
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-8-2"  
class="td11"> 
1799339
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-8-3"  
class="td11"> 
69
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-8-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-8-5"  
class="td11">   
150
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-8-6"  
class="td11"> 
2<span class="cmmi-12">.</span>01 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-8-7"  
class="td11">  
2<span class="cmmi-12">.</span>76 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-9-1"  
class="td11"> 
L13a-mediated translational silencing of Ceruloplasmin <a 
href="#glo:gene expression">expression</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-9-2"  
class="td11"> 
156827
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-9-3"  
class="td11"> 
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-9-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-9-5"  
class="td11">   
148
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-9-6"  
class="td11"> 
5<span class="cmmi-12">.</span>91 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-9-7"  
class="td11">  
6<span class="cmmi-12">.</span>44 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-10-1"  
class="td11"> 
3’ -UTR-mediated translational regulation
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-10-2"  
class="td11"> 
157279
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-10-3"  
class="td11"> 
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-10-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-10-5"  
class="td11">   
148
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-10-6"  
class="td11"> 
5<span class="cmmi-12">.</span>91 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-10-7"  
class="td11">  
6<span class="cmmi-12">.</span>44 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-11-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-67" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-67-1g"><col 
id="TBL-67-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-67-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-67-1-1"  
class="td00">
Activation of the <a 
href="#glo:mRNA">mRNA</a> upon binding of the cap-binding complex and eIFs,
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-67-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-67-2-1"  
class="td00">
and subsequent binding to 43S
</td></tr></table>        </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-11-2"  
class="td11">  
72662
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-11-3"  
class="td11"> 
70
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-11-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-11-5"  
class="td11">   
147
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-11-6"  
class="td11"> 
1<span class="cmmi-12">.</span>14 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-11-7"  
class="td11">  
9<span class="cmmi-12">.</span>28 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-12-1"  
class="td11"> 
Formation of the ternary complex, and subsequently, the 43S complex
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-12-2"  
class="td11">  
72695
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-12-3"  
class="td11"> 
70
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-12-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-12-5"  
class="td11">   
147
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-12-6"  
class="td11"> 
1<span class="cmmi-12">.</span>14 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-12-7"  
class="td11">  
9<span class="cmmi-12">.</span>28 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-13-1"  
class="td11"> 
Ribosomal scanning and start codon recognition
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-13-2"  
class="td11">  
72702
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-13-3"  
class="td11"> 
70
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-13-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-13-5"  
class="td11">   
147
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-13-6"  
class="td11"> 
1<span class="cmmi-12">.</span>14 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-13-7"  
class="td11">  
9<span class="cmmi-12">.</span>28 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-14-1"  
class="td11"> 
Eukaryotic Translation Elongation
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-14-2"  
class="td11"> 
156842
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-14-3"  
class="td11"> 
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-14-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-14-5"  
class="td11">   
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-14-6"  
class="td11"> 
1<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-14-7"  
class="td11">  
9<span class="cmmi-12">.</span>28 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-15-1"  
class="td11"> 
Nonsense Mediated Decay independent of the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-15-2"  
class="td11"> 
975956
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-15-3"  
class="td11"> 
71
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-15-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-15-5"  
class="td11">   
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-15-6"  
class="td11"> 
1<span class="cmmi-12">.</span>24 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-15-7"  
class="td11">  
9<span class="cmmi-12">.</span>28 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-16-1"  
class="td11"> 
Viral <a 
href="#glo:mRNA">mRNA</a> Translation
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-16-2"  
class="td11"> 
192823
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-16-3"  
class="td11"> 
70
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-16-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-16-5"  
class="td11">   
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-16-6"  
class="td11"> 
1<span class="cmmi-12">.</span>51 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-16-7"  
class="td11">  
1<span class="cmmi-12">.</span>04 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-17-1"  
class="td11"> 
Eukaryotic Translation Termination
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-17-2"  
class="td11">  
72764
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-17-3"  
class="td11"> 
70
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-17-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-17-5"  
class="td11">   
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-17-6"  
class="td11"> 
1<span class="cmmi-12">.</span>51 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-17-7"  
class="td11">  
1<span class="cmmi-12">.</span>04 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-18-1"  
class="td11"> 
NF-kB is activated and signals survival
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-18-2"  
class="td11"> 
209560
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-18-3"  
class="td11"> 
71
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-18-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-18-5"  
class="td11">   
145
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-18-6"  
class="td11"> 
1<span class="cmmi-12">.</span>90 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-18-7"  
class="td11">  
1<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-19-1"  
class="td11"> 
Peptide chain elongation
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-19-2"  
class="td11"> 
156902
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-19-3"  
class="td11"> 
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-19-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-19-5"  
class="td11">   
145
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-19-6"  
class="td11"> 
1<span class="cmmi-12">.</span>91 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-19-7"  
class="td11">  
1<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-20-1"  
class="td11"> 
Influenza Life Cycle
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-20-2"  
class="td11"> 
168255
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-20-3"  
class="td11"> 
70
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-20-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-20-5"  
class="td11">   
145
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-20-6"  
class="td11"> 
1<span class="cmmi-12">.</span>95 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-20-7"  
class="td11">  
1<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-21-1"  
class="td11"> 
Formation of a pool of free 40S subunits
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-21-2"  
class="td11">  
72689
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-21-3"  
class="td11"> 
73
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-21-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-21-5"  
class="td11">   
145
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-21-6"  
class="td11"> 
2<span class="cmmi-12">.</span>01 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-21-7"  
class="td11">  
1<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-22-1"  
class="td11"> 
Nonsense-Mediated Decay
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-22-2"  
class="td11"> 
927802
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-22-3"  
class="td11"> 
71
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-22-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-22-5"  
class="td11">   
145
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-22-6"  
class="td11"> 
2<span class="cmmi-12">.</span>44 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-22-7"  
class="td11">  
1<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-23-1"  
class="td11"> 
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-23-2"  
class="td11"> 
975957
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-23-3"  
class="td11"> 
71
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-23-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-23-5"  
class="td11">   
145
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-23-6"  
class="td11"> 
2<span class="cmmi-12">.</span>44 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-23-7"  
class="td11">  
1<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-24-1"  
class="td11"> 
GTP hydrolysis and joining of the 60S ribosomal subunit
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-24-2"  
class="td11">  
72706
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-24-3"  
class="td11"> 
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-24-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-24-5"  
class="td11">   
145
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-24-6"  
class="td11"> 
2<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-24-7"  
class="td11">  
1<span class="cmmi-12">.</span>37 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-25-1"  
class="td11"> 
Influenza Viral <a 
href="#glo:RNA">RNA</a> Transcription and Replication
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-25-2"  
class="td11"> 
168273
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-25-3"  
class="td11"> 
72
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-25-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-25-5"  
class="td11">   
144
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-25-6"  
class="td11"> 
4<span class="cmmi-12">.</span>01 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-25-7"  
class="td11">  
2<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-66-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-26-1"  
class="td11"> 
Signalling by NOTCH1 HD Domain Mutants in Cancer
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-66-26-2"  
class="td11"> 
2691230
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-26-3"  
class="td11"> 
79
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-26-4"  
class="td11">   
130
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-26-5"  
class="td11">   
143
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-26-6"  
class="td11"> 
5<span class="cmmi-12">.</span>99 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-66-26-7"  
class="td11">  
2<span class="cmmi-12">.</span>82 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-66-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-66-27-1"  
class="td11">                                                                         </td></tr></table>
<!--l. 1148--><p class="noindent" ></div>
<p>Strongest candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners for <span class="cmti-12">CDH1 </span>by <a 
href="#glo:SLIPT">SLIPT</a> with observed and expected numbers of <a 
href="#glo:TCGA">TCGA</a> breast cancer samples with low <a 
href="#glo:gene expression">expression</a> of both <span class="cmti-12">CDH1 </span>and the <a 
href="#glo:metagene">metagene</a>. The most significant 26 pathways are reported to show the diversity of candidate synthetic lethal <a 
href="#glo:metagene">metagene</a> and demonstrate the highly significant results of <a 
href="#glo:SLIPT">SLIPT</a> when performed on <a 
href="#glo:pathway">pathway</a> <a 
href="#glo:metagene">metagenes</a>.  </p>
</p>
</div><hr class="endfloat" />
<!--l. 1151--><p class="indent" >
Signalling <a 
href="#glo:pathway">pathways</a> were more strongly supported by <a 
href="#glo:mtSLIPT">mtSLIPT</a> analysis of <a 
href="#glo:metagene">metagene</a> <a 
href="#glo:pathway">pathways</a> <a 
href="#glo:gene expression">expression</a> against <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutation</a>, as shown in Table <a 
href="#x1-1790011">D.1<!--tex4ht:ref: tab:metagene_mtSL --></a>, although these results were generally less statistically significant than <a 
href="#glo:gene expression">expression</a> analyses. Signalling <a 
href="#glo:pathway">pathways</a> detected as <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:metagene">metagenes</a> include G<sub><span class="cmmi-8">αz</span></sub>, insulin-related growth factor (IGF), GABA receptor, G<sub><span class="cmmi-8">αs</span></sub>, S6K1 and various toxin responses mediated by <a 
href="#glo:GPCR">GPCRs</a>. Metabolic processes including processing of carbohydrates and fatty acids were also implicated across these analyses.
</p><!--l. 1153--><p class="indent" >
The <a 
href="#glo:metagene">metagene</a> analyses differ more between expresssion and <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutation</a> than previous analyses, with more specific signalling <a 
href="#glo:pathway">pathways</a> identified in the <a 
href="#glo:mutation">mutation</a> analysis. This supports the usage of a complete null <a 
href="#glo:mutant">mutant</a> model in experimental testing for <a 
href="#glo:synthetic lethal">synthetic lethality</a> of signalling <a 
href="#glo:pathway">pathways</a> against CDH1 inactivation rather than a knockdown in <a 
href="#glo:gene expression">expression</a>. However, low <a 
href="#glo:gene expression">expression</a> of partners has been used in either case to be applicable to dose-dependent pharmacological inhibition and across genes where <a 
href="#glo:mutation">mutations</a> have different functional consequences, including variants of unknown significance.
</p><!--l. 1155--><p class="indent" >
These results show an alternative <a 
href="#glo:pathway">pathways</a>-based approach to detecting <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene functions interacting with <span class="cmti-12">CDH1</span>. The use of <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:metagene">metagenes</a> replicates support for these <a 
href="#glo:pathway">pathways</a> independent of <a 
href="#glo:pathway">pathways</a> size (as genes are weighted equally). Having verified that the direction of <a 
href="#glo:metagene">metagenes</a> recapitulates the activity of a <a 
href="#glo:pathway">pathways</a>, these demonstrate that many of the <a 
href="#glo:pathway">pathways</a> previously identified (from <a 
href="#glo:SLIPT">SLIPT</a> candidate genes) are <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a>, with their activity dependent on <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes, rather than containing <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes as inhibitors or peripheral regulators of the <a 
href="#glo:pathway">pathways</a>.
</p><!--l. 1159--><p class="indent" >
The <a 
href="#glo:synthetic lethal">synthetic lethal</a> analysis against low <span class="cmti-12">CDH1 </span><a 
href="#glo:gene expression">expression</a> supports prior findings in translational and immune <a 
href="#glo:pathway">pathways</a> even if they were not able to detected in an experimental screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). Together these findings support the role of <span class="cmti-12">CDH1</span> loss in cancer disrupting cell signalling with wider effects on protein translation and metabolism necessary for the proliferation of cancer cells. This is consistent with the <a 
href="#glo:GPCR">GPCR</a> <a 
href="#glo:pathway">pathways</a>, such as G<sub><span class="cmmi-8">αs</span></sub> signalling, being supported by <a 
href="#glo:SLIPT">SLIPT</a> gene candidates and the experimental primary <a 
href="#glo:siRNA">siRNA</a> screen, as shown by resampling in Section <a 
href="#x1-1070001">4.2.5.1<!--tex4ht:ref: chapt3:compare_pathway_perm --></a>. <a 
 id="x1-109002r214"></a>
</p>
                                                                                    

<h3 class="sectionHead">
<span class="titlemark">4.4 </span> <a 
 id="x1-1100004"></a>Replication in Stomach Cancer
</h3>
<!--l. 1184--><p class="noindent" >
<span class="cmti-12">CDH1 </span>is also important in stomach cancer biology as a <a 
href="#glo:driver mutation">driver</a> <a 
href="#glo:tumour suppressor">tumour suppressor</a> gene, including as a <a 
href="#glo:germline">germline mutation</a> <a 
href="#glo:mutation">mutation</a> in many cases of <a 
href="#glo:hereditary">hereditary</a> diffuse gastric cancer. The <a 
href="#glo:synthetic lethal">synthetic lethal</a> analysis of genes and <a 
href="#glo:pathway">pathways</a> (previously identified for <a 
href="#glo:TCGA">TCGA</a> breast cancer data) was replicated in <a 
href="#glo:TCGA">TCGA</a> stomach cancer. The accompanying data for <a 
href="#glo:SLIPT">SLIPT</a> analysis against <span class="cmti-12">CDH1 </span><a 
href="#glo:gene expression">expression</a> is provided in Appendix <a 
href="#x1-181000F">F<!--tex4ht:ref: appendix:stad_exprSL --></a>.
</p><!--l. 1186--><p class="indent" >
While the sample size was lower for <a 
href="#glo:TCGA">TCGA</a> stomach cancer (particularly for <a 
href="#glo:mutation">mutations</a>), the results serve to support the findings in breast cancer in an independent patient cohort and tissue samples. The molecular profiling, including <a 
href="#glo:RNA-Seq">RNA-Seq</a> <a 
href="#glo:gene expression">expression</a>, were performed by <a 
href="#glo:TCGA">TCGA</a> using the sample procedures as for breast cancer and the findings reported here were performed useing data analysis techniques identical to those presented previously. These procedures should ensure as close a comparison as feasible across both of the cancer types most relevant to <a 
href="#glo:HDGC">HDGC</a> and recurrent <span class="cmti-12">CDH1</span> <a 
href="#glo:mutation">mutations</a>.
</p><!--l. 1188--><p class="indent" >
<p>The strongest <a 
href="#glo:SLIPT">SLIPT</a> genes for stomach cancer (shown in Table <a 
href="#x1-1820011">F.1<!--tex4ht:ref: tab:gene_stad_SL --></a>) did not necessarily directly correspond to those observed in breast cancer (shown in Appendix Table <a 
href="#x1-970011">4.1<!--tex4ht:ref: tab:gene_SL --></a>). However, several gene functions were replicated in stomach cancer. Together, these gene candidates indicate widespread functions of <span class="cmti-12">CDH1 </span>and strongly detectable <a 
href="#glo:synthetic lethal">synthetic lethality</a> with many genes, using a strategy that can be applied across cancer types. More specifically, the signalling genes included <a 
href="#glo:GPCR">GPCR</a> signalling genes, which was one of the most supported <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> in breast cancer analysis and the experimental screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>).These findings were further supported by the <a 
href="#glo:pathway">pathways</a> over-represented in <a 
href="#glo:SLIPT">SLIPT</a> candidates from <a 
href="#glo:TCGA">TCGA</a> stomach cancer (shown in Appendix Table <a 
href="#x1-1820022">F.2<!--tex4ht:ref: tab:pathway_stad_exprSL --></a>) which replicated the translational and immune <a 
href="#glo:pathway">pathways</a> observed in <a 
href="#glo:TCGA">TCGA</a> breast cancer (shown in Table <a 
href="#x1-980012">4.2<!--tex4ht:ref: tab:pathway_exprSL --></a>) and further supported GCPR signalling <a 
href="#glo:pathway">pathways</a>, including the class A/1 receptors. The extracellular matrix was also detected at the <a 
href="#glo:pathway">pathways</a> level in stomach cancer, including elastic fibres, glycosylation, collagen, and integrin cell-surface interactions. While fewer <a 
href="#glo:pathway">pathways</a> were supported by resampling for the intersection of <a 
href="#glo:SLIPT">SLIPT</a> and experimental screen candidate partners in stomach cancer than breast cancer, many of those detected (shown in Appendix Table <a 
href="#x1-1840026">F.6<!--tex4ht:ref: tab:pathway_perm_overlap_stad --></a>) replicate those detected in breast cancer (shown in Table <a 
href="#x1-1070038">4.8<!--tex4ht:ref: tab:pathway_perm_overlap --></a>). The <a 
href="#glo:pathway">pathways</a> detected by both permutation and over-representation analysis were more likely to be replicated across stomach and breast cancer than those detected by over-representation alone, supporting the use of this</p>
procedure to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> applicable across cancer types. The include G<sub><span class="cmmi-8">αs</span> </sub> signalling and elastic fibre formation as discussed for breast cancer (in Section <a 
href="#x1-1070001">4.2.5.1<!--tex4ht:ref: chapt3:compare_pathway_perm --></a>). <a 
 id="x1-110001r216"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">4.5 </span> <a 
 id="x1-1110005"></a>Discussion
</h3>
<a 
 id="x1-111001r213"></a>
<h4 class="subsectionHead">
<span class="titlemark">4.5.1 </span> <a 
 id="x1-1120001"></a>Strengths of the SLIPT Methodology
</h4>
<!--l. 1577--><p class="noindent" >
Synthetic lethal discovery with <a 
href="#glo:SLIPT">SLIPT</a> used established statistical procedures to identify putative partner genes from <a 
href="#glo:gene expression">gene expression</a> data. Such use of the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>-value is amenable to <a 
href="#glo:pathway">pathways</a> or permutation analyses and could feasibly be applied to other disease gene or pair-wise across the <a 
href="#glo:genome">genomes</a>, although previous <a 
href="#glo:genome">genomes</a>-wide approaches were unable to find informative candidate genes for <a 
href="#glo:E-cadherin">E-cadherin</a> (<a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>). <a 
href="#glo:synthetic lethal">Synthetic lethal</a> discovery in cancer has focused on genes with severe cellular <a 
href="#glo:mutant">mutant</a> phenotypes, such as <a 
href="#glo:essential">essential</a> genes or the <a 
href="#glo:oncogene">oncogenes</a> <span class="cmti-12">TP53 </span>and <span class="cmti-12">AKT </span>(<a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>; <a 
href="#XTiong2014">Tiong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTiong2014">2014</a>; <a 
href="#XWang2013">Wang and Simon</a>, <a 
href="#XWang2013">2013</a>), with other cancer genes, such as <span class="cmti-12">CDH1</span>, requiring more focused investigations. Prior computational approaches for <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery, in cancer, vary widely (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>; <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>; <a 
href="#XTiong2014">Tiong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTiong2014">2014</a>; <a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWappett2016">2016</a>). There is no consensus as to which approach is more appropriate, and the methods are difficult to compare, as they either do not have a released code implementation or do not make predictions solely from normalised <a 
href="#glo:gene expression">expression</a> data.
</p><!--l. 1579--><p class="indent" >
The query-based approach demonstrated by <a 
href="#glo:SLIPT">SLIPT</a> analysis is suitable for wider application on <a 
href="#glo:gene expression">expression</a> data and for augmenting experimental studies such as high-throughput screens. This approach has identified biologically plausible <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> for <span class="cmti-12">CDH1</span>, triaged candidates from experimental screening (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), and replicates genes and <a 
href="#glo:pathway">pathways</a> across breast and stomach cancer datasets. In addition, <a 
href="#glo:SLIPT">SLIPT</a> avoids critical assumptions underlying the design of some approaches such as co-expression of synthetic candidates or that interacting gene pairs will have known (annotated) similarities in function.
</p><!--l. 1581--><p class="indent" >
<p>The DAISY methodology <a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XJerby2014">2014</a>), which took a similar query-based approach with the <a 
href="#glo:tumour suppressor">tumour suppressor</a> <span class="cmti-12">VHL</span>, has been critiqued for being too stringent (<a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>) which impedes <a 
href="#glo:pathway">pathways</a> analysis. Since <a 
href="#glo:functional redundancy">functional redundancy</a> does not require genes to be expressed at the same time, the <a 
href="#glo:SLIPT">SLIPT</a> approach does not assume co-expression of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes which may enrich for <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes in established</p>
coregulated <a 
href="#glo:pathway">pathways</a>. Rather, the interpretation of <a 
href="#glo:synthetic lethal">synthetic lethality</a> for <a 
href="#glo:SLIPT">SLIPT</a> was similar to other computational methods based on ‘co-loss under-representation’, ‘compensation’, or ‘simultaneous differential <a 
href="#glo:gene expression">expression</a>’ (<a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>; <a 
href="#XTiong2014">Tiong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTiong2014">2014</a>; <a 
href="#XWang2013">Wang and Simon</a>, <a 
href="#XWang2013">2013</a>).
</p><!--l. 1583--><p class="indent" >
Genomics analyses are prone to false-positives and require statistical caution, particularly where working with gene-pairs scale sup the number of multiple tests drastically, at the expense of statistical power. Experimental screens for <a 
href="#glo:synthetic lethal">synthetic lethality</a> are also error-prone (<a 
href="#XFece2015">Fece de la Cruz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFece2015">2015</a>; <a 
href="#XLord2014">Lord <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLord2014">2015</a>; <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>), especially with false-positives, raising the need for understanding the expected behaviour and number of functional relationships and genetic interactions in the <a 
href="#glo:genome">genomes</a>, or in discovery of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of a particular query gene. Thus analyses throughout this thesis have focused on querying for partners of a particular gene of interest. Statistical modelling and simulations (in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a> and Chapter <a 
href="#x1-1340006">6<!--tex4ht:ref: chap:simulation --></a>) will further support the design decisions underlying <a 
href="#glo:SLIPT">SLIPT</a> analysis and its strengths over other approaches. <a 
 id="x1-112001r218"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.5.2 </span> <a 
 id="x1-1130002"></a>Synthetic Lethal Pathways for E-cadherin
</h4>
<!--l. 1587--><p class="noindent" >
Specific genes were difficult to replicate across experiments. This is consistent with <a 
href="#glo:gene expression">gene expression</a> profiles for <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners reflecting the complexity of biological <a 
href="#glo:pathway">pathways</a> which are subject to higher-order interactions and do not consistently compensate for loss of gene function across all samples (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>; <a 
href="#XKelly2013">Kelly</a>, <a 
href="#XKelly2013">2013</a>; <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>). The predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">CDH1 </span>(with <a 
href="#glo:FDR">FDR</a> correction) were investigated with <a 
href="#glo:gene expression">gene expression</a> profiles and clinical variables to find relationships in <a 
href="#glo:gene expression">gene expression</a>, gene function, and clinical characteristics. The large number of genes detected indicates that <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection is potentially error-prone, and that identifying genes relevant for clinical application will be difficult without a supporting biological <a 
href="#glo:pathway">pathways</a> rationale. As such, investigations into the genes identified by <a 
href="#glo:SLIPT">SLIPT</a>, the correlation structure between them, and those which were validated by experimental screening (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) focused at the <a 
href="#glo:pathway">pathways</a> level throughout this Chapter. Similarly, comparisons across analyses were largely made at the <a 
href="#glo:pathway">pathways</a> level, including comparisons between <a 
href="#glo:gene expression">expression</a> and <a 
href="#glo:mutation">mutation</a>, breast and stomach <a 
href="#glo:TCGA">TCGA</a> datasets.
</p><!--l. 1589--><p class="indent" >
<p>Potential <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">CDH1 </span>identified by <a 
href="#glo:SLIPT">SLIPT</a> had many distinct functions, with each gene cluster highly expressed in different patient subgroups</p>
(Figure <a 
href="#x1-990011">4.1<!--tex4ht:ref: fig:slipt_expr --></a>). The <a 
href="#glo:gene expression">expression</a> profiles of the SL partners of <span class="cmti-12">CDH1 </span>predicted from <a 
href="#glo:TCGA">TCGA</a> breast cancer <a 
href="#glo:RNA-Seq">RNA-Seq</a> data (expected to have compensating high or stable <a 
href="#glo:gene expression">expression</a>) and their corresponding functional enrichment found in subgroups of genes, particularly among <span class="cmti-12">CDH1 </span>low breast tumours. Ductal breast cancers showed higher <a 
href="#glo:gene expression">expression</a> of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners, suggesting treatmemay be more effective in this tumour subtype. However, there was consistently low <a 
href="#glo:gene expression">expression</a> of SL partners in estrogen receptor negative tumours, although this is independent of tumour stage and consistent with poor prognosis in these patients and could inform other treatment strategies or prevent ineffective treatment further impacting quality of life in these patients. These results suggest that <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner <a 
href="#glo:gene expression">expression</a> varies between patients; that these different tumour classes would react differently to the same treatment; that treatment of different <a 
href="#glo:pathway">pathways</a> and combinations in different patients is the most effective approach to target genes compensating for <span class="cmti-12">CDH1 </span>gene loss; and that the <a 
href="#glo:gene expression">expression</a> of synthetic partners could be a clinically important biomarker.
</p><!--l. 1591--><p class="indent" >
The <a 
href="#glo:pathway">pathways</a> that <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">CDH1 </span>identified by <a 
href="#glo:SLIPT">SLIPT</a> were involved in a diverse range of biological functions and differed to those detected experimentally. This discrepancy may be accounted for by <a 
href="#glo:gene expression">gene expression</a> analyses detecting both <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners, as screened for experimentally by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>), and their downstream targets (not detected by <a 
href="#glo:siRNA">siRNA</a>), capturing the wider <a 
href="#glo:pathway">pathways</a> and mechanisms involved in <a 
href="#glo:synthetic lethal">synthetic lethality</a> with <span class="cmti-12">CDH1 </span>inactivation. In particular, <a 
href="#glo:GPCR">GPCR</a> phosphorylation cascades (which regulate <a 
href="#glo:gene expression">gene expression</a> and translation in cancers (<a 
href="#XGao2015">Gao and Roux</a>, <a 
href="#XGao2015">2015</a>)) were predicted to be <a 
href="#glo:synthetic lethal">synthetic lethal</a> with <span class="cmti-12">CDH1</span>. The predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners occurred across functionally distinct <a 
href="#glo:pathway">pathways</a>, including characterised functions of <span class="cmti-12">CDH1</span>. The most consistently supported <a 
href="#glo:pathway">pathways</a> included elastic fibres in the extracelullar matrix, <a 
href="#glo:GPCR">GPCR</a> signalling, and translation presenting vulnerabilities for <span class="cmti-12">CDH1</span> deficient cancer cells from extracellular stimuli to the core growth mechanisms of a cell.
</p><!--l. 1593--><p class="indent" >
<p>This diversity in <a 
href="#glo:synthetic lethal">synthetic lethal</a> functions is consistent with the wide ranging role of <span class="cmti-12">CDH1 </span>in cell-cell adhesion, cell signalling, and the cytoskeletal structure of epithelial tissues. Pathway structure may be relevant to identifying potential drug targets from <a 
href="#glo:gene expression">gene expression</a> signatures, indicating downstream effector genes and mechanisms leading to cell inviability. Identification of distinct <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene clusters may further lead to the elucidation of drug resistance mechanisms. While these <a 
href="#glo:pathway">pathways</a> are indicative of the main functions of <a 
href="#glo:E-cadherin">E-cadherin</a> and <a 
href="#glo:synthetic lethal">synthetic lethal</a></p>
partners, it remains to identify the genes within these <a 
href="#glo:pathway">pathways</a> that are the most actionable or supported across <a 
href="#glo:SLIPT">SLIPT</a> analysis in patient samples and detected by experiments in preclinical models (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>; <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). The specific genes within key <a 
href="#glo:pathway">pathways</a> will be be discussed in Chapter <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>, along with further investigations into their relation to <a 
href="#glo:graph">pathway</a> structure. While these are important clinical implications, the <a 
href="#glo:synthetic lethal">synthetic lethal</a> predictions lack enough confidence for direct translation into pre-clinical models or clinical applications leading to a need for statistical modelling and simulation of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in <a 
href="#glo:genomic">genomics</a> <a 
href="#glo:gene expression">expression</a> data.
</p><!--l. 1595--><p class="indent" >
These <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> have potential clinical implications, particularly those supported in pre-clinical models and in patient <a 
href="#glo:gene expression">expression</a> data. However, further validation of gene candidates will be necessary to ensure that these are able to reproduced in further pre-clinical studies, they are applicable to tumours <span class="cmti-12">in</span> <span class="cmti-12">vivo</span>, and that effective inhibitory agents can be repurposed or designed against them. <a 
 id="x1-113001r219"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.5.3 </span> <a 
 id="x1-1140003"></a>Replication and Validation
</h4>
<a 
 id="x1-114001r209"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">4.5.3.1 </span> <a 
 id="x1-1150001"></a>Integration with siRNA Screening
</h5>
<!--l. 1601--><p class="noindent" >
The <a 
href="#glo:pathway">pathways</a> composition across computational and experimental <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates was informative with over-representation (Table <a 
href="#x1-1060016">4.6<!--tex4ht:ref: tab:Venn_over-representation --></a>) and supported by resampling analysis (Table <a 
href="#x1-1070038">4.8<!--tex4ht:ref: tab:pathway_perm_overlap --></a>), despite a modest intersection of genes between them (Figure <a 
href="#x1-1020012">4.2<!--tex4ht:ref: fig:Venn_allgenes --></a>). Either approach may be significant for a <a 
href="#glo:pathway">pathways</a> in this intersection without being supported by the other: resampling analysis may support <a 
href="#glo:pathway">pathways</a> that were not over-represented due to small effect sizes, thus both tests are required for a candidate <a 
href="#glo:pathway">pathways</a>.
</p><!--l. 1603--><p class="indent" >
<p>The <a 
href="#glo:pathway">pathways</a> detected by both over-representation and resampling are the strongest candidates for further investigation and the <a 
href="#glo:graph">pathway</a> structure analyses in Chapter <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a> will focus on these <a 
href="#glo:pathway">pathways</a> detected by both over-representation and resampling. Particularly, those replicated across datasets or with <a 
href="#glo:pathway">pathways</a> <a 
href="#glo:metagene">metagenes</a>. In addition to GCPR <a 
href="#glo:pathway">pathways</a> detected across these analyses, the <a 
href="#glo:PI3K">PI3K</a> cascade will also be investigated in Chapter <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>, this signalling <a 
href="#glo:pathway">pathways</a> is a well characterised mediator between GCPR receptors and regulation of translation (<a 
href="#XGao2015">Gao and Roux</a>, <a 
href="#XGao2015">2015</a>) (both detected throughout this Chapter) and exhibited unexpected</p>
behaviour with <a 
href="#glo:pathway">pathways</a> the <a 
href="#glo:metagene">metagenes</a> (in Section <a 
href="#x1-177000D">D<!--tex4ht:ref: chapt3:metagene_results --></a>). This <a 
href="#glo:pathway">pathways</a> is activated by protein phosphorylation states and thus inactivation may not be detectable with <a 
href="#glo:gene expression">expression</a>.
</p><!--l. 1605--><p class="indent" >
The <a 
href="#glo:SLIPT">SLIPT</a> approach was shown to be predictive of which <a 
href="#glo:siRNA">siRNA</a> primary screen candidate partners of <span class="cmti-12">CDH1 </span>were validated in a secondary screen (as shown in Section <a 
href="#x1-1050004">4.2.4<!--tex4ht:ref: chapt3:secondary_screen --></a>). These results further support <a 
href="#glo:SLIPT">SLIPT</a> for identifying robust <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates which can be validated and as a triage approach for interpreting screening experiments. <a 
 id="x1-115001r221"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">4.5.3.2 </span> <a 
 id="x1-1160002"></a>Replication across Tissues
</h5>
<!--l. 1609--><p class="noindent" >
Furthermore, <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners identified by <a 
href="#glo:SLIPT">SLIPT</a> were replicated across breast and stomach cancer. These were particularly concordant at the <a 
href="#glo:pathway">pathways</a> level, as expected between tissues, since <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> have higher conservation between species (<a 
href="#XDixon2008">Dixon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDixon2008">2008</a>). These findings support gene functions conserved across <span class="cmti-12">CDH1 </span>deficient cancers in breast and stomach tissues, presenting vulnerabilities that could be applied against molecular targets in both cancers. In addition, these analyses serve as a replication across independent patient cohorts from breast and stomach cancers, decreasing the likelihood of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> detected being false positives or artifacts of either dataset.
</p><!--l. 1611--><p class="indent" >
Synthetic lethal <a 
href="#glo:pathway">pathways</a> were also replicated across <a 
href="#glo:gene expression">expression</a> analyses of <a 
href="#glo:TCGA">TCGA</a> patient samples in heterogeneous tumours and homogeneous cell line isolates. This further supports that the subset of <a 
href="#glo:synthetic lethal">synthetic lethal</a> functions detectable in experimental models (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>; <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) would be applicable tumours of patients with <span class="cmti-12">CDH1 </span>deficient cancers.
</p><!--l. 1613--><p class="indent" >
There are many gene functions replicated across breast cancer <a 
href="#glo:gene expression">gene expression</a> analyses. Many of these were also replicated with <a 
href="#glo:mutation">mutation</a> analysis and with stomach cancer or cell line <a 
href="#glo:gene expression">expression</a> data. These <a 
href="#glo:pathway">pathways</a> were more consistent across replication analyses than previous investigations with <a 
href="#glo:TCGA">TCGA</a> <a 
href="#glo:microarray">microarray</a> data (<a 
href="#XKelly2013">Kelly</a>, <a 
href="#XKelly2013">2013</a>). <a 
 id="x1-116001r217"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">4.6 </span> <a 
 id="x1-1170006"></a>Summary
</h3>
<!--l. 1617--><p class="noindent" >
<p>We have developed a simple, interpretable, computational approach to predict <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners from <a 
href="#glo:genomic">genomics</a> data. The analyses focus on <a 
href="#glo:gene expression">gene expression</a> data as it is</p>
widely available for applications in other cancers and other disease genes, particularly those with malignant loss of function.
</p><!--l. 1619--><p class="indent" >
This approach has been applied to robustly detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> for <a 
href="#glo:E-cadherin">E-cadherin</a> (the <span class="cmti-12">CDH1 </span>gene) in <a 
href="#glo:TCGA">TCGA</a> breast cancer molecular profiles, with comparisons to experimental screening (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) in cell lines and replication in <a 
href="#glo:TCGA">TCGA</a> stomach cancer molecular profiles. The <a 
href="#glo:pathway">pathways</a> replicated across several analyses included extracellular matrix <a 
href="#glo:pathway">pathways</a> (e.g., elastic fibres formation), cell signalling (including <a 
href="#glo:GPCR">GPCRs</a>), and core gene regulation and translation processes crucial for the growth and proliferation of cancer cells. These <a 
href="#glo:pathway">pathways</a> show evidence of <a 
href="#glo:non-oncogene addiction">non-oncogene addiction</a> for <span class="cmti-12">CDH1 </span>deficient cells, which presents vulnerabilities that could be exploited for specific treatment against <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutations</a> in HCGC and <a 
href="#glo:sporadic">sporadic</a> cancers. There was also support for <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> with <span class="cmti-12">CDH1 </span>in cell adhesion and cytoskeletal processes to which <span class="cmti-12">CDH1 </span>belongs, supporting the finding that <a 
href="#glo:synthetic lethal">synthetic lethality</a> occurs within biological <a 
href="#glo:pathway">pathways</a> (<a 
href="#XBoone2007">Boone <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoone2007">2007</a>; <a 
href="#XKelley2005">Kelley and Ideker</a>, <a 
href="#XKelley2005">2005</a>).
</p><!--l. 1622--><p class="indent" >
While translational and immune <a 
href="#glo:pathway">pathways</a> detected by <a 
href="#glo:SLIPT">SLIPT</a> were not supported by primary <a 
href="#glo:siRNA">siRNA</a> screening (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), these were replicated across various analyses. Due to the differences between an experimental cell line model (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>; <a 
href="#XFece2015">Fece de la Cruz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFece2015">2015</a>) and patient molecular profiles (<a 
href="#XTCGA2014GC">Bass <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014GC">2014</a>; <a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>), these would not be expected to be completely concordant. Furthermore, many <a 
href="#glo:pathway">pathways</a> are difficult to test in an isolated experimental system. Nevertheless, many of the genes and <a 
href="#glo:pathway">pathways</a> detected by <a 
href="#glo:SLIPT">SLIPT</a> are suitable to inform further investigations and triage of therapeutic targets against <span class="cmti-12">CDH1 </span>deficient tumours in combination with experimental screening.
</p><!--l. 1627--><p class="indent" >
<p>Thus <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> have been identified using <a 
href="#glo:TCGA">TCGA</a> patient molecular profiles and experimental screening results. Some these were robustly replicated across these datasets and against <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutation</a> or <a 
href="#glo:gene expression">expression</a> analysis. However, there remains the need to identify actionable genes within these <a 
href="#glo:pathway">pathways</a>, relationships with experimental candidates, and how these <a 
href="#glo:pathway">pathways</a> may affect viability when lost. While the genes identified between these analyses were less concordant the results of the <a 
href="#glo:TCGA">TCGA</a> breast cancer analysis will be used to test <a 
href="#glo:graph">pathway</a> structure relationships and further examine the <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes detected in the following Chapter.</p>
</p>
<h2 class="chapterHead">
<span class="titlemark">Chapter 5</span><br /><a 
 id="x1-1180005"></a>Synthetic Lethal Pathway Structure
</h2>
<!--l. 4--><p class="noindent" >
Having identified key <a 
href="#glo:pathway">pathways</a> implicated in <a 
href="#glo:synthetic lethal">synthetic lethal</a> genetic interactions with <span class="cmti-12">CDH1 </span>(in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>), these were investigated in this Chapter for the <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes within them, and for their relationships to <a 
href="#glo:graph">pathway</a> structure. This chapter will focus on the Reactome biological <a 
href="#glo:pathway">pathways</a> detected across analyses in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>. Specifically, investigations were performed to determine whether <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates, detected by <a 
href="#glo:SLIPT">SLIPT</a> or <a 
href="#glo:siRNA">siRNA</a>, exhibited differences with respect to metrics of <a 
href="#glo:graph">pathway</a> structure of network connectivity and importance (as described in Sections <a 
href="#x1-680004">2.4.4<!--tex4ht:ref: methods:network_metrics --></a> and <a 
href="#x1-900003">3.5.3<!--tex4ht:ref: methods:igraph_extensions --></a>). The relationships between <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates, detected by either approach, were also examined to determine whether <a 
href="#glo:SLIPT">SLIPT</a> candidate genes were upstream or downstream <a 
href="#glo:siRNA">siRNA</a> candidate genes. These directional relationships were tested by resampling (as described in Sections <a 
href="#x1-840001">3.4.1<!--tex4ht:ref: methods:pathway_str --></a> and <a 
href="#x1-850001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>) and comparisons to the <a 
href="#glo:pathway">pathway</a> hierarchical score based on biological context (as derived in Section <span class="cmbx-12">??</span>). Together these investigations into structural relationships demonstrate how a combination of network biology and statistical techniques can be performed with genes identified by a <a 
href="#glo:bioinformatics">bioinformatics</a> analysis. <a 
 id="x1-118001r223"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">5.1 </span> <a 
 id="x1-1190001"></a>Synthetic Lethal Genes in Reactome Pathways
</h3>
<!--l. 18--><p class="noindent" >
The <a 
href="#glo:graph">graph</a> structure for Reactome <a 
href="#glo:pathway">pathways</a> was obtained from Pathway Commons via <a 
href="#glo:BioPAX">BioPAX</a> (as described in Section <a 
href="#x1-660002">2.4.2<!--tex4ht:ref: methods:graph_data --></a>). The <a 
href="#glo:pathway">pathways</a> describe the (directional) relationships between biomolecules, including genes that encode proteins in biological <a 
href="#glo:pathway">pathways</a>. These relationships include cell signalling (e.g., kinase phosphorylation cascades), gene regulation (e.g., transcription factors, chromatin modifiers, <a 
href="#glo:RNA">RNA</a> binding proteins), and metabolism (e.g., the product of an enzyme being the substrate of another). Together these relationships describe the known functional <a 
href="#glo:pathway">pathways</a> in a human cell with a reasonable resolution, from a curated database supported by publications documenting <a 
href="#glo:pathway">pathway</a> relationships.
</p><!--l. 20--><p class="indent" >
<p>Pathway structures from the Reactome network (as described in Section <a 
href="#x1-670003">2.4.3<!--tex4ht:ref: methods:subgraphs --></a>) were used to derive the <a 
href="#glo:graph">graph</a> structure of each biological <a 
href="#glo:pathway">pathway</a>. The <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate genes for notable <a 
href="#glo:pathway">pathways</a> discussed in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>, including candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> of <span class="cmti-12">CDH1</span>, were examined to show the <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidates within these <a 
href="#glo:pathway">pathways</a>. The <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes considered here are</p>
those candidates detected by <a 
href="#glo:SLIPT">SLIPT</a> (as described in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) in <a 
href="#glo:TCGA">TCGA</a> breast cancer <a 
href="#glo:gene expression">expression</a> and <a 
href="#glo:mutation">mutation</a> data (<a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>) in comparison to the candidate gene partners from the <a 
href="#glo:siRNA">siRNA</a> screening in breast cell lines (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). <a 
 id="x1-119001r220"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">5.1.1 </span> <a 
 id="x1-1200001"></a>The PI3K/AKT Pathway
</h4>
<!--l. 28--><p class="noindent" >
The <a 
href="#glo:PI3K">phosphoinositide 3-kinase</a> (<a 
href="#glo:PI3K">PI3K</a>) cascade signalling <a 
href="#glo:pathway">pathway</a> is important in cancer because it is involved in mediating signals between the <a 
href="#glo:GPCR">GPCRs</a> and regulation of protein translation, which have both been strongly implicated to be <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> with loss of <span class="cmti-12">CDH1 </span>function (Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>). These <a 
href="#glo:pathway">pathways</a> are all subject to dysregulation in cancer (<a 
href="#XCourtney2010">Courtney <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCourtney2010">2010</a>; <a 
href="#XDorsam2007">Dorsam and Gutkind</a>, <a 
href="#XDorsam2007">2007</a>; <a 
href="#XGao2015">Gao and Roux</a>, <a 
href="#XGao2015">2015</a>). Thus the PI3K cascade will be examined along with the most supported <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> (as identified in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>).
</p><!--l. 30--><p class="indent" >
<p>The <a 
href="#glo:PI3K">PI3K</a> <a 
href="#glo:pathway">pathway</a> is well characterised and has an established direction of signal transduction from extracellular stimuli (and membrane bound receptors) to the inner mechanisms of the cell, namely, the regulation of protein translation. The production of proteins is necessary for the growth of the cell so it is reasonable to suggest that these processes may be subject to (non-<a 
href="#glo:oncogene">oncogene</a>) addiction in some cancer cells which rely upon them for sustained protein production and cell growth. This is also supported by the <a 
href="#glo:oncogene">oncogenes</a> <span class="cmti-12">PIK3CA </span>and <span class="cmti-12">AKT1 </span>being involved with the PI3K cascade and the related PI3K/AKT <a 
href="#glo:pathway">pathway</a>, which may be subject to <a 
href="#glo:oncogene addiction">oncogene addiction</a> when these proto-oncogenes are activated.</p>
</p><!--l. 32--><p class="indent" >
<a 
 id="x1-1200011"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 34--><p class="noindent" >
</p><!--l. 35--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_Pi3k_exprSL2".png" alt="pict"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 5.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality in the PI3K cascade.</span></strong> <span class="cmr-10x-x-109">The Reactome </span><a 
href="#glo:PI3K"><span class="cmr-10x-x-109">PI3K</span></a> <span class="cmr-10x-x-109">Cascade</span> <a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathway</span></a> <span class="cmr-10x-x-109">with </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">candidates coloured as shown in the legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-1200011 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 48--><p class="indent" >
The <a 
href="#glo:PI3K">PI3K</a> cascade was not supported across <a 
href="#glo:SLIPT">SLIPT</a> in <a 
href="#glo:TCGA">TCGA</a> breast <a 
href="#glo:gene expression">expression</a> data and the <a 
href="#glo:siRNA">siRNA</a> primary screen by over-representation (in Section <a 
href="#x1-1060005">4.2.5<!--tex4ht:ref: chapt3:compare_pathway --></a>) or resampling (in Section <a 
href="#x1-1070001">4.2.5.1<!--tex4ht:ref: chapt3:compare_pathway_perm --></a>) but genes within this <a 
href="#glo:pathway">pathway</a> were detectable by either approach (as shown in Figure <a 
href="#x1-1200011">5.1<!--tex4ht:ref: fig:SL_Pathway_Pi3K --></a>). While few genes were identified by both approaches, they include genes that are highly connected in the PI3K cascade and are hubs to information transmission such as <span class="cmti-12">FGF9</span>,<span class="cmti-12">PDE3B</span>, and <span class="cmti-12">PDE4A</span>. The key upstream genes <span class="cmti-12">PIK3CA </span>and <span class="cmti-12">PIK3CG </span>were detected by <a 
href="#glo:siRNA">siRNA</a> whereas the downstream <span class="cmti-12">PIK3R1 </span>and <span class="cmti-12">AKT2 </span>genes were detected by <a 
href="#glo:SLIPT">SLIPT</a>. Genes detected by either method were also prevalent in the <a 
href="#glo:PI3K">PI3K</a>, <a 
href="#glo:PDE">phosphodiesterase (PDE)</a>, and <a 
href="#glo:AMPK"><a 
href="#glo:AMP">AMP</a>-activated protein kinase (AMPK)</a> modules, in addition to the downstream translation factors and ribosomal genes (<span class="cmti-12">EIF4B</span>, <span class="cmti-12">EEF2K</span>, and <span class="cmti-12">RPS6</span>). Together these suggest that there may be further structure between the <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidate partners of <span class="cmti-12">CDH1 </span>in <a 
href="#glo:pathway">pathways</a> as illustrated by <a 
href="#glo:PI3K">PI3K</a>. As such, <a 
href="#glo:graph">pathway</a> structure will be investigated to detect differences in the upstream and downstream gene candidates detected by either method. Pathway structure may account for the disparity between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> genes, even in <a 
href="#glo:pathway">pathways</a> such as PI3K where they did not significantly intersect. For instance, <a 
href="#glo:SLIPT">SLIPT</a> gene partners may be downstream of <a 
href="#glo:siRNA">siRNA</a> candidates rather than replicating them directly.
</p><!--l. 50--><p class="indent" >
This disparity between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of CDH1(i.e., a high number of genes detected by either approach with few detected by both) was replicated in the related PI3K/AKT <a 
href="#glo:pathway">pathway</a> and the “PI3K/AKT in cancer” <a 
href="#glo:pathway">pathway</a> (shown in Appendix Figures <a 
href="#x1-1860011">G.1<!--tex4ht:ref: fig:SL_Pathway_Pi3KAkt --></a> and <a 
href="#x1-1860022">G.2<!--tex4ht:ref: fig:SL_Pathway_Pi3KAktCancer --></a>). Many <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates were at the upstream core of these <a 
href="#glo:pathway">pathway</a> networks and the downstream extremities. It is particularly notable that many genes important in cell signalling and gene regulation were detected by either <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection approach. These include <span class="cmti-12">AKT1</span>, <span class="cmti-12">AKT2</span>, and <span class="cmti-12">AKT3</span>, the Calmodulin signalling genes <span class="cmti-12">CALM1 </span>and <span class="cmti-12">CAMK4</span>, and the forkhead family transcription factors <span class="cmti-12">FOXO1 </span>(a <a 
href="#glo:tumour suppressor">tumour suppressor</a>) and <span class="cmti-12">FOXO4 </span>(an inhibitor of <a 
href="#glo:EMT">EMT</a>). <a 
 id="x1-120002r226"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">5.1.2 </span> <a 
 id="x1-1210002"></a>The Extracellular Matrix
</h4>
<!--l. 57--><p class="noindent" >
<p>The extracellular <a 
href="#glo:pathway">pathways</a> “elastic fibre formation” and “fibrin clot formation” (shown in Figures <a 
href="#x1-1210012">5.2<!--tex4ht:ref: fig:SL_Pathway_ElasticFibre --></a> and <a 
href="#x1-1210023">5.3<!--tex4ht:ref: fig:SL_Pathway_FibrinFormation --></a> respectively) were both supported across analyses (in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>). These <a 
href="#glo:pathway">pathways</a> were identified by both <a 
href="#glo:SLIPT">SLIPT</a> (for <a 
href="#glo:TCGA">TCGA</a> breast cancer) and</p>
<p><a 
href="#glo:siRNA">siRNA</a> gene candidates as they had significant over-representation and resampling analyses.</p>
</p><!--l. 59--><p class="indent" >
<a 
 id="x1-1210012"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 61--><p class="noindent" >
</p><!--l. 62--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_ElasticFibre_exprSL2".png" alt="pict"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 5.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality in Elastic Fibre Formation.</span></strong> <span class="cmr-10x-x-109">The Reactome Elastic</span> <span class="cmr-10x-x-109">Fibre Formation </span><a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathway</span></a> <span class="cmr-10x-x-109">with </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">candidates coloured as shown in the legend.</span> </span></p>
</div>
<!--tex4ht:label?: x1-1210012 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 75--><p class="indent" >
Particularly for elastic fibres (Figure <a 
href="#x1-1210012">5.2<!--tex4ht:ref: fig:SL_Pathway_ElasticFibre --></a>), the vast majority of genes were detected by either approach, in addition to a significant proportion of genes detected by both approaches (as determined in Section <a 
href="#x1-1060005">4.2.5<!--tex4ht:ref: chapt3:compare_pathway --></a>). The genes detected by both approaches also appeared to have a non-random distribution in the network, with <span class="cmti-12">TFGB1</span>, <span class="cmti-12">ITGB8</span>, and <span class="cmti-12">MFAP2 </span>exhibiting high connectivity, and having a central role in their respective <a 
href="#glo:pathway">pathway</a> modules. In addition to a structural role in the extracellular matrix and connective tissue (including the tumour microenvironment), these proteins including Furin, <a 
href="#glo:TGFB">transforming growth factor <span class="cmmi-12">β </span>(TGF<span class="cmmi-12">β</span>)</a>, and the <a 
href="#glo:BMP">bone morphogenic proteins (BMPs)</a>, are also involved in responses to endocrine signals and interact with the cellular receptors for signalling <a 
href="#glo:pathway">pathways</a>. Therefore it is plausible that <span class="cmti-12">CDH1 </span>deficient tumours will be subject to <a 
href="#glo:non-oncogene addiction">non-oncogene addiction</a> to the extracellular environment and growth signals arising from this <a 
href="#glo:pathway">pathway</a>. The <a 
href="#glo:graph">pathway</a> structure also indicated that the genes detected by <a 
href="#glo:siRNA">siRNA</a> (or by both approaches) may be be downstream of those detected by <a 
href="#glo:SLIPT">SLIPT</a>, in addition to whether connectivity or <a 
href="#glo:centrality">centrality</a> is higher for <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates than other genes in the <a 
href="#glo:pathway">pathway</a>.
</p><!--l. 77--><p class="indent" >
<p>Genes detected as <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">CDH1 </span>by <a 
href="#glo:SLIPT">SLIPT</a> or <a 
href="#glo:siRNA">siRNA</a> screening were also common in the Fibrin clot formation <a 
href="#glo:pathway">pathway</a> (shown in Figure <a 
href="#x1-1210023">5.3<!--tex4ht:ref: fig:SL_Pathway_FibrinFormation --></a>). This is consistent with the established pleiotropic role of <span class="cmti-12">CDH1 </span>in regulating fibrin clotting. It is also notable that the genes detected by either method appear to be highly connected such as <span class="cmti-12">C1QBP</span> <span class="cmti-12">KNG1</span>, <span class="cmti-12">F8</span>, <span class="cmti-12">F10</span>, <span class="cmti-12">F12</span>, <span class="cmti-12">F13A</span>, and <span class="cmti-12">PROC </span>(including many of the coagulation factors). <a 
href="#glo:synthetic lethal">Synthetic lethal</a> candidates also include <span class="cmti-12">SERPINE2 </span>and <span class="cmti-12">PRCP</span>, which only affect downstream genes, in addition to <span class="cmti-12">PROCR </span>and <span class="cmti-12">VWF</span>, which are only affected by upstream genes.</p>
</p><!--l. 79--><p class="indent" >
<a 
 id="x1-1210023"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 81--><p class="noindent" >
</p><!--l. 82--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_FibrinFormation_exprSL2".png" alt="pict"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 5.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality in Fibrin Clot Formation.</span></strong> <span class="cmr-10x-x-109">The Reactome Fibrin Clot</span> <span class="cmr-10x-x-109">Formation </span><a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathway</span></a> <span class="cmr-10x-x-109">with </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">candidates coloured as shown in the legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-1210023 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 96--><p class="indent" >
Many of these genes are involved in the larger Extracellular Matrix <a 
href="#glo:pathway">pathway</a> (shown in Appendix Figure <a 
href="#x1-1860033">G.3<!--tex4ht:ref: fig:SL_Pathway_ExtracellularMatrix --></a>), including many of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates discussed for elastic fibres. The number of <a 
href="#glo:SLIPT">SLIPT</a> candidate genes outnumbers those identified by <a 
href="#glo:siRNA">siRNA</a>, as expected from an isolated cell model. However, the endocrine response genes (e.g., <span class="cmti-12">TGFB1</span> and <span class="cmti-12">LTBP4 </span>) which are potentially artifacts of the cell line growth process were replicated with <a 
href="#glo:SLIPT">SLIPT</a> analysis in patient tumours (TCGA breast cancer data). There is also additional support for <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes (e.g., <span class="cmti-12">ITGB2</span>, <span class="cmti-12">MFAP2</span>, and <span class="cmti-12">SPARC </span>) being highly connected networks hubs of the <a 
href="#glo:pathway">pathway</a>. The complexity of the extracellular matrix <a 
href="#glo:pathway">pathway</a> lends credence to the need for formal network analysis approaches to interpret the <a 
href="#glo:graph">pathway</a> structure of <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates. Furthermore, statistical approaches are needed to determine whether the apparent structural relationships between <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates could have occurred by chance <a 
 id="x1-121003r228"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">5.1.3 </span> <a 
 id="x1-1220003"></a>G Protein Coupled Receptors
</h4>
<!--l. 102--><p class="noindent" >
<p><a 
href="#glo:GPCR">G protein coupled receptor</a> (<a 
href="#glo:GPCR">GPCR</a>) <a 
href="#glo:pathway">pathways</a> are highly complex (as shown in Figure <a 
href="#x1-1220014">5.4<!--tex4ht:ref: fig:SL_Pathway_GPCR --></a> and Appendix Figure <a 
href="#x1-1860044">G.4<!--tex4ht:ref: fig:SL_Pathway_GPCR_Downstream --></a>). Many of genes in these <a 
href="#glo:pathway">pathways</a> were <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates, detected by either <a 
href="#glo:SLIPT">SLIPT</a> or <a 
href="#glo:siRNA">siRNA</a> screening, including genes frequently detected by both approaches, consistent with these <a 
href="#glo:pathway">pathways</a> being supported by prior analyses (in Sections <a 
href="#x1-1060005">4.2.5<!--tex4ht:ref: chapt3:compare_pathway --></a> and <a 
href="#x1-1070001">4.2.5.1<!--tex4ht:ref: chapt3:compare_pathway_perm --></a>). <a 
href="#glo:synthetic lethal">Synthetic lethal</a> candidates include the <a 
href="#glo:PDE">PDE</a> and Calmodulin genes (as discussed in Section <a 
href="#x1-1220003">5.1.3<!--tex4ht:ref: chapt4:SL_Genes_GPCR --></a>) in addition to others such as the regulators of <a 
href="#glo:RGS">G-protein signalling (RGS)</a>, <a 
href="#glo:CXCR">chemokine receptors (CXCR)</a>, <a 
href="#glo:JAK">Janus kinase</a> (<a 
href="#glo:JAK">JAK</a>), and the <a 
href="#glo:RHO">Ras homolog family (RHO)</a> genes. These are important regulatory signalling <a 
href="#glo:pathway">pathways</a> necessary for cellular growth and cancer proliferation. Thus the <a 
href="#glo:GPCR">GPCR</a> <a 
href="#glo:pathway">pathways</a> (and downstream PI3K/AKT signals) are a potentially actionable vulnerability against <span class="cmti-12">CDH1 </span>deficient cancers, particularly since many existing drug targets are in these signalling <a 
href="#glo:pathway">pathways</a>, some of which have been experimentally validated (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). While statistically significant numbers of genes in GCPR <a 
href="#glo:pathway">pathways</a> were detected by both approaches (in Sections <a 
href="#x1-1060005">4.2.5<!--tex4ht:ref: chapt3:compare_pathway --></a> and <a 
href="#x1-1070001">4.2.5.1<!--tex4ht:ref: chapt3:compare_pathway_perm --></a>), the complexity of <a 
href="#glo:GPCR">GPCR</a> networks (containing hundreds of genes) further support the needs for a rational network-based approach to the relationships between <a 
href="#glo:SLIPT">SLIPT</a> and experimental candidates.</p>
</p><!--l. 104--><p class="indent" >
<a 
 id="x1-1220014"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 106--><p class="noindent" >
</p><!--l. 107--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_GPCR_exprSL2".png" alt="PIC"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 5.4: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality in the GPCRs.</span></strong> <span class="cmr-10x-x-109">The Reactome G</span><sub><span class="cmmi-8">αi</span></sub> <a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathway</span></a> <span class="cmr-10x-x-109">with</span> <a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">candidates, coloured as shown in the legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-1220014 -->
                                                                                    

</div><hr class="endfloat" />
<a 
 id="x1-122002r231"></a>
<h4 class="subsectionHead">
<span class="titlemark">5.1.4 </span> <a 
 id="x1-1230004"></a>Gene Regulation and Translation
</h4>
<!--l. 126--><p class="noindent" >
While very few <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes were detected in translational <a 
href="#glo:pathway">pathways</a> in an experimental screen against <span class="cmti-12">CDH1 </span>(<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), these were highly over-represented in translational elongation (as shown in Appendix Figure <a 
href="#x1-1860055">G.5<!--tex4ht:ref: fig:SL_Pathway_TranslationElongation --></a>). These <a 
href="#glo:SLIPT">SLIPT</a> genes include many ribosomal proteins and the regulatory “elongation factors” which may be subject to responses in the upstream signalling <a 
href="#glo:pathway">pathways</a>. This observation further indicates that <a 
href="#glo:graph">pathway</a> structure may be used to identify relationships between <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates detected by <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a>. The computational approach with <a 
href="#glo:SLIPT">SLIPT</a> may exhibit the ability to detect downstream genes in the core translational processes, which experimental screening did not identify. The experimental screening may similarly detect upstream regulatory genes less sensitive to inactivation, that is, genes that are less likely to be indiscriminately lethal to both genotypes at high doses of inactivation.
</p><!--l. 128--><p class="indent" >
<p>Many of these <a 
href="#glo:SLIPT">SLIPT</a> candidate genes are also among the <a 
href="#glo:NMD">nonsense-mediated decay (NMD)</a> <a 
href="#glo:pathway">pathway</a> (shown in Appendix Figure <a 
href="#x1-1860066">G.6<!--tex4ht:ref: fig:SL_Pathway_NMD --></a>) or 3<sup><span class="cmsy-8">′</span></sup> <a 
href="#glo:UTR">untranslated region (UTR)</a> mediated translational regulation (shown in Appendix Figure <a 
href="#x1-1860077">G.7<!--tex4ht:ref: fig:SL_Pathway_Three_prime_UTR --></a>). While genes in these <a 
href="#glo:pathway">pathways</a> were also supported by experimental screening with <a 
href="#glo:siRNA">siRNA</a>, there were differences in which genes were detected within the <a 
href="#glo:graph">pathway</a> structures. In particular, <span class="cmti-12">UPF1 </span>was detected in the <a 
href="#glo:siRNA">siRNA</a> screen and is the focal downstream gene for the entire <a 
href="#glo:NMD">NMD</a> <a 
href="#glo:pathway">pathway</a> showing that (in this case) <a 
href="#glo:siRNA">siRNA</a> genes are downstream effectors of those detected by <a 
href="#glo:SLIPT">SLIPT</a>. 3<sup><span class="cmsy-8">′</span></sup> <a 
href="#glo:UTR">UTR</a> mediated translational regulation has a similar structure with two modules connected solely by <span class="cmti-12">RPL13A</span>, giving an example of <a 
href="#glo:SLIPT">SLIPT</a> candidate genes with high connectivity, although there were many ribosomal proteins detected by <a 
href="#glo:SLIPT">SLIPT</a>. However, the detection of <span class="cmti-12">EIF3K</span>, a regulatory elongation factor (not <a 
href="#glo:essential">essential</a> to ribosomal function) was replicated across <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> screening, while the majority of the elongation factors were not detected by either approach. Regulatory genes, being more amenable to experimental validation, also support further investigation into <a 
href="#glo:graph">pathway</a> structure. The <a 
href="#glo:SLIPT">SLIPT</a> candidates may support experimental candidates in biological <a 
href="#glo:pathway">pathways</a> by detecting downstream genes, which may not be detectable by experimental screening with high dose inhibitors. This difference between the approaches may explain the greater number of <a 
href="#glo:SLIPT">SLIPT</a> candidate partners of <span class="cmti-12">CDH1 </span>than those experimentally identified.</p>
<a 
 id="x1-123001r225"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">5.2 </span> <a 
 id="x1-1240002"></a>Network Analysis of Synthetic Lethal Genes
</h3>
<!--l. 137--><p class="noindent" >
To demonstrate the network properties of <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates in a pathway, a network analsis was performed on the genes detected as <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">CDH1</span> with the <a 
href="#glo:SLIPT">SLIPT</a> computational approach and the <a 
href="#glo:siRNA">siRNA</a> screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) in G<sub><span class="cmmi-8">αi</span></sub> signalling, a <a 
href="#glo:GPCR">GPCR</a> <a 
href="#glo:pathway">pathway</a>. This <a 
href="#glo:pathway">pathway</a> was used to demonstrate deeper network analysis approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> candiates within complex <a 
href="#glo:pathway">pathways</a>, as it was supported across analyses (in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>), with significant over-representation in both <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> screening, and the genes differed considerably between <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection methods (shown in Appendix Figures <a 
href="#x1-1220014">5.4<!--tex4ht:ref: fig:SL_Pathway_GPCR --></a>). These network metrics were used to measure whether the network properties differed between groups of genes detected by either or both approaches. These analyses serve to test both whether <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene candidates had higher connectivity or importance in a network and whether either detection approach is biased towards genes with different network properties. <a 
 id="x1-124001r233"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">5.2.1 </span> <a 
 id="x1-1250001"></a>Gene Connectivity and Vertex Degree
</h4>
<!--l. 148--><p class="noindent" >
<p>Vertex degree (the number of connections) for each gene is a fundamental property of a network. The vast majority of genes had a relatively modest number of connections, each with only a few genes in the G<sub><span class="cmmi-8">αi</span></sub> <a 
href="#glo:pathway">pathway</a> (shown in Figure <a 
href="#x1-1250015">5.5<!--tex4ht:ref: fig:SL_Pathway_GPCR_Vertex_Degree --></a>) having <a 
href="#glo:pathway">pathway</a> relationships with a high number of genes, consistent with the <a 
href="#glo:scale-free">scale-free</a> property of biological networks (<a 
href="#XBarabasi2004">Barabási and Oltvai</a>, <a 
href="#XBarabasi2004">2004</a>). The number of connections was similar between gene groups (by <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection). Genes detected by <a 
href="#glo:siRNA">siRNA</a> included those with the fewest connections, despite there being fewer genes that were detected by either approach. There was no statistically significant effect of either computational or experimental <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection method on <a 
href="#glo:vertex">vertex</a> degree, as determined by <a 
href="#glo:ANOVA">analysis of variance (ANOVA)</a> (shown by Table <a 
href="#x1-1250021">5.1<!--tex4ht:ref: tab:SL_Pathway_GPCR_Vertex_Degree --></a>).</p>
</p><!--l. 152--><p class="indent" >
<a 
 id="x1-1250015"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 154--><p class="noindent" >
</p><!--l. 155--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Centrality_exprSL/GPCR_network_vertex_degree_stripchart2".png" alt="pict"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 5.5: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality and vertex degree.</span></strong> <span class="cmr-10x-x-109">The number of connected genes</span> <span class="cmr-10x-x-109">(</span><a 
href="#glo:vertex degree"><span class="cmr-10x-x-109">vertex degree</span></a><span class="cmr-10x-x-109">) was compared (on a log-scale) across genes detected by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screening in the Reactome G</span><sub><span class="cmmi-8">αi</span></sub> <span class="cmr-10x-x-109">cascade </span><a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathway</span></a><span class="cmr-10x-x-109">. There were no differences in </span><a 
href="#glo:vertex"><span class="cmr-10x-x-109">vertex</span></a> <span class="cmr-10x-x-109">degree</span> <span class="cmr-10x-x-109">between the groups (shown in Table</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-1250021"><span class="cmr-10x-x-109">5.1</span><!--tex4ht:ref: tab:SL_Pathway_GPCR_Vertex_Degree --></a><span class="cmr-10x-x-109">), although genes detected by </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">included those</span> <span class="cmr-10x-x-109">with the fewest connections. </span></span></p>
</div>
<!--tex4ht:label?: x1-1250015 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 168--><p class="indent" >
<a 
 id="x1-1250021"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 5.1: </span><span class="content"><a 
href="#glo:ANOVA">ANOVA</a> for synthetic lethality and vertex degree</span>
</div><!--tex4ht:label?: x1-1250021 -->
<div class="tabular">
<table id="TBL-68" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-68-1g"><col 
id="TBL-68-1" /><col 
id="TBL-68-2" /><col 
id="TBL-68-3" /><col 
id="TBL-68-4" /><col 
id="TBL-68-5" /><col 
id="TBL-68-6" /></colgroup><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-68-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-68-1-1"  
class="td11">              </td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-1-2"  
class="td11"> 
DF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-1-3"  
class="td11"> 
Sum Squares
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-1-4"  
class="td11"> 
Mean Squares
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-1-5"  
class="td11"> 
F-value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-1-6"  
class="td11"> 
p-value
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-68-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-68-2-1"  
class="td11"> 
siRNA
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-2-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-2-3"  
class="td11">     
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-2-4"  
class="td11">     
20.8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-2-5"  
class="td11"> 
0.0030
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-2-6"  
class="td11"> 
0.9561
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-68-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-68-3-1"  
class="td11"> 
SLIPT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-3-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-3-3"  
class="td11">    
16215
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-3-4"  
class="td11">    
16215
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-3-5"  
class="td11"> 
2.3722
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-3-6"  
class="td11"> 
0.1246
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-68-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-68-4-1"  
class="td11"> 
siRNA<span class="cmsy-10x-x-120">×</span>SLIPT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-4-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-4-3"  
class="td11">     
17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-4-4"  
class="td11">      
17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-4-5"  
class="td11"> 
0.0025
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-68-4-6"  
class="td11"> 
0.9603
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-68-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-68-5-1"  
class="td11">               </td></tr></table>
<!--l. 192--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Analysis of variance for </span><a 
href="#glo:vertex"><span class="cmr-10x-x-109">vertex</span></a> <span class="cmr-10x-x-109">degree against </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">detection approaches (with an</span> <span class="cmr-10x-x-109">interaction term)</span></p>
</p>
</div><hr class="endfloat" />
<!--l. 195--><p class="indent" >
The results for the G<sub><span class="cmmi-8">αi</span></sub> <a 
href="#glo:pathway">pathway</a> were very similar when testing <a 
href="#glo:synthetic lethal">synthetic lethality</a> against <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutation</a> (<a 
href="#glo:mtSLIPT">mtSLIPT</a>). In either case, there was no significant evidence that <a 
href="#glo:SLIPT">SLIPT</a> or <a 
href="#glo:mtSLIPT">mtSLIPT</a>-specific genes had higher connectivity than those detected by <a 
href="#glo:siRNA">siRNA</a> screening (shown in Appendix Figure <a 
href="#x1-1870011">H.1<!--tex4ht:ref: fig:mtSL_Pathway_GPCR_Vertex_Degree --></a> and Appendix Table <a 
href="#x1-1870041">H.1<!--tex4ht:ref: tab:mtSL_Pathway_GPCR_Vertex_Degree --></a>). Thus <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection does not discriminate among genes by their connectivity in this <a 
href="#glo:pathway">pathway</a> network, nor is either approach constrained to detecting highly connected genes. Both approaches have been demonstrated to detect genes with many and very few connections in the G<sub><span class="cmmi-8">αi</span></sub> signalling <a 
href="#glo:pathway">pathway</a>. <a 
 id="x1-125003r235"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">5.2.2 </span> <a 
 id="x1-1260002"></a>Gene Importance and Centrality
</h4>
<a 
 id="x1-126001r222"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">5.2.2.1 </span> <a 
 id="x1-1270001"></a>Information Centrality
</h5>
<!--l. 204--><p class="noindent" >
<a 
href="#glo:information centrality">Information centrality</a> is a measure of the importance of <a 
href="#glo:vertex">nodes</a> in a network in terms of how vital they are to the transmission of information throughout the network. This applies well to biological <a 
href="#glo:pathway">pathways</a>, partcularly gene regulation and cell signalling. The <a 
href="#glo:vertex">nodes</a> with the highest <a 
href="#glo:information centrality">information centrality</a> are not necessarily the most connected, as they may also include <a 
href="#glo:vertex">nodes</a> that pass signals between highly connected network hubs. <a 
href="#glo:information centrality">Information centrality</a> therefore provides a distinct metric for the connectivity of a gene in a <a 
href="#glo:pathway">pathway</a>, which has the added benefit of being directly related to the disruption of <a 
href="#glo:pathway">pathway</a> function were it to be inactivated or removed. <a 
href="#glo:information centrality">Information centrality</a> has also been suggested to be indicative of the <a 
href="#glo:essential">essentiality</a> of genes or proteins (<a 
href="#XKranthi2013">Kranthi <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XKranthi2013">2013</a>).
</p><!--l. 210--><p class="indent" >
<p>Within the G<sub><span class="cmmi-8">αi</span></sub> <a 
href="#glo:pathway">pathway</a> (shown in Figure <a 
href="#x1-1270016">5.6<!--tex4ht:ref: fig:SL_Pathway_GPCR_InfoCent --></a>), the <a 
href="#glo:information centrality">information centrality</a> across gene groups detected by either <a 
href="#glo:synthetic lethal">synthetic lethal</a> approach did not differ significantly (shown by Table <a 
href="#x1-1270022">5.2<!--tex4ht:ref: tab:SL_Pathway_GPCR_InfoCent --></a>). Genes detected by <a 
href="#glo:SLIPT">SLIPT</a> span the complete range of <a 
href="#glo:PageRank centrality">PageRank centrality</a> values for this <a 
href="#glo:pathway">pathway</a>. These findings were replicated (shown in Appendix Figure <a 
href="#x1-1870022">H.2<!--tex4ht:ref: fig:mtSL_Pathway_GPCR_InfoCent --></a> and Appendix Table <a 
href="#x1-1870052">H.2<!--tex4ht:ref: tab:mtSL_Pathway_GPCR_InfoCent --></a>). Thus neither method was unable to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes in the G<sub><span class="cmmi-8">αi</span></sub> <a 
href="#glo:pathway">pathway</a> with particular <a 
href="#glo:centrality">centrality</a> constraints but they were also not detecting genes with higher <a 
href="#glo:centrality">centrality</a> than expected by chance.</p>
</p><!--l. 216--><p class="indent" >
<a 
 id="x1-1270016"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 218--><p class="noindent" >
</p><!--l. 219--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Centrality_exprSL/GPCR_network_Info_Centrality(Log)_stripchart2".png" alt="pict"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 5.6: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality and centrality.</span></strong> <span class="cmr-10x-x-109">The </span><a 
href="#glo:information centrality"><span class="cmr-10x-x-109">information centrality</span></a> <span class="cmr-10x-x-109">was compared</span> <span class="cmr-10x-x-109">(on a log-scale) across genes detected by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screening in the Reactome G</span><sub><span class="cmmi-8">αi</span></sub> <a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathway</span></a><span class="cmr-10x-x-109">. Genes detected by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">or </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">did not have higher </span><a 
href="#glo:centrality"><span class="cmr-10x-x-109">centrality</span></a> <span class="cmr-10x-x-109">than other genes</span> <span class="cmr-10x-x-109">(shown in Table</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-1270022"><span class="cmr-10x-x-109">5.2</span><!--tex4ht:ref: tab:SL_Pathway_GPCR_InfoCent --></a><span class="cmr-10x-x-109">). Genes detected by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">spanned the range of </span><a 
href="#glo:centrality"><span class="cmr-10x-x-109">centrality</span></a> <span class="cmr-10x-x-109">values. </span></span></p>
</div>
<!--tex4ht:label?: x1-1270016 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 233--><p class="indent" >
<a 
 id="x1-1270022"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 5.2: </span><span class="content"><a 
href="#glo:ANOVA">ANOVA</a> for synthetic lethality and information centrality</span>
</div><!--tex4ht:label?: x1-1270022 -->
<div class="tabular">
<table id="TBL-69" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-69-1g"><col 
id="TBL-69-1" /><col 
id="TBL-69-2" /><col 
id="TBL-69-3" /><col 
id="TBL-69-4" /><col 
id="TBL-69-5" /><col 
id="TBL-69-6" /></colgroup><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-69-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-69-1-1"  
class="td11">              </td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-1-2"  
class="td11"> 
DF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-1-3"  
class="td11"> 
Sum Squares
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-1-4"  
class="td11"> 
Mean Squares
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-1-5"  
class="td11"> 
F-value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-1-6"  
class="td11"> 
p-value
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-69-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-69-2-1"  
class="td11"> 
siRNA
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-2-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-2-3"  
class="td11">  
0.00000000
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-2-4"  
class="td11"> 
2<span class="cmmi-12">.</span>7000 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-2-5"  
class="td11"> 
0.0016
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-2-6"  
class="td11"> 
0.96783
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-69-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-69-3-1"  
class="td11"> 
SLIPT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-3-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-3-3"  
class="td11">  
0.00000548
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-3-4"  
class="td11"> 
5<span class="cmmi-12">.</span>4831 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-3-5"  
class="td11"> 
3.3253
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-3-6"  
class="td11"> 
0.06926
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-69-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-69-4-1"  
class="td11"> 
siRNA<span class="cmsy-10x-x-120">×</span>SLIPT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-4-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-4-3"  
class="td11">  
0.00000002
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-4-4"  
class="td11"> 
1<span class="cmmi-12">.</span>8800 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-4-5"  
class="td11"> 
0.0114
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-69-4-6"  
class="td11"> 
0.91511
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-69-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-69-5-1"  
class="td11">               </td></tr></table>
<!--l. 257--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Analysis of variance for </span><a 
href="#glo:information centrality"><span class="cmr-10x-x-109">information centrality</span></a> <span class="cmr-10x-x-109">against </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">detection approaches</span> <span class="cmr-10x-x-109">(with an interaction term)</span></p>
</p>
</div><hr class="endfloat" />
<a 
 id="x1-127003r239"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">5.2.2.2 </span> <a 
 id="x1-1280002"></a>PageRank Centrality
</h5>
<!--l. 267--><p class="noindent" >
<a 
href="#glo:PageRank centrality">PageRank centrality</a> is another network analysis procedure to infer a hierarchy of gene importance from a network using connections and structure (<a 
href="#XBrin1998">Brin and Page</a>, <a 
href="#XBrin1998">1998</a>). In contrast to the <a 
href="#glo:information centrality">information centrality</a> approach of removing <a 
href="#glo:vertex">nodes</a>, PageRank uses the eigenvalue properties of the adjacency matrix to rank genes according to the number of connections and paths they are involved in.
</p><!--l. 269--><p class="indent" >
This distinction is immediately clear within the G<sub><span class="cmmi-8">αi</span></sub> <a 
href="#glo:pathway">pathway</a> (shown in Figure <a 
href="#x1-1280017">5.7<!--tex4ht:ref: fig:SL_Pathway_GPCR_PageRank --></a>), which differs considerably from the <a 
href="#glo:information centrality">information centrality</a> scores (as shown in Figure <a 
href="#x1-1270016">5.6<!--tex4ht:ref: fig:SL_Pathway_GPCR_InfoCent --></a>). Genes detected by either <a 
href="#glo:synthetic lethal">synthetic lethal</a> approach did not include those with the highest PageRank centrality. There was a significant association between genes detected by <a 
href="#glo:SLIPT">SLIPT</a> (which had a lower median) with PageRank <a 
href="#glo:centrality">centrality</a> (shown by Table <a 
href="#x1-1280023">5.3<!--tex4ht:ref: tab:SL_Pathway_GPCR_PageRank --></a>).
</p><!--l. 273--><p class="indent" >
<p>The genes detected by <a 
href="#glo:SLIPT">SLIPT</a> span the range of <a 
href="#glo:centrality">centrality</a> values of <a 
href="#glo:siRNA">siRNA</a> showing that both approaches were capable of detecting genes of moderately high <a 
href="#glo:centrality">centrality</a> (as shown for information centrality) and that the lower <a 
href="#glo:centrality">centrality</a> of <a 
href="#glo:SLIPT">SLIPT</a> candidates in the G<sub><span class="cmmi-8">αi</span> </sub> <a 
href="#glo:pathway">pathway</a> may be due to <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners being less critical to the <a 
href="#glo:pathway">pathway</a>, rather than a limitation of the methodology. While it is expected that some <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes will be important to the function of the <a 
href="#glo:pathway">pathway</a>, it is possible that genes with high <a 
href="#glo:centrality">centrality</a> were avoided if they are <a 
href="#glo:essential">essential</a> to cellular viability.</p>
</p><!--l. 276--><p class="indent" >
<a 
 id="x1-1280017"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 278--><p class="noindent" >
</p><!--l. 279--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Centrality_exprSL/GPCR_network_pagerank_stripchart2".png" alt="pict"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 5.7: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality and PageRank.</span></strong> <span class="cmr-10x-x-109">The </span><a 
href="#glo:PageRank centrality"><span class="cmr-10x-x-109">PageRank centrality</span></a> <span class="cmr-10x-x-109">was compared</span> <span class="cmr-10x-x-109">(on a log-scale) across genes detected by </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screening in the Reactome</span> <span class="cmr-10x-x-109">G</span><sub><span class="cmmi-8">αi</span> </sub> <a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathway</span></a><span class="cmr-10x-x-109">. Genes detected by with either </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">detection approach had a more</span> <span class="cmr-10x-x-109">restricted range of </span><a 
href="#glo:centrality"><span class="cmr-10x-x-109">centrality</span></a> <span class="cmr-10x-x-109">values but only </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">genes had a significant association with</span> <a 
href="#glo:centrality"><span class="cmr-10x-x-109">centrality</span></a> <span class="cmr-10x-x-109">(shown in Table</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-1280023"><span class="cmr-10x-x-109">5.3</span><!--tex4ht:ref: tab:SL_Pathway_GPCR_PageRank --></a><span class="cmr-10x-x-109">). </span></span></p>
</div>
<!--tex4ht:label?: x1-1280017 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 293--><p class="indent" >
<p>There was not a significant association between <a 
href="#glo:siRNA">siRNA</a> candidates and PageRank centrality. The significant result for <a 
href="#glo:SLIPT">SLIPT</a> was not replicated when testing <a 
href="#glo:synthetic lethal">synthetic lethality</a> against <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutation</a> (shown in Appendix Figure <a 
href="#x1-1870033">H.3<!--tex4ht:ref: fig:mtSL_Pathway_GPCR_PageRank --></a> and Appendix Table <a 
href="#x1-1870063">H.3<!--tex4ht:ref: tab:mtSL_Pathway_GPCR_PageRank --></a>). However, this may be due to fewer genes being detected by <a 
href="#glo:mtSLIPT">mtSLIPT</a> and <a 
href="#glo:siRNA">siRNA</a>.</p>
</p><!--l. 295--><p class="indent" >
<a 
 id="x1-1280023"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 5.3: </span><span class="content"><a 
href="#glo:ANOVA">ANOVA</a> for synthetic lethality and PageRank centrality</span>
</div><!--tex4ht:label?: x1-1280023 -->
<div class="tabular">
<table id="TBL-70" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-70-1g"><col 
id="TBL-70-1" /><col 
id="TBL-70-2" /><col 
id="TBL-70-3" /><col 
id="TBL-70-4" /><col 
id="TBL-70-5" /><col 
id="TBL-70-6" /></colgroup><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-70-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-70-1-1"  
class="td11">              </td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-1-2"  
class="td11"> 
DF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-1-3"  
class="td11"> 
Sum Squares
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-1-4"  
class="td11"> 
Mean Squares
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-1-5"  
class="td11"> 
F-value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-1-6"  
class="td11"> 
p-value
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-70-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-70-2-1"  
class="td11"> 
siRNA
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-2-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-2-3"  
class="td11">  
0.0001059
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-2-4"  
class="td11"> 
1<span class="cmmi-12">.</span>0589 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-2-5"  
class="td11"> 
2.1021
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-2-6"  
class="td11"> 
0.14818
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-70-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-70-3-1"  
class="td11"> 
SLIPT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-3-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-3-3"  
class="td11">  
0.0002881
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-3-4"  
class="td11"> 
2<span class="cmmi-12">.</span>8808 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-3-5"  
class="td11"> 
5.7188
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-3-6"  
class="td11"> 
0.01743
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-70-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-70-4-1"  
class="td11"> 
siRNA<span class="cmsy-10x-x-120">×</span>SLIPT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-4-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-4-3"  
class="td11">  
0.0000477
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-4-4"  
class="td11"> 
4<span class="cmmi-12">.</span>7704 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-4-5"  
class="td11"> 
0.9470
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-70-4-6"  
class="td11"> 
0.33131
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-70-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-70-5-1"  
class="td11">               </td></tr></table>
<!--l. 319--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Analysis of variance for </span><a 
href="#glo:PageRank centrality"><span class="cmr-10x-x-109">PageRank centrality</span></a> <span class="cmr-10x-x-109">against </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">detection approaches</span> <span class="cmr-10x-x-109">(with an interaction term)</span></p>
</p>
</div><hr class="endfloat" />
<a 
 id="x1-128003r234"></a>
<h3 class="sectionHead">
<span class="titlemark">5.3 </span> <a 
 id="x1-1290003"></a>Relationships between Synthetic Lethal Genes
</h3>
<!--l. 455--><p class="noindent" >
The network analyses so far have tested whether <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate genes were more connected or important within a <a 
href="#glo:graph">pathway</a> structure, such as the G<sub><span class="cmmi-8">αi</span></sub> <a 
href="#glo:pathway">pathway</a>. However these metrics do not ascertain whether there were relationships between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidate partners of <span class="cmti-12">CDH1</span>. In particular, it is plausible that they may be upstream or downstream of one and other within a <a 
href="#glo:pathway">pathway</a>.
</p><!--l. 457--><p class="indent" >
The direction of a biological <a 
href="#glo:pathway">pathway</a> is important, particularly those involved in cell signalling which respond to extracellular stimuli and transmit these signals via intermediary proteins to regulate core functions and responses of the cell. These <a 
href="#glo:pathway">pathways</a> regulate process such as <a 
href="#glo:gene expression">gene expression</a> and protein translation, which are important in the proliferation of cancers (<a 
href="#XGao2015">Gao and Roux</a>, <a 
href="#XGao2015">2015</a>). Therefore it is important to determine which <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates were upstream or downstream in the context of a biological <a 
href="#glo:pathway">pathway</a>. In particular, <a 
href="#glo:graph">pathway</a> structure may be used to identify relationships between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates.
</p><!--l. 460--><p class="indent" >
A <a 
href="#glo:graph">pathway</a> structure method was devised to use <a 
href="#glo:graph">network</a> structures to identify directional relationships between individual <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> genes. This <a 
href="#glo:graph">pathway</a> structure methodology was applied (as described in Section <a 
href="#x1-840001">3.4.1<!--tex4ht:ref: methods:pathway_str --></a>) to detect the direction of <a 
href="#glo:shortest path">shortest paths</a> between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates. This is used to demonstrate the methodology on the <a 
href="#glo:PI3K">PI3K</a> and G<sub><span class="cmmi-8">αi</span></sub> <a 
href="#glo:pathway">pathways</a>, to develop a statistical test for <a 
href="#glo:graph">pathway</a> structure between between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates using resampling (as described in Section <a 
href="#x1-850001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>), and to apply this test for <a 
href="#glo:graph">pathway</a> structure among <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene candidates to the <a 
href="#glo:pathway">pathways</a> identified in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a> and discussed in Section <a 
href="#x1-1190001">5.1<!--tex4ht:ref: chapt4:SL_Genes --></a>. <a 
 id="x1-129001r238"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">5.3.1 </span> <a 
 id="x1-1300001"></a>Detecting Upstream or Downstream Synthetic Lethality
</h4>
<!--l. 469--><p class="noindent" >
<p>Shortest paths in a <a 
href="#glo:pathway">pathway</a> network were used to devise a strategy to detect <a 
href="#glo:graph">pathway</a> structure between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates partners of <span class="cmti-12">CDH1 </span>(as described in Section <a 
href="#x1-840001">3.4.1<!--tex4ht:ref: methods:pathway_str --></a>). Thus we can determine whether individual <a 
href="#glo:SLIPT">SLIPT</a> genes have upstream or downstream <a 
href="#glo:siRNA">siRNA</a> candidates (scored as “up” or “down” events respectively). This procedure enables the detection of directional relationships between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene</p>
<p>candidates (e.g., if genes detected by <a 
href="#glo:siRNA">siRNA</a> are more likely to be downstream of genes detected by <a 
href="#glo:SLIPT">SLIPT</a> in the same pathway).</p>
</p><!--l. 472--><p class="indent" >
<a 
 id="x1-1300039"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 474--><p class="noindent" >
</p><!--l. 478--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/test_GPCR_exprSL".png" alt="pict"  
 /><a 
 id="x1-130001r1"></a> <span class="cmr-10x-x-109">Resampling</span> <span class="cmr-10x-x-109">in</span> <span class="cmr-10x-x-109">G</span><sub><span class="cmmi-8">αi</span> </sub> <span class="cmr-10x-x-109">signalling</span><img 
src=""/home/tomkelly/Downloads/Pathway_Structure/test_PI3K_exprSL".png" alt="pict"  
 /><a 
 id="x1-130002r2"></a> <span class="cmr-10x-x-109">Resampling</span> <span class="cmr-10x-x-109">in the</span> <span class="cmr-10x-x-109">PI3K</span> <span class="cmr-10x-x-109">cascade</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 5.9: </span><span class="content"><strong><span class="cmbx-10x-x-109">Structure of synthetic lethality resampling.</span></strong> <span class="cmr-10x-x-109">A null distribution with</span> <span class="cmr-10x-x-109">10,000 iterations of the number of </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes upstream or downstream of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">genes</span> <span class="cmr-10x-x-109">(depicted as the difference of these) in each </span><a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathway</span></a><span class="cmr-10x-x-109">. To assess significance, the observed</span> <span class="cmr-10x-x-109">events (with </span><a 
href="#glo:shortest path"><span class="cmr-10x-x-109">shortest paths</span></a><span class="cmr-10x-x-109">) were compared to the 90% and 95% intervals for the null</span> <span class="cmr-10x-x-109">distribution (shown in blue). Genes detected by both methods were not fixed to the same</span> <span class="cmr-10x-x-109">number as observed for the alternative null distribution (shown in red), although the</span> <span class="cmr-10x-x-109">significance of the observed number of events (red) was changed in either case. The genes</span> <span class="cmr-10x-x-109">detected by both approaches were included in computing the number of </span><a 
href="#glo:shortest path"><span class="cmr-10x-x-109">shortest paths</span></a> <span class="cmr-10x-x-109">(in</span> <span class="cmr-10x-x-109">either direction) between </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes. The permutations show (a) a significant</span> <a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathway</span></a> <span class="cmr-10x-x-109">relationship for G</span><sub><span class="cmmi-8">αi</span></sub> <span class="cmr-10x-x-109">signalling and (b) and non-significant relationship for the </span><a 
href="#glo:PI3K"><span class="cmr-10x-x-109">PI3K</span></a> <span class="cmr-10x-x-109">cascade. </span></span></p>
</div>
<!--tex4ht:label?: x1-1300039 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 489--><p class="indent" >
The total number of gene candidate pairs in either direction can be compared within a <a 
href="#glo:pathway">pathway</a> network to assess the overall directional relationships in a <a 
href="#glo:pathway">pathway</a>. This directionality is detectable by the difference between the number of <a 
href="#glo:SLIPT">SLIPT</a> candidate genes with upstream and downstream <a 
href="#glo:siRNA">siRNA</a> gene partners. However, this measure alone is not sufficient to determine whether there is evidence of <a 
href="#glo:graph">pathway</a> structure between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates partners of <span class="cmti-12">CDH1 </span>in a <a 
href="#glo:pathway">pathway</a> network. Nevertheless, it does serve to measure the magnitude (and direction) of the consensus of directional relationships (upstream and downstream) between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates partners. This measure of <a 
href="#glo:graph">pathway</a> structure can be used for testing for statistical significance of <a 
href="#glo:graph">pathway</a> structure by resampling, using a permutation procedure to test whether these relationships are detectable among randomly selected gene groups rather than the detected <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates partners (as described in Sections <a 
href="#x1-630006">2.3.6<!--tex4ht:ref: methods:permutation --></a> and <a 
href="#x1-850001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>).
</p><!--l. 491--><p class="indent" >
This resampling procedure was performed for the G<sub><span class="cmmi-8">αi</span></sub> and <a 
href="#glo:PI3K">PI3K</a> <a 
href="#glo:pathway">pathways</a> to generate a null distribution for the difference in the number of “up events” and “down events” for these <a 
href="#glo:pathway">pathway</a> structures (as shown in Figures <a 
href="#x1-1200011">5.1<!--tex4ht:ref: fig:SL_Pathway_Pi3K --></a> and <a 
href="#x1-1220014">5.4<!--tex4ht:ref: fig:SL_Pathway_GPCR --></a>). The resulting null distributions (as shown in Figure <a 
href="#x1-1300039">5.9<!--tex4ht:ref: fig:SL_Pathway_Perm --></a>) were used to detect whether genes detected by <a 
href="#glo:SLIPT">SLIPT</a> had significantly more upstream or downstream <a 
href="#glo:siRNA">siRNA</a> candidates in either <a 
href="#glo:pathway">pathway</a>. It can be seen that <a 
href="#glo:siRNA">siRNA</a> genes were significantly downstream of <a 
href="#glo:SLIPT">SLIPT</a> candidate genes by resampling for the G<sub><span class="cmmi-8">αi</span></sub> signalling <a 
href="#glo:pathway">pathway</a> (Figure <a 
href="#x1-130001r1">5.8a<!--tex4ht:ref: fig:SL_Pathway_GPCR_Perm --></a>). This demonstrates that <a 
href="#glo:pathway">pathway</a> relationships can be detected between <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates by this procedure and that <a 
href="#glo:siRNA">siRNA</a> genes were downstream of gene detected by <a 
href="#glo:SLIPT">SLIPT</a> in an example of GPCR signalling expanding on support for <a 
href="#glo:synthetic lethal">synthetic lethality</a> in this <a 
href="#glo:pathway">pathway</a> (as shown in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>). These structural relationships may also account for why each the computational and experimental approaches did not detect many of the same specific genes because they are detecting different parts of the <a 
href="#glo:pathway">pathway</a>.
</p><!--l. 493--><p class="indent" >
<p>In contrast, there was not significant evidence of such <a 
href="#glo:pathway">pathway</a> structure between <a 
href="#glo:siRNA">siRNA</a> and <a 
href="#glo:SLIPT">SLIPT</a> candidate genes when resampling within the <a 
href="#glo:PI3K">PI3K</a> cascade <a 
href="#glo:pathway">pathway</a> (as shown in Figure <a 
href="#x1-130002r2">5.8b<!--tex4ht:ref: fig:SL_Pathway_PI3K_Perm --></a>). This indicates that such relationships may be <a 
href="#glo:pathway">pathway</a> specific rather than a general property of these <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection methods. These results were robustly reproducible, with similar findings (as shown in Appendix Figure <a 
href="#x1-1880032">I.2<!--tex4ht:ref: fig:mtSL_Pathway_Perm --></a>) for each <a 
href="#glo:pathway">pathway</a> when testing for <a 
href="#glo:synthetic lethal">synthetic lethality</a> against <span class="cmti-12">CDH1 </span><a 
href="#glo:mutation">mutation</a> (<a 
href="#glo:mtSLIPT">mtSLIPT</a>).</p>
</p><!--l. 495--><p class="indent" >
The number of genes detected by both approaches was fixed (to the number observed) for deriving p-values for pathway relationships (as described in Section <a 
href="#x1-850001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>). These genes were included in the analysis because they can be disproportionately upstream (or downstream) of more <a 
href="#glo:siRNA">siRNA</a> genes than <a 
href="#glo:SLIPT">SLIPT</a> genes, which may lead to them having different proportions of genes detected by either approach upstream (or downstream) of them. However, allowing the number of jointly detected genes to vary during resampling (as shown in Figure <a 
href="#x1-1300039">5.9<!--tex4ht:ref: fig:SL_Pathway_Perm --></a> and Appendix Figure <a 
href="#x1-1880032">I.2<!--tex4ht:ref: fig:mtSL_Pathway_Perm --></a>) or excluding these jointly detected genes did not alter the findings of this approach. Furthermore, expanding the range of <a 
href="#glo:shortest path">shortest paths</a> to consider <a 
href="#glo:edge">links</a> in related <a 
href="#glo:pathway">pathways</a> (using the “meta-<a 
href="#glo:pathway">pathways</a>” constructed in Section <a 
href="#x1-670003">2.4.3<!--tex4ht:ref: methods:subgraphs --></a>) also had little effect on the null distribution generated, despite increasing the computational complexity of the procedure. <a 
 id="x1-130004r246"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">5.3.2 </span> <a 
 id="x1-1310002"></a>Resampling for Synthetic Lethal Pathway Structure
</h4>
<!--l. 504--><p class="noindent" >
The permutation procedure (as described in Section <a 
href="#x1-850001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>) that was performed in Section <a 
href="#x1-1300001">5.3.1<!--tex4ht:ref: chapt4:Structure_GPCR --></a> for the G<sub><span class="cmmi-8">αi</span></sub> and <a 
href="#glo:PI3K">PI3K</a> <a 
href="#glo:pathway">pathways</a> was also applied to other <a 
href="#glo:pathway">pathways</a> identified in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a> and discussed in Section <a 
href="#x1-1190001">5.1<!--tex4ht:ref: chapt4:SL_Genes --></a>. In addition to the cell signalling <a 
href="#glo:pathway">pathways</a> (PI3K/AKT and GCPRs demonstrated in Section <a 
href="#x1-1300001">5.3.1<!--tex4ht:ref: chapt4:Structure_GPCR --></a>), the <a 
href="#glo:pathway">pathways</a> tested include extracellular matrix (with constituent elastic fibre and fibrin <a 
href="#glo:pathway">pathways</a>), and translational <a 
href="#glo:pathway">pathways</a> (with <a 
href="#glo:NMD">NMD</a> and 3<sup><span class="cmsy-8">′</span></sup><a 
href="#glo:UTR">UTR</a> regulation).
</p><!--l. 506--><p class="indent" >
The resampling results across these <a 
href="#glo:pathway">pathways</a> (as shown in Table <a 
href="#x1-1310014">5.4<!--tex4ht:ref: tab:pathway_str_exprSL --></a>) had limited support for association between <a 
href="#glo:graph">pathway</a> structure and detection of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes, with the majority of these being non-significant (as shown for <a 
href="#glo:PI3K">PI3K</a> in Figure <a 
href="#x1-130002r2">5.8b<!--tex4ht:ref: fig:SL_Pathway_PI3K_Perm --></a>), with the exception of G<sub><span class="cmmi-8">αi</span></sub> signalling (as shown in Figure <a 
href="#x1-130001r1">5.8a<!--tex4ht:ref: fig:SL_Pathway_GPCR_Perm --></a>). However, the exact distribution for these <a 
href="#glo:pathway">pathways</a> will differ depending on their structure, the number of genes they contain, and the proportion of <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates among them (including a higher frequency of genes detected by both methods for the <a 
href="#glo:pathway">pathways</a> identified in Section <a 
href="#x1-1070001">4.2.5.1<!--tex4ht:ref: chapt3:compare_pathway_perm --></a>). This resampling is therefore an appropriate procedure to use to detect structural relationships across <a 
href="#glo:pathway">pathways</a> as it does not assume an underlying test statistic distribution.
</p><!--l. 508--><p class="indent" >
<p>Pathway structure was supported for the <a 
href="#glo:NMD">NMD</a> <a 
href="#glo:pathway">pathway</a> (which is consistent with <a 
href="#glo:siRNA">siRNA</a> being downstream in Appendix Figure <a 
href="#x1-1860066">G.6<!--tex4ht:ref: fig:SL_Pathway_NMD --></a>). However, this observation rests upon a single gene and was not replicated when testing <a 
href="#glo:synthetic lethal">synthetic lethality</a> (<a 
href="#glo:mtSLIPT">mtSLIPT</a>) against <span class="cmti-12">CDH1</span></p>
<p><a 
href="#glo:mutation">mutation</a> (as shown in Appendix Table <a 
href="#x1-1880041">I.1<!--tex4ht:ref: tab:pathway_str_mtSL --></a>) nor was it supported by the related 3<sup><span class="cmsy-8">′</span></sup><a 
href="#glo:UTR">UTR</a> regulation and translational elongation <a 
href="#glo:pathway">pathways</a>.</p>
</p><!--l. 510--><p class="indent" >
<a 
 id="x1-1310014"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table 5.4: </span><span class="content">Resampling for <a 
href="#glo:graph">pathway</a> structure of <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection methods</span>
</div><!--tex4ht:label?: x1-1310014 -->
<div class="tabular">
                                             <table id="TBL-71" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-71-1g"><col 
id="TBL-71-1" /></colgroup><colgroup id="TBL-71-2g"><col 
id="TBL-71-2" /><col 
id="TBL-71-3" /></colgroup><colgroup id="TBL-71-4g"><col 
id="TBL-71-4" /><col 
id="TBL-71-5" /></colgroup><colgroup id="TBL-71-6g"><col 
id="TBL-71-6" /><col 
id="TBL-71-7" /><col 
id="TBL-71-8" /><col 
id="TBL-71-9" /></colgroup><colgroup id="TBL-71-10g"><col 
id="TBL-71-10" /><col 
id="TBL-71-11" /></colgroup><colgroup id="TBL-71-12g"><col 
id="TBL-71-12" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-71-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-71-1-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-71-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-71-2-1"  
class="td11">                                     </td><td colspan="2" style="white-space:nowrap; text-align:center;" id="TBL-71-2-2"  
class="td11">      <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<strong><span class="cmbx-12">Graph</span></strong>
</div>       </td><td colspan="2" style="white-space:nowrap; text-align:center;" id="TBL-71-2-4"  
class="td11">    <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<strong><span class="cmbx-12">Candidates</span></strong>
</div>     </td><td colspan="3" style="white-space:nowrap; text-align:center;" id="TBL-71-2-6"  
class="td11"></td>                 <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<strong><span class="cmbx-12">Observed</span></strong>
</div>                    <td colspan="2" style="white-space:nowrap; text-align:center;" id="TBL-71-2-10"  
class="td11">   <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<strong><span class="cmbx-12">Permutation p-value</span></strong>
</div>    </td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-2-12"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<strong><span class="cmbx-12">p-value (FDR)</span></strong>
</div>
</td></tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-71-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-71-3-1"  
class="td11"> 
<strong><span class="cmbx-12">Pathway</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-3-2"  
class="td11"> 
<strong><span class="cmbx-12">Nodes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-3-3"  
class="td11"> 
<strong><span class="cmbx-12">Edges</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-3-4"  
class="td11"> 
<strong><span class="cmbx-12">SLIPT</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-3-5"  
class="td11"> 
<strong><span class="cmbx-12">siRNA</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-3-6"  
class="td11"> 
<strong><span class="cmbx-12">Up</span></strong><sup class="textsuperscript"><span class="cmr-10x-x-109">1</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-3-7"  
class="td11"> 
<strong><span class="cmbx-12">Down</span></strong><sup class="textsuperscript"><span class="cmr-10x-x-109">2</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-3-8"  
class="td11"> 
<strong><span class="cmbx-12">Up</span><span class="cmsy-10x-x-120">-</span><span class="cmbx-12">Down</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-3-9"  
class="td11"> 
<strong><span class="cmbx-12">Up</span><span class="cmmi-12">∕</span><span class="cmbx-12">Down</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-3-10"  
class="td11"> 
<strong><span class="cmbx-12">Up</span><span class="cmsy-10x-x-120">-</span><span class="cmbx-12">Down</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-3-11"  
class="td11"> 
<strong><span class="cmbx-12">Down</span><span class="cmsy-10x-x-120">-</span><span class="cmbx-12">Up</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-3-12"  
class="td11">   
<strong><span class="cmbx-12">Down</span><span class="cmsy-10x-x-120">-</span><span class="cmbx-12">Up</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-71-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-71-4-1"  
class="td11"> 
PI3K Cascade
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-4-2"  
class="td11">   
138
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-4-3"  
class="td11">  
1495
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-4-4"  
class="td11">   
38
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-4-5"  
class="td11">   
25
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-4-6"  
class="td11"> 
122
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-4-7"  
class="td11">  
128
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-4-8"  
class="td11">     
-6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-4-9"  
class="td11">   
0.953
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-4-10"  
class="td11">   
0.5326
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-4-11"  
class="td11">   
0.4606
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-4-12"  
class="td11">     
0.6734
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-71-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-71-5-1"  
class="td11"> 
PI3K/AKT Signalling in Cancer
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-5-2"  
class="td11">   
275
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-5-3"  
class="td11"> 
12882
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-5-4"  
class="td11">   
98
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-5-5"  
class="td11">   
44
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-5-6"  
class="td11"> 
779
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-5-7"  
class="td11">  
679
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-5-8"  
class="td11">    
100
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-5-9"  
class="td11">   
1.147
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-5-10"  
class="td11">   
0.3255
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-5-11"  
class="td11">   
0.6734
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-5-12"  
class="td11">     
0.6734
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-71-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-71-6-1"  
class="td11"> 
<strong><span class="cmbx-12">G</span><sub><span class="cmmi-8">αi</span> </sub> <span class="cmbx-12">Signalling</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-6-2"  
class="td11">   
292
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-6-3"  
class="td11"> 
22003
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-6-4"  
class="td11">   
95
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-6-5"  
class="td11">   
58
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-6-6"  
class="td11"> 
836
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-6-7"  
class="td11">  
1546
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-6-8"  
class="td11">    
-710
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-6-9"  
class="td11">   
0.541
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-6-10"  
class="td11">   
0.9971
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-6-11"  
class="td11">   
0.0029
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-6-12"  
class="td11">     
0.0145
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-71-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-71-7-1"  
class="td11"> 
GPCR downstream
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-7-2"  
class="td11">  
1270
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-7-3"  
class="td11"> 
142071
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-7-4"  
class="td11">   
312
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-7-5"  
class="td11">   
160
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-7-6"  
class="td11"> 
9755
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-7-7"  
class="td11">  
9261
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-7-8"  
class="td11">    
494
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-7-9"  
class="td11">   
1.053
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-7-10"  
class="td11">   
0.3692
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-7-11"  
class="td11">   
0.6305
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-7-12"  
class="td11">     
0.6734
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-71-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-71-8-1"  
class="td11"> 
Elastic fibre formation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-8-2"  
class="td11">   
42
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-8-3"  
class="td11">  
175
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-8-4"  
class="td11">   
24
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-8-5"  
class="td11">    
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-8-6"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-8-7"  
class="td11">   
2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-8-8"  
class="td11">     
-1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-8-9"  
class="td11">   
0.500
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-8-10"  
class="td11">   
0.5461
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-8-11"  
class="td11">   
0.3865
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-8-12"  
class="td11">     
0.6734
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-71-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-71-9-1"  
class="td11"> 
Extracellular matrix
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-9-2"  
class="td11">   
299
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-9-3"  
class="td11">  
3677
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-9-4"  
class="td11">   
127
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-9-5"  
class="td11">   
29
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-9-6"  
class="td11"> 
547
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-9-7"  
class="td11">  
455
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-9-8"  
class="td11">     
92
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-9-9"  
class="td11">   
1.202
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-9-10"  
class="td11">   
0.3351
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-9-11"  
class="td11">   
0.6636
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-9-12"  
class="td11">     
0.6734
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-71-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-71-10-1"  
class="td11"> 
Formation of Fibrin
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-10-2"  
class="td11">   
52
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-10-3"  
class="td11">  
243
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-10-4"  
class="td11">   
18
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-10-5"  
class="td11">    
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-10-6"  
class="td11">  
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-10-7"  
class="td11">   
17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-10-8"  
class="td11">     
-5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-10-9"  
class="td11">   
0.706
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-10-10"  
class="td11">   
0.6198
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-10-11"  
class="td11">   
0.3564
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-10-12"  
class="td11">     
0.6734
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-71-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-71-11-1"  
class="td11"> 
<strong><span class="cmbx-12">Nonsense-Mediated Decay</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-11-2"  
class="td11">   
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-11-3"  
class="td11">  
102
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-11-4"  
class="td11">   
74
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-11-5"  
class="td11">    
2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-11-6"  
class="td11">  
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-11-7"  
class="td11">   
74
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-11-8"  
class="td11">     
-74
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-11-9"  
class="td11">     
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-11-10"  
class="td11">   
1.0000
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-11-11"  
class="td11">  
<span class="cmmi-12">&lt; </span>0<span class="cmmi-12">.</span>0001
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-11-12"  
class="td11">    
<span class="cmmi-12">&lt; </span>0<span class="cmmi-12">.</span>0010
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-71-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-71-12-1"  
class="td11"> 
3’ -UTR-mediated translational regulation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-12-2"  
class="td11">   
107
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-12-3"  
class="td11">  
2860
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-12-4"  
class="td11">   
77
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-12-5"  
class="td11">    
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-12-6"  
class="td11">  
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-12-7"  
class="td11">   
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-12-8"  
class="td11">     
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-12-9"  
class="td11">           </td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-12-10"  
class="td11">   
0.4902
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-12-11"  
class="td11">   
0.5027
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-12-12"  
class="td11">     
0.6734
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-71-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-71-13-1"  
class="td11"> 
Eukaryotic Translation Elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-13-2"  
class="td11">   
92
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-13-3"  
class="td11">  
3746
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-13-4"  
class="td11">   
76
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-13-5"  
class="td11">    
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-13-6"  
class="td11">  
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-13-7"  
class="td11">   
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-13-8"  
class="td11">     
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-13-9"  
class="td11">           </td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-13-10"  
class="td11">   
0.4943
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-13-11"  
class="td11">   
0.4933
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-71-13-12"  
class="td11">     
0.6734
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-71-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-71-14-1"  
class="td11">                                     </td></tr></table>
<!--l. 557--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Pathways in the Reactome network tested for structural relationships between </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes by resampling. The raw p-value (computed without adjusting for multiple comparisons over </span><a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathways</span></a><span class="cmr-10x-x-109">) is given for</span> <span class="cmr-10x-x-109">the difference in upstream and downstream paths from </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">to </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">gene candidate partners of </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">with significant </span><a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathways</span></a> <span class="cmr-10x-x-109">highlighted in bold. Sampling was performed only in the target </span><a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathway</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:shortest path"><span class="cmr-10x-x-109">shortest</span> <span class="cmr-10x-x-109">paths</span></a> <span class="cmr-10x-x-109">were computed within it. Loops or paths in either direction that could not be resolved were excluded from the analysis. The genes detected by both </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">(or resampling for them) were included in</span> <span class="cmr-10x-x-109">the analysis and the number of these were fixed to the number observed.</span> <sup class="textsuperscript"><span class="cmr-10x-x-109">1</span></sup><span class="cmr-10x-x-109">The number of paths where the </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">candidate was upstream of a </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">candidate</span> <sup class="textsuperscript"><span class="cmr-10x-x-109">2</span></sup><span class="cmr-10x-x-109">The number of paths where the </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">candidate was downstream of a </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">candidate</span></p>
</p>
</div><hr class="endfloat" />
<!--l. 560--><p class="indent" >
There does not appear to be a consensus on the directionality of <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidates across <a 
href="#glo:pathway">pathways</a> as distinct <a 
href="#glo:pathway">pathways</a> showed stronger tendency for <a 
href="#glo:siRNA">siRNA</a> genes to be either upstream or downstream. Even related <a 
href="#glo:pathway">pathways</a> such as <a 
href="#glo:PI3K">PI3K</a> and PI3K/AKT signalling showed directional events in opposite directions. The strongest <a 
href="#glo:pathway">pathway</a> (among those tested) with support for directional <a 
href="#glo:pathway">pathways</a> structure is G<sub><span class="cmmi-8">αi</span> </sub> signalling (as shown in Figure <a 
href="#x1-130001r1">5.8a<!--tex4ht:ref: fig:SL_Pathway_GPCR_Perm --></a>). In contrast to the other <a 
href="#glo:pathway">pathways</a>s G<sub><span class="cmmi-8">αi</span></sub> signalling showed significant downstream <a 
href="#glo:siRNA">siRNA</a> genes for <a 
href="#glo:SLIPT">SLIPT</a> from a large number of <a 
href="#glo:shortest path">shortest paths</a> (in Table <a 
href="#x1-1310014">5.4<!--tex4ht:ref: tab:pathway_str_exprSL --></a>). This would indicate that <a 
href="#glo:SLIPT">SLIPT</a> detects upstream regulators of genes experimentally validated by <a 
href="#glo:siRNA">siRNA</a> in this <a 
href="#glo:pathway">pathway</a>. This result was <a 
href="#glo:pathway">pathway</a> was also the strongest result in <a 
href="#glo:mtSLIPT">mtSLIPT</a> results (Appendix Table <a 
href="#x1-1880041">I.1<!--tex4ht:ref: tab:pathway_str_mtSL --></a>), although it was not significant after adjusting for multiple testing in this case.
</p><!--l. 564--><p class="indent" >
There is insufficient evidence to determine whether there is <a 
href="#glo:graph">pathway</a> structure, that genes were detected upstream or downstream by either method, between the <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidates in many of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> (identified in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>). In particular, directional structure among <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates for <span class="cmti-12">CDH1 </span>was not strongly supported in most of the signalling <a 
href="#glo:pathway">pathways</a> (with the exception of G<sub><span class="cmmi-8">αi</span></sub> signalling) upon which the rationale for <a 
href="#glo:graph">pathway</a> structure hypotheses were based. While there is statistically significant over-representation of many of these <a 
href="#glo:pathway">pathways</a> in genes detected by both <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> (as described in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>), many of these did not show relationships with respect to <a 
href="#glo:graph">pathway</a> structure. Despite the design of a robust resampling approach to test relationships between gene groups, the detection of structural relationships between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates did not generalise across <a 
href="#glo:pathway">pathways</a> (and was specific to a few). Such structural relationships may apply more broadly to gene networkss as different biological <a 
href="#glo:pathway">pathways</a> were more over-represented among <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates. Furthermore, <a 
href="#glo:graph">pathway</a> structure did not account for the discrepancy between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates which did not significantly intersect, such as the <a 
href="#glo:PI3K">PI3K</a> cascade. <a 
 id="x1-131002r245"></a>
</p>
                                                                                    

<h3 class="sectionHead">
<span class="titlemark">5.4 </span> <a 
 id="x1-1320004"></a>Discussion
</h3>
<!--l. 571--><p class="noindent" >
These investigations used a functional <a 
href="#glo:pathway">pathway</a> network that encapsulates protein complexes and functional modules. The Reactome network uses curated, experimentally identified <a 
href="#glo:pathway">pathways</a> to determine relationships between genes and does not have the limitation of relying solely on protein binding or text-mining which are prone to false positives (<a 
href="#XReactome">Croft <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XReactome">2014</a>) . While it is not documented whether these relationships are activating or inhibitory, the Reactome network (<a 
href="#XReactome">Croft <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XReactome">2014</a>) is sufficient to test <a 
href="#glo:pathway">pathway</a> relationships with directional information.
</p><!--l. 573--><p class="indent" >
Synthetic lethal genes and <a 
href="#glo:pathway">pathways</a> (for <span class="cmti-12">CDH1 </span>loss in cancer) were identified across <a 
href="#glo:gene expression">gene expression</a> and <a 
href="#glo:mutation">mutation</a> datasets in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>. The investigations presented here extend those findings to consider <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene candidates within <a 
href="#glo:graph">pathway</a> structures, including exploring whether the discrepancy between individual <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidate genes can be accounted for within a <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathway</a>. Pathways with replicated <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes across these detection methods, breast and stomach cancer data, were investigated, including <a 
href="#glo:pathway">pathways</a> from the extracellular microenvironment to core translational <a 
href="#glo:pathway">pathways</a> and the signalling <a 
href="#glo:pathway">pathways</a> between them.
</p><!--l. 579--><p class="indent" >
Examining <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene candidates in the context of <a 
href="#glo:graph">pathway</a> structures may also provide additional mechanisms by which the function of particular genes is subject to <a 
href="#glo:induced essentiality">induced essentiality</a> and support for them belonging to a <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathway</a>. Gene candidates with characterised functions important to cellular viability are ideal for triage of targets specific to <span class="cmti-12">CDH1 </span>deficient tumours and for further experimental validation in preclinical models. This chapter presents computational methods to use <a 
href="#glo:graph">pathway</a> structure in an attempt to detect genes with importance in a <a 
href="#glo:pathway">pathway</a> and reconcile the differences between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidate genes with <a 
href="#glo:pathway">pathway</a> relationships (e.g., one group being downstream of the other).
</p><!--l. 583--><p class="indent" >
<p>Many genes were detected by either <a 
href="#glo:SLIPT">SLIPT</a> or <a 
href="#glo:siRNA">siRNA</a>. The differences between these computational and experimental screening approaches could feasibly lead to differences in which genes within a <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathway</a> are identified. Genes detected by <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection strategies included those of biological importance within <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a>, those which are actionable drug targets, and those with functional implications for the biological growth mechanisms or vulnerabilities of <span class="cmti-12">CDH1 </span>deficient tumours. It appeared that genes detected by both approaches were highly connected (or of importance) in the <a 
href="#glo:graph">network</a> structure of some <a 
href="#glo:pathway">pathways</a>, and that there may be some structure with</p>
<a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidates tending to appear upstream or downstream of each other.
</p><!--l. 585--><p class="indent" >
The complexity of biological <a 
href="#glo:pathway">pathways</a> meant that relationships between gene candidates were difficult to discern without formal mathematical and computational approaches, and thus these were used to analyse large biological networks. Network analysis techniques were applied to formalise and quantify the connectivity and importance (centrality) of genes within <a 
href="#glo:pathway">pathways</a> (using G<sub><span class="cmmi-8">αi</span></sub> signalling as an example). However, these network techniques were unable to identify distinct differences in many of the network properties of genes between those detected as <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates by computational or experimental methods. These network metrics support the application of <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection across <a 
href="#glo:pathway">pathways</a> (and the findings using <a 
href="#glo:pathway">pathways</a> as gene sets in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>) as neither <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection approach was pre-disposed towards genes of higher importance or connectivity and neither approach was insensitive to genes of lower importance or connectivity. <a 
href="#glo:SLIPT">SLIPT</a> did not detect genes with a significantly more crucial role in the G<sub><span class="cmmi-8">αi</span></sub> <a 
href="#glo:pathway">pathway</a>, as inferred by <a 
href="#glo:pathway">pathway</a> connectivity and <a 
href="#glo:centrality">centrality</a> measures. However, <a 
href="#glo:SLIPT">SLIPT</a> genes had significantly lower centrality in the G<sub><span class="cmmi-8">αi</span> </sub> <a 
href="#glo:pathway">pathway</a> by <a 
href="#glo:PageRank centrality">PageRank centrality</a> (as shown in Section <a 
href="#x1-1280002">5.2.2.2<!--tex4ht:ref: chapt4:Network_PageRank --></a>), and so the highest scoring genes may be too <a 
href="#glo:essential">essential</a> to cellular viability to be <a 
href="#glo:synthetic lethal">synthetic lethal</a>.
</p><!--l. 589--><p class="indent" >
A measure of <a 
href="#glo:graph">pathway</a> structure between individual <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidate genes within a <a 
href="#glo:pathway">pathway</a> was devised using the direction of <a 
href="#glo:shortest path">shortest paths</a> in a directed <a 
href="#glo:graph">graph</a> structure. This is amenable to detecting the consensus directionality of the <a 
href="#glo:pathway">pathway</a> across pairs of genes detected by either method. The <a 
href="#glo:graph">pathway</a> structure methodology developed here is generally applicable to comparison of <a 
href="#glo:vertex">node</a> groups (which may intersect), including genes in biological <a 
href="#glo:pathway">pathways</a> and their detection by different methodologies. While the <a 
href="#glo:graph">pathway</a> structure measure alone is not able to detect structural relationships between gene groups (e.g., <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates), it is amenable to resampling to determine whether these relationships are statistically significant. This approach successfully detected a statistically robust relationship between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidate genes on the G<sub><span class="cmmi-8">αi</span></sub> signalling <a 
href="#glo:pathway">pathway</a>, despite there being few differences between these genes with respect to network metrics of connectivity or centrality. <a 
 id="x1-132001r252"></a>
</p>
                                                                                    

<h3 class="sectionHead">
<span class="titlemark">5.5 </span> <a 
 id="x1-1330005"></a>Summary
</h3>
<!--l. 593--><p class="noindent" >
Together these analyses of biological <a 
href="#glo:pathway">pathways</a>, network metrics, and statistical procedures devised specifically for this purpose were applied to Reactome <a 
href="#glo:graph">pathway</a> structures to test whether structural relationships existed between <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates. Of particular interest was whether these relationships related to the differences between the computational (<a 
href="#glo:SLIPT">SLIPT</a>) and experimental (<a 
href="#glo:siRNA">siRNA</a>) <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate partners of <span class="cmti-12">CDH1 </span>(in the <a 
href="#glo:pathway">pathways</a> discussed in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>).
</p><!--l. 595--><p class="indent" >
While biologically relevant relationships were observed in specific <a 
href="#glo:pathway">pathways</a>, there were few detectable structural relationships between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates, apart from structural relationships specific to G<sub><span class="cmmi-8">αi</span></sub> signalling. In this <a 
href="#glo:pathway">pathway</a>, <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates did not exhibit significant differences in network connectivity or <a 
href="#glo:centrality">centrality</a> measures. These network analyses were also unable to ascertain whether the candidates detected by either method stratified into upstream and downstream genes on the <a 
href="#glo:pathway">pathway</a>.
</p><!--l. 597--><p class="indent" >
A statistical resampling procedure was applied to <a 
href="#glo:shortest path">shortest path</a> analysis to test whether pairs of <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates were more likely to be upstream or downstream of each other. This approach did not detect many structural relationships in the <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> identified in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>. Overall, support for <a 
href="#glo:graph">pathway</a> structure between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates was weak and the direction was inconsistent across <a 
href="#glo:pathway">pathways</a>. Therefore <a 
href="#glo:graph">pathway</a> structure does not appear to generally account for the differences between the <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates, although it may apply in specific <a 
href="#glo:pathway">pathways</a> as demonstrated with G<sub><span class="cmmi-8">αi</span></sub> signalling. It was possible to detect some <a 
href="#glo:pathway">pathway</a> relationships between candidate genes in <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a>, in addition to the significantly over-represented genes shared between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> (as identified in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>).
</p><!--l. 599--><p class="indent" >
<p>Furthermore, the resampling procedure demonstrated in this chapter is more widely applicable to gene states in <a 
href="#glo:graph">network</a> structures and may be of further utility in the analysis of biological <a 
href="#glo:pathway">pathways</a> or networks. This approach was able to quantify structural relationships that were otherwise difficult to interpret and to conclusively exclude many potential relationships. In this respect, the network resampling methodology may also be applicable to triage of <a 
href="#glo:targeted therapy">therapeutic targets</a> for experimental validation.</p>
</p>
<h2 class="chapterHead">
<span class="titlemark">Chapter 6</span><br /><a 
 id="x1-1340006"></a>Simulation and Modelling of Synthetic Lethal Pathways
</h2>
<!--l. 4--><p class="noindent" >
Simulation and modelling of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in <a 
href="#glo:gene expression">gene expression</a> was revisited in greater detail in this chapter, building upon the results which supported the use of <a 
href="#glo:SLIPT">SLIPT</a> (in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>). In Chapter <a 
href="#x1-730003">3<!--tex4ht:ref: chap:methods_dev --></a>, a procedure for generating simulated data with underlying (known) <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of a query gene, such as <span class="cmti-12">CDH1</span>, was developed (as described in Section <a 
href="#x1-770002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a>) by sampling from a multivariate normal distribution based on a statistical model of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in <a 
href="#glo:gene expression">gene expression</a> data (as described in Section <a 
href="#x1-760001">3.2.1<!--tex4ht:ref: methods:SL_Model --></a>). This simulation framework was applied to simulated data (in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>), including simple correlation structures to assess the statistical performance of the <a 
href="#glo:SLIPT">SLIPT</a> methodology and support its use as a computational approach for detecting <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates from <a 
href="#glo:gene expression">expression</a> data throughout this thesis (Chapters <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a> and <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>).
</p><!--l. 6--><p class="indent" >
While this basic framework provided some support for the use of <a 
href="#glo:SLIPT">SLIPT</a>, further investigations with simulations were conducted to assess the strengths and limitations of the <a 
href="#glo:SLIPT">SLIPT</a> methodology, compare it to alternative statistical approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection, and assess its performance under more complex correlation structures. Together these simulation investigations assess the performance of the <a 
href="#glo:SLIPT">SLIPT</a> methodology, including on pathway <a 
href="#glo:graph">graph</a> structures (e.g., those discussed in Chapter <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>). These results can indicate whether the <a 
href="#glo:SLIPT">SLIPT</a> methodology robustly detects known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners (and how it compares to other <a 
href="#glo:bioinformatics">bioinformatics</a> strategies) or is suitable for wider <a 
href="#glo:genomic">genomics</a> applications.
</p><!--l. 8--><p class="indent" >
These simulation investigations continue to utilise the multivariate normal simulation procedure (as applied in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>) with further refinements. The <a 
href="#glo:SLIPT">SLIPT</a> methodology (and the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test) were applied across a range of parameters (including altering the quantiles for detecting <a 
href="#glo:synthetic lethal">synthetic lethal</a> direction) and compared to correlation as a predictor of <a 
href="#glo:synthetic lethal">synthetic lethality</a>. These simulations included thousands of non-synthetic lethal genes and correlations with the query gene (as performed in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>).
</p><!--l. 10--><p class="indent" >
<p>A refined simulation procedure was developed specifically to extend the methodology described in Section <a 
href="#x1-750002">3.2<!--tex4ht:ref: methods:simulation_SL_expression --></a> to utilise pathway <a 
href="#glo:graph">graph</a> structures for the correlation structures of simulated datasets (as described in Section <a 
href="#x1-860002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>). This methodology can be applied to simulated correlation structures across simple <a 
href="#glo:graph">graph</a> structures to test specific network modules or use <a 
href="#glo:graph">pathway</a> structures based on biological pathways. Thus <a 
href="#glo:graph">graph</a> structure and simulation approaches were combined to test whether a gene locus in a</p>
pathway affects detection by <a 
href="#glo:SLIPT">SLIPT</a> and whether <a 
href="#glo:SLIPT">SLIPT</a> performance is affected by <a 
href="#glo:graph">pathway</a> structure. The simulation procedure based on <a 
href="#glo:graph">graph</a> structures was applied in a computational pipeline across many parameter combinations using high-performance computing resources (as discussed in Section <a 
href="#x1-720003">2.5.3<!--tex4ht:ref: methods:HPC --></a>) and the core simulation functions have been released as a software package for wider use to test <a 
href="#glo:bioinformatics">bioinformatics</a> and statistical methods on <a 
href="#glo:graph">graph</a> structures (as described in Section <a 
href="#x1-900003">3.5.3<!--tex4ht:ref: methods:igraph_extensions --></a>). <a 
 id="x1-134001r253"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">6.1 </span> <a 
 id="x1-1350001"></a>Synthetic Lethal Detection Methods
</h3>
<!--l. 22--><p class="noindent" >
The <a 
href="#glo:SLIPT">SLIPT</a> methodology (as it has been applied throughout Chapters <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a> and <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>) was compared for alternative computational approaches to detecting <a 
href="#glo:synthetic lethal">synthetic lethality</a> in simulated <a 
href="#glo:gene expression">gene expression</a> data. As discussed in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>, this procedure enabled testing the ability of <a 
href="#glo:SLIPT">SLIPT</a> to detect known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes by sampling from a statistical model of <a 
href="#glo:synthetic lethal">synthetic lethality</a>. While comprehensive benchmarking has not been performed, several approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection are considered (e.g., Pearson correlation, the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test, and testing for bimodality) to evaluate the strengths of the <a 
href="#glo:SLIPT">SLIPT</a> methodology, including modifications to the parameters of <a 
href="#glo:SLIPT">SLIPT</a>. The following comparisons of simulations of computational detection of <a 
href="#glo:synthetic lethal">synthetic lethality</a> with different statistical rationales were perfomed to show the strengths of <a 
href="#glo:SLIPT">SLIPT</a>, evaluate whether it is appropriate for further application in <a 
href="#glo:genomic">genomics</a> research, and identify limitations which may be addressed with further developments. <a 
 id="x1-135001r250"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.1.1 </span> <a 
 id="x1-1360001"></a>Performance of SLIPT and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> across Quantiles
</h4>
<!--l. 30--><p class="noindent" >
<p>Simulated datasets with <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes were generated using the multivariate normal simulation procedure (as described in Section <a 
href="#x1-770002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a>) with performance assessed using <a 
href="#glo:AUROC">AUROC</a> analysis (as described in Section <a 
href="#x1-610005">2.3.5<!--tex4ht:ref: methods:simulation --></a>). <a 
href="#glo:synthetic lethal">Synthetic lethal</a> detection was compared for modifications to the <a 
href="#glo:SLIPT">SLIPT</a> methodology (as described in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>), namely that the quantiles used to define low and high <a 
href="#glo:gene expression">expression</a> were varied. Rather, than <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span> (as used throughout this thesis) the samples below the lowest <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmmi-8">n</span> quantile and above the highest <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmmi-8">n</span> quantile were used for <a 
href="#glo:SLIPT">SLIPT</a> (and the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>-test) to detect samples that exhibited low and high <a 
href="#glo:gene expression">expression</a> levels respectively. The quantiles tested ranged from two, splitting at</p>
the <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">2</span> quantile (the median), to 100, using the lowest (1%) and highest (99%) percentiles.
</p><!--l. 32--><p class="indent" >
This enabled testing of the threshold for low <a 
href="#glo:gene expression">expression</a> of genes which is most able to distinguish <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes, even with higher-order <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions (as discussed in Section <a 
href="#x1-760001">3.2.1<!--tex4ht:ref: methods:SL_Model --></a>). Both <a 
href="#glo:SLIPT">SLIPT</a> with the directional criteria for <a 
href="#glo:synthetic lethal">synthetic lethality</a> and significance of the equivalent <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test were performed for each quantile. Pearson correlation was also tested on simulated continuous <a 
href="#glo:gene expression">expression</a> data for <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection in simulated data, considering both positive and negative correlations separately as predictors of <a 
href="#glo:synthetic lethal">synthetic lethality</a> for comparison with <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> based approaches, using discrete categories of gene function deriving from quantiles.
</p><!--l. 34--><p class="indent" >
<p>The results presented throughout this Section use the example of five <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners to illustrate the differences in performance between the standard <a 
href="#glo:SLIPT">SLIPT</a> procedure (slipt-3) to <span class="cmmi-12">n </span>quantiles (slipt-<span class="cmmi-12">n</span>), the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>-test on the same quantiles, and positive or negative correlation. However, similar results across different numbers of known <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes are shown in Appendix <a 
href="#x1-189000J">J<!--tex4ht:ref: appendix:compare_chisq --></a>. The <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection procedures were compared with 10,000 simulations of a small dataset of 100 genes and 1000 samples without correlation structure between genes (as performed in Section <a 
href="#x1-800002">3.3.2<!--tex4ht:ref: chapt2:simulation_mvtnorm --></a>).</p>
</p><!--l. 36--><p class="indent" >
<a 
 id="x1-1360032"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 38--><p class="noindent" >
</p><!--l. 43--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(5)".png  /><a 
 id="x1-136001r1"></a> <span class="cmr-10x-x-109">Barplot</span> <span class="cmr-10x-x-109">of </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span> </sup> <span class="cmr-10x-x-109">,</span> <span class="cmr-10x-x-109">SLIPT,</span> <span class="cmr-10x-x-109">and</span> <span class="cmr-10x-x-109">correlation.</span>
</p><!--l. 48--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png  /><a 
 id="x1-136002r2"></a> <span class="cmr-10x-x-109">Lineplot</span> <span class="cmr-10x-x-109">of </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span> </sup> <span class="cmr-10x-x-109">,</span> <span class="cmr-10x-x-109">SLIPT,</span> <span class="cmr-10x-x-109">and</span> <span class="cmr-10x-x-109">correlation.</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 6.2: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles</span></strong>. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> detection (of 5 genes) with quantiles as on the axes. The barplot uses the same hues for each quantile (grey for correlation) and darker for <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> (and positive correlation). The line plot (with log-scale quantiles) is coloured according to the legend. <a 
href="#glo:SLIPT">SLIPT</a> and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> perform similarly, peaking at <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span>-quantiles and converging to random (0.5). Negative correlation had higher performance than positive correlation but not optimal quantiles for <a 
href="#glo:SLIPT">SLIPT</a> or <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>.</span></p>
</div>
<!--tex4ht:label?: x1-1360032 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 55--><p class="indent" >
As shown in Figure <a 
href="#x1-1360032">6.2<!--tex4ht:ref: fig:simulation1108_Graph5All5SL --></a>, the <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span>-quantiles previously used have optimal performance and <a 
href="#glo:SLIPT">SLIPT</a> has a comparable or higher performance than the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>-test alone across quantiles. Pearson correlation was also tested as a predictor of <a 
href="#glo:synthetic lethal">synthetic lethality</a> (i.e., whether highly positive or negative correlations with the query gene detected <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners). Positive correlation performed worse than random (with an <a 
href="#glo:AUROC">AUROC</a> lower than 0.5) as thus coexpression of genes was not predictive of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in simulated data. Conversely, negative correlation was predictive of <a 
href="#glo:synthetic lethal">synthetic lethality</a>, consistent with <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene activity being mutually exclusive. However, neither correlation approach performed as well as the optimal quantiles for the <a 
href="#glo:SLIPT">SLIPT</a> procedure or the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> -test.
</p><!--l. 58--><p class="indent" >
<p>These results are shown in both a bargraph and lineplot to show the individual results of each parameter, and to compare <a 
href="#glo:SLIPT">SLIPT</a> with the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>-test side-by-side across quantiles. Similarly, these plots are given for detecting a range of known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in the simulations in Appendix Figures <a 
href="#x1-1890174">J.4<!--tex4ht:ref: fig:simulation1108_Graph5Allbarplot --></a> and <a 
href="#x1-1890348">J.8<!--tex4ht:ref: fig:simulation1108_Graph5Alllineplot --></a>. These demonstrate that the findings shown for five <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes are robust across different numbers of underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes.</p>
</p><!--l. 60--><p class="indent" >
<a 
 id="x1-1360043"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 62--><p class="noindent" >

</p><!--l. 71--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png  />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 6.3: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles with more genes</span></strong>. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> detection (of 5 genes in 20,000) with quantiles as in axis labels. The line plot (with log-scale quantiles) is coloured according to the legend. As for simulations with fewer genes, <a 
href="#glo:SLIPT">SLIPT</a> and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> perform similarly, peaking at <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span>-quantiles and converging to random (0.5). Negative correlation had higher performance than positive correlation but not optimal quantiles for <a 
href="#glo:SLIPT">SLIPT</a> or <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>.</span></p>
</div>
<!--tex4ht:label?: x1-1360043 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 79--><p class="indent" >
The <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection procedures were also tested with 1000 simulations of a larger dataset of 20,000 genes and 1000 samples. While fewer simulations gives a less accurate <a 
href="#glo:ROC">ROC</a> result, this is sufficient to replicate the above findings with a feasible number of genes in a human <a 
href="#glo:gene expression">gene expression</a> dataset and assess the impact of a higher proportion of non-synthetic lethal genes (potential false positives). Simulated datasets of this size were also used in Section  <a 
href="#x1-800002">3.3.2<!--tex4ht:ref: chapt2:simulation_mvtnorm --></a> to test the specificity in a number of genes similar to that in experimental datasets for cancer <a 
href="#glo:genome">genomes</a>. As shown in Figure <a 
href="#x1-1360043">6.3<!--tex4ht:ref: fig:simulation1107_Graph5All5SL --></a>, the above findings were replicated in simulations of a larger dataset with 20,000 genes. These were also robustly replicated across varying numbers of underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes (as shown in Appendix Figure <a 
href="#x1-18905112">J.12<!--tex4ht:ref: fig:simulation1107_Graph5Alllineplot --></a>). <a 
 id="x1-136005r242"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">6.1.1.1 </span> <a 
 id="x1-1370001"></a>Correlated Query Genes affects Specificity
</h5>
                                                                                    

<!--l. 88--><p class="indent" >
<a 
 id="x1-1370014"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 90--><p class="noindent" >

</p><!--l. 99--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png  />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 6.4: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles with query</span> <span class="cmbx-12">correlation</span></strong>. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> detection (of 5 genes in 100 including 5 query correlated) with quantiles as in axis labels. The line plot (with log-scale quantiles) is coloured according to the legend. <a 
href="#glo:SLIPT">SLIPT</a> performs consistently higher than <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> due to higher specificity. Negative correlation performed modestly.</span></p>
</div>
<!--tex4ht:label?: x1-1370014 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 108--><p class="noindent" >
As discussed in Section <a 
href="#x1-820002">3.3.2.2<!--tex4ht:ref: chapt2:simulation_mvtnorm_query_cor --></a>, positively correlated genes (with the query gene) have an impact on the performance of <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection. <a 
href="#glo:SLIPT">SLIPT</a> was able to distinguish these correlated genes from <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners and hence is likely to have a higher specificity in datasets which include positively correlated genes with the query gene (as expected in <a 
href="#glo:gene expression">gene expression</a> data). The <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection procedures were compared with 10,000 simulations of a small dataset of 100 genes (with 5 correlated with the query gene) and 1000 samples otherwise without correlation structure between genes. As shown in Figure <a 
href="#x1-1370014">6.4<!--tex4ht:ref: fig:simulation1108_query_cor_Graph5All5SL --></a>, this specificity is reflected in the increased <a 
href="#glo:AUROC">AUROC</a> performance values for <a 
href="#glo:SLIPT">SLIPT</a> (in contrast to Figure <a 
href="#x1-1360032">6.2<!--tex4ht:ref: fig:simulation1108_Graph5All5SL --></a>). This specificity can be attributed to the directional criteria (as described in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) since the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> -test alone performs comparatively poorly with positively correlated genes.
</p><!--l. 110--><p class="indent" >
<p>The <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection procedures were also compared with 1000 simulations of a larger dataset of 20,000 genes (with 1000 correlated with the query gene) and 1000 samples otherwise without correlation structure between genes. This simulation increases the number of genes (and proportion of negative genes) to those comparable with a human <a 
href="#glo:gene expression">gene expression</a> dataset while maintaining a comparable 5% of positively correlated genes. As shown in Figure <a 
href="#x1-1370025">6.5<!--tex4ht:ref: fig:simulation1107_query_cor_Graph5All5SL --></a>, <a 
href="#glo:SLIPT">SLIPT</a> still outperforms <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> or negative correlation and is optimal at the <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span> -quantile. The difference between <a 
href="#glo:SLIPT">SLIPT</a> and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> was less pronounced in a larger dataset with many weakly correlated genes. The greater specificity of <a 
href="#glo:SLIPT">SLIPT</a> than the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>-test to distinguish positively correlated non-synthetic lethal genes is not as evident with a large number of negative genes (as potential false positives). However, specificity is an important consideration in large-scale <a 
href="#glo:genomic">genomics</a> analysis where there are potentially many false positives.</p>
</p><!--l. 113--><p class="indent" >
<a 
 id="x1-1370025"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 115--><p class="noindent" >

</p><!--l. 124--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png  />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 6.5: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles with query</span> <span class="cmbx-12">correlation and more genes</span></strong>. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> detection (of 5 genes in 20,000 including 1000 query correlated) with quantiles as in axis labels. The line plot (with log-scale quantiles) is coloured according to the legend. <a 
href="#glo:SLIPT">SLIPT</a> performs consistently higher than <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> due to higher specificity. Negative correlation performed modestly.</span></p>
</div>
<!--tex4ht:label?: x1-1370025 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 132--><p class="indent" >
Nevertheless, <a 
href="#glo:SLIPT">SLIPT</a> with <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span>-quantiles (as performed throughout Chapters <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a> and <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>), had higher performance than when other quantile thresholds were used, particularly when positive correlations were present (replicating the Section <a 
href="#x1-820002">3.3.2.2<!--tex4ht:ref: chapt2:simulation_mvtnorm_query_cor --></a>). These findings hold across different numbers of underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes (as shown in Figures <a 
href="#x1-19003420">J.20<!--tex4ht:ref: fig:simulation1108_query_cor_Graph5Alllineplot --></a> and  <a 
href="#x1-19005124">J.24<!--tex4ht:ref: fig:simulation1107_query_cor_Graph5Alllineplot --></a>).
</p><!--l. 134--><p class="indent" >
Together these results support the use of <a 
href="#glo:SLIPT">SLIPT</a>, particularly the use of quantiles as thresholds for gene function and specific use of <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span>-quantiles which perform well compared to other quantiles. A particular concern in the design of <a 
href="#glo:SLIPT">SLIPT</a> for <a 
href="#glo:gene expression">expression</a> data whether the samples sizes are sufficient when the data are divided into quantiles. The <a 
href="#glo:SLIPT">SLIPT</a> methodology further performed better for <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span>-quantiles (and other moderate values) than <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> or correlation as a predictor of <a 
href="#glo:synthetic lethal">synthetic lethality</a>. These results were irrespective of sample size or p-value threshold since the results replicated across sample sizes and the <a 
href="#glo:AUROC">AUROC</a> values were independent significance thresholds. Using a moderate number of quantiles for <a 
href="#glo:SLIPT">SLIPT</a> ensures that there are a sufficient number of samples expected below and above them so that deviations from these are statistically detectable. These quantiles were also optimal for the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test which uses the same expected values asthe <a 
href="#glo:SLIPT">SLIPT</a> directional conditions. <a 
 id="x1-137003r256"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.1.2 </span> <a 
 id="x1-1380002"></a>Alternative Synthetic Lethal Detection Strategies
</h4>
<!--l. 140--><p class="noindent" >
The <a 
href="#glo:SLIPT">SLIPT</a> approach (and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>) to detect <a 
href="#glo:synthetic lethal">synthetic lethality</a> from binning <a 
href="#glo:gene expression">expression</a> to estimate gene function also outperformed correlations, which use continuous data directly. Correlation performing poorly as a <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection strategy is consistent with there not necessarily being a relationship between <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners, which can be in distinct biological pathways, expressed at different times or in different cell types. Nevertheless, correlation is among the alternative detection methods considered in further detail.
</p><!--l. 142--><p class="indent" >
<p>The <a 
href="#glo:BiSEp">BiSEp</a> R package (<a 
href="#XWappett2014">Wappett</a>, <a 
href="#XWappett2014">2014</a>) for using bimodality to detect <a 
href="#glo:synthetic lethal">synthetic lethality</a> (<a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWappett2016">2016</a>) was also considered, along with a linear regression approach. These statistical methods span a range of computational approaches to detecting <a 
href="#glo:synthetic lethal">synthetic lethality</a> and serve to compare alternatives to <a 
href="#glo:SLIPT">SLIPT</a>, supporting its design and application. However, these comparisons are able provide supporting data from statistical modelling and simulations for the viability of the <a 
href="#glo:SLIPT">SLIPT</a> methodology for</p>
<a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery in cancer (as demonstrated in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>) and further applications. <a 
 id="x1-138001r261"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">6.1.2.1 </span> <a 
 id="x1-1390001"></a>Correlation for Synthetic Lethal Detection
</h5>
<!--l. 151--><p class="noindent" >
<p>As expected, negative (Pearson) correlation performed better than positive correlation at detecting <a 
href="#glo:synthetic lethal">synthetic lethality</a> (shown in Section <a 
href="#x1-1360001">6.1.1<!--tex4ht:ref: chapt5:compare_chisq --></a>). However, neither correlation approach performed as well as <a 
href="#glo:SLIPT">SLIPT</a> or the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test as a predictor of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene partners. It is notable that negative correlation still often performed considerably better than random chance.</p>
</p><!--l. 154--><p class="indent" >
<a 
 id="x1-1390016"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 156--><p class="noindent" >
</p><!--l. 157--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161207_Cor_randx/SL_Model_Test_Graph_10K_Graph4_ROC_Compare(1)_SLIPT_v_Cor".png  />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 6.6: </span><span class="content"><strong><span class="cmbx-12">Performance of negative correlation and SLIPT</span></strong>. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> detection with <a 
href="#glo:SLIPT">SLIPT</a> was compared to negative (Pearson) correlation across parameters. SLIPT consistently outperformed correlation. Both approaches had lower performance for more <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners and for lower sample sizes. 10,000 simulations were performed with correlation structure. </span></p>
</div>
<!--tex4ht:label?: x1-1390016 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 168--><p class="indent" >
Negative correlation was compared directly to the <a 
href="#glo:SLIPT">SLIPT</a> methodology (as described in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) across numbers of known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners and sample size (ranging from 500 to 5000). This comparison used 1000 simulations of a dataset with 20,000 genes and <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes from within a network (sampled as in Section <a 
href="#x1-860002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>) with a 0.8 correlation between adjacent genes. In a direct comparison of <a 
href="#glo:SLIPT">SLIPT</a> and negative correlation (shown in Figure <a 
href="#x1-1390016">6.6<!--tex4ht:ref: fig:simulation1205_randx_Graph4cfnCor --></a>), <a 
href="#glo:SLIPT">SLIPT</a> consistently has higher performance in simulated data across parameter values and (inverse) correlation-based approaches perform modestly in comparison. Thus using thresholds to categorise <a 
href="#glo:gene expression">expression</a> data (as performed by <a 
href="#glo:SLIPT">SLIPT</a> and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>) does not compromise the performance of these methods by losing continuous data that would be used for calculating correlations.
</p><!--l. 172--><p class="indent" >
Both <a 
href="#glo:SLIPT">SLIPT</a> and correlation had poorer performance with increasing numbers of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes to detect, while they had higher performance in higher sample sizes, as expected (as previously observed for <a 
href="#glo:SLIPT">SLIPT</a> in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>). Thus the issue with detection of greater numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes is not specific to <a 
href="#glo:SLIPT">SLIPT</a> but occurs across computational methods of <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery in (simulated) <a 
href="#glo:gene expression">expression</a> data and likely stems from cryptic higher-order <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions (as conservatively assumed in Section <a 
href="#x1-760001">3.2.1<!--tex4ht:ref: methods:SL_Model --></a>). <a 
 id="x1-139002r265"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">6.1.2.2 </span> <a 
 id="x1-1400002"></a>Testing for Bimodality with BiSEp
</h5>
<!--l. 179--><p class="noindent" >
Extensive attempts were also made to compare <a 
href="#glo:SLIPT">SLIPT</a> to the <a 
href="#glo:BiSEp">BiSEp</a> methodology (<a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWappett2016">2016</a>), a statistical approach to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs from mutually exclusive relationships using bimodal distributions. This <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection methodology is also designed for <a 
href="#glo:gene expression">expression</a> analysis in cancer and is readily available as an (open-source) R package (<a 
href="#XWappett2014">Wappett</a>, <a 
href="#XWappett2014">2014</a>), a practice which facilitates adoption and testing of the methodology on the same datasets and simulations procedures as previously used for <a 
href="#glo:SLIPT">SLIPT</a>.
</p><!--l. 181--><p class="indent" >
<p>The <a 
href="#glo:BiSEp">BiSEp</a> package is designed for global testing of all potential gene pairs in the <a 
href="#glo:genome">genomes</a> for <a 
href="#glo:synthetic lethal">synthetic lethality</a> rather than focusing on the search space of potential partners of the query gene. This approach was unable to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs in the <a 
href="#glo:TCGA">TCGA</a> breast cancer <a 
href="#glo:gene expression">expression</a> dataset (<a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>). However, this may be due to stringent thresholds under the multiple testing of millions of potential gene pairs.</p>
</p><!--l. 183--><p class="indent" >
For a direct comparison with the query-based <a 
href="#glo:SLIPT">SLIPT</a> approach, the source code of the <a 
href="#glo:BiSEp">BiSEp</a> R functions was modified to test solely for the partners of a specific gene. This approach was still unable to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">CDH1 </span>in <a 
href="#glo:TCGA">TCGA</a> breast cancer <a 
href="#glo:gene expression">expression</a> data (<a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>), even with the detection thresholds for bimodality and significance greatly relaxed from those which the package defaults to.
</p><!--l. 185--><p class="indent" >
To circumvent multiple testing issues, <a 
href="#glo:BiSEp">BiSEp</a> only tests gene pairs for <a 
href="#glo:synthetic lethal">synthetic lethality</a> between genes with a detectable bimodal distribution. However, even with relaxed thresholds, bimodal distributions were not detectable in the normalised <a 
href="#glo:TCGA">TCGA</a> data (<a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>). Such normalisation <a 
href="#Xlimma">Ritchie <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#Xlimma">2015</a>) is standard practice for <a 
href="#glo:gene expression">expression</a> datasets generated from <a 
href="#glo:microarray">microarrays</a> or <a 
href="#glo:RNA-Seq">RNA-Seq</a> and therefore <a 
href="#glo:BiSEp">BiSEp</a> may not be appropriate to apply to this data. However, it is noted that <a 
href="#glo:BiSEp">BiSEp</a> may also use other data types such as <a 
href="#glo:DNA">DNA</a> copy number or cell line data for which it may be more applicable (<a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWappett2016">2016</a>).
</p><!--l. 187--><p class="indent" >
Nevertheless, attempts were made to test <a 
href="#glo:BiSEp">BiSEp</a> on simulated datasets with underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes (using the procedures described in Sections <a 
href="#x1-770002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a> and <a 
href="#x1-860002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>). However, <a 
href="#glo:BiSEp">BiSEp</a> was also unable to detect genes with bimodal distributions of genes (and thus unable to detect <a 
href="#glo:synthetic lethal">synthetic lethality</a>) in a limited number of computationally intensive simulations. Therefore investigations on a wider range of parameters were not performed. <a 
 id="x1-140001r255"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">6.2 </span> <a 
 id="x1-1410002"></a>Simulations with Graph Structures
</h3>
<!--l. 247--><p class="noindent" >
<p>The simulations of <a 
href="#glo:synthetic lethal">synthetic lethality</a> performed in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a> included correlated blocks of genes as a rudimentary representation of <a 
href="#glo:graph">pathway</a> structure and co-regulated genes. The simulation procedure was enhanced here to account for more complex <a 
href="#glo:graph">pathway</a> structures by sampling from multivariate normal distributions with correlation structure derived from <a 
href="#glo:graph">graph</a> structures (as described in Section <a 
href="#x1-860002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>). This approach enabled the simulation of <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways with known correlation structure and synthetic lethal partners (of a gene not in the pathway). Using this procedure, the performance of <a 
href="#glo:SLIPT">SLIPT</a> was evaluated under simple controlled correlation structures and complex correlations, such as those derived from biological networks (e.g., those described in Chapter <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>). The <a 
href="#glo:SLIPT">SLIPT</a> methodology was tested in artificially constructed networks to evaluate the effect of <a 
href="#glo:graph">pathway</a> structure on <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection These included large biologically feasible pathways to ensure that the <a 
href="#glo:SLIPT">SLIPT</a> methodology is robust</p>
under complex correlation structures and applicable to such complex <a 
href="#glo:genomic">genomics</a> data.
</p><!--l. 249--><p class="indent" >
These simulations combine the approach of prior simulation analyses (in Sections <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a> and <a 
href="#x1-1350001">6.1<!--tex4ht:ref: chapt5:compare_methods --></a>) with the <a 
href="#glo:graph">graph</a> structures for biological pathways (as used in Chapter <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>). This enabled testing whether subtle or large differences in <a 
href="#glo:graph">pathway</a> structure affect <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection, whether inhibiting relationships (or inverse correlations) between genes affect <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection, and whether <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection varies by which gene is <a 
href="#glo:synthetic lethal">synthetic lethal</a> and which genes are closely linked within the <a 
href="#glo:graph">pathway</a> structure. In addition, large numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes and biologically feasible numbers of genes (with many non-synthetic lethal genes) were tested to replicate the findings of Sections <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a> and <a 
href="#x1-1350001">6.1<!--tex4ht:ref: chapt5:compare_methods --></a> in correlated structures derived from pathway graphs, including examples of biological pathways from Reactome (<a 
href="#XReactome">Croft <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XReactome">2014</a>).
</p><!--l. 251--><p class="indent" >
Simple and more complex constructed <a 
href="#glo:graph">graph</a> structures were used to demonstrate the impact of <a 
href="#glo:graph">pathway</a> structure of the performance of <a 
href="#glo:SLIPT">SLIPT</a> for <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection in simulations. In addition, more complex constructed <a 
href="#glo:graph">graph</a> structures were compared to the <a 
href="#glo:PI3K">PI3K</a> and G<sub><span class="cmmi-8">αi</span> </sub> signalling pathways derived from Reactome which were used for simulation of <a 
href="#glo:graph">pathway</a> structures of biological complexity (as shown in Figure <a 
href="#x1-1200011">5.1<!--tex4ht:ref: fig:SL_Pathway_Pi3K --></a> and Appendix Figure <a 
href="#x1-1220014">5.4<!--tex4ht:ref: fig:SL_Pathway_GPCR --></a>). <a 
 id="x1-141001r264"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.2.1 </span> <a 
 id="x1-1420001"></a>Performance over Graph Structures
</h4>
<a 
 id="x1-142001r267"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">6.2.1.1 </span> <a 
 id="x1-1430001"></a>Simple Graph Structures
</h5>
<!--l. 263--><p class="noindent" >
<p>Simple pathway modules were used to test the effect of <a 
href="#glo:graph">pathway</a> structure on the performance of detecting <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners within <a 
href="#glo:graph">graph</a> structures. For an intial comparison, the <a 
href="#glo:graph">graph</a> structures (shown by Figure <a 
href="#x1-1430038">6.8<!--tex4ht:ref: fig:simple_graph1 --></a>) were used where a gene has one upstream regulator and two downstream (Figure <a 
href="#x1-143002r2">6.7b<!--tex4ht:ref: fig:simple_graph2:first --></a>) or a gene has two upstream regulators and one downstream gene (Figure <a 
href="#x1-143002r2">6.7b<!--tex4ht:ref: fig:simple_graph2:first --></a>). <a 
href="#glo:SLIPT">SLIPT</a> had a high performance in these simulations, detecting randomly selected <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in both of these small simple networks (shown in Figure <a 
href="#x1-14300810">6.10<!--tex4ht:ref: fig:simulation1205_randx_Graph1 --></a> and Appendix Figure <a 
href="#x1-1910052">K.2<!--tex4ht:ref: fig:simulation1205_randx_Graph2 --></a>).</p>
</p><!--l. 265--><p class="indent" >
<a 
 id="x1-1430038"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 268--><p class="noindent" >
</p><!--l. 274--><p class="noindent" >
<span class="fbox"><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph1".png /></span><a 
 id="x1-143001r1"></a> <span class="cmr-10x-x-109">Diverging</span> <span class="cmr-10x-x-109">Module</span><span class="fbox"><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph2".png /></span><a 
 id="x1-143002r2"></a> <span class="cmr-10x-x-109">Converging</span> <span class="cmr-10x-x-109">module</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 6.8: </span><span class="content"><strong><span class="cmbx-10x-x-109">Simple </span><a 
href="#glo:graph"><span class="cmbx-10x-x-109">graph</span></a> <span class="cmbx-10x-x-109">structures.</span></strong> <span class="cmr-10x-x-109">These simple </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structures used to demonstrate</span> <span class="cmr-10x-x-109">the simulation procedure. These are examples of a pathway diverging or converging</span> <span class="cmr-10x-x-109">respectively which enabled testing the importance of direction in </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">pathway</span></a> <span class="cmr-10x-x-109">structures. These</span> <span class="cmr-10x-x-109">are used with both activating and inhibiting relationships as shown.</span></span></p>
</div>
<!--tex4ht:label?: x1-1430038 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 312--><p class="indent" >
<a 
 id="x1-14300810"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 315--><p class="noindent" >
</p><!--l. 319--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1_ROC1_samplesx".png  /><a 
 id="x1-143004r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1_ROC2_samplesx".png  /><a 
 id="x1-143005r2"></a> <span class="cmr-10x-x-109">Receiver</span> <span class="cmr-10x-x-109">operating</span> <span class="cmr-10x-x-109">characteristic</span>
</p><!--l. 328--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph1".png /> <a 
 id="x1-143006r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1_AUC_samplesx".png  /><a 
 id="x1-143007r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 6.10: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a simple graph.</span></strong> <span class="cmr-10x-x-109">Simulation of </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic</span> <span class="cmr-10x-x-109">lethality</span></a> <span class="cmr-10x-x-109">was performed by sampling from a multivariate normal distribution generated from</span> <span class="cmr-10x-x-109">a diverging </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure. Performance of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">declines for more synthetic partners but</span> <span class="cmr-10x-x-109">this is mitigated by increased sample sizes (in darker colours). This manifests as a decline in</span> <span class="cmr-10x-x-109">specificity and the false positive rate. For each parameter value, 10,000 simulations were used.</span> <span class="cmr-10x-x-109">Colours of the </span><a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a> <span class="cmr-10x-x-109">curves in Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-143005r2"><span class="cmr-10x-x-109">6.9b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph1ROC:ROC --></a> <span class="cmr-10x-x-109">correspond to the parameters in Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-143007r4"><span class="cmr-10x-x-109">6.9d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph1ROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-14300810 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 339--><p class="indent" >
As previously observed (in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>), performance declined with higher numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes and lower sample sizes. However, the sensitivity of <a 
href="#glo:SLIPT">SLIPT</a> as a binary classifier was high. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> partners are often distinguishable for non-synthetic lethal genes, even in simple highly correlated networks. The small number of genes and their high correlation has an impact on the <a 
href="#glo:ROC">ROC</a> curves for higher numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners which are skewed compared to those observed previously. Specificity cannot be tested if all potential partner genes are <a 
href="#glo:synthetic lethal">synthetic lethal</a>, which limits the number of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes that can be tested.
</p><!--l. 341--><p class="indent" >
<p>These results were consistent between the pathway modules of diverging (as shown in Figure <a 
href="#x1-143009r1">6.11a<!--tex4ht:ref: fig:simulation1205_randx_Graph1cf2:Graph1 --></a>) and converging signals (as shown in Figure <a 
href="#x1-143010r2">6.11b<!--tex4ht:ref: fig:simulation1205_randx_Graph1cf2:Graph2 --></a>). The <a 
href="#glo:AUROC">AUROC</a> performance and underlying curves were strikingly similar between these <a 
href="#glo:graph">graph</a> structures (as shown in Figure <a 
href="#x1-14300810">6.10<!--tex4ht:ref: fig:simulation1205_randx_Graph1 --></a> and Appendix Figure <a 
href="#x1-1910052">K.2<!--tex4ht:ref: fig:simulation1205_randx_Graph2 --></a>). Thus the performance of <a 
href="#glo:SLIPT">SLIPT</a> was not perturbed by <a 
href="#glo:graph">pathway</a> structure, specifically the direction of pathway relationships since these <a 
href="#glo:graph">graph</a> structures also demonstrate pathways in opposite direction. In a direct comparison (shown in Figure <a 
href="#x1-143011r3">6.11c<!--tex4ht:ref: fig:simulation1205_randx_Graph1cf2:Compare --></a>), the performance of simulations did not differ across parameter values in these simple <a 
href="#glo:graph">graphs</a> and therefore <a 
href="#glo:SLIPT">SLIPT</a> is robust to pathway direction.</p>
</p><!--l. 344--><p class="indent" >
<a 
 id="x1-14301212"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 347--><p class="noindent" >
</p><!--l. 351--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph1".png /><a 
 id="x1-143009r1"></a> <span class="cmr-10x-x-109">Diverging</span> <span class="cmr-10x-x-109">Module</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph2".png /><a 
 id="x1-143010r2"></a> <span class="cmr-10x-x-109">Converging</span> <span class="cmr-10x-x-109">module</span>
</p><!--l. 359--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph1_ROC_Compare_Graph2(1)".png /><a 
 id="x1-143011r3"></a> <span class="cmr-10x-x-109">Performance</span> <span class="cmr-10x-x-109">between</span> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structures</span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.12: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations is similar in simple graphs.</span></strong> <span class="cmr-10x-x-109">The </span><a 
href="#glo:AUROC"><span class="cmr-10x-x-109">AUROC</span></a> <span class="cmr-10x-x-109">values for simulations of multivariate normal distributions based on each </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure</span> <span class="cmr-10x-x-109">yielded indistinguishable performance across parameter values in 10,000 simulations.</span></span></p>
</div>
<!--tex4ht:label?: x1-14301212 -->
                                                                                    

</div><hr class="endfloat" />
<a 
 id="x1-143013r270"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">6.2.1.2 </span> <a 
 id="x1-1440002"></a>Constructed Graph Structures
</h5>
<!--l. 379--><p class="noindent" >
<p>A more complex <a 
href="#glo:graph">graph</a> structure was used to examine the performance of detecting <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners with <a 
href="#glo:SLIPT">SLIPT</a> in simulated <a 
href="#glo:gene expression">expression</a> data with pathway correlation structures. For a simple chain of genes representing a very linear pathway (shown in Figure <a 
href="#x1-14400514">6.14<!--tex4ht:ref: fig:simulation1205_randx_Graph4 --></a>), the above findings were generally replicated. <a 
href="#glo:SLIPT">SLIPT</a> had high performance across parameter values in small networks but was still lower for higher numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes and lower sample sizes.</p>
</p><!--l. 381--><p class="indent" >
<a 
 id="x1-14400514"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 384--><p class="noindent" >
</p><!--l. 388--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4_ROC1_samplesx".png  /><a 
 id="x1-144001r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4_ROC2_samplesx".png  /><a 
 id="x1-144002r2"></a> <span class="cmr-10x-x-109">Receiver</span> <span class="cmr-10x-x-109">operating</span> <span class="cmr-10x-x-109">characteristic</span>
</p><!--l. 397--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4".png /> <a 
 id="x1-144003r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4_AUC_samplesx".png  /><a 
 id="x1-144004r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 6.14: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a pathway.</span></strong> <span class="cmr-10x-x-109">Simulation of </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic</span> <span class="cmr-10x-x-109">lethality</span></a> <span class="cmr-10x-x-109">was performed by sampling from a multivariate normal distribution generated from</span> <span class="cmr-10x-x-109">a </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">pathway</span></a> <span class="cmr-10x-x-109">structure. Performance of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">declines for more synthetic partners and lower</span> <span class="cmr-10x-x-109">sample sizes (in darker colours). For each parameter value, 10,000 simulations were used.</span> <span class="cmr-10x-x-109">Colours of the </span><a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a> <span class="cmr-10x-x-109">curves in Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-144002r2"><span class="cmr-10x-x-109">6.13b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph4ROC:ROC --></a> <span class="cmr-10x-x-109">correspond to the parameters in Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-144004r4"><span class="cmr-10x-x-109">6.13d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph4ROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-14400514 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 408--><p class="indent" >
When detecting <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes with <a 
href="#glo:SLIPT">SLIPT</a> as a binary classifier, the performance differences were primarily due to changes in specificity, as the small numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes still had highly significant p-values. Despite lower specificity and performance in <a 
href="#glo:ROC">ROC</a> curves, the accuracy increased and false positive rate decreased desirably with higher numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes due to the high sensitivity and the high proportion of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes detected. Therefore the use of adjusted p-values for <a 
href="#glo:SLIPT">SLIPT</a> as a binary classifier appear to be appropriate for detecting <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners, even in strongly correlated pathways, at least in these small-scale test cases.
</p><!--l. 410--><p class="indent" >
An artifact of these small test cases led to the skewed <a 
href="#glo:ROC">ROC</a> curves (as discussed in Section <a 
href="#x1-1430001">6.2.1.1<!--tex4ht:ref: chapt5:graphsim_performance_simple --></a>), which may be the result of the low number of non-synthetic lethal genes to identify as true negatives, affecting the accuracy of specificity. This issue does not occur in larger, more complex <a 
href="#glo:graph">graph</a> structures, even with modest total numbers of genes and high correlations (as shown in Section <a 
href="#x1-1480003">6.3<!--tex4ht:ref: chapt5:complex_graphs --></a>). This issue is unlikely to occur in large <a 
href="#glo:gene expression">expression</a> datasets with many non-synthetic lethal genes, as shown previously (in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a> and <a 
href="#x1-1430001">6.2.1.1<!--tex4ht:ref: chapt5:graphsim_performance_simple --></a>) with <a 
href="#glo:graph">graph</a> structures in larger datasets (in Section <a 
href="#x1-1470004">6.2.4<!--tex4ht:ref: chapt5:graphsim_performance_20K --></a>). <a 
 id="x1-144006r269"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.2.2 </span> <a 
 id="x1-1450002"></a>Performance with Inhibitions
</h4>
<!--l. 420--><p class="noindent" >
<p>Simulations of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in <a 
href="#glo:gene expression">expression</a> data were also performed with correlation structures derived from <a 
href="#glo:graph">graphs</a> containing inhibiting relationships (as are commonplace in biological pathways) which produce negative correlations. As shown in Figure <a 
href="#x1-14500516">6.16<!--tex4ht:ref: fig:simulation1205_randx_Graph1i --></a>, these are not an issue for detection by <a 
href="#glo:SLIPT">SLIPT</a>. Rather, the <a 
href="#glo:SLIPT">SLIPT</a> procedure performs well on simple graph modules with highly negative correlations. With <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection based on p-value (adjusted by <a 
href="#glo:FDR">FDR</a>), there was higher specificity, higher accuracy, and lower false positive rate in an inhibitory graph than the same graph with activating relationships (as shown by Figure <a 
href="#x1-14300810">6.10<!--tex4ht:ref: fig:simulation1205_randx_Graph1 --></a>).</p>
</p><!--l. 422--><p class="indent" >
<a 
 id="x1-14500516"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 425--><p class="noindent" >
</p><!--l. 429--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1i_ROC1_samplesx".png  /><a 
 id="x1-145001r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1i_ROC2_samplesx".png  /><a 
 id="x1-145002r2"></a> <span class="cmr-10x-x-109">Receiver</span> <span class="cmr-10x-x-109">operating</span> <span class="cmr-10x-x-109">characteristic</span>
</p><!--l. 438--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph1i".png /> <a 
 id="x1-145003r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph1i_AUC_samplesx".png  /><a 
 id="x1-145004r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 6.16: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a simple graph with inhibition.</span></strong> <span class="cmr-10x-x-109">Simulation of </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a> <span class="cmr-10x-x-109">was performed by sampling from a multivariate normal</span> <span class="cmr-10x-x-109">distribution generated from an inhibiting graph. Performance of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">declined for more</span> <span class="cmr-10x-x-109">synthetic partners and lower sample sizes. For each parameter value, 10,000 simulations</span> <span class="cmr-10x-x-109">were used. Colours of the </span><a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a> <span class="cmr-10x-x-109">curves in Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-145002r2"><span class="cmr-10x-x-109">6.15b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph1iROC:ROC --></a> <span class="cmr-10x-x-109">correspond to the parameters in</span> <span class="cmr-10x-x-109">Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-145004r4"><span class="cmr-10x-x-109">6.15d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph1iROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-14500516 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 449--><p class="indent" >
The <a 
href="#glo:ROC">ROC</a> curves for an inhibiting graph also showed consistently high specificity, irrespective of detection threshold, with only the upper extreme of the curve exhibiting a skew below random performance (as shown in Figure <a 
href="#x1-14500516">6.16<!--tex4ht:ref: fig:simulation1205_randx_Graph1i --></a>). Nevertheless, the <a 
href="#glo:AUROC">AUROC</a> values show a high performance across parameter values, particularly avoiding issues with higher numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners (as observed in Section <a 
href="#x1-1430001">6.2.1.1<!--tex4ht:ref: chapt5:graphsim_performance_simple --></a>). However, performance was marginally lower for higher numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes to detect and lower sample sizes, consistent with previously observations.
</p><!--l. 451--><p class="indent" >
Negatively correlated simulated datasets are also unperturbed by minor differences in <a 
href="#glo:graph">graph</a> structure, such as changing in the direction of the graph module. As observed for activating relationships in these graph modules, the performance was highly concordant between the graph modules (shown by similar results in Figures <a 
href="#x1-14500516">6.16<!--tex4ht:ref: fig:simulation1205_randx_Graph1i --></a> and <a 
href="#x1-1920054">K.4<!--tex4ht:ref: fig:simulation1205_randx_Graph2i --></a>).
</p><!--l. 453--><p class="indent" >
Detection of <a 
href="#glo:synthetic lethal">synthetic lethality</a> by <a 
href="#glo:SLIPT">SLIPT</a> in simulated data with inhibiting relationships outperforms simulations with activating relationships in the same <a 
href="#glo:graph">graph</a> structure (as shown in Figure <a 
href="#x1-14500918">6.18<!--tex4ht:ref: fig:simulation1205_randx_Graph1cf1i --></a>). Thus <a 
href="#glo:SLIPT">SLIPT</a> was robust in <a 
href="#glo:gene expression">gene expression</a> datasets with inverse correlations and performed well in them, at least in simple test cases. This is important because such relationships occur frequently in biological pathways and therefore the findings inferred from <a 
href="#glo:graph">graph</a> structures without inhibiting relationships are a conservative estimate.
</p><!--l. 455--><p class="indent" >
<p>The <a 
href="#glo:SLIPT">SLIPT</a> methodology likely performs better in biological pathways (which contain negative correlations) than the <a 
href="#glo:graph">graph</a> structures discussed previously (in Section <a 
href="#x1-1420001">6.2.1<!--tex4ht:ref: chapt5:graphsim_performance --></a>). This is likely since negative correlations lead to <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners and inversely correlated genes which are positively correlated with the query gene. As previously shown, the <a 
href="#glo:SLIPT">SLIPT</a> methodology performs well with specificity against positively correlated query genes (in Sections <a 
href="#x1-820002">3.3.2.2<!--tex4ht:ref: chapt2:simulation_mvtnorm_query_cor --></a> and <a 
href="#x1-1390001">6.1.2.1<!--tex4ht:ref: chapt5:compare_correlation --></a>).</p>
</p><!--l. 457--><p class="indent" >
<a 
 id="x1-14500918"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 460--><p class="noindent" >
</p><!--l. 465--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph1".png /><a 
 id="x1-145006r1"></a> <span class="cmr-10x-x-109">Activating</span> <span class="cmr-10x-x-109">Graph</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph1i".png /><a 
 id="x1-145007r2"></a> <span class="cmr-10x-x-109">Inhibiting</span> <span class="cmr-10x-x-109">Graph</span>
</p><!--l. 475--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph1_ROC_Compare(1)".png /><a 
 id="x1-145008r3"></a> <span class="cmr-10x-x-109">Performance</span> <span class="cmr-10x-x-109">between</span> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structures</span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.18: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance is higher on a simple inhibiting graph.</span></strong> <span class="cmr-10x-x-109">The </span><a 
href="#glo:AUROC"><span class="cmr-10x-x-109">AUROC</span></a> <span class="cmr-10x-x-109">values</span> <span class="cmr-10x-x-109">for simulations of multivariate normal distributions based on inhibitions in the </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure</span> <span class="cmr-10x-x-109">yielded consistently higher performance across parameter values in 10,000 simulations.</span></span></p>
</div>
<!--tex4ht:label?: x1-14500918 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 488--><p class="indent" >
<p>Similarly, more complex <a 
href="#glo:graph">graph</a> structures with entirely inhibiting relationships (negative correlations) also perform desirably with <a 
href="#glo:SLIPT">SLIPT</a> as a binary classifier and have high performance across increasing numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes, particularly for sufficiently high sample sizes (as shown by Appendix Figure <a 
href="#x1-1920106">K.6<!--tex4ht:ref: fig:simulation1205_randx_Graph4i --></a>). However, this is not necessarily the case for <a 
href="#glo:graph">graph</a> structures with a combination of activating and inhibiting relationships (i.e., containing positive and negative correlations). As shown by Appendix Figure <a 
href="#x1-1920158">K.8<!--tex4ht:ref: fig:simulation1205_randx_Graph4i2 --></a>, such a mixed <a 
href="#glo:graph">network</a> structure does not necessarily have high performance across parameters as observed for purely inhibiting networks.</p>
</p><!--l. 490--><p class="indent" >
<a 
 id="x1-14501420"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 493--><p class="noindent" >
</p><!--l. 498--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4i2_ROC1_samplesx".png  /><a 
 id="x1-145010r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4i2_ROC2_samplesx".png  /><a 
 id="x1-145011r2"></a> <span class="cmr-10x-x-109">Receiver</span> <span class="cmr-10x-x-109">operating</span> <span class="cmr-10x-x-109">characteristic</span>
</p><!--l. 509--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4i2".png /> <a 
 id="x1-145012r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4i2_AUC_samplesx".png  /><a 
 id="x1-145013r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 6.20: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a constructed graph with inhibition.</span></strong> <span class="cmr-10x-x-109">Simulation of </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a> <span class="cmr-10x-x-109">was performed by sampling from a multivariate normal</span> <span class="cmr-10x-x-109">distribution generated from </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">pathway</span></a> <span class="cmr-10x-x-109">structure with a combination of inhibitions. Performance</span> <span class="cmr-10x-x-109">of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">declines for more synthetic partners and lower sample sizes. For each parameter</span> <span class="cmr-10x-x-109">value, 10,000 simulations were used.</span></span></p>
</div>
<!--tex4ht:label?: x1-14501420 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 521--><p class="indent" >
These still appear to have desirably high sensitivity, high accuracy, and low false positive rate for detecting more <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes, despite poor specificity. The <a 
href="#glo:ROC">ROC</a> curves were particularly skewed for high proportions of the network being <a 
href="#glo:synthetic lethal">synthetic lethal</a> and may stem from low numbers of true negative genes to detect (as discussed in Section <a 
href="#x1-1430001">6.2.1.1<!--tex4ht:ref: chapt5:graphsim_performance_simple --></a>). In a direct comparison of performance (shown in Figure <a 
href="#x1-14502022">6.22<!--tex4ht:ref: fig:simulation1205_randx_Graph4cf4i --></a>), the purely inhibiting graph had consistently higher performance than the activating one, as observed for simpler <a 
href="#glo:graph">graphs</a> (as shown in Figure <a 
href="#x1-14500918">6.18<!--tex4ht:ref: fig:simulation1205_randx_Graph1cf1i --></a>).
</p><!--l. 523--><p class="indent" >
<p>In contrast, the combination of activating and inhibiting relationships had slightly lower performance across parameters compared to the same <a 
href="#glo:graph">graph</a> structure with activating relationships. Therefore correlation structure can impact on the performance of <a 
href="#glo:SLIPT">SLIPT</a> in a graph network, in either direction, specifically the addition of negative correlations. However, this may be an artifact of the simulation procedure as <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes from the correlation structure were randomly selected (without regard to their relationships), with the query gene added to ensure that conditions for <a 
href="#glo:synthetic lethal">synthetic lethal</a> relationships were met.</p>
</p><!--l. 525--><p class="indent" >
<a 
 id="x1-14502022"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 528--><p class="noindent" >
</p><!--l. 533--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4".png /><a 
 id="x1-145015r1"></a> <span class="cmr-10x-x-109">Activating</span> <span class="cmr-10x-x-109">Graph</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4i".png /><a 
 id="x1-145016r2"></a> <span class="cmr-10x-x-109">Inhibiting</span> <span class="cmr-10x-x-109">Graph</span> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4i2".png /><a 
 id="x1-145017r3"></a> <span class="cmr-10x-x-109">Mixed</span> <span class="cmr-10x-x-109">Graph</span>
</p><!--l. 547--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph4_ROC_Compare(1)".png /><a 
 id="x1-145018r4"></a> <span class="cmr-10x-x-109">Performance</span> <span class="cmr-10x-x-109">between</span> <span class="cmr-10x-x-109">Graphs</span> <span class="cmr-10x-x-109">(a)</span> <span class="cmr-10x-x-109">and</span> <span class="cmr-10x-x-109">(b)</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph4_ROC_Compare(4)".png /><a 
 id="x1-145019r5"></a> <span class="cmr-10x-x-109">Performance</span> <span class="cmr-10x-x-109">between</span> <span class="cmr-10x-x-109">Graphs</span> <span class="cmr-10x-x-109">(a)</span> <span class="cmr-10x-x-109">and</span> <span class="cmr-10x-x-109">(c)</span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.22: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance is affected by inhibition in graphs.</span></strong> <span class="cmr-10x-x-109">The </span><a 
href="#glo:AUROC"><span class="cmr-10x-x-109">AUROC</span></a> <span class="cmr-10x-x-109">values</span> <span class="cmr-10x-x-109">for simulations of multivariate normal distributions based on </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure containing only</span> <span class="cmr-10x-x-109">inhibitions in the </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure yielded consistently higher performance across parameter</span> <span class="cmr-10x-x-109">values in 10,000 simulations. A combination of activating and inhibiting relationships had</span> <span class="cmr-10x-x-109">lower performance but was more similar to the activating graph.</span></span></p>
</div>
<!--tex4ht:label?: x1-14502022 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 564--><p class="indent" >
This system for simulating inhibitory pathways is not ideal since it can lead to <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene combinations, by randomly selecting them, which are unlikely to occur in biological pathways. These randomly selected <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes may account for the detection results being suboptimal (i.e., difficult to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners) compared to previous investigations. It is expected that inversely correlated synthetic partner genes will be highly expressed in a mutually exclusive manner such that at least one of them will be compensating for loss of the query gene in most samples, leading to a weak <a 
href="#glo:synthetic lethal">synthetic lethal</a> signature in <a 
href="#glo:gene expression">expression</a> data in this case. Furthermore, this case may not be representative of empirical biological data with <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <a 
href="#glo:tumour suppressor">tumour suppressor</a> genes which are commonly inversely correlated to the query gene (to some extent) and therefore it is unlikely that they are strongly negative correlated with each other, unless they are <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of each other as well. It is plausible that many <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes will serve to separately compensate for the loss of query gene function and be positively correlated with each other. Nonetheless, these simulations demonstrate that correlation structure (particularly negative correlations) have an impact on the detection of <a 
href="#glo:synthetic lethal">synthetic lethality</a>. However, <a 
href="#glo:SLIPT">SLIPT</a> was still able to perform well across <a 
href="#glo:graph">graphs</a> with different activating and inhibiting relationships, and the perturbations in performance were marginal, particularly those reducing performance compared to an activating network. <a 
 id="x1-145021r289"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.2.3 </span> <a 
 id="x1-1460003"></a>Synthetic Lethality across Graph Structures
</h4>
<!--l. 573--><p class="noindent" >
<p><a 
href="#glo:synthetic lethal">Synthetic lethal</a> genes were distinguishable from highly correlated genes in simple cases (as shown by <a 
href="#glo:ROC">ROC</a> analysis). However, correlated genes may lead to low specificity and high false positive rates. Negative correlations do not affect specificity this way but they may perturb the correlation structure between <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes, making it difficult to detect many of them with high sensitivity. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> genes have been selected randomly in simulations so far, which is a limited approach. To examine the impact of pathway relationships in more detail, specific genes were selected to be <a 
href="#glo:synthetic lethal">synthetic lethal</a> within a network over replicate simulations. These simulations with a fixed <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene were performed to demonstrate their impact on the detection of other genes in the network.</p>
</p><!--l. 577--><p class="indent" >
<p>For detection of a <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene in an activating <a 
href="#glo:graph">graph</a> structure (as shown in Figure <a 
href="#x1-146001r1">6.23a<!--tex4ht:ref: fig:simulation1206_Str_randx_Graph4D:Graph4 --></a>), the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values were clearly distinguishable from other genes (shown in Figure <a 
href="#x1-146003r3">6.23c<!--tex4ht:ref: fig:simulation1206_Str_randx_Graph4D:vioplot --></a>). Simulations were performed for each gene being the <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner. For each <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene, it had the highest <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> value amongst 20,000 genes, including the highly correlated graph network (as shown in Appendix Figure <a 
href="#x1-19300710">K.10<!--tex4ht:ref: fig:simulation1206_Str_randx_Graph4 --></a>). Despite optimal performance for <a 
href="#glo:SLIPT">SLIPT</a> detecting one <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene in a <a 
href="#glo:ROC">ROC</a> curve (as shown in Figure <a 
href="#x1-14400514">6.14<!--tex4ht:ref: fig:simulation1205_randx_Graph4 --></a>), irrespective of detection threshold, the highly correlated genes would be detected as false positives by <a 
href="#glo:SLIPT">SLIPT</a> as a binary classifier (as described in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>). In particular, the genes that were adjacent in the <a 
href="#glo:pathway">pathway</a> to the <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene “D” had high test statistics which could be false positives (as shown in Figure <a 
href="#x1-146003r3">6.23c<!--tex4ht:ref: fig:simulation1206_Str_randx_Graph4D:vioplot --></a>). This was not specific to example of gene “D”, with the neighbouring genes of each <a 
href="#glo:synthetic lethal">synthetic lethal</a> having higher <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values (as shown in Appendix Figure <a 
href="#x1-19300710">K.10<!--tex4ht:ref: fig:simulation1206_Str_randx_Graph4 --></a>).</p>
</p><!--l. 582--><p class="indent" >
<a 
 id="x1-14600524"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 584--><p class="noindent" >
</p><!--l. 587--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4D".png /><a 
 id="x1-146001r1"></a> <span class="cmr-10x-x-109">Activating</span> <span class="cmr-10x-x-109">Graph</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4iD".png /><a 
 id="x1-146002r2"></a> <span class="cmr-10x-x-109">Inhibiting</span> <span class="cmr-10x-x-109">Graph</span>
</p><!--l. 595--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(4)".png /><a 
 id="x1-146003r3"></a> <span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span> </sup> <span class="cmr-10x-x-109">distribution</span> <span class="cmr-10x-x-109">for</span> <span class="cmr-10x-x-109">Activating</span> <span class="cmr-10x-x-109">Graph</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i_ROC_samples_SLstry_vioplot(4)".png /><a 
 id="x1-146004r4"></a> <span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">distribution</span> <span class="cmr-10x-x-109">for</span> <span class="cmr-10x-x-109">Inhibiting</span> <span class="cmr-10x-x-109">Graph</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 6.24: </span><span class="content"><strong><span class="cmbx-10x-x-109">Detection of synthetic lethality within a graph structure.</span></strong> <span class="cmr-10x-x-109">The gene</span> <span class="cmr-10x-x-109">“D” was designated to be </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">and the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">value from </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">was computed for</span> <span class="cmr-10x-x-109">each gene across each </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure. The </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">values were computed in 100 simulations of</span> <span class="cmr-10x-x-109">datasets of 20,000 genes including the </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure and 1000 samples. Adjacent genes</span> <span class="cmr-10x-x-109">exhibited lower </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">values with inhibiting relationships.</span></span></p>
</div>
<!--tex4ht:label?: x1-14600524 -->
<p><a 
 id="x1-146006r23"></a></p>
</div><hr class="endfloat" />
<!--l. 642--><p class="indent" >
The <a 
href="#glo:synthetic lethal">synthetic lethal</a> signal propagates from the true <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene throughout the network. As such, the genes nearer to (i.e., more highly correlated with) the true <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene had higher test statistics and were more likely to be detected by <a 
href="#glo:SLIPT">SLIPT</a> as false positives. The adjacent genes of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners being false positives may account for the higher concordance of <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways than genes between <a 
href="#glo:SLIPT">SLIPT</a> in <a 
href="#glo:TCGA">TCGA</a> data (<a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>) and the <a 
href="#glo:siRNA">siRNA</a> screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) than individual gene results (in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>). False positive genes are more likely to be involved in a <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathway, being correlated with a true <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> pathways are likely to contain many genes detected by <a 
href="#glo:SLIPT">SLIPT</a>, giving a consensus in the pathway over-representation analysis. <a 
href="#glo:SLIPT">SLIPT</a> is also able to detect true <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners or prioritise those most likely to be experimentally validated. Genes with the strongest support (i.e, higher <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values and more significant p-values) are more likely to be the underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene.
</p><!--l. 644--><p class="indent" >
The immediately adjacent genes in an inhibiting graph (Figure <a 
href="#x1-146002r2">6.23b<!--tex4ht:ref: fig:simulation1206_Str_randx_Graph4iD:Graph4i --></a>) did not have elevated <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> test statistics or a significant inverse effect (as shown in Figure <a 
href="#x1-146004r4">6.23d<!--tex4ht:ref: fig:simulation1206_Str_randx_Graph4iD:vioplot --></a>). Therefore true <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners were highly distinguishable from other genes with inhibiting relationships. This was shown for each gene in the <a 
href="#glo:graph">graph</a> structure as the <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner (shown in Appendix Figure <a 
href="#x1-193024r13">K.13<!--tex4ht:ref: fig:simulation1206_Str_randx_Graph4i --></a>). These results support <a 
href="#glo:SLIPT">SLIPT</a> as an appropriate approach to distinguish <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in biological pathways (which frequently have inhibitions), including those relevant to cancer growth and inhibition.
</p><!--l. 652--><p class="indent" >
The 2<sup class="textsuperscript"><span class="cmr-10x-x-109">nd</span> </sup> degree neighbours of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene still exhibited moderate <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values (and are moderately correlated with the <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene). These genes could be false positives, as shown for an activating <a 
href="#glo:graph">graph</a> structure, although inhibitory relationships (i.e., negative correlations) lead to more differences in test statistics between correlated genes and underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners (as shown in Appendix Figure <a 
href="#x1-193024r13">K.13<!--tex4ht:ref: fig:simulation1206_Str_randx_Graph4i --></a>). Simulations in a graph containing a combination of activating and inhibiting relationships exhibits either of these <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> profiles, depending on which gene is <a 
href="#glo:synthetic lethal">synthetic lethal</a> and the relationships between genes (as shown in Appendix Figure <a 
href="#x1-193036r14">K.14<!--tex4ht:ref: fig:simulation1206_Str_randx_Graph4i2 --></a>). In this case, the <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene is distinguishable and inhibitory relationships make it easier to detect with <a 
href="#glo:SLIPT">SLIPT</a>.
</p><!--l. 679--><p class="indent" >
<p>These results contrast with randomly selecting multiple <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes (as shown in Figure <a 
href="#x1-14502022">6.22<!--tex4ht:ref: fig:simulation1205_randx_Graph4cf4i --></a>), where the performance of <a 
href="#glo:SLIPT">SLIPT</a> was impeded by the</p>
inhibitory relationships between <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners. The randomly selected <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes, with negative correlations between them, which had poor performance due to an artifact in the simulation process resulting in biologically implausible <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes. The results with one <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner were sufficient to show the impact of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners on neighbouring (correlated) genes. It is plausible that the <a 
href="#glo:synthetic lethal">synthetic lethal</a> signatures in <a 
href="#glo:gene expression">expression</a> data would propagate similarly through a network from multiple <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners. <a 
 id="x1-146007r310"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.2.4 </span> <a 
 id="x1-1470004"></a>Performance within a Large Simulated Datasets
</h4>
<!--l. 691--><p class="noindent" >
<p>The performance of <a 
href="#glo:SLIPT">SLIPT</a> with higher numbers of true partners to detect may have been affected by the high proportion of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners (i.e., fewer true negatives) in small networks (as noted in Section <a 
href="#x1-1430001">6.2.1.1<!--tex4ht:ref: chapt5:graphsim_performance_simple --></a>). The performance of <a 
href="#glo:SLIPT">SLIPT</a> increased with the addition of more non-synthetic lethal genes, particularly the specificity (as shown in Sections <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a> and <a 
href="#x1-1350001">6.1<!--tex4ht:ref: chapt5:compare_methods --></a>). The correlated genes from <a 
href="#glo:graph">graph</a> structures (as used in Section <a 
href="#x1-1420001">6.2.1<!--tex4ht:ref: chapt5:graphsim_performance --></a>) were included in a larger simulated dataset to assess the performance of <a 
href="#glo:SLIPT">SLIPT</a> for a <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathway in the context of thousands of genes, as occurs in <a 
href="#glo:gene expression">expression</a> datasets.</p>
</p><!--l. 700--><p class="indent" >
<a 
 id="x1-14700525"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 703--><p class="noindent" >
</p><!--l. 708--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph1_ROC1_samplesx".png  /><a 
 id="x1-147001r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph1_ROC2_samplesx".png  /><a 
 id="x1-147002r2"></a> <span class="cmr-10x-x-109">Receiver</span> <span class="cmr-10x-x-109">operating</span> <span class="cmr-10x-x-109">characteristic</span>
</p><!--l. 719--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph1".png /> <a 
 id="x1-147003r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Graph1_AUC_samplesx".png  /><a 
 id="x1-147004r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 6.25: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations including a simple graph.</span></strong> <span class="cmr-10x-x-109">Simulation</span> <span class="cmr-10x-x-109">of </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a> <span class="cmr-10x-x-109">was performed by sampling from a multivariate normal distribution</span> <span class="cmr-10x-x-109">(without correlation structure apart from the graph shown). Performance of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">was high</span> <span class="cmr-10x-x-109">across parameters for detecting </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a> <span class="cmr-10x-x-109">in the </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure within a larger</span> <span class="cmr-10x-x-109">dataset. The sensitivity decreased for a greater number of true positives to detect but the</span> <span class="cmr-10x-x-109">specificity remained high with a low false positive rate.</span></span></p>
</div>
<!--tex4ht:label?: x1-14700525 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 732--><p class="indent" >
The simulations performed in Section <a 
href="#x1-1430001">6.2.1.1<!--tex4ht:ref: chapt5:graphsim_performance_simple --></a> were replicated within a dataset of 20,000 genes with the remainder being composed of non-synthetic lethal genes without correlation structure. The specificity in a higher number of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes did not affect performance in a simple <a 
href="#glo:graph">graph</a> structure (as shown in Figure <a 
href="#x1-14700525">6.25<!--tex4ht:ref: fig:simulation1207_randx_Graph1 --></a>). For a graph of highly correlated genes within a <a 
href="#glo:gene expression">gene expression</a> dataset, <a 
href="#glo:SLIPT">SLIPT</a> had high the performance detecting of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes in the network within a larger dataset. In this case, a reduction in sensitivity resulted in poorer performance. A high number of non-synthetic lethal genes were correctly identified, with a low false positive rate and high accuracy. Thus the use of stringent <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> p-value thresholds (adjusted by <a 
href="#glo:FDR">FDR</a>) are suitable for testing for <a 
href="#glo:synthetic lethal">synthetic lethality</a> in <a 
href="#glo:gene expression">gene expression</a> data across the number of genes in human and cancer data.
</p><!--l. 734--><p class="indent" >
In a direct comparison with simulations in the <a 
href="#glo:graph">graph</a> structure alone (as performed in Section <a 
href="#x1-1430001">6.2.1.1<!--tex4ht:ref: chapt5:graphsim_performance_simple --></a>), detection of <a 
href="#glo:synthetic lethal">synthetic lethality</a> with <a 
href="#glo:SLIPT">SLIPT</a> performed consistently better in a larger dataset with many true negative genes to detect (as shown in Figure <a 
href="#x1-14701027">6.27<!--tex4ht:ref: fig:simulation1205_randx_Graph1cf20K --></a>). The <a 
href="#glo:SLIPT">SLIPT</a> methodology had a high specificity and low false positive rate, which is desirable. <a 
href="#glo:SLIPT">SLIPT</a> is therefore applicable to large <a 
href="#glo:gene expression">gene expression</a> datasets, where these are important considerations since the number of negative genes often vastly outnumbers the number of positive genes to detect.
</p><!--l. 738--><p class="indent" >
<p>Performance was not necessarily higher with more non-synthetic lethal genes in an inhibiting <a 
href="#glo:graph">graph</a> structure. The performance of simulations of an entirely inhibiting <a 
href="#glo:graph">graph</a> structure did not improve within a larger dataset. Rather, the performance in the inhibiting <a 
href="#glo:graph">graph</a> structure was similar to simulations of the <a 
href="#glo:graph">graph</a> structure in isolation. Biological pathways commonly contain inhibiting relationships (and inverse correlations), although they are unlikely to occur across an entire pathway. In <a 
href="#glo:graph">graph</a> structures with inhibitions included in a larger dataset, the performance of <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection by <a 
href="#glo:SLIPT">SLIPT</a> was sometimes higher than in <a 
href="#glo:graph">graph</a> structure simulated alone (as shown in Figure <a 
href="#x1-14701529">6.29<!--tex4ht:ref: fig:simulation1205_randx_Graph4cf20K --></a>). However, these did not perform as well as the equivalent <a 
href="#glo:graph">graph</a> structures without inhibitory relationships within a similar dataset. It is expected that the findings based on these simulations of genes with <a 
href="#glo:graph">pathway</a> structures in smaller datasets (as described in Section <a 
href="#x1-1420001">6.2.1<!--tex4ht:ref: chapt5:graphsim_performance --></a>) will be relevant to larger datasets. The simulation results in these inhibiting <a 
href="#glo:graph">graph</a> structures perform comparably or higher with more non-synthetic lethal genes to distinguish from them even with inhibitory relationships within the <a 
href="#glo:graph">graph</a> structure</p>
</p><!--l. 750--><p class="indent" >
<a 
 id="x1-14701027"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 753--><p class="noindent" >
</p><!--l. 758--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph1".png /><a 
 id="x1-147006r1"></a> <span class="cmr-10x-x-109">Diverging</span> <span class="cmr-10x-x-109">Module</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph2".png /><a 
 id="x1-147007r2"></a> <span class="cmr-10x-x-109">Converging</span> <span class="cmr-10x-x-109">module</span>
</p><!--l. 772--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph1_ROC_Compare(2)".png /><a 
 id="x1-147008r3"></a> <span class="cmr-10x-x-109">Performance</span> <span class="cmr-10x-x-109">in</span> <span class="cmr-10x-x-109">Diverging</span> <span class="cmr-10x-x-109">Module</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph2_ROC_Compare(2)".png /><a 
 id="x1-147009r4"></a> <span class="cmr-10x-x-109">Performance</span> <span class="cmr-10x-x-109">in</span> <span class="cmr-10x-x-109">Converging</span> <span class="cmr-10x-x-109">module</span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.27: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance on a simple graph improves with more genes.</span></strong> <span class="cmr-10x-x-109">Simulations</span> <span class="cmr-10x-x-109">were performed with each of the </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structures to detect </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners within</span> <span class="cmr-10x-x-109">them. In either structure, performance of detection in a dataset containing on the </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure (in colour) was lower than testing the </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure within a larger dataset of</span> <span class="cmr-10x-x-109">non-synthetic lethal genes (without correlations).</span></span></p>
</div>
<!--tex4ht:label?: x1-14701027 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 829--><p class="indent" >
<a 
 id="x1-14701529"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 832--><p class="noindent" >
</p><!--l. 837--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4".png /><a 
 id="x1-147011r1"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4i2".png /><a 
 id="x1-147012r2"></a> <span class="cmr-10x-x-109">Inhibiting</span> <span class="cmr-10x-x-109">Structure</span>
</p><!--l. 851--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph4_ROC_Compare(2)".png /><a 
 id="x1-147013r3"></a> <span class="cmr-10x-x-109">Performance</span> <span class="cmr-10x-x-109">in</span> <span class="cmr-10x-x-109">Graph</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Graph4_ROC_Compare(5)".png /><a 
 id="x1-147014r4"></a> <span class="cmr-10x-x-109">Performance</span> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">Inhibitions</span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.29: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance on an inhibiting graph improves with more genes.</span></strong> <span class="cmr-10x-x-109">Simulations were performed in a </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure with activating and inhibiting relationships</span> <span class="cmr-10x-x-109">to detect </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners within them. In contrast to an activating graph,</span> <span class="cmr-10x-x-109">performance of detection in a dataset containing only the </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure (in colour) was as</span> <span class="cmr-10x-x-109">much lower than testing the </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure within a larger dataset of non-synthetic lethal</span> <span class="cmr-10x-x-109">genes (without correlations) in an inhibiting </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure with negative correlations.</span></span></p>
</div>
<!--tex4ht:label?: x1-14701529 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 864--><p class="indent" >
This poorer performance of inhibitory <a 
href="#glo:graph">graph</a> structures may be due to highly negatively correlated genes being false positives. These genes will be positively correlated with the query gene if they are negatively correlated with a <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner (i.e., within a <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathway). The <a 
href="#glo:SLIPT">SLIPT</a> procedure performs well at distinguishing these positively correlated genes, as previously shown (in Sections <a 
href="#x1-820002">3.3.2.2<!--tex4ht:ref: chapt2:simulation_mvtnorm_query_cor --></a> and <a 
href="#x1-1370001">6.1.1.1<!--tex4ht:ref: chapt5:compare_chisq_query_cor --></a>). These false positives will also be a minority amongst a larger dataset of non-synthetic lethal genes without correlation to the query or <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes.
</p><!--l. 866--><p class="indent" >
It more likely that the poorer performance stems from negative correlations between <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes which makes them more difficult to individually detect (as observed in Section <a 
href="#x1-1450002">6.2.2<!--tex4ht:ref: chapt5:graphsim_performance_inhib --></a>). As discussed in Section <a 
href="#x1-1460003">6.2.3<!--tex4ht:ref: chapt5:graphsim_str --></a>, this is likely an artifact of the simulation procedure selecting random <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes with strong inhibitory relationships between them. Therefore the poorer performance for inhibiting <a 
href="#glo:graph">graphs</a> within larger datasets is not cause for concern because the cases where <a 
href="#glo:SLIPT">SLIPT</a> performs poorly are likely to be combinations of simulated <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes that are not likely to occur within biological pathways. This simulation procedure has included higher-order <a 
href="#glo:synthetic lethal">synthetic lethal</a> to produce the weakest signal of <a 
href="#glo:synthetic lethal">synthetic lethality</a> for individual partner genes which are still detectable by <a 
href="#glo:SLIPT">SLIPT</a>. <a 
 id="x1-147016r268"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">6.3 </span> <a 
 id="x1-1480003"></a>Simulations in More Complex Graph Structures
</h3>
<!--l. 876--><p class="noindent" >
<p>Investigations with simulations based on <a 
href="#glo:graph">graph</a> structures were extended to larger <a 
href="#glo:graph">graphs</a>, enabling more synthetic lethal genes within a pathway and modelling the complexity of a biological pathway. Sensitivity declines over a greater range for the number of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in a larger network with a trade-off with specificity (as shown in Appendix Figures <a 
href="#x1-19400516">K.16<!--tex4ht:ref: fig:simulation1205_randx_Graph6 --></a>–<a 
href="#x1-19401520">K.20<!--tex4ht:ref: fig:simulation1205_randx_Graph5 --></a>). However, the accuracy declined for greater numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners and the false positive rate peaks at intermediate values. In this range, differencse between simulations varied with greater sample size. The <a 
href="#glo:AUROC">AUROC</a> results were similar between these more complex <a 
href="#glo:graph">graph</a> structures, although the larger <a 
href="#glo:graph">graph</a> (Appendix Figure <a 
href="#x1-19401520">K.20<!--tex4ht:ref: fig:simulation1205_randx_Graph5 --></a>) differed in sensitivity and specificity for <a 
href="#glo:SLIPT">SLIPT</a> as a binary classifier. This difference be due to different proportions of <a 
href="#glo:synthetic lethal">synthetic lethal</a> and non-synthetic lethal genes to</p>
detect, since these <a 
href="#glo:graph">graphs</a> (as shown in Appendix Figures <a 
href="#x1-19400516">K.16<!--tex4ht:ref: fig:simulation1205_randx_Graph6 --></a> and <a 
href="#x1-19401018">K.18<!--tex4ht:ref: fig:simulation1205_randx_Graph7 --></a>) had fewer genes.
</p><!--l. 878--><p class="indent" >
While the <a 
href="#glo:graph">graph</a> structures (of similar size) were highly distinct, they had similar performance profiles across parameters. <a 
href="#glo:SLIPT">SLIPT</a> is therefore robust across <a 
href="#glo:graph">pathway</a> structures and is more affected by the number or proportion of genes to detect. Findings from previous simulations in similar correlation structures (in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>) should be applicable to <a 
href="#glo:gene expression">expression</a> data with more complex correlation structures, such as those occurring in biological pathways. Specifically, <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners are distinguishable from closely correlated geness in the context of a biological pathway networks, irrespective of thresholds (shown by <a 
href="#glo:ROC">ROC</a>) and with the sensitivity and specificity of <a 
href="#glo:SLIPT">SLIPT</a> as a binary classifier (as used in Chapters <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a> and <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>).
</p><!--l. 885--><p class="indent" >
The findings for inhibitory <a 
href="#glo:graph">graph</a> structures were replicated with larger more complex <a 
href="#glo:graph">graph</a> structures with inhibiting relationships and more <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes to detect (shown in Appendix Figures <a 
href="#x1-19500522">K.22<!--tex4ht:ref: fig:simulation1205_randx_Graph6i --></a>–<a 
href="#x1-19502028">K.28<!--tex4ht:ref: fig:simulation1205_randx_Graph7i2 --></a>). In each <a 
href="#glo:graph">graph</a> structure, simulations entirely with inhibiting relationships (Appendix Figures <a 
href="#x1-19500522">K.22<!--tex4ht:ref: fig:simulation1205_randx_Graph6i --></a>, <a 
href="#x1-19501526">K.26<!--tex4ht:ref: fig:simulation1205_randx_Graph7i --></a>, and <a 
href="#x1-19502530">K.30<!--tex4ht:ref: fig:simulation1205_randx_Graph5i --></a>) had higher performance than the equivalent graph with entirely activating relationships (Appendix Figures <a 
href="#x1-19400516">K.16<!--tex4ht:ref: fig:simulation1205_randx_Graph6 --></a>, <a 
href="#x1-19401018">K.18<!--tex4ht:ref: fig:simulation1205_randx_Graph7 --></a>, and <a 
href="#x1-19401520">K.20<!--tex4ht:ref: fig:simulation1205_randx_Graph5 --></a>) or a combination of activating and inhibiting relationships (Appendix Figures <a 
href="#x1-19501024">K.24<!--tex4ht:ref: fig:simulation1205_randx_Graph6i2 --></a>, <a 
href="#x1-19502028">K.28<!--tex4ht:ref: fig:simulation1205_randx_Graph7i2 --></a>, and <a 
href="#x1-19503032">K.32<!--tex4ht:ref: fig:simulation1205_randx_Graph5i2 --></a>). While the presence of negative correlations subtly affects the performance of <a 
href="#glo:SLIPT">SLIPT</a>, the methodology is robust across the exact structures of genes and is therefore applicable to detecting <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes in a range of (synthetic lethal) biological pathways with different structural relationships. <a 
 id="x1-148001r316"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.3.1 </span> <a 
 id="x1-1490001"></a>Simulations over Pathway-based Graphs
</h4>
<!--l. 901--><p class="noindent" >
<p>Simulations of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in <a 
href="#glo:gene expression">gene expression</a> with correlation structures thus far have used simple blocks of correlated genes (as used in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>) or have ben derived from constructed <a 
href="#glo:graph">graph</a> structures (as used in Section <a 
href="#x1-1410002">6.2<!--tex4ht:ref: chapt5:graphsim --></a>). These have been used to make inferences on the impact of correlation structure but it remains to be shown whether these findings are reproducible in the complexity of the biological <a 
href="#glo:graph">network</a> structure. Specifically, <a 
href="#glo:SLIPT">SLIPT</a> was tested on simulated data with known underlying simulated <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners (as described in Section <a 
href="#x1-770002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a>) with multivariate normal correlation structure derived from biological pathways (as described in Section <a 
href="#x1-860002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>).</p>
</p><!--l. 903--><p class="indent" >
The Reactome <a 
href="#glo:graph">pathway</a> structure for the <a 
href="#glo:PI3K">PI3K</a> cascade (as used in Chapter <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>) was used to demonstrate the simulation procedure for detecting <a 
href="#glo:synthetic lethal">synthetic lethality</a> in the <a 
href="#glo:graph">graph</a> structure of a biological pathway. This pathway has clear directionality, with related signalling pathways among those identified to be <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates (in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>). The <a 
href="#glo:PI3K">PI3K</a> pathway has a relatively moderate size (138 genes) and complexity. It is therefore suitable for comparison to previous <a 
href="#glo:graph">graph</a> structures of a similar scale (50–100 genes) with the complexity of a biological pathway.
</p><!--l. 905--><p class="indent" >
<p>The performance of <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection with <a 
href="#glo:SLIPT">SLIPT</a>, in simulated <a 
href="#glo:gene expression">expression</a> data based on the Reactome <a 
href="#glo:PI3K">PI3K</a> pathway (as shown in Figure <a 
href="#x1-14900531">6.31<!--tex4ht:ref: fig:simulation1205_randx_Graph_pi3k --></a>), was concordant with previous findings. <a 
href="#glo:SLIPT">SLIPT</a> had high performance when detecting a low number of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes which decreased for high numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes or lower sample sizes. In particular, the performance of simulations in the <a 
href="#glo:PI3K">PI3K</a> pathway closely resembled the simulation results for constructed <a 
href="#glo:graph">graphs</a> of similar scale and complexity (as shown in Appendix Figures <a 
href="#x1-19400516">K.16<!--tex4ht:ref: fig:simulation1205_randx_Graph6 --></a> and <a 
href="#x1-19401018">K.18<!--tex4ht:ref: fig:simulation1205_randx_Graph7 --></a>). Using thresholds based on the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> p-value (adjusted by <a 
href="#glo:FDR">FDR</a>), simulations in the biological <a 
href="#glo:PI3K">PI3K</a> pathway had a higher sensitivity and lower specificity. While the performance decreased for more <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes to detect within the simulated <a 
href="#glo:PI3K">PI3K</a> pathway, this primarily involved a reduction in sensitivity to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes rather than false positives, as the false positive rate decreased, the accuracy increased, and the specificity was relatively unperturbed (being more dependent on sample size). Thus <a 
href="#glo:SLIPT">SLIPT</a> was stringent in an example of a biological <a 
href="#glo:graph">graph</a> structure and is appropriate for detection of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes in complex correlation structures in <a 
href="#glo:gene expression">gene expression</a> data involving biological pathways.</p>
</p><!--l. 907--><p class="indent" >
<a 
 id="x1-14900531"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 910--><p class="noindent" >
</p><!--l. 915--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_pi3k_ROC1_samplesx".png  /><a 
 id="x1-149001r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_pi3k_ROC2_samplesx".png  /><a 
 id="x1-149002r2"></a> <span class="cmr-10x-x-109">Receiver</span> <span class="cmr-10x-x-109">operating</span> <span class="cmr-10x-x-109">characteristic</span>
</p><!--l. 926--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph_pi3k".png /><a 
 id="x1-149003r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_pi3k_AUC_samplesx_prop".png  /><a 
 id="x1-149004r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 6.31: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on the </span><a 
href="#glo:PI3K"><span class="cmbx-10x-x-109">PI3K</span></a> <span class="cmbx-10x-x-109">cascade.</span></strong> <span class="cmr-10x-x-109">Simulation of</span> <a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a> <span class="cmr-10x-x-109">was performed by sampling from a multivariate normal distribution based</span> <span class="cmr-10x-x-109">on the Reactome </span><a 
href="#glo:PI3K"><span class="cmr-10x-x-109">PI3K</span></a> <span class="cmr-10x-x-109">cascade. Performance of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">was high across parameters for</span> <span class="cmr-10x-x-109">detecting </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a> <span class="cmr-10x-x-109">in the </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure within a larger dataset. The performance</span> <span class="cmr-10x-x-109">decreased for a greater number of true positives to detect but the accuracy increased with a</span> <span class="cmr-10x-x-109">low false positive rate.</span></span></p>
</div>
<!--tex4ht:label?: x1-14900531 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 938--><p class="indent" >
These simulations were replicated in the more complex G<sub><span class="cmmi-8">αi</span></sub> signalling pathway (of 292 genes), which was one of the most well supported <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways with loss of <span class="cmti-12">CDH1 </span>in cancer (in Chapters <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a> and <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>). This pathway showed similar relationships between sensitivity, specificity, and false positive rate with number of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners and sample size (as shown in Appendix Figure <a 
href="#x1-19600534">K.34<!--tex4ht:ref: fig:simulation1205_randx_Graph_Gai --></a>). While the overall performance was lower than for smaller networks structures, many of the findings from previous networks were replicated in a larger more complex biological network. In the G<sub><span class="cmmi-8">αi</span></sub> signalling pathway, <a 
href="#glo:SLIPT">SLIPT</a> had high performance for detecting low numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes and was highly stringent against false positives for higher numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes. <a 
 id="x1-149006r333"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.3.2 </span> <a 
 id="x1-1500002"></a>Pathway Structures in a Large Simulated Datasets
</h4>
<!--l. 946--><p class="noindent" >
<p>Simulations were also performed with <a 
href="#glo:graph">graph</a> structures from biological pathways included in a larger dataset to simulate <a 
href="#glo:gene expression">gene expression</a> data of the scale typical for human and cancer studies. These simulations (as discussed in Section <a 
href="#x1-1470004">6.2.4<!--tex4ht:ref: chapt5:graphsim_performance_20K --></a>) showed a higher specificity and therefore <a 
href="#glo:SLIPT">SLIPT</a> had higher performance. The simulated <a 
href="#glo:PI3K">PI3K</a> pathway (as shown in Figure <a 
href="#x1-15000533">6.33<!--tex4ht:ref: fig:simulation1207_randx_Graph_pi3k --></a>) was no exception, with high performance across parameter values, which remained high up to many genes. While the sensitivity decreased for high numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes to detect within the <a 
href="#glo:PI3K">PI3K</a> pathway, the <a 
href="#glo:SLIPT">SLIPT</a> methodology remained accurate, with high specificity in a large simulated <a 
href="#glo:gene expression">gene expression</a> dataset.</p>
</p><!--l. 948--><p class="indent" >
<a 
 id="x1-15000533"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 951--><p class="noindent" >
</p><!--l. 956--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_pi3k_ROC1_samplesx".png  /><a 
 id="x1-150001r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_pi3k_ROC2_samplesx".png  /><a 
 id="x1-150002r2"></a> <span class="cmr-10x-x-109">Receiver</span> <span class="cmr-10x-x-109">operating</span> <span class="cmr-10x-x-109">characteristic</span>
</p><!--l. 967--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph_pi3k".png /><a 
 id="x1-150003r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_pi3k_AUC_samplesx_prop".png  /><a 
 id="x1-150004r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure 6.33: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations including the </span><a 
href="#glo:PI3K"><span class="cmbx-10x-x-109">PI3K</span></a> <span class="cmbx-10x-x-109">cascade.</span></strong> <span class="cmr-10x-x-109">Simulation</span> <span class="cmr-10x-x-109">of </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a> <span class="cmr-10x-x-109">was performed by sampling from a multivariate normal distribution</span> <span class="cmr-10x-x-109">(without correlation structure apart from the Reactome </span><a 
href="#glo:PI3K"><span class="cmr-10x-x-109">PI3K</span></a> <span class="cmr-10x-x-109">cascade). Performance of</span> <a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">was high across parameters for detecting </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a> <span class="cmr-10x-x-109">in the </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure</span> <span class="cmr-10x-x-109">within a larger dataset. The sensitivity decreases for a greater number of true positives to</span> <span class="cmr-10x-x-109">detect but the specificity remains high with a low false positive rate.</span></span></p>
</div>
<!--tex4ht:label?: x1-15000533 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 979--><p class="indent" >
Therefore the <a 
href="#glo:SLIPT">SLIPT</a> methodology is a highly stringent approach suitable to be applied for detecting <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes and pathways within highly complex <a 
href="#glo:gene expression">expression</a> data with biological <a 
href="#glo:graph">pathway</a> structure. Even the poorly performing simulations were highly stringent, with low false positive rates, which are an important consideration in a <a 
href="#glo:gene expression">gene expression</a> data with many non-synthetic lethal genes. The enrichment of true <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners among detected genes makes <a 
href="#glo:SLIPT">SLIPT</a> valuable for triage of candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners for further validation and for pathway analysis.
</p><!--l. 981--><p class="indent" >
<p>The performance of <a 
href="#glo:SLIPT">SLIPT</a> in simulations of <a 
href="#glo:synthetic lethal">synthetic lethality</a> within biological pathways was markedly higher in the context of a larger dataset of thousands of genes. As shown in a direct comparison with the <a 
href="#glo:graph">graph</a> structures alone (as shown in Figure <a 
href="#x1-150008r3">6.34c<!--tex4ht:ref: fig:simulation1205_randx_Graphpwaycf20K:Compare1 --></a>), performance was consistently higher across parameters in pathways of biological complexity from the Reactome database (<a 
href="#XReactome">Croft <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XReactome">2014</a>) such as <a 
href="#glo:PI3K">PI3K</a> cascade and the G<sub><span class="cmmi-8">αi</span></sub> signalling pathway (shown in Figure <a 
href="#x1-150009r4">6.34d<!--tex4ht:ref: fig:simulation1205_randx_Graphpwaycf20K:Compare2 --></a> and Appendix Figure <a 
href="#x1-19601036">K.36<!--tex4ht:ref: fig:simulation1207_randx_Graph_Gai --></a>).</p>
</p><!--l. 983--><p class="indent" >
<a 
 id="x1-15001035"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 986--><p class="noindent" >
</p><!--l. 990--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph_pi3k".png /><a 
 id="x1-150006r1"></a> <span class="cmr-10x-x-109">The</span> <span class="cmr-10x-x-109">PI3K</span> <span class="cmr-10x-x-109">cascade</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph_Gai".png /><a 
 id="x1-150007r2"></a> <span class="cmr-10x-x-109">G</span><sub><span class="cmmi-8">αi</span></sub> <span class="cmr-10x-x-109">signalling</span>
</p><!--l. 1002--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_pi3k_ROC_Compare(2)".png /><a 
 id="x1-150008r3"></a> <span class="cmr-10x-x-109">Performance</span> <span class="cmr-10x-x-109">in the</span> <span class="cmr-10x-x-109">PI3K</span> <span class="cmr-10x-x-109">cascade</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_10K_Gai_ROC_Compare(2)".png /><a 
 id="x1-150009r4"></a> <span class="cmr-10x-x-109">Performance</span> <span class="cmr-10x-x-109">in</span> <span class="cmr-10x-x-109">G</span><sub><span class="cmmi-8">αi</span></sub> <span class="cmr-10x-x-109">signalling</span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.35: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance on pathways improves with more genes.</span></strong> <span class="cmr-10x-x-109">Simulations were</span> <span class="cmr-10x-x-109">performed in a </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structures for the </span><a 
href="#glo:PI3K"><span class="cmr-10x-x-109">PI3K</span></a> <span class="cmr-10x-x-109">cascade and G</span><sub><span class="cmmi-8">αi</span></sub> <span class="cmr-10x-x-109">signalling pathways structures</span> <span class="cmr-10x-x-109">to detect </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners within them. As for constructed graphs, performance of</span> <span class="cmr-10x-x-109">detection in a dataset containing only the </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure (in colour) was as much lower than</span> <span class="cmr-10x-x-109">testing the </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graph</span></a> <span class="cmr-10x-x-109">structure within a larger dataset of non-synthetic lethal genes (without</span> <span class="cmr-10x-x-109">correlations) for both </span><a 
href="#glo:graph"><span class="cmr-10x-x-109">graphs</span></a> <span class="cmr-10x-x-109">of biological complexity.</span></span></p>
</div>
<!--tex4ht:label?: x1-15001035 -->
                                                                                    

</div><hr class="endfloat" />
<!--l. 1014--><p class="indent" >
These biologically complex <a 
href="#glo:graph">graph</a> structures, based on the Reactome pathways, assumed activating relationships to test <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection with <a 
href="#glo:SLIPT">SLIPT</a> in the context of complex correlation structures. Inhibiting relationships were not distinguished in the Reactome database (<a 
href="#XReactome">Croft <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XReactome">2014</a>). However, these investigations with pathway-based <a 
href="#glo:graph">graph</a> structures indicate that the findings in constructed <a 
href="#glo:graph">graphs</a> (as used in Section <a 
href="#x1-1410002">6.2<!--tex4ht:ref: chapt5:graphsim --></a>) are relevant to <a 
href="#glo:gene expression">gene expression</a> data containing real correlated pathways. Furthermore, previous comparisons between simulations with inhibiting relationships indicated that the performance of <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection in an equivalent <a 
href="#glo:graph">graph</a> structure with inhibitory relationships will likely be higher.
</p><!--l. 1016--><p class="indent" >
Non synthetic lethal genes, inversely correlated with the underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners, were distinguishable by <a 
href="#glo:SLIPT">SLIPT</a> with high specificity. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> genes were detectable with reasonable performance in large scale simulated <a 
href="#glo:gene expression">gene expression</a> data and highly (positively) correlated genes in <a 
href="#glo:graph">pathway</a> structures. These findings serve as a conservative estimate for the performance of <a 
href="#glo:SLIPT">SLIPT</a> to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes within a <a 
href="#glo:synthetic lethal">synthetic lethal</a> biological pathway in empirical data. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> genes are distinguishable from correlated genes, to varying extents, in simulations. False positives are also more likely to occur within the same (synthetic lethal) pathways. Therefore <a 
href="#glo:SLIPT">SLIPT</a> is both effective at triage of <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates within a biological pathway and at identifying <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways in high dimensional <a 
href="#glo:gene expression">gene expression</a> data. <a 
 id="x1-150011r332"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">6.4 </span> <a 
 id="x1-1510004"></a>Discussion
</h3>
<a 
 id="x1-151001r339"></a>
<h4 class="subsectionHead">
<span class="titlemark">6.4.1 </span> <a 
 id="x1-1520001"></a>Simulation Procedure
</h4>
<!--l. 1025--><p class="noindent" >
<p>Simulations were performed to assess the performance of the <a 
href="#glo:SLIPT">SLIPT</a> methodology (as described in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a> and with modifications) for detecting known underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of a query gene. The simulation results supported the findings in empirical data (in Chapters <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a> and <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>) by addressing whether the methodology used to generate them was accurate or had desirable statistical performance in controlled simulated conditions. These investigations included adjusting parameters such as the numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes which were known in empirical data to assess the performance of the <a 
href="#glo:SLIPT">SLIPT</a></p>
methodology across simulation parameters and characterise the datasets for which <a 
href="#glo:SLIPT">SLIPT</a> performs well. Simulation and statistically modelling also enabled comparison of the <a 
href="#glo:SLIPT">SLIPT</a> methodology to other statistical approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection in <a 
href="#glo:gene expression">expression</a> data.
</p><!--l. 1027--><p class="indent" >
These simulations were based on a statistical model of <a 
href="#glo:synthetic lethal">synthetic lethality</a> (as described in Section <a 
href="#x1-760001">3.2.1<!--tex4ht:ref: methods:SL_Model --></a>) which was designed stringently to ensure that if <a 
href="#glo:synthetic lethal">synthetic lethality</a> was detectable in the simulated datasets it would also be detectable by the same methodology in empirical <a 
href="#glo:gene expression">expression</a> data. The model of <a 
href="#glo:synthetic lethal">synthetic lethality</a> made conservative assumptions such as the low threshold of <a 
href="#glo:gene expression">expression</a> for gene function or the inclusion of cryptic higher-order <a 
href="#glo:synthetic lethal">synthetic lethality</a> (when testing pairwise). These assumptions decrease the likelihood that <a 
href="#glo:synthetic lethal">synthetic lethal</a> signatures would be detectable in <a 
href="#glo:gene expression">expression</a> data. Thus it is reassuring that <a 
href="#glo:synthetic lethal">synthetic lethality</a> wes still detectable in under many simulation parameters as the performance of <a 
href="#glo:SLIPT">SLIPT</a> would be expected to be higher were these assumptions to be violated in empirical data.
</p><!--l. 1033--><p class="indent" >
The simulation procedure (as described in Section <a 
href="#x1-770002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a>) wes designed as a computational pipeline with arguments passes to scripts. The <a 
href="#glo:SLIPT">SLIPT</a> methodology and simulation of <a 
href="#glo:gene expression">expression</a> from <a 
href="#glo:graph">graph</a> structures were both used as R (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>) software packages developed and released for this project (as described in Section <a 
href="#x1-870005">3.5<!--tex4ht:ref: methods:r_packages --></a>). This design ensurd that the simulations can be robustly applied across parameters with consistency between simulations apart from the differences discussed. The simulation procedure is also flexible to simulating other datasets, including <a 
href="#glo:synthetic lethal">synthetic lethal</a> relationships and pathway correlation structures, should these be relevant to future investigations or <a 
href="#glo:bioinformatics">bioinformatics</a> tool development. The computational pipeline is also compatible with parallel computing and made use of <a 
href="#glo:HPC">High Performance Computing (HPC)</a> infrastructure provided by <a 
href="#glo:NeSI">the New Zealand eScience Infrastructure (NeSI)</a> using the <a 
href="#glo:Slurm">Simple Linux Utility for Resource Management (Slurm)</a> submission system (as described in Section <a 
href="#x1-720003">2.5.3<!--tex4ht:ref: methods:HPC --></a>). This parallel computing pipeline enabled extensive investigations into <a 
href="#glo:synthetic lethal">synthetic lethality</a> in simulated data, including approximately 2 million cpu-hours on <a 
href="#glo:NeSI">NeSI</a>. <a 
 id="x1-152001r351"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.4.2 </span> <a 
 id="x1-1530002"></a>Comparing Methods with Simulated Data
</h4>
<!--l. 1038--><p class="noindent" >
<p>Attempts were made to implement alternative <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection approaches such as linear regression and the <a 
href="#glo:BiSEp">BiSEp</a> R package (discussed in Section <a 
href="#x1-1350001">6.1<!--tex4ht:ref: chapt5:compare_methods --></a>). However, those tested</p>
were ineffective at detecting <a 
href="#glo:synthetic lethal">synthetic lethality</a> in multivariate normal simulated data in comparison to <a 
href="#glo:SLIPT">SLIPT</a>. While some of the published <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection methods (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>; <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>) did not provide reproducible software releases for direct comparison, some of the central assumptions used in their design were tested by the statistical methods considered for <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection in <a 
href="#glo:gene expression">expression</a> data.
</p><!--l. 1040--><p class="indent" >
<a 
href="#glo:BiSEp">BiSEp</a> took considerably more time to compute predictions than <a 
href="#glo:SLIPT">SLIPT</a> or <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>, which limited the number of simulations that were feasible and made it difficult to apply across parameters in the simulation pipeline (even when using supercomputing infrastructure as discussed in Section <a 
href="#x1-720003">2.5.3<!--tex4ht:ref: methods:HPC --></a>). The computationally intensive nature of the <a 
href="#glo:BiSEp">BiSEp</a> procedure does not appear to be the issue for detecting <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes in <a 
href="#glo:TCGA">TCGA</a> data or simulations, although it has made more extensive simulations challenging. Rather, <a 
href="#glo:BiSEp">BiSEp</a> was not suitable in either case since the <a 
href="#glo:TCGA">TCGA</a> data was normalised with <span class="cmtt-12">voom </span>(<a 
href="#Xlimma">Ritchie <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xlimma">2015</a>) and simulated data was generated by sampling from a multivariate normal distribution. In either case, even subtle bimodal signatures in <a 
href="#glo:gene expression">expression</a> data were not consistently detectable or sufficient to detect <a 
href="#glo:synthetic lethal">synthetic lethality</a>.
</p><!--l. 1042--><p class="indent" >
The <a 
href="#glo:BiSEp">BiSEp</a> methodology may perform better on other data types but it cannot be directly compared with the results for <a 
href="#glo:SLIPT">SLIPT</a> throughout this thesis which have used normalised or (multivariate) normally distributed data. Since it requires bimodal distributions, <a 
href="#glo:BiSEp">BiSEp</a> was not suitable for stringently normalised <a 
href="#glo:gene expression">expression</a> data nor would it be expected to perform on (ranked) pathway <a 
href="#glo:metagene">metagenes</a>. Thus <a 
href="#glo:SLIPT">SLIPT</a> represents a distinct approach more suitable for these data types whereas <a 
href="#glo:BiSEp">BiSEp</a> may be applicable to other applications in which bimodal distributions are more frequent.
</p><!--l. 1044--><p class="indent" >
<p>This investigation also demonstrate that implementing scientific software from other research groups is not a trivial exercise, even when released as an open-source R package. Therefore, the above results were used to evaluate <a 
href="#glo:SLIPT">SLIPT</a> and compare it to other statistical rationales. A comprehensive comparison to contemporary <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection approaches (and those released in the future) or further benchmarking is left to an impartial researcher to evaluate. The above findings showed that the <a 
href="#glo:SLIPT">SLIPT</a> approach was able to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes in simulated data with comparable or better performance than a range of distinct statistical techniques and was appropriate for use throughout this thesis. <a 
 id="x1-153001r352"></a></p>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.4.3 </span> <a 
 id="x1-1540003"></a>Design and Performance of SLIPT
</h4>
<!--l. 1048--><p class="noindent" >
The simulation procedure using sampling from a multivariate normal distribution was used throughout the majority of the simulation investigations in this thesis. This approach has the advantages of emulating the continuous normalised <a 
href="#glo:gene expression">expression</a> data used for <a 
href="#glo:gene expression">gene expression</a> analysis and enabled the simulation of correlation structures (as discussed in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>). These simulations scaled to datasets of comparable scale to those used in <a 
href="#glo:gene expression">gene expression</a> anlaysis with thousands of genes. The <a 
href="#glo:SLIPT">SLIPT</a> methodology was shown to perform robustly across large numbers of genes and simple correlation structures. This included high specificity against genes positively correlated with the query gene for which the directional <a 
href="#glo:SLIPT">SLIPT</a> methodology was more suited to distinguishing <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes from than the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test without directional criteria on the number of samples observed.
</p><!--l. 1050--><p class="indent" >
These findings were expanded upon in this chapter. Specifically, different quantiles were compared for <a 
href="#glo:SLIPT">SLIPT</a> and the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test. These approaches using threshold based discrete gene function were compared to the Pearson correlation without loss of the continuous <a 
href="#glo:gene expression">expression</a> data. The <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span> -quantiles for <a 
href="#glo:SLIPT">SLIPT</a> (as described in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) were optimal for both <a 
href="#glo:SLIPT">SLIPT</a> and the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> alone. In addition to being optimal for estimating the significance of <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions, these quantiles were also optimal for the directional criteria of <a 
href="#glo:SLIPT">SLIPT</a> since this method outperformed the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test and was the most different at the <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span> -quantile. As previously noted, this difference was more pronounced with positively correlated genes (with the query gene) for which the specificity of <a 
href="#glo:SLIPT">SLIPT</a> improved, and was replicated in large datasets with thousands of genes, as occur in human <a 
href="#glo:gene expression">expression</a> data. These results were not simply due to sufficient samples for significant p-values since the performance as determined by <a 
href="#glo:AUROC">AUROC</a> analysis which was independent from significance thresholds. This indicated that the <a 
href="#glo:SLIPT">SLIPT</a> methodology (as it has been used in Chapters <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a> and <a 
href="#x1-1180005">5<!--tex4ht:ref: chap:Pathways --></a>) was optimal, with the <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span>-quantile having the highest performance, and as such the parameters used to design it were appropriate.
</p><!--l. 1052--><p class="indent" >
<p>Both discrete functional approaches (<a 
href="#glo:SLIPT">SLIPT</a> and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>) were able to outperform negative correlation which supports their use. In particular, this result addressed the concern that arbitary thresholds of low and high gene function (as used by <a 
href="#glo:SLIPT">SLIPT</a>) may lose useful data by compressing the spectrum of <a 
href="#glo:gene expression">gene expression</a> into categorical data. However, this does not impede the performance of <a 
href="#glo:SLIPT">SLIPT</a> if the quantiles used were optimal. The poorer performance of correlation-based detection of <a 
href="#glo:synthetic lethal">synthetic lethality</a> was consistent with gene</p>
function for <a 
href="#glo:synthetic lethal">synthetic lethality</a> being qualitative, that is <a 
href="#glo:gene expression">expression</a> must be sufficient for cell viability and higher <a 
href="#glo:gene expression">expression</a> was not necessary for function (as this is not the case for all genes). Furthermore, the finding that negative correlation outperforms positive correlation is also consistent with co-expression being a poor predictor of <a 
href="#glo:synthetic lethal">synthetic lethality</a> compared to other approaches (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>), supporting the claims of <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2015">2015</a>).
</p><!--l. 1054--><p class="indent" >
Compared with <a 
href="#glo:SLIPT">SLIPT</a>, neither correlation approaches nor bimodality signatures were suitable for detecting <a 
href="#glo:synthetic lethal">synthetic lethality</a> in <a 
href="#glo:gene expression">expression</a> data. The correlation-based approaches made assumptions about the relationship between <a 
href="#glo:gene expression">gene expression</a> and function which do not necessarily hold for all genes. Similarly, the bimodal approach was not appropriate for normalised data since deviations from a normal distribution had already been used for ensuring data quality, as is common practice for <a 
href="#glo:RNA-Seq">RNA-Seq</a> data. A linear model or regression approach may also be used to detect <a 
href="#glo:synthetic lethal">synthetic lethality</a> from relationships between <a 
href="#glo:gene expression">expression</a> of genes, which may be improved with conditioning on known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners with multivariate regression or Bayesian priors. Similarly, <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection could be performed by iteratively conditioning upon the strong candidate from previous analysis. These approaches may be able to better circumvent the issues of high-order <a 
href="#glo:synthetic lethal">synthetic lethality</a> and multiple testing.
</p><!--l. 1058--><p class="indent" >
Nevertheless, the above findings were sufficient to assess the performance of <a 
href="#glo:SLIPT">SLIPT</a> and present an effective straightforward approach to <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection in <a 
href="#glo:gene expression">gene expression</a> data. Further development with linear models, Bayesian inference approaches, or comparison to existing <a 
href="#glo:synthetic lethal">synthetic lethal</a> approaches (e.g., machine learning) remain as future directions. Developing and testing more sophisticated statistical approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection may benefit from the concepts discussed with regard to the relatively simple <a 
href="#glo:SLIPT">SLIPT</a> methodology. Similarly, further comparisons and benchmarking of <a 
href="#glo:SLIPT">SLIPT</a> against other computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection in <a 
href="#glo:gene expression">gene expression</a> data is more suitable for an independent researcher and the <span class="cmtt-12">slipt </span>R package has been released (as described in Section <a 
href="#x1-870005">3.5<!--tex4ht:ref: methods:r_packages --></a>) for this purpose, in addition to further application in research. <a 
 id="x1-154001r353"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.4.4 </span> <a 
 id="x1-1550004"></a>Simulations from Graph Structures
</h4>
<!--l. 1064--><p class="noindent" >
<p>The simple correlation structures (as used in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>) were expanded upon using the multivariate normal simulation procedure to produce correlation structures based on <a 
href="#glo:graph">graph</a></p>
structures (as described in Section <a 
href="#x1-1410002">6.2<!--tex4ht:ref: chapt5:graphsim --></a>). These simulations enabled further investigations into the performance of <a 
href="#glo:SLIPT">SLIPT</a> in the context of more complex correlation structures. The simulation of <a 
href="#glo:gene expression">expression</a> from <a 
href="#glo:graph">network</a> structures is widely applicable to simulating pathway <a 
href="#glo:gene expression">expression</a> data and as such the <span class="cmtt-12">graphsim </span>R package has been released (as described in Section <a 
href="#x1-870005">3.5<!--tex4ht:ref: methods:r_packages --></a>).
</p><!--l. 1067--><p class="indent" >
These investigations show that <a 
href="#glo:SLIPT">SLIPT</a> performs robustly across datasets with different correlation structures, including those derived from <a 
href="#glo:graph">graphs</a> with the complexity of biological pathways. The <a 
href="#glo:SLIPT">SLIPT</a> methodology was able to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes within <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways across many <a 
href="#glo:graph">graph</a> structures. This methodology performed particularly well with <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways in the context of a larger dataset with a high specificity which supports <a 
href="#glo:SLIPT">SLIPT</a> as a stringent approach to <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection in highly dimensional <a 
href="#glo:gene expression">gene expression</a> data. Together these results support the use of <a 
href="#glo:SLIPT">SLIPT</a> in biological gene expresssion data since it was able to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes in highly complex correlation structures.
</p><!--l. 1069--><p class="indent" >
Similarly, the inclusion of inhibitory relationships in <a 
href="#glo:graph">graph</a> structures was shown to increase the performance in simple networks, supporting <a 
href="#glo:SLIPT">SLIPT</a> being applicable to biological data in which these relationships are common. While these results were not replicated in more complex inhibitory <a 
href="#glo:graph">graph</a> structures, this is likely an artifact of the simulation procedure (which randomly selects <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes) generating biologically implausible combinations of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes which were difficult to detect. When the test statistics in simulations with a <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene were examined in more detail, the test statistics of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene were consistently higher and distinguishable from nearby genes in the <a 
href="#glo:graph">graph</a> structure. In contrast to previous concerns with inhibiting relationships, these differences were more pronounced with genes which had inhibitory relationships with <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes. While distinguishable from nearby genes in a <a 
href="#glo:graph">pathway</a> structure, the genes correlated with <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners still had higher test statistics than more distant genes (similar to observations with correlated genes in Section <a 
href="#x1-780003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>).
</p><!--l. 1071--><p class="indent" >
<p>In addition to being able to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes in a pathway, the proximal genes in a pathway were most likely to be false positives and therefore <a 
href="#glo:SLIPT">SLIPT</a> is also able to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways. <a 
href="#glo:SLIPT">SLIPT</a> identifies genes which are likely to be constituent of a <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathway and is more likely to rank underlying <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes with greater significance. Together these findings support the use of <a 
href="#glo:SLIPT">SLIPT</a> throughout this thesis, further application of <a 
href="#glo:SLIPT">SLIPT</a>, and further</p>
development of such strategies for <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection. Similarly, the simulation procedures developed and demonstrated for examining <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection in <a 
href="#glo:gene expression">expression</a> data using <a 
href="#glo:graph">graph</a> structures is amenable to further development and investigations into <a 
href="#glo:graph">pathway</a> structure in <a 
href="#glo:gene expression">expression</a> data such as predicting biological pathways from <a 
href="#glo:gene expression">expression</a> data or the impact of pathways on differential <a 
href="#glo:gene expression">expression</a> analyses. <a 
 id="x1-155001r350"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">6.5 </span> <a 
 id="x1-1560005"></a>Summary
</h3>
<!--l. 1076--><p class="noindent" >
A statistical model and simulation procedure has been developed to test the performance of the <a 
href="#glo:SLIPT">SLIPT</a> methodology in controlled conditions, using multivariate normal distributions. This simulation procedure has been developed into a computational pipeline which was able to test the statistical performance (using stringent assumptions) of <a 
href="#glo:SLIPT">SLIPT</a> across many parameters and compare it to alternative <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection strategies. The <a 
href="#glo:SLIPT">SLIPT</a> methodology performed well at detecting small numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes in simple systems. It did not perform as well in more complex systems but neither did alternative strategies. The <a 
href="#glo:SLIPT">SLIPT</a> methodology performed well compared to Pearson correlation and similar methods based on the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test. Thus <a 
href="#glo:SLIPT">SLIPT</a> is an effective detection method for <a 
href="#glo:synthetic lethal">synthetic lethal</a> relationships in <a 
href="#glo:gene expression">expression</a> data despite its relatively simple design.
</p><!--l. 1078--><p class="indent" >
<p>Simulations of more complex datasets were performed, including large numbers of genes, complex correlation structure derived from <a 
href="#glo:graph">graph</a> structures, and correlations with the query gene. <a 
href="#glo:SLIPT">SLIPT</a> performed robustly across these, including correlation structures based on complex biological pathways. The performance of <a 
href="#glo:SLIPT">SLIPT</a> improved in larger datasets, datasets with positive correlations with the query genes, and some <a 
href="#glo:graph">graph</a> structures which included inhibiting relationships, namely those datasets that were more representative of <a 
href="#glo:gene expression">gene expression</a> in biological data. <a 
href="#glo:SLIPT">SLIPT</a> was both capable of recurrently detecting genes within a <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathway, and distinguishing <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes from correlated with them, even in highly complex correlation structures. Therefore <a 
href="#glo:SLIPT">SLIPT</a> is a stringent <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection strategy and is applicable to <a 
href="#glo:gene expression">gene expression</a> as previously demonstrated for the partners of <span class="cmti-12">CDH1 </span>in breast and stomach cancer in this thesis.</p>
</p>
<h2 class="chapterHead">
<span class="titlemark">Chapter 7</span><br /><a 
 id="x1-1570007"></a>Discussion
</h2>
<!--l. 4--><p class="noindent" >
This thesis combines analysis of <a 
href="#glo:gene expression">gene expression</a> data from <a 
href="#glo:TCGA">TCGA</a> with experimental screening results (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) to demonstrate <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery for partners of <span class="cmti-12">CDH1</span>. Together these findings further elucidate the functions of <span class="cmti-12">CDH1 </span>in the cell, <a 
href="#glo:functional redundancy">functional redundancy</a> in cancer, and represent potential <a 
href="#glo:targeted therapy">therapeutic targets</a> against loss of <span class="cmti-12">CDH1 </span>function. These candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes were further investigated for relationships within <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways, and in the process a network-based approach to compare genes identified in <a 
href="#glo:genomic">genomics</a> experiments was developed.
</p><!--l. 7--><p class="indent" >
The <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection methodology, <a 
href="#glo:SLIPT">SLIPT</a>, was applied to <a 
href="#glo:gene expression">gene expression</a> data throughout this thesis and was evaluated with simulated data. A procedure was developed to stringently generate <a 
href="#glo:gene expression">gene expression</a> data from known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in simulated data. These simulations included simple and complex correlation structures, and modelling <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes within pathways. Together, these results demonstrate <a 
href="#glo:SLIPT">SLIPT</a> as a robust widely applicable <a 
href="#glo:gene expression">gene expression</a> analysis procedure (for which an R package has been made available) for discovery of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes. Performance of <a 
href="#glo:SLIPT">SLIPT</a> on simulated data also highlights the strengths of the procedure and future directions to improve upon it. <a 
 id="x1-157001r355"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">7.1 </span> <a 
 id="x1-1580001"></a>Synthetic Lethality and <span class="cmti-12">CDH1 </span>Biology
</h3>
<!--l. 12--><p class="noindent" >
The <span class="cmti-12">CDH1 </span><a 
href="#glo:tumour suppressor">tumour suppressor</a> gene was the focus of identifying <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners to demonstrate the novel <a 
href="#glo:SLIPT">SLIPT</a> methodology. This gene is important in <a 
href="#glo:sporadic">sporadic</a> breast and stomach cancers, in addition to <a 
href="#glo:familial">familial</a> syndromes, such as <a 
href="#glo:HDGC">HDGC</a>. The analysis of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">CDH1 </span>in breast and stomach cancers was enabled by the availability of molecular data (<a 
href="#XTCGA2014GC">Bass <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014GC">2014</a>; <a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>) and a <a 
href="#glo:synthetic lethal">synthetic lethal</a> screen conducted in MCF10A breast cells (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>; <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>).
</p><!--l. 14--><p class="indent" >
<p>Synthetic lethal interactions arise due to <a 
href="#glo:functional redundancy">functional redundancy</a> (<a 
href="#XBoone2007">Boone <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoone2007">2007</a>; <a 
href="#XFece2015">Fece de la Cruz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFece2015">2015</a>; <a 
href="#XKaelin2005">Kaelin, Jr</a>, <a 
href="#XKaelin2005">2005</a>) and as such the <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">CDH1</span> indicate the wide-ranging biological functions that <a 
href="#glo:E-cadherin">E-cadherin</a> is involved in. The diverse <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways identified support the known pleiotropic nature of the <span class="cmti-12">CDH1 </span>gene (<a 
href="#XKroepil2012">Kroepil <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XKroepil2012">2012</a>), by detecting established functions of <span class="cmti-12">CDH1</span>, replicating candidates</p>
from an experimental screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), and identifying novel interactions with candidate genes and pathways for further investigation. The highly pleiotropic functions of <a 
href="#glo:E-cadherin">E-cadherin</a> was also consistent with <span class="cmti-12">CDH1 </span>being a <a 
href="#glo:tumour suppressor">tumour suppressor</a> gene. <a 
 id="x1-158001r354"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">7.1.1 </span> <a 
 id="x1-1590001"></a>Established Functions of <span class="cmti-12">CDH1</span>
</h4>
<!--l. 19--><p class="noindent" >
<span class="cmti-12">CDH1 </span>has established functions in cell-cell communication and maintaining the cytoskeleton, specifically with cell-cell adhesion by forming tight junctions and the adherens complex (<a 
href="#XJeanes2008">Jeanes <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJeanes2008">2008</a>). More recently, additional functions of <span class="cmti-12">CDH1 </span>in the extracellular matrix and fibrin clotting have also been identified (<a 
href="#XCardiff2011">Cardiff <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCardiff2011">2011</a>; <a 
href="#XTunggal2005">Tunggal <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTunggal2005">2005</a>; <a 
href="#XWojtukiewicz2016">Wojtukiewicz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWojtukiewicz2016">2016</a>). <a 
href="#glo:synthetic lethal">Synthetic lethal</a> interactions within biological pathways (i.e., partners in the same pathway as the query gene) are expected according to previous <a 
href="#glo:synthetic lethal">synthetic lethal</a> experiments (<a 
href="#XBoone2007">Boone <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoone2007">2007</a>; <a 
href="#XKelley2005">Kelley and Ideker</a>, <a 
href="#XKelley2005">2005</a>). <a 
href="#glo:synthetic lethal">Synthetic lethal</a> interactions identified in these pathways are consistent with these being functions of <span class="cmti-12">CDH1</span>, in addition to potentially actionable targets against cancers. <a 
 id="x1-159001r358"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">7.1.2 </span> <a 
 id="x1-1600002"></a>The Molecular Role of <span class="cmti-12">CDH1 </span>in Cancer
</h4>
<!--l. 26--><p class="noindent" >
The involvement of <span class="cmti-12">CDH1 </span>in the extracellular matrix is important in cancers as it indicates a mechanism by which <span class="cmti-12">CDH1 </span>loss may affect the tumour microenvironment, contributing to its role as a tumour and invasion suppressor. Furthermore, perturbations in the extracellular matrix and tumour microenvironment present a means by which to specifically inhibit (cancerous) <span class="cmti-12">CDH1</span>-deficient cells, in addition to those currently being considered. These may be further supported in further investigations with 3D cell culture, “organoid”, or mouse xenograft cancer models.
</p><!--l. 30--><p class="indent" >
<p>In contrast, many of the pathways involved in cell signalling, including <a 
href="#glo:GPCR">GPCRs</a>, were identified by <a 
href="#glo:SLIPT">SLIPT</a> in addition to the experimental screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). These support the previous results in cell line models, that these pathways are <a 
href="#glo:essential">essential</a> to the growth of <span class="cmti-12">CDH1</span>-deficient cancers and present a potential vulnerability specific to these (cancerous) cells. Furthermore, the replication of <a 
href="#glo:synthetic lethal">synthetic lethality</a> of <span class="cmti-12">CDH1 </span>with cell signalling pathways in <a 
href="#glo:TCGA">TCGA</a> data across cancer types and genetic backgrounds robustly supports these pathways being clinically applicable beyond the genetic background of the model system of <span class="cmti-12">CDH1</span><sup class="textsuperscript"><span class="cmr-10x-x-109">-/-</span></sup> MCF10A cells (<a 
href="#XChen2014">Chen</p>
<em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>). While the specific <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes were not as consistently detected between the <a 
href="#glo:SLIPT">SLIPT</a> analyses and <a 
href="#glo:siRNA">siRNA</a> screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), they were sufficient to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways for further experimental investigation, which are more likely to be replicated between genetic backgrounds (<a 
href="#XDixon2008">Dixon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDixon2008">2008</a>). Together these results demonstrate how <a 
href="#glo:SLIPT">SLIPT</a> can be integrated with an experimental screen to triage potential therapeutic targets for further pre-clinical investigation.
</p><!--l. 32--><p class="indent" >
The analysis of <a 
href="#glo:gene expression">expression</a> data with <a 
href="#glo:SLIPT">SLIPT</a> is also indicative of additional biological mechanisms of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in pathways beyond those identified in screening experiments (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). In particular, translation and regulatory pathways, involving 3<sup><span class="cmsy-8">′</span> </sup> <a 
href="#glo:UTR">UTRs</a> and <a 
href="#glo:NMD">NMD</a>, were identified as candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways with <span class="cmti-12">CDH1 </span>by <a 
href="#glo:SLIPT">SLIPT</a>. These pathways represent downstream targets regulated by the putative <a 
href="#glo:synthetic lethal">synthetic lethal</a> signalling pathways which cancer cells are dependent on to proliferate and evade host defense processes such as apoptosis and immune responses (<a 
href="#XGao2015">Gao and Roux</a>, <a 
href="#XGao2015">2015</a>) . <a 
 id="x1-160001r357"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">7.2 </span> <a 
 id="x1-1610002"></a>Significance
</h3>
<a 
 id="x1-161001r359"></a>
<h4 class="subsectionHead">
<span class="titlemark">7.2.1 </span> <a 
 id="x1-1620001"></a>Synthetic Lethality in the Genomic Era
</h4>
<!--l. 45--><p class="noindent" >
Development of an effective <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery tool for <a 
href="#glo:bioinformatics">bioinformatic</a> analysis has a wide range of applications in genetics research including functional <a 
href="#glo:genomic">genomics</a>, medical and agricultural applications. The <a 
href="#glo:SLIPT">SLIPT</a> approach demonstrated in this thesis is widely applicable to other genes and biological questions. In addition to further query of cancer genes, including other tissues, <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene functions are also of wider interest for their implications for <a 
href="#glo:functional redundancy">genetic redundancy</a>. Highly redundant genes, and the genetically robust systems they give rise to, are of further relevance to evolutionary, developmental, and systems biology to understand how these change over time and play a role in fundamental development of cell types, in addition to cancers (<a 
href="#XBoone2007">Boone <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoone2007">2007</a>; <a 
href="#XNowak1997">Nowak <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XNowak1997">1997</a>; <a 
href="#XTischler2008">Tischler <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTischler2008">2008</a>).
</p><!--l. 47--><p class="indent" >
<p>Developmental genes in particular, are highly evolutionarily conserved and subject to high rates of <a 
href="#glo:functional redundancy">redundancy</a> (<a 
href="#XFromental-Ramain1996">Fromental-Ramain <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFromental-Ramain1996">1996</a>; <a 
href="#XKockel1997">Kockel <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XKockel1997">1997</a>; <a 
href="#XNowak1997">Nowak <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XNowak1997">1997</a>). These are often difficult to study with conventional functional genetics since individual knockouts of redundant genes do not necessarily have a <a 
href="#glo:mutant">mutant</a> phenotype. Identifying genes with a common function is therefore also important</p>
to the study of developmental genes with unknown functions. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> discovery methods such as <a 
href="#glo:SLIPT">SLIPT</a> provide a <a 
href="#glo:genomic">genomic</a> approach to further systematic characterisation of gene function including such highly redundant developmental genes.
</p><!--l. 49--><p class="indent" >
Similarly, variants of unknown significance and modifier loci are a major concerns in human genetics, including “monogenic” and “rare” diseases. Many of these could potentially be difficult to characterise individually due to <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions where additional loci contribute to the disease (or only compensate for some variants). As such systematic identification of <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions also has applications in the study of such “oligogenic” diseases along with similar applications in the study of heritability for traits including agricultural <a 
href="#glo:genome">genomic</a> selection.
</p><!--l. 51--><p class="indent" >
<a 
href="#glo:functional redundancy">Genetic redundancy</a> is also a concern in pharmacology. Polypharmacology and network medicine are rationales to account for this by using drugs with multiple (known and specific) targets (<a 
href="#XBarabasi2011">Barabási <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBarabasi2011">2011</a>; <a 
href="#XHopkins2008">Hopkins</a>, <a 
href="#XHopkins2008">2008</a>). Further characterisation of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes will be valuable to the design of effective multi-target drugs or combination therapies in a range of therapeutic applications including molecular targeted therapies against cancer for which combination therapies are a popular solution for acquired resistance against individual targeted therapies. Characterisation of genetic interactions and combination therapies also has the potential to expand pharmacogenomic investigations. These may elucidate the impact of genotypes at multiple loci, which lead to adverse effects in a subset of the population due to variants in <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes.
</p><!--l. 53--><p class="indent" >
Furthermore, redundant functions and <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions also present a means to expand upon the concept of the “minimal” <a 
href="#glo:genome">genome</a> (<a 
href="#XHutchison2016">Hutchison <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XHutchison2016">2016</a>). It is important to account for <a 
href="#glo:essential">essential</a> gene functions that are performed by redundant genes (or in combination with <a 
href="#glo:pleiotropy">pleiotropic</a> genes), rather than simply those that are perturbed by individual genes. An <a 
href="#glo:essential">essential</a> gene approach is likely to produce an underestimate that does not account for <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions.
</p><!--l. 55--><p class="indent" >
<p><a 
href="#glo:synthetic lethal">Synthetic lethal</a> interactions are fundamentally important throughout genetics. Further understanding of them in a <a 
href="#glo:genomic">genomic</a> context, facilitated by methods such as <a 
href="#glo:SLIPT">SLIPT</a>, would contribute towards deeper understanding of gene functions and their role in traits or diseases in the post-genomic era. Genes do not function in isolation and understanding them in the context of the complexity of a cell and across genetic backgrounds is <a 
href="#glo:essential">essential</a> to further characterise their functions and ensure that findings can be validated or applied beyond experimental systems.</p>
<a 
 id="x1-162001r361"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">7.2.2 </span> <a 
 id="x1-1630002"></a>Clinical Interventions based on Synthetic Lethality
</h4>
<!--l. 61--><p class="noindent" >
Synthetic lethal discovery with <a 
href="#glo:SLIPT">SLIPT</a> is of particular interest in cancer research as a complementary approach to discovery of <a 
href="#glo:synthetic lethal">synthetic lethal</a> drug targets. The cancer research community relies on cell line and mouse models for screening and validation experiments (<a 
href="#XFece2015">Fece de la Cruz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFece2015">2015</a>) which would benefit from integration with <a 
href="#glo:gene expression">gene expression</a> analysis as demonstrated for <span class="cmti-12">CDH1 </span>and the screen conducted by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>). <a 
href="#glo:synthetic lethal">Synthetic lethal</a> drug design against cancer <a 
href="#glo:mutation">mutations</a>, including gene loss or over-expression, could lead to a revolution in cancer <a 
href="#glo:treatment">therapy</a> and <a 
href="#glo:chemoprevention">chemoprevention</a>. Such <a 
href="#glo:targeted therapy">therapeutics</a> would enable personalised treatment for cancer patients and high risk individuals. Examples of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> strategy (<a 
href="#XBryant2005">Bryant <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBryant2005">2005</a>; <a 
href="#XFarmer2005">Farmer <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFarmer2005">2005</a>) for cancer treatment have been shown to be clinically effective (<a 
href="#XMcLachlan2016">McLachlan <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMcLachlan2016">2016</a>). Many large-scale <a 
href="#glo:RNAi">RNAi</a> screens have been conducted recently, aiming to discover gene function and drug targets for similar application with other cancer genes, including cancers in other tissues (<a 
href="#XFece2015">Fece de la Cruz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFece2015">2015</a>).
</p><!--l. 68--><p class="indent" >
While <a 
href="#glo:SLIPT">SLIPT</a> analysis and <a 
href="#glo:RNAi">RNAi</a> screens represent a significant step towards anti-cancer medicines, further validation is required to ensure that the <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate genes and pathways identified for <span class="cmti-12">CDH1 </span>in breast and stomach cancer are applicable against <span class="cmti-12">CDH1 </span>-deficient cancers in the clinic. Validation with <a 
href="#glo:RNAi">RNAi</a> or pharmacological inhibitors is needed, since false positives may occur in <a 
href="#glo:SLIPT">SLIPT</a> analysis or <a 
href="#glo:siRNA">siRNA</a> screens. These candidates will need to be tested in pre-clinical models (cell lines and mouse xenografts) before proceeding to clinical trials. A therapeutic intervention will also require a <a 
href="#glo:targeted therapy">targeted therapeutic</a> to develop developed or repurposed against the <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner. Drug targets could be triaged from <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes by functions known to be amenable to drugs or structure with conserved specific sites that are not homologous to other genes, or those with existing drugs approved in trial for other applications. Both structure-aided drug design and compound screening are viable ways to target <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners.
</p><!--l. 72--><p class="indent" >
<p><a 
href="#glo:targeted therapy">Targeted therapeutics</a> designed based on <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions could expand the applications of “precision medicine” against molecular targets. <a 
href="#glo:synthetic lethal">Synthetic lethality</a> expands the range of cancer genes which can be (indirectly) targeted to include <a 
href="#glo:tumour suppressor">tumour suppressor</a> genes with loss of function, such as <span class="cmti-12">CDH1</span>. <a 
href="#glo:oncogene">Oncogenes</a> with disrupted functions that are over-expressed or highly homologous to non-cancerous proto-oncogenes, such as <span class="cmti-12">MYC</span>, <span class="cmti-12">EGFR</span></p>
or <span class="cmti-12">KRAS</span>, may also be targeted by <a 
href="#glo:synthetic lethal">synthetic lethality</a>. Applications against <a 
href="#glo:tumour suppressor">tumour suppressor</a> genes is particularly important, as these cannot be approached by careful dosing. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> drug design has the benefit of being highly specific against a particular genotype (such as CDH1<sup class="textsuperscript"><span class="cmr-10x-x-109">-/-</span></sup>) with the potential for <a 
href="#glo:targeted therapy">targeted therapies</a> with a wide therapeutic index and few adverse effects, in contrast to many current anti-cancer drug regimens (<a 
href="#XHopkins2008">Hopkins</a>, <a 
href="#XHopkins2008">2008</a>; <a 
href="#XKaelin2009">Kaelin, Jr</a>, <a 
href="#XKaelin2009">2009</a>). These properties are highly desirable for <a 
href="#glo:chemoprevention">chemoprevention</a> applications, such as treatment against <span class="cmti-12">CDH1 </span>-deficient in <a 
href="#glo:HDGC">HDGC</a> patients (<a 
href="#XGuilford2010">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford2010">2010</a>), as an alternative to monitoring or surgery. <a 
 id="x1-163001r360"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">7.3 </span> <a 
 id="x1-1640003"></a>Future Directions
</h3>
<!--l. 99--><p class="noindent" >
While further validation and pre-clinical testing is required to translate the findings for <span class="cmti-12">CDH1 </span>to cancer therapy or prevention, there are also further avenues for research into the detection of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in <a 
href="#glo:gene expression">gene expression</a> and other <a 
href="#glo:genomic">genomics</a> data. The <a 
href="#glo:SLIPT">SLIPT</a> methodology is amenable to wider application against a range of genes for which loss of function is deleterious, including other cancer genes in breast cancer or other tissues. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> interactions are functionally informative, particularly for mode-of-action of known drug targets, and are also relevant for identifying functions of newly characterised genes in <a 
href="#glo:genomic">genomics</a> studies and designing specific interventions against cells with loss of function in cancer and other diseases. Thus <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection using <a 
href="#glo:SLIPT">SLIPT</a> in <a 
href="#glo:gene expression">expression</a> data could be further used for many other genes, including others relevant to human health and disease.
</p><!--l. 101--><p class="indent" >
These investigations do not need to be limited to <a 
href="#glo:gene expression">expression</a> data. While <a 
href="#glo:gene expression">expression</a> as a measure of gene function has been the focus of this thesis, other <a 
href="#glo:genomic">genomics</a> data could be used for a similar purpose for <a 
href="#glo:SLIPT">SLIPT</a> analysis. These include <a 
href="#glo:DNA">DNA</a> copy number, <a 
href="#glo:DNA">DNA</a> methylation, histone activation, <a 
href="#glo:mutation">mutation</a> status, protein abundance, and protein activation state. In particular, <a 
href="#glo:DNA">DNA</a> copy number and <a 
href="#glo:mutation">mutations</a> have been demonstrated by other approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> analysis (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>; <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>; <a 
href="#XSrihari2015">Srihari <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XSrihari2015">2015</a>; <a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWappett2016">2016</a>), although some of these have not been released for wider application.
</p><!--l. 103--><p class="indent" >
<p>For some applications or genes, these molecular profiles may be more informative of gene function and <a 
href="#glo:synthetic lethal">synthetic lethal</a> relationships. However, <a 
href="#glo:gene expression">expression</a> was the focus of the investigations thus far as a widely accepted measure of gene function which has widely available <a 
href="#glo:genomic">genomics</a> data. <a 
href="#glo:SLIPT">SLIPT</a> is compatible with each of these data types (if the thresholds</p>
are selected appropriately) and may perform better for some applications with these molecular profiles or a weighted combination of these. As demonstrated, <a 
href="#glo:SLIPT">SLIPT</a> is also suitable for future investigations with pathway <a 
href="#glo:metagene">metagenes</a> and other summary data as well.
</p><!--l. 105--><p class="indent" >
It may also be possible to improve the performance of <a 
href="#glo:SLIPT">SLIPT</a> with refinements to the statistical or computational approach. This thesis has focused on rational query-based approach which computes relatively quickly in R (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>), and is relatively intuitive to interpret. These computations are compatible with parallel computing and the computational resources may be further reduced by using a different computing language. The <span class="cmtt-12">slipt </span>R package has been documented and released as open-source software (as described in Section <a 
href="#x1-870005">3.5<!--tex4ht:ref: methods:r_packages --></a>) to facilitate further development, wider adoption, or comparison with other scientific software for similar purposes.
</p><!--l. 107--><p class="indent" >
Alternative methods may be also improve on the statistical performance of <a 
href="#glo:SLIPT">SLIPT</a>. In particular, the sensitivity was generally as issue with higher numbers of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in simulated data. While approaches using continuous data such as Pearson correlation and linear regression did not perform as well as <a 
href="#glo:SLIPT">SLIPT</a>, they could be improved. A least squares regression approach in particular, enables multiple measures of relationships such as the coefficients of the fitted curve and significance of the fit (computed from the residuals). A linear modelling approach using regression is also amenable to refinement such as extending from fitting a linear relationship to a polynomial or logistic regression. Another benefit to fitting linear models is that these would enable the conditioning of known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners to identify subtle signatures of further interacting partners.
</p><!--l. 109--><p class="indent" >
<p>This approach could also be applied iteratively on the strongest candidates from previous <a 
href="#glo:synthetic lethal">synthetic lethal</a> analyses in further rounds of prediction conditioned upon them. Similarly, <a 
href="#glo:synthetic lethal">synthetic lethal</a> prediction could also be approached with a Bayesian framework (<a 
href="#XFriedman2000">Friedman <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFriedman2000">2000</a>; <a 
href="#XImoto2004">Imoto <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XImoto2004">2004</a>; <a 
href="#XJansen2003">Jansen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJansen2003">2003</a>) which is also amenable to Bayesian priors on known or previously predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners. Either of these approaches has the potential to improve upon the <a 
href="#glo:synthetic lethal">synthetic lethal</a> predictions which have been demonstrated as possible and biologically relevant by <a 
href="#glo:SLIPT">SLIPT</a>.</p>
<a 
 id="x1-164001r363"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">7.4 </span> <a 
 id="x1-1650004"></a>Conclusions
</h3>
<!--l. 7--><p class="noindent" >
Synthetic lethal interactions are important for understanding gene function and the development of highly specific <a 
href="#glo:targeted therapy">targeted</a> cancer <a 
href="#glo:treatment">treatments</a>. In particular, <a 
href="#glo:synthetic lethal">synthetic lethality</a> could expand the repertoire of applications for precision cancer medicine by indirectly targeting loss of function in <a 
href="#glo:tumour suppressor">tumour suppressor</a> genes. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> discovery with experimental screening is error-prone and limited by the model systems in which it is performed. Thus there is a need for a <a 
href="#glo:bioinformatics">bioinformatics</a> tool to predict <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions from <a 
href="#glo:gene expression">gene expression</a> data, which would facilitate the rapid identification of <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates, and augment functional genetic screens and triage of cancer drug targets. This thesis develops the <a 
href="#glo:SLIPT">Synthetic Lethal Interaction Prediction Tool</a> (<a 
href="#glo:SLIPT">SLIPT</a>) methodology as a statistically robust procedure to perform this analysis.
</p><!--l. 9--><p class="indent" >
The <a 
href="#glo:SLIPT">SLIPT</a> methodology has been demonstrated to identify biologically relevant genes and pathways. A comprehensive analysis of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of the <span class="cmti-12">CDH1 </span>gene was performed in <a 
href="#glo:TCGA">TCGA</a> breast cancer data (<a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>), with many of these findings replicated in stomach cancer data (<a 
href="#XTCGA2014GC">Bass <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014GC">2014</a>). These genes clustered into several distinct groups, with distinct biological functions and elevated <a 
href="#glo:gene expression">expression</a> in different clinical subtypes. These analyses identified <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates in the <span class="cmmi-12">G</span><sub><span class="cmmi-8">αi</span> </sub> signalling, cytoplasmic microfibres, and extracellular fibrin clotting pathways. These <a 
href="#glo:pathway">pathways</a> were supported by an <a 
href="#glo:siRNA">siRNA</a> screen performed by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) and were consistent with the known cytoskeletal and cell signalling roles of <a 
href="#glo:E-cadherin">E-cadherin</a>. <a 
href="#glo:SLIPT">SLIPT</a> also identified <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in novel pathways for <span class="cmti-12">CDH1</span>, including the regulation of immune signalling and translational elongation, which extend the range of established functions of <span class="cmti-12">CDH1 </span>and present further biological mechanisms that can be investigated to exploit the vulnerabilities of <span class="cmti-12">CDH1</span>-deficient cancers.
</p><!--l. 15--><p class="indent" >
<p>While some of these pathways are not expected to be detected in an isolated experimental cell line model, <a 
href="#glo:graph">pathway</a> structure may have accounted for this disparity. Thus <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates detected by <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> were compared within <a 
href="#glo:graph">graph</a> structures of the candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways. However, this did not generally account for differences between these approaches. Neither <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection methodology preferentially detected genes of more importance or connectivity in <a 
href="#glo:graph">pathway</a> structures using established network metrics, nor could it be generally established that</p>
<a 
href="#glo:SLIPT">SLIPT</a> gene candidates were upstream or downstream of <a 
href="#glo:siRNA">siRNA</a> gene candidates in <a 
href="#glo:graph">pathway</a> structures across biological pathways. However, it could be shown that <a 
href="#glo:SLIPT">SLIPT</a> genes had lower centrality and were upstream of <a 
href="#glo:siRNA">siRNA</a> candidates, specifically in the <span class="cmmi-12">G</span><sub><span class="cmmi-8">αi</span></sub> signalling pathway.
</p><!--l. 17--><p class="indent" >
<p>Pathway <a 
href="#glo:graph">graph</a> structures were also included in investigations with simulated data to ascertain whether the <a 
href="#glo:SLIPT">SLIPT</a> procedure performed well in data with complex correlation structures derived based on biological pathways. A simulation procedure was developed based on a statistical model of <a 
href="#glo:synthetic lethal">synthetic lethality</a> which generates multivariate normal data with known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners and correlation structures. The <a 
href="#glo:SLIPT">SLIPT</a> methodology had high statistical performance, particularly when detecting few known <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes, with large sample sizes, and a background of many non-synthetic lethal genes to distinguish true partners from. This method had high specificity, performed better than Pearson correlation or the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> -test, and had had optimal performance across simulation parameter combinations for the thresholds used throughout this thesis. These findings were robust across correlation structures, including those derived from complex <a 
href="#glo:graph">pathway</a> structures containing strong positive and negative correlations between genes. Together, these findings support the release of the <a 
href="#glo:SLIPT">SLIPT</a> software R packages and the application of the method to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes within pathways and use candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> pathways, as demonstrated in this thesis.</p>
<p><a 
 id="Q1-1-365"></a></p>
</p>
<h2 class="likechapterHead">
<a 
 id="x1-1660004"></a>Bibliography
</h2>
<div class="thebibliography">
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XAarts2015"></a><span class="bibsp">   </span></span>Aarts, M., Bajrami, I., Herrera-Abreu, M.T., Elliott, R., Brough, R., Ashworth, A., Lord, C.J., and Turner, N.C. (2015) Functional genetic screen identifies increased sensitivity to wee1 inhibition in cells with defects in fanconi anemia and hr pathways. <em><span class="cmti-12">Mol Cancer Ther</span></em>, <strong><span class="cmbx-12">14</span></strong>(4): 865–76.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTCGA2015PR"></a><span class="bibsp">   </span></span>Abeshouse, A., Ahn, J., Akbani, R., Ally, A., Amin, S., Andry, C.D., Annala, M., Aprikian, A., Armenia, J., Arora, A., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) The Molecular Taxonomy of Primary Prostate Cancer. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">163</span></strong>(4): 1011–1025.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="Xvioplot"></a><span class="bibsp">   </span></span>Adler, D. (2005) <em><span class="cmti-12">vioplot: Violin plot</span></em>. R package version 0.2.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTCGA2015SK"></a><span class="bibsp">   </span></span>Akbani, R., Akdemir, K.C., Aksoy, B.A., Albert, M., Ally, A., Amin, S.B., Arachchi, H., Arora, A., Auman, J.T., Ayala, B., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) Genomic Classification of Cutaneous Melanoma. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">161</span></strong>(7): 1681–1696.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XAkobeng2007"></a><span class="bibsp">   </span></span>Akobeng, A.K. (2007) Understanding diagnostic tests 3: receiver operating characteristic curves. <em><span class="cmti-12">Acta Pædiatrica</span></em>, <strong><span class="cmbx-12">96</span></strong>(5): 644–647.
</p>
                                                                                    

<p class="bibitem" >
<span class="biblabel"> <a 
 id="XACS2017"></a><span class="bibsp">   </span></span>American Cancer Society (2017) Genetics and cancer. <a 
href="https://www.cancer.org/cancer/cancer-causes/genetics.html" class="url" ><span class="cmtt-12">https://www.cancer.org/cancer/cancer-_causes/genetics.html</span></a>. Accessed: 22/03/2017.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XAnjomshoaa2008"></a><span class="bibsp">   </span></span>Anjomshoaa, A., Lin, Y.H., Black, M.A., McCall, J.L., Humar, B., Song, S., Fukuzawa, R., Yoon, H.S., Holzmann, B., Friederichs, J., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2008) Reduced expression of a gene proliferation signature is associated with enhanced malignancy in colon cancer. <em><span class="cmti-12">Br J Cancer</span></em>, <strong><span class="cmbx-12">99</span></strong>(6): 966–973.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="Xgenesetdb"></a><span class="bibsp">   </span></span>Araki, H., Knapp, C., Tsai, P., and Print, C. (2012) GeneSetDB: A comprehensive meta-database, statistical and visualisation framework for gene set analysis. <em><span class="cmti-12">FEBS</span> <span class="cmti-12">Open Bio</span></em>, <strong><span class="cmbx-12">2</span></strong>: 76–82.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XAshburner2000"></a><span class="bibsp">   </span></span>Ashburner, M., Ball, C.A., Blake, J.A., Botstein, D., Butler, H., Cherry, J.M., Davis, A.P., Dolinski, K., Dwight, S.S., Eppig, J.T., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2000) Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">25</span></strong>(1): 25–29.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XAshworth2008"></a><span class="bibsp">   </span></span>Ashworth, A. (2008) A synthetic lethal therapeutic approach: poly(adp) ribose polymerase inhibitors for the treatment of cancers deficient in dna double-strand break repair. <em><span class="cmti-12">J Clin Oncol</span></em>, <strong><span class="cmbx-12">26</span></strong>(22): 3785–90.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XAshworth2011"></a><span class="bibsp">   </span></span>Ashworth, A., Lord, C.J., and Reis-Filho, J.S. (2011) Genetic interactions in cancer progression and treatment. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">145</span></strong>(1): 30–38.
</p>
                                                                                    

<p class="bibitem" >
<span class="biblabel"> <a 
 id="XAudeh2010"></a><span class="bibsp">   </span></span>Audeh, M.W., Carmichael, J., Penson, R.T., Friedlander, M., Powell, B., Bell-McGuinn, K.M., Scott, C., Weitzel, J.N., Oaknin, A., Loman, N., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2010) Oral poly(adp-ribose) polymerase inhibitor olaparib in patients with <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>mutations and recurrent ovarian cancer: a proof-of-concept trial. <em><span class="cmti-12">Lancet</span></em>, <strong><span class="cmbx-12">376</span></strong>(9737): 245–51.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBabyak2004"></a><span class="bibsp">   </span></span>Babyak, M.A. (2004) What you see may not be what you get: a brief, nontechnical introduction to overfitting in regression-type models. <em><span class="cmti-12">Psychosom Med</span></em>, <strong><span class="cmbx-12">66</span></strong>(3): 411–21.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBamford2004"></a><span class="bibsp">   </span></span>Bamford, S., Dawson, E., Forbes, S., Clements, J., Pettett, R., Dogan, A., Flanagan, A., Teague, J., Futreal, P.A., Stratton, M.R., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2004) The COSMIC (Catalogue of Somatic Mutations in Cancer) database and website. <em><span class="cmti-12">Br J Cancer</span></em>, <strong><span class="cmbx-12">91</span></strong>(2): 355–358.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBarabasi1999"></a><span class="bibsp">   </span></span>Barabási, A.L. and Albert, R. (1999) Emergence of scaling in random networks. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">286</span></strong>(5439): 509–12.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBarabasi2011"></a><span class="bibsp">   </span></span>Barabási, A.L., Gulbahce, N., and Loscalzo, J. (2011) Network medicine: a network-based approach to human disease. <em><span class="cmti-12">Nat Rev Genet</span></em>, <strong><span class="cmbx-12">12</span></strong>(1): 56–68.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBarabasi2004"></a><span class="bibsp">   </span></span>Barabási, A.L. and Oltvai, Z.N. (2004) Network biology: understanding the cell’s functional organization. <em><span class="cmti-12">Nat Rev Genet</span></em>, <strong><span class="cmbx-12">5</span></strong>(2): 101–13.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XBarrat2000"></a><span class="bibsp">   </span></span>Barrat, A. and Weigt, M. (2000) On the properties of small-world network models.</p>
<em><span class="cmti-12">The European Physical Journal B - Condensed Matter and Complex Systems</span></em>, <strong><span class="cmbx-12">13</span></strong>(3): 547–560.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBarretina2012"></a><span class="bibsp">   </span></span>Barretina, J., Caponigro, G., Stransky, N., Venkatesan, K., Margolin, A.A., Kim, S., Wilson, C.J., Lehar, J., Kryukov, G.V., Sonkin, D., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2012) The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">483</span></strong>(7391): 603–607.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="Xsafe"></a><span class="bibsp">   </span></span>Barry, W.T. (2016) <em><span class="cmti-12">safe: Significance Analysis of Function and Expression</span></em>. R package version 3.14.0.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBaryshnikova2010a"></a><span class="bibsp">   </span></span>Baryshnikova, A., Costanzo, M., Dixon, S., Vizeacoumar, F.J., Myers, C.L., Andrews, B., and Boone, C. (2010a) Synthetic genetic array (sga) analysis in saccharomyces cerevisiae and schizosaccharomyces pombe. <em><span class="cmti-12">Methods Enzymol</span></em>, <strong><span class="cmbx-12">470</span></strong>: 145–79.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBaryshnikova2010b"></a><span class="bibsp">   </span></span>Baryshnikova, A., Costanzo, M., Kim, Y., Ding, H., Koh, J., Toufighi, K., Youn, J.Y., Ou, J., San Luis, B.J., Bandyopadhyay, S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2010b) Quantitative analysis of fitness and genetic interactions in yeast on a genome scale. <em><span class="cmti-12">Nat Meth</span></em>, <strong><span class="cmbx-12">7</span></strong>(12): 1017–1024.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTCGA2014GC"></a><span class="bibsp">   </span></span>Bass, A.J., Thorsson, V., Shmulevich, I., Reynolds, S.M., Miller, M., Bernard, B., Hinoue, T., Laird, P.W., Curtis, C., Shen, H., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Comprehensive molecular characterization of gastric adenocarcinoma. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">513</span></strong>(7517): 202–209.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XMatrix"></a><span class="bibsp">   </span></span>Bates, D. and Maechler, M. (2016) <em><span class="cmti-12">Matrix: Sparse and Dense Matrix Classes and</span> <span class="cmti-12">Methods</span></em>. R package version 1.2-7.1.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBateson1909"></a><span class="bibsp">   </span></span>Bateson, W. and Mendel, G. (1909) <em><span class="cmti-12">Mendel’s principles of heredity, by W. Bateson</span></em>. University Press, Cambridge [Eng.].
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBecker1994"></a><span class="bibsp">   </span></span>Becker, K.F., Atkinson, M.J., Reich, U., Becker, I., Nekarda, H., Siewert, J.R., and Höfler, H. (1994) E-cadherin gene mutations provide clues to diffuse type gastric carcinomas. <em><span class="cmti-12">Cancer Research</span></em>, <strong><span class="cmbx-12">54</span></strong>(14): 3845–3852.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTCGA2011OV"></a><span class="bibsp">   </span></span>Bell, D., Berchuck, A., Birrer, M., Chien, J., Cramer, D., Dao, F., Dhir, R., DiSaia, P., Gabra, H., Glenn, P., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2011) Integrated genomic analyses of ovarian carcinoma. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">474</span></strong>(7353): 609–615.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="Xfdr1995"></a><span class="bibsp">   </span></span>Benjamini, Y. and Hochberg, Y. (1995) Controlling the false discovery rate: A practical and powerful approach to multiple testing. <em><span class="cmti-12">Journal of the Royal Statistical</span> <span class="cmti-12">Society Series B (Methodological)</span></em>, <strong><span class="cmbx-12">57</span></strong>(1): 289–300.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBerx1995"></a><span class="bibsp">   </span></span>Berx, G., Cleton-Jansen, A.M., Nollet, F., de Leeuw, W.J., van de Vijver, M., Cornelisse, C., and van Roy, F. (1995) E-cadherin is a tumour/invasion suppressor gene mutated in human lobular breast cancers. <em><span class="cmti-12">EMBO J</span></em>, <strong><span class="cmbx-12">14</span></strong>(24): 6107–15.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBerx1996"></a><span class="bibsp">   </span></span>Berx, G., Cleton-Jansen, A.M., Strumane, K., de Leeuw, W.J., Nollet, F., van Roy, F., and Cornelisse, C. (1996) E-cadherin is inactivated in a majority of invasive human lobular breast cancers by truncation mutations throughout its extracellular domain. <em><span class="cmti-12">Oncogene</span></em>, <strong><span class="cmbx-12">13</span></strong>(9): 1919–25.
</p>
                                                                                    

<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBerx2009"></a><span class="bibsp">   </span></span>Berx, G. and van Roy, F. (2009) Involvement of members of the cadherin superfamily in cancer. <em><span class="cmti-12">Cold Spring Harb Perspect Biol</span></em>, <strong><span class="cmbx-12">1</span></strong>: a003129.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBitler2015"></a><span class="bibsp">   </span></span>Bitler, B.G., Aird, K.M., Garipov, A., Li, H., Amatangelo, M., Kossenkov, A.V., Schultz, D.C., Liu, Q., Shih Ie, M., Conejo-Garcia, J.R., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) Synthetic lethality by targeting ezh2 methyltransferase activity in arid1a-mutated cancers. <em><span class="cmti-12">Nat</span> <span class="cmti-12">Med</span></em>, <strong><span class="cmbx-12">21</span></strong>(3): 231–8.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBlake2015"></a><span class="bibsp">   </span></span>Blake, J.A., Christie, K.R., Dolan, M.E., Drabkin, H.J., Hill, D.P., Ni, L., Sitnikov, D., Burgess, S., Buza, T., Gresham, C., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) Gene Ontology Consortium: going forward. <em><span class="cmti-12">Nucleic Acids Res</span></em>, <strong><span class="cmbx-12">43</span></strong>(Database issue): D1049–1056.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBoettcher2014"></a><span class="bibsp">   </span></span>Boettcher, M., Lawson, A., Ladenburger, V., Fredebohm, J., Wolf, J., Hoheisel, J.D., Frezza, C., and Shlomi, T. (2014) High throughput synthetic lethality screen reveals a tumorigenic role of adenylate cyclase in fumarate hydratase-deficient cancer cells. <em><span class="cmti-12">BMC Genomics</span></em>, <strong><span class="cmbx-12">15</span></strong>: 158.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBoone2007"></a><span class="bibsp">   </span></span>Boone, C., Bussey, H., and Andrews, B.J. (2007) Exploring genetic interactions and networks with yeast. <em><span class="cmti-12">Nat Rev Genet</span></em>, <strong><span class="cmbx-12">8</span></strong>(6): 437–49.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBorgatti2005"></a><span class="bibsp">   </span></span>Borgatti, S.P. (2005) Centrality and network flow. <em><span class="cmti-12">Social Networks</span></em>, <strong><span class="cmbx-12">27</span></strong>(1): 55 – 71.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XBoucher2013"></a><span class="bibsp">   </span></span>Boucher, B. and Jenna, S. (2013) Genetic interaction networks: better understand to better predict. <em><span class="cmti-12">Front Genet</span></em>, <strong><span class="cmbx-12">4</span></strong>: 290.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBozovic-Spasojevic2012"></a><span class="bibsp">   </span></span>Bozovic-Spasojevic, I., Azambuja, E., McCaskill-Stevens, W., Dinh, P., and Cardoso, F. (2012) Chemoprevention for breast cancer. <em><span class="cmti-12">Cancer treatment reviews</span></em>, <strong><span class="cmbx-12">38</span></strong>(5): 329–339.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBreiman2001"></a><span class="bibsp">   </span></span>Breiman, L. (2001) Random forests. <em><span class="cmti-12">Machine Learning</span></em>, <strong><span class="cmbx-12">45</span></strong>(1): 5–32.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBrin1998"></a><span class="bibsp">   </span></span>Brin, S. and Page, L. (1998) The anatomy of a large-scale hypertextual web search engine. <em><span class="cmti-12">Computer Networks and ISDN Systems</span></em>, <strong><span class="cmbx-12">30</span></strong>(1): 107 – 117.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBruckner2009"></a><span class="bibsp">   </span></span>Brückner, A., Polge, C., Lentze, N., Auerbach, D., and Schlattner, U. (2009) Yeast two-hybrid, a powerful tool for systems biology. <em><span class="cmti-12">Int J Mol Sci</span></em>, <strong><span class="cmbx-12">10</span></strong>(6): 2763–2788.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBryant2005"></a><span class="bibsp">   </span></span>Bryant, H.E., Schultz, N., Thomas, H.D., Parker, K.M., Flower, D., Lopez, E., Kyle, S., Meuth, M., Curtin, N.J., and Helleday, T. (2005) Specific killing of <span class="cmti-12">BRCA2 </span>deficient tumours with inhibitors of poly<img 
src="thesis10x.png" alt="adpribose  "  class="math"  /> polymerase. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">434</span></strong>(7035): 913–7.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XBussey2006"></a><span class="bibsp">   </span></span>Bussey, H., Andrews, B., and Boone, C. (2006) From worm genetic networks to complex human diseases. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">38</span></strong>(8): 862–3.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XButland2008"></a><span class="bibsp">   </span></span>Butland, G., Babu, M., Diaz-Mejia, J.J., Bohdana, F., Phanse, S., Gold, B., Yang, W., Li, J., Gagarinova, A.G., Pogoutse, O., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2008) esga: E. coli synthetic genetic array analysis. <em><span class="cmti-12">Nat Methods</span></em>, <strong><span class="cmbx-12">5</span></strong>(9): 789–95.
</p>
                                                                                    

<p class="bibitem" >
<span class="biblabel"> <a 
 id="XCardiff2011"></a><span class="bibsp">   </span></span>Cardiff, R.D., Couto, S., and Bolon, B. (2011) Three interrelated themes in current breast cancer research: gene addiction, phenotypic plasticity, and cancer stem cells. <em><span class="cmti-12">Breast Cancer Res</span></em>, <strong><span class="cmbx-12">13</span></strong>(5): 216.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XcBioPortal"></a><span class="bibsp">   </span></span>cBioPortal for Cancer Genomics (cBioPortal) (2017) cBioPortal for Cancer Genomics. <a 
href="http://www.cbioportal.org/" class="url" ><span class="cmtt-12">http://www.cbioportal.org/</span></a>. Accessed: 26/03/2017.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XPathwayCommons"></a><span class="bibsp">   </span></span>Cerami, E.G., Gross, B.E., Demir, E., Rodchenkov, I., Babur, O., Anwar, N., Schultz, N., Bader, G.D., and Sander, C. (2011) Pathway Commons, a web resource for biological pathway data. <em><span class="cmti-12">Nucleic Acids Res</span></em>, <strong><span class="cmbx-12">39</span></strong>(Database issue): D685–690.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XChen2014"></a><span class="bibsp">   </span></span>Chen, A., Beetham, H., Black, M.A., Priya, R., Telford, B.J., Guest, J., Wiggins, G.A.R., Godwin, T.D., Yap, A.S., and Guilford, P.J. (2014) E-cadherin loss alters cytoskeletal organization and adhesion in non-malignant breast cells but is insufficient to induce an epithelial-mesenchymal transition. <em><span class="cmti-12">BMC Cancer</span></em>, <strong><span class="cmbx-12">14</span></strong>(1): 552.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XChen2007"></a><span class="bibsp">   </span></span>Chen, S. and Parmigiani, G. (2007) Meta-analysis of BRCA1 and BRCA2 penetrance. <em><span class="cmti-12">J Clin Oncol</span></em>, <strong><span class="cmbx-12">25</span></strong>(11): 1329–1333.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XChipman2009"></a><span class="bibsp">   </span></span>Chipman, K. and Singh, A. (2009) Predicting genetic interactions with random walks on biological networks. <em><span class="cmti-12">BMC Bioinformatics</span></em>, <strong><span class="cmbx-12">10</span></strong>(1): 17.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XChristofori1999"></a><span class="bibsp">   </span></span>Christofori, G. and Semb, H. (1999) The role of the cell-adhesion molecule E-cadherin as a tumour-suppressor gene. <em><span class="cmti-12">Trends in Biochemical Sciences</span></em>, <strong><span class="cmbx-12">24</span></strong>(2): 73 – 76.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTCGA2015LBC"></a><span class="bibsp">   </span></span>Ciriello, G., Gatza, M.L., Beck, A.H., Wilkerson, M.D., Rhie, S.K., Pastore, A., Zhang, H., McLellan, M., Yau, C., Kandoth, C., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) Comprehensive Molecular Portraits of Invasive Lobular Breast Cancer. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">163</span></strong>(2): 506–519.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XClark2004"></a><span class="bibsp">   </span></span>Clark, M.J. (2004) Endogenous Regulator of G Protein Signaling Proteins Suppress G o-Dependent  -Opioid Agonist-Mediated Adenylyl Cyclase Supersensitization. <em><span class="cmti-12">Journal of Pharmacology and Experimental Therapeutics</span></em>, <strong><span class="cmbx-12">310</span></strong>(1): 215–222.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XCollingridge2013"></a><span class="bibsp">   </span></span>Collingridge, D.S. (2013) A primer on quantitized data analysis and permutation testing. <em><span class="cmti-12">Journal of Mixed Methods Research</span></em>, <strong><span class="cmbx-12">7</span></strong>(1): 81–97.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XCollins2007"></a><span class="bibsp">   </span></span>Collins, F.S. and Barker, A.D. (2007) Mapping the cancer genome. Pinpointing the genes involved in cancer will help chart a new course across the complex landscape of human malignancies. <em><span class="cmti-12">Sci Am</span></em>, <strong><span class="cmbx-12">296</span></strong>(3): 50–57.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTCGA2014LU"></a><span class="bibsp">   </span></span>Collisson, E., Campbell, J., Brooks, A., Berger, A., Lee, W., Chmielecki, J., Beer, D., Cope, L., Creighton, C., Danilova, L., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Comprehensive molecular profiling of lung adenocarcinoma. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">511</span></strong>(7511): 543–550.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XCostanzo2010"></a><span class="bibsp">   </span></span>Costanzo, M., Baryshnikova, A., Bellay, J., Kim, Y., Spear, E.D., Sevier, C.S., Ding, H., Koh, J.L., Toufighi, K., Mostafavi, S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2010) The genetic landscape of a cell. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">327</span></strong>(5964): 425–31.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XCostanzo2011"></a><span class="bibsp">   </span></span>Costanzo, M., Baryshnikova, A., Myers, C.L., Andrews, B., and Boone, C. (2011) Charting the genetic interaction map of a cell. <em><span class="cmti-12">Curr Opin Biotechnol</span></em>, <strong><span class="cmbx-12">22</span></strong>(1): 66–74.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XCourtney2010"></a><span class="bibsp">   </span></span>Courtney, K.D., Corcoran, R.B., and Engelman, J.A. (2010) The PI3K pathway as drug target in human cancer. <em><span class="cmti-12">J Clin Oncol</span></em>, <strong><span class="cmbx-12">28</span></strong>(6): 1075–1083.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTCGA2013RCC"></a><span class="bibsp">   </span></span>Creighton, C.J., Morgan, M., Gunaratne, P.H., Wheeler, D.A., Gibbs, R.A., Robertson, A., Chu, A., Beroukhim, R., Cibulskis, K., Signoretti, S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2013) Comprehensive molecular characterization of clear cell renal cell carcinoma. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">499</span></strong>(7456): 43–49.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XReactome"></a><span class="bibsp">   </span></span>Croft, D., Mundo, A.F., Haw, R., Milacic, M., Weiser, J., Wu, G., Caudy, M., Garapati, P., Gillespie, M., Kamdar, M.R., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) The Reactome pathway knowledgebase. <em><span class="cmti-12">Nucleic Acids Res</span></em>, <strong><span class="cmbx-12">42</span></strong>(database issue): D472–D477.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XCrunkhorn2014"></a><span class="bibsp">   </span></span>Crunkhorn, S. (2014) Cancer: Predicting synthetic lethal interactions. <em><span class="cmti-12">Nat Rev</span> <span class="cmti-12">Drug Discov</span></em>, <strong><span class="cmbx-12">13</span></strong>(11): 812.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="Xigraph"></a><span class="bibsp">   </span></span>Csardi, G. and Nepusz, T. (2006) The igraph software package for complex network research. <em><span class="cmti-12">InterJournal</span></em>, <strong><span class="cmbx-12">Complex Systems</span></strong>: 1695.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XDai2015"></a><span class="bibsp">   </span></span>Dai, X., Li, T., Bai, Z., Yang, Y., Liu, X., Zhan, J., and Shi, B. (2015) Breast cancer intrinsic subtype classification, clinical use and future trends. <em><span class="cmti-12">Am J Cancer</span> <span class="cmti-12">Res</span></em>, <strong><span class="cmbx-12">5</span></strong>(10): 2929–2943.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XDavierwala2005"></a><span class="bibsp">   </span></span>Davierwala, A.P., Haynes, J., Li, Z., Brost, R.L., Robinson, M.D., Yu, L., Mnaimneh, S., Ding, H., Zhu, H., Chen, Y., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2005) The synthetic genetic interaction spectrum of essential genes. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">37</span></strong>(10): 1147–1152.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XDeLeeuw1997"></a><span class="bibsp">   </span></span>De Leeuw, W.J., Berx, G., Vos, C.B., Peterse, J.L., Van de Vijver, M.J., Litvinov, S., Van Roy, F., Cornelisse, C.J., and Cleton-Jansen, A.M. (1997) Simultaneous loss of E-cadherin and catenins in invasive lobular breast cancer and lobular carcinoma in situ. <em><span class="cmti-12">J Pathol</span></em>, <strong><span class="cmbx-12">183</span></strong>(4): 404–11.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XDeSantis2009"></a><span class="bibsp">   </span></span>De Santis, G., Miotti, S., Mazzi, M., Canevari, S., and Tomassetti, A. (2009) E-cadherin directly contributes to PI3K/AKT activation by engaging the PI3K-p85 regulatory subunit to adherens junctions of ovarian carcinoma cells. <em><span class="cmti-12">Oncogene</span></em>, <strong><span class="cmbx-12">28</span></strong>(9): 1206–1217.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="Xpaxtools"></a><span class="bibsp">   </span></span>Demir, E., Babur, O., Rodchenkov, I., Aksoy, B.A., Fukuda, K.I., Gross, B., Sumer, O.S., Bader, G.D., and Sander, C. (2013) Using biological pathway data with Paxtools. <em><span class="cmti-12">PLoS Comput Biol</span></em>, <strong><span class="cmbx-12">9</span></strong>(9): e1003194.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XDeshpande2013"></a><span class="bibsp">   </span></span>Deshpande, R., Asiedu, M.K., Klebig, M., Sutor, S., Kuzmin, E., Nelson, J., Piotrowski, J., Shin, S.H., Yoshida, M., Costanzo, M., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2013) A comparative genomic approach for identifying synthetic lethal interactions in human cancer. <em><span class="cmti-12">Cancer Res</span></em>, <strong><span class="cmbx-12">73</span></strong>(20): 6128–36.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XDickson1999"></a><span class="bibsp">   </span></span>Dickson, D. (1999) Wellcome funds cancer database. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">401</span></strong>(6755): 729.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XDijkstra1959"></a><span class="bibsp">   </span></span>Dijkstra, E.W. (1959) A note on two problems in connexion with graphs. <em><span class="cmti-12">Numerische Mathematik</span></em>, <strong><span class="cmbx-12">1</span></strong>(1): 269–271.
</p>
                                                                                    

<p class="bibitem" >
<span class="biblabel"> <a 
 id="XDixon2009a"></a><span class="bibsp">   </span></span>Dixon, S.J., Andrews, B.J., and Boone, C. (2009) Exploring the conservation of synthetic lethal genetic interaction networks. <em><span class="cmti-12">Commun Integr Biol</span></em>, <strong><span class="cmbx-12">2</span></strong>(2): 78–81.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XDixon2008"></a><span class="bibsp">   </span></span>Dixon, S.J., Fedyshyn, Y., Koh, J.L., Prasad, T.S., Chahwan, C., Chua, G., Toufighi, K., Baryshnikova, A., Hayles, J., Hoe, K.L., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2008) Significant conservation of synthetic lethal genetic interaction networks between distantly related eukaryotes. <em><span class="cmti-12">Proc Natl Acad Sci U S A</span></em>, <strong><span class="cmbx-12">105</span></strong>(43): 16653–8.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XDorsam2007"></a><span class="bibsp">   </span></span>Dorsam, R.T. and Gutkind, J.S. (2007) G-protein-coupled receptors and cancer. <em><span class="cmti-12">Nat Rev Cancer</span></em>, <strong><span class="cmbx-12">7</span></strong>(2): 79–94.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XErdos1959"></a><span class="bibsp">   </span></span>Erd<img 
src="thesis11x.png" alt="&#x02DD;o "  class="Huml"  />s, P. and Rényi, A. (1959) On random graphs I. <em><span class="cmti-12">Publ Math Debrecen</span></em>, <strong><span class="cmbx-12">6</span></strong>: 290–297.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XErdos1960"></a><span class="bibsp">   </span></span>Erd<img 
src="thesis12x.png" alt="&#x02DD;o "  class="Huml"  />s, P. and Rényi, A. (1960) On the evolution of random graphs. <em><span class="cmti-12">Publ Math</span> <span class="cmti-12">Inst Hung Acad Sci</span></em>, <strong><span class="cmbx-12">5</span></strong>(1): 17–61.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XEroles2012"></a><span class="bibsp">   </span></span>Eroles, P., Bosch, A., Perez-Fidalgo, J.A., and Lluch, A. (2012) Molecular biology in breast cancer: intrinsic subtypes and signaling pathways. <em><span class="cmti-12">Cancer Treat Rev</span></em>, <strong><span class="cmbx-12">38</span></strong>(6): 698–707.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XFarmer2005"></a><span class="bibsp">   </span></span>Farmer, H., McCabe, N., Lord, C.J., Tutt, A.N., Johnson, D.A., Richardson, T.B., Santarosa, M., Dillon, K.J., Hickson, I., Knights, C., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2005) Targeting the dna repair defect in BRCA mutant cells as a therapeutic strategy. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">434</span></strong>(7035): 917–21.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XFawcett2006"></a><span class="bibsp">   </span></span>Fawcett, T. (2006) An introduction to ROC analysis. <em><span class="cmti-12">Pattern Recognition Letters</span></em>, <strong><span class="cmbx-12">27</span></strong>(8): 861 – 874. <span class="cmsy-10x-x-120">{</span>ROC<span class="cmsy-10x-x-120">} </span>Analysis in Pattern Recognition.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XFece2015"></a><span class="bibsp">   </span></span>Fece de la Cruz, F., Gapp, B.V., and Nijman, S.M. (2015) Synthetic lethal vulnerabilities of cancer. <em><span class="cmti-12">Annu Rev Pharmacol Toxicol</span></em>, <strong><span class="cmbx-12">55</span></strong>: 513–531.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XFerlay2015"></a><span class="bibsp">   </span></span>Ferlay, J., Soerjomataram, I., Dikshit, R., Eser, S., Mathers, C., Rebelo, M., Parkin, D.M., Forman, D., and Bray, F. (2015) Cancer incidence and mortality worldwide: sources, methods and major patterns in GLOBOCAN 2012. <em><span class="cmti-12">Int J Cancer</span></em>, <strong><span class="cmbx-12">136</span></strong>(5): E359–386.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XFisher1919"></a><span class="bibsp">   </span></span>Fisher, R.A. (1919) Xv.—the correlation between relatives on the supposition of mendelian inheritance. <em><span class="cmti-12">Earth and Environmental Science Transactions of the Royal</span> <span class="cmti-12">Society of Edinburgh</span></em>, <strong><span class="cmbx-12">52</span></strong>(02): 399–433.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XFong2009"></a><span class="bibsp">   </span></span>Fong, P.C., Boss, D.S., Yap, T.A., Tutt, A., Wu, P., Mergui-Roelvink, M., Mortimer, P., Swaisland, H., Lau, A., O’Connor, M.J., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2009) Inhibition of poly(adp-ribose) polymerase in tumors from BRCA mutation carriers. <em><span class="cmti-12">N Engl J</span> <span class="cmti-12">Med</span></em>, <strong><span class="cmbx-12">361</span></strong>(2): 123–34.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XFong2010"></a><span class="bibsp">   </span></span>Fong, P.C., Yap, T.A., Boss, D.S., Carden, C.P., Mergui-Roelvink, M., Gourley, C., De Greve, J., Lubinski, J., Shanley, S., Messiou, C., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2010) Poly(adp)-ribose polymerase inhibition: frequent durable responses in BRCA carrier ovarian cancer correlating with platinum-free interval. <em><span class="cmti-12">J Clin Oncol</span></em>, <strong><span class="cmbx-12">28</span></strong>(15): 2512–9.
</p>
                                                                                    

<p class="bibitem" >
<span class="biblabel"> <a 
 id="XForbes2015"></a><span class="bibsp">   </span></span>Forbes, S.A., Beare, D., Gunasekaran, P., Leung, K., Bindal, N., Boutselakis, H., Ding, M., Bamford, S., Cole, C., Ward, S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) COSMIC: exploring the world’s knowledge of somatic mutations in human cancer. <em><span class="cmti-12">Nucleic Acids Res</span></em>, <strong><span class="cmbx-12">43</span></strong>(Database issue): D805–811.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XFraser2004"></a><span class="bibsp">   </span></span>Fraser, A. (2004) Towards full employment: using RNAi to find roles for the redundant. <em><span class="cmti-12">Oncogene</span></em>, <strong><span class="cmbx-12">23</span></strong>(51): 8346–52.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XFriedman2000"></a><span class="bibsp">   </span></span>Friedman, N., Linial, M., Nachman, I., and Pe’er, D. (2000) Using Bayesian networks to analyze expression data. <em><span class="cmti-12">J Comput Biol</span></em>, <strong><span class="cmbx-12">7</span></strong>(3-4): 601–620.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XFromental-Ramain1996"></a><span class="bibsp">   </span></span>Fromental-Ramain, C., Warot, X., Lakkaraju, S., Favier, B., Haack, H., Birling, C., Dierich, A., Doll e, P., and Chambon, P. (1996) Specific and redundant functions of the paralogous Hoxa-9 and Hoxd-9 genes in forelimb and axial skeleton patterning. <em><span class="cmti-12">Development</span></em>, <strong><span class="cmbx-12">122</span></strong>(2): 461–472.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XFutreal2004"></a><span class="bibsp">   </span></span>Futreal, P.A., Coin, L., Marshall, M., Down, T., Hubbard, T., Wooster, R., Rahman, N., and Stratton, M.R. (2004) A census of human cancer genes. <em><span class="cmti-12">Nat Rev Cancer</span></em>, <strong><span class="cmbx-12">4</span></strong>(3): 177–183.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XFutreal2001"></a><span class="bibsp">   </span></span>Futreal, P.A., Kasprzyk, A., Birney, E., Mullikin, J.C., Wooster, R., and Stratton, M.R. (2001) Cancer and genomics. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">409</span></strong>(6822): 850–852.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XGao2015"></a><span class="bibsp">   </span></span>Gao, B. and Roux, P.P. (2015) Translational control by oncogenic signaling pathways. <em><span class="cmti-12">Biochimica et Biophysica Acta</span></em>, <strong><span class="cmbx-12">1849</span></strong>(7): 753–65.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XGatza2011"></a><span class="bibsp">   </span></span>Gatza, M.L., Kung, H.N., Blackwell, K.L., Dewhirst, M.W., Marks, J.R., and Chi, J.T. (2011) Analysis of tumor environmental response and oncogenic pathway activation identifies distinct basal and luminal features in HER2-related breast tumor subtypes. <em><span class="cmti-12">Breast Cancer Res</span></em>, <strong><span class="cmbx-12">13</span></strong>(3): R62.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XGatza2010"></a><span class="bibsp">   </span></span>Gatza, M.L., Lucas, J.E., Barry, W.T., Kim, J.W., Wang, Q., Crawford, M.D., Datto, M.B., Kelley, M., Mathey-Prevot, B., Potti, A., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2010) A pathway-based classification of human breast cancer. <em><span class="cmti-12">Proc Natl Acad Sci USA</span></em>, <strong><span class="cmbx-12">107</span></strong>(15): 6994–6999.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XGatza2014"></a><span class="bibsp">   </span></span>Gatza, M.L., Silva, G.O., Parker, J.S., Fan, C., and Perou, C.M. (2014) An integrated genomics approach identifies drivers of proliferation in luminal-subtype human breast cancer. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">46</span></strong>(10): 1051–1059.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XGentleman2004"></a><span class="bibsp">   </span></span>Gentleman, R.C., Carey, V.J., Bates, D.M., Bolstad, B., Dettling, M., Dudoit, S., Ellis, B., Gautier, L., Ge, Y., Gentry, J., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2004) Bioconductor: open software development for computational biology and bioinformatics. <em><span class="cmti-12">Genome Biol</span></em>, <strong><span class="cmbx-12">5</span></strong>(10): R80.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XGenz2009"></a><span class="bibsp">   </span></span>Genz, A. and Bretz, F. (2009) Computation of multivariate normal and t probabilities. In <em><span class="cmti-12">Lecture Notes in Statistics</span></em>, volume 195. Springer-Verlag, Heidelberg.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="Xmvtnorm"></a><span class="bibsp">   </span></span>Genz, A., Bretz, F., Miwa, T., Mi, X., Leisch, F., Scheipl, F., and Hothorn, T. (2016) <em><span class="cmti-12">mvtnorm: Multivariate Normal and t Distributions</span></em>. R package version 1.0-5. URL.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XGlaire2017"></a><span class="bibsp">   </span></span>Glaire, M.A., Brown, M., Church, D.N., and Tomlinson, I. (2017) Cancer predisposition syndromes: lessons for truly precision medicine. <em><span class="cmti-12">J Pathol</span></em>, <strong><span class="cmbx-12">241</span></strong>(2): 226–235.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XGlobus"></a><span class="bibsp">   </span></span>Globus (Globus) (2017) Research data management simplified. <a 
href="https://www.globus.org/" class="url" ><span class="cmtt-12">https://www.globus.org/</span></a>. Accessed: 25/03/2017.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XGoodwin2016"></a><span class="bibsp">   </span></span>Goodwin, S., McPherson, J.D., and McCombie, W.R. (2016) Coming of age: ten years of next-generation sequencing technologies. <em><span class="cmti-12">Nat Rev Genet</span></em>, <strong><span class="cmbx-12">17</span></strong>(6): 333–351.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XGrady2000"></a><span class="bibsp">   </span></span>Grady, W.M., Willis, J., Guilford, P.J., Dunbier, A.K., Toro, T.T., Lynch, H., Wiesner, G., Ferguson, K., Eng, C., Park, J.G., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2000) Methylation of the CDH1 promoter as the second genetic hit in hereditary diffuse gastric cancer. <em><span class="cmti-12">Nat</span> <span class="cmti-12">Genet</span></em>, <strong><span class="cmbx-12">26</span></strong>(1): 16–17.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XGraziano2003"></a><span class="bibsp">   </span></span>Graziano, F., Humar, B., and Guilford, P. (2003) The role of the E-cadherin gene (<span class="cmti-12">CDH1</span>) in diffuse gastric cancer susceptibility: from the laboratory to clinical practice. <em><span class="cmti-12">Annals of Oncology</span></em>, <strong><span class="cmbx-12">14</span></strong>(12): 1705–1713.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XGuaragnella2014"></a><span class="bibsp">   </span></span>Guaragnella, N., Palermo, V., Galli, A., Moro, L., Mazzoni, C., and Giannattasio, S. (2014) The expanding role of yeast in cancer research and diagnosis: insights into the function of the oncosuppressors p53 and BRCA1/2. <em><span class="cmti-12">FEMS Yeast Res</span></em>, <strong><span class="cmbx-12">14</span></strong>(1): 2–16.
</p>
                                                                                    

<p class="bibitem" >
<span class="biblabel"> <a 
 id="XGuell2014"></a><span class="bibsp">   </span></span>Güell, O., Sagués, F., and Serrano, M. (2014) Essential plasticity and redundancy of metabolism unveiled by synthetic lethality analysis. <em><span class="cmti-12">PLoS Comput Biol</span></em>, <strong><span class="cmbx-12">10</span></strong>(5): e1003637.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XGuilford1998"></a><span class="bibsp">   </span></span>Guilford, P., Hopkins, J., Harraway, J., McLeod, M., McLeod, N., Harawira, P., Taite, H., Scoular, R., Miller, A., and Reeve, A.E. (1998) E-cadherin germline mutations in familial gastric cancer. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">392</span></strong>(6674): 402–5.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XGuilford2010"></a><span class="bibsp">   </span></span>Guilford, P., Humar, B., and Blair, V. (2010) Hereditary diffuse gastric cancer: translation of <span class="cmti-12">CDH1 </span>germline mutations into clinical practice. <em><span class="cmti-12">Gastric Cancer</span></em>, <strong><span class="cmbx-12">13</span></strong>(1): 1–10.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XHDGC"></a><span class="bibsp">   </span></span>Guilford, P.J., Hopkins, J.B., Grady, W.M., Markowitz, S.D., Willis, J., Lynch, H., Rajput, A., Wiesner, G.L., Lindor, N.M., Burgart, L.J., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (1999) E-cadherin germline mutations define an inherited cancer syndrome dominated by diffuse gastric cancer. <em><span class="cmti-12">Hum Mutat</span></em>, <strong><span class="cmbx-12">14</span></strong>(3): 249–55.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XGuo2016"></a><span class="bibsp">   </span></span>Guo, J., Liu, H., and Zheng, J. (2016) SynLethDB: synthetic lethality database toward discovery of selective and sensitive anticancer drug targets. <em><span class="cmti-12">Nucleic Acids</span> <span class="cmti-12">Res</span></em>, <strong><span class="cmbx-12">44</span></strong>(D1): D1011–1017.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XHajian-Tilaki2013"></a><span class="bibsp">   </span></span>Hajian-Tilaki, K. (2013) Receiver Operating Characteristic (ROC) Curve Analysis for Medical Diagnostic Test Evaluation. <em><span class="cmti-12">Caspian J Intern Med</span></em>, <strong><span class="cmbx-12">4</span></strong>(2): 627–635.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XHall2009"></a><span class="bibsp">   </span></span>Hall, M., Frank, E., Holmes, G., Pfahringer, B., Reutemann, P., and Witten, I.H.</p>
<ol start="2009" style="list-style-type: decimal">
<li>The weka data mining software: an update. <em><span class="cmti-12">SIGKDD Explor Newsl</span></em>, <strong><span class="cmbx-12">11</span></strong>(1): 10–18.
</p>
    <p class="bibitem" >
    
<span class="biblabel"> <a 
     id="XTCGA2012LUSC"></a><span class="bibsp">   </span></span>Hammerman, P.S., Lawrence, M.S., Voet, D., Jing, R., Cibulskis, K., Sivachenko, A., Stojanov, P., McKenna, A., Lander, E.S., Gabriel, S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em>
<ol start="2012" style="list-style-type: decimal">
<li>Comprehensive genomic characterization of squamous cell lung cancers. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">489</span></strong>(7417): 519–525.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XHanahan2000"></a><span class="bibsp">   </span></span>Hanahan, D. and Weinberg, R.A. (2000) The hallmarks of cancer. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">100</span></strong>(1): 57–70.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XHanahan2011"></a><span class="bibsp">   </span></span>Hanahan, D. and Weinberg, R.A. (2011) Hallmarks of cancer: the next generation. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">144</span></strong>(5): 646–674.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XCIX2013"></a><span class="bibsp">   </span></span>Hanna, S.</li>
<li>Cancer incidence in new zealand (2003-2007). In D. Forman, D. Bray F Brewster, C. Gombe Mbalawa, B. Kohler, M. Piñeros, E. Steliarova-Foucher, R. Swaminathan, and J. Ferlay (editors), <em><span class="cmti-12">Cancer Incidence</span> <span class="cmti-12">in Five Continents</span></em>, volume X, 902–907. International Agency for Research on Cancer, Lyon, France. Electronic version <a 
href="http://ci5.iarc.fr" class="url" ><span class="cmtt-12">http://ci5.iarc.fr</span></a> Accessed 22/03/2017.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XHansford2015"></a><span class="bibsp">   </span></span>Hansford, S., Kaurah, P., Li-Chang, H., Woo, M., Senz, J., Pinheiro, H., Schrader, K.A., Schaeffer, D.F., Shumansky, K., Zogopoulos, G., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></li>
<li>Hereditary Diffuse Gastric Cancer Syndrome: CDH1 Mutations and Beyond. <em><span class="cmti-12">JAMA Oncol</span></em>, <strong><span class="cmbx-12">1</span></strong>(1): 23–32.
</p>

</li>
</ol></li>
</ol>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XHeiskanen2012"></a><span class="bibsp">   </span></span>Heiskanen, M.A. and Aittokallio, T. (2012) Mining high-throughput screens for cancer drug targets-lessons from yeast chemical-genomic profiling and synthetic lethality. <em><span class="cmti-12">Wiley Interdisciplinary Reviews: Data Mining and Knowledge Discovery</span></em>, <strong><span class="cmbx-12">2</span></strong>(3): 263–272.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XHell1976"></a><span class="bibsp">   </span></span>Hell, P. (1976) Graphs with given neighbourhoods i. problémes combinatorics at theorie des graphes. <em><span class="cmti-12">Proc Coil Int CNRS, Orsay</span></em>, <strong><span class="cmbx-12">260</span></strong>: 219–223.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XHiggins2007"></a><span class="bibsp">   </span></span>Higgins, M.E., Claremont, M., Major, J.E., Sander, C., and Lash, A.E. (2007) CancerGenes: a gene selection resource for cancer genome projects. <em><span class="cmti-12">Nucleic Acids</span> <span class="cmti-12">Res</span></em>, <strong><span class="cmbx-12">35</span></strong>(Database issue): D721–726.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XHillenmeyer2008"></a><span class="bibsp">   </span></span>Hillenmeyer, M.E. (2008) The chemical genomic portrait of yeast: uncovering a phenotype for all genes. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">320</span></strong>: 362–365.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTCGA2014PAN"></a><span class="bibsp">   </span></span>Hoadley, K.A., Yau, C., Wolf, D.M., Cherniack, A.D., Tamborero, D., Ng, S., Leiserson, M.D., Niu, B., McLellan, M.D., Uzunangelov, V., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Multiplatform analysis of 12 cancer types reveals molecular classification within and across tissues of origin. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">158</span></strong>(4): 929–944.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XHoehndorf2013"></a><span class="bibsp">   </span></span>Hoehndorf, R., Hardy, N.W., Osumi-Sutherland, D., Tweedie, S., Schofield, P.N., and Gkoutos, G.V. (2013) Systematic analysis of experimental phenotype data reveals gene functions. <em><span class="cmti-12">PLoS ONE</span></em>, <strong><span class="cmbx-12">8</span></strong>(4): e60847.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XHolm1979"></a><span class="bibsp">   </span></span>Holm, S. (1979) A simple sequentially rejective multiple test procedure. <em><span class="cmti-12">Scandinavian Journal of Statistics</span></em>, <strong><span class="cmbx-12">6</span></strong>(2): 65–70.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XHopkins2008"></a><span class="bibsp">   </span></span>Hopkins, A.L. (2008) Network pharmacology: the next paradigm in drug discovery. <em><span class="cmti-12">Nat Chem Biol</span></em>, <strong><span class="cmbx-12">4</span></strong>(11): 682–690.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XHu2006"></a><span class="bibsp">   </span></span>Hu, Z., Fan, C., Oh, D.S., Marron, J.S., He, X., Qaqish, B.F., Livasy, C., Carey, L.A., Reynolds, E., Dressler, L., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2006) The molecular portraits of breast tumors are conserved across microarray platforms. <em><span class="cmti-12">BMC Genomics</span></em>, <strong><span class="cmbx-12">7</span></strong>: 96.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XHuang2003"></a><span class="bibsp">   </span></span>Huang, E., Cheng, S., Dressman, H., Pittman, J., Tsou, M., Horng, C., Bild, A., Iversen, E., Liao, M., Chen, C., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2003) Gene expression predictors of breast cancer outcomes. <em><span class="cmti-12">Lancet</span></em>, <strong><span class="cmbx-12">361</span></strong>: 1590–1596.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XHutchison2016"></a><span class="bibsp">   </span></span>Hutchison, C.A., Chuang, R.Y., Noskov, V.N., Assad-Garcia, N., Deerinck, T.J., Ellisman, M.H., Gill, J., Kannan, K., Karas, B.J., Ma, L., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2016) Design and synthesis of a minimal bacterial genome. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">351</span></strong>(6280): aad6253.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XImoto2004"></a><span class="bibsp">   </span></span>Imoto, S., Higuchi, T., Goto, T., Tashiro, K., Kuhara, S., and Miyano, S. (2004) Combining microarrays and biological knowledge for estimating gene networks via bayesian networks. <em><span class="cmti-12">J Bioinform Comput Biol</span></em>, <strong><span class="cmbx-12">2</span></strong>(1): 77–98.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XHapMap2003"></a><span class="bibsp">   </span></span>International HapMap 3 Consortium (HapMap) (2003) The International HapMap Project. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">426</span></strong>(6968): 789–796.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XJansen2003"></a><span class="bibsp">   </span></span>Jansen, R., Yu, H., Greenbaum, D., Kluger, Y., Krogan, N.J., Chung, S., Emili, A., Snyder, M., Greenblatt, J.F., and Gerstein, M. (2003) A Bayesian networks approach</p>
for predicting protein-protein interactions from genomic data. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">302</span></strong>(5644): 449–453.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XJeanes2008"></a><span class="bibsp">   </span></span>Jeanes, A., Gottardi, C.J., and Yap, A.S. (2008) Cadherins and cancer: how does cadherin dysfunction promote tumor progression? <em><span class="cmti-12">Oncogene</span></em>, <strong><span class="cmbx-12">27</span></strong>(55): 6920–6929.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XJerby2014"></a><span class="bibsp">   </span></span>Jerby-Arnon, L., Pfetzer, N., Waldman, Y., McGarry, L., James, D., Shanks, E., Seashore-Ludlow, B., Weinstock, A., Geiger, T., Clemons, P., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Predicting cancer-specific vulnerability via data-driven detection of synthetic lethality. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">158</span></strong>(5): 1199–1209.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XJoachims1999"></a><span class="bibsp">   </span></span>Joachims, T. (1999) Making large-scale support vector machine learning practical. In S. Bernhard, lkopf, J.C.B. Christopher, and J.S. Alexander (editors), <em><span class="cmti-12">Advances</span> <span class="cmti-12">in kernel methods</span></em>, 169–184. MIT Press.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XKaelin2005"></a><span class="bibsp">   </span></span>Kaelin, Jr, W. (2005) The concept of synthetic lethality in the context of anticancer therapy. <em><span class="cmti-12">Nat Rev Cancer</span></em>, <strong><span class="cmbx-12">5</span></strong>(9): 689–98.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XKaelin2009"></a><span class="bibsp">   </span></span>Kaelin, Jr, W. (2009) Synthetic lethality: a framework for the development of wiser cancer therapeutics. <em><span class="cmti-12">Genome Med</span></em>, <strong><span class="cmbx-12">1</span></strong>: 99.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XKamada1989"></a><span class="bibsp">   </span></span>Kamada, T. and Kawai, S. (1989) An algorithm for drawing general undirected graphs. <em><span class="cmti-12">Information Processing Letters</span></em>, <strong><span class="cmbx-12">31</span></strong>(1): 7–15.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XFANTOM2001"></a><span class="bibsp">   </span></span>Kawai, J., Shinagawa, A., Shibata, K., Yoshino, M., Itoh, M., Ishii, Y., Arakawa,</p>
T., Hara, A., Fukunishi, Y., Konno, H., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2001) Functional annotation of a full-length mouse cDNA collection. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">409</span></strong>(6821): 685–690.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XKelley2005"></a><span class="bibsp">   </span></span>Kelley, R. and Ideker, T. (2005) Systematic interpretation of genetic interactions using protein networks. <em><span class="cmti-12">Nat Biotech</span></em>, <strong><span class="cmbx-12">23</span></strong>(5): 561–566.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XKelly2013"></a><span class="bibsp">   </span></span>Kelly, S.T. (2013) <em><span class="cmti-12">Statistical Predictions of Synthetic Lethal Interactions in Cancer</span></em>. Dissertation, University of Otago.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XKeshavaPrasad2009"></a><span class="bibsp">   </span></span>Keshava Prasad, T.S., Goel, R., Kandasamy, K., Keerthikumar, S., Kumar, S., Mathivanan, S., Telikicherla, D., Raju, R., Shafreen, B., Venugopal, A., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2009) Human Protein Reference Database–2009 update. <em><span class="cmti-12">Nucleic Acids Res</span></em>, <strong><span class="cmbx-12">37</span></strong>(Database issue): D767–772.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XKim2011"></a><span class="bibsp">   </span></span>Kim, N.G., Koh, E., Chen, X., and Gumbiner, B.M. (2011) E-cadherin mediates contact inhibition of proliferation through Hippo signaling-pathway components. <em><span class="cmti-12">Proc Natl Acad Sci USA</span></em>, <strong><span class="cmbx-12">108</span></strong>(29): 11930–11935.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTCGA2012"></a><span class="bibsp">   </span></span>Koboldt, D.C., Fulton, R.S., McLellan, M.D., Schmidt, H., Kalicki-Veizer, J., McMichael, J.F., Fulton, L.L., Dooling, D.J., Ding, L., Mardis, E.R., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2012) Comprehensive molecular portraits of human breast tumours. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">490</span></strong>(7418): 61–70.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XKockel1997"></a><span class="bibsp">   </span></span>Kockel, L., Zeitlinger, J., Staszewski, L.M., Mlodzik, M., and Bohmann, D. (1997) Jun in drosophila development: redundant and nonredundant functions and regulation by two mapk signal transduction pathways. <em><span class="cmti-12">Genes &amp; Development</span></em>, <strong><span class="cmbx-12">11</span></strong>(13): 1748–1758.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XKozlov2015"></a><span class="bibsp">   </span></span>Kozlov, K.N., Gursky, V.V., Kulakovskiy, I.V., and Samsonova, M.G. (2015) Sequence-based model of gap gene regulation network. <em><span class="cmti-12">BMC Genomics</span></em>, <strong><span class="cmbx-12">15</span></strong>(Suppl 12): S6.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XKranthi2013"></a><span class="bibsp">   </span></span>Kranthi, S., Rao, S., and Manimaran, P. (2013) Identification of synthetic lethal pairs in biological systems through network information centrality. <em><span class="cmti-12">Mol BioSyst</span></em>, <strong><span class="cmbx-12">9</span></strong>(8): 2163–2167.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XKroepil2012"></a><span class="bibsp">   </span></span>Kroepil, F., Fluegen, G., Totikov, Z., Baldus, S.E., Vay, C., Schauer, M., Topp, S.A., Esch, J.S., Knoefel, W.T., and Stoecklein, N.H. (2012) Down-regulation of CDH1 is associated with expression of SNAI1 in colorectal adenomas. <em><span class="cmti-12">PLoS ONE</span></em>, <strong><span class="cmbx-12">7</span></strong>(9): e46665.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLander2011"></a><span class="bibsp">   </span></span>Lander, E.S. (2011) Initial impact of the sequencing of the human genome. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">470</span></strong>(7333): 187–197.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLander2001"></a><span class="bibsp">   </span></span>Lander, E.S., Linton, L.M., Birren, B., Nusbaum, C., Zody, M.C., Baldwin, J., Devon, K., Dewar, K., Doyle, M., FitzHugh, W., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2001) Initial sequencing and analysis of the human genome. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">409</span></strong>(6822): 860–921.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="Xbowtie"></a><span class="bibsp">   </span></span>Langmead, B., Trapnell, C., Pop, M., and Salzberg, S.L. (2009) Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. <em><span class="cmti-12">Genome</span> <span class="cmti-12">Biol</span></em>, <strong><span class="cmbx-12">10</span></strong>(3): R25.
</p>
                                                                                    

<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLatora2001"></a><span class="bibsp">   </span></span>Latora, V. and Marchiori, M. (2001) Efficient behavior of small-world networks. <em><span class="cmti-12">Phys Rev Lett</span></em>, <strong><span class="cmbx-12">87</span></strong>: 198701.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLaufer2013"></a><span class="bibsp">   </span></span>Laufer, C., Fischer, B., Billmann, M., Huber, W., and Boutros, M. (2013) Mapping genetic interactions in human cancer cells with RNAi and multiparametric phenotyping. <em><span class="cmti-12">Nat Methods</span></em>, <strong><span class="cmbx-12">10</span></strong>(5): 427–31.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLaw2014"></a><span class="bibsp">   </span></span>Law, C.W., Chen, Y., Shi, W., and Smyth, G.K. (2014) voom: precision weights unlock linear model analysis tools for RNA-seq read counts. <em><span class="cmti-12">Genome Biol</span></em>, <strong><span class="cmbx-12">15</span></strong>(2): R29.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLeMeur2008"></a><span class="bibsp">   </span></span>Le Meur, N. and Gentleman, R. (2008) Modeling synthetic lethality. <em><span class="cmti-12">Genome Biol</span></em>, <strong><span class="cmbx-12">9</span></strong>(9): R135.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLeMeur2014"></a><span class="bibsp">   </span></span>Le Meur, N., Jiang, Z., Liu, T., Mar, J., and Gentleman, R.C. (2014) Slgi: Synthetic lethal genetic interaction. r package version 1.26.0.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLee2010a"></a><span class="bibsp">   </span></span>Lee, A.Y., Perreault, R., Harel, S., Boulier, E.L., Suderman, M., Hallett, M., and Jenna, S. (2010a) Searching for signaling balance through the identification of genetic interactors of the rab guanine-nucleotide dissociation inhibitor gdi-1. <em><span class="cmti-12">PLoS ONE</span></em>, <strong><span class="cmbx-12">5</span></strong>(5): e10624.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XLee2010b"></a><span class="bibsp">   </span></span>Lee, I., Lehner, B., Vavouri, T., Shin, J., Fraser, A.G., and Marcotte, E.M. (2010b) Predicting genetic modifier loci using functional gene networks. <em><span class="cmti-12">Genome Research</span></em>, <strong><span class="cmbx-12">20</span></strong>(8): 1143–1153.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLee2009"></a><span class="bibsp">   </span></span>Lee, I. and Marcotte, E.M. (2009) Effects of functional bias on supervised learning of a gene network model. <em><span class="cmti-12">Methods Mol Biol</span></em>, <strong><span class="cmbx-12">541</span></strong>: 463–75.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLee2012"></a><span class="bibsp">   </span></span>Lee, M.J., Ye, A.S., Gardino, A.K., Heijink, A.M., Sorger, P.K., MacBeath, G., and Yaffe, M.B. (2012) Sequential application of anticancer drugs enhances cell death by rewiring apoptotic signaling networks. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">149</span></strong>(4): 780–94.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLehner2006"></a><span class="bibsp">   </span></span>Lehner, B., Crombie, C., Tischler, J., Fortunato, A., and Fraser, A.G. (2006) Systematic mapping of genetic interactions in caenorhabditis elegans identifies common modifiers of diverse signaling pathways. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">38</span></strong>(8): 896–903.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XRSEM"></a><span class="bibsp">   </span></span>Li, B., Ruotti, V., Stewart, R.M., Thomson, J.A., and Dewey, C.N. (2010) RNA-Seq gene expression estimation with read mapping uncertainty. <em><span class="cmti-12">Bioinformatics</span></em>, <strong><span class="cmbx-12">26</span></strong>(4): 493–500.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLi2014"></a><span class="bibsp">   </span></span>Li, X.J., Mishra, S.K., Wu, M., Zhang, F., and Zheng, J. (2014) Syn-lethality: An integrative knowledge base of synthetic lethality towards discovery of selective anticancer therapies. <em><span class="cmti-12">Biomed Res Int</span></em>, <strong><span class="cmbx-12">2014</span></strong>: 196034.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTCGA2016PRC"></a><span class="bibsp">   </span></span>Linehan, W.M., Spellman, P.T., Ricketts, C.J., Creighton, C.J., Fei, S.S., Davis, C., Wheeler, D.A., Murray, B.A., Schmidt, L., Vocke, C.D., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2016) Comprehensive Molecular Characterization of Papillary Renal-Cell Carcinoma. <em><span class="cmti-12">N Engl J Med</span></em>, <strong><span class="cmbx-12">374</span></strong>(2): 135–145.
</p>
                                                                                    

<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLokody2014"></a><span class="bibsp">   </span></span>Lokody, I. (2014) Computational modelling: A computational crystal ball. <em><span class="cmti-12">Nature</span> <span class="cmti-12">Reviews Cancer</span></em>, <strong><span class="cmbx-12">14</span></strong>(10): 649–649.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLord2014"></a><span class="bibsp">   </span></span>Lord, C.J., Tutt, A.N., and Ashworth, A. (2015) Synthetic lethality and cancer therapy: lessons learned from the development of PARP inhibitors. <em><span class="cmti-12">Annu Rev Med</span></em>, <strong><span class="cmbx-12">66</span></strong>: 455–470.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLu2013"></a><span class="bibsp">   </span></span>Lu, X., Kensche, P.R., Huynen, M.A., and Notebaart, R.A. (2013) Genome evolution predicts genetic interactions in protein complexes and reveals cancer drug targets. <em><span class="cmti-12">Nat Commun</span></em>, <strong><span class="cmbx-12">4</span></strong>: 2124.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLu2015"></a><span class="bibsp">   </span></span>Lu, X., Megchelenbrink, W., Notebaart, R.A., and Huynen, M.A. (2015) Predicting human genetic interactions from cancer genome evolution. <em><span class="cmti-12">PLoS One</span></em>, <strong><span class="cmbx-12">10</span></strong>(5): e0125795.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLum2004"></a><span class="bibsp">   </span></span>Lum, P.Y., Armour, C.D., Stepaniants, S.B., Cavet, G., Wolf, M.K., Butler, J.S., Hinshaw, J.C., Garnier, P., Prestwich, G.D., Leonardson, A., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2004) Discovering modes of action for therapeutic compounds using a genome-wide screen of yeast heterozygotes. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">116</span></strong>(1): 121–137.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XLuo2009"></a><span class="bibsp">   </span></span>Luo, J., Solimini, N.L., and Elledge, S.J. (2009) Principles of Cancer Therapy: Oncogene and Non-oncogene Addiction. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">136</span></strong>(5): 823–837.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XMachado2001"></a><span class="bibsp">   </span></span>Machado, J., Olivera, C., Carvalh, R., Soares, P., Berx, G., Caldas, C., Sercuca, R., Carneiro, F., and Sorbrinho-Simoes, M. (2001) E-cadherin gene (<span class="cmti-12">CDH1</span>) promoter</p>
methylation as the second hit in sporadic diffuse gastric carcinoma. <em><span class="cmti-12">Oncogene</span></em>, <strong><span class="cmbx-12">20</span></strong>: 1525–1528.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XMarkowetz2017"></a><span class="bibsp">   </span></span>Markowetz, F. (2017) All biology is computational biology. <em><span class="cmti-12">PLoS Biol</span></em>, <strong><span class="cmbx-12">15</span></strong>(3): e2002050.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XMasciari2007"></a><span class="bibsp">   </span></span>Masciari, S., Larsson, N., Senz, J., Boyd, N., Kaurah, P., Kandel, M.J., Harris, L.N., Pinheiro, H.C., Troussard, A., Miron, P., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2007) Germline E-cadherin mutations in familial lobular breast cancer. <em><span class="cmti-12">J Med Genet</span></em>, <strong><span class="cmbx-12">44</span></strong>(11): 726–31.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XMattison2009"></a><span class="bibsp">   </span></span>Mattison, J., van der Weyden, L., Hubbard, T., and Adams, D.J. (2009) Cancer gene discovery in mouse and man. <em><span class="cmti-12">Biochim Biophys Acta</span></em>, <strong><span class="cmbx-12">1796</span></strong>(2): 140–161.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XMcLachlan2016"></a><span class="bibsp">   </span></span>McLachlan, J., George, A., and Banerjee, S. (2016) The current status of parp inhibitors in ovarian cancer. <em><span class="cmti-12">Tumori</span></em>, <strong><span class="cmbx-12">102</span></strong>(5): 433–440.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTCGA2008GBM"></a><span class="bibsp">   </span></span>McLendon, R., Friedman, A., Bigner, D., Van Meir, E.G., Brat, D.J., Mastrogianakis, G.M., Olson, J.J., Mikkelsen, T., Lehman, N., Aldape, K., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2008) Comprehensive genomic characterization defines human glioblastoma genes and core pathways. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">455</span></strong>(7216): 1061–1068.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XMiles2001"></a><span class="bibsp">   </span></span>Miles, D.W. (2001) Update on HER-2 as a target for cancer therapy: herceptin in the clinical setting. <em><span class="cmti-12">Breast Cancer Res</span></em>, <strong><span class="cmbx-12">3</span></strong>(6): 380–384.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XTCGA2012CRC"></a><span class="bibsp">   </span></span>Muzny, D.M., Bainbridge, M.N., Chang, K., Dinh, H.H., Drummond, J.A.,</p>
Fowler, G., Kovar, C.L., Lewis, L.R., Morgan, M.B., Newsham, I.F., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2012) Comprehensive molecular characterization of human colon and rectal cancer. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">487</span></strong>(7407): 330–337.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XNagalla2013"></a><span class="bibsp">   </span></span>Nagalla, S., Chou, J.W., Willingham, M.C., Ruiz, J., Vaughn, J.P., Dubey, P., Lash, T.L., Hamilton-Dutoit, S.J., Bergh, J., Sotiriou, C., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2013) Interactions between immunity, proliferation and molecular subtype in breast cancer prognosis. <em><span class="cmti-12">Genome Biol</span></em>, <strong><span class="cmbx-12">14</span></strong>(4): R34.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="Xmatrixcalc"></a><span class="bibsp">   </span></span>Novomestky, F. (2012) <em><span class="cmti-12">matrixcalc: Collection of functions for matrix calculations</span></em>. R package version 1.0-3.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XNowak1997"></a><span class="bibsp">   </span></span>Nowak, M.A., Boerlijst, M.C., Cooke, J., and Smith, J.M. (1997) Evolution of genetic redundancy. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">388</span></strong>(6638): 167–171.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XOliveira2009"></a><span class="bibsp">   </span></span>Oliveira, C., Senz, J., Kaurah, P., Pinheiro, H., Sanges, R., Haegert, A., Corso, G., Schouten, J., Fitzgerald, R., Vogelsang, H., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2009) Germline <span class="cmti-12">CDH1 </span>deletions in hereditary diffuse gastric cancer families. <em><span class="cmti-12">Human Molecular Genetics</span></em>, <strong><span class="cmbx-12">18</span></strong>(9): 1545–1555.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XOliveira2013"></a><span class="bibsp">   </span></span>Oliveira, C., Seruca, R., Hoogerbrugge, N., Ligtenberg, M., and Carneiro, F. (2013) Clinical utility gene card for: Hereditary diffuse gastric cancer (HDGC). <em><span class="cmti-12">Eur J Hum</span> <span class="cmti-12">Genet</span></em>, <strong><span class="cmbx-12">21</span></strong>(8).
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XPandey2010"></a><span class="bibsp">   </span></span>Pandey, G., Zhang, B., Chang, A.N., Myers, C.L., Zhu, J., Kumar, V., and Schadt, E.E. (2010) An integrative multi-network and multi-classifier approach to predict genetic interactions. <em><span class="cmti-12">PLoS Comput Biol</span></em>, <strong><span class="cmbx-12">6</span></strong>(9).</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XParker2009"></a><span class="bibsp">   </span></span>Parker, J., Mullins, M., Cheung, M., Leung, S., Voduc, D., Vickery, T., Davies, S., Fauron, C., He, X., Hu, Z., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2009) Supervised risk predictor of breast cancer based on intrinsic subtypes. <em><span class="cmti-12">Journal of Clinical Oncology</span></em>, <strong><span class="cmbx-12">27</span></strong>(8): 1160–1167.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XMETABRIC2016"></a><span class="bibsp">   </span></span>Pereira, B., Chin, S.F., Rueda, O.M., Vollan, H.K., Provenzano, E., Bardwell, H.A., Pugh, M., Jones, L., Russell, R., Sammut, S.J., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2016) Erratum: The somatic mutation profiles of 2,433 breast cancers refine their genomic and transcriptomic landscapes. <em><span class="cmti-12">Nat Commun</span></em>, <strong><span class="cmbx-12">7</span></strong>: 11908.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XPerou2000"></a><span class="bibsp">   </span></span>Perou, C.M., Sørlie, T., Eisen, M.B., van de Rijn, M., Jeffrey, S.S., Rees, C.A., Pollack, J.R., Ross, D.T., Johnsen, H., Akslen, L.A., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2000) Molecular portraits of human breast tumours. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">406</span></strong>(6797): 747–752.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XR_core"></a><span class="bibsp">   </span></span>R Core Team (2016) <em><span class="cmti-12">R: A Language and Environment for Statistical Computing</span></em>. R Foundation for Statistical Computing, Vienna, Austria. R version 3.3.2.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="Xlimma"></a><span class="bibsp">   </span></span>Ritchie, M.E., Phipson, B., Wu, D., Hu, Y., Law, C.W., Shi, W., and Smyth, G.K. (2015) limma powers differential expression analyses for RNA-sequencing and microarray studies. <em><span class="cmti-12">Nucleic Acids Research</span></em>, <strong><span class="cmbx-12">43</span></strong>(7): e47.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XRoguev2008"></a><span class="bibsp">   </span></span>Roguev, A., Bandyopadhyay, S., Zofall, M., Zhang, K., Fischer, T., Collins, S.R., Qu, H., Shales, M., Park, H.O., Hayles, J., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2008) Conservation and rewiring of functional modules revealed by an epistasis map in fission yeast. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">322</span></strong>(5900): 405–10.
</p>
                                                                                    

<p class="bibitem" >
<span class="biblabel"> <a 
 id="XRoychowdhury2016"></a><span class="bibsp">   </span></span>Roychowdhury, S. and Chinnaiyan, A.M. (2016) Translating cancer genomes and transcriptomes for precision oncology. <em><span class="cmti-12">CA Cancer J Clin</span></em>, <strong><span class="cmbx-12">66</span></strong>(1): 75–88.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XRung2013"></a><span class="bibsp">   </span></span>Rung, J. and Brazma, A. (2013) Reuse of public genome-wide gene expression data. <em><span class="cmti-12">Nat Rev Genet</span></em>, <strong><span class="cmbx-12">14</span></strong>(2): 89–99.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XRyan2014"></a><span class="bibsp">   </span></span>Ryan, C., Lord, C., and Ashworth, A. (2014) Daisy: Picking synthetic lethals from cancer genomes. <em><span class="cmti-12">Cancer Cell</span></em>, <strong><span class="cmbx-12">26</span></strong>(3): 306–308.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XSchena1996"></a><span class="bibsp">   </span></span>Schena, M. (1996) Genome analysis with gene expression microarrays. <em><span class="cmti-12">Bioessays</span></em>, <strong><span class="cmbx-12">18</span></strong>(5): 427–431.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XScheuer2002"></a><span class="bibsp">   </span></span>Scheuer, L., Kauff, N., Robson, M., Kelly, B., Barakat, R., Satagopan, J., Ellis, N., Hensley, M., Boyd, J., Borgen, P., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2002) Outcome of preventive surgery and screening for breast and ovarian cancer in BRCA mutation carriers. <em><span class="cmti-12">J Clin Oncol</span></em>, <strong><span class="cmbx-12">20</span></strong>(5): 1260–1268.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XSemb1998"></a><span class="bibsp">   </span></span>Semb, H. and Christofori, G. (1998) The tumor-suppressor function of E-cadherin. <em><span class="cmti-12">Am J Hum Genet</span></em>, <strong><span class="cmbx-12">63</span></strong>(6): 1588–93.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XROCR"></a><span class="bibsp">   </span></span>Sing, T., Sander, O., Beerenwinkel, N., and Lengauer, T. (2005) Rocr: visualizing classifier performance in r. <em><span class="cmti-12">Bioinformatics</span></em>, <strong><span class="cmbx-12">21</span></strong>(20): 7881.
</p>
                                                                                    

<p class="bibitem" >
<span class="biblabel"> <a 
 id="Xslurm"></a><span class="bibsp">   </span></span>Slurm development team (Slurm) (2017) Slurm workload manager. <a 
href="https://slurm.schedmd.com/" class="url" ><span class="cmtt-12">https://slurm.schedmd.com/</span></a>. Accessed: 25/03/2017.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XSorlie2001"></a><span class="bibsp">   </span></span>Sørlie, T., Perou, C.M., Tibshirani, R., Aas, T., Geisler, S., Johnsen, H., Hastie, T., Eisen, M.B., van de Rijn, M., Jeffrey, S.S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2001) Gene expression patterns of breast carcinomas distinguish tumor subclasses with clinical implications. <em><span class="cmti-12">Proc Natl</span> <span class="cmti-12">Acad Sci USA</span></em>, <strong><span class="cmbx-12">98</span></strong>(19): 10869–10874.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XSrihari2015"></a><span class="bibsp">   </span></span>Srihari, S., Singla, J., Wong, L., and Ragan, M.A. (2015) Inferring synthetic lethal interactions from mutual exclusivity of genetic events in cancer. <em><span class="cmti-12">Biology Direct</span></em>, <strong><span class="cmbx-12">10</span></strong>(1): 57.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XStajich2006"></a><span class="bibsp">   </span></span>Stajich, J.E. and Lapp, H. (2006) Open source tools and toolkits for bioinformatics: significance, and where are we? <em><span class="cmti-12">Brief Bioinformatics</span></em>, <strong><span class="cmbx-12">7</span></strong>(3): 287–296.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XStratton2009"></a><span class="bibsp">   </span></span>Stratton, M.R., Campbell, P.J., and Futreal, P.A. (2009) The cancer genome. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">458</span></strong>(7239): 719–724.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XStrom2012"></a><span class="bibsp">   </span></span>Ström, C. and Helleday, T. (2012) Strategies for the use of poly(adenosine diphosphate ribose) polymerase (parp) inhibitors in cancer therapy. <em><span class="cmti-12">Biomolecules</span></em>, <strong><span class="cmbx-12">2</span></strong>(4): 635–649.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XTarazona2011"></a><span class="bibsp">   </span></span>Tarazona, S., Garcia-Alcalde, F., Dopazo, J., Ferrer, A., and Conesa, A. (2011) Differential expression in RNA-seq: a matter of depth. <em><span class="cmti-12">Genome Res</span></em>, <strong><span class="cmbx-12">21</span></strong>(12): 2213–2223.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTelford2015"></a><span class="bibsp">   </span></span>Telford, B.J., Chen, A., Beetham, H., Frick, J., Brew, T.P., Gould, C.M., Single, A., Godwin, T., Simpson, K.J., and Guilford, P. (2015) Synthetic lethal screens identify vulnerabilities in gpcr signalling and cytoskeletal organization in E-cadherin-deficient cells. <em><span class="cmti-12">Mol Cancer Ther</span></em>, <strong><span class="cmbx-12">14</span></strong>(5): 1213–1223.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="X1000Genomes2010"></a><span class="bibsp">   </span></span>The 1000 Genomes Project Consortium (1000 Genomes) (2010) A map of human genome variation from population-scale sequencing. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">467</span></strong>(7319): 1061–1073.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTCGA2017web"></a><span class="bibsp">   </span></span>The Cancer Genome Atlas Research Network (TCGA) (2017) The Cancer Genome Atlas Project. <a 
href="https://cancergenome.nih.gov/" class="url" ><span class="cmtt-12">https://cancergenome.nih.gov/</span></a>. Accessed: 26/03/2017.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XCOSMICdb"></a><span class="bibsp">   </span></span>The Catalogue Of Somatic Mutations In Cancer (COSMIC) (2016) Cosmic: The catalogue of somatic mutations in cancer. <a 
href="http://cancer.sanger.ac.uk/cosmic" class="url" ><span class="cmtt-12">http://cancer.sanger.ac.uk/cosmic</span></a>. Release 79 (23/08/2016), Accessed: 05/02/2017.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XCRAN"></a><span class="bibsp">   </span></span>The Comprehensive R Archive Network (CRAN) (2017) Cran. <a 
href="https://cran.r-project.org/" class="url" ><span class="cmtt-12">https://cran.r-_project.org/</span></a>. Accessed: 24/03/2017.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XENCODE2004"></a><span class="bibsp">   </span></span>The ENCODE Project Consortium (ENCODE) (2004) The ENCODE (ENCyclopedia Of DNA Elements) Project. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">306</span></strong>(5696): 636–640.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XNCI2015"></a><span class="bibsp">   </span></span>The National Cancer Institute (NCI) (2015) The genetics of cancer. <a 
href="https://www.cancer.gov/about-cancer/causes-prevention/genetics" class="url" ><span class="cmtt-12">https://www.cancer.gov/about-<em>cancer/causes-</em>prevention/genetics</span></a>. Published: 22/04/2015, Accessed: 22/03/2017.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XNeSI"></a><span class="bibsp">   </span></span>The New Zealand eScience Infrastructure (NeSI) (2017) NeSI. <a 
href="https://www.nesi.org.nz/" class="url" ><span class="cmtt-12">https://www.nesi.org.nz/</span></a>. Accessed: 25/03/2017.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="Xsnow"></a><span class="bibsp">   </span></span>Tierney, L., Rossini, A.J., Li, N., and Sevcikova, H. (2015) <em><span class="cmti-12">snow: Simple Network</span> <span class="cmti-12">of Workstations</span></em>. R package version 0.4-2.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTiong2014"></a><span class="bibsp">   </span></span>Tiong, K.L., Chang, K.C., Yeh, K.T., Liu, T.Y., Wu, J.H., Hsieh, P.H., Lin, S.H., Lai, W.Y., Hsu, Y.C., Chen, J.Y., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Csnk1e/ctnnb1 are synthetic lethal to tp53 in colorectal cancer and are markers for prognosis. <em><span class="cmti-12">Neoplasia</span></em>, <strong><span class="cmbx-12">16</span></strong>(5): 441–50.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTischler2008"></a><span class="bibsp">   </span></span>Tischler, J., Lehner, B., and Fraser, A.G. (2008) Evolutionary plasticity of genetic interaction networks. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">40</span></strong>(4): 390–391.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTomasetti2015"></a><span class="bibsp">   </span></span>Tomasetti, C. and Vogelstein, B. (2015) Cancer etiology. Variation in cancer risk among tissues can be explained by the number of stem cell divisions. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">347</span></strong>(6217): 78–81.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTong2001"></a><span class="bibsp">   </span></span>Tong, A.H., Evangelista, M., Parsons, A.B., Xu, H., Bader, G.D., Page, N., Robinson, M., Raghibizadeh, S., Hogue, C.W., Bussey, H., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2001) Systematic genetic analysis with ordered arrays of yeast deletion mutants. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">294</span></strong>(5550): 2364–8.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XTong2004"></a><span class="bibsp">   </span></span>Tong, A.H., Lesage, G., Bader, G.D., Ding, H., Xu, H., Xin, X., Young, J., Berriz, G.F., Brost, R.L., Chang, M., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2004) Global mapping of the yeast genetic interaction network. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">303</span></strong>(5659): 808–13.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTran2012"></a><span class="bibsp">   </span></span>Tran, B., Dancey, J.E., Kamel-Reid, S., McPherson, J.D., Bedard, P.L., Brown, A.M., Zhang, T., Shaw, P., Onetto, N., Stein, L., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2012) Cancer genomics: technology, discovery, and translation. <em><span class="cmti-12">J Clin Oncol</span></em>, <strong><span class="cmbx-12">30</span></strong>(6): 647–660.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTravers1969"></a><span class="bibsp">   </span></span>Travers, J. and Milgram, S. (1969) An experimental study of the small world problem. <em><span class="cmti-12">Sociometry</span></em>, <strong><span class="cmbx-12">32</span></strong>(4): 425–443.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTunggal2005"></a><span class="bibsp">   </span></span>Tunggal, J.A., Helfrich, I., Schmitz, A., Schwarz, H., Gunzel, D., Fromm, M., Kemler, R., Krieg, T., and Niessen, C.M. (2005) E-cadherin is essential for in vivo epidermal barrier function by regulating tight junctions. <em><span class="cmti-12">EMBO J</span></em>, <strong><span class="cmbx-12">24</span></strong>(6): 1146–1156.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTutt2010"></a><span class="bibsp">   </span></span>Tutt, A., Robson, M., Garber, J.E., Domchek, S.M., Audeh, M.W., Weitzel, J.N., Friedlander, M., Arun, B., Loman, N., Schmutzler, R.K., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2010) Oral poly(adp-ribose) polymerase inhibitor olaparib in patients with <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2</span> mutations and advanced breast cancer: a proof-of-concept trial. <em><span class="cmti-12">Lancet</span></em>, <strong><span class="cmbx-12">376</span></strong>(9737): 235–44.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XUCSC2012"></a><span class="bibsp">   </span></span>University of California, Santa Cruz (UCSC) (2012) Ucsc cancer browser. Accessed 29/03/2012.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XvanderMeer2014"></a><span class="bibsp">   </span></span>van der Meer, R., Song, H.Y., Park, S.H., Abdulkadir, S.A., and Roh, M. (2014) RNAi screen identifies a synthetic lethal interaction between PIM1 overexpression and PLK1 inhibition. <em><span class="cmti-12">Clinical Cancer Research</span></em>, <strong><span class="cmbx-12">20</span></strong>(12): 3211–3221.
</p>
                                                                                    

<p class="bibitem" >
<span class="biblabel"> <a 
 id="XvanderPost2015"></a><span class="bibsp">   </span></span>van der Post, R.S., Vogelaar, I.P., Carneiro, F., Guilford, P., Huntsman, D., Hoogerbrugge, N., Caldas, C., Schreiber, K.E., Hardwick, R.H., Ausems, M.G., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) Hereditary diffuse gastric cancer: updated clinical guidelines with an emphasis on germline CDH1 mutation carriers. <em><span class="cmti-12">J Med Genet</span></em>, <strong><span class="cmbx-12">52</span></strong>(6): 361–374.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XvanSteen2011"></a><span class="bibsp">   </span></span>van Steen, K. (2012) Travelling the world of gene–gene interactions. <em><span class="cmti-12">Briefings in</span> <span class="cmti-12">Bioinformatics</span></em>, <strong><span class="cmbx-12">13</span></strong>(1): 1–19.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XvanSteen2010"></a><span class="bibsp">   </span></span>van Steen, M. (2010) <em><span class="cmti-12">Graph Theory and Complex Networks: An Introduction</span></em>. Maarten van Steen, VU Amsterdam.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XVapnik1995"></a><span class="bibsp">   </span></span>Vapnik, V.N. (1995) <em><span class="cmti-12">The nature of statistical learning theory</span></em>. Springer-Verlag New York, Inc.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XVizeacoumar2013"></a><span class="bibsp">   </span></span>Vizeacoumar, F.J., Arnold, R., Vizeacoumar, F.S., Chandrashekhar, M., Buzina, A., Young, J.T., Kwan, J.H., Sayad, A., Mero, P., Lawo, S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2013) A negative genetic interaction map in isogenic cancer cell lines reveals cancer cell vulnerabilities. <em><span class="cmti-12">Mol Syst Biol</span></em>, <strong><span class="cmbx-12">9</span></strong>: 696.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XVogelstein2013"></a><span class="bibsp">   </span></span>Vogelstein, B., Papadopoulos, N., Velculescu, V.E., Zhou, S., Diaz, L.A., and Kinzler, K.W. (2013) Cancer genome landscapes. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">339</span></strong>(6127): 1546–1558.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XVos1997"></a><span class="bibsp">   </span></span>Vos, C.B., Cleton-Jansen, A.M., Berx, G., de Leeuw, W.J., ter Haar, N.T., van Roy, F., Cornelisse, C.J., Peterse, J.L., and van de Vijver, M.J. (1997) E-cadherin inactivation in lobular carcinoma in situ of the breast: an early event in tumorigenesis. <em><span class="cmti-12">Br J Cancer</span></em>, <strong><span class="cmbx-12">76</span></strong>(9): 1131–3.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XWaldron2016"></a><span class="bibsp">   </span></span>Waldron, D. (2016) Cancer genomics: A multi-layer omics approach to cancer. <em><span class="cmti-12">Nat</span> <span class="cmti-12">Rev Genet</span></em>, <strong><span class="cmbx-12">17</span></strong>(8): 436–437.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="Xmapsplice"></a><span class="bibsp">   </span></span>Wang, K., Singh, D., Zeng, Z., Coleman, S.J., Huang, Y., Savich, G.L., He, X., Mieczkowski, P., Grimm, S.A., Perou, C.M., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2010) MapSplice: accurate mapping of RNA-seq reads for splice junction discovery. <em><span class="cmti-12">Nucleic Acids Res</span></em>, <strong><span class="cmbx-12">38</span></strong>(18): e178.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XWang2013"></a><span class="bibsp">   </span></span>Wang, X. and Simon, R. (2013) Identification of potential synthetic lethal genes to p53 using a computational biology approach. <em><span class="cmti-12">BMC Medical Genomics</span></em>, <strong><span class="cmbx-12">6</span></strong>(1): 30.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XWappett2014"></a><span class="bibsp">   </span></span>Wappett, M. (2014) Bisep: Toolkit to identify candidate synthetic lethality. r package version 2.0.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XWappett2016"></a><span class="bibsp">   </span></span>Wappett, M., Dulak, A., Yang, Z.R., Al-Watban, A., Bradford, J.R., and Dry, J.R. (2016) Multi-omic measurement of mutually exclusive loss-of-function enriches for candidate synthetic lethal gene pairs. <em><span class="cmti-12">BMC Genomics</span></em>, <strong><span class="cmbx-12">17</span></strong>: 65.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="Xgplots"></a><span class="bibsp">   </span></span>Warnes, G.R., Bolker, B., Bonebakker, L., Gentleman, R., Liaw, W.H.A., Lumley, T., Maechler, M., Magnusson, A., Moeller, S., Schwartz, M., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) <em><span class="cmti-12">gplots:</span> <span class="cmti-12">Various R Programming Tools for Plotting Data</span></em>. R package version 2.17.0.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XWatts1998"></a><span class="bibsp">   </span></span>Watts, D.J. and Strogatz, S.H. (1998) Collective dynamics of ’small-world’ networks. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">393</span></strong>(6684): 440–2.</p>
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XWeinstein2000"></a><span class="bibsp">   </span></span>Weinstein, I.B. (2000) Disorders in cell circuitry during multistage carcinogenesis: the role of homeostasis. <em><span class="cmti-12">Carcinogenesis</span></em>, <strong><span class="cmbx-12">21</span></strong>(5): 857–864.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTCGA2014BL"></a><span class="bibsp">   </span></span>Weinstein, J.N., Akbani, R., Broom, B.M., Wang, W., Verhaak, R.G., McConkey, D., Lerner, S., Morgan, M., Creighton, C.J., Smith, C., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Comprehensive molecular characterization of urothelial bladder carcinoma. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">507</span></strong>(7492): 315–322.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XTCGA2013PAN"></a><span class="bibsp">   </span></span>Weinstein, J.N., Collisson, E.A., Mills, G.B., Shaw, K.R., Ozenberger, B.A., Ellrott, K., Shmulevich, I., Sander, C., Stuart, J.M., Chang, K., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2013) The Cancer Genome Atlas Pan-Cancer analysis project. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">45</span></strong>(10): 1113–1120.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="Xdevtools"></a><span class="bibsp">   </span></span>Wickham, H. and Chang, W. (2016) <em><span class="cmti-12">devtools: Tools to Make Developing R Packages</span> <span class="cmti-12">Easier</span></em>. R package version 1.12.0.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="Xroxygen"></a><span class="bibsp">   </span></span>Wickham, H., Danenberg, P., and Eugster, M. (2017) <em><span class="cmti-12">roxygen2: In-Line</span> <span class="cmti-12">Documentation for R</span></em>. R package version 6.0.1.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XWojtukiewicz2016"></a><span class="bibsp">   </span></span>Wojtukiewicz, M.Z., Hempel, D., Sierko, E., Tucker, S.C., and Honn, K.V. (2016) Thrombin-unique coagulation system protein with multifaceted impacts on cancer and metastasis. <em><span class="cmti-12">Cancer Metastasis Rev</span></em>, <strong><span class="cmbx-12">35</span></strong>(2): 213–233.
</p>
<p class="bibitem" >
<p><span class="biblabel"> <a 
 id="XWong2004"></a><span class="bibsp">   </span></span>Wong, S.L., Zhang, L.V., Tong, A.H.Y., Li, Z., Goldberg, D.S., King, O.D., Lesage, G., Vidal, M., Andrews, B., Bussey, H., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2004) Combining biological networks</p>
to predict genetic interactions. <em><span class="cmti-12">Proceedings of the National Academy of Sciences of</span> <span class="cmti-12">the United States of America</span></em>, <strong><span class="cmbx-12">101</span></strong>(44): 15682–15687.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XWorldHealthOrg2017"></a><span class="bibsp">   </span></span>World Health Organization (WHO) (2017) Fact sheet: Cancer. <a 
href="http://www.who.int/mediacentre/factsheets/fs297/en/" class="url" ><span class="cmtt-12">http://www.who.int/mediacentre/factsheets/fs297/en/</span></a>. Updated February 2017, Accessed: 22/03/2017.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XWu2014"></a><span class="bibsp">   </span></span>Wu, M., Li, X., Zhang, F., Li, X., Kwoh, C.K., and Zheng, J. (2014) In silico prediction of synthetic lethality by meta-analysis of genetic interactions, functions, and pathways in yeast and human cancer. <em><span class="cmti-12">Cancer Inform</span></em>, <strong><span class="cmbx-12">13</span></strong>(Suppl 3): 71–80.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XRmpi"></a><span class="bibsp">   </span></span>Yu, H. (2002) Rmpi: Parallel statistical computing in r. <em><span class="cmti-12">R News</span></em>, <strong><span class="cmbx-12">2</span></strong>(2): 10–14.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XZhang2015"></a><span class="bibsp">   </span></span>Zhang, F., Wu, M., Li, X.J., Li, X.L., Kwoh, C.K., and Zheng, J. (2015) Predicting essential genes and synthetic lethality via influence propagation in signaling pathways of cancer cell fates. <em><span class="cmti-12">J Bioinform Comput Biol</span></em>, <strong><span class="cmbx-12">13</span></strong>(3): 1541002.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XICGC2011"></a><span class="bibsp">   </span></span>Zhang, J., Baran, J., Cros, A., Guberman, J.M., Haider, S., Hsu, J., Liang, Y., Rivkin, E., Wang, J., Whitty, B., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2011) International cancer genome consortium data portal—a one-stop shop for cancer genomics data. <em><span class="cmti-12">Database: The</span> <span class="cmti-12">Journal of Biological Databases and Curation</span></em>, <strong><span class="cmbx-12">2011</span></strong>: bar026.
</p>
<p class="bibitem" >
<span class="biblabel"> <a 
 id="XZhong2006"></a><span class="bibsp">   </span></span>Zhong, W. and Sternberg, P.W. (2006) Genome-wide prediction of c. elegans genetic interactions. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">311</span></strong>(5766): 1481–1484.
</p>
                                                                                    

<p class="bibitem" >
<span class="biblabel"> <a 
 id="XZweig1993"></a><span class="bibsp">   </span></span>Zweig, M.H. and Campbell, G. (1993) Receiver-operating characteristic (roc) plots: a fundamental evaluation tool in clinical medicine. <em><span class="cmti-12">Clinical Chemistry</span></em>, <strong><span class="cmbx-12">39</span></strong>(4): 561–577.
</p>
</div>
                                                                                    



<a 
 id="x1-166001r356"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix A</span><br /><a 
 id="x1-167000A"></a>Sample Quality
</h2>
<a 
 id="x1-167001r364"></a>
<h3 class="sectionHead">
<span class="titlemark">A.1 </span> <a 
 id="x1-1680001"></a>Sample Correlation
</h3>
                                                                                    

<!--l. 24--><p class="indent" >
<a 
 id="x1-1680011"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 26--><p class="noindent" >
</p><!--l. 27--><p class="noindent" >
<img 
src="Raw(log)_Corr_eudist_Cut_part_Pub.png" alt="PIC"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure A.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Correlation profiles of removed samples.</span></strong> <span class="cmr-10x-x-109">Heatmap (Euclidean distance)</span> <span class="cmr-10x-x-109">of samples in </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">breast cancer dataset (left) clustered for all samples against removed</span> <span class="cmr-10x-x-109">samples (top): tissue source site (TSS), sample type with reds for tumour and greens for</span> <span class="cmr-10x-x-109">normal, patient (A2QH in pink), with varied analyte and plate. Excluded samples clustered</span> <span class="cmr-10x-x-109">at the bottom and annotation (left) show shared properties between samples in the dataset.</span> </span></p>
</div>
<!--tex4ht:label?: x1-1680011 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 39--><p class="indent" >
<a 
 id="x1-1680022"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 41--><p class="noindent" >
</p><!--l. 42--><p class="noindent" >
<img 
src="Raw(log)_Corr_eudist_Ann_Pub.png" alt="PIC"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure A.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Correlation analysis and sample removal.</span></strong> <span class="cmr-10x-x-109">Correlation matrix heatmap</span> <span class="cmr-10x-x-109">(Euclidean distance) of all samples in </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">breast cancer dataset against each other</span> <span class="cmr-10x-x-109">annotated for sample clinical data: sample type, tissue type, tumour stage, </span><a 
href="#glo:ER"><span class="cmr-10x-x-109">ER</span></a> <span class="cmr-10x-x-109">and intrinsic</span> <span class="cmr-10x-x-109">subtype (from the </span><a 
href="#glo:PAM50"><span class="cmr-10x-x-109">PAM50</span></a> <span class="cmr-10x-x-109">method). </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">somatic mutation, gene expression, and status</span> <span class="cmr-10x-x-109">for </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">prediction were also annotated. Discrete variables were coloured as displayed in</span> <span class="cmr-10x-x-109">the legend and continuous variables on a blue–red scale as shown in the colour key. Trimmed</span> <span class="cmr-10x-x-109">samples cluster at the bottom of the heatmap and the colour bars of the left show which were</span> <span class="cmr-10x-x-109">removed for quality concerns.</span></span></p>
</div>
<!--tex4ht:label?: x1-1680022 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<a 
 id="x1-168003r368"></a>
<h3 class="sectionHead">
<span class="titlemark">A.2 </span> <a 
 id="x1-1690002"></a>Replicate Samples in TCGA Breast Cancer Data
</h3>
<!--l. 56--><p class="noindent" >
Replicate samples were picked where possible from the <a 
href="#glo:TCGA">TCGA</a> breast cancer gene expression data to examine for sample quality. Independent samples of the same tumour were expected to have very high Pearson correlation between their expression profiles unless there were issues with sample collection or preparation and were thus an indicator of sample quality. The log-transformed raw read counts for replicate samples were examined in Figures <a 
href="#x1-1690013">A.3<!--tex4ht:ref: fig:rep_cutcut --></a>–<a 
href="#x1-1690249">A.9<!--tex4ht:ref: fig:rep_keepcut --></a>. These were examined before normalisation which would be expected to increase sample concordance.
</p><!--l. 58--><p class="indent" >
<p>Another consideration was the samples which were removed for quality concerns (in Section <a 
href="#x1-530002">2.2.2<!--tex4ht:ref: methods:sample_qc --></a>). While these were selected by unbiased hierarchical clustering (See Figure <a 
href="#x1-1680022">A.2<!--tex4ht:ref: fig:corr_map --></a>), many of the excluded (tumour) samples were performed in replicate despite relatively few replicate samples in the overall dataset. These samples correlate poorly with the rest of the dataset, in addition to correlation with replicate samples.</p>
</p><!--l. 60--><p class="indent" >
<a 
 id="x1-1690013"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<img 
src="Rep_picking_CutCut2.png" alt="pict"  
 /> <br />
<div class="caption">
<p><span class="id">Figure A.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">Replicate excluded samples.</span></strong> <span class="cmr-10x-x-109">Both tumour samples of patient A2QH were</span> <span class="cmr-10x-x-109">excluded as they were poorly correlated with other samples, although they were highly similar</span> <span class="cmr-10x-x-109">to each other as shown by Pearson correlation of log-raw counts. </span></span></p>
</div>
<!--tex4ht:label?: x1-1690013 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 72--><p class="indent" >
<a 
 id="x1-1690085"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 74--><p class="noindent" >
</p><!--l. 79--><p class="noindent" >
<img 
src="Rep_picking_KeepKeep(1).png" alt="pict"  
 /><a 
 id="x1-169002r1"></a> <span class="cmr-10x-x-109">Remaining</span> <span class="cmr-10x-x-109">triplet</span><img 
src="Rep_picking_KeepKeep(2).png" alt="pict"  
 /><a 
 id="x1-169003r2"></a> <span class="cmr-10x-x-109">Remaining</span> <span class="cmr-10x-x-109">triplet</span> <img 
src="Rep_picking_KeepKeep(3).png" alt="pict"  
 /><a 
 id="x1-169004r3"></a> <span class="cmr-10x-x-109">Remaining</span> <span class="cmr-10x-x-109">triplet</span>
</p><!--l. 94--><p class="noindent" >
<img 
src="Rep_picking_KeepKeep(4).png" alt="pict"  
 /><a 
 id="x1-169005r4"></a> <span class="cmr-10x-x-109">Remaining</span> <span class="cmr-10x-x-109">pair</span><img 
src="Rep_picking_KeepKeep(5).png" alt="pict"  
 /><a 
 id="x1-169006r5"></a> <span class="cmr-10x-x-109">Remaining</span> <span class="cmr-10x-x-109">pair</span> <img 
src="Rep_picking_KeepKeep(6).png" alt="pict"  
 /><a 
 id="x1-169007r6"></a> <span class="cmr-10x-x-109">Remaining</span> <span class="cmr-10x-x-109">pair</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure A.5: </span><span class="content"><strong><span class="cmbx-10x-x-109">Replicate samples with all remaining.</span></strong> <span class="cmr-10x-x-109">Patient A26J was sampled 3 times</span> <span class="cmr-10x-x-109">and compared pairwise. Pairs of samples were also compared for other patients with replicate</span> <span class="cmr-10x-x-109">samples. In all cases, replicate samples remaining in the dataset were highly concordant, as</span> <span class="cmr-10x-x-109">shown by Pearson correlation of log-raw counts. </span></span></p>
</div>
<!--tex4ht:label?: x1-1690085 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 113--><p class="indent" >
<a 
 id="x1-1690187"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 115--><p class="noindent" >
</p><!--l. 120--><p class="noindent" >
<img 
src="Rep_picking_KeepCut(1).png" alt="pict"  
 /><a 
 id="x1-169009r1"></a> <span class="cmr-10x-x-109">Remaining</span><img 
src="Rep_picking_KeepCut(2).png" alt="pict"  
 /><a 
 id="x1-169010r2"></a> <span class="cmr-10x-x-109">Compare</span> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">excluded</span><img 
src="Rep_picking_KeepCut(3).png" alt="pict"  
 /><a 
 id="x1-169011r3"></a> <span class="cmr-10x-x-109">Compare</span> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">excluded</span>
</p><!--l. 135--><p class="noindent" >
<img 
src="Rep_picking_KeepCut(4).png" alt="pict"  
 /><a 
 id="x1-169012r4"></a> <span class="cmr-10x-x-109">Remaining</span><img 
src="Rep_picking_KeepCut(5).png" alt="pict"  
 /><a 
 id="x1-169013r5"></a> <span class="cmr-10x-x-109">Compare</span> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">excluded</span><img 
src="Rep_picking_KeepCut(6).png" alt="pict"  
 /><a 
 id="x1-169014r6"></a> <span class="cmr-10x-x-109">Compare</span> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">excluded</span>
</p><!--l. 150--><p class="noindent" >
<img 
src="Rep_picking_KeepCut(7).png" alt="pict"  
 /><a 
 id="x1-169015r7"></a> <span class="cmr-10x-x-109">Remaining</span><img 
src="Rep_picking_KeepCut(8).png" alt="pict"  
 /><a 
 id="x1-169016r8"></a> <span class="cmr-10x-x-109">Compare</span> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">excluded</span><img 
src="Rep_picking_KeepCut(9).png" alt="pict"  
 /><a 
 id="x1-169017r9"></a> <span class="cmr-10x-x-109">Compare</span> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">excluded</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure A.7: </span><span class="content"><strong><span class="cmbx-10x-x-109">Replicate samples with some excluded.</span></strong> <span class="cmr-10x-x-109">(continued on next page) </span></span></p>
</div>
<!--tex4ht:label?: x1-1690187 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 166--><p class="indent" >
<a 
 id="x1-1690249"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 168--><p class="noindent" >
</p><!--l. 174--><p class="noindent" >
<img 
src="Rep_picking_KeepCut(10).png" alt="pict"  
 /><a 
 id="x1-169019r1"></a> <span class="cmr-10x-x-109">Remaining</span><img 
src="Rep_picking_KeepCut(11).png" alt="pict"  
 /><a 
 id="x1-169020r2"></a> <span class="cmr-10x-x-109">Compare</span> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">excluded</span><img 
src="Rep_picking_KeepCut(12).png" alt="pict"  
 /><a 
 id="x1-169021r3"></a> <span class="cmr-10x-x-109">Compare</span> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">excluded</span>
</p><!--l. 189--><p class="noindent" >
<img 
src="Rep_picking_KeepCut(13).png" alt="pict"  
 /><a 
 id="x1-169022r4"></a> <span class="cmr-10x-x-109">Compare</span> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">excluded</span><img 
src="Rep_picking_KeepCut(14).png" alt="pict"  
 /><a 
 id="x1-169023r5"></a> <span class="cmr-10x-x-109">Compare</span> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">excluded</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure A.9: </span><span class="content"><strong><span class="cmbx-10x-x-109">Replicate samples with some excluded.</span></strong> <span class="cmr-10x-x-109">Patients A0DB, A13D, A13E, and</span> <span class="cmr-10x-x-109">A26E were each sampled 3 times and compared pairwise. Pairs of samples were also compared</span> <span class="cmr-10x-x-109">for other patients with replicate samples. In all cases, the replicate samples remaining in the</span> <span class="cmr-10x-x-109">dataset more were highly concordant than those excluded from the analysis, as shown by</span> <span class="cmr-10x-x-109">Pearson correlation of log-raw countsr. </span></span></p>
</div>
<!--tex4ht:label?: x1-1690249 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<a 
 id="x1-169025r356"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix B</span><br /><a 
 id="x1-170000B"></a>Software Used for Thesis
</h2>
<a 
 id="x1-170001r1"></a> <!--l. 213-->
<div class="longtable">
<table id="TBL-72" class="longtable" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-72-1g"><col 
id="TBL-72-1" /><col 
id="TBL-72-2" /><col 
id="TBL-72-3" /><col 
id="TBL-72-4" /><col 
id="TBL-72-5" /><col 
id="TBL-72-6" /></colgroup>
<tr  
 style="vertical-align:baseline;" id="TBL-72-1-"><td colspan="3" style="white-space:nowrap; text-align:left;" id="TBL-72-1-1"  
class="td11"></td> <div class="multicolumn"  style="white-space:nowrap; text-align:center;"> <div class="caption" 
>
<p><span class="id">Table B.1: </span><span class="content">Complete list of R packages used during this thesis</span></p>
</div>
<!--tex4ht:label?: x1-170001r1 -->                                                                  </div>                   
<a 
 id="x1-170002"></a>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-2-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Package</span>
</div></td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-2-2"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Repository</span>
</div></td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-2-3"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Laptop</span>
</div></td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-2-4"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Lab</span>
</div></td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-2-5"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">Server</span>
</div></td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-2-6"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">
<span class="cmbx-12">NeSI</span>
</div>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-72-3-"><td colspan="3" style="white-space:nowrap; text-align:left;" id="TBL-72-3-1"  
class="td11"></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-4-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-5-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-6-"><td colspan="3" style="white-space:nowrap; text-align:left;" id="TBL-72-6-1"  
class="td11"></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-7-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-8-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-9-1"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-9-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-9-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-9-4"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-9-5"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-9-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-10-1"  
class="td11"> 
abind
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-10-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-10-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-10-4"  
class="td11"> 
1.4-5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-10-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-10-6"  
class="td11"> 
1.4-3
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-11-1"  
class="td11"> 
acepack
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-11-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-11-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-11-4"  
class="td11"> 
1.4.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-11-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-11-6"  
class="td11"> 
1.3-3.3
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-12-1"  
class="td11"> 
ade4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-12-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-12-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-12-4"  
class="td11"> 
1.7-5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-12-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-12-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-13-1"  
class="td11"> 
annaffy
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-13-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-13-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-13-4"  
class="td11"> 
1.46.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-13-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-13-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-14-1"  
class="td11"> 
AnnotationDbi
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-14-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-14-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-14-4"  
class="td11"> 
1.36.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-14-5"  
class="td11"> 
1.36.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-14-6"  
class="td11"> 
1.34.4
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-15-1"  
class="td11"> 
apComplex
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-15-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-15-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-15-4"  
class="td11"> 
2.40.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-15-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-15-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-16-1"  
class="td11"> 
ape
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-16-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-16-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-16-4"  
class="td11"> 
4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-16-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-16-6"  
class="td11"> 
3.4
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-17-1"  
class="td11"> 
arm
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-17-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-17-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-17-4"  
class="td11"> 
1.9-3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-17-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-17-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-18-1"  
class="td11"> 
assertthat
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-18-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-18-3"  
class="td11"> 
0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-18-4"  
class="td11"> 
0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-18-5"  
class="td11"> 
0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-18-6"  
class="td11"> 
0.1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-19-1"  
class="td11"> 
backports
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-19-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-19-3"  
class="td11"> 
1.0.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-19-4"  
class="td11"> 
1.0.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-19-5"  
class="td11"> 
1.0.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-19-6"  
class="td11"> 
1.0.2
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-20-1"  
class="td11"> 
base64
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-20-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-20-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-20-4"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-20-5"  
class="td11"> 
2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-20-6"  
class="td11"> 
2
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-21-1"  
class="td11"> 
base64enc
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-21-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-21-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-21-4"  
class="td11"> 
0.1-3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-21-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-21-6"  
class="td11"> 
0.1-3
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-22-1"  
class="td11"> 
beanplot
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-22-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-22-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-22-4"  
class="td11"> 
1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-22-5"  
class="td11"> 
1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-22-6"  
class="td11"> 
1.2
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-23-1"  
class="td11"> 
BH
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-23-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-23-3"  
class="td11"> 
1.60.0-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-23-4"  
class="td11"> 
1.62.0-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-23-5"  
class="td11"> 
1.62.0-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-23-6"  
class="td11"> 
1.60.0-2
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-24-1"  
class="td11"> 
Biobase
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-24-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-24-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-24-4"  
class="td11"> 
2.34.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-24-5"  
class="td11"> 
2.34.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-24-6"  
class="td11"> 
2.32.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-25-1"  
class="td11"> 
BiocGenerics
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-25-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-25-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-25-4"  
class="td11"> 
0.20.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-25-5"  
class="td11"> 
0.20.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-25-6"  
class="td11"> 
0.18.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-26-1"  
class="td11"> 
BiocInstaller
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-26-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-26-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-26-4"  
class="td11"> 
1.24.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-26-5"  
class="td11"> 
1.20.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-26-6"  
class="td11"> 
1.22.3
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-27-1"  
class="td11"></td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-27-2"  
class="td11"></td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-27-3"  
class="td11"></td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-27-4"  
class="td11"></td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-27-5"  
class="td11"></td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-27-6"  
class="td11"></td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-72-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-28-1"  
class="td11"> 
BiocParallel
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-28-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-28-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-28-4"  
class="td11"> 
1.8.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-28-5"  
class="td11"> 
1.8.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-28-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-29-1"  
class="td11"> 
Biostrings
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-29-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-29-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-29-4"  
class="td11"> 
2.42.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-29-5"  
class="td11"> 
2.42.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-29-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-30-1"  
class="td11"> 
BiSEp
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-30-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-30-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-30-4"  
class="td11"> 
2.0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-30-5"  
class="td11"> 
2.0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-30-6"  
class="td11"> 
2.0.1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-31-1"  
class="td11"> 
bitops
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-31-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-31-3"  
class="td11"> 
1.0-6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-31-4"  
class="td11"> 
1.0-6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-31-5"  
class="td11"> 
1.0-6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-31-6"  
class="td11"> 
1.0-6
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-32-1"  
class="td11"> 
boot
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-32-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-32-3"  
class="td11"> 
1.3-18
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-32-4"  
class="td11"> 
1.3-18
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-32-5"  
class="td11"> 
1.3-18
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-32-6"  
class="td11"> 
1.3-18
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-33-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-33-1"  
class="td11"> 
brew
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-33-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-33-3"  
class="td11"> 
1.0-6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-33-4"  
class="td11"> 
1.0-6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-33-5"  
class="td11"> 
1.0-6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-33-6"  
class="td11"> 
1.0-6
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-34-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-34-1"  
class="td11"> 
broom
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-34-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-34-3"  
class="td11"> 
0.4.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-34-4"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-34-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-34-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-35-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-35-1"  
class="td11"> 
caTools
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-35-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-35-3"  
class="td11"> 
1.17.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-35-4"  
class="td11"> 
1.17.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-35-5"  
class="td11"> 
1.17.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-35-6"  
class="td11"> 
1.17.1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-36-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-36-1"  
class="td11"> 
cgdsr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-36-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-36-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-36-4"  
class="td11"> 
1.2.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-36-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-36-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-37-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-37-1"  
class="td11"> 
checkmate
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-37-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-37-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-37-4"  
class="td11"> 
1.8.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-37-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-37-6"  
class="td11"> 
1.7.4
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-38-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-38-1"  
class="td11"> 
chron
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-38-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-38-3"  
class="td11"> 
2.3-47
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-38-4"  
class="td11"> 
2.3-48
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-38-5"  
class="td11"> 
2.3-50
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-38-6"  
class="td11"> 
2.3-47
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-39-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-39-1"  
class="td11"> 
class
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-39-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-39-3"  
class="td11"> 
7.3-14
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-39-4"  
class="td11"> 
7.3-14
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-39-5"  
class="td11"> 
7.3-14
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-39-6"  
class="td11"> 
7.3-14
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-40-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-40-1"  
class="td11"> 
cluster
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-40-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-40-3"  
class="td11"> 
2.0.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-40-4"  
class="td11"> 
2.0.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-40-5"  
class="td11"> 
2.0.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-40-6"  
class="td11"> 
2.0.4
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-41-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-41-1"  
class="td11"> 
coda
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-41-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-41-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-41-4"  
class="td11"> 
0.19-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-41-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-41-6"  
class="td11"> 
0.18-1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-42-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-42-1"  
class="td11"> 
codetools
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-42-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-42-3"  
class="td11"> 
0.2-15
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-42-4"  
class="td11"> 
0.2-15
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-42-5"  
class="td11"> 
0.2-15
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-42-6"  
class="td11"> 
0.2-14
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-43-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-43-1"  
class="td11"> 
colorRamps
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-43-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-43-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-43-4"  
class="td11"> 
2.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-43-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-43-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-44-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-44-1"  
class="td11"> 
colorspace
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-44-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-44-3"  
class="td11"> 
1.2-6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-44-4"  
class="td11"> 
1.3-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-44-5"  
class="td11"> 
1.3-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-44-6"  
class="td11"> 
1.2-6
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-45-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-45-1"  
class="td11"> 
commonmark
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-45-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-45-3"  
class="td11"> 
1.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-45-4"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-45-5"  
class="td11"> 
1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-45-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-46-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-46-1"  
class="td11"> 
compiler
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-46-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-46-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-46-4"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-46-5"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-46-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-47-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-47-1"  
class="td11"> 
corpcor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-47-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-47-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-47-4"  
class="td11"> 
1.6.8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-47-5"  
class="td11"> 
1.6.8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-47-6"  
class="td11"> 
1.6.8
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-48-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-48-1"  
class="td11"> </td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-72-49-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-49-1"  
class="td11"> 
Cprob
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-49-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-49-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-49-4"  
class="td11"> 
1.2.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-49-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-49-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-50-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-50-1"  
class="td11"> 
crayon
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-50-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-50-3"  
class="td11"> 
1.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-50-4"  
class="td11"> 
1.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-50-5"  
class="td11"> 
1.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-50-6"  
class="td11"> 
1.3.2
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-51-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-51-1"  
class="td11"> 
crop
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-51-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-51-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-51-4"  
class="td11"> 
0.0-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-51-5"  
class="td11"> 
0.0-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-51-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-52-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-52-1"  
class="td11"> 
curl
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-52-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-52-3"  
class="td11"> 
1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-52-4"  
class="td11"> 
2.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-52-5"  
class="td11"> 
2.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-52-6"  
class="td11"> 
0.9.7
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-53-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-53-1"  
class="td11"> 
d3Network
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-53-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-53-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-53-4"  
class="td11"> 
0.5.2.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-53-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-53-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-54-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-54-1"  
class="td11"> 
data.table
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-54-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-54-3"  
class="td11"> 
1.9.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-54-4"  
class="td11"> 
1.10.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-54-5"  
class="td11"> 
1.10.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-54-6"  
class="td11"> 
1.9.6
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-55-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-55-1"  
class="td11"> 
data.tree
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-55-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-55-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-55-4"  
class="td11"> 
0.7.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-55-5"  
class="td11"> 
0.7.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-55-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-56-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-56-1"  
class="td11"> 
datasets
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-56-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-56-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-56-4"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-56-5"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-56-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-57-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-57-1"  
class="td11"> 
DBI
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-57-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-57-3"  
class="td11"> 
0.5-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-57-4"  
class="td11"> 
0.5-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-57-5"  
class="td11"> 
0.5-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-57-6"  
class="td11"> 
0.5-1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-58-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-58-1"  
class="td11"> 
dendextend
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-58-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-58-3"  
class="td11"> 
1.4.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-58-4"  
class="td11"> 
1.4.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-58-5"  
class="td11"> 
1.4.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-58-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-59-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-59-1"  
class="td11"> 
DEoptimR
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-59-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-59-3"  
class="td11"> 
1.0-8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-59-4"  
class="td11"> 
1.0-8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-59-5"  
class="td11"> 
1.0-8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-59-6"  
class="td11"> 
1.0-4
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-60-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-60-1"  
class="td11"> 
desc
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-60-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-60-3"  
class="td11"> 
1.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-60-4"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-60-5"  
class="td11"> 
1.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-60-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-61-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-61-1"  
class="td11"> 
devtools
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-61-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-61-3"  
class="td11"> 
1.12.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-61-4"  
class="td11"> 
1.12.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-61-5"  
class="td11"> 
1.12.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-61-6"  
class="td11"> 
1.12.0
</td>
                                                                                    

</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-62-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-62-1"  
class="td11"> 
DiagrammeR
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-62-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-62-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-62-4"  
class="td11"> 
0.9.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-62-5"  
class="td11"> 
0.9.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-62-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-63-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-63-1"  
class="td11"> 
dichromat
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-63-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-63-3"  
class="td11"> 
2.0-0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-63-4"  
class="td11"> 
2.0-0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-63-5"  
class="td11"> 
2.0-0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-63-6"  
class="td11"> 
2.0-0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-64-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-64-1"  
class="td11"> 
digest
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-64-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-64-3"  
class="td11"> 
0.6.10
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-64-4"  
class="td11"> 
0.6.11
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-64-5"  
class="td11"> 
0.6.12
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-64-6"  
class="td11"> 
0.6.9
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-65-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-65-1"  
class="td11"> 
diptest
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-65-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-65-3"  
class="td11"> 
0.75-7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-65-4"  
class="td11"> 
0.75-7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-65-5"  
class="td11"> 
0.75-7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-65-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-66-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-66-1"  
class="td11"> 
doParallel
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-66-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-66-3"  
class="td11"> 
1.0.10
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-66-4"  
class="td11"> 
1.0.10
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-66-5"  
class="td11"> 
1.0.10
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-66-6"  
class="td11"> 
1.0.10
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-67-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-67-1"  
class="td11"> 
dplyr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-67-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-67-3"  
class="td11"> 
0.5.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-67-4"  
class="td11"> 
0.5.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-67-5"  
class="td11"> 
0.5.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-67-6"  
class="td11"> 
0.5.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-68-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-68-1"  
class="td11"> 
ellipse
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-68-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-68-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-68-4"  
class="td11"> 
0.3-8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-68-5"  
class="td11"> 
0.3-8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-68-6"  
class="td11"> 
0.3-8
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-69-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-69-1"  
class="td11"> </td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-72-70-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-70-1"  
class="td11"> 
evaluate
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-70-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-70-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-70-4"  
class="td11"> 
0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-70-5"  
class="td11"> 
0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-70-6"  
class="td11"> 
0.9
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-71-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-71-1"  
class="td11"> 
fdrtool
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-71-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-71-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-71-4"  
class="td11"> 
1.2.15
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-71-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-71-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-72-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-72-1"  
class="td11"> 
fields
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-72-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-72-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-72-4"  
class="td11"> 
8.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-72-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-72-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-73-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-73-1"  
class="td11"> 
flexmix
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-73-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-73-3"  
class="td11"> 
2.3-13
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-73-4"  
class="td11"> 
2.3-13
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-73-5"  
class="td11"> 
2.3-13
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-73-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-74-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-74-1"  
class="td11"> 
forcats
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-74-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-74-3"  
class="td11"> 
0.2.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-74-4"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-74-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-74-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-75-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-75-1"  
class="td11"> 
foreach
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-75-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-75-3"  
class="td11"> 
1.4.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-75-4"  
class="td11"> 
1.4.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-75-5"  
class="td11"> 
1.4.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-75-6"  
class="td11"> 
1.4.3
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-76-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-76-1"  
class="td11"> 
foreign
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-76-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-76-3"  
class="td11"> 
0.8-67
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-76-4"  
class="td11"> 
0.8-67
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-76-5"  
class="td11"> 
0.8-67
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-76-6"  
class="td11"> 
0.8-66
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-77-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-77-1"  
class="td11"> 
formatR
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-77-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-77-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-77-4"  
class="td11"> 
1.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-77-5"  
class="td11"> 
1.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-77-6"  
class="td11"> 
1.4
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-78-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-78-1"  
class="td11"> 
Formula
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-78-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-78-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-78-4"  
class="td11"> 
1.2-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-78-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-78-6"  
class="td11"> 
1.2-1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-79-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-79-1"  
class="td11"> 
fpc
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-79-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-79-3"  
class="td11"> 
2.1-10
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-79-4"  
class="td11"> 
2.1-10
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-79-5"  
class="td11"> 
2.1-10
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-79-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-80-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-80-1"  
class="td11"> 
futile.logger
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-80-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-80-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-80-4"  
class="td11"> 
1.4.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-80-5"  
class="td11"> 
1.4.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-80-6"  
class="td11"> 
1.4.1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-81-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-81-1"  
class="td11"> 
futile.options
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-81-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-81-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-81-4"  
class="td11"> 
1.0.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-81-5"  
class="td11"> 
1.0.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-81-6"  
class="td11"> 
1.0.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-82-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-82-1"  
class="td11"> 
gdata
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-82-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-82-3"  
class="td11"> 
2.17.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-82-4"  
class="td11"> 
2.17.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-82-5"  
class="td11"> 
2.17.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-82-6"  
class="td11"> 
2.17.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-83-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-83-1"  
class="td11"> 
geepack
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-83-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-83-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-83-4"  
class="td11"> 
1.2-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-83-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-83-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-84-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-84-1"  
class="td11"> 
GenomeInfoDb
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-84-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-84-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-84-4"  
class="td11"> 
1.10.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-84-5"  
class="td11"> 
1.10.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-84-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-85-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-85-1"  
class="td11"> 
GenomicAlignments
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-85-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-85-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-85-4"  
class="td11"> 
1.10.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-85-5"  
class="td11"> 
1.10.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-85-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-86-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-86-1"  
class="td11"> 
GenomicRanges
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-86-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-86-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-86-4"  
class="td11"> 
1.26.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-86-5"  
class="td11"> 
1.26.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-86-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-87-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-87-1"  
class="td11"> 
ggm
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-87-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-87-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-87-4"  
class="td11"> 
2.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-87-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-87-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-88-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-88-1"  
class="td11"> 
ggplot2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-88-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-88-3"  
class="td11"> 
2.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-88-4"  
class="td11"> 
2.2.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-88-5"  
class="td11"> 
2.2.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-88-6"  
class="td11"> 
2.1.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-89-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-89-1"  
class="td11"> 
git2r
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-89-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-89-3"  
class="td11"> 
0.15.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-89-4"  
class="td11"> 
0.18.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-89-5"  
class="td11"> 
0.16.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-89-6"  
class="td11"> 
0.15.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-90-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-90-1"  
class="td11"> </td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-72-91-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-91-1"  
class="td11"> 
glasso
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-91-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-91-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-91-4"  
class="td11"> 
1.8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-91-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-91-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-92-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-92-1"  
class="td11"> 
GO.db
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-92-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-92-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-92-4"  
class="td11"> 
3.4.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-92-5"  
class="td11"> 
3.2.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-92-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-93-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-93-1"  
class="td11"> 
GOSemSim
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-93-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-93-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-93-4"  
class="td11"> 
2.0.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-93-5"  
class="td11"> 
1.28.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-93-6"  
class="td11"> 
1.30.3
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-94-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-94-1"  
class="td11"> 
gplots
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-94-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-94-3"  
class="td11"> 
3.0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-94-4"  
class="td11"> 
3.0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-94-5"  
class="td11"> 
3.0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-94-6"  
class="td11"> 
3.0.1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-95-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-95-1"  
class="td11"> 
graph
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-95-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-95-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-95-4"  
class="td11"> 
1.52.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-95-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-95-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-96-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-96-1"  
class="td11"> 
graphics
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-96-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-96-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-96-4"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-96-5"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-96-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-97-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-97-1"  
class="td11"> 
graphsim
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-97-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-73" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-73-1g"><col 
id="TBL-73-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-73-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-73-1-1"  
class="td00">
GitHub
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-73-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-73-2-1"  
class="td00">
TomKellyGenetics
</td> </tr></table> </div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-97-3"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-97-4"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-97-5"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-97-6"  
class="td11"> 
0.1.0
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-98-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-98-1"  
class="td11"> 
grDevices
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-98-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-98-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-98-4"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-98-5"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-98-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-99-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-99-1"  
class="td11"> 
grid
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-99-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-99-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-99-4"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-99-5"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-99-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-100-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-100-1"  
class="td11"> 
gridBase
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-100-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-100-3"  
class="td11"> 
0.4-7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-100-4"  
class="td11"> 
0.4-7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-100-5"  
class="td11"> 
0.4-7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-100-6"  
class="td11"> 
0.4-7
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-101-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-101-1"  
class="td11"> 
gridExtra
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-101-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-101-3"  
class="td11"> 
2.2.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-101-4"  
class="td11"> 
2.2.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-101-5"  
class="td11"> 
2.2.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-101-6"  
class="td11"> 
2.2.1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-102-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-102-1"  
class="td11"> 
gridGraphics
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-102-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-102-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-102-4"  
class="td11"> 
0.1-5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-102-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-102-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-103-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-103-1"  
class="td11"> 
gtable
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-103-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-103-3"  
class="td11"> 
0.2.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-103-4"  
class="td11"> 
0.2.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-103-5"  
class="td11"> 
0.2.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-103-6"  
class="td11"> 
0.2.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-104-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-104-1"  
class="td11"> 
gtools
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-104-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-104-3"  
class="td11"> 
3.5.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-104-4"  
class="td11"> 
3.5.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-104-5"  
class="td11"> 
3.5.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-104-6"  
class="td11"> 
3.5.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-105-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-105-1"  
class="td11"> 
haven
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-105-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-105-3"  
class="td11"> 
1.0.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-105-4"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-105-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-105-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-106-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-106-1"  
class="td11"> 
heatmap.2x
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-106-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-74" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-74-1g"><col 
id="TBL-74-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-74-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-74-1-1"  
class="td00">
GitHub
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-74-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-74-2-1"  
class="td00">
TomKellyGenetics
</td> </tr></table> </div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-106-3"  
class="td11"> 
0.0.0.9000
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-106-4"  
class="td11"> 
0.0.0.9000
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-106-5"  
class="td11"> 
0.0.0.9000
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-106-6"  
class="td11"> 
0.0.0.9000
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-107-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-107-1"  
class="td11"> 
hgu133plus2.db
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-107-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-107-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-107-4"  
class="td11"> 
3.2.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-107-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-107-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-108-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-108-1"  
class="td11"> 
highr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-108-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-108-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-108-4"  
class="td11"> 
0.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-108-5"  
class="td11"> 
0.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-108-6"  
class="td11"> 
0.6
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-109-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-109-1"  
class="td11"> 
Hmisc
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-109-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-109-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-109-4"  
class="td11"> 
4.0-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-109-5"  
class="td11"> 
4.0-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-109-6"  
class="td11"> 
3.17-4
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-110-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-110-1"  
class="td11"> 
hms
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-110-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-110-3"  
class="td11"> 
0.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-110-4"  
class="td11"> 
0.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-110-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-110-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-111-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-111-1"  
class="td11"> </td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-72-112-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-112-1"  
class="td11"> 
htmlTable
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-112-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-112-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-112-4"  
class="td11"> 
1.8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-112-5"  
class="td11"> 
1.9
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-112-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-113-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-113-1"  
class="td11"> 
htmltools
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-113-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-113-3"  
class="td11"> 
0.3.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-113-4"  
class="td11"> 
0.3.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-113-5"  
class="td11"> 
0.3.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-113-6"  
class="td11"> 
0.3.5
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-114-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-114-1"  
class="td11"> 
htmlwidgets
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-114-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-114-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-114-4"  
class="td11"> 
0.8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-114-5"  
class="td11"> 
0.8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-114-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-115-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-115-1"  
class="td11"> 
httpuv
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-115-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-115-3"  
class="td11"> 
1.3.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-115-4"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-115-5"  
class="td11"> 
1.3.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-115-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-116-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-116-1"  
class="td11"> 
httr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-116-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-116-3"  
class="td11"> 
1.2.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-116-4"  
class="td11"> 
1.2.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-116-5"  
class="td11"> 
1.2.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-116-6"  
class="td11"> 
1.1.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-117-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-117-1"  
class="td11"> 
huge
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-117-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-117-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-117-4"  
class="td11"> 
1.2.7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-117-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-117-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-118-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-118-1"  
class="td11"> 
hunspell
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-118-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-118-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-118-4"  
class="td11"> 
2.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-118-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-118-6"  
class="td11"> 
2
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-119-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-119-1"  
class="td11"> 
hypergraph
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-119-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-119-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-119-4"  
class="td11"> 
1.46.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-119-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-119-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-120-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-120-1"  
class="td11"> 
igraph
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-120-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-120-3"  
class="td11"> 
1.0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-120-4"  
class="td11"> 
1.0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-120-5"  
class="td11"> 
1.0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-120-6"  
class="td11"> 
1.0.1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-121-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-121-1"  
class="td11"> 
igraph.extensions
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-121-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-75" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-75-1g"><col 
id="TBL-75-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-75-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-75-1-1"  
class="td00">
GitHub
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-75-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-75-2-1"  
class="td00">
TomKellyGenetics
</td> </tr></table> </div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-121-3"  
class="td11"> 
0.1.0.9001
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-121-4"  
class="td11"> 
0.1.0.9001
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-121-5"  
class="td11"> 
0.1.0.9001
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-121-6"  
class="td11"> 
0.1.0.9001
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-122-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-122-1"  
class="td11"> 
influenceR
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-122-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-122-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-122-4"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-122-5"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-122-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-123-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-123-1"  
class="td11"> 
info.centrality
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-123-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-76" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-76-1g"><col 
id="TBL-76-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-76-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-76-1-1"  
class="td00">
GitHub
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-76-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-76-2-1"  
class="td00">
TomKellyGenetics
</td> </tr></table> </div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-123-3"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-123-4"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-123-5"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-123-6"  
class="td11"> 
0.1.0
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-124-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-124-1"  
class="td11"> 
IRanges
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-124-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-124-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-124-4"  
class="td11"> 
2.8.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-124-5"  
class="td11"> 
2.8.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-124-6"  
class="td11"> 
2.6.1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-125-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-125-1"  
class="td11"> 
irlba
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-125-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-125-3"  
class="td11"> 
2.1.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-125-4"  
class="td11"> 
2.1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-125-5"  
class="td11"> 
2.1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-125-6"  
class="td11"> 
2.0.0
</td>
                                                                                    

</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-126-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-126-1"  
class="td11"> 
iterators
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-126-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-126-3"  
class="td11"> 
1.0.8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-126-4"  
class="td11"> 
1.0.8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-126-5"  
class="td11"> 
1.0.8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-126-6"  
class="td11"> 
1.0.8
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-127-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-127-1"  
class="td11"> 
jpeg
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-127-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-127-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-127-4"  
class="td11"> 
0.1-8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-127-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-127-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-128-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-128-1"  
class="td11"> 
jsonlite
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-128-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-128-3"  
class="td11"> 
1.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-128-4"  
class="td11"> 
1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-128-5"  
class="td11"> 
1.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-128-6"  
class="td11"> 
0.9.20
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-129-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-129-1"  
class="td11"> 
KEGG.db
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-129-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-129-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-129-4"  
class="td11"> 
3.2.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-129-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-129-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-130-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-130-1"  
class="td11"> 
kernlab
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-130-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-130-3"  
class="td11"> 
0.9-25
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-130-4"  
class="td11"> 
0.9-25
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-130-5"  
class="td11"> 
0.9-25
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-130-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-131-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-131-1"  
class="td11"> 
KernSmooth
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-131-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-131-3"  
class="td11"> 
2.23-15
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-131-4"  
class="td11"> 
2.23-15
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-131-5"  
class="td11"> 
2.23-15
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-131-6"  
class="td11"> 
2.23-15
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-132-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-132-1"  
class="td11"> </td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-72-133-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-133-1"  
class="td11"> 
knitr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-133-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-133-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-133-4"  
class="td11"> 
1.15.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-133-5"  
class="td11"> 
1.15.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-133-6"  
class="td11"> 
1.14
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-134-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-134-1"  
class="td11"> 
labeling
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-134-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-134-3"  
class="td11"> 
0.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-134-4"  
class="td11"> 
0.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-134-5"  
class="td11"> 
0.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-134-6"  
class="td11"> 
0.3
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-135-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-135-1"  
class="td11"> 
lambda.r
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-135-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-135-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-135-4"  
class="td11"> 
1.1.9
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-135-5"  
class="td11"> 
1.1.9
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-135-6"  
class="td11"> 
1.1.7
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-136-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-136-1"  
class="td11"> 
lattice
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-136-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-136-3"  
class="td11"> 
0.20-34
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-136-4"  
class="td11"> 
0.20-34
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-136-5"  
class="td11"> 
0.20-34
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-136-6"  
class="td11"> 
0.20-33
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-137-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-137-1"  
class="td11"> 
latticeExtra
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-137-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-137-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-137-4"  
class="td11"> 
0.6-28
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-137-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-137-6"  
class="td11"> 
0.6-28
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-138-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-138-1"  
class="td11"> 
lava
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-138-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-138-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-138-4"  
class="td11"> 
1.4.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-138-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-138-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-139-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-139-1"  
class="td11"> 
lavaan
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-139-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-139-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-139-4"  
class="td11"> 
0.5-22
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-139-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-139-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-140-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-140-1"  
class="td11"> 
lazyeval
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-140-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-140-3"  
class="td11"> 
0.2.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-140-4"  
class="td11"> 
0.2.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-140-5"  
class="td11"> 
0.2.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-140-6"  
class="td11"> 
0.2.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-141-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-141-1"  
class="td11"> 
les
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-141-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-141-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-141-4"  
class="td11"> 
1.24.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-141-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-141-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-142-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-142-1"  
class="td11"> 
lgtdl
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-142-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-142-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-142-4"  
class="td11"> 
1.1.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-142-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-142-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-143-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-143-1"  
class="td11"> 
limma
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-143-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-143-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-143-4"  
class="td11"> 
3.30.7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-143-5"  
class="td11"> 
3.30.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-143-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-144-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-144-1"  
class="td11"> 
lme4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-144-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-144-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-144-4"  
class="td11"> 
1.1-12
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-144-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-144-6"  
class="td11"> 
1.1-12
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-145-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-145-1"  
class="td11"> 
lubridate
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-145-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-145-3"  
class="td11"> 
1.6.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-145-4"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-145-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-145-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-146-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-146-1"  
class="td11"> 
magrittr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-146-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-146-3"  
class="td11"> 
1.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-146-4"  
class="td11"> 
1.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-146-5"  
class="td11"> 
1.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-146-6"  
class="td11"> 
1.5
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-147-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-147-1"  
class="td11"> 
maps
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-147-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-147-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-147-4"  
class="td11"> 
3.1.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-147-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-147-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-148-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-148-1"  
class="td11"> 
markdown
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-148-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-148-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-148-4"  
class="td11"> 
0.7.7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-148-5"  
class="td11"> 
0.7.7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-148-6"  
class="td11"> 
0.7.7
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-149-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-149-1"  
class="td11"> 
MASS
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-149-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-149-3"  
class="td11"> 
7.3-45
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-149-4"  
class="td11"> 
7.3-45
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-149-5"  
class="td11"> 
7.3-45
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-149-6"  
class="td11"> 
7.3-45
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-150-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-150-1"  
class="td11"> 
Matrix
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-150-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-150-3"  
class="td11"> 
1.2-7.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-150-4"  
class="td11"> 
1.2-7.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-150-5"  
class="td11"> 
1.2-8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-150-6"  
class="td11"> 
1.2-6
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-151-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-151-1"  
class="td11"> 
matrixcalc
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-151-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-151-3"  
class="td11"> 
1.0-3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-151-4"  
class="td11"> 
1.0-3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-151-5"  
class="td11"> 
1.0-3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-151-6"  
class="td11"> 
1.0-3
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-152-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-152-1"  
class="td11"> 
mclust
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-152-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-152-3"  
class="td11"> 
5.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-152-4"  
class="td11"> 
5.2.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-152-5"  
class="td11"> 
5.2.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-152-6"  
class="td11"> 
5.2
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-153-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-153-1"  
class="td11"> </td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-72-154-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-154-1"  
class="td11"> 
memoise
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-154-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-154-3"  
class="td11"> 
1.0.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-154-4"  
class="td11"> 
1.0.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-154-5"  
class="td11"> 
1.0.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-154-6"  
class="td11"> 
1.0.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-155-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-155-1"  
class="td11"> 
methods
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-155-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-155-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-155-4"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-155-5"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-155-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-156-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-156-1"  
class="td11"> 
mgcv
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-156-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-156-3"  
class="td11"> 
1.8-16
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-156-4"  
class="td11"> 
1.8-16
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-156-5"  
class="td11"> 
1.8-17
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-156-6"  
class="td11"> 
1.8-12
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-157-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-157-1"  
class="td11"> 
mi
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-157-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-157-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-157-4"  
class="td11"> 
1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-157-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-157-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-158-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-158-1"  
class="td11"> 
mime
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-158-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-158-3"  
class="td11"> 
0.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-158-4"  
class="td11"> 
0.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-158-5"  
class="td11"> 
0.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-158-6"  
class="td11"> 
0.4
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-159-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-159-1"  
class="td11"> 
minqa
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-159-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-159-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-159-4"  
class="td11"> 
1.2.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-159-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-159-6"  
class="td11"> 
1.2.4
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-160-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-160-1"  
class="td11"> 
mnormt
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-160-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-160-3"  
class="td11"> 
1.5-5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-160-4"  
class="td11"> 
1.5-5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-160-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-160-6"  
class="td11"> 
1.5-4
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-161-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-161-1"  
class="td11"> 
modelr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-161-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-161-3"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-161-4"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-161-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-161-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-162-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-162-1"  
class="td11"> 
modeltools
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-162-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-162-3"  
class="td11"> 
0.2-21
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-162-4"  
class="td11"> 
0.2-21
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-162-5"  
class="td11"> 
0.2-21
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-162-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-163-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-163-1"  
class="td11"> 
multtest
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-163-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-163-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-163-4"  
class="td11"> 
2.30.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-163-5"  
class="td11"> 
2.30.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-163-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-164-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-164-1"  
class="td11"> 
munsell
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-164-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-164-3"  
class="td11"> 
0.4.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-164-4"  
class="td11"> 
0.4.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-164-5"  
class="td11"> 
0.4.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-164-6"  
class="td11"> 
0.4.3
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-165-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-165-1"  
class="td11"> 
mvtnorm
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-165-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-165-3"  
class="td11"> 
1.0-5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-165-4"  
class="td11"> 
1.0-5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-165-5"  
class="td11"> 
1.0-6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-165-6"  
class="td11"> 
1.0-5
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-166-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-166-1"  
class="td11"> 
network
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-166-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-166-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-166-4"  
class="td11"> 
1.13.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-166-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-166-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-167-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-167-1"  
class="td11"> 
nlme
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-167-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-167-3"  
class="td11"> 
3.1-128
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-167-4"  
class="td11"> 
3.1-128
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-167-5"  
class="td11"> 
3.1-131
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-167-6"  
class="td11"> 
3.1-128
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-168-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-168-1"  
class="td11"> 
nloptr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-168-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-168-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-168-4"  
class="td11"> 
1.0.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-168-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-168-6"  
class="td11"> 
1.0.4
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-169-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-169-1"  
class="td11"> 
NMF
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-169-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-169-3"  
class="td11"> 
0.20.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-169-4"  
class="td11"> 
0.20.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-169-5"  
class="td11"> 
0.20.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-169-6"  
class="td11"> 
0.20.6
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-170-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-170-1"  
class="td11"> 
nnet
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-170-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-170-3"  
class="td11"> 
7.3-12
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-170-4"  
class="td11"> 
7.3-12
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-170-5"  
class="td11"> 
7.3-12
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-170-6"  
class="td11"> 
7.3-12
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-171-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-171-1"  
class="td11"> 
numDeriv
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-171-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-171-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-171-4"  
class="td11"> 
2016.8-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-171-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-171-6"  
class="td11"> 
2014.2-1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-172-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-172-1"  
class="td11"> 
openssl
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-172-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-172-3"  
class="td11"> 
0.9.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-172-4"  
class="td11"> 
0.9.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-172-5"  
class="td11"> 
0.9.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-172-6"  
class="td11"> 
0.9.4
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-173-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-173-1"  
class="td11"> 
org.Hs.eg.db
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-173-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-173-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-173-4"  
class="td11"> 
3.1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-173-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-173-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-174-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-174-1"  
class="td11"> </td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-72-175-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-175-1"  
class="td11"> 
org.Sc.sgd.db
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-175-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-175-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-175-4"  
class="td11"> 
3.4.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-175-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-175-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-176-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-176-1"  
class="td11"> 
parallel
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-176-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-176-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-176-4"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-176-5"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-176-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-177-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-177-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-77" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-77-1g"><col 
id="TBL-77-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-77-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-77-1-1"  
class="td00">
pathway.structure
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-77-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-77-2-1"  
class="td00">
.permutation
</td> </tr></table> </div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-177-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-78" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-78-1g"><col 
id="TBL-78-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-78-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-78-1-1"  
class="td00">
GitHub
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-78-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-78-2-1"  
class="td00">
TomKellyGenetics
</td> </tr></table>                                                                                                                 </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-177-3"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-177-4"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-177-5"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-177-6"  
class="td11"> 
0.1.0
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-178-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-178-1"  
class="td11"> 
pbivnorm
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-178-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-178-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-178-4"  
class="td11"> 
0.6.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-178-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-178-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-179-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-179-1"  
class="td11"> 
PGSEA
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-179-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-179-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-179-4"  
class="td11"> 
1.48.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-179-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-179-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-180-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-180-1"  
class="td11"> 
pkgmaker
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-180-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-180-3"  
class="td11"> 
0.22
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-180-4"  
class="td11"> 
0.22
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-180-5"  
class="td11"> 
0.22
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-180-6"  
class="td11"> 
0.22
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-181-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-181-1"  
class="td11"> 
PKI
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-181-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-181-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-181-4"  
class="td11"> 
0.1-3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-181-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-181-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-182-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-182-1"  
class="td11"> 
plogr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-182-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-182-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-182-4"  
class="td11"> 
0.1-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-182-5"  
class="td11"> 
0.1-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-182-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-183-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-183-1"  
class="td11"> 
plot.igraph
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-183-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-79" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-79-1g"><col 
id="TBL-79-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-79-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-79-1-1"  
class="td00">
GitHub
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-79-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-79-2-1"  
class="td00">
TomKellyGenetics
</td> </tr></table>                                                                                                                 </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-183-3"  
class="td11"> 
0.0.0.9001
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-183-4"  
class="td11"> 
0.0.0.9001
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-183-5"  
class="td11"> 
0.0.0.9001
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-183-6"  
class="td11"> 
0.0.0.9001
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-184-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-184-1"  
class="td11"> 
plotrix
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-184-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-184-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-184-4"  
class="td11"> 
3.6-4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-184-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-184-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-185-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-185-1"  
class="td11"> 
plyr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-185-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-185-3"  
class="td11"> 
1.8.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-185-4"  
class="td11"> 
1.8.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-185-5"  
class="td11"> 
1.8.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-185-6"  
class="td11"> 
1.8.3
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-186-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-186-1"  
class="td11"> 
png
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-186-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-186-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-186-4"  
class="td11"> 
0.1-7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-186-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-186-6"  
class="td11"> 
0.1-7
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-187-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-187-1"  
class="td11"> 
prabclus
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-187-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-187-3"  
class="td11"> 
2.2-6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-187-4"  
class="td11"> 
2.2-6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-187-5"  
class="td11"> 
2.2-6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-187-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-188-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-188-1"  
class="td11"> 
praise
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-188-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-188-3"  
class="td11"> 
1.0.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-188-4"  
class="td11"> 
1.0.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-188-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-188-6"  
class="td11"> 
1.0.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-189-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-189-1"  
class="td11"> 
pROC
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-189-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-189-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-189-4"  
class="td11"> 
1.8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-189-5"  
class="td11"> 
1.9.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-189-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-190-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-190-1"  
class="td11"> 
prodlim
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-190-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-190-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-190-4"  
class="td11"> 
1.5.7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-190-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-190-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-191-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-191-1"  
class="td11"> 
prof.tree
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-191-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-191-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-191-4"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-191-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-191-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-192-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-192-1"  
class="td11"> 
proftools
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-192-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-192-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-192-4"  
class="td11"> 
0.99-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-192-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-192-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-193-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-193-1"  
class="td11"> 
progress
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-193-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-193-3"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-193-4"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-193-5"  
class="td11"> 
1.1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-193-6"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-194-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-194-1"  
class="td11"> 
psych
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-194-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-194-3"  
class="td11"> 
1.6.12
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-194-4"  
class="td11"> 
1.6.12
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-194-5"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-194-6"  
class="td11">          </td>
                                                                                    

</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-195-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-195-1"  
class="td11"> </td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-72-196-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-196-1"  
class="td11"> 
purrr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-196-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-196-3"  
class="td11"> 
0.2.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-196-4"  
class="td11"> 
0.2.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-196-5"  
class="td11"> 
0.2.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-196-6"  
class="td11"> 
0.2.2
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-197-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-197-1"  
class="td11"> 
qgraph
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-197-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-197-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-197-4"  
class="td11"> 
1.4.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-197-5"  
class="td11">           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-197-6"  
class="td11">           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-198-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-198-1"  
class="td11"> 
quadprog
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-198-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-198-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-198-4"  
class="td11"> 
1.5-5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-198-5"  
class="td11"> 
1.5-5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-198-6"  
class="td11"> 
1.5-5
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-199-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-199-1"  
class="td11"> 
R.methodsS3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-199-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-199-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-199-4"  
class="td11"> 
1.7.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-199-5"  
class="td11">           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-199-6"  
class="td11"> 
1.7.1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-200-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-200-1"  
class="td11"> 
R.oo
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-200-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-200-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-200-4"  
class="td11"> 
1.21.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-200-5"  
class="td11">           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-200-6"  
class="td11"> 
1.20.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-201-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-201-1"  
class="td11"> 
R.utils
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-201-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-201-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-201-4"  
class="td11"> 
2.5.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-201-5"  
class="td11">           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-201-6"  
class="td11">           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-202-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-202-1"  
class="td11"> 
R6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-202-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-202-3"  
class="td11"> 
2.1.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-202-4"  
class="td11"> 
2.2.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-202-5"  
class="td11"> 
2.2.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-202-6"  
class="td11"> 
2.1.3
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-203-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-203-1"  
class="td11"> 
RBGL
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-203-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-203-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-203-4"  
class="td11"> 
1.50.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-203-5"  
class="td11">           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-203-6"  
class="td11">           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-204-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-204-1"  
class="td11"> 
RColorBrewer
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-204-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-204-3"  
class="td11"> 
1.1-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-204-4"  
class="td11"> 
1.1-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-204-5"  
class="td11"> 
1.1-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-204-6"  
class="td11"> 
1.1-2
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-205-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-205-1"  
class="td11"> 
Rcpp
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-205-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-205-3"  
class="td11"> 
0.12.7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-205-4"  
class="td11"> 
0.12.9
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-205-5"  
class="td11"> 
0.12.9
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-205-6"  
class="td11"> 
0.12.7
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-206-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-206-1"  
class="td11"> 
RcppArmadillo
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-206-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-206-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-206-4"  
class="td11">         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-206-5"  
class="td11"> 
0.7.700.0.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-206-6"  
class="td11"> 
0.6.700.6.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-207-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-207-1"  
class="td11"> 
RcppEigen
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-207-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-207-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-207-4"  
class="td11"> 
0.3.2.9.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-207-5"  
class="td11">           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-207-6"  
class="td11"> 
0.3.2.8.1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-208-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-208-1"  
class="td11"> 
RCurl
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-208-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-208-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-208-4"  
class="td11"> 
1.95-4.8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-208-5"  
class="td11"> 
1.95-4.8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-208-6"  
class="td11"> 
1.95-4.8
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-209-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-209-1"  
class="td11"> 
reactome.db
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-209-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-209-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-209-4"  
class="td11"> 
1.52.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-209-5"  
class="td11"> 
1.52.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-209-6"  
class="td11">           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-210-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-210-1"  
class="td11"> 
reactometree
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-210-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-80" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-80-1g"><col 
id="TBL-80-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-80-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-80-1-1"  
class="td00">
GitHub
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-80-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-80-2-1"  
class="td00">
TomKellyGenetics
</td> </tr></table> </div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-210-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-210-4"  
class="td11"> 
0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-210-5"  
class="td11">           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-210-6"  
class="td11">           </td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-211-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-211-1"  
class="td11"> 
readr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-211-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-211-3"  
class="td11"> 
1.0.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-211-4"  
class="td11"> 
1.0.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-211-5"  
class="td11">           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-211-6"  
class="td11">           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-212-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-212-1"  
class="td11"> 
readxl
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-212-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-212-3"  
class="td11"> 
0.1.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-212-4"  
class="td11">         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-212-5"  
class="td11">           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-212-6"  
class="td11">           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-213-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-213-1"  
class="td11"> 
registry
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-213-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-213-3"  
class="td11"> 
0.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-213-4"  
class="td11"> 
0.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-213-5"  
class="td11"> 
0.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-213-6"  
class="td11"> 
0.3
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-214-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-214-1"  
class="td11"> 
reshape2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-214-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-214-3"  
class="td11"> 
1.4.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-214-4"  
class="td11"> 
1.4.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-214-5"  
class="td11"> 
1.4.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-214-6"  
class="td11"> 
1.4.1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-215-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-215-1"  
class="td11"> 
rgexf
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-215-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-215-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-215-4"  
class="td11"> 
0.15.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-215-5"  
class="td11"> 
0.15.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-215-6"  
class="td11">           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-216-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-216-1"  
class="td11"> </td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-72-217-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-217-1"  
class="td11"> 
rgl
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-217-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-217-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-217-4"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-217-5"  
class="td11"> 
0.97.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-217-6"  
class="td11"> 
0.95.1441
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-218-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-218-1"  
class="td11"> 
Rgraphviz
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-218-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-218-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-218-4"  
class="td11"> 
2.18.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-218-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-218-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-219-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-219-1"  
class="td11"> 
rjson
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-219-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-219-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-219-4"  
class="td11"> 
0.2.15
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-219-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-219-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-220-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-220-1"  
class="td11"> 
RJSONIO
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-220-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-220-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-220-4"  
class="td11"> 
1.3-0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-220-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-220-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-221-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-221-1"  
class="td11"> 
rmarkdown
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-221-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-221-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-221-4"  
class="td11"> 
1.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-221-5"  
class="td11"> 
1.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-221-6"  
class="td11"> 
1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-222-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-222-1"  
class="td11"> 
Rmpi
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-222-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-222-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-222-4"  
class="td11"> 
0.6-6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-222-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-222-6"  
class="td11"> 
0.6-5
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-223-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-223-1"  
class="td11"> 
rngtools
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-223-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-223-3"  
class="td11"> 
1.2.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-223-4"  
class="td11"> 
1.2.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-223-5"  
class="td11"> 
1.2.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-223-6"  
class="td11"> 
1.2.4
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-224-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-224-1"  
class="td11"> 
robustbase
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-224-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-224-3"  
class="td11"> 
0.92-7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-224-4"  
class="td11"> 
0.92-7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-224-5"  
class="td11"> 
0.92-7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-224-6"  
class="td11"> 
0.92-5
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-225-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-225-1"  
class="td11"> 
ROCR
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-225-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-225-3"  
class="td11"> 
1.0-7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-225-4"  
class="td11"> 
1.0-7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-225-5"  
class="td11"> 
1.0-7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-225-6"  
class="td11"> 
1.0-7
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-226-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-226-1"  
class="td11"> 
Rook
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-226-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-226-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-226-4"  
class="td11"> 
1.1-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-226-5"  
class="td11"> 
1.1-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-226-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-227-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-227-1"  
class="td11"> 
roxygen2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-227-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-227-3"  
class="td11"> 
6.0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-227-4"  
class="td11"> 
5.0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-227-5"  
class="td11"> 
6.0.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-227-6"  
class="td11"> 
5.0.1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-228-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-228-1"  
class="td11"> 
rpart
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-228-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-228-3"  
class="td11"> 
4.1-10
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-228-4"  
class="td11"> 
4.1-10
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-228-5"  
class="td11"> 
4.1-10
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-228-6"  
class="td11"> 
4.1-10
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-229-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-229-1"  
class="td11"> 
rprojroot
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-229-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-229-3"  
class="td11"> 
1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-229-4"  
class="td11"> 
1.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-229-5"  
class="td11"> 
1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-229-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-230-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-230-1"  
class="td11"> 
Rsamtools
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-230-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-230-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-230-4"  
class="td11"> 
1.26.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-230-5"  
class="td11"> 
1.26.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-230-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-231-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-231-1"  
class="td11"> 
rsconnect
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-231-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-231-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-231-4"  
class="td11"> 
0.7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-231-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-231-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-232-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-232-1"  
class="td11"> 
RSQLite
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-232-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-232-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-232-4"  
class="td11"> 
1.1-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-232-5"  
class="td11"> 
1.1-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-232-6"  
class="td11"> 
1.0.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-233-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-233-1"  
class="td11"> 
rstudioapi
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-233-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-233-3"  
class="td11"> 
0.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-233-4"  
class="td11"> 
0.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-233-5"  
class="td11"> 
0.6
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-233-6"  
class="td11"> 
0.6
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-234-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-234-1"  
class="td11"> 
rvest
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-234-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-234-3"  
class="td11"> 
0.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-234-4"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-234-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-234-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-235-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-235-1"  
class="td11"> 
S4Vectors
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-235-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-235-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-235-4"  
class="td11"> 
0.12.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-235-5"  
class="td11"> 
0.12.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-235-6"  
class="td11"> 
0.10.3
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-236-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-236-1"  
class="td11"> 
safe
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-236-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-236-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-236-4"  
class="td11"> 
3.14.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-236-5"  
class="td11"> 
3.10.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-236-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-237-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-237-1"  
class="td11"> </td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-72-238-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-238-1"  
class="td11"> 
scales
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-238-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-238-3"  
class="td11"> 
0.4.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-238-4"  
class="td11"> 
0.4.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-238-5"  
class="td11"> 
0.4.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-238-6"  
class="td11"> 
0.4.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-239-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-239-1"  
class="td11"> 
selectr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-239-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-239-3"  
class="td11"> 
0.3-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-239-4"  
class="td11">       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-239-5"  
class="td11">       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-239-6"  
class="td11">       </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-240-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-240-1"  
class="td11"> 
sem
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-240-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-240-3"  
class="td11">       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-240-4"  
class="td11"> 
3.1-8
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-240-5"  
class="td11">       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-240-6"  
class="td11">       </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-241-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-241-1"  
class="td11"> 
shiny
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-241-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-241-3"  
class="td11"> 
0.14
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-241-4"  
class="td11">       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-241-5"  
class="td11"> 
1.0.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-241-6"  
class="td11">       </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-242-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-242-1"  
class="td11"> 
slipt
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-242-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-81" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-81-1g"><col 
id="TBL-81-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-81-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-81-1-1"  
class="td00">
GitHub
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-81-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-81-2-1"  
class="td00">
TomKellyGenetics
</td> </tr></table>                                                                                                                 </div>  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-242-3"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-242-4"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-242-5"  
class="td11"> 
0.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-242-6"  
class="td11"> 
0.1.0
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-243-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-243-1"  
class="td11"> 
sm
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-243-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-243-3"  
class="td11"> 
2.2-5.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-243-4"  
class="td11"> 
2.2-5.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-243-5"  
class="td11">       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-243-6"  
class="td11">       </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-244-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-244-1"  
class="td11"> 
sna
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-244-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-244-3"  
class="td11">       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-244-4"  
class="td11"> 
2.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-244-5"  
class="td11">       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-244-6"  
class="td11">       </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-245-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-245-1"  
class="td11"> 
snow
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-245-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-245-3"  
class="td11"> 
0.4-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-245-4"  
class="td11"> 
0.4-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-245-5"  
class="td11"> 
0.4-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-245-6"  
class="td11"> 
0.3-13
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-246-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-246-1"  
class="td11"> 
sourcetools
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-246-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-246-3"  
class="td11"> 
0.1.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-246-4"  
class="td11">       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-246-5"  
class="td11"> 
0.1.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-246-6"  
class="td11">       </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-247-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-247-1"  
class="td11"> 
SparseM
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-247-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-247-3"  
class="td11">       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-247-4"  
class="td11"> 
1.74
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-247-5"  
class="td11">       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-247-6"  
class="td11"> 
1.7
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-248-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-248-1"  
class="td11"> 
spatial
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-248-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-248-3"  
class="td11"> 
7.3-11
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-248-4"  
class="td11"> 
7.3-11
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-248-5"  
class="td11"> 
7.3-11
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-248-6"  
class="td11"> 
7.3-11
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-249-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-249-1"  
class="td11"> 
splines
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-249-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-249-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-249-4"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-249-5"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-249-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-250-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-250-1"  
class="td11"> 
statnet.common
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-250-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-250-3"  
class="td11">       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-250-4"  
class="td11"> 
3.3.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-250-5"  
class="td11">       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-250-6"  
class="td11">       </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-251-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-251-1"  
class="td11"> 
stats
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-251-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-251-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-251-4"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-251-5"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-251-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-252-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-252-1"  
class="td11"> 
stats4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-252-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-252-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-252-4"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-252-5"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-252-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-253-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-253-1"  
class="td11"> 
stringi
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-253-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-253-3"  
class="td11"> 
1.1.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-253-4"  
class="td11"> 
1.1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-253-5"  
class="td11"> 
1.1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-253-6"  
class="td11"> 
1.0-1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-254-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-254-1"  
class="td11"> 
stringr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-254-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-254-3"  
class="td11"> 
1.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-254-4"  
class="td11"> 
1.1.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-254-5"  
class="td11"> 
1.2.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-254-6"  
class="td11"> 
1.0.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-255-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-255-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-82" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-82-1g"><col 
id="TBL-82-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-82-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-82-1-1"  
class="td00">
Summarized
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-82-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-82-2-1"  
class="td00">
Experiment
</td> </tr></table> </div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-255-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-255-3"  
class="td11">       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-255-4"  
class="td11"> 
1.4.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-255-5"  
class="td11"> 
1.4.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-255-6"  
class="td11">       </td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-256-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-256-1"  
class="td11"> 
survival
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-256-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-256-3"  
class="td11"> 
2.39-4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-256-4"  
class="td11"> 
2.40-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-256-5"  
class="td11"> 
2.40-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-256-6"  
class="td11"> 
2.39-4
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-257-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-257-1"  
class="td11"> 
tcltk
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-257-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-257-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-257-4"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-257-5"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-257-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-258-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-258-1"  
class="td11"> </td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-72-259-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-259-1"  
class="td11"> 
testthat
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-259-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-259-3"  
class="td11"> 
1.0.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-259-4"  
class="td11"> 
1.0.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-259-5"  
class="td11">        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-259-6"  
class="td11"> 
1.0.2
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-260-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-260-1"  
class="td11"> 
tibble
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-260-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-260-3"  
class="td11"> 
1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-260-4"  
class="td11"> 
1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-260-5"  
class="td11"> 
1.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-260-6"  
class="td11"> 
1.2
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-261-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-261-1"  
class="td11"> 
tidyr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-261-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-261-3"  
class="td11"> 
0.6.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-261-4"  
class="td11"> 
0.6.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-261-5"  
class="td11"> 
0.6.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-261-6"  
class="td11">        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-262-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-262-1"  
class="td11"> 
tidyverse
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-262-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-83" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-83-1g"><col 
id="TBL-83-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-83-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-83-1-1"  
class="td00">
GitHub
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-83-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-83-2-1"  
class="td00">
hadley
</td> </tr></table> </div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-262-3"  
class="td11"> 
1.1.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-262-4"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-262-5"  
class="td11">        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-262-6"  
class="td11">        </td>



</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-263-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-263-1"  
class="td11"> 
timeline
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-263-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-263-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-263-4"  
class="td11"> 
0.9
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-263-5"  
class="td11">        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-263-6"  
class="td11">        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-264-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-264-1"  
class="td11"> 
tools
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-264-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-264-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-264-4"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-264-5"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-264-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-265-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-265-1"  
class="td11"> 
tpr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-265-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-265-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-265-4"  
class="td11"> 
0.3-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-265-5"  
class="td11">        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-265-6"  
class="td11">        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-266-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-266-1"  
class="td11"> 
trimcluster
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-266-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-266-3"  
class="td11"> 
0.1-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-266-4"  
class="td11"> 
0.1-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-266-5"  
class="td11"> 
0.1-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-266-6"  
class="td11">        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-267-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-267-1"  
class="td11"> 
Unicode
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-267-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-267-3"  
class="td11"> 
9.0.0-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-267-4"  
class="td11"> 
9.0.0-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-267-5"  
class="td11"> 
9.0.0-1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-267-6"  
class="td11">        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-268-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-268-1"  
class="td11"> 
utils
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-268-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-268-3"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-268-4"  
class="td11"> 
3.3.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-268-5"  
class="td11"> 
3.3.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-268-6"  
class="td11"> 
3.3.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-269-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-269-1"  
class="td11"> 
vioplot
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-269-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-269-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-269-4"  
class="td11"> 
0.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-269-5"  
class="td11">        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-269-6"  
class="td11">        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-270-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-270-1"  
class="td11"> 
vioplotx
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-270-2"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-84" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-84-1g"><col 
id="TBL-84-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-84-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-84-1-1"  
class="td00">
GitHub
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-84-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-84-2-1"  
class="td00">
TomKellyGenetics
</td> </tr></table> </div> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-270-3"  
class="td11"> 
0.0.0.9000
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-270-4"  
class="td11"> 
0.0.0.9000
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-270-5"  
class="td11">        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-270-6"  
class="td11">        </td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-271-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-271-1"  
class="td11"> 
viridis
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-271-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-271-3"  
class="td11"> 
0.3.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-271-4"  
class="td11"> 
0.3.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-271-5"  
class="td11"> 
0.3.4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-271-6"  
class="td11">        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-272-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-272-1"  
class="td11"> 
visNetwork
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-272-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-272-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-272-4"  
class="td11"> 
1.0.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-272-5"  
class="td11"> 
1.0.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-272-6"  
class="td11">        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-273-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-273-1"  
class="td11"> 
whisker
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-273-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-273-3"  
class="td11"> 
0.3-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-273-4"  
class="td11"> 
0.3-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-273-5"  
class="td11"> 
0.3-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-273-6"  
class="td11"> 
0.3-2
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-274-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-274-1"  
class="td11"> 
withr
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-274-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-274-3"  
class="td11"> 
1.0.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-274-4"  
class="td11"> 
1.0.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-274-5"  
class="td11"> 
1.0.2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-274-6"  
class="td11"> 
1.0.2
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-275-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-275-1"  
class="td11"> 
XML
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-275-2"  
class="td11"> 
base
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-275-3"  
class="td11"> 
3.98-1.3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-275-4"  
class="td11"> 
3.98-1.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-275-5"  
class="td11"> 
3.98-1.5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-275-6"  
class="td11"> 
3.98-1.4
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-276-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-276-1"  
class="td11"> 
xml2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-276-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-276-3"  
class="td11"> 
1.1.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-276-4"  
class="td11">          </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-276-5"  
class="td11"> 
1.1.1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-276-6"  
class="td11"> 
1.0.0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-277-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-277-1"  
class="td11"> 
xtable
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-277-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-277-3"  
class="td11"> 
1.8-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-277-4"  
class="td11"> 
1.8-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-277-5"  
class="td11"> 
1.8-2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-277-6"  
class="td11"> 
1.8-2
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-278-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-278-1"  
class="td11"> 
XVector
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-278-2"  
class="td11"> 
Bioconductor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-278-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-278-4"  
class="td11"> 
0.14.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-278-5"  
class="td11"> 
0.14.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-278-6"  
class="td11">        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-279-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-279-1"  
class="td11"> </td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-72-280-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-280-1"  
class="td11"> 
yaml
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-280-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-280-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-280-4"  
class="td11"> 
2.1.14
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-280-5"  
class="td11"> 
2.1.14
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-280-6"  
class="td11"> 
2.1.13
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-281-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-281-1"  
class="td11"> 
zlibbioc
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-281-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-281-3"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-281-4"  
class="td11"> 
1.20.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-281-5"  
class="td11"> 
1.20.0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-281-6"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-282-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-282-1"  
class="td11"> 
zoo
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-282-2"  
class="td11"> 
<a 
href="#glo:CRAN">CRAN</a>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-282-3"  
class="td11"> 
1.7-13
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-282-4"  
class="td11"> 
1.7-14
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-282-5"  
class="td11"> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-72-282-6"  
class="td11"> 
1.7-13
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-283-"><td colspan="3" style="white-space:nowrap; text-align:left;" id="TBL-72-283-1"  
class="td11"></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-284-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-284-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-285-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-285-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-286-"><td colspan="3" style="white-space:nowrap; text-align:left;" id="TBL-72-286-1"  
class="td11"></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-287-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-287-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-288-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-288-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-289-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-289-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-290-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-290-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-291-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-291-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-292-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-292-1"  
class="td11"> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-72-293-"><td  style="white-space:nowrap; text-align:left;" id="TBL-72-293-1"  
class="td11"> </td></tr>
</table></div>
                                                                                    





<a 
 id="x1-170003r356"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix C</span><br /><a 
 id="x1-171000C"></a>Mutation Analysis in Breast Cancer
</h2>
<a 
 id="x1-171001r371"></a>
<h3 class="sectionHead">
<span class="titlemark">C.1 </span> <a 
 id="x1-1720001"></a>Synthetic Lethal Genes and Pathways
</h3>
<!--l. 78--><p class="noindent" >
<p>SLIPT expression analysis (described in Section <a 
href="#x1-740001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) on <a 
href="#glo:TCGA">TCGA</a> breast cancer data (<span class="cmmi-12">n </span>= 969) found the following genes and pathways, described in Sections <a 
href="#x1-970001">4.1<!--tex4ht:ref: chapt3:exprSL_genes --></a> and <a 
href="#x1-980001">4.1.1<!--tex4ht:ref: chapt3:exprSL_pathways --></a>.</p>
</p><!--l. 80--><p class="indent" >
<a 
 id="x1-1720011"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table C.1: </span><span class="content">Candidate synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from mtSLIPT</span>
</div><!--tex4ht:label?: x1-1720011 -->
<div class="tabular">
<table id="TBL-85" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-85-1g"><col 
id="TBL-85-1" /><col 
id="TBL-85-2" /><col 
id="TBL-85-3" /><col 
id="TBL-85-4" /><col 
id="TBL-85-5" /><col 
id="TBL-85-6" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-85-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-1-1"  
class="td11"> 
<span class="cmbx-12">Gene </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-1-2"  
class="td11"> 
Observed<sup class="textsuperscript"><span class="cmr-10x-x-109">*</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-1-3"  
class="td11"> 
Expected<sup class="textsuperscript"><span class="cmr-10x-x-109">*</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-1-4"  
class="td11"> 
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-1-5"  
class="td11">   
p-value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-1-6"  
class="td11"> 
p-value (<a 
href="#glo:FDR">FDR</a>)
</td>

</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-85-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-2-1"  
class="td11"> 
<span class="cmti-12">TFAP2B </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-2-2"  
class="td11">    
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-2-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-2-4"  
class="td11">   
89.5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-2-5"  
class="td11"> 
3<span class="cmmi-12">.</span>60 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-2-6"  
class="td11">  
8<span class="cmmi-12">.</span>37 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-3-1"  
class="td11"> 
<span class="cmti-12">ZNF423 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-3-2"  
class="td11">    
15
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-3-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-3-4"  
class="td11">   
78.8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-3-5"  
class="td11"> 
7<span class="cmmi-12">.</span>89 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-3-6"  
class="td11">  
1<span class="cmmi-12">.</span>22 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-4-1"  
class="td11"> 
<span class="cmti-12">CALCOCO1 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-4-2"  
class="td11">    
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-4-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-4-4"  
class="td11">   
76.8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-4-5"  
class="td11"> 
2<span class="cmmi-12">.</span>09 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-4-6"  
class="td11">  
2<span class="cmmi-12">.</span>59 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-5-1"  
class="td11"> 
<span class="cmti-12">RBM5 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-5-2"  
class="td11">    
13
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-5-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-5-4"  
class="td11">   
75.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-5-5"  
class="td11"> 
3<span class="cmmi-12">.</span>65 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-5-6"  
class="td11">  
4<span class="cmmi-12">.</span>00 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-6-1"  
class="td11"> 
<span class="cmti-12">BTG2 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-6-2"  
class="td11">    
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-6-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-6-4"  
class="td11">   
71.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-6-5"  
class="td11"> 
2<span class="cmmi-12">.</span>72 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-6-6"  
class="td11">  
1<span class="cmmi-12">.</span>81 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-7-1"  
class="td11"> 
<span class="cmti-12">RXRA </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-7-2"  
class="td11">    
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-7-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-7-4"  
class="td11">   
70.5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-7-5"  
class="td11"> 
5<span class="cmmi-12">.</span>00 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-7-6"  
class="td11">  
2<span class="cmmi-12">.</span>97 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-8-1"  
class="td11"> 
<span class="cmti-12">SLC27A1 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-8-2"  
class="td11">    
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-8-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-8-4"  
class="td11">   
70.3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-8-5"  
class="td11"> 
5<span class="cmmi-12">.</span>42 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-8-6"  
class="td11">  
2<span class="cmmi-12">.</span>97 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-9-1"  
class="td11"> 
<span class="cmti-12">MEF2D </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-9-2"  
class="td11">    
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-9-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-9-4"  
class="td11">   
69.6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-9-5"  
class="td11"> 
7<span class="cmmi-12">.</span>86 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-9-6"  
class="td11">  
3<span class="cmmi-12">.</span>95 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-10-1"  
class="td11"> 
<span class="cmti-12">NISCH </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-10-2"  
class="td11">    
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-10-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-10-4"  
class="td11">   
69.6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-10-5"  
class="td11"> 
7<span class="cmmi-12">.</span>86 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-10-6"  
class="td11">  
3<span class="cmmi-12">.</span>95 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-11-1"  
class="td11"> 
<span class="cmti-12">AVPR2 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-11-2"  
class="td11">    
9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-11-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-11-4"  
class="td11">   
69.2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-11-5"  
class="td11"> 
9<span class="cmmi-12">.</span>36 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-11-6"  
class="td11">  
4<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-12-1"  
class="td11"> 
<span class="cmti-12">CRY2 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-12-2"  
class="td11">    
13
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-12-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-12-4"  
class="td11">   
68.9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-12-5"  
class="td11"> 
1<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-12-6"  
class="td11">  
4<span class="cmmi-12">.</span>98 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-13-1"  
class="td11"> 
<span class="cmti-12">RAPGEF3 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-13-2"  
class="td11">    
13
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-13-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-13-4"  
class="td11">   
68.9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-13-5"  
class="td11"> 
1<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-13-6"  
class="td11">  
4<span class="cmmi-12">.</span>98 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-14-1"  
class="td11"> 
<span class="cmti-12">NRIP2 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-14-2"  
class="td11">    
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-14-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-14-4"  
class="td11">   
68.2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-14-5"  
class="td11"> 
1<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-14-6"  
class="td11">  
7<span class="cmmi-12">.</span>18 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-15-1"  
class="td11"> 
<span class="cmti-12">DARC </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-15-2"  
class="td11">    
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-15-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-15-4"  
class="td11">   
66.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-15-5"  
class="td11"> 
3<span class="cmmi-12">.</span>76 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-15-6"  
class="td11">  
1<span class="cmmi-12">.</span>54 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-16-1"  
class="td11"> 
<span class="cmti-12">SFRS5 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-16-2"  
class="td11">    
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-16-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-16-4"  
class="td11">   
66.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-16-5"  
class="td11"> 
3<span class="cmmi-12">.</span>76 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-16-6"  
class="td11">  
1<span class="cmmi-12">.</span>54 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-17-1"  
class="td11"> 
<span class="cmti-12">NOSTRIN </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-17-2"  
class="td11">    
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-17-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-17-4"  
class="td11">   
65.1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-17-5"  
class="td11"> 
7<span class="cmmi-12">.</span>40 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-17-6"  
class="td11">  
2<span class="cmmi-12">.</span>70 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-18-1"  
class="td11"> 
<span class="cmti-12">KIF13B </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-18-2"  
class="td11">    
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-18-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-18-4"  
class="td11">   
63.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-18-5"  
class="td11"> 
1<span class="cmmi-12">.</span>69 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-18-6"  
class="td11">  
5<span class="cmmi-12">.</span>16 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-19-1"  
class="td11"> 
<span class="cmti-12">TENC1 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-19-2"  
class="td11">    
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-19-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-19-4"  
class="td11">   
62.5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-19-5"  
class="td11"> 
2<span class="cmmi-12">.</span>67 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-19-6"  
class="td11">  
7<span class="cmmi-12">.</span>40 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-20-1"  
class="td11"> 
<span class="cmti-12">MFAP4 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-20-2"  
class="td11">    
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-20-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-20-4"  
class="td11">   
60.5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-20-5"  
class="td11"> 
7<span class="cmmi-12">.</span>17 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-20-6"  
class="td11">  
1<span class="cmmi-12">.</span>67 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-21-1"  
class="td11"> 
<span class="cmti-12">ELN </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-21-2"  
class="td11">    
13
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-21-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-21-4"  
class="td11">   
59.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-21-5"  
class="td11"> 
1<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-21-6"  
class="td11">  
2<span class="cmmi-12">.</span>32 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-22-1"  
class="td11"> 
<span class="cmti-12">SGK223 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-22-2"  
class="td11">    
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-22-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-22-4"  
class="td11">   
59
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-22-5"  
class="td11"> 
1<span class="cmmi-12">.</span>51 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-22-6"  
class="td11">  
3<span class="cmmi-12">.</span>05 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-23-1"  
class="td11"> 
<span class="cmti-12">KIF12 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-23-2"  
class="td11">    
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-23-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-23-4"  
class="td11">   
58.8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-23-5"  
class="td11"> 
1<span class="cmmi-12">.</span>74 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-23-6"  
class="td11">  
3<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-24-1"  
class="td11"> 
<span class="cmti-12">SELP </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-24-2"  
class="td11">    
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-24-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-24-4"  
class="td11">   
58.8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-24-5"  
class="td11"> 
1<span class="cmmi-12">.</span>74 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-24-6"  
class="td11">  
3<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-25-1"  
class="td11"> 
<span class="cmti-12">CIRBP </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-25-2"  
class="td11">    
9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-25-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-25-4"  
class="td11">   
58.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-25-5"  
class="td11"> 
1<span class="cmmi-12">.</span>83 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-25-6"  
class="td11">  
3<span class="cmmi-12">.</span>41 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-85-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-26-1"  
class="td11"> 
<span class="cmti-12">CTDSP1 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-26-2"  
class="td11">    
9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-26-3"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-26-4"  
class="td11">   
58.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-26-5"  
class="td11"> 
1<span class="cmmi-12">.</span>83 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-85-26-6"  
class="td11">  
3<span class="cmmi-12">.</span>41 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-85-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-85-27-1"  
class="td11">             </td></tr></table>
<!--l. 150--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Strongest candidate </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">by </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">in </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">in breast cancer</span> <span class="cmr-10x-x-109">expression and mutation data</span> <sup class="textsuperscript"><span class="cmr-10x-x-109">*</span></sup><span class="cmr-10x-x-109">Observed and expected numbers of </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutant </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">breast tumours with low expression of</span> <span class="cmr-10x-x-109">partner genes</span></p>
</p>
</div><hr class="endfloat" />
                                                                                    

<!--l. 154--><p class="indent" >
<a 
 id="x1-1720022"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table C.2: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT</span>
</div><!--tex4ht:label?: x1-1720022 -->
<div class="tabular">
<table id="TBL-86" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-86-1g"><col 
id="TBL-86-1" /><col 
id="TBL-86-2" /><col 
id="TBL-86-3" /><col 
id="TBL-86-4" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-86-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-1-1"  
class="td11"> 
<strong><span class="cmbx-12">Pathways Over-represented</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-1-2"  
class="td11"> 
<strong><span class="cmbx-12">Pathway Size</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-1-3"  
class="td11"> 
<strong><span class="cmbx-12">SL Genes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-1-4"  
class="td11"> 
<strong><span class="cmbx-12">p-value (</span><a 
href="#glo:FDR"><span class="cmbx-12">FDR</span></a><span class="cmbx-12">)</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-86-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-2-1"  
class="td11"> 
Eukaryotic Translation Elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-2-2"  
class="td11">      
86
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-2-3"  
class="td11">     
60
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-2-4"  
class="td11">   
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">128</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-3-1"  
class="td11"> 
Peptide chain elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-3-2"  
class="td11">      
83
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-3-3"  
class="td11">     
59
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-3-4"  
class="td11">   
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">128</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-4-1"  
class="td11"> 
Eukaryotic Translation Termination
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-4-2"  
class="td11">      
83
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-4-3"  
class="td11">     
58
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-4-4"  
class="td11">   
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">125</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-5-1"  
class="td11"> 
Viral <a 
href="#glo:mRNA">mRNA</a> Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-5-2"  
class="td11">      
81
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-5-3"  
class="td11">     
57
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-5-4"  
class="td11">   
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">124</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-6-1"  
class="td11"> 
Nonsense Mediated Decay independent of the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-6-2"  
class="td11">      
88
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-6-3"  
class="td11">     
59
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-6-4"  
class="td11">   
8<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">124</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-7-1"  
class="td11"> 
Nonsense-Mediated Decay
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-7-2"  
class="td11">      
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-7-3"  
class="td11">     
61
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-7-4"  
class="td11">   
5<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">117</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-8-1"  
class="td11"> 
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-8-2"  
class="td11">      
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-8-3"  
class="td11">     
61
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-8-4"  
class="td11">   
5<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">117</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-9-1"  
class="td11"> 
Formation of a pool of free 40S subunits
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-9-2"  
class="td11">      
93
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-9-3"  
class="td11">     
58
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-9-4"  
class="td11">   
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">116</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-10-1"  
class="td11"> 
L13a-mediated translational silencing of Ceruloplasmin expression
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-10-2"  
class="td11">      
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-10-3"  
class="td11">     
59
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-10-4"  
class="td11">   
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">111</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-11-1"  
class="td11"> 
3’ -UTR-mediated translational regulation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-11-2"  
class="td11">      
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-11-3"  
class="td11">     
59
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-11-4"  
class="td11">   
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">111</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-12-1"  
class="td11"> 
GTP hydrolysis and joining of the 60S ribosomal subunit
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-12-2"  
class="td11">      
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-12-3"  
class="td11">     
59
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-12-4"  
class="td11">   
6<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">111</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-13-1"  
class="td11"> 
SRP-dependent cotranslational protein targeting to membrane
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-13-2"  
class="td11">      
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-13-3"  
class="td11">     
58
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-13-4"  
class="td11">   
2<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">108</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-14-1"  
class="td11"> 
Eukaryotic Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-14-2"  
class="td11">      
111
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-14-3"  
class="td11">     
59
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-14-4"  
class="td11">   
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">106</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-15-1"  
class="td11"> 
Cap-dependent Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-15-2"  
class="td11">      
111
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-15-3"  
class="td11">     
59
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-15-4"  
class="td11">   
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">106</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-16-1"  
class="td11"> 
Influenza Viral <a 
href="#glo:RNA">RNA</a> Transcription and Replication
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-16-2"  
class="td11">      
108
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-16-3"  
class="td11">     
57
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-16-4"  
class="td11">   
5<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">103</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-17-1"  
class="td11"> 
Influenza Infection
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-17-2"  
class="td11">      
117
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-17-3"  
class="td11">     
59
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-17-4"  
class="td11">   
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">102</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-18-1"  
class="td11"> 
Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-18-2"  
class="td11">      
141
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-18-3"  
class="td11">     
64
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-18-4"  
class="td11">   
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">101</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-19-1"  
class="td11"> 
Influenza Life Cycle
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-19-2"  
class="td11">      
112
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-19-3"  
class="td11">     
57
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-19-4"  
class="td11">   
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">100</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-20-1"  
class="td11"> 
GPCR downstream signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-20-2"  
class="td11">      
472
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-20-3"  
class="td11">    
116
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-20-4"  
class="td11">   
1<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-86-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-21-1"  
class="td11"> 
Hemostasis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-21-2"  
class="td11">      
422
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-21-3"  
class="td11">    
105
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-86-21-4"  
class="td11">   
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">78</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-86-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-86-22-1"  
class="td11">                                                             </td></tr></table>
<!--l. 210--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Gene set over-representation analysis (hypergeometric test) for Reactome pathways in </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">partners for </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">.</span></p>
</p>
</div><hr class="endfloat" />
<!--l. 215--><p class="indent" >
The genes and pathways identified in Tables <a 
href="#x1-1720011">C.1<!--tex4ht:ref: tab:gene_mtSL --></a> and <a 
href="#x1-1720022">C.2<!--tex4ht:ref: tab:pathway_mtSL --></a> were derived from comparing the expression profiles of potential partners to the mutation status of <span class="cmti-12">CDH1 </span>(as shown in Figure <a 
href="#x1-740022">3.2<!--tex4ht:ref: fig:SLIPT_Method_mtSL --></a>). The following analysis was limited to the samples for which both expression and somatic mutation data were available from <a 
href="#glo:TCGA">TCGA</a>. <a 
 id="x1-172003r399"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">C.2 </span> <a 
 id="x1-1730002"></a>Synthetic Lethal Expression Profiles
</h3>
<!--l. 222--><p class="noindent" >
<p>Similar to the analysis of synthetic lethal partners against low <span class="cmti-12">CDH1 </span>expression in <a 
href="#x1-990002">4.1.2<!--tex4ht:ref: chapt3:exprSL_clusters --></a>, the partners detected from <span class="cmti-12">CDH1 </span>mutation were also examined for their expression profiles and the pathway composition of gene clusters. Hierachical clustering was performed on <a 
href="#glo:mtSLIPT">mtSLIPT</a> partners for <span class="cmti-12">CDH1 </span>as showing in Figure <a 
href="#x1-1730011">C.1<!--tex4ht:ref: fig:slipt_expr_mtSL --></a>. Over-representation for Reactome pathways for each of the gene clusters identified is given in Table <a 
href="#x1-1730023">C.3<!--tex4ht:ref: tab:pathway_clusters_mtSL --></a>.</p>
</p><!--l. 224--><p class="indent" >
<a 
 id="x1-1730011"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<img 
src="CDH1_Heatmaps_Genes_Split_By_CDH1_z-trans_mtSL_cordistx_Pub.png" alt="PIC"  
 /> <br />
<div class="caption">
<p><span class="id">Figure C.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethal expression profiles of analysed samples.</span></strong> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">expression profile heatmap (correlation distance) of all samples (separated by </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">somatic</span> <span class="cmr-10x-x-109">mutation status) analysed in </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">breast cancer dataset for gene expression of 3743</span> <span class="cmr-10x-x-109">candidate partners of </span><a 
href="#glo:E-cadherin"><span class="cmr-10x-x-109">E-cadherin</span></a> <span class="cmr-10x-x-109">(</span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">) from </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">prediction (with significant </span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a> <span class="cmr-10x-x-109">adjusted </span><span class="cmmi-10x-x-109">p &lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05). Deeply clustered, inter-correlated genes form several main groups, each</span> <span class="cmr-10x-x-109">containing genes that were SL candidates or toxic in an </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screen </span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">Telford </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">2015</span></a><span class="cmr-10x-x-109">).</span> <span class="cmr-10x-x-109">Clusters had different sample groups highly expressing the synthetic lethal candidates in</span> <span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutant samples and often lowly expressing </span><span class="cmti-10x-x-109">CDH1</span><a 
href="#glo:wild-type"><span class="cmr-10x-x-109">wild-type</span></a> <span class="cmr-10x-x-109">samples (which were</span> <span class="cmr-10x-x-109">not tested for), although many of the </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutant samples had among the lowest </span><span class="cmti-10x-x-109">CDH1</span> <span class="cmr-10x-x-109">expression. In contrast to the expression analysis the (predominantly </span><span class="cmti-10x-x-109">CDH1</span><a 
href="#glo:wild-type"><span class="cmr-10x-x-109">wild-type</span></a><span class="cmr-10x-x-109">) basal</span> <span class="cmr-10x-x-109">subtype and </span><a 
href="#glo:ER"><span class="cmr-10x-x-109">ER</span></a> <span class="cmr-10x-x-109">negative samples have depleted expression among most candidate synthetic</span> <span class="cmr-10x-x-109">lethal partners. </span></span></p>
</div>
<!--tex4ht:label?: x1-1730011 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 240--><p class="indent" >
<a 
 id="x1-1730023"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table C.3: </span><span class="content">Pathways for clusters of <span class="cmti-12">CDH1 </span>partners from mtSLIPT</span>
</div><!--tex4ht:label?: x1-1730023 -->
<div class="tabular">
<table id="TBL-87" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-87-1g"><col 
id="TBL-87-1" /><col 
id="TBL-87-2" /><col 
id="TBL-87-3" /><col 
id="TBL-87-4" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-87-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-1-1"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 1</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-1-2"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-1-3"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-1-4"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">p-value (</span><a 
href="#glo:FDR"><span class="cmbx-12x-x-120">FDR</span></a><span class="cmbx-12x-x-120">)</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-87-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-2-1"  
class="td11"> 
Olfactory Signalling Pathway
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-2-2"  
class="td11">        
57
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-2-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-2-4"  
class="td11">     
7<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-3-1"  
class="td11"> 
Assembly of the primary cilium
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-3-2"  
class="td11">       
149
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-3-3"  
class="td11">        
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-3-4"  
class="td11">     
8<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-4-1"  
class="td11"> 
Sphingolipid metabolism
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-4-2"  
class="td11">        
62
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-4-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-4-4"  
class="td11">     
9<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-5-1"  
class="td11"> 
Signalling by ERBB4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-5-2"  
class="td11">       
133
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-5-3"  
class="td11">        
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-5-4"  
class="td11">     
5<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-6-1"  
class="td11"> 
PI3K Cascade
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-6-2"  
class="td11">        
65
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-6-3"  
class="td11">         
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-6-4"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-7-1"  
class="td11"> 
Circadian Clock
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-7-2"  
class="td11">        
33
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-7-3"  
class="td11">         
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-7-4"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-8-1"  
class="td11"> 
Nuclear signalling by ERBB4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-8-2"  
class="td11">        
34
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-8-3"  
class="td11">         
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-8-4"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-9-1"  
class="td11"> 
Intraflagellar transport
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-9-2"  
class="td11">        
35
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-9-3"  
class="td11">         
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-9-4"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-10-1"  
class="td11"> 
PI3K events in ERBB4 signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-10-2"  
class="td11">        
87
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-10-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-10-4"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-11-1"  
class="td11"> 
PIP3 activates AKT signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-11-2"  
class="td11">        
87
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-11-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-11-4"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-12-1"  
class="td11"> 
PI3K events in ERBB2 signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-12-2"  
class="td11">        
87
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-12-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-12-4"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-13-1"  
class="td11"> 
PI-3K cascade:FGFR1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-13-2"  
class="td11">        
87
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-13-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-13-4"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-14-1"  
class="td11"> 
PI-3K cascade:FGFR2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-14-2"  
class="td11">        
87
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-14-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-14-4"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-15-1"  
class="td11"> 
PI-3K cascade:FGFR3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-15-2"  
class="td11">        
87
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-15-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-15-4"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-16-1"  
class="td11"> 
PI-3K cascade:FGFR4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-16-2"  
class="td11">        
87
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-16-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-16-4"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-17-1"  
class="td11"> 
Deadenylation of <a 
href="#glo:mRNA">mRNA</a>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-17-2"  
class="td11">        
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-17-3"  
class="td11">         
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-17-4"  
class="td11">     
5<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-18-1"  
class="td11"> 
PI3K/AKT activation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-18-2"  
class="td11">        
90
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-18-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-18-4"  
class="td11">     
5<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-19-1"  
class="td11"> 
Cargo trafficking to the periciliary membrane
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-19-2"  
class="td11">        
38
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-19-3"  
class="td11">         
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-19-4"  
class="td11">     
5<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-87-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-20-1"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 2</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-20-2"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-20-3"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-20-4"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">p-value (</span><a 
href="#glo:FDR"><span class="cmbx-12x-x-120">FDR</span></a><span class="cmbx-12x-x-120">)</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-87-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-21-1"  
class="td11"> 
G<sub><span class="cmmi-8">αs</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-21-2"  
class="td11">        
83
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-21-3"  
class="td11">        
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-21-4"  
class="td11">     
5<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-22-1"  
class="td11"> 
Extracellular matrix organization
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-22-2"  
class="td11">       
238
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-22-3"  
class="td11">        
30
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-22-4"  
class="td11">     
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-23-1"  
class="td11"> 
Hemostasis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-23-2"  
class="td11">       
422
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-23-3"  
class="td11">        
46
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-23-4"  
class="td11">     
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-24-1"  
class="td11"> 
Aquaporin-mediated transport
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-24-2"  
class="td11">        
32
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-24-3"  
class="td11">         
9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-24-4"  
class="td11">     
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-25-1"  
class="td11"> 
Transcriptional regulation of white adipocyte differentiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-25-2"  
class="td11">        
56
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-25-3"  
class="td11">        
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-25-4"  
class="td11">     
1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-26-1"  
class="td11"> 
Degradation of the extracellular matrix
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-26-2"  
class="td11">       
102
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-26-3"  
class="td11">        
15
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-26-4"  
class="td11">     
1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-27-1"  
class="td11"> 
Integration of energy metabolism
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-27-2"  
class="td11">        
84
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-27-3"  
class="td11">        
13
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-27-4"  
class="td11">     
8<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-28-1"  
class="td11"> 
GPCR downstream signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-28-2"  
class="td11">       
472
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-28-3"  
class="td11">        
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-28-4"  
class="td11">     
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-29-1"  
class="td11"> 
G<sub><span class="cmmi-8">αz</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-29-2"  
class="td11">        
15
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-29-3"  
class="td11">         
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-29-4"  
class="td11">     
5<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-30-1"  
class="td11"> 
Molecules associated with elastic fibres
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-30-2"  
class="td11">        
33
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-30-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-30-4"  
class="td11">     
5<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-31-1"  
class="td11"> 
Phase 1 - Functionalization of compounds
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-31-2"  
class="td11">        
67
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-31-3"  
class="td11">        
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-31-4"  
class="td11">     
5<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-32-1"  
class="td11"> 
Platelet activation, signalling and aggregation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-32-2"  
class="td11">       
179
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-32-3"  
class="td11">        
20
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-32-4"  
class="td11">     
5<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-33-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-33-1"  
class="td11"> 
Vasopressin regulates renal water homeostasis via Aquaporins
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-33-2"  
class="td11">        
24
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-33-3"  
class="td11">         
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-33-4"  
class="td11">     
6<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-34-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-34-1"  
class="td11"> 
Elastic fibre formation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-34-2"  
class="td11">        
37
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-34-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-34-4"  
class="td11">     
<span class="cmmi-12">.</span>03 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-35-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-35-1"  
class="td11"> 
Calmodulin induced events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-35-2"  
class="td11">        
27
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-35-3"  
class="td11">         
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-35-4"  
class="td11">     
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-36-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-36-1"  
class="td11"> 
CaM pathway
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-36-2"  
class="td11">        
27
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-36-3"  
class="td11">         
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-36-4"  
class="td11">     
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-37-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-37-1"  
class="td11"> 
cGMP effects
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-37-2"  
class="td11">        
18
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-37-3"  
class="td11">         
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-37-4"  
class="td11">     
3<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-38-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-38-1"  
class="td11"> 
G<sub><span class="cmmi-8">αi</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-38-2"  
class="td11">       
167
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-38-3"  
class="td11">        
18
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-38-4"  
class="td11">     
6<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-87-39-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-39-1"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 3</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-39-2"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-39-3"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-39-4"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">p-value (</span><a 
href="#glo:FDR"><span class="cmbx-12x-x-120">FDR</span></a><span class="cmbx-12x-x-120">)</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-87-40-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-40-1"  
class="td11"> 
Eukaryotic Translation Elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-40-2"  
class="td11">        
86
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-40-3"  
class="td11">        
55
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-40-4"  
class="td11">    
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">112</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-41-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-41-1"  
class="td11"> 
Peptide chain elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-41-2"  
class="td11">        
83
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-41-3"  
class="td11">        
54
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-41-4"  
class="td11">    
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">112</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-42-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-42-1"  
class="td11"> 
Viral <a 
href="#glo:mRNA">mRNA</a> Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-42-2"  
class="td11">        
81
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-42-3"  
class="td11">        
53
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-42-4"  
class="td11">    
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">111</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-43-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-43-1"  
class="td11"> 
Eukaryotic Translation Termination
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-43-2"  
class="td11">        
83
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-43-3"  
class="td11">        
53
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-43-4"  
class="td11">    
7<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">110</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-44-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-44-1"  
class="td11"> 
Nonsense Mediated Decay independent of the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-44-2"  
class="td11">        
88
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-44-3"  
class="td11">        
54
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-44-4"  
class="td11">    
1<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">108</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-45-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-45-1"  
class="td11"> 
Formation of a pool of free 40S subunits
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-45-2"  
class="td11">        
93
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-45-3"  
class="td11">        
53
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-45-4"  
class="td11">    
4<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">102</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-46-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-46-1"  
class="td11"> 
Nonsense-Mediated Decay
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-46-2"  
class="td11">       
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-46-3"  
class="td11">        
54
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-46-4"  
class="td11">     
3<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">98</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-47-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-47-1"  
class="td11"> 
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-47-2"  
class="td11">       
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-47-3"  
class="td11">        
54
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-47-4"  
class="td11">     
3<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">98</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-48-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-48-1"  
class="td11"> 
L13a-mediated translational silencing of Ceruloplasmin expression
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-48-2"  
class="td11">       
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-48-3"  
class="td11">        
53
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-48-4"  
class="td11">     
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">95</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-49-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-49-1"  
class="td11"> 
3’ -UTR-mediated translational regulation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-49-2"  
class="td11">       
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-49-3"  
class="td11">        
53
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-49-4"  
class="td11">     
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">95</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-50-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-50-1"  
class="td11"> 
SRP-dependent cotranslational protein targeting to membrane
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-50-2"  
class="td11">       
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-50-3"  
class="td11">        
53
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-50-4"  
class="td11">     
4<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">95</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-51-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-51-1"  
class="td11"> 
GTP hydrolysis and joining of the 60S ribosomal subunit
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-51-2"  
class="td11">       
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-51-3"  
class="td11">        
53
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-51-4"  
class="td11">     
4<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">95</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-52-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-52-1"  
class="td11"> 
Influenza Viral <a 
href="#glo:RNA">RNA</a> Transcription and Replication
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-52-2"  
class="td11">       
108
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-52-3"  
class="td11">        
53
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-52-4"  
class="td11">     
9<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">93</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-53-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-53-1"  
class="td11"> 
Eukaryotic Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-53-2"  
class="td11">       
111
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-53-3"  
class="td11">        
53
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-53-4"  
class="td11">     
4<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">91</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-54-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-54-1"  
class="td11"> 
Cap-dependent Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-54-2"  
class="td11">       
111
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-54-3"  
class="td11">        
53
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-54-4"  
class="td11">     
4<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">91</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-55-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-55-1"  
class="td11"> 
Influenza Life Cycle
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-55-2"  
class="td11">       
112
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-55-3"  
class="td11">        
53
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-55-4"  
class="td11">     
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-56-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-56-1"  
class="td11"> 
Influenza Infection
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-56-2"  
class="td11">       
117
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-56-3"  
class="td11">        
53
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-56-4"  
class="td11">     
6<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">88</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-57-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-57-1"  
class="td11"> 
Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-57-2"  
class="td11">       
141
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-57-3"  
class="td11">        
55
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-57-4"  
class="td11">     
3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">81</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-87-58-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-58-1"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 4</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-58-2"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-58-3"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-58-4"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">p-value (</span><a 
href="#glo:FDR"><span class="cmbx-12x-x-120">FDR</span></a><span class="cmbx-12x-x-120">)</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-87-59-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-59-1"  
class="td11"> 
ECM proteoglycans
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-59-2"  
class="td11">        
66
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-59-3"  
class="td11">        
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-59-4"  
class="td11">     
2<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-60-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-60-1"  
class="td11"> 
deactivation of the beta-catenin transactivating complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-60-2"  
class="td11">        
38
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-60-3"  
class="td11">         
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-60-4"  
class="td11">     
5<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-61-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-61-1"  
class="td11"> 
Arachidonic acid metabolism
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-61-2"  
class="td11">        
41
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-61-3"  
class="td11">         
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-61-4"  
class="td11">     
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-62-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-62-1"  
class="td11"> 
G<sub><span class="cmmi-8">αq</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-62-2"  
class="td11">       
149
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-62-3"  
class="td11">        
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-62-4"  
class="td11">     
4<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-63-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-63-1"  
class="td11"> 
HS-GAG degradation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-63-2"  
class="td11">        
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-63-3"  
class="td11">         
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-63-4"  
class="td11">     
4<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-64-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-64-1"  
class="td11"> 
Uptake and actions of bacterial toxins
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-64-2"  
class="td11">        
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-64-3"  
class="td11">         
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-64-4"  
class="td11">     
6<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-65-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-65-1"  
class="td11"> 
Gastrin-CREB signalling pathway via PKC and MAPK
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-65-2"  
class="td11">       
170
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-65-3"  
class="td11">        
15
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-65-4"  
class="td11">     
6<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-66-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-66-1"  
class="td11"> 
<a 
href="#glo:RNA">RNA</a> Polymerase I, <a 
href="#glo:RNA">RNA</a> Polymerase III, and Mitochondrial Transcription <span id="textcolor30">ll</span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-66-2"  
class="td11">        
64
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-66-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-66-4"  
class="td11">     
6<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-67-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-67-1"  
class="td11"> 
Non-integrin membrane-ECM interactions
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-67-2"  
class="td11">        
53
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-67-3"  
class="td11">         
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-67-4"  
class="td11">     
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-68-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-68-1"  
class="td11"> 
Syndecan interactions
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-68-2"  
class="td11">        
25
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-68-3"  
class="td11">         
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-68-4"  
class="td11">     
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-69-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-69-1"  
class="td11"> 
NOTCH1 Intracellular Domain Regulates Transcription
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-69-2"  
class="td11">        
40
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-69-3"  
class="td11">         
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-69-4"  
class="td11">     
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-70-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-70-1"  
class="td11"> 
Synthesis of Leukotrienes and Eoxins
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-70-2"  
class="td11">        
15
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-70-3"  
class="td11">         
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-70-4"  
class="td11">     
3<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-71-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-71-1"  
class="td11"> 
Signalling by NOTCH1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-71-2"  
class="td11">        
59
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-71-3"  
class="td11">         
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-71-4"  
class="td11">     
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-72-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-72-1"  
class="td11"> 
Regulation of insulin secretion
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-72-2"  
class="td11">        
44
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-72-3"  
class="td11">         
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-72-4"  
class="td11">     
6<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-73-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-73-1"  
class="td11"> 
Metabolism of lipids and lipoproteins
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-73-2"  
class="td11">       
471
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-73-3"  
class="td11">        
37
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-73-4"  
class="td11">     
8<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-74-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-74-1"  
class="td11"> 
Signalling by NOTCH
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-74-2"  
class="td11">        
80
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-74-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-74-4"  
class="td11">     
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-75-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-75-1"  
class="td11"> 
Platelet activation, signalling and aggregation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-75-2"  
class="td11">       
179
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-75-3"  
class="td11">        
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-75-4"  
class="td11">     
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-87-76-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-76-1"  
class="td11"> 
Recruitment of mitotic centrosome proteins and complexes
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-76-2"  
class="td11">        
64
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-76-3"  
class="td11">         
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-87-76-4"  
class="td11">     
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-87-77-"><td  style="white-space:nowrap; text-align:left;" id="TBL-87-77-1"  
class="td11">                                                                  </td></tr></table>
<!--l. 432--><p class="noindent" ></div>
<p>Pathway over-representation analysis for Reactome pathways with the number of genes in each pathway (Pathway Size), number of genes within the pathway identified (Cluster Genes), and the pathway over-representation p-value (adjusted by <a 
href="#glo:FDR">FDR</a>) from the hypergeometric test.</p>
</p>
</div><hr class="endfloat" />
                                                                                    

<a 
 id="x1-173003r402"></a>
<h3 class="sectionHead">
<span class="titlemark">C.3 </span> <a 
 id="x1-1740003"></a>Comparison to Primary Screen
</h3>
<!--l. 439--><p class="noindent" >
The mutation synthetic lethal partners with <span class="cmti-12">CDH1 </span>were also compared to <a 
href="#glo:siRNA">siRNA</a> primary screen data (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), as performed in Section <a 
href="#x1-1020001">4.2.1<!--tex4ht:ref: chapt3:primary_screen --></a>. These were expected to be more concordant with the experimental results performed on a null mutant, however this was not the case at the gene level: less genes overlapped with experimental candidates in Figure <a 
href="#x1-1740012">C.2<!--tex4ht:ref: fig:Venn_allgenes_mtSL --></a>. This discrepancy was may be due to lower sample size for mutations in <a 
href="#glo:TCGA">TCGA</a> data or lower frequency (expected value) of <span class="cmti-12">CDH1 </span>mutations compared to low expression.
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                    

<p><a 
 id="x1-1740012"></a></p>
<!--l. 444--><p class="noindent" >
<img 
src="Venn_mtSL_siRNA_allgenes_reduced_Pub.png" alt="PIC"  
 /> <br />
</p><div class="caption" 
>
<span class="id">Figure C.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Comparison of </span><a 
href="#glo:mtSLIPT"><span class="cmbx-10x-x-109">mtSLIPT</span></a> <span class="cmbx-10x-x-109">to </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a><span class="cmbx-10x-x-109">.</span></strong> <span class="cmr-10x-x-109">Testing the overlap of gene candidates</span> <span class="cmr-10x-x-109">for </span><a 
href="#glo:E-cadherin"><span class="cmr-10x-x-109">E-cadherin</span></a> <span class="cmr-10x-x-109">synthetic lethal partners between computational (SLIPT) and experimental</span> <span class="cmr-10x-x-109">screening (siRNA) approaches. The </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test suggests that the overlap is no more than would</span> <span class="cmr-10x-x-109">be expected by chance (</span><span class="cmmi-10x-x-109">p </span><span class="cmr-10x-x-109">= 0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">281). </span></span>
</div><!--tex4ht:label?: x1-1740012 -->
                                                                                    

</div><hr class="endfigure" />
<!--l. 455--><p class="indent" >
<p>Despite a lower sample size (and low number of a predicted partners) for mutation analysis, the pathway composition (Tables <a 
href="#x1-1720022">C.2<!--tex4ht:ref: tab:pathway_mtSL --></a> and <a 
href="#x1-1740024">C.4<!--tex4ht:ref: tab:Venn_over-representation_mtSL --></a>) was similar to expression analysis, as described in Section <a 
href="#x1-1060005">4.2.5<!--tex4ht:ref: chapt3:compare_pathway --></a>. In particular, the resampling analysis (Section <a 
href="#x1-1750001">C.3.1<!--tex4ht:ref: appendix:compare_pathway_perm_mtSL --></a>) supported many of the results of expression analysis (Section <a 
href="#x1-1070001">4.2.5.1<!--tex4ht:ref: chapt3:compare_pathway_perm --></a>). Tables <a 
href="#x1-1750015">C.5<!--tex4ht:ref: tab:pathway_perm_mtSL --></a> and <a 
href="#x1-1750026">C.6<!--tex4ht:ref: tab:pathway_perm_overlap_mtSL --></a> detected many of the same or functionally-related pathways.</p>
</p><!--l. 458--><p class="indent" >
<a 
 id="x1-1740024"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table C.4: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from <a 
href="#glo:mtSLIPT">mtSLIPT</a> and <a 
href="#glo:siRNA">siRNA</a></span>
</div><!--tex4ht:label?: x1-1740024 -->
<!--tex4ht:inline--><div class="tabular"><table id="TBL-88" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-88-1g"><col 
id="TBL-88-1" /><col 
id="TBL-88-2" /><col 
id="TBL-88-3" /><col 
id="TBL-88-4" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-88-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-1-1"  
class="td11"> 
<span class="cmbx-12">Predicted only by SLIPT (2901 genes) </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-1-2"  
class="td11"> 
Pathway Size
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-1-3"  
class="td11"> 
Genes Identified
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-1-4"  
class="td11"> 
p-value (<a 
href="#glo:FDR">FDR</a>)
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-88-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-2-1"  
class="td11"> 
Eukaryotic Translation Elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-2-2"  
class="td11">      
87
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-2-3"  
class="td11">       
57
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-2-4"  
class="td11">  
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">120</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-3-1"  
class="td11"> 
Peptide chain elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-3-2"  
class="td11">      
84
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-3-3"  
class="td11">       
56
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-3-4"  
class="td11">  
3<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">120</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-4-1"  
class="td11"> 
Eukaryotic Translation Termination
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-4-2"  
class="td11">      
84
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-4-3"  
class="td11">       
55
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-4-4"  
class="td11">  
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">117</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-5-1"  
class="td11"> 
Viral <a 
href="#glo:mRNA">mRNA</a> Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-5-2"  
class="td11">      
82
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-5-3"  
class="td11">       
54
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-5-4"  
class="td11">  
4<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">116</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-6-1"  
class="td11"> 
Nonsense Mediated Decay independent of the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-6-2"  
class="td11">      
89
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-6-3"  
class="td11">       
55
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-6-4"  
class="td11">  
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">113</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-7-1"  
class="td11"> 
Formation of a pool of free 40S subunits
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-7-2"  
class="td11">      
94
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-7-3"  
class="td11">       
55
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-7-4"  
class="td11">  
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">109</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-8-1"  
class="td11"> 
Nonsense-Mediated Decay
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-8-2"  
class="td11">     
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-8-3"  
class="td11">       
57
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-8-4"  
class="td11">  
8<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">108</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-9-1"  
class="td11"> 
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-9-2"  
class="td11">     
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-9-3"  
class="td11">       
57
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-9-4"  
class="td11">  
8<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">108</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-10-1"  
class="td11"> 
L13a-mediated translational silencing of Ceruloplasmin expression
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-10-2"  
class="td11">     
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-10-3"  
class="td11">       
56
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-10-4"  
class="td11">  
3<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">105</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-11-1"  
class="td11"> 
3’ -UTR-mediated translational regulation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-11-2"  
class="td11">     
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-11-3"  
class="td11">       
56
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-11-4"  
class="td11">  
3<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">105</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-12-1"  
class="td11"> 
GTP hydrolysis and joining of the 60S ribosomal subunit
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-12-2"  
class="td11">     
105
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-12-3"  
class="td11">       
56
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-12-4"  
class="td11">  
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">104</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-13-1"  
class="td11"> 
Eukaryotic Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-13-2"  
class="td11">     
112
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-13-3"  
class="td11">       
56
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-13-4"  
class="td11">  
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">100</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-14-1"  
class="td11"> 
Cap-dependent Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-14-2"  
class="td11">     
112
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-14-3"  
class="td11">       
56
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-14-4"  
class="td11">  
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">100</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-15-1"  
class="td11"> 
SRP-dependent cotranslational protein targeting to membrane
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-15-2"  
class="td11">     
105
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-15-3"  
class="td11">       
54
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-15-4"  
class="td11">  
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">99</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-16-1"  
class="td11"> 
Influenza Viral <a 
href="#glo:RNA">RNA</a> Transcription and Replication
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-16-2"  
class="td11">     
109
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-16-3"  
class="td11">       
54
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-16-4"  
class="td11">  
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-17-1"  
class="td11"> 
Influenza Life Cycle
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-17-2"  
class="td11">     
113
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-17-3"  
class="td11">       
54
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-17-4"  
class="td11">  
9<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">95</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-18-1"  
class="td11"> 
Influenza Infection
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-18-2"  
class="td11">     
118
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-18-3"  
class="td11">       
55
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-18-4"  
class="td11">  
1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">94</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-19-1"  
class="td11"> 
Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-19-2"  
class="td11">     
142
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-19-3"  
class="td11">       
60
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-19-4"  
class="td11">  
3<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">94</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-20-1"  
class="td11"> 
Infectious disease
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-20-2"  
class="td11">     
349
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-20-3"  
class="td11">       
77
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-20-4"  
class="td11">  
5<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">62</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-21-1"  
class="td11"> 
Extracellular matrix organization
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-21-2"  
class="td11">     
241
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-21-3"  
class="td11">       
54
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-21-4"  
class="td11">  
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">52</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-88-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-22-1"  
class="td11">                                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-23-1"  
class="td11"> 
<span class="cmbx-12">Detected only by </span><a 
href="#glo:siRNA"><span class="cmbx-12">siRNA</span></a> <span class="cmbx-12">screen (1752 genes) </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-23-2"  
class="td11"> 
Pathway Size
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-23-3"  
class="td11"> 
Genes Identified
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-23-4"  
class="td11"> 
p-value (<a 
href="#glo:FDR">FDR</a>)
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-88-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-24-1"  
class="td11"> 
Class A/1 (Rhodopsin-like receptors)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-24-2"  
class="td11">     
282
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-24-3"  
class="td11">       
69
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-24-4"  
class="td11">  
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">59</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-25-1"  
class="td11"> 
GPCR ligand binding
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-25-2"  
class="td11">     
363
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-25-3"  
class="td11">       
78
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-25-4"  
class="td11">  
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">54</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-26-1"  
class="td11"> 
Peptide ligand-binding receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-26-2"  
class="td11">     
175
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-26-3"  
class="td11">       
41
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-26-4"  
class="td11">  
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">42</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-27-1"  
class="td11"> 
<span class="cmmi-12">G</span><sub><span class="cmmi-8">αi</span></sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-27-2"  
class="td11">     
184
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-27-3"  
class="td11">       
41
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-27-4"  
class="td11">  
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">40</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-28-1"  
class="td11"> 
Gastrin-CREB signalling pathway via PKC and MAPK
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-28-2"  
class="td11">     
180
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-28-3"  
class="td11">       
37
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-28-4"  
class="td11">  
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">35</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-29-1"  
class="td11"> 
<span class="cmmi-12">G</span><sub><span class="cmmi-8">αq</span></sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-29-2"  
class="td11">     
159
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-29-3"  
class="td11">       
34
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-29-4"  
class="td11">  
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">35</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-30-1"  
class="td11"> 
DAP12 interactions
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-30-2"  
class="td11">     
159
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-30-3"  
class="td11">       
27
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-30-4"  
class="td11">  
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-31-1"  
class="td11"> 
VEGFA-VEGFR2 Pathway
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-31-2"  
class="td11">      
91
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-31-3"  
class="td11">       
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-31-4"  
class="td11">  
1<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">23</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-32-1"  
class="td11"> 
Downstream signal transduction
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-32-2"  
class="td11">     
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-32-3"  
class="td11">       
24
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-32-4"  
class="td11">  
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-33-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-33-1"  
class="td11"> 
Signalling by VEGF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-33-2"  
class="td11">      
99
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-33-3"  
class="td11">       
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-33-4"  
class="td11">  
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-34-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-34-1"  
class="td11"> 
DAP12 signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-34-2"  
class="td11">     
149
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-34-3"  
class="td11">       
24
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-34-4"  
class="td11">  
4<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-35-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-35-1"  
class="td11"> 
Organelle biogenesis and maintenance
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-35-2"  
class="td11">     
264
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-35-3"  
class="td11">       
34
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-35-4"  
class="td11">  
4<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-36-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-36-1"  
class="td11"> 
Downstream signalling of activated FGFR1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-36-2"  
class="td11">     
134
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-36-3"  
class="td11">       
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-36-4"  
class="td11">  
4<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-37-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-37-1"  
class="td11"> 
Downstream signalling of activated FGFR2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-37-2"  
class="td11">     
134
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-37-3"  
class="td11">       
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-37-4"  
class="td11">  
4<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-38-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-38-1"  
class="td11"> 
Downstream signalling of activated FGFR3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-38-2"  
class="td11">     
134
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-38-3"  
class="td11">       
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-38-4"  
class="td11">  
4<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-39-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-39-1"  
class="td11"> 
Downstream signalling of activated FGFR4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-39-2"  
class="td11">     
134
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-39-3"  
class="td11">       
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-39-4"  
class="td11">  
4<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-40-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-40-1"  
class="td11"> 
Signalling by ERBB2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-40-2"  
class="td11">     
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-40-3"  
class="td11">       
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-40-4"  
class="td11">  
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-41-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-41-1"  
class="td11"> 
Signalling by FGFR
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-41-2"  
class="td11">     
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-41-3"  
class="td11">       
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-41-4"  
class="td11">  
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-42-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-42-1"  
class="td11"> 
Signalling by FGFR1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-42-2"  
class="td11">     
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-42-3"  
class="td11">       
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-42-4"  
class="td11">  
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-43-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-43-1"  
class="td11"> 
Signalling by FGFR2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-43-2"  
class="td11">     
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-43-3"  
class="td11">       
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-43-4"  
class="td11">  
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-88-44-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-44-1"  
class="td11">                                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-45-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-45-1"  
class="td11"> 
<span class="cmbx-12">Intersection of SLIPT and </span><a 
href="#glo:siRNA"><span class="cmbx-12">siRNA</span></a> <span class="cmbx-12">screen (450 genes) </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-45-2"  
class="td11"> 
Pathway Size
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-45-3"  
class="td11"> 
Genes Identified
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-45-4"  
class="td11"> 
p-value (<a 
href="#glo:FDR">FDR</a>)
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-88-46-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-46-1"  
class="td11"> 
HS-GAG degradation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-46-2"  
class="td11">      
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-46-3"  
class="td11">       
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-46-4"  
class="td11">   
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-47-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-47-1"  
class="td11"> 
Retinoid metabolism and transport
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-47-2"  
class="td11">      
39
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-47-3"  
class="td11">       
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-47-4"  
class="td11">   
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-48-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-48-1"  
class="td11"> 
Platelet activation, signalling and aggregation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-48-2"  
class="td11">     
186
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-48-3"  
class="td11">       
13
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-48-4"  
class="td11">   
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-49-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-49-1"  
class="td11"> 
Signalling by NOTCH4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-49-2"  
class="td11">      
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-49-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-49-4"  
class="td11">   
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-50-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-50-1"  
class="td11"> 
<span class="cmmi-12">G</span><sub><span class="cmmi-8">αs</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-50-2"  
class="td11">     
100
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-50-3"  
class="td11">       
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-50-4"  
class="td11">   
5<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-51-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-51-1"  
class="td11"> 
Defective EXT2 causes exostoses 2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-51-2"  
class="td11">      
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-51-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-51-4"  
class="td11">   
5<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-52-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-52-1"  
class="td11"> 
Defective EXT1 causes exostoses 1, TRPS2 and CHDS
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-52-2"  
class="td11">      
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-52-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-52-4"  
class="td11">   
5<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-53-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-53-1"  
class="td11"> 
Class A/1 (Rhodopsin-like receptors)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-53-2"  
class="td11">     
289
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-53-3"  
class="td11">       
18
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-53-4"  
class="td11">   
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-54-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-54-1"  
class="td11"> 
Signalling by PDGF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-54-2"  
class="td11">     
173
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-54-3"  
class="td11">       
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-54-4"  
class="td11">   
2<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-55-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-55-1"  
class="td11"> 
Circadian Clock
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-55-2"  
class="td11">      
34
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-55-3"  
class="td11">       
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-55-4"  
class="td11">   
2<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-56-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-56-1"  
class="td11"> 
Signalling by ERBB4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-56-2"  
class="td11">     
139
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-56-3"  
class="td11">       
9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-56-4"  
class="td11">   
4<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-57-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-57-1"  
class="td11"> 
Role of LAT2/NTAL/LAB on calcium mobilization
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-57-2"  
class="td11">      
99
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-57-3"  
class="td11">       
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-57-4"  
class="td11">   
4<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-58-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-58-1"  
class="td11"> 
Peptide ligand-binding receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-58-2"  
class="td11">     
181
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-58-3"  
class="td11">       
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-58-4"  
class="td11">   
4<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-59-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-59-1"  
class="td11"> 
Defective B4GALT7 causes EDS, progeroid type
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-59-2"  
class="td11">      
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-59-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-59-4"  
class="td11">   
4<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-60-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-60-1"  
class="td11"> 
Defective B3GAT3 causes JDSSDHD
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-60-2"  
class="td11">      
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-60-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-60-4"  
class="td11">   
4<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-61-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-61-1"  
class="td11"> 
Signalling by NOTCH
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-61-2"  
class="td11">      
80
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-61-3"  
class="td11">       
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-61-4"  
class="td11">   
4<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-62-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-62-1"  
class="td11"> 
<span class="cmmi-12">G</span><sub><span class="cmmi-8">αq</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-62-2"  
class="td11">     
164
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-62-3"  
class="td11">       
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-62-4"  
class="td11">   
5<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-63-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-63-1"  
class="td11"> 
Response to elevated platelet cytosolic Ca<sup><span class="cmr-8">2+</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-63-2"  
class="td11">      
84
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-63-3"  
class="td11">       
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-63-4"  
class="td11">   
7<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-64-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-64-1"  
class="td11"> 
Signalling by ERBB2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-64-2"  
class="td11">     
148
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-64-3"  
class="td11">       
9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-64-4"  
class="td11">   
7<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-88-65-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-65-1"  
class="td11"> 
Signalling by SCF-KIT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-65-2"  
class="td11">     
129
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-65-3"  
class="td11">       
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-88-65-4"  
class="td11">   
8<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-88-66-"><td  style="white-space:nowrap; text-align:left;" id="TBL-88-66-1"  
class="td11">                                                             </td></tr></table>                                                         </div>
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<a 
 id="x1-174003r362"></a>
<h4 class="subsectionHead">
<span class="titlemark">C.3.1 </span> <a 
 id="x1-1750001"></a>Resampling Analysis
</h4>
                                                                                    

<!--l. 606--><p class="indent" >
<a 
 id="x1-1750015"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table C.5: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT</span>
</div><!--tex4ht:label?: x1-1750015 -->
<div class="tabular">
<table id="TBL-89" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-89-1g"><col 
id="TBL-89-1" /><col 
id="TBL-89-2" /><col 
id="TBL-89-3" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-89-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-1-1"  
class="td11"> 
<span class="cmbx-12">Reactome Pathway </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-1-2"  
class="td11"> 
Over-representation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-1-3"  
class="td11">  
Permutation
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-89-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-2-1"  
class="td11"> 
<strong><span class="cmbx-12">Eukaryotic Translation Elongation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-2-2"  
class="td11">    
3<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">128</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-2-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-3-1"  
class="td11"> 
<strong><span class="cmbx-12">Peptide chain elongation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-3-2"  
class="td11">    
3<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">128</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-3-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-4-1"  
class="td11"> 
<strong><span class="cmbx-12">Eukaryotic Translation Termination</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-4-2"  
class="td11">    
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">125</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-4-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-5-1"  
class="td11"> 
<strong><span class="cmbx-12">Viral </span><a 
href="#glo:mRNA"><span class="cmbx-12">mRNA</span></a> <span class="cmbx-12">Translation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-5-2"  
class="td11">    
4<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">124</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-5-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-6-1"  
class="td11"> 
<strong><span class="cmbx-12">Nonsense Mediated Decay independent of the Exon Junction Complex</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-6-2"  
class="td11">    
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">123</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-6-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-7-1"  
class="td11"> 
<strong><span class="cmbx-12">Nonsense-Mediated Decay</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-7-2"  
class="td11">    
8<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">117</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-7-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-8-1"  
class="td11"> 
<strong><span class="cmbx-12">Nonsense Mediated Decay enhanced by the Exon Junction Complex</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-8-2"  
class="td11">    
8<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">117</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-8-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-9-1"  
class="td11"> 
<strong><span class="cmbx-12">Formation of a pool of free 40S subunits</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-9-2"  
class="td11">    
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">116</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-9-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-10-1"  
class="td11"> 
<strong><span class="cmbx-12">L13a-mediated translational silencing of Ceruloplasmin expression</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-10-2"  
class="td11">    
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">111</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-10-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-11-1"  
class="td11"> 
<strong><span class="cmbx-12">3’ -UTR-mediated translational regulation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-11-2"  
class="td11">    
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">111</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-11-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-12-1"  
class="td11"> 
<strong><span class="cmbx-12">GTP hydrolysis and joining of the 60S ribosomal subunit</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-12-2"  
class="td11">    
9<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">111</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-12-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-13-1"  
class="td11"> 
<strong><span class="cmbx-12">SRP-dependent cotranslational protein targeting to membrane</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-13-2"  
class="td11">    
4<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">108</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-13-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-14-1"  
class="td11"> 
<strong><span class="cmbx-12">Eukaryotic Translation Initiation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-14-2"  
class="td11">    
4<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">106</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-14-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-15-1"  
class="td11"> 
<strong><span class="cmbx-12">Cap-dependent Translation Initiation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-15-2"  
class="td11">    
4<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">106</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-15-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-16-1"  
class="td11"> 
<strong><span class="cmbx-12">Influenza Viral </span><a 
href="#glo:RNA"><span class="cmbx-12">RNA</span></a> <span class="cmbx-12">Transcription and Replication</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-16-2"  
class="td11">    
8<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">103</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-16-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-17-1"  
class="td11"> 
<strong><span class="cmbx-12">Influenza Infection</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-17-2"  
class="td11">    
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">102</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-17-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-18-1"  
class="td11"> 
<strong><span class="cmbx-12">Translation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-18-2"  
class="td11">    
6<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">101</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-18-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-19-1"  
class="td11"> 
<strong><span class="cmbx-12">Influenza Life Cycle</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-19-2"  
class="td11">    
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">100</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-19-3"  
class="td11"> 
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-20-1"  
class="td11"> 
<strong><span class="cmbx-12">Disease</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-20-2"  
class="td11">     
2<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-20-3"  
class="td11">    
0<span class="cmmi-12">.</span>013347
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-21-1"  
class="td11"> 
<strong><span class="cmbx-12">GPCR downstream signalling</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-21-2"  
class="td11">     
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-21-3"  
class="td11">    
0<span class="cmmi-12">.</span>095478
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-22-1"  
class="td11"> 
Hemostasis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-22-2"  
class="td11">     
2<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">78</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-22-3"  
class="td11">     
0<span class="cmmi-12">.</span>2671
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-23-1"  
class="td11"> 
Signalling by GPCR
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-23-2"  
class="td11">     
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">73</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-23-3"  
class="td11">    
0<span class="cmmi-12">.</span>44939
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-24-1"  
class="td11"> 
<span class="cmti-12">Extracellular matrix organization </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-24-2"  
class="td11">     
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">67</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-24-3"  
class="td11">    
0<span class="cmmi-12">.</span>054008
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-25-1"  
class="td11"> 
Metabolism of proteins
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-25-2"  
class="td11">     
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">66</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-25-3"  
class="td11">     
0<span class="cmmi-12">.</span>9607
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-26-1"  
class="td11"> 
Signal Transduction
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-26-2"  
class="td11">     
2<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">66</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-26-3"  
class="td11">    
0<span class="cmmi-12">.</span>48184
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-27-1"  
class="td11"> 
Developmental Biology
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-27-2"  
class="td11">     
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">66</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-27-3"  
class="td11">    
0<span class="cmmi-12">.</span>54075
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-28-1"  
class="td11"> 
Innate Immune System
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-28-2"  
class="td11">     
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">66</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-28-3"  
class="td11">     
0<span class="cmmi-12">.</span>9589
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-29-1"  
class="td11"> 
Infectious disease
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-29-2"  
class="td11">     
9<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">66</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-29-3"  
class="td11">    
0<span class="cmmi-12">.</span>21075
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-30-1"  
class="td11"> 
Signalling by NGF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-30-2"  
class="td11">     
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">62</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-30-3"  
class="td11">    
0<span class="cmmi-12">.</span>43356
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-89-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-31-1"  
class="td11"> 
Immune System
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-31-2"  
class="td11">     
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">62</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-89-31-3"  
class="td11">    
0<span class="cmmi-12">.</span>23052
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-89-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-89-32-1"  
class="td11">                                                                      </td></tr></table>
<!--l. 725--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Over-representation (hypergeometric test) and Permutation p-values adjusted for multiple tests across pathways (</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a><span class="cmr-10x-x-109">). Significant</span> <span class="cmr-10x-x-109">pathways were marked in bold (</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a> <span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05) and italics (</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a> <span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">1).</span></p>
</p>
</div><hr class="endfloat" />
                                                                                    

<!--l. 729--><p class="indent" >
<a 
 id="x1-1750026"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table C.6: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from <a 
href="#glo:mtSLIPT">mtSLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> primary screen</span>
</div><!--tex4ht:label?: x1-1750026 -->
<div class="tabular">
<table id="TBL-90" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-90-1g"><col 
id="TBL-90-1" /><col 
id="TBL-90-2" /><col 
id="TBL-90-3" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-90-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-1-1"  
class="td11"> 
<span class="cmbx-12">Reactome Pathway </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-1-2"  
class="td11"> 
Over-representation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-1-3"  
class="td11"> 
Permutation
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-90-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-2-1"  
class="td11"> 
Visual phototransduction
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-2-2"  
class="td11">     
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-2-3"  
class="td11">   
0<span class="cmmi-12">.</span>86279
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-3-1"  
class="td11"> 
<strong><span class="cmbx-12">G</span><sub><span class="cmmi-8">αs</span> </sub> <span class="cmbx-12">signalling events</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-3-2"  
class="td11">     
2<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-3-3"  
class="td11">   
0<span class="cmmi-12">.</span>023066
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-4-1"  
class="td11"> 
Retinoid metabolism and transport
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-4-2"  
class="td11">     
2<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-4-3"  
class="td11">    
0<span class="cmmi-12">.</span>299
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-5-1"  
class="td11"> 
Acyl chain remodelling of PS
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-5-2"  
class="td11">     
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-5-3"  
class="td11">   
0<span class="cmmi-12">.</span>42584
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-6-1"  
class="td11"> 
Transcriptional regulation of white adipocyte differentiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-6-2"  
class="td11">     
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-6-3"  
class="td11">   
0<span class="cmmi-12">.</span>53928
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-7-1"  
class="td11"> 
Chemokine receptors bind chemokines
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-7-2"  
class="td11">     
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-7-3"  
class="td11">   
0<span class="cmmi-12">.</span>95259
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-8-1"  
class="td11"> 
<span class="cmti-12">Signalling by NOTCH4 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-8-2"  
class="td11">     
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-8-3"  
class="td11">   
0<span class="cmmi-12">.</span>079229
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-9-1"  
class="td11"> 
Defective EXT2 causes exostoses 2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-9-2"  
class="td11">     
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-9-3"  
class="td11">   
0<span class="cmmi-12">.</span>22292
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-10-1"  
class="td11"> 
Defective EXT1 causes exostoses 1, TRPS2 and CHDS
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-10-2"  
class="td11">     
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-10-3"  
class="td11">   
0<span class="cmmi-12">.</span>22292
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-11-1"  
class="td11"> 
Platelet activation, signalling and aggregation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-11-2"  
class="td11">     
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-11-3"  
class="td11">   
0<span class="cmmi-12">.</span>48853
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-12-1"  
class="td11"> 
Serotonin receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-12-2"  
class="td11">     
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-12-3"  
class="td11">   
0<span class="cmmi-12">.</span>34596
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-13-1"  
class="td11"> 
Nicotinamide salvaging
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-13-2"  
class="td11">     
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-13-3"  
class="td11">   
0<span class="cmmi-12">.</span>70881
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-14-1"  
class="td11"> 
Phase 1 - Functionalization of compounds
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-14-2"  
class="td11">      
2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-14-3"  
class="td11">   
0<span class="cmmi-12">.</span>31142
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-15-1"  
class="td11"> 
Amine ligand-binding receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-15-2"  
class="td11">     
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-15-3"  
class="td11">   
0<span class="cmmi-12">.</span>34934
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-16-1"  
class="td11"> 
Acyl chain remodelling of PE
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-16-2"  
class="td11">     
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-16-3"  
class="td11">   
0<span class="cmmi-12">.</span>42615
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-17-1"  
class="td11"> 
Signalling by GPCR
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-17-2"  
class="td11">     
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-17-3"  
class="td11">   
0<span class="cmmi-12">.</span>93888
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-18-1"  
class="td11"> 
<strong><span class="cmbx-12">Molecules associated with elastic fibres</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-18-2"  
class="td11">     
3<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-18-3"  
class="td11">   
0<span class="cmmi-12">.</span>017982
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-19-1"  
class="td11"> 
DAP12 interactions
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-19-2"  
class="td11">     
3<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-19-3"  
class="td11">   
0<span class="cmmi-12">.</span>71983
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-20-1"  
class="td11"> 
Beta defensins
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-20-2"  
class="td11">     
3<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-20-3"  
class="td11">   
0<span class="cmmi-12">.</span>91458
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-21-1"  
class="td11"> 
Cytochrome P<sub><span class="cmr-8">450</span></sub> - arranged by substrate type
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-21-2"  
class="td11">     
4<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-21-3"  
class="td11">   
0<span class="cmmi-12">.</span>83493
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-22-1"  
class="td11"> 
GPCR ligand binding
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-22-2"  
class="td11">     
5<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-22-3"  
class="td11">   
0<span class="cmmi-12">.</span>95258
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-23-1"  
class="td11"> 
Acyl chain remodelling of PC
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-23-2"  
class="td11">     
6<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-23-3"  
class="td11">   
0<span class="cmmi-12">.</span>42584
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-24-1"  
class="td11"> 
Response to elevated platelet cytosolic Ca<sup><span class="cmr-8">2+</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-24-2"  
class="td11">     
6<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-24-3"  
class="td11">   
0<span class="cmmi-12">.</span>54046
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-25-1"  
class="td11"> 
<strong><span class="cmbx-12">Arachidonic acid metabolism</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-25-2"  
class="td11">     
6<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-25-3"  
class="td11">   
0<span class="cmmi-12">.</span>026696
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-26-1"  
class="td11"> 
Defective B4GALT7 causes EDS, progeroid type
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-26-2"  
class="td11">     
7<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-26-3"  
class="td11">   
0<span class="cmmi-12">.</span>24921
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-27-1"  
class="td11"> 
Defective B3GAT3 causes JDSSDHD
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-27-2"  
class="td11">     
7<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-27-3"  
class="td11">   
0<span class="cmmi-12">.</span>24921
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-28-1"  
class="td11"> 
Hydrolysis of LPC
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-28-2"  
class="td11">     
7<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-28-3"  
class="td11">   
0<span class="cmmi-12">.</span>80663
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-29-1"  
class="td11"> 
<strong><span class="cmbx-12">Elastic fibre formation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-29-2"  
class="td11">     
7<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-29-3"  
class="td11">  
0<span class="cmmi-12">.</span>0058768
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-30-1"  
class="td11"> 
<strong> <span class="cmbx-12">HS-GAG degradation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-30-2"  
class="td11">     
9<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-30-3"  
class="td11">  
0<span class="cmmi-12">.</span>0083179
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-31-1"  
class="td11"> 
<span class="cmti-12">Bile acid and bile salt metabolism </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-31-2"  
class="td11">     
9<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-31-3"  
class="td11">   
0<span class="cmmi-12">.</span>079905
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-32-1"  
class="td11"> 
Netrin-1 signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-32-2"  
class="td11">      
0<span class="cmmi-12">.</span>00011
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-32-3"  
class="td11">   
0<span class="cmmi-12">.</span>92216
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-33-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-33-1"  
class="td11"> 
<strong><span class="cmbx-12">Integration of energy metabolism</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-33-2"  
class="td11">      
0<span class="cmmi-12">.</span>00011
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-33-3"  
class="td11">   
0<span class="cmmi-12">.</span>011152
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-34-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-34-1"  
class="td11"> 
Dectin-2 family
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-34-2"  
class="td11">      
0<span class="cmmi-12">.</span>00012
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-34-3"  
class="td11">   
0<span class="cmmi-12">.</span>10385
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-35-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-35-1"  
class="td11"> 
Platelet sensitization by LDL
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-35-2"  
class="td11">      
0<span class="cmmi-12">.</span>00012
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-35-3"  
class="td11">   
0<span class="cmmi-12">.</span>34596
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-36-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-36-1"  
class="td11"> 
DAP12 signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-36-2"  
class="td11">      
0<span class="cmmi-12">.</span>00012
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-36-3"  
class="td11">   
0<span class="cmmi-12">.</span>62787
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-37-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-37-1"  
class="td11"> 
Defensins
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-37-2"  
class="td11">      
0<span class="cmmi-12">.</span>00012
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-37-3"  
class="td11">   
0<span class="cmmi-12">.</span>77542
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-38-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-38-1"  
class="td11"> 
GPCR downstream signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-38-2"  
class="td11">      
0<span class="cmmi-12">.</span>00012
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-38-3"  
class="td11">   
0<span class="cmmi-12">.</span>79454
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-39-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-39-1"  
class="td11"> 
<span class="cmti-12">Diseases associated with glycosaminoglycan metabolism </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-39-2"  
class="td11">      
0<span class="cmmi-12">.</span>00013
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-39-3"  
class="td11">   
0<span class="cmmi-12">.</span>065927
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-40-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-40-1"  
class="td11"> 
<span class="cmti-12">Diseases of glycosylation </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-40-2"  
class="td11">      
0<span class="cmmi-12">.</span>00013
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-40-3"  
class="td11">   
0<span class="cmmi-12">.</span>065927
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-41-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-41-1"  
class="td11"> 
Signalling by Retinoic Acid
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-41-2"  
class="td11">      
0<span class="cmmi-12">.</span>00013
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-41-3"  
class="td11">   
0<span class="cmmi-12">.</span>22292
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-42-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-42-1"  
class="td11"> 
Signalling by Leptin
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-42-2"  
class="td11">      
0<span class="cmmi-12">.</span>00013
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-42-3"  
class="td11">   
0<span class="cmmi-12">.</span>34596
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-43-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-43-1"  
class="td11"> 
Signalling by SCF-KIT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-43-2"  
class="td11">      
0<span class="cmmi-12">.</span>00013
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-43-3"  
class="td11">   
0<span class="cmmi-12">.</span>70881
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-44-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-44-1"  
class="td11"> 
Opioid Signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-44-2"  
class="td11">      
0<span class="cmmi-12">.</span>00013
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-44-3"  
class="td11">   
0<span class="cmmi-12">.</span>96053
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-45-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-45-1"  
class="td11"> 
Signalling by NOTCH
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-45-2"  
class="td11">      
0<span class="cmmi-12">.</span>00015
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-45-3"  
class="td11">   
0<span class="cmmi-12">.</span>26884
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-46-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-46-1"  
class="td11"> 
Platelet homeostasis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-46-2"  
class="td11">      
0<span class="cmmi-12">.</span>00015
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-46-3"  
class="td11">    
0<span class="cmmi-12">.</span>4878
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-47-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-47-1"  
class="td11"> 
Signalling by NOTCH1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-47-2"  
class="td11">      
0<span class="cmmi-12">.</span>00016
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-47-3"  
class="td11">   
0<span class="cmmi-12">.</span>13043
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-48-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-48-1"  
class="td11"> 
Class B/2 (Secretin family receptors)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-48-2"  
class="td11">      
0<span class="cmmi-12">.</span>00016
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-48-3"  
class="td11">   
0<span class="cmmi-12">.</span>13994
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-49-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-49-1"  
class="td11"> 
<span class="cmti-12">Diseases of Immune System </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-49-2"  
class="td11">       
0<span class="cmmi-12">.</span>0002
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-49-3"  
class="td11">    
0<span class="cmmi-12">.</span>0795
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-50-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-50-1"  
class="td11"> 
<span class="cmti-12">Diseases associated with the TLR signalling cascade </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-50-2"  
class="td11">       
0<span class="cmmi-12">.</span>0002
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-50-3"  
class="td11">    
0<span class="cmmi-12">.</span>0795
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-90-51-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-51-1"  
class="td11"> 
A tetrasaccharide linker sequence is required for GAG synthesis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-51-2"  
class="td11">       
0<span class="cmmi-12">.</span>0002
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-90-51-3"  
class="td11">   
0<span class="cmmi-12">.</span>42615
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-90-52-"><td  style="white-space:nowrap; text-align:left;" id="TBL-90-52-1"  
class="td11">                                                       </td></tr></table>
<!--l. 846--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Over-representation (hypergeometric test) and Permutation p-values adjusted for multiple tests across pathways</span> <span class="cmr-10x-x-109">(</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a><span class="cmr-10x-x-109">). Significant pathways were marked in bold (</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a> <span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05) and italics (</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a> <span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">1).</span></p>
</p>
</div><hr class="endfloat" />
                                                                                    

<a 
 id="x1-175003r405"></a>
<h3 class="sectionHead">
<span class="titlemark">C.4 </span> <a 
 id="x1-1760004"></a>Compare SLIPT genes
</h3>
<!--l. 890--><p class="noindent" >
<p>The mutation synthetic lethal partners with <span class="cmti-12">CDH1 </span>were also compared to <a 
href="#glo:siRNA">siRNA</a> primary screen data (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), by correlation and <a 
href="#glo:siRNA">siRNA</a> viability as described in Sections <a 
href="#x1-1030002">4.2.2<!--tex4ht:ref: chapt3:compare_correlation --></a> and <a 
href="#x1-1040003">4.2.3<!--tex4ht:ref: chapt3:compare_viability --></a>.</p>
</p><!--l. 892--><p class="indent" >
<a 
 id="x1-1760013"></a><a 
 id="x1-1760024"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 894--><p class="noindent" >
</p><!--l. 895--><p class="noindent" >
<img 
src="mtSLIPT_siRNA_vs_Correlation_with_CDH1.png" alt="PIC"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure C.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">Compare </span><a 
href="#glo:mtSLIPT"><span class="cmbx-10x-x-109">mtSLIPT</span></a> <span class="cmbx-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">genes with correlation.</span></strong> <span class="cmr-10x-x-109">The </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">p-values were compared against Pearson correlation of expression with </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">. Genes detected</span> <span class="cmr-10x-x-109">by SLIPT or </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">were coloured according to the legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-1760013 -->
<div class="center" 
>
<!--l. 907--><p class="noindent" >
</p><!--l. 908--><p class="noindent" >
<img 
src="vioplotx_mtSLIPT_siRNA_vs_CDH1_Correlation_with_CDH1.png" alt="pict"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure C.4: </span><span class="content"><strong><span class="cmbx-10x-x-109">Compare </span><a 
href="#glo:mtSLIPT"><span class="cmbx-10x-x-109">mtSLIPT</span></a> <span class="cmbx-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">genes with correlation.</span></strong> <span class="cmr-10x-x-109">Genes detected</span> <span class="cmr-10x-x-109">by </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">against </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutation and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screening were compared against Pearson</span> <span class="cmr-10x-x-109">correlation of expression with </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">. There were no differences in correlation between the</span> <span class="cmr-10x-x-109">gene groups. </span></span></p>
</div>
<!--tex4ht:label?: x1-1760024 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 918--><p class="indent" >
<a 
 id="x1-1760035"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 920--><p class="noindent" >
</p><!--l. 921--><p class="noindent" >
<img 
src="vioplotx_mtSLIPT_siRNA_vs_Viability_Ratio_with_CDH1.png" alt="pict"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure C.5: </span><span class="content"><strong><span class="cmbx-10x-x-109">Compare </span><a 
href="#glo:mtSLIPT"><span class="cmbx-10x-x-109">mtSLIPT</span></a> <span class="cmbx-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">genes with </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">viability.</span></strong> <span class="cmr-10x-x-109">Genes</span> <span class="cmr-10x-x-109">detected as candidate synthetic lethal partners by </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">(in </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">breast cancer)</span> <span class="cmr-10x-x-109">expression analysis against </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutation and experimental screening (with </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a><span class="cmr-10x-x-109">) were</span> <span class="cmr-10x-x-109">compared against the viability ratio of </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutant and</span><a 
href="#glo:wild-type"><span class="cmr-10x-x-109">wild-type</span></a> <span class="cmr-10x-x-109">cells in the primary </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screen. There were clear no differences in viability between genes detected by </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">and</span> <span class="cmr-10x-x-109">those not with the differences being primarily due to viability thresholds that were used to</span> <span class="cmr-10x-x-109">detect synthetic lethality by </span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">Telford </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">2015</span></a><span class="cmr-10x-x-109">). </span></span></p>
</div>
<!--tex4ht:label?: x1-1760035 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    



<a 
 id="x1-176004r356"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix D</span><br /><a 
 id="x1-177000D"></a>Metagene Analysis
</h2>
<!--l. 936--><p class="noindent" >
Well characterised gene signatures from previous publications in breast cancer (<a 
href="#XGatza2011">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2011">2011</a>, <a 
href="#XGatza2014">2014</a>) were used to demonstrate to utility of the <a 
href="#glo:metagene">metagene</a> approach for use on a wider range of <a 
href="#glo:pathway">pathways</a> as was performed with the Reactome (<a 
href="#XReactome">Croft <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XReactome">2014</a>) <a 
href="#glo:pathway">pathways</a> as an alternative approach to identification of <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a>. The direction of <a 
href="#glo:metagene">metagenes</a> is arbitrary but they have been corrected to ensure the metagene increases in a direction which reflects overall activation of the <a 
href="#glo:pathway">pathways</a> (as described in Section <a 
href="#x1-540003">2.2.3<!--tex4ht:ref: methods:metagene --></a>) which was verfied by examining in the pathway signatures in breast cancer. <a 
href="#glo:metagene">Metagenes</a> were derived for these <a 
href="#glo:pathway">pathways</a> signatures (<a 
href="#XGatza2011">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2011">2011</a>, <a 
href="#XGatza2014">2014</a>), which were expected to have particular molecular properties in clinical and molecular subtypes (<a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>; <a 
href="#XPerou2000">Perou <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XPerou2000">2000</a>). This was performed by examining the <a 
href="#glo:pathway">pathways</a> <a 
href="#glo:gene expression">expression</a> of breast cancer gene signatures in <a 
href="#glo:TCGA">TCGA</a> <a 
href="#glo:gene expression">expression</a> data.
</p><!--l. 942--><p class="indent" >
These gene signatures were used to establish that <a 
href="#glo:metagene">metagenes</a> generated with this procedure reflect gene activity. The same <a 
href="#glo:metagene">metagene</a> procedure (in Section <a 
href="#x1-540003">2.2.3<!--tex4ht:ref: methods:metagene --></a>) was applied to the Reactome <a 
href="#glo:pathway">pathways</a> (<a 
href="#XReactome">Croft <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XReactome">2014</a>). These Reactome <a 
href="#glo:metagene">metagenes</a> were used for <a 
href="#glo:synthetic lethal">synthetic lethal</a> analysis of <a 
href="#glo:pathway">pathways</a> with <a 
href="#glo:SLIPT">SLIPT</a>, directly using <a 
href="#glo:pathway">pathways</a> activity for identifying <a 
href="#glo:synthetic lethal">synthetic lethal</a> <a 
href="#glo:pathway">pathways</a> with <span class="cmti-12">CDH1</span>. <a 
 id="x1-177001r411"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">D.1 </span> <a 
 id="x1-1780001"></a>Pathway Signature Expression
</h3>
<!--l. 948--><p class="noindent" >
<p>Pathway <a 
href="#glo:metagene">metagenes</a> (generated as described in Section <a 
href="#x1-540003">2.2.3<!--tex4ht:ref: methods:metagene --></a>) for gene signatures of key processes in breast cancer (<a 
href="#XGatza2011">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2011">2011</a>) were used to check that <a 
href="#glo:metagene">metagenes</a> were generated in the correct direction to indicate <a 
href="#glo:pathway">pathways</a> activation. Some of these gene signatures are plotted in Figure <a 
href="#x1-1780011">D.1<!--tex4ht:ref: fig:metagene_expr_Gatza2011 --></a> for comparison with clinical factors and <a 
href="#glo:somatic">somatic</a> <a 
href="#glo:mutation">mutations</a>. The “<a 
href="#glo:intrinsic subtype">intrinsic subtypes</a>” was computed by performing the <a 
href="#glo:PAM50">PAM50</a> procedure <a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XParker2009">2009</a>) for <a 
href="#glo:RNA-Seq">RNA-Seq</a> data which was highly concordant (<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> = 1305<span class="cmmi-12">.</span>9, <span class="cmmi-12">p </span>= 2<span class="cmmi-12">.</span>73 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">268</span></sup>) with the subtypes provided by <a 
href="#glo:UCSC">University of California, Santa Cruz (UCSC)</a> (<a 
href="#XUCSC2012">UCSC</a>, <a 
href="#XUCSC2012">2012</a>) for <a 
href="#glo:TCGA">TCGA</a> samples (<a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>) previously analysed by <a 
href="#glo:microarray">microarrays</a> (as shown in Appendix <a 
href="#x1-180000E">E<!--tex4ht:ref: appendix:intrinsic_subtypes --></a>). <a 
href="#glo:somatic">Somatic</a> <a 
href="#glo:mutation">mutations</a> were reported for glslinkrecurrent mutationrecurrently mutated genes in breast cancer, as reported by <a 
href="#glo:TCGA">TCGA</a> (<a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>), related genes, and those previously discussed to be important in <a 
href="#glo:hereditary">hereditary</a> breast cancers (<span class="cmti-12">BRCA1</span>, <span class="cmti-12">BRCA2</span>, and <span class="cmti-12">CDH1 </span>).</p>
</p><!--l. 950--><p class="indent" >
<a 
 id="x1-1780011"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="minipage"><div class="center" 
>
<!--l. 964--><p class="noindent" >
</p><!--l. 964--><p class="noindent" > 
<img 
src="CDH1_Heatmaps_Gatza2011(ranked)_Full_split_by_Subtype_and_CDH1_Stat_corr2.png" alt="pict"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure D.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Pathway meta</span><a 
href="#glo:gene expression"><span class="cmbx-10x-x-109">gene expression</span></a> <span class="cmbx-10x-x-109">profiles.</span></strong> <span class="cmr-10x-x-109">Expression profiles for </span><a 
href="#glo:metagene"><span class="cmr-10x-x-109">metagene</span></a> <span class="cmr-10x-x-109">signatures from </span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">Gatza </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">2011</span></a><span class="cmr-10x-x-109">) in </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">breast data, annotated for clinical factors</span> <span class="cmr-10x-x-109">(with sample types and histological results coloured according to the legend) and cancer</span> <span class="cmr-10x-x-109">gene </span><a 
href="#glo:mutation"><span class="cmr-10x-x-109">mutations</span></a> <span class="cmr-10x-x-109">(Negative values for </span><a 
href="#glo:mutation"><span class="cmr-10x-x-109">mutation</span></a> <span class="cmr-10x-x-109">are light grey with missing data in white).</span> <a 
href="#glo:intrinsic subtype"><span class="cmr-10x-x-109">Intrinsic subtypes</span></a> <span class="cmr-10x-x-109">are shown as derived from </span><a 
href="#glo:microarray"><span class="cmr-10x-x-109">microarray</span></a> <span class="cmr-10x-x-109">(</span><a 
href="#glo:UCSC"><span class="cmr-10x-x-109">UCSC</span></a><span class="cmr-10x-x-109">) and </span><a 
href="#glo:RNA-Seq"><span class="cmr-10x-x-109">RNA-Seq</span></a> <span class="cmr-10x-x-109">(</span><a 
href="#glo:PAM50"><span class="cmr-10x-x-109">PAM50</span></a><span class="cmr-10x-x-109">)</span> <span class="cmr-10x-x-109">data (</span><a 
href="#XTCGA2012"><span class="cmr-10x-x-109">Koboldt </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109">,</span><span class="cmr-10x-x-109"> </span><a 
href="#XTCGA2012"><span class="cmr-10x-x-109">2012</span></a><span class="cmr-10x-x-109">;</span><span class="cmr-10x-x-109"> </span><a 
href="#XParker2009"><span class="cmr-10x-x-109">Parker </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109">,</span><span class="cmr-10x-x-109"> </span><a 
href="#XParker2009"><span class="cmr-10x-x-109">2009</span></a><span class="cmr-10x-x-109">). Samples were clustered independently for</span> <span class="cmr-10x-x-109">each </span><a 
href="#glo:intrinsic subtype"><span class="cmr-10x-x-109">intrinsic subtypes</span></a> <span class="cmr-10x-x-109">and by </span><span class="cmti-10x-x-109">CDH1 </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">status. Pathway </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">signatures are</span> <span class="cmr-10x-x-109">consistent with </span><a 
href="#glo:mutation"><span class="cmr-10x-x-109">mutations</span></a> <span class="cmr-10x-x-109">and clinical subgroups. </span></span></p>
</div>
<!--tex4ht:label?: x1-1780011 -->                                      </div>
                                                                                    

</div><hr class="endfloat" />
<!--l. 967--><p class="indent" >
These gene signatures reflect <a 
href="#glo:intrinsic subtype">intrinsic subtypes</a> as expected. In particular, the estrogen and progesterone receptor signatures are low in the predominantly <a 
href="#glo:ER">ER</a><sup><span class="cmsy-8">-</span></sup> and <a 
href="#glo:PR">Progesterone receptor (PR)</a><sup><span class="cmsy-8">-</span></sup> basal-like subtype tumours. These tumours also had the highest frequency of <span class="cmti-12">TP53 </span><a 
href="#glo:mutation">mutations</a> and a corresponding reduction of p53 <a 
href="#glo:metagene">metagene</a> activity, as expected for loss of a <a 
href="#glo:tumour suppressor">tumour suppressor</a>. The luminal A and luminal B tumour subtypes are the most similar, which is reflected in these <a 
href="#glo:metagene">metagenes</a> signatures, although they are distinguishable molecular subtypes as shown by elevated <a 
href="#glo:PI3K">PI3K</a>, AKT, RAS, and <span class="cmmi-12">β</span>-catenin signalling in luminal B tumours. However, these <a 
href="#glo:pathway">pathways</a> were also elevated in basal-like and HER2-enriched subtypes and lowly expressed in the “normal-like” subtype (which contained the normal samples).
</p><!--l. 971--><p class="indent" >
<p>The direction of each <a 
href="#glo:metagene">metagene</a> was consistent with the clinical characteristics, which formed a consensus of gene activity as shown for the <a 
href="#glo:PI3K">PI3K</a> and <a 
href="#glo:ER">ER</a> signatures (<a 
href="#XGatza2011">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2011">2011</a>) in Figure <a 
href="#x1-1780022">D.2<!--tex4ht:ref: fig:metagene_expr_Gatza2011_ER --></a>. The <a 
href="#glo:gene expression">expression</a> of the majority of the genes were highly concordant with the <a 
href="#glo:metagene">metagene</a>, being either positively or negatively correlated. These were generally consistent with established clinical and molecular subtypes of breast cancer and the <a 
href="#glo:recurrent mutation">recurrent mutations</a> shown.</p>
</p><!--l. 974--><p class="indent" >
<a 
 id="x1-1780022"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="minipage"><div class="center" 
>
<!--l. 988--><p class="noindent" >
</p><!--l. 988--><p class="noindent" >
<img 
src="CDH1_Heatmaps_Gatza2011(ER)_Full_Metagene_mgorder.png" alt="PIC"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure D.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Expression profiles for estrogen receptor related genes.</span></strong> <span class="cmr-10x-x-109">Expression profiles the genes</span> <span class="cmr-10x-x-109">contained in the estrogen receptor (ER) gene signature from </span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">Gatza </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">2011</span></a><span class="cmr-10x-x-109">) in </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">breast data,</span> <span class="cmr-10x-x-109">annotated for clinical factors and cancer gene </span><a 
href="#glo:mutation"><span class="cmr-10x-x-109">mutations</span></a><span class="cmr-10x-x-109">. Samples are separated by </span><span class="cmti-10x-x-109">CDH1 </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">status</span> <span class="cmr-10x-x-109">and sorted by the </span><a 
href="#glo:metagene"><span class="cmr-10x-x-109">metagene</span></a><span class="cmr-10x-x-109">. In both cases, the majority of genes were consistent with the direction of the</span> <a 
href="#glo:metagene"><span class="cmr-10x-x-109">metagene</span></a><span class="cmr-10x-x-109">, with very few exceptions being inversely correlated. Estrogen receptor (by antibody staining)</span> <span class="cmr-10x-x-109">negative samples had low meta</span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">gene expression</span></a><span class="cmr-10x-x-109">, as expected. These were more likely to be ductal and basal</span> <span class="cmr-10x-x-109">subtypes, lacking </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">or </span><span class="cmti-10x-x-109">PIK3CA </span><a 
href="#glo:mutation"><span class="cmr-10x-x-109">mutations</span></a><span class="cmr-10x-x-109">. </span></span></p>
</div>
<!--tex4ht:label?: x1-1780022 -->                                                    </div>
                                                                                    

</div><hr class="endfloat" />
<a 
 id="x1-178003r416"></a>
<h3 class="sectionHead">
<span class="titlemark">D.2 </span> <a 
 id="x1-1790002"></a>Synthetic Lethal Reactome Metagenes
</h3>
<!--l. 1164--><p class="noindent" >
<p>Metagene analysis was performed for synthetic lethal pathways against <span class="cmti-12">CDH1 </span>mutation. These were described and compared to expression analysis in Section <a 
href="#x1-1090003">4.3<!--tex4ht:ref: chapt3:metagene_SL --></a>.</p>
</p><!--l. 1166--><p class="indent" >
<a 
 id="x1-1790011"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table D.1: </span><span class="content">Candidate synthetic lethal metagenes against <span class="cmti-12">CDH1 </span>from mtSLIPT</span>
</div><!--tex4ht:label?: x1-1790011 -->
<div class="tabular">
<table id="TBL-91" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-91-1g"><col 
id="TBL-91-1" /><col 
id="TBL-91-2" /><col 
id="TBL-91-3" /><col 
id="TBL-91-4" /><col 
id="TBL-91-5" /><col 
id="TBL-91-6" /><col 
id="TBL-91-7" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-91-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-1-1"  
class="td11"> 
<span class="cmbx-12">Pathway </span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-1-2"  
class="td11">   
ID
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-1-3"  
class="td11"> 
Observed
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-1-4"  
class="td11"> 
Expected
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-1-5"  
class="td11"> 
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-1-6"  
class="td11">   
p-value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-1-7"  
class="td11"> 
p-value (<a 
href="#glo:FDR">FDR</a>)
</td>

</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-91-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-2-1"  
class="td11"> 
Neurotoxicity of clostridium toxins
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-2-2"  
class="td11"> 
168799
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-2-3"  
class="td11"> 
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-2-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-2-5"  
class="td11">  
79.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-2-6"  
class="td11"> 
5<span class="cmmi-12">.</span>71 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-2-7"  
class="td11">  
3<span class="cmmi-12">.</span>14 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-3-1"  
class="td11"> 
Aquaporin-mediated transport
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-3-2"  
class="td11"> 
445717
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-3-3"  
class="td11"> 
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-3-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-3-5"  
class="td11">  
76.3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-3-6"  
class="td11"> 
2<span class="cmmi-12">.</span>73 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-3-7"  
class="td11">  
9<span class="cmmi-12">.</span>01 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-4-1"  
class="td11"> 
Toxicity of botulinum toxin type G (BoNT/G)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-4-2"  
class="td11"> 
5250989
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-4-3"  
class="td11"> 
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-4-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-4-5"  
class="td11">  
76.3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-4-6"  
class="td11"> 
2<span class="cmmi-12">.</span>73 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-4-7"  
class="td11">  
9<span class="cmmi-12">.</span>01 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-5-1"  
class="td11"> 
ABC-family proteins mediated transport
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-5-2"  
class="td11"> 
382556
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-5-3"  
class="td11"> 
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-5-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-5-5"  
class="td11">  
68.2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-5-6"  
class="td11"> 
1<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-5-7"  
class="td11">  
1<span class="cmmi-12">.</span>86 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-6-1"  
class="td11"> 
G<sub><span class="cmmi-8">αz</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-6-2"  
class="td11"> 
418597
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-6-3"  
class="td11"> 
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-6-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-6-5"  
class="td11">  
59.9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-6-6"  
class="td11"> 
9<span class="cmmi-12">.</span>97 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-6-7"  
class="td11">  
5<span class="cmmi-12">.</span>48 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-7-1"  
class="td11"> 
Regulation of IGF transport and uptake by IGFBPs
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-7-2"  
class="td11"> 
381426
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-7-3"  
class="td11"> 
9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-7-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-7-5"  
class="td11">  
56.3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-7-6"  
class="td11"> 
5<span class="cmmi-12">.</span>88 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-7-7"  
class="td11">  
2<span class="cmmi-12">.</span>11 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-8-1"  
class="td11"> 
GP1b-IX-V activation signalling
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-8-2"  
class="td11"> 
430116
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-8-3"  
class="td11"> 
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-8-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-8-5"  
class="td11">  
55.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-8-6"  
class="td11"> 
8<span class="cmmi-12">.</span>20 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-8-7"  
class="td11">  
2<span class="cmmi-12">.</span>76 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-9-1"  
class="td11"> 
GABA receptor activation
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-9-2"  
class="td11"> 
977443
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-9-3"  
class="td11"> 
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-9-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-9-5"  
class="td11">  
55.1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-9-6"  
class="td11"> 
1<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-9-7"  
class="td11">  
3<span class="cmmi-12">.</span>26 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-10-1"  
class="td11"> 
Vasopressin regulates renal water homeostasis via Aquaporins
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-10-2"  
class="td11"> 
432040
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-10-3"  
class="td11"> 
9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-10-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-10-5"  
class="td11">  
54.1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-10-6"  
class="td11"> 
1<span class="cmmi-12">.</span>77 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-10-7"  
class="td11">  
4<span class="cmmi-12">.</span>88 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-11-1"  
class="td11"> 
Toxicity of botulinum toxin type D (BoNT/D)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-11-2"  
class="td11"> 
5250955
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-11-3"  
class="td11"> 
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-11-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-11-5"  
class="td11">  
53.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-11-6"  
class="td11"> 
2<span class="cmmi-12">.</span>54 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-11-7"  
class="td11">  
6<span class="cmmi-12">.</span>64 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-12-1"  
class="td11"> 
Toxicity of botulinum toxin type F (BoNT/F)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-12-2"  
class="td11"> 
5250981
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-12-3"  
class="td11"> 
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-12-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-12-5"  
class="td11">  
53.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-12-6"  
class="td11"> 
2<span class="cmmi-12">.</span>54 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-12-7"  
class="td11">  
6<span class="cmmi-12">.</span>64 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-13-1"  
class="td11"> 
STAT6-mediated induction of chemokines
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-13-2"  
class="td11"> 
3249367
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-13-3"  
class="td11"> 
16
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-13-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-13-5"  
class="td11">  
52.2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-13-6"  
class="td11"> 
4<span class="cmmi-12">.</span>72 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-13-7"  
class="td11">  
1<span class="cmmi-12">.</span>13 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-14-1"  
class="td11"> 
Toxicity of botulinum toxin type B (BoNT/B)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-14-2"  
class="td11"> 
5250958
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-14-3"  
class="td11"> 
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-14-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-14-5"  
class="td11">  
50.8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-14-6"  
class="td11"> 
9<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-14-7"  
class="td11">  
1<span class="cmmi-12">.</span>98 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-15-1"  
class="td11"> 
S6K1 signalling
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-15-2"  
class="td11"> 
165720
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-15-3"  
class="td11"> 
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-15-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-15-5"  
class="td11">  
50.2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-15-6"  
class="td11"> 
1<span class="cmmi-12">.</span>24 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-15-7"  
class="td11">  
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-16-1"  
class="td11"> 
G<sub><span class="cmmi-8">αs</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-16-2"  
class="td11"> 
418555
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-16-3"  
class="td11"> 
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-16-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-16-5"  
class="td11">  
49.2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-16-6"  
class="td11"> 
2<span class="cmmi-12">.</span>08 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-16-7"  
class="td11">  
3<span class="cmmi-12">.</span>85 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-17-1"  
class="td11"> 
RHO GTPases activate CIT
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-17-2"  
class="td11"> 
5625900
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-17-3"  
class="td11"> 
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-17-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-17-5"  
class="td11">  
48.2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-17-6"  
class="td11"> 
3<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-17-7"  
class="td11">  
5<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-18-1"  
class="td11"> 
NADE modulates death signalling
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-18-2"  
class="td11"> 
205025
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-18-3"  
class="td11"> 
15
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-18-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-18-5"  
class="td11">  
47.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-18-6"  
class="td11"> 
5<span class="cmmi-12">.</span>00 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-18-7"  
class="td11">  
8<span class="cmmi-12">.</span>32 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-19-1"  
class="td11"> 
Keratan sulfate degradation
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-19-2"  
class="td11"> 
2022857
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-19-3"  
class="td11"> 
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-19-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-19-5"  
class="td11">  
46.6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-19-6"  
class="td11"> 
7<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-19-7"  
class="td11">  
1<span class="cmmi-12">.</span>15 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-20-1"  
class="td11"> 
Signalling by Retinoic Acid
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-20-2"  
class="td11"> 
5362517
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-20-3"  
class="td11"> 
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-20-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-20-5"  
class="td11">  
46.6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-20-6"  
class="td11"> 
7<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-20-7"  
class="td11">  
1<span class="cmmi-12">.</span>15 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-21-1"  
class="td11"> 
Adenylate cyclase inhibitory pathway
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-21-2"  
class="td11"> 
170670
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-21-3"  
class="td11"> 
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-21-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-21-5"  
class="td11">  
45.9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-21-6"  
class="td11"> 
1<span class="cmmi-12">.</span>11 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-21-7"  
class="td11">  
1<span class="cmmi-12">.</span>59 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-22-1"  
class="td11"> 
Inhibition of adenylate cyclase pathway
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-22-2"  
class="td11"> 
997269
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-22-3"  
class="td11"> 
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-22-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-22-5"  
class="td11">  
45.9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-22-6"  
class="td11"> 
1<span class="cmmi-12">.</span>11 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-22-7"  
class="td11">  
1<span class="cmmi-12">.</span>59 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-23-1"  
class="td11"> 
Fatty acids
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-23-2"  
class="td11"> 
211935
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-23-3"  
class="td11"> 
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-23-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-23-5"  
class="td11">  
45.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-23-6"  
class="td11"> 
1<span class="cmmi-12">.</span>21 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-23-7"  
class="td11">  
1<span class="cmmi-12">.</span>72 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-24-1"  
class="td11"> 
Ionotropic activity of Kainate Receptors
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-24-2"  
class="td11"> 
451306
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-24-3"  
class="td11"> 
13
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-24-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-24-5"  
class="td11">  
44.6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-24-6"  
class="td11"> 
2<span class="cmmi-12">.</span>03 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-24-7"  
class="td11">  
2<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-25-1"  
class="td11"> 
Activation of Ca-permeable Kainate Receptor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-25-2"  
class="td11"> 
451308
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-25-3"  
class="td11"> 
13
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-25-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-25-5"  
class="td11">  
44.6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-25-6"  
class="td11"> 
2<span class="cmmi-12">.</span>03 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-25-7"  
class="td11">  
2<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-91-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-26-1"  
class="td11"> 
RA biosynthesis pathway
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-91-26-2"  
class="td11"> 
5365859
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-26-3"  
class="td11"> 
13
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-26-4"  
class="td11">   
36.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-26-5"  
class="td11">  
44.6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-26-6"  
class="td11"> 
2<span class="cmmi-12">.</span>03 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-91-26-7"  
class="td11">  
2<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-91-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-91-27-1"  
class="td11">                                                     </td></tr></table>
<!--l. 1233--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Strongest candidate </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">by </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">with observed and expected numbers of mutant </span><span class="cmti-10x-x-109">CDH1 </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">breast cancer tumours with low</span> <span class="cmr-10x-x-109">expression of partner metagenes.</span></p>
</p>
</div><hr class="endfloat" />
                                                                                    



<a 
 id="x1-179002r356"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix E</span><br /><a 
 id="x1-180000E"></a>Intrinsic Subtyping
</h2> 
<p>The intrinsic subtypes for <a 
href="#glo:TCGA">TCGA</a> breast cancer samples provided by <a 
href="#glo:UCSC">UCSC</a> (<a 
href="#XTCGA2012">Koboldt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012">2012</a>; <a 
href="#XUCSC2012">UCSC</a>, <a 
href="#XUCSC2012">2012</a>) that were derived from microarray analysis have been compared to the <a 
href="#glo:PAM50">PAM50</a> results for performing subtyping from <a 
href="#glo:RNA-Seq">RNA-Seq</a> data (<a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>). As shown in Table <a 
href="#x1-1800011">E.1<!--tex4ht:ref: tab:intrinsic_subtypes --></a>, these subtypes were highly concordant for samples which had both procedures performed upon them (<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> = 1305<span class="cmmi-12">.</span>9, <span class="cmmi-12">p </span>= 2<span class="cmmi-12">.</span>73 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">268</span></sup>). The main exception were the luminal A samples some of which were reclassified as luminal B or “normal-like”.</p>
<!--l. 1356--><p class="indent" >
<a 
 id="x1-1800011"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table E.1: </span><span class="content">Comparison of intrinsic subtypes</span>
</div><!--tex4ht:label?: x1-1800011 -->
<div class="tabular">
<table id="TBL-92" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-92-1g"><col 
id="TBL-92-1" /><col 
id="TBL-92-2" /><col 
id="TBL-92-3" /><col 
id="TBL-92-4" /><col 
id="TBL-92-5" /><col 
id="TBL-92-6" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-92-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-1-1"  
class="td11">                 </td><td colspan="3" style="white-space:nowrap; text-align:left;" id="TBL-92-1-2"  
class="td11"></td>                          <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<a 
href="#glo:UCSC"><span class="cmbx-12">UCSC</span></a> <span class="cmbx-12">Subtype</span>
</div>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-92-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-2-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-92-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-3-1"  
class="td11">                  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-3-2"  
class="td11"> 
Basal-like
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-3-3"  
class="td11"> 
HER2-enriched
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-3-4"  
class="td11"> 
Luminal A
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-3-5"  
class="td11"> 
Luminal B
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-3-6"  
class="td11"> 
Normal-like
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-92-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-4-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-92-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-5-1"  
class="td11">                  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-5-2"  
class="td11"> 
100
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-5-3"  
class="td11"> 
58
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-5-4"  
class="td11"> 
232
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-5-5"  
class="td11"> 
128
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-5-6"  
class="td11"> 
30
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-92-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-6-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-92-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-7-1"  
class="td11">                  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-92-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-8-1"  
class="td11">                  </td><td colspan="3" style="white-space:nowrap; text-align:left;" id="TBL-92-8-2"  
class="td11"></td>                         <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<a 
href="#glo:PAM50"><span class="cmbx-12">PAM50</span></a> <span class="cmbx-12">Subtype</span>
</div>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-92-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-9-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-92-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-10-1"  
class="td11">                  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-10-2"  
class="td11"> 
Basal-like
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-10-3"  
class="td11"> 
HER2-enriched
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-10-4"  
class="td11"> 
Luminal A
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-10-5"  
class="td11"> 
Luminal B
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-10-6"  
class="td11"> 
Normal-like
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-92-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-11-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-92-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-12-1"  
class="td11">                  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-12-2"  
class="td11"> 
208
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-12-3"  
class="td11"> 
94
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-12-4"  
class="td11"> 
314
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-12-5"  
class="td11"> 
334
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-12-6"  
class="td11"> 
227
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-92-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-13-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-92-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-14-1"  
class="td11">                  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-92-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-15-1"  
class="td11">                  </td><td colspan="3" style="white-space:nowrap; text-align:left;" id="TBL-92-15-2"  
class="td11"></td>                          <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<a 
href="#glo:UCSC"><span class="cmbx-12">UCSC</span></a> <span class="cmbx-12">Subtype</span>
</div>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-92-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-16-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-92-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-17-1"  
class="td11"> 
<strong><a 
href="#glo:PAM50"><span class="cmbx-12">PAM50</span></a> <span class="cmbx-12">Subtype</span></strong>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-17-2"  
class="td11"> 
Basal-like
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-17-3"  
class="td11"> 
HER2-enriched
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-17-4"  
class="td11"> 
Luminal A
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-17-5"  
class="td11"> 
Luminal B
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-17-6"  
class="td11"> 
Normal-like
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-92-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-18-1"  
class="td11"> 
Basal-like
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-18-2"  
class="td11"> 
96
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-18-3"  
class="td11"> 
4
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-18-4"  
class="td11"> 
2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-18-5"  
class="td11"> 
2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-18-6"  
class="td11"> 
1
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-92-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-19-1"  
class="td11"> 
HER2-enriched
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-19-2"  
class="td11"> 
0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-19-3"  
class="td11"> 
47
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-19-4"  
class="td11"> 
5
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-19-5"  
class="td11"> 
3
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-19-6"  
class="td11"> 
0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-92-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-20-1"  
class="td11"> 
Luminal A
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-20-2"  
class="td11"> 
1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-20-3"  
class="td11"> 
0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-20-4"  
class="td11"> 
141
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-20-5"  
class="td11"> 
1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-20-6"  
class="td11"> 
0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-92-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-21-1"  
class="td11"> 
Luminal B
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-21-2"  
class="td11"> 
2
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-21-3"  
class="td11"> 
7
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-21-4"  
class="td11"> 
49
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-21-5"  
class="td11"> 
121
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-21-6"  
class="td11"> 
0
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-92-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-22-1"  
class="td11"> 
Normal-like
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-22-2"  
class="td11"> 
1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-22-3"  
class="td11"> 
0
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-22-4"  
class="td11"> 
35
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-22-5"  
class="td11"> 
1
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-92-22-6"  
class="td11"> 
29
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-92-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-92-23-1"  
class="td11">                  </td></tr></table>
<!--l. 1400--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">The intrinsic subtypes of </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">breast samples were compared between those provided by </span><a 
href="#glo:UCSC"><span class="cmr-10x-x-109">UCSC</span></a> <span class="cmr-10x-x-109">(</span><a 
href="#XTCGA2012"><span class="cmr-10x-x-109">Koboldt</span> <em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109">,</span><span class="cmr-10x-x-109"> </span><a 
href="#XTCGA2012"><span class="cmr-10x-x-109">2012</span></a><span class="cmr-10x-x-109">) from microarray expression to those derived from </span><a 
href="#glo:RNA-Seq"><span class="cmr-10x-x-109">RNA-Seq</span></a> <span class="cmr-10x-x-109">data (</span><a 
href="#XParker2009"><span class="cmr-10x-x-109">Parker </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109">,</span><span class="cmr-10x-x-109"> </span><a 
href="#XParker2009"><span class="cmr-10x-x-109">2009</span></a><span class="cmr-10x-x-109">).</span> <span class="cmr-10x-x-109">Comparisons between these were limited to samples for which both data types were available.</span></p>
</p>
</div><hr class="endfloat" />
<!--l. 1403--><p class="noindent" >
<p>The <a 
href="#glo:PAM50">PAM50</a> subtypes could be more accurate given similarity of these subtypes and that the remainder of the subtypes were accurately recapitulated with <a 
href="#glo:RNA-Seq">RNA-Seq</a> data. Furthermore, <a 
href="#glo:UCSC">UCSC</a> subtypes correctly identified <span class="cmr-8">22</span> <span class="cmmi-12">∕</span> <span class="cmr-8">22</span> normal samples as “normal-like” and <a 
href="#glo:PAM50">PAM50</a> subtyping in <a 
href="#glo:RNA-Seq">RNA-Seq</a> data had a success rate of <span class="cmr-8">112</span> <span class="cmmi-12">∕</span> <span class="cmr-8">113</span> (including all of those identified from microarrays). Therefore the <a 
href="#glo:PAM50">PAM50</a> subtypes (performed on a larger cohort of samples) are appropriate to use for further interpretation, superseding the <a 
href="#glo:UCSC">UCSC</a> subtypes available for a limited set of samples.</p>
<a 
 id="x1-180002r356"></a>
</p>
<h2 class="appendixHead">
<span class="titlemark">Appendix F</span><br /><a 
 id="x1-181000F"></a>Stomach Expression Analysis
</h2>
<!--l. 1411--><p class="noindent" >
The following results are a replication of the <a 
href="#glo:TCGA">TCGA</a> results (in Chapter <a 
href="#x1-960004">4<!--tex4ht:ref: chap:SLIPT --></a>) with stomach cancer data, using synthetic lethality (SLIPT) against <span class="cmti-12">CDH1</span>. <a 
 id="x1-181001r419"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">F.1 </span> <a 
 id="x1-1820001"></a>Synthetic Lethal Genes and Pathways
</h3>
                                                                                    

<!--l. 1415--><p class="indent" >
<a 
 id="x1-1820011"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table F.1: </span><span class="content">Synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from SLIPT in stomach cancer</span>
</div><!--tex4ht:label?: x1-1820011 -->
<div class="tabular">
<table id="TBL-93" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-93-1g"><col 
id="TBL-93-1" /><col 
id="TBL-93-2" /><col 
id="TBL-93-3" /><col 
id="TBL-93-4" /><col 
id="TBL-93-5" /><col 
id="TBL-93-6" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-93-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-1-1"  
class="td11"> 
<span class="cmbx-12">Gene </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-1-2"  
class="td11"> 
Observed<sup class="textsuperscript"><span class="cmr-10x-x-109">*</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-1-3"  
class="td11"> 
Expected<sup class="textsuperscript"><span class="cmr-10x-x-109">*</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-1-4"  
class="td11"> 
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-1-5"  
class="td11">   
p-value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-1-6"  
class="td11"> 
p-value (<a 
href="#glo:FDR">FDR</a>)
</td>

</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-93-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-2-1"  
class="td11"> 
<span class="cmti-12">PRAF2 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-2-2"  
class="td11">    
17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-2-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-2-4"  
class="td11">   
121
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-2-5"  
class="td11"> 
3<span class="cmmi-12">.</span>54 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-2-6"  
class="td11">  
1<span class="cmmi-12">.</span>45 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-3-1"  
class="td11"> 
<span class="cmti-12">EMP3 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-3-2"  
class="td11">    
17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-3-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-3-4"  
class="td11">   
115
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-3-5"  
class="td11"> 
5<span class="cmmi-12">.</span>06 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-3-6"  
class="td11">  
1<span class="cmmi-12">.</span>48 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-4-1"  
class="td11"> 
<span class="cmti-12">PLEKHO1 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-4-2"  
class="td11">    
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-4-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-4-4"  
class="td11">   
112
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-4-5"  
class="td11"> 
2<span class="cmmi-12">.</span>14 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">23</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-4-6"  
class="td11">  
4<span class="cmmi-12">.</span>75 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-5-1"  
class="td11"> 
<span class="cmti-12">SELM </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-5-2"  
class="td11">    
20
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-5-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-5-4"  
class="td11">   
111
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-5-5"  
class="td11"> 
5<span class="cmmi-12">.</span>13 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">23</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-5-6"  
class="td11">  
8<span class="cmmi-12">.</span>09 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-6-1"  
class="td11"> 
<span class="cmti-12">GYPC </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-6-2"  
class="td11">    
20
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-6-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-6-4"  
class="td11">   
110
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-6-5"  
class="td11"> 
5<span class="cmmi-12">.</span>77 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">23</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-6-6"  
class="td11">  
8<span class="cmmi-12">.</span>45 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-7-1"  
class="td11"> 
<span class="cmti-12">COX7A1 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-7-2"  
class="td11">    
18
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-7-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-7-4"  
class="td11">   
109
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-7-5"  
class="td11"> 
1<span class="cmmi-12">.</span>15 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-7-6"  
class="td11">  
1<span class="cmmi-12">.</span>39 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-8-1"  
class="td11"> 
<span class="cmti-12">TNFSF12 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-8-2"  
class="td11">    
20
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-8-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-8-4"  
class="td11">   
106
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-8-5"  
class="td11"> 
4<span class="cmmi-12">.</span>06 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-8-6"  
class="td11">  
4<span class="cmmi-12">.</span>38 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-9-1"  
class="td11"> 
<span class="cmti-12">SEPT4 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-9-2"  
class="td11">    
17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-9-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-9-4"  
class="td11">   
106
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-9-5"  
class="td11"> 
6<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-9-6"  
class="td11">  
5<span class="cmmi-12">.</span>91 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-10-1"  
class="td11"> 
<span class="cmti-12">LGALS1 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-10-2"  
class="td11">    
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-10-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-10-4"  
class="td11">   
105
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-10-5"  
class="td11"> 
6<span class="cmmi-12">.</span>64 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-10-6"  
class="td11">  
5<span class="cmmi-12">.</span>91 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-11-1"  
class="td11"> 
<span class="cmti-12">RARRES2 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-11-2"  
class="td11">    
27
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-11-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-11-4"  
class="td11">   
105
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-11-5"  
class="td11"> 
8<span class="cmmi-12">.</span>02 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-11-6"  
class="td11">  
6<span class="cmmi-12">.</span>85 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-12-1"  
class="td11"> 
<span class="cmti-12">VEGFB </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-12-2"  
class="td11">    
16
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-12-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-12-4"  
class="td11">   
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-12-5"  
class="td11"> 
1<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-12-6"  
class="td11">  
9<span class="cmmi-12">.</span>74 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-13-1"  
class="td11"> 
<span class="cmti-12">PRR24 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-13-2"  
class="td11">    
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-13-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-13-4"  
class="td11">   
102
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-13-5"  
class="td11"> 
2<span class="cmmi-12">.</span>96 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-13-6"  
class="td11">  
2<span class="cmmi-12">.</span>02 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-14-1"  
class="td11"> 
<span class="cmti-12">SYNC </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-14-2"  
class="td11">    
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-14-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-14-4"  
class="td11">   
102
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-14-5"  
class="td11"> 
3<span class="cmmi-12">.</span>73 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-14-6"  
class="td11">  
2<span class="cmmi-12">.</span>39 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-15-1"  
class="td11"> 
<span class="cmti-12">MAGEH1 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-15-2"  
class="td11">    
17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-15-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-15-4"  
class="td11">   
100
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-15-5"  
class="td11"> 
9<span class="cmmi-12">.</span>52 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-15-6"  
class="td11">  
5<span class="cmmi-12">.</span>01 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-16-1"  
class="td11"> 
<span class="cmti-12">HSPB2 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-16-2"  
class="td11">    
23
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-16-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-16-4"  
class="td11">   
99.6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-16-5"  
class="td11"> 
1<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-16-6"  
class="td11">  
5<span class="cmmi-12">.</span>82 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-17-1"  
class="td11"> 
<span class="cmti-12">SMARCD3 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-17-2"  
class="td11">    
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-17-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-17-4"  
class="td11">   
99
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-17-5"  
class="td11"> 
1<span class="cmmi-12">.</span>59 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-17-6"  
class="td11">  
7<span class="cmmi-12">.</span>57 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-18-1"  
class="td11"> 
<span class="cmti-12">CREM </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-18-2"  
class="td11">    
13
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-18-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-18-4"  
class="td11">   
98.1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-18-5"  
class="td11"> 
2<span class="cmmi-12">.</span>48 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-18-6"  
class="td11">  
1<span class="cmmi-12">.</span>13 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-19-1"  
class="td11"> 
<span class="cmti-12">GNG11 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-19-2"  
class="td11">    
20
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-19-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-19-4"  
class="td11">   
97.3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-19-5"  
class="td11"> 
3<span class="cmmi-12">.</span>68 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-19-6"  
class="td11">  
1<span class="cmmi-12">.</span>59 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-20-1"  
class="td11"> 
<span class="cmti-12">GNAI2 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-20-2"  
class="td11">    
17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-20-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-20-4"  
class="td11">   
96.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-20-5"  
class="td11"> 
5<span class="cmmi-12">.</span>75 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-20-6"  
class="td11">  
2<span class="cmmi-12">.</span>36 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-21-1"  
class="td11"> 
<span class="cmti-12">FUNDC2 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-21-2"  
class="td11">    
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-21-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-21-4"  
class="td11">   
95.9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-21-5"  
class="td11"> 
7<span class="cmmi-12">.</span>39 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-21-6"  
class="td11">  
2<span class="cmmi-12">.</span>91 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-22-1"  
class="td11"> 
<span class="cmti-12">CNRIP1 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-22-2"  
class="td11">    
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-22-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-22-4"  
class="td11">   
95.3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-22-5"  
class="td11"> 
1<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-22-6"  
class="td11">  
3<span class="cmmi-12">.</span>66 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-23-1"  
class="td11"> 
<span class="cmti-12">CALHM2 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-23-2"  
class="td11">    
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-23-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-23-4"  
class="td11">   
93.1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-23-5"  
class="td11"> 
2<span class="cmmi-12">.</span>94 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-23-6"  
class="td11">  
1<span class="cmmi-12">.</span>06 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-24-1"  
class="td11"> 
<span class="cmti-12">ARID5A </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-24-2"  
class="td11">    
18
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-24-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-24-4"  
class="td11">   
92.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-24-5"  
class="td11"> 
3<span class="cmmi-12">.</span>47 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-24-6"  
class="td11">  
1<span class="cmmi-12">.</span>22 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-25-1"  
class="td11"> 
<span class="cmti-12">ST3GAL3 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-25-2"  
class="td11">    
27
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-25-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-25-4"  
class="td11">   
92.2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-25-5"  
class="td11"> 
4<span class="cmmi-12">.</span>49 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-25-6"  
class="td11">  
1<span class="cmmi-12">.</span>56 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-93-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-26-1"  
class="td11"> 
<span class="cmti-12">LOC339524 </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-26-2"  
class="td11">    
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-26-3"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-26-4"  
class="td11">   
92.1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-26-5"  
class="td11"> 
4<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-93-26-6"  
class="td11">  
1<span class="cmmi-12">.</span>59 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-93-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-93-27-1"  
class="td11">            </td></tr></table>
<!--l. 1484--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Strongest candidate </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">in </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">stomach cancer</span> <span class="cmr-10x-x-109">expression data</span> <sup class="textsuperscript"><span class="cmr-10x-x-109">*</span></sup><span class="cmr-10x-x-109">Observed and expected numbers of samples which had low </span><a 
href="#glo:gene expression"><span class="cmr-10x-x-109">expression</span></a> <span class="cmr-10x-x-109">of both genes</span></p>
</p>
</div><hr class="endfloat" />
                                                                                    

<!--l. 1488--><p class="indent" >
<a 
 id="x1-1820022"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table F.2: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in stomach cancer</span>
</div><!--tex4ht:label?: x1-1820022 -->
<div class="tabular">
<table id="TBL-94" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-94-1g"><col 
id="TBL-94-1" /><col 
id="TBL-94-2" /><col 
id="TBL-94-3" /><col 
id="TBL-94-4" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-94-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-1-1"  
class="td11"> 
<strong><span class="cmbx-12">Pathways Over-represented</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-1-2"  
class="td11"> 
<strong><span class="cmbx-12">Pathway Size</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-1-3"  
class="td11"> 
<strong><span class="cmbx-12">SL Genes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-1-4"  
class="td11"> 
<strong><span class="cmbx-12">p-value (</span><a 
href="#glo:FDR"><span class="cmbx-12">FDR</span></a><span class="cmbx-12">)</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-94-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-2-1"  
class="td11"> 
Extracellular matrix organization
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-2-2"  
class="td11">      
241
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-2-3"  
class="td11">    
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-2-4"  
class="td11">   
7<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">140</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-3-1"  
class="td11"> 
Hemostasis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-3-2"  
class="td11">      
445
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-3-3"  
class="td11">    
138
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-3-4"  
class="td11">   
1<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">121</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-4-1"  
class="td11"> 
Developmental Biology
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-4-2"  
class="td11">      
432
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-4-3"  
class="td11">    
125
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-4-4"  
class="td11">   
9<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">107</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-5-1"  
class="td11"> 
Axon guidance
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-5-2"  
class="td11">      
289
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-5-3"  
class="td11">     
94
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-5-4"  
class="td11">   
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">102</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-6-1"  
class="td11"> 
Eukaryotic Translation Termination
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-6-2"  
class="td11">      
84
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-6-3"  
class="td11">     
49
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-6-4"  
class="td11">   
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">99</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-7-1"  
class="td11"> 
GPCR ligand binding
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-7-2"  
class="td11">      
373
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-7-3"  
class="td11">    
108
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-7-4"  
class="td11">   
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">99</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-8-1"  
class="td11"> 
Viral <a 
href="#glo:mRNA">mRNA</a> Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-8-2"  
class="td11">      
82
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-8-3"  
class="td11">     
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-8-4"  
class="td11">   
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">98</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-9-1"  
class="td11"> 
Formation of a pool of free 40S subunits
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-9-2"  
class="td11">      
94
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-9-3"  
class="td11">     
51
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-9-4"  
class="td11">   
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">98</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-10-1"  
class="td11"> 
Eukaryotic Translation Elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-10-2"  
class="td11">      
87
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-10-3"  
class="td11">     
49
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-10-4"  
class="td11">   
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-11-1"  
class="td11"> 
Peptide chain elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-11-2"  
class="td11">      
84
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-11-3"  
class="td11">     
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-11-4"  
class="td11">   
7<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-12-1"  
class="td11"> 
Class A/1 (Rhodopsin-like receptors)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-12-2"  
class="td11">      
289
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-12-3"  
class="td11">     
90
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-12-4"  
class="td11">   
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">96</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-13-1"  
class="td11"> 
Nonsense Mediated Decay independent of the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-13-2"  
class="td11">      
89
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-13-3"  
class="td11">     
49
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-13-4"  
class="td11">   
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">96</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-14-1"  
class="td11"> 
Infectious disease
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-14-2"  
class="td11">      
349
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-14-3"  
class="td11">    
100
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-14-4"  
class="td11">   
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">94</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-15-1"  
class="td11"> 
GTP hydrolysis and joining of the 60S ribosomal subunit
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-15-2"  
class="td11">      
105
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-15-3"  
class="td11">     
52
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-15-4"  
class="td11">   
3<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">94</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-16-1"  
class="td11"> 
L13a-mediated translational silencing of Ceruloplasmin expression
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-16-2"  
class="td11">      
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-16-3"  
class="td11">     
51
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-16-4"  
class="td11">   
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-17-1"  
class="td11"> 
3’ -UTR-mediated translational regulation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-17-2"  
class="td11">      
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-17-3"  
class="td11">     
51
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-17-4"  
class="td11">   
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-18-1"  
class="td11"> 
Neuronal System
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-18-2"  
class="td11">      
272
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-18-3"  
class="td11">     
84
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-18-4"  
class="td11">   
8<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-19-1"  
class="td11"> 
SRP-dependent cotranslational protein targeting to membrane
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-19-2"  
class="td11">      
105
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-19-3"  
class="td11">     
51
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-19-4"  
class="td11">   
9<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-20-1"  
class="td11"> 
Eukaryotic Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-20-2"  
class="td11">      
112
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-20-3"  
class="td11">     
52
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-20-4"  
class="td11">   
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-94-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-21-1"  
class="td11"> 
Cap-dependent Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-21-2"  
class="td11">      
112
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-21-3"  
class="td11">     
52
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-94-21-4"  
class="td11">   
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-94-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-94-22-1"  
class="td11">                                                             </td></tr></table>
<!--l. 1544--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Gene set over-representation analysis (hypergeometric test) for Reactome pathways in </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">partners for </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">.</span></p>
</p>
</div><hr class="endfloat" />
                                                                                    

<!--l. 1555--><p class="indent" >
<a 
 id="x1-1820031"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<img 
src="CDH1_Heatmaps_Genes_Split_By_CDH1_z-trans_exprSL_cordistx_Pub_stad.png" alt="PIC"  
 /> <br />
<div class="caption">
<p><span class="id">Figure F.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethal expression profiles of analysed samples.</span></strong> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">expression profile heatmap (correlation distance) of all samples (separated by the</span> <span class="cmr-8">1</span> <span class="cmmi-10x-x-109">∕</span> <span class="cmr-8">3</span> <span class="cmr-10x-x-109">quantile</span> <span class="cmr-10x-x-109">of </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">expression) analysed in </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">stomach cancer dataset for gene expression of 4365</span> <span class="cmr-10x-x-109">candidate partners of </span><a 
href="#glo:E-cadherin"><span class="cmr-10x-x-109">E-cadherin</span></a> <span class="cmr-10x-x-109">(</span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">) from </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">prediction (with significant </span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a> <span class="cmr-10x-x-109">adjusted </span><span class="cmmi-10x-x-109">p &lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05). Deeply clustered, inter-correlated genes form several main groups, each</span> <span class="cmr-10x-x-109">containing genes that were SL candidates or toxic in an </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screen </span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">Telford </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">2015</span></a><span class="cmr-10x-x-109">).</span> <span class="cmr-10x-x-109">Clusters had different sample groups highly expressing the synthetic lethal candidates in</span> <span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">low samples. Notably, diffuse and </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutant samples had elevated expression</span> <span class="cmr-10x-x-109">in one or more distinct clusters, although there was less complexity and variation among</span> <span class="cmr-10x-x-109">candidate synthetic lethal partners than in breast data. </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">low samples also contained</span> <span class="cmr-10x-x-109">most of samples with </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutations. </span></span></p>
</div>
<!--tex4ht:label?: x1-1820031 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 1573--><p class="indent" >
<a 
 id="x1-1820043"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table F.3: </span><span class="content">Pathways for clusters of <span class="cmti-12">CDH1 </span>partners in stomach SLIPT</span>
</div><!--tex4ht:label?: x1-1820043 -->
<div class="tabular">
<table id="TBL-95" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-95-1g"><col 
id="TBL-95-1" /><col 
id="TBL-95-2" /><col 
id="TBL-95-3" /><col 
id="TBL-95-4" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-95-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-1-1"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 1</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-1-2"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-1-3"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-1-4"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">p-value (</span><a 
href="#glo:FDR"><span class="cmbx-12x-x-120">FDR</span></a><span class="cmbx-12x-x-120">)</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-95-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-2-1"  
class="td11"> 
Viral <a 
href="#glo:mRNA">mRNA</a> Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-2-2"  
class="td11">        
82
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-2-3"  
class="td11">        
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-2-4"  
class="td11">     
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-3-1"  
class="td11"> 
Formation of a pool of free 40S subunits
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-3-2"  
class="td11">        
94
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-3-3"  
class="td11">        
51
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-3-4"  
class="td11">     
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-4-1"  
class="td11"> 
Eukaryotic Translation Elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-4-2"  
class="td11">        
87
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-4-3"  
class="td11">        
49
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-4-4"  
class="td11">     
4<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-5-1"  
class="td11"> 
Peptide chain elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-5-2"  
class="td11">        
84
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-5-3"  
class="td11">        
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-5-4"  
class="td11">     
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">96</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-6-1"  
class="td11"> 
Eukaryotic Translation Termination
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-6-2"  
class="td11">        
84
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-6-3"  
class="td11">        
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-6-4"  
class="td11">     
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">96</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-7-1"  
class="td11"> 
GTP hydrolysis and joining of the 60S ribosomal subunit
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-7-2"  
class="td11">       
105
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-7-3"  
class="td11">        
52
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-7-4"  
class="td11">     
7<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">94</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-8-1"  
class="td11"> 
Nonsense Mediated Decay independent of the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-8-2"  
class="td11">        
89
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-8-3"  
class="td11">        
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-8-4"  
class="td11">     
3<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">93</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-9-1"  
class="td11"> 
L13a-mediated translational silencing of Ceruloplasmin expression
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-9-2"  
class="td11">       
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-9-3"  
class="td11">        
51
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-9-4"  
class="td11">     
5<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-10-1"  
class="td11"> 
3’ -UTR-mediated translational regulation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-10-2"  
class="td11">       
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-10-3"  
class="td11">        
51
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-10-4"  
class="td11">     
5<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-11-1"  
class="td11"> 
SRP-dependent cotranslational protein targeting to membrane
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-11-2"  
class="td11">       
105
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-11-3"  
class="td11">        
51
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-11-4"  
class="td11">     
1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">91</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-12-1"  
class="td11"> 
Eukaryotic Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-12-2"  
class="td11">       
112
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-12-3"  
class="td11">        
52
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-12-4"  
class="td11">     
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-13-1"  
class="td11"> 
Cap-dependent Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-13-2"  
class="td11">       
112
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-13-3"  
class="td11">        
52
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-13-4"  
class="td11">     
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-14-1"  
class="td11"> 
Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-14-2"  
class="td11">       
142
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-14-3"  
class="td11">        
56
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-14-4"  
class="td11">     
3<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">85</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-15-1"  
class="td11"> 
Nonsense-Mediated Decay
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-15-2"  
class="td11">       
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-15-3"  
class="td11">        
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-15-4"  
class="td11">     
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">84</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-16-1"  
class="td11"> 
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-16-2"  
class="td11">       
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-16-3"  
class="td11">        
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-16-4"  
class="td11">     
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">84</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-17-1"  
class="td11"> 
Influenza Viral <a 
href="#glo:RNA">RNA</a> Transcription and Replication
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-17-2"  
class="td11">       
109
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-17-3"  
class="td11">        
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-17-4"  
class="td11">     
4<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">82</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-18-1"  
class="td11"> 
Influenza Life Cycle
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-18-2"  
class="td11">       
113
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-18-3"  
class="td11">        
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-18-4"  
class="td11">     
3<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-19-1"  
class="td11"> 
Influenza Infection
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-19-2"  
class="td11">       
118
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-19-3"  
class="td11">        
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-19-4"  
class="td11">     
6<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">78</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-95-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-20-1"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 2</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-20-2"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-20-3"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-20-4"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">p-value (</span><a 
href="#glo:FDR"><span class="cmbx-12x-x-120">FDR</span></a><span class="cmbx-12x-x-120">)</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-95-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-21-1"  
class="td11"> 
Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-21-2"  
class="td11">        
65
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-21-3"  
class="td11">        
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-21-4"  
class="td11">     
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-22-1"  
class="td11"> 
Phosphorylation of CD3 and TCR zeta chains
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-22-2"  
class="td11">        
18
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-22-3"  
class="td11">         
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-22-4"  
class="td11">     
1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-23-1"  
class="td11"> 
Generation of second messenger molecules
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-23-2"  
class="td11">        
29
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-23-3"  
class="td11">         
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-23-4"  
class="td11">     
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-24-1"  
class="td11"> 
PD-1 signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-24-2"  
class="td11">        
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-24-3"  
class="td11">         
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-24-4"  
class="td11">     
7.4 <span class="cmsy-10x-x-120">×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-25-1"  
class="td11"> 
TCR signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-25-2"  
class="td11">        
62
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-25-3"  
class="td11">         
9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-25-4"  
class="td11">     
4.3 <span class="cmsy-10x-x-120">×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-26-1"  
class="td11"> 
Translocation of ZAP-70 to Immunological synapse
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-26-2"  
class="td11">        
16
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-26-3"  
class="td11">         
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-26-4"  
class="td11">     
1.1 <span class="cmsy-10x-x-120">×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-27-1"  
class="td11"> 
Interferon alpha/beta signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-27-2"  
class="td11">        
68
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-27-3"  
class="td11">         
9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-27-4"  
class="td11">     
1.6 <span class="cmsy-10x-x-120">×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-28-1"  
class="td11"> 
Initial triggering of complement
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-28-2"  
class="td11">        
17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-28-3"  
class="td11">         
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-28-4"  
class="td11">     
1.6 <span class="cmsy-10x-x-120">×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-29-1"  
class="td11"> 
IKK complex recruitment mediated by RIP1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-29-2"  
class="td11">        
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-29-3"  
class="td11">         
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-29-4"  
class="td11">     
5.1 <span class="cmsy-10x-x-120">×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-30-1"  
class="td11"> 
TRIF-mediated programmed cell death
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-30-2"  
class="td11">        
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-30-3"  
class="td11">         
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-30-4"  
class="td11">     
6.2 <span class="cmsy-10x-x-120">×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-31-1"  
class="td11"> 
Creation of C4 and C2 activators
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-31-2"  
class="td11">        
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-31-3"  
class="td11">         
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-31-4"  
class="td11">     
1.3 <span class="cmsy-10x-x-120">×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-32-1"  
class="td11"> 
RHO GTPases Activate NADPH Oxidases
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-32-2"  
class="td11">        
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-32-3"  
class="td11">         
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-32-4"  
class="td11">     
1.3 <span class="cmsy-10x-x-120">×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-33-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-33-1"  
class="td11"> 
Interferon Signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-33-2"  
class="td11">       
175
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-33-3"  
class="td11">        
15
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-33-4"  
class="td11">     
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-34-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-34-1"  
class="td11"> 
Chemokine receptors bind chemokines
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-34-2"  
class="td11">        
52
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-34-3"  
class="td11">         
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-34-4"  
class="td11">     
4<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-35-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-35-1"  
class="td11"> 
Interferon gamma signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-35-2"  
class="td11">        
74
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-35-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-35-4"  
class="td11">     
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-36-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-36-1"  
class="td11"> 
TRAF6 mediated induction of TAK1 complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-36-2"  
class="td11">        
15
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-36-3"  
class="td11">         
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-36-4"  
class="td11">     
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-37-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-37-1"  
class="td11"> 
Activation of IRF3/IRF7 mediated by TBK1/IKK epsilon
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-37-2"  
class="td11">        
16
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-37-3"  
class="td11">         
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-37-4"  
class="td11">     
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-38-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-38-1"  
class="td11"> 
Downstream TCR signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-38-2"  
class="td11">        
45
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-38-3"  
class="td11">         
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-38-4"  
class="td11">     
3<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-95-39-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-39-1"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 3</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-39-2"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-39-3"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-39-4"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">p-value (</span><a 
href="#glo:FDR"><span class="cmbx-12x-x-120">FDR</span></a><span class="cmbx-12x-x-120">)</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-95-40-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-40-1"  
class="td11"> 
Uptake and actions of bacterial toxins
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-40-2"  
class="td11">        
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-40-3"  
class="td11">         
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-40-4"  
class="td11">     
3<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-41-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-41-1"  
class="td11"> 
Neurotoxicity of clostridium toxins
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-41-2"  
class="td11">        
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-41-3"  
class="td11">         
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-41-4"  
class="td11">     
3<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-42-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-42-1"  
class="td11"> 
Activation of PPARGC1A (PGC-1alpha) by phosphorylation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-42-2"  
class="td11">        
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-42-3"  
class="td11">         
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-42-4"  
class="td11">     
3<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-43-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-43-1"  
class="td11"> 
SMAD2/SMAD3:SMAD4 heterotrimer regulates transcription
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-43-2"  
class="td11">        
28
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-43-3"  
class="td11">         
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-43-4"  
class="td11">     
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-44-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-44-1"  
class="td11"> 
Assembly of the primary cilium
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-44-2"  
class="td11">       
149
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-44-3"  
class="td11">        
10
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-44-4"  
class="td11">     
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-45-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-45-1"  
class="td11"> 
Serotonin Neurotransmitter Release Cycle
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-45-2"  
class="td11">        
15
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-45-3"  
class="td11">         
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-45-4"  
class="td11">     
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-46-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-46-1"  
class="td11"> 
Glycosaminoglycan metabolism
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-46-2"  
class="td11">       
114
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-46-3"  
class="td11">         
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-46-4"  
class="td11">     
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-47-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-47-1"  
class="td11"> 
Platelet homeostasis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-47-2"  
class="td11">        
54
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-47-3"  
class="td11">         
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-47-4"  
class="td11">     
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-48-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-48-1"  
class="td11"> 
Norepinephrine Neurotransmitter Release Cycle
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-48-2"  
class="td11">        
17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-48-3"  
class="td11">         
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-48-4"  
class="td11">     
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-49-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-49-1"  
class="td11"> 
Acetylcholine Neurotransmitter Release Cycle
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-49-2"  
class="td11">        
17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-49-3"  
class="td11">         
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-49-4"  
class="td11">     
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-50-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-50-1"  
class="td11"> 
G<sub><span class="cmmi-8">αs</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-50-2"  
class="td11">       
100
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-50-3"  
class="td11">         
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-50-4"  
class="td11">     
5<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-51-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-51-1"  
class="td11"> 
GABA synthesis, release, reuptake and degradation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-51-2"  
class="td11">        
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-51-3"  
class="td11">         
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-51-4"  
class="td11">     
5<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-52-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-52-1"  
class="td11"> 
deactivation of the beta-catenin transactivating complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-52-2"  
class="td11">        
39
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-52-3"  
class="td11">         
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-52-4"  
class="td11">     
6<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-53-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-53-1"  
class="td11"> 
Dopamine Neurotransmitter Release Cycle
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-53-2"  
class="td11">        
20
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-53-3"  
class="td11">         
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-53-4"  
class="td11">     
6<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-54-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-54-1"  
class="td11"> 
IRS-related events triggered by IGF1R
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-54-2"  
class="td11">        
83
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-54-3"  
class="td11">         
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-54-4"  
class="td11">     
7<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-55-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-55-1"  
class="td11"> 
Generic Transcription Pathway
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-55-2"  
class="td11">       
186
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-55-3"  
class="td11">        
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-55-4"  
class="td11">     
7<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-56-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-56-1"  
class="td11"> 
Termination of O-glycan biosynthesis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-56-2"  
class="td11">        
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-56-3"  
class="td11">         
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-56-4"  
class="td11">     
7<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-57-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-57-1"  
class="td11"> 
Kinesins
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-57-2"  
class="td11">        
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-57-3"  
class="td11">         
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-57-4"  
class="td11">     
8<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-95-58-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-58-1"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 4</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-58-2"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-58-3"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-58-4"  
class="td11"> 
<strong><span class="cmbx-12x-x-120">p-value (</span><a 
href="#glo:FDR"><span class="cmbx-12x-x-120">FDR</span></a><span class="cmbx-12x-x-120">)</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-95-59-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-59-1"  
class="td11"> 
Extracellular matrix organization
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-59-2"  
class="td11">       
241
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-59-3"  
class="td11">        
97
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-59-4"  
class="td11">    
8<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">126</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-60-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-60-1"  
class="td11"> 
Axon guidance
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-60-2"  
class="td11">       
289
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-60-3"  
class="td11">        
75
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-60-4"  
class="td11">     
8<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">72</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-61-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-61-1"  
class="td11"> 
Hemostasis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-61-2"  
class="td11">       
445
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-61-3"  
class="td11">        
101
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-61-4"  
class="td11">     
8<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">72</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-62-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-62-1"  
class="td11"> 
Developmental Biology
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-62-2"  
class="td11">       
432
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-62-3"  
class="td11">        
95
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-62-4"  
class="td11">     
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">67</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-63-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-63-1"  
class="td11"> 
Response to elevated platelet cytosolic Ca<sup><span class="cmr-8">2+</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-63-2"  
class="td11">        
84
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-63-3"  
class="td11">        
37
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-63-4"  
class="td11">     
5<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">67</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-64-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-64-1"  
class="td11"> 
Platelet degranulation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-64-2"  
class="td11">        
79
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-64-3"  
class="td11">        
36
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-64-4"  
class="td11">     
5<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">67</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-65-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-65-1"  
class="td11"> 
Degradation of the extracellular matrix
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-65-2"  
class="td11">       
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-65-3"  
class="td11">        
39
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-65-4"  
class="td11">     
6<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">63</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-66-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-66-1"  
class="td11"> 
Platelet activation, signalling and aggregation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-66-2"  
class="td11">       
186
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-66-3"  
class="td11">        
52
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-66-4"  
class="td11">     
6<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">62</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-67-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-67-1"  
class="td11"> 
ECM proteoglycans
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-67-2"  
class="td11">        
66
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-67-3"  
class="td11">        
31
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-67-4"  
class="td11">     
8<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">61</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-68-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-68-1"  
class="td11"> 
Neuronal System
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-68-2"  
class="td11">       
272
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-68-3"  
class="td11">        
64
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-68-4"  
class="td11">     
5<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">60</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-69-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-69-1"  
class="td11"> 
Signalling by PDGF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-69-2"  
class="td11">       
173
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-69-3"  
class="td11">        
47
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-69-4"  
class="td11">     
9<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">57</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-70-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-70-1"  
class="td11"> 
Integrin cell surface interactions
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-70-2"  
class="td11">        
82
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-70-3"  
class="td11">        
31
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-70-4"  
class="td11">     
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">53</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-71-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-71-1"  
class="td11"> 
Collagen biosynthesis and modifying enzymes
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-71-2"  
class="td11">        
56
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-71-3"  
class="td11">        
26
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-71-4"  
class="td11">     
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">52</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-72-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-72-1"  
class="td11"> 
Collagen formation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-72-2"  
class="td11">        
67
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-72-3"  
class="td11">        
28
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-72-4"  
class="td11">     
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">52</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-73-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-73-1"  
class="td11"> 
Class A/1 (Rhodopsin-like receptors)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-73-2"  
class="td11">       
289
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-73-3"  
class="td11">        
61
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-73-4"  
class="td11">     
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">52</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-74-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-74-1"  
class="td11"> 
GPCR ligand binding
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-74-2"  
class="td11">       
373
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-74-3"  
class="td11">        
73
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-74-4"  
class="td11">     
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">52</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-75-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-75-1"  
class="td11"> 
Elastic fibre formation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-75-2"  
class="td11">        
38
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-75-3"  
class="td11">        
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-75-4"  
class="td11">     
4<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">52</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-95-76-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-76-1"  
class="td11"> 
Non-integrin membrane-ECM interactions
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-76-2"  
class="td11">        
53
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-76-3"  
class="td11">        
24
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-95-76-4"  
class="td11">     
7<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">49</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-95-77-"><td  style="white-space:nowrap; text-align:left;" id="TBL-95-77-1"  
class="td11">                                                                   </td></tr></table>
<!--l. 1765--><p class="noindent" ></div>
<p>Pathway over-representation analysis for Reactome pathways with the number of genes in each pathway (Pathway Size), number of genes within the pathway identified (Cluster Genes), and the pathway over-representation p-value (adjusted by <a 
href="#glo:FDR">FDR</a>) from the hypergeometric test.</p>
</p>
</div><hr class="endfloat" />
<a 
 id="x1-182005r424"></a>
<h3 class="sectionHead">
<span class="titlemark">F.2 </span> <a 
 id="x1-1830002"></a>Comparison to Primary Screen
</h3>
<!--l. 1775--><p class="noindent" >
The synthetic lethal partners with <span class="cmti-12">CDH1 </span>expression in stomach cancers were also compared to <a 
href="#glo:siRNA">siRNA</a> primary screen data (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), as performed in Section <a 
href="#x1-1020001">4.2.1<!--tex4ht:ref: chapt3:primary_screen --></a>. These were expected to be more concordant with the experimental results performed on a null mutant, however this was not the case at the gene level: less genes overlapped with experimental candidates in Figure <a 
href="#x1-1830012">F.2<!--tex4ht:ref: fig:Venn_allgenes_stad --></a>. This may be due to lower sample size for mutations in <a 
href="#glo:TCGA">TCGA</a> data or lower frequency (expected value) of <span class="cmti-12">CDH1 </span>mutations compared to low expression.
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                    

<p><a 
 id="x1-1830012"></a></p>
<!--l. 1782--><p class="noindent" >
<img 
src="Venn_exprSL_siRNA_allgenes_reduced_Pub_stad.png" alt="PIC"  
 /> <br />
</p><div class="caption" 
>
<span class="id">Figure F.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Comparison of SLIPT in stomach to </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a><span class="cmbx-10x-x-109">.</span></strong> <span class="cmr-10x-x-109">The overlap of gene</span> <span class="cmr-10x-x-109">candidates for </span><a 
href="#glo:E-cadherin"><span class="cmr-10x-x-109">E-cadherin</span></a> <span class="cmr-10x-x-109">synthetic lethal partners between computational (SLIPT) and</span> <span class="cmr-10x-x-109">experimental screening (siRNA) approaches. The </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test suggests that the overlap is no more</span> <span class="cmr-10x-x-109">than would be expected by chance (</span><span class="cmmi-10x-x-109">p </span><span class="cmr-10x-x-109">= 0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">281). </span></span>
</div><!--tex4ht:label?: x1-1830012 -->
                                                                                    

</div><hr class="endfigure" />
                                                                                    

<!--l. 1796--><p class="indent" >
<a 
 id="x1-1830024"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table F.4: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT and <a 
href="#glo:siRNA">siRNA</a></span>
</div><!--tex4ht:label?: x1-1830024 -->
<!--tex4ht:inline--><div class="tabular"><table id="TBL-96" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-96-1g"><col 
id="TBL-96-1" /><col 
id="TBL-96-2" /><col 
id="TBL-96-3" /><col 
id="TBL-96-4" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-96-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-1-1"  
class="td11"> 
<span class="cmbx-12">Predicted only by SLIPT (3392 genes) </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-1-2"  
class="td11"> 
Pathway Size
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-1-3"  
class="td11"> 
Genes Identified
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-1-4"  
class="td11"> 
p-value (<a 
href="#glo:FDR">FDR</a>)
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-96-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-2-1"  
class="td11"> 
Extracellular matrix organization
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-2-2"  
class="td11">     
238
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-2-3"  
class="td11">       
90
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-2-4"  
class="td11">  
3<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">107</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-3-1"  
class="td11"> 
Eukaryotic Translation Termination
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-3-2"  
class="td11">      
79
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-3-3"  
class="td11">       
46
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-3-4"  
class="td11">  
7<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">91</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-4-1"  
class="td11"> 
Viral <a 
href="#glo:mRNA">mRNA</a> Translation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-4-2"  
class="td11">      
77
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-4-3"  
class="td11">       
45
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-4-4"  
class="td11">  
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">89</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-5-1"  
class="td11"> 
Eukaryotic Translation Elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-5-2"  
class="td11">      
82
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-5-3"  
class="td11">       
46
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-5-4"  
class="td11">  
5<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">89</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-6-1"  
class="td11"> 
Peptide chain elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-6-2"  
class="td11">      
79
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-6-3"  
class="td11">       
45
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-6-4"  
class="td11">  
2<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">88</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-7-1"  
class="td11"> 
Nonsense Mediated Decay independent of the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-7-2"  
class="td11">      
84
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-7-3"  
class="td11">       
46
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-7-4"  
class="td11">  
9<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">88</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-8-1"  
class="td11"> 
Formation of a pool of free 40S subunits
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-8-2"  
class="td11">      
89
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-8-3"  
class="td11">       
47
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-8-4"  
class="td11">  
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">87</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-9-1"  
class="td11"> 
GTP hydrolysis and joining of the 60S ribosomal subunit
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-9-2"  
class="td11">     
100
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-9-3"  
class="td11">       
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-9-4"  
class="td11">  
3<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">83</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-10-1"  
class="td11"> 
Axon guidance
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-10-2"  
class="td11">     
284
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-10-3"  
class="td11">       
84
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-10-4"  
class="td11">  
3<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">82</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-11-1"  
class="td11"> 
Developmental Biology
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-11-2"  
class="td11">     
426
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-11-3"  
class="td11">      
111
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-11-4"  
class="td11">  
4<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">82</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-12-1"  
class="td11"> 
L13a-mediated translational silencing of Ceruloplasmin expression
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-12-2"  
class="td11">      
99
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-12-3"  
class="td11">       
47
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-12-4"  
class="td11">  
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">81</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-13-1"  
class="td11"> 
3’ -UTR-mediated translational regulation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-13-2"  
class="td11">      
99
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-13-3"  
class="td11">       
47
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-13-4"  
class="td11">  
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">81</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-14-1"  
class="td11"> 
SRP-dependent cotranslational protein targeting to membrane
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-14-2"  
class="td11">      
99
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-14-3"  
class="td11">       
47
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-14-4"  
class="td11">  
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">81</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-15-1"  
class="td11"> 
Nonsense-Mediated Decay
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-15-2"  
class="td11">      
99
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-15-3"  
class="td11">       
47
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-15-4"  
class="td11">  
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">81</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-16-1"  
class="td11"> 
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-16-2"  
class="td11">      
99
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-16-3"  
class="td11">       
47
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-16-4"  
class="td11">  
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">81</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-17-1"  
class="td11"> 
Hemostasis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-17-2"  
class="td11">     
438
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-17-3"  
class="td11">      
112
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-17-4"  
class="td11">  
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-18-1"  
class="td11"> 
Eukaryotic Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-18-2"  
class="td11">     
107
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-18-3"  
class="td11">       
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-18-4"  
class="td11">  
8<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-19-1"  
class="td11"> 
Cap-dependent Translation Initiation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-19-2"  
class="td11">     
107
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-19-3"  
class="td11">       
48
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-19-4"  
class="td11">  
8<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-20-1"  
class="td11"> 
Infectious disease
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-20-2"  
class="td11">     
338
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-20-3"  
class="td11">       
90
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-20-4"  
class="td11">  
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">76</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-21-1"  
class="td11"> 
Neuronal System
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-21-2"  
class="td11">     
267
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-21-3"  
class="td11">       
77
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-21-4"  
class="td11">  
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">76</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-96-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-22-1"  
class="td11">                                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-23-1"  
class="td11"> 
<span class="cmbx-12">Detected only by </span><a 
href="#glo:siRNA"><span class="cmbx-12">siRNA</span></a> <span class="cmbx-12">screen (1803 genes) </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-23-2"  
class="td11"> 
Pathway Size
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-23-3"  
class="td11"> 
Genes Identified
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-23-4"  
class="td11"> 
p-value (<a 
href="#glo:FDR">FDR</a>)
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-96-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-24-1"  
class="td11"> 
Class A/1 (Rhodopsin-like receptors)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-24-2"  
class="td11">     
282
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-24-3"  
class="td11">       
62
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-24-4"  
class="td11">  
8<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">50</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-25-1"  
class="td11"> 
GPCR ligand binding
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-25-2"  
class="td11">     
363
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-25-3"  
class="td11">       
71
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-25-4"  
class="td11">  
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">46</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-26-1"  
class="td11"> 
Peptide ligand-binding receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-26-2"  
class="td11">     
175
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-26-3"  
class="td11">       
38
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-26-4"  
class="td11">  
7<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">38</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-27-1"  
class="td11"> 
G<sub><span class="cmmi-8">αi</span></sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-27-2"  
class="td11">     
184
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-27-3"  
class="td11">       
37
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-27-4"  
class="td11">  
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">34</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-28-1"  
class="td11"> 
Gastrin-CREB signalling pathway via PKC and MAPK
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-28-2"  
class="td11">     
180
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-28-3"  
class="td11">       
35
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-28-4"  
class="td11">  
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">32</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-29-1"  
class="td11"> 
G<sub><span class="cmmi-8">αq</span></sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-29-2"  
class="td11">     
159
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-29-3"  
class="td11">       
32
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-29-4"  
class="td11">  
4<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">32</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-30-1"  
class="td11"> 
DAP12 interactions
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-30-2"  
class="td11">     
159
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-30-3"  
class="td11">       
29
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-30-4"  
class="td11">  
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-31-1"  
class="td11"> 
Downstream signal transduction
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-31-2"  
class="td11">     
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-31-3"  
class="td11">       
26
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-31-4"  
class="td11">  
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-32-1"  
class="td11"> 
DAP12 signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-32-2"  
class="td11">     
149
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-32-3"  
class="td11">       
26
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-32-4"  
class="td11">  
6<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-33-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-33-1"  
class="td11"> 
VEGFA-VEGFR2 Pathway
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-33-2"  
class="td11">      
91
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-33-3"  
class="td11">       
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-33-4"  
class="td11">  
8<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-34-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-34-1"  
class="td11"> 
Signalling by PDGF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-34-2"  
class="td11">     
172
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-34-3"  
class="td11">       
27
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-34-4"  
class="td11">  
5<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">23</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-35-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-35-1"  
class="td11"> 
Signalling by ERBB2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-35-2"  
class="td11">     
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-35-3"  
class="td11">       
24
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-35-4"  
class="td11">  
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-36-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-36-1"  
class="td11"> 
Signalling by VEGF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-36-2"  
class="td11">      
99
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-36-3"  
class="td11">       
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-36-4"  
class="td11">  
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-37-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-37-1"  
class="td11"> 
Visual phototransduction
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-37-2"  
class="td11">      
85
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-37-3"  
class="td11">       
17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-37-4"  
class="td11">  
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-38-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-38-1"  
class="td11"> 
Downstream signalling of activated FGFR1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-38-2"  
class="td11">     
134
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-38-3"  
class="td11">       
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-38-4"  
class="td11">  
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-39-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-39-1"  
class="td11"> 
Downstream signalling of activated FGFR2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-39-2"  
class="td11">     
134
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-39-3"  
class="td11">       
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-39-4"  
class="td11">  
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-40-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-40-1"  
class="td11"> 
Downstream signalling of activated FGFR3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-40-2"  
class="td11">     
134
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-40-3"  
class="td11">       
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-40-4"  
class="td11">  
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-41-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-41-1"  
class="td11"> 
Downstream signalling of activated FGFR4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-41-2"  
class="td11">     
134
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-41-3"  
class="td11">       
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-41-4"  
class="td11">  
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-42-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-42-1"  
class="td11"> 
Signalling by FGFR
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-42-2"  
class="td11">     
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-42-3"  
class="td11">       
23
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-42-4"  
class="td11">  
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-43-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-43-1"  
class="td11"> 
Signalling by FGFR1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-43-2"  
class="td11">     
146
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-43-3"  
class="td11">       
23
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-43-4"  
class="td11">  
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-96-44-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-44-1"  
class="td11">                                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-45-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-45-1"  
class="td11"> 
<span class="cmbx-12">Intersection of SLIPT and </span><a 
href="#glo:siRNA"><span class="cmbx-12">siRNA</span></a> <span class="cmbx-12">screen (547 genes) </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-45-2"  
class="td11"> 
Pathway Size
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-45-3"  
class="td11"> 
Genes Identified
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-45-4"  
class="td11"> 
p-value (<a 
href="#glo:FDR">FDR</a>)
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-96-46-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-46-1"  
class="td11"> 
Class A/1 (Rhodopsin-like receptors)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-46-2"  
class="td11">     
282
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-46-3"  
class="td11">       
25
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-46-4"  
class="td11">   
3<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-47-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-47-1"  
class="td11"> 
Platelet activation, signalling and aggregation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-47-2"  
class="td11">     
182
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-47-3"  
class="td11">       
17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-47-4"  
class="td11">   
3<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-48-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-48-1"  
class="td11"> 
Response to elevated platelet cytosolic Ca2<sup><span class="cmr-8">+</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-48-2"  
class="td11">      
82
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-48-3"  
class="td11">       
9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-48-4"  
class="td11">   
5<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-49-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-49-1"  
class="td11"> 
Platelet homeostasis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-49-2"  
class="td11">      
53
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-49-3"  
class="td11">       
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-49-4"  
class="td11">   
5<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-50-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-50-1"  
class="td11"> 
Nucleotide-like (purinergic) receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-50-2"  
class="td11">      
16
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-50-3"  
class="td11">       
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-50-4"  
class="td11">   
1<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-51-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-51-1"  
class="td11"> 
Platelet degranulation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-51-2"  
class="td11">      
77
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-51-3"  
class="td11">       
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-51-4"  
class="td11">   
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-52-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-52-1"  
class="td11"> 
Peptide ligand-binding receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-52-2"  
class="td11">     
175
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-52-3"  
class="td11">       
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-52-4"  
class="td11">   
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-53-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-53-1"  
class="td11"> 
Molecules associated with elastic fibres
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-53-2"  
class="td11">      
34
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-53-3"  
class="td11">       
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-53-4"  
class="td11">   
7<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-54-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-54-1"  
class="td11"> 
Amine ligand-binding receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-54-2"  
class="td11">      
35
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-54-3"  
class="td11">       
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-54-4"  
class="td11">   
8<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-55-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-55-1"  
class="td11"> 
G<sub><span class="cmmi-8">αi</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-55-2"  
class="td11">     
184
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-55-3"  
class="td11">       
14
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-55-4"  
class="td11">   
9<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-56-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-56-1"  
class="td11"> 
GPCR ligand binding
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-56-2"  
class="td11">     
363
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-56-3"  
class="td11">       
27
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-56-4"  
class="td11">   
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-57-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-57-1"  
class="td11"> 
Elastic fibre formation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-57-2"  
class="td11">      
38
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-57-3"  
class="td11">       
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-57-4"  
class="td11">   
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-58-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-58-1"  
class="td11"> 
G<sub><span class="cmmi-8">αq</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-58-2"  
class="td11">     
159
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-58-3"  
class="td11">       
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-58-4"  
class="td11">   
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-59-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-59-1"  
class="td11"> 
Serotonin receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-59-2"  
class="td11">      
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-59-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-59-4"  
class="td11">   
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-60-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-60-1"  
class="td11"> 
P2Y receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-60-2"  
class="td11">      
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-60-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-60-4"  
class="td11">   
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-61-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-61-1"  
class="td11"> 
Signal amplification
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-61-2"  
class="td11">      
16
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-61-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-61-4"  
class="td11">   
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-62-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-62-1"  
class="td11"> 
Gastrin-CREB signalling pathway via PKC and MAPK
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-62-2"  
class="td11">     
180
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-62-3"  
class="td11">       
12
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-62-4"  
class="td11">   
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-63-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-63-1"  
class="td11"> 
Complement cascade
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-63-2"  
class="td11">      
33
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-63-3"  
class="td11">       
4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-63-4"  
class="td11">   
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-64-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-64-1"  
class="td11"> 
Glycosaminoglycan metabolism
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-64-2"  
class="td11">     
110
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-64-3"  
class="td11">       
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-64-4"  
class="td11">   
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-96-65-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-65-1"  
class="td11"> 
Glycogen breakdown (glycogenolysis)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-65-2"  
class="td11">      
17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-65-3"  
class="td11">       
3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-96-65-4"  
class="td11">   
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-96-66-"><td  style="white-space:nowrap; text-align:left;" id="TBL-96-66-1"  
class="td11">                                                             </td></tr></table>                                                         </div>
                                                                                    

</div><hr class="endfloat" />
<a 
 id="x1-183003r408"></a>
<h4 class="subsectionHead">
<span class="titlemark">F.2.1 </span> <a 
 id="x1-1840001"></a>Resampling Analysis
</h4>
                                                                                    

<!--l. 1954--><p class="indent" >
<a 
 id="x1-1840015"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table F.5: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in stomach cancer</span>
</div><!--tex4ht:label?: x1-1840015 -->
<div class="tabular">
<table id="TBL-97" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-97-1g"><col 
id="TBL-97-1" /><col 
id="TBL-97-2" /><col 
id="TBL-97-3" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-97-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-1-1"  
class="td11"> 
<span class="cmbx-12">Reactome Pathway </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-1-2"  
class="td11"> 
Over-representation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-1-3"  
class="td11">  
Permutation
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-97-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-2-1"  
class="td11"> 
<span class="cmti-12">Extracellular matrix organization </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-2-2"  
class="td11">    
7<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">140</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-2-3"  
class="td11">    
0<span class="cmmi-12">.</span>070215
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-3-1"  
class="td11"> 
Hemostasis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-3-2"  
class="td11">    
1<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">121</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-3-3"  
class="td11">    
0<span class="cmmi-12">.</span>25804
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-4-1"  
class="td11"> 
Developmental Biology
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-4-2"  
class="td11">    
9<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">107</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-4-3"  
class="td11">    
0<span class="cmmi-12">.</span>53032
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-5-1"  
class="td11"> 
Axon guidance
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-5-2"  
class="td11">    
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">102</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-5-3"  
class="td11">     
0<span class="cmmi-12">.</span>6704
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-6-1"  
class="td11"> 
<strong><span class="cmbx-12">Eukaryotic Translation Termination</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-6-2"  
class="td11">     
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">99</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-6-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-7-1"  
class="td11"> 
GPCR ligand binding
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-7-2"  
class="td11">     
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">99</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-7-3"  
class="td11">    
0<span class="cmmi-12">.</span>54914
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-8-1"  
class="td11"> 
<strong><span class="cmbx-12">Viral </span><a 
href="#glo:mRNA"><span class="cmbx-12">mRNA</span></a> <span class="cmbx-12">Translation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-8-2"  
class="td11">     
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">98</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-8-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-9-1"  
class="td11"> 
<strong><span class="cmbx-12">Formation of a pool of free 40S subunits</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-9-2"  
class="td11">     
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">98</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-9-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-10-1"  
class="td11"> 
<strong><span class="cmbx-12">Eukaryotic Translation Elongation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-10-2"  
class="td11">     
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-10-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-11-1"  
class="td11"> 
<strong><span class="cmbx-12">Peptide chain elongation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-11-2"  
class="td11">     
7<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-11-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-12-1"  
class="td11"> 
Class A/1 (Rhodopsin-like receptors)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-12-2"  
class="td11">     
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">96</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-12-3"  
class="td11">    
0<span class="cmmi-12">.</span>58174
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-13-1"  
class="td11"> 
<strong><span class="cmbx-12">Nonsense Mediated Decay independent of the Exon Junction Complex</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-13-2"  
class="td11">     
3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">96</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-13-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-14-1"  
class="td11"> 
Infectious disease
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-14-2"  
class="td11">     
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">94</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-14-3"  
class="td11">    
0<span class="cmmi-12">.</span>25484
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-15-1"  
class="td11"> 
<strong><span class="cmbx-12">GTP hydrolysis and joining of the 60S ribosomal subunit</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-15-2"  
class="td11">     
3<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">94</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-15-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-16-1"  
class="td11"> 
<strong><span class="cmbx-12">L13a-mediated translational silencing of Ceruloplasmin expression</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-16-2"  
class="td11">     
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-16-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-17-1"  
class="td11"> 
<strong><span class="cmbx-12">3’ -UTR-mediated translational regulation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-17-2"  
class="td11">     
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-17-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-18-1"  
class="td11"> 
Neuronal System
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-18-2"  
class="td11">     
8<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-18-3"  
class="td11">    
0<span class="cmmi-12">.</span>53433
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-19-1"  
class="td11"> 
<strong><span class="cmbx-12">SRP-dependent cotranslational protein targeting to membrane</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-19-2"  
class="td11">     
9<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-19-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-20-1"  
class="td11"> 
<strong><span class="cmbx-12">Eukaryotic Translation Initiation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-20-2"  
class="td11">     
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-20-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-21-1"  
class="td11"> 
<strong><span class="cmbx-12">Cap-dependent Translation Initiation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-21-2"  
class="td11">     
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-21-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-22-1"  
class="td11"> 
<strong><span class="cmbx-12">Nonsense-Mediated Decay</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-22-2"  
class="td11">     
7<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-22-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-23-1"  
class="td11"> 
<strong><span class="cmbx-12">Nonsense Mediated Decay enhanced by the Exon Junction Complex</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-23-2"  
class="td11">     
7<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-23-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-24-1"  
class="td11"> 
Adaptive Immune System
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-24-2"  
class="td11">     
8<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">88</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-24-3"  
class="td11">    
0<span class="cmmi-12">.</span>14116
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-25-1"  
class="td11"> 
<strong><span class="cmbx-12">Translation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-25-2"  
class="td11">     
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">87</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-25-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-26-1"  
class="td11"> 
Platelet activation, signalling and aggregation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-26-2"  
class="td11">     
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">86</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-26-3"  
class="td11">    
0<span class="cmmi-12">.</span>28959
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-27-1"  
class="td11"> 
<strong><span class="cmbx-12">Influenza Infection</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-27-2"  
class="td11">     
1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">82</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-27-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-28-1"  
class="td11"> 
<strong><span class="cmbx-12">Influenza Viral </span><a 
href="#glo:RNA"><span class="cmbx-12">RNA</span></a> <span class="cmbx-12">Transcription and Replication</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-28-2"  
class="td11">     
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">82</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-28-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-29-1"  
class="td11"> 
<strong><span class="cmbx-12">Influenza Life Cycle</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-29-2"  
class="td11">     
2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-29-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-30-1"  
class="td11"> 
Response to elevated platelet cytosolic Ca2<sup><span class="cmr-8">+</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-30-2"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">78</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-30-3"  
class="td11">    
0<span class="cmmi-12">.</span>50817
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-31-1"  
class="td11"> 
Signalling by NGF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-31-2"  
class="td11">     
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">75</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-31-3"  
class="td11">    
0<span class="cmmi-12">.</span>38518
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-32-1"  
class="td11"> 
Rho GTPase cycle
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-32-2"  
class="td11">     
5<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">75</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-32-3"  
class="td11">    
0<span class="cmmi-12">.</span>14864
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-33-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-33-1"  
class="td11"> 
Signalling by PDGF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-33-2"  
class="td11">     
7<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">74</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-33-3"  
class="td11">    
0<span class="cmmi-12">.</span>40493
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-34-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-34-1"  
class="td11"> 
<span class="cmti-12">Signalling by Rho GTPases </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-34-2"  
class="td11">     
5<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">73</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-34-3"  
class="td11">    
0<span class="cmmi-12">.</span>077217
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-35-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-35-1"  
class="td11"> 
Glycosaminoglycan metabolism
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-35-2"  
class="td11">     
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">68</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-35-3"  
class="td11">    
0<span class="cmmi-12">.</span>52984
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-36-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-36-1"  
class="td11"> 
G<sub><span class="cmmi-8">αi</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-36-2"  
class="td11">     
1<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">66</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-36-3"  
class="td11">     
0<span class="cmmi-12">.</span>9254
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-37-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-37-1"  
class="td11"> 
Metabolism of carbohydrates
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-37-2"  
class="td11">     
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">65</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-37-3"  
class="td11">    
0<span class="cmmi-12">.</span>39501
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-38-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-38-1"  
class="td11"> 
<strong><span class="cmbx-12">G</span><sub><span class="cmmi-8">αs</span> </sub> <span class="cmbx-12">signalling events</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-38-2"  
class="td11">     
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">65</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-38-3"  
class="td11">   
0<span class="cmmi-12">.</span>0050293
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-39-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-39-1"  
class="td11"> 
Potassium Channels
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-39-2"  
class="td11">     
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">65</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-39-3"  
class="td11">    
0<span class="cmmi-12">.</span>53359
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-40-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-40-1"  
class="td11"> 
Transmission across Chemical Synapses
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-40-2"  
class="td11">     
1<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">64</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-40-3"  
class="td11">    
0<span class="cmmi-12">.</span>81833
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-41-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-41-1"  
class="td11"> 
ECM proteoglycans
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-41-2"  
class="td11">     
3<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">64</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-41-3"  
class="td11">    
0<span class="cmmi-12">.</span>083482
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-42-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-42-1"  
class="td11"> 
Peptide ligand-binding receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-42-2"  
class="td11">     
4<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">64</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-42-3"  
class="td11">    
0<span class="cmmi-12">.</span>62817
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-43-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-43-1"  
class="td11"> 
Degradation of the extracellular matrix
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-43-2"  
class="td11">     
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">63</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-43-3"  
class="td11">    
0<span class="cmmi-12">.</span>80879
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-44-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-44-1"  
class="td11"> 
Platelet homeostasis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-44-2"  
class="td11">     
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">63</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-44-3"  
class="td11">    
0<span class="cmmi-12">.</span>53134
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-45-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-45-1"  
class="td11"> 
NGF signalling via TRKA from the plasma membrane
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-45-2"  
class="td11">     
6<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">63</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-45-3"  
class="td11">     
0<span class="cmmi-12">.</span>5717
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-46-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-46-1"  
class="td11"> 
Integration of energy metabolism
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-46-2"  
class="td11">     
4<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">61</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-46-3"  
class="td11">    
0<span class="cmmi-12">.</span>10889
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-47-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-47-1"  
class="td11"> 
Collagen formation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-47-2"  
class="td11">     
5<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">61</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-47-3"  
class="td11">    
0<span class="cmmi-12">.</span>29896
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-48-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-48-1"  
class="td11"> 
Integrin cell surface interactions
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-48-2"  
class="td11">     
7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">59</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-48-3"  
class="td11">    
0<span class="cmmi-12">.</span>18167
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-49-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-49-1"  
class="td11"> 
Collagen biosynthesis and modifying enzymes
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-49-2"  
class="td11">     
7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">59</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-49-3"  
class="td11">    
0<span class="cmmi-12">.</span>30208
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-50-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-50-1"  
class="td11"> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-98" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-98-1g"><col 
id="TBL-98-1" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-98-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-98-1-1"  
class="td00">
Neurotransmitter Receptor Binding And Downstream Transmission
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-98-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-98-2-1"  
class="td00">
In The Postsynaptic Cell
</td></tr></table>                    </div>  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-50-2"  
class="td11">     
8<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">57</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-50-3"  
class="td11">    
0<span class="cmmi-12">.</span>82522
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-97-51-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-51-1"  
class="td11"> 
Signalling by Wnt
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-51-2"  
class="td11">     
8<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">57</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-97-51-3"  
class="td11">    
0<span class="cmmi-12">.</span>25468
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-97-52-"><td  style="white-space:nowrap; text-align:left;" id="TBL-97-52-1"  
class="td11">                                                                         </td></tr></table>
<!--l. 2071--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Over-representation (hypergeometric test) and Permutation p-values adjusted for multiple tests across pathways (</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a><span class="cmr-10x-x-109">). Significant pathways</span> <span class="cmr-10x-x-109">were marked in bold (</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a> <span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05) and italics (</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a> <span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">1).</span></p>
</p>
</div><hr class="endfloat" />
                                                                                    

<!--l. 2075--><p class="indent" >
<a 
 id="x1-1840026"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table F.6: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in stomach and <a 
href="#glo:siRNA">siRNA</a></span>
</div><!--tex4ht:label?: x1-1840026 -->
<div class="tabular">
<table id="TBL-99" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-99-1g"><col 
id="TBL-99-1" /><col 
id="TBL-99-2" /><col 
id="TBL-99-3" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-99-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-1-1"  
class="td11"> 
<span class="cmbx-12">Reactome Pathway </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-1-2"  
class="td11"> 
Over-representation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-1-3"  
class="td11">  
Permutation
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-99-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-2-1"  
class="td11"> 
Platelet activation, signalling and aggregation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-2-2"  
class="td11">     
3<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-2-3"  
class="td11">     
0<span class="cmmi-12">.</span>49557
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-3-1"  
class="td11"> 
Class A/1 (Rhodopsin-like receptors)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-3-2"  
class="td11">     
3<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-3-3"  
class="td11">     
0<span class="cmmi-12">.</span>98432
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-4-1"  
class="td11"> 
Response to elevated platelet cytosolic Ca2<sup><span class="cmr-8">+</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-4-2"  
class="td11">     
5<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-4-3"  
class="td11">     
0<span class="cmmi-12">.</span>54349
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-5-1"  
class="td11"> 
Platelet homeostasis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-5-2"  
class="td11">     
5<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-5-3"  
class="td11">     
0<span class="cmmi-12">.</span>45017
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-6-1"  
class="td11"> 
Nucleotide-like (purinergic) receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-6-2"  
class="td11">     
1<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-6-3"  
class="td11">     
0<span class="cmmi-12">.</span>36966
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-7-1"  
class="td11"> 
Peptide ligand-binding receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-7-2"  
class="td11">     
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-7-3"  
class="td11">     
0<span class="cmmi-12">.</span>91294
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-8-1"  
class="td11"> 
<strong><span class="cmbx-12">Molecules associated with elastic fibres</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-8-2"  
class="td11">     
7<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-8-3"  
class="td11">    
0<span class="cmmi-12">.</span>0025868
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-9-1"  
class="td11"> 
Amine ligand-binding receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-9-2"  
class="td11">     
8<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-9-3"  
class="td11">     
0<span class="cmmi-12">.</span>43303
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-10-1"  
class="td11"> 
G<sub><span class="cmmi-8">αi</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-10-2"  
class="td11">     
9<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-10-3"  
class="td11">     
0<span class="cmmi-12">.</span>99626
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-11-1"  
class="td11"> 
GPCR ligand binding
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-11-2"  
class="td11">     
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-11-3"  
class="td11">     
0<span class="cmmi-12">.</span>97733
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-12-1"  
class="td11"> 
<strong><span class="cmbx-12">Elastic fibre formation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-12-2"  
class="td11">     
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-12-3"  
class="td11">    
0<span class="cmmi-12">.</span>0025868
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-13-1"  
class="td11"> 
G<sub><span class="cmmi-8">αq</span> </sub> signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-13-2"  
class="td11">     
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-13-3"  
class="td11">     
0<span class="cmmi-12">.</span>86089
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-14-1"  
class="td11"> 
P2Y receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-14-2"  
class="td11">     
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-14-3"  
class="td11">     
0<span class="cmmi-12">.</span>18795
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-15-1"  
class="td11"> 
Serotonin receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-15-2"  
class="td11">     
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-15-3"  
class="td11">     
0<span class="cmmi-12">.</span>37853
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-16-1"  
class="td11"> 
Signal amplification
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-16-2"  
class="td11">     
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-16-3"  
class="td11">     
0<span class="cmmi-12">.</span>47856
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-17-1"  
class="td11"> 
Gastrin-CREB signalling pathway via PKC and MAPK
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-17-2"  
class="td11">     
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-17-3"  
class="td11">     
0<span class="cmmi-12">.</span>98567
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-18-1"  
class="td11"> 
<strong><span class="cmbx-12">Complement cascade</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-18-2"  
class="td11">     
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-18-3"  
class="td11"> 
<span class="cmmi-12">&gt; </span>3<span class="cmmi-12">.</span>4628 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-19-1"  
class="td11"> 
Glycosaminoglycan metabolism
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-19-2"  
class="td11">     
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-19-3"  
class="td11">     
0<span class="cmmi-12">.</span>38953
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-20-1"  
class="td11"> 
Glycogen breakdown (glycogenolysis)
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-20-2"  
class="td11">     
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-20-3"  
class="td11">     
0<span class="cmmi-12">.</span>83772
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-21-1"  
class="td11"> 
Defective B4GALT7 causes EDS, progeroid type
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-21-2"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-21-3"  
class="td11">     
0<span class="cmmi-12">.</span>10792
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-22-1"  
class="td11"> 
Defective B3GAT3 causes JDSSDHD
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-22-2"  
class="td11">     
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-22-3"  
class="td11">     
0<span class="cmmi-12">.</span>10792
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-23-1"  
class="td11"> 
Role of LAT2/NTAL/LAB on calcium mobilization
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-23-2"  
class="td11">     
5<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-23-3"  
class="td11">     
0<span class="cmmi-12">.</span>35373
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-24-1"  
class="td11"> 
Cell surface interactions at the vascular wall
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-24-2"  
class="td11">     
5<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-24-3"  
class="td11">     
0<span class="cmmi-12">.</span>47642
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-25-1"  
class="td11"> 
<strong><span class="cmbx-12">G</span><sub><span class="cmmi-8">αs</span> </sub> <span class="cmbx-12">signalling events</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-25-2"  
class="td11">      
6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-25-3"  
class="td11">    
0<span class="cmmi-12">.</span>019858
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-26-1"  
class="td11"> 
Signalling by NOTCH
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-26-2"  
class="td11">      
6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-26-3"  
class="td11">     
0<span class="cmmi-12">.</span>19008
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-27-1"  
class="td11"> 
A tetrasaccharide linker sequence is required for GAG synthesis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-27-2"  
class="td11">      
0<span class="cmmi-12">.</span>00017
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-27-3"  
class="td11">     
0<span class="cmmi-12">.</span>47642
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-28-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-28-1"  
class="td11"> 
<strong><span class="cmbx-12">Extracellular matrix organization</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-28-2"  
class="td11">      
0<span class="cmmi-12">.</span>00018
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-28-3"  
class="td11">    
0<span class="cmmi-12">.</span>0047308
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-29-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-29-1"  
class="td11"> 
Collagen formation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-29-2"  
class="td11">      
0<span class="cmmi-12">.</span>00018
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-29-3"  
class="td11">     
0<span class="cmmi-12">.</span>19245
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-30-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-30-1"  
class="td11"> 
Effects of PIP2 hydrolysis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-30-2"  
class="td11">       
0<span class="cmmi-12">.</span>0002
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-30-3"  
class="td11">     
0<span class="cmmi-12">.</span>37779
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-31-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-31-1"  
class="td11"> 
Syndecan interactions
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-31-2"  
class="td11">       
0<span class="cmmi-12">.</span>0002
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-31-3"  
class="td11">     
0<span class="cmmi-12">.</span>37779
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-32-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-32-1"  
class="td11"> 
<strong><span class="cmbx-12">Diseases associated with glycosaminoglycan metabolism</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-32-2"  
class="td11">      
0<span class="cmmi-12">.</span>00023
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-32-3"  
class="td11">     
0<span class="cmmi-12">.</span>01028
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-33-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-33-1"  
class="td11"> 
<strong><span class="cmbx-12">Diseases of glycosylation</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-33-2"  
class="td11">      
0<span class="cmmi-12">.</span>00023
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-33-3"  
class="td11">     
0<span class="cmmi-12">.</span>01028
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-34-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-34-1"  
class="td11"> 
<span class="cmti-12">Chondroitin sulfate/dermatan sulfate metabolism </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-34-2"  
class="td11">      
0<span class="cmmi-12">.</span>00023
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-34-3"  
class="td11">    
0<span class="cmmi-12">.</span>085541
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-35-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-35-1"  
class="td11"> 
Integrin alphaIIb beta3 signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-35-2"  
class="td11">      
0<span class="cmmi-12">.</span>00028
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-35-3"  
class="td11">     
0<span class="cmmi-12">.</span>76936
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-36-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-36-1"  
class="td11"> 
Keratan sulfate biosynthesis
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-36-2"  
class="td11">      
0<span class="cmmi-12">.</span>00034
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-36-3"  
class="td11">     
0<span class="cmmi-12">.</span>68744
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-37-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-37-1"  
class="td11"> 
Rho GTPase cycle
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-37-2"  
class="td11">      
0<span class="cmmi-12">.</span>00034
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-37-3"  
class="td11">     
0<span class="cmmi-12">.</span>15675
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-38-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-38-1"  
class="td11"> 
Creation of C4 and C2 activators
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-38-2"  
class="td11">      
0<span class="cmmi-12">.</span>00035
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-38-3"  
class="td11">     
0<span class="cmmi-12">.</span>12275
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-39-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-39-1"  
class="td11"> 
Abacavir transport and metabolism
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-39-2"  
class="td11">      
0<span class="cmmi-12">.</span>00035
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-39-3"  
class="td11">     
0<span class="cmmi-12">.</span>12443
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-40-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-40-1"  
class="td11"> 
Amine compound SLC transporters
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-40-2"  
class="td11">      
0<span class="cmmi-12">.</span>00037
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-40-3"  
class="td11">     
0<span class="cmmi-12">.</span>69773
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-41-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-41-1"  
class="td11"> 
FCERI mediated NF-kB activation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-41-2"  
class="td11">      
0<span class="cmmi-12">.</span>00037
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-41-3"  
class="td11">     
0<span class="cmmi-12">.</span>69846
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-42-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-42-1"  
class="td11"> 
Fc epsilon receptor (FCERI) signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-42-2"  
class="td11">      
0<span class="cmmi-12">.</span>00056
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-42-3"  
class="td11">     
0<span class="cmmi-12">.</span>43303
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-43-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-43-1"  
class="td11"> 
Defective EXT2 causes exostoses 2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-43-2"  
class="td11">      
0<span class="cmmi-12">.</span>00067
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-43-3"  
class="td11">     
0<span class="cmmi-12">.</span>16053
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-44-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-44-1"  
class="td11"> 
Defective EXT1 causes exostoses 1, TRPS2 and CHDS
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-44-2"  
class="td11">      
0<span class="cmmi-12">.</span>00067
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-44-3"  
class="td11">     
0<span class="cmmi-12">.</span>16053
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-45-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-45-1"  
class="td11"> 
<span class="cmti-12">Collagen biosynthesis and modifying enzymes </span>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-45-2"  
class="td11">      
0<span class="cmmi-12">.</span>00071
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-45-3"  
class="td11">    
0<span class="cmmi-12">.</span>052911
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-46-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-46-1"  
class="td11"> 
Keratan sulfate/keratin metabolism
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-46-2"  
class="td11">      
0<span class="cmmi-12">.</span>00073
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-46-3"  
class="td11">     
0<span class="cmmi-12">.</span>46533
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-47-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-47-1"  
class="td11"> 
G alpha (12/13) signalling events
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-47-2"  
class="td11">      
0<span class="cmmi-12">.</span>00078
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-47-3"  
class="td11">     
0<span class="cmmi-12">.</span>59164
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-48-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-48-1"  
class="td11"> 
<strong><span class="cmbx-12">SEMA3A-Plexin repulsion signalling by inhibiting Integrin adhesion</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-48-2"  
class="td11">      
0<span class="cmmi-12">.</span>00084
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-48-3"  
class="td11">    
0<span class="cmmi-12">.</span>038504
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-49-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-49-1"  
class="td11"> 
Signal attenuation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-49-2"  
class="td11">      
0<span class="cmmi-12">.</span>00084
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-49-3"  
class="td11">     
0<span class="cmmi-12">.</span>37779
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-50-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-50-1"  
class="td11"> 
Eicosanoid ligand-binding receptors
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-50-2"  
class="td11">       
0<span class="cmmi-12">.</span>0011
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-50-3"  
class="td11">     
0<span class="cmmi-12">.</span>11117
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-99-51-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-51-1"  
class="td11"> 
SOS-mediated signalling
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-51-2"  
class="td11">       
0<span class="cmmi-12">.</span>0011
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-99-51-3"  
class="td11">     
0<span class="cmmi-12">.</span>25387
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-99-52-"><td  style="white-space:nowrap; text-align:left;" id="TBL-99-52-1"  
class="td11">                                                                   </td></tr></table>
<!--l. 2192--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Over-representation (hypergeometric test) and Permutation p-values adjusted for multiple tests across pathways (</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a><span class="cmr-10x-x-109">). Significant</span> <span class="cmr-10x-x-109">pathways were marked in bold (</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a> <span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05) and italics (</span><a 
href="#glo:FDR"><span class="cmr-10x-x-109">FDR</span></a> <span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">1).</span></p>
</p>
</div><hr class="endfloat" />
<a 
 id="x1-184003r429"></a>
<h3 class="sectionHead">
<span class="titlemark">F.3 </span> <a 
 id="x1-1850003"></a>Metagene Analysis
</h3>
<!--l. 2204--><p class="noindent" >
<p>Metagenes used to detect synthetic lethal pathways with <span class="cmti-12">CDH1 </span>in stomach cancer.</p>
</p><!--l. 2206--><p class="indent" >
<a 
 id="x1-1850017"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table F.7: </span><span class="content">Synthetic lethal metagenes against <span class="cmti-12">CDH1 </span>in stomach cancer</span>
</div><!--tex4ht:label?: x1-1850017 -->
<div class="tabular">
<table id="TBL-100" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-100-1g"><col 
id="TBL-100-1" /><col 
id="TBL-100-2" /><col 
id="TBL-100-3" /><col 
id="TBL-100-4" /><col 
id="TBL-100-5" /><col 
id="TBL-100-6" /><col 
id="TBL-100-7" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-100-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-1-1"  
class="td11"> 
<span class="cmbx-12">Pathway </span>
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-1-2"  
class="td11">   
ID
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-1-3"  
class="td11"> 
Observed
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-1-4"  
class="td11"> 
Expected
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-1-5"  
class="td11"> 
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-1-6"  
class="td11">   
p-value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-1-7"  
class="td11"> 
p-value (<a 
href="#glo:FDR">FDR</a>)
</td>

</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-100-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-2-1"  
class="td11"> 
Cell-Cell communication
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-2-2"  
class="td11"> 
1500931
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-2-3"  
class="td11"> 
18
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-2-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-2-5"  
class="td11">   
110
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-2-6"  
class="td11"> 
7<span class="cmmi-12">.</span>43 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">23</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-2-7"  
class="td11">  
1<span class="cmmi-12">.</span>53 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-3-1"  
class="td11"> 
VEGFR2 mediated vascular permeability
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-3-2"  
class="td11"> 
5218920
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-3-3"  
class="td11"> 
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-3-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-3-5"  
class="td11">   
109
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-3-6"  
class="td11"> 
1<span class="cmmi-12">.</span>36 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-3-7"  
class="td11">  
2<span class="cmmi-12">.</span>49 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-4-1"  
class="td11"> 
Sema4D in semaphorin signalling
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-4-2"  
class="td11"> 
400685
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-4-3"  
class="td11"> 
20
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-4-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-4-5"  
class="td11">   
104
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-4-6"  
class="td11"> 
1<span class="cmmi-12">.</span>62 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-4-7"  
class="td11">  
2<span class="cmmi-12">.</span>12 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-5-1"  
class="td11"> 
Ion transport by P-type ATPases
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-5-2"  
class="td11"> 
936837
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-5-3"  
class="td11"> 
17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-5-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-5-5"  
class="td11">   
100
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-5-6"  
class="td11"> 
8<span class="cmmi-12">.</span>29 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-5-7"  
class="td11">  
8<span class="cmmi-12">.</span>06 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-6-1"  
class="td11"> 
Sialic acid metabolism
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-6-2"  
class="td11"> 
4085001
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-6-3"  
class="td11"> 
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-6-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-6-5"  
class="td11">  
95.3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-6-6"  
class="td11"> 
9<span class="cmmi-12">.</span>95 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-6-7"  
class="td11">  
7<span class="cmmi-12">.</span>82 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-7-1"  
class="td11"> 
Synthesis of pyrophosphates in the cytosol
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-7-2"  
class="td11"> 
1855167
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-7-3"  
class="td11"> 
26
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-7-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-7-5"  
class="td11">   
94
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-7-6"  
class="td11"> 
1<span class="cmmi-12">.</span>86 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-7-7"  
class="td11">  
1<span class="cmmi-12">.</span>23 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-8-1"  
class="td11"> 
Keratan sulfate/keratin metabolism
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-8-2"  
class="td11"> 
1638074
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-8-3"  
class="td11"> 
25
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-8-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-8-5"  
class="td11">  
93.5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-8-6"  
class="td11"> 
2<span class="cmmi-12">.</span>36 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-8-7"  
class="td11">  
1<span class="cmmi-12">.</span>44 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-9-1"  
class="td11"> 
Ion channel transport
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-9-2"  
class="td11"> 
983712
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-9-3"  
class="td11"> 
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-9-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-9-5"  
class="td11">  
92.8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-9-6"  
class="td11"> 
3<span class="cmmi-12">.</span>37 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-9-7"  
class="td11">  
1<span class="cmmi-12">.</span>99 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-10-1"  
class="td11"> 
Keratan sulfate biosynthesis
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-10-2"  
class="td11"> 
2022854
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-10-3"  
class="td11"> 
26
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-10-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-10-5"  
class="td11">  
91.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-10-6"  
class="td11"> 
6<span class="cmmi-12">.</span>79 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-10-7"  
class="td11">  
3<span class="cmmi-12">.</span>62 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-11-1"  
class="td11"> 
Arachidonic acid metabolism
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-11-2"  
class="td11"> 
2142753
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-11-3"  
class="td11"> 
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-11-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-11-5"  
class="td11">  
90.6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-11-6"  
class="td11"> 
9<span class="cmmi-12">.</span>81 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-11-7"  
class="td11">  
5<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-12-1"  
class="td11"> 
RHO GTPases activate CIT
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-12-2"  
class="td11"> 
5625900
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-12-3"  
class="td11"> 
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-12-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-12-5"  
class="td11">   
87
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-12-6"  
class="td11"> 
5<span class="cmmi-12">.</span>80 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-12-7"  
class="td11">  
2<span class="cmmi-12">.</span>66 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-13-1"  
class="td11"> 
Stimuli-sensing channels
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-13-2"  
class="td11"> 
2672351
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-13-3"  
class="td11"> 
25
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-13-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-13-5"  
class="td11">  
85.8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-13-6"  
class="td11"> 
1<span class="cmmi-12">.</span>03 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-13-7"  
class="td11">  
4<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-14-1"  
class="td11"> 
Synthesis of PI
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-14-2"  
class="td11"> 
1483226
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-14-3"  
class="td11"> 
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-14-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-14-5"  
class="td11">  
85.6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-14-6"  
class="td11"> 
1<span class="cmmi-12">.</span>15 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-14-7"  
class="td11">  
4<span class="cmmi-12">.</span>89 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-15-1"  
class="td11"> 
G-protein activation
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-15-2"  
class="td11"> 
202040
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-15-3"  
class="td11"> 
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-15-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-15-5"  
class="td11">  
85.3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-15-6"  
class="td11"> 
1<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-15-7"  
class="td11">  
5<span class="cmmi-12">.</span>53 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-16-1"  
class="td11"> 
NrCAM interactions
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-16-2"  
class="td11"> 
447038
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-16-3"  
class="td11"> 
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-16-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-16-5"  
class="td11">  
84.3
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-16-6"  
class="td11"> 
2<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-16-7"  
class="td11">  
8<span class="cmmi-12">.</span>27 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-17-1"  
class="td11"> 
Inwardly rectifying <span class="cmmi-12">K</span><sup><span class="cmr-8">+</span></sup> channels
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-17-2"  
class="td11"> 
1296065
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-17-3"  
class="td11"> 
24
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-17-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-17-5"  
class="td11">  
83.5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-17-6"  
class="td11"> 
3<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-17-7"  
class="td11">  
1<span class="cmmi-12">.</span>22 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-18-1"  
class="td11"> 
Calcitonin-like ligand receptors
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-18-2"  
class="td11"> 
419812
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-18-3"  
class="td11"> 
20
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-18-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-18-5"  
class="td11">  
82.2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-18-6"  
class="td11"> 
6<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-18-7"  
class="td11">  
2<span class="cmmi-12">.</span>13 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-19-1"  
class="td11"> 
Prostacyclin signalling through prostacyclin receptor
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-19-2"  
class="td11"> 
392851
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-19-3"  
class="td11"> 
24
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-19-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-19-5"  
class="td11">  
81.8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-19-6"  
class="td11"> 
7<span class="cmmi-12">.</span>27 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-19-7"  
class="td11">  
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-20-1"  
class="td11"> 
Presynaptic function of Kainate receptors
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-20-2"  
class="td11"> 
500657
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-20-3"  
class="td11"> 
26
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-20-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-20-5"  
class="td11">  
79.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-20-6"  
class="td11"> 
2<span class="cmmi-12">.</span>00 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-20-7"  
class="td11">  
6<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-21-1"  
class="td11"> 
ADP signalling through P2Y purinoceptor 12
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-21-2"  
class="td11"> 
392170
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-21-3"  
class="td11"> 
23
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-21-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-21-5"  
class="td11">  
79.2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-21-6"  
class="td11"> 
2<span class="cmmi-12">.</span>57 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-21-7"  
class="td11">  
7<span class="cmmi-12">.</span>71 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-22-1"  
class="td11"> 
regulation of FZD by ubiquitination
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-22-2"  
class="td11"> 
4641263
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-22-3"  
class="td11"> 
22
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-22-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-22-5"  
class="td11">  
78.8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-22-6"  
class="td11"> 
3<span class="cmmi-12">.</span>15 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-22-7"  
class="td11">  
9<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-23-1"  
class="td11"> 
Toxicity of tetanus toxin (TeNT)
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-23-2"  
class="td11"> 
5250982
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-23-3"  
class="td11"> 
27
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-23-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-23-5"  
class="td11">  
78.7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-23-6"  
class="td11"> 
3<span class="cmmi-12">.</span>36 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-23-7"  
class="td11">  
9<span class="cmmi-12">.</span>75 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-24-1"  
class="td11"> 
Gap junction degradation
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-24-2"  
class="td11"> 
190873
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-24-3"  
class="td11"> 
21
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-24-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-24-5"  
class="td11">  
78.5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-24-6"  
class="td11"> 
3<span class="cmmi-12">.</span>66 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-24-7"  
class="td11">  
1<span class="cmmi-12">.</span>04 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-25-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-25-1"  
class="td11"> 
Nephrin interactions
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-25-2"  
class="td11"> 
373753
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-25-3"  
class="td11"> 
25
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-25-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-25-5"  
class="td11">  
78.2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-25-6"  
class="td11"> 
4<span class="cmmi-12">.</span>21 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-25-7"  
class="td11">  
1<span class="cmmi-12">.</span>14 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-100-26-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-26-1"  
class="td11"> 
GABA synthesis, release, reuptake and degradation
</td><td  style="white-space:nowrap; text-align:left;" id="TBL-100-26-2"  
class="td11"> 
888590
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-26-3"  
class="td11"> 
26
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-26-4"  
class="td11">   
50.4
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-26-5"  
class="td11">   
77
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-26-6"  
class="td11"> 
7<span class="cmmi-12">.</span>69 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-100-26-7"  
class="td11">  
1<span class="cmmi-12">.</span>95 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-100-27-"><td  style="white-space:nowrap; text-align:left;" id="TBL-100-27-1"  
class="td11">                                              </td></tr></table>
<!--l. 2273--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Strongest candidate </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">by SLIPT with observed and expected numbers of </span><a 
href="#glo:TCGA"><span class="cmr-10x-x-109">TCGA</span></a> <span class="cmr-10x-x-109">stomach cancer samples with low</span> <span class="cmr-10x-x-109">expression of both genes.</span></p>
</p>
</div><hr class="endfloat" />
                                                                                    





<a 
 id="x1-185002r356"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix G</span><br /><a 
 id="x1-186000G"></a>Synthetic Lethal Genes in Pathways
</h2>
                                                                                    

<!--l. 41--><p class="indent" >
<a 
 id="x1-1860011"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 43--><p class="noindent" >
</p><!--l. 44--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_Pi3kAkt_exprSL2".png" alt="PIC"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure G.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality in the PI3K/AKT pathway.</span></strong> <span class="cmr-10x-x-109">The Reactome</span> <span class="cmr-10x-x-109">PI3K/AKT pathway with synthetic lethal candidates, coloured as shown in the legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-1860011 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 57--><p class="indent" >
<a 
 id="x1-1860022"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 59--><p class="noindent" >
</p><!--l. 60--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_Pi3kAktCancer_exprSL2".png" alt="PIC"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure G.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality in the PI3K/AKT pathway in cancer.</span></strong> <span class="cmr-10x-x-109">The</span> <span class="cmr-10x-x-109">Reactome PI3K/AKT in cancer pathway with synthetic lethal candidates, coloured as shown</span> <span class="cmr-10x-x-109">in the legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-1860022 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 110--><p class="indent" >
<a 
 id="x1-1860033"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 112--><p class="noindent" >
</p><!--l. 113--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_ExtracellularMatrix_exprSL2".png" alt="pict"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure G.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality in the Extracellular Matrix.</span></strong> <span class="cmr-10x-x-109">The Reactome</span> <span class="cmr-10x-x-109">Extracellular Matrix pathway with synthetic lethal candidates, coloured as shown in the</span> <span class="cmr-10x-x-109">legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-1860033 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 128--><p class="indent" >
<a 
 id="x1-1860044"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 130--><p class="noindent" >
</p><!--l. 131--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_GPCR_Downstream_exprSL2".png" alt="PIC"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure G.4: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality in the GPCR Downstream.</span></strong> <span class="cmr-10x-x-109">The Reactome </span><a 
href="#glo:GPCR"><span class="cmr-10x-x-109">GPCR</span></a> <span class="cmr-10x-x-109">Downstream pathway with synthetic lethal candidates, coloured as shown in the legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-1860044 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 144--><p class="indent" >
<a 
 id="x1-1860055"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 146--><p class="noindent" >
</p><!--l. 147--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_TranslationElongation_exprSL2".png" alt="PIC"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure G.5: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality in the Translation Elongation.</span></strong> <span class="cmr-10x-x-109">The Reactome</span> <span class="cmr-10x-x-109">Translation Elongation pathway with synthetic lethal candidates, coloured as shown in the</span> <span class="cmr-10x-x-109">legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-1860055 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 161--><p class="indent" >
<a 
 id="x1-1860066"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 163--><p class="noindent" >
</p><!--l. 164--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_NMD_exprSL2".png" alt="pict"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure G.6: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality in the Nonsense-mediated Decay.</span></strong> <span class="cmr-10x-x-109">The Reactome</span> <a 
href="#glo:NMD"><span class="cmr-10x-x-109">NMD</span></a> <span class="cmr-10x-x-109">pathway with synthetic lethal candidates, coloured as shown in the legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-1860066 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 177--><p class="indent" >
<a 
 id="x1-1860077"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 179--><p class="noindent" >
</p><!--l. 180--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_Three_prime_UTR_exprSL2".png" alt="PIC"  
 />
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure G.7: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality in the 3</span><sup><span class="cmsy-8">′</span></sup> <span class="cmbx-10x-x-109">UTR.</span></strong> <span class="cmr-10x-x-109">The Reactome 3</span><sup><span class="cmsy-8">′</span></sup> <a 
href="#glo:UTR"><span class="cmr-10x-x-109">UTR</span></a> <span class="cmr-10x-x-109">pathway with</span> <span class="cmr-10x-x-109">synthetic lethal candidates, coloured as shown in the legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-1860077 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<a 
 id="x1-186008r356"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix H</span><br /><a 
 id="x1-187000H"></a>Network Analysis for Mutation SLIPT
</h2>
                                                                                    

<!--l. 198--><p class="indent" >
<a 
 id="x1-1870011"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 200--><p class="noindent" >
</p><!--l. 201--><p class="noindent" >
<span class="fbox"><img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Centrality_mtSL/GPCR_network_vertex_degree_stripchart2".png" alt="pict"  
 /></span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure H.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality and vertex degree.</span></strong> <span class="cmr-10x-x-109">The number of connected genes</span> <span class="cmr-10x-x-109">(</span><a 
href="#glo:vertex degree"><span class="cmr-10x-x-109">vertex degree</span></a><span class="cmr-10x-x-109">) was compared (on a log-scale) across genes detected by </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screening in the Reactome G</span><sub><span class="cmmi-8">αi</span></sub> <span class="cmr-10x-x-109">pathway. There were no differences in </span><a 
href="#glo:vertex"><span class="cmr-10x-x-109">vertex</span></a> <span class="cmr-10x-x-109">degree between</span> <span class="cmr-10x-x-109">the groups (shown in Table</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-1250021"><span class="cmr-10x-x-109">5.1</span><!--tex4ht:ref: tab:SL_Pathway_GPCR_Vertex_Degree --></a><span class="cmr-10x-x-109">), although genes detected by </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">included those with the</span> <span class="cmr-10x-x-109">fewest connections. </span></span></p>
</div>
<!--tex4ht:label?: x1-1870011 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 216--><p class="indent" >
<a 
 id="x1-1870022"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 218--><p class="noindent" >
</p><!--l. 219--><p class="noindent" >
<span class="fbox"><img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Centrality_mtSL/GPCR_network_Info_Centrality(Log)_stripchart2".png" alt="pict"  
 /></span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure H.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality and centrality.</span></strong> <span class="cmr-10x-x-109">The </span><a 
href="#glo:information centrality"><span class="cmr-10x-x-109">information centrality</span></a> <span class="cmr-10x-x-109">was</span> <span class="cmr-10x-x-109">compared (on a log-scale) across genes detected by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screening in the</span> <span class="cmr-10x-x-109">Reactome G</span><sub><span class="cmmi-8">αi</span> </sub> <span class="cmr-10x-x-109">pathway. Genes detected by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">or </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">did not have higher centrality</span> <span class="cmr-10x-x-109">than other genes (shown in Table</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-1870052"><span class="cmr-10x-x-109">H.2</span><!--tex4ht:ref: tab:mtSL_Pathway_GPCR_InfoCent --></a><span class="cmr-10x-x-109">). Genes detected by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">spanned the range of</span> <a 
href="#glo:centrality"><span class="cmr-10x-x-109">centrality</span></a> <span class="cmr-10x-x-109">values. </span></span></p>
</div>
<!--tex4ht:label?: x1-1870022 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 238--><p class="indent" >
<a 
 id="x1-1870033"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 240--><p class="noindent" >
</p><!--l. 241--><p class="noindent" >
<span class="fbox"><img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Centrality_mtSL/GPCR_network_pagerank_stripchart2".png" alt="pict"  
 /></span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure H.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality and PageRank.</span></strong> <span class="cmr-10x-x-109">The </span><a 
href="#glo:PageRank centrality"><span class="cmr-10x-x-109">PageRank centrality</span></a> <span class="cmr-10x-x-109">was compared</span> <span class="cmr-10x-x-109">(on a log-scale) across genes detected by </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screening in the Reactome</span> <span class="cmr-10x-x-109">G</span><sub><span class="cmmi-8">αi</span> </sub> <span class="cmr-10x-x-109">pathway. Genes detected by with either </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">detection approach had a</span> <span class="cmr-10x-x-109">more restricted range of </span><a 
href="#glo:centrality"><span class="cmr-10x-x-109">centrality</span></a> <span class="cmr-10x-x-109">values neither of these had a significant association with</span> <span class="cmr-10x-x-109">centrality (shown in Table</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-1870063"><span class="cmr-10x-x-109">H.3</span><!--tex4ht:ref: tab:mtSL_Pathway_GPCR_PageRank --></a><span class="cmr-10x-x-109">). </span></span></p>
</div>
<!--tex4ht:label?: x1-1870033 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 255--><p class="indent" >
<a 
 id="x1-1870041"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table H.1: </span><span class="content"><a 
href="#glo:ANOVA">ANOVA</a> for synthetic lethality and vertex degree</span>
</div><!--tex4ht:label?: x1-1870041 -->
<div class="tabular">
<table id="TBL-101" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-101-1g"><col 
id="TBL-101-1" /><col 
id="TBL-101-2" /><col 
id="TBL-101-3" /><col 
id="TBL-101-4" /><col 
id="TBL-101-5" /><col 
id="TBL-101-6" /></colgroup><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-101-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-101-1-1"  
class="td11">                </td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-1-2"  
class="td11"> 
DF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-1-3"  
class="td11"> 
Sum Squares
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-1-4"  
class="td11"> 
Mean Squares
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-1-5"  
class="td11"> 
F-value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-1-6"  
class="td11"> 
p-value
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-101-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-101-2-1"  
class="td11"> 
siRNA
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-2-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-2-3"  
class="td11">     
15
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-2-4"  
class="td11">     
15.50
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-2-5"  
class="td11"> 
0.0134
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-2-6"  
class="td11"> 
0.9084
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-101-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-101-3-1"  
class="td11"> 
mtSLIPT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-3-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-3-3"  
class="td11">     
196
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-3-4"  
class="td11">    
195.94
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-3-5"  
class="td11"> 
0.1689
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-3-6"  
class="td11"> 
0.6825
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-101-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-101-4-1"  
class="td11"> 
siRNA<span class="cmsy-10x-x-120">×</span>mtSLIPT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-4-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-4-3"  
class="td11">      
9
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-4-4"  
class="td11">     
9.17
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-4-5"  
class="td11"> 
0.0079
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-101-4-6"  
class="td11"> 
0.9294
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-101-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-101-5-1"  
class="td11">                 </td></tr></table>
<!--l. 279--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Analysis of variance for </span><a 
href="#glo:vertex degree"><span class="cmr-10x-x-109">vertex degree</span></a> <span class="cmr-10x-x-109">against </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">detection approaches (with an</span> <span class="cmr-10x-x-109">interaction term)</span></p>
</p>
</div><hr class="endfloat" />
                                                                                    

<!--l. 283--><p class="indent" >
<a 
 id="x1-1870052"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table H.2: </span><span class="content"><a 
href="#glo:ANOVA">ANOVA</a> for synthetic lethality and information centrality</span>
</div><!--tex4ht:label?: x1-1870052 -->
<div class="tabular">
<table id="TBL-102" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-102-1g"><col 
id="TBL-102-1" /><col 
id="TBL-102-2" /><col 
id="TBL-102-3" /><col 
id="TBL-102-4" /><col 
id="TBL-102-5" /><col 
id="TBL-102-6" /></colgroup><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-102-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-102-1-1"  
class="td11">                </td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-1-2"  
class="td11"> 
DF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-1-3"  
class="td11"> 
Sum Squares
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-1-4"  
class="td11"> 
Mean Squares
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-1-5"  
class="td11"> 
F-value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-1-6"  
class="td11"> 
p-value
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-102-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-102-2-1"  
class="td11"> 
siRNA
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-2-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-2-3"  
class="td11">   
0.000256
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-2-4"  
class="td11">   
0.0002561
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-2-5"  
class="td11"> 
0.1851
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-2-6"  
class="td11"> 
0.6685
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-102-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-102-3-1"  
class="td11"> 
mtSLIPT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-3-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-3-3"  
class="td11">   
0.003225
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-3-4"  
class="td11">   
0.0032247
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-3-5"  
class="td11"> 
2.3308
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-3-6"  
class="td11"> 
0.1318
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-102-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-102-4-1"  
class="td11"> 
siRNA<span class="cmsy-10x-x-120">×</span>mtSLIPT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-4-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-4-3"  
class="td11">   
0.001238
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-4-4"  
class="td11">   
0.0012385
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-4-5"  
class="td11"> 
0.8952
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-102-4-6"  
class="td11"> 
0.3476
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-102-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-102-5-1"  
class="td11">                 </td></tr></table>
<!--l. 307--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Analysis of variance for </span><a 
href="#glo:information centrality"><span class="cmr-10x-x-109">information centrality</span></a> <span class="cmr-10x-x-109">against </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">detection approaches (with</span> <span class="cmr-10x-x-109">an interaction term)</span></p>
</p>
</div><hr class="endfloat" />
                                                                                    

<!--l. 312--><p class="indent" >
<a 
 id="x1-1870063"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table H.3: </span><span class="content"><a 
href="#glo:ANOVA">ANOVA</a> for synthetic lethality and PageRank centrality</span>
</div><!--tex4ht:label?: x1-1870063 -->
<div class="tabular">
<table id="TBL-103" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-103-1g"><col 
id="TBL-103-1" /><col 
id="TBL-103-2" /><col 
id="TBL-103-3" /><col 
id="TBL-103-4" /><col 
id="TBL-103-5" /><col 
id="TBL-103-6" /></colgroup><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-103-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-103-1-1"  
class="td11">                </td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-1-2"  
class="td11"> 
DF
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-1-3"  
class="td11"> 
Sum Squares
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-1-4"  
class="td11"> 
Mean Squares
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-1-5"  
class="td11"> 
F-value
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-1-6"  
class="td11"> 
p-value
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-103-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-103-2-1"  
class="td11"> 
siRNA
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-2-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-2-3"  
class="td11">  
0.0002038
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-2-4"  
class="td11"> 
2<span class="cmmi-12">.</span>0385 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-2-5"  
class="td11"> 
1.1423
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-2-6"  
class="td11"> 
0.2892
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-103-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-103-3-1"  
class="td11"> 
mtSLIPT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-3-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-3-3"  
class="td11">  
0.0000208
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-3-4"  
class="td11"> 
2<span class="cmmi-12">.</span>0752 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-3-5"  
class="td11"> 
0.1163
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-3-6"  
class="td11"> 
0.7342
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-103-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-103-4-1"  
class="td11"> 
siRNA<span class="cmsy-10x-x-120">×</span>mtSLIPT
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-4-2"  
class="td11">  
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-4-3"  
class="td11">  
0.0000137
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-4-4"  
class="td11"> 
1<span class="cmmi-12">.</span>3743 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-4-5"  
class="td11"> 
0.0770
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-103-4-6"  
class="td11"> 
0.7823
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-103-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-103-5-1"  
class="td11">                 </td></tr></table>
<!--l. 336--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Analysis of variance for </span><a 
href="#glo:PageRank centrality"><span class="cmr-10x-x-109">PageRank centrality</span></a> <span class="cmr-10x-x-109">against </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethal</span></a> <span class="cmr-10x-x-109">detection approaches (with</span> <span class="cmr-10x-x-109">an interaction term)</span></p>
</p>
</div><hr class="endfloat" />
                                                                                    

<a 
 id="x1-187007r356"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix I</span><br /><a 
 id="x1-188000I"></a>Pathway Structure for Mutation SLIPT
</h2>
                                                                                    

<!--l. 548--><p class="indent" >
<a 
 id="x1-1880032"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 550--><p class="noindent" >
</p><!--l. 554--><p class="noindent" >
<img 
src=""/home/tomkelly/Downloads/Pathway_Structure/test_GPCR_mtSL".png" alt="pict"  
 /><a 
 id="x1-188001r1"></a> <span class="cmr-10x-x-109">Resampling</span> <span class="cmr-10x-x-109">in</span> <span class="cmr-10x-x-109">G</span><sub><span class="cmmi-8">αi</span> </sub> <span class="cmr-10x-x-109">signalling</span><img 
src=""/home/tomkelly/Downloads/Pathway_Structure/test_PI3K_mtSL".png" alt="pict"  
 /><a 
 id="x1-188002r2"></a> <span class="cmr-10x-x-109">Resampling</span> <span class="cmr-10x-x-109">in the</span> <span class="cmr-10x-x-109">PI3K</span> <span class="cmr-10x-x-109">cascade</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure I.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Structure of synthetic lethality resampling.</span></strong> <span class="cmr-10x-x-109">A null distribution with 10,000</span> <span class="cmr-10x-x-109">iterations of the number of </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes upstream or downstream of </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">genes (depicted</span> <span class="cmr-10x-x-109">as the difference of these) in each </span><a 
href="#glo:pathway"><span class="cmr-10x-x-109">pathway</span></a><span class="cmr-10x-x-109">. To assess significance, the observed events (with</span> <a 
href="#glo:shortest path"><span class="cmr-10x-x-109">shortest paths</span></a><span class="cmr-10x-x-109">) were compared to the 90% and 95% intervals for the null distribution (shown</span> <span class="cmr-10x-x-109">in blue). Genes detected by both methods were not fixed to the same number as observed</span> <span class="cmr-10x-x-109">for the alternative null distribution (shown in red), although the significance of the observed</span> <span class="cmr-10x-x-109">number of events (red) was changed in either case. The genes detected by both approaches</span> <span class="cmr-10x-x-109">were included in computing the number of </span><a 
href="#glo:shortest path"><span class="cmr-10x-x-109">shortest paths</span></a> <span class="cmr-10x-x-109">(in either direction) between </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes. The permutations show (a) a significant pathway relationship for G</span><sub><span class="cmmi-8">αi</span></sub> <span class="cmr-10x-x-109">signalling and (b) and non-significant relationship for the </span><a 
href="#glo:PI3K"><span class="cmr-10x-x-109">PI3K</span></a> <span class="cmr-10x-x-109">cascade. </span></span></p>
</div>
<!--tex4ht:label?: x1-1880032 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 566--><p class="indent" >
<a 
 id="x1-1880041"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="caption" 
>
<span class="id">Table I.1: </span><span class="content">Resampling for pathway structure of <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection methods</span>
</div><!--tex4ht:label?: x1-1880041 -->
<div class="tabular">
                                            <table id="TBL-104" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-104-1g"><col 
id="TBL-104-1" /></colgroup><colgroup id="TBL-104-2g"><col 
id="TBL-104-2" /><col 
id="TBL-104-3" /></colgroup><colgroup id="TBL-104-4g"><col 
id="TBL-104-4" /><col 
id="TBL-104-5" /></colgroup><colgroup id="TBL-104-6g"><col 
id="TBL-104-6" /><col 
id="TBL-104-7" /><col 
id="TBL-104-8" /><col 
id="TBL-104-9" /></colgroup><colgroup id="TBL-104-10g"><col 
id="TBL-104-10" /><col 
id="TBL-104-11" /></colgroup><colgroup id="TBL-104-12g"><col 
id="TBL-104-12" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-104-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-104-1-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-104-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-104-2-1"  
class="td11">                                     </td><td colspan="2" style="white-space:nowrap; text-align:center;" id="TBL-104-2-2"  
class="td11">      <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<strong><span class="cmbx-12">Graph</span></strong>
</div>       </td><td colspan="2" style="white-space:nowrap; text-align:center;" id="TBL-104-2-4"  
class="td11">    <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<strong><span class="cmbx-12">Candidates</span></strong>
</div>     </td><td colspan="3" style="white-space:nowrap; text-align:center;" id="TBL-104-2-6"  
class="td11"></td>                 <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<strong><span class="cmbx-12">Observed</span></strong>
</div>                    <td colspan="2" style="white-space:nowrap; text-align:center;" id="TBL-104-2-10"  
class="td11">   <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<strong><span class="cmbx-12">Permutation p-value</span></strong>
</div>    </td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-2-12"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:center;">
<strong><span class="cmbx-12">p-value (FDR)</span></strong>
</div>
</td></tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-104-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-104-3-1"  
class="td11"> 
P<strong><span class="cmbx-12">Pathway</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-3-2"  
class="td11"> 
<strong><span class="cmbx-12">Nodes</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-3-3"  
class="td11"> 
<strong><span class="cmbx-12">Edges</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-3-4"  
class="td11"> 
<strong><span class="cmbx-12">SLIPT</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-3-5"  
class="td11"> 
<strong><span class="cmbx-12">siRNA</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-3-6"  
class="td11"> 
<strong><span class="cmbx-12">Up</span></strong><sup class="textsuperscript"><span class="cmr-10x-x-109">1</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-3-7"  
class="td11"> 
<strong><span class="cmbx-12">Down</span></strong><sup class="textsuperscript"><span class="cmr-10x-x-109">2</span></sup>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-3-8"  
class="td11"> 
<strong><span class="cmbx-12">Up</span><span class="cmsy-10x-x-120">-</span><span class="cmbx-12">Down</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-3-9"  
class="td11"> 
<strong><span class="cmbx-12">Up</span><span class="cmmi-12">∕</span><span class="cmbx-12">Down</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-3-10"  
class="td11"> 
<strong><span class="cmbx-12">Up</span><span class="cmsy-10x-x-120">-</span><span class="cmbx-12">Down</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-3-11"  
class="td11"> 
<strong><span class="cmbx-12">Down</span><span class="cmsy-10x-x-120">-</span><span class="cmbx-12">Up</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-3-12"  
class="td11">   
<strong><span class="cmbx-12">Down</span><span class="cmsy-10x-x-120">-</span><span class="cmbx-12">Up</span></strong>
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-104-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-104-4-1"  
class="td11"> 
PI3K Cascade
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-4-2"  
class="td11">   
138
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-4-3"  
class="td11">  
1495
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-4-4"  
class="td11">   
42
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-4-5"  
class="td11">   
25
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-4-6"  
class="td11"> 
131
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-4-7"  
class="td11">  
123
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-4-8"  
class="td11">     
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-4-9"  
class="td11">   
1.065
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-4-10"  
class="td11">   
0.4473
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-4-11"  
class="td11">   
0.5466
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-4-12"  
class="td11">     
0.7263
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-104-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-104-5-1"  
class="td11"> 
PI3K/AKT Signalling in Cancer
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-5-2"  
class="td11">   
275
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-5-3"  
class="td11"> 
12882
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-5-4"  
class="td11">   
56
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-5-5"  
class="td11">   
44
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-5-6"  
class="td11"> 
478
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-5-7"  
class="td11">  
440
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-5-8"  
class="td11">     
38
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-5-9"  
class="td11">   
1.086
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-5-10"  
class="td11">   
0.4163
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-5-11"  
class="td11">   
0.5810
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-5-12"  
class="td11">     
0.7263
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-104-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-104-6-1"  
class="td11"> 
<strong><span class="cmbx-12">G</span><sub><span class="cmmi-8">αi</span> </sub> <span class="cmbx-12">Signalling</span></strong>
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-6-2"  
class="td11">   
292
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-6-3"  
class="td11"> 
22003
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-6-4"  
class="td11">   
57
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-6-5"  
class="td11">   
58
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-6-6"  
class="td11"> 
543
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-6-7"  
class="td11">  
866
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-6-8"  
class="td11">    
-323
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-6-9"  
class="td11">   
0.627
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-6-10"  
class="td11">   
0.9507
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-6-11"  
class="td11">   
0.0488
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-6-12"  
class="td11">      
0.488
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-104-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-104-7-1"  
class="td11"> 
GPCR downstream
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-7-2"  
class="td11">  
1270
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-7-3"  
class="td11"> 
142071
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-7-4"  
class="td11">   
218
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-7-5"  
class="td11">   
160
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-7-6"  
class="td11"> 
7632
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-7-7"  
class="td11">  
6500
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-7-8"  
class="td11">    
1132
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-7-9"  
class="td11">   
1.174
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-7-10"  
class="td11">   
0.1707
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-7-11"  
class="td11">   
0.8291
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-7-12"  
class="td11">     
0.8751
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-104-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-104-8-1"  
class="td11"> 
Elastic fibre formation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-8-2"  
class="td11">   
42
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-8-3"  
class="td11">  
175
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-8-4"  
class="td11">   
16
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-8-5"  
class="td11">    
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-8-6"  
class="td11">  
6
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-8-7"  
class="td11">   
7
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-8-8"  
class="td11">     
-1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-8-9"  
class="td11">   
0.857
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-8-10"  
class="td11">   
0.5512
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-8-11"  
class="td11">   
0.3681
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-8-12"  
class="td11">     
0.7263
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-104-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-104-9-1"  
class="td11"> 
Extracellular matrix
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-9-2"  
class="td11">   
299
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-9-3"  
class="td11">  
3677
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-9-4"  
class="td11">   
81
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-9-5"  
class="td11">   
29
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-9-6"  
class="td11"> 
313
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-9-7"  
class="td11">  
347
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-9-8"  
class="td11">     
-34
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-9-9"  
class="td11">   
0.902
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-9-10"  
class="td11">   
0.5762
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-9-11"  
class="td11">   
0.4215
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-9-12"  
class="td11">     
0.7263
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-104-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-104-10-1"  
class="td11"> 
Formation of Fibrin
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-10-2"  
class="td11">   
52
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-10-3"  
class="td11">  
243
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-10-4"  
class="td11">   
11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-10-5"  
class="td11">    
5
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-10-6"  
class="td11">  
8
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-10-7"  
class="td11">   
19
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-10-8"  
class="td11">     
-11
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-10-9"  
class="td11">   
0.421
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-10-10"  
class="td11">   
0.7993
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-10-11"  
class="td11">   
0.1800
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-10-12"  
class="td11">     
0.6000
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-104-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-104-11-1"  
class="td11"> 
Nonsense-Mediated Decay
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-11-2"  
class="td11">   
103
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-11-3"  
class="td11">  
102
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-11-4"  
class="td11">   
56
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-11-5"  
class="td11">    
2
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-11-6"  
class="td11">  
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-11-7"  
class="td11">   
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-11-8"  
class="td11">     
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-11-9"  
class="td11">           </td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-11-10"  
class="td11">    
0.197
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-11-11"  
class="td11">   
0.1373
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-11-12"  
class="td11">     
0.6000
</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-104-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-104-12-1"  
class="td11"> 
3<sup><span class="cmsy-8">′</span> </sup> -UTR-mediated translational regulation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-12-2"  
class="td11">   
107
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-12-3"  
class="td11">  
2860
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-12-4"  
class="td11">   
56
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-12-5"  
class="td11">    
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-12-6"  
class="td11">  
52
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-12-7"  
class="td11">   
1
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-12-8"  
class="td11">     
51
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-12-9"  
class="td11">     
52
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-12-10"  
class="td11">   
0.1210
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-12-11"  
class="td11">   
0.8751
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-12-12"  
class="td11">     
0.8751
</td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-104-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-104-13-1"  
class="td11"> 
Eukaryotic Translation Elongation
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-13-2"  
class="td11">   
92
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-13-3"  
class="td11">  
3746
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-13-4"  
class="td11">   
57
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-13-5"  
class="td11">    
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-13-6"  
class="td11">  
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-13-7"  
class="td11">   
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-13-8"  
class="td11">     
0
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-13-9"  
class="td11">           </td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-13-10"  
class="td11">   
0.4952
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-13-11"  
class="td11">   
0.4892
</td><td  style="white-space:nowrap; text-align:center;" id="TBL-104-13-12"  
class="td11">     
0.7263
</td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-104-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-104-14-1"  
class="td11">                                     </td></tr></table>
<!--l. 611--><p class="noindent" ></div>
<p><span class="cmr-10x-x-109">Pathways in the Reactome network tested for structural relationships between </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes by resampling. The raw p-value (computed without adjusting for multiple comparisons over pathways) is given</span> <span class="cmr-10x-x-109">for the difference in upstream and downstream paths from </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">to </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">gene candidate partners of CDH1 with significant pathways highlighted in bold. Sampling was performed only in the target pathway and</span> <span class="cmr-10x-x-109">shortest paths were computed within it. Loops or paths in either direction that could not be resolved were excluded from the analysis. The genes detected by both </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">(or resampling for them) were</span> <span class="cmr-10x-x-109">included in the analysis and the number of these were fixed to the number observed.</span> <sup class="textsuperscript"><span class="cmr-10x-x-109">1</span></sup><span class="cmr-10x-x-109">The number of paths where the </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">candidate was upstream of a </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">candidate</span> <sup class="textsuperscript"><span class="cmr-10x-x-109">2</span></sup><span class="cmr-10x-x-109">The number of paths where the </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">candidate was downstream of a </span><a 
href="#glo:mtSLIPT"><span class="cmr-10x-x-109">mtSLIPT</span></a> <span class="cmr-10x-x-109">candidate</span></p>
</p>
</div><hr class="endfloat" />
                                                                                    





<a 
 id="x1-188005r356"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix J</span><br /><a 
 id="x1-189000J"></a>Performance of SLIPT and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>
</h2>
                                                                                    

<!--l. 20--><p class="indent" >
<a 
 id="x1-1890072"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 22--><p class="noindent" >
</p><!--l. 27--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(1)".png  /><a 
 id="x1-189001r1"></a> <span class="cmr-10x-x-109">1 SL</span> <span class="cmr-10x-x-109">partner</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(2)".png  /><a 
 id="x1-189002r2"></a> <span class="cmr-10x-x-109">2 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(3)".png  /><a 
 id="x1-189003r3"></a> <span class="cmr-10x-x-109">3 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 40--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(4)".png  /><a 
 id="x1-189004r4"></a> <span class="cmr-10x-x-109">4 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(5)".png  /><a 
 id="x1-189005r5"></a> <span class="cmr-10x-x-109">5 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(6)".png  /><a 
 id="x1-189006r6"></a> <span class="cmr-10x-x-109">6 SL</span> <span class="cmr-10x-x-109">partners</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure J.2: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles</span></strong>. (continued on next page) </span></p>
</div>
<!--tex4ht:label?: x1-1890072 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 55--><p class="indent" >
<a 
 id="x1-1890174"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 57--><p class="noindent" >
</p><!--l. 61--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(7)".png  /><a 
 id="x1-189008r1"></a> <span class="cmr-10x-x-109">7 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(8)".png  /><a 
 id="x1-189009r2"></a> <span class="cmr-10x-x-109">8 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(9)".png  /><a 
 id="x1-189010r3"></a> <span class="cmr-10x-x-109">9 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 75--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(10)".png  /><a 
 id="x1-189011r4"></a> <span class="cmr-10x-x-109">10 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(15)".png  /><a 
 id="x1-189012r5"></a> <span class="cmr-10x-x-109">15 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(20)".png  /><a 
 id="x1-189013r6"></a> <span class="cmr-10x-x-109">20 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 88--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(30)".png  /><a 
 id="x1-189014r7"></a> <span class="cmr-10x-x-109">30 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(40)".png  /><a 
 id="x1-189015r8"></a> <span class="cmr-10x-x-109">40 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(50)".png  /><a 
 id="x1-189016r9"></a> <span class="cmr-10x-x-109">50 SL</span> <span class="cmr-10x-x-109">partners</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure J.4: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles</span></strong>. Synthetic lethal detection with quantiles as in axis labels. The barplots have the same hue for each quantile (grey for correlation) and darker for <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> (and positive correlation). <a 
href="#glo:SLIPT">SLIPT</a> and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> performed similarly, peaking at <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span>-quantiles and converging to random (0.5). Negative correlation was higher than positive but not optimal quantiles for <a 
href="#glo:SLIPT">SLIPT</a> or <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> . These findings were robust across different numbers of underlying synthetic lethal genes in 10,000 simulations of 100 genes and 1000 samples. <a 
href="#glo:SLIPT">SLIPT</a> performed better than <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> for higher numbers of synthetic lethal genes and finer quantiles.</span></p>
</div>
<!--tex4ht:label?: x1-1890174 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 103--><p class="indent" >
<a 
 id="x1-1890276"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 105--><p class="noindent" >
</p><!--l. 111--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(1)".png  /><a 
 id="x1-189018r1"></a> <span class="cmr-10x-x-109">1 SL</span> <span class="cmr-10x-x-109">partner</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(2)".png  /><a 
 id="x1-189019r2"></a> <span class="cmr-10x-x-109">2 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(3)".png  /><a 
 id="x1-189020r3"></a> <span class="cmr-10x-x-109">3 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 124--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(4)".png  /><a 
 id="x1-189021r4"></a> <span class="cmr-10x-x-109">4 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png  /><a 
 id="x1-189022r5"></a> <span class="cmr-10x-x-109">5 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(6)".png  /><a 
 id="x1-189023r6"></a> <span class="cmr-10x-x-109">6 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 137--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(7)".png  /><a 
 id="x1-189024r7"></a> <span class="cmr-10x-x-109">7 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(8)".png  /><a 
 id="x1-189025r8"></a> <span class="cmr-10x-x-109">8 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(9)".png  /><a 
 id="x1-189026r9"></a> <span class="cmr-10x-x-109">9 SL</span> <span class="cmr-10x-x-109">partners</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure J.6: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles</span></strong>. (continued on next page) </span></p>
</div>
<!--tex4ht:label?: x1-1890276 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 151--><p class="indent" >
<a 
 id="x1-1890348"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 153--><p class="noindent" >
</p><!--l. 158--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(10)".png  /><a 
 id="x1-189028r1"></a> <span class="cmr-10x-x-109">10 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(15)".png  /><a 
 id="x1-189029r2"></a> <span class="cmr-10x-x-109">15 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(20)".png  /><a 
 id="x1-189030r3"></a> <span class="cmr-10x-x-109">20 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 171--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(30)".png  /><a 
 id="x1-189031r4"></a> <span class="cmr-10x-x-109">30 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(40)".png  /><a 
 id="x1-189032r5"></a> <span class="cmr-10x-x-109">40 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(50)".png  /><a 
 id="x1-189033r6"></a> <span class="cmr-10x-x-109">50 SL</span> <span class="cmr-10x-x-109">partners</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure J.8: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles</span></strong>. Synthetic lethal detection with quantiles as in axis labels. The line plots are coloured for SLIPT (red), <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> (blue) and correlation (grey), according to the legend. <a 
href="#glo:SLIPT">SLIPT</a> and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> performed similarly, peaking at <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span>-quantiles and converging to random (0.5). Negative correlation was higher than positive but not optimal quantiles for <a 
href="#glo:SLIPT">SLIPT</a> or <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>. These findings were robust across different numbers of underlying synthetic lethal genes in 10,000 simulations of 100 genes and 1000 samples. <a 
href="#glo:SLIPT">SLIPT</a> performed better than <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> for higher numbers of synthetic lethal genes and finer quantiles.</span></p>
</div>
<!--tex4ht:label?: x1-1890348 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 187--><p class="indent" >
<a 
 id="x1-18904410"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 189--><p class="noindent" >
</p><!--l. 195--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(1)".png  /><a 
 id="x1-189035r1"></a> <span class="cmr-10x-x-109">1 SL</span> <span class="cmr-10x-x-109">partner</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(2)".png  /><a 
 id="x1-189036r2"></a> <span class="cmr-10x-x-109">2 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(3)".png  /><a 
 id="x1-189037r3"></a> <span class="cmr-10x-x-109">3 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 208--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(4)".png  /><a 
 id="x1-189038r4"></a> <span class="cmr-10x-x-109">4 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png  /><a 
 id="x1-189039r5"></a> <span class="cmr-10x-x-109">5 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(6)".png  /><a 
 id="x1-189040r6"></a> <span class="cmr-10x-x-109">6 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 221--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(7)".png  /><a 
 id="x1-189041r7"></a> <span class="cmr-10x-x-109">7 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(8)".png  /><a 
 id="x1-189042r8"></a> <span class="cmr-10x-x-109">8 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(9)".png  /><a 
 id="x1-189043r9"></a> <span class="cmr-10x-x-109">9 SL</span> <span class="cmr-10x-x-109">partners</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure J.10: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles with more genes</span></strong>. (continued on next page) </span></p>
</div>
<!--tex4ht:label?: x1-18904410 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 235--><p class="indent" >
<a 
 id="x1-18905112"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 237--><p class="noindent" >
</p><!--l. 242--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(10)".png  /><a 
 id="x1-189045r1"></a> <span class="cmr-10x-x-109">10 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(15)".png  /><a 
 id="x1-189046r2"></a> <span class="cmr-10x-x-109">15 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(20)".png  /><a 
 id="x1-189047r3"></a> <span class="cmr-10x-x-109">20 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 255--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(30)".png  /><a 
 id="x1-189048r4"></a> <span class="cmr-10x-x-109">30 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(40)".png  /><a 
 id="x1-189049r5"></a> <span class="cmr-10x-x-109">40 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(50)".png  /><a 
 id="x1-189050r6"></a> <span class="cmr-10x-x-109">50 SL</span> <span class="cmr-10x-x-109">partners</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure J.12: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles with more genes</span></strong>. Synthetic lethal detection with quantiles as in axis labels. The line plots are coloured for SLIPT (red), <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> (blue) and correlation (grey), according to the legend. <a 
href="#glo:SLIPT">SLIPT</a> and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> performed similarly, peaking at <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span>-quantiles and converging to random (0.5). Negative correlation was higher than positive but not optimal quantiles for <a 
href="#glo:SLIPT">SLIPT</a> or <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> . These findings were robust across different numbers of underlying synthetic lethal genes in 1000 simulations of 20,000 genes and 1000 samples. <a 
href="#glo:SLIPT">SLIPT</a> performed better than <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> for higher numbers of synthetic lethal genes and finer quantiles.</span></p>
</div>
<!--tex4ht:label?: x1-18905112 -->
                                                                                    

</div><hr class="endfloat" />
<a 
 id="x1-189052r435"></a>
<h3 class="sectionHead">
<span class="titlemark">J.1 </span> <a 
 id="x1-1900001"></a>Correlated Query Genes affects Specificity
</h3>
                                                                                    

<!--l. 277--><p class="indent" >
<a 
 id="x1-19001014"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 279--><p class="noindent" >
</p><!--l. 284--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(1)".png  /><a 
 id="x1-190001r1"></a> <span class="cmr-10x-x-109">1 SL</span> <span class="cmr-10x-x-109">partner</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(2)".png  /><a 
 id="x1-190002r2"></a> <span class="cmr-10x-x-109">2 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(3)".png  /><a 
 id="x1-190003r3"></a> <span class="cmr-10x-x-109">3 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 297--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(4)".png  /><a 
 id="x1-190004r4"></a> <span class="cmr-10x-x-109">4 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(5)".png  /><a 
 id="x1-190005r5"></a> <span class="cmr-10x-x-109">5 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(6)".png  /><a 
 id="x1-190006r6"></a> <span class="cmr-10x-x-109">6 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 310--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(7)".png  /><a 
 id="x1-190007r7"></a> <span class="cmr-10x-x-109">7 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(8)".png  /><a 
 id="x1-190008r8"></a> <span class="cmr-10x-x-109">8 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(9)".png  /><a 
 id="x1-190009r9"></a> <span class="cmr-10x-x-109">9 SL</span> <span class="cmr-10x-x-109">partners</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure J.14: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles with query</span> <span class="cmbx-12">correlation</span></strong>. (continued on next page) </span></p>
</div>
<!--tex4ht:label?: x1-19001014 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 324--><p class="indent" >
<a 
 id="x1-19001716"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 326--><p class="noindent" >
</p><!--l. 331--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(10)".png  /><a 
 id="x1-190011r1"></a> <span class="cmr-10x-x-109">10 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(15)".png  /><a 
 id="x1-190012r2"></a> <span class="cmr-10x-x-109">15 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(20)".png  /><a 
 id="x1-190013r3"></a> <span class="cmr-10x-x-109">20 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 344--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(30)".png  /><a 
 id="x1-190014r4"></a> <span class="cmr-10x-x-109">30 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(40)".png  /><a 
 id="x1-190015r5"></a> <span class="cmr-10x-x-109">40 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_Cor_v_nCor_All(50)".png  /><a 
 id="x1-190016r6"></a> <span class="cmr-10x-x-109">50 SL</span> <span class="cmr-10x-x-109">partners</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure J.16: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles with query</span> <span class="cmbx-12">correlation</span></strong>. Synthetic lethal detection with quantiles as in axis labels. The barplots have the same hue for each quantile (grey for correlation) and darker for <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> (and positive correlation). <a 
href="#glo:SLIPT">SLIPT</a> and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> performed similarly, peaking at <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span>-quantiles and converging to random (0.5). Negative correlation was higher than positive but not optimal quantiles for <a 
href="#glo:SLIPT">SLIPT</a> or <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>. These findings were robust across different numbers of underlying synthetic lethal genes in 10,000 simulations of 100 genes (including 10 correlated with the query) and 1000 samples. <a 
href="#glo:SLIPT">SLIPT</a> performed consistently better than <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> with positively correlated genes.</span></p>
</div>
<!--tex4ht:label?: x1-19001716 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 359--><p class="indent" >
<a 
 id="x1-19002718"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 361--><p class="noindent" >
</p><!--l. 367--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(1)".png  /><a 
 id="x1-190018r1"></a> <span class="cmr-10x-x-109">1 SL</span> <span class="cmr-10x-x-109">partner</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(2)".png  /><a 
 id="x1-190019r2"></a> <span class="cmr-10x-x-109">2 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(3)".png  /><a 
 id="x1-190020r3"></a> <span class="cmr-10x-x-109">3 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 380--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(4)".png  /><a 
 id="x1-190021r4"></a> <span class="cmr-10x-x-109">4 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png  /><a 
 id="x1-190022r5"></a> <span class="cmr-10x-x-109">5 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(6)".png  /><a 
 id="x1-190023r6"></a> <span class="cmr-10x-x-109">6 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 393--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(7)".png  /><a 
 id="x1-190024r7"></a> <span class="cmr-10x-x-109">7 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(8)".png  /><a 
 id="x1-190025r8"></a> <span class="cmr-10x-x-109">8 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(9)".png  /><a 
 id="x1-190026r9"></a> <span class="cmr-10x-x-109">9 SL</span> <span class="cmr-10x-x-109">partners</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure J.18: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles with query</span> <span class="cmbx-12">correlation</span></strong>. (continued on next page) </span></p>
</div>
<!--tex4ht:label?: x1-19002718 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 407--><p class="indent" >
<a 
 id="x1-19003420"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 409--><p class="noindent" >
</p><!--l. 414--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(10)".png  /><a 
 id="x1-190028r1"></a> <span class="cmr-10x-x-109">10 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(15)".png  /><a 
 id="x1-190029r2"></a> <span class="cmr-10x-x-109">15 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(20)".png  /><a 
 id="x1-190030r3"></a> <span class="cmr-10x-x-109">20 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 427--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(30)".png  /><a 
 id="x1-190031r4"></a> <span class="cmr-10x-x-109">30 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(40)".png  /><a 
 id="x1-190032r5"></a> <span class="cmr-10x-x-109">40 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161108_query_cor/SL_Model_Test_Graph_10K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(50)".png  /><a 
 id="x1-190033r6"></a> <span class="cmr-10x-x-109">50 SL</span> <span class="cmr-10x-x-109">partners</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure J.20: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles with query</span> <span class="cmbx-12">correlation</span></strong>. Synthetic lethal detection with quantiles as in axis labels. The line plots are coloured for SLIPT (red), <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> (blue) and correlation (grey), according to the legend. <a 
href="#glo:SLIPT">SLIPT</a> and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> performed similarly, peaking at <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span>-quantiles and converging to random (0.5). Negative correlation was higher than positive but not optimal quantiles for <a 
href="#glo:SLIPT">SLIPT</a> or <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> . These findings were robust across different numbers of underlying synthetic lethal genes in 10,000 simulations of 100 genes (including 10 correlated with the query) and 1000 samples. <a 
href="#glo:SLIPT">SLIPT</a> performed consistently better than <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> with positively correlated genes.</span></p>
</div>
<!--tex4ht:label?: x1-19003420 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 443--><p class="indent" >
<a 
 id="x1-19004422"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 445--><p class="noindent" >
</p><!--l. 451--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(1)".png  /><a 
 id="x1-190035r1"></a> <span class="cmr-10x-x-109">1 SL</span> <span class="cmr-10x-x-109">partner</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(2)".png  /><a 
 id="x1-190036r2"></a> <span class="cmr-10x-x-109">2 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(3)".png  /><a 
 id="x1-190037r3"></a> <span class="cmr-10x-x-109">3 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 464--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(4)".png  /><a 
 id="x1-190038r4"></a> <span class="cmr-10x-x-109">4 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(5)".png  /><a 
 id="x1-190039r5"></a> <span class="cmr-10x-x-109">5 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(6)".png  /><a 
 id="x1-190040r6"></a> <span class="cmr-10x-x-109">6 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 477--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(7)".png  /><a 
 id="x1-190041r7"></a> <span class="cmr-10x-x-109">7 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(8)".png  /><a 
 id="x1-190042r8"></a> <span class="cmr-10x-x-109">8 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(9)".png  /><a 
 id="x1-190043r9"></a> <span class="cmr-10x-x-109">9 SL</span> <span class="cmr-10x-x-109">partners</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure J.22: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles with query</span> <span class="cmbx-12">correlation and more genes</span></strong>. (continued on next page) </span></p>
</div>
<!--tex4ht:label?: x1-19004422 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 492--><p class="indent" >
<a 
 id="x1-19005124"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 494--><p class="noindent" >
</p><!--l. 499--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(10)".png  /><a 
 id="x1-190045r1"></a> <span class="cmr-10x-x-109">10 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(15)".png  /><a 
 id="x1-190046r2"></a> <span class="cmr-10x-x-109">15 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(20)".png  /><a 
 id="x1-190047r3"></a> <span class="cmr-10x-x-109">20 SL</span> <span class="cmr-10x-x-109">partners</span>
</p><!--l. 512--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(30)".png  /><a 
 id="x1-190048r4"></a> <span class="cmr-10x-x-109">30 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(40)".png  /><a 
 id="x1-190049r5"></a> <span class="cmr-10x-x-109">40 SL</span> <span class="cmr-10x-x-109">partners</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161107_query_cor/SL_Model_Test_Graph_1K_Graph5_ROC_SLIPT_v_ChiSq_v_nCor_Ally(50)".png  /><a 
 id="x1-190050r6"></a> <span class="cmr-10x-x-109">50 SL</span> <span class="cmr-10x-x-109">partners</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure J.24: </span><span class="content"><strong><span class="cmbx-12">Performance of </span><span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmbx-12">and SLIPT across quantiles with query</span> <span class="cmbx-12">correlation and more genes</span></strong>. Synthetic lethal detection with quantiles as in axis labels. The line plots are coloured for SLIPT (red), <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> (blue) and correlation (grey), according to the legend. <a 
href="#glo:SLIPT">SLIPT</a> and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> performed similarly, peaking at <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span>-quantiles and converging to random (0.5). Negative correlation was higher than positive but not optimal quantiles for <a 
href="#glo:SLIPT">SLIPT</a> or <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>. These findings were robust across different numbers of underlying synthetic lethal genes in 1000 simulations of 20,000 genes (including 1000 correlated with the query) and 1000 samples. <a 
href="#glo:SLIPT">SLIPT</a> performed consistently better than <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> with positively correlated genes.</span></p>
</div>
<!--tex4ht:label?: x1-19005124 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<a 
 id="x1-190052r356"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix K</span><br /><a 
 id="x1-191000K"></a>Simulations on Graph Structures
</h2>
                                                                                    

<!--l. 827--><p class="indent" >
<a 
 id="x1-1910052"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 830--><p class="noindent" >
</p><!--l. 834--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph2_ROC1_samplesx".png  /><a 
 id="x1-191001r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph2_ROC2_samplesx".png  /><a 
 id="x1-191002r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a>
</p><!--l. 843--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph2".png /> <a 
 id="x1-191003r3"></a> <span class="cmr-10x-x-109">Graph</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph2_AUC_samplesx".png  /><a 
 id="x1-191004r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a simple graph.</span></strong> <span class="cmr-10x-x-109">Simulation of synthetic</span> <span class="cmr-10x-x-109">lethality was performed using a multivariate normal distribution from a converging graph. For</span> <span class="cmr-10x-x-109">each parameter, 10,000 simulations were used. Colours in Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-191002r2"><span class="cmr-10x-x-109">K.1b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph2ROC:ROC --></a> <span class="cmr-10x-x-109">match Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-191004r4"><span class="cmr-10x-x-109">K.1d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph2ROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-1910052 -->
                                                                                    

</div><hr class="endfloat" />
<a 
 id="x1-191006r432"></a>
<h4 class="subsectionHead">
<span class="titlemark">K.0.1 </span> <a 
 id="x1-1920001"></a>Simulations from Inhibiting Graph Structures
</h4>
                                                                                    

<!--l. 1060--><p class="indent" >
<a 
 id="x1-1920054"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 1063--><p class="noindent" >
</p><!--l. 1067--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph2i_ROC1_samplesx".png  /><a 
 id="x1-192001r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph2i_ROC2_samplesx".png  /><a 
 id="x1-192002r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a>
</p><!--l. 1076--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph2i".png /> <a 
 id="x1-192003r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph2i_AUC_samplesx".png  /><a 
 id="x1-192004r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.4: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on an inhibiting graph.</span></strong> <span class="cmr-10x-x-109">Simulation of</span> <span class="cmr-10x-x-109">synthetic lethality used a multivariate normal distribution from a converging graph. For each</span> <span class="cmr-10x-x-109">parameter, 10,000 simulations were used. Colours in Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-192002r2"><span class="cmr-10x-x-109">K.3b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph2iROC:ROC --></a> <span class="cmr-10x-x-109">match Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-192004r4"><span class="cmr-10x-x-109">K.3d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph2iROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-1920054 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 1121--><p class="indent" >
<a 
 id="x1-1920106"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 1124--><p class="noindent" >
</p><!--l. 1129--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4i_ROC1_samplesx".png  /><a 
 id="x1-192006r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4i_ROC2_samplesx".png  /><a 
 id="x1-192007r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a>
</p><!--l. 1139--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4i".png /> <a 
 id="x1-192008r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4i_AUC_samplesx".png  /><a 
 id="x1-192009r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.6: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a constructed graph with inhibition.</span></strong> <span class="cmr-10x-x-109">Simulation of synthetic lethality used a multivariate normal distribution from a pathway with</span> <span class="cmr-10x-x-109">only inhibitions. For each parameter, 10,000 simulations were used. Colours in Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-192007r2"><span class="cmr-10x-x-109">K.5b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph4iROC:ROC --></a> <span class="cmr-10x-x-109">match Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-192009r4"><span class="cmr-10x-x-109">K.5d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph4iROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-1920106 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 1151--><p class="indent" >
<a 
 id="x1-1920158"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 1154--><p class="noindent" >
</p><!--l. 1159--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4i2_ROC1_samplesx".png  /><a 
 id="x1-192011r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4i2_ROC2_samplesx".png  /><a 
 id="x1-192012r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a>
</p><!--l. 1169--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4i2".png /> <a 
 id="x1-192013r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph4i2_AUC_samplesx".png  /><a 
 id="x1-192014r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.8: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a constructed graph with inhibition.</span></strong> <span class="cmr-10x-x-109">Simulation of synthetic lethality used a multivariate normal distribution from a pathway with</span> <span class="cmr-10x-x-109">a combination of inhibitions. For each parameter, 10,000 simulations were used. Colours in</span> <span class="cmr-10x-x-109">Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-192012r2"><span class="cmr-10x-x-109">K.7b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph4i2ROC:ROC --></a> <span class="cmr-10x-x-109">match Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-192014r4"><span class="cmr-10x-x-109">K.7d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph4i2ROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-1920158 -->
                                                                                    

</div><hr class="endfloat" />
<a 
 id="x1-192016r509"></a>
<h3 class="sectionHead">
<span class="titlemark">K.1 </span> <a 
 id="x1-1930001"></a>Simulation across Graph Structures
</h3>
                                                                                    

<!--l. 1190--><p class="indent" >
<a 
 id="x1-19300710"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 1192--><p class="noindent" >
</p><!--l. 1196--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4_coloured".png /><a 
 id="x1-193001r1"></a> <span class="cmr-10x-x-109">Activating</span> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(1)".png /><a 
 id="x1-193002r2"></a> <span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">distribution</span> <span class="cmr-10x-x-109">for</span> <span class="cmr-10x-x-109">“A”</span> <span class="cmr-10x-x-109">SL</span>
</p><!--l. 1205--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(2)".png /><a 
 id="x1-193003r3"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“B”</span> <span class="cmr-10x-x-109">SL</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(3)".png /><a 
 id="x1-193004r4"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“C”</span> <span class="cmr-10x-x-109">SL</span>
</p><!--l. 1214--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(4)".png /><a 
 id="x1-193005r5"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“D”</span> <span class="cmr-10x-x-109">SL</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(5)".png /><a 
 id="x1-193006r6"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“E”</span> <span class="cmr-10x-x-109">SL</span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure K.10: </span><span class="content"><strong><span class="cmbx-10x-x-109">Detection of synthetic lethality within a graph Structure.</span></strong> <span class="cmr-10x-x-109">(continued</span> <span class="cmr-10x-x-109">on next page) </span></span></p>
</div>
<!--tex4ht:label?: x1-19300710 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 1226--><p class="indent" >
<a 
 id="x1-19301212"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 1228--><p class="noindent" >
</p><!--l. 1232--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(6)".png /><a 
 id="x1-193008r1"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“F”</span> <span class="cmr-10x-x-109">SL</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(7)".png /><a 
 id="x1-193009r2"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“G”</span> <span class="cmr-10x-x-109">SL</span>
</p><!--l. 1242--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(9)".png /><a 
 id="x1-193010r3"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“H”</span> <span class="cmr-10x-x-109">SL</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4_ROC_samples_SLstry_vioplot(8)".png /><a 
 id="x1-193011r4"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“I” SL</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.12: </span><span class="content"><strong><span class="cmbx-10x-x-109">Detection of synthetic lethality within a graph structure.</span></strong> <span class="cmr-10x-x-109">Each gene</span> <span class="cmr-10x-x-109">was designated to be synthetic lethal separately and the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">value from </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">was computed</span> <span class="cmr-10x-x-109">for each gene across the graph. For each synthetic lethal gene (highlighted in the respective</span> <span class="cmr-10x-x-109">colours), the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">values were computed in 100 simulations of datasets of 20,000 genes including</span> <span class="cmr-10x-x-109">the graph structure and 1000 samples. For each synthetic lethal gene, the adjacent genes in</span> <span class="cmr-10x-x-109">the network also had elevated test statistics.</span></span></p>
</div>
<!--tex4ht:label?: x1-19301212 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 1253--><p class="indent" >
<a 
 id="x1-19302314"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 1255--><p class="noindent" >
</p><!--l. 1259--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4i_coloured".png /><a 
 id="x1-193013r1"></a> <span class="cmr-10x-x-109">Inhibiting</span> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i_ROC_samples_SLstry_vioplot(1)".png /><a 
 id="x1-193014r2"></a> <span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">distribution</span> <span class="cmr-10x-x-109">for</span> <span class="cmr-10x-x-109">“A”</span> <span class="cmr-10x-x-109">SL</span>
</p><!--l. 1268--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i_ROC_samples_SLstry_vioplot(2)".png /><a 
 id="x1-193015r3"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“B”</span> <span class="cmr-10x-x-109">SL</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i_ROC_samples_SLstry_vioplot(3)".png /><a 
 id="x1-193016r4"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“C”</span> <span class="cmr-10x-x-109">SL</span> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i_ROC_samples_SLstry_vioplot(4)".png /><a 
 id="x1-193017r5"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“D”</span> <span class="cmr-10x-x-109">SL</span> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i_ROC_samples_SLstry_vioplot(5)".png /><a 
 id="x1-193018r6"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“E”</span> <span class="cmr-10x-x-109">SL</span>
</p><!--l. 1286--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i_ROC_samples_SLstry_vioplot(6)".png /><a 
 id="x1-193019r7"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“F”</span> <span class="cmr-10x-x-109">SL</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i_ROC_samples_SLstry_vioplot(7)".png /><a 
 id="x1-193020r8"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“G”</span> <span class="cmr-10x-x-109">SL</span> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i_ROC_samples_SLstry_vioplot(9)".png /><a 
 id="x1-193021r9"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“H”</span> <span class="cmr-10x-x-109">SL</span> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i_ROC_samples_SLstry_vioplot(8)".png /><a 
 id="x1-193022r10"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“I” SL</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.14: </span><span class="content"><strong><span class="cmbx-10x-x-109">Detection of synthetic lethality within an inhibiting graph.</span></strong> <span class="cmr-10x-x-109">Each</span> <span class="cmr-10x-x-109">gene was designated to be synthetic lethal separately and the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">value from </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">was</span> <span class="cmr-10x-x-109">computed for each gene across the graph structure with inhibiting relationships. For each</span> <span class="cmr-10x-x-109">synthetic lethal gene (highlighted in the respective colours), the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">values were computed in</span> <span class="cmr-10x-x-109">100 simulations of datasets of 20,000 genes including the graph structure and 1000 samples.</span> <span class="cmr-10x-x-109">For each synthetic lethal gene, the adjacent genes exhibited lower </span><span class="cmmi-10x-x-109">χ</span><sup></sup>values with inhibiting relationships.</span></p>
</div>
<!--tex4ht:label?: x1-19302314 -->
<p><a 
 id="x1-193024r13"></a></p>
</div><hr class="endfloat" />
                                                                                    

<!--l. 1308--><p class="indent" >
<a 
 id="x1-19303515"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 1310--><p class="noindent" >
</p><!--l. 1314--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4i2_coloured".png /><a 
 id="x1-193025r1"></a> <span class="cmr-10x-x-109">Inhibiting</span> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i2_ROC_samples_SLstry_vioplot(1)".png /><a 
 id="x1-193026r2"></a> <span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">distribution</span> <span class="cmr-10x-x-109">for</span> <span class="cmr-10x-x-109">“A”</span> <span class="cmr-10x-x-109">SL</span>
</p><!--l. 1323--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i2_ROC_samples_SLstry_vioplot(2)".png /><a 
 id="x1-193027r3"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“B”</span> <span class="cmr-10x-x-109">SL</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i2_ROC_samples_SLstry_vioplot(3)".png /><a 
 id="x1-193028r4"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“C”</span> <span class="cmr-10x-x-109">SL</span> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i2_ROC_samples_SLstry_vioplot(4)".png /><a 
 id="x1-193029r5"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“D”</span> <span class="cmr-10x-x-109">SL</span> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i2_ROC_samples_SLstry_vioplot(5)".png /><a 
 id="x1-193030r6"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“E”</span> <span class="cmr-10x-x-109">SL</span>
</p><!--l. 1341--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i2_ROC_samples_SLstry_vioplot(6)".png /><a 
 id="x1-193031r7"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“F”</span> <span class="cmr-10x-x-109">SL</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i2_ROC_samples_SLstry_vioplot(7)".png /><a 
 id="x1-193032r8"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“G”</span> <span class="cmr-10x-x-109">SL</span> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i2_ROC_samples_SLstry_vioplot(9)".png /><a 
 id="x1-193033r9"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“H”</span> <span class="cmr-10x-x-109">SL</span> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model//RUN_20161206_Str_randx/SL_Model_Test_Graph_1K_Graph4i2_ROC_samples_SLstry_vioplot(8)".png /><a 
 id="x1-193034r10"></a> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">“I” SL</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.15: </span><span class="content"><strong><span class="cmbx-10x-x-109">Detection of synthetic lethality within an inhibiting graph.</span></strong> <span class="cmr-10x-x-109">Each</span> <span class="cmr-10x-x-109">gene was designated to be synthetic lethal separately and the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">value from </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">was</span> <span class="cmr-10x-x-109">computed for each gene across the graph structure with inhibiting and relationships. For each</span> <span class="cmr-10x-x-109">synthetic lethal gene (highlighted in the respective colours), the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">values were computed in</span> <span class="cmr-10x-x-109">100 simulations of datasets of 20,000 genes including the graph structure and 1000 samples.</span></span></p>
</div>
<!--tex4ht:label?: x1-19303515 -->
<p><a 
 id="x1-193036r14"></a></p>
</div><hr class="endfloat" />
<a 
 id="x1-193037r583"></a>
<h3 class="sectionHead">
<span class="titlemark">K.2 </span> <a 
 id="x1-1940002"></a>Simulations from Complex Graph Structures
</h3>
                                                                                    

<!--l. 1967--><p class="indent" >
<a 
 id="x1-19400516"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 1970--><p class="noindent" >
</p><!--l. 1974--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph6_ROC1_samplesx".png  /><a 
 id="x1-194001r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph6_ROC2_samplesx".png  /><a 
 id="x1-194002r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a>
</p><!--l. 1983--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph6".png /> <a 
 id="x1-194003r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph6_AUC_samplesx".png  /><a 
 id="x1-194004r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.16: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a branching graph.</span></strong> <span class="cmr-10x-x-109">Simulation of</span> <span class="cmr-10x-x-109">synthetic lethality used a multivariate normal distribution from a branching graph. For each</span> <span class="cmr-10x-x-109">parameter, 10,000 simulations were used. Colours in Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-194002r2"><span class="cmr-10x-x-109">K.15b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph6ROC:ROC --></a> <span class="cmr-10x-x-109">match Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-194004r4"><span class="cmr-10x-x-109">K.15d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph6ROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-19400516 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 1995--><p class="indent" >
<a 
 id="x1-19401018"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 1998--><p class="noindent" >
</p><!--l. 2002--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph7_ROC1_samplesx".png  /><a 
 id="x1-194006r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph7_ROC2_samplesx".png  /><a 
 id="x1-194007r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a>
</p><!--l. 2010--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph7".png /><a 
 id="x1-194008r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span>
</p><!--l. 2015--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph7_AUC_samplesx".png  /><a 
 id="x1-194009r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.18: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a complex graph.</span></strong> <span class="cmr-10x-x-109">Simulation of synthetic</span> <span class="cmr-10x-x-109">lethality used a multivariate normal distribution from a complex graph. For each parameter,</span> <span class="cmr-10x-x-109">10,000 simulations were used. Colours in Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-194007r2"><span class="cmr-10x-x-109">K.17b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph7ROC:ROC --></a> <span class="cmr-10x-x-109">match Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-194009r4"><span class="cmr-10x-x-109">K.17d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph7ROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-19401018 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 2024--><p class="indent" >
<a 
 id="x1-19401520"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 2027--><p class="noindent" >
</p><!--l. 2031--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph5_ROC1_samplesx".png  /><a 
 id="x1-194011r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph5_ROC2_samplesx".png  /><a 
 id="x1-194012r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a>
</p><!--l. 2041--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph5".png /><a 
 id="x1-194013r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span>
</p><!--l. 2045--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph5_AUC_samplesx_prop".png  /><a 
 id="x1-194014r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.20: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a large graph.</span></strong> <span class="cmr-10x-x-109">Simulation of synthetic</span> <span class="cmr-10x-x-109">lethality used a multivariate normal distribution from a large graph. For each parameter,</span> <span class="cmr-10x-x-109">10,000 simulations were used. Colours in Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-194012r2"><span class="cmr-10x-x-109">K.19b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph5ROC:ROC --></a> <span class="cmr-10x-x-109">match Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-194014r4"><span class="cmr-10x-x-109">K.19d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph5ROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-19401520 -->
                                                                                    

</div><hr class="endfloat" />
<a 
 id="x1-194016r567"></a>
<h4 class="subsectionHead">
<span class="titlemark">K.2.1 </span> <a 
 id="x1-1950001"></a>Simulations from Complex Inhibiting Graphs
</h4>
                                                                                    

<!--l. 2062--><p class="indent" >
<a 
 id="x1-19500522"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 2065--><p class="noindent" >
</p><!--l. 2070--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph6i_ROC1_samplesx".png  /><a 
 id="x1-195001r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph6i_ROC2_samplesx".png  /><a 
 id="x1-195002r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a>
</p><!--l. 2080--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph6i".png /> <a 
 id="x1-195003r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph6i_AUC_samplesx_prop".png  /><a 
 id="x1-195004r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.22: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a branching graph with inhibition.</span></strong> <span class="cmr-10x-x-109">Simulation of synthetic lethality used a multivariate normal distribution from a branching</span> <span class="cmr-10x-x-109">graph with only inhibitions. For each parameter, 10,000 simulations were used. Colours in</span> <span class="cmr-10x-x-109">Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-195002r2"><span class="cmr-10x-x-109">K.21b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph6iROC:ROC --></a> <span class="cmr-10x-x-109">match Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-195004r4"><span class="cmr-10x-x-109">K.21d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph6iROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-19500522 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 2092--><p class="indent" >
<a 
 id="x1-19501024"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 2095--><p class="noindent" >
</p><!--l. 2100--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph6i2_ROC1_samplesx".png  /><a 
 id="x1-195006r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph6i2_ROC2_samplesx".png  /><a 
 id="x1-195007r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a>
</p><!--l. 2110--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph6i2".png /><a 
 id="x1-195008r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span>
</p><!--l. 2114--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph6i2_AUC_samplesx_prop".png  /><a 
 id="x1-195009r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.24: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a branching graph with inhibition.</span></strong> <span class="cmr-10x-x-109">Simulation of synthetic lethality used a multivariate normal distribution from a branching</span> <span class="cmr-10x-x-109">graph with alternating inhibitions. For each parameter, 10,000 simulations were used. Colours</span> <span class="cmr-10x-x-109">in Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-195007r2"><span class="cmr-10x-x-109">K.23b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph6i2ROC:ROC --></a> <span class="cmr-10x-x-109">match Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-195009r4"><span class="cmr-10x-x-109">K.23d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph6i2ROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-19501024 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 2123--><p class="indent" >
<a 
 id="x1-19501526"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 2126--><p class="noindent" >
</p><!--l. 2131--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph7i_ROC1_samplesx".png  /><a 
 id="x1-195011r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph7i_ROC2_samplesx".png  /><a 
 id="x1-195012r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a>
</p><!--l. 2141--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph7i".png /><a 
 id="x1-195013r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span>
</p><!--l. 2145--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph7i_AUC_samplesx_prop".png  /><a 
 id="x1-195014r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.26: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a complex graph with inhibition.</span></strong> <span class="cmr-10x-x-109">Simulation of synthetic lethality used a multivariate normal distribution from a complex</span> <span class="cmr-10x-x-109">graph with only inhibitions. For each parameter, 10,000 simulations were used. Colours in</span> <span class="cmr-10x-x-109">Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-195012r2"><span class="cmr-10x-x-109">K.25b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph7iROC:ROC --></a> <span class="cmr-10x-x-109">match Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-195014r4"><span class="cmr-10x-x-109">K.25d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph7iROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-19501526 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 2155--><p class="indent" >
<a 
 id="x1-19502028"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 2158--><p class="noindent" >
</p><!--l. 2163--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph7i2_ROC1_samplesx".png  /><a 
 id="x1-195016r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph7i2_ROC2_samplesx".png  /><a 
 id="x1-195017r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a>
</p><!--l. 2173--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph7i2".png /><a 
 id="x1-195018r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span>
</p><!--l. 2177--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph7i2_AUC_samplesx_prop".png  /><a 
 id="x1-195019r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.28: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a complex graph with inhibition.</span></strong> <span class="cmr-10x-x-109">Simulation of synthetic lethality used a multivariate normal distribution from a complex</span> <span class="cmr-10x-x-109">graph with a combination of relationships. For each parameter, 10,000 simulations were used.</span> <span class="cmr-10x-x-109">Colours in Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-195017r2"><span class="cmr-10x-x-109">K.27b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph7i2ROC:ROC --></a> <span class="cmr-10x-x-109">match Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-195019r4"><span class="cmr-10x-x-109">K.27d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph7i2ROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-19502028 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 2186--><p class="indent" >
<a 
 id="x1-19502530"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 2189--><p class="noindent" >
</p><!--l. 2194--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph5i_ROC1_samplesx".png  /><a 
 id="x1-195021r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph5i_ROC2_samplesx".png  /><a 
 id="x1-195022r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a>
</p><!--l. 2204--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph5i".png /><a 
 id="x1-195023r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span>
</p><!--l. 2208--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph5i_AUC_samplesx_prop".png  /><a 
 id="x1-195024r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.30: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a large constructed graph with</span> <span class="cmbx-10x-x-109">inhibition.</span></strong> <span class="cmr-10x-x-109">Simulation of synthetic lethality used a multivariate normal distribution from a</span> <span class="cmr-10x-x-109">large graph with only inhibitions. For each parameter, 10,000 simulations were used. Colours</span> <span class="cmr-10x-x-109">in Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-195022r2"><span class="cmr-10x-x-109">K.29b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph5iROC:ROC --></a> <span class="cmr-10x-x-109">match Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-195024r4"><span class="cmr-10x-x-109">K.29d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph5iROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-19502530 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 2217--><p class="indent" >
<a 
 id="x1-19503032"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 2220--><p class="noindent" >
</p><!--l. 2225--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph5i2_ROC1_samplesx".png  /><a 
 id="x1-195026r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph5i2_ROC2_samplesx".png  /><a 
 id="x1-195027r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a>
</p><!--l. 2235--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph5i2".png /><a 
 id="x1-195028r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span>
</p><!--l. 2240--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Graph5i2_AUC_samplesx_prop".png  /><a 
 id="x1-195029r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.32: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on a large constructed graph with</span> <span class="cmbx-10x-x-109">inhibition.</span></strong> <span class="cmr-10x-x-109">Simulation of synthetic lethality used a multivariate normal distribution from a</span> <span class="cmr-10x-x-109">large graph with alternating inhibitions. For each parameter, 10,000 simulations were used.</span> <span class="cmr-10x-x-109">Colours in Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-195027r2"><span class="cmr-10x-x-109">K.31b</span><!--tex4ht:ref: fig:simulation1205_randx_Graph5i2ROC:ROC --></a> <span class="cmr-10x-x-109">match Figure</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-195029r4"><span class="cmr-10x-x-109">K.31d</span><!--tex4ht:ref: fig:simulation1205_randx_Graph5i2ROC:AUC --></a><span class="cmr-10x-x-109">.</span></span></p>
</div>
<!--tex4ht:label?: x1-19503032 -->
                                                                                    

</div><hr class="endfloat" />
<a 
 id="x1-195031r618"></a>
<h3 class="sectionHead">
<span class="titlemark">K.3 </span> <a 
 id="x1-1960003"></a>Simulations from Pathway Graph Structures
</h3>
                                                                                    

<!--l. 2323--><p class="indent" >
<a 
 id="x1-19600534"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 2326--><p class="noindent" >
</p><!--l. 2331--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Gai_ROC1_samplesx".png  /><a 
 id="x1-196001r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Gai_ROC2_samplesx".png  /><a 
 id="x1-196002r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a>
</p><!--l. 2342--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph_Gai".png /><a 
 id="x1-196003r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161205_randx/SL_Model_Test_Graph_10K_Gai_AUC_samplesx_prop".png  /><a 
 id="x1-196004r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.34: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations on theG</span><sub><span class="cmmi-8">αi</span></sub> <span class="cmbx-10x-x-109">signalling pathway.</span></strong> <span class="cmr-10x-x-109">Simulation</span> <span class="cmr-10x-x-109">of synthetic lethality used a multivariate normal distribution based on the Reactome G</span><sub><span class="cmmi-8">αi</span></sub> <span class="cmr-10x-x-109">signalling pathway. Performance of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">was high across parameters for detecting synthetic</span> <span class="cmr-10x-x-109">lethality in the graph structure within a larger dataset. The performance decreased for a</span> <span class="cmr-10x-x-109">greater number of true positives to detect but the accuracy increased with a low false positive</span> <span class="cmr-10x-x-109">rate.</span></span></p>
</div>
<!--tex4ht:label?: x1-19600534 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    

<!--l. 2355--><p class="indent" >
<a 
 id="x1-19601036"></a>
</p><hr class="float" /><div class="float" 
>
                                                                                    

<div class="center" 
>
<!--l. 2358--><p class="noindent" >
</p><!--l. 2363--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Gai_ROC1_samplesx".png  /><a 
 id="x1-196006r1"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">evaluation</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Gai_ROC2_samplesx".png  /><a 
 id="x1-196007r2"></a> <a 
href="#glo:ROC"><span class="cmr-10x-x-109">ROC</span></a>
</p><!--l. 2374--><p class="noindent" >
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph_Gai".png /><a 
 id="x1-196008r3"></a> <span class="cmr-10x-x-109">Graph</span> <span class="cmr-10x-x-109">Structure</span><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/RUN_20161207_randx/SL_Model_Test_Graph_1K_Gai_AUC_samplesx_prop".png  /><a 
 id="x1-196009r4"></a> <span class="cmr-10x-x-109">Statistical</span> <span class="cmr-10x-x-109">performance</span>
</p></div>
<br />
<div class="caption">
<p><span class="id">Figure K.36: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of simulations including the G</span><sub><span class="cmmi-8">αi</span></sub> <span class="cmbx-10x-x-109">signalling pathway.</span></strong> <span class="cmr-10x-x-109">Simulation of synthetic lethality used a multivariate normal distribution (without correlation</span> <span class="cmr-10x-x-109">structure apart from the Reactome G</span><sub><span class="cmmi-8">αi</span></sub> <span class="cmr-10x-x-109">signalling pathway. Performance of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">was high</span> <span class="cmr-10x-x-109">across parameters for detecting synthetic lethality in the graph structure within a larger</span> <span class="cmr-10x-x-109">dataset. The sensitivity decreased for a greater number of true positives to detect but the</span> <span class="cmr-10x-x-109">specificity remained high with a low false positive rate.</span></span></p>
</div>
<!--tex4ht:label?: x1-19601036 -->
                                                                                    

</div><hr class="endfloat" />
                                                                                    


</body></html>
</body>
</html>
