<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
</head>
<body>
<?xml version="1.0" encoding="iso-8859-1" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" 
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!--http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd-->
<html xmlns="http://www.w3.org/1999/xhtml"  
>
<head>
<title>
A Bioinformatics Approach to Synthetic Lethal Interactions in Breast Cancer with Gene Expression Data
</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" /> <meta name="generator" content="TeX4ht (http://www.cse.ohio-state.edu/~gurari/TeX4ht/)" /> <meta name="originator" content="TeX4ht (http://www.cse.ohio-state.edu/~gurari/TeX4ht/)" /> <!-- html,xhtml --> <meta name="src" content="thesis.tex" /> <meta name="date" content="2017-05-12 03:38:00" /> <link rel="stylesheet" type="text/css" href="thesis.css" />
</head>
<body 
>
<div class="center">
<!--l. 146-->
<p class="noindent">
</p>
<!--l. 146-->
<p class="noindent">
<span class="cmbx-12">Library Declaration Form</span><br /> <img 
src="logo.png" alt="pict"  
 /> <br /> University of Otago Library
</p>
</div>
<table cellpadding="0" border="0" cellspacing="0" class="tabbing">
<tr style="vertical-align:baseline;" class="tabbing">
<td class="tabbing">
</td>
</tr>
</table>
<!--l. 146-->
<p class="noindent" >
<pre><code>                      &lt;/p&gt;&lt;table  </code></pre>
cellpadding=“0” border=“0” cellspacing=“0” class=“tabbing”&gt;
<tr style="vertical-align:baseline;" class="tabbing">
<td class="tabbing">
Author’s full name and year of birth: Simon Thomas Kelly,
</td>
</tr>
</table>
<!--l. 146-->
<p class="noindent" >
<pre><code>                          &lt;/p&gt;&lt;table  </code></pre>
cellpadding=“0” border=“0” cellspacing=“0” class=“tabbing”&gt;
<tr style="vertical-align:baseline;" class="tabbing">
<td style="width:292;" class="tabbing">
   (for cataloguing purposes)
</td>
<td class="tabbing">
24 February 1992
</td>
</tr>
</table>
<!--l. 146-->
<p class="noindent" >
<pre><code>                                                                                &lt;/p&gt;&lt;table  </code></pre>
cellpadding=“0” border=“0” cellspacing=“0” class=“tabbing”&gt;
<tr style="vertical-align:baseline;" class="tabbing">
<td class="tabbing">
</td>
</tr>
</table>
<!--l. 146-->
<p class="noindent">
</p>
<!--l. 146-->
<p class="noindent">
Title of thesis: A Bioinformatics Approach to Synthetic Lethal Interactions in Breast Cancer with Gene Expression Data<br 
class="newline" />
</p>
<!--l. 146-->
<p class="noindent">
Degree: Doctor of Philosophy<br 
class="newline" />
</p>
<!--l. 146-->
<p class="noindent">
Department: Department of Biochemistry<br 
class="newline" />
</p>
<!--l. 146-->
<p class="noindent">
Permanent Address: 710 Cumberland Street, Dunedin, NZ<br 
class="newline" />
</p>
<!--l. 146-->
<p class="noindent">
I agree that this thesis may be consulted for research and study purposes and that reasonable quotation may be made from it, provided that proper acknowledgement of its use is made.<br 
class="newline" />
</p>
<!--l. 146-->
<p class="noindent">
I consent to this thesis being copied in part or in whole for
</p>
<dl class="list1">
<dt class="list">
<ol style="list-style-type: lower-roman">
<li></li>
</ol>
</dt>
<dd class="list">
<!--l. 146-->
<p class="noindent">
a library
</p>
</dd>
<dt class="list">
<ol start="2" style="list-style-type: lower-roman">
<li></li>
</ol>
</dt>
<dd class="list">
<!--l. 146-->
<p class="noindent">
an individual
</p>
</dd>
</dl>
<!--l. 146-->
<p class="noindent">
at the discretion of the University of Otago.<br 
class="newline" />
</p>
<!--l. 146-->
<p class="noindent">
      Signature:<br 
class="newline" />
</p>
<!--l. 146-->
<p class="noindent">
<p>      Date:</p>
</p>
<div class="maketitle">
<h2 class="titleHead">
A Bioinformatics Approach to Synthetic Lethal Interactions in Breast Cancer with Gene Expression Data
</h2>
<div class="author">
<p><span class="cmr-12x-x-120">S. Thomas Kelly</span></p>
</div>
a thesis submitted for the degree of <span class="cmr-12x-x-120">Doctor of Philosophy</span> at the University of Otago, Dunedin, New Zealand.
<div class="date">
<p>31 March 2017</p>
</div>
</div>
<div class="abstract">
<div class="center">
<!--l. 146-->
<p class="noindent">
</p>
<!--l. 146-->
<p class="noindent">
<span class="cmbx-12">Abstract</span>
</p>
</div>
<div class="quote">
<!--l. 2-->
<p class="noindent">
<strong><span class="cmbx-12">Background</span></strong>
</p>
<!--l. 4-->
<p class="noindent">
Synthetic lethal interactions are re-emerging in genetics research in the genomics era driven by potential applications in precision medicine against cancers. This approach aims to exploit functional redundancy at the genetic level against mutations in cancers for developing specific treatments against them, including loss of function events in tumour suppressors. Of particular interest is the targeting loss of function of E-cadherin, encoded by <span class="cmti-12">CDH1</span>, a tumour supressor gene involved in Breast and Stomach cancers. Experimental screens have been used to identify candidate synthetic lethal interactions and here bioinformatics analysis used to augment the triage drug target triage process. Furthermore the pathway composition of synthetic lethal candidates and the effect of pathway structure on their detection in genomics data.
</p>
<!--l. 7-->
<p class="noindent">
<strong><span class="cmbx-12">Approach</span></strong>
</p>
<!--l. 9-->
<p class="noindent">
<p>A computational statistics methodology, the Synthetic Lethal Prediction Tool (SLIPT) has been developed to detect synthetic lethal interactions in gene expression datasets. The methodology has been demonstrated on Breast and Stomach cancer datasets from The Cancer Genome Atlas (TCGA) database, testing for interactions with <span class="cmti-12">CDH1</span>. Various analyses have been applied to further euclidate these candidates, including differential</p>
gene expression, correlation co-expression, unsupervised clustering, gene set over-representation analysis, singular-value decomposition “metagenes”, and permutation re-sampling analysis. A particular challenge of performing these analyses was to compare SLIPT gene candidates to the results of an experimental synthetic lethal siRNA screen of E-cadherin <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) at the pathway level. Graph theory methods including information centrality and shortest paths were applied to the most supported pathways from both the computational and experimental synthetic lethal candidates to test for graph structure among hits from each approach. Simulation and modelling was performed to test the statistical performance of the SLIPT methodology and further applied to datasets with simulated correlation structures, including those derived from known graph stuctures.
</p>
<!--l. 12-->
<p class="noindent">
<strong><span class="cmbx-12">Findings</span></strong>
</p>
<!--l. 15-->
<p class="noindent">
A vast number of genes having expression consistent with being synthetic lethal partners of <span class="cmti-12">CDH1 </span>were detected in both TCGA Breast and Stomach cancer genes. For breast cancers, these genes clustered into several distinct groups, with distinct enriched biological functions and elevated expression in different clinical subclasses such as normal-like, basal, or estrogen receptor negative samples. While the number of genes detected by both computational and experimental approaches were not significant, there was significant pathway composition in the overlapping genes. In particular <span class="cmmi-12">G</span><sub><span class="cmmi-8">αi</span> </sub> signalling, cytoplasmic microfibres, and extracelluar fibrin clotting were supported by both approaches even after permutation testing. These findings are consistent with the known roles of E-cadherin in cytoskeletal or cell signalling roles and the proposed downstream targets of GCPR singalling of Telford <span class="cmti-12">et al</span>. (2015). Many of these and related pathways were replicated in the separate stomach cancer dataset. Furthermore other candidate pathways uniquely supported by the computational predictions included regulation of immune signaling and translational elongation, both unlikely to have been detected with high dose siRNA in an isogenic cell line and these are still candidates for further testing in mouse xenograft models.
</p>
<!--l. 18-->
<p class="noindent">
<p>A number of approaches were adapted or developed to test whether there was a connection between synthetic lethal candidates in the graph</p>
structures of the pathways most supported by prior analyses. Network centrality measures were used to compare the importance or connectivity of genes in the pathway subnetworks but no significant difference was found between synthetic candidates and other genes within the same pathway. Another hypothesis was that computational synthetic lethal candidates would be downstream of experimental candidates within a pathway but no evidence of directionality between the candidates was detected.
</p>
<!--l. 22-->
<p class="noindent">
A model of synthetic lethality was developed and was sucessfully implemented to simulate gene expression datasets with known underlying synthetic lethal partners of a query gene. For small numbers of known synthetic lethal partners, the SLIPT methdology performed well respect to reciever operator characteristic curves. As the number of true partners to detect increases, the power to detect them diminishes. Increasing sample sizes, however, was able to mitigate this effect somewhat as expected. This finding was replicated in simulations up to a feasible number of human genes (20,000) with more true negatives and correlations structures. The SLIPT methdology performs similarly across these conditions and performs better than Pearson’s correlation (for co-expression) of the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>-test without a directional criterion. However, correlation structure of the dataset does impact on synthetic lethal predictions, genes correlated with (or in a pathway structure near to) true synthetic lethal partners having elevated test statistic values over other true negatives. A quadratic (second order polynomial) least squares linear regression methodology has been developed as a comparable alternative with the added benefit of conditioning against known partners (or strongest candidates prior analyses).
</p>
<!--l. 26-->
<p class="noindent">
<p>Thus my thesis has developed, evaluated and refined a bioinformatics approach to discovery of synthetic lethal genes solely from gene expression data.</p>
</p>
<div class="center">
<!--l. 30-->
<p class="noindent">
</p>
<!--l. 31-->
<p class="noindent">
<strong><span class="cmbx-12">Research Contributions During Candidature</span></strong>
</p>
</div>
<!--l. 34-->
<p class="noindent">
<strong><span class="cmbx-12">Publications Under Peer-Review</span></strong>
</p>
<!--l. 38-->
<p class="noindent">
Kelly, S. T. and Spencer, H. G. (2017) Population-Genetics Models of Sex-Limited Genomic Imprinting. <span class="cmti-12">Theoretical Population Biology </span><strong><span class="cmbx-12">115</span></strong>:35-44 doi:10.1016/j.tpb.2017.03.004
</p>
<!--l. 40-->
<p class="noindent">
Kelly, S. T., Single, A. B., Telford, B. J., Beetham, H. G, Godwin, T. D., Chen, A., Black, M., A., and Guilford, P. J. (2017) Towards HDGC chemoprevention: vulnerabilities in E-cadherin-negative cells identified by genomic interrogation of isogenic cell lines and whole tumors. Submitted to <span class="cmti-12">Cancer Prevention</span> <span class="cmti-12">Research</span>.
</p>
<!--l. 42-->
<p class="noindent">
Kelly, S. T., Chen, A., Guilford, P. J., and Black, M. A. (2017) Synthetic lethal interaction prediction of target pathways in E-cadherin deficient breast cancers. Submitted to <span class="cmti-12">BMC Genomics</span>.
</p>
<!--l. 44-->
<p class="noindent">
<strong><span class="cmbx-12">Community Blog Posts</span></strong>
</p>
<!--l. 46-->
<p class="noindent">
Black, M. A., Kelly, S. T., and Cadzow, M. Posted on the <span class="cmti-12">Software Carpentry </span>website 2016 July 4<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup>: “Software Carpentry workshop at the University of Otago, New Zealand” <a 
href="https://software-carpentry.org/blog/2016/07/otago-workshop.html" class="url" ><span class="cmtt-12">https://software-_carpentry.org/blog/2016/07/otago-_workshop.html</span></a>
</p>
<!--l. 51-->
<p class="noindent">
Kelly, S. T., Black, M., A., Bae, S., Hayek, W., and Pawlik, A. Posted on the <span class="cmti-12">Software Carpentry </span>website 2016 September 28<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup>: “Software Carpentry Workshop Attendance: a New Zealand Perspective“ <a 
href="https://software-carpentry.org/blog/2016/09/attendance-nz.html" class="url" ><span class="cmtt-12">https://software-_carpentry.org/blog/2016/09/attendance-_nz.html</span></a>
</p>
<!--l. 55-->
<p class="noindent">
<strong><span class="cmbx-12">Software Packages</span></strong>
</p>
<!--l. 57-->
<p class="noindent">
<p>Several software packages in the R language have been released on GitHub while preparing this thesis. Please see the appropriate GitHub repository for more information on installing and running these packages, on the following account: <a 
href="https://github.com/TomKellyGenetics" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics</span></a></p>
</p>
<!--l. 59-->
<p class="noindent">
<a 
href="https://github.com/TomKellyGenetics/slipt" ><span class="cmtt-12">slipt</span></a> is the Synthetic Lethal interaction Prediction Tool, released to accompany the synthetic lethal publication above. <a 
href="https://github.com/TomKellyGenetics/slipt-app" ><span class="cmtt-12">slipt-app</span></a> contains an application developed in the R <span class="cmtt-12">shiny </span>environment as part of a related project.
</p>
<!--l. 61-->
<p class="noindent">
Several plotting functions were customised for the Figures in this thesis (and the above publications), notably <a 
href="https://github.com/TomKellyGenetics/heatmap.2x" ><span class="cmtt-12">heatmap.2x</span></a> and <a 
href="https://github.com/TomKellyGenetics/vioplotx" ><span class="cmtt-12">vioplotx</span></a> have been prepared largely for use during this project but are also documented and available to other R users. These are enhancements to the CRAN <span class="cmtt-12">gplots </span>and <span class="cmtt-12">vioplot </span>packages respectively and are intended be user-friendly for those familiar with <span class="cmtt-12">heatmap.2</span> or <span class="cmtt-12">vioplot </span>and <span class="cmtt-12">boxplot </span>(base R) functions. These are backwards compatible with these functions, taking similar imputs as demonstrated in the appropriate vignettes.
</p>
<!--l. 63-->
<p class="noindent">
The use of iGraph (the R <span class="cmtt-12">igraph </span>package) operations of graph-network structure in the analysis and simulations of pathways involved several original or customised functions to manipuate or plot <span class="cmtt-12">igraph </span>objects and adjacency matrices. These can be install separately from their respective repositories of with the metapackage: <a 
href="https://github.com/TomKellyGenetics/igraph.extensions" ><span class="cmtt-12">igraph.extensions</span></a>. <a 
href="https://github.com/TomKellyGenetics/plot.igraph" ><span class="cmtt-12">plot.igraph</span></a> enables plotting graph networks with customised inhibitor arrow and node colours.<a 
href="https://github.com/TomKellyGenetics/info.centrality" ><span class="cmtt-12">info.centrality</span></a> enables the calculation of additional node and network centrality metrics not available in the <span class="cmtt-12">igraph </span>package. <a 
href="https://github.com/TomKellyGenetics/pathway.structure.permutation" ><span class="cmtt-12">pathway.structure.permutation</span></a> enables testing of related states or node groups in a network by directionality of shortest paths. The <a 
href="https://github.com/TomKellyGenetics/graphsim" ><span class="cmtt-12">graphsim</span></a> package has been set up to simulate a multi-variate normal gene expression dataset with <span class="cmtt-12">mvtnorm </span>while deriving the correlation structure, Σ, from a graph structure. Note that these require various packages for graph theory, statistics and matrix operations and these will be installed as dependencies.
</p>
<!--l. 65-->
<p class="noindent">
<strong><span class="cmbx-12">Conference Participation</span></strong>
</p>
<!--l. 67-->
<p class="noindent">
eResearch 2017 (Queenstown) Speaker February 20<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup>-22<sup class="textsuperscript"><span class="cmr-10x-x-109">nd</span></sup> “Detecting Synthetic Lethality from Cancer Gene Expression: A PhD project on genetic interactions with CDH1 inactivation in TCGA data”
</p>
<!--l. 70-->
<p class="noindent">
<p>RIKEN Division of Genomic Technologies (Yokohama, Japan) Seminar 2016 October 20<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup>; National Institute of Genetics (Mishima, Japan) Seminar 2016 October 21<sup class="textsuperscript"><span class="cmr-10x-x-109">st</span></sup>; Tokyo University Institute of Medical Science (Shirokanedai</p>
Campus, Japan) Seminar 2016 October 24<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup>; Sokendai Graduate University (Hayama, Japan) Seminar 2016 October 25<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup> “Analysis of Synthetic Lethal Pathways in Breast Cancer: A PhD project on genetic interactions with CDH1 inactivation in TCGA data”
</p>
<!--l. 76-->
<p class="noindent">
Next Generation Sequencing Asia 2016 (Singapore) Poster October 11<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup>-12<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup> “Bioinformatic Investigations of Synthetic Lethal Interactions with E-cadherin in Breast Cancer” (Supported by the University of Otago Division of Health Sciences; Maurice and Phyllis Paykel Trust)
</p>
<!--l. 80-->
<p class="noindent">
eResearch 2016 (Queenstown, New Zealand) Speaker February 9<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup>-11<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup> “Sifting the Needles in the Haystack: Permutation Resampling Biological Pathways in Cancer Genomic Interaction Data” (Supported by REANNZ)
</p>
<!--l. 83-->
<p class="noindent">
Genetics Otago Symposium 2016 (Dunedin, New Zealand) Student Speaker March 7<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup>-8<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup> “A Bioinformatics approach to Genetic Interactions: Synthetic Lethal Pathways with E-cadherin in Breast Cancer Genomics Data”
</p>
<!--l. 86-->
<p class="noindent">
Research Bazaar 2015 (Melbourne, Australia) February 16<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup>-18<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup> “My digital research toolkit” (Supported by the New Zealand eScience Infrastructure)
</p>
<!--l. 90-->
<p class="noindent">
QMB Cancer Drugs Satellite 2014 (Queenstown, New Zealand) Poster August 24<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span> </sup>-25<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup>; Otago School of Medical Sciences 2015 (Dunedin, New Zealand) Poster Postgraduate Symposium April 28<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup>-29<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup> Bioinformatics Prioritisation of Synthetic Lethal Targets for Drug Activity Against E-Cadherin Deficient Cancers
</p>
<!--l. 94-->
<p class="noindent">
DunDead: Zombie Science and Popular Culture Festival 2014 (Dunedin, New Zealand) Ignite Speaker August 16<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup>-17<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup> “Hidden in Plain Sight - The Genetics of Zombies”
</p>
<!--l. 98-->
<p class="noindent">
eResearch 2014 (Waikato University, Hamilton, New Zealand) Ignite Speaker June 30<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup>-July 2<sup class="textsuperscript"><span class="cmr-10x-x-109">nd</span></sup> “Bioinformatic analysis of synthetic lethal genetic interactions in breast cancer” (Supported by Google)
</p>
</div>
</div>
<div class="center">
<!--l. 146-->
<p class="noindent">
</p>
<!--l. 146-->
<p class="noindent">
<span class="cmbx-12">Acknowledgements</span>
</p>
</div>
<div class="quote">
<!--l. 1-->
<p class="noindent">
I thank my supervisors A/Prof. Mik Black and Prof. Parry Guilford for their support and guidance throughout this my postgraduate studies. It has been a great experience, I look forward to seeing what your research groups produce in the future, may this not be the end.
</p>
<!--l. 3-->
<p class="noindent">
I am also thankful for the guidance and mentorship of Prof. Hamish Spencer for career advice throughout my studies and time in his research group.
</p>
<!--l. 5-->
<p class="noindent">
I am also grateful to the past and current members of these research groups, and my peers at the laboratory benches and computers across campus. The peer support, comraderie, and guidance to newer students has been an incredible part of my time at Otago and has made my thesis studies not just easier but possible at all. The postgraduate community is very special here and have truly made some lifelong friends from all over the world, you are talented researchers and amazing people. May we meet again some day. Whereever you may end up, there’s always time to catch up and I’d be delighted to host some visits while working abroad.
</p>
<!--l. 7-->
<p class="noindent">
<p>I cannot thank my friends, flatmates and family enough for their patience and support during such as massive, challenging, and (I’m sure you’ve heard too many times) stressful undertaking during both my PhD and the study leading up to it. There are too many of you to name everyone here without leaving someone out, so thank you all for everything you’ve done, both the good times and the tough. Thank you for pretending to understand when I</p>
try to discuss complex math at the wrong moment. Thank you for checking my writing or slides, even if I should have given you more time. Thank for your time when all I really needed was a chat over a walk or a pint and a moment to think clearly.
</p>
<!--l. 9-->
<p class="noindent">
I must also thank various organisations supported this research project:
</p>
<ul class="itemize1">
<li class="itemize">
This thesis was supported by the Postgraduate Tassell Scholarship in Cancer Research, a University of Otago Doctoral Scholarship.
</li>
<li class="itemize">
The New Zealand eScience Infrastucture (NeSI) provided access to the Intel Pan high-performance computing cluster, support, and training to use it effectively. Various aspects of this thesis would not have been possible without access to such a resource.
</li>
<li class="itemize">
The Health Research Council (HRC) of New Zealand provided funding for experimental research in the Cancer Genetics Laboratory. Again some aspects of this project would not have been possible without access to the data and findings funded by this grant.
</li>
<li class="itemize">
The Allan Wilson Centre and Otago School of Biomedical Sciences provided funding for summer research placements which was a valuable opportunity to gain experience and training used in this thesis project.
</li>
</ul>
<!--l. 26-->
<p class="noindent">
I thank the following organisations for support towards presenting findings in this thesis at conference and seminars:
</p>
<ul class="itemize1">
<li class="itemize">
Google (towards eResearch 2014 conference, Hamilton)
</li>
<li class="itemize">
<p>NeSI (towards Software Carpentry training and Research Bazaar 2015, Melbourne)</p>
</li>
<li class="itemize">
REANNZ, NZGL, and NeSI (towards eResearch 2016 conference, Queenstown)
</li>
<li class="itemize">
Otago Division of Health Sciences, Department of Biochemistry, Oxford Global, and Maurice and Phyllis Paykel Trust (towards NGS Asia 2016, Singapore)
</li>
<li class="itemize">
RIKEN Division of Genomics Technologies and the Okinawa Institue of Science and Technology (for hosting seminars in Japan)
</li>
</ul>
<!--l. 47-->
<p class="noindent">
Thanks most of all to my fianceé, Dr Yui Kawagishi, you’ve been an inspiration. Thank you for your support, help, and encouragement, even from afar times, it has always made a difference. It’s been incredible to see you flourish in your career and I look forward to joining you again soon. May the next Chapter of our adventures involve a bit less Skype across timezones.
</p>
</div>
<h2 class="likechapterHead">
<a 
 id="x1-1000"></a>Contents
</h2>
<div class="tableofcontents">
<p><span class="chapterToc"><a 
href="#Q1-1-4">Glossary</a></span> <br /><span class="chapterToc"><a 
href="#Q1-1-6">Acronyms</a></span> <br /><span class="chapterToc">1 <a 
href="#x1-60001" id="QQ2-1-8">Introduction</a></span> <br /> <span class="sectionToc">1.1 <a 
href="#x1-70001" id="QQ2-1-9">Cancer Research in the Post-Genomic Era</a></span> <br />  <span class="subsectionToc">1.1.1 <a 
href="#x1-80001" id="QQ2-1-10">Cancer as a Global Health Concern</a></span> <br />   <span class="subsubsectionToc">1.1.1.1 <a 
href="#x1-90001" id="QQ2-1-11">The Genetics and Molecular Biology of Cancers</a></span> <br />  <span class="subsectionToc">1.1.2 <a 
href="#x1-100002" id="QQ2-1-12">The Human Genome Revolution</a></span> <br />   <span class="subsubsectionToc">1.1.2.1 <a 
href="#x1-110001" id="QQ2-1-13">The First Human Genome Sequence</a></span> <br />   <span class="subsubsectionToc">1.1.2.2 <a 
href="#x1-120002" id="QQ2-1-14">Impact of Genomics</a></span> <br />  <span class="subsectionToc">1.1.3 <a 
href="#x1-130003" id="QQ2-1-15">Technologies to Enable Genetics Research</a></span> <br />   <span class="subsubsectionToc">1.1.3.1 <a 
href="#x1-140001" id="QQ2-1-16">DNA Sequencing and Genotyping Technologies</a></span> <br />   <span class="subsubsectionToc">1.1.3.2 <a 
href="#x1-150002" id="QQ2-1-17">Microarrays and Quantitative Technologies</a></span> <br />   <span class="subsubsectionToc">1.1.3.3 <a 
href="#x1-160003" id="QQ2-1-18">Massively Parallel “Next Generation” Sequencing</a></span> <br />   <span class="subsubsectionToc">1.1.3.4 <a 
href="#x1-190004" id="QQ2-1-21">Bioinformatics as Interdisciplinary Genomic Analysis</a></span> <br />  <span class="subsectionToc">1.1.4 <a 
href="#x1-200004" id="QQ2-1-22">Follow-up Large-Scale Genomics Projects</a></span> <br />  <span class="subsectionToc">1.1.5 <a 
href="#x1-210005" id="QQ2-1-23">Cancer Genomes</a></span> <br />   <span class="subsubsectionToc">1.1.5.1 <a 
href="#x1-220001" id="QQ2-1-24">The Cancer Genome Atlas Project</a></span> <br />  <span class="subsectionToc">1.1.6 <a 
href="#x1-260006" id="QQ2-1-28">Genomic Cancer Medicine</a></span> <br />   <span class="subsubsectionToc">1.1.6.1 <a 
href="#x1-270001" id="QQ2-1-29">Cancer Genes and Driver Mutations</a></span> <br />   <span class="subsubsectionToc">1.1.6.2 <a 
href="#x1-280002" id="QQ2-1-30">Personalised or Precision Cancer Medicine</a></span> <br />   <span class="subsubsectionToc">1.1.6.3 <a 
href="#x1-300003" id="QQ2-1-32">Targeted Therapeutics and Pharmacogenomics</a></span> <br />   <span class="subsubsectionToc">1.1.6.4 <a 
href="#x1-320004" id="QQ2-1-34">Systems and Network Biology</a></span> <br /> <span class="sectionToc">1.2 <a 
href="#x1-340002" id="QQ2-1-36">A Synthetic Lethal Approach to Cancer Medicine</a></span> <br />  <span class="subsectionToc">1.2.1 <a 
href="#x1-350001" id="QQ2-1-37">Synthetic Lethal Genetic Interactions</a></span> <br />  <span class="subsectionToc">1.2.2 <a 
href="#x1-360002" id="QQ2-1-39">Synthetic Lethal Concepts in Genetics</a></span> <br />  <span class="subsectionToc">1.2.3 <a 
href="#x1-370003" id="QQ2-1-40">Studies of Synthetic Lethality</a></span> <br />   <span class="subsubsectionToc">1.2.3.1 <a 
href="#x1-380001" id="QQ2-1-41">Synthetic Lethal Pathways and Networks</a></span> <br />  <span class="subsectionToc">1.2.4 <a 
href="#x1-400004" id="QQ2-1-43">Synthetic Lethal Concepts in Cancer</a></span> <br />  <span class="subsectionToc">1.2.5 <a 
href="#x1-410005" id="QQ2-1-45">Clinical Impact of Synthetic Lethality in Cancer</a></span> <br />  <span class="subsectionToc">1.2.6 <a 
href="#x1-420006" id="QQ2-1-46">High-throughput Screening for Synthetic Lethality</a></span> <br />   <span class="subsubsectionToc">1.2.6.1 <a 
href="#x1-430001" id="QQ2-1-47">Synthetic Lethal Screens</a></span> <br />  <span class="subsectionToc">1.2.7 <a 
href="#x1-440007" id="QQ2-1-48">Computational Prediction of Synthetic Lethality</a></span> <br />   <span class="subsubsectionToc">1.2.7.1 <a 
href="#x1-450001" id="QQ2-1-49">Bioinformatics Approaches to Genetic Interactions</a></span> <br />   <span class="subsubsectionToc">1.2.7.2 <a 
href="#x1-460002" id="QQ2-1-52">Comparative Genomics</a></span> <br />   <span class="subsubsectionToc">1.2.7.3 <a 
href="#x1-470003" id="QQ2-1-54">Analysis and Modelling of Protein Data</a></span> <br />   <span class="subsubsectionToc">1.2.7.4 <a 
href="#x1-480004" id="QQ2-1-55">Differential Gene Expression</a></span> <br />   <span class="subsubsectionToc">1.2.7.5 <a 
href="#x1-490005" id="QQ2-1-56">Data Mining and Machine Learning</a></span></p>
<p><br />   <span class="subsubsectionToc">1.2.7.6 <a 
href="#x1-500006" id="QQ2-1-57">Bimodality</a></span> <br />   <span class="subsubsectionToc">1.2.7.7 <a 
href="#x1-510007" id="QQ2-1-58">Rationale for Further Development</a></span> <br /> <span class="sectionToc">1.3 <a 
href="#x1-520003" id="QQ2-1-59">E-cadherin as a Synthetic Lethal Target</a></span> <br />  <span class="subsectionToc">1.3.1 <a 
href="#x1-530001" id="QQ2-1-60">The <span class="cmti-12">CDH1 </span>gene and it’s Biological Functions</a></span> <br />   <span class="subsubsectionToc">1.3.1.1 <a 
href="#x1-540001" id="QQ2-1-61">Cytoskeleton</a></span> <br />   <span class="subsubsectionToc">1.3.1.2 <a 
href="#x1-550002" id="QQ2-1-62">Extracellular and Tumour Micro-Environment</a></span> <br />   <span class="subsubsectionToc">1.3.1.3 <a 
href="#x1-560003" id="QQ2-1-63">Cell-Cell Adhesion and Signalling</a></span> <br />  <span class="subsectionToc">1.3.2 <a 
href="#x1-570002" id="QQ2-1-64"><span class="cmti-12">CDH1 </span>as a Tumour (and Invasion) Suppressor</a></span> <br />   <span class="subsubsectionToc">1.3.2.1 <a 
href="#x1-580001" id="QQ2-1-65">Breast Cancers and Invasion</a></span> <br />  <span class="subsectionToc">1.3.3 <a 
href="#x1-590003" id="QQ2-1-66">Hereditary Diffuse Gastric Cancer and Lobular Breast Cancer</a></span> <br />  <span class="subsectionToc">1.3.4 <a 
href="#x1-600004" id="QQ2-1-67">Somatic Mutations</a></span> <br />   <span class="subsubsectionToc">1.3.4.1 <a 
href="#x1-610001" id="QQ2-1-68">Mutation Rate</a></span> <br />   <span class="subsubsectionToc">1.3.4.2 <a 
href="#x1-620002" id="QQ2-1-69">Co-occurring Mutations</a></span> <br />  <span class="subsectionToc">1.3.5 <a 
href="#x1-630005" id="QQ2-1-70">Models of <span class="cmti-12">CDH1 </span>loss in cell lines</a></span> <br /> <span class="sectionToc">1.4 <a 
href="#x1-640004" id="QQ2-1-71">Summary and Research Direction of Thesis</a></span> <br /><span class="chapterToc">2 <a 
href="#x1-650002" id="QQ2-1-72">Methods and Resources</a></span> <br /> <span class="sectionToc">2.1 <a 
href="#x1-660001" id="QQ2-1-73">Bioinformatics Resources for Genomics Research</a></span> <br />  <span class="subsectionToc">2.1.1 <a 
href="#x1-670001" id="QQ2-1-74">Public Data and Software Packages</a></span> <br />   <span class="subsubsectionToc">2.1.1.1 <a 
href="#x1-680001" id="QQ2-1-75">Cancer Genome Atlas Data</a></span> <br />   <span class="subsubsectionToc">2.1.1.2 <a 
href="#x1-690002" id="QQ2-1-76">Reactome and Annotation Data</a></span> <br /> <span class="sectionToc">2.2 <a 
href="#x1-700002" id="QQ2-1-77">Data Handling</a></span> <br />  <span class="subsectionToc">2.2.1 <a 
href="#x1-710001" id="QQ2-1-78">Normalisation</a></span> <br />  <span class="subsectionToc">2.2.2 <a 
href="#x1-720002" id="QQ2-1-79">Sample Triage</a></span> <br />  <span class="subsectionToc">2.2.3 <a 
href="#x1-730003" id="QQ2-1-83">Metagenes and the Singular Value Decomposition</a></span> <br />   <span class="subsubsectionToc">2.2.3.1 <a 
href="#x1-740001" id="QQ2-1-84">Candidate Triage and Integration with Screen Data</a></span> <br /> <span class="sectionToc">2.3 <a 
href="#x1-750003" id="QQ2-1-85">Techniques</a></span> <br />  <span class="subsectionToc">2.3.1 <a 
href="#x1-760001" id="QQ2-1-86">Statistical Procedures and Tests</a></span> <br />  <span class="subsectionToc">2.3.2 <a 
href="#x1-770002" id="QQ2-1-87">Gene Set Over-representation Analysis</a></span> <br />  <span class="subsectionToc">2.3.3 <a 
href="#x1-780003" id="QQ2-1-88">Clustering</a></span> <br />  <span class="subsectionToc">2.3.4 <a 
href="#x1-790004" id="QQ2-1-89">Heatmap</a></span> <br />  <span class="subsectionToc">2.3.5 <a 
href="#x1-800005" id="QQ2-1-90">Modeling and Simulations</a></span> <br />   <span class="subsubsectionToc">2.3.5.1 <a 
href="#x1-810001" id="QQ2-1-91">Receiver Operating Characteristic (Performance)</a></span> <br />  <span class="subsectionToc">2.3.6 <a 
href="#x1-820006" id="QQ2-1-92">Resampling Analysis</a></span> <br /> <span class="sectionToc">2.4 <a 
href="#x1-830004" id="QQ2-1-93">Pathway Structure Methods</a></span> <br />  <span class="subsectionToc">2.4.1 <a 
href="#x1-840001" id="QQ2-1-94">Network and Graph Analysis</a></span> <br />  <span class="subsectionToc">2.4.2 <a 
href="#x1-850002" id="QQ2-1-95">Sourcing Graph Structure Data</a></span> <br />  <span class="subsectionToc">2.4.3 <a 
href="#x1-860003" id="QQ2-1-96">Constructing Pathway Subgraphs</a></span> <br />  <span class="subsectionToc">2.4.4 <a 
href="#x1-870004" id="QQ2-1-97">Network Analysis Metrics</a></span> <br /> <span class="sectionToc">2.5 <a 
href="#x1-880005" id="QQ2-1-98">Implementation</a></span> <br />  <span class="subsectionToc">2.5.1 <a 
href="#x1-890001" id="QQ2-1-99">Computational Resources and Linux Utilities</a></span> <br />  <span class="subsectionToc">2.5.2 <a 
href="#x1-900002" id="QQ2-1-102">R Language and Packages</a></span> <br />  <span class="subsectionToc">2.5.3 <a 
href="#x1-910003" id="QQ2-1-106">High Performance and Parallel Computing</a></span> <br /><span class="chapterToc">3 <a 
href="#x1-920003" id="QQ2-1-107">Methods Developed During Thesis</a></span> <br /> <span class="sectionToc">3.1 <a 
href="#x1-930001" id="QQ2-1-108">A Synthetic Lethal Detection Methodology</a></span> <br /> <span class="sectionToc">3.2 <a 
href="#x1-940002" id="QQ2-1-111">Synthetic Lethal Simulation and Modelling</a></span> <br />  <span class="subsectionToc">3.2.1 <a 
href="#x1-950001" id="QQ2-1-112">A Model of Synthetic Lethality in Expression Data</a></span></p>
<p><br />  <span class="subsectionToc">3.2.2 <a 
href="#x1-960002" id="QQ2-1-116">Simulation Procedure</a></span> <br /> <span class="sectionToc">3.3 <a 
href="#x1-970003" id="QQ2-1-120">Detecting Simulated Synthetic Lethal Partners</a></span> <br />  <span class="subsectionToc">3.3.1 <a 
href="#x1-980001" id="QQ2-1-121">Binomial Simulation of Synthetic lethality</a></span> <br />  <span class="subsectionToc">3.3.2 <a 
href="#x1-990002" id="QQ2-1-124">Multivariate Normal Simulation of Synthetic lethality</a></span> <br />   <span class="subsubsectionToc">3.3.2.1 <a 
href="#x1-1000001" id="QQ2-1-126">Multivariate Normal Simulation with Correlated Genes</a></span> <br />   <span class="subsubsectionToc">3.3.2.2 <a 
href="#x1-1010002" id="QQ2-1-133">Specificity with Query-Correlated Pathways</a></span> <br /> <span class="sectionToc">3.4 <a 
href="#x1-1030004" id="QQ2-1-137">Graph Structure Methods</a></span> <br />  <span class="subsectionToc">3.4.1 <a 
href="#x1-1040001" id="QQ2-1-138">Upstream and Downstream Gene Detection</a></span> <br />   <span class="subsubsectionToc">3.4.1.1 <a 
href="#x1-1050001" id="QQ2-1-139">Permutation Analysis for Statistical Significance</a></span> <br />   <span class="subsubsectionToc">3.4.1.2 <a 
href="#x1-1060002" id="QQ2-1-140">Hierarchy Based on Biological Context</a></span> <br />  <span class="subsectionToc">3.4.2 <a 
href="#x1-1070002" id="QQ2-1-141">Simulating Gene Expression from Graph Structures</a></span> <br /> <span class="sectionToc">3.5 <a 
href="#x1-1080005" id="QQ2-1-145">Customised Functions and Packages Developed</a></span> <br />  <span class="subsectionToc">3.5.1 <a 
href="#x1-1090001" id="QQ2-1-146">Synthetic Lethal Interaction Prediction Tool</a></span> <br />  <span class="subsectionToc">3.5.2 <a 
href="#x1-1100002" id="QQ2-1-147">Data Visualisation</a></span> <br />  <span class="subsectionToc">3.5.3 <a 
href="#x1-1110003" id="QQ2-1-150">Extensions to the iGraph Package</a></span> <br />   <span class="subsubsectionToc">3.5.3.1 <a 
href="#x1-1120001" id="QQ2-1-151">Sampling Simulated Data from Graph Structures</a></span> <br />   <span class="subsubsectionToc">3.5.3.2 <a 
href="#x1-1130002" id="QQ2-1-152">Plotting Directed Graph Structures</a></span> <br />   <span class="subsubsectionToc">3.5.3.3 <a 
href="#x1-1140003" id="QQ2-1-154">Computing Information Centrality</a></span> <br />   <span class="subsubsectionToc">3.5.3.4 <a 
href="#x1-1150004" id="QQ2-1-155">Testing Pathway Structure with Permutation Testing</a></span> <br />   <span class="subsubsectionToc">3.5.3.5 <a 
href="#x1-1160005" id="QQ2-1-156">Metapackage to Install iGraph Functions</a></span> <br /><span class="chapterToc">4 <a 
href="#x1-1170004" id="QQ2-1-157">Synthetic Lethal Analysis of Gene Expression Data</a></span> <br /> <span class="sectionToc">4.1 <a 
href="#x1-1180001" id="QQ2-1-158">Synthetic lethal genes in breast cancer</a></span> <br />  <span class="subsectionToc">4.1.1 <a 
href="#x1-1190001" id="QQ2-1-160">Synthetic lethal pathways in breast cancer</a></span> <br />  <span class="subsectionToc">4.1.2 <a 
href="#x1-1200002" id="QQ2-1-162">Expression profiles of synthetic lethal partners</a></span> <br />   <span class="subsubsectionToc">4.1.2.1 <a 
href="#x1-1210001" id="QQ2-1-164">Subgroup pathway analysis</a></span> <br /> <span class="sectionToc">4.2 <a 
href="#x1-1220002" id="QQ2-1-166">Comparison of synthetic lethal gene candidates</a></span> <br />  <span class="subsectionToc">4.2.1 <a 
href="#x1-1230001" id="QQ2-1-167">Comparison with siRNA screen candidates</a></span> <br />   <span class="subsubsectionToc">4.2.1.1 <a 
href="#x1-1240001" id="QQ2-1-169">Comparison with correlation</a></span> <br />   <span class="subsubsectionToc">4.2.1.2 <a 
href="#x1-1250002" id="QQ2-1-172">Comparison with viability</a></span> <br />   <span class="subsubsectionToc">4.2.1.3 <a 
href="#x1-1260003" id="QQ2-1-176">Comparison with secondary siRNA screen candidates</a></span> <br />   <span class="subsubsectionToc">4.2.1.4 <a 
href="#x1-1270004" id="QQ2-1-177">Comparison of screen at pathway level</a></span> <br /> <span class="sectionToc">4.3 <a 
href="#x1-1290003" id="QQ2-1-183">Metagene Analysis</a></span> <br />  <span class="subsectionToc">4.3.1 <a 
href="#x1-1300001" id="QQ2-1-184">Pathway expression</a></span> <br />  <span class="subsectionToc">4.3.2 <a 
href="#x1-1310002" id="QQ2-1-186">Somatic mutation</a></span> <br />  <span class="subsectionToc">4.3.3 <a 
href="#x1-1320003" id="QQ2-1-188">Mutation locus</a></span> <br />  <span class="subsectionToc">4.3.4 <a 
href="#x1-1330004" id="QQ2-1-190">Synthetic lethal metagenes</a></span> <br /> <span class="sectionToc">4.4 <a 
href="#x1-1340004" id="QQ2-1-192">Replication in stomach cancer</a></span> <br />  <span class="subsectionToc">4.4.1 <a 
href="#x1-1350001" id="QQ2-1-193">Synthetic Lethal Genes and Pathways</a></span> <br />  <span class="subsectionToc">4.4.2 <a 
href="#x1-1360002" id="QQ2-1-195">Synthetic Lethal Expression Profiles</a></span> <br />  <span class="subsectionToc">4.4.3 <a 
href="#x1-1370003" id="QQ2-1-197">Comparison to Primary Screen</a></span> <br />   <span class="subsubsectionToc">4.4.3.1 <a 
href="#x1-1380001" id="QQ2-1-198">Resampling Analysis</a></span> <br />  <span class="subsectionToc">4.4.4 <a 
href="#x1-1390004" id="QQ2-1-199">Metagene Analysis</a></span> <br /> <span class="sectionToc">4.5 <a 
href="#x1-1400005" id="QQ2-1-200">Global Synthetic Lethality</a></span> <br />  <span class="subsectionToc">4.5.1 <a 
href="#x1-1410001" id="QQ2-1-202">Hub Genes</a></span> <br />  <span class="subsectionToc">4.5.2 <a 
href="#x1-1420002" id="QQ2-1-204">Hub Pathways</a></span> <br /> <span class="sectionToc">4.6 <a 
href="#x1-1430006" id="QQ2-1-206">Replication in cell line encyclopaedia</a></span></p>
<p><br /> <span class="sectionToc">4.7 <a 
href="#x1-1440007" id="QQ2-1-209">Discussion</a></span> <br />  <span class="subsectionToc">4.7.1 <a 
href="#x1-1450001" id="QQ2-1-210">Strengths of the SLIPT Methodology</a></span> <br />  <span class="subsectionToc">4.7.2 <a 
href="#x1-1460002" id="QQ2-1-211">Syntheic Lethal Pathways for E-cadherin</a></span> <br />  <span class="subsectionToc">4.7.3 <a 
href="#x1-1470003" id="QQ2-1-212">Replication and Validation</a></span> <br />   <span class="subsubsectionToc">4.7.3.1 <a 
href="#x1-1480001" id="QQ2-1-213">Integration with siRNA Screening</a></span> <br />   <span class="subsubsectionToc">4.7.3.2 <a 
href="#x1-1490002" id="QQ2-1-214">Replication across Tissues and Cell lines</a></span> <br /> <span class="sectionToc">4.8 <a 
href="#x1-1500008" id="QQ2-1-215">Summary</a></span> <br />   <span class="paragraphToc"><a 
href="#x1-1510008" id="QQ2-1-216">Aims</a></span> <br />   <span class="paragraphToc"><a 
href="#x1-1520008" id="QQ2-1-217">Summary</a></span> <br /><span class="chapterToc">5 <a 
href="#x1-1530005" id="QQ2-1-218">Synthetic Lethal Pathway Structure</a></span> <br /> <span class="sectionToc">5.1 <a 
href="#x1-1540001" id="QQ2-1-219">Synthetic Lethal Genes in Reactome Pathways</a></span> <br />  <span class="subsectionToc">5.1.1 <a 
href="#x1-1550001" id="QQ2-1-220">The PI3K/AKT Pathway</a></span> <br />  <span class="subsectionToc">5.1.2 <a 
href="#x1-1560002" id="QQ2-1-222">The Extracellular Matrix</a></span> <br />  <span class="subsectionToc">5.1.3 <a 
href="#x1-1570003" id="QQ2-1-225">G Protein Coupled Receptors</a></span> <br />  <span class="subsectionToc">5.1.4 <a 
href="#x1-1580004" id="QQ2-1-226">Gene Regulation and Translation</a></span> <br /> <span class="sectionToc">5.2 <a 
href="#x1-1590002" id="QQ2-1-227">Network Analysis of Synthetic Lethal Genes</a></span> <br />  <span class="subsectionToc">5.2.1 <a 
href="#x1-1600001" id="QQ2-1-228">Gene Connectivity and Vertex Degree</a></span> <br />  <span class="subsectionToc">5.2.2 <a 
href="#x1-1610002" id="QQ2-1-231">Gene Importance and Centrality</a></span> <br />   <span class="subsubsectionToc">5.2.2.1 <a 
href="#x1-1620001" id="QQ2-1-232">Information Centrality</a></span> <br />   <span class="subsubsectionToc">5.2.2.2 <a 
href="#x1-1630002" id="QQ2-1-235">PageRank Centrality</a></span> <br /> <span class="sectionToc">5.3 <a 
href="#x1-1640003" id="QQ2-1-238">Testing Pathway Structure of Synthetic Lethal Genes</a></span> <br />  <span class="subsectionToc">5.3.1 <a 
href="#x1-1650001" id="QQ2-1-239">Hierarchical Pathway Structure</a></span> <br />   <span class="subsubsectionToc">5.3.1.1 <a 
href="#x1-1660001" id="QQ2-1-240">Contextual Hierarchy of PI3K</a></span> <br />   <span class="subsubsectionToc">5.3.1.2 <a 
href="#x1-1670002" id="QQ2-1-242">Testing Contextual Hierarchy of Synthetic Lethal Genes</a></span> <br />  <span class="subsectionToc">5.3.2 <a 
href="#x1-1680002" id="QQ2-1-247">Upstream or Downstream Synthetic Lethality</a></span> <br />   <span class="subsubsectionToc">5.3.2.1 <a 
href="#x1-1690001" id="QQ2-1-248">Measuring Structure of Candidates within PI3K</a></span> <br />   <span class="subsubsectionToc">5.3.2.2 <a 
href="#x1-1700002" id="QQ2-1-250">Resampling for Synthetic Lethal Pathway Structure</a></span> <br /> <span class="sectionToc">5.4 <a 
href="#x1-1710004" id="QQ2-1-252">Discussion</a></span> <br /> <span class="sectionToc">5.5 <a 
href="#x1-1720005" id="QQ2-1-253">Summary</a></span> <br />   <span class="paragraphToc"><a 
href="#x1-1730005" id="QQ2-1-254">Aims</a></span> <br />   <span class="paragraphToc"><a 
href="#x1-1740005" id="QQ2-1-255">Summary</a></span> <br /><span class="chapterToc">6 <a 
href="#x1-1750006" id="QQ2-1-256">Simulation and Modeling of Synthetic Lethal Pathways</a></span> <br /> <span class="sectionToc">6.1 <a 
href="#x1-1760001" id="QQ2-1-257">Comparing methods</a></span> <br />  <span class="subsectionToc">6.1.1 <a 
href="#x1-1770001" id="QQ2-1-258">Performance of SLIPT and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> across Quantiles</a></span> <br />   <span class="subsubsectionToc">6.1.1.1 <a 
href="#x1-1780001" id="QQ2-1-259">Correlated Query Genes affects Specificity</a></span> <br />  <span class="subsectionToc">6.1.2 <a 
href="#x1-1790002" id="QQ2-1-260">Correlation as a Synthetic Lethal Detection Strategy</a></span> <br />  <span class="subsectionToc">6.1.3 <a 
href="#x1-1800003" id="QQ2-1-261">Testing for Bimodality with BiSEp</a></span> <br /> <span class="sectionToc">6.2 <a 
href="#x1-1810002" id="QQ2-1-262">Simulations with Graph Structures</a></span> <br />  <span class="subsectionToc">6.2.1 <a 
href="#x1-1820001" id="QQ2-1-273">Performance over a Graph Structure</a></span> <br />  <span class="subsectionToc">6.2.2 <a 
href="#x1-1830002" id="QQ2-1-274">Synthetic Lethality across Graph Stuctures</a></span> <br />  <span class="subsectionToc">6.2.3 <a 
href="#x1-1840003" id="QQ2-1-275">Performance with inhibition links</a></span> <br />  <span class="subsectionToc">6.2.4 <a 
href="#x1-1850004" id="QQ2-1-276">Performance with 20,000 genes</a></span> <br /> <span class="sectionToc">6.3 <a 
href="#x1-1860003" id="QQ2-1-277">Simulations over pathway-based graphs</a></span> <br /> <span class="sectionToc">6.4 <a 
href="#x1-1870004" id="QQ2-1-278">Discussion</a></span> <br /> <span class="sectionToc">6.5 <a 
href="#x1-1880005" id="QQ2-1-279">Summary</a></span> <br />   <span class="paragraphToc"><a 
href="#x1-1890005" id="QQ2-1-280">Aims</a></span></p>
<p><br />   <span class="paragraphToc"><a 
href="#x1-1900005" id="QQ2-1-281">Summary</a></span> <br /><span class="chapterToc">7 <a 
href="#x1-1910007" id="QQ2-1-282">Discussion</a></span> <br />   <span class="paragraphToc"><a 
href="#x1-1920007" id="QQ2-1-283">Aims</a></span> <br />   <span class="paragraphToc"><a 
href="#x1-1930007" id="QQ2-1-284">Summary</a></span> <br /> <span class="sectionToc">7.1 <a 
href="#x1-1940001" id="QQ2-1-285">Significance</a></span> <br /> <span class="sectionToc">7.2 <a 
href="#x1-1950002" id="QQ2-1-286">Future Directions</a></span> <br /> <span class="sectionToc">7.3 <a 
href="#x1-1960003" id="QQ2-1-287">Conclusion</a></span> <br /><span class="chapterToc">8 <a 
href="#x1-1970008" id="QQ2-1-288">Conclusion</a></span> <br /><span class="chapterToc"><a 
href="#Q1-1-289"> References</a></span> <br /><span class="appendixToc">A <a 
href="#x1-199000A" id="QQ2-1-291">Sample Quality</a></span> <br /> <span class="sectionToc">A.1 <a 
href="#x1-2000001" id="QQ2-1-292">Sample Correlation</a></span> <br /> <span class="sectionToc">A.2 <a 
href="#x1-2010002" id="QQ2-1-295">Replicate Samples in TCGA Breast</a></span> <br /><span class="appendixToc">B <a 
href="#x1-202000B" id="QQ2-1-300">Software Used for Thesis</a></span> <br /><span class="appendixToc">C <a 
href="#x1-203000C" id="QQ2-1-302">Secondary Screen Data</a></span> <br /><span class="appendixToc">D <a 
href="#x1-204000D" id="QQ2-1-306">Mutation Analysis in Breast Cancer</a></span> <br /> <span class="sectionToc">D.1 <a 
href="#x1-2050001" id="QQ2-1-307">Synthetic Lethal Genes and Pathways</a></span> <br /> <span class="sectionToc">D.2 <a 
href="#x1-2060002" id="QQ2-1-310">Synthetic Lethal Expression Profiles</a></span> <br /> <span class="sectionToc">D.3 <a 
href="#x1-2070003" id="QQ2-1-313">Comparison to Primary Screen</a></span> <br />  <span class="subsectionToc">D.3.1 <a 
href="#x1-2080001" id="QQ2-1-316">Resampling Analysis</a></span> <br /> <span class="sectionToc">D.4 <a 
href="#x1-2090004" id="QQ2-1-319">Compare SLIPT genes</a></span> <br /> <span class="sectionToc">D.5 <a 
href="#x1-2100005" id="QQ2-1-323">Metagene Analysis</a></span> <br /> <span class="sectionToc">D.6 <a 
href="#x1-2110006" id="QQ2-1-325">Mutation Variation</a></span> <br />  <span class="subsectionToc">D.6.1 <a 
href="#x1-2120001" id="QQ2-1-326">Mutation Frequency</a></span> <br />  <span class="subsectionToc">D.6.2 <a 
href="#x1-2130002" id="QQ2-1-328">PI3K Mutation Expression</a></span> <br /><span class="appendixToc">E <a 
href="#x1-214000E" id="QQ2-1-332">Metagene Expression Profiles</a></span> <br /><span class="appendixToc">F <a 
href="#x1-215000F" id="QQ2-1-338">Stomach Expression Analysis</a></span> <br /> <span class="sectionToc">F.1 <a 
href="#x1-2160001" id="QQ2-1-339">Synthetic Lethal Genes and Pathways</a></span> <br /> <span class="sectionToc">F.2 <a 
href="#x1-2170002" id="QQ2-1-342">Comparison to Primary Screen</a></span> <br />  <span class="subsectionToc">F.2.1 <a 
href="#x1-2180001" id="QQ2-1-345">Resampling Analysis</a></span> <br /> <span class="sectionToc">F.3 <a 
href="#x1-2190003" id="QQ2-1-348">Metagene Analysis</a></span> <br /><span class="appendixToc">G <a 
href="#x1-220000G" id="QQ2-1-350">Stomach Mutation Analysis</a></span> <br /> <span class="sectionToc">G.1 <a 
href="#x1-2210001" id="QQ2-1-351">Synthetic Lethal Genes and Pathways</a></span> <br /> <span class="sectionToc">G.2 <a 
href="#x1-2220002" id="QQ2-1-354">Synthetic Lethal Expression Profiles</a></span> <br /> <span class="sectionToc">G.3 <a 
href="#x1-2230003" id="QQ2-1-357">Comparison to Primary Screen</a></span> <br />  <span class="subsectionToc">G.3.1 <a 
href="#x1-2240001" id="QQ2-1-360">Resampling Analysis</a></span> <br /> <span class="sectionToc">G.4 <a 
href="#x1-2250004" id="QQ2-1-363">Metagene Analysis</a></span> <br /><span class="appendixToc">H <a 
href="#x1-226000H" id="QQ2-1-365">Global Synthetic Lethality in Stomach Cancer</a></span> <br /> <span class="sectionToc">H.1 <a 
href="#x1-2270001" id="QQ2-1-367">Hub Genes</a></span> <br /> <span class="sectionToc">H.2 <a 
href="#x1-2280002" id="QQ2-1-369">Hub Pathways</a></span> <br /><span class="appendixToc">I <a 
href="#x1-229000I" id="QQ2-1-371">Replication in cell line encyclopaedia</a></span> <br /><span class="appendixToc">J <a 
href="#x1-230000J" id="QQ2-1-377">Synthetic Lethal Genes in Pathways</a></span> <br /><span class="appendixToc">K <a 
href="#x1-231000K" id="QQ2-1-386">Pathway Connectivity for Mutation SLIPT</a></span> <br /><span class="appendixToc">L <a 
href="#x1-232000L" id="QQ2-1-393">Information Centrality for Gene Essentiality</a></span> <br /><span class="appendixToc">M <a 
href="#x1-233000M" id="QQ2-1-396">Pathway Structure for Mutation SLIPT</a></span> <br /><span class="appendixToc">N <a 
href="#x1-234000N" id="QQ2-1-403">Simulation</a></span></p>
</div>
<h2 class="likechapterHead">
<a 
 id="x1-2000"></a>List of Tables
</h2>
<div class="tableofcontents">
<p><span class="lotToc">1.1 <a 
href="#x1-45001r1">Methods for Predicting Genetic Interactions</a></span><br /><span class="lotToc">1.2 <a 
href="#x1-45002r2">Methods for Predicting Synthetic Lethality in Cancer</a></span><br /><span class="lotToc">1.3 <a 
href="#x1-46001r3">Methods used by Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014)</a></span><br /><span class="lotToc">2.1 <a 
href="#x1-72007r1">Excluded Samples by Batch and Clinical Characteristics.</a></span><br /><span class="lotToc">2.2 <a 
href="#x1-89001r2">Computers used during Thesis</a></span><br /><span class="lotToc">2.3 <a 
href="#x1-89002r3">Linux Utilities and Applications used during Thesis</a></span><br /><span class="lotToc">2.4 <a 
href="#x1-90001r4">R Installations used during Thesis</a></span><br /><span class="lotToc"> <a 
href="#x1-90003">2.5 R Packages used during Thesis</a></span><br /><span class="lotToc">2.6 <a 
href="#x1-90004r6">R Packages Developed during Thesis</a></span><br /><span class="lotToc">4.1 <a 
href="#x1-118001r1">Candidate synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from SLIPT</a></span><br /><span class="lotToc">4.2 <a 
href="#x1-119001r2">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT</a></span><br /><span class="lotToc">4.3 <a 
href="#x1-121001r3">Pathway composition for clusters of <span class="cmti-12">CDH1 </span>partners from SLIPT</a></span><br /><span class="lotToc">4.4 <a 
href="#x1-127001r4">Pathway composition for <span class="cmti-12">CDH1</span> partners from SLIPT and siRNA screening</a></span><br /><span class="lotToc">4.5 <a 
href="#x1-128002r5">Pathways for <span class="cmti-12">CDH1</span> partners from SLIPT</a></span><br /><span class="lotToc">4.6 <a 
href="#x1-128003r6">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT and siRNA primary screen</a></span><br /><span class="lotToc">4.7 <a 
href="#x1-133001r7">Candidate synthetic lethal metagenes against <span class="cmti-12">CDH1 </span>from SLIPT</a></span><br /><span class="lotToc">4.8 <a 
href="#x1-135001r8">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in stomach cancer</a></span><br /><span class="lotToc">4.9 <a 
href="#x1-141001r9">Query synthetic lethal genes with the most SLIPT partners</a></span><br /><span class="lotToc">4.10 <a 
href="#x1-142001r10">Pathways for genes with the most SLIPT partners</a></span><br /><span class="lotToc">4.11 <a 
href="#x1-143001r11">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in CCLE</a></span><br /><span class="lotToc">4.12 <a 
href="#x1-143002r12">Pathways for <span class="cmti-12">CDH1</span> partners from SLIPT in breast CCLE</a></span><br /><span class="lotToc">5.1 <a 
href="#x1-160002r1">analysis of variance (ANOVA) for Synthetic Lethality and Vertex Degree</a></span><br /><span class="lotToc">5.2 <a 
href="#x1-162002r2">ANOVA for Synthetic Lethality and Information Centrality</a></span><br /><span class="lotToc">5.3 <a 
href="#x1-163002r3">ANOVA for Synthetic Lethality and PageRank Centrality</a></span><br /><span class="lotToc">5.4 <a 
href="#x1-167003r4">ANOVA for Synthetic Lethality and PI3K Hierarchy</a></span><br /><span class="lotToc">5.5 <a 
href="#x1-170001r5">Resampling for pathway structure of synthetic lethal detection methods</a></span><br /><span class="lotToc"> <a 
href="#x1-202002">B.1 R Packages used during Thesis</a></span><br /><span class="lotToc">C.1 <a 
href="#x1-203001r1">Comparing SLIPT genes against Secondary siRNA Screen in breast cancer</a></span><br /><span class="lotToc">C.2 <a 
href="#x1-203002r2">Comparing mtSLIPT genes against Secondary siRNA Screen in breast cancer</a></span><br /><span class="lotToc">C.3 <a 
href="#x1-203003r3">Comparing SLIPT genes against Secondary siRNA Screen in stomach cancer</a></span><br /><span class="lotToc">D.1 <a 
href="#x1-205001r1">Candidate synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from mtSLIPT</a></span><br /><span class="lotToc">D.2 <a 
href="#x1-205002r2">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT</a></span><br /><span class="lotToc">D.3 <a 
href="#x1-206002r3">Pathway composition for clusters of <span class="cmti-12">CDH1 </span>partners from mtSLIPT</a></span><br /><span class="lotToc">D.4 <a 
href="#x1-207002r4">Pathway composition for <span class="cmti-12">CDH1</span> partners from mtSLIPT and siRNA</a></span><br /><span class="lotToc">D.5 <a 
href="#x1-208001r5">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT</a></span><br /><span class="lotToc">D.6 <a 
href="#x1-208002r6">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT and siRNA primary screen</a></span><br /><span class="lotToc">D.7 <a 
href="#x1-210001r7">Candidate synthetic lethal metagenes against <span class="cmti-12">CDH1</span> from mtSLIPT</a></span><br /><span class="lotToc">F.1 <a 
href="#x1-216001r1">Synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from SLIPT in stomach cancer</a></span><br /><span class="lotToc">F.2 <a 
href="#x1-216002r2">Pathway composition for clusters of <span class="cmti-12">CDH1 </span>partners in stomach SLIPT</a></span><br /><span class="lotToc">F.3 <a 
href="#x1-217002r3">Pathway composition for <span class="cmti-12">CDH1 </span>partners from SLIPT and siRNA screening</a></span><br /><span class="lotToc">F.4 <a 
href="#x1-218001r4">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in stomach cancer</a></span><br /><span class="lotToc" >F.5 <a 
href="#x1-218002r5">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in stomach</p>
<p>and siRNA screen</a></span><br /><span class="lotToc">F.6 <a 
href="#x1-219001r6">Candidate synthetic lethal metagenes against <span class="cmti-12">CDH1</span> from SLIPT in stomach cancer</a></span><br /><span class="lotToc">G.1 <a 
href="#x1-221001r1">Synthetic lethal gene partners of <span class="cmti-12">CDH1</span> from mtSLIPT in stomach cancer</a></span><br /><span class="lotToc">G.2 <a 
href="#x1-221002r2">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT in stomach cancer</a></span><br /><span class="lotToc">G.3 <a 
href="#x1-222002r3">Pathway composition for clusters of <span class="cmti-12">CDH1 </span>partners in stomach mtSLIPT</a></span><br /><span class="lotToc">G.4 <a 
href="#x1-223002r4">Pathway composition for <span class="cmti-12">CDH1</span> partners from mtSLIPT and siRNA</a></span><br /><span class="lotToc">G.5 <a 
href="#x1-224001r5">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT in stomach cancer</a></span><br /><span class="lotToc">G.6 <a 
href="#x1-224002r6">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT in stomach and siRNA screen</a></span><br /><span class="lotToc">G.7 <a 
href="#x1-225001r7">Candidate synthetic lethal metagenes against <span class="cmti-12">CDH1 </span>from mtSLIPT in stomach cancer</a></span><br /><span class="lotToc">H.1 <a 
href="#x1-227001r1">Query synthetic lethal genes with the most SLIPT partners</a></span><br /><span class="lotToc">H.2 <a 
href="#x1-228001r2">Pathways for genes with the most SLIPT partners</a></span><br /><span class="lotToc">I.1 <a 
href="#x1-229001r1">Candidate synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from SLIPT in CCLE</a></span><br /><span class="lotToc">I.2 <a 
href="#x1-229002r2">Candidate synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from SLIPT in breast CCLE</a></span><br /><span class="lotToc">I.3 <a 
href="#x1-229003r3">Candidate synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from SLIPT in stomach CCLE</a></span><br /><span class="lotToc">I.4 <a 
href="#x1-229004r4">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in stomach CCLE</a></span><br /><span class="lotToc">I.5 <a 
href="#x1-229005r5">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in breast and stomach CCLE</a></span><br /><span class="lotToc">K.1 <a 
href="#x1-231004r1">ANOVA for Synthetic Lethality and Vertex Degree</a></span><br /><span class="lotToc">K.2 <a 
href="#x1-231005r2">ANOVA for Synthetic Lethality and Information Centrality</a></span><br /><span class="lotToc">K.3 <a 
href="#x1-231006r3">ANOVA for Synthetic Lethality and PageRank Centrality</a></span><br /><span class="lotToc">L.1 <a 
href="#x1-232001r1">Information centrality for genes and molecules in the Reactome network</a></span><br /><span class="lotToc">M.1 <a 
href="#x1-233002r1">ANOVA for Synthetic Lethality and PI3K Hierarchy</a></span><br /><span class="lotToc">M.2 <a 
href="#x1-233006r2">Resampling for pathway structure of synthetic lethal detection methods</a></span><br /></p>
</div>
<h2 class="likechapterHead">
<a 
 id="x1-3000"></a>List of Figures
</h2>
<div class="tableofcontents">
<p><span class="lofToc">1.1 <a 
href="#x1-35001r1">Synthetic genetic interactions</a></span><br /><span class="lofToc">1.2 <a 
href="#x1-40001r2">Synthetic lethality in cancer</a></span><br /><span class="lofToc">2.1 <a 
href="#x1-72003r1">Read count density</a></span><br /><span class="lofToc">2.2 <a 
href="#x1-72006r2">Read count sample mean</a></span><br /><span class="lofToc">3.1 <a 
href="#x1-93001r1">Framework for synthetic lethal prediction</a></span><br /><span class="lofToc">3.2 <a 
href="#x1-93002r2">Synthetic lethal prediction adapted for mutation</a></span><br /><span class="lofToc">3.3 <a 
href="#x1-95001r3">A model of synthetic lethal gene expression</a></span><br /><span class="lofToc">3.4 <a 
href="#x1-95002r4">Modeling synthetic lethal gene expression</a></span><br /><span class="lofToc">3.5 <a 
href="#x1-95003r5">Synthetic lethality with multiple genes</a></span><br /><span class="lofToc">3.6 <a 
href="#x1-96003r6">Simulating gene function</a></span><br /><span class="lofToc">3.7 <a 
href="#x1-96006r7">Simulating synthetic lethal gene function</a></span><br /><span class="lofToc">3.8 <a 
href="#x1-96009r8">Simulating synthetic lethal gene expression</a></span><br /><span class="lofToc">3.9 <a 
href="#x1-98001r9">Performance of binomial simulations</a></span><br /><span class="lofToc">3.10 <a 
href="#x1-98002r10">Comparison of statistical performance</a></span><br /><span class="lofToc">3.11 <a 
href="#x1-99004r11">Performance of multivariate normal simulations</a></span><br /><span class="lofToc">3.12 <a 
href="#x1-100005r12">Simulating expression with correlated gene blocks</a></span><br /><span class="lofToc">3.13 <a 
href="#x1-100010r13">Simulating expression with correlated gene blocks</a></span><br /><span class="lofToc">3.14 <a 
href="#x1-100013r14">Synthetic lethal prediction across simulations</a></span><br /><span class="lofToc">3.15 <a 
href="#x1-100017r15">Performance with correlations</a></span><br /><span class="lofToc">3.16 <a 
href="#x1-100018r16">Comparison of statistical performance with correlation structure</a></span><br /><span class="lofToc">3.17 <a 
href="#x1-100022r17">Performance with query correlations</a></span><br /><span class="lofToc">3.18 <a 
href="#x1-102003r18">Statistical evaluation of directional criteria</a></span><br /><span class="lofToc">3.19 <a 
href="#x1-102007r19">Performance of directional criteria</a></span><br /><span class="lofToc">3.20 <a 
href="#x1-107003r20">Simulated graph structures</a></span><br /><span class="lofToc">3.21 <a 
href="#x1-107010r21">Simulating expression from a graph structure</a></span><br /><span class="lofToc">3.22 <a 
href="#x1-107017r22">Simulating expression from graph structure with inhibitions</a></span><br /><span class="lofToc">3.23 <a 
href="#x1-110003r23">Demonstration of violin plots with custom features</a></span><br /><span class="lofToc">3.24 <a 
href="#x1-110004r24">Demonstration of annotated heatmap</a></span><br /><span class="lofToc">3.25 <a 
href="#x1-113001r25">Simulating graph structures</a></span><br /><span class="lofToc">4.1 <a 
href="#x1-120001r1">Synthetic lethal expression profiles of analysed samples</a></span><br /><span class="lofToc">4.2 <a 
href="#x1-123001r2">Comparison of SLIPT to siRNA</a></span><br /><span class="lofToc">4.3 <a 
href="#x1-124001r3">Compare SLIPT and siRNA genes with correlation</a></span><br /><span class="lofToc">4.4 <a 
href="#x1-124002r4">Compare SLIPT and siRNA genes with correlation</a></span><br /><span class="lofToc">4.5 <a 
href="#x1-125001r5">Compare SLIPT and siRNA genes with siRNA viability</a></span><br /><span class="lofToc">4.6 <a 
href="#x1-125002r6">Compare SLIPT and siRNA genes with viability</a></span><br /><span class="lofToc">4.7 <a 
href="#x1-125003r7">Compare SLIPT and siRNA genes with siRNA viability</a></span><br /><span class="lofToc">4.8 <a 
href="#x1-128001r8">Resampled intersection of SLIPT and siRNA candidates</a></span><br /><span class="lofToc">4.9 <a 
href="#x1-130001r9">Pathway metagene expression profiles</a></span><br /><span class="lofToc">4.10 <a 
href="#x1-131005r10">Somatic mutation against PI3K metagene</a></span><br /><span class="lofToc">4.11 <a 
href="#x1-132008r11">Somatic mutation locus against expression</a></span><br /><span class="lofToc">4.12 <a 
href="#x1-136001r12">Synthetic lethal expression profiles of stomach samples</a></span><br /><span class="lofToc">4.13 <a 
href="#x1-140001r13">Synthetic lethal partners across query genes</a></span><br /><span class="lofToc">5.1 <a 
href="#x1-155001r1">Synthetic Lethality in the PI3K Cascade</a></span><br /><span class="lofToc">5.2 <a 
href="#x1-156001r2">Synthetic Lethality in the Elastic Fibre Formation Pathway</a></span><br /><span class="lofToc">5.3 <a 
href="#x1-156002r3">Synthetic Lethality in the Fibrin Clot Formation</a></span><br /><span class="lofToc">5.4 <a 
href="#x1-160001r4">Synthetic Lethality and Vertex Degree</a></span><br /><span class="lofToc">5.5 <a 
href="#x1-162001r5">Synthetic Lethality and Centrality</a></span><br /><span class="lofToc">5.6 <a 
href="#x1-163001r6">Synthetic Lethality and PageRank</a></span><br /><span class="lofToc">5.7 <a 
href="#x1-166001r7">Structure of PI3K Ranking</a></span><br /><span class="lofToc">5.8 <a 
href="#x1-167001r8">Synthetic Lethality and Hierarchy Score in PI3K</a></span><br /><span class="lofToc">5.9 <a 
href="#x1-167002r9">Hierarchy Score in PI3K against Synthetic Lethality in PI3K</a></span><br /><span class="lofToc">5.10 <a 
href="#x1-167004r10">Structure of Synthetic Lethality in PI3K</a></span><br /><span class="lofToc">5.11 <a 
href="#x1-169001r11">Structure of Synthetic Lethality Resampling in PI3K</a></span><br /><span class="lofToc">6.1 <a 
href="#x1-181003r1">Simulated graph structures</a></span><br /><span class="lofToc">6.2 <a 
href="#x1-181006r2">Simulated graph structures</a></span><br /><span class="lofToc">6.3 <a 
href="#x1-181009r3">Simulated graph structures</a></span><br /><span class="lofToc" >6.4 <a 
href="#x1-181013r4">Simulated</p>
<p>graph structures</a></span><br /><span class="lofToc">6.5 <a 
href="#x1-181014r5">Simulated graph structures</a></span><br /><span class="lofToc">6.6 <a 
href="#x1-181015r6">Simulated graph structures</a></span><br /><span class="lofToc">6.7 <a 
href="#x1-181016r7">Simulated graph structures</a></span><br /><span class="lofToc">6.8 <a 
href="#x1-181017r8">Simulated graph structures</a></span><br /><span class="lofToc">6.9 <a 
href="#x1-181018r9">Simulated graph structures</a></span><br /><span class="lofToc">6.10 <a 
href="#x1-181019r10">Simulated graph structures</a></span><br /><span class="lofToc">A.1 <a 
href="#x1-200001r1">Correlation profiles of removed samples</a></span><br /><span class="lofToc">A.2 <a 
href="#x1-200002r2">Correlation analysis and sample removal</a></span><br /><span class="lofToc">A.3 <a 
href="#x1-201001r3">Replicate excluded samples</a></span><br /><span class="lofToc">A.4 <a 
href="#x1-201008r4">Replicate samples with all remaining</a></span><br /><span class="lofToc">A.5 <a 
href="#x1-201018r5">Replicate samples with some excluded</a></span><br /><span class="lofToc">A.6 <a 
href="#x1-201024r6">Replicate samples with some excluded</a></span><br /><span class="lofToc">D.1 <a 
href="#x1-206001r1">Synthetic lethal expression profiles of analysed samples</a></span><br /><span class="lofToc">D.2 <a 
href="#x1-207001r2">Comparison of mtSLIPT to siRNA</a></span><br /><span class="lofToc">D.3 <a 
href="#x1-209001r3">Compare mtSLIPT and siRNA genes with correlation</a></span><br /><span class="lofToc">D.4 <a 
href="#x1-209002r4">Compare mtSLIPT and siRNA genes with correlation</a></span><br /><span class="lofToc">D.5 <a 
href="#x1-209003r5">Compare mtSLIPT and siRNA genes with siRNA viability</a></span><br /><span class="lofToc">D.6 <a 
href="#x1-212005r6">Somatic mutation locus</a></span><br /><span class="lofToc">D.7 <a 
href="#x1-213005r7">Somatic mutation against PIK3CA metagene</a></span><br /><span class="lofToc">D.8 <a 
href="#x1-213010r8">Somatic mutation against PI3K protein</a></span><br /><span class="lofToc">D.9 <a 
href="#x1-213015r9">Somatic mutation against AKT protein</a></span><br /><span class="lofToc">E.1 <a 
href="#x1-214001r1">Pathway metagene expression profiles</a></span><br /><span class="lofToc">E.2 <a 
href="#x1-214002r2">Expression profiles for constituent genes of PI3K</a></span><br /><span class="lofToc">E.3 <a 
href="#x1-214003r3">Expression profiles for p53 related genes</a></span><br /><span class="lofToc">E.4 <a 
href="#x1-214004r4">Expression profiles for estrogen receptor related genes</a></span><br /><span class="lofToc">E.5 <a 
href="#x1-214005r5">Expression profiles for BRCA related genes</a></span><br /><span class="lofToc">F.1 <a 
href="#x1-217001r1">Comparison of SLIPT in stomach to siRNA</a></span><br /><span class="lofToc">G.1 <a 
href="#x1-222001r1">Synthetic lethal expression profiles of stomach samples</a></span><br /><span class="lofToc">G.2 <a 
href="#x1-223001r2">Comparison of mtSLIPT in stomach to siRNA</a></span><br /><span class="lofToc">H.1 <a 
href="#x1-226001r1">Synthetic lethal partners across query genes</a></span><br /><span class="lofToc">J.1 <a 
href="#x1-230001r1">Synthetic Lethality in the PI3K/AKT Pathway</a></span><br /><span class="lofToc">J.2 <a 
href="#x1-230002r2">Synthetic Lethality in the PI3K/AKT Pathway in Cancer</a></span><br /><span class="lofToc">J.3 <a 
href="#x1-230003r3">Synthetic Lethality in the Extracellular Matrix</a></span><br /><span class="lofToc">J.4 <a 
href="#x1-230004r4">Synthetic Lethality in the GPCRs</a></span><br /><span class="lofToc">J.5 <a 
href="#x1-230005r5">Synthetic Lethality in the GPCR Downstream</a></span><br /><span class="lofToc">J.6 <a 
href="#x1-230006r6">Synthetic Lethality in the Translation Elongation</a></span><br /><span class="lofToc">J.7 <a 
href="#x1-230007r7">Synthetic Lethality in the Nonsense-mediated Decay</a></span><br /><span class="lofToc">J.8 <a 
href="#x1-230008r8">Synthetic Lethality in the 3<sup><span class="cmsy-8">′</span></sup> UTR</a></span><br /><span class="lofToc">K.1 <a 
href="#x1-231001r1">Synthetic Lethality and Vertex Degree</a></span><br /><span class="lofToc">K.2 <a 
href="#x1-231002r2">Synthetic Lethality and Centrality</a></span><br /><span class="lofToc">K.3 <a 
href="#x1-231003r3">Synthetic Lethality and PageRank</a></span><br /><span class="lofToc">L.1 <a 
href="#x1-232002r1">Information centrality distribution</a></span><br /><span class="lofToc">M.1 <a 
href="#x1-233001r1">Synthetic Lethality and Heirarchy Score in PI3K</a></span><br /><span class="lofToc">M.2 <a 
href="#x1-233003r2">Heirarchy Score in PI3K against Synthetic Lethality in PI3K</a></span><br /><span class="lofToc">M.3 <a 
href="#x1-233004r3">Structure of Synthetic Lethality in PI3K</a></span><br /><span class="lofToc">M.4 <a 
href="#x1-233005r4">Structure of Synthetic Lethality Resampling</a></span><br /></p>
</div>
<p><a 
 id="likesection.1"></a><a 
 id="Q1-1-4"></a></p>
<h2 class="likechapterHead">
<a 
 id="x1-4000"></a>Glossary
</h2>
<div class="supertabular">
<table id="TBL-2" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-2-1g">
<col 
id="TBL-2-1" />
<col 
id="TBL-2-2" />
</colgroup>
<tr  
 style="vertical-align:baseline;" id="TBL-2-1-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-2-1-1"  
class="td11">
<pre><code>          &lt;/td&gt;&lt;/tr&gt;&lt;tr  </code></pre>
style=“vertical-align:baseline;” id=“TBL-2-2-”&gt;
<td style="white-space:nowrap; text-align:left;" id="TBL-2-2-1" class="td11">
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-2-3-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-2-3-1"  
class="td11">
<pre><code>           &lt;/td&gt;&lt;/tr&gt;&lt;tr  </code></pre>
style=“vertical-align:baseline;” id=“TBL-2-4-”&gt;
<td style="white-space:nowrap; text-align:left;" id="TBL-2-4-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-5-1" class="td11">
<a 
 id="glo:bioinformatics"></a>bioinformatics
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-5-2" class="td11">
Statistical or computational approaches to biological data or research tools.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-6-1" class="td11">
<a 
 id="glo:bisulfite-Seq"></a>bisulfite-Seq
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-6-2" class="td11">
Methylome data from sequencing bisulfite treated DNA.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-7-1" class="td11">
<pre><code>           &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-2-7-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-8-1" class="td11">
<a 
 id="glo:CAGE-Seq"></a>CAGE-Seq
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-8-2" class="td11">
Transcriptome data from cap analysis of gene expression.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-9-1" class="td11">
<pre><code>           &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-2-9-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-10-1" class="td11">
<a 
 id="glo:E-cadherin"></a>E-cadherin
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-10-2" class="td11">
Epithelial cadherin (calcium-dependent adhesion), a cell-adhesion protein encoded by the tumour suppressor gene <span class="cmti-12">CDH1</span>.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-11-1" class="td11">
<a 
 id="glo:epigenome"></a>epigenome
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-11-2" class="td11">
An analysis of epigenetic modifications of all genes in the genome.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-12-1" class="td11">
<a 
 id="glo:exome"></a>exome
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-12-2" class="td11">
An approach or technology designed to generate or use data from all genes in the genome.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-13-1" class="td11">
<pre><code>           &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-2-13-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-14-1" class="td11">
<a 
 id="glo:genomics"></a>genomics
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-14-2" class="td11">
An approach or technology designed to generate or use data from all genes in the genome.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-15-1" class="td11">
<pre><code>           &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-2-15-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-16-1" class="td11">
<a 
 id="glo:metabolome"></a>metabolome
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-16-2" class="td11">
An analysis of all the metabolites and enzymes in the cell.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-17-1" class="td11">
<a 
 id="glo:metagenomics"></a>metagenomics
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-17-2" class="td11">
An analysis of all the genes and genomes in a community.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-18-1" class="td11">
<a 
 id="glo:microRNA"></a>microRNA
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-18-2" class="td11">
Short RNA molecules generally regarded to regulate gene expression by binding to mRNA.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-19-1" class="td11">
<pre><code>           &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-2-19-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-20-1" class="td11">
<a 
 id="glo:omics"></a>omics
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-20-2" class="td11">
A rationale for applying molecular studies across all of the genes in the genome or biomolecules in the cell.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-21-1" class="td11">
<pre><code>           &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-2-21-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-22-1" class="td11">
<a 
 id="glo:Pan cancer"></a>Pan cancer
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-22-2" class="td11">
Study or analysis or cancers across tissue of origin.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-23-1" class="td11">
<a 
 id="glo:pleiotropy"></a>pleiotropy
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-23-2" class="td11">
A gene which has multiple biological functions.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-24-1" class="td11">
<a 
 id="glo:proteome"></a>proteome
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-24-2" class="td11">
An analysis of all the proteins expressed from the genome.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-25-1" class="td11">
<pre><code>           &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-2-25-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-26-1" class="td11">
<a 
 id="glo:RNA-Seq"></a>RNA-Seq
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-26-2" class="td11">
Transcriptome data from sequencing RNA.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-27-1" class="td11">
<pre><code>           &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-2-27-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-28-1" class="td11">
<a 
 id="glo:Sanger"></a>Sanger
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-28-2" class="td11">
A dideoxy chain termination method for DNA sequencing (named after Fred Sanger).
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-29-1" class="td11">
<a 
 id="glo:synthetic lethal"></a>synthetic lethal
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-29-2" class="td11">
Genetic interactions where inactivation of multiple genes is inviable (or deleterious) when they are viable if inactivated separately.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-30-1" class="td11">
<pre><code>           &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-2-30-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-2-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-2-31-1" class="td11">
<a 
 id="glo:transcriptome"></a>transcriptome
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-2-31-2" class="td11">
An analysis of all the genes expressed in the genome.
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-2-32-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-2-32-1"  
class="td11">
<pre><code>           &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/div&gt;
                                                                                
                                                                                </code></pre>
<!--l. 1-->
<p class="indent">
<p><a 
 id="likesection.2"></a><a 
 id="Q1-1-6"></a></p>
</p>
<h2 class="likechapterHead">
<a 
 id="x1-5000"></a>Acronyms
</h2>
<div class="supertabular">
<table id="TBL-4" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-4-1g">
<col 
id="TBL-4-1" />
<col 
id="TBL-4-2" />
</colgroup>
<tr  
 style="vertical-align:baseline;" id="TBL-4-1-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-4-1-1"  
class="td11">
<pre><code>     &lt;/td&gt;&lt;/tr&gt;&lt;tr  </code></pre>
style=“vertical-align:baseline;” id=“TBL-4-2-”&gt;
<td style="white-space:nowrap; text-align:left;" id="TBL-4-2-1" class="td11">
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-4-3-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-4-3-1"  
class="td11">
<pre><code>      &lt;/td&gt;&lt;/tr&gt;&lt;tr  </code></pre>
style=“vertical-align:baseline;” id=“TBL-4-4-”&gt;
<td style="white-space:nowrap; text-align:left;" id="TBL-4-4-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-5-1" class="td11">
<a 
 id="glo:AMPK"></a>AMPK
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-5-2" class="td11">
AMP-activated protein kinase.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-6-1" class="td11">
<a 
 id="glo:ANOVA"></a>ANOVA
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-6-2" class="td11">
Analysis of Variance.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-7-1" class="td11">
<a 
 id="glo:AUROC"></a>AUROC
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-7-2" class="td11">
Area under the reciever operating characteristic (curve).
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-8-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-8-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-9-1" class="td11">
<a 
 id="glo:BioPAX"></a>BioPAX
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-9-2" class="td11">
Biological Pathway Exchange.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-10-1" class="td11">
<a 
 id="glo:BiSEp"></a>BiSEp
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-10-2" class="td11">
Bimodal Subsetting Expression.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-11-1" class="td11">
<a 
 id="glo:BMP"></a>BMP
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-11-2" class="td11">
Bone morphogenic protein.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-12-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-12-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-13-1" class="td11">
<a 
 id="glo:CCLE"></a>CCLE
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-13-2" class="td11">
Cancer Cell Line Encyclopaedia.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-14-1" class="td11">
<a 
 id="glo:cDNA"></a>cDNA
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-14-2" class="td11">
Complementary deoxyribonucleic acid (from mRNA).
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-15-1" class="td11">
<a 
 id="glo:ChIP-Seq"></a>ChIP-Seq
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-15-2" class="td11">
Chromatin immunopreciptation sequencing.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-16-1" class="td11">
<a 
 id="glo:COSMIC"></a>COSMIC
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-16-2" class="td11">
Catalogue Of Somatic Mutations In Cancer.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-17-1" class="td11">
<a 
 id="glo:CXCR"></a>CXCR
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-17-2" class="td11">
Chemokine receptors.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-18-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-18-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-19-1" class="td11">
<a 
 id="glo:DAISY"></a>DAISY
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-19-2" class="td11">
Data mining synthetic lethal identification pipeline.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-20-1" class="td11">
<a 
 id="glo:DDBJ"></a>DDBJ
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-20-2" class="td11">
DNA Data Bank of Japan.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-21-1" class="td11">
<a 
 id="glo:DNA"></a>DNA
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-21-2" class="td11">
Deoxyribonucleic acid.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-22-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-22-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-23-1" class="td11">
<a 
 id="glo:EMBL"></a>EMBL
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-23-2" class="td11">
European Molecular Biology Laboratory.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-24-1" class="td11">
<a 
 id="glo:EMT"></a>EMT
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-24-2" class="td11">
Epithelial-mesenchymal transition.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-25-1" class="td11">
<a 
 id="glo:ENA"></a>ENA
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-25-2" class="td11">
The European Nucleotide Archive.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-26-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-26-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-27-1" class="td11">
<a 
 id="glo:GEO"></a>GEO
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-27-2" class="td11">
Gene Expression Omnibus.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-28-1" class="td11">
<a 
 id="glo:GO"></a>GO
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-28-2" class="td11">
Gene Ontology.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-29-1" class="td11">
<a 
 id="glo:GPCR"></a>GPCR
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-29-2" class="td11">
G protein coupled receptor.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-30-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-30-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-31-1" class="td11">
<a 
 id="glo:HDAC"></a>HDAC
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-31-2" class="td11">
Histone deacetylase.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-32-1" class="td11">
<a 
 id="glo:HDGC"></a>HDGC
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-32-2" class="td11">
Hereditary diffuse gastric cancer.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-33-1" class="td11">
<a 
 id="glo:HLRCC"></a>HLRCC
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-33-2" class="td11">
Hereditary leiomyomatosis and renal cell carcinoma.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-34-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-34-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-35-1" class="td11">
<a 
 id="glo:ICGC"></a>ICGC
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-35-2" class="td11">
International Cancer Genome Consortium.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-36-1" class="td11">
<a 
 id="glo:InDel"></a>InDel
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-36-2" class="td11">
Insertion or deletion (in DNA sequence).
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-37-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-37-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-38-1" class="td11">
<a 
 id="glo:JAK"></a>JAK
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-38-2" class="td11">
Janus kinase.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-39-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-39-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-40-1" class="td11">
<a 
 id="glo:lncRNA"></a>lncRNA
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-40-2" class="td11">
Long non-coding ribonucleic acid.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-41-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-41-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-42-1" class="td11">
<a 
 id="glo:miRNA"></a>miRNA
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-42-2" class="td11">
microRNA.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-43-1" class="td11">
<a 
 id="glo:mRNA"></a>mRNA
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-43-2" class="td11">
Messenger ribonucleic acid.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-44-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-44-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-44-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-45-1" class="td11">
<a 
 id="glo:NCBI"></a>NCBI
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-45-2" class="td11">
National Center for Biotechnology Information (in the USA).
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-46-1" class="td11">
<a 
 id="glo:NCI"></a>NCI
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-46-2" class="td11">
National Cancer Institute (in the USA).
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-47-1" class="td11">
<a 
 id="glo:NGS"></a>NGS
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-47-2" class="td11">
Next-generation sequencing.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-48-1" class="td11">
<a 
 id="glo:NHGRI"></a>NHGRI
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-48-2" class="td11">
National Human Genome Research Institute (in the USA).
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-49-1" class="td11">
<a 
 id="glo:NIG"></a>NIG
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-49-2" class="td11">
National Institute of Genetics (in Japan).
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-50-1" class="td11">
<a 
 id="glo:NIH"></a>NIH
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-50-2" class="td11">
National Institutes of Health (in the USA).
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-51-1" class="td11">
<a 
 id="glo:NMD"></a>NMD
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-51-2" class="td11">
Nonsense-mediated decay.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-52-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-52-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-52-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-53-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-53-1" class="td11">
<a 
 id="glo:PCR"></a>PCR
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-53-2" class="td11">
Polymerase chain reaction.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-54-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-54-1" class="td11">
<a 
 id="glo:PDE"></a>PDE
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-54-2" class="td11">
Phosphodiesterase.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-55-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-55-1" class="td11">
<a 
 id="glo:PI3K"></a>PI3K
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-55-2" class="td11">
Phosphoinositide 3-kinase.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-56-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-56-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-56-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-57-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-57-1" class="td11">
<a 
 id="glo:qPCR"></a>qPCR
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-57-2" class="td11">
Quantitative (real-time) polymerase chain reaction.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-58-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-58-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-58-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-59-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-59-1" class="td11">
<a 
 id="glo:RFLP"></a>RFLP
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-59-2" class="td11">
Restriction fragment length polymorphism.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-60-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-60-1" class="td11">
<a 
 id="glo:RGS"></a>RGS
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-60-2" class="td11">
G-protein signaling.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-61-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-61-1" class="td11">
<a 
 id="glo:RHO"></a>RHO
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-61-2" class="td11">
Ras Homolog Family.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-62-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-62-1" class="td11">
<a 
 id="glo:RNA"></a>RNA
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-62-2" class="td11">
Ribonucleic acid.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-63-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-63-1" class="td11">
<a 
 id="glo:RNAi"></a>RNAi
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-63-2" class="td11">
Ribonucleic acid interference.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-64-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-64-1" class="td11">
<a 
 id="glo:RRBS"></a>RRBS
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-64-2" class="td11">
Reduced representation bisulfite sequencing.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-65-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-65-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-65-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-66-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-66-1" class="td11">
<a 
 id="glo:SGI"></a>SGI
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-66-2" class="td11">
synthetic genetic interaction.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-67-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-67-1" class="td11">
<a 
 id="glo:siRNA"></a>siRNA
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-67-2" class="td11">
Short interfering ribonucleic acid.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-68-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-68-1" class="td11">
<a 
 id="glo:SLIPT"></a>SLIPT
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-68-2" class="td11">
Synthetic lethal interaction prediction tool.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-69-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-69-1" class="td11">
<a 
 id="glo:SNP"></a>SNP
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-69-2" class="td11">
Single nucleotide polymorphism.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-70-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-70-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-70-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-71-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-71-1" class="td11">
<a 
 id="glo:TCGA"></a>TCGA
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-71-2" class="td11">
The Cancer Genome Atlas (genomics project).
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-72-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-72-1" class="td11">
<a 
 id="glo:TGFB"></a>TGF<span class="cmmi-12">β </span>
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-72-2" class="td11">
Transforming growth factor <span class="cmmi-12">β</span>.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-73-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-73-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-73-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-74-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-74-1" class="td11">
<a 
 id="glo:UTR"></a>UTR
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-74-2" class="td11">
Untranslated region (of mRNA).
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-75-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-75-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:wrap; text-align:left;&quot; id=&quot;TBL-4-75-2&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-4-76-">
<td style="white-space:nowrap; text-align:left;" id="TBL-4-76-1" class="td11">
<a 
 id="glo:WNT"></a>WNT
</td>
<td style="white-space:wrap; text-align:left;" id="TBL-4-76-2" class="td11">
Wingless-related integration site.
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-4-77-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-4-77-1"  
class="td11">
<pre><code>      &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/div&gt;                                                                  
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                </code></pre>
<h2 class="chapterHead">
<span class="titlemark">Chapter 1</span><br /><a 
 id="x1-60001"></a>Introduction
</h2>
<!--l. 11-->
<p class="noindent">
The thesis presents research into genetic interactions based on <a 
href="#glo:genomics">genomics</a> data and <a 
href="#glo:bioinformatics">bioinformatics</a> approaches. This Chapter introduces the recent developments in <a 
href="#glo:genomics">genomics</a> and <a 
href="#glo:bioinformatics">bioinformatics</a>, particularly in their application to cancer research. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> interactions are a long standing area of research in genetics in both model organisms and cancer biology. Various reasons why these interactions are of interest in fundamental and translational biology will be outlined but first these and similar interactions will be defined. A <a 
href="#glo:bioinformatics">bioinformatics</a> approach to <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions enables much wider exploration of the inter-connected nature of genes and proteins within a cancer cell than previous candidate-based approaches. An alternative approach is experimental screening which will be presented and contrasted with <a 
href="#glo:bioinformatics">bioinformatics</a> approaches in more detail. An emerging application of <a 
href="#glo:synthetic lethal">synthetic lethality</a> is the design of treatments with specificity against loss of function mutations in tumour suppressor genes. <a 
href="#glo:E-cadherin">E-cadherin</a> (encoded by <span class="cmti-12">CDH1 </span>) is a prime example of this which will be the focus of the analysis in this thesis and as such the role of this gene in cellular and cancer biology will be briefly reviewed. <a 
 id="x1-6001r1"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">1.1 </span> <a 
 id="x1-70001"></a>Cancer Research in the Post-Genomic Era
</h3>
<!--l. 19-->
<p class="noindent">
<p><a 
href="#glo:genomics">Genomics</a> technologies have the potential to vastly impact upon various areas including health and cancer medicine. Considering the progress in recent <a 
href="#glo:genomics">genomics</a> research, this technology and the findings from it have considerable potential for significant impacts in the clinic and wider applications of genetics either directly or by enabling more focused genetics research on candidates selected from <a 
href="#glo:genomics">genomics</a> or <a 
href="#glo:bioinformatics">bioinformatics</a> analysis. The completion of the draft Human Genome (<a 
href="#XLander2001">Lander <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLander2001">2001</a>) marks a major accomplishment in genetics research and raises new challenges to utilise this genomic scale information effectively. Technologies in this area have rapidly developed since completion of the human genome project and many global large-scale projects have expanded upon the human genome, to populations (<a 
href="#X1000Genomes2010">1000 Genomes</a>, <a 
href="#X1000Genomes2010">2010</a>), to cancers (<a 
href="#XDickson1999">Dickson</a>, <a 
href="#XDickson1999">1999</a>; <a 
href="#XICGC2011">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XICGC2011">2011</a>), and to deeper functional understanding (<a 
href="#XFANTOM2001">Kawai <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFANTOM2001">2001</a>; <a 
href="#XENCODE2004">ENCODE</a>, <a 
href="#XENCODE2004">2004</a>). However, impact on the clinic has been slower than initially anticipated following the completion of the “draft” genome with <a 
href="#glo:genomics">genomics</a> technologies yet to become widely adopted in healthcare and oncology. Here we outline the <a 
href="#glo:genomics">genomics</a> technologies and <a 
href="#glo:bioinformatics">bioinformatics</a> approaches which have led to availability of <a 
href="#glo:genomics">genomics</a> data and techiques used in</p>
this thesis and potential for applications in cancer research or the clinic in the future. <a 
 id="x1-7001r1"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.1.1 </span> <a 
 id="x1-80001"></a>Cancer as a Global Health Concern
</h4>
<!--l. 23-->
<p class="noindent">
Cancer is a class of diseases involving malignant cellular growth, invasion of tissues, and spread to other organs. While there are also environmental factors, most cancers occur more frequently with age and family history. Accordingly, genetics has become widely acknowledged as having an important role in cancer risk (in addition to environmental factors). Cancers arise from dysregulated cellular growth or differentiation from stem cells. These can occur through genetic mutations or alterations in gene regulation or expression.
</p>
<!--l. 25-->
<p class="indent">
Cancers are a major global health concern, being the second leading cause of death globally (<a 
href="#XWorldHealthOrg2017">WHO</a>, <a 
href="#XWorldHealthOrg2017">2017</a>), with an estimated annual incidence of 14.1 million cases and annual mortality of 8.2 million people (<a 
href="#XFerlay2015">Ferlay <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFerlay2015">2015</a>). Breast and stomach cancers are among the 5 most frequent cancers globally, with breast cancer affecting women more than other cancer tissue types. Breast cancer has an estimated annual incidence of 1.6 million cases and mortality of 520 thousand people. Stomach cancer has an estimated annual incidence of 950 thousand cases and a mortality of 723 thousand people. Cancer is also a major health concern here in New Zealand, with 19.1 thousand people (including 2.5 thousand cases of breast cancer and 370 cases of stomach cancer) diagnosed annually (<a 
href="#XCIX2013">Hanna</a>, <a 
href="#XCIX2013">2003</a>), among the highest incidence (age-standardised per capita) of cancer in the world (<a 
href="#XFerlay2015">Ferlay <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFerlay2015">2015</a>).
</p>
<!--l. 27-->
<p class="indent">
<p>While the genetic contribution to cancer risk and many of the molecular changes occurring in cancers are widely acknowledged (<a 
href="#XASCO2017">ASCO</a>, <a 
href="#XASCO2017">2017</a>; <a 
href="#XCancerResearchUK2017">Cancer Research UK</a>, <a 
href="#XCancerResearchUK2017">2017</a>; <a 
href="#XCSNZ2017">Cancer Society of NZ</a>, <a 
href="#XCSNZ2017">2017</a>), the majority of these findings have yet to impact on clinical practice. Diagnostics are traditionally based on pathological examination of tissue samples where histological staining for cell type, biomolecules and biomarkers continue to be widely used, although genetic and biochemical markers are being adopted for some cancer types. In general, the current standard of care includes surgery, radiation, and cytotoxic chemotherapy, depending on whether the cancer is localised or has become systemic (via metastasis) and spread to other organ systems. These approaches can be effective against certain cancers, particularly in early stage cancer or in patients with particular subtypes (such as acute myeloid leukaemia) which respond well to modern treatment regimens. Thus early diagnosis is important to patient survival and quality of life.</p>
National screening programs (which prioritise patients with a high risk of cancer) therefore aim to diagnose cancers early and subtypes more accurately. Therefore identification of patients with genetic variants or family histories (for inclusion in national databases) for high risk of particular cancers is an important health issue, particularly where effective interventions exist if these cancers are diagnosed early. Thus high risk individuals being regularly monitored for some cancers are sometimes offered preventative surgery or treatment for pre-cancerous tissue (<a 
href="#XGuilford2010">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford2010">2010</a>; <a 
href="#XScheuer2002">Scheuer <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XScheuer2002">2002</a>).
</p>
<!--l. 29-->
<p class="indent">
Chemotherapy is a last-resort treatment for many advanced stage (systemic) cancers which is designed to inhibit the growth and spread of cancer throughout the body by targeting rapidly growing cells. However, this approach often has severe adverse effects, a narrow therapeutic window, and is not suitable for chemopreventative application in many cases (<a 
href="#XKaelin2009">Kaelin, Jr</a>, <a 
href="#XKaelin2009">2009</a>). Since surveillence preventative surgery (removing the tissue at risk of cancer) is not completely effective at preventing cancers and may impact on quality of life, depending on the cancer tissue types they are at risk of, alternative treatment strategies based on molecular biology and other fields are being investigated. These alternatives include immunological, endocrine, and targeted therapeutics, with a particular interest in treatments with specificity against cancer cells and wider applications (i.e., tolerable effective doses in applciations as a chemopreventative or against advanced stage cancers). <a 
 id="x1-8001r1"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.1.1 </span> <a 
 id="x1-90001"></a>The Genetics and Molecular Biology of Cancers
</h5>
<!--l. 37-->
<p class="noindent">
<p>Cancers involve dysregulation of genes with both somatic mutations or regulatory disruptions which accumulate during a patient’s lifetime and germline mutations which predispose individuals to high-risk early onset cancers (<a 
href="#XACS2017">American Cancer Society</a>, <a 
href="#XACS2017">2017</a>; <a 
href="#XGuilford1998">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford1998">1998</a>; <a 
href="#XNCI2015">NCI</a>, <a 
href="#XNCI2015">2015</a>). Cancer is widely viewed to be a genetic disease due to these familial cancer syndromes, hereditary risk factors, and the molecular changes occurring in cancers, including numerous cancer genes which have been identified <a 
href="#XStratton2009">Stratton <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XStratton2009">2009</a>); <a 
href="#XVogelstein2013">Vogelstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XVogelstein2013">2013</a>). Cancer genes are generally classified into two classes: “oncogenes” which are activated in cancers, driving tumour growth and invasion, or “tumour suppressors” which are inactivated in cancers, removing cellular regulation and genomic maintenance functions. The mutations which cause cancers accumulate with age and have been suggested to be inevitably coupled with aging due to the association of cancer incidence with the stem</p>
cell divisions in which mutations could occur across tissue types (<a 
href="#XTomasetti2015">Tomasetti and Vogelstein</a>, <a 
href="#XTomasetti2015">2015</a>).
</p>
<!--l. 47-->
<p class="indent">
<a 
href="#XHanahan2000">Hanahan and Weinberg</a> (<a 
href="#XHanahan2000">2000</a>) identified several key molecular and cellular traits shared across most cancers as a rational approach to the complex changes that occur in cancer initiation and progression due to common molecular machinery underlying all cells. A cancer cell must possess limitless replication potential, modulate growth signals to grow indefinitely, and gain invasive or metastatic capabilities. In addition, cancers must evade apoptosis, the immune system, and sustain angiogenesis and energy metabolism in order to survive (<a 
href="#XHanahan2000">Hanahan and Weinberg</a>, <a 
href="#XHanahan2000">2000</a>, <a 
href="#XHanahan2011">2011</a>). In order to achieve this, cancer cells undergo changes to their genomes and the surrounding cells to create a tumour microenvironment. Thus genomic instability has a key role in the survival and proliferation of cancer cells and the progression of further disease, as these malignant characteristics are acquired. Identifying the mechanisms of these acquired traits and the underlying genetic mutation or dysregulation behind them, such as <a 
href="#glo:E-cadherin">E-cadherin</a> mutation in metastasis or p53 mutation in genomic instability (<a 
href="#XHanahan2000">Hanahan and Weinberg</a>, <a 
href="#XHanahan2000">2000</a>), will be an important step in understanding and inhibiting cancer with the next generation of genomically-informed treatments.
</p>
<!--l. 49-->
<p class="indent">
<p>Molecular biological processes have particular importance in characterising breast cancers. Gene expression and regulatory signals confer cell identity and response to the environment. Therefore gene expression has been investigated with microarray technologies <a 
href="#XPerou2000">Perou <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XPerou2000">2000</a>), with “intrinsic subtypes” identified characterised by estrogen receptor, <span class="cmti-12">HER2</span>, and basal, epithelial signalling. The expression profiles were similar across independent samples of the same tumour and between primary and metastatic tumours of the same patient. Thus expression profiles represent the molecular state of a tumour rather than the sample and the molecular configuration of the cells regulation is carried through the cellular lineage of during metastasis preserving the molecular subtype. These molecular intrinsic subtypes “luminal A”, “luminal B”, “HER2-enriched”, “basal-like”, and “normal-like” have been replicated across microarray studies (<a 
href="#XHu2006">Hu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XHu2006">2006</a>), with their relevance to prognosis (including predicting survival and response to neoadjuvant chemotherapy) demonstrated and a 50-gene subtype predictor from microarray and <a 
href="#glo:qPCR">quantitative PCR (qPCR)</a> analysis has been provided (<a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>; <a 
href="#XSorlie2001">Sørlie <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XSorlie2001">2001</a>). This has been further updated with the “claudin-low” subtype (<a 
href="#XHerschkowitz2007">Herschkowitz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XHerschkowitz2007">2007</a>) and stimulated further investigations into subtyping of breast cancers by molecular properties.</p>
</p>
<!--l. 51-->
<p class="indent">
Despite differences in subtyping performed by different research groups and companies, there is widespread agreement that distinguishing luminal, HER2-enriched, and triple negative tumours can be performed with expression profiles and have value in our understanding of cancer progression and prognostic importance for patients <a 
href="#XDai2015">Dai <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XDai2015">2015</a>). High-throughput technologies have the potential to enable such subtyping on a vast scale in discovery of further subtypes in breast cancer or other diseases and in identification of these subtypes along with mutations in routine clinical diagnostic and prognostic testing. The “Pan cancer” approaches by the cancer genome atlas project (as discussed in more detail in Section <a 
href="#x1-230001">1.1.5.1.1<!--tex4ht:ref: TCGA_Findings --></a>) expand on the importance of molecular differences between cancers by examining molecular profiles across cancer tissue types (<a 
href="#XTCGA2013PAN">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013PAN">2013</a>).
</p>
<!--l. 53-->
<p class="indent">
<p>The molecular variability of cancer has also been approached rationally at a pathway level with patients subgroups activating different molecular pathways reflecting differences in disease mechanisms (<a 
href="#XGatza2010">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2010">2010</a>). A robust approach to measuring pathway activation in cancer is with a“metagene” which gives a consistent signal as a consensus of expression across genes even if they are inversely correlated (<a 
href="#XAnjomshoaa2008">Anjomshoaa <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XAnjomshoaa2008">2008</a>; <a 
href="#XHuang2003">Huang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XHuang2003">2003</a>; <a 
href="#XNagalla2013">Nagalla <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XNagalla2013">2013</a>). These are derived from the first principal component or eigenvector of a singular matrix decomposition, capturing the the most consistent variation across genes in a pathway or gene signature. <a 
href="#XGatza2010">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XGatza2010">2010</a>) used gene signatures for 18 cellular pathways in breast cancer to define subtypes with distinct molecular pathway activity. In a meta-analysis of Affymetrix microarray expression data for 1,143 samples and 50 cell lines, unsupervised hierarchical clustering robustly defined subtypes with common homogeneous pathway activity despite variation in the specific mutations giving rise to them. These subtypes with shared pathway activity have similar molecular characteristics (such as DNA copy number), clinical properties and prognosis, building upon the intrinsic subypes (<a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>) and providing a functional interpretation for molecular stratification (<a 
href="#XGatza2010">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2010">2010</a>). The pathway-based subtypes often correspond to intrinsic subtypes (<a 
href="#XGatza2010">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2010">2010</a>, <a 
href="#XGatza2014">2014</a>) and provide finer molecular stratification such as environmental stress response (to hypoxia within HER2-enriched cancers) (<a 
href="#XGatza2011">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2011">2011</a>) and have pathway-specfic DNA copy number variation or essential genes (<a 
href="#XGatza2014">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2014">2014</a>). <a 
href="#XGatza2014">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XGatza2014">2014</a>) extend these analyses include 52 pathway signatures from previous publications in breast cancer, replicating known characteristics (such as hormone re) of each subtype and identifying novel subtype-specific driver genes of proliferation by analysis of microarray</p>
expression from 476 from <a 
href="#glo:TCGA">The Cancer Genome Atlas (TCGA)</a>. In addition to distinct biological functions driving growth of breast cancer subtypes, these molecular subtypes provide a rational approach based on molecular properties to cancer treatment with combination and targeted therapies (<a 
href="#XGatza2010">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2010">2010</a>, <a 
href="#XGatza2014">2014</a>; <a 
href="#XHanahan2000">Hanahan and Weinberg</a>, <a 
href="#XHanahan2000">2000</a>).
</p>
<!--l. 55-->
<p class="indent">
Cancer is a major health concern with a well-established genetic contribution, in risk and in the molecular changes occurring during progression (<a 
href="#XStratton2009">Stratton <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XStratton2009">2009</a>). Many genes have been discovered to be important in different cancers with molecular differences between cancers, including alterations across the genome, being of clinical importance. As such cancers were among the first samples investigated with <a 
href="#glo:genomics">genomics</a> following the sequencing of the human genome <a 
href="#XDickson1999">Dickson</a> (<a 
href="#XDickson1999">1999</a>) and continue to be the subject of <a 
href="#glo:genomics">genomics</a> and <a 
href="#glo:bioinformatics">bioinformatics</a> investigations. <a 
 id="x1-9001r10"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.1.2 </span> <a 
 id="x1-100002"></a>The Human Genome Revolution
</h4>
<!--l. 58-->
<p class="noindent">
The advent of the Human Genome sequence (<a 
href="#XLander2001">Lander <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLander2001">2001</a>) has transformed genetics research including the study of health and disease (<a 
href="#XLander2011">Lander</a>, <a 
href="#XLander2011">2011</a>; <a 
href="#XPeltonen2001">Peltonen and McKusick</a>, <a 
href="#XPeltonen2001">2001</a>). Systematic, unbiased studies across all of the genes in the genome are viable in unprecedented ways. The successful undertaking of such an international scientific megaproject has set an example for numerous initiatives to follow, including many <a 
href="#glo:genomics">genomics</a> investigations expanding to species, to the functional, or to the population level (<a 
href="#XCollins2003">Collins <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCollins2003">2003</a>). These projects serve as an excellent resource for genetics research globally, particularly for cancers where <a 
href="#glo:genomics">genomics</a> investigation have been widely applied to different tissues across molecular profiles <a 
href="#XBamford2004">Bamford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XBamford2004">2004</a>); <a 
href="#XTCGA2013PAN">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTCGA2013PAN">2013</a>); <a 
href="#XICGC2011">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XICGC2011">2011</a>) . Genome sequencing technologies continue to improve, decrease in price, and become increasingly feasible in a wider range of research and clinical applications. <a 
 id="x1-10001r11"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.2.1 </span> <a 
 id="x1-110001"></a>The First Human Genome Sequence
</h5>
<!--l. 61-->
<p class="noindent">
<p>The first human genome is a good example of a large-scale <a 
href="#glo:genomics">genomics</a> project for it’s success as an international collaboration and releasing their data as a resource for the wider scientific community (<a 
href="#XCollins2003">Collins <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCollins2003">2003</a>; <a 
href="#XLander2001">Lander <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLander2001">2001</a>). This particular project generated significant public interest due to it being a landmark achievement, the first of it’s scale, and some controversial findings. Namely, the number of genes discovered (particularly those</p>
specific to vertebrates) was much lower than most estimates for a genome of it’s size and the number of repetitious transposon elements was very high. Even the figure of 30–40,000 genes given by the original publication is now regarded to be an overestimate (<a 
href="#XEzkurdia2014">Ezkurdia <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XEzkurdia2014">2014</a>; <a 
href="#XIHGSC2004">IHGSC</a>, <a 
href="#XIHGSC2004">2004</a>).
</p>
<!--l. 63-->
<p class="indent">
Accounting for the “complexity” encoded by the human genome with so few genes has led to investigations into molecular function, expression profiling, and population variation. When announcing the draft genome, <a 
href="#XLander2001">Lander <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLander2001">2001</a>) conceded that genomic information alone was not sufficient for biological understanding and that many investigations remained to carried out, with their objective being to share the raw genome data so that it was available for further inquiry rather than interpreting it themselves. While <a 
href="#glo:genomics">genomics</a> technologies and <a 
href="#glo:genomics">genomics</a> projects have flourished since then, the need in turn for systematic means of interpreting data of such scale and for the interdisciplinary expertise to do so has only grown. <a 
 id="x1-11001r13"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.2.2 </span> <a 
 id="x1-120002"></a>Impact of Genomics
</h5>
<!--l. 77-->
<p class="noindent">
<a 
href="#glo:genomics">Genomics</a> has stimulated investigations into many of these previously largely explored areas of functional genetics and thus been of immense value in genetics research, attracting high expectations for further applications. <a 
href="#glo:genomics">Genomics</a> research created widespread anticipation for potential applications in healthcare, agriculture, ecology, conservation, and evolutionary biology despite few of these being realised yet.
</p>
<!--l. 79-->
<p class="indent">
Cancer research is an area of particularly high expectations for the clinical impact of <a 
href="#glo:genomics">genomics</a> in oncology. <a 
href="#glo:genomics">Genomics</a> technologies have potential applications across cancer diagnostics, prognosis, management, and developing treatment. Cancers often involve genetic mutation or dysregulated gene expression which can be detected in a genome or transcriptome with potential to improve patient care. While direct impact of <a 
href="#glo:genomics">genomics</a> on the clinic has been limited, compared to initial expectations following the publication of the human genome, diagnostic cancer genes and therapeutic targets identified with <a 
href="#glo:genomics">genomics</a> research have begun to be introduced in the clinic (<a 
href="#XStratton2009">Stratton <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XStratton2009">2009</a>). <a 
 id="x1-12001r12"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.1.3 </span> <a 
 id="x1-130003"></a>Technologies to Enable Genetics Research
</h4>
<p><a 
 id="x1-13001r14"></a></p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.3.1 </span> <a 
 id="x1-140001"></a>DNA Sequencing and Genotyping Technologies
</h5>
<!--l. 88-->
<p class="noindent">
Genotyping was once commonly performed on variable regions of the genome with <a 
href="#glo:RFLP">restriction fragment length polymorphism (RFLP)</a> or repetitious microsatellite regions. These exploited sequence variation at target sites of restriction enzymes or measured the length of repetitious regions, using <a 
href="#glo:PCR">polymerase chain reaction (PCR)</a>, restriction enzymes, and gel electrophoresis to measure <a 
href="#glo:DNA">deoxyribonucleic acid (DNA)</a> genotypes at particular sites. This is laborious and limited to well characterised variable regions of the genome, generally genes or nearby marker regions.
</p>
<!--l. 90-->
<p class="indent">
The <a 
href="#glo:Sanger">Sanger</a> (dideoxy) chain termination method (<a 
href="#XSanger1975">Sanger and Coulson</a>, <a 
href="#XSanger1975">1975</a>) enabled DNA sequencing and genotyping at a widespread scale, being less technically difficult than the Maxam-Gilbert sequencing by degradation method (<a 
href="#XGilbert1973">Gilbert and Maxam</a>, <a 
href="#XGilbert1973">1973</a>; <a 
href="#XMaxam1977">Maxam and Gilbert</a>, <a 
href="#XMaxam1977">1977</a>), which required more radioactive and toxic reactants. The <a 
href="#glo:Sanger">Sanger</a> methodology has relatively long read length (particularly compared to early versions of more recent technologies), with read lengths of 500–700 base pairs accurately sequenced in most applications, usually following targeted amplification with <a 
href="#glo:PCR">PCR</a>. <a 
href="#glo:Sanger">Sanger</a> sequencing by gel electrophoresis takes around 6-8 hours and has been further refined with the “capillary” approach to 1–3 hours and requiring less input DNA and reactants. The capillary approach has been scaled up to run in parallel from a 96 well plate, at 166 kilobases per hour. The 96 well parallel capillary method was one of the main innovations which made the first Human Genome Project feasible and was used throughout (<a 
href="#XLander2001">Lander <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLander2001">2001</a>). Due to the quality of the <a 
href="#glo:Sanger">Sanger</a> sequence reads and low cost, it is still widely used in smaller scale applications, clinical testing, and to validate the findings of newer approaches. <a 
 id="x1-14001r16"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.3.2 </span> <a 
 id="x1-150002"></a>Microarrays and Quantitative Technologies
</h5>
<!--l. 94-->
<p class="noindent">
<p>Real-time or <a 
href="#glo:qPCR">qPCR</a> is another adaptation of genetic technologies to quantitatively study nucleic acids, often reverse transcribed “<a 
href="#glo:cDNA">deoxyribonucleic acid (cDNA)</a>” or messenger “<a 
href="#glo:mRNA">messenger ribonucleic acid (mRNA)</a>” to measure (relative) gene expression or transcript abundance. While numerous quality control measures are required to correctly interpret a qPCR experiment, these have similarly become widely adopted and are still used for smaller scale experiments and as a “gold standard” for measuring gene expression (<a 
href="#XAdamski2014">Adamski <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XAdamski2014">2014</a>). This also represents a</p>
shift in the application of <a 
href="#glo:qPCR">qPCR</a> and sequencing technology, where the primary interest is quantifying the amount of input material (by the rate of amplification to a certain level) rather than the qualitative nature of the sequence itself. The more recent technologies of microarrays and <a 
href="#glo:RNA-Seq">RNA-Seq</a> have similarly embraced this application to quantify DNA copy number, <a 
href="#glo:RNA">ribonucleic acid (RNA)</a> expression, and DNA methylation levels. Due to results of comparable or arguably better quality from these newer technologies (<a 
href="#XBeck2016">Beck <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBeck2016">2016</a>; <a 
href="#XGit2010">Git <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGit2010">2010</a>; <a 
href="#XMcCourt2013">McCourt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMcCourt2013">2013</a>; <a 
href="#XRobin2016">Robin <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XRobin2016">2016</a>), this “gold standard” status has begun to come under scrutiny.
</p>
<!--l. 96-->
<p class="indent">
Microarrays represent a truly high-throughput molecular technique, reducing the cost, time, and labour required to study molecular factors such as genotype, expression, or methylation across many genes, making it feasible to do so over a statistically meaningful number of samples. Microarrays are manufactured with probes which measure binding of particular nucleotide sequences to either quantitatively detect the presence of a sequence such as a <a 
href="#glo:SNP">single nucleotide polymorphism (SNP)</a> or quantify DNA copy number, gene expression, or DNA CpG methylation. Microarray technologies have popularised “genome scale” studies of genetic variation and expression.
</p>
<!--l. 98-->
<p class="indent">
In addition to being more versatile and higher-throughput than <a 
href="#glo:PCR">PCR</a> based techniques, microarrays are considered cost-effective, particularly when scaled up to a large number of probes. They are also available with established gene panels or customised probes from a number of commercial manufacturers. These remained popular during the introduction of newer technologies due to reliability and relatively lower cost, especially in large-scale projects involving many samples. However, microarrays have issues with signal-to-noise ratio, with both sensitivity to low nucleic acid abundance and “saturation” of probes at high abundance, edge effects, and requiring more starting material than <a 
href="#glo:qPCR">qPCR</a>. Thus <a 
href="#glo:qPCR">qPCR</a> is still used for many small gene panel studies. <a 
 id="x1-15001r17"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.3.3 </span> <a 
 id="x1-160003"></a>Massively Parallel “Next Generation” Sequencing
</h5>
<!--l. 103-->
<p class="noindent">
<p>Similar to microarrays, the introduction of massively parallel sequencing technologies have further expanded the availability of high-throughput molecular studies to researchers, with corresponding availability of <a 
href="#glo:genomics">genomics</a> data from these studies. This <a 
href="#glo:NGS">“Next-Generation Sequencing” (NGS)</a> expands not only gene expression studies (compared to microarrays) but extends to genome sequencing <span class="cmti-12">de novo </span>for previously unknown genome and transcriptome</p>
sequences at an unprecedented scale. This has been a particularly important technological revolution in <a 
href="#glo:genomics">genomics</a>, as the cost and time of genome sequencing has dropped dramatically and enabled sequencing projects of far more samples and applications beyond the Human Genome Project. Particularly, when dealing with variants in a species with an existing reference sequence such as humans, where there is a low computational cost of mapping to a reference compared to a genome assembly. However, the cost of sequencing (<a 
href="#glo:RNA-Seq">RNA-Seq</a>) for gene expression or DNA methylation studies is still considerably higher than a microarray study (limiting feasible sample sizes).
</p>
<!--l. 105-->
<p class="indent">
Compared with arrays, <a 
href="#glo:NGS">NGS</a> studies have additional challenges, particularly regarding large data and compute requirements to handle the raw output data. Compared with the the established methods to analyse microarray data, handling <a 
href="#glo:NGS">NGS</a> data can be more technically difficult. While methods developed for analysing microarray data can be repurposed for sequence analysis in many cases, more <a 
href="#glo:bioinformatics">bioinformatics</a> expertise is required particularly to handle the raw read data and changing approaches for various developments in sequencing technologies. One of the main computational challenges is the assembly of reads or mapping to a reference genome due to the inherently small reads of most <a 
href="#glo:NGS">NGS</a> technologies compared to the Sanger methodology. Furthermore, there are fewer software releases and best practices established specifically for <a 
href="#glo:RNA-Seq">RNA-Seq</a> data, thus many analyses are still conducted with customised analysis approaches and command-line tools. Compared to existing graphical tools or pipelines for microarray analysis, this is a more active technology for <a 
href="#glo:bioinformatics">bioinformatics</a> research where many applications of <a 
href="#glo:genomics">genomics</a> data have yet to be explored.
</p>
<!--l. 107-->
<p class="indent">
<p>However, the methodology itself has challenges with the sample preparation, requiring a relatively high quantity of input material and “contamination” with over abundant ribosomal rRNA taking up the majority of the sequencing if not purified correctly. This abundance of rRNA is a particularly important issue in microarray and <a 
href="#glo:RNA">RNA</a> experiments in Eukaryotes where it is commonplace to target the <a 
href="#glo:mRNA">mRNA</a> by binding to the poly-A tail (<a 
href="#glo:RNA-Seq">RNA-Seq</a>) or 5’ cap (<a 
href="#glo:CAGE-Seq">CAGE-Seq</a>). However, this has the potential to exclude <a 
href="#glo:miRNA">miRNA (miRNA)</a> and <a 
href="#glo:lncRNA">long non-coding ribonucleic acid (lncRNA)</a> of interest unless the sample is prepared specifically to study these. Similarly capturing a subsection of the genome for <a 
href="#glo:exome">exome</a> analysis or <a 
href="#glo:RRBS">reduced representation bisulfite sequencing (RRBS)</a>, focuses on sequencing DNA sequences and methylation levels of CpG sites near known genes to reduce cost, noise, and incidental findings.</p>
</p>
<!--l. 110-->
<p class="indent">
In many cases, the benefits of <a 
href="#glo:NGS">NGS</a> technologies over microarrays still outweigh the additional cost. <a 
href="#glo:NGS">NGS</a> technologies have the advantage of greater potential accuracy and sensitivity than microarrays, depending on the sequencing depth or “coverage”, theoretically sensitive down to the exact number of molecules for each transcript. <a 
href="#glo:NGS">NGS</a> experiments are regarded as “reproducible” with no need for technical replicates, although these are still performed for a subset of samples in many projects for quality assurance purposes. <a 
href="#glo:NGS">NGS</a> has a wider dynamic range than microarrays and is able to detect <a 
href="#glo:SNP">SNPs</a>, <a 
href="#glo:InDel">insertions or deletions</a>, and splice variants in addition to quantifying DNA copy number or transcript abundance. <a 
href="#glo:NGS">NGS</a> scales to all genes and beyond for these molecular applications without having to design new probes as required for a microarray. Thus <a 
href="#glo:NGS">NGS</a> technologies are not limited to genes with already characterised sequence or functions, do not need to be updated with new probes for each genome annotation release, and do not require a reference genome at all for new species. A “transcriptome” can be assembled <span class="cmti-12">de novo </span>for an expression study in any organism by sequencing the mRNA extracted from a cell. <a 
 id="x1-16001r1"></a>
</p>
<a 
 id="x1-170001"></a>Molecular Profiling with Genomics Technology <!--l. 117-->
<p class="indent">
<p><a 
href="#glo:NGS">NGS</a> is highly adaptable to different applications: DNA sequencing (whole <a 
href="#glo:genomics">genome</a> or <a 
href="#glo:exome">exome</a>), DNA methylation (<a 
href="#glo:bisulfite-Seq">bisulfite-Seq</a>), <a 
href="#glo:RNA-Seq">RNA-Seq</a>, <a 
href="#glo:miRNA">miRNA</a>, <a 
href="#glo:lncRNA">lncRNA</a>, or <a 
href="#glo:ChIP-Seq">chromatin immunopreciptation sequencing (ChIP-Seq)</a>. <a 
href="#glo:RNA-Seq">RNA-Seq</a> of the <a 
href="#glo:transcriptome">transcriptome</a> is a common adaptation where <a 
href="#glo:RNA">RNA</a> is reverse transcribed and sequenced from the resulting <a 
href="#glo:cDNA">cDNA</a>. This is utilised to quantify the levels of <a 
href="#glo:RNA">RNA</a> and identify which regions of DNA are expressed. Similar bisulfite treatment converts cytosine residues to uracil (sequenced as thymidine), sparing methylated cytosine enabling it to be distinguished with bisulfite-Seq for high-throughput detection of the notable epigenetic mark and is a common procedure to generate an <a 
href="#glo:epigenome">epigenome</a>. Subsets of the nucleic acid may be extracted for sequencing such as the coding regions of DNA (for the “<a 
href="#glo:exome">exome</a>”), the mRNA 5<span class="cmsy-10x-x-120">′ </span>cap (<a 
href="#glo:CAGE-Seq">CAGE-Seq</a>), mRNA 3<span class="cmsy-10x-x-120">′ </span>poly-A tail (<a 
href="#glo:RNA-Seq">RNA-Seq</a>), <a 
href="#glo:microRNA">microRNA</a>, or an enriched subset of variable regions for DNA sequencing (“genotyping by sequencing”) and methylation studies (“reduced-representation bisulfite sequencing). High-throughput gel and mass spectrometry techniques have been applied to proteins and metabolites to generate the <a 
href="#glo:proteome">proteome</a> and <a 
href="#glo:metabolome">metabolome</a> respectively. These “<a 
href="#glo:omics">omics</a>” technologies are applicable across a wide range of biomolecules in a cell and these “molecular profiles” are produced in many experimental laboratories.</p>
<a 
 id="x1-17001r19"></a>
</p>
<a 
 id="x1-180002"></a>Sequencing Technologies <!--l. 128-->
<p class="indent">
Illumina sequencing (developed by Solexa and later acquired by Illumina) was released in 2006. It utilises reversible terminating dyes to sequence by synthesis with a lower accuracy (98%) and read lengths of 150–250bp. Illumina more than makes up for relatively short reads (along with improving the read length of the technology) and low accuracy with high-throughput and cost effectiveness, with a Hi-Seq 4000 platform producing up to 10 billion paired-end reads (1500Gbp) in a run of appropriately 3 days, capable of sequencing 12 human genomes (30<span class="cmsy-10x-x-120">×</span> coverage) or 100 human transcriptomes simultaneously (<a 
href="#XIllumina">Illumina</a>, <a 
href="#XIllumina">2017</a>). Illumina has further reduced the cost of sequencing with the economies of scale with the Hi-Seq X 10 claiming to produce a human genome (with 30<span class="cmsy-10x-x-120">× </span>coverage) for less than US<span class="tcrm-1200">$</span>1000, the first platform to achieve this long-standing goal in <a 
href="#glo:genomics">genomics</a>. The high-throughput of Illumina sequencing also makes deep sequencing for high coverage, high quality consensus reads, and sensitive <a 
href="#glo:RNA-Seq">RNA-Seq</a> experiments feasible. Illumina sequencing now has a dominating market share of the <a 
href="#glo:NGS">NGS</a> technologies. Their Hi-Seq platforms were used in large-scale gemomics projects (such as the cancer genome atlas discussed in Section <a 
href="#x1-220001">1.1.5.1<!--tex4ht:ref: TCGA_Intro --></a>) to generate the sequence data used throughout this thesis.
</p>
<!--l. 143-->
<p class="indent">
<a 
href="#glo:NGS">NGS</a> technologies continue to be refined with Illumina (and competitors) continuing to release refined productions, offer additional genomics-based services, and decreases overhead and operating costs to enable a wider range of research projects. As such <a 
href="#glo:RNA-Seq">RNA-Seq</a> for examining the transcriptome of an organisms and for expression studies in diseases is a growing field of research and expression data will continue to be generated for a range of samples in many healthy and diseased tissues. The technology be also be further improved developments in speed and accuracy (such as Ion Torrent platforms) and towards long reads of single molecules (such as Pacific Biosciences, Oxford Nanopore, and Quantum Biosystems Japan).
</p>
<!--l. 149-->
<p class="indent">
<p>Due to such benefits of sequencing over previous technologies (and their continued refinement), this thesis has focused on gene expression data generated by <a 
href="#glo:RNA-Seq">RNA-Seq</a> rather by microarrays. <a 
href="#glo:RNA-Seq">RNA-Seq</a> data is widely available as a resource from large-scale cancer <a 
href="#glo:genomics">genomics</a> projects and methods to make inferences from <a 
href="#glo:RNA-Seq">RNA-Seq</a> experiments could feasibly be applied to many other studies based on these current (or similar future) technologies.</p>
<a 
 id="x1-18001r18"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.3.4 </span> <a 
 id="x1-190004"></a>Bioinformatics as Interdisciplinary Genomic Analysis
</h5>
<!--l. 154-->
<p class="noindent">
<a 
href="#glo:genomics">Genomics</a> technologies have given rise to data at a scale previously rarely encountered in molecular biology, making inference with conventional techniques difficult. Computational, Mathematical, and Statistical skills are required to handle this data effectively, in addition to biological background to frame and interpret research questions. Drawing upon these disciplines to handle biological data has become the field of “<a 
href="#glo:bioinformatics">Bioinformatics</a>”, focusing specifically on making inferences from <a 
href="#glo:genomics">genomics</a> and high-throughput molecular data or developing the tools to do so. This contrasts with the existing fields of “theoretical” or “computational biology” which existed prior to <a 
href="#glo:genomics">genomics</a> data, focusing on modelling and simulating aspects of biology without necessarily addressing the <a 
href="#glo:genomics">genomics</a> data or detecting the phenomena in nature, extending beyond genetics to cell modelling, neuroscience, cancer development, ecology, and evolution.
</p>
<!--l. 156-->
<p class="indent">
In practice, many researchers identify with both <a 
href="#glo:bioinformatics">bioinformatics</a> and computational biology, or draw upon the findings and methods of the other field. This thesis uses many approaches in <a 
href="#glo:bioinformatics">bioinformatics</a> to biological research questions and established mathematical or <a 
href="#glo:bioinformatics">bioinformatics</a> resources.
</p>
<!--l. 158-->
<p class="indent">
Gene expression analysis is the focus of many <a 
href="#glo:bioinformatics">bioinformatics</a> research groups, drawing upon statistical approaches to appropriately handle microarray and <a 
href="#glo:RNA-Seq">RNA-Seq</a> data along with making biological inferences from a large number of statistical tests. This presents various challenges from normalising sample data and accounting for batch effects to developing or applying statistical tests tailored to biological hypotheses and testing them at a genome-wide scale, generally across thousands of genes. There are numerous approaches for dealing with these challenges, some of which will be described in Chapter <a 
href="#x1-650002">2<!--tex4ht:ref: chap:methods --></a>. <a 
 id="x1-19001r15"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.1.4 </span> <a 
 id="x1-200004"></a>Follow-up Large-Scale Genomics Projects
</h4>
<!--l. 164-->
<p class="noindent">
<p>A number of projects have attempted to follow up on the human genome project to varying degrees of success. The genomes have since been sequenced for a variety of model organisms, organisms of importance in health, agriculture, <a 
href="#glo:metagenomics">metagenomics</a> of microorganisms (microbiome), ecology and conservation. <a 
href="#glo:genomics">Genomics</a> projects have also been applied functional genetics (<a 
href="#XFANTOM2001">Kawai <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFANTOM2001">2001</a>; <a 
href="#XENCODE2004">ENCODE</a>, <a 
href="#XENCODE2004">2004</a>) and to human populations with an interest</p>
variability between individuals and health or disease risk (<a 
href="#XHapMap2003">HapMap</a>, <a 
href="#XHapMap2003">2003</a>; <a 
href="#X1000Genomes2010">1000 Genomes</a>, <a 
href="#X1000Genomes2010">2010</a>).
</p>
<!--l. 168-->
<p class="indent">
<a 
href="#glo:genomics">Genomics</a> databases have also focused on faciltating distribution of genomic data generated by researchers, rather than generating it themselves. GenBank hosted by the <a 
href="#glo:NCBI">National Center for Biotechnology Information (NCBI)</a> in the US, the <a 
href="#glo:ENA">The European Nucleotide Archive (ENA)</a> hosted by the <a 
href="#glo:EMBL">European Molecular Biology Laboratory (EMBL)</a>, and the <a 
href="#glo:DDBJ">DNA Data Bank of Japan (DDBJ)</a> hosted by the <a 
href="#glo:NIG">National Institute of Genetics (NIG)</a> do so by serving as public repositories of DNA sequence data. <a 
href="#glo:GEO">Gene Expression Omnibus (GEO)</a> (<a 
href="#XGEO2016">Clough and Barrett</a>, <a 
href="#XGEO2016">2016</a>), arrayExpress (<a 
href="#XArrayExpress2013">Rustici <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XArrayExpress2013">2013</a>), and caArray (<a 
href="#XcaArray2014">Heiskanen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XcaArray2014">2014</a>) serve a similar purpose as a resource for gene expression datasets, originally developed for microarray data but <a 
href="#glo:RNA-Seq">RNA-Seq</a> data is now supported by some platforms. They are repositories for researchers to deposit, share, and access gene expression data, serving as a resource to support ongoing research where larger datasets than would were previously accessible for many individual laboratories are available (<a 
href="#XRung2013">Rung and Brazma</a>, <a 
href="#XRung2013">2013</a>). These resources cover not only DNA sequence across the genome but also molecular profiles of other factors by adapting genomic sequencing or other high throughput technologies for quantifying gene expression or DNA methylation. Sharing the expression datasets generated in a publication is now required by some journals.
</p>
<!--l. 170-->
<p class="indent">
Similarly, international projects and consortiums have begun to release data gathered using common agreed upon protocols in laboratories across the world, often hosting public databases of these themselves, publishing their own investigations into the datasets as they are released, or offering basic searches and analytics of the data via a web portal. These databases include many of the <a 
href="#glo:genomics">genomics</a> projects discussed above and the cancer-specific projects discussed below. In many ways, the quality, consistency, and accessibility of these international projects has become more appealing than accessing smaller studies, particularly for gene expression datasets where the more recent, larger projects have switched from microarray to <a 
href="#glo:RNA-Seq">RNA-Seq</a> technologies. This distinction will also be discussed later. <a 
 id="x1-20001r22"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.1.5 </span> <a 
 id="x1-210005"></a>Cancer Genomes
</h4>
<!--l. 173-->
<p class="noindent">
<p>The importance of genomics technologies in the future of cancer research was noticed, even in the early days of <a 
href="#glo:genomics">genomics</a> (<a 
href="#XDickson1999">Dickson</a>, <a 
href="#XDickson1999">1999</a>). The Cancer Genome Project (CGP) based at</p>
Wellcome Trust Sanger Institute in the UK were among the first to launch investigations into cancer after the publication of the Human Genome, using this genome sequence, consensus across the cancer research literature, and sequencing the genes of cancers themselves. Initially, the Sanger Institute set out to sequence 20 genes across 378 samples while the Human Genome project was still ongoing (<a 
href="#XCollins2007">Collins and Barker</a>, <a 
href="#XCollins2007">2007</a>), optimising sequencing and computation infrastructure for a larger project while doing so. The main aim of the Cancer Genome Project was to discover “cancer genes”, those frequently mutated in cancers by comparing the genes of cancer and normal tissue samples, both “oncogenes” and “tumour suppressors” which are activated and inactivated respectively in cancers. This project is ongoing and the UK continues to be involved in international sequencing initiatives and those focused on particular tissue types.
</p>
<!--l. 175-->
<p class="indent">
The Sanger Institute also hosts the Catalogue of Somatic Mutations in Cancer (<a 
href="#XCOSMICdb">COSMIC</a>, <a 
href="#XCOSMICdb">2016</a>), a database and website of cancer genes. This launched with 66,634 samples and 10,647 mutations from initial investigations into <span class="cmti-12">BRAF</span>, HRAS, <span class="cmti-12">KRAS</span>, and NRAS (<a 
href="#XBamford2004">Bamford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBamford2004">2004</a>). It has since expanded to include 1,257,487 samples with 4,175,8787 gene mutations curated from 23,870 publications, including 29,112 whole genomes (<a 
href="#XCOSMICdb">COSMIC</a>, <a 
href="#XCOSMICdb">2016</a>). This database now also identifies cancer genes from DNA copy number, differential gene expression and differential DNA methylation. <a 
 id="x1-21001r21"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.5.1 </span> <a 
 id="x1-220001"></a>The Cancer Genome Atlas Project
</h5>
<!--l. 178-->
<p class="noindent">
Based in the US, <a 
href="#glo:TCGA">The Cancer Genome Atlas</a> (<a 
href="#glo:TCGA">TCGA</a>) project was established in 2005, a combined effort of the <a 
href="#glo:NCI">National Cancer Institute (NCI)</a> and the <a 
href="#glo:NHGRI">National Human Genome Research Institute (NHGRI)</a> of the <a 
href="#glo:NIH">National Institutes of Health (NIH)</a> (<a 
href="#XTCGA2017web">TCGA</a>, <a 
href="#XTCGA2017web">2017a</a>). They first set out to demonstrate the pilot project on brain (<a 
href="#XTCGA2008GBM">McLendon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2008GBM">2008</a>), ovarian (<a 
href="#XTCGA2011OV">Bell <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2011OV">2011</a>), and squamous cell lung (<a 
href="#XTCGA2012LUSC">Hammerman <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012LUSC">2012</a>) cancers. In 2009, the project expanded aiming to analyse 500 samples each for 20-25 tumour tissue types. They have since exceeded that goal, with data available for 33 cancer types including 10 “rare” cancers, a total of over 10,000 samples.
</p>
<!--l. 180-->
<p class="indent">
<p>The <a 
href="#glo:TCGA">TCGA</a> projects set out to generate a molecular “profile” of the tumour (and some matched normal tissue) samples: the genotype, somatic mutations, gene expression, DNA copy number, and <a 
href="#glo:RNA">RNA</a> methylation levels. While these were originally performed</p>
largely with microarray technologies, exome and <a 
href="#glo:RNA-Seq">RNA-Seq</a> has been since adopted and performed for many <a 
href="#glo:TCGA">TCGA</a> samples, with whole genomes being performed for some samples. Data which cannot be used to identify the patients (such as somatic mutation, expression, methylation, and various clincal factors) are publicly available. <a 
 id="x1-22001r20"></a>
</p>
<a 
 id="x1-230001"></a>Findings from Cancer Genomes <!--l. 189-->
<p class="indent">
The Cancer Genome Atlas pilot projects (<a 
href="#XTCGA2011OV">Bell <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2011OV">2011</a>; <a 
href="#XTCGA2012LUSC">Hammerman <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012LUSC">2012</a>; <a 
href="#XTCGA2008GBM">McLendon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2008GBM">2008</a>) serve to demonstrate the power of applying <a 
href="#glo:genomics">genomics</a> technologies to cancer research at such as scale. In addition to sequencing the whole genome or a subset (exome), DNA copy number, gene expression, DNA methylation, and somatic mutations were also analysed. The initial projects used microarray technologies for expression and methylation data but these have since been replaced by <a 
href="#glo:RNA-Seq">RNA-Seq</a> for expression. <a 
href="#glo:TCGA">TCGA</a> demonstrated the potential discovery of the molecular basis of cancer by analysing 206 glioblastoma brain cancer samples (<a 
href="#XTCGA2008GBM">McLendon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2008GBM">2008</a>), highlighting the roles of <span class="cmti-12">ERBB2</span>, <span class="cmti-12">NF1</span>, <span class="cmti-12">TP53</span>, and <span class="cmti-12">PIK3R1 </span>mutations, along with altered methylation of <span class="cmti-12">MGMT</span>, and the core pathways of RTK, p53, and RB signalling in brain cancer. An analysis of 489 serious ovarian cancers (<a 
href="#XTCGA2011OV">Bell <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2011OV">2011</a>) similarly reported <span class="cmti-12">TP53 </span>mutations specifically over-represented in high grade tumours and reported 133 copy number variants, 168 differentially methylated regions, and recurrently somatic mutations in 9 genes in low grade tumours including <span class="cmti-12">NF1</span>, <span class="cmti-12">BRCA1</span>, <span class="cmti-12">BRCA2</span>, <span class="cmti-12">RB1</span>, and <span class="cmti-12">CDK12</span>. Four transcriptional subtypes of ovarian cancers were identified, alterations in <span class="cmti-12">BRCA1</span>, <span class="cmti-12">BRCA2</span>, and <span class="cmti-12">CCLE </span>had an impact on patient survival, and the homologous recombination, NOTCH and FOXM1 signalling pathways were involved in ovarian cancer growth. The <a 
href="#glo:genomics">genomics</a> of 178 squamous cell lung cancers (<a 
href="#XTCGA2012LUSC">Hammerman <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012LUSC">2012</a>) were highly complex, averaging at 360 muations in coding regions. While no targeted therapies existed for this cancer subtype, 11 recurrently mutated genes were identified including <span class="cmti-12">TP53 </span>and <span class="cmti-12">HLA-A</span>. The pathways altered in various squamous cell lung cancers were NFE2L2, KEAP1, differentiation genes, PI3K, CDKN2A and RB1. These aberrant genes and pathways represent potential therapeutic targets which could be identified for most samples.
</p>
<!--l. 191-->
<p class="indent">
<p>The <a 
href="#glo:TCGA">TCGA</a> breast cancer analysis (<a 
href="#XTCGA2012">TCGA</a>, <a 
href="#XTCGA2012">2012</a>) consisted of 802 samples with exomes, copy number variants, RPPA protein quantification, and DNA methylation, mRNA, and</p>
microRNA arrays with 97 whole genomes sequenced. Four main molecular classes were identified to subtype the samples, despite considerable heterogeneity between samples. Recurrent mutations across more than 10% of samples were identified in <span class="cmti-12">TP53</span>,<span class="cmti-12">PIK3CA</span>, and <span class="cmti-12">GATA</span>. <a 
href="#glo:TCGA">TCGA</a> further suggests subtypes by HER2 and EGFR protein levels. In a further analysis of 817 breast cancer samples including 127 invasive lobular breast and 88 mixed type samples (<a 
href="#XTCGA2015LBC">Ciriello <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2015LBC">2015</a>), 3 molecular subtypes of lobular breast cancer were identified. Lobular breast cancer was also characterised by recurrent mutations in <span class="cmti-12">CDH1</span>, <span class="cmti-12">PTEN</span>, <span class="cmti-12">TBX2</span>, and <span class="cmti-12">FOXA1</span>.
</p>
<!--l. 193-->
<p class="indent">
<a 
href="#glo:TCGA">TCGA</a> reported results of colon and rectal cancers in a combined analysis of 267 samples (<a 
href="#XTCGA2012CRC">Muzny <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012CRC">2012</a>), finding no genomic distinction between colorectal cancers. Apart from 16% of hypermutated colorectal cancers, the remaining samples were very similar at the molecular level with 24 significantly recurrently mutated genes identified. These include the expected <span class="cmti-12">APC</span>, <span class="cmti-12">TP53</span>, <span class="cmti-12">SMAD4</span>, <span class="cmti-12">PIK3CA</span>, and <span class="cmti-12">KRAS</span> genes. Additionally, novel recurrent mutations were identified in <span class="cmti-12">ARID1A</span>, <span class="cmti-12">SOX9</span>, and <span class="cmti-12">FAM123 </span>along with recurrent copy number alterations in <span class="cmti-12">ERBB2 </span>and <span class="cmti-12">IFG2</span>. Thus the molecular findings of colon and rectal tumours can be applicable across colorectal cancers, including the known characteristics of microsatellite instability (MSI) and CpG island methylator phenotype (CIMP) found in some colorectal tumours.
</p>
<!--l. 195-->
<p class="indent">
The <a 
href="#glo:TCGA">TCGA</a> stomach cancer analysis of 295 samples (<a 
href="#XTCGA2014GC">Bass <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014GC">2014</a>) identified 4 molecular subtypes of stomach cancers characterised by: the Epstein-Barr virus, MSI, <a 
href="#glo:genomics">genomics</a> instability, and chromosomal instability. Abberrations in <span class="cmti-12">PD-L1</span>, <span class="cmti-12">PIK3CA</span>, and <span class="cmti-12">JAK2 </span>were also identified in stomach cancers which may present therapeutic targets. <a 
 id="x1-23001r25"></a>
</p>
<a 
 id="x1-240002"></a>Genomic Comparisons Across Cancer Tissues <!--l. 199-->
<p class="indent">
<p><a 
href="#glo:TCGA">TCGA</a> have identified various genes as recurrent, driver mutations across cancer types which are likely to have a role in driving the proliferation of these cancers and present a molecular target that could be applied across tissue types. These include <span class="cmti-12">TP53 </span>(in brain, lung/head/neck squamous cell, breast, colorectal, uterine, and endometrial cancers), <span class="cmti-12">ERBB2 </span>/HER2/NEU (in brain, breast, colorectal, bladder, and lung cancers), <span class="cmti-12">PIK3CA,</span> <span class="cmti-12">PIK3R1 </span>(in brain, breast, colorectal, endometrial, bladder, clear cell renal, and lung cancers), <span class="cmti-12">BRCA1</span>/<span class="cmti-12">BRCA2 </span>(in breast and ovarian cancers), <span class="cmti-12">NF1 </span>(in brain, ovarian, and skin cancers), <span class="cmti-12">ARID1A </span>(in colorectal, endometrial, and clear cell</p>
renal cancers), <span class="cmti-12">KRAS </span>(in colorectal, endometrial, and skin cancers), <span class="cmti-12">BRAF </span>(in colorectal, thyroid, and skin cancers), <span class="cmti-12">EGFR </span>(in brain, breast, and lung cancers), and <span class="cmti-12">PTEN </span>(in breast, endometrial, and uterine cancers) (<a 
href="#XTCGA2014TH">Agrawal <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014TH">2014</a>; <a 
href="#XTCGA2015SK">Akbani <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2015SK">2015</a>; <a 
href="#XTCGA2014GC">Bass <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014GC">2014</a>; <a 
href="#XTCGA2011OV">Bell <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2011OV">2011</a>; <a 
href="#XTCGA2017CERV">Burk <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2017CERV">2017</a>; <a 
href="#XTCGA2017UT">Cherniack <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2017UT">2017</a>; <a 
href="#XTCGA2015LBC">Ciriello <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2015LBC">2015</a>; <a 
href="#XTCGA2014LU">Collisson <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014LU">2014</a>; <a 
href="#XTCGA2013RCC">Creighton <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013RCC">2013</a>; <a 
href="#XTCGA2012LUSC">Hammerman <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012LUSC">2012</a>; <a 
href="#XTCGA2013ENDO">Kandoth <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013ENDO">2013</a>; <a 
href="#XTCGA2015HNSC">Lawrence <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2015HNSC">2015</a>; <a 
href="#XTCGA2008GBM">McLendon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2008GBM">2008</a>; <a 
href="#XTCGA2012CRC">Muzny <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012CRC">2012</a>; <a 
href="#XTCGA2012">TCGA</a>, <a 
href="#XTCGA2012">2012</a>; <a 
href="#XTCGA2014BL">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014BL">2014</a>). In addition to disregarding the tissue-based distinction between colon and rectal cancers based on molecular similarlity (<a 
href="#XTCGA2012CRC">Muzny <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012CRC">2012</a>), the <a 
href="#glo:TCGA">TCGA</a> project have observed differences within tumour types and proposed molecular subtyping for breast, clear cell renal, papillary renal, stomach, skin, bladder, and prostate cancers (<a 
href="#XTCGA2015PR">Abeshouse <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2015PR">2015</a>; <a 
href="#XTCGA2015SK">Akbani <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2015SK">2015</a>; <a 
href="#XTCGA2014GC">Bass <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014GC">2014</a>; <a 
href="#XTCGA2015LBC">Ciriello <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2015LBC">2015</a>; <a 
href="#XTCGA2013RCC">Creighton <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013RCC">2013</a>; <a 
href="#XTCGA2012LUSC">Hammerman <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012LUSC">2012</a>; <a 
href="#XTCGA2016PRC">Linehan <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2016PRC">2016</a>; <a 
href="#XTCGA2012CRC">Muzny <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2012CRC">2012</a>; <a 
href="#XTCGA2012">TCGA</a>, <a 
href="#XTCGA2012">2012</a>; <a 
href="#XTCGA2014BL">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014BL">2014</a>).
</p>
<!--l. 201-->
<p class="indent">
<p>The “<a 
href="#glo:Pan cancer">Pan cancer</a>” project (<a 
href="#XTCGA2014PAN">Hoadley <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014PAN">2014</a>; <a 
href="#XTCGA2013PAN">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013PAN">2013</a>) analysed 3527 samples across 12 tissue types for DNA, RNA, protein, and epigenetic molecular profiles. This project was initated in 2012 to perform a comprehensive analysis of molecular data across cancer types to identify molecular simliarities and differences. Recurrent <span class="cmti-12">TP53</span> mutations characterised high grade tumours across breast, ovarian, and endometrial cancers. HER2 was identified in brain, endometrial, bladder, and lung cancers, in addition to the known role of HER2 in breast cancers. <span class="cmti-12">BRCA1 </span>and <span class="cmti-12">BRCA2 </span>mutations were also detected across cancers, mainly breast and ovarian cancers as expected. Microsatellite instability characterised both endometrial and colorectal cancers. The Pan cancer project (<a 
href="#XTCGA2014PAN">Hoadley <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014PAN">2014</a>) has identified 11 molecular subtypes across these tissues, 5 of corresponding to tissue cancer types and the remainder reassigned due to molecular similarities shared across cancer types. Squamous cell lung, head, and neck and a subset bladder cancers were grouped together by molecular similarities, characteried by a high frequency of <span class="cmti-12">TP53 </span>mutations. Conversely, bladder cancers were divided into 3 of these molecular subtypes with distinct profiles. This project further supports the genomic stratification of patients, demonstrated in breast cancer (<a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>; <a 
href="#XMETABRIC2016">Pereira <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMETABRIC2016">2016</a>; <a 
href="#XPerou2000">Perou <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XPerou2000">2000</a>), which may apply to other cancer types and to molecular characteristics across them targeting recurrent mechanisms of cancer growth and progression (<a 
href="#XHanahan2000">Hanahan and Weinberg</a>, <a 
href="#XHanahan2000">2000</a>, <a 
href="#XHanahan2011">2011</a>). <a 
 id="x1-24001r26"></a></p>
</p>
<a 
 id="x1-250003"></a>Cancer Genomic Data Resources <!--l. 204-->
<p class="indent">
While the findings from the <a 
href="#glo:TCGA">TCGA</a> projects themselves are a considerable contribution to understanding cancer biology within and across tissue types, the main eventual benefit of such projects will be the avaialability of the data for the research community to analyse further and use to inform future investigations (<a 
href="#XTCGA2008GBM">McLendon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2008GBM">2008</a>; <a 
href="#XTCGA2017web">TCGA</a>, <a 
href="#XTCGA2017web">2017a</a>; <a 
href="#XTCGA2013PAN">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013PAN">2013</a>). These serve as a vast resource of common and rare cancer types and are publicly available for further analysis (<a 
href="#XcBioPortal">cBioPortal</a>, <a 
href="#XcBioPortal">2017</a>; <a 
href="#XTCGA2017web">TCGA</a>, <a 
href="#XTCGA2017web">2017a</a>; <a 
href="#XICGC2011">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XICGC2011">2011</a>). This also applies to the Molecular Taxonomy of Breast Cancer International Consortium (METABRIC) project which focuses on breast cancer which also aimed to identify novel molecular subtypes (<a 
href="#XMETABRIC2012">Curtis <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMETABRIC2012">2012</a>). They performed an analysis of 2433 breast cancer samples with long-term clincal data, gene expression, copy number variants, and 173 genes sequenced which identified 40 driver mutations in breast cancer in addition to further support for molecular subtyping to identify patient groups with different clinical outcomes (<a 
href="#XMETABRIC2016">Pereira <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMETABRIC2016">2016</a>). <a 
 id="x1-25001r23"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.1.6 </span> <a 
 id="x1-260006"></a>Genomic Cancer Medicine
</h4>
<!--l. 216-->
<p class="noindent">
There is much anticipation in cancer research for <a 
href="#glo:genomics">genomics</a> technologies to have a clinical impact in cancer medicine: from diagnosis and prognosis to treatment developments and strategies. These may result either from direct use of genome or <a 
href="#glo:RNA-Seq">RNA-Seq</a> in clinical laboratories or indirectly from biomarkers and treatments developed with research faciltated by <a 
href="#glo:genomics">genomics</a>. This second strategy is likely to have a more immediate patient benefit due to the cost of genome sequencing, particularly considering adoption in public healthcare systems with a limited budget. <a 
 id="x1-26001r24"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.6.1 </span> <a 
 id="x1-270001"></a>Cancer Genes and Driver Mutations
</h5>
<!--l. 219-->
<p class="noindent">
<p>There are two main categories of “cancer genes” (<a 
href="#XFutreal2001">Futreal <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFutreal2001">2001</a>). Oncogenes are those activated in cancers either by gain of function mutations in proto-oncogenes, amplification of DNA copies, or evelated gene expression. Their normal functions are typically to regulate stem cells or to promote cellular growth and recurrent mutations are typically concentrated to particular gene regions. Conversely, tumour suppressor genes are those inactivated in</p>
cancer either by loss of function mutations, deletion of DNA copies, repression of gene expression, or hypermethylation. Their normal functions are typically to regulate cell division, DNA repair, and cell signalling.
</p>
<!--l. 221-->
<p class="indent">
Detecting these cancer genes is a major challenge in cancer biology and has been revolutionised by genomic technologies. Recurrent mutations, or DNA copy number variants and differential gene expression or DNA methylation are all indicative of cancer genes (<a 
href="#XMattison2009">Mattison <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMattison2009">2009</a>), which can be detected in <a 
href="#glo:genomics">genomics</a> data (<a 
href="#XMETABRIC2016">Pereira <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMETABRIC2016">2016</a>; <a 
href="#XTCGA2013PAN">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013PAN">2013</a>). Important “driver” cancer genes (<a 
href="#XStratton2009">Stratton <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XStratton2009">2009</a>) are difficult to detect from “passenger” mutations due to patient variation, tumour heterogeneity, and genomic instability. However, many cancer genes have been replicated from previous studies or well supported from <a 
href="#glo:genomics">genomics</a> data. There remains the challenge of translating the identification of cancer genes to patient benefit with characterisation of variants of unknown significance, which mutation or gene expression markers can be used to monitor tumour progression or treament response, and design of therapeutic intervention against many molecular target for which they have yet to be developed or repurposed from other disease to cancers.
</p>
<!--l. 223-->
<p class="indent">
Driver mutations can be identified by whether they co-occur or are mutually exclusive with mutations in other genes in cancers, are recurrently mutated across a significant proportion of samples for a specific tissue type, or if mutations are recurrent across different cancer tissue types (<a 
href="#XcBioPortal">cBioPortal</a>, <a 
href="#XcBioPortal">2017</a>; <a 
href="#XMETABRIC2016">Pereira <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMETABRIC2016">2016</a>; <a 
href="#XCOSMICdb">COSMIC</a>, <a 
href="#XCOSMICdb">2016</a>; <a 
href="#XTCGA2013PAN">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2013PAN">2013</a>; <a 
href="#XICGC2011">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XICGC2011">2011</a>). Approximately 140 driver mutations have been identified, including many novel genes in particular cancers from <a 
href="#glo:genomics">genomics</a> studies, with 2–8 in typically occurring in each tumour usually affecting cell fate, survival, or genome maintenance (<a 
href="#XVogelstein2013">Vogelstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XVogelstein2013">2013</a>). <a 
 id="x1-27001r29"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.6.2 </span> <a 
 id="x1-280002"></a>Personalised or Precision Cancer Medicine
</h5>
<!--l. 226-->
<p class="noindent">
<p>The notion of using a patient’s genome to tailor healthcare to an individual has been appealing since the advent of <a 
href="#glo:genomics">genomics</a>, popularised with the term “personalised medicine”. This approach was expected to span from preventative lifestyle advice to effective treatments. Personalised medicine was described in contrast with current strategies of health advice, screening, prognostics, and treatments based on what works well for the majority of the population. Adverse effects of these treatments occur in a</p>
significant subpopulation, particularly demographics under-represented in clinical studies.
</p>
<!--l. 228-->
<p class="indent">
The importance of <a 
href="#glo:genomics">genomics</a> is still recognised in translational cancer research. Applications are particularly emphasised in molecular diagnosis, prognosis, and treatments of patients already presenting with cancers in the clinic rather than preventative medicine. This is in part due to the vast number of variants of unknown clinical significance, the ethical issue of reporting on incidental findings, and the regulatory issues direct-to-consumer genetics companies have encountered offering health risk assessment.
</p>
<!--l. 230-->
<p class="indent">
More recently the term “genomic medicine” has been preffered the describe the paradigm of treating cancers by their genomic features, particularly grouping patients by the mutation, expression, or DNA methylation profiles of their cancers. Radical proponents advocate for these molecular subtypes to supercede tissue or cell type specific diagnosis of cancers. However, in practice they are often used in combination, with clinical and pathological factors being informative of prognosis and the medical expertise required for treatment. The related term of “precision medicine” also stems from this trend with the rationale to target these molecular subtypes with separate treatment strategies, particularly in developing and applying treatments targeted against a particular mutation specific to cancers. To this end much research in this field is focused on identifying mutations and gene expression signatures amenable to distinguishing cancers, particularly oncogenic driver mutations, and developing treatments against them. <a 
 id="x1-28001r27"></a>
</p>
<a 
 id="x1-290001"></a>Molecular Diagnostics and Pan-Cancer Medicine <!--l. 233-->
<p class="indent">
<p>There is growing support for the use of molecular tools such as mutations or gene expression signatures to diagnose tumour subtypes addition to (or in lieu of) tissue of origin or histology. This is particularly important in breast cancer where analysis of molecular data detected several distinct “intrinsic subtypes” with differences in malignancy and patient outcome which were distinguished by molecular mechanisms rather than tissue or cellular phenotype (<a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>; <a 
href="#XPerou2000">Perou <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XPerou2000">2000</a>). Conversely, common molecular mechanisms may be shared between cancers across tissue types as discovered by the “Pan cancer” studies, such as those conducted by the <a 
href="#glo:TCGA">TCGA</a> and <a 
href="#glo:ICGC">International Cancer Genome Consortium (ICGC)</a> projects, which combined molecular profiles across tissue types <a 
href="#XTCGA2013PAN">Weinstein <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTCGA2013PAN">2013</a>). The molecular subtypes could feasibly be included in clinical</p>
testing as a panel of biomarkers for diagnostics and prognosis. Such biomarkers also have the potential to monitor drug response or risk of recurrence. This is also raises the need for development of treatments that target these molecular subtypes. <a 
 id="x1-29001r30"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.6.3 </span> <a 
 id="x1-300003"></a>Targeted Therapeutics and Pharmacogenomics
</h5>
<!--l. 236-->
<p class="noindent">
Targeted therapies with specificity against a molecular target are emerging as precision cancer medicine. Molecular targets can be tested in laboratory conditions with <a 
href="#glo:RNA">RNA</a> interference or pharmacological agents. Identification of molecular targets is important for developing novel anti-cancer treatments along with validation and drug testing. For oncogenic mutations, the recurrent mutant variant or overexpressed gene is directly inhibited using structure-aided drug design or compound screening. However, oncogenes with high homology to other genes or tumour suppressor genes (where lost in cancers) are not amenable to direct targeting (<a 
href="#XKaelin2009">Kaelin, Jr</a>, <a 
href="#XKaelin2009">2009</a>).
</p>
<!--l. 241-->
<p class="indent">
Despite controversy over their prohibitively high cost (<a 
href="#XPharmac2016">PHARMAC</a>, <a 
href="#XPharmac2016">2016</a>), targeted therapeutics have been applied as monoclonal antibodies against oncogenes (such as <span class="cmti-12">HER2</span>) with relative sucess in clinical trials (<a 
href="#XMiles2001">Miles</a>, <a 
href="#XMiles2001">2001</a>), generating considerable interest in wider application of this approach. Targeted therapeutics have potential to have applications across cancer tissue types, specificity against tumour cells, wide therapeutic windows, and combination therapies (even in advanced disease or as a chemopreventative in high-risk individuals). <a 
 id="x1-30001r31"></a>
</p>
<a 
 id="x1-310001"></a>Targeting Oncogenic Driver Mutations <!--l. 247-->
<p class="indent">
<p>Oncogene targeted therapies have also been developed with some examples of effective clinical application against cancers. However, they have already begun to manifest problems with resistance, recurrence, tissue specificity, and design of inhibitors specific to oncogenic variants rather than proto-oncogene precusors. Targeted anticancer therapeutics can exploit complex interactions to distinguish normal and cancerous cells which may benefit from studies of gene regulation or interaction networks. The unexpected synergy between inhibitors of the oncogenes <span class="cmti-12">BRAF </span><sup class="textsuperscript"><span class="cmr-10x-x-109">V600E</span> </sup> and <span class="cmti-12">EGFR </span>in colorectal cancer is an example of such a system <a 
href="#XPrahallad2012">Prahallad <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XPrahallad2012">2012</a>).</p>
</p>
<!--l. 253-->
<p class="indent">
Despite successful application of vemurafenib against <span class="cmti-12">BRAF</span><sup class="textsuperscript"><span class="cmr-10x-x-109">V600E</span></sup> in melanomas <a 
href="#XDienstmann2011">Dienstmann and Tabernero</a> (<a 
href="#XDienstmann2011">2011</a>); <a 
href="#XRavnan2012">Ravnan and Matalka</a> (<a 
href="#XRavnan2012">2012</a>), colorectal cancers with <span class="cmti-12">BRAF </span><sup class="textsuperscript"><span class="cmr-10x-x-109">V600E</span> </sup> mutations have poor prognosis and lack drug response. <a 
href="#XPrahallad2012">Prahallad <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XPrahallad2012">2012</a>) used an <a 
href="#glo:RNAi">RNA interference (RNAi)</a> screen and found that <span class="cmti-12">EGFR </span>inhibition is synergistic with vemurafenib aginst <span class="cmti-12">BRAF</span><sup class="textsuperscript"><span class="cmr-10x-x-109">V600E</span></sup> in colon cell lines and xenografts due to feedback activation of <span class="cmti-12">EGFR</span>. Vemurafenib induced rapid reactivation of MAPK/ERK signalling via <span class="cmti-12">EGFR </span>in colorectal cell lines in a tissue-specific manner <a 
href="#XCorcoran2012">Corcoran <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XCorcoran2012">2012</a>) and may be be relevant to acquired resistance in melanoma <a 
href="#XSun2014">Sun <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XSun2014">2014</a>). Thus combination therapies against several molecular pathways may be necessary to anticipate acquired resistance <a 
href="#XRavnan2012">Ravnan and Matalka</a> (<a 
href="#XRavnan2012">2012</a>) and targeted therapeutics may be further refined from understanding the pathway structure and functional interactions across cancer cells. <a 
 id="x1-31001r32"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.1.6.4 </span> <a 
 id="x1-320004"></a>Systems and Network Biology
</h5>
<!--l. 257-->
<p class="noindent">
It is also important to consider that driver mutations in oncogenes and tumour suppressor genes do not occur in isolation. The genetic interaction, regulatory and cellular signalling, and metabolic reactions of are all inter-related and may each be perturbed by aberrations in gene function occurring in cancers. These relationships can be represented by biological networks by mapping pairs of genes with a particular relationship. Due to the complexity of a cell, these molecular networks are very large consisting of thousands of nodes comprised by genes or proteins.
</p>
<!--l. 259-->
<p class="indent">
The properties of large networks were first studied by constructing random networks by randomly linking a fixed number of nodes (<a 
href="#XErdos1959">Erd<img 
src="thesis0x.png" alt="&#x02DD;o  "  class="Huml"  />s and Rényi</a>, <a 
href="#XErdos1959">1959</a>, <a 
href="#XErdos1960">1960</a>). Despite the random nature of these networks, properties such as their connectivity were well characterised. The vertex degree (number of partners for each node) of random network follows a Poisson distribution, however this property does not hold in nature, suggesting that natural networks are non-random or not formed in this way <a 
href="#XBarabasi2004">Barabási and Oltvai</a> (<a 
href="#XBarabasi2004">2004</a>).
</p>
<!--l. 261-->
<p class="indent">
<p>This work formed the foundation for studying complex networks (<a 
href="#XvanSteen2010">van Steen</a>, <a 
href="#XvanSteen2010">2010</a>), which model features of real world networks not found in Erd<img 
src="thesis1x.png" alt="&#x02DD;o  "  class="Huml"  />s and Rényi‘s random networks (<a 
href="#XErdos1959">Erd<img 
src="thesis2x.png" alt="&#x02DD;o  "  class="Huml"  />s and Rényi</a>, <a 
href="#XErdos1959">1959</a>, <a 
href="#XErdos1960">1960</a>). The small world property, made popular by findings in social networks (<a 
href="#XTravers1969">Travers and Milgram</a>, <a 
href="#XTravers1969">1969</a>), is the remarkably short path lengths between any nodes in a small world network. A small world network is</p>
well-connected with a characteristic path length (the average length of shortest paths between all pairs of nodes) proportional to the logarithm of the number of nodes. <a 
href="#XWatts1998">Watts and Strogatz</a> (<a 
href="#XWatts1998">1998</a>) developed a model of random rewiring of a regular network to construct random networks with the small world property and a high clustering coefficient. While these properties are more representative of networks occurring in nature, their model is limited by the degree distribution which converges to a Poisson distribution as it is rewired <a 
href="#XBarrat2000">Barrat and Weigt</a> (<a 
href="#XBarrat2000">2000</a>).
</p>
<!--l. 263-->
<p class="indent">
The vertex degree distribution of naturally occurring networks often follows a power law distribution with the majority of nodes having far fewer connections than average and a small subset of highly connected network ‘hubs’ <a 
href="#XBarabasi1999">Barabási and Albert</a> (<a 
href="#XBarabasi1999">1999</a>). Hubs further differentiate into ‘party’ hubs (which interact simultaneously with many partners) and ‘date’ hubs (which interact with different partners in different conditions) <a 
href="#XHan2004">Han <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XHan2004">2004</a>). Network hubs can also be classed as associative or dissociative depending on whether they tend toward or away from connecting directly to other network hubs (<a 
href="#XvanSteen2010">van Steen</a>, <a 
href="#XvanSteen2010">2010</a>). The associative and dissociative properties can also be used to test whether nodes of a particular subgroup (e.g., gene function) associate with each other.
</p>
<!--l. 265-->
<p class="indent">
<a 
href="#XBarabasi1999">Barabási and Albert</a> (<a 
href="#XBarabasi1999">1999</a>) constructed a network model in an entirely different way to randomly generate scale-free networks which have a power law degree distribution. They constructed random networks by preferential attachment, modelling growth of a network by sequentially adding nodes with links to existing nodes. The scale-free nature of the random networks was ensured by adding new nodes with an increasing probability of attachment to an existing node if it has higher degree. These networks successfully capture the scale-free nature of many real world networks with short characteristic path length and low eccentricity resulting in super small worlds <a 
href="#XBarabasi1999">Barabási and Albert</a> (<a 
href="#XBarabasi1999">1999</a>). Scale-free networks are limited by a low clustering coefficient and lack of modular structure; however, they have enabled the study of scale-free network topology and served as a basis for modified scale-free models (<a 
href="#XDorogovtsev2003">Dorogovtsev and Mendes</a>, <a 
href="#XDorogovtsev2003">2003</a>; <a 
href="#XHolme2002">Holme and Kim</a>, <a 
href="#XHolme2002">2002</a>).
</p>
<!--l. 267-->
<p class="indent">
<p><a 
href="#XHan2004">Han <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XHan2004">2004</a>) observed dynamic modularity in biological networks and suggested the network structure may underpin genetic robustness and plasticity. They focus on network hubs which are more likely to be essential genes and define the subgroups of hubs based on correlation of gene expression with protein-protein interaction partners: ‘party’ hubs (which interact simultaneously with many partners) and ‘date’ hubs (which interact with different partners in different conditions). Party</p>
and date hubs occurred most frequently within and between network modules respectively. Party hubs were considered local regulators, whereas date hubs were considered important to network connectivity as global regulators. This distinction between classes of network hubs was supported by differences in tissue specificity and clinical relevance as a proposed predictor of clinical outcome in breast cancer with an <a 
href="#glo:AUROC">area under the reciever operating characteristic (AUROC)</a> of 0.784 <a 
href="#XTaylor2009">Taylor <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTaylor2009">2009</a>). However, correlation between expression and protein interactions were not robustly reproduced. The importance of date hubs has been criticised for assuming a bimodal distribution and basing the global importance of data hubs on a small subset <a 
href="#XAgarwal2010">Agarwal <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XAgarwal2010">2010</a>). As an alternative interpretation, (<a 
href="#XAgarwal2010">Agarwal <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XAgarwal2010">2010</a>) suggest the importance of interactions rather than network hubs as interactions important to the network were between functionally similar proteins. Network hubs can also be classed as associative or dissociative depending on whether they tend toward or away from connecting directly to other network hubs (van Steen 2010). The associative and dissociative properties can also be used to test whether nodes of a particular subgroup (e.g., gene function) associate with each other.
</p>
<!--l. 269-->
<p class="indent">
<p>Applications of network theory are diverse, including uses in social sciences, engineering, and computer science. Due to their complexity and difficulty of gathering sufficient empirical data, biological applications of network theory are relatively unexplored. High-throughput technologies such as siRNA screens, two-hybrid screens, microarrays and massively parallel sequencing have made generating genome-scale molecular data feasible and enabled analysis of biological networks at the molecular level. Many types of inter-related molecular networks can be constructed and analysed, depending on the biological application. Genetic interaction networks will be the focus of this project because they are relatively unexplored compared to other molecular networks, have potential for applications in drug discovery (particularly cancer treatment), and may lead to better understanding of the role of genetics in cellular function and disease. Genetic interactions are usually studied at a high-throughput scale in simple model organisms such as bacteria, yeasts or the nematode worm; studies in humans, mammals, and non-model organisms (where applications would have the most societal impact) are limited by cost, time and labour constraints. Computational approaches with effective predictive models are the only feasible approach to study the connectivity of a biological network in a complex metazoan cell at the genome-scale.</p>
<a 
 id="x1-32001r33"></a>
</p>
<a 
 id="x1-330001"></a>Network Medicine, and Polypharmacology <!--l. 273-->
<p class="indent">
Molecular networks are biological networks consisting of biological molecules including genes, transcripts (with non-coding and microRNAs), or proteins related by known interactions and gene regulatory or metabolic pathways. Targeted therapeutics have had some success for drug discovery, particularly in anticancer applications, including exploiting these molecular networks by designing combination therapies and applying a network pharmacology framework <a 
href="#XHopkins2008">Hopkins</a> (<a 
href="#XHopkins2008">2008</a>). Rational design of drugs selective to a single target has often failed to deliver clinical efficacy. Many existing effective drugs modulate multiple proteins, having been selected for biological effects or clinical outcome rather than molecular targets. Proponents of network biology and polypharmacology (specific binding to multiple targets) recommend developing drugs with a desired target profile designed for the target topology <a 
href="#XBarabasi2004">Barabási and Oltvai</a> (<a 
href="#XBarabasi2004">2004</a>); <a 
href="#XHopkins2008">Hopkins</a> (<a 
href="#XHopkins2008">2008</a>). Multi-target treatments aim to achieve a clinical outcome through modulation of molecular networks since the genetic robustness of a cell often compensates for loss of a single molecular target.
</p>
<!--l. 275-->
<p class="indent">
While multi-target drugs may be more difficult to design, they are faster to test clinically than drug combinations which are usually required to be tested separately first <a 
href="#XHopkins2008">Hopkins</a> (<a 
href="#XHopkins2008">2008</a>). <a 
href="#glo:synthetic lethal">Synthetic lethal</a> treatments for cancer, drug combinations and multi-target drugs to combat resistance to chemotherapy and antibiotics can be informed by biological networks <a 
href="#XBarabasi2004">Barabási and Oltvai</a> (<a 
href="#XBarabasi2004">2004</a>); <a 
href="#XHopkins2008">Hopkins</a> (<a 
href="#XHopkins2008">2008</a>). Further optimisation of timing and dosing of drug combinations may increase efficacy of treatments with low efficacy applied separately. Low doses and drug holidays are other counter intuitive approaches which may increase clinical efficacy, reduce adverse effects, and reduce drug resistance (<a 
href="#XSun2014">Sun <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XSun2014">2014</a>; <a 
href="#XTsai2012">Tsai <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTsai2012">2012</a>).
</p>
<!--l. 277-->
<p class="indent">
<p>A molecular map of the interactions and pathways in the mammalian cellular network has the potential to impact upon drug design and clinical practice, particularly in treatment of cancer and infectious disease. Characterisation of the target system and impact of existing treatments, such as <span class="cmti-12">BRAF</span><sup class="textsuperscript"><span class="cmr-10x-x-109">V600E</span></sup> and <span class="cmti-12">EGFR </span>inhibitors, enable wider application of the mechanisms for such interventions exploiting genetic interactions or pathways. This could lead to development of more effective treatment interventions for these systems and prediction of similar molecular systems for development of novel drug targets and combinations. <a 
 id="x1-33001r9"></a></p>
</p>
<h3 class="sectionHead">
<span class="titlemark">1.2 </span> <a 
 id="x1-340002"></a>A Synthetic Lethal Approach to Cancer Medicine
</h3>
<!--l. 285-->
<p class="noindent">
<a 
href="#glo:synthetic lethal">Synthetic lethality</a> has vast potential to improve cancer medicine by expanding application of targeted therapeutics to include inactivation of tumour suppressors and genes that are difficult to target directly. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> interactions are also studied for gene function and drug mode-of-action in model organisms. This section introduces the concept of <a 
href="#glo:synthetic lethal">synthetic lethality</a> as it was originally conceived and how it has been adopted conceptually in cancer research. Detecting these interactions at scale and interpreting them is the focus of this thesis, hence we start with an overview of the concepts involved, initial work on the interaction, and the rationale for applications to cancer. Specific investigations into <a 
href="#glo:synthetic lethal">synthetic lethality</a> in cancer, detection by experimental screening, and prediction by computational analysis will then be reviewed. <a 
 id="x1-34001r28"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.2.1 </span> <a 
 id="x1-350001"></a>Synthetic Lethal Genetic Interactions
</h4>
<!--l. 294-->
<p class="noindent">
Genetic interactions are a core concept of molecular biology, discovered among earliest investigations of Mendelian genetics, and receiving revived interest with new technologies and potential applications. Biological epistasis is the effect of an allele at one locus “masking” the phenotype of another locus (<a 
href="#XBateson1909">Bateson and Mendel</a>, <a 
href="#XBateson1909">1909</a>). Statistical epistasis is where there is significant disparity between the observed and expected phenotype of a double mutant, compared to the respective phenotypes of single mutants and the wild-type (<a 
href="#XFisher1919">Fisher</a>, <a 
href="#XFisher1919">1919</a>). Fisher’s definition lends itself to quantitative traits and more broadly encompasses synthetic genetic interactions (<a 
href="#glo:SGI">synthetic genetic interactions</a>). These have become popular for studies in yeast genetics and cancer drug design (<a 
href="#XBoone2007">Boone <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoone2007">2007</a>; <a 
href="#XKaelin2005">Kaelin, Jr</a>, <a 
href="#XKaelin2005">2005</a>).
</p>
<!--l. 297-->
<p class="indent">
<p>Synthetic genetic interactions are substantial deviations of growth or viability from the expected null mutant phenotype (of an organism or cell) assuming additive (deleterious) effects of the single mutants. The double mutant does not necessarily have either single mutant phenotype (as shown for cellular growth phenotypes in Figure <a 
href="#x1-35001r1">1.1<!--tex4ht:ref: fig:Costanzo2011 --></a>). Most <a 
href="#glo:SGI">SGIs</a> are more viable than either single mutant or less viable than the expected double mutant. Mutations are “synergistic” in negative <a 
href="#glo:SGI">SGIs</a> with more deviation from the wild-type than expected. Formally, “synthetic sick” (SSL) and “synthetic lethal” (SL) interactions are negative <a 
href="#glo:SGI">SGIs</a> giving growth inhibition and inviability respectively. <a 
href="#glo:synthetic lethal">Synthetic lethality</a> in cancer research more broadly describes any negative <a 
href="#glo:SGI">SGI</a></p>
with specific inhibition of a mutant cell, including SSL interactions. Mutations are “alleviating” in positive <a 
href="#glo:SGI">SGIs</a> with less deviation from the wild-type than expected. For viability, “suppression” and “rescue” are positive <a 
href="#glo:SGI">SGIs</a> giving at least partial restoration of wild-type growth from single mutants with growth impairment and lethal phenotypes respectively. Negative <a 
href="#glo:SGI">SGIs</a> were markedly more common than positive <a 
href="#glo:SGI">SGIs</a> in a number of studies in model systems <a 
href="#XBoucher2013">Boucher and Jenna</a> (<a 
href="#XBoucher2013">2013</a>); <a 
href="#XTong2004">Tong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTong2004">2004</a>).
</p>
<hr class="figure" />
<div class="figure">
<p><a 
 id="x1-35001r1"></a></p>
<hr />
____________________________________________________________________________________      <!--l. 301-->
<p class="noindent">
<img 
src="Costanzo_2011_SL_mega_no_border.png" alt="pict"  
 /> <br />
</p>
<div class="caption">
<p><span class="id">Figure 1.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic genetic interactions.</span></strong> <span class="cmr-10x-x-109">Impact of various negative and positive</span> <a 
href="#glo:SGI"><span class="cmr-10x-x-109">SGIs</span></a><span class="cmr-10x-x-109">: negative interactions involve deleterious (sick) or inviable (lethal) phenotypess</span> <span class="cmr-10x-x-109">whereas positive interactions involve restoring viability by masking or suppressing the</span> <span class="cmr-10x-x-109">other mutation or complete rescue of the wildtype phenotype. Figure adapted from</span> <span class="cmr-10x-x-109">(</span><a 
href="#XCostanzo2011"><span class="cmr-10x-x-109">Costanzo </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109">,</span><span class="cmr-10x-x-109"> </span><a 
href="#XCostanzo2011"><span class="cmr-10x-x-109">2011</span></a><span class="cmr-10x-x-109">) concerning growth viability fitness in yeast.</span></span></p>
</div>
<!--tex4ht:label?: x1-35001r1 -->
</div>
<hr class="endfigure" />
<a 
 id="x1-35002r37"></a>
<h4 class="subsectionHead">
<span class="titlemark">1.2.2 </span> <a 
 id="x1-360002"></a>Synthetic Lethal Concepts in Genetics
</h4>
<!--l. 312-->
<p class="noindent">
<a 
href="#glo:synthetic lethal">Synthetic lethal</a> genes are generally regarded to arise due to functional redundancy. Due to the functional level of <a 
href="#glo:SGI">SGIs</a>, <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes do not need directly interact, nor be expressed in the same cell or at the same developmental stage: serving related functions is sufficient to affect cell (or organism) viability and be relevant to drug-mode-of-action cancer biology. Combined loss of genes performing an essential or important function in a cell are therefore deleterious. <a 
href="#glo:synthetic lethal">Synthetic lethal</a> gene pairs are therefore pairwise essential with “induced essentiality”: each <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene becomes essential to the cell upon loss of the other.
</p>
<!--l. 314-->
<p class="indent">
Since <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene partners can be affected by extracellular stimuli such as chemicals, essentiality of <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes can be induced by the environment of a cell. An environmental stress condition may inhibit one or the other <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene, such as exposure to chemicals, in which case the <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner gene is “conditionally essential” (<a 
href="#XHillenmeyer2008">Hillenmeyer</a>, <a 
href="#XHillenmeyer2008">2008</a>). Thus the evolutionary rationale for the abundance of <a 
href="#glo:SGI">SGIs</a> (compared to the surprisingly low number of essential genes) in a Eukaryotic genome attributed to genetic functional redundancy and network robustness of a cell which are advantageous to survival.
</p>
<!--l. 316-->
<p class="indent">
Biological functions are typically performed by a pathway of genes (or their products). Many genes of the same pathway may be fucntionally interchangable as <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of a particular gene since loss of the pathway is deleterious without the <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner gene. Therefore biological pathways can be subject to induced essentiality under loss of a gene and <a 
href="#glo:synthetic lethal">synthetic lethality</a> may occur at pathway level or in a gene regulation network. <a 
 id="x1-36001r39"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.2.3 </span> <a 
 id="x1-370003"></a>Studies of Synthetic Lethality
</h4>
<!--l. 319-->
<p class="noindent">
<p>Genetic high-throughput screens have identified unexpected, functionally informative, and clinically relevant <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions; including <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of genes recurrently mutated in cancer or attributed to familial early-onset cancers. While screening presents an appealing strategy for <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery, computational approaches are becoming popular as an alternative or complement to experimental methods to overcome inherent bias and limitations of experimental screens. An array of recently</p>
developed computational methods (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>; <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>; <a 
href="#XTiong2014">Tiong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTiong2014">2014</a>; <a 
href="#XWang2013">Wang and Simon</a>, <a 
href="#XWang2013">2013</a>; <a 
href="#XWappett2014">Wappett</a>, <a 
href="#XWappett2014">2014</a>) show the need for <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery in the fundamental genetics and translational cancer research community. However, existing computational methods are not suitable for queries of genomic data for interacting partners of a particular gene: they have been applied pairwise across the genome, do not have software released to apply the methodology, or lack statistical measures of error for further analysis. A robust prediction of gene interactions is an effective and practical approach at a scale of the entire genome for ideal translational applications, analysis of biological systems, and constructing functional gene networks. <a 
 id="x1-37001r34"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.3.1 </span> <a 
 id="x1-380001"></a>Synthetic Lethal Pathways and Networks
</h5>
<!--l. 322-->
<p class="noindent">
<a 
href="#glo:SGI">SGIs</a> are very common in genomes, with 4<span class="cmsy-10x-x-120">× </span>more interactions detected with synthetic gene array mating screens than protein-protein interactions yeast-2-hybrid studies (<a 
href="#XTong2004">Tong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTong2004">2004</a>). The <a 
href="#glo:SGI">SGI</a> network is scale-free with power-law vertex degree distribution and low average shortest path length (3.3) as expected for a complex biological network (<a 
href="#XBarabasi2004">Barabási and Oltvai</a>, <a 
href="#XBarabasi2004">2004</a>). Highly connected “hub” genes with the highest number of links (vertex degree) are functionally important with many negative <a 
href="#glo:SGI">SGI</a> hubs involved in cell cycle regulation and many positive <a 
href="#glo:SGI">SGI</a> hubs involved in translation (<a 
href="#XBaryshnikova2010b">Baryshnikova <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBaryshnikova2010b">2010b</a>; <a 
href="#XCostanzo2010">Costanzo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCostanzo2010">2010</a>). Negative <a 
href="#glo:SGI">SGIs</a> were far more common than positive <a 
href="#glo:SGI">SGIs</a>, with synthetic gene loss being more likely to be deleterious to cell than advantageous which indicates than <a 
href="#glo:synthetic lethal">synthetic lethality</a> may be comparably easier to detect than other <a 
href="#glo:SGI">SGIs</a>.
</p>
<!--l. 324-->
<p class="indent">
Essential pathways are highly buffered with 5<span class="cmsy-10x-x-120">× </span>more interactions than other <a 
href="#glo:SGI">SGIs</a>, consistent with strong selection for survival, as found with conditional and partial mutations in essential genes (<a 
href="#XDavierwala2005">Davierwala <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDavierwala2005">2005</a>). This <a 
href="#glo:SGI">SGI</a> network had scale-free topology and rarely shared interactions with the protein-protein interaction network. These networks are related by an “orthogonal” relationship: shared partners in one network tend to be themselves connected directly in the other network. Essential genes were likely to have closely related functions, whereas non-essential networks were relatively more inclined to have <a 
href="#glo:SGI">SGIs</a> between distinct biological pathways. <a 
 id="x1-38001r35"></a>
</p>
<a 
 id="x1-390001"></a>Evolution of Synthetic Lethality <!--l. 327-->
<p class="indent">
There is poor conservation of specific <a 
href="#glo:SGI">SGIs</a> between <span class="cmti-12">S. cerevisiae </span>and <span class="cmti-12">S. pombe </span>with 29% of the interactions tested in both distantly related species being conserved between them (<a 
href="#XDixon2008">Dixon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDixon2008">2008</a>). The remaining interactions show high species-specific differences; however, many of the species-specific interactions were still conserved between biological pathways, protein complexes, or protein-protein interaction modules. Similarly, conservation of pathway redundancy was also found between Eukaryotes (<span class="cmti-12">S.</span> <span class="cmti-12">cerevisiae</span>) and prokaryotes (<span class="cmti-12">E. coli</span>) (<a 
href="#XButland2008">Butland <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XButland2008">2008</a>). Negative <a 
href="#glo:SGI">SGIs</a> were more likely to be conserved between biological pathways, whereas positive <a 
href="#glo:SGI">SGIs</a> were more likely to be conserved within a pathway or protein complex (<a 
href="#XRoguev2008">Roguev <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XRoguev2008">2008</a>).
</p>
<!--l. 329-->
<p class="indent">
A modest 5% of interactions were conserved between unicellular (<span class="cmti-12">S. cerevisiae</span>) and multicellular (<span class="cmti-12">C. elegans</span>) organisms. However, the nematode <a 
href="#glo:SGI">SGI</a> network had similar scale-free topology and modularity despite differences in methodology: metazoan <a 
href="#glo:RNAi">RNAi</a> screens are incomplete knockouts whereas screening null mutations is feasible in yeast (<a 
href="#XBussey2006">Bussey <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBussey2006">2006</a>). The nematode <a 
href="#glo:SGI">SGI</a> screen identified network hubs with important interactions to orthologues of known human disease genes (<a 
href="#XLehner2006">Lehner <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLehner2006">2006</a>). Despite the lack of direct conservation of <a 
href="#glo:SGI">SGIs</a> between yeasts and nematode worms, genetic redundancy at the gene or pathway level may yet be consistent with an induced essentiality model of <a 
href="#glo:SGI">SGIs</a> where gene functions are conserved with network restructuring over evolutionary change (<a 
href="#XTischler2008">Tischler <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTischler2008">2008</a>). While nematode models are more closely related to human cells, cancer cells can present growth and viability phenotypes more comparable to yeast models. Therefore findings from both SGA and <a 
href="#glo:RNAi">RNAi</a> models are relevant to understanding cellular network structure and in healthy and cancerous human cells. <a 
href="#glo:RNAi">RNAi</a> has also been applied to human and mouse cancer cells in cell culture and genetic screening experiments. These findings suggest that <a 
href="#glo:SGI">SGI</a> network “rewiring” is a concern for identifying specific <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions in cancer and a pathway approach may be more robust in the context of evolution, patient variation, tumour heterogeneity, and disease progression. <a 
 id="x1-39001r40"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.2.4 </span> <a 
 id="x1-400004"></a>Synthetic Lethal Concepts in Cancer
</h4>
<!--l. 333-->
<p class="noindent">
Loss of function occurs in many genes in cancers including tumour suppressors and yet few interventions target such mutations compared to targeted therapies for gain of function mutation in oncogenes (<a 
href="#XKaelin2005">Kaelin, Jr</a>, <a 
href="#XKaelin2005">2005</a>). <a 
href="#glo:synthetic lethal">Synthetic lethality</a> is a powerful design strategy for therapies selective against loss of gene function with potential for application against a range of genes and diseases (<a 
href="#XFece2015">Fece de la Cruz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFece2015">2015</a>; <a 
href="#XKaelin2009">Kaelin, Jr</a>, <a 
href="#XKaelin2009">2009</a>). Since <a 
href="#glo:synthetic lethal">synthetic lethality</a> affects cellular viability by indirect functional relationships between genes, it is suitable for indirectly targeting mutations in cancers. Once <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of cancer genes are identified, targeted therapeutics can be applied against them. When genes are disrupted in cancers, the induced essentiality of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners is a vulnerability that may be exploited for anti-cancer therapy. This has the potential to be very specific against cancer cells (with the target mutation) over non-cancer cells (with a functional compensating gene). Analogous to “oncogene addiction”, where cancer cells adapt to particular oncogenic growth signals and become reliant on them to remain viable (<a 
href="#XLuo2009">Luo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLuo2009">2009</a>; <a 
href="#XWeinstein2000">Weinstein</a>, <a 
href="#XWeinstein2000">2000</a>), <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of inactivated tumour suppressors are required to maintain cancer cell viability and proliferation. As such cancers are subject to “non-oncogene addiction” and are feasible anti-cancer drug targets.
</p>
<hr class="figure" />
<div class="figure">
<p><a 
 id="x1-40001r2"></a></p>
<hr />
____________________________________________________________________________________      <!--l. 338-->
<p class="noindent">
<img 
src="SL_Concept_no_border.png" alt="PIC"  
 /> <br />
</p>
<div class="caption">
<p><span class="id">Figure 1.2: </span><span class="content"><strong><a 
href="#glo:synthetic lethal"><span class="cmbx-10x-x-109">Synthetic lethality</span></a> <span class="cmbx-10x-x-109">in cancer.</span></strong> <span class="cmr-10x-x-109">Rationale of exploiting </span><a 
href="#glo:synthetic lethal"><span class="cmr-10x-x-109">synthetic lethality</span></a> <span class="cmr-10x-x-109">for specifity against a tumour suppressor gene (e.g., </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">) while other cells are spared</span> <span class="cmr-10x-x-109">under the inhibition of a partner gene.</span></span></p>
</div>
<!--tex4ht:label?: x1-40001r2 -->
</div>
<hr class="endfigure" />
<!--l. 347-->
<p class="indent">
The <a 
href="#glo:synthetic lethal">synthetic lethal</a> approach to cancer medicine is most amenable to loss of function mutations in tumour suppressor genes, where it would feasibly be effective against any loss of function mutation across the tumour suppressor with a viable <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner gene (as shown in Figure <a 
href="#x1-40001r2">1.2<!--tex4ht:ref: fig:SL_Concept --></a>). However, the approach may also be suitable for cases where cancer cells have mutations where the normal function of the gene is disrupted such as if it were overexpression (“synthetic dosage lethality”) or if an oncogene interfered with the function of the proto-oncogenic variant such as competitive inhibition. Thus <a 
href="#glo:synthetic lethal">synthetic lethality</a> expands the range of cancer-specific mutations feasible to target with targeted therapeutics to absence of tumour suppressor genes and distinguishing highly homologous oncogenes by functional differences by targeting their <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners. <a 
 id="x1-40002r43"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.2.5 </span> <a 
 id="x1-410005"></a>Clinical Impact of Synthetic Lethality in Cancer
</h4>
<!--l. 355-->
<p class="noindent">
The <a 
href="#glo:synthetic lethal">synthetic lethal</a> interaction of <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>with <span class="cmti-12">PARP1 </span>in breast cancer is an example of how gene interactions are important in cancer, including translation to the clinic. These genetic interactions enable specific targeting of mutations in <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>tumour suppressor genes with PARP inhibitors by inducing <a 
href="#glo:synthetic lethal">synthetic lethality</a> in breast cancer (<a 
href="#XFarmer2005">Farmer <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFarmer2005">2005</a>). PARP inhibitors are one of the first targeted therapeutics against a tumour suppressor mutation with success in clinical trials.
</p>
<!--l. 357-->
<p class="indent">
<p><span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>and <span class="cmti-12">PARP1 </span>genes demonstrate the application of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> approach to cancer therapy <a 
href="#XAshworth2008">Ashworth</a> (<a 
href="#XAshworth2008">2008</a>); <a 
href="#XKaelin2005">Kaelin, Jr</a> (<a 
href="#XKaelin2005">2005</a>). <span class="cmti-12">BRCA1 </span>and <span class="cmti-12">BRCA2 </span>are homologous DNA repair genes, widely known as tumour suppressors; mutation carriers have substantially increased risk of breast (risk by age 70 of 57% for <span class="cmti-12">BRCA1 </span>and 59% for <span class="cmti-12">BRCA2 </span>) and ovarian cancers (risk by age 70 of 40% for <span class="cmti-12">BRCA1 </span>and 18% for <span class="cmti-12">BRCA2</span>) (<a 
href="#XChen2007">Chen and Parmigiani</a>, <a 
href="#XChen2007">2007</a>). The <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>genes, which usually repair DNA or destroy the cell if it cannot be repaired, have inactivating somatic mutations in some familial and sporadic cancers. Poly-ADP-ribose polymerase (PARP) genes are tumour suppressor genes involved in base excision DNA repair. Loss of PARP activity results in single-stranded DNA breaks. However, <span class="cmti-12">PARP1</span><sup><span class="cmsy-8">-</span><span class="cmmi-8">∕</span><span class="cmsy-8">-</span></sup> knockout mice are viable and healthy indicating low toxicity from PARP inhibition (<a 
href="#XBryant2005">Bryant <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBryant2005">2005</a>).</p>
</p>
<!--l. 359-->
<p class="indent">
<a 
href="#XBryant2005">Bryant <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XBryant2005">2005</a>) showed that <span class="cmti-12">BRCA2 </span>cells were sensitive to PARP inhibition by siRNA of <span class="cmti-12">PARP1 </span>or drug inhibition (which targets <span class="cmti-12">PARP1 </span>and <span class="cmti-12">PARP2</span>) using Chinese hamster ovary cells, MCF7 and MDA-MB-231 breast cell lines. This effect was sufficient to kill mouse tumour xenografts and showed high specificity to <span class="cmti-12">BRCA2 </span>deficient cells in culture and xenografts. <a 
href="#XFarmer2005">Farmer <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XFarmer2005">2005</a>) replicated these results in embryonic stem cells and showed that <span class="cmti-12">BRCA1 </span>cells were also sensitive to PARP inhibition relative to the wild-type with siRNA and drug experiments in cell culture and drug activity against <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>deficient embryonic stem cell mouse xenografts. They found evidence that PARP inhibition causes DNA lesions, usually repaired in wild-type cells, which lead to chromosomal instability, cell cycle arrest, and induction of apoptosis in <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>deficient cells. Therefore, the pathways cooperate to repair DNA giving a plausible mechanism for combined loss as an effective anti-cancer treatment.
</p>
<!--l. 361-->
<p class="indent">
Thus PARP inhibitors have potential for clinical use against <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2</span> mutations in hereditary and sporadic cancers (Ashworth 2008; Kaelin2005). PARP inhibition has been found to be effective in cancer patients carrying <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>mutations and some other ovarian cancers, suggesting <a 
href="#glo:synthetic lethal">synthetic lethality</a> between PARP and other DNA repair pathways (<a 
href="#XStrom2012">Ström and Helleday</a>, <a 
href="#XStrom2012">2012</a>). This supports the potential for PARP inhibition as a chemo-preventative alternative to prophylactic surgery for high risk individuals with <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>mutations (<a 
href="#XStrom2012">Ström and Helleday</a>, <a 
href="#XStrom2012">2012</a>). Hormone-based therapy has also been suggested as a chemo-preventative in such high risk individuals and aromatase inhibitors have completed phase I clinical trials for this purpose (Bozovic-Spasojevic2012). <a 
href="#XStrom2012">Ström and Helleday</a> (<a 
href="#XStrom2012">2012</a>) also postulate increased efficacy of PARP inhibitors in the hypoxic DNA-damaging tumour micro-environment.
</p>
<!--l. 363-->
<p class="indent">
<p>A PARP inhibitor, olaparib, showed fewer adverse effects than cytotoxic chemotherapy and anti-tumour activity in phase I trials against <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>deficient familial breast, ovarian, and prostate cancers (<a 
href="#XFong2009">Fong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFong2009">2009</a>) and sporadic ovarian cancer (<a 
href="#XFong2010">Fong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFong2010">2010</a>). AstraZeneca has reported phase II trials showing the treatment is effective in <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>deficient breast (<a 
href="#XTutt2010">Tutt <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTutt2010">2010</a>) and ovarian cancers (<a 
href="#XAudeh2010">Audeh <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XAudeh2010">2010</a>) with a favourable therapeutic window and similar toxicity between carriers of <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>mutations and sporadic cases. AstraZeneca announced that olaparib has begun phase III trials for breast and ovarian cancers in 2013. Mixed results in phase II trials in ovarian cancer are behind the delays addressed by retrospective analysis of the cohort subgroup with confirmed mutation of <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>genes in the tumour;</p>
unsurprisingly these patients, benefit most from the PARP inhibitor treatment and have increased platinum sensitivity in combination treatment. These PARP inhibitors are FDA approved for some cancers <a 
href="#XMcLachlan2016">McLachlan <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XMcLachlan2016">2016</a>), are effective against germline and sporadic <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>mutations, and are a potential prevention alternative to prophylactic surgery for high risk mutation carriers <a 
href="#XStrom2012">Ström and Helleday</a> (<a 
href="#XStrom2012">2012</a>).
</p>
<!--l. 365-->
<p class="indent">
This demonstrates the clinical impact of a well characterised system of <a 
href="#glo:synthetic lethal">synthetic lethality</a> with known cancer risk genes. <a 
href="#glo:synthetic lethal">Synthetic lethality</a> has the benefit of being effective against inactivation of tumour suppressor genes by any means, broader than targeting a particular oncogenic mutation (<a 
href="#XKaelin2005">Kaelin, Jr</a>, <a 
href="#XKaelin2005">2005</a>). The targeted therapy is effective in both sporadic and hereditary <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>deficient tumours acting against an oncogenic molecular aberration across several tissues. <a 
 id="x1-41001r45"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.2.6 </span> <a 
 id="x1-420006"></a>High-throughput Screening for Synthetic Lethality
</h4>
<!--l. 382-->
<p class="noindent">
The function of signalling pathways and combinations of interacting genes are important in cancer research but classical genetics approaches have been limited to non-redundant pathways (<a 
href="#XFraser2004">Fraser</a>, <a 
href="#XFraser2004">2004</a>). The emerging <a 
href="#glo:RNAi">RNAi</a> technologies have vastly expanded the potential for studying genetic redundancy in mammalian experimental models including testing experimentally for <a 
href="#glo:synthetic lethal">synthetic lethality</a> (<a 
href="#XFraser2004">Fraser</a>, <a 
href="#XFraser2004">2004</a>). Identifying <a 
href="#glo:synthetic lethal">synthetic lethality</a> is crucial to study gene function, drug mechanisms, and design novel therapies (<a 
href="#XLum2004">Lum <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLum2004">2004</a>). Candidate selection of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs relevant to cancer has shown some success but is limited because interactions are difficult to predict; they can occur between seemingly unrelated pathways in model organisms (<a 
href="#XCostanzo2011">Costanzo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCostanzo2011">2011</a>). While biologically informed hypotheses have had some success in <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery (<a 
href="#XBitler2015">Bitler <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBitler2015">2015</a>; <a 
href="#XBryant2005">Bryant <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBryant2005">2005</a>; <a 
href="#XFarmer2005">Farmer <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFarmer2005">2005</a>), interactions occurring indirectly between distinct pathways would be missed (<a 
href="#XBoone2007">Boone <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoone2007">2007</a>; <a 
href="#XCostanzo2011">Costanzo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCostanzo2011">2011</a>). Scanning the entire genome for interactions against a clinically relevant gene is an emerging strategy being explored with high-throughput screens (<a 
href="#XFece2015">Fece de la Cruz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFece2015">2015</a>) and computational approaches (<a 
href="#XBoucher2013">Boucher and Jenna</a>, <a 
href="#XBoucher2013">2013</a>; <a 
href="#XvanSteen2011">van Steen</a>, <a 
href="#XvanSteen2011">2012</a>).
</p>
<!--l. 384-->
<p class="indent">
<p>Experimental screening for <a 
href="#glo:synthetic lethal">synthetic lethality</a> is an appealing strategy for wider discovery of functional interactions <span class="cmti-12">in vivo </span>despite many potential sources of error which must be considered. The <a 
href="#glo:synthetic lethal">synthetic lethal</a> concept has both genetic and pharmacological screening applications to cancer research. Genetic screens, with <a 
href="#glo:RNAi">RNAi</a> to discover the specific genes</p>
involved, inform development of targeted therapies with a known mode of action, anticipated mechanisms of resistance, and biomarkers for treatment response. <a 
href="#glo:RNAi">RNAi</a> is a transient knockdown of gene expression more similar to the effect of drugs than complete gene loss and makes comparison to screens in model organisms difficult (<a 
href="#XBussey2006">Bussey <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBussey2006">2006</a>). The <a 
href="#glo:RNAi">RNAi</a> gene knockdown process has inherent toxicity to some cells, potential off-target effects, and issues with a high false positive rate. Therefore, it is important to validate any candidates in a secondary screen and replicate knockdown experiments with a number of independent shRNAs. Alternative gene knockout procedures have also been proposed for <a 
href="#glo:synthetic lethal">synthetic lethal</a> screening including a genome-wide application of the CRISPR/Cas9/sgRNA genome editing technology (<a 
href="#XSander2014">Sander and Joung</a>, <a 
href="#XSander2014">2014</a>), episomal gene transfer (<a 
href="#XVargas2004">Vargas <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XVargas2004">2004</a>), or <a 
href="#glo:RNAi">RNAi</a> with lentiviral transfection for delivery of shRNA (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). Genetic screens have potential for quantitative gene disruption experiments to selectively target overexpressed genes in cancer via synthetic dosage lethality. While powerful for understanding fundamental cellular function, analysis of isogenic cell lines is inherently limited by assuming only a single mutation differs between them despite susceptibility to “genetic drift” and cannot account for diverse genetic backgrounds or tumour heterogeneity (<a 
href="#XFece2015">Fece de la Cruz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFece2015">2015</a>). Genetic screens thus identify targets to develop or repurpose targeted therapies for disease but alone will not directly identify a lead compound to develop for the market or clinical translation.
</p>
<!--l. 386-->
<p class="indent">
<p>Chemical screens are immediately applicable to the clinic by directly screening for selective lead compounds with suitable pharmacological properties. However chemical screens lack a known mode of action, may affect many targets, and screen a narrow range of genes with existing drugs. With either approach there are many challenges translating candidates into the clinic such as finding targets relevant to a range of patients, validation of targets, accounting for a range of genetic (and epigenetic) contexts or tumour micro-environment, identifying effective synergistic combinations, enhancers of existing radiation or cytotoxic treatments, avoiding inherent or acquired drug resistance, and developing biomarkers for patients which will respond to <a 
href="#glo:synthetic lethal">synthetic lethal</a> treatment, including integrating these into clinical trials and clinical practice. Identifying specific target genes is an effective way to anticipate such challenges, which can be approached with genetic screens, so we will focus on these and computational alternatives. Screening methods have proven a fruitful area of research, despite being costly, laborious, and having many different sources of error. These limitations</p>
suggest a need for complementary computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery. <a 
 id="x1-42001r41"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.6.1 </span> <a 
 id="x1-430001"></a>Synthetic Lethal Screens
</h5>
<!--l. 390-->
<p class="noindent">
Overexpression of genes is another suitable application for <a 
href="#glo:synthetic lethal">synthetic lethality</a> since overexpressed genes cannot be distinguished from the wild-type by direct sequence specific targeted therapy. Overexpression of oncogenes, such as <span class="cmti-12">EGFR</span>, <span class="cmti-12">MYC</span>, and <span class="cmti-12">PIM1</span>, has been found to drive many cancers. <span class="cmti-12">PIM1 </span>is a candidate for <a 
href="#glo:synthetic lethal">synthetic lethal</a> drug design in lymphomas and prostate cancers, where it interacts with <span class="cmti-12">MYC </span>to drive cancer growth. <a 
href="#XvanderMeer2014">van der Meer <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XvanderMeer2014">2014</a>) performed an <a 
href="#glo:RNAi">RNAi</a> screen for <a 
href="#glo:synthetic lethal">synthetic lethality</a> between <span class="cmti-12">PIM1 </span>overexpression and gene knockdown in RWPE prostate cancer cell lines. <span class="cmti-12">PLK1 </span>gene knockdown and drug inhibition was effective as a specific inhibitor of <span class="cmti-12">PIM1 </span>overexpressing prostate cells in cell culture and mouse tumour xenografts. <span class="cmti-12">PLK1 </span>inhibition reduced <span class="cmti-12">MYC </span>expression in pre-clinical models, consistent with expression in human tumours in which <span class="cmti-12">PIM1 </span>and <span class="cmti-12">PLK1</span> are co-expressed and correlated with tumour grade. Thus <a 
href="#glo:RNAi">RNAi</a> screening was valuable to identify therapeutic targets and biomarkers for patient response as demonstrated with the finding of <span class="cmti-12">PLK1 </span>as a candidate drug target against prostate cancer progression.
</p>
<!--l. 392-->
<p class="indent">
<a 
href="#glo:HLRCC">Hereditary leiomyomatosis and renal cell carcinoma (HLRCC)</a> is a cancer syndrome of predisposition to benign tumours in the uterus and risk of malignant cancer of the kidney attributed to inherited mutations in fumarate hydratase (<span class="cmti-12">FH</span>). <a 
href="#XBoettcher2014">Boettcher <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XBoettcher2014">2014</a>) performed an <a 
href="#glo:RNAi">RNAi</a> screen on HEK293T renal cells for <a 
href="#glo:synthetic lethal">synthetic lethality</a> with <span class="cmti-12">FH</span>. They found enrichment of haem metabolism (consistent with the literature) and adenylate cyclase pathways (consistent with cAMP dysregulation in <span class="cmti-12">FH </span>mutant cells). <a 
href="#glo:synthetic lethal">Synthetic lethality</a> between <span class="cmti-12">FH </span>mutation and adenylate cyclases was validated with gene knockdown, drug experiments, and replicated across both HEK293T renal cells and VOK262 cells derived from a HLRCC patient, suggesting new potential treatments against the disease.
</p>
<!--l. 394-->
<p class="indent">
<p>Similarly, <a 
href="#glo:HDGC">hereditary diffuse gastric cancer (HDGC)</a> is a cancer syndrome of predisposition to early-onset malignant stomach and breast cancers attributed to mutations in <a 
href="#glo:E-cadherin">E-cadherin</a> (<span class="cmti-12">CDH1</span>). <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) performed an <a 
href="#glo:RNAi">RNAi</a> screen on MCF10A breast cells for <a 
href="#glo:synthetic lethal">synthetic lethality</a> with <span class="cmti-12">CDH1</span>. They found enrichment of G-protein coupled receptors (GPCRs) and cytoskeletal gene functions. The results were consistent with a</p>
concurrent drug compound screen with a number of candidates validated by lentiviral shRNA gene knockdown and drug testing including inhibitors of Janus kinase, histone deacetylases, phosphoinositide 3-kinase, aurora kinase, and tyrosine kinases. Therefore the <a 
href="#glo:synthetic lethal">synthetic lethal</a> strategy has potential for clinical impact against <a 
href="#glo:HDGC">HDGC</a>, with particular interest in interventions with low adverse effects for chemo-prevention, including repurposing existing approved drugs for activity against <span class="cmti-12">CDH1 </span>deficient cancers.
</p>
<!--l. 396-->
<p class="indent">
<a 
href="#glo:RNAi">RNAi</a> screening for <a 
href="#glo:synthetic lethal">synthetic lethality</a> is also useful for functional genetics to understand drug sensitivity. <a 
href="#XAarts2015">Aarts <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XAarts2015">2015</a>) screened WiDr colorectal cells for <a 
href="#glo:synthetic lethal">synthetic lethality</a> between <span class="cmti-12">WEE1 </span>inhibitor treatment and an <a 
href="#glo:RNAi">RNAi</a> library of 1206 genes with functions known to be amenable to drug treatment or important in cancer such as kinases, phosphatases, tumour suppressors, and DNA repair (a pathway <span class="cmti-12">WEE1 </span>regulates). Screening identified a number of <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates including genes involved in cell cycle regulation, DNA replication, repair, homologous recombination, and Fanconi anaemia. <a 
href="#glo:synthetic lethal">Synthetic lethality</a> with cell-cycle and DNA repair genes was consistent with the literature and validation in a panel of breast and colorectal cell lines supported checkpoint kinases, Fanconi anaemia, and homologous recombination as <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">WEE1</span>. These results show that <a 
href="#glo:synthetic lethal">synthetic lethality</a> can be used to improve drug sensitivity as a combination treatment, especially to exploit genomic instability and DNA repair, which are known to be clinically applicable from previous results with <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2</span> genes and PARP inhibitors (<a 
href="#XLord2014">Lord <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLord2014">2015</a>). Therefore, <span class="cmti-12">WEE1 </span>inhibitors are an example of treatment which could be repurposed with the <a 
href="#glo:synthetic lethal">synthetic lethal</a> strategy. Similar findings would be valuable to clinicians as a source of biomarkers and novel treatments. While using a panel of cell lines to replicate findings across genetic background is a promising approach to ensure wide clinical application of validated <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners, a computational approach may be more effective as it could account for wider patient variation than scaling up intensive experiments on a wide array of cell lines and could screen beyond limited candidates from an <a 
href="#glo:RNAi">RNAi</a> library.
</p>
<!--l. 398-->
<p class="indent">
<p>Chemical genetic screens are also a viable strategy to identify therapeutically relevant <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions. <a 
href="#XBitler2015">Bitler <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XBitler2015">2015</a>) investigated <span class="cmti-12">ARID1A</span> mutations, aberrations in chromatin remodelling known to be common in ovarian cancers, for drug response. Ovarian RMG1 cells were screened for drug response specific to <span class="cmti-12">ARID1A </span>knockdown cells. They used <span class="cmti-12">ARID1A </span>gene knockdown for</p>
consistent genetic background, with control experiments and 3D cell culture to ensure relevance to drug activity in the tumour micro-environment. Screening a panel of commercially available drugs targeting epigenetic regulators found <span class="cmti-12">ESH2 </span>methyltransferase inhibitors effective and specific against <span class="cmti-12">ARID1A </span>mutation with validation in a panel of ovarian cell lines. <a 
href="#glo:synthetic lethal">Synthetic lethality</a> between <span class="cmti-12">ARID1A </span>and <span class="cmti-12">ESH2 </span>was supported by decreases in H3K27me3 epigenetic marks and markers of apoptosis in response to <span class="cmti-12">ESH2 </span>inhibitors. This was mechanistically supported with differential expression of <span class="cmti-12">PIK3IP1 </span>and association of both <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes with the <span class="cmti-12">PIK3IP1 </span>promoter identifying the PI3K-AKT signalling pathway as disrupted when both genes were inhibited.
</p>
<!--l. 401-->
<p class="indent">
This successfully demonstrates the importance of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in epigenetic regulators, identifies a therapeutically relevant <a 
href="#glo:synthetic lethal">synthetic lethal</a> interaction, and shows that chemical genetic screens could model drug response and combination therapy in cancer cells. However this approach is limited to finding <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions between genes with known similar function, which may not be the most suitable for treatment. Further limiting experiments to genes with existing targeted drugs reduces the number of <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions detected, assumes the specificity of drugs to a particular target, and many of these drugs are not yet clinically available yet anyway, although they are clinical trials for other diseases or limited to access by patients from a particular countries.
</p>
<!--l. 403-->
<p class="indent">
<p>The examples above show that high-throughput screens are an effective approach to discover <a 
href="#glo:synthetic lethal">synthetic lethality</a> in cancer with a wide range of applications. Screens are more comprehensive than hypothesis-driven candidate gene approaches and successfully find known and novel <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions with potential for rapid clinical application. They have the power to test mode of action of drugs, find unexpected <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions between pathways, or identify effective treatment strategies without needing a clear mechanism. However, <a 
href="#glo:synthetic lethal">synthetic lethal</a> screens are costly, labour-intensive, error-prone, and biased towards genes with effective <a 
href="#glo:RNAi">RNAi</a> knockdown libraries. Limited genetic background, lethality to wild-type cell during gene knockdown, off-target effects, and difficultly replicating <a 
href="#glo:synthetic lethal">synthetic lethality</a> across different cell lines, tissues, laboratories, or conditions stems from a high false positive rate and a lack of standardised thresholds to identify <a 
href="#glo:synthetic lethal">synthetic lethality</a> in a high-throughput screen. Therefore there is a need for replication, validation, and alternative approaches to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates. Varied conditions between</p>
experimental screens and differences between <a 
href="#glo:RNAi">RNAi</a> and drug screens renders meta-analysis ineffective.
</p>
<!--l. 405-->
<p class="indent">
Genome-scale <a 
href="#glo:synthetic lethal">synthetic lethal</a> experiments (across gene pairs) are not feasible, even in model organisms, and they typically focus on specific gene candidates or the partners of a gene of interest (such as importance in health). Therefore a computational approach is more suitable for this task and may further augment experimental screening to replicate screen candidates beyond experimental models. <a 
 id="x1-43001r46"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.2.7 </span> <a 
 id="x1-440007"></a>Computational Prediction of Synthetic Lethality
</h4>
<a 
 id="x1-44001r47"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.7.1 </span> <a 
 id="x1-450001"></a>Bioinformatics Approaches to Genetic Interactions
</h5>
<!--l. 431-->
<p class="noindent">
Prediction of gene interaction networks is a feasible alternative to high-throughput screening with biological importance and clinical relevance. There are many existing methods to predict gene networks, as reviewed by <a 
href="#XvanSteen2011">van Steen</a> (<a 
href="#XvanSteen2011">2012</a>) and <a 
href="#XBoucher2013">Boucher and Jenna</a> (<a 
href="#XBoucher2013">2013</a>) and summarised in Table <a 
href="#x1-45001r1">1.1<!--tex4ht:ref: tab:methods_model --></a>. However, many of these methods have limitations including the requirement for existing <a 
href="#glo:SGI">SGI</a> data, several data inputs, and reliability of gene function annotation. Many of the existing methods also assume conservation of individual interactions between species, which has been found not to hold in yeast studies (<a 
href="#XDixon2008">Dixon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDixon2008">2008</a>). Tissue specificity is important in gene regulation and gene expression, which are used as predictors of genetic interaction. However, tissue specificity of genetic interactions cannot be explored in yeast studies and has not been considered in many studies of multicellular model organisms, human networks, or cancers. Similarly, investigation into tissue specificity of protein-protein interactions (PPIs) , an important predictor of genetic interactions, is difficult given that high-throughput two-hybrid screens occur out of cellular context for multicellular organisms.
</p>
<div class="table">


<!--l. 433-->
<p class="indent">
<a 
 id="x1-45001r1"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 1.1: </span><span class="content">Methods for Predicting Genetic Interactions</span></p>
</div>
<!--tex4ht:label?: x1-45001r1 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-5" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-5-1g">
<col 
id="TBL-5-1" />
<col 
id="TBL-5-2" />
<col 
id="TBL-5-3" />
<col 
id="TBL-5-4" />
<col 
id="TBL-5-5" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-5-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-5-1-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><strong><span class="cmbx-12">Method</span></strong></p>
</div>
<pre><code>                                                                                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-5-1-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><strong><span class="cmbx-12">Input Data</span></strong></p>
</div>
<pre><code>                           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-5-1-3&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><strong><span class="cmbx-12">Species</span></strong></p>
</div>
<pre><code>                                                                          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-5-1-4&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><strong><span class="cmbx-12">Source</span></strong></p>
</div>
<pre><code>                                                                          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-5-1-5&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><strong><span class="cmbx-12">Tool Offered</span></strong></p>
</div>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-5-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-5-2-1" class="td11">
<span id="textcolor1">Between Pathways Model</span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-2-2" class="td11">
PPI, <a 
href="#glo:SGI">SGI</a>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-2-3" class="td11">
<span id="textcolor2"><span class="cmti-12">S. cerevisiae</span></span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-2-4" class="td11">
<a 
href="#XKelley2005">Kelley and Ideker</a> (<a 
href="#XKelley2005">2005</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-2-5" class="td11">
 
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-5-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-5-3-1" class="td11">
Within Pathways Model
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-3-2" class="td11">
PPI, <a 
href="#glo:SGI">SGI</a>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-3-3" class="td11">
<span class="cmti-12">S. cerevisiae </span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-3-4" class="td11">
<a 
href="#XKelley2005">Kelley and Ideker</a> (<a 
href="#XKelley2005">2005</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-3-5" class="td11">
 
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-5-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-5-4-1" class="td11">
<span id="textcolor3">Decision Tree</span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-4-2" class="td11">
PPI, expression, phenotype
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-4-3" class="td11">
<span id="textcolor4"><span class="cmti-12">S. cerevisiae</span></span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-4-4" class="td11">
<a 
href="#XWong2004">Wong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWong2004">2004</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-4-5" class="td11">
2 Hop
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-5-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-5-5-1" class="td11">
Logistic Regression
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-5-2" class="td11">
<a 
href="#glo:SGI">SGI</a>, PPI, co-expression, phenotype
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-5-3" class="td11">
<span class="cmti-12">C. elegans </span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-5-4" class="td11">
<a 
href="#XZhong2006">Zhong and Sternberg</a> (<a 
href="#XZhong2006">2006</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-5-5" class="td11">
Gene Orienteer
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-5-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-5-6-1" class="td11">
<span id="textcolor5">Network Sampling</span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-6-2" class="td11">
<a 
href="#glo:SGI">SGI</a>, PPI, GO
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-6-3" class="td11">
<span id="textcolor6"><span class="cmti-12">S. cerevisiae</span></span>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-5-6-4"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-6" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-6-1g">
<col 
id="TBL-6-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-6-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-6-1-1" class="td00">
<a 
href="#XLeMeur2008">Le Meur and Gentleman</a> (<a 
href="#XLeMeur2008">2008</a>)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-6-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-6-2-1" class="td00">
<a 
href="#XLeMeur2014">Le Meur <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLeMeur2014">2014</a>)
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-6-5" class="td11">
SLGI(R)
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-5-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-5-7-1" class="td11">
Random Walk
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-7-2" class="td11">
GO, PPI, expression
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-5-7-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-7" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-7-1g">
<col 
id="TBL-7-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-7-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-7-1-1" class="td00">
<span class="cmti-12">S. cerevisiae</span>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-7-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-7-2-1" class="td00">
<span class="cmti-12">C. elegans</span>
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-7-4" class="td11">
<a 
href="#XChipman2009">Chipman and Singh</a> (<a 
href="#XChipman2009">2009</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-7-5" class="td11">
 
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-5-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-5-8-1" class="td11">
<span id="textcolor7">Shared Function</span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-8-2" class="td11">
Co-expression, PPI, text mining, phylogeny
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-8-3" class="td11">
<span id="textcolor8"><span class="cmti-12">C. elegans</span></span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-8-4" class="td11">
<a 
href="#XLee2010b">Lee <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLee2010b">2010b</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-8-5" class="td11">
WormNet
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-5-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-5-9-1" class="td11">
Logistic Regression
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-9-2" class="td11">
Co-expression, PPI, phenotype
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-9-3" class="td11">
<span class="cmti-12">C. elegans </span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-9-4" class="td11">
<a 
href="#XLee2010a">Lee <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLee2010a">2010a</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-9-5" class="td11">
GI Finder
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-5-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-5-10-1" class="td11">
<span id="textcolor9">Jaccard Index</span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-10-2" class="td11">
GO, <a 
href="#glo:SGI">SGI</a>, PPI, phenotype
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-10-3" class="td11">
Eukarya
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-10-4" class="td11">
<a 
href="#XHoehndorf2013">Hoehndorf <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XHoehndorf2013">2013</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-10-5" class="td11">
 
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-5-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-5-11-1" class="td11">
<span id="textcolor10">Machine Learning</span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-11-2" class="td11">
 
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-11-3" class="td11">
 
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-11-4" class="td11">
<a 
href="#XPandey2010">Pandey <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XPandey2010">2010</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-11-5" class="td11">
MNMC
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-5-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-5-12-1" class="td11">
Machine Learning Meta-Analysis
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-12-2" class="td11">
 
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-12-3" class="td11">
 
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-12-4" class="td11">
<a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWu2014">2014</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-12-5" class="td11">
MetaSL
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-5-13-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-5-13-1"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-8" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-8-1g">
<col 
id="TBL-8-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-8-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-8-1-1" class="td00">
<span id="textcolor11">Flux Variability Analysis</span>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-8-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-8-2-1" class="td00">
<span id="textcolor12">Flux Balance Analysis</span>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-8-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-8-3-1" class="td00">
<span id="textcolor13">Network Simulation</span>
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-13-2" class="td11">
Metabolism
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-5-13-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-9" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-9-1g">
<col 
id="TBL-9-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-9-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-9-1-1" class="td00">
<span id="textcolor14"><span class="cmti-12">E. coli</span></span>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-9-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-9-2-1" class="td00">
<span id="textcolor15"><span class="cmti-12">M. pneumoniae</span></span>
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-13-4" class="td11">
<a 
href="#XGuell2014">Güell <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XGuell2014">2014</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-5-13-5" class="td11">
 
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-5-14-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-5-14-1"  
class="td11">
<pre><code>                                                                      &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                                                                                                                                                                                                                      &lt;/div&gt;
                                                                                
                                                                                </code></pre>
</div>
<hr class="endfloat" />
</div>
<!--l. 597-->
<p class="indent">
<a 
 id="x1-45002r2"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 1.2: </span><span class="content">Methods for Predicting Synthetic Lethality in Cancer</span></p>
</div>
<!--tex4ht:label?: x1-45002r2 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-10" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-10-1g">
<col 
id="TBL-10-1" />
<col 
id="TBL-10-2" />
<col 
id="TBL-10-3" />
<col 
id="TBL-10-4" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-10-1-">
<td style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-10-1-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left; vertical-align:middle;">
<p><span class="cmbx-12">Method</span></p>
</div>
<pre><code>                                                                           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left; vertical-align:middle;&quot; id=&quot;TBL-10-1-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left; vertical-align:middle;">
<p><span id="textcolor16"><span class="cmbx-12">Input Data</span></span></p>
</div>
<pre><code>                                                                      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left; vertical-align:middle;&quot; id=&quot;TBL-10-1-3&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left; vertical-align:middle;">
<p><span class="cmbx-12">Source</span></p>
</div>
<pre><code>                                                                            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-10-1-4&quot;  </code></pre>
class=“td11”&gt; <span class="cmbx-12">Tool Offered</span>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-10-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-10-2-1" class="td11">
<span id="textcolor17">Network Centrality</span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-2-2" class="td11">
protein-protein interactions
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-2-3" class="td11">
<a 
href="#XKranthi2013">Kranthi <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XKranthi2013">2013</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-2-4" class="td11">
 
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-10-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-10-3-1" class="td11">
Differential Expression
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-3-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-11" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-11-1g">
<col 
id="TBL-11-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-11-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-11-1-1" class="td00">
Expression
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-11-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-11-2-1" class="td00">
Mutation
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-3-3" class="td11">
<a 
href="#XWang2013">Wang and Simon</a> (<a 
href="#XWang2013">2013</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-3-4" class="td11">
 
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-10-4-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-4-1"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-12" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-12-1g">
<col 
id="TBL-12-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-12-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-12-1-1" class="td00">
<span id="textcolor18">Comparative <a 
href="#glo:genomics">Genomics</a></span>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-12-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-12-2-1" class="td00">
<span id="textcolor19">Chemical-<a 
href="#glo:genomics">Genomics</a></span>
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-4-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-13" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-13-1g">
<col 
id="TBL-13-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-13-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-13-1-1" class="td00">
Yeast synthetic gene interactions
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-13-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-13-2-1" class="td00">
Homology
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-4-3" class="td11">
<a 
href="#XHeiskanen2012">Heiskanen and Aittokallio</a> (<a 
href="#XHeiskanen2012">2012</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-4-4" class="td11">
 
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-10-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-10-5-1" class="td11">
Comparative <a 
href="#glo:genomics">Genomics</a>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-5-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-14" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-14-1g">
<col 
id="TBL-14-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-14-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-14-1-1" class="td00">
Yeast synthetic gene interactions
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-14-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-14-2-1" class="td00">
Homology
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-5-3" class="td11">
<a 
href="#XDeshpande2013">Deshpande <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XDeshpande2013">2013</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-5-4" class="td11">
 
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-10-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-10-6-1" class="td11">
Machine Learning
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-6-2" class="td11">
 
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-6-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-15" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-15-1g">
<col 
id="TBL-15-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-15-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-15-1-1" class="td00">
Discussed by <a 
href="#XBabyak2004">Babyak</a> (<a 
href="#XBabyak2004">2004</a>)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-15-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-15-2-1" class="td00">
and <a 
href="#XLee2009">Lee and Marcotte</a> (<a 
href="#XLee2009">2009</a>)
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-6-4" class="td11">
 
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-10-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-10-7-1" class="td11">
<span id="textcolor20">Differential Expression</span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-7-2" class="td11">
Expression
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-7-3" class="td11">
<a 
href="#XTiong2014">Tiong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTiong2014">2014</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-7-4" class="td11">
 
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-10-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-10-8-1" class="td11">
Literature Database
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-8-2" class="td11">
 
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-8-3" class="td11">
<a 
href="#XLi2014">Li <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLi2014">2014</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-8-4" class="td11">
Syn-Lethality
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-10-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-10-9-1" class="td11">
<span id="textcolor21">Meta-Analysis</span>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-9-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-16" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-16-1g">
<col 
id="TBL-16-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-16-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-16-1-1" class="td00">
Meta-Analysis
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-16-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-16-2-1" class="td00">
Machine Learning
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-9-3" class="td11">
<a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWu2014">2014</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-9-4" class="td11">
MetaSL
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-10-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-10-10-1" class="td11">
Pathway Analysis
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-10-2" class="td11">
 
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-10-3" class="td11">
<a 
href="#XZhang2015">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XZhang2015">2015</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-10-4" class="td11">
 
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-10-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-10-11-1" class="td11">
<span id="textcolor22">Protein Domains</span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-11-2" class="td11">
Homology
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-11-3" class="td11">
<a 
href="#XKozlov2015">Kozlov <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XKozlov2015">2015</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-11-4" class="td11">
 
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-10-12-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-12-1"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-17" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-17-1g">
<col 
id="TBL-17-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-17-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-17-1-1" class="td00">
Data-Mining
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-17-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-17-2-1" class="td00">
Machine Learning
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-12-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-18" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-18-1g">
<col 
id="TBL-18-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-18-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-18-1-1" class="td00">
Expression
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-18-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-18-2-1" class="td00">
Somatic mutation and DNA CNV
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-18-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-18-3-1" class="td00">
siRNA in cell lines
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-12-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-19" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-19-1g">
<col 
id="TBL-19-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-19-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-19-1-1" class="td00">
<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XJerby2014">2014</a>)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-19-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-19-2-1" class="td00">
<a 
href="#XRyan2014">Ryan <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XRyan2014">2014</a>)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-19-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-19-3-1" class="td00">
<a 
href="#XCrunkhorn2014">Crunkhorn</a> (<a 
href="#XCrunkhorn2014">2014</a>)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-19-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-19-4-1" class="td00">
<a 
href="#XLokody2014">Lokody</a> (<a 
href="#XLokody2014">2014</a>)
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-12-4" class="td11">
<a 
href="#glo:DAISY">DAISY</a> (method)
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-10-13-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-13-1"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-20" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-20-1g">
<col 
id="TBL-20-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-20-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-20-1-1" class="td00">
<span id="textcolor23">Genome Evolution</span>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-20-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-20-2-1" class="td00">
<span id="textcolor24">Hypothesis Test</span>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-20-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-20-3-1" class="td00">
<span id="textcolor25">Machine Learning</span>
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-13-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-21" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-21-1g">
<col 
id="TBL-21-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-21-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-21-1-1" class="td00">
Expression
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-21-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-21-2-1" class="td00">
DNA CNV
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-21-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-21-3-1" class="td00">
Known SL
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-13-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-22" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-22-1g">
<col 
id="TBL-22-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-22-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-22-1-1" class="td00">
<a 
href="#XLu2013">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2013">2013</a>)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-22-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-22-2-1" class="td00">
<a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2015">2015</a>)
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-13-4" class="td11">
 
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-10-14-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-14-1"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-23" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-23-1g">
<col 
id="TBL-23-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-23-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-23-1-1" class="td00">
<span id="textcolor26">Bimodality</span>
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-14-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-24" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-24-1g">
<col 
id="TBL-24-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-24-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-24-1-1" class="td00">
Expression
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-24-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-24-2-1" class="td00">
DNA CNV
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-24-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-24-3-1" class="td00">
Somatic Mutation
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-14-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-25" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-25-1g">
<col 
id="TBL-25-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-25-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-25-1-1" class="td00">
<a 
href="#XWappett2014">Wappett</a> (<a 
href="#XWappett2014">2014</a>)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-25-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-25-2-1" class="td00">
<a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWappett2016">2016</a>)
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-10-14-4" class="td11">
<a 
href="#glo:BiSEp">BImodal Subsetting ExPression (BiSEp)</a>
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-10-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-10-15-1" class="td11">
Directional Chi-Square
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-15-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-26" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-26-1g">
<col 
id="TBL-26-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-26-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-26-1-1" class="td00">
Expression (microarray)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-26-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-26-2-1" class="td00">
Somatic mutation
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-15-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-27" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-27-1g">
<col 
id="TBL-27-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-27-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-27-1-1" class="td00">
Kelly, S. T., Guilford, P. J., and Black, M. A.
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-27-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-27-2-1" class="td00">
Dissertation (Kelly, 2013) and developed here
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-27-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-27-3-1" class="td00">
<pre><code>                                   &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                               &lt;/div&gt; &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-10-15-4&quot;  </code></pre>
class=“td11”&gt; SLIPT
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-10-16-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-10-16-1"  
class="td11">
<pre><code>                                                                      &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                                                                                                                                                                                               &lt;/div&gt;
                                                                                
                                                                                </code></pre>
</div>
<hr class="endfloat" />
<!--l. 795-->
<p class="indent">
There are a number of existing computational methods for predicting <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs in humans with a specfic interest in cancer (as summarised in <a 
href="#x1-45002r2">1.2<!--tex4ht:ref: tab:methods_SL --></a>). While these demonstrate the power and need for predictions of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in human and cancer contexts, limitations of previous methods could be met with a different approach. Existing computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> prediction are often difficult to interpret, replicate for new genes, or are reliant on data types not available for a wider range of genes to test. <a 
 id="x1-45003r49"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.7.2 </span> <a 
 id="x1-460002"></a>Comparative Genomics
</h5>
<!--l. 799-->
<p class="noindent">
A comparative <a 
href="#glo:genomics">genomics</a> approach by <a 
href="#XDeshpande2013">Deshpande <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XDeshpande2013">2013</a>) used the results of well characterised high-throughput mutation screens in <span class="cmti-12">S. cerevisiae </span>as candidates for <a 
href="#glo:synthetic lethal">synthetic lethality</a> in humans (<a 
href="#XBaryshnikova2010a">Baryshnikova <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBaryshnikova2010a">2010a</a>; <a 
href="#XCostanzo2010">Costanzo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCostanzo2010">2010</a>, <a 
href="#XCostanzo2011">2011</a>; <a 
href="#XTong2001">Tong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTong2001">2001</a>, <a 
href="#XTong2004">2004</a>). Yeast <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners were compared to human orthologues to find cancer relevant <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate pairs with direct therapeutic potential. Proposed as a complementary approach to siRNA screens, approximately 24,000 of the 116,000 negative <a 
href="#glo:SGI">SGIs</a> in yeast (<a 
href="#XCostanzo2011">Costanzo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCostanzo2011">2011</a>) were matched to human orthologues, with over 500 involving a cancer gene (<a 
href="#XFutreal2004">Futreal <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFutreal2004">2004</a>). Under strict criteria of one-to-one orthologues, large effect size and significant interaction in yeast data (<span class="cmmi-12">ϵ &lt; </span><span class="cmsy-10x-x-120">-</span>0<span class="cmmi-12">.</span>2, <span class="cmmi-12">p &lt; </span>0<span class="cmmi-12">.</span>05), 1,522 interactions were identified with 70 involving cancer genes. Of the 21 gene interactions tested with pairs of siRNA in IMR1 fibroblast cells, 6 exhibited <a 
href="#glo:synthetic lethal">synthetic lethal</a> effects. The two strongest interactions (<span class="cmti-12">SMARCB1 </span>with <span class="cmti-12">PSMA4 </span>and <span class="cmti-12">ASPSCR1 </span>with <span class="cmti-12">PSMC2</span>) were successfully validated by protein analysis of human cells and replication with tetrad analysis for yeast orthologues.
</p>
<!--l. 801-->
<p class="indent">
<p>Another approach to systematic <a 
href="#glo:synthetic lethal">synthetic lethality</a> discovery specific to human cancer (in contrast to the plethora of yeast <a 
href="#glo:synthetic lethal">synthetic lethality</a> data) was to build a database as done by <a 
href="#XLi2014">Li <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLi2014">2014</a>). In their relational database, called “Syn-lethality”, they have curated both known experimentally discovered <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs in humans (113 pairs) from the literature and those predicted from <a 
href="#glo:synthetic lethal">synthetic lethality</a> between orthologous genes in <span class="cmti-12">S. cerevisiae </span>yeast (1114 pairs). This knowledge-based database is the first dedicated to human cancer <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions and integrates gene function annotation, pathway and molecular mechanism data with experimental and predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs. This combination of data sources is intended to tackle</p>
<p>the trade-off between more conclusive <a 
href="#glo:synthetic lethal">synthetic lethal</a> experiments in yeast and more clinically relevant <a 
href="#glo:synthetic lethal">synthetic lethal</a> experiments in human cancer models, such as <a 
href="#glo:RNAi">RNAi</a>, especially when high-throughput screens are costly and prone to false positives in either system and difficult to replicate across gene backgrounds. This database centralises a wealth of knowledge scattered in the literature including cancer relevant genes (<span class="cmti-12">BRCA1</span>, <span class="cmti-12">BRCA2</span>, <span class="cmti-12">PARP1</span>, <span class="cmti-12">PTEN</span>, <span class="cmti-12">VHL</span>, <span class="cmti-12">MYC</span>, <span class="cmti-12">EGFR</span>, <span class="cmti-12">MSH2</span>, <span class="cmti-12">KRAS</span>, and <span class="cmti-12">TP53</span>) and is publicly available as a Java App. These included the previously mentioned interactions of <span class="cmti-12">BRCA1 </span>and <span class="cmti-12">BRCA2 </span>with <span class="cmti-12">PARP1 </span>and <span class="cmti-12">TP53 </span>with <span class="cmti-12">WEE1 </span>and <span class="cmti-12">PLK1</span>. However, the computational methodology was not released, so it is not possible to replicate their results, nor to add to the findings with new datasets, which are limited to 647 human genes. Suggested future directions were promising, such as constructing networks of known <a 
href="#glo:synthetic lethal">synthetic lethality</a>, applying known <a 
href="#glo:synthetic lethal">synthetic lethality</a> to cancer treatment, data mining, replicating the approach for <a 
href="#glo:synthetic lethal">synthetic lethality</a> in model organisms, signalling pathways, and developing a complete global network in human cancer or yeast (both of which are still incomplete with experimental data), some of which has been implemented in “SynLethDB” (<a 
href="#XGuo2016">Guo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuo2016">2016</a>).</p>
</p>
<!--l. 804-->
<p class="indent">
<a 
 id="x1-46001r3"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 1.3: </span><span class="content">Machine Learning Methods used by <a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWu2014">2014</a>)</span></p>
</div>
<!--tex4ht:label?: x1-46001r3 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-28" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-28-1g">
<col 
id="TBL-28-1" />
<col 
id="TBL-28-2" />
<col 
id="TBL-28-3" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-28-1-">
<td style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-28-1-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left; vertical-align:middle;">
<p><span class="cmbx-12">Method</span></p>
</div>
<pre><code>                                                                          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left; vertical-align:middle;&quot; id=&quot;TBL-28-1-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left; vertical-align:middle;">
<p><span class="cmbx-12">Source</span></p>
</div>
<pre><code>                 &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-28-1-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmbx-12">Tool Offered</span>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-28-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-28-2-1" class="td
11">
<span id="textcolor27">Random Forest</span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-28-2-2" class="td11">
<a 
href="#XBreiman2001">Breiman</a> (<a 
href="#XBreiman2001">2001</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-28-2-3" class="td11">
 
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-28-3-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-28-3-1"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-29" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-29-1g">
<col 
id="TBL-29-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-29-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-29-1-1" class="td00">
Random Forest
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-29-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-29-2-1" class="td00">
J48 (decision tree)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-29-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-29-3-1" class="td00">
Bayes (Log Regression)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-29-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-29-4-1" class="td00">
Bayes (Network)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-29-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-29-5-1" class="td00">
PART (Rule-based)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-29-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-29-6-1" class="td00">
RBF Network
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-29-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-29-7-1" class="td00">
Bagging / Bootstrap
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-29-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-29-8-1" class="td00">
Classification via Regression
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-28-3-2" class="td11">
<a 
href="#XHall2009">Hall <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XHall2009">2009</a>)  
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-28-3-3" class="td11">
WEKA
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-28-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-28-4-1" class="td11">
<span id="textcolor28">Support Vector Machine (Linear)</span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-28-4-2" class="td11">
<a 
href="#XVapnik1995">Vapnik</a> (<a 
href="#XVapnik1995">1995</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-28-4-3" class="td11">
 
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-28-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-28-5-1" class="td11">
Support Vector Machine (RBF – Gaussian)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-28-5-2" class="td11">
<a 
href="#XJoachims1999">Joachims</a> (<a 
href="#XJoachims1999">1999</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-28-5-3" class="td11">
 
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-28-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-28-6-1" class="td11">
<span id="textcolor29">Multi-Network Multi-Class (MNMC)</span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-28-6-2" class="td11">
<a 
href="#XPandey2010">Pandey <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XPandey2010">2010</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-28-6-3" class="td11">
 
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-28-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-28-7-1" class="td11">
MetaSL (Meta-Analysis)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-28-7-2" class="td11">
<a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWu2014">2014</a>)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-28-7-3" class="td11">
MetaSL
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-28-8-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-28-8-1"  
class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                &lt;/div&gt;
                                                                                
                                                                                </code></pre>
</div>
<hr class="endfloat" />
<!--l. 862-->
<p class="indent">
Machine learning approaches have also been proposed for <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery (<a 
href="#XBabyak2004">Babyak</a>, <a 
href="#XBabyak2004">2004</a>; <a 
href="#XLee2009">Lee and Marcotte</a>, <a 
href="#XLee2009">2009</a>). Due to concerns that these may be subject to overfitting or noise, <a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWu2014">2014</a>) developed a meta-analysis method (based on the machine learning methods in Table <a 
href="#x1-46001r3">1.3<!--tex4ht:ref: tab:methods_meta --></a>) for <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs relevant to developing selective drugs against human cancer, building upon their previous database (<a 
href="#XLi2014">Li <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLi2014">2014</a>). They used training data of 10,885 <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions from yeast experiments of which 7347 occurred between the 5,504 non-essential genes. Their “metaSL” approach utilises genomic, proteomic and annotation data (including GO terms <a 
href="#XAshburner2000">Ashburner <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XAshburner2000">2000</a>), PPI, protein complexes, and biological pathway) with strong statistical performance in yeast data (<a 
href="#glo:AUROC">AUROC</a> of 0.871). The predicted orthologous <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in human data were not experimentally validated but several would be relevant to cancer such as <span class="cmti-12">EGFR </span>with <span class="cmti-12">PRKCZ</span>. They note that computational approaches scale-up across the genome at lower cost than experimental screen and share their most supported interactions online. However, the method is not available for analysis of other genes studied by the cancer research community. While machine learning has great potential as a predictor, the results vary greatly depending on the predictive features selected and it is not clear which threshold should be used to report reliably detected genes. Syn-Lethality (<a 
href="#XLi2014">Li <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLi2014">2014</a>) and MetaSL (<a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWu2014">2014</a>) demonstrate the value of computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethality</a> but omit many genes of importance in cancer, such as <span class="cmti-12">CDH1</span>, and there remains a need to enable biological researchers to query such genes in a particular tissue or genetic background.
</p>
<!--l. 864-->
<p class="indent">
<p>There is also concern for analyses based on yeast data that many <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions may not be conserved between species <a 
href="#XDixon2009a">Dixon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XDixon2009a">2009</a>), although interactions between pathways may be more comparable. It is unsurprising that many of the interactions identified were not experimentally validated. There have been many gene duplications in the separate evolutionary histories of humans and yeast which may lead to differences in genetic redundancy. Yeast are not an ideal human cancer model because they are do not have tissue specificity, multicellular gene regulation, or orthologues to a number of known cancer genes such as p53. Although these studies have tried to anticipate these issues with stringent criteria such as requiring one-to-one orthologues, there remains the possibility that changes in gene function may affect whether these are solely redundant such as if functions had coevolved without sequence</p>
homology. Many genes will also be excluded since they lack homologues in yeast, the corresponding experimental data, or having paralogues in either species. Thus conservation of yeast interactions is not an ideal strategy and analysis of human data directly for comparison with human experimental data will be the focus of this thesis. <a 
 id="x1-46002r52"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.7.3 </span> <a 
 id="x1-470003"></a>Analysis and Modelling of Protein Data
</h5>
<!--l. 868-->
<p class="noindent">
<a 
href="#XKranthi2013">Kranthi <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XKranthi2013">2013</a>) took a network approach to discovery of <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate selection applying the concept to “centrality” to a human PPI network involving interacting partners of known cancer genes. The effect of removing pairs of genes on connectivity of the network was used as a surrogate for viability which is supported by observations that the PPI and <a 
href="#glo:synthetic lethal">synthetic lethal</a> networks are orthogonal in <span class="cmti-12">S. cerevisiae </span>studies (<a 
href="#XTong2004">Tong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTong2004">2004</a>). They showed that the human cancer protein interaction network (of 1539 proteins and 6471 interactions) exhibits the power law distribution expected of a scale-free <a 
href="#glo:synthetic lethal">synthetic lethal</a> network with high connectivity (average vertex degree of 23.67 and network efficiency of 0.2952). Their top 100 candidate interactions included interactions of the tumour suppressor <span class="cmti-12">TP53</span> with <span class="cmti-12">BRCA1</span>, <span class="cmti-12">CDKNA1</span>, <span class="cmti-12">CDKNA2</span>, <span class="cmti-12">MET</span>, and <span class="cmti-12">RB1 </span>which have been detected by prior studies. The gene pairs were often observed to be in the same or a plausible compensatory pathway. Thus the network structure is important in the biological functions of cancers and could be exploited for targeting <span class="cmti-12">TP53 </span>loss of function mutations.
</p>
<!--l. 870-->
<p class="indent">
<p>However, their approach was limited to known cancer genes and is not applicable to genes that do not have PPI data. Other nucleotide sequencing data types are more commonly available for cancer studies at a genomic scale. Of further concern is that the results were enriched for p53 <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners which is relevant to many cancer researchers but this genome-wide approach did not detect many other cancer genes due to multiple testing. This enrichment may be due to the known drastic effect of removing p53 itself from the network as a master regulator, cancer driving tumour suppressor gene, and highly connected network “hub”. The focus on cancer genes is useful for translation into therapeutics but does not account for variable genetic backgrounds or effect of protein removal on the whole cellular network.</p>
</p>
<!--l. 872-->
<p class="indent">
Focusing on the potential for <a 
href="#glo:synthetic lethal">synthetic lethality</a> to be an effective anti-cancer drug target, <a 
href="#XZhang2015">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XZhang2015">2015</a>) used modelling of signalling pathways to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions between known drug targets and cancer genes by simulating gene knockdowns. A computational approach was applied to avoid the limitations of experimental <a 
href="#glo:RNAi">RNAi</a> screens such as scale, instability of knockdown, and off-target effects. This ‘hybrid’ method of a data-driven model and known signalling pathways showed potential as a means to predict cell death in single and combination gene knockouts. They used time series protein phosphorylation data (<a 
href="#XLee2012">Lee <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLee2012">2012</a>) for 28 signalling proteins and <a 
href="#glo:GO">Gene Ontology (GO)</a> (GO) pathways <a 
href="#XAshburner2000">Ashburner <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XAshburner2000">2000</a>); <a 
href="#XBlake2015">Blake <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XBlake2015">2015</a>). This approach successfully detected many known essential genes in the human gene essentiality database, known <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners in the Syn-Lethality database (<a 
href="#XLi2014">Li <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLi2014">2014</a>), and predicted novel <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs. The strongest essential genes in single knockdowns were <span class="cmti-12">AKT</span>, <span class="cmti-12">TP53</span>, <span class="cmti-12">CHK1</span>, <span class="cmti-12">S6K1</span>, and <span class="cmti-12">CYCLIND1</span>. Pairwise knockdowns identified 252 candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions including <span class="cmti-12">TP53 </span>with <span class="cmti-12">CHK1</span>, <span class="cmti-12">S6K1</span>, <span class="cmti-12">WEE1</span>, <span class="cmti-12">CYCLIND1</span>, and <span class="cmti-12">CASP9</span>; <span class="cmti-12">AKT </span>with <span class="cmti-12">WEE1</span>; and <span class="cmti-12">CDK1 </span>with <span class="cmti-12">CYCLIND1</span>. These novel results contained many <span class="cmti-12">TP53 </span>and AKT <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners, genes known to be important in many cancers, however these also have a severe impact on the signalling pathways in their essentiality analysis of single gene disruptions and large phenotypic changes in cancer. This approach is amenable to detect functionally related pathways and protein complexes across the molecular function, cellular component, and biological process annotations provided by GO. The results were consistent with the experimental results in the literature but the novel <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions have yet to be validated. While the mathematical reasoning and algorithms are given, the code was not released to replicate the findings or apply the methodology beyond the signalling pathways analysed by <a 
href="#XZhang2015">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XZhang2015">2015</a>). While this is an interesting approach, the analysis of this thesis will focus on gene expression and <a 
href="#glo:RNAi">RNAi</a> data which is available to test a wider range of candidate gene pairs. <a 
 id="x1-47001r54"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.7.4 </span> <a 
 id="x1-480004"></a>Differential Gene Expression
</h5>
<!--l. 878-->
<p class="noindent">
<p>Differential gene expression has been explored to predict <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs in cancer which would be widely applicable due to the availability of public gene expression data for a large number of samples and cancer types. <a 
href="#XWang2013">Wang and Simon</a> (<a 
href="#XWang2013">2013</a>) found differentially expressed genes (by the t-test, adjusted by FDR) between tumours with or</p>
without functional p53 mutations in <a 
href="#glo:TCGA">TCGA</a> (<a 
href="#XTCGA2008GBM">McLendon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2008GBM">2008</a>) and <a 
href="#glo:CCLE">Cancer Cell Line Encyclopaedia (CCLE)</a> (<a 
href="#XBarretina2012">Barretina <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBarretina2012">2012</a>) <a 
href="#glo:RNA-Seq">RNA-Seq</a> gene expression data as candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner pathways of p53. They identified 2, 8, and 21 candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes in 3 microarray datasets from the NCI60 cell lines, 31 partner genes from the <a 
href="#glo:CCLE">CCLE</a> <a 
href="#glo:RNA-Seq">RNA-Seq</a> data, and 50 in <a 
href="#glo:TCGA">TCGA</a> <a 
href="#glo:RNA-Seq">RNA-Seq</a> data. <span class="cmti-12">PLK1 </span>was replicated across 4 of these analyses and 17 other genes were replicated across 2 analyses (including <span class="cmti-12">MTOR</span>, <span class="cmti-12">PLK4</span>, <span class="cmti-12">MAST2</span>, <span class="cmti-12">MAP3K4</span>, <span class="cmti-12">AURKA</span>, <span class="cmti-12">BUB1 </span>and 6 CDK genes) with many playing a role in cell cycle regulation. This was supported by a drug sensitivity experiment on the NCI60 cell lines which found that cells which lacked functional p53 were more sensitive to paclitaxel (which targets <span class="cmti-12">PLK1</span>, <span class="cmti-12">AURKA</span>, and <span class="cmti-12">BUB1</span>). This demonstrated the potential of gene expression as a surrogate for gene function and use of public genomic data to predict <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs in cancer. <a 
href="#XWang2013">Wang and Simon</a> (<a 
href="#XWang2013">2013</a>) advocated for pre-screening of expression profiles to augment future <a 
href="#glo:RNAi">RNAi</a> screens. However, the analyses were limited to kinase genes and focused on currently druggable genes, lacking wider application of <a 
href="#glo:synthetic lethal">synthetic lethal</a> prediction methodology. This approach may not be feasible or applicable in cancer genes with a lower mutation rate than p53.
</p>
<!--l. 880-->
<p class="indent">
<p><a 
href="#XTiong2014">Tiong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTiong2014">2014</a>) also investigated gene expression as a predictor of synthetic lescale-freethal pairs with colorectal cancer microarrays from a Han Chinese population with a sample size of 70 tumour and 12 normal tissue samples. Simultaneously differentially expression of “tumour dependent” gene pairs (which includes co-expression) between cancer and normal tissue was used to rank 663 candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions identified in cell line siRNA experiments. Of the top 20 genes, 17 were tested for testing differential expression at the protein level with immunohistochemistry staining and correlation with clinical characteristics, with 11 pairs exhibiting synergistic effects. Some of the predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs were consistent with the literature (including <span class="cmti-12">TP53</span> with <span class="cmti-12">S6K1 </span>and partners of <span class="cmti-12">KRAS</span>, <span class="cmti-12">PTEN</span>, <span class="cmti-12">BRCA1</span>, and <span class="cmti-12">BRCA2</span>) and two novel <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions (<span class="cmti-12">TP53 </span>with <span class="cmti-12">CSNK1E </span>and <span class="cmti-12">CTNNB1</span>) were validated in pre-clinical models. This serves a valuable proof-of-concept for integration of <span class="cmti-12">in silico</span> approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery in cancer demonstrating it’s utility to triage and identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of p53 applicable to colorectal tissues. Although the experimental work was the focus of the paper, these findings show that <a 
href="#glo:bioinformatics">bioinformatics</a> <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates can be validated in patient tissue</p>
samples (from a non-caucasian population) to find those applicable to colorectal cancers. <a 
 id="x1-48001r55"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.7.5 </span> <a 
 id="x1-490005"></a>Data Mining and Machine Learning
</h5>
<!--l. 886-->
<p class="noindent">
Recognising the utility of <a 
href="#glo:synthetic lethal">synthetic lethality</a> to drug inhibition and specificity of anti-cancer treatments, <a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XJerby2014">2014</a>) also saw the need for effective prediction of gene essentiality and <a 
href="#glo:synthetic lethal">synthetic lethality</a> to augment experimental studies of SL. They developed the <a 
href="#glo:DAISY">DAta mIning SYnthetic lethal identification pipeline (DAISY)</a>, a data-driven approach for genome-wide analysis of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in public cancer <a 
href="#glo:genomics">genomics</a> data from <a 
href="#glo:TCGA">TCGA</a> and <a 
href="#glo:CCLE">CCLE</a> (<a 
href="#XBarretina2012">Barretina <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBarretina2012">2012</a>). <a 
href="#glo:DAISY">DAISY</a> is intended to predict the candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of a query gene such as genes recurrently mutated in cancer.
</p>
<!--l. 888-->
<p class="indent">
<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XJerby2014">2014</a>) combined a computational approach to triage candidates with a conventional <a 
href="#glo:RNAi">RNAi</a> screen to validate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners. They screened a selection of computationally predicted candidates and randomly selected genes with <a 
href="#glo:RNAi">RNAi</a> against <span class="cmti-12">VHL </span>loss of function mutation in RCC4 renal cell lines. The computational method had a high <a 
href="#glo:AUROC">AUROC</a> of 0.779 and predictions were enriched 4<span class="cmsy-10x-x-120">× </span>for validated <a 
href="#glo:RNAi">RNAi</a> hits over randomly selected genes. This approach detected known <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs such as <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>genes with <span class="cmti-12">PARP1 </span>and <span class="cmti-12">MSH2 </span>with <span class="cmti-12">DHFR</span>. The <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates identified with both <a 
href="#glo:RNAi">RNAi</a> screening and computational prediction formed an extensive network of 2077 genes with 2816 <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions and similar network of 3158 genes with 3635 synthetic dosage lethal interactions (for <a 
href="#glo:synthetic lethal">synthetic lethality</a> with over-expression). Each network was scale-free as expected of a biological network and was enriched for known cancer genes, essential genes in mice, and could be harnessed for predicting prognosis and drug response. While demonstrating the feasibility of combining experimental and computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethality</a> in cancer, there remain challenges in predicting <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes, novel drug targets, and translation into the clinic.
</p>
<!--l. 890-->
<p class="indent">
<p>The <a 
href="#glo:DAISY">DAISY</a> methodology (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>) compares the results of analysis of several data types to predict <a 
href="#glo:synthetic lethal">synthetic lethality</a>, namely: DNA copy number and somatic mutation for <a 
href="#glo:TCGA">TCGA</a> patient samples and <a 
href="#glo:CCLE">CCLE</a> cell lines. The cell lines were also analysed with gene expression and gene essentiality (shRNA screening) profiles. Genes were classed as inactivated by copy number deletion, somatic loss of</p>
function mutation, or low expression and tested for <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene partners which are either essential in screens or not deleted with copy number variants. Co-expression is also used for <a 
href="#glo:synthetic lethal">synthetic lethality</a> prediction based on studies in yeast (<a 
href="#XCostanzo2010">Costanzo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XCostanzo2010">2010</a>; <a 
href="#XKelley2005">Kelley and Ideker</a>, <a 
href="#XKelley2005">2005</a>). Copy number, gene expression and, essentiality analyses are stringently compared by adjusting each for multiple tests with Bonferroni correction and only taking hits which occur in all analyses. This methodology was also adapted for synthetic dosage lethality by testing for partner genes where genes are overactive with high copy number or expression. As discussed above, the predictions performed well and an <a 
href="#glo:RNAi">RNAi</a> screen for the example of <span class="cmti-12">VHL </span>in renal cancer validated predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">VHL</span> demonstrating the feasibility of combining approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery in cancer and using computational predictions to enable more efficient high-throughput screening. <a 
href="#glo:DAISY">DAISY</a> performs well statistically with a <a 
href="#glo:AUROC">AUROC</a> of 0.779 on a set of gene pairs with experimental screen data, although co-expression and shRNA functional examination contributes much less of this than the mutation and copy number analysis (<a 
href="#glo:AUROC">AUROC</a> 0.683 alone). However, this methodology is very stringent, missing potentially valuable <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates, may not be applicable to genes of interest to other groups and the software for the procedure has not been publicly released for replication.
</p>
<!--l. 892-->
<p class="indent">
Although the <a 
href="#glo:DAISY">DAISY</a> procedure performs well and has been well received by the scientific community (<a 
href="#XCrunkhorn2014">Crunkhorn</a>, <a 
href="#XCrunkhorn2014">2014</a>; <a 
href="#XLokody2014">Lokody</a>, <a 
href="#XLokody2014">2014</a>; <a 
href="#XRyan2014">Ryan <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XRyan2014">2014</a>), showing a need for such methodology, there is no indication of adoption of the methodology in the community yet. The co-expression analysis may not be the most effective way to test gene expression for directional <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions (where inverse correlation would be expected). In the interests of a large sample size, tissue types were not tested separately despite tissue-specific <a 
href="#glo:synthetic lethal">synthetic lethality</a> being likely since gene function (and by extension expression, isoforms, and clinical characteristics) in cancers may often be tissue-dependent. Some data forms and analyses used, such as gene essentiality, may not be available for all cancers, genes, or tissues, and may not be reproduced.
</p>
<!--l. 894-->
<p class="indent">
<p><a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2015">2015</a>) critique the assumption of co-expression in the <a 
href="#glo:DAISY">DAISY</a> methodology and propose an alternative computational prediction of <a 
href="#glo:synthetic lethal">synthetic lethality</a> based on machine learning methods and a “cancer genome evolution” hypothesis. Using DNA copy number and gene expression data from <a 
href="#glo:TCGA">TCGA</a> patient samples, a cancer genome evolution model assumes that <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs behave in two</p>
distinct ways in response to an inactive <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner gene, either a “compensation” pattern where the other <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner is overactive or a “co-loss underrepresentation” pattern where the other <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner is less likely to be lost, since loss of both genes would cause death of the cancer cell. During the genome evolution of cancers, the cell becomes addicted to the remaining <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner due to induced gene essentiality. These patterns would explain why <a 
href="#glo:DAISY">DAISY</a> detects only a small number of <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs, compared to the large number expected based on model organism studies (<a 
href="#XBoone2007">Boone <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoone2007">2007</a>), and the disparity between screening and computationally predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates due to testing different classes of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs.
</p>
<!--l. 896-->
<p class="indent">
<a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2015">2015</a>) compared a genome-wide computational model of genome evolution and gene expression patterns to the experimental data of <a 
href="#XVizeacoumar2013">Vizeacoumar <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XVizeacoumar2013">2013</a>) and <a 
href="#XLaufer2013">Laufer <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLaufer2013">2013</a>). This simpler model performing well with an <a 
href="#glo:AUROC">AUROC</a> of 0.751 but was less than <a 
href="#glo:DAISY">DAISY</a>, although it did not rely on data from cell lines which may not represent patient disease. They predict a larger comprehensive list of 591,000 human <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners with a probability score threshold of 0.81, giving a precision of 67% and 14<span class="cmsy-10x-x-120">× </span>enrichment of <a 
href="#glo:synthetic lethal">synthetic lethal</a> true positives compared to randomly selected gene pairs. Discovery of such a vast number of cancer-relevant <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions in humans would not be feasible experimentally and is a valuable resource for research and clinical applications. These predictions are not limited by assuming co-expression of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners or evolutionary conservation with model organisms enabling wider <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery. However, there remains a lack of basis for an expectation of how many <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners a particular gene will have, how many pairs there are in the human genome, and whether pathways or correlation structure would influence predicted <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners.
</p>
<!--l. 898-->
<p class="indent">
<p>Large scale, computational approaches have yet to determine whether <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions are tissue-specific since <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2015">2015</a>) used pan-cancer data for 14136 patients with 31 cancer types. Experimental data used for comparison was a small training dataset specific to colorectal cancer, and based on screens for other phenotypes, which may limit performance of the model or application to other cancers. Proposed expansion of the computational approach to mutation, microRNA, or epigenetic modulation of gene function and tumour micro-environment or heterogeneity suggests that <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery could be widely applied to the</p>
current challenges in cancer <a 
href="#glo:genomics">genomics</a>. This approach was also based on machine learning methodology and not supported by a software release for the community to develop, contribute to, or reproduce beyond the gene pairs given in the supplementary results. <a 
 id="x1-49001r56"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.7.6 </span> <a 
 id="x1-500006"></a>Bimodality
</h5>
<!--l. 902-->
<p class="noindent">
<a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWappett2016">2016</a>) demonstrate a multi-omic approach to identification of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in cancer with a strategy to detect bimodal patterns in molecular profiles. They released this solution as the <a 
href="#glo:BiSEp">BiSEp</a> R package <a 
href="#XWappett2014">Wappett</a> (<a 
href="#XWappett2014">2014</a>) which aims to detect subtle bimodal and non-normal patterns in expression data. Since loss of gene function is not consistently genetic, <a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWappett2016">2016</a>) advocate the use of gene expression (loss of <a 
href="#glo:mRNA">mRNA</a>) and deletion (loss of copy number) data in addition to mutation. The <a 
href="#glo:BiSEp">BiSEp</a> procedure was demonstrated on an analysis of 881 cell lines from <a 
href="#glo:CCLE">CCLE</a> (<a 
href="#XBarretina2012">Barretina <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBarretina2012">2012</a>), 442 cell lines from <a 
href="#glo:COSMIC">Catalogue Of Somatic Mutations In Cancer (COSMIC)</a> (<a 
href="#XForbes2015">Forbes <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XForbes2015">2015</a>), and RSEM normalised <a 
href="#glo:RNA-Seq">RNA-Seq</a> data for 178 <a 
href="#glo:TCGA">TCGA</a> lung patient samples (<a 
href="#XTCGA2014LU">Collisson <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014LU">2014</a>). <a 
href="#glo:BiSEp">BiSEp</a> was demonstrated to have significant enrichment of validated tumour suppressor, <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs (detecting 76 experimentally supported gene pairs) and was improved (detecting 420) with expression data rather than relying on detecting loss of gene function by mutation or deletion. They identified interactions with genes relevant to cancer with support in experimental screens including <span class="cmti-12">ERCC4 </span>with <span class="cmti-12">XRCC1</span>, <span class="cmti-12">BRCA1 </span>with <span class="cmti-12">PARP3</span>, and <span class="cmti-12">SMARCA1 </span>with <span class="cmti-12">SMARCA4</span>.
</p>
<!--l. 904-->
<p class="indent">
<a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XWappett2016">2016</a>) demonstrated that analysis of <a 
href="#glo:genomics">genomics</a> data, particularly expression data, is relevant to augment the identification of <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions with screening experiments. They further show that this is applicable in both genetically homogeneous cell lines and heterogeneous cell population from patient samples. This approach is limited however to genes which exhibit bimodal expression patterns which do not commonly occur, particularly in normalised gene expression data, and other approaches may need to be considered for gene such as <span class="cmti-12">CDH1 </span>which were not identified by <a 
href="#glo:BiSEp">BiSEp</a>. <a 
 id="x1-50001r57"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.2.7.7 </span> <a 
 id="x1-510007"></a>Rationale for Further Development
</h5>
<!--l. 908-->
<p class="noindent">
Many of the approaches discussed here aimed to identify the strongest <a 
href="#glo:synthetic lethal">synthetic lethal</a> pairs across the yeast or human genomes (<a 
href="#XDeshpande2013">Deshpande <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDeshpande2013">2013</a>; <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>; <a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWappett2016">2016</a>; <a 
href="#XWu2014">Wu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWu2014">2014</a>), which may not be an ideal strategy to identify interactions in particular functions or relevance to particular cancers. These demonstrate a need for computational approaches to prioritise candidate gene pairs for validation but this thesis will focus on the interactions with <span class="cmti-12">CDH1 </span>with particular importance in breast and stomach cancers, although these partners may be applicable in other cancers. As such, this thesis presents a query-based method, amenable to identification of candidate partners for a selected gene of functional or translational importance such as <span class="cmti-12">CDH1</span>. <a 
 id="x1-51001r36"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">1.3 </span> <a 
 id="x1-520003"></a>E-cadherin as a Synthetic Lethal Target
</h3>
<!--l. 915-->
<p class="noindent">
<a 
href="#glo:E-cadherin">E-cadherin</a> is a transmembrane protein (encoded by <span class="cmti-12">CDH1</span>) with several characterised functions in the cytoskeleton and cell-to-cell signalling. Here we outline the key known functions of <a 
href="#glo:E-cadherin">E-cadherin</a> and it’s importance in cancer biology. <span class="cmti-12">CDH1 </span>is a tumour supressor gene, with loss of function occurring in both familial (germline mutations) and sporadic (somatic mutations) cancers. As such, <span class="cmti-12">CDH1 </span>inactivation is a prime example of a genetic event that could be targeted by <a 
href="#glo:synthetic lethal">synthetic lethality</a> for anti-cancer treatments. Most notably this includes patients at risk of developing hereditary breast and stomach cancers for which conventional surgical or cytotoxic chemotherapy is not ideal (due to impact of quality of life) and who have a known genetic aberration in their familial syndromic cancers. Effective treatments against <span class="cmti-12">CDH1 </span>inactivation would also benefit patients with sporadic diffuse gastric cancers since they often present with symptoms at a late stage. <a 
 id="x1-52001r48"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.3.1 </span> <a 
 id="x1-530001"></a>The <span class="cmti-12">CDH1 </span>gene and it’s Biological Functions
</h4>
<!--l. 918-->
<p class="noindent">
<p>The tumour suppressor gene <span class="cmti-12">CDH1 </span>is implicated in hereditary and sporadic lobular breast cancers (<a 
href="#XBerx1996">Berx <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBerx1996">1996</a>; <a 
href="#XBerx2009">Berx and van Roy</a>, <a 
href="#XBerx2009">2009</a>; <a 
href="#XDeLeeuw1997">De Leeuw <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDeLeeuw1997">1997</a>; <a 
href="#XMasciari2007">Masciari <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMasciari2007">2007</a>; <a 
href="#XSemb1998">Semb and Christofori</a>, <a 
href="#XSemb1998">1998</a>; <a 
href="#XVos1997">Vos <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XVos1997">1997</a>). The <span class="cmti-12">CDH1 </span>gene encodes the <a 
href="#glo:E-cadherin">E-cadherin</a> protein and is normally expressed in epithelial tissues, where it has also been identified as an invasion suppressor and loss of <span class="cmti-12">CDH1 </span>function has been implicated in breast</p>
cancer progression and metastasis (<a 
href="#XBecker1994">Becker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBecker1994">1994</a>; <a 
href="#XBerx1995">Berx <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBerx1995">1995</a>; <a 
href="#XChristofori1999">Christofori and Semb</a>, <a 
href="#XChristofori1999">1999</a>). <a 
 id="x1-53001r58"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.3.1.1 </span> <a 
 id="x1-540001"></a>Cytoskeleton
</h5>
<!--l. 921-->
<p class="noindent">
The primary function of <span class="cmti-12">CDH1 </span>is cell-cell adhesion forming the adherens junction, maintaining the cytoskeleton and mediating molecular signals between cells. The function of the adherens complex is particularly important for cell structure and regulation because it interacts with cytoskeletal actins and microtubules. The cytoskeletal role of <a 
href="#glo:E-cadherin">E-cadherin</a> maintains healthy cellular viability and growth in epithelial tissues including cellular polarity. <a 
href="#glo:E-cadherin">E-cadherin</a> is not essential to cellular viability but loss in epithelial cells does lead to defects in cytoskeletal structure and proliferation. In addition to a central role in the adherens complex, <a 
href="#glo:E-cadherin">E-cadherin</a> is involved in many other cellular functions and thus <span class="cmti-12">CDH1 </span>is regarded as a highly <a 
href="#glo:pleiotropy">pleiotropic</a> gene. <a 
 id="x1-54001r61"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.3.1.2 </span> <a 
 id="x1-550002"></a>Extracellular and Tumour Micro-Environment
</h5>
<!--l. 924-->
<p class="noindent">
As a transmembrane signalling protein <a 
href="#glo:E-cadherin">E-cadherin</a> also interacts with the extracellular environment and other cells, most notably forming tight junctions between cells. These junctions serve to both regulate movement of ion signals between cells and separate membrane proteins on the apical and basal surfaces of a cell, maintaining cell polarity. Thus <a 
href="#glo:E-cadherin">E-cadherin</a> is an important regulator of epithelial tissues by intercellular communication. It also has important roles in the extracellular matrix, including fibrin clot formation. The role of intercellular interactions and the tissue micro-environment are important themes in cancer research, being a potential mechanism for cancer progression and malignancy in a addition to it’s potential for specifically targeting tumour cells. <a 
 id="x1-55001r62"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.3.1.3 </span> <a 
 id="x1-560003"></a>Cell-Cell Adhesion and Signalling
</h5>
<!--l. 927-->
<p class="noindent">
<p>The signals mediated by tight junctions are also passed on to intracellular signalling pathways and thus <a 
href="#glo:E-cadherin">E-cadherin</a> also has a role in maintaining cellular function and growth. One such example is the regulation of <span class="cmmi-12">β</span>-catenin which interacts with both the actin</p>
cytoskeleton and acts as a transcription factor via the WNT pathway. Similarly, the Hippo and PI3K/AKT pathways are implicated in being mediated by <a 
href="#glo:E-cadherin">E-cadherin</a>, having roles in promoting cell survival, proliferation, and repressing apoptosis. <a 
href="#glo:E-cadherin">E-cadherin</a> shares several downstream pathways with signalling pathways such as integrins and thus indirectly interacts with them, particularly since feedback loops may occur in such pathways. Conversely, the multifaceted roles of <a 
href="#glo:E-cadherin">E-cadherin</a> have been shown with differing overexpression in ovarian cells promoting tumour growth, while it maintains healthy cellular functions in other cells. <a 
 id="x1-56001r60"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.3.2 </span> <a 
 id="x1-570002"></a><span class="cmti-12">CDH1 </span>as a Tumour (and Invasion) Suppressor
</h4>
<!--l. 930-->
<p class="noindent">
<a 
href="#glo:E-cadherin">E-cadherin</a> has key roles in maintaining cellular structure and regulating growth, consistent with <span class="cmti-12">CDH1 </span>being a tumour suppressor gene. Loss of <span class="cmti-12">CDH1 </span>in epithelial tissues leads to disrupted cell polarity, differentiation, and migration. <a 
href="#glo:E-cadherin">E-cadherin</a> loss has been identified as a recurrent driver tumour suppressor mutation in sporadic cancers of many tissues including breast, stomach, lung, colon, and pancreas tissue. <a 
 id="x1-57001r63"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.3.2.1 </span> <a 
 id="x1-580001"></a>Breast Cancers and Invasion
</h5>
<!--l. 933-->
<p class="noindent">
<a 
href="#glo:E-cadherin">E-cadherin</a> loss in breast cancers has been shown to cause increased proliferation, lymph node invasion, and metastasis with poor cell-cell contact. Thus <span class="cmti-12">CDH1 </span>gene has also been implicated as an invasion suppressor, with a key role in the <a 
href="#glo:EMT">epithelial-mesenchymal transition (EMT)</a>, an established mechanism of cancer progression (<a 
href="#XHanahan2011">Hanahan and Weinberg</a>, <a 
href="#XHanahan2011">2011</a>). The epithelial-mesenchymal transition is important during development and wound healing but such changes in cellular differentiation also occur in cancers. If <span class="cmti-12">CDH1 </span>is inactivated by mutation or DNA methylation (<a 
href="#XBerx1996">Berx <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBerx1996">1996</a>; <a 
href="#XGuilford1999">Guilford</a>, <a 
href="#XGuilford1999">1999</a>; <a 
href="#XMachado2001">Machado <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMachado2001">2001</a>), it is likely that <a 
href="#glo:EMT">EMT</a> will drive growth of <a 
href="#glo:E-cadherin">E-cadherin</a> deficient cancers (<a 
href="#XBerx2009">Berx and van Roy</a>, <a 
href="#XBerx2009">2009</a>; <a 
href="#XGraziano2003">Graziano <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGraziano2003">2003</a>; <a 
href="#XPolyak2009">Polyak and Weinberg</a>, <a 
href="#XPolyak2009">2009</a>). While loss of <a 
href="#glo:E-cadherin">E-cadherin</a> is not sufficient to cause <a 
href="#glo:EMT">EMT</a> or tumourigenesis, it is an important step in this mechanism of tumour progression and a potential therapeutic intervention may therefore also impede cancer progression and have activity against advanced stage cancers. <a 
 id="x1-58001r64"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.3.3 </span> <a 
 id="x1-590003"></a>Hereditary Diffuse Gastric Cancer and Lobular Breast Cancer
</h4>
<!--l. 939-->
<p class="noindent">
<span class="cmti-12">CDH1 </span>loss of function mutations also causes familial cancers, including diffuse gastric cancer and lobular breast cancer (<a 
href="#XGraziano2003">Graziano <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGraziano2003">2003</a>; <a 
href="#XGuilford2010">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford2010">2010</a>, <a 
href="#XHDGC">1999</a>; <a 
href="#XOliveira2009">Oliveira <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XOliveira2009">2009</a>). Individuals carrying a null mutation in <span class="cmti-12">CDH1 </span>have a syndromic predisposition to early-onset these cancers, known as <a 
href="#glo:HDGC">hereditary diffuse gastric cancer (HDGC)</a> (<a 
href="#XGuilford1998">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford1998">1998</a>). Due to the loss of an allele, these individuals are prone to carcinogenic lesion in the breast and stomach when the other allele is inactivated, occurring much more frequently and thus younger than in individuals without a second functional allele of <span class="cmti-12">CDH1</span>. The loss of the second allele is most often hypermethylation suppressing expression rather than mutation, although loss of heterozygousity may also occur. Therefore <a 
href="#glo:HDGC">HDGC</a> is an autosomal dominant cancer syndrome with incomplete penetrance. The “lifetime” (until age 80 years) risk for mutation carriers of diffuse gastric cancer is 70% in males and 56% in females. In addition, the lifetime risk of lobular breast cancer is 42% in female mutation carriers.
</p>
<!--l. 941-->
<p class="indent">
<a 
href="#glo:HDGC">HDGC</a> affects less than 1 in a million people globally (<a 
href="#XFerlay2015">Ferlay <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFerlay2015">2015</a>) and less than 1% of gastric cancers. However, <a 
href="#glo:HDGC">HDGC</a> is documented to affect several hundred families globally. <a 
href="#glo:E-cadherin">E-cadherin</a> mutations in the germline is implicated in 1-3% of gastric cancers presenting with a family history, varing between high and low incidence populations. <a 
href="#glo:E-cadherin">E-cadherin</a> is also mutated in 13% of sporadic gastric cancers.
</p>
<!--l. 943-->
<p class="indent">
<p>While diagnostic testing for <span class="cmti-12">CDH1 </span>genotype has enabled more effective management of <a 
href="#glo:HDGC">HDGC</a> and improved patient outcomes, there are still limited options for clinical interventions (<a 
href="#XGuilford2010">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford2010">2010</a>). Individuals with a family history of <a 
href="#glo:HDGC">HDGC</a> are recommended to be tested for <span class="cmti-12">CDH1 </span>mutations in late adolescence and are offered prophylactic stomach surgery before the risk of developing cancers increases with age. Another option is annual endoscopic screening to diagnose early stage stomach cancers with surgical intervention once they are detected (<a 
href="#XOliveira2013">Oliveira <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XOliveira2013">2013</a>). However, these early stage cancers are difficult to detect and may be missed in regular screening. Thus patients carrying <span class="cmti-12">CDH1 </span>mutations either have surgical interventions with a significant impact on quality of life and risk of complications or remain at risk of developing advanced stage stomach cancers. Due to the lower mortality rate due to stomach cancers, there is increasing concerns among these <a 
href="#glo:HDGC">HDGC</a> families on the elevated risk of lobular breast cancers for women later in life.</p>
</p>
<!--l. 945-->
<p class="indent">
The current clinical management of <a 
href="#glo:HDGC">HDGC</a> still has significant risks for patients and therefore a greater understanding of the molecular and cellular function of <span class="cmti-12">CDH1 </span>is important for its role in these cancers. Such studies may lead to alternative treament strategies such as pharmacological treatments with specificity against <span class="cmti-12">CDH1 </span>null cells, once they lose the second allele. While a loss of gene function cannot be targeted directly, designing a treatment with specifity against <span class="cmti-12">CDH1 </span>may also have activity in sporadic cancers in a range of epithelial cancers. Thus an effective treatment against <span class="cmti-12">CDH1 </span>mutant cancers would potentially have significant therapeutic and preventative applications in a large number of patients. <a 
 id="x1-59001r66"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.3.4 </span> <a 
 id="x1-600004"></a>Somatic Mutations
</h4>
<a 
 id="x1-60001r65"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">1.3.4.1 </span> <a 
 id="x1-610001"></a>Mutation Rate
</h5>
<!--l. 950-->
<p class="noindent">
Estimates for the prevalence of <span class="cmti-12">CDH1 </span>somatic mutations in sporadic cancers varies. The Cancer Gene Census (<a 
href="#XFutreal2004">Futreal <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFutreal2004">2004</a>; <a 
href="#XPleasance2010">Pleasance <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XPleasance2010">2010</a>) detected 994 distinct mutations in 10,143 tumour samples (at a rate of 7.52%), <a 
href="#XCOSMICdb">COSMIC</a> (<a 
href="#XCOSMICdb">2016</a>) detected 632 distinct mutations in 43,865 tumour samples (at a rate of 1.71%), and detected mutations in 13.2% of 53 of the NCI60 cancer cell lines. While there is no consensus on the prevalence of <span class="cmti-12">CDH1 </span>mutations, the vast variability of mutations is consistent with it’s role as a tumour supressor and it has been found to be recurrently mutated in a wide range of cancers of epithelial tissues.
</p>
<!--l. 952-->
<p class="indent">
<p><a 
href="#XCOSMICdb">COSMIC</a> (<a 
href="#XCOSMICdb">2016</a>) reports <span class="cmti-12">CDH1 </span>mutations in 40 cancer tissue types including stomach (11.40% in 1342 samples), breast (10.29% in 3343 samples), large colon (2.87%), skin (2.83%), endometrial (2.81%), and bladder (1.9%) cancer. <a 
href="#XICGC2017web">ICGC</a> (<a 
href="#XICGC2017web">2017</a>) reports <span class="cmti-12">CDH1</span> mutations in 29 cancer tissue types including skin (23.41% in 598 samples), breast (14.50% in 1696 samples), ovary (13.98% in 93 samples), and stomach (11.41% in 289 samples) cancer samples. <span class="cmti-12">CDH1 </span>mutations are reported at similar rates in breast and stomach cancer in other cancer <a 
href="#glo:genomics">genomics</a> projects and studies across distinct populations. <a 
href="#XcBioPortal">cBioPortal</a> (<a 
href="#XcBioPortal">2017</a>) reports <span class="cmti-12">CDH1 </span>mutation prevalence in stomach cancer at 16.7% (<a 
href="#XKakiuchi2014">Kakiuchi <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XKakiuchi2014">2014</a>, 30 samples), 15% (<a 
href="#XWang2014">Wang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWang2014">2014</a>, 100 samples), 14.1% (<a 
href="#XChen2015">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2015">2015</a>, 78 samples), and 9.4% (<a 
href="#XTCGA2017prov">TCGA</a>, <a 
href="#XTCGA2017prov">2017b</a>, 393 samples). <a 
href="#XcBioPortal">cBioPortal</a> (<a 
href="#XcBioPortal">2017</a>) also reports <span class="cmti-12">CDH1 </span>mutation prevalence in breast cancer at 12.7% (<a 
href="#XTCGA2017prov">TCGA</a>, <a 
href="#XTCGA2017prov">2017b</a>, 963 samples) and 10.8% (<a 
href="#XMETABRIC2012">Curtis <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMETABRIC2012">2012</a>; <a 
href="#XMETABRIC2016">Pereira <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMETABRIC2016">2016</a>, 2051 samples). The rare plasmacytoid bladder cancer subtype also has a high prevalence of <span class="cmti-12">CDH1</span></p>
mutations in <a 
href="#XCOSMICdb">COSMIC</a> (<a 
href="#XCOSMICdb">2016</a>) at a rate of 81.8% (N=33). These demonstrate that <span class="cmti-12">CDH1 </span>is important in many cancers and targeting <span class="cmti-12">CDH1 </span>may be widely applied against sporadic cancers in addition to hereditary cancers. However, some of these studies have focused on disease subgroups (such as lobular subtype or estogren receptor negative breast cancers) with poor patient outcomes which may have inflated the prevalence of <span class="cmti-12">CDH1 </span>mutations which are more common in some of these subtypes. <a 
 id="x1-61001r68"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">1.3.4.2 </span> <a 
 id="x1-620002"></a>Co-occurring Mutations
</h5>
<!--l. 956-->
<p class="noindent">
Another concern is that <span class="cmti-12">CDH1 </span>mutations may co-occur with other known cancer driver genes such as highly prevalent tumour suppressor gene <span class="cmti-12">TP53 </span>or the proto-oncogene <span class="cmti-12">PIK3CA</span>. <a 
href="#XcBioPortal">cBioPortal</a> (<a 
href="#XcBioPortal">2017</a>) reports the prevalence of the mutations in these genes at 10% for <span class="cmti-12">CDH1</span>, 49% for <span class="cmti-12">TP53</span>, 22% for <span class="cmti-12">PIK3CA </span>in stomach cancer (<a 
href="#XTCGA2017prov">TCGA</a>, <a 
href="#XTCGA2017prov">2017b</a>, 393 samples). There is no evidence of significant co-occurring mutations between <span class="cmti-12">CDH1 </span>and <span class="cmti-12">PIK3CA </span>(mutex <span class="cmmi-12">p </span>= 0<span class="cmmi-12">.</span>231) but there is evidence for significant mutually exclusive mutations for <span class="cmti-12">CDH1 </span>(mutex <span class="cmmi-12">p </span>= 0<span class="cmmi-12">.</span>002) and <span class="cmti-12">PIK3CA </span>(mutex <span class="cmmi-12">p </span>= 0<span class="cmmi-12">.</span>004) with <span class="cmti-12">TP53</span>. <a 
href="#XcBioPortal">cBioPortal</a> (<a 
href="#XcBioPortal">2017</a>) also reports the prevalence of the mutations in these genes at 13% for <span class="cmti-12">CDH1</span>, 32% for <span class="cmti-12">TP53</span>, 36% for <span class="cmti-12">PIK3CA </span>in breast cancer (<a 
href="#XTCGA2017prov">TCGA</a>, <a 
href="#XTCGA2017prov">2017b</a>, 963 samples). There is evidence of significant co-occurring mutations with <span class="cmti-12">CDH1</span> and <span class="cmti-12">PIK3CA </span>(mutex <span class="cmmi-12">p &lt; </span>0<span class="cmmi-12">.</span>0001) and evidence for significant mutually exclusive mutations for <span class="cmti-12">CDH1 </span>(mutex <span class="cmmi-12">p </span>= 0<span class="cmmi-12">.</span>003) and <span class="cmti-12">PIK3CA </span>(mutex <span class="cmmi-12">p </span>= 0<span class="cmmi-12">.</span>032) with <span class="cmti-12">TP53</span>.
</p>
<!--l. 958-->
<p class="indent">
These cancer driver mutations have distinct molecular features, leading to disease progression in distinct ways which is a concern for drug resistance when several mutations may accumulate, particularly for sporadic cancers where this is common. Targeting <span class="cmti-12">CDH1 </span>specifically is most suitable for hereditary cancers and combination therapies may be required for sporadic cancers. However, <span class="cmti-12">CDH1 </span>and <span class="cmti-12">TP53 </span>mutant cancers appear to be distinct pathways of tumour progression so the high impact of <span class="cmti-12">TP53 </span>mutation on cancer cells need not be considered for the purposes of studying <span class="cmti-12">CDH1</span>. <a 
 id="x1-62001r67"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">1.3.5 </span> <a 
 id="x1-630005"></a>Models of <span class="cmti-12">CDH1 </span>loss in cell lines
</h4>
<!--l. 961-->
<p class="noindent">
Previous work our research group has published used a model of homozygous <span class="cmti-12">CDH1</span><sup><span class="cmsy-8">-</span><span class="cmmi-8">∕</span><span class="cmsy-8">-</span></sup> null mutation in non-malignant MCF10A breast cells to show that loss of <span class="cmti-12">CDH1 </span>alone was not sufficient to induce <a 
href="#glo:EMT">EMT</a> with compensatory changes in the expression of other cell adhesion genes occurring (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>). However, <span class="cmti-12">CDH1 </span>deficient cells did manifest changes in morphology, migration, and weaker cell adhesion (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>).
</p>
<!--l. 963-->
<p class="indent">
This <span class="cmti-12">CDH1</span><sup><span class="cmsy-8">-</span><span class="cmmi-8">∕</span><span class="cmsy-8">-</span></sup> MCF10A model has been used in a genome-wide screen of 18,120 genes using <a 
href="#glo:siRNA">short interfering ribonucleic acid (siRNA)</a> and a complementary drug screen using 4,057 compounds to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners to <a 
href="#glo:E-cadherin">E-cadherin</a> (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). One of the strongest candidate pathways identified by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) were the GPCR signalling cascades, which were highly enriched by <a 
href="#glo:GO">GO</a> analysis of the candidate <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners the primary <a 
href="#glo:siRNA">siRNA</a> screen. This was supported by validation with Pertussis toxin, known to target G<sub><span class="cmmi-8">αi</span></sub> signalling (<a 
href="#XClark2004">Clark</a>, <a 
href="#XClark2004">2004</a>), as were various candidate cytoskeletal pathways by inhibition of <a 
href="#glo:JAK">Janus kinase (JAK)</a> and aurora kinase. The drug screen also produced candidates in <a 
href="#glo:HDAC">histone deacetylase (HDAC)</a> and <a 
href="#glo:PI3K">phosphoinositide 3-kinase (PI3K)</a> which were supported by validation and time course experiments. <a 
 id="x1-63001r59"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">1.4 </span> <a 
 id="x1-640004"></a>Summary and Research Direction of Thesis
</h3>
<!--l. 985-->
<p class="noindent">
<a 
href="#glo:genomics">Genomics</a> technologies and the data available from them have immense potential for understanding of genetics and improving healthcare, including identification of genes altered in cancer for molecular diagnosis, prognostic biomarkers, and therapeutic targets. This has been demonstrated with the identification of cancer genes in many cancers, distinguishing tumour subtypes by expression profiles, and the development of targeted therapies against oncogenes (such as <span class="cmti-12">BRAF </span>and tumour suppressors (such as <span class="cmti-12">BRCA1</span>). <a 
href="#glo:synthetic lethal">Synthetic lethality</a> is an important genetic interaction to study fundamental cellular functions and exploit them for biomarkers and cancer treatment. They present a means to target loss of function mutations and genetic dysregulation in tumour suppressor genes by identifying interacting partners with redundant or compensating molecular functions.
</p>
<!--l. 987-->
<p class="indent">
<p><span class="cmti-12">CDH1 </span>(encoding <a 
href="#glo:E-cadherin">E-cadherin</a>) is an example of a tumour suppressor gene implicated in sporadic breast and stomach cancers. Germline mutations in <span class="cmti-12">CDH1 </span>are also found in many patients with familial early onset cancers (<a 
href="#glo:HDGC">HDGC</a>). Discovery of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners</p>
would be contribute to an understanding on the molecular mechanisms driving the growth of <span class="cmti-12">CDH1 </span>deficient tumours and identification of potential therapeutic targets or chemopreventative agents for management of <a 
href="#glo:HDGC">HDGC</a>. The clinical potential of the <a 
href="#glo:synthetic lethal">synthetic lethal</a> approach has been demonstrated with the application of olaparib against <span class="cmti-12">BRCA1 </span>and <span class="cmti-12">BRCA2 </span>mutations <a 
href="#XLord2014">Lord <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLord2014">2015</a>) but there remains the need to systematically identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes for other tumour suppressors such as <span class="cmti-12">CDH1</span>. A <a 
href="#glo:synthetic lethal">synthetic lethal</a> screen has been conducted on breast cell lines <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) but computational approaches to identification of <a 
href="#glo:synthetic lethal">synthetic lethal</a> partners of <span class="cmti-12">CDH1 </span>remains to be done.
</p>
<!--l. 992-->
<p class="indent">
While there are a wide range of experimental and computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery, many are limited to particular applications, prone to false positives, inconsistent across independent approaches, or enriched for particular genes of interest. Therefore <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions are difficult to replicate or apply in the clinic. Computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethality</a> are not widely adopted by the cancer research community and experimental approaches cannot be combined to study <a 
href="#glo:synthetic lethal">synthetic lethality</a> at a genome-wide scale. However, these show interest in <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery in the community and the need for robust predictions of <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions in cancer and human tissues.
</p>
<!--l. 994-->
<p class="indent">
Effective screening, prediction, and analysis of <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions are a crucial part of developing next generation anti-cancer strategies. Therefore, we propose developing a computational statistical procedure to identify <a 
href="#glo:synthetic lethal">synthetic lethal</a> interactions and construct gene networks. This will enable the development of personalised medicine targeted to particular molecular aberrations. Genetic tests and <a 
href="#glo:genomics">genomics</a> have the potential to revolutionise cancer screening, diagnosis, and prognostics; targeted therapeutics, similarly, have applications in prevention and therapy of sporadic or hereditary cancers with known molecular properties.
</p>
<!--l. 996-->
<p class="indent">
<p>To address the concerns raised by recent computational approaches to <a 
href="#glo:synthetic lethal">synthetic lethal</a> discovery in cancer (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>; <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>; <a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWappett2016">2016</a>), I present similar analysis using solely gene expression data which is widely available for a large number of samples in many different cancers. This uses a statistical methodology the <a 
href="#glo:SLIPT">Synthetic Lethal Interaction Prediction Tool (SLIPT)</a> developed for this purpose. To further determine the limitations and implications of <a 
href="#glo:synthetic lethal">synthetic lethal</a> predictions, modelling and simulation was performed upon the statistical behaviour of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs in <a 
href="#glo:genomics">genomics</a> data. Comparison of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene candidates from public data analysis and experimental candidates, pathway analysis, and networks structure will also be</p>
presented to investigate the relationships between <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates. Release of R codes used for simulation, prediction, and analysis will enable adoption of the methodology in the cancer research community and comparison to existing methods.
</p>
<!--l. 998-->
<p class="indent">
My thesis aims to develop such predictions for <a 
href="#glo:synthetic lethal">synthetic lethal</a> partner genes with a focus on the example of <a 
href="#glo:E-cadherin">E-cadherin</a> to compare to the findings of <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>), develop of network approaches for pathway structure, and simulate gene expression on pathway structre with the following <a 
href="#glo:bioinformatics">bioinformatics</a> and computational biology investigations:
</p>
<ul class="itemize1">
<li class="itemize">
<!--l. 1004-->
<p class="noindent">
Developed a query-based <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection methodology (<a 
href="#glo:SLIPT">SLIPT</a>) for use on gene expression data
</p>
</li>
<li class="itemize">
<!--l. 1008-->
<p class="noindent">
Adapt this methodology to utilise somatic mutation for query genes or candidate pathway metagenes
</p>
</li>
<li class="itemize">
<!--l. 1012-->
<p class="noindent">
Apply <a 
href="#glo:synthetic lethal">Synthetic lethal</a> prediction to public breast cancer <a 
href="#glo:genomics">genomics</a> data from <a 
href="#glo:TCGA">TCGA</a> (<a 
href="#XTCGA2012">TCGA</a>, <a 
href="#XTCGA2012">2012</a>)
</p>
</li>
<li class="itemize">
<!--l. 1016-->
<p class="noindent">
Identify over-represented biological pathways using Reactome (<a 
href="#XReactome">Croft <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XReactome">2014</a>) among <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidate partner genes
</p>
</li>
<li class="itemize">
<!--l. 1020-->
<p class="noindent">
Compare these at the gene and pathway level to experimental screen data in breast cell lines from <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>)
</p>
</li>
<li class="itemize">
<!--l. 1024-->
<p class="noindent">
Replicate these analyses in stomach cancer <a 
href="#glo:genomics">genomics</a> data from <a 
href="#glo:TCGA">TCGA</a> (<a 
href="#XTCGA2014GC">Bass <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014GC">2014</a>)
</p>
</li>
<li class="itemize">
<!--l. 1028-->
<p class="noindent">
Determine whether <a 
href="#glo:synthetic lethal">synthetic lethal</a> candidates have importance in biological networks of candidate partner pathways
</p>
</li>
<li class="itemize">
<!--l. 1032-->
<p class="noindent">
Determine whether there are relationships within biological network structures between experimental and predicted gene candidate partners
</p>
</li>
<li class="itemize">
<!--l. 1036-->
<p class="noindent">
Develop a statistical model of <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene expression
</p>
</li>
<li class="itemize">
<!--l. 1040-->
<p class="noindent">
Simulate gene expression with <a 
href="#glo:synthetic lethal">synthetic lethal</a> genes and pathway structures
</p>
</li>
<li class="itemize">
<!--l. 1044-->
<p class="noindent">
Evaluate the effects of modification to the <a 
href="#glo:SLIPT">SLIPT</a> procedure on it’s statistical performance
</p>
</li>
<li class="itemize">
<!--l. 1048-->
<p class="noindent">
Compare the statistical performance of the <a 
href="#glo:SLIPT">SLIPT</a> procedure to alternative statistical methods
</p>
</li>
<li class="itemize">
<!--l. 1052-->
<p class="noindent">
<p>Release a <a 
href="#glo:synthetic lethal">synthetic lethal</a> prediction methodology (<a 
href="#glo:SLIPT">SLIPT</a>) to the research community for wider application</p>
</p>
<div class="center">
<!--l. 1057-->
<p class="noindent">
</p>
<!--l. 1058-->
<p class="noindent">
<strong><span class="cmbx-12">Thesis Aims</span></strong>
</p>
</div>
</li>
</ul>
<ul class="itemize1">
<li class="itemize">
To develop a statistical approach to detect <a 
href="#glo:synthetic lethal">synthetic lethal</a> gene pairs in cancer from expression data
</li>
<li class="itemize">
To apply this methodology to public cancer gene expression data against <span class="cmti-12">CDH1</span> and analyse pathway structure with comparisons to experimental screen data
</li>
<li class="itemize">
To construct a statistical model of <a 
href="#glo:synthetic lethal">synthetic lethality</a> in multivariate normal expression data
</li>
<li class="itemize">
To develop a simulation pipeline of expression with pathway structure on a high-performance computing cluster
</li>
<li class="itemize">
To examine the statistical performance of the methodology with simulated expression including pathways and compare it to other approaches
</li>
<li class="itemize">
To release the <a 
href="#glo:synthetic lethal">synthetic lethal</a> detection methodology and pathway simulation procedure as R software packages
</li>
</ul>
<h2 class="chapterHead">
<span class="titlemark">Chapter 2</span><br /><a 
 id="x1-650002"></a>Methods and Resources
</h2>
<!--l. 5-->
<p class="noindent">
In this Chapter, I will outline the various existing resources and methods utilised throughout this project. This includes public data repositories, stable and development releases of software packages (mostly for the R programming environment), and implementation of bioinformatics methods and statistical concepts with Shell or R scripts developed for this purpose. Methods and packages developed specifically for this project will be covered in more detail along with preliminary data to demonstrate and support their use in Chapter <a 
href="#x1-920003">3<!--tex4ht:ref: chap:methods_dev --></a>. <a 
 id="x1-65001r71"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">2.1 </span> <a 
 id="x1-660001"></a>Bioinformatics Resources for Genomics Research
</h3>
<a 
 id="x1-66001r70"></a>
<h4 class="subsectionHead">
<span class="titlemark">2.1.1 </span> <a 
 id="x1-670001"></a>Public Data and Software Packages
</h4>
<!--l. 9-->
<p class="noindent">
Various bioinformatics resources, such as databases and methods, have become integral parts of genetics and genomics research. Reference genomes, genotyped variants, gene expression, and epigenetics profiles are among the most commonly used resources. Gene expression data in particular is widely available from many microarray and RNA-Seq studies, from repositories such as Gene Expression Omnibus (GEO) (<a 
href="#XGEO2016">Clough and Barrett</a>, <a 
href="#XGEO2016">2016</a>), caArray (<a 
href="#XcaArray2014">Heiskanen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XcaArray2014">2014</a>), and ArrayExpress (<a 
href="#XArrayExpress2013">Rustici <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XArrayExpress2013">2013</a>). Such profiles are excellent resources to examine the changes of gene expression occurring in cancers and the variation between samples. These microarray initiatives have set a precedent for data sharing, data mining, and the wider benefits of publicly available data for enabling the scientific community to further utilise the data rather than a single research group or consortium (<a 
href="#XRung2013">Rung and Brazma</a>, <a 
href="#XRung2013">2013</a>). The practice of integrating findings from publicly available genomics data with the research questions and experimental results of individual research groups has carried over into RNA-Seq datasets including the large-scale cancer genomics projects (<a 
href="#XICGC2011">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XICGC2011">2011</a>). This thesis is one such example of an investigation enabled by this wider movement and tools developed in various disciplines to generate, process, and disseminate genomic-scale data.
</p>
<!--l. 11-->
<p class="indent">
<p>Along with databases, it is also becoming common practice for bioinformatics researchers to release their code as open-source or provide a software package to enable replication of the findings or further applications of the methods (<a 
href="#XStajich2006">Stajich and Lapp</a>, <a 
href="#XStajich2006">2006</a>). This is part of a wider movement in software and data analysis with many tools to facilitate such work being released for use in Linux or the R</p>
programming environment (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>). In addition to the R packages hosted on CRAN (<a 
href="#XCRAN">CRAN</a>, <a 
href="#XCRAN">2017</a>), the Bioconductor repositories (<a 
href="#XGentleman2004">Gentleman <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGentleman2004">2004</a>) also contain many packages specifically for applications in bioinformatics, and the GitHub site hosts many packages in various stages of development and early release. Packages from these various sources have been used throughout this project and cited where-ever possible. Several R packages have been developed during this thesis project and either publicly released on GitHub or prepared to accompany a publication. <a 
 id="x1-67001r69"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">2.1.1.1 </span> <a 
 id="x1-680001"></a>Cancer Genome Atlas Data
</h5>
<!--l. 14-->
<p class="noindent">
Molecular profile data from normal and tumour samples was downloaded from publicly available sources, using the <a 
href="#glo:TCGA">TCGA</a> (<a 
href="#XTCGA2012">TCGA</a>, <a 
href="#XTCGA2012">2012</a>) and the International Cancer Genome Consortium (ICGC) web portals (<a 
href="#XICGC2011">Zhang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XICGC2011">2011</a>). These include gene expression (RNA-Seq), somatic mutations, and anonymous clinical data. These versions downloaded were on the 6<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup> of August 2015 (Release 19) and the 2<sup class="textsuperscript"><span class="cmr-10x-x-109">nd</span></sup> of May 2016 (Release 20) for breast and stomach cancer respectively via the ICGC data portal (<a 
href="https://dcc.icgc.org/" class="url" ><span class="cmtt-12">https://dcc.icgc.org/</span></a>).
</p>
<!--l. 17-->
<p class="indent">
Performing a genomic alignment in remains a challenge in bioinformatics and methods to do so may yet be improved (<a 
href="#XChen2010">Chen and Tompa</a>, <a 
href="#XChen2010">2010</a>). However, the statistical and biological aspects of bioinformatics are the focus of this thesis, comparing alignment methods is outside the scope of these investigations. The <a 
href="#glo:TCGA">TCGA</a> project (<a 
href="#XTCGA2012">TCGA</a>, <a 
href="#XTCGA2012">2012</a>) used widely adopted tools: “Bowtie” for alignment (<a 
href="#Xbowtie">Langmead <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xbowtie">2009</a>), “mapslice” to detect splice sites (<a 
href="#Xmapsplice">Wang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xmapsplice">2010</a>), and the Reads Per Kilobase per Million mapped reads (RPKM) approach to qualify reads per transcript as a measure of gene expression (<a 
href="#XMortazavi2008">Mortazavi <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XMortazavi2008">2008</a>). These are widely acceptable tools for processing RNA-Seq data which were used to produce the raw counts of mapped reads (tier 1) and normalised expression data (tier 3) publicly available from TCGA.
</p>
<!--l. 19-->
<p class="indent">
<p>Raw count and RSEM normalised <a 
href="#glo:TCGA">TCGA</a> expression data from Illumina RNA-Seq protocols were available from 1,177 breast samples (113 normal, 1,057 primary tumour, and 7 metastases) for 20,501 genes. <a 
href="#glo:TCGA">TCGA</a> breast somatic mutation data for 981 samples (976 primary tumours and 5 metastases) across 25,836 genes were available including 969 samples (964 primary tumours and 5 metastases) with corresponding RNA-Seq expression data and 19,166 genes mapped from Ensembl identifiers to gene symbols, of which 16,156 had</p>
corresponding gene expression information. Unless otherwise stated, the raw counts were used for further processing rather than the RSEM normalised data (provided by <a 
href="#glo:TCGA">TCGA</a> tier 3). For the purposes of this analysis somatic mutations were reported if they were detected to non-synonymous substitutions, frameshifts, or truncations (by premature stop codons) which would likely disrupt the wild-type gene function. Normalised protein expression was used (as provided by <a 
href="#glo:TCGA">TCGA</a> tier 3), generated from reverse phase protein arrays (RPPA) for 142 antibodies targeting 115 genes for 298 <a 
href="#glo:TCGA">TCGA</a> breast samples.
</p>
<!--l. 21-->
<p class="indent">
Raw count <a 
href="#glo:TCGA">TCGA</a> expression data (<a 
href="#glo:TCGA">TCGA</a> tier 1) from Illumina RNA-Seq was also available for 450 stomach samples (35 normal, 415 primary tumour) for 20,501 genes. <a 
href="#glo:TCGA">TCGA</a> stomach mutation data was also available for 289 samples across 25807 genes, corresponding to 19436 genes with expression data. Normalised protein expression (RPPA) data was also sourced (from <a 
href="#glo:TCGA">TCGA</a> tier 3) for 201 antibodies targeting 158 genes for 357 stomach samples.
</p>
<!--l. 23-->
<p class="indent">
Cell line data was downloaded from the <a 
href="#glo:CCLE">CCLE</a> on the 7<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup> of November 2014 (<a 
href="#XBarretina2012">Barretina <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBarretina2012">2012</a>; <a 
href="#XCCLE">CCLE</a>, <a 
href="#XCCLE">2014</a>). This includes expression data (gnerated by Affymetrix U133 Plus 2.0 arrays) for 1037 cell lines across 19544 genes (last updated on the 18<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span> </sup> of October 2012), DNA copy number, somatic mutation, drug response, and sample information. Samples include 59 breast cell lines and 38 stomach cell lines. <a 
 id="x1-68001r75"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">2.1.1.2 </span> <a 
 id="x1-690002"></a>Reactome and Annotation Data
</h5>
<!--l. 28-->
<p class="noindent">
Unless otherwise specified, pathway analysis was performed for human pathway annotation from the Reactome database (version 52) with pathway gene sets derived from the <span class="cmtt-12">reactome.db </span>R package. Entrez identifiers were mapped to gene symbols or aliases to match to <a 
href="#glo:TCGA">TCGA</a> expression and mutation data using the <span class="cmtt-12">org.Hs.eg.db </span>R package. Further pathway analysis used breast cancer gene signatures from Gatza and colleagues (Gatza <span class="cmti-12">et al</span>., 2011; Gatza <span class="cmti-12">et al</span>., 2014). These gene symbols were matched to the relevant dataset and used to construct a matrix of category membership using the <span class="cmtt-12">safe </span>R package (<a 
href="#Xsafe">Barry</a>, <a 
href="#Xsafe">2016</a>). <a 
 id="x1-69001r73"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">2.2 </span> <a 
 id="x1-700002"></a>Data Handling
</h3>
<p><a 
 id="x1-70001r74"></a></p>
<h4 class="subsectionHead">
<span class="titlemark">2.2.1 </span> <a 
 id="x1-710001"></a>Normalisation
</h4>
<!--l. 34-->
<p class="noindent">
Apart from the PAM50 subtyping procedure (<a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>), which required RSEM normalised data (J.S. Parker personal communication), the analysis of the RNA-Seq data presented here was based on raw read count data. Raw read counts were log-scaled; samples removed for consistency (based on a Euclidean distance correlation matrix as described in Section <a 
href="#x1-720002">2.2.2<!--tex4ht:ref: methods:sample_qc --></a>); and the final dataset was TMM normalised (<a 
href="#XRobinson2010">Robinson and Oshlack</a>, <a 
href="#XRobinson2010">2010</a>) then processed using the <span class="cmtt-12">voom </span>function (<a 
href="#XLaw2014">Law <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLaw2014">2014</a>) in the <span class="cmtt-12">limma </span>R package (<a 
href="#Xlimma">Ritchie <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xlimma">2015</a>). Protein expression data generated from RPPA was normalised to dilution curves using the <span class="cmtt-12">SuperCurve </span>R package (<a 
href="#XJu2015">Ju <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJu2015">2015</a>; <a 
href="#XNeeley2009">Neeley <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XNeeley2009">2009</a>).
</p>
<!--l. 36-->
<p class="indent">
The microarray expression data sourced from the <a 
href="#glo:CCLE">CCLE</a> was used as provided, using the Robust Multi-array Average (RMA) and normalized by quantile normalization. <a 
 id="x1-71001r78"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.2.2 </span> <a 
 id="x1-720002"></a>Sample Triage
</h4>
<!--l. 42-->
<p class="noindent">
<p>The <a 
href="#glo:TCGA">TCGA</a> breast RNA-Seq data were assessed for batch effects using a correlation matrix of the log-transformed raw counts for which a heatmap (Euclidean distance, complete linkage) is shown in Figure <a 
href="#x1-200002r2">A.2<!--tex4ht:ref: fig:corr_map --></a>. While no major batch effects were detectable between the samples, 9 samples were excluded due to poor correlation with the remaining samples, as detailed in Table <a 
href="#x1-72007r1">2.1<!--tex4ht:ref: tab:qc --></a>. These samples showed unusual density plots compared to the rest of the dataset, and exhibited low mean read count in Figures <a 
href="#x1-72003r1">2.1<!--tex4ht:ref: fig:density --></a> and <a 
href="#x1-72006r2">2.2<!--tex4ht:ref: fig:boxplot --></a>. A heatmap showing key clinical properties of these excluded samples and their correlation with the remainder of the samples is shown in Figure <a 
href="#x1-200001r1">A.1<!--tex4ht:ref: fig:corr_map_part --></a>, and a full correlation heatmap (Figure <a 
href="#x1-200002r2">A.2<!--tex4ht:ref: fig:corr_map --></a>) shows these samples as relatively poorly correlated outliers in the bottom rows and left columns. In addition to the clustering analysis (in Appendix <a 
href="#x1-2000001">A.1<!--tex4ht:ref: appendix:sample_correlation --></a>), replicate tumour samples were also examined for sample quality in Appendix <a 
href="#x1-2010002">A.2<!--tex4ht:ref: appendix:replicate_samples --></a>. After removal of these samples, the <a 
href="#glo:TCGA">TCGA</a> dataset used for analysis consisted of the remaining 1168 samples (from 1040 patients): 1049 tumour samples, 112 normal tissue for matched samples, and 7 metastases.</p>
</p>
<!--l. 49-->
<p class="indent">
<a 
 id="x1-72003r1"></a><a 
 id="x1-72006r2"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 52-->
<p class="noindent">
<a 
 id="x1-72001r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="fig2a.png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Raw</span> <span class="cmr-10">counts</span> <span class="cmr-10">(log-scale)</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-72002r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="fig2b.png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Voom</span> <span class="cmr-10">normalised</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 2.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Read count density.</span></strong> <span class="cmr-10x-x-109">Sample density plots of raw counts on log-scale and</span> <span class="cmr-10x-x-109">voom normalised showing samples removed due to quality concerns.</span></span></p>
</div>
<!--tex4ht:label?: x1-72003r1 -->
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 73-->
<p class="noindent">
<a 
 id="x1-72004r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="fig3a.png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Mean</span> <span class="cmr-10">raw</span> <span class="cmr-10">counts</span> <span class="cmr-10">(log-scale)</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-72005r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="fig3b.png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Mean</span> <span class="cmr-10">voom</span> <span class="cmr-10">normalised</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 2.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Read count sample mean.</span></strong> <span class="cmr-10x-x-109">Boxplots of sample means for raw counts on</span> <span class="cmr-10x-x-109">log-scale and voom normalised show removed tumour samples with low mean read count.</span></span></p>
</div>
<!--tex4ht:label?: x1-72006r2 -->
</div>
<hr class="endfloat" />
<!--l. 92-->
<p class="indent">
<a 
 id="x1-72007r1"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 2.1: </span><span class="content">Excluded Samples by Batch and Clinical Characteristics.</span></p>
</div>
<!--tex4ht:label?: x1-72007r1 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-30" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-30-1g">
<col 
id="TBL-30-1" />
<col 
id="TBL-30-2" />
<col 
id="TBL-30-3" />
<col 
id="TBL-30-4" />
<col 
id="TBL-30-5" />
<col 
id="TBL-30-6" />
<col 
id="TBL-30-7" />
<col 
id="TBL-30-8" />
<col 
id="TBL-30-9" />
<col 
id="TBL-30-10" />
<col 
id="TBL-30-11" />
<col 
id="TBL-30-12" />
<col 
id="TBL-30-13" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-30-1-">
<td style="white-space:nowrap; text-align:center;" id="TBL-30-1-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><span class="cmbx-12">Tissue Source</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-1-3" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><span class="cmbx-12">Type</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-1-4" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><span class="cmbx-12">Batch</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-1-5" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><span class="cmbx-12">Plate</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-1-6" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><span class="cmbx-12">Patient</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-1-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><span class="cmbx-12">Samples</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-1-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><span class="cmbx-12">p53</span></p>
</div>
<pre><code>&lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-30-1-9&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><span class="cmbx-12">Subtype</span></p>
</div>
</td>
<td colspan="2" style="white-space:nowrap; text-align:center;" id="TBL-30-1-10" class="td11">
<pre><code>       &lt;div class=&quot;multicolumn&quot;  style=&quot;white-space:nowrap; text-align:center;&quot;&gt;&lt;span </code></pre>
class=“cmbx-12”&gt;Treatment (History)</span>
</div>
</td>
<td colspan="3" style="white-space:nowrap; text-align:center;" id="TBL-30-1-12" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><span class="cmbx-12">Clinical Subtypes (Stage)</span></p>
</div>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-30-2-">
<td style="white-space:nowrap; text-align:center;" id="TBL-30-2-1" class="td11">
A7 Christiana
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-2-2" class="td11">
Tumour
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-2-3" class="td11">
47
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-2-4" class="td11">
A227
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-2-5" class="td11">
A0DB
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-2-6" class="td11">
1 of 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-2-7" class="td11">
NA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-2-8" class="td11">
Luminal A
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-2-9" class="td11">
<pre><code>        Mastectomy                   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-30-2-10&quot;  </code></pre>
class=“td11”&gt; (no)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-2-11" class="td11">
ER+
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-2-12" class="td11">
Ductal
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-30-2-13"  
class="td11">
<ol start="2" style="list-style-type: decimal">
<li></td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-30-3-">
<td style="white-space:nowrap; text-align:center;" id="TBL-30-3-1" class="td11">
A7 Christiana
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-3-2" class="td11">
Tumour
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-3-3" class="td11">
96
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-3-4" class="td11">
A220
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-3-5" class="td11">
A13D
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-3-6" class="td11">
1 of 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-3-7" class="td11">
Wildtype
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-3-8" class="td11">
Luminal A
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-3-9" class="td11">
<pre><code>        Mastectomy                   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-30-3-10&quot;  </code></pre>
class=“td11”&gt; (no)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-3-11" class="td11">
ER+
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-3-12" class="td11">
Ductal
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-30-3-13"  
class="td11">
<ol start="2" style="list-style-type: decimal">
<li></td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-30-4-">
<td style="white-space:nowrap; text-align:center;" id="TBL-30-4-1" class="td11">
A7 Christiana
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-4-2" class="td11">
Tumour
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-4-3" class="td11">
96
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-4-4" class="td11">
A227
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-4-5" class="td11">
A13E
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-4-6" class="td11">
1 of 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-4-7" class="td11">
NA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-4-8" class="td11">
Basal
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-4-9" class="td11">
<pre><code>       Lumpectomy                  &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-30-4-10&quot;  </code></pre>
class=“td11”&gt; (no)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-4-11" class="td11">
ER<span class="cmsy-10x-x-120">- </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-4-12" class="td11">
Ductal
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-30-4-13"  
class="td11">
<ol start="2" style="list-style-type: decimal">
<li></td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-30-5-">
<td style="white-space:nowrap; text-align:center;" id="TBL-30-5-1" class="td11">
A7 Christiana
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-5-2" class="td11">
Tumour
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-5-3" class="td11">
142
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-5-4" class="td11">
A277
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-5-5" class="td11">
A26E
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-5-6" class="td11">
1 of 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-5-7" class="td11">
NA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-5-8" class="td11">
Basal
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-5-9" class="td11">
<pre><code>       Lumpectomy                  &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-30-5-10&quot;  </code></pre>
class=“td11”&gt; (no)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-5-11" class="td11">
ER+
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-5-12" class="td11">
Ductal
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-30-5-13"  
class="td11">
<ol start="2" style="list-style-type: decimal">
<li></td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-30-6-">
<td style="white-space:nowrap; text-align:center;" id="TBL-30-6-1" class="td11">
A7 Christiana
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-6-2" class="td11">
Tumour
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-6-3" class="td11">
47
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-6-4" class="td11">
A277
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-6-5" class="td11">
A0DC
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-6-6" class="td11">
1 of 2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-6-7" class="td11">
NA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-6-8" class="td11">
Luminal A
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-6-9" class="td11">
<pre><code>        Mastectomy                   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-30-6-10&quot;  </code></pre>
class=“td11”&gt; (yes)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-6-11" class="td11">
ER+
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-6-12" class="td11">
Lobular
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-30-6-13"  
class="td11">
<ol start="3" style="list-style-type: decimal">
<li></td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-30-7-">
<td style="white-space:nowrap; text-align:center;" id="TBL-30-7-1" class="td11">
A7 Christiana
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-7-2" class="td11">
Tumour
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-7-3" class="td11">
142
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-7-4" class="td11">
A220
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-7-5" class="td11">
A26I
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-7-6" class="td11">
1 of 2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-7-7" class="td11">
Mutant
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-7-8" class="td11">
Basal
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-7-9" class="td11">
<pre><code>       Lumpectomy                  &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-30-7-10&quot;  </code></pre>
class=“td11”&gt; (yes)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-7-11" class="td11">
ER<span class="cmsy-10x-x-120">- </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-7-12" class="td11">
Ductal
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-30-7-13"  
class="td11">
<ol start="2" style="list-style-type: decimal">
<li></td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-30-8-">
<td style="white-space:nowrap; text-align:center;" id="TBL-30-8-1" class="td11">
AC Intl Genomics
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-8-2" class="td11">
Tumour
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-8-3" class="td11">
177
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-8-4" class="td11">
A18M
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-8-5" class="td11">
A2QH
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-8-6" class="td11">
2 of 2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-8-7" class="td11">
Mutant
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-8-8" class="td11">
Basal
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-8-9" class="td11">
<pre><code>    Radical Mastectomy             &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-30-8-10&quot;  </code></pre>
class=“td11”&gt; (no)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-8-11" class="td11">
ER<span class="cmsy-10x-x-120">- </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-8-12" class="td11">
Metaplastic
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-30-8-13"  
class="td11">
<ol start="2" style="list-style-type: decimal">
<li></td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-30-9-">
<td style="white-space:nowrap; text-align:center;" id="TBL-30-9-1" class="td11">
AC Intl Genomics
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-9-2" class="td11">
Tumour
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-9-3" class="td11">
177
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-9-4" class="td11">
A220
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-9-5" class="td11">
A2QH
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-9-6" class="td11">
2 of 2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-9-7" class="td11">
Mutant
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-9-8" class="td11">
Basal
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-9-9" class="td11">
<pre><code>    Radical Mastectomy             &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-30-9-10&quot;  </code></pre>
class=“td11”&gt; (no)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-9-11" class="td11">
ER<span class="cmsy-10x-x-120">- </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-9-12" class="td11">
Metaplastic
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-30-9-13"  
class="td11">
<ol start="2" style="list-style-type: decimal">
<li></td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-30-10-">
<td style="white-space:nowrap; text-align:center;" id="TBL-30-10-1" class="td11">
GI ABS IUPUI
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-10-2" class="td11">
Normal
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-10-3" class="td11">
177
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-10-4" class="td11">
A16F
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-10-5" class="td11">
A2C8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-10-6" class="td11">
1 of 1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-10-7" class="td11">
NA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-10-8" class="td11">
Luminal A
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-10-9" class="td11">
Radical Mastectomy and Neoadjuvant
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-10-10" class="td11">
(no)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-10-11" class="td11">
ER+
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-30-10-12" class="td11">
Ductal
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-30-10-13"  
class="td11">
<ol start="2" style="list-style-type: decimal">
<li></td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-30-11-">
<td  style="white-space:nowrap; text-align:center;" id="TBL-30-11-1"  
class="td11">
<pre><code>              &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                                                                                                                                              &lt;/div&gt;</code></pre></li>
</ol></li>
</ol></li>
</ol></li>
</ol></li>
</ol></li>
</ol></li>
</ol></li>
</ol></li>
</ol>
</div>
<hr class="endfloat" />
<!--l. 123-->
<p class="indent">
Similarly, a correlation matrix of log-transformed raw counts was used to evaluate sample quality for <a 
href="#glo:TCGA">TCGA</a> stomach RNASeq. A tumour sample (patient 4294) was removed due to similar quality concerns leaving a final dataset for 449 samples (from 417 patients): 414 tumour samples and 35 normal tissue samples. <a 
 id="x1-72008r79"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.2.3 </span> <a 
 id="x1-730003"></a>Metagenes and the Singular Value Decomposition
</h4>
<!--l. 126-->
<p class="noindent">
A “metagene” offers a consistent signal of pathway (expression) activation or inactivation by dimension reduction of a matrix, avoiding negatively correlated genes averaging out the signal of a mean-based centroid (<a 
href="#XHuang2003">Huang <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XHuang2003">2003</a>). Constructing these pathway metagenes used gene sets for Reactome and Gatza signatures (<a 
href="#XGatza2011">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2011">2011</a>, <a 
href="#XGatza2014">2014</a>) as specified above (see Section <a 
href="#x1-690002">2.1.1.2<!--tex4ht:ref: methods:gene_set --></a>). The singular-value decomposition was performed (<span class="cmmi-12">X </span>= <span class="cmmi-12">U</span><sup><span class="cmmi-8">T</span> </sup><span class="cmmi-12">DV </span>where <span class="cmmi-12">X </span>is the data matrix of the gene set with genes <span class="cmsy-10x-x-120">× </span>samples) and the leading eigenvector (first column of <span class="cmmi-12">V </span>) corresponding to the largest singular value was used as a metagene for the pathway gene set. To ensure consistent directionality of metagene signals, the median of the gene set in each sample was calculated and correlated against the metagene with the (arbitrary) metagene sign adjusted as needed to conform with the majority of the gene set (i.e., positive correlation between metagene and the median-based centroid). To ensure that genes and pathways were weighted equally, metagenes were derived from a z-transformed dataset of gene expression and samples were scaled (by fractional ranking) for each metagene so that they were comparable on a [0<span class="cmmi-12">, </span>1] scale. <a 
 id="x1-73001r76"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">2.2.3.1 </span> <a 
 id="x1-740001"></a>Candidate Triage and Integration with Screen Data
</h5>
<!--l. 129-->
<p class="noindent">
<p>Candidate triage in combination with the experimental data was intended to integrate findings of the <a 
href="#glo:SLIPT">SLIPT</a> analysis with an ongoing experiment project (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>; <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). The first procedure to compare the <a 
href="#glo:SLIPT">SLIPT</a> gene candidates for <span class="cmti-12">CDH1 </span>with an <a 
href="#glo:siRNA">siRNA</a> experimental screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) was a direct comparison of the overlapping candidates, presented in a Venn diagram and tested with the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test. Since these candidates modestly overlapped at the gene level (even when excluding genes not contained in both datasets), further gene set over-representation analysis was</p>
performed for pathways specific to each detection approach and the intersection of the two.
</p>
<!--l. 131-->
<p class="indent">
The pathway composition of the intersection was further verified by a permutation resampling analysis (as described in Section <a 
href="#x1-820006">2.3.6<!--tex4ht:ref: methods:permutation --></a>): the same number of genes detected by <a 
href="#glo:SLIPT">SLIPT</a> were sampled randomly from the universe of genes tested by both approaches. These samplings were performed over 1 million iterations and the pathway over-representation was compared for each of the 1,652 reactome pathways. These over-representation scores (<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>) were compared the observed over-representation in the intersection of the <a 
href="#glo:SLIPT">SLIPT</a> candidates, with the proportion of resamplings with higher <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values used for empirical p-values of pathway composition. The <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test was used as an appropriation of Fisher’s exact test on a hypergeometric distribution for resampling to computationally scale pathway over-representation tests across iterations. Pathways for which no resamplings were occurred as high as the observed were reported as <span class="cmmi-12">p &lt; </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>. These empirical p-values were adjusted for multiple comparisons (FDR). Intersection size was not assumed to be constant across resamplings so similarly with the proportion of resamplings with higher or lower intersection size were used to evaluate significance of enrichment or depletion respectively (of <a 
href="#glo:siRNA">siRNA</a> candidate among <a 
href="#glo:SLIPT">SLIPT</a> candidate genes). <a 
 id="x1-74001r77"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">2.3 </span> <a 
 id="x1-750003"></a>Techniques
</h3>
<!--l. 136-->
<p class="noindent">
Various statistical, computational, and bioinformatics techniques were performed throughout this thesis. This section describes these techniques and gives the parameters used unless otherwise specified. Where relevant, the R package implementation which provided the technique will be acknowledged. <a 
 id="x1-75001r83"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.3.1 </span> <a 
 id="x1-760001"></a>Statistical Procedures and Tests
</h4>
<!--l. 140-->
<p class="noindent">
<p>As described in sections <a 
href="#x1-790004">2.3.4<!--tex4ht:ref: methods:heatmap --></a> and <a 
href="#x1-730003">2.2.3<!--tex4ht:ref: methods:metagene --></a>, the z-transform has been used to generate z-scores in various analyses in this thesis. Each row of dataset (<span class="cmmi-12">x</span>) is transformed into a scores (<span class="cmmi-12">z</span>) using the mean (<span class="cmmi-12">μ</span>) and standard deviation (<span class="cmmi-12">σ</span>) of the data such that:</p>
</p>
<center class="math-display">
<img 
src="thesis3x.png" alt="    x - &#x03BC;(x )
z = ---------
      &#x03C3;(x)  " class="math-display"  />
</center>
This generates data where each row (gene) has a mean of 0 and standard deviation of 1. Where plotted as aa heatmap, any data more than 3 standard deviations above or below the mean is plotted as 3 or <span class="cmsy-10x-x-120">-</span>3 respectively. <!--l. 143-->
<p class="indent">
Empirical Bayes differential expression analysis was performed using the <span class="cmtt-12">limma </span>R package (<a 
href="#Xlimma">Ritchie <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xlimma">2015</a>). Where specified, the Fisher’s exact test, <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test, and correlation were used to measure associations between variables (as implemented in the <span class="cmtt-12">stats </span>R package (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>)). Unless otherwise specified, Pearson’s correlation was used for correlation analyses (<span class="cmmi-12">r</span>) and coefficient of determination (<span class="cmmi-12">R</span><sup><span class="cmr-8">2</span></sup>). Where these comparisons are discussed in more detail, Fisher’s exact test and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> tests are supported by a table or Venn diagram, rendered with the <span class="cmtt-12">limma </span>R package (<a 
href="#Xlimma">Ritchie <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xlimma">2015</a>). In some analyses, correlation is furter supported by a scatter plot and a line of best fit dervied by least squares linear regression.
</p>
<!--l. 145-->
<p class="indent">
The <span class="cmtt-12">t.test </span>function (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>) has also been used to implement the t-test to compare pairs of data. Where relevant, an <a 
href="#glo:ANOVA">ANOVA</a> has been performed to report significance of multivariate predictors of outcomes, or least squares linear regression performed for the adjusted coefficient of determination (<span class="cmmi-12">R</span><sup><span class="cmr-8">2</span></sup>) and F-statistic p-value to evaluate the fit of the predictor variables. For some analyses these are supported by boxplot or violinplot visualisation (rendered in R).
</p>
<!--l. 147-->
<p class="indent">
Multiple comparisons are adjusted by the Benjamini-Hochberg procedure to control the false discovery rate (FDR) unless otherwise specified (<a 
href="#Xfdr1995">Benjamini and Hochberg</a>, <a 
href="#Xfdr1995">1995</a>). This procedure adjusts p-values to achieve an average of the proportion of false-positives among significant tests below a threshold, <span class="cmmi-12">α</span>. The more stringent Holm-Bonferroni (Holm) procedure (<a 
href="#XHolm1979">Holm</a>, <a 
href="#XHolm1979">1979</a>) was also applied in some cases to adjust for multiple comparisons and control the family-wise error rate which adjusts p-values so that the probability that any one of the tests is a false-positive (type-1 error) below a threshold, <span class="cmmi-12">α</span>. <a 
 id="x1-76001r86"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.3.2 </span> <a 
 id="x1-770002"></a>Gene Set Over-representation Analysis
</h4>
<!--l. 150-->
<p class="noindent">
Gene set enrichment over-representation was performed to test whether there is an enrichment of a gene set (such as a biological pathway) among a group of input genes. Such input genes may be predicted synthetic lethal candidates or a subset defined by clustering (in Section <a 
href="#x1-780003">2.3.3<!--tex4ht:ref: methods:clustering --></a>) or comparison with experimental candidates (see Section <a 
href="#x1-740001">2.2.3.1<!--tex4ht:ref: methods:venn_analysis --></a>). Initially, these tests were performed using the GeneSetDB web tool (<a 
href="#Xgenesetdb">Araki <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xgenesetdb">2012</a>) hosted by the University of Auckland on the Reactome pathways (<a 
href="#XReactome">Croft <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XReactome">2014</a>). Since the GeneSetDB tool used an older version of Reactome (version 40), it was difficult to directly compare with the results of other analysis (see sections <a 
href="#x1-740001">2.2.3.1<!--tex4ht:ref: methods:venn_analysis --></a> and <a 
href="#x1-820006">2.3.6<!--tex4ht:ref: methods:permutation --></a>) performed on version 52 (as described in Section <a 
href="#x1-690002">2.1.1.2<!--tex4ht:ref: methods:gene_set --></a>). Thus an implementation of the hypergeometric test in R (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>) was used to test for over-representation against Reactome (version 52) pathways. Pathways containing less than 10 genes or more than 500 (as performed in GeneSetDB by <a 
href="#Xgenesetdb">Araki <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xgenesetdb">2012</a>) were excluded before adjusting for multiple comparisons. <a 
 id="x1-77001r87"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.3.3 </span> <a 
 id="x1-780003"></a>Clustering
</h4>
<!--l. 153-->
<p class="noindent">
Clustering analysis when performed uses unsupervised hierarchical clustering with complete linkage (distance calculated from the furthest possible pairing). For correlation matrices or multivariate normal parameters (e.g., Σ), the distance metric used was Euclidean distance. For empirical or simulated gene and pathway expression data correlation distance was used, calculated by <span class="cmmi-12">distance </span>= 1 <span class="cmsy-10x-x-120">- </span><span class="cmmi-12">cor</span>(<span class="cmmi-12">t</span>(<span class="cmmi-12">x</span>)) where <span class="cmmi-12">cor </span>is Pearson’s correlation and <span class="cmmi-12">t</span>(<span class="cmmi-12">x</span>) is the transpose of the expression matrix. <a 
 id="x1-78001r88"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.3.4 </span> <a 
 id="x1-790004"></a>Heatmap
</h4>
<!--l. 156-->
<p class="noindent">
<p>Standardised z-scores of the data were used to plot heatmaps on an appropriate scale. Raw (log-scale) read counts or voom normalised counts per gene (as specified) were plotted as normalised z-scores on a [<span class="cmsy-10x-x-120">-</span>3<span class="cmmi-12">, </span>+3] blue-red scale. Similarly, correlations were plotted on a [<span class="cmsy-10x-x-120">-</span>1<span class="cmmi-12">, </span>+1] blue-red scale. These heatmaps were performed using the linkage and distance specified for the clustering performed in Section <a 
href="#x1-780003">2.3.3<!--tex4ht:ref: methods:clustering --></a>. The <span class="cmtt-12">gplots </span>R package (<a 
href="#Xgplots">Warnes <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xgplots">2015</a>) was used to generate many of the heatmaps throughout this thesis, along with a customised heatmap function (released</p>
as <span class="cmtt-12">heatmap.2x</span>). Where clearly specified, data have been split into subsets with clustering performed separately on each subset with these plotted alongside each other. <a 
 id="x1-79001r89"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.3.5 </span> <a 
 id="x1-800005"></a>Modeling and Simulations
</h4>
<!--l. 159-->
<p class="noindent">
Statistical modeling and simulations have been used to test various synthetic lethal detection procedures on simulated data. This involves constructing a statistical model of how synthetic lethality would appear in (continuous normally distributed) gene expression data. Where presented (in Section <a 
href="#x1-950001">3.2.1<!--tex4ht:ref: methods:SL_Model --></a>), the assumptions of the model are stated clearly. The model allows sampling from a multivariate normal distribution (using the <span class="cmtt-12">mvtnorm </span>R package (<a 
href="#XGenz2009">Genz and Bretz</a>, <a 
href="#XGenz2009">2009</a>; <a 
href="#Xmvtnorm">Genz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xmvtnorm">2016</a>)) to generate simulated data with known underlying synthetic lethal partners (detailed in Section <a 
href="#x1-960002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a>). We can test whether statistical procedures, including those developed in this thesis (presented in Section <a 
href="#x1-930001">3.1<!--tex4ht:ref: methods:SLIPT --></a>), are capable of detecting them upon this simulated data. This multivariate normal simulation procedure also enables the inclusion of correlation structure which is either given as correlated blocks of genes or derived from pathway structures (as detailed in Section <a 
href="#x1-1070002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>).
</p>
<!--l. 161-->
<p class="indent">
If this multivariate normal distribution was sampled once and the procedure to add known synthetic lethal partners was performed, it generates a simulated dataset. Performing this simulation procedure and testing with a synthetic lethal detection procedure iteratively, these simulations can be used to assess the statistical performance of the detection procedure. The number of iterations (<span class="cmtt-12">Reps</span>) will be given for each simulation result. Typically, these are performed 1000 or 10,000 times depending on computational feasibility of doing so on larger datasets.
</p>
<!--l. 163-->
<p class="indent">
<p>Several measures of statistical performance were used to assess the simulations. The following measures used the final classification of the detection procedure, statistical significance for <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>, significance and directional criteria met for <a 
href="#glo:SLIPT">SLIPT</a> (see Section <a 
href="#x1-930001">3.1<!--tex4ht:ref: methods:SLIPT --></a>), and an arbitrary threshold: <span class="cmmi-12">&lt; </span><span class="cmsy-10x-x-120">-</span>0<span class="cmmi-12">.</span>2 and <span class="cmmi-12">&gt; </span>+0<span class="cmmi-12">.</span>2 for negative correlation and correlation respectively. Sensitivity (or “true positive rate”) was measured as the proportion of known synthetic lethal partners predicted to be synthetic lethal. Specificity (or “true negative rate”) was measured as the proportion of known non-synthetic lethal partners predicted not to be synthetic lethal. The “false discovery rate” (also used in adjusting for multiple comparisons) was measured here as the proportion of known non-synthetic lethal partners out of all putative partners</p>
predicted by the detection procedure. Statistical “accuracy” is the proportion of true predictions for a detection procedure, which is both the correctly predicted known synthetic lethal partners and correctly negative known non-synthetic lethal partners. <a 
 id="x1-80001r84"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">2.3.5.1 </span> <a 
 id="x1-810001"></a>Receiver Operating Characteristic (Performance)
</h5>
<!--l. 166-->
<p class="noindent">
A more general procedure to measure the statistical performance of a simulation is the Receiver Operating Characteristic (ROC) curve which does not assume a threshold for classification of synthetic lethality but demonstrates the trade-off of sensitivity and specificity (<a 
href="#XAkobeng2007">Akobeng</a>, <a 
href="#XAkobeng2007">2007</a>; <a 
href="#XFawcett2006">Fawcett</a>, <a 
href="#XFawcett2006">2006</a>; <a 
href="#XZweig1993">Zweig and Campbell</a>, <a 
href="#XZweig1993">1993</a>). These curves (implemented with the <span class="cmtt-12">ROCR </span>R package (<a 
href="#XROCR">Sing <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XROCR">2005</a>)) plot the True Positive Rate (sensitivity) against the False Positive Rate (1<span class="cmsy-10x-x-120">-</span>specificity) as the prediction threshold is varied. An ideal detection method will have a true positive rate of 1 and a false positive rate of 0, hence the Area Under the ROC curve (AUC or AUROC) is a measure of statistical performance for a detection procedure accounting for this trade-off. AUROC values are typically range from 0.5 the value expected by random chance to 1 for an optimal detection method, however it is possible for an AUROC below 0.5 for a poor detection method that performs worse than random chance. An cancer biology, an AUROC of approximately 0<span class="cmmi-12">.</span>8 is a predictive biomarker suitable for publication (<a 
href="#XHajian-Tilaki2013">Hajian-Tilaki</a>, <a 
href="#XHajian-Tilaki2013">2013</a>) but predictors with lower AUROC values may still be informative depending on the context. In this thesis, the AUROC values varies widely across simulation parameters and a primarily used for comparisons across these parameters, although they can also be used to refine thresholds for optimal classification. <a 
 id="x1-81001r90"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.3.6 </span> <a 
 id="x1-820006"></a>Resampling Analysis
</h4>
<!--l. 169-->
<p class="noindent">
<p>Resampling analyses (e.g., “permutation” analysis) are used to statistically test the significance of an observation without assuming the underlying distribution of expected test statistics <a 
href="#XCollingridge2013">Collingridge</a> (<a 
href="#XCollingridge2013">2013</a>). Instead these are derived from randomly shuffling test statistics or randomly sampling predicted candidates. For the purposes of this thesis, this involved randomly sampling genes from those tested to be analysed as putative synthetic lethal candidates. This was performed both for testing the significance of pathway composition in the intersection with experimental gene candidates (Section <a 
href="#x1-740001">2.2.3.1<!--tex4ht:ref: methods:venn_analysis --></a>) and for</p>
assessing the significance of pathway structure among synthetic lethal candidates (Section <a 
href="#x1-1050001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>).
</p>
<!--l. 171-->
<p class="indent">
These were analysed to compare the observed synthetic lethal genes against values derived from randomly sampling the same number of genes as observed by synthetic lethal from among the genes tested. Sampling iteratively across many resampling procedures, these resampling-based values form a null distribution that would be observed if the null hypothesis were true. Thus the proportion of resampling-based values across these iterations that are greater than or equal to that observed, forms an empirically derived p-value to test significance.
</p>
<!--l. 173-->
<p class="indent">
Resampling was performed for comparison (in Section <a 
href="#x1-740001">2.2.3.1<!--tex4ht:ref: methods:venn_analysis --></a>) with fixed experimental screen candidates (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) both resampling the number of genes overlapping with the screen candidates and test statistics for pathway enrichment. Resampling analysis was also applied to shortest paths and network metrics (in Section <a 
href="#x1-1050001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>) to test significance of directional relationships between synthetic lethal candidate genes within pathway structures.
</p>
<!--l. 175-->
<p class="indent">
The number of iterations determines the accuracy of these p-values. For pathway composition (in Section <a 
href="#x1-740001">2.2.3.1<!--tex4ht:ref: methods:venn_analysis --></a>), a million iterations were performed using high performance computing (as detailed in Section <a 
href="#x1-910003">2.5.3<!--tex4ht:ref: methods:HPC --></a>) to provide sufficient accuracy after adjusting for multiple comparisons across pathways. For the purposes of network analysis (in Section <a 
href="#x1-1050001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>), a thousand iterations were sufficient to reject the null hypothesis for the majority of pathways tested before adjusting for multiple comparisons, and thus further iterations were not performed. <a 
 id="x1-82001r85"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">2.4 </span> <a 
 id="x1-830004"></a>Pathway Structure Methods
</h3>
<a 
 id="x1-83001r92"></a>
<h4 class="subsectionHead">
<span class="titlemark">2.4.1 </span> <a 
 id="x1-840001"></a>Network and Graph Analysis
</h4>
<!--l. 181-->
<p class="noindent">
<p>Networks are important in considering the structure of relationships in molecular biology, including gene regulation, kinase cellular signaling, and metabolic pathways (<a 
href="#XBarabasi2004">Barabási and Oltvai</a>, <a 
href="#XBarabasi2004">2004</a>). Network theory is an interdisciplinary field which combines the approaches of computer science with the metrics and fundamental principles of graph theory, an area of pure mathematics dealing with relationships between sets of discrete elements. The vast amounts of molecular and cellular data from high-throughput technologies have enabled the application of network-based and genome-wide bioinformatics analysis to examine the complexity of a cell at the molecular level and understand</p>
aberrations in cancer. This thesis uses various metrics and analysis procedures developed in Graph and Network theory to analyse graph structure of biological pathways. Where feasible, these have been implemented using the <span class="cmtt-12">igraph </span>R package with such procedures described below (<a 
href="#Xigraph">Csardi and Nepusz</a>, <a 
href="#Xigraph">2006</a>). Custom R functions to perform more complex analysis and visualisation of iGraph data objects will be described later.
</p>
<!--l. 183-->
<p class="indent">
Graph theory is a branch of pure mathematics which deals with the properties of sets of discrete objects (referred to as a ‘node’ or ‘vertex‘) with some pairs are joined (by a ‘link’ or an ‘edge‘). While a seemingly reductionist abstraction to mathematically study relationships, graph theory serves has applications in a wide range of studies including life sciences. Network theory is the sub-discipline of graph theory which deals with networks which has become popular due to the vast potential for applications of networks (<a 
href="#XvanSteen2010">van Steen</a>, <a 
href="#XvanSteen2010">2010</a>).
</p>
<!--l. 185-->
<p class="indent">
Applications vary depending on the situation modelled, particularly in how the edges between vertices are defined, whether they are directed or weighted, and whether multiple redundant edges between a pair of vertices (referred to as ‘parallel edges‘) or edges connecting a vertex to itself (referred to as ‘loops‘) are permitted in the model. Networks are defined such that the edges represent a relationship between the vertices and may be directed, weighted, or contain parallel edges or loops depending on the application (<a 
href="#XvanSteen2010">van Steen</a>, <a 
href="#XvanSteen2010">2010</a>). Unless otherwise stated, graph structures and networks in thesis will be unweighted and have no parallel edges or loops. Where a directional relationship is known or modelled, it will be represented with a directed edge in a directed graph. <a 
 id="x1-84001r94"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.4.2 </span> <a 
 id="x1-850002"></a>Sourcing Graph Structure Data
</h4>
<!--l. 188-->
<p class="noindent">
<p>Pathway Commons interaction data was sourced using <a 
href="#glo:BioPAX">Biological pathway exchange (BioPAX)</a> with the paxtools-4.3.0 Java application on October 6<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup> 2015 (<a 
href="#XPathwayCommons">Cerami <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XPathwayCommons">2011</a>; <a 
href="#Xpaxtools">Demir <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xpaxtools">2013</a>). This utility was used to source ‘sif’ format interaction data into R (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>), from which the human Reactome (version 52) dataset of interactions was imported (<a 
href="#XReactome">Croft <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XReactome">2014</a>), matching those used for pathway enrichment analysis. These interactions were used to construct an adjacency matrix for the Reactome network and subnetworks corresponding to each relevant biological pathway. <a 
 id="x1-85001r95"></a></p>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.4.3 </span> <a 
 id="x1-860003"></a>Constructing Pathway Subgraphs
</h4>
<!--l. 191-->
<p class="noindent">
Subgraphs for each relevant pathway were constructed by matching the nodes in the complete Reactome network to the pathway gene sets (as derived in Section <a 
href="#x1-690002">2.1.1.2<!--tex4ht:ref: methods:gene_set --></a>). A subgraph with adjacent nodes was constructed by adding nodes which have an edge with a gene in the pathway gene set. The pathways these adjacent nodes belong to were added to form a “meta-pathway” to account for the possibility for nodes within the pathway being linked by the surrounding graph structure. <a 
 id="x1-86001r96"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">2.4.4 </span> <a 
 id="x1-870004"></a>Network Analysis Metrics
</h4>
<!--l. 194-->
<p class="noindent">
The existing network analysis measures applied in this thesis (as described below) used an implementation in the <span class="cmtt-12">igraph </span>R package where it was available (<a 
href="#Xigraph">Csardi and Nepusz</a>, <a 
href="#Xigraph">2006</a>). Otherwise, custom features were developed for analysis of iGraph objects in R and released as <span class="cmtt-12">igraph.extensions </span>(as described in Section <a 
href="#x1-1110003">3.5.3<!--tex4ht:ref: methods:igraph_extensions --></a>).
</p>
<!--l. 196-->
<p class="indent">
Vertex degree is the number of edges a node has and is a fundamental measure of the importance and connectivity of a network (<a 
href="#XvanSteen2010">van Steen</a>, <a 
href="#XvanSteen2010">2010</a>). More connected nodes, such as network hubs, will have a higher vertex degree relative to other nodes. For the purposes of this thesis, vertex degree ignored edge direction with loops (edges with itself) and double edges to the same node excluded.
</p>
<!--l. 198-->
<p class="indent">
A fundamental concept in network analysis is a “shortest path”, that is the shortest route via edges between any two particular nodes in a network. These are computed by Dijstra’s algorithm (<a 
href="#XDijkstra1959">Dijkstra</a>, <a 
href="#XDijkstra1959">1959</a>) in the <span class="cmtt-12">igraph </span>R package (<a 
href="#Xigraph">Csardi and Nepusz</a>, <a 
href="#Xigraph">2006</a>). Where applicable paths will only use directed edges in a particular direction. Shortests paths are a useful measure of how close nodes are in a network. This is used to compute information centrality, and for further analysis of pathway structure (as described in Section <a 
href="#x1-1040001">3.4.1<!--tex4ht:ref: methods:pathway_str --></a>).
</p>
<!--l. 200-->
<p class="indent">
<p>Network centrality is an alternative measure of the importance or influence of a node to the graph structure (<a 
href="#XBorgatti2005">Borgatti</a>, <a 
href="#XBorgatti2005">2005</a>). Various strategies are used to derive centrality, typically based on how connected the node is or the impact of node removal on the connectivity of the network. One of the most notable is the “PageRank” algorithm, a refinement of eigenvector centrality based on the eigenvectors of the adjacency matrix (<a 
href="#XBrin1998">Brin and Page</a>, <a 
href="#XBrin1998">1998</a>). This is implemented in the <span class="cmtt-12">igraph </span>R package (<a 
href="#Xigraph">Csardi and Nepusz</a>, <a 
href="#Xigraph">2006</a>).</p>
</p>
<!--l. 202-->
<p class="indent">
Another network centrality measure that has been previously applied to biological protein interaction networks (<a 
href="#XKranthi2013">Kranthi <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XKranthi2013">2013</a>) is the “information centrality”. The information centrality of a node is the relative impact on efficiency (transmission of information via shortest paths) of the network when the node is removed. That is the centrality (<span class="cmmi-12">C</span>) (<a 
href="#XKranthi2013">Kranthi <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XKranthi2013">2013</a>) for node <span class="cmmi-12">n </span>in graph <span class="cmmi-12">G </span>is defined as:
</p>
<center class="math-display">
<img 
src="thesis4x.png" alt="                   &#x2032;
Cn =  E-(G)---E-(G-)
          E (G )  " class="math-display"  />
</center>
where <span class="cmmi-12">G</span><span class="cmsy-10x-x-120">′ </span>is the subgraph with the node removed and <span class="cmmi-12">E </span>is the efficiency (<a 
href="#XLatora2001">Latora and Marchiori</a>, <a 
href="#XLatora2001">2001</a>) derived from shortest paths (<span class="cmmi-12">d</span><sub><span class="cmmi-8">ij</span></sub> between nodes <span class="cmmi-12">i </span>and <span class="cmmi-12">j</span>) as:
<center class="math-display">
<img 
src="thesis5x.png" alt="         ---2----- &#x2211;   -1-
E (G) =  N (N  - 1      dij
                  i&#x003C;j&#x2208;G  " class="math-display"  />
</center>
The efficiency of the network can be derived from shortest paths implemented in the <span class="cmtt-12">igraph </span>R package and the iterative network centrality computation of each node has been released as an R package (<span class="cmtt-12">info.centrality</span>) and included in the <span class="cmtt-12">igraph.extensions</span> package. <a 
 id="x1-87001r93"></a>
<h3 class="sectionHead">
<span class="titlemark">2.5 </span> <a 
 id="x1-880005"></a>Implementation
</h3>
<p><a 
 id="x1-88001r97"></a></p>
<h4 class="subsectionHead">
<span class="titlemark">2.5.1 </span> <a 
 id="x1-890001"></a>Computational Resources and Linux Utilities
</h4>
<!--l. 213-->
<p class="noindent">
Several computers were used to process and store data during this thesis (as summarised in Table <a 
href="#x1-89001r2">2.2<!--tex4ht:ref: tab:computers --></a>), running different versions of Linux operating systems, including a personal laptop computer, laboratory desktop machine, departmental server, and the New Zealand eScience Infrastructure Intel Pan high-performance computing cluster (a supercomputer based at the University of Auckland). Each of these systems support a 64-bit architecture. Current workflows on local machines use Elementary OS (based on the Ubuntu versions given in Table <a 
href="#x1-89001r2">2.2<!--tex4ht:ref: tab:computers --></a>) and interacting with these via ZSH shell. However, Ubuntu OS and the Bourne Again SHell (bash) were used at the inception of this project and bash is continues to be used for running scripts. Various Linux applications and command-line utilities were used on these machines (as summarised in Table <a 
href="#x1-89002r3">2.3<!--tex4ht:ref: tab:computers_linux --></a>). As such, the workflows developed in this project should be backwards-compatible with Ubuntu Linux (and other derivatives). The majority of novel methodology and implementations were performed in R which is a cross-platform language, packages developed in R will be available for users of Linux, Mac, and Windows machines.
</p>
<div class="table">


<!--l. 216-->
<p class="indent">
<a 
 id="x1-89001r2"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 2.2: </span><span class="content">Computers used during Thesis</span></p>
</div>
<!--tex4ht:label?: x1-89001r2 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-31" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
>
<colgroup id="TBL-31-1g">
<col 
id="TBL-31-1" />
</colgroup>
<colgroup id="TBL-31-2g">
<col 
id="TBL-31-2" />
</colgroup>
<colgroup id="TBL-31-3g">
<col 
id="TBL-31-3" />
</colgroup>
<colgroup id="TBL-31-4g">
<col 
id="TBL-31-4" />
</colgroup>
<colgroup id="TBL-31-5g">
<col 
id="TBL-31-5" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-31-1-">
<td style="white-space:nowrap; text-align:right;" id="TBL-31-1-1" class="td11">
<pre><code>                &lt;div class=&quot;multicolumn&quot;  style=&quot;white-space:nowrap; text-align:right;&quot;&gt;&lt;/div&gt; &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-31-1-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Viao Laptop</span></p>
</div>
<pre><code>                                                                   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-31-1-3&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Lab Machine</span></p>
</div>
<pre><code>                                                                   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-31-1-4&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Biochem Server</span></p>
</div>
<pre><code>                                                               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-31-1-5&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">NeSI Pan Cluster</span></p>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-31-2-">
<td style="white-space:nowrap; text-align:right;" id="TBL-31-2-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-31-3-">
<td style="white-space:nowrap; text-align:right;" id="TBL-31-3-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p>Operating System (OS)</p>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-31-3-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-32" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-32-1g">
<col 
id="TBL-32-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-32-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-32-1-1" class="td00">
Elementary OS
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-32-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-32-2-1" class="td00">
Freya 0.3.2
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-31-3-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-33" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-33-1g">
<col 
id="TBL-33-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-33-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-33-1-1" class="td00">
Elementary OS
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-33-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-33-2-1" class="td00">
Loki 0.4
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-31-3-4"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-34" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-34-1g">
<col 
id="TBL-34-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-34-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-34-1-1" class="td00">
Red Hat Enterprise
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-34-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-34-2-1" class="td00">
Maipo 7.2
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-31-3-5"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-35" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-35-1g">
<col 
id="TBL-35-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-35-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-35-1-1" class="td00">
Cent OS
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-35-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-35-2-1" class="td00">
Final 6.4
</td>
</tr>
</table>
</div>
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-31-4-">
<td  style="white-space:nowrap; text-align:right;" id="TBL-31-4-1"  
class="td11">
<pre><code>     &lt;div class=&quot;multicolumn&quot;  style=&quot;white-space:nowrap; text-align:right;&quot;&gt;Upstream OS&lt;/div&gt; &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-31-4-2&quot;  </code></pre>
class=“td11”&gt; <!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-36" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-36-1g">
<col 
id="TBL-36-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-36-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-36-1-1" class="td00">
Ubuntu LTS
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-36-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-36-2-1" class="td00">
Trusty 14.04
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-31-4-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-37" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-37-1g">
<col 
id="TBL-37-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-37-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-37-1-1" class="td00">
Ubuntu LTS
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-37-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-37-2-1" class="td00">
Xenial 16.04
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-31-4-4" class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-31-4-5&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-31-5-">
<td style="white-space:nowrap; text-align:right;" id="TBL-31-5-1" class="td11">
<pre><code>      &lt;div class=&quot;multicolumn&quot;  style=&quot;white-space:nowrap; text-align:right;&quot;&gt;Linux Kernel&lt;/div&gt; &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-31-5-2&quot;  </code></pre>
class=“td11”&gt; 3.19.0-65-generic
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-31-5-3" class="td11">
4.4.0-36-generic
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-31-5-4" class="td11">
3.10.0-327.36.2.el7.x86_64
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-31-5-5" class="td11">
2.6.32-504.16.2.el6.x86_64
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-31-6-">
<td style="white-space:nowrap; text-align:right;" id="TBL-31-6-1" class="td11">
<pre><code>        &lt;div class=&quot;multicolumn&quot;  style=&quot;white-space:nowrap; text-align:right;&quot;&gt;Shell: bash&lt;/div&gt; &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-31-6-2&quot;  </code></pre>
class=“td11”&gt; 4.3.11(1)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-31-6-3" class="td11">
4.3.46(1)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-31-6-4" class="td11">
4.2.46(1)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-31-6-5" class="td11">
4.2.1(1)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-31-7-">
<td style="white-space:nowrap; text-align:right;" id="TBL-31-7-1" class="td11">
<pre><code>         &lt;div class=&quot;multicolumn&quot;  style=&quot;white-space:nowrap; text-align:right;&quot;&gt;Shell: zsh&lt;/div&gt; &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-31-7-2&quot;  </code></pre>
class=“td11”&gt; 5.0.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-31-7-3" class="td11">
5.1.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-31-7-4" class="td11">
5.0.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-31-7-5" class="td11">
5.2
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-31-8-">
<td style="white-space:nowrap; text-align:right;" id="TBL-31-8-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-31-9-">
<td  style="white-space:nowrap; text-align:right;" id="TBL-31-9-1"  
class="td11">
<pre><code>                  &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                                                                                                                                                                                                                                                                                                                                   &lt;/div&gt;
                                                                                
                                                                                </code></pre>
</div>
<hr class="endfloat" />
</div>
<div class="table">


<!--l. 246-->
<p class="indent">
<a 
 id="x1-89002r3"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 2.3: </span><span class="content">Linux Utilities and Applications used during Thesis</span></p>
</div>
<!--tex4ht:label?: x1-89002r3 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-38" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
>
<colgroup id="TBL-38-1g">
<col 
id="TBL-38-1" />
<col 
id="TBL-38-2" />
</colgroup>
<colgroup id="TBL-38-3g">
<col 
id="TBL-38-3" />
</colgroup>
<colgroup id="TBL-38-4g">
<col 
id="TBL-38-4" />
</colgroup>
<colgroup id="TBL-38-5g">
<col 
id="TBL-38-5" />
</colgroup>
<colgroup id="TBL-38-6g">
<col 
id="TBL-38-6" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-38-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-1-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">

</div>
<pre><code>                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-1-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">

</div>
<pre><code>                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-1-3&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Viao Laptop</span></p>
</div>
<pre><code>                                                                   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-1-4&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Lab Machine</span></p>
</div>
<pre><code>                                                                   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-1-5&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Biochem Server</span></p>
</div>
<pre><code>                                                               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-1-6&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">NeSI Pan Cluster</span></p>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-2-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-38-3-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-38-3-1"  
class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">

</div>
<pre><code>                 &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-3-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>OS</p>
</div>
<pre><code>                  &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-3-3&quot;  </code></pre>
class=“td11”&gt; <!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-39" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-39-1g">
<col 
id="TBL-39-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-39-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-39-1-1" class="td00">
Elementary OS
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-39-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-39-2-1" class="td00">
Freya 0.3.2
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-38-3-4"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-40" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-40-1g">
<col 
id="TBL-40-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-40-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-40-1-1" class="td00">
Elementary OS
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-40-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-40-2-1" class="td00">
Loki 0.4
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-38-3-5"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-41" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-41-1g">
<col 
id="TBL-41-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-41-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-41-1-1" class="td00">
Red Hat Enterprise
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-41-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-41-2-1" class="td00">
Maipo 7.2
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-38-3-6"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-42" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-42-1g">
<col 
id="TBL-42-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-42-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-42-1-1" class="td00">
Cent OS
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-42-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-42-2-1" class="td00">
Final 6.4
</td>
</tr>
</table>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-4-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">

</div>
<pre><code>                 &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-4-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>Linux Kernel</p>
</div>
<pre><code>        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-4-3&quot;  </code></pre>
class=“td11”&gt; 3.19.0-65-generic
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-4-4" class="td11">
4.4.0-36-generic
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-4-5" class="td11">
3.10.0-327.36.2.el7.x86_64
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-4-6" class="td11">
2.6.32-504.16.2.el6.x86_64
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-5-1" class="td11">
Scripting
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-5-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>Shell bash</p>
</div>
<pre><code>          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-5-3&quot;  </code></pre>
class=“td11”&gt; 4.3.11(1)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-5-4" class="td11">
4.3.46(1)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-5-5" class="td11">
4.2.46(1)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-5-6" class="td11">
4.2.1(1)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-6-1" class="td11">
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-6-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>Shell zsh</p>
</div>
<pre><code>            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-6-3&quot;  </code></pre>
class=“td11”&gt; 5.0.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-6-4" class="td11">
5.1.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-6-5" class="td11">
5.0.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-6-6" class="td11">
5.2
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-7-1" class="td11">
Programming
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-7-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>Python</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-7-3" class="td11">
2.7.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-7-4" class="td11">
2.7.12
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-7-5" class="td11">
2.7.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-7-6" class="td11">
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-38-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-8-1" class="td11">
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-8-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>Java</p>
</div>
<pre><code>                &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-8-3&quot;  </code></pre>
class=“td11”&gt; 1.8.0_101
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-8-4" class="td11">
9-ea
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-8-5" class="td11">
1.8.0_101
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-38-8-6"  
class="td11">
<pre><code>                                                                     &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-9-1" class="td11">
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-9-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>C++</p>
</div>
<pre><code>                &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-9-3&quot;  </code></pre>
class=“td11”&gt; 4.8.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-9-4" class="td11">
5.4.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-9-5" class="td11">
4.8.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-9-6" class="td11">
4.4.7
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-10-1" class="td11">
Text Editor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-10-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>nano</p>
</div>
<pre><code>                &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-10-3&quot;  </code></pre>
class=“td11”&gt; 2.2.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-10-4" class="td11">
2.5.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-10-5" class="td11">
2.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-10-6" class="td11">
2.0.9
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-11-1" class="td11">
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-11-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>kile (<span class="LATEX">L<span class="A">A</span><span class="TEX">T<span class="E">E</span>X</span></span>)</p>
</div>
<pre><code>       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-11-3&quot;  </code></pre>
class=“td11”&gt; 2.1.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-11-4" class="td11">
2.1.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-11-5" class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-11-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-12-1" class="td11">
Version Control
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-12-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>git</p>
</div>
<pre><code>                  &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-12-3&quot;  </code></pre>
class=“td11”&gt; 1.9.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-12-4" class="td11">
2.11.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-12-5" class="td11">
1.7.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-12-6" class="td11">
1.8.3.1
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-13-1" class="td11">
Shell Utilities
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-13-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>sed</p>
</div>
<pre><code>                  &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-13-3&quot;  </code></pre>
class=“td11”&gt; 4.4.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-13-4" class="td11">
4.4.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-13-5" class="td11">
4.4.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-13-6" class="td11">
4.4.1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-14-1" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-14-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>grep</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-14-3" class="td11">
2.16-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-14-4" class="td11">
2.25-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-14-5" class="td11">
2.20
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-14-6" class="td11">
2.6.3
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-15-1" class="td11">
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-15-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>nohup</p>
</div>
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-15-3&quot;  </code></pre>
class=“td11”&gt; 8.21
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-15-4" class="td11">
8.25
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-15-5" class="td11">
8.22
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-15-6" class="td11">
8.4
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-16-1" class="td11">
Typesetting
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-16-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="TEX">T<span class="E">E</span>X</span></p>
</div>
<pre><code>                &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-16-3&quot;  </code></pre>
class=“td11”&gt; 3.1415926
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-16-4" class="td11">
3.14159265
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-16-5" class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-16-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-17-1" class="td11">
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-17-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>TexLive (<span class="LATEX">L<span class="A">A</span><span class="TEX">T<span class="E">E</span>X</span></span>)</p>
</div>
<pre><code>   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-17-3&quot;  </code></pre>
class=“td11”&gt; 2013
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-17-4" class="td11">
2015
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-17-5" class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-17-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-18-1" class="td11">
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-18-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>PDF<span class="TEX">T<span class="E">E</span>X</span></p>
</div>
<pre><code>           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-18-3&quot;  </code></pre>
class=“td11”&gt; 2.5-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-18-4" class="td11">
2.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-18-5" class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-18-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-19-1" class="td11">
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-19-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>pandoc</p>
</div>
<pre><code>              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-19-3&quot;  </code></pre>
class=“td11”&gt; 1.12.2.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-19-4" class="td11">
1.16.0.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-19-5" class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-19-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-20-1" class="td11">
Remote Computing
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-20-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>slurm scheduler</p>
</div>
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-20-3&quot;  </code></pre>
class=“td11”&gt;
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-20-4" class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-20-5&quot;  </code></pre>
class=“td11”&gt;
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-20-6" class="td11">
16.05.6
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-21-1" class="td11">
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-21-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>OpenSSH</p>
</div>
<pre><code>           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-21-3&quot;  </code></pre>
class=“td11”&gt; 7.2p2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-21-4" class="td11">
7.2p2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-21-5" class="td11">
6.6.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-21-6" class="td11">
5.3p1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-22-1" class="td11">
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-22-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>OpenSSL</p>
</div>
<pre><code>            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-22-3&quot;  </code></pre>
class=“td11”&gt; 1.0.2g
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-22-4" class="td11">
1.0.2g
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-22-5" class="td11">
1.0.01e-fips
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-22-6" class="td11">
1.0.01e-fips
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-38-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-23-1" class="td11">
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-23-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>rsync</p>
</div>
<pre><code>                &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-23-3&quot;  </code></pre>
class=“td11”&gt; 3.1.0p31
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-23-4" class="td11">
3.1.1p31
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-23-5" class="td11">
3.0.9p30
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-38-23-6"  
class="td11">
<pre><code>                                                                     &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-24-1" class="td11">
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-24-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>Globus Online Transfer</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-24-3" class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-24-4&quot;  </code></pre>
class=“td11”&gt;
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-24-5" class="td11">
3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-24-6" class="td11">
3.1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-25-1" class="td11">
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-25-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>Cisco AnyConnect VPN</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-25-3" class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-25-4&quot;  </code></pre>
class=“td11”&gt; 3.1.05170
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-25-5" class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-25-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-26-1" class="td11">
Image Processing
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-26-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>Inkscape</p>
</div>
<pre><code>            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-26-3&quot;  </code></pre>
class=“td11”&gt; 0.48.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-26-4" class="td11">
0.91
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-26-5" class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-26-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-38-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-27-1" class="td11">
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-27-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>GIMP</p>
</div>
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-27-3&quot;  </code></pre>
class=“td11”&gt; 2.8.10
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-27-4" class="td11">
2.8.16
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-27-5" class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-27-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-38-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-38-28-1" class="td11">
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-28-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>ImageMagick</p>
</div>
<pre><code>        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-28-3&quot;  </code></pre>
class=“td11”&gt; 6.7.7.10-6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-38-28-4" class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-38-28-5&quot;  </code></pre>
class=“td11”&gt;
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-38-28-6"  
class="td11">
<pre><code>                                                                     &lt;/td&gt;</code></pre>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-38-29-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-38-29-1"  
class="td11">
<pre><code>               &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                                                                                                                                                                                                                                                                                                                                                               &lt;/div&gt;
                                                                                
                                                                                </code></pre>
</div>
<hr class="endfloat" />
</div>
<a 
 id="x1-89003r99"></a>
<h4 class="subsectionHead">
<span class="titlemark">2.5.2 </span> <a 
 id="x1-900002"></a>R Language and Packages
</h4>
<!--l. 344-->
<p class="noindent">
The R programming language has been used for the majority of this thesis. Current R installations across the machines used are given in Table <a 
href="#x1-90001r4">2.4<!--tex4ht:ref: tab:computers_r --></a>. Local machines currently run the latest version of the R (at the time of writing) and remote machines run the versions and modules as managed by the system administrator.
</p>
<!--l. 346-->
<p class="indent">
Various scripts and packages in this thesis were developed or run in previous versions of RStudio and R but these run without error in the current version of R (and the older versions on remote machines). The R packages which were used throughout this thesis (as detailed in Table <a 
href="#x1-90002r5">2.5<!--tex4ht:ref: tab:computers_r_packages --></a> with versions specified) were installed from the Comprehensive R Archive Network (<a 
href="#XCRAN">CRAN</a>, <a 
href="#XCRAN">2017</a>), Bioconductor (<a 
href="#XGentleman2004">Gentleman <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGentleman2004">2004</a>, version 3.4; BiocInstaller 1.24.0), or GitHub. These packages were not updated when they would change the functionality of scripts or functions in packages, in particular imported data from annotation packages (used to define gene sets) have been saved as local files to continue using stable versions of these pathway data (across machines).
</p>
<!--l. 348-->
<p class="indent">
This is a summary of the key packages which (in addition to their dependencies) have been used throughout this project. Where a package implementation has been central to the methods applied, they are described in more detail in the relevant section. A full table of packages used in this thesis can be found in Appendix <a 
href="#x1-202000B">B<!--tex4ht:ref: appendix:software --></a> (Table <a 
href="#x1-202001r1">B.1<!--tex4ht:ref: tab:computers_r_packages_full --></a>). The R packages developed during this thesis are given in Table <a 
href="#x1-90004r6">2.6<!--tex4ht:ref: tab:computers_r_packages_dev --></a> with the relevant sections describing their implementation and use where appropriate, in addition to further details on these functions in Section <a 
href="#x1-1080005">3.5<!--tex4ht:ref: methods:r_packages --></a>.
</p>
<div class="table">


<!--l. 350-->
<p class="indent">
<a 
 id="x1-90001r4"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 2.4: </span><span class="content">R Installations used during Thesis</span></p>
</div>
<!--tex4ht:label?: x1-90001r4 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-43" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
>
<colgroup id="TBL-43-1g">
<col 
id="TBL-43-1" />
<col 
id="TBL-43-2" />
</colgroup>
<colgroup id="TBL-43-3g">
<col 
id="TBL-43-3" />
</colgroup>
<colgroup id="TBL-43-4g">
<col 
id="TBL-43-4" />
</colgroup>
<colgroup id="TBL-43-5g">
<col 
id="TBL-43-5" />
</colgroup>
<colgroup id="TBL-43-6g">
<col 
id="TBL-43-6" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-43-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-43-1-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">

</div>
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-43-1-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">

</div>
<pre><code>     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-43-1-3&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Viao Laptop</span></p>
</div>
<pre><code>                                                                   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-43-1-4&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Lab Machine</span></p>
</div>
<pre><code>                                                                   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-43-1-5&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Biochem Server</span></p>
</div>
<pre><code>                                                               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-43-1-6&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">NeSI Pan Cluster</span></p>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-43-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-43-2-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-43-3-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-43-3-1"  
class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">

</div>
<pre><code>           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-43-3-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>OS</p>
</div>
<pre><code>  &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-43-3-3&quot;  </code></pre>
class=“td11”&gt; <!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-44" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-44-1g">
<col 
id="TBL-44-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-44-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-44-1-1" class="td00">
Elementary OS
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-44-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-44-2-1" class="td00">
Freya 0.3.2
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-43-3-4"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-45" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-45-1g">
<col 
id="TBL-45-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-45-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-45-1-1" class="td00">
Elementary OS
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-45-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-45-2-1" class="td00">
Loki 0.4
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-43-3-5"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-46" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-46-1g">
<col 
id="TBL-46-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-46-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-46-1-1" class="td00">
Red Hat Enterprise
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-46-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-46-2-1" class="td00">
Maipo 7.2
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-43-3-6"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-47" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-47-1g">
<col 
id="TBL-47-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-47-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-47-1-1" class="td00">
Cent OS
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-47-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-47-2-1" class="td00">
Final 6.4
</td>
</tr>
</table>
</div>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-43-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-43-4-1" class="td11">
Programming
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-43-4-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>R</p>
</div>
<pre><code>   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-43-4-3&quot;  </code></pre>
class=“td11”&gt; 3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-43-4-4" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-43-4-5" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-43-4-6" class="td11">
3.3.0-intel (module)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-43-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-43-5-1" class="td11">
Development
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-43-5-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>RStudio</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-43-5-3" class="td11">
1.0.136
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-43-5-4" class="td11">
1.0.136
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-43-5-5" class="td11">
1.0.136 (server)
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-43-5-6"  
class="td11">
<pre><code>                                                                     &lt;/td&gt;</code></pre>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-43-6-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-43-6-1"  
class="td11">
<pre><code>          &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                                                                                                                                                                                                                                                                                                                                               &lt;/div&gt;
                                                                                
                                                                                </code></pre>
</div>
<hr class="endfloat" />
</div>
<a 
 id="x1-90002r5"></a> <!--l. 378-->
<div class="longtable">
<table id="TBL-48" class="longtable" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-48-1g">
<col 
id="TBL-48-1" />
<col 
id="TBL-48-2" />
<col 
id="TBL-48-3" />
<col 
id="TBL-48-4" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-48-1-">
<td colspan="4" style="white-space:nowrap; text-align:center;" id="TBL-48-1-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<div class="caption">
<p><span class="id">Table 2.5: </span><span class="content">R Packages used during Thesis</span></p>
</div>
<!--tex4ht:label?: x1-90002r5 -->
</div>
<a 
 id="x1-90003"></a>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-2-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Package</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-2-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Version Used</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-2-3" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Built</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-2-4" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Repository</span></p>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-3-">
<td colspan="4" style="white-space:nowrap; text-align:left;" id="TBL-48-3-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-4-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-5-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-6-">
<td colspan="4" style="white-space:nowrap; text-align:left;" id="TBL-48-6-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-7-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-8-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-9-1" class="td11">
colorspace
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-9-2" class="td11">
1.3-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-9-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-9-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-10-1" class="td11">
curl
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-10-2" class="td11">
2.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-10-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-10-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-11-1" class="td11">
data.table
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-11-2" class="td11">
1.9.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-11-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-11-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-12-1" class="td11">
dendextend
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-12-2" class="td11">
1.4.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-12-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-12-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-13-1" class="td11">
DBI
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-13-2" class="td11">
0.5-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-13-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-13-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-14-1" class="td11">
devtools
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-14-2" class="td11">
1.12.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-14-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-14-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-15-1" class="td11">
dplyr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-15-2" class="td11">
0.5.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-15-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-15-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-16-1" class="td11">
ggplot2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-16-2" class="td11">
2.2.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-16-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-16-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-17-1" class="td11">
git2r
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-17-2" class="td11">
0.18.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-17-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-17-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-18-1" class="td11">
gplots
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-18-2" class="td11">
3.0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-18-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-18-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-19-1" class="td11">
gtools
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-19-2" class="td11">
3.5.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-19-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-19-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-20-1" class="td11">
igraph
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-20-2" class="td11">
1.0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-20-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-20-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-21-1" class="td11">
matrixcalc
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-21-2" class="td11">
1.0-3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-21-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-21-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-22-1" class="td11">
mclust
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-22-2" class="td11">
5.2.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-22-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-22-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-23-1" class="td11">
mvtnorm
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-23-2" class="td11">
1.0-6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-23-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-23-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-24-1" class="td11">
org.Hs.eg.db
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-24-2" class="td11">
3.1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-24-3" class="td11">
3.1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-24-4" class="td11">
Bioconductor
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-25-1" class="td11">
openssl
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-25-2" class="td11">
0.9.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-25-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-25-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-26-1" class="td11">
plyr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-26-2" class="td11">
1.8.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-26-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-26-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-27-1" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-27-2" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-27-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-27-4" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-28-1" class="td11">
purrr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-28-2" class="td11">
0.2.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-28-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-28-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-29-1" class="td11">
reactome.db
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-29-2" class="td11">
1.52.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-29-3" class="td11">
3.2.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-29-4" class="td11">
Bioconductor
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-30-1" class="td11">
RColorBrewer
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-30-2" class="td11">
1.1-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-30-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-30-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-31-1" class="td11">
Rcpp
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-31-2" class="td11">
0.12.9
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-31-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-31-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-32-1" class="td11">
ROCR
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-32-2" class="td11">
1.0-7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-32-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-32-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-33-1" class="td11">
roxygen2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-33-2" class="td11">
6.0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-33-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-33-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-34-1" class="td11">
shiny
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-34-2" class="td11">
1.0.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-34-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-34-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-35-1" class="td11">
snow
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-35-2" class="td11">
0.4-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-35-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-35-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-36-1" class="td11">
testthat
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-36-2" class="td11">
1.0.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-36-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-36-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-37-1" class="td11">
tidyr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-37-2" class="td11">
0.6.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-37-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-37-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-38-1" class="td11">
tidyverse
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-38-2" class="td11">
1.1.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-38-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-38-4" class="td11">
GitHub (hadley)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-39-1" class="td11">
sm
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-39-2" class="td11">
2.2-5.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-39-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-39-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-40-1" class="td11">
Unicode
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-40-2" class="td11">
9.0.0-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-40-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-40-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-41-1" class="td11">
vioplot
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-41-2" class="td11">
0.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-41-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-41-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-42-1" class="td11">
viridis
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-42-2" class="td11">
0.3.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-42-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-42-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-43-1" class="td11">
xml2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-43-2" class="td11">
1.1.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-43-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-43-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-44-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-44-1" class="td11">
xtable
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-44-2" class="td11">
1.8-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-44-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-44-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-45-1" class="td11">
zoo
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-45-2" class="td11">
1.7-14
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-45-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-45-4" class="td11">
CRAN
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-46-1" class="td11">
graphics
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-46-2" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-46-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-46-4" class="td11">
base
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-47-1" class="td11">
grDevices
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-47-2" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-47-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-47-4" class="td11">
base
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-48-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-49-1" class="td11">
cluster
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-49-2" class="td11">
2.0.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-49-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-49-4" class="td11">
base
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-50-1" class="td11">
graphics
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-50-2" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-50-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-50-4" class="td11">
base
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-51-1" class="td11">
grDevices
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-51-2" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-51-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-51-4" class="td11">
base
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-52-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-52-1" class="td11">
Matrix
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-52-2" class="td11">
1.2-8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-52-3" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-52-4" class="td11">
base
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-53-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-53-1" class="td11">
stats
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-53-2" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-53-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-48-53-4" class="td11">
base
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-54-">
<td colspan="4" style="white-space:nowrap; text-align:left;" id="TBL-48-54-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-55-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-55-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-56-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-56-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-57-">
<td colspan="4" style="white-space:nowrap; text-align:left;" id="TBL-48-57-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-58-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-58-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-59-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-59-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-60-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-60-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-61-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-61-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-62-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-62-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-63-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-63-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-48-64-">
<td style="white-space:nowrap; text-align:left;" id="TBL-48-64-1" class="td11">
</td>
</tr>
</table>
</div>
<div class="table">


<!--l. 514-->
<p class="indent">
<a 
 id="x1-90004r6"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 2.6: </span><span class="content">R Packages Developed during Thesis</span></p>
</div>
<!--tex4ht:label?: x1-90004r6 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-49" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
>
<colgroup id="TBL-49-1g">
<col 
id="TBL-49-1" />
</colgroup>
<colgroup id="TBL-49-2g">
<col 
id="TBL-49-2" />
<col 
id="TBL-49-3" />
<col 
id="TBL-49-4" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-49-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-49-1-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">

</div>
<pre><code>                     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-49-1-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Package Name</span></p>
</div>
<pre><code>               &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-49-1-3&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Description and GitHub Repository</span></p>
</div>
<pre><code>                                         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-49-1-4&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><span class="cmbx-12">Section</span></p>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-49-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-49-2-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-49-3-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-49-3-1"  
class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">

</div>
<pre><code>                  &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-49-3-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmtt-12">slipt</span></p>
</div>
<pre><code>                         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-49-3-3&quot;  </code></pre>
class=“td11”&gt; <!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-50" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-50-1g">
<col 
id="TBL-50-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-50-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-50-1-1" class="td00">
Synthetic lethal detection by <a 
href="#glo:SLIPT">SLIPT</a> (to accompany publication)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-50-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-50-2-1" class="td00">
<a 
href="https://github.com/TomKellyGenetics/slipt" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/slipt</span> </a>
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-49-3-4" class="td11">
<a 
href="#x1-930001">3.1<!--tex4ht:ref: methods:SLIPT --></a>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-49-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-49-4-1" class="td11">
visualisation
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-49-4-2" class="td11">
<span class="cmtt-12">vioplotx </span>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-49-4-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-51" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-51-1g">
<col 
id="TBL-51-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-51-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-51-1-1" class="td00">
Customised violin plots (based on <span class="cmtt-12">vioplot</span>)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-51-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-51-2-1" class="td00">
<a 
href="https://github.com/TomKellyGenetics/vioplotx" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/vioplotx</span></a>
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-49-4-4"  
class="td11">
<pre><code>     &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-49-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-49-5-1" class="td11">
<pre><code>                &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-49-5-2&quot;  </code></pre>
class=“td11”&gt; <span class="cmtt-12">heatmap.2x </span>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-49-5-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-52" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-52-1g">
<col 
id="TBL-52-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-52-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-52-1-1" class="td00">
Customised heatmaps (based on <span class="cmtt-12">gplots</span>)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-52-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-52-2-1" class="td00">
<a 
href="https://github.com/TomKellyGenetics/heatmap.2x" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/heatmap.2x</span></a>
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-49-5-4" class="td11">
<a 
href="#x1-790004">2.3.4<!--tex4ht:ref: methods:heatmap --></a>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-49-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-49-6-1" class="td11">
<span class="cmtt-12">igraph.extensions </span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-49-6-2" class="td11">
<span class="cmtt-12">igraph.extensions </span>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-49-6-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-53" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-53-1g">
<col 
id="TBL-53-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-53-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-53-1-1" class="td00">
Meta-package to install the follow iGraph functions
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-53-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-53-2-1" class="td00">
<a 
href="https://github.com/TomKellyGenetics/igraph.extensions" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/igraph.extensions</span></a>
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-49-6-4" class="td11">
<a 
href="#x1-1110003">3.5.3<!--tex4ht:ref: methods:igraph_extensions --></a>
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-49-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-49-7-1" class="td11">
<pre><code>                &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-49-7-2&quot;  </code></pre>
class=“td11”&gt; <span class="cmtt-12">plot.igraph </span>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-49-7-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-54" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-54-1g">
<col 
id="TBL-54-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-54-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-54-1-1" class="td00">
Custom plotting of directed graphs
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-54-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-54-2-1" class="td00">
<a 
href="https://github.com/TomKellyGenetics/plot.igraph" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/plot.igraph</span></a>
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-49-7-4" class="td11">
<a 
href="#x1-870004">2.4.4<!--tex4ht:ref: methods:network_metrics --></a>
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-49-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-49-8-1" class="td11">
<pre><code>                &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-49-8-2&quot;  </code></pre>
class=“td11”&gt; <span class="cmtt-12">info.centrality </span>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-49-8-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-55" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-55-1g">
<col 
id="TBL-55-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-55-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-55-1-1" class="td00">
Computing information centrality from network efficiency
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-55-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-55-2-1" class="td00">
<a 
href="https://github.com/TomKellyGenetics/info.centrality" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/info.centrality</span></a>
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-49-8-4" class="td11">
<a 
href="#x1-1070002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-49-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-49-9-1" class="td11">
<pre><code>                &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-49-9-2&quot;  </code></pre>
class=“td11”&gt; <span class="cmtt-12">pathway.structure.permutation </span>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-49-9-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-56" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-56-1g">
<col 
id="TBL-56-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-56-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-56-1-1" class="td00">
Testing pathway structure with resampling analysis
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-56-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-56-2-1" class="td00">
<a 
href="https://github.com/TomKellyGenetics/pathway.structure.permutation" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/pathway.structure.permutation</span></a>
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-49-9-4" class="td11">
<a 
href="#x1-1050001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-49-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-49-10-1" class="td11">
<pre><code>                &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-49-10-2&quot;  </code></pre>
class=“td11”&gt; <span class="cmtt-12">graphsim </span>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-49-10-3"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-57" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-57-1g">
<col 
id="TBL-57-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-57-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-57-1-1" class="td00">
Generating simulated expression from graph structures
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-57-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-57-2-1" class="td00">
<a 
href="https://github.com/TomKellyGenetics/graphsim" class="url" ><span class="cmtt-12">https://github.com/TomKellyGenetics/graphsim</span> </a>
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-49-10-4" class="td11">
<a 
href="#x1-1070002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-49-11-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-49-11-1"  
class="td11">
<pre><code>                &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                                                                                                            &lt;/div&gt;
                                                                                
                                                                                </code></pre>
</div>
<hr class="endfloat" />
</div>
<a 
 id="x1-90005r102"></a>
<h4 class="subsectionHead">
<span class="titlemark">2.5.3 </span> <a 
 id="x1-910003"></a>High Performance and Parallel Computing
</h4>
<!--l. 555-->
<p class="noindent">
Another enabling technology for bioinformatics is parallel computing, performing independent operations in separate cores: this “multithreading” is widely used to increase the time to compute results. Bioinformatics is particularly amenable to this since performing multiple iterations of a simulation or testing separate genes is often “embarrassingly parallel“, being completely independent of the results of each other. As such parallel computing is offered by many high-performance “supercomputers” including national research infrastructure.
</p>
<!--l. 557-->
<p class="indent">
The New Zealand eScience Infrastructure (NeSI) is a computating resource providing the Intel Pan cluster hosted by the University of Auckland (<a 
href="#XNeSI">NeSI</a>, <a 
href="#XNeSI">2017</a>). The Pan cluster used throughout this thesis project to optimise and perform computations which would have otherwise been infeasible in the timeframe of thesis. Such technological developments and infrastructure initiatives have enabled bioinformatics research including this project. High performance computing on the Pan cluster was used extensively in this project including for resampling analysis (in sections <a 
href="#x1-820006">2.3.6<!--tex4ht:ref: methods:permutation --></a> and <a 
href="#x1-1050001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>), calculating information centrality (in Section <a 
href="#x1-870004">2.4.4<!--tex4ht:ref: methods:network_metrics --></a>), and in simulations (in sections <a 
href="#x1-800005">2.3.5<!--tex4ht:ref: methods:simulation --></a>, <a 
href="#x1-940002">3.2<!--tex4ht:ref: methods:simulation_SL_expression --></a>, and <a 
href="#x1-1070002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>)
</p>
<!--l. 559-->
<p class="indent">
<p>Scripts and data were transferred between the Pan cluster and University of Otago computing resources by <span class="cmtt-12">rsync </span>or the Globus file transfer service (<a 
href="#XGlobus">Globus</a>, <a 
href="#XGlobus">2017</a>). R scripts (<a 
href="#XR_core">R Core Team</a>, <a 
href="#XR_core">2016</a>) were run in parallel with the “simple network of workstations” <span class="cmtt-12">snow </span>R package <a 
href="#Xsnow">Tierney <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#Xsnow">2015</a>). This utilised the “message passing interface” (<a 
href="#XRmpi">Yu</a>, <a 
href="#XRmpi">2002</a>) when it was feasible with memory requirements to run in parallel across multiple compute nodes, otherwise SOCKS was used to access multiple cores within an instance of R and pass input data to them. R jobs were submitted to queue for available resources and run on the Pan cluster via the Slurm (Simple Linux Utility for Resource Management) workload manager (<a 
href="#Xslurm">Slurm</a>, <a 
href="#Xslurm">2017</a>). When running R scripts across many parameters or for memory-intensive jobs, Slurm array job submission and independent submission of different parameters via shell commands with arguments passed to R. In some cases, this submission was automated across a range of parameters with Bash scripts.</p>
</p>
<h2 class="chapterHead">
<span class="titlemark">Chapter 3</span><br /><a 
 id="x1-920003"></a>Methods Developed During Thesis
</h2>
<!--l. 5-->
<p class="noindent">
In this Chapter, I will outline the rationale and development of various methods used throughout this thesis to examine synthetic lethality in gene expression data, graph structures, models and simulations. First by describing the <a 
href="#glo:SLIPT">Synthetic Lethal Interaction Prediction Tool</a> (<a 
href="#glo:SLIPT">SLIPT</a>), a bioinformatics approach to triage of synthetic lethal candidate genes. This is considered one of the main research outputs of the thesis, which is supported by comparisons to an experimental screen from a related project and performance on simulated data. These supporting data will be covered in further Chapters but preliminary data to support the use and design of <a 
href="#glo:SLIPT">SLIPT</a> are provided alongside description of the method. This includes the construction of a statistical model of synthetic lethality in (continuous multivariate Gaussian) gene expression data, which enables testing <a 
href="#glo:SLIPT">SLIPT</a> upon simulated data with known synthetic lethal partners. Another key component of the simulation pipeline used later is the generation of simulated data from a known graph structure or simulated biological pathway. The development of this simulation procedure and other statistical treatment of graph and network structures will also be covered. Various R packages have been developed to support this project, most notably the <span class="cmtt-12">slipt </span>package to implement the <a 
href="#glo:SLIPT">SLIPT</a> methodology. The additional R packages for handling graph structures, simulations, and custom plotting features will also be described as research outputs of this thesis, methods applied throughout, and contributions to the open-source software community that made this project feasible. <a 
 id="x1-92001r98"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">3.1 </span> <a 
 id="x1-930001"></a>A Synthetic Lethal Detection Methodology
</h3>
<!--l. 11-->
<p class="noindent">
<p>The <a 
href="#glo:SLIPT">SLIPT</a> methodology identifies gene expression patterns consistent with synthetic lethal interactions between a query gene and a panel of candidate interacting partners. Gene expression is called low, medium, or high by separating samples into tertiles (3-quantiles) for each gene. Genes with insufficient expression across all samples were excluded by requiring that the first tertile of raw counts is above zero. Then a <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test is performed between the query gene and each candidate partner, with the p-values for the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test being corrected for multiple testing using false discovery rate (FDR) error control to reduce false positives for large candidate gene panels (<a 
href="#Xfdr1995">Benjamini and Hochberg</a>, <a 
href="#Xfdr1995">1995</a>). Significance was called only if FDR adjusted p-values were below the threshold <span class="cmmi-12">p &lt; </span>0<span class="cmmi-12">.</span>05. A synthetic lethal interaction is predicted (as</p>
<p>shown in Figure <a 
href="#x1-93001r1">3.1<!--tex4ht:ref: fig:SLIPT_Method --></a>) when (i) the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test is significant; (ii) observed low-query, low-candidate samples are less frequent than expected; and (iii) observed low-query, high-candidate and high-query, low-candidate samples are more frequent than expected.</p>
</p>
<!--l. 15-->
<p class="indent">
<a 
 id="x1-93001r1"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 17-->
<p class="noindent">
</p>
<!--l. 18-->
<p class="noindent">
<span class="fbox"><img 
src="thesis6x.png" alt="PICT"  /></span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Framework for synthetic lethal prediction.</span></strong> <a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">was designed to</span> <span class="cmr-10x-x-109">identify candidate interacting genes from gene expression data using the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test against</span> <span class="cmr-10x-x-109">a query gene. Samples are sorted into low, medium, and high expression quantiles for</span> <span class="cmr-10x-x-109">each gene to test for a directional shift. A sample being low in both genes of a synthetic</span> <span class="cmr-10x-x-109">lethal pair is unlikely, since loss of both genes will be deleterious, and is expected to be</span> <span class="cmr-10x-x-109">statistically under-represented in a gene expression dataset. We expect a corresponding</span> <span class="cmr-10x-x-109">(symmetric) increase in frequency of sample with low-high gene pairs. Synthetic lethal</span> <span class="cmr-10x-x-109">candidate (exprSL) partners of a gene are identified by running this procedure on all</span> <span class="cmr-10x-x-109">possible partner genes, selecting those with an FDR-adjusted </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">p-value of </span><span class="cmmi-10x-x-109">p &lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05,</span> <span class="cmr-10x-x-109">and meeting the directional criteria. Since synthetic lethal genes are partners of each</span> <span class="cmr-10x-x-109">other commutatively, the symmetric direction criteria are all required such that synthetic</span> <span class="cmr-10x-x-109">lethal genes will predicted to be partners of each other. </span></span></p>
</div>
<!--tex4ht:label?: x1-93001r1 -->
</div>
<hr class="endfloat" />
<!--l. 29-->
<p class="indent">
<p>The synthetic lethal prediction procedure has also been adapted to utilise somatic mutation data for the query gene. This is intended to utilise a query gene known to be recurrently mutated in the disease (and dataset), with the majority of mutations inactivating gene function (such as null or frameshift mutations). A synthetic lethal interaction is predicted (as shown in Figure <a 
href="#x1-93002r2">3.2<!--tex4ht:ref: fig:SLIPT_Method_mtSL --></a>) when (i) the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test is significant; (ii) observed mutant-query, low-candidate samples are less frequent than expected; and (iii) observed mutant-query, high-candidate and wild-type-query, low-candidate samples are more frequent than expected. Unless otherwise specified, computationally predicted synthetic lethal gene candidates from <a 
href="#glo:SLIPT">SLIPT</a> used expression data (exprSL) for both genes (as shown in Figure <a 
href="#x1-93001r1">3.1<!--tex4ht:ref: fig:SLIPT_Method --></a>) rather than mutation data (mtSL) for the query gene (as shown in Figure <a 
href="#x1-93002r2">3.2<!--tex4ht:ref: fig:SLIPT_Method_mtSL --></a>).</p>
</p>
<!--l. 31-->
<p class="indent">
<a 
 id="x1-93002r2"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 33-->
<p class="noindent">
</p>
<!--l. 34-->
<p class="noindent">
<span class="fbox"><img 
src="thesis7x.png" alt="PICT"  /></span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethal prediction adapted for mutation.</span></strong> <a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">was also</span> <span class="cmr-10x-x-109">adapted to identify candidate interacting genes using (somatic) mutation data of the</span> <span class="cmr-10x-x-109">query gene in the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test. Samples are sorted into low, medium, and high expression</span> <span class="cmr-10x-x-109">quantiles for each candidate gene and tested for a directional shift against mutation status</span> <span class="cmr-10x-x-109">of the query gene. A sample having low expression or mutation for the synthetic lethal</span> <span class="cmr-10x-x-109">pair is expected to be unlikely with a corresponding increase in frequency of sample with</span> <span class="cmr-10x-x-109">mutant-high or wild-type-low gene pairs. Synthetic lethal candidate (mtSL) partners of</span> <span class="cmr-10x-x-109">a gene are identified by running this procedure on all possible partner genes, selecting</span> <span class="cmr-10x-x-109">those with an FDR-adjusted </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">p-value of </span><span class="cmmi-10x-x-109">p &lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05, and meeting the directional criteria.</span> </span></p>
</div>
<!--tex4ht:label?: x1-93002r2 -->
</div>
<hr class="endfloat" />
<!--l. 44-->
<p class="indent">
The <a 
href="#glo:SLIPT">SLIPT</a> methodology is amenable for use on expression data including pathway metagenes (as generated in Section <a 
href="#x1-730003">2.2.3<!--tex4ht:ref: methods:metagene --></a>). The suitability of the <a 
href="#glo:SLIPT">SLIPT</a> methodology to application on public gene expression data will further be supported by simulation results in Section <a 
href="#x1-970003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a> and Chapter <a 
href="#x1-1750006">6<!--tex4ht:ref: chap:simulation --></a>, including comparison to other statistical methods. <a 
href="#glo:SLIPT">SLIPT</a> results for <span class="cmti-12">CDH1 </span>will also compared experimental screen results in a breast cell line (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), primary screen results are discussed in Section <a 
href="#x1-1220002">4.2<!--tex4ht:ref: chapt3:compare_SL_genes --></a> and secondary (validation) screen results are presented in Appendix <a 
href="#x1-203000C">C<!--tex4ht:ref: appendix:secondary_screen --></a>. <a 
 id="x1-93003r108"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">3.2 </span> <a 
 id="x1-940002"></a>Synthetic Lethal Simulation and Modelling
</h3>
<!--l. 55-->
<p class="noindent">
A statistical model of Synthetic Lethality was developed to generate simulated data to test the <a 
href="#glo:SLIPT">SLIPT</a> procedure. This section will describe the synthetic lethal model and the simulation procedure for generating gene expression data with known synthetic lethal partners. Some preliminary results to support usage of the <a 
href="#glo:SLIPT">SLIPT</a> methodology throughout this thesis will be presented here. The simulation procedure will be applied in more depth in Chapter <a 
href="#x1-1750006">6<!--tex4ht:ref: chap:simulation --></a>, including in combination with simulations from graph structures. <a 
 id="x1-94001r106"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.2.1 </span> <a 
 id="x1-950001"></a>A Model of Synthetic Lethality in Expression Data
</h4>
<!--l. 59-->
<p class="noindent">
<p>A conceptual model of synthetic lethality was constructed (see Figure <a 
href="#x1-95001r3">3.3<!--tex4ht:ref: fig:SL_Model --></a>), which will be used to build a statistical model of synthetic lethal gene expression from which to simulate expression data to on which test SLIPT and various potential synthetic lethal prediction methods. In the model, synthetic lethality arises between genes with related functions as a cell death phenotype when these functions are removed.</p>
</p>
<!--l. 61-->
<p class="indent">
<a 
 id="x1-95001r3"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 63-->
<p class="noindent">
</p>
<!--l. 64-->
<p class="noindent">
<span class="fbox"> <img 
src=""SL_Model".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">A model of synthetic lethal gene expresion.</span></strong> <span class="cmr-10x-x-109">A conceptual model of</span> <span class="cmr-10x-x-109">synthetic lethal interactions between a Query gene and partner gene (</span><span class="cmmi-10x-x-109">G</span><sub><span class="cmmi-8">X</span></sub><span class="cmr-10x-x-109">). Genes that</span> <span class="cmr-10x-x-109">are synthetic lethal may not both be non-functional in the same sample without another</span> <span class="cmr-10x-x-109">gene compensating for the loss of function. This is most likely to be detectable as low gene</span> <span class="cmr-10x-x-109">expression, whether they are lost by mutation, deletion, DNA methylation, or suppressing</span> <span class="cmr-10x-x-109">regulatory signals. This could manifest as coexpression, mutual exclusivity, or directional</span> <span class="cmr-10x-x-109">shifts in sample frequency. Thus the alternative hypothesis (</span><span class="cmmi-10x-x-109">H</span><sub><span class="cmmi-8">A</span></sub><span class="cmr-10x-x-109">) is that synthetic lethal</span> <span class="cmr-10x-x-109">genes will have a reduced frequency of co-loss samples while the null hypothesis (</span><span class="cmmi-10x-x-109">H</span><sub><span class="cmr-8">0</span></sub><span class="cmr-10x-x-109">) is</span> <span class="cmr-10x-x-109">that non-synthetic lethal gene pairs would show no such relationship, even if they may be</span> <span class="cmr-10x-x-109">correlated for other means such as pathway relationships. In this model synthetic lethal</span> <span class="cmr-10x-x-109">genes may compensate for the loss of each other but this is not assumed, only that loss of</span> <span class="cmr-10x-x-109">both is unfavourable to cell viability and probability of detecting samples with combined</span> <span class="cmr-10x-x-109">gene loss. </span></span></p>
</div>
<!--tex4ht:label?: x1-95001r3 -->
</div>
<hr class="endfloat" />
<!--l. 78-->
<p class="indent">
This model suggests that synthetic lethality is detectable in measures of gene inactivation across a sample population, namely mutation, DNA copy number, DNA methylation, and suppression of expression. While any of these mechanisms of gene inactivation could lead to synthetic lethality, expression data is readily available and changes in these alternative mechanisms are likely to impact on the amount of expressed (functional) RNA or protein detectable. There are several ways that functional relationships between genes could manifest in expression data, including coexpression, mutual exclusivity and directional shifts. Co-expression is overly simplistic and has previously performed poorly as a predictor of synthetic lethality (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>), although this will still be tested with correlation measures in later simulations. Here the alternative hypothesis is that synthetic lethality will lead to a detectable directional shift in the number of samples exhibiting low or high expression of either gene. This model does not preclude mutual exclusivity (<a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWappett2016">2016</a>), compensating expression or co-loss under-representation (<a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>) as previously postulated to occur between synthetic lethal genes.
</p>
<!--l. 80-->
<p class="indent">
<p>The first condition of the synthetic lethal model is that if there are only two synthetic lethal genes (e.g., <span class="cmti-12">CDH1 </span>and one SL partner), then they will not both be non-functional in the same sample (in an ideal model). Gene function is thus determined for each sample in a model of synthetic lethal with the proportion of samples with a functional or non-functional gene being arbitrary. Whether a gene is functional can similarly be modelled by an arbitrary threshold of continuous and normally distributed gene expression data to define gene function (as shown in Figure <a 
href="#x1-95002r4">3.4<!--tex4ht:ref: fig:SL_Model_Expression --></a>). For the purposes of modeling synthetic lethality in breast cancer expression data, a threshold of the 30<sup class="textsuperscript"><span class="cmr-10x-x-109">th</span></sup> percentile of the expression levels was used because approximately 30% of samples analysed had <span class="cmti-12">CDH1 </span>inactivation. This was generalised for a model of the proportion of samples inactivated for each gene. In this ideal case, no samples lowly expressing both of these genes are expected to be observed. While this is not observed, that is to be expected as it is unlikely that only 2 genes will have an exclusive synthetic lethal partnership. The threshold of the 0.3 quantile was used in simulations dervied from this model throughout this thesis.</p>
</p>
<!--l. 82-->
<p class="indent">
<a 
 id="x1-95002r4"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 84-->
<p class="noindent">
</p>
<!--l. 85-->
<p class="noindent">
<span class="fbox"> <img 
src=""SL_Model_Expression".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.4: </span><span class="content"><strong><span class="cmbx-10x-x-109">Modeling synthetic lethal gene expression.</span></strong> <span class="cmr-10x-x-109">When modeling synthetic</span> <span class="cmr-10x-x-109">lethal interactions between a Query gene with partner genes (</span><span class="cmmi-10x-x-109">G</span><sub><span class="cmmi-8">X</span></sub> <span class="cmr-10x-x-109">and </span><span class="cmmi-10x-x-109">G</span><sub><span class="cmmi-8">Y</span> </sub><span class="cmr-10x-x-109">) above, cellular</span> <span class="cmr-10x-x-109">viability requires that at least of genes is not inactivated. Expression below a threshold</span> <span class="cmr-10x-x-109">is used as a model of loss of function, where genes are regarded as non-functional for the</span> <span class="cmr-10x-x-109">purposes of modelling synthetic lethality. Tumour suppressor genes with loss of function</span> <span class="cmr-10x-x-109">also have cancer specific phenotypes (although these thresholds are not necessarily the</span> <span class="cmr-10x-x-109">same). Expression is modeled by a normally (Gaussian) distributed continuous data such</span> <span class="cmr-10x-x-109">as (log-scale) data from RNA (microarray or RNA-Seq), protein, or pathway metagenes.</span> <span class="cmr-10x-x-109">This rationale generalises for several genes on a multivariate normal distribution. </span></span></p>
</div>
<!--tex4ht:label?: x1-95002r4 -->
</div>
<hr class="endfloat" />
<!--l. 98-->
<p class="indent">
<a 
 id="x1-95003r5"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 100-->
<p class="noindent">
</p>
<!--l. 101-->
<p class="noindent">
<span class="fbox"> <img 
src=""SL_Model_Higher".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.5: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethality with multiple genes.</span></strong> <span class="cmr-10x-x-109">Higher order synthetic lethal</span> <span class="cmr-10x-x-109">interactions may occur between 3 or more genes, affecting the simulated expression</span> <span class="cmr-10x-x-109">(or synthetic lethal predictions) even if undetected when observed pairwise. Consider</span> <span class="cmr-10x-x-109">interactions between a Query gene and two partner genes (</span><span class="cmmi-10x-x-109">G</span><sub><span class="cmmi-8">X</span></sub> <span class="cmr-10x-x-109">and </span><span class="cmmi-10x-x-109">G</span><sub><span class="cmmi-8">Y</span> </sub><span class="cmr-10x-x-109">). They may</span> <span class="cmr-10x-x-109">interact with the Query pairwise (inviable when either gene pair is lost) or form a</span> <span class="cmr-10x-x-109">higher-order interaction such as the “synthetic lethal triplet” if any of the genes provide</span> <span class="cmr-10x-x-109">an essential function (inviable only when all are lost). Either is plausible with the</span> <span class="cmr-10x-x-109">potential pathway structures. A synthetic lethal triple has 8 potential combinations of</span> <span class="cmr-10x-x-109">gene functional but one is not expected to be observed (due to inviability) but pairwise</span> <span class="cmr-10x-x-109">inactivation may be observed if additional partner genes are functional. The proportion</span> <span class="cmr-10x-x-109">of these combinations vary depending on the functional threshold. </span></span></p>
</div>
<!--tex4ht:label?: x1-95003r5 -->
</div>
<hr class="endfloat" />
<!--l. 116-->
<p class="indent">
A synthetic lethal pair of genes is unlikely to act in isolation, therefore higher-order synthetic lethal interactions (i.e., 3 or more genes) must be considered in the model as shown in Figure <a 
href="#x1-95003r5">3.5<!--tex4ht:ref: fig:SL_Model_Higher --></a>. Even when testing pairwise interactions, modelling higher level interactions that may interfere is important. If there are additional synthetic lethal partners, there are two possibilities for adding these: 1) that they are independent partners of the query genes interacting pairwise (and not with each other) or 2) that an addition partner gene interacts with both of the synthetic lethal genes already in the system and any of the three (or more) are required to be functional for the cell to survive.
</p>
<!--l. 118-->
<p class="indent">
The signal (in terms of gene expression data) will be weaker for this latter case and this model has the more stringent assumption that all synthetic lethal partner genes interact with each other: that only one of these must be expressed to satisfy the model of synthetic lethality. In this model any of the synthetic lethal genes in a higher-order interaction is able to provide the missing function of the others, allowing for higher-level synthetic lethal partners to compensate for loss a synthetic lethal gene pair. While samples expressing low levels of the synthetic lethal gene pairs will be under-represented, they may not be completely absent from the dataset due to these higher-level interactions.
</p>
<!--l. 120-->
<p class="indent">
In the example of 3 synthetic lethal genes <a 
href="#x1-95003r5">3.5<!--tex4ht:ref: fig:SL_Model_Higher --></a>, only one of genes involved in the higher-order synthetic lethal interaction is required for cell viability. For synthetic lethal pairs, only a subset of these samples will be inviable (i.e., removed from simulated data), leading to an under-representation.
</p>
<!--l. 122-->
<p class="indent">
In practice, samples are not removed from a simulated dataset, rather the expression and function of the query gene is generated across samples separately from the pool of potential partner genes. The query gene data is matched to simulated samples (as shown in Figure <a 
href="#x1-96006r7">3.7<!--tex4ht:ref: fig:simulate_add_query --></a>), satisfying the synthetic lethal condition with the procedure described in Section <a 
href="#x1-960002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a>. This is performed to maintain a comparable samples size across simulations and the preserve the assumed (multivariate) normal distribution of the data. <a 
 id="x1-95004r112"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.2.2 </span> <a 
 id="x1-960002"></a>Simulation Procedure
</h4>
<!--l. 128-->
<p class="noindent">
<p>Simulations were developed to simulate normal distributions of expression data and define function with a threshold cut-off. This is the reverse to the procedure of <a 
href="#glo:SLIPT">SLIPT</a> to predict</p>
<p>synthetic lethal partners (although the threshold is assumed to be unknown when testing upon simulated data). While gene function is used as an intermediary step in modelling synthetic lethal genes in expression data, the normal distribution is sampled for simulated data to represent normalised empirical gene expression data for which <a 
href="#glo:SLIPT">SLIPT</a> (and other methods) will be applicable.</p>
</p>
<!--l. 130-->
<p class="indent">
<a 
 id="x1-96003r6"></a><a 
 id="x1-96006r7"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 133-->
<p class="noindent">
<a 
 id="x1-96001r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_mat".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Simulated</span> <span class="cmr-10">expression</span> <span class="cmr-10">matrix</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-96002r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_disc_mat".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Corresponding</span> <span class="cmr-10">gene</span> <span class="cmr-10">function</span> <span class="cmr-10">calls</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.6: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulating gene function.</span></strong></strong> <span class="cmr-10x-x-109">A simulated dataset with samples (columns)</span> <span class="cmr-10x-x-109">and genes A–H (rows) is transformed from a continuous (coloured blue–red) scale to a</span> <span class="cmr-10x-x-109">discrete matrix of gene function (black for functional levels and grey for non-functional).</span></span></p>
</div>
<!--tex4ht:label?: x1-96003r6 -->
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 183-->
<p class="noindent">
<a 
 id="x1-96004r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_SL_disc_mat".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Simulated</span> <span class="cmr-10">gene</span> <span class="cmr-10">function</span> <span class="cmr-10">with</span> <span class="cmr-10">SL</span> <span class="cmr-10">genes</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-96005r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_disc_query_mat_graph".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Query</span> <span class="cmr-10">gene</span> <span class="cmr-10">added</span> <span class="cmr-10">with</span> <span class="cmr-10">SL</span> <span class="cmr-10">condition</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.7: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulating synthetic lethal gene function.</span></strong></strong> <span class="cmr-10x-x-109">In a discrete simulated gene</span> <span class="cmr-10x-x-109">function dataset (shaded for functional levels and pale otherwise) with samples (columns)</span> <span class="cmr-10x-x-109">and genes (rows), genes A and I are SL partners of a “Query” gene. A partner is selected</span> <span class="cmr-10x-x-109">(highlighted in green) randomly in each sample for simulating synthetic lethality, then</span> <span class="cmr-10x-x-109">ordered such that the query gene or an SL partner are functional in each sample. </span></span></p>
</div>
<!--tex4ht:label?: x1-96006r7 -->
</div>
<hr class="endfloat" />
<!--l. 208-->
<p class="indent">
Sampling a distribution for expression profiles has the added advantage of being amenable to simulating correlation structures with the multivariate normal distribution (using the <span class="cmtt-12">mvtnorm </span>R package (<a 
href="#XGenz2009">Genz and Bretz</a>, <a 
href="#XGenz2009">2009</a>; <a 
href="#Xmvtnorm">Genz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xmvtnorm">2016</a>)). The parameter Σ is a covariance matrix defines the correlation structure between simulated genes being sampled. With a diagonal of one, this Σ matrix simulates genes with a standard deviation of one and the covariance parameters between them are the correlations between each gene. In Figure <a 
href="#x1-96003r6">3.6<!--tex4ht:ref: fig:simulate_function --></a>, an example of such a simulated multivariate normal dataset is shown with the functional threshold applied.
</p>
<!--l. 210-->
<p class="indent">
<p>Once we have generated a simulated dataset, the samples are compared by gene function (as derived from a functional threshold). Known underlying synthetic lethal partners are selected within the dataset and a query gene is generated by sampling from the normal distribution. These are matched (as shown for 2 synthetic lethal partners in Figure <a 
href="#x1-96006r7">3.7<!--tex4ht:ref: fig:simulate_add_query --></a>) such that the synthetic lethal condition is met: that at least one of the synthetic partner genes and the query gene are functional in any particular cell. The samples are ordered by functional data (without assuming correlation of underyling expression values) with the query gene in one direction and the remaining dataset ordered by the selected synthetic lethal partner.</p>
</p>
<!--l. 213-->
<p class="indent">
<a 
 id="x1-96009r8"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 216-->
<p class="noindent">
<a 
 id="x1-96007r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_mat".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Initial</span> <span class="cmr-10">expression</span> <span class="cmr-10">matrix</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-96008r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_query_mat_graph".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Simulated</span> <span class="cmr-10">synthetic</span> <span class="cmr-10">lethal</span> <span class="cmr-10">dataset</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.8: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulating synthetic lethal gene expression.</span></strong></strong> <span class="cmr-10x-x-109">A simluated continuous</span> <span class="cmr-10x-x-109">expression dataset (blue–red scale) with samples (columns) and genes (rows) is matched</span> <span class="cmr-10x-x-109">to a query gene such that at least one synthetic lethal partner is above a functional</span> <span class="cmr-10x-x-109">threshold when the query gene is below it satisfying the synthetic lethal model.</span></span></p>
</div>
<!--tex4ht:label?: x1-96009r8 -->
</div>
<hr class="endfloat" />
<!--l. 237-->
<p class="indent">
This results a simulated dataset where samples with non-functional query gene have at least one functional partner gene. Similarly, the query gene is functional in all samples where all of the synthetic lethal partner genes are non-functional. Therefore a dataset has been generated with known synthetic lethal partners (see Figure <a 
href="#x1-96009r8">3.8<!--tex4ht:ref: fig:simulate_SL --></a>) by as few assumptions about the relationships between the each synthetic lethal pair as possible (and allowing compensating functions from higher-order interactions). This has been designed to have the most stringent (least detectable) synthetic lethal relationships where higher-order interactions are possible for the purposes of testing pairwise detection procedures such as <a 
href="#glo:SLIPT">SLIPT</a>. <a 
 id="x1-96010r111"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">3.3 </span> <a 
 id="x1-970003"></a>Detecting Simulated Synthetic Lethal Partners
</h3>
<!--l. 244-->
<p class="noindent">
The synthetic lethal detection methodology (<a 
href="#glo:SLIPT">SLIPT</a>), as described in Section <a 
href="#x1-930001">3.1<!--tex4ht:ref: methods:SLIPT --></a>, was tested on simulated data with known synthetic lethal partners, generated using the procedure described in Section <a 
href="#x1-960002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a>. This section will present basic simulations to demonstrate the methodology and support it’s use throughout this thesis. These will be performed with sampling from basic statistical distributions as described, including multivariate normal distribution with correlated blocks of genes, with the Σ matrix show in the plots where relevant. A more complex multivariate normal sampling procedure based on pathway graph structures, as described in section  <a 
href="#x1-1070002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>, will be applied in Chapter <a 
href="#x1-1750006">6<!--tex4ht:ref: chap:simulation --></a>. <a 
 id="x1-97001r116"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.3.1 </span> <a 
 id="x1-980001"></a>Binomial Simulation of Synthetic lethality
</h4>
<!--l. 250-->
<p class="noindent">
A previous version of the synthetic lethal simulation procedure (described in Section <a 
href="#x1-960002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a>), used gene function sampled directly from a binomial distribution using the binomial probability of observing functional gene levels (<span class="cmmi-12">p </span>= 0<span class="cmmi-12">.</span>3) in one observation (<span class="cmmi-12">n </span>= 1) for each samples:
</p>
<center class="math-display">
<img 
src="thesis8x.png" alt="X  ~ Bin (n,p)  " class="math-display"  />
</center>
Once a query gene consistent with synthetic lethality has been added, these functional levels were passed directly into <a 
href="#glo:SLIPT">SLIPT</a> as “low” and “high” categories. <!--l. 252-->
<p class="indent">
<p>The simulation procedure was performed with 20,000 total genes (as feasible in the human genome and expression datasets) with a variable number of true synthetic lethal partners and sample sizes of 500, 1000, 2000, and 5000. Each ROC curve was derived from the results of 10,000 replicate simulations. The statistical performance (as shown in Figure <a 
href="#x1-98001r9">3.9<!--tex4ht:ref: fig:Binomial_AUC --></a>) of such an approach based on the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> p-value declines towards random predictions (an AUROC of 0.5) with an increasing number of underlying true synthetic lethal partners to detect. However, increased sample size mitigates this decline to some extent, as expected with a statistical predictor, particularly for moderate numbers of synthetic lethal partners.</p>
</p>
<!--l. 254-->
<p class="indent">
<a 
 id="x1-98001r9"></a><a 
 id="x1-98002r10"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 256-->
<p class="noindent">
</p>
<!--l. 257-->
<p class="noindent">
<span class="fbox"> <img 
src=""SL_Model_Binomial_1K_AUC_samples_prop".png" alt="PIC"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.9: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of binomial simulations.</span></strong> <span class="cmr-10x-x-109">Gene function was simulated by</span> <span class="cmr-10x-x-109">binomial sampling and tested for synthetic lethal genes. Statistical performance declines</span> <span class="cmr-10x-x-109">with additional known synthetic partners but this is mitigated by increased sample sizes.</span></span></p>
</div>
<!--tex4ht:label?: x1-98001r9 -->
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 268-->
<p class="noindent">
</p>
<!--l. 269-->
<p class="noindent">
<span class="fbox"> <img 
src=""SL_Model_Test_Graph_10K_Graph1_ROC_Compare_Binom(Feb)_v_Mvtn(Aprxy)_Full".png" alt="PIC"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.10: </span><span class="content"><strong><span class="cmbx-10x-x-109">Comparison of statistical performance.</span></strong> <span class="cmr-10x-x-109">Binomial simulation of</span> <span class="cmr-10x-x-109">synthetic lethality (in colour) is compared (in greyscale) to multivariate normal</span> <span class="cmr-10x-x-109">simulations (detailed below) which consistently outperforms binomial simulation across</span> <span class="cmr-10x-x-109">parameters.</span></span></p>
</div>
<!--tex4ht:label?: x1-98002r10 -->
</div>
<hr class="endfloat" />
<!--l. 280-->
<p class="indent">
Simulations based on a simple binomial model of synthetic lethality are limited but form a basis for builing a more complex model including expression and correlation structures. While this does not represent the data that <a 
href="#glo:SLIPT">SLIPT</a> will be applied to, binomial simulations do demonstrate that <a 
href="#glo:SLIPT">SLIPT</a> is able to distinguish small numbers of synthetic lethal partners in a simplistic simulated system with behavious expected with resepect to sample size. This supported further development of the synthetic lethal model and simulation pipeline (as described in Section <a 
href="#x1-940002">3.2<!--tex4ht:ref: methods:simulation_SL_expression --></a>) using the multivariate normal distribution.
</p>
<!--l. 282-->
<p class="indent">
The multivariate normal simulation procedure is more representative of the (normalised) expression data <a 
href="#glo:SLIPT">SLIPT</a> is intended for and enables the prediction procedure to be tested without changes to the methodology (presented in more detail in Section <a 
href="#x1-990002">3.3.2<!--tex4ht:ref: chapt2:simulation_mvtnorm --></a>). Sampling continuous expression values from a normal distribution allows the expression threshold for gene function to differ from the categorical “low” and “high” expression binning performed by <a 
href="#glo:SLIPT">SLIPT</a> (as discussed in Section <a 
href="#x1-950001">3.2.1<!--tex4ht:ref: methods:SL_Model --></a>) which represents that the <a 
href="#glo:SLIPT">SLIPT</a> procedure does not assume a known threshold for expression but rather uses expression as an estimate of gene function. This functionality can be included in the multivariate normal simulation without compromising the statistical performance of the <a 
href="#glo:SLIPT">SLIPT</a>, rather the performance estimates (shown in Figure <a 
href="#x1-98002r10">3.10<!--tex4ht:ref: fig:Binomial_Compare --></a>) were a marked improvement over the binomial simulation procedure across simulation parameters in an equivalent simulation (without correlation structure). This imprivement may be due to binomial model defining the synthetic lethal condition in a way that, while ensuring at least on synthetic lethal partner is active in query deficient samples, disrupts the number of samples with functional synthetic lethal genes compared to other genes affecting the expected sample proportions of <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test. <a 
 id="x1-98003r121"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.3.2 </span> <a 
 id="x1-990002"></a>Multivariate Normal Simulation of Synthetic lethality
</h4>
<!--l. 288-->
<p class="noindent">
The multivariate normal simulation procedure was initially performed using the <span class="cmtt-12">mvtnorm </span>R package (<a 
href="#XGenz2009">Genz and Bretz</a>, <a 
href="#XGenz2009">2009</a>; <a 
href="#Xmvtnorm">Genz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xmvtnorm">2016</a>) (as described in Section <a 
href="#x1-940002">3.2<!--tex4ht:ref: methods:simulation_SL_expression --></a>) without correlation structure.
</p>
<!--l. 290-->
<p class="indent">
<p>Expression is sampled from multivariate normal distribution with a mean (<span class="cmmi-12">μ </span>= 0), standard deviation (<span class="cmmi-12">σ </span>= 1), and no correlation between genes (<span class="cmmi-12">r </span>= 0):</p>
</p>
<center class="math-display">
<img 
src="thesis9x.png" alt="X ~  N (¯&#x03BC;,&#x03A3; )  " class="math-display"  />
</center>
<p>Once a query gene consistent with synthetic lethality has been added, the simulated expression values are tested by <a 
href="#glo:SLIPT">SLIPT</a> exactly as described in Section <a 
href="#x1-930001">3.1<!--tex4ht:ref: methods:SLIPT --></a>.</p>
<!--l. 292-->
<p class="indent">
<a 
 id="x1-99004r11"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 295-->
<p class="noindent">
<a 
 id="x1-99001r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="SL_Model_Apr15mvnormCor_1K_ROC1_samplesx_prop".png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Statistical</span> <span class="cmr-10">evaluation</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-99002r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_Apr15mvnormCor_1K_ROC2_samplesx_prop".png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Receiver</span> <span class="cmr-10">operating</span> <span class="cmr-10">characteristic</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-99003r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_Apr15mvnormCor_1K_AUC_samplesx_prop".png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmr-10">Statistical</span> <span class="cmr-10">performance</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.11: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance of multivariate normal simulations.</span></strong> <span class="cmr-10x-x-109">Simulation of</span> <span class="cmr-10x-x-109">synthetic lethality was performed sampling from a multivariate normal distribution</span> <span class="cmr-10x-x-109">(without correlation structure). Performance of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">declines for more synthetic</span> <span class="cmr-10x-x-109">partners but this is mitigated by increased sample sizes (in darker colours). This generally</span> <span class="cmr-10x-x-109">occurs as the sensitivity decreases for a greater number of true positives to detect, leading</span> <span class="cmr-10x-x-109">to a trade off in accuracy as seen in a trough for false discovery rate and the ROC curves.</span></span></p>
</div>
<!--tex4ht:label?: x1-99004r11 -->
</div>
<hr class="endfloat" />
<!--l. 317-->
<p class="indent">
As shown in Figure <a 
href="#x1-99001r1">3.11(a)<!--tex4ht:ref: fig:simulation_Apr15ROC:Perf --></a>, the statistical accuracy of <a 
href="#glo:SLIPT">SLIPT</a> as a binary classifier is considerably high across simulations of a full human dataset of 20,000 genes. However, with the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> p-value as a threshold for prediction, this is largely to desirable specificity: the majority of non-SL genes are distinguished from the few underlying synthetic lethal genes. In this regard, the <a 
href="#glo:SLIPT">SLIPT</a> methodology generally performs better with larger datasets with more expected negatives and thus the results of simulations of smaller numbers of genes (such as the graph structures analysed in Chapter <a 
href="#x1-1750006">6<!--tex4ht:ref: chap:simulation --></a>) can be applied to larger datasets where they are expected to perform comparably or better with a lower false negative rate. Accordingly, key results will be supported by replication with larger numbers of non-SL genes added to the simulations.
</p>
<!--l. 319-->
<p class="indent">
However, with higher numbers of synthetic lethal genes to detect, the sensitivity (in Figure <a 
href="#x1-99001r1">3.11(a)<!--tex4ht:ref: fig:simulation_Apr15ROC:Perf --></a>) of <a 
href="#glo:SLIPT">SLIPT</a> as a binary classifier of synthetic lethality declines, although this is somewhat mitigated by higher sample sizes (shown in darker colours). Thus the minority of true synthetic lethal partners are more difficult to distinguish when there are more of them (and a weaker expression signal from each). While a reasonable reduction of the false discovery rate can be achieved for moderate numbers of underlying synthetic lethal partners, we can not be sure how many partners are expected to be detected in analyses of expression data. However this simulation procedure is amenable to assessing the performance of <a 
href="#glo:SLIPT">SLIPT</a> across simulation parameters, graph structures and comparisons to other approaches (presented in more detail in Chapter <a 
href="#x1-1750006">6<!--tex4ht:ref: chap:simulation --></a>).
</p>
<!--l. 321-->
<p class="indent">
<p>Not all of the genes detected by <a 
href="#glo:SLIPT">SLIPT</a> will be true synthetic lethals but these will be among the strongest candidates and it performs better with fewer underlying synthetic lethals to detect. This supports a focus on pathway analyses, in particular detecting pathways for further investigation. Since individually gene candidates are not necessarily gene synthetic lethal themselves, pathway over-representation analysis will be performed to detect functional groups recurrently detected by <a 
href="#glo:SLIPT">SLIPT</a> as these detection of functionally related genes further support their role in synthetic lethal relationships in addition to being biologically informative. Alternatively, pathway metagenes will reduce the number of underlying synthetic lethals to identify synthetic lethal pathways. Both of these approaches will be applied in Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a> to identify and replicate synthetic pathways of <span class="cmti-12">CDH1</span>. Pathways are also more likely to replicate across experimental models as demonstrated by <a 
href="#XDixon2008">Dixon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XDixon2008">2008</a>).</p>
</p>
<!--l. 323-->
<p class="indent">
The receiver operating characteristic curves (in Figure <a 
href="#x1-99002r2">3.11(b)<!--tex4ht:ref: fig:simulation_Apr15ROC:ROC --></a>) demonstrate that <a 
href="#glo:SLIPT">SLIPT</a> is subject to near equal trade-off between sensitivity and specificity across threshold values. The lower sensitivity and higher specificity with a binary classifiation (in Figure <a 
href="#x1-99001r1">3.11(a)<!--tex4ht:ref: fig:simulation_Apr15ROC:Perf --></a>) stems from stringent testing by <a 
href="#glo:SLIPT">SLIPT</a> with (FDR) p-values adjusted for multiple tests. The area under these curves is also used to compare statistical performce (in Figure <a 
href="#x1-99003r3">3.11(c)<!--tex4ht:ref: fig:simulation_Apr15ROC:AUC --></a>), with declining performance across increased underlying synthetic lethal partners and increased performance with sample size in multivariate normal simulations. <a 
 id="x1-99005r91"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">3.3.2.1 </span> <a 
 id="x1-1000001"></a>Multivariate Normal Simulation with Correlated Genes
</h5>
<!--l. 330-->
<p class="noindent">
<p>Correlation structures can be added to the simulation procedure (as discussed in Section <a 
href="#x1-940002">3.2<!--tex4ht:ref: methods:simulation_SL_expression --></a>), starting with simple correlated blocks of genes as the Σ parameter depicted in Figure <a 
href="#x1-100001r1">3.12(a)<!--tex4ht:ref: fig:simulation_May4SL:first --></a>. These correlated blocks represent genes with correlated expression such as that expected by coregulation or biological pathways. Figure <a 
href="#x1-100005r12">3.12<!--tex4ht:ref: fig:simulation_May4SL --></a> gives an example of 4 synthetic lethal genes (out of 100), each with 5 correlated genes that are not themselves synthetic lethal partners of the query gene. This serves to test whether synthetic lethal genes are distinguishable from correlated partners. This Σ matrix produces a similar correlation structure (Figure <a 
href="#x1-100002r2">3.12(b)<!--tex4ht:ref: fig:simulation_May4SL:second --></a>) in the resulting expression profiles (Figure <a 
href="#x1-100003r3">3.12(c)<!--tex4ht:ref: fig:simulation_May4SL:third --></a>) where apart from correlated blocks of genes (<span class="cmmi-12">r </span>= 0<span class="cmmi-12">.</span>8), the remaining genes have only slight variations due to random sampling. The structure of the dataset, particularly between synthetic lethal genes and the query, is shown at the gene expression (Figure <a 
href="#x1-100003r3">3.12(c)<!--tex4ht:ref: fig:simulation_May4SL:third --></a>) and function (Figure <a 
href="#x1-100004r4">3.12(d)<!--tex4ht:ref: fig:simulation_May4SL:fifth --></a>). These are ordered by the <a 
href="#glo:SLIPT">SLIPT</a> results and the synthetic lethal genes are ranked high, with the majority of them being distinguishable from highly correlated genes.</p>
</p>
<!--l. 332-->
<p class="indent">
<a 
 id="x1-100005r12"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 335-->
<p class="noindent">
<a 
 id="x1-100001r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_top(1)".png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Input</span> <span class="cmr-10">Σ</span> <span class="cmr-10">matrix</span> <span class="cmr-10">parameter</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-100002r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_top(2)".png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Simulated</span> <span class="cmr-10">correlation</span> <span class="cmr-10">matrix</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-100003r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_top(4)".png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmr-10">Simulated</span> <span class="cmr-10">gene</span> <span class="cmr-10">expression</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-100004r4"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_top(3)".png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(d)</span> <span class="cmr-10">Simulated</span> <span class="cmr-10">gene</span> <span class="cmr-10">function</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.12: </span><span class="content"><strong><span class="cmbx-10x-x-109">Simulating expression with correlated gene blocks.</span></strong> <span class="cmr-10x-x-109">A Σ matrix (a)</span> <span class="cmr-10x-x-109">is used generate a multivariate normal distribution with of 100 genes correlated blocks</span> <span class="cmr-10x-x-109">of genes (correlated by 0.8) with a comparable structure (b) to the input Σ, as shown by</span> <span class="cmr-10x-x-109">correlation on a red–green scale. The annotation bars for genes give the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">(in blue if the</span> <span class="cmr-10x-x-109">direction of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">is met or red otherwise) and the gene category (blue for query, cyan for</span> <span class="cmr-10x-x-109">query-correlated, red for SL, orange for SL-correlated, forest green for non-SL-correlated,</span> <span class="cmr-10x-x-109">and green for non-SL). The simulated gene expression (c) and function (d) generated are</span> <span class="cmr-10x-x-109">ordered by </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">showing the functional structure of synthetic lethal genes and that they</span> <span class="cmr-10x-x-109">are among the strongest </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">results.</span></span></p>
</div>
<!--tex4ht:label?: x1-100005r12 -->
</div>
<hr class="endfloat" />
<!--l. 361-->
<p class="indent">
The use of correlation structures generalises to larger datasets, such as 1000 genes shown in Figure <a 
href="#x1-100010r13">3.13<!--tex4ht:ref: fig:simulation_May4SL1K --></a>. Synthetic lethal genes are highly ranked by <a 
href="#glo:SLIPT">SLIPT</a> and still largely distinguishable from correlated genes. As previously discussed in Section <a 
href="#x1-990002">3.3.2<!--tex4ht:ref: chapt2:simulation_mvtnorm --></a>, these synthetic lethal genes are still detectable among a larger number of true negatives and the <a 
href="#glo:SLIPT">SLIPT</a> methodology performs better on such datasets.
</p>
<!--l. 363-->
<p class="indent">
<p>These plots (Figures <a 
href="#x1-100005r12">3.12<!--tex4ht:ref: fig:simulation_May4SL --></a> and <a 
href="#x1-100010r13">3.13<!--tex4ht:ref: fig:simulation_May4SL1K --></a>) also show similar correlated blocks with a non-synthetic lethal gene (true negative) and the query gene (which is not synthetic lethal with itself). Neither of these should be synthetic lethal (or detected to be) but they may impact upon the performance of the model, particularly the specificity as correlated negative genes may be distinguishable from true synthetic lethals. The non-synthetic lethal correlated block has no impact on synthetic lethal detection but the impact of query correlated genes will be discussed in Section <a 
href="#x1-1010002">3.3.2.2<!--tex4ht:ref: chapt2:simulation_mvtnorm_query_cor --></a> and Chapter <a 
href="#x1-1750006">6<!--tex4ht:ref: chap:simulation --></a>.</p>
</p>
<!--l. 365-->
<p class="indent">
<a 
 id="x1-100010r13"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 368-->
<p class="noindent">
<a 
 id="x1-100006r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_1K_top(1)".png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Input</span> <span class="cmr-10">Σ</span> <span class="cmr-10">matrix</span> <span class="cmr-10">parameter</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-100007r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_1K_top(2)".png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Simulated</span> <span class="cmr-10">correlation</span> <span class="cmr-10">matrix</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-100008r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_1K_top(4)".png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmr-10">Simulated</span> <span class="cmr-10">gene</span> <span class="cmr-10">expression</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-100009r4"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_May15mvnorm_heatmap_4SL_cor_comp_1K_top(3)".png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(d)</span> <span class="cmr-10">Simulated</span> <span class="cmr-10">gene</span> <span class="cmr-10">function</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.13: </span><span class="content"><strong><span class="cmbx-10x-x-109">Simulating expression with correlated gene blocks.</span></strong> <span class="cmr-10x-x-109">Using the (a)</span> <span class="cmr-10x-x-109">Σ matrix, sampling from a multivariate normal distribution with of 1000 genes produced</span> <span class="cmr-10x-x-109">(b) correlated blocks of genes (correlated by 0.8) on a red–green scale. The simulated gene</span> <span class="cmr-10x-x-109">expression (c) and function (d) generated are ordered by </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">and </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">direction show</span> <span class="cmr-10x-x-109">that synthetic lethal genes are among the strongest </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">results with high specficity</span> <span class="cmr-10x-x-109">against many potential false positives. These are annotated for </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">(on a red–green</span> <span class="cmr-10x-x-109">scale) and category (blue for query, cyan for query-correlated, red for SL, orange for</span> <span class="cmr-10x-x-109">SL-correlated, forest green for non-SL-correlated, and green for non-SL) for each gene.</span></span></p>
</div>
<!--tex4ht:label?: x1-100010r13 -->
</div>
<hr class="endfloat" />
<!--l. 395-->
<p class="indent">
<p>These simulations (on 100 genes) were repeated to examine the variation between detection on different samples and varying the number of underlying synthetic lethal partners, in simulated gene expression data with correlations structure. A small nuber (10 for each) simulations are shown in Figure <a 
href="#x1-100013r14">3.14<!--tex4ht:ref: fig:simulation_May4SLreps --></a> to demonstrate the variation between replicate simulations, with iterative sampling from the same multivariate normal distribution. These simulations show synthetic lethal genes are not only highly ranked by <a 
href="#glo:SLIPT">SLIPT</a> when there are few of them but also that they are fairly consistent across replicate simulations. Whereas they become less consistent for increasing numbers of true synthetic lethal partners to detect and thus more difficult to distinguish from other geners, particularly those correlated with them. Similarly, the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values show a marked stepwise increase with clear thresholds for SL and correlated genes in simple simulations, whereas these become less evident for higher numbers of SL partners.</p>
</p>
<!--l. 397-->
<p class="indent">
<a 
 id="x1-100013r14"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 400-->
<p class="noindent">
<a 
 id="x1-100011r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_May15mvnorm_heatmap_10XSL_cor_comp2(1)".png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Gene</span> <span class="cmr-10">category</span> <span class="cmr-10">in</span> <span class="cmr-10">simulations</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-100012r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_May15mvnorm_heatmap_10XSL_cor_comp2(2)".png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Corresponding</span> <span class="cmmi-10">χ</span><sup><span class="cmr-7">2</span></sup> <span class="cmr-10">values</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.14: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethal prediction across simulations.</span></strong> <span class="cmr-10x-x-109">The gene category</span> <span class="cmr-10x-x-109">(blue for query, cyan for query-correlated, red for SL, orange for SL-correlated, forest</span> <span class="cmr-10x-x-109">green for non-SL-correlated, and green for non-SL) ordered by </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">signed by the </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">directional condition is shown across simulations. For each of 1–10 SL partners, 10</span> <span class="cmr-10x-x-109">simulations demonstrate that the increasing numbers of SL partners become harder</span> <span class="cmr-10x-x-109">detect. The </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">values show a clear threshold for SL and correlated genes when there are</span> <span class="cmr-10x-x-109">fewer of them, distinguishable from correlated genes in this case.</span></span></p>
</div>
<!--tex4ht:label?: x1-100013r14 -->
</div>
<hr class="endfloat" />
<!--l. 419-->
<p class="indent">
<a 
 id="x1-100017r15"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 422-->
<p class="noindent">
<a 
 id="x1-100014r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_Apr15mvnorm_1K_ROC1_samplesx_prop".png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Statistical</span> <span class="cmr-10">evaluation</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-100015r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_Apr15mvnorm_1K_ROC2_samplesx_prop".png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Receiver</span> <span class="cmr-10">operating</span> <span class="cmr-10">characteristic</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-100016r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_Apr15mvnorm_1K_AUC_samplesx_prop".png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmr-10">Statistical</span> <span class="cmr-10">performance</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.15: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance with correlations.</span></strong> <span class="cmr-10x-x-109">Simulation of synthetic lethality was</span> <span class="cmr-10x-x-109">performed sampling from a multivariate normal distribution (with correlation structure).</span> <span class="cmr-10x-x-109">Performance of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">declines for more synthetic partners but this is mitigated by</span> <span class="cmr-10x-x-109">increased sample sizes (darker colours). This generally occurs as the sensitivity decreases</span> <span class="cmr-10x-x-109">for a greater number of true positives to detect, leading to a trade off in accuracy as seen</span> <span class="cmr-10x-x-109">in a trough for false discovery rate and the ROC curves.</span></span></p>
</div>
<!--tex4ht:label?: x1-100017r15 -->
</div>
<hr class="endfloat" />
<!--l. 444-->
<p class="indent">
<a 
 id="x1-100018r16"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 446-->
<p class="noindent">
</p>
<!--l. 447-->
<p class="noindent">
<span class="fbox"> <img 
src=""SL_Model_Test_Graph_10K_Graph1_ROC_Compare_Mvtn(Apry)_v_Cor(Aprxy)_Full".png" alt="PIC"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.16: </span><span class="content"><strong><span class="cmbx-10x-x-109">Comparison of statistical performance with correlation</span> <span class="cmbx-10x-x-109">structure.</span></strong> <span class="cmr-10x-x-109">Multivariate simulation of synthetic lethality with correlation structure (in</span> <span class="cmr-10x-x-109">colour) has comparable performance to simulation without correlations (in greyscale)</span> <span class="cmr-10x-x-109">with known synthetic partners across parameters.</span></span></p>
</div>
<!--tex4ht:label?: x1-100018r16 -->
</div>
<hr class="endfloat" />
<!--l. 459-->
<p class="indent">
Whether the synthetic lethal genes detected in simple simulations (in Figure <a 
href="#x1-100013r14">3.14<!--tex4ht:ref: fig:simulation_May4SLreps --></a>) are robustly detectable across greater number of simulations, in addition to further comparions, was tested with a supporting ROC analysis. These results (in Figure <a 
href="#x1-100017r15">3.15<!--tex4ht:ref: fig:simulation_Apr15ROC2 --></a>) are very similar to simulations without correlation structure, with <a 
href="#glo:SLIPT">SLIPT</a> as a binary classifier having a poor sensitivity with increasing numbers of synthetic lethal partners to detect but high specificity in a total of 20,000 genes with the vast majority being true negatives. This is reflected in a similar decline in statistical performance for increasing numbers of synthetic lethal partners and a compensating increase in performance with higher sample size. Overall, the statistical performance is very similar to simulations without correlation structure (as shown in Figure <a 
href="#x1-100018r16">3.16<!--tex4ht:ref: fig:mvtnorm_cor_compare --></a>).
</p>
<!--l. 461-->
<p class="indent">
<p>Thus <a 
href="#glo:SLIPT">SLIPT</a> is robust across correlation structures and applicable to real gene expresssion data where pathway structures and correlations are a consideration. These correlation structures are not intended to model specific biological pathways or represent them, rather they serve to test the impact of correlation structure on the performance of <a 
href="#glo:SLIPT">SLIPT</a> with an extreme example of closely correlated (<span class="cmmi-12">r </span>= 0<span class="cmmi-12">.</span>8) gene blocks. More complex correlation structures, such as genes postively correlated with the query gene and derived from pathway graph structures (as described in <a 
href="#x1-1070002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>) will be examined below (in Section <a 
href="#x1-1010002">3.3.2.2<!--tex4ht:ref: chapt2:simulation_mvtnorm_query_cor --></a>) and in Chapter <a 
href="#x1-1750006">6<!--tex4ht:ref: chap:simulation --></a> respectively.</p>
</p>
<!--l. 465-->
<p class="indent">
<a 
 id="x1-100022r17"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 468-->
<p class="noindent">
<a 
 id="x1-100019r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_May15mvnorm_dir_1K_ROC1_samplesx_prop".png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Statistical</span> <span class="cmr-10">evaluation</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-100020r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_May15mvnorm_dir_1K_ROC2_samplesx_prop".png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Receiver</span> <span class="cmr-10">operating</span> <span class="cmr-10">characteristic</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-100021r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_May15mvnorm_dir_1K_AUC_samplesx_prop".png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmr-10">Statistical</span> <span class="cmr-10">performance</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.17: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance with query correlations.</span></strong> <span class="cmr-10x-x-109">Simulation of synthetic</span> <span class="cmr-10x-x-109">lethality was performed sampling from a multivariate normal distribution (with</span> <span class="cmr-10x-x-109">correlation structure including correlated genes with non-SL and query genes). As before,</span> <span class="cmr-10x-x-109">performance of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">declines for more synthetic partners and is mitigated by increased</span> <span class="cmr-10x-x-109">sample sizes (darker colours)but the sensitivity remains higher for a greater number of</span> <span class="cmr-10x-x-109">true positives with corresponding improvements in ROC curves.</span></span></p>
</div>
<!--tex4ht:label?: x1-100022r17 -->
</div>
<hr class="endfloat" />
<!--l. 490-->
<p class="indent">
In particular, genes correlated with true synthetic lethal genes have little impact on the performance of <a 
href="#glo:SLIPT">SLIPT</a> detection: synthetic lethal genes are as distinguishable from true negative genes as without correlated genes. Synthetic lethal correlated genes will not interfere detect of true synthetic lethals, although they may be ranked next below them and be biologically informative with related gene functions. <a 
 id="x1-100023r126"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">3.3.2.2 </span> <a 
 id="x1-1010002"></a>Specificity with Query-Correlated Pathways
</h5>
<!--l. 496-->
<p class="noindent">
Another consideration for correlation structures is postively correlated genes with the query that are not synthetic lethal. As described in Section <a 
href="#x1-1000001">3.3.2.1<!--tex4ht:ref: chapt2:simulation_mvtnorm_cor --></a>, 5 highly correlated (<span class="cmmi-12">r </span>= 0<span class="cmmi-12">.</span>8) with the query gene were added. These simulations perform similarly to before (in Figure <a 
href="#x1-100022r17">3.17<!--tex4ht:ref: fig:simulation_May15ROC2 --></a>) with a higher specficity and a lower false discovery rate being feasible (as shown in <a 
href="#x1-100019r1">3.17(a)<!--tex4ht:ref: fig:simulation_May15ROC:Perf --></a>). <a 
 id="x1-101001r42"></a>
</p>
<a 
 id="x1-1020001"></a>Importance of Directional Testing <!--l. 519-->
<p class="indent">
<p>It is important to notice here that the directional criteria of the <a 
href="#glo:SLIPT">SLIPT</a> procedure is enhancing it’s performance, particularly in distinguishing positively correlated true negatives. The multivariate normal simulation results, with 20,000 genes including all of the correlation structures discussed (SL, non-SL, and query correlated genes), are compared here for <a 
href="#glo:SLIPT">SLIPT</a> with and without (<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>) directional testing. There is a marked improvement in statistical performance with directional criteria, particularly with increased sensitivity and lower false discovery rate (as shown in Figure <a 
href="#x1-102003r18">3.18<!--tex4ht:ref: fig:mvtnorm_dir_compare --></a>).</p>
</p>
<!--l. 521-->
<p class="indent">
<a 
 id="x1-102003r18"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 524-->
<p class="noindent">
<a 
 id="x1-102001r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<span class="fbox"> <img 
src=""SL_Model_May15mvnorm_1K_ROC1_samplesx_prop".png" alt="PIC"  
 /> </span>
<div class="subfigcaption">
<p><span class="cmr-10">(a) </span><span class="cmmi-10">χ</span><sup><span class="cmr-7">2</span></sup> <span class="cmr-10">testing</span> <span class="cmr-10">without</span> <span class="cmr-10">direction</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-102002r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<span class="fbox"> <img 
src=""SL_Model_May15mvnorm_dir_1K_ROC1_samplesx_prop".png" alt="PIC"  
 /> </span>
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <a 
href="#glo:SLIPT"><span class="cmr-10">SLIPT</span></a> <span class="cmr-10">with</span> <span class="cmr-10">directional</span> <span class="cmr-10">criteria</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.18: </span><span class="content"><strong><span class="cmbx-10x-x-109">Statistical evaluation of directional criteria.</span></strong> <span class="cmr-10x-x-109">A simulated</span> <span class="cmr-10x-x-109">multivariate normal dataset of 20,000 genes with correlation structures was tested by</span> <a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">with the directional condition and and the equivalent </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test without. </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">exhibited a consistently higher sensitivity and lower false discovery rate.</span></span></p>
</div>
<!--tex4ht:label?: x1-102003r18 -->
</div>
<hr class="endfloat" />
<!--l. 546-->
<p class="indent">
<a 
 id="x1-102007r19"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 549-->
<p class="noindent">
<a 
 id="x1-102004r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<span class="fbox"> <img 
src=""SL_Model_May15mvnorm_1K_ROC2_samplesx_prop".png" alt="PIC"  
 /> </span>
<div class="subfigcaption">
<p><span class="cmr-10">(a) </span><span class="cmmi-10">χ</span><sup><span class="cmr-7">2</span></sup> <span class="cmr-10">testing</span> <span class="cmr-10">without</span> <span class="cmr-10">direction</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-102005r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<span class="fbox"> <img 
src=""SL_Model_May15mvnorm_dir_1K_ROC2_samplesx_prop".png" alt="PIC"  
 /> </span>
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <a 
href="#glo:SLIPT"><span class="cmr-10">SLIPT</span></a> <span class="cmr-10">with</span> <span class="cmr-10">directional</span> <span class="cmr-10">criteria</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-102006r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""SL_Model_Test_Graph_10K_Graph1_ROC_Compare_Mvtn(Mayy)_v_Cor(Mayxy)_Full".png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmr-10">Statistical</span> <span class="cmr-10">performance</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.19: </span><span class="content"><strong><span class="cmbx-10x-x-109">Performance with directional criteria.</span></strong> <span class="cmr-10x-x-109">A simulated multivariate</span> <span class="cmr-10x-x-109">normal dataset of 20,000 genes with correlation structures was tested by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">with</span> <span class="cmr-10x-x-109">the directional condition and and the equivalent </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test without. </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">has higher</span> <span class="cmr-10x-x-109">performance across simulation parameters, clearly differing from random (grey diagonal)</span> <span class="cmr-10x-x-109">in ROC curves up to 100 SL genes (b). The performance (c) of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">(in greyscale) was</span> <span class="cmr-10x-x-109">consistently higher than the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test (in color).</span></span></p>
</div>
<!--tex4ht:label?: x1-102007r19 -->
</div>
<hr class="endfloat" />
<!--l. 577-->
<p class="indent">
This is encouraging for the application of <a 
href="#glo:SLIPT">SLIPT</a> to empirical expression datasets as postively correlated genes are likely to occur and the directional condition robustly improves the performance of <a 
href="#glo:SLIPT">SLIPT</a> across simulation parameters. Without assuming the underlying number of synthetic lethal genes, <a 
href="#glo:SLIPT">SLIPT</a> will perform better than the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test alone at detecting them. This is further supported irrespective of significance threshold for the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test by the ROC analysis in Figure <a 
href="#x1-102007r19">3.19<!--tex4ht:ref: fig:mvtnorm_dir_compare2 --></a>. The directional <a 
href="#glo:SLIPT">SLIPT</a> methodology outperforms the ordinary <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> test at detecting synthetic lethal partners with some predictive power (above random and AUROC of 0.5) even up to 100 synthetic lethal genes.
</p>
<!--l. 579-->
<p class="indent">
Together these simulation results support the application of the <a 
href="#glo:SLIPT">SLIPT</a> methodology as it has been performed throughout Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a> and <a 
href="#x1-1530005">5<!--tex4ht:ref: chap:Pathways --></a>. However, the methodology and simulation procedure will explored in more detail in Chapter <a 
href="#x1-1750006">6<!--tex4ht:ref: chap:simulation --></a>, with the inclusion of graph structures and comparison to other synthetic lethal detection approaches. <a 
 id="x1-102008r120"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">3.4 </span> <a 
 id="x1-1030004"></a>Graph Structure Methods
</h3>
<!--l. 584-->
<p class="noindent">
Graph structures have been used in several ways in this project with novel approaches to analysis and simulations. Procedures were developed for statistical and network analysis of gene states in pathway structures. Specifically, the relationships between <a 
href="#glo:siRNA">siRNA</a> and <a 
href="#glo:SLIPT">SLIPT</a> genes were tested within biological pathways in Chapter <a 
href="#x1-1530005">5<!--tex4ht:ref: chap:Pathways --></a>. These graph structures were also used in Chapter <a 
href="#x1-1750006">6<!--tex4ht:ref: chap:simulation --></a> for the simulation of synthetic lethality to derive correlation structure between simulated gene expression profiles in manner that resembles biological pathways. <a 
 id="x1-103001r124"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.4.1 </span> <a 
 id="x1-1040001"></a>Upstream and Downstream Gene Detection
</h4>
<!--l. 588-->
<p class="noindent">
<p>Comparison of experimental and computational candidate synthetic lethal partner genes within pathway structures arose from the hypothesis that these sets of genes were related by pathway structure. Due to differences in how these candidates were generated, it should not be expected that they detect the identical genes within the candidate biological pathways, rather they may be related by being upstream or downstream of each other.</p>
</p>
<!--l. 590-->
<p class="indent">
Using the Reactome version 52 data (<a 
href="#XReactome">Croft <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XReactome">2014</a>) as described in Section <a 
href="#x1-850002">2.4.2<!--tex4ht:ref: methods:graph_data --></a>, genes identified by each synthetic lethal discovery approach were mapped to the graph structure for the candidate pathways identified in Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a> (with subgraphs defined as described in Section <a 
href="#x1-860003">2.4.3<!--tex4ht:ref: methods:subgraphs --></a>). To test whether <a 
href="#glo:siRNA">siRNA</a> candidate genes were upstream of <a 
href="#glo:SLIPT">SLIPT</a> candidate genes, shortest paths were traced between each potential pair of these genes in a directed network. The number of genes where the <a 
href="#glo:siRNA">siRNA</a> candidate was upstream were scored “up” and where the <a 
href="#glo:siRNA">siRNA</a> candidate was downstream were scored “down”. This procedure enabled counting the total number of shortest paths which supported <a 
href="#glo:siRNA">siRNA</a> genes being upstream or downstream of the <a 
href="#glo:SLIPT">SLIPT</a> genes and measuring the difference between these to determine if there is an imbalance in a particular direction. While this difference is indicative of the number of paths between the gene candidate groups in either direction, alone it is not sufficient to statistically support structure or relationships between <a 
href="#glo:siRNA">siRNA</a> and <a 
href="#glo:SLIPT">SLIPT</a> genes. However, it may be combined with a permutation resampling procedure (as described in Section <a 
href="#x1-1050001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>) to test for directional relationships in either direction.
</p>
<!--l. 592-->
<p class="indent">
The original version of this procedure excluded gene detected by both approaches since they would count in both directions. Upon further consideration, the intersection genes were restored to being accounted for by the shortest paths counts since they may count unequally to being upstream or downstream of each gene set if there are unequal numbers above or below them in the pathway structure. <a 
 id="x1-104001r133"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">3.4.1.1 </span> <a 
 id="x1-1050001"></a>Permutation Analysis for Statistical Significance
</h5>
<!--l. 595-->
<p class="noindent">
A permutation procedure was developed to randomly assign members of the pathway to <a 
href="#glo:siRNA">siRNA</a> and/or <a 
href="#glo:SLIPT">SLIPT</a> groups, with the same number of each candidate partner gene set as observed in the pathway. These permuted genes are measured for pathway structure between the permuted gene groups as performed for the observed candidates (as performed in Section <a 
href="#x1-1040001">3.4.1<!--tex4ht:ref: methods:pathway_str --></a>). A distribution of pathway structure relationships expected by chance is generated by permuting iteratively over these pathways. This null distribution can be compared to the observed counts of relationships (in either direction), which yields a permutation p-value as the proportion of permutations in which had value or greater or more extreme magnitude than the observed value.
</p>
<!--l. 597-->
<p class="indent">
<p>The null hypothesis is that there is no relationship between these gene groups that would not have occurred had the genes been selected at random. Thus we can test both the</p>
alternate hypothesis that the <a 
href="#glo:siRNA">siRNA</a> genes were upstream of the <a 
href="#glo:SLIPT">SLIPT</a> genes or that they are downstream of them.
</p>
<!--l. 599-->
<p class="indent">
The permutation procedure does not assume the underlying distribution of the data under the null hypothesis and accounts for the total number of nodes, edges, <a 
href="#glo:siRNA">siRNA</a>, and <a 
href="#glo:SLIPT">SLIPT</a> genes in each pathway network structure. The intersection size of the <a 
href="#glo:siRNA">siRNA</a> and <a 
href="#glo:SLIPT">SLIPT</a> genes was originally not accounted for under the shortest path counts procedure that excluded them. A refined version of this procedure ensured that the number of intersecting genes was equal to the number observed to test for pathway structure without changing the intersection size, the subject of prior analyses. <a 
 id="x1-105001r139"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">3.4.1.2 </span> <a 
 id="x1-1060002"></a>Hierarchy Based on Biological Context
</h5>
<!--l. 602-->
<p class="noindent">
An alternative approach to pathway structure was performed based on the biological context that genes at the upstream and downstream ends of a pathway perform different functions, such as a kinase signalling cascade recieving signals from external stimuli and passes these on ribosomes or the nucleus. The genes were assigned to a hierarchy to determine if genes of either candidate group (or those with stronger support for either group) performed upstream or downstream functions disproportionately.
</p>
<!--l. 604-->
<p class="indent">
A network-based approach was used to determine the pathway hierarchy of genes in a computationally rational way when applied to different biological pathways with a directed graph structure, <span class="cmmi-12">G </span>(without loops). The diameter of the network (i.e., the length of longest possible shortest path between the most distant genes) was used to identify a gene (<span class="cmmi-12">z</span>) at the downstream end of the pathway (at the end of a diameter spanning shortest path), assigned a hierarchy of:
</p>
<center class="math-display">
<img 
src="thesis10x.png" alt="hierarchy (z) = 1 + diameter (G)  " class="math-display"  />
</center>
<p>Having identified the downstream end of the pathway, genes upstream (e.g., gene <span class="cmmi-12">i</span>) of</p>
this are assigned a hierarchy by the length of their shortest path (<span class="cmmi-12">d</span>) to this gene, <span class="cmmi-12">z</span>.
<center class="math-display">
<img 
src="thesis11x.png" alt="hierarchy (i) = hierarchy (z ) - diz  " class="math-display"  />
</center>
The remaining unassigned genes (e.g., gene <span class="cmmi-12">j</span>) gain the hierarchy of the length of the shortest path downstream from the nearest assigned gene if possible.
<center class="math-display">
<img 
src="thesis12x.png" alt="hierarchy (j) = hierarchy (i) + dij  " class="math-display"  />
</center>
This process may be performed iteratively to fill in pathway hierarchy but it was not necessary to perform further iterations for the candidate synthetic lethal pathways investigated (amenable to this procedure) which exhibited clear directional structure and the small world property (with a low diameter). Thus genes in a pathway graph structure were assigned integer valued hierarchy upstream to downstream by this procedure:
<center class="math-display">
<img 
src="thesis13x.png" alt="hierarchy &#x2208; {1, 2,3,...,1 + diameter (G)} " class="math-display"  />
</center>
<p>This</p>
hierarchy of pathway directionality (such as that shown in Figure <a 
href="#x1-166001r7">5.7<!--tex4ht:ref: fig:SL_Pathway_PI3K_Ranking --></a>) can be used for comparison with measures of the number of genes of each candidate group and the support for being synthetic lethal partners with either approach. <a 
 id="x1-106001r138"></a>
<h4 class="subsectionHead">
<span class="titlemark">3.4.2 </span> <a 
 id="x1-1070002"></a>Simulating Gene Expression from Graph Structures
</h4>
<!--l. 609-->
<p class="noindent">
<p>A further refinement of the simulation procedure generated expression data with correlation structure, derived from a known graph structure. This enables modelling of synthetic lethal partners within a biological pathway and the investigation of impact of pathway structure on synthetic lethal prediction. A simulated pathway is first constructed as a graph structure, with the <span class="cmtt-12">igraph </span>R package <a 
href="#Xigraph">Csardi and Nepusz</a> (<a 
href="#Xigraph">2006</a>), with the added annotation of the state of the edges (i.e, whether they activate or inhibit downstream pathway members). This simulation procedure was intended for biological pathway members with correlated gene expression (higher than the background of genes in other pathways) but it may also be applicable to modelling protein levels (in a kinase regulation cascade) or substrates and products (in a metabolic pathway).</p>
</p>
<!--l. 612-->
<p class="indent">
<a 
 id="x1-107003r20"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 615-->
<p class="noindent">
<a 
 id="x1-107001r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<span class="fbox"> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/simple_graph".png  /> </span>
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Activating</span> <span class="cmr-10">pathway</span> <span class="cmr-10">structure</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-107002r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<span class="fbox"> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/simple_graph_inhibiting".png  /> </span>
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Pathway</span> <span class="cmr-10">structure</span> <span class="cmr-10">including</span> <span class="cmr-10">inhibitions</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.20: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulated graph structures.</span></strong></strong> <span class="cmr-10x-x-109">A constructed graph structure used as</span> <span class="cmr-10x-x-109">an example to demonstrate the simulation procedure. Activating links are denoted by</span> <span class="cmr-10x-x-109">blue arrows and inhibiting links by red edges.</span></span></p>
</div>
<!--tex4ht:label?: x1-107003r20 -->
</div>
<hr class="endfloat" />
<!--l. 637-->
<p class="indent">
<p>First, the graph structure is constructed for simulated data to be generated from (by sampling from a multivariate normal distribution using the <span class="cmtt-12">mvtnorm </span>R package (<a 
href="#XGenz2009">Genz and Bretz</a>, <a 
href="#XGenz2009">2009</a>; <a 
href="#Xmvtnorm">Genz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xmvtnorm">2016</a>)). Throughout this section, the simulation procedure will be demonstrated with the relatively simple constructed graph structure shown in Figure <a 
href="#x1-107003r20">3.20<!--tex4ht:ref: fig:simple_graph --></a>. This graph structure visualisation was specifically developed for (directed) iGraph objects in R and has been released in the <span class="cmtt-12">plot.igraph</span> package and <span class="cmtt-12">igraph.extensions </span>library (see Table <a 
href="#x1-90004r6">2.6<!--tex4ht:ref: tab:computers_r_packages_dev --></a> and Section <a 
href="#x1-1110003">3.5.3<!--tex4ht:ref: methods:igraph_extensions --></a>). The <span class="cmtt-12">plot</span><span class="cmtt-12">_directed </span>function allows customisation of plot parameters for each node or edge and mixed (directed) edge types for indicating activation or inhibition. These inhibition links (which often occur in biological pathways) are demonstrated in Figure <a 
href="#x1-107002r2">3.20(b)<!--tex4ht:ref: fig:simple_graph:second --></a>.</p>
</p>
<!--l. 639-->
<p class="indent">
<a 
 id="x1-107010r21"></a>
</p>
<hr class="float" />
<div class="float" 
>


<hr />
____________________________________________________________________________________     
<div class="center" 
>
<!--l. 642-->
<p class="noindent">
<a 
 id="x1-107004r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/simple_graph".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Activating</span> <span class="cmr-10">pathway</span> <span class="cmr-10">structure</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-107005r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/dist_mat".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Distance</span> <span class="cmr-10">matrix</span></p>
</div>
<pre><code> &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/div&gt;</code></pre>
<a 
 id="x1-107006r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/sigma_mat".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmr-10">Sigma,</span> <span class="cmr-10">Σ</span> <span class="cmr-10">(expected</span> <span class="cmr-10">correlation)</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-107007r4"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_cor_mat".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(d)</span> <span class="cmr-10">Simulated</span> <span class="cmr-10">correlation</span> <span class="cmr-10">structure</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-107008r5"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_mat".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(e)</span> <span class="cmr-10">Simulated</span> <span class="cmr-10">expression</span> <span class="cmr-10">data</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-107009r6"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_disc_mat".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(f)</span> <span class="cmr-10">Simulated</span> <span class="cmr-10">gene</span> <span class="cmr-10">function</span> <span class="cmr-10">calls</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.21: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulating expression from a graph structure.</span></strong></strong> <span class="cmr-10x-x-109">An example graph</span> <span class="cmr-10x-x-109">structure is used to derive a correlation structure from the relative distances between</span> <span class="cmr-10x-x-109">nodes and simulate continuous gene expression with sampling from the multivariate</span> <span class="cmr-10x-x-109">normal distribution.</span></span></p>
</div>
<!--tex4ht:label?: x1-107010r21 -->
</div>
<hr class="endfloat" />
<!--l. 677-->
<p class="indent">
The simulation procedure is designed to use such graph structures to inform development of a “Sigma” variance-covariance matrix (Σ) for sampling from a multivariate normal distribution (using the <span class="cmtt-12">mvtnorm </span>R package). Given a graph structure (or adjacency matrix), such as Figure <a 
href="#x1-107004r1">3.21(a)<!--tex4ht:ref: fig:simulation_activating:first --></a>, a relation matrix is calculated based on distance such that nearer nodes are given higher weight than farther nodes. For the purposes of this thesis a geometrically decreasing (relative) distance weighting is used, with each more distant node being related by <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">2</span> compared to the next nearest as shown in Figure <a 
href="#x1-107005r2">3.21(b)<!--tex4ht:ref: fig:simulation_activating:second --></a>. However, an arithmetically decreasing (absolute) distance weighting is also available in the <span class="cmtt-12">graphsim </span>R package release of this procedure.
</p>
<!--l. 679-->
<p class="indent">
<p>A Σ matrix is derived from this distance weighting matrix, creating a matrix (with a diagonal of 1) where each node has a variance and standard deviation of 1. Thus covariances between adjacent nodes are assigned by a correlation parameter and the remaining matrix based on weighting these correlations with by the distance matrix (or the nearest “positive definite” matrix). For the purposes of this thesis, the correlation parameter is 0<span class="cmmi-12">.</span>8 unless otherwise specified (as used for the example in Figure <a 
href="#x1-107006r3">3.21(c)<!--tex4ht:ref: fig:simulation_activating:third --></a>). This Σ matrix is used to sample from a multivariate normal distribution with each gene having a mean of 0, standard deviation 1, and covariance within the range [0<span class="cmmi-12">, </span>1] such that they are correlations. This procedure generates a simulated (continuous normally distributed) expression profile for each node (as shown in Figure <a 
href="#x1-107008r5">3.21(e)<!--tex4ht:ref: fig:simulation_activating:fourth --></a>) with corresponding correlation structure (Figure <a 
href="#x1-107007r4">3.21(d)<!--tex4ht:ref: fig:simulation_activating:fifth --></a>). The simulated correlation structure closely resembles the expected correlation structure (Sigma in <a 
href="#x1-107006r3">3.21(c)<!--tex4ht:ref: fig:simulation_activating:third --></a>) even for the relatively modest sample size (<span class="cmmi-12">N </span>= 100) illustrated in <a 
href="#x1-107010r21">3.21<!--tex4ht:ref: fig:simulation_activating --></a>. Once a simulated gene expression dataset has been generated (as in Figure <a 
href="#x1-107008r5">3.21(e)<!--tex4ht:ref: fig:simulation_activating:fourth --></a>), then a discrete matrix of gene function can be constructed with a functional threshold quantile to simulate functional relationships of synthetic lethality (as shown in Figure <a 
href="#x1-95002r4">3.4<!--tex4ht:ref: fig:SL_Model_Expression --></a>). For the purposes of this thesis, this threshold is the 0.3 quantile (as discussed in Section <a 
href="#x1-950001">3.2.1<!--tex4ht:ref: methods:SL_Model --></a>) which generates functional discrete matrices such as those used for synthetic lethal simulation in Section <a 
href="#x1-960002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a> (as shown Figure <a 
href="#x1-107009r6">3.21(f)<!--tex4ht:ref: fig:simulation_activating:sixth --></a>)</p>
</p>
<!--l. 681-->
<p class="indent">
<a 
 id="x1-107017r22"></a>
</p>
<hr class="float" />
<div class="float" 
>


<hr />
____________________________________________________________________________________     
<div class="center" 
>
<!--l. 684-->
<p class="noindent">
<a 
 id="x1-107011r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/simple_graph_inhibiting".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Pathway</span> <span class="cmr-10">structure</span> <span class="cmr-10">with</span> <span class="cmr-10">inhibition</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-107012r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/dist_mat".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Distance</span> <span class="cmr-10">matrix</span></p>
</div>
<pre><code> &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/div&gt;</code></pre>
<a 
 id="x1-107013r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/sigma_mat_inhibiting".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmr-10">Sigma,</span> <span class="cmr-10">Σ</span> <span class="cmr-10">(expected</span> <span class="cmr-10">correlation)</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-107014r4"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_cor_mat_inhibiting".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(d)</span> <span class="cmr-10">Simulated</span> <span class="cmr-10">correlation</span> <span class="cmr-10">structure</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-107015r5"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_mat_inhibiting".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(e)</span> <span class="cmr-10">Simulated</span> <span class="cmr-10">expression</span> <span class="cmr-10">data</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-107016r6"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/expr_inhib_disc_mat".png  />
<div class="subfigcaption">
<p><span class="cmr-10">(f)</span> <span class="cmr-10">Simulated</span> <span class="cmr-10">gene</span> <span class="cmr-10">function</span> <span class="cmr-10">calls</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.22: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulating expression from graph structure with inhibitions.</span></strong></strong> <span class="cmr-10x-x-109">An example graph structure is used to derive a correlation structure from the relative</span> <span class="cmr-10x-x-109">distances between nodes and simulate continuous gene expression with sampling from</span> <span class="cmr-10x-x-109">the multivariate normal distribution.</span></span></p>
</div>
<!--tex4ht:label?: x1-107017r22 -->
</div>
<hr class="endfloat" />
<!--l. 719-->
<p class="indent">
The simulation procedure (depicted in Figure <a 
href="#x1-107010r21">3.21<!--tex4ht:ref: fig:simulation_activating --></a>) is amenable to pathways containing inhibition links (as shown in Figure <a 
href="#x1-107017r22">3.22<!--tex4ht:ref: fig:simulation_inhibiting --></a>) with several refinements. With the inhibition links (as shown in Figure <a 
href="#x1-107011r1">3.22(a)<!--tex4ht:ref: fig:simulation_inhibiting:first --></a>), distances are calculated in the same manner as before (Figure <a 
href="#x1-107012r2">3.22(b)<!--tex4ht:ref: fig:simulation_inhibiting:second --></a>) with inhibitions accounted for by iteratively multiplying downstream nodes by <span class="cmsy-10x-x-120">-</span>1 to form blocks of negative correlations (as shown in Figures <a 
href="#x1-107013r3">3.22(c)<!--tex4ht:ref: fig:simulation_inhibiting:third --></a> and <a 
href="#x1-107014r4">3.22(d)<!--tex4ht:ref: fig:simulation_inhibiting:fifth --></a>). As before, a multivariate normal distribution with these negative correlations can be sampled to generate simulated data (as shown in Figures <a 
href="#x1-107015r5">3.22(e)<!--tex4ht:ref: fig:simulation_inhibiting:fourth --></a> and <a 
href="#x1-107016r6">3.22(f)<!--tex4ht:ref: fig:simulation_inhibiting:sixth --></a>).
</p>
<!--l. 721-->
<p class="indent">
These simulated datasets are amenable to simulating synthetic lethal partners of a query gene within a graph network. The query gene is assumed to be separate from the graph network pathway and is added to the dataset using the procedure in Section <a 
href="#x1-960002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a>. Thus we can simulate known synthetic lethal partner genes within a synthetic lethal partner pathway structure. <a 
 id="x1-107018r137"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">3.5 </span> <a 
 id="x1-1080005"></a>Customised Functions and Packages Developed
</h3>
<!--l. 744-->
<p class="noindent">
[Move to Appendix?]
</p>
<!--l. 746-->
<p class="indent">
Various R packages have been developed throughout this thesis using <span class="cmtt-12">devtools</span> (<a 
href="#Xdevtools">Wickham and Chang</a>, <a 
href="#Xdevtools">2016</a>) and <span class="cmtt-12">roxygen </span>(<a 
href="#Xroxygen">Wickham <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xroxygen">2017</a>) to enable reproducibility of customised analysis and visualisation. Many of these have the added benefit of the functions being documented, demonstrated in example vignettes, and released on GitHub to enable the research community to access utilise them in their own analysis. These are summarised in Table <a 
href="#x1-90004r6">2.6<!--tex4ht:ref: tab:computers_r_packages_dev --></a>, along with the corresponding urls for their GitHub repository which contains a README file with instructions for installation with the <span class="cmtt-12">devtools </span>R package (<a 
href="#Xdevtools">Wickham and Chang</a>, <a 
href="#Xdevtools">2016</a>) and links to the relevant vignette(s) where available. <a 
 id="x1-108001r141"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.5.1 </span> <a 
 id="x1-1090001"></a>Synthetic Lethal Interaction Prediction Tool
</h4>
<!--l. 749-->
<p class="noindent">
<p>The statistical methodology for detection of synthetic lethality in gene expression data (<a 
href="#glo:SLIPT">SLIPT</a>) is one of the main novel procedures developed in this thesis, as described in Section <a 
href="#x1-930001">3.1<!--tex4ht:ref: methods:SLIPT --></a>. The <span class="cmtt-12">slipt </span>R package has been prepared for release to accompany a publication</p>
demonstrating the applications of the methodology for identifying candidate interacting genes and pathways with <span class="cmti-12">CDH1 </span>in breast cancer (<a 
href="#XTCGA2012">TCGA</a>, <a 
href="#XTCGA2012">2012</a>).
</p>
<!--l. 751-->
<p class="indent">
<a 
href="#glo:SLIPT">SLIPT</a> is amenable to analysis of any effectively continuous measure of gene activity (e.g., microarray, RNA-Seq, protein abundance, or pathway metagenes). Executing <span class="cmtt-12">slipt </span>is straightforward: the <span class="cmtt-12">prep</span><span class="cmtt-12">_data</span><span class="cmtt-12">_for</span><span class="cmtt-12">_SL </span>function scores samples as “low”, “medium”, or “high” for each gene, then the <span class="cmtt-12">detect</span><span class="cmtt-12">_SL </span>function tests a given query gene against all potential partners by performing the chi-squared test and directional conditions. This function returns a table summarising the observed and expected sample numbers used for the directional criteria, the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values, and corresponding p-values including adjusting for multiple comparisons. The <span class="cmtt-12">count</span><span class="cmtt-12">_of</span><span class="cmtt-12">_SL </span>and <span class="cmtt-12">table</span><span class="cmtt-12">_of</span><span class="cmtt-12">_SL </span>functions serve to facilitate summary and extraction of the positive <a 
href="#glo:SLIPT">SLIPT</a> hits, respectively, from the table of predictions of synthetic lethal partners.
</p>
<!--l. 753-->
<p class="indent">
The <a 
href="#glo:SLIPT">SLIPT</a> methodology in this package release has been used in later analyses rather than the corresponding source R code, including use on remote machines and upon simulated data. In particular, the functions in the package facilitate alterations to parameters, such as the proportion of samples called as exhibiting low or high gene activity. This release support reproducibility and enables wider use of <a 
href="#glo:SLIPT">SLIPT</a> in future investigations into other disease genes. <a 
 id="x1-109001r146"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.5.2 </span> <a 
 id="x1-1100002"></a>Data Visualisation
</h4>
<!--l. 756-->
<p class="noindent">
<p>Customisations to existing data visualisations in R have been developed to present data throughout this thesis. The <span class="cmtt-12">vioplotx </span>and <span class="cmtt-12">heatmap.2x </span>packages are enhancements of the <span class="cmtt-12">vioplot </span>package (<a 
href="#Xvioplot">Adler</a>, <a 
href="#Xvioplot">2005</a>) and <span class="cmtt-12">heatmap.2 </span>provided by the <span class="cmtt-12">gplots </span>package (<a 
href="#Xgplots">Warnes <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xgplots">2015</a>).</p>
</p>
<!--l. 758-->
<p class="indent">
<a 
 id="x1-110003r23"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 761-->
<p class="noindent">
<a 
 id="x1-110001r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""vioplot1".png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Customised</span> <span class="cmr-10">violin</span> <span class="cmr-10">plot</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-110002r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src=""vioplot2".png" alt="PIC"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Split</span> <span class="cmr-10">violin</span> <span class="cmr-10">plot</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.23: </span><span class="content"><strong><span class="cmbx-10x-x-109">Demonstration of violin plots with custom features.</span></strong> <span class="cmr-10x-x-109">An example</span> <span class="cmr-10x-x-109">of the </span><span class="cmtt-10x-x-109">iris </span><span class="cmr-10x-x-109">dataset is plotted to show the custom features of the </span><span class="cmtt-10x-x-109">vioplotx </span><span class="cmr-10x-x-109">package</span> <span class="cmr-10x-x-109">including a) individual colour, shape and size parameters of each violin, scaling violin</span> <span class="cmr-10x-x-109">widths by area, and b) splitting violins to compare subsets of data.</span></span></p>
</div>
<!--tex4ht:label?: x1-110003r23 -->
</div>
<hr class="endfloat" />
<!--l. 778-->
<p class="indent">
<p>The <span class="cmtt-12">vioplotx </span>package provides an alternative visualisation (of continuous variables against categories) to the more familiar boxplot, showing variability of the data by the width of the plots. As demonstrated in Figure <a 
href="#x1-110003r23">3.23<!--tex4ht:ref: fig:vioplot --></a>, the customised version enables separate plotting parameters for each violin with vector inputs for colour, shape, and size of various elements of the median point, central boxplot, borders, and fill colour for the violin. Scaling violin width to adjust violin area and splitting data by a second categorical variable is also enabled. This function is intended to be backwards compatible with the inputs of <span class="cmtt-12">vioplot </span>(applying scalar inputs across all violins) and <span class="cmtt-12">boxplot </span>(by enabling formula inputs as an S3 method). Each of these features is demonstrated with examples in respective vignettes on the package <a 
href="https://github.com/TomKellyGenetics/vioplotx" >GitHub repository</a>.</p>
</p>
<!--l. 780-->
<p class="indent">
<a 
 id="x1-110004r24"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 783-->
<p class="noindent">
</p>
<!--l. 784-->
<p class="noindent">
<img 
src=""heatmap2x".png" alt="PIC"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.24: </span><span class="content"><strong><span class="cmbx-10x-x-109">Demonstration of annotated heatmap</span></strong><span class="cmr-10x-x-109">. The example heatmap depicts</span> <span class="cmr-10x-x-109">the additional row and column annotation bars enabled by </span><span class="cmtt-10x-x-109">heatmap.2x</span><span class="cmr-10x-x-109">, extending the</span> <span class="cmr-10x-x-109">features of </span><span class="cmtt-10x-x-109">gplots </span><span class="cmr-10x-x-109">with backwards compatible inputs.</span></span></p>
</div>
<!--tex4ht:label?: x1-110004r24 -->
</div>
<hr class="endfloat" />
<!--l. 792-->
<p class="indent">
The <span class="cmtt-12">heatmap.2x </span>provides extensions for annotation colour bars for both the rows and columns (as shown in Figure <a 
href="#x1-110004r24">3.24<!--tex4ht:ref: fig:heatmap.2x --></a>). Multiple bars are enabled on both axes with matrix inputs (rather than single vector for <span class="cmtt-12">heatmap.2</span>) which facilitates additional plotting of gene and sample characteristics for comparison with correlation matrices, expression profiles, or pathway metagenes. Annotation bar inputs correspond to their orientation on the plot, each colour bar is provided as a column for the row annotation on the left of the heatmap and as a row for the column annotation on top of the heatmap. Row and column annotation bars are labelled with the column or row names respectively. Additional parameters enable resizing of these annotation bar labels and control of reordering columns for if samples are ordered in advance (e.g., ranked by a metagene or split into groups clustered separately). These features were used through this thesis and are provided in a package <a 
href="https://github.com/TomKellyGenetics/heatmap.2x" >GitHub repository</a>. <a 
 id="x1-110005r147"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">3.5.3 </span> <a 
 id="x1-1110003"></a>Extensions to the iGraph Package
</h4>
<!--l. 798-->
<p class="noindent">
The following features were developed during this thesis using “iGraph” data objects, building upon the <span class="cmtt-12">igraph </span>package (<a 
href="#Xigraph">Csardi and Nepusz</a>, <a 
href="#Xigraph">2006</a>). These have been released as separate packages for each respective procedure and can be installed together as a collection of extensions to the <span class="cmtt-12">igraph </span>package. <a 
 id="x1-111001r140"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">3.5.3.1 </span> <a 
 id="x1-1120001"></a>Sampling Simulated Data from Graph Structures
</h5>
<!--l. 801-->
<p class="noindent">
<p>The <span class="cmtt-12">graphsim </span>package implements the procedure for simulating gene expression from graph structures (as described in Section <a 
href="#x1-1070002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>). By default, this derives a matrix with a geometrically decreasing weighting by distance (by shortest paths) between each pair of nodes with. An absolute decreasing weighting is also available with the option of to derive correlation structures from adjacency matrices or the number of links common partners (i.e., size of the shared “neighbourhood” (<a 
href="#XHell1976">Hell</a>, <a 
href="#XHell1976">1976</a>)) between each pair of nodes. Functions to compute these are called directly by passing parameters to them when running the <span class="cmtt-12">generate</span><span class="cmtt-12">_expression </span>or <span class="cmtt-12">make</span><span class="cmtt-12">_sigma</span><span class="cmtt-12">_mat </span>commands. This package enables simulating expression data directly from a graph structure (with the intermediate steps automated) or generating Σ parameters for <span class="cmtt-12">mvtnorm </span>from graph structures or matrices derived from them.</p>
These functions support assigning activating or inhibiting to each edge (with a <span class="cmtt-12">state</span> parameter). <a 
 id="x1-112001r151"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">3.5.3.2 </span> <a 
 id="x1-1130002"></a>Plotting Directed Graph Structures
</h5>
<!--l. 804-->
<p class="noindent">
<p>The <span class="cmtt-12">plot.igraph </span>package provides the <span class="cmtt-12">plot</span><span class="cmtt-12">_directed </span>function specifically developed for directed graph structures and to plot activating or inhibiting for each edge (as described in Section <a 
href="#x1-1070002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>). As shown in Figure <a 
href="#x1-181006r2">6.2<!--tex4ht:ref: fig:simple_graph2 --></a>, this function supports separate plotting parameters for each node, node label, and edge. This includes colours of node fill, border, label text, and edges and size of nodes, edge widths, arrowhead lengths, and font size of labels. The <span class="cmtt-12">state</span> parameter for assigning activating or inhibiting to each edge determines whether edges are dipected with 30<span class="tcrm-1200">°</span> or 90<span class="tcrm-1200">°</span> arrowheads. Colours are assigned separately and so they may be customised. Vectorised parameters are applied across each node or edge, whereas scalar parameters apply the same plotting parameters across them. The default layout function is <span class="cmtt-12">layout.fruchterman.reingold </span>but any layout function supported by <span class="cmtt-12">plot </span>function in <span class="cmtt-12">igraph </span>(<a 
href="#Xigraph">Csardi and Nepusz</a>, <a 
href="#Xigraph">2006</a>) is compatible such as <span class="cmtt-12">layout.kamada.kawai </span>used to implement the Kamada–Kawai algorithm (<a 
href="#XKamada1989">Kamada and Kawai</a>, <a 
href="#XKamada1989">1989</a>) for graph plots throughout this thesis.</p>
</p>
<!--l. 807-->
<p class="indent">
<a 
 id="x1-113001r25"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 810-->
<p class="noindent">
</p>
<div class="fbox">
<p><img 
src=""/home/tomkelly/Documents/PhD.png" alt="PIC"  
Otago Uni/SL_Model/graph_sim_method/simple_graph_inhibiting".png  /></p>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 3.25: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulating graph structures.</span></strong></strong> <span class="cmr-10x-x-109">An example graph structure which will</span> <span class="cmr-10x-x-109">be used throughout demonstrating the simulation procedure from graph structures. Here</span> <span class="cmr-10x-x-109">activating links are denoted by blue arrows and inhibiting links by red edges.</span></span></p>
</div>
<!--tex4ht:label?: x1-113001r25 -->
</div>
<hr class="endfloat" />
<a 
 id="x1-113002r152"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">3.5.3.3 </span> <a 
 id="x1-1140003"></a>Computing Information Centrality
</h5>
<!--l. 824-->
<p class="noindent">
The shortest paths of a network are computed by the <span class="cmtt-12">igraph </span>package <a 
href="#Xigraph">Csardi and Nepusz</a> (<a 
href="#Xigraph">2006</a>) which can be extended to calculate the network efficiency but is not provided by the package itself (ss described in Section <a 
href="#x1-870004">2.4.4<!--tex4ht:ref: methods:network_metrics --></a>). The “information centrality” of a vertex is computed as the relative change in the network efficiency when the vertex is removed. Information centrality is calculated iteratively for each node and the sum of information centrality for each vertex is the information centrality for the network. These metrics are provided by the <span class="cmtt-12">info.centrality</span> package. <a 
 id="x1-114001r154"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">3.5.3.4 </span> <a 
 id="x1-1150004"></a>Testing Pathway Structure with Permutation Testing
</h5>
<!--l. 827-->
<p class="noindent">
A network-based procedure developed was used for comparison of <a 
href="#glo:siRNA">siRNA</a> and <a 
href="#glo:SLIPT">SLIPT</a> candidate genes in a pathway structure. Such pathway structure relationships were tested by computing the number of shortest paths between two different groups of nodes in either direction within a graph . This pathway relationship metric was implemented in the <span class="cmtt-12">pathway.structure.permutation </span>package with permutation testing (as described in sections <a 
href="#x1-1040001">3.4.1<!--tex4ht:ref: methods:pathway_str --></a> and <a 
href="#x1-1050001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>). <a 
 id="x1-115001r155"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">3.5.3.5 </span> <a 
 id="x1-1160005"></a>Metapackage to Install iGraph Functions
</h5>
<!--l. 830-->
<p class="noindent">
<p>These features may be installed together with <span class="cmtt-12">igraph.extensions </span>which can be accessed from a <a 
href="https://github.com/TomKellyGenetics/igraph.extensions" >GitHub repository</a>. This meta-package installs <span class="cmtt-12">igraph </span>(<a 
href="#Xigraph">Csardi and Nepusz</a>, <a 
href="#Xigraph">2006</a>) and the packages described in Section <a 
href="#x1-1110003">3.5.3<!--tex4ht:ref: methods:igraph_extensions --></a> including their dependencies for matrix operations and statistical procedures: <span class="cmtt-12">Matrix</span>, <span class="cmtt-12">matrixcalc</span>, and <span class="cmtt-12">mvtnorm </span>(<a 
href="#XMatrix">Bates and Maechler</a>, <a 
href="#XMatrix">2016</a>; <a 
href="#XGenz2009">Genz and Bretz</a>, <a 
href="#XGenz2009">2009</a>; <a 
href="#Xmvtnorm">Genz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xmvtnorm">2016</a>; <a 
href="#Xmatrixcalc">Novomestky</a>, <a 
href="#Xmatrixcalc">2012</a>).</p>
</p>
<h2 class="chapterHead">
<span class="titlemark">Chapter 4</span><br /><a 
 id="x1-1170004"></a>Synthetic Lethal Analysis of Gene Expression Data
</h2>
<!--l. 4-->
<p class="noindent">
Having developed a statistical synthetic lethal detection methodology (<a 
href="#glo:SLIPT">SLIPT</a>), it was applied to empirical (publicly available) cancer gene expression datasets in this Chapter. The analysis largely focuses findings from the TCGA breast cancer data (<a 
href="#XTCGA2012">TCGA</a>, <a 
href="#XTCGA2012">2012</a>) which covers a range of clinical subtypes and is more closely modelled by <a 
href="#glo:siRNA">siRNA</a> data (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) generated from screening experiments conducted in MCF10A breast cells. Although stomach cancer data will also be considered to replicate findings in an independent dataset and for it’s relevance to syndromic hereditary diffuse gastric cancer. The TCGA data also has the advantages of other clinical and molecular profiles (e.g., somatic mutation and DNA copy number) for many of the same samples, in addition to a considerable sample size for RNASeq expression data, treated with a rigorous procedure to minimise batch effects. Some findings will be replicated in the Cancer Cell Line Encyclopaedia (CCLE) (<a 
href="#XBarretina2012">Barretina <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBarretina2012">2012</a>) which may be more comparable to the cell line experiments.
</p>
<!--l. 6-->
<p class="indent">
Synthetic lethal candidate partners for <span class="cmti-12">CDH1 </span>will be described at both the gene and pathway level. <a 
href="#glo:SLIPT">SLIPT</a> gene candidates will be analysed by cluster analysis for common expression profiles across samples and relationships with clinical factors and mutations in key breast cancer genes. These genes will also be compared to the gene candidates from a primary and secondary (validation) screens conducted by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) on isogenic cell lines. For comparison, an alternative <a 
href="#glo:SLIPT">SLIPT</a> methodology which uses mutation data for <span class="cmti-12">CDH1 </span>against expression of candidate partners will also be presented which may better represent the null mutations in HDGC patients and the experiment cell model (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>). Pathways will be analysed by over-representation analysis (with resampling for comparisons with <a 
href="#glo:siRNA">siRNA</a> data) and supported by a metagene analysis of pathway gene signatures. The pathway metagene expression profiles will be used to replicate known relationships between clinical and molecular characteristics for breast cancer and to demonstrate application of <a 
href="#glo:SLIPT">SLIPT</a> directly on metagenes to detect synthetic lethal pathways.
</p>
<!--l. 8-->
<p class="indent">
<p>Together these results will demonstrate the wide range of applications for <a 
href="#glo:SLIPT">SLIPT</a> analysis and examine the synthetic lethal partners of <span class="cmti-12">CDH1 </span>in breast and stomach cancer. These synthetic lethal genes and pathways will be described in both context of the functional implications of novel synthetic lethal relationships and as potential actionable targets against <span class="cmti-12">CDH1 </span>deficient tumours, in addition to replication of</p>
established functions of E-cadherin. In particular, the focus of these analysis will be in comparisons with experimental screening data to explore the potential for <a 
href="#glo:SLIPT">SLIPT</a> to augment such triage of candidate partners and support further experimental investigations. The key synthetic lethal partner pathways for <span class="cmti-12">CDH1</span>, supported by both approaches, will be examined in more detail at the gene and pathway structure level in Chapter <a 
href="#x1-1530005">5<!--tex4ht:ref: chap:Pathways --></a>.
</p>
<!--l. 10-->
<p class="indent">
Some of the findings presented in this Chapter have also been included in manuscripts submitted for publication (<a 
href="#XKellyBMC">Kelly <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XKellyBMC">2017a</a>,<a 
href="#XKellyHDGC">b</a>) and may bear similarity to them, although the results in this thesis have been edited to cohesively fit with additional findings (including consistent data versions). These findings are the result of investigations conducted throughout this thesis project and only these contributions to the articles are included in this Chapter, not that conducted by co-authors. <a 
 id="x1-117001r145"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">4.1 </span> <a 
 id="x1-1180001"></a>Synthetic lethal genes in breast cancer
</h3>
<!--l. 14-->
<p class="noindent">
The <a 
href="#glo:SLIPT">SLIPT</a> methodology (as described in Section <a 
href="#x1-930001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) was applied to the normalised TCGA breast cancer gene expression dataset (<span class="cmmi-12">n </span>= 1168). As shown in Table <a 
href="#x1-118001r1">4.1<!--tex4ht:ref: tab:gene_SL --></a>, the most significant genes had strong evidence of expression-based association with <span class="cmti-12">CDH1 </span>(high <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values) with fewer samples exhibiting low expression of both genes than expected statistically. Eukaryotic translation gene were among the highest gene candidates, including initiation factors, elongation factors, and ribosomal proteins. These are clearly neccessary for cancer cells to grow and proliferate, with sustained gene expression needed to maintain growth signaling pathways and resist apoptosis or immune factors translation may be subject to non-oncogene addiction for <span class="cmti-12">CDH1</span>-deficient cells.
</p>
<!--l. 16-->
<p class="indent">
<p>While these are among the strongest synthetic lethal candidates, translational genes are cruicial to the viability of healthy cells and dosing for a selective synthetic lethal effect against these may be difficult compared to other biological functions which may also be supported among the <a 
href="#glo:SLIPT">SLIPT</a> candidate genes. Furthermore, few known biological functions of <span class="cmti-12">CDH1 </span>were among the strongest SL candidates so the remaining candidate genes may also be informative since they are likely to contain these expected functions in addition to novel relationships for <span class="cmti-12">CDH1</span>. Thus further pathway level analyses were also conducted to examine biological functions over-represented among synthetic candidate genes and identify synthetic lethal pathways.</p>
</p>
<!--l. 18-->
<p class="indent">
<a 
 id="x1-118001r1"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 4.1: </span><span class="content">Candidate synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from SLIPT</span></p>
</div>
<!--tex4ht:label?: x1-118001r1 -->
<div class="tabular">
<table id="TBL-58" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-58-1g">
<col 
id="TBL-58-1" />
<col 
id="TBL-58-2" />
<col 
id="TBL-58-3" />
<col 
id="TBL-58-4" />
<col 
id="TBL-58-5" />
<col 
id="TBL-58-6" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-58-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-1-1" class="td11">
<span class="cmbx-12">Gene </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-1-2" class="td11">
Observed
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-1-3" class="td11">
Expected
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-1-4" class="td11">
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-1-5" class="td11">
p-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-1-6" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-2-1" class="td11">
<span class="cmti-12">TRIP10 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-2-2" class="td11">
<pre><code>62      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-2-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-2-4" class="td11">
162
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-2-5" class="td11">
5<span class="cmmi-12">.</span>65 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">34</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-2-6" class="td11">
1<span class="cmmi-12">.</span>84 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-3-1" class="td11">
<span class="cmti-12">EEF1B2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-3-2" class="td11">
<pre><code>56      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-3-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-3-4" class="td11">
158
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-3-5" class="td11">
3<span class="cmmi-12">.</span>10 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">33</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-3-6" class="td11">
9<span class="cmmi-12">.</span>45 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-4-1" class="td11">
<span class="cmti-12">GBGT1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-4-2" class="td11">
<pre><code>61      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-4-3&quot;  </code></pre>
class=“td11”&gt; 131
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-4-4" class="td11">
156
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-4-5" class="td11">
1<span class="cmmi-12">.</span>08 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">32</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-4-6" class="td11">
3<span class="cmmi-12">.</span>14 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-5-1" class="td11">
<span class="cmti-12">ELN </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-5-2" class="td11">
<pre><code>81      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-5-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-5-4" class="td11">
149
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-5-5" class="td11">
3<span class="cmmi-12">.</span>46 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-5-6" class="td11">
8<span class="cmmi-12">.</span>82 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-6-1" class="td11">
<span class="cmti-12">TSPAN4 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-6-2" class="td11">
<pre><code>78      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-6-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-6-4" class="td11">
146
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-6-5" class="td11">
1<span class="cmmi-12">.</span>63 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-6-6" class="td11">
3<span class="cmmi-12">.</span>79 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-7-1" class="td11">
<span class="cmti-12">GLIPR2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-7-2" class="td11">
<pre><code>72      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-7-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-7-4" class="td11">
146
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-7-5" class="td11">
1<span class="cmmi-12">.</span>68 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-7-6" class="td11">
3<span class="cmmi-12">.</span>86 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-8-1" class="td11">
<span class="cmti-12">RPS20 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-8-2" class="td11">
<pre><code>73      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-8-3&quot;  </code></pre>
class=“td11”&gt; 131
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-8-4" class="td11">
145
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-8-5" class="td11">
1<span class="cmmi-12">.</span>89 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-8-6" class="td11">
4<span class="cmmi-12">.</span>28 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-9-1" class="td11">
<span class="cmti-12">RPS27A </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-9-2" class="td11">
<pre><code>80      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-9-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-9-4" class="td11">
143
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-9-5" class="td11">
5<span class="cmmi-12">.</span>53 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-9-6" class="td11">
1<span class="cmmi-12">.</span>18 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-10-1" class="td11">
<span class="cmti-12">EEF1A1P9 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-10-2" class="td11">
<pre><code>63      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-10-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-10-4" class="td11">
141
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-10-5" class="td11">
1<span class="cmmi-12">.</span>91 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-10-6" class="td11">
3<span class="cmmi-12">.</span>74 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-11-1" class="td11">
<span class="cmti-12">C1R </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-11-2" class="td11">
<pre><code>73      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-11-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-11-4" class="td11">
141
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-11-5" class="td11">
2<span class="cmmi-12">.</span>05 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-11-6" class="td11">
3<span class="cmmi-12">.</span>97 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-12-1" class="td11">
<span class="cmti-12">LYL1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-12-2" class="td11">
<pre><code>73      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-12-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-12-4" class="td11">
140
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-12-5" class="td11">
2<span class="cmmi-12">.</span>99 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-12-6" class="td11">
5<span class="cmmi-12">.</span>74 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-13-1" class="td11">
<span class="cmti-12">RPLP2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-13-2" class="td11">
<pre><code>71      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-13-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-13-4" class="td11">
139
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-13-5" class="td11">
4<span class="cmmi-12">.</span>88 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-13-6" class="td11">
9<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-14-1" class="td11">
<span class="cmti-12">C10orf10 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-14-2" class="td11">
<pre><code>73      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-14-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-14-4" class="td11">
138
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-14-5" class="td11">
6<span class="cmmi-12">.</span>72 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-14-6" class="td11">
1<span class="cmmi-12">.</span>20 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-15-1" class="td11">
<span class="cmti-12">DULLARD </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-15-2" class="td11">
<pre><code>74      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-15-3&quot;  </code></pre>
class=“td11”&gt; 131
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-15-4" class="td11">
138
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-15-5" class="td11">
9<span class="cmmi-12">.</span>29 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-15-6" class="td11">
1<span class="cmmi-12">.</span>61 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-16-1" class="td11">
<span class="cmti-12">PPM1F </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-16-2" class="td11">
<pre><code>64      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-16-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-16-4" class="td11">
136
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-16-5" class="td11">
1<span class="cmmi-12">.</span>61 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-16-6" class="td11">
2<span class="cmmi-12">.</span>65 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-17-1" class="td11">
<span class="cmti-12">OBFC2A </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-17-2" class="td11">
<pre><code>69      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-17-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-17-4" class="td11">
136
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-17-5" class="td11">
2<span class="cmmi-12">.</span>49 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-17-6" class="td11">
3<span class="cmmi-12">.</span>93 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-18-1" class="td11">
<span class="cmti-12">RPL11 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-18-2" class="td11">
<pre><code>70      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-18-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-18-4" class="td11">
136
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-18-5" class="td11">
2<span class="cmmi-12">.</span>56 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-18-6" class="td11">
3<span class="cmmi-12">.</span>97 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-19-1" class="td11">
<span class="cmti-12">RPL18A </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-19-2" class="td11">
<pre><code>70      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-19-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-19-4" class="td11">
135
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-19-5" class="td11">
3<span class="cmmi-12">.</span>08 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-19-6" class="td11">
4<span class="cmmi-12">.</span>70 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-20-1" class="td11">
<span class="cmti-12">MFNG </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-20-2" class="td11">
<pre><code>76      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-20-3&quot;  </code></pre>
class=“td11”&gt; 131
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-20-4" class="td11">
133
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-20-5" class="td11">
7<span class="cmmi-12">.</span>73 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-20-6" class="td11">
1<span class="cmmi-12">.</span>12 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-21-1" class="td11">
<span class="cmti-12">RPS17 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-21-2" class="td11">
<pre><code>77      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-21-3&quot;  </code></pre>
class=“td11”&gt; 131
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-21-4" class="td11">
133
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-21-5" class="td11">
8<span class="cmmi-12">.</span>94 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-21-6" class="td11">
1<span class="cmmi-12">.</span>29 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-22-1" class="td11">
<span class="cmti-12">MGAT1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-22-2" class="td11">
<pre><code>73      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-22-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-22-4" class="td11">
132
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-22-5" class="td11">
1<span class="cmmi-12">.</span>44 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-22-6" class="td11">
2<span class="cmmi-12">.</span>03 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-23-1" class="td11">
<span class="cmti-12">RPS12 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-23-2" class="td11">
<pre><code>72      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-23-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-23-4" class="td11">
128
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-23-5" class="td11">
8<span class="cmmi-12">.</span>57 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-23-6" class="td11">
1<span class="cmmi-12">.</span>12 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-24-1" class="td11">
<span class="cmti-12">C10orf54 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-24-2" class="td11">
<pre><code>73      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-24-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-24-4" class="td11">
127
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-24-5" class="td11">
1<span class="cmmi-12">.</span>37 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-24-6" class="td11">
1<span class="cmmi-12">.</span>75 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-25-1" class="td11">
<span class="cmti-12">LOC286367 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-25-2" class="td11">
<pre><code>72      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-25-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-25-4" class="td11">
126
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-25-5" class="td11">
2<span class="cmmi-12">.</span>20 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-25-6" class="td11">
2<span class="cmmi-12">.</span>70 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-58-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-58-26-1" class="td11">
<span class="cmti-12">GMFG </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-26-2" class="td11">
<pre><code>70      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-58-26-3&quot;  </code></pre>
class=“td11”&gt; 130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-26-4" class="td11">
126
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-26-5" class="td11">
2<span class="cmmi-12">.</span>20 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-58-26-6" class="td11">
2<span class="cmmi-12">.</span>70 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-58-27-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-58-27-1"  
class="td11">
<pre><code>        &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 85-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Strongest candidate SL partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">with observed and expected samples with low</span> <span class="cmr-10x-x-109">expression of both genes</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 88-->
<p class="indent">
The modified mtSLIPT methodology (as described in Section <a 
href="#x1-930001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) was also applied to the normalised TCGA breast cancer gene expression dataset, against somatic loss of function mutations in <span class="cmti-12">CDH1</span>. As shown in Table <a 
href="#x1-205001r1">D.1<!--tex4ht:ref: tab:gene_mtSL --></a>, the most significant genes also had strong evidence of expression associated with <span class="cmti-12">CDH1 </span>mutations (high <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values) with fewer samples exhibiting both low expression and mutations of each gene than expected statistically. Although, these were not a strongly supported as the expression analysis (in Table <a 
href="#x1-118001r1">4.1<!--tex4ht:ref: tab:gene_SL --></a>) nor were as many genes detected. This is unsurprising due to the lower sample size with matching somatic mutation data and the lower frequency of <span class="cmti-12">CDH1 </span>mutations compared to low expression by <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span> quantiles.
</p>
<!--l. 90-->
<p class="indent">
The mtSLIPT candidates had more genes involved in cell and gene regulation, particularly DNA and RNA binding factors. The strongest candidates also include microtubule (<span class="cmti-12">KIF12</span>), microfibril (<span class="cmti-12">MFAP4</span>), and cell adhesion (<span class="cmti-12">TENC1</span>) genes consistent with the established cytoskeletal role of <span class="cmti-12">CDH1</span>. The elastin gene (<span class="cmti-12">ELN</span>) was notably strongly supported by both expression and mutation SLIPT analysis of CDH1 supporting a interactions with extracellular proteins and the tumour microenvironment. <a 
 id="x1-118002r150"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.1.1 </span> <a 
 id="x1-1190001"></a>Synthetic lethal pathways in breast cancer
</h4>
<!--l. 97-->
<p class="noindent">
Translational pathways were strongly over-represented in <a 
href="#glo:SLIPT">SLIPT</a> partners, as shown in Table <a 
href="#x1-119001r2">4.2<!--tex4ht:ref: tab:pathway_exprSL --></a>. These include ribosomal subunits, initiation, peptide elongation, and termination. Regulatory processes involving mRNA including 3’ untranslated region (UTR) binding, L13a-mediated translational silencing, and nonsense-mediated decay were also implicated. These are consistent with protein translation being subject to “non-oncogene addiction” (<a 
href="#XLuo2009">Luo <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLuo2009">2009</a>), as a core process that is dysregulated to sustain cancer proliferation and survival (<a 
href="#XGao2015">Gao and Roux</a>, <a 
href="#XGao2015">2015</a>).
</p>
<!--l. 99-->
<p class="indent">
<p>Immune pathways, including the adaptive immune system and responses to infectious diseases were also strongly implicated as synthetic lethal with loss of E-cadherin. This is consistent with the alterations of immune response being a hallmark of cancer <a 
href="#XHanahan2000">Hanahan and Weinberg</a> (<a 
href="#XHanahan2000">2000</a>), since evading the immune system is necessary for cancer survival. Either of these systems are potential means to target <span class="cmti-12">CDH1 </span>deficient cells, although these were not detected in an isolated cell line experimental screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) and the</p>
<p>differences between to findings in patient data will be described in more detail in Section <a 
href="#x1-1270004">4.2.1.4<!--tex4ht:ref: chapt3:compare_pathway --></a>.</p>
</p>
<!--l. 101-->
<p class="indent">
<a 
 id="x1-119001r2"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 4.2: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT</span></p>
</div>
<!--tex4ht:label?: x1-119001r2 -->
<div class="tabular">
<table id="TBL-59" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-59-1g">
<col 
id="TBL-59-1" />
<col 
id="TBL-59-2" />
<col 
id="TBL-59-3" />
<col 
id="TBL-59-4" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-59-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-1-1" class="td11">
<strong><span class="cmbx-12">Pathways Over-represented</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-1-2" class="td11">
<strong><span class="cmbx-12">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-1-3" class="td11">
<strong><span class="cmbx-12">SL Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-1-4" class="td11">
<strong><span class="cmbx-12">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-2-1" class="td11">
Eukaryotic Translation Elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-2-2" class="td11">
<pre><code>  86          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-2-3&quot;  </code></pre>
class=“td11”&gt; 81
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-2-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">207</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-3-1" class="td11">
Peptide chain elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-3-2" class="td11">
<pre><code>  83          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-3-3&quot;  </code></pre>
class=“td11”&gt; 78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-3-4" class="td11">
5<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">201</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-4-1" class="td11">
Eukaryotic Translation Termination
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-4-2" class="td11">
<pre><code>  83          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-4-3&quot;  </code></pre>
class=“td11”&gt; 77
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-4-4" class="td11">
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">196</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-5-1" class="td11">
Viral mRNA Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-5-2" class="td11">
<pre><code>  81          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-5-3&quot;  </code></pre>
class=“td11”&gt; 76
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-5-4" class="td11">
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">196</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-6-1" class="td11">
Formation of a pool of free 40S subunits
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-6-2" class="td11">
<pre><code>  93          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-6-3&quot;  </code></pre>
class=“td11”&gt; 81
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-6-4" class="td11">
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">194</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-7-1" class="td11">
Nonsense Mediated Decay independent of the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-7-2" class="td11">
<pre><code>  88          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-7-3&quot;  </code></pre>
class=“td11”&gt; 77
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-7-4" class="td11">
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">187</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-8-1" class="td11">
L13a-mediated translational silencing of Ceruloplasmin expression
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-8-2" class="td11">
<pre><code>  103         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-8-3&quot;  </code></pre>
class=“td11”&gt; 82
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-8-4" class="td11">
9<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">183</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-9-1" class="td11">
3’ -UTR-mediated translational regulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-9-2" class="td11">
<pre><code>  103         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-9-3&quot;  </code></pre>
class=“td11”&gt; 82
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-9-4" class="td11">
9<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">183</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-10-1" class="td11">
GTP hydrolysis and joining of the 60S ribosomal subunit
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-10-2" class="td11">
<pre><code>  104         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-10-3&quot;  </code></pre>
class=“td11”&gt; 82
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-10-4" class="td11">
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">181</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-11-1" class="td11">
Nonsense-Mediated Decay
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-11-2" class="td11">
<pre><code>  103         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-11-3&quot;  </code></pre>
class=“td11”&gt; 80
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-11-4" class="td11">
6<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">176</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-12-1" class="td11">
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-12-2" class="td11">
<pre><code>  103         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-12-3&quot;  </code></pre>
class=“td11”&gt; 80
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-12-4" class="td11">
6<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">176</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-13-1" class="td11">
Adaptive Immune System
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-13-2" class="td11">
<pre><code>  412         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-13-3&quot;  </code></pre>
class=“td11”&gt; 167
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-13-4" class="td11">
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">174</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-14-1" class="td11">
Eukaryotic Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-14-2" class="td11">
<pre><code>  111         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-14-3&quot;  </code></pre>
class=“td11”&gt; 82
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-14-4" class="td11">
5<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">173</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-15-1" class="td11">
Cap-dependent Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-15-2" class="td11">
<pre><code>  111         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-15-3&quot;  </code></pre>
class=“td11”&gt; 82
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-15-4" class="td11">
5<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">173</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-16-1" class="td11">
SRP-dependent cotranslational protein targeting to membrane
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-16-2" class="td11">
<pre><code>  104         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-16-3&quot;  </code></pre>
class=“td11”&gt; 79
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-16-4" class="td11">
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">171</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-17-1" class="td11">
Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-17-2" class="td11">
<pre><code>  141         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-17-3&quot;  </code></pre>
class=“td11”&gt; 91
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-17-4" class="td11">
6<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">170</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-18-1" class="td11">
Infectious disease
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-18-2" class="td11">
<pre><code>  347         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-18-3&quot;  </code></pre>
class=“td11”&gt; 146
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-18-4" class="td11">
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">166</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-19-1" class="td11">
Influenza Infection
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-19-2" class="td11">
<pre><code>  117         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-19-3&quot;  </code></pre>
class=“td11”&gt; 81
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-19-4" class="td11">
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">163</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-20-1" class="td11">
Influenza Viral RNA Transcription and Replication
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-20-2" class="td11">
<pre><code>  108         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-20-3&quot;  </code></pre>
class=“td11”&gt; 77
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-20-4" class="td11">
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">160</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-59-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-59-21-1" class="td11">
Influenza Life Cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-21-2" class="td11">
<pre><code>  112         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-59-21-3&quot;  </code></pre>
class=“td11”&gt; 77
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-59-21-4" class="td11">
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">156</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-59-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-59-22-1"  
class="td11">
<pre><code>                                                         &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 157-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Gene set over-representation analysis (hypergeometric test) for Reactome pathways in </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">partners for </span><span class="cmti-10x-x-109">CDH1</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 160-->
<p class="indent">
It is also notable that the pathways over-represented in <a 
href="#glo:SLIPT">SLIPT</a> candidate genes have strongly significant over-representation of Reactome pathways from the hypergeometric test (as described in Section <a 
href="#x1-770002">2.3.2<!--tex4ht:ref: methods:enrichment --></a>). Even after adjusting stringently for multiple tests, biologically related pathways give consensus support to these pathways. These pathways are further supported by testing for synthetic lethality against <span class="cmti-12">CDH1 </span>mutations (mtSLIPT) with many of these pathways also among the most strongly supported in this analysis (shown in Table <a 
href="#x1-205002r2">D.2<!--tex4ht:ref: tab:pathway_mtSL --></a>). This analysis more closely represents the null <span class="cmti-12">CDH1 </span>mutations in HDGC (<a 
href="#XGuilford1998">Guilford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGuilford1998">1998</a>) and the experimental MCF10A cell model (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>). Although it still supports translational and immune pathways not detected in the isolated experimental system, G-protein-coupled receptors (GPCRs) were also among the most strongly supported pathways, supporting the experimental findings of <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) for these intracellular signalling pathways already being targeted for other diseases. <a 
 id="x1-119002r160"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.1.2 </span> <a 
 id="x1-1200002"></a>Expression profiles of synthetic lethal partners
</h4>
<!--l. 170-->
<p class="noindent">
<p>Due to the sheer number of gene candidates and to examine their expression patterns, investigations proceeded into correlation structure and pathway over-representation. This serves to explore the functional similarity of the synthetic lethal partners of <span class="cmti-12">CDH1</span>, with the eventual aim to assess their utility as drug targets. As shown in Figure <a 
href="#x1-120001r1">4.1<!--tex4ht:ref: fig:slipt_expr --></a> (which clusters <span class="cmti-12">CDH1 </span>lowly expressing samples separately), there were several large clusters of genes among the expression profiles of the <span class="cmti-12">CDH1 </span>synthetic lethal candidate partners. The clustering suggests co-regulation of genes or pathway correlation between partner gene candidates. A number of candidates from an experimental RNAi screen study performed by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) were also identified by this approach. In addition, we identified novel gene candidates, which had little effect on viability in isogenic cell line experiments.</p>
</p>
<!--l. 172-->
<p class="indent">
<a 
 id="x1-120001r1"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________      <!--l. 175-->
<p class="noindent">
<img 
src="CDH1_Heatmaps_Genes_Split_By_CDH1_z-trans_exprSL_cordistx_Pub.png" alt="PIC"  
 /> <br />
</p>
<div class="caption">
<p><span class="id">Figure 4.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethal expression profiles of analysed samples.</span></strong> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">expression profile heatmap (correlation distance) of all samples (separated by the</span> <span class="cmr-8">1</span><span class="cmmi-10x-x-109">∕</span><span class="cmr-8">3</span> <span class="cmr-10x-x-109">quantile of </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">expression) analysed in TCGA breast cancer dataset for gene</span> <span class="cmr-10x-x-109">expression of 5,165 candidate partners of E-cadherin (</span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">) from </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">prediction</span> <span class="cmr-10x-x-109">(with significant FDR adjusted </span><span class="cmmi-10x-x-109">p &lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05). Deeply clustered, inter-correlated genes form</span> <span class="cmr-10x-x-109">several main groups, each containing genes that were SL candidates or toxic in an </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screen </span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">Telford </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">2015</span></a><span class="cmr-10x-x-109">). Clusters had different sample groups highly expressing</span> <span class="cmr-10x-x-109">the synthetic lethal candidates in </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">low samples, notably ‘normal-like’, basal, and</span> <span class="cmr-10x-x-109">estrogen receptor negative samples have elevated expression in one or more distinct</span> <span class="cmr-10x-x-109">clusters showing complexity and variation among candidate synthetic lethal partners.</span> <span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">low samples also contained most of samples with </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutations. </span></span></p>
</div>
<!--tex4ht:label?: x1-120001r1 -->
</div>
<hr class="endfloat" />
<!--l. 185-->
<p class="indent">
In these expression profiles, a gene with a moderate or high signal across samples exhibiting low <span class="cmti-12">CDH1 </span>expression would represent a potential drug target. However, it appears that several molecular subtypes of cancer have elevation of different clusters of synthetic lethal candidates in samples with low <span class="cmti-12">CDH1</span>. This clustering suggests that different targets or combinations could be effective in different patients suggesting potential utility for stratification. In particular, estrogen receptor negative, basal subtype, and “normal-like” samples <a 
href="#XDai2015">Dai <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XDai2015">2015</a>); <a 
href="#XEroles2012">Eroles <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XEroles2012">2012</a>); <a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XParker2009">2009</a>) have elevation of genes specific to particular clusters which is indicative of some synthetic lethal interactions being specific to a particular molecular subtype or genetic background. Thus synthetic lethal drug therapy against these subtypes may be ineffective if it were designed against genes in another cluster.
</p>
<!--l. 188-->
<p class="indent">
A similar correlation structure was observed among the candidates tested against <span class="cmti-12">CDH1</span> mutation (mtSLIPT), as shown in Figure <a 
href="#x1-206001r1">D.1<!--tex4ht:ref: fig:slipt_expr_mtSL --></a>. This clustering analysis similarly identified several major clusters of putative synthetic lethal partner genes. Although in this case many partner genes had consistently high expression across most of the (predominantly lobular subtype) <span class="cmti-12">CDH1 </span>breast cancer samples. However, a major exception to this in the <span class="cmti-12">CDH1</span> expression analysis were the normal samples which were excluded from the mutation data (as they were not tested for tumour-specific genotypes). This supports synthetic lethal interventions being more applicable to <span class="cmti-12">CDH1 </span>mutant tumours and genotyping tumours for loss of function will be essential for clinical application. There was still considerable correlation structure, particularly among <span class="cmti-12">CDH1 </span>wildtype samples, sufficient to distinguish gene clusters. In contrast to the expression analysis the (predominantly ductal <span class="cmti-12">CDH1 </span>wildtype) basal subtype and estrogen receptor negative samples have depleted expression among most candidate synthetic lethal partners. This is consistent with synthetic lethal interventions only being effective in lobular estrogen receptor positive breast cancers in which they are a more common, as recurrent (driver) mutation. However, the remaining samples are still informative for synthetic lethal analysis (by <a 
href="#glo:SLIPT">SLIPT</a>) as it requires highly expressing <span class="cmti-12">CDH1 </span>samples for comparison.
</p>
<!--l. 190-->
<p class="indent">
<p>The <span class="cmti-12">CDH1 </span>mutant samples (in Figure <a 
href="#x1-120001r1">4.1<!--tex4ht:ref: fig:slipt_expr --></a>) were predominantly among the <span class="cmti-12">CDH1 </span>lowly expressing samples and distributed throughout <span class="cmti-12">CDH1 </span>samples with clustering analysis. Thus the molecular profiles of <span class="cmti-12">CDH1 </span>low samples a indistinguishable from <span class="cmti-12">CDH1 </span>mutant samples with the exception of normal samples (that do not have somatic mutation data as it is inferred from comparison to them to tumour-specific genotypes). Conversely, many of the</p>
<span class="cmti-12">CDH1 </span>mutant samples (in Figure <a 
href="#x1-206001r1">D.1<!--tex4ht:ref: fig:slipt_expr_mtSL --></a>) had among the lowest <span class="cmti-12">CDH1 </span>expression and some of the synthetic lethal partners were also highly expressed in lowly expressing <span class="cmti-12">CDH1</span> wildtype samples, despite these not being considered as “inactivated” by mtSLIPT analysis.
</p>
<!--l. 192-->
<p class="indent">
Together these results support the use for low <span class="cmti-12">CDH1 </span>expression as a strategy for detecting <span class="cmti-12">CDH1 </span>inactivation. This has the benefit of increasing sample size (including samples such as normal tissue which do not have somatic mutation data available) and increasing the expected number of mutually inactive (low-low) samples for the directional criteria of (mt)SLIPT which enabling it to better distinguish significant deviations below this (as discussed in Section <a 
href="#x1-1760001">6.1<!--tex4ht:ref: chap5:compare_ methods --></a>). This also circumvents the assumption that all (detected) mutations are inactivating (although synonymous mutations were excluded from the analysis), which may not the case for several highly expressing <span class="cmti-12">CDH1 </span>mutant samples that do not cluster together in Figures <a 
href="#x1-120001r1">4.1<!--tex4ht:ref: fig:slipt_expr --></a> or <a 
href="#x1-206001r1">D.1<!--tex4ht:ref: fig:slipt_expr_mtSL --></a>. One of these exhibits among the lowest expression for many predicted synthetic lethal partners and would not be vulnerable to inactivation of these genes. As such correctly genotyping inactivating mutations will be essential in clinical practice for synthetic lethal targeting tumour suppressor genes, particularly for other genes such as <span class="cmti-12">TP53 </span>where oncogenic and tumour suppressor mutations (with different molecular consequences) are both common in cancers. Using expression as a measure of gene expression also avoids the assumptions that mutations are somatic rather than germline and that gene inactivation is by detectable mutations rather than other mechanisms such as epigenetic changes which is supported by many lowly expressing <span class="cmti-12">CDH1 </span>wildtype samples clustering with similar profiles to mutant samples. <a 
 id="x1-120002r156"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">4.1.2.1 </span> <a 
 id="x1-1210001"></a>Subgroup pathway analysis
</h5>
<!--l. 206-->
<p class="indent">
<a 
 id="x1-121001r3"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 4.3: </span><span class="content">Pathway composition for clusters of <span class="cmti-12">CDH1 </span>partners from SLIPT</span></p>
</div>
<!--tex4ht:label?: x1-121001r3 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-60" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-60-1g">
<col 
id="TBL-60-1" />
<col 
id="TBL-60-2" />
<col 
id="TBL-60-3" />
<col 
id="TBL-60-4" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-60-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-1-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 1</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-1-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-1-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-1-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-2-1" class="td11">
Collagen formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-2-2" class="td11">
<pre><code>    67            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-2-3&quot;  </code></pre>
class=“td11”&gt; 10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-2-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-3-1" class="td11">
Extracellular matrix organisation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-3-2" class="td11">
<pre><code>   238           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-3-3&quot;  </code></pre>
class=“td11”&gt; 21
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-3-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-4-1" class="td11">
Collagen biosynthesis and modifying enzymes
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-4-2" class="td11">
<pre><code>    56            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-4-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-4-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-5-1" class="td11">
Uptake and actions of bacterial toxins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-5-2" class="td11">
<pre><code>    22            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-5-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-5-4" class="td11">
<pre><code> 9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-6-1" class="td11">
Elastic fibre formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-6-2" class="td11">
<pre><code>    37            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-6-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-6-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-7-1" class="td11">
Muscle contraction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-7-2" class="td11">
<pre><code>    62            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-7-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-7-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-8-1" class="td11">
Fatty acid, triacylglycerol, and ketone body metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-8-2" class="td11">
<pre><code>   117           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-8-3&quot;  </code></pre>
class=“td11”&gt; 10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-8-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-9-1" class="td11">
XBP1(S) activates chaperone genes
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-9-2" class="td11">
<pre><code>    51            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-9-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-9-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-10-1" class="td11">
IRE1alpha activates chaperones
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-10-2" class="td11">
<pre><code>    54            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-10-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-10-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-11-1" class="td11">
Neurotoxicity of clostridium toxins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-11-2" class="td11">
<pre><code>    10            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-11-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-11-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-12-1" class="td11">
Retrograde neurotrophin signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-12-2" class="td11">
<pre><code>    10            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-12-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-12-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-13-1" class="td11">
Assembly of collagen fibrils and other multimeric structures
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-13-2" class="td11">
<pre><code>    40            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-13-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-13-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-14-1" class="td11">
Collagen degradation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-14-2" class="td11">
<pre><code>    58            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-14-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-14-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-15-1" class="td11">
Arachidonic acid metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-15-2" class="td11">
<pre><code>    41            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-15-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-15-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-16-1" class="td11">
Synthesis of PA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-16-2" class="td11">
<pre><code>    26            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-16-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-16-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-17-1" class="td11">
Signaling by NOTCH
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-17-2" class="td11">
<pre><code>    80            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-17-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-17-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-18-1" class="td11">
Signalling to RAS
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-18-2" class="td11">
<pre><code>    27            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-18-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-18-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-19-1" class="td11">
Integrin cell surface interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-19-2" class="td11">
<pre><code>    82            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-19-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-19-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-20-1" class="td11">
Smooth Muscle Contraction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-20-2" class="td11">
<pre><code>    28            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-20-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-20-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-21-1" class="td11">
ECM proteoglycans
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-21-2" class="td11">
<pre><code>    66            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-21-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-21-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-60-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-60-22-1"  
class="td11">
<pre><code>                                                               &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-23-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 2</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-23-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-23-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-23-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-24-1" class="td11">
Eukaryotic Translation Elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-24-2" class="td11">
<pre><code>    86            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-24-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-24-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">181</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-25-1" class="td11">
Viral mRNA Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-25-2" class="td11">
<pre><code>    81            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-25-3&quot;  </code></pre>
class=“td11”&gt; 72
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-25-4" class="td11">
<pre><code>9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">179</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-26-1" class="td11">
Peptide chain elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-26-2" class="td11">
<pre><code>    83            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-26-3&quot;  </code></pre>
class=“td11”&gt; 72
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-26-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">175</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-27-1" class="td11">
Eukaryotic Translation Termination
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-27-2" class="td11">
<pre><code>    83            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-27-3&quot;  </code></pre>
class=“td11”&gt; 72
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-27-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">175</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-28-1" class="td11">
Formation of a pool of free 40S subunits
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-28-2" class="td11">
<pre><code>    93            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-28-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-28-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">171</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-29-1" class="td11">
Nonsense Mediated Decay independent of the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-29-2" class="td11">
<pre><code>    88            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-29-3&quot;  </code></pre>
class=“td11”&gt; 72
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-29-4" class="td11">
<pre><code>9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">168</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-30-1" class="td11">
L13a-mediated translational silencing of Ceruloplasmin expression
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-30-2" class="td11">
<pre><code>   103           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-30-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-30-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">159</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-31-1" class="td11">
3’ -UTR-mediated translational regulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-31-2" class="td11">
<pre><code>   103           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-31-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-31-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">159</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-32-1" class="td11">
Nonsense-Mediated Decay
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-32-2" class="td11">
<pre><code>   103           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-32-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-32-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">159</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-33-1" class="td11">
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-33-2" class="td11">
<pre><code>   103           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-33-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-33-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">159</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-34-1" class="td11">
SRP-dependent cotranslational protein targeting to membrane
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-34-2" class="td11">
<pre><code>   104           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-34-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-34-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">158</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-35-1" class="td11">
GTP hydrolysis and joining of the 60S ribosomal subunit
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-35-2" class="td11">
<pre><code>   104           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-35-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-35-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">158</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-36-1" class="td11">
Eukaryotic Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-36-2" class="td11">
<pre><code>   111           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-36-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-36-4" class="td11">
<pre><code>4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">151</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-37-1" class="td11">
Cap-dependent Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-37-2" class="td11">
<pre><code>   111           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-37-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-37-4" class="td11">
<pre><code>4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">151</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-38-1" class="td11">
Influenza Infection
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-38-2" class="td11">
<pre><code>   117           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-38-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-38-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">145</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-39-1" class="td11">
Influenza Viral RNA Transcription and Replication
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-39-2" class="td11">
<pre><code>   108           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-39-3&quot;  </code></pre>
class=“td11”&gt; 72
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-39-4" class="td11">
<pre><code>5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">145</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-40-1" class="td11">
Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-40-2" class="td11">
<pre><code>   141           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-40-3&quot;  </code></pre>
class=“td11”&gt; 81
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-40-4" class="td11">
<pre><code>8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">143</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-41-1" class="td11">
Influenza Life Cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-41-2" class="td11">
<pre><code>   112           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-41-3&quot;  </code></pre>
class=“td11”&gt; 72
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-41-4" class="td11">
<pre><code>2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">141</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-42-1" class="td11">
Infectious disease
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-42-2" class="td11">
<pre><code>   347           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-42-3&quot;  </code></pre>
class=“td11”&gt; 103
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-42-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">95</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-43-1" class="td11">
Formation of the ternary complex, and subsequently, the 43S complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-43-2" class="td11">
<pre><code>    47            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-43-3&quot;  </code></pre>
class=“td11”&gt; 33
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-43-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-60-44-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-60-44-1"  
class="td11">
<pre><code>                                                               &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-45-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 3</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-45-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-45-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-45-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-46-1" class="td11">
Adaptive Immune System
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-46-2" class="td11">
<pre><code>   412           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-46-3&quot;  </code></pre>
class=“td11”&gt; 90
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-46-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">61</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-47-1" class="td11">
Chemokine receptors bind chemokines
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-47-2" class="td11">
<pre><code>    52            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-47-3&quot;  </code></pre>
class=“td11”&gt; 27
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-47-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">56</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-48-1" class="td11">
Generation of second messenger molecules
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-48-2" class="td11">
<pre><code>    29            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-48-3&quot;  </code></pre>
class=“td11”&gt; 21
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-48-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">55</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-49-1" class="td11">
Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-49-2" class="td11">
<pre><code>    64            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-49-3&quot;  </code></pre>
class=“td11”&gt; 29
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-49-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">55</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-50-1" class="td11">
TCR signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-50-2" class="td11">
<pre><code>    62            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-50-3&quot;  </code></pre>
class=“td11”&gt; 27
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-50-4" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">51</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-51-1" class="td11">
Peptide ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-51-2" class="td11">
<pre><code>   161           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-51-3&quot;  </code></pre>
class=“td11”&gt; 40
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-51-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">45</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-52-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-52-1" class="td11">
Translocation of ZAP-70 to Immunological synapse
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-52-2" class="td11">
<pre><code>    16            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-52-3&quot;  </code></pre>
class=“td11”&gt; 14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-52-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">43</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-53-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-53-1" class="td11">
Costimulation by the CD28 family
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-53-2" class="td11">
<pre><code>    51            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-53-3&quot;  </code></pre>
class=“td11”&gt; 22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-53-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">43</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-54-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-54-1" class="td11">
PD-1 signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-54-2" class="td11">
<pre><code>    21            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-54-3&quot;  </code></pre>
class=“td11”&gt; 15
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-54-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">41</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-55-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-55-1" class="td11">
Class A/1 (Rhodopsin-like receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-55-2" class="td11">
<pre><code>   258           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-55-3&quot;  </code></pre>
class=“td11”&gt; 50
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-55-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">41</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-56-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-56-1" class="td11">
Phosphorylation of CD3 and TCR zeta chains
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-56-2" class="td11">
<pre><code>    18            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-56-3&quot;  </code></pre>
class=“td11”&gt; 14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-56-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">40</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-57-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-57-1" class="td11">
Interferon gamma signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-57-2" class="td11">
<pre><code>    74            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-57-3&quot;  </code></pre>
class=“td11”&gt; 24
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-57-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">39</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-58-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-58-1" class="td11">
GPCR ligand binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-58-2" class="td11">
<pre><code>   326           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-58-3&quot;  </code></pre>
class=“td11”&gt; 57
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-58-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">38</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-59-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-59-1" class="td11">
Cytokine Signaling in Immune system
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-59-2" class="td11">
<pre><code>   268           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-59-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-59-4" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">37</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-60-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-60-1" class="td11">
Downstream TCR signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-60-2" class="td11">
<pre><code>    45            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-60-3&quot;  </code></pre>
class=“td11”&gt; 18
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-60-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">35</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-61-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-61-1" class="td11">
G<sub><span class="cmmi-8">αi</span> </sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-61-2" class="td11">
<pre><code>   167           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-61-3&quot;  </code></pre>
class=“td11”&gt; 33
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-61-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">33</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-62-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-62-1" class="td11">
Cell surface interactions at the vascular wall
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-62-2" class="td11">
<pre><code>    99            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-62-3&quot;  </code></pre>
class=“td11”&gt; 21
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-62-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-63-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-63-1" class="td11">
Interferon Signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-63-2" class="td11">
<pre><code>   164           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-63-3&quot;  </code></pre>
class=“td11”&gt; 28
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-63-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-64-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-64-1" class="td11">
Extracellular matrix organisation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-64-2" class="td11">
<pre><code>   238           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-64-3&quot;  </code></pre>
class=“td11”&gt; 35
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-64-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-65-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-65-1" class="td11">
Antigen activates B Cell Receptor leading to generation of second messengers
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-65-2" class="td11">
<pre><code>    32            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-65-3&quot;  </code></pre>
class=“td11”&gt; 12
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-65-4" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-60-66-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-60-66-1"  
class="td11">
<pre><code>                                                               &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-67-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-67-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 4</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-67-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-67-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-67-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-68-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-68-1" class="td11">
Extracellular matrix organisation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-68-2" class="td11">
<pre><code>   238           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-68-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-68-4" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">41</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-69-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-69-1" class="td11">
Class A/1 (Rhodopsin-like receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-69-2" class="td11">
<pre><code>   258           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-69-3&quot;  </code></pre>
class=“td11”&gt; 47
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-69-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">36</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-70-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-70-1" class="td11">
GPCR ligand binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-70-2" class="td11">
<pre><code>   326           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-70-3&quot;  </code></pre>
class=“td11”&gt; 54
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-70-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">34</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-71-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-71-1" class="td11">
G<sub><span class="cmmi-8">αs</span> </sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-71-2" class="td11">
<pre><code>    83            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-71-3&quot;  </code></pre>
class=“td11”&gt; 22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-71-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-72-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-72-1" class="td11">
GPCR downstream signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-72-2" class="td11">
<pre><code>   472           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-72-3&quot;  </code></pre>
class=“td11”&gt; 68
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-72-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-73-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-73-1" class="td11">
Haemostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-73-2" class="td11">
<pre><code>   423           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-73-3&quot;  </code></pre>
class=“td11”&gt; 61
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-73-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-74-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-74-1" class="td11">
Platelet activation, signalling and aggregation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-74-2" class="td11">
<pre><code>   180           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-74-3&quot;  </code></pre>
class=“td11”&gt; 31
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-74-4" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-75-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-75-1" class="td11">
Binding and Uptake of Ligands by Scavenger Receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-75-2" class="td11">
<pre><code>    40            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-75-3&quot;  </code></pre>
class=“td11”&gt; 14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-75-4" class="td11">
<pre><code> 9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-76-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-76-1" class="td11">
RA biosynthesis pathway
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-76-2" class="td11">
<pre><code>    22            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-76-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-76-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-77-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-77-1" class="td11">
Response to elevated platelet cytosolic Ca<sup><span class="cmr-8">2+</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-77-2" class="td11">
<pre><code>    82            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-77-3&quot;  </code></pre>
class=“td11”&gt; 19
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-77-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-78-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-78-1" class="td11">
Developmental Biology
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-78-2" class="td11">
<pre><code>   420           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-78-3&quot;  </code></pre>
class=“td11”&gt; 57
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-78-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-79-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-79-1" class="td11">
G<sub><span class="cmmi-8">αi</span> </sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-79-2" class="td11">
<pre><code>   167           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-79-3&quot;  </code></pre>
class=“td11”&gt; 28
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-79-4" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-80-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-80-1" class="td11">
Platelet degranulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-80-2" class="td11">
<pre><code>    77            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-80-3&quot;  </code></pre>
class=“td11”&gt; 18
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-80-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-81-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-81-1" class="td11">
Gastrin-CREB signalling pathway via PKC and MAPK
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-81-2" class="td11">
<pre><code>   171           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-81-3&quot;  </code></pre>
class=“td11”&gt; 28
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-81-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-82-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-82-1" class="td11">
Muscle contraction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-82-2" class="td11">
<pre><code>    62            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-82-3&quot;  </code></pre>
class=“td11”&gt; 16
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-82-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-83-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-83-1" class="td11">
G<sub><span class="cmmi-8">αq</span> </sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-83-2" class="td11">
<pre><code>   150           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-83-3&quot;  </code></pre>
class=“td11”&gt; 25
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-83-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-84-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-84-1" class="td11">
Retinoid metabolism and transport
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-84-2" class="td11">
<pre><code>    34            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-84-3&quot;  </code></pre>
class=“td11”&gt; 12
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-84-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-85-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-85-1" class="td11">
Phase 1 - Functionalisation of compounds
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-85-2" class="td11">
<pre><code>    67            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-85-3&quot;  </code></pre>
class=“td11”&gt; 16
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-85-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-86-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-86-1" class="td11">
Signalling by Retinoic Acid
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-86-2" class="td11">
<pre><code>    42            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-86-3&quot;  </code></pre>
class=“td11”&gt; 13
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-86-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-60-87-">
<td style="white-space:nowrap; text-align:left;" id="TBL-60-87-1" class="td11">
Degradation of the extracellular matrix
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-87-2" class="td11">
<pre><code>   102           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-60-87-3&quot;  </code></pre>
class=“td11”&gt; 19
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-60-87-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-60-88-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-60-88-1"  
class="td11">
<pre><code>                                                               &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                                       &lt;/div&gt;
                                                                  
                                                                                </code></pre>
</div>
<hr class="endfloat" />
<!--l. 396-->
<p class="noindent">
Synthetic lethal gene candidates for <span class="cmti-12">CDH1 </span>from <a 
href="#glo:SLIPT">SLIPT</a> performed on RNA-Seq expression data were also used for pathway over-representation analyses (as described in Section <a 
href="#x1-770002">2.3.2<!--tex4ht:ref: methods:enrichment --></a>). The correlation structure in the expression of candidates synthetic lethal genes in <span class="cmti-12">CDH1 </span>low tumours (lowest <span class="cmr-8">1</span> <span class="cmmi-12">∕</span> <span class="cmr-8">3</span><sup class="textsuperscript"><span class="cmr-10x-x-109">rd</span></sup> quantile of expression) was examined for distinct biological pathways in subgroups of genes elevated in different clusters of samples. These gene were highly expressed in different samples with their clinical factors including estrogen receptor status and intrinsic (PAM50) subtype (<a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XParker2009">2009</a>) shown in Figure <a 
href="#x1-120001r1">4.1<!--tex4ht:ref: fig:slipt_expr --></a>.
</p>
<!--l. 399-->
<p class="indent">
As shown by the most over-represented pathways in Table <a 
href="#x1-121001r3">4.3<!--tex4ht:ref: tab:pathway_clusters --></a>, each correlated cluster of candidate synthetic lethal partners of <span class="cmti-12">CDH1 </span>contains functionally different genes. Cluster 1 contains genes with less evidence of over-represented pathways than other clusters, corresponding to less correlation between genes within the cluster, and to it being a relatively small group. While there is some indication that collagen biosynthesis, microfibril elastic fibres, extracellular matrix, and metabolic pathways may be over-represented in Cluster 1, these results are mainly based on small pathways containing few synthetic lethal genes. Genes in Cluster 2 exhibited low expression in normal tissue samples compared to tumour samples (see Figure <a 
href="#x1-120001r1">4.1<!--tex4ht:ref: fig:slipt_expr --></a>) and show compelling evidence of over-representation of post-transcriptional gene regulation and protein translation processes. Similarly, Cluster 3 has over-representation of immune signalling pathways (including chemokines, secondary messenger, and TCR signaling) and downstream intracellular signalling cascades such as G protein coupled receptor (GPCR) and G<sub><span class="cmmi-8">αi</span></sub> signalling events. While pathway over-representation was weaker among genes in Cluster 4, they contained intracellular signalling pathways and were highly expressed in normal samples (in contrast to Cluster 2). Cluster 4 also involved extracellular factors and stimuli such as extracellular matrix, platelet activation, ligand receptors, and retinoic acid signalling.
</p>
<!--l. 402-->
<p class="indent">
Based on these results, potential synthetic lethal partners of <span class="cmti-12">CDH1 </span>include processes known to be dysregulated in cancer, such as translational, cytoskeletal, and immune processes. Intracellular signalling cascades such as the GPCRs and extracellular stimuli for these pathways were also implicated in potential synthetic lethality with <span class="cmti-12">CDH1</span>.
</p>
<!--l. 404-->
<p class="indent">
<p>Similar translational, cytoskeletal, and immune processes were identified among <a 
href="#glo:SLIPT">SLIPT</a> partners with respect to <span class="cmti-12">CDH1 </span>mutation, shown in Table <a 
href="#x1-206002r3">D.3<!--tex4ht:ref: tab:pathway_clusters_mtSL --></a>. While GPCR signalling was replicated in mtSLIPT analysis, there was also stronger over-representation for NOTCH,</p>
ERBB2, and PI3K/AKT signalling in mutation analysis consistent with these signals being important for proliferation of <span class="cmti-12">CDH1 </span>deficient tumours. The GCPR and PI3K/AKT pathways are of particular interest as pathways with oncogenic mutations that can be targeted and downstream effects on translation (a strongly supported process across analyses). Extracellular matrix pathways (such as elastic fibre formation) were also supported across analyses (in Tables <a 
href="#x1-121001r3">4.3<!--tex4ht:ref: tab:pathway_clusters --></a> and <a 
href="#x1-206002r3">D.3<!--tex4ht:ref: tab:pathway_clusters_mtSL --></a>) consistent with the established cell-cell signalling role of <span class="cmti-12">CDH1 </span>and the importance of the tumour microenvironment for cancer proliferation. <a 
 id="x1-121002r158"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">4.2 </span> <a 
 id="x1-1220002"></a>Comparison of synthetic lethal gene candidates
</h3>
<a 
 id="x1-122001r162"></a>
<h4 class="subsectionHead">
<span class="titlemark">4.2.1 </span> <a 
 id="x1-1230001"></a>Comparison with siRNA screen candidates
</h4>
<!--l. 420-->
<p class="noindent">
Gene candidates were compared between computational (<a 
href="#glo:SLIPT">SLIPT</a> in TCGA breast cancer data) and experimental (the primary <a 
href="#glo:siRNA">siRNA</a> screen performed by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>)) approaches in Figure <a 
href="#x1-123001r2">4.2<!--tex4ht:ref: fig:Venn_allgenes --></a>. The number of genes detected by both methods did not produce a significant overlap but these may be difficult to compare due to vast differences between the detection methods. There were similar issues comparison of mtSLIPT genes tested against <span class="cmti-12">CDH1 </span>mutations (in Figure <a 
href="#x1-223001r2">G.2<!--tex4ht:ref: fig:Venn_allgenes_stad_mtSL --></a>), despite exlcuded genes not tested by both methods in either test. However, these intersecting genes may still be functionally informative or amenable to drug triage as they were replicated across both methods and pathway over-representation differed between the sections of the Venn diagram (see Figure <a 
href="#x1-123001r2">4.2<!--tex4ht:ref: fig:Venn_allgenes --></a>).
</p>
<hr class="figure" />
<div class="figure">
<p><a 
 id="x1-123001r2"></a></p>
<hr />
____________________________________________________________________________________      <!--l. 425-->
<p class="noindent">
<img 
src="Venn_exprSL_siRNA_allgenes_reduced_Pub.png" alt="PIC"  
 /> <br />
</p>
<div class="caption">
<p><span class="id">Figure 4.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Comparison of </span><a 
href="#glo:SLIPT"><span class="cmbx-10x-x-109">SLIPT</span></a> <span class="cmbx-10x-x-109">to </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a><span class="cmbx-10x-x-109">.</span></strong> <span class="cmr-10x-x-109">Testing the overlap of gene</span> <span class="cmr-10x-x-109">candidates for E-cadherin synthetic lethal partners between computational (</span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a><span class="cmr-10x-x-109">) and</span> <span class="cmr-10x-x-109">experimental screening (</span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a><span class="cmr-10x-x-109">) approaches. The </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test suggests that the overlap is no</span> <span class="cmr-10x-x-109">more than would be expected by chance (</span><span class="cmmi-10x-x-109">p </span><span class="cmr-10x-x-109">= 0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">281). </span></span></p>
</div>
<!--tex4ht:label?: x1-123001r2 -->
</div>
<hr class="endfigure" />
<a 
 id="x1-123002r164"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">4.2.1.1 </span> <a 
 id="x1-1240001"></a>Comparison with correlation
</h5>
<!--l. 441-->
<p class="indent">
<a 
 id="x1-124001r3"></a><a 
 id="x1-124002r4"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 443-->
<p class="noindent">
</p>
<!--l. 444-->
<p class="noindent">
<img 
src="exprSLIPT_siRNA_vs_Correlation_with_CDH1.png" alt="PIC"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 4.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">Compare </span><a 
href="#glo:SLIPT"><span class="cmbx-10x-x-109">SLIPT</span></a> <span class="cmbx-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">genes with correlation.</span></strong> <span class="cmr-10x-x-109">The </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">p-values</span> <span class="cmr-10x-x-109">for genes tested by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">(in TCGA breast cancer) expression analysis were compared</span> <span class="cmr-10x-x-109">against Pearson’s correlation of gene expression with </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">. Genes detected by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">or </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">are coloured according to the legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-124001r3 -->
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 456-->
<p class="noindent">
</p>
<!--l. 457-->
<p class="noindent">
<img 
src="vioplotx_exprSLIPT_siRNA_vs_CDH1_Correlation_with_CDH1.png" alt="pict"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 4.4: </span><span class="content"><strong><span class="cmbx-10x-x-109">Compare </span><a 
href="#glo:SLIPT"><span class="cmbx-10x-x-109">SLIPT</span></a> <span class="cmbx-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">genes with correlation.</span></strong> <span class="cmr-10x-x-109">Genes detected</span> <span class="cmr-10x-x-109">as candidate synthetic lethal partners by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">(in TCGA breast cancer) expression</span> <span class="cmr-10x-x-109">analysis and experimental screening (with </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a><span class="cmr-10x-x-109">) were compared against Pearson’s</span> <span class="cmr-10x-x-109">correlation of gene expression with </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">. There were no differences in correlation</span> <span class="cmr-10x-x-109">between gene groups detected by either approach. </span></span></p>
</div>
<!--tex4ht:label?: x1-124002r4 -->
</div>
<hr class="endfloat" />
<!--l. 467-->
<p class="noindent">
Another potential means to triage drug target candidates is correlation of expression profiles with <span class="cmti-12">CDH1</span>. Correlation with <span class="cmti-12">CDH1 </span>was compared to <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> results in Figure <a 
href="#x1-124001r3">4.3<!--tex4ht:ref: fig:compare_points_correlation_SL --></a>. The genes not detected by <a 
href="#glo:SLIPT">SLIPT</a> (including <a 
href="#glo:siRNA">siRNA</a> candidates) included genes with high (insignificant) <a 
href="#glo:SLIPT">SLIPT</a> p-values. As expected, these genes were distributed around a correlation of zero and genes with higher correlation with <span class="cmti-12">CDH1 </span>(either direction) had more significant <a 
href="#glo:SLIPT">SLIPT</a> p-values, although there were exceptions to this trend and larger positive correlations were negative correations.
</p>
<!--l. 469-->
<p class="indent">
The majority of <a 
href="#glo:SLIPT">SLIPT</a> candidates appeared to have negative correlations and moreso for those genes detected by both approaches, although these were typically weak correlations and are unlikely to be sufficient to detect such genes on their own. This is supported by simulation results in Section <a 
href="#x1-1760001">6.1<!--tex4ht:ref: chap5:compare_ methods --></a>.
</p>
<!--l. 471-->
<p class="indent">
There were not strong postive correlations with <span class="cmti-12">CDH1 </span>among <a 
href="#glo:siRNA">siRNA</a> candidates, consistent with previous findings that co-expression is not predictive of synthetic lethality (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>; <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>). Negative correlation may not be indicative of synthetic lethality either as many <a 
href="#glo:siRNA">siRNA</a> candidates also had positive correlations. The <a 
href="#glo:SLIPT">SLIPT</a> methodology has shown to detect genes with both positive and negative correlations, although it does appear to preferentially detect negatively correlated genes to some extent. These findings were replicated with the mtSLIPT approach against <span class="cmti-12">CDH1 </span>mutation (in Figure <a 
href="#x1-209001r3">D.3<!--tex4ht:ref: fig:compare_points_correlation_mtSL --></a>), although the range of the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> p-values differ due to lower sample size for mutation analysis.
</p>
<!--l. 473-->
<p class="indent">
However, the apparent tendancy for genes detected by <a 
href="#glo:SLIPT">SLIPT</a> or <a 
href="#glo:siRNA">siRNA</a> to have negative correlations with <span class="cmti-12">CDH1 </span>expression may be due to the smaller number of genes in these groups. The distribution of <span class="cmti-12">CDH1 </span>correations does not differ across these gene groups (as shown by Figures <a 
href="#x1-124002r4">4.4<!--tex4ht:ref: fig:compare_correlation_SL --></a> and <a 
href="#x1-209002r4">D.4<!--tex4ht:ref: fig:compare_correlation_mtSL --></a>). Therefore further triage of gene candidates by correaltion is not suitable, nor is use of correlation itself to preduct synthetic lethal partners in the first place. <a 
 id="x1-124003r169"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">4.2.1.2 </span> <a 
 id="x1-1250002"></a>Comparison with viability
</h5>
<!--l. 479-->
<p class="noindent">
<p>A similar comparison of <a 
href="#glo:SLIPT">SLIPT</a> results was made with the viability ratio (of <span class="cmti-12">CDH1 </span>mutant to wildtype) in the primary <a 
href="#glo:siRNA">siRNA</a> screen performed by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>). The significance and viability thresholds used for <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> detection of synthetic lethal candidate partners of <span class="cmti-12">CDH1 </span>are clear in Figure <a 
href="#x1-125001r5">4.5<!--tex4ht:ref: fig:compare_points_viability_SL --></a>. However note that not all of the gene below these thresholds are neccessarily selected to be</p>
<p>candidate partners as additional criteria were used in each case: directional criteria as for <a 
href="#glo:SLIPT">SLIPT</a> (see Section <a 
href="#x1-930001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) and minimum wildtype viability for <a 
href="#glo:siRNA">siRNA</a> (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>).</p>
</p>
<!--l. 481-->
<p class="indent">
<a 
 id="x1-125001r5"></a><a 
 id="x1-125002r6"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 483-->
<p class="noindent">
</p>
<!--l. 484-->
<p class="noindent">
<img 
src="exprSLIPT_siRNA_vs_Viability_Ratio_with_CDH1.png" alt="PIC"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 4.5: </span><span class="content"><strong><span class="cmbx-10x-x-109">Compare </span><a 
href="#glo:SLIPT"><span class="cmbx-10x-x-109">SLIPT</span></a> <span class="cmbx-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">genes with </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">viability.</span></strong> <span class="cmr-10x-x-109">The </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">p-values for genes tested by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">(in TCGA breast cancer) expression analysis were</span> <span class="cmr-10x-x-109">compared against the viability ratio of </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutant and wildtype cells in the primary</span> <a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screen. Genes detected by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">or </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">are coloured according to the legend.</span> </span></p>
</div>
<!--tex4ht:label?: x1-125001r5 -->
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 496-->
<p class="noindent">
</p>
<!--l. 497-->
<p class="noindent">
<img 
src="exprSLIPT(p-val)_siRNA_vs_Viability_Ratio_with_CDH1.png" alt="pict"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 4.6: </span><span class="content"><strong><span class="cmbx-10x-x-109">Compare </span><a 
href="#glo:SLIPT"><span class="cmbx-10x-x-109">SLIPT</span></a> <span class="cmbx-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">genes with viability.</span></strong> <span class="cmr-10x-x-109">The </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">p-values</span> <span class="cmr-10x-x-109">for genes tested by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">(in TCGA breast cancer) expression analysis were compared</span> <span class="cmr-10x-x-109">(on a log-scale) against the viability ratio of </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutant and wildtype cells in the</span> <span class="cmr-10x-x-109">primary </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screen. Genes detected by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">or </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">are coloured according to</span> <span class="cmr-10x-x-109">the legend with a grey line for </span><span class="cmmi-10x-x-109">p </span><span class="cmr-10x-x-109">= 0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05. </span></span></p>
</div>
<!--tex4ht:label?: x1-125002r6 -->
</div>
<hr class="endfloat" />
<!--l. 507-->
<p class="indent">
There does not appear to be a clear relationship between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidates. Many genes not detected by both approaches were numerous in Figures <a 
href="#x1-123001r2">4.2<!--tex4ht:ref: fig:Venn_allgenes --></a> and <a 
href="#x1-207001r2">D.2<!--tex4ht:ref: fig:Venn_allgenes_mtSL --></a>. These genes detected by either are not neccessarily near the thresholds for the other. In this respect the <a 
href="#glo:SLIPT">SLIPT</a> approach with patient data and cell line experiments are independent means to identify synthetic lethal candidates. While genes detected by both approaches were not neccessarily more strongly supported by either, the genes with a viability closer to 1 (no synthetic lethal effect) in <a 
href="#glo:siRNA">siRNA</a> included those with more significant <a 
href="#glo:SLIPT">SLIPT</a> p-values whereas more extreme viability ratios tended to be less significant (as shown by a logarithmic plot in Figure <a 
href="#x1-125002r6">4.6<!--tex4ht:ref: fig:compare_pval_points_correlation_SL --></a>). Although it should be noted that genes with more moderate viability ratios were more common and <a 
href="#glo:SLIPT">SLIPT</a> was capable (despite adjusting for multiple testing) of detecting significant genes with extreme viability ratios, particularly those considerably lower than 1.
</p>
<!--l. 509-->
<p class="indent">
<p>However, there was not support for <a 
href="#glo:SLIPT">SLIPT</a> candidates or those detected by both approaches having considerably different viability ratios (as shown in Figures <a 
href="#x1-125003r7">4.7<!--tex4ht:ref: fig:compare_viability_SL --></a> and <a 
href="#x1-209003r5">D.5<!--tex4ht:ref: fig:compare_viability_mtSL --></a>). The difference between the gene groups stems largely from the viability thresholds used by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) to detect synthetic lethal candidates in the primary screen, rather than more extreme viability ratios for genes identified by <a 
href="#glo:SLIPT">SLIPT</a>.</p>
</p>
<!--l. 511-->
<p class="indent">
<a 
 id="x1-125003r7"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 513-->
<p class="noindent">
</p>
<!--l. 514-->
<p class="noindent">
<img 
src="vioplotx_exprSLIPT_siRNA_vs_Viability_Ratio_with_CDH1.png" alt="pict"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 4.7: </span><span class="content"><strong><span class="cmbx-10x-x-109">Compare </span><a 
href="#glo:SLIPT"><span class="cmbx-10x-x-109">SLIPT</span></a> <span class="cmbx-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">genes with </span><a 
href="#glo:siRNA"><span class="cmbx-10x-x-109">siRNA</span></a> <span class="cmbx-10x-x-109">viability.</span></strong> <span class="cmr-10x-x-109">Genes</span> <span class="cmr-10x-x-109">detected as candidate synthetic lethal partners by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">(in TCGA breast cancer)</span> <span class="cmr-10x-x-109">expression analysis and experimental screening (with </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a><span class="cmr-10x-x-109">) were compared against the</span> <span class="cmr-10x-x-109">viability ratio of </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutant and wildtype cells in the primary </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screen. There</span> <span class="cmr-10x-x-109">were clear no differences in viability between genes detected by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and those not with</span> <span class="cmr-10x-x-109">the differences being primarily due to viability thresholds being used to detect synthetic</span> <span class="cmr-10x-x-109">lethality by </span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">Telford </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">2015</span></a><span class="cmr-10x-x-109">). </span></span></p>
</div>
<!--tex4ht:label?: x1-125003r7 -->
</div>
<hr class="endfloat" />
<a 
 id="x1-125004r172"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">4.2.1.3 </span> <a 
 id="x1-1260003"></a>Comparison with secondary siRNA screen candidates
</h5>
<!--l. 528-->
<p class="noindent">
However, it should be noted that genes with a lower viability ratio were not necessarily the strongest supported by experimental screening. The primary screen (with 4 pooled <a 
href="#glo:siRNA">siRNAs</a>) has been used for the majority of comparisons in this thesis because the genome-wide panel of target genes screened enables a large number of genes to be compared with <a 
href="#glo:SLIPT">SLIPT</a> results from gene expression and somatic mutation analysis. A secondary screen was also performed by <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) on the isogenic MCF10A breast cell lines to individually validate the <a 
href="#glo:siRNA">siRNAs</a> separately, with the strongest candidates being those exhibiting synthetic lethal viability ratios replicated across independently targeting <a 
href="#glo:siRNA">siRNAs</a>. This was performed for the top 500 candidates (with the lowest viability ratio) from the primary screen and the 482 of these genes also tested by <a 
href="#glo:SLIPT">SLIPT</a> in breast cancer (and the 486 genes tested by <a 
href="#glo:SLIPT">SLIPT</a> in stomach cancer).
</p>
<!--l. 530-->
<p class="indent">
The secondary screen results are given in Appendix <a 
href="#x1-203000C">C<!--tex4ht:ref: appendix:secondary_screen --></a> which show that <a 
href="#glo:SLIPT">SLIPT</a> candidate genes are more significantly (<span class="cmmi-12">p </span>= 7<span class="cmmi-12">.</span>49 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">3</span></sup> by Fisher’s exact test) more likely to be validated in the secondary screen and are thus informative of more robust partner genes, in addition to providing support that these interactions are consistent with expression profiles from heterogeneous patient samples across genetic backgrounds. While the individual genes detected by either approach do not neccessarily match (and are potentially false-positives), the biological functions important in <span class="cmti-12">CDH1 </span>deficient cancers and potential mechanisms for specific targeting of them can be further supported by pathway analysis of the gene detected by either method. The genes detected by both approaches may therefore be more informative at the pathway level, where it is unlikely for a pathway to be consistently detected by chance. As the <a 
href="#glo:SLIPT">SLIPT</a> candidates differ from the siRNA candidates (and are more likely to be validated), they can provide additional mechanisms by which <span class="cmti-12">CDH1 </span>deficient cancers proliferate and vulnerabilities that may be exploited against them by using the synthetic lethal pathways. <a 
 id="x1-126001r176"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">4.2.1.4 </span> <a 
 id="x1-1270004"></a>Comparison of screen at pathway level
</h5>
<!--l. 536-->
<p class="indent">
<a 
 id="x1-127001r4"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 4.4: </span><span class="content">Pathway composition for <span class="cmti-12">CDH1 </span>partners from SLIPT and siRNA screening</span></p>
</div>
<!--tex4ht:label?: x1-127001r4 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-61" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-61-1g">
<col 
id="TBL-61-1" />
<col 
id="TBL-61-2" />
<col 
id="TBL-61-3" />
<col 
id="TBL-61-4" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-61-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-1-1" class="td11">
<span class="cmbx-12">Predicted only by </span><a 
href="#glo:SLIPT"><span class="cmbx-12">SLIPT</span></a> <span class="cmbx-12">(4025 genes) </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-1-2" class="td11">
Pathway Size
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-1-3" class="td11">
Genes Identified
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-1-4" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-2-1" class="td11">
Eukaryotic Translation Elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-2-2" class="td11">
<pre><code>  80         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-2-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-2-4" class="td11">
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">182</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-3-1" class="td11">
Peptide chain elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-3-2" class="td11">
<pre><code>  77         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-3-3&quot;  </code></pre>
class=“td11”&gt; 72
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-3-4" class="td11">
2<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">176</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-4-1" class="td11">
Viral mRNA Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-4-2" class="td11">
<pre><code>  75         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-4-3&quot;  </code></pre>
class=“td11”&gt; 70
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-4-4" class="td11">
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">172</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-5-1" class="td11">
Eukaryotic Translation Termination
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-5-2" class="td11">
<pre><code>  76         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-5-3&quot;  </code></pre>
class=“td11”&gt; 70
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-5-4" class="td11">
5<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">170</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-6-1" class="td11">
Formation of a pool of free 40S subunits
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-6-2" class="td11">
<pre><code>  87         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-6-3&quot;  </code></pre>
class=“td11”&gt; 74
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-6-4" class="td11">
9<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">166</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-7-1" class="td11">
Nonsense Mediated Decay independent of the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-7-2" class="td11">
<pre><code>  81         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-7-3&quot;  </code></pre>
class=“td11”&gt; 70
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-7-4" class="td11">
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">160</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-8-1" class="td11">
L13a-mediated translational silencing of Ceruloplasmin expression
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-8-2" class="td11">
<pre><code>  97         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-8-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-8-4" class="td11">
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">155</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-9-1" class="td11">
3’ -UTR-mediated translational regulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-9-2" class="td11">
<pre><code>  97         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-9-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-9-4" class="td11">
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">155</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-10-1" class="td11">
GTP hydrolysis and joining of the 60S ribosomal subunit
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-10-2" class="td11">
<pre><code>  98         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-10-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-10-4" class="td11">
6<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">154</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-11-1" class="td11">
Nonsense-Mediated Decay
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-11-2" class="td11">
<pre><code>  96         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-11-3&quot;  </code></pre>
class=“td11”&gt; 73
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-11-4" class="td11">
5<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">150</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-12-1" class="td11">
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-12-2" class="td11">
<pre><code>  96         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-12-3&quot;  </code></pre>
class=“td11”&gt; 73
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-12-4" class="td11">
5<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">150</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-13-1" class="td11">
SRP-dependent cotranslational protein targeting to membrane
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-13-2" class="td11">
<pre><code>  97         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-13-3&quot;  </code></pre>
class=“td11”&gt; 73
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-13-4" class="td11">
7<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">149</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-14-1" class="td11">
Eukaryotic Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-14-2" class="td11">
<pre><code> 105        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-14-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-14-4" class="td11">
4<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">146</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-15-1" class="td11">
Cap-dependent Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-15-2" class="td11">
<pre><code> 105        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-15-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-15-4" class="td11">
4<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">146</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-16-1" class="td11">
Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-16-2" class="td11">
<pre><code> 133        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-16-3&quot;  </code></pre>
class=“td11”&gt; 83
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-16-4" class="td11">
4<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">142</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-17-1" class="td11">
Influenza Viral RNA Transcription and Replication
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-17-2" class="td11">
<pre><code> 102        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-17-3&quot;  </code></pre>
class=“td11”&gt; 71
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-17-4" class="td11">
2<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">137</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-18-1" class="td11">
Influenza Infection
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-18-2" class="td11">
<pre><code> 111        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-18-3&quot;  </code></pre>
class=“td11”&gt; 74
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-18-4" class="td11">
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">137</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-19-1" class="td11">
Influenza Life Cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-19-2" class="td11">
<pre><code> 106        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-19-3&quot;  </code></pre>
class=“td11”&gt; 71
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-19-4" class="td11">
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">133</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-20-1" class="td11">
Infectious disease
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-20-2" class="td11">
<pre><code> 326        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-20-3&quot;  </code></pre>
class=“td11”&gt; 125
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-20-4" class="td11">
4<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">120</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-21-1" class="td11">
Extracellular matrix organisation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-21-2" class="td11">
<pre><code> 189        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-21-3&quot;  </code></pre>
class=“td11”&gt; 77
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-21-4" class="td11">
5<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">95</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-61-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-61-22-1"  
class="td11">
<pre><code>                                                         &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-23-1" class="td11">
<span class="cmbx-12">Detected only by siRNA screen (1599 genes) </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-23-2" class="td11">
Pathway Size
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-23-3" class="td11">
Genes Identified
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-23-4" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-24-1" class="td11">
Class A/1 (Rhodopsin-like receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-24-2" class="td11">
<pre><code> 282        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-24-3&quot;  </code></pre>
class=“td11”&gt; 44
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-24-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-25-1" class="td11">
GPCR ligand binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-25-2" class="td11">
<pre><code> 363        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-25-3&quot;  </code></pre>
class=“td11”&gt; 52
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-25-4" class="td11">
5<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">26</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-26-1" class="td11">
G<sub><span class="cmmi-8">αq</span></sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-26-2" class="td11">
<pre><code> 159        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-26-3&quot;  </code></pre>
class=“td11”&gt; 26
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-26-4" class="td11">
6<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">23</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-27-1" class="td11">
Gastrin-CREB signalling pathway via PKC and MAPK
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-27-2" class="td11">
<pre><code> 180        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-27-3&quot;  </code></pre>
class=“td11”&gt; 27
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-27-4" class="td11">
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-28-1" class="td11">
G<sub><span class="cmmi-8">αi</span></sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-28-2" class="td11">
<pre><code> 184        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-28-3&quot;  </code></pre>
class=“td11”&gt; 27
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-28-4" class="td11">
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-29-1" class="td11">
Downstream signal transduction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-29-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-29-3&quot;  </code></pre>
class=“td11”&gt; 23
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-29-4" class="td11">
7<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-30-1" class="td11">
Signalling by PDGF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-30-2" class="td11">
<pre><code> 172        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-30-3&quot;  </code></pre>
class=“td11”&gt; 25
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-30-4" class="td11">
4<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-31-1" class="td11">
Peptide ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-31-2" class="td11">
<pre><code> 175        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-31-3&quot;  </code></pre>
class=“td11”&gt; 25
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-31-4" class="td11">
8<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-32-1" class="td11">
Signalling by ERBB2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-32-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-32-3&quot;  </code></pre>
class=“td11”&gt; 22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-32-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-33-1" class="td11">
DAP12 interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-33-2" class="td11">
<pre><code> 159        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-33-3&quot;  </code></pre>
class=“td11”&gt; 23
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-33-4" class="td11">
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-34-1" class="td11">
DAP12 signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-34-2" class="td11">
<pre><code> 149        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-34-3&quot;  </code></pre>
class=“td11”&gt; 22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-34-4" class="td11">
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-35-1" class="td11">
Organelle biogenesis and maintenance
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-35-2" class="td11">
<pre><code> 264        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-35-3&quot;  </code></pre>
class=“td11”&gt; 33
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-35-4" class="td11">
5<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-36-1" class="td11">
Signalling by NGF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-36-2" class="td11">
<pre><code> 266        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-36-3&quot;  </code></pre>
class=“td11”&gt; 33
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-36-4" class="td11">
8<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-37-1" class="td11">
Downstream signalling of activated FGFR1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-37-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-37-3&quot;  </code></pre>
class=“td11”&gt; 20
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-37-4" class="td11">
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-38-1" class="td11">
Downstream signalling of activated FGFR2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-38-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-38-3&quot;  </code></pre>
class=“td11”&gt; 20
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-38-4" class="td11">
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-39-1" class="td11">
Downstream signalling of activated FGFR3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-39-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-39-3&quot;  </code></pre>
class=“td11”&gt; 20
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-39-4" class="td11">
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-40-1" class="td11">
Downstream signalling of activated FGFR4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-40-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-40-3&quot;  </code></pre>
class=“td11”&gt; 20
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-40-4" class="td11">
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-41-1" class="td11">
Signalling by FGFR
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-41-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-41-3&quot;  </code></pre>
class=“td11”&gt; 21
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-41-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-42-1" class="td11">
Signalling by FGFR1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-42-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-42-3&quot;  </code></pre>
class=“td11”&gt; 21
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-42-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-43-1" class="td11">
Signalling by FGFR2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-43-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-43-3&quot;  </code></pre>
class=“td11”&gt; 21
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-43-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-61-44-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-61-44-1"  
class="td11">
<pre><code>                                                         &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-45-1" class="td11">
<span class="cmbx-12">Intersection of </span><a 
href="#glo:SLIPT"><span class="cmbx-12">SLIPT</span></a> <span class="cmbx-12">and siRNA screen (604 genes) </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-45-2" class="td11">
Pathway Size
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-45-3" class="td11">
Genes Identified
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-45-4" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-46-1" class="td11">
Visual phototransduction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-46-2" class="td11">
<pre><code>  54         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-46-3&quot;  </code></pre>
class=“td11”&gt; 9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-46-4" class="td11">
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-47-1" class="td11">
G<sub><span class="cmmi-8">αs</span> </sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-47-2" class="td11">
<pre><code>  48         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-47-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-47-4" class="td11">
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-48-1" class="td11">
Retinoid metabolism and transport
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-48-2" class="td11">
<pre><code>  24         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-48-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-48-4" class="td11">
1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-49-1" class="td11">
Acyl chain remodelling of PS
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-49-2" class="td11">
<pre><code>  10         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-49-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-49-4" class="td11">
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-50-1" class="td11">
Transcriptional regulation of white adipocyte differentiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-50-2" class="td11">
<pre><code>  51         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-50-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-50-4" class="td11">
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-51-1" class="td11">
Chemokine receptors bind chemokines
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-51-2" class="td11">
<pre><code>  22         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-51-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-51-4" class="td11">
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-52-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-52-1" class="td11">
Signalling by NOTCH4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-52-2" class="td11">
<pre><code>  11         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-52-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-52-4" class="td11">
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-53-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-53-1" class="td11">
Defective EXT2 causes exostoses 2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-53-2" class="td11">
<pre><code>  11         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-53-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-53-4" class="td11">
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-54-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-54-1" class="td11">
Defective EXT1 causes exostoses 1, TRPS2 and CHDS
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-54-2" class="td11">
<pre><code>  11         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-54-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-54-4" class="td11">
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-55-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-55-1" class="td11">
Platelet activation, signalling and aggregation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-55-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-55-3&quot;  </code></pre>
class=“td11”&gt; 12
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-55-4" class="td11">
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-56-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-56-1" class="td11">
Phase 1 - Functionalisation of compounds
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-56-2" class="td11">
<pre><code>  41         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-56-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-56-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-57-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-57-1" class="td11">
Amine ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-57-2" class="td11">
<pre><code>  13         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-57-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-57-4" class="td11">
1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-58-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-58-1" class="td11">
Acyl chain remodelling of PE
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-58-2" class="td11">
<pre><code>  14         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-58-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-58-4" class="td11">
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-59-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-59-1" class="td11">
Signalling by GPCR
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-59-2" class="td11">
<pre><code> 300        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-59-3&quot;  </code></pre>
class=“td11”&gt; 23
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-59-4" class="td11">
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-60-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-60-1" class="td11">
Molecules associated with elastic fibres
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-60-2" class="td11">
<pre><code>  29         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-60-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-60-4" class="td11">
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-61-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-61-1" class="td11">
DAP12 interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-61-2" class="td11">
<pre><code> 128        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-61-3&quot;  </code></pre>
class=“td11”&gt; 10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-61-4" class="td11">
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-62-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-62-1" class="td11">
Cytochrome P<sub><span class="cmr-8">450</span></sub> - arranged by substrate type
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-62-2" class="td11">
<pre><code>  30         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-62-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-62-4" class="td11">
3<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-63-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-63-1" class="td11">
GPCR ligand binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-63-2" class="td11">
<pre><code> 147        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-63-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-63-4" class="td11">
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-64-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-64-1" class="td11">
Acyl chain remodelling of PC
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-64-2" class="td11">
<pre><code>  16         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-64-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-64-4" class="td11">
4<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-61-65-">
<td style="white-space:nowrap; text-align:left;" id="TBL-61-65-1" class="td11">
Response to elevated platelet cytosolic Ca<sup><span class="cmr-8">2+</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-65-2" class="td11">
<pre><code>  66         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-61-65-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-61-65-4" class="td11">
4<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-61-66-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-61-66-1"  
class="td11">
<pre><code>                                                         &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                         &lt;/div&gt;
                                                                                
                                                                                </code></pre>
</div>
<hr class="endfloat" />
<!--l. 680-->
<p class="noindent">
These pathway over-representation analyses (performed as described in Section <a 
href="#x1-770002">2.3.2<!--tex4ht:ref: methods:enrichment --></a>) correspond to genes separated into <a 
href="#glo:SLIPT">SLIPT</a> or siRNA screen candidates unique to either method or detected by both (Table <a 
href="#x1-127001r4">4.4<!--tex4ht:ref: tab:Venn_over-representation --></a>). The <a 
href="#glo:SLIPT">SLIPT</a>-specific gene candidates were involved most strongly with translational and immune regulatory pathways, although extracellular matrix pathways were also supported. These pathways were largely consistent with those identified in Tabel <a 
href="#x1-119001r2">4.2<!--tex4ht:ref: tab:pathway_exprSL --></a> and in the clustering analysis (Table <a 
href="#x1-121001r3">4.3<!--tex4ht:ref: tab:pathway_clusters --></a>). The genes detected only by the siRNA screen had over-representation of cell signalling pathways, including many containing genes known to be involved in cancer (e.g., MAPK, PDGF, ERBB2, and FGFR), with the detection of Class A GPCRs supporting the independent analyses by Telford <span class="cmti-12">et al. </span><a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>). The intersection of computational and experimental synthetic lethal partners of <span class="cmti-12">CDH1 </span>has stronger evidence for over-representation of GPCR pathways and more specific subclasses, such as visual phototransduction (<span class="cmmi-12">p </span>= 6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>) and G<sub> <span class="cmmi-8">αs</span></sub> signalling events (<span class="cmmi-12">p </span>= 1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>), than other signalling pathways.
</p>
<!--l. 682-->
<p class="indent">
The pathway analysis for mtSLIPT against <span class="cmti-12">CDH1 </span>mutations (in Table <a 
href="#x1-207002r4">D.4<!--tex4ht:ref: tab:Venn_over-representation_mtSL --></a>) had concordant results for both mtSLIPT-specific and siRNA-specific pathways. While the specific pathway composition of the intersection of these analyses differed from <a 
href="#glo:SLIPT">SLIPT</a> against low <span class="cmti-12">CDH1 </span>expression, signalling pathways including GPCRs, NOTCH, EERB2, PDGF, and SCF-KIT. These findings indicate the signalling pathways are among the most suitable vulnerability to exploit in targeting <span class="cmti-12">CDH1 </span>deficient tumours as they can be detected in both a patient cohort (with TCGA expression data) and tested in a laboratory system. However, it is possible that the isolated experimental system is set up to preferentially detect kinase singalling pathways (which are amenable to pharmacological inhibition and translation to the clinic) and the other pathways identified by <a 
href="#glo:SLIPT">SLIPT</a> may still be informative of the role of <span class="cmti-12">CDH1 </span>loss of function in cancers or mechanisms by which further gene loss leads to specific inviability. <a 
 id="x1-127002r134"></a>
</p>
<a 
 id="x1-1280001"></a>Resampling of genes for pathway enrichment <!--l. 692-->
<p class="indent">
<p>Comparing genes between experimental screen candidates and prediction from TCGA expression data has been less consistent than pathways. Although this is not unexpected since synthetic lethal pathways more more robustly conserved (<a 
href="#XDixon2008">Dixon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDixon2008">2008</a>) and the</p>
computational approach using patient samples from complex tumour microenvironment has considerably different strengths to an experimental screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) based on genetically homogenous cell line models in an isolated laboratory environment. For instance, it is unlikely for immune signaling to be detected in an isolated cell culture system.
</p>
<hr class="figure" />
<div class="figure">
<p><a 
 id="x1-128001r8"></a></p>
<hr />
____________________________________________________________________________________      <!--l. 697-->
<p class="noindent">
<img 
src="sample_size_dist_exprSL_1M_Pub.png" alt="PIC"  
 /> <br />
</p>
<div class="caption">
<p><span class="id">Figure 4.8: </span><span class="content"><strong><span class="cmbx-10x-x-109">Resampled intersection of </span><a 
href="#glo:SLIPT"><span class="cmbx-10x-x-109">SLIPT</span></a> <span class="cmbx-10x-x-109">and siRNA candidates.</span></strong> <span class="cmr-10x-x-109">Resampling analysis of intersect size from genes detected by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and siRNA screening</span> <span class="cmr-10x-x-109">approaches over 1 million replicates. The proportion of expected intersection sizes for</span> <span class="cmr-10x-x-109">random samples below or above the observed intersection size respectively, lacking</span> <span class="cmr-10x-x-109">significant over-representation or depletion of siRNA screen candidates within the </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">predictions for </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">. </span></span></p>
</div>
<!--tex4ht:label?: x1-128001r8 -->
</div>
<hr class="endfigure" />
<!--l. 707-->
<p class="indent">
The overlap between synthetic lethal from bioinformatics <a 
href="#glo:SLIPT">SLIPT</a> predictions and siRNA screening has raised other questions including whether the pathways over-represented would be expected by chance. This of particular concern since the siRNA candidate genes themselves are highly over-represented for particular pathways (such as GPCRs) so selecting any intersect with them would be enriched for these pathways. Another pathway approach is to test whether pathways are over-represented in randomly sampled genes, comparing many “resamplings” or “permutations” of these genes to the enrichment statistics observed for these pathways in the <a 
href="#glo:SLIPT">SLIPT</a> candidates and their intersection with the siRNA hits shows whether we detect these pathways more than we expect by chance (as described in Section <a 
href="#x1-820006">2.3.6<!--tex4ht:ref: methods:permutation --></a>).
</p>
<!--l. 709-->
<p class="indent">
Of particular concern are the over-represented pathways in genes detected by both methods. Pathway over-representation alone does not detect whether <a 
href="#glo:SLIPT">SLIPT</a> predicted genes or siRNA candidates are enriched within each other. This resampling analysis therefore detects whether over-represented pathways were detected by <a 
href="#glo:SLIPT">SLIPT</a> independently of their over-representation among siRNA candidates (without assuming an underlying test statistic distribution).
</p>
<!--l. 711-->
<p class="indent">
A resampling approach is also applicable to testing whether the number of genes detected by each approach significantly intersected. As shown in Figure <a 
href="#x1-128001r8">4.8<!--tex4ht:ref: fig:perm_sample --></a>, resampling did not find evidence of significant depletion or over-representation for experimental synthetic lethal candidates in the computationally predicted synthetic lethal partners of <span class="cmti-12">CDH1 </span>and the overlap may be observed by chance. This is consistent with previous findings (see Figure <a 
href="#x1-123001r2">4.2<!--tex4ht:ref: fig:Venn_allgenes --></a>) and does not preclude pathway relationships being supported by resampling.
</p>
<!--l. 713-->
<p class="indent">
<p>A permutation analysis was performed to resample the genes tested by both approaches to investigate whether the observed pathway over-representation could have occurred in a randomly selected sample of genes from the experimental candidates, that is, whether the pathway predictions from <a 
href="#glo:SLIPT">SLIPT</a> could be expected by chance (as described in sections <a 
href="#x1-740001">2.2.3.1<!--tex4ht:ref: methods:venn_analysis --></a> and <a 
href="#x1-820006">2.3.6<!--tex4ht:ref: methods:permutation --></a>). While the number of siRNA candidate genes detected by <a 
href="#glo:SLIPT">SLIPT</a> was not statistically significant (<span class="cmmi-12">p </span>= 0<span class="cmmi-12">.</span>281), this may be due to the vastly different limitations of the approaches and the correlation structure of gene expression not being independent (as assumed for multiple testing procedures). The intersection may still be functionally relevant to <span class="cmti-12">CDH1</span>-deficient cancers, such as the pathway data in Table <a 
href="#x1-127001r4">4.4<!--tex4ht:ref: tab:Venn_over-representation --></a>. The resampling analysis for pathways was compared to the pathway over-representation for <a 
href="#glo:SLIPT">SLIPT</a> predicted synthetic lethal partners in Table <a 
href="#x1-128002r5">4.5<!--tex4ht:ref: tab:pathway_perm --></a>. Similarly, the pathway resampling</p>
for intersection between <a 
href="#glo:SLIPT">SLIPT</a> predictions and experimental screen candidates was compared to pathway over-representation in Table <a 
href="#x1-128003r6">4.6<!--tex4ht:ref: tab:pathway_perm_overlap --></a> for intersection with siRNA data.
</p>
<!--l. 717-->
<p class="indent">
<p>The pathway resampling approach for <a 
href="#glo:SLIPT">SLIPT</a>-specific gene candidates (Table <a 
href="#x1-128002r5">4.5<!--tex4ht:ref: tab:pathway_perm --></a>) replicates the gene set over-representation analysis for all <a 
href="#glo:SLIPT">SLIPT</a> genes, detecting evidence of synthetic lethal pathways for <span class="cmti-12">CDH1 </span>in translational, immune, and cell signalling pathways including G<sub><span class="cmmi-8">αi</span> </sub> signalling, GPCR downstream signalling, and chemokine receptor binding. While the immune and signal transduction pathways were not significantly over-represented in the resampling analysis, the results for the two approaches were largely consistent for translation and post-transcriptional gene regulation, supporting gene set over-representation of the <a 
href="#glo:SLIPT">SLIPT</a>-specific pathways in Table <a 
href="#x1-128002r5">4.5<!--tex4ht:ref: tab:pathway_perm --></a>. In particular, some of the most significantly over-represented pathways had higher observed <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values than any of the 1 million random permutations. Similar pathways were also replicated by permutation analysis for mt<a 
href="#glo:SLIPT">SLIPT</a> candidate partners against <span class="cmti-12">CDH1 </span>mutation (shown in Table <a 
href="#x1-208001r5">D.5<!--tex4ht:ref: tab:pathway_perm_mtSL --></a>). This shows that many of the pathways detected specfically by <a 
href="#glo:SLIPT">SLIPT</a> are replicated by permutation procedures and that the permutation approach is capable of detecting many of the most strongly over-represented pathways.</p>
</p>
<!--l. 720-->
<p class="indent">
<a 
 id="x1-128002r5"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 4.5: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT</span></p>
</div>
<!--tex4ht:label?: x1-128002r5 -->
<div class="tabular">
<table id="TBL-62" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-62-1g">
<col 
id="TBL-62-1" />
<col 
id="TBL-62-2" />
<col 
id="TBL-62-3" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-62-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-1-1" class="td11">
<span class="cmbx-12">Reactome Pathway </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-1-2" class="td11">
Over-representation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-1-3" class="td11">
Permutation
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-2-1" class="td11">
<strong><span class="cmbx-12">Eukaryotic Translation Elongation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-2-2" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">207</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-2-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-3-1" class="td11">
<strong><span class="cmbx-12">Peptide chain elongation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-3-2" class="td11">
<pre><code>5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">201</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-3-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-4-1" class="td11">
<strong><span class="cmbx-12">Viral mRNA Translation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-4-2" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">196</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-4-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-5-1" class="td11">
<strong><span class="cmbx-12">Eukaryotic Translation Termination</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-5-2" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">196</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-5-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-6-1" class="td11">
<strong><span class="cmbx-12">Formation of a pool of free 40S subunits</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-6-2" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">194</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-6-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-7-1" class="td11">
<strong><span class="cmbx-12">Nonsense Mediated Decay independent of the Exon Junction Complex</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-7-2" class="td11">
<pre><code>5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">187</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-7-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-8-1" class="td11">
<strong><span class="cmbx-12">L13a-mediated translational silencing of Ceruloplasmin expression</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-8-2" class="td11">
<pre><code>9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">183</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-8-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-9-1" class="td11">
<strong><span class="cmbx-12">3’ -UTR-mediated translational regulation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-9-2" class="td11">
<pre><code>9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">183</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-9-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-10-1" class="td11">
<strong><span class="cmbx-12">GTP hydrolysis and joining of the 60S ribosomal subunit</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-10-2" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">181</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-10-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-11-1" class="td11">
<strong><span class="cmbx-12">Nonsense-Mediated Decay</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-11-2" class="td11">
<pre><code>6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">176</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-11-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-12-1" class="td11">
<strong><span class="cmbx-12">Nonsense Mediated Decay enhanced by the Exon Junction Complex</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-12-2" class="td11">
<pre><code>6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">176</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-12-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-13-1" class="td11">
Adaptive Immune System
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-13-2" class="td11">
<pre><code>6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">174</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-13-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>15753
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-14-1" class="td11">
<strong><span class="cmbx-12">Eukaryotic Translation Initiation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-14-2" class="td11">
<pre><code>5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">173</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-14-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-15-1" class="td11">
<strong><span class="cmbx-12">Cap-dependent Translation Initiation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-15-2" class="td11">
<pre><code>5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">173</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-15-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-16-1" class="td11">
<strong><span class="cmbx-12">SRP-dependent cotranslational protein targeting to membrane</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-16-2" class="td11">
<pre><code>2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">171</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-16-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-17-1" class="td11">
<strong><span class="cmbx-12">Translation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-17-2" class="td11">
<pre><code>6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">170</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-17-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-18-1" class="td11">
Infectious disease
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-18-2" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">166</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-18-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>23231
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-19-1" class="td11">
<strong><span class="cmbx-12">Influenza Infection</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-19-2" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">163</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-19-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-20-1" class="td11">
<strong><span class="cmbx-12">Influenza Viral RNA Transcription and Replication</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-20-2" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">160</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-20-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-21-1" class="td11">
<strong><span class="cmbx-12">Influenza Life Cycle</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-21-2" class="td11">
<pre><code>2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">156</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-21-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-22-1" class="td11">
<span class="cmti-12">Extracellular matrix organisation </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-22-2" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">152</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-22-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>071761
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-23-1" class="td11">
GPCR ligand binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-23-2" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">143</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-23-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>55801
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-24-1" class="td11">
Class A/1 (Rhodopsin-like receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-24-2" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">142</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-24-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>58901
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-25-1" class="td11">
<span class="cmti-12">GPCR downstream signalling </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-25-2" class="td11">
<pre><code>7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">140</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-25-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>098357
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-26-1" class="td11">
Haemostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-26-2" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">134</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-26-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>27059
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-27-1" class="td11">
Developmental Biology
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-27-2" class="td11">
<pre><code>2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">123</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-27-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>52737
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-28-1" class="td11">
Metabolism of lipids and lipoproteins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-28-2" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">120</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-28-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>724
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-29-1" class="td11">
Cytokine Signalling in Immune system
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-29-2" class="td11">
<pre><code>2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">119</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-29-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>39661
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-30-1" class="td11">
Peptide ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-30-2" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">109</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-30-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>61102
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-62-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-62-31-1" class="td11">
<strong><span class="cmbx-12">G</span><sub><span class="cmmi-8">αi</span> </sub> <span class="cmbx-12">signalling events</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-31-2" class="td11">
<pre><code>8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">100</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-62-31-3" class="td11">
<span class="cmmi-12">&lt; </span>1<span class="cmmi-12">.</span>241 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-62-32-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-62-32-1"  
class="td11">
<pre><code>                                                                  &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 839-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Over-representation (hypergeometric test) and Permutation p-values adjusted for multiple tests across pathways (FDR). Significant</span> <span class="cmr-10x-x-109">pathways are marked in bold (FDR </span><span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05) and italics (FDR </span><span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">1).</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 842-->
<p class="indent">
The permutation approach was then also applied to the intersection between computational and experimental candidates. Where the permutation anlaysis is testing for consistent detection of pathways independent of their pre-existing status as experimental candidates. The pathway results for these candidate partners (in Table <a 
href="#x1-128003r6">4.6<!--tex4ht:ref: tab:pathway_perm_overlap --></a>) differed between over-representation and resampling analyses.
</p>
<!--l. 844-->
<p class="indent">
<p>Namely, many of the over-represented pathways were not significant in the resampling analysis, including visual phototransduction and retinoic acid signalling, although pathways involving defective <span class="cmti-12">EXT1 </span>or <span class="cmti-12">EXT2 </span>genes approach significance after FDR adjustment for multiple tests. Of the highest over-represented pathways in the intersection, only G<sub><span class="cmmi-8">αs</span></sub> signalling events were supported by both over-representation and resampling analyses. Other pathways supported by both analyses were cytoplasmic elastic fibre formation, associated HS-GAG protein modification pathways, energy metabolism, and the fibrin clotting cascade.</p>
</p>
<!--l. 846-->
<p class="indent">
<a 
 id="x1-128003r6"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 4.6: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT and siRNA primary screen</span></p>
</div>
<!--tex4ht:label?: x1-128003r6 -->
<div class="tabular">
<table id="TBL-63" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-63-1g">
<col 
id="TBL-63-1" />
<col 
id="TBL-63-2" />
<col 
id="TBL-63-3" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-63-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-1-1" class="td11">
<span class="cmbx-12">Reactome Pathway </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-1-2" class="td11">
Over-representation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-1-3" class="td11">
Permutation
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-2-1" class="td11">
Visual phototransduction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-2-2" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-2-3" class="td11">
0<span class="cmmi-12">.</span>91116
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-3-1" class="td11">
<strong><span class="cmbx-12">G</span><sub><span class="cmmi-8">αs</span> </sub> <span class="cmbx-12">signalling events</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-3-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-3-3" class="td11">
0<span class="cmmi-12">.</span>012988
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-4-1" class="td11">
Retinoid metabolism and transport
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-4-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-4-3" class="td11">
0<span class="cmmi-12">.</span>20487
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-5-1" class="td11">
Transcriptional regulation of white adipocyte differentiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-5-2" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-5-3" class="td11">
0<span class="cmmi-12">.</span>38197
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-6-1" class="td11">
Acyl chain remodelling of PS
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-6-2" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-6-3" class="td11">
0<span class="cmmi-12">.</span>58485
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-7-1" class="td11">
Chemokine receptors bind chemokines
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-7-2" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-7-3" class="td11">
0<span class="cmmi-12">.</span>97255
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-8-1" class="td11">
<span class="cmti-12">Defective EXT2 causes exostoses 2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-8-2" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-8-3" class="td11">
0<span class="cmmi-12">.</span>056437
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-9-1" class="td11">
<span class="cmti-12">Defective EXT1 causes exostoses 1, TRPS2 and CHDS </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-9-2" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-9-3" class="td11">
0<span class="cmmi-12">.</span>056437
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-10-1" class="td11">
Signalling by NOTCH4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-10-2" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-10-3" class="td11">
0<span class="cmmi-12">.</span>15497
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-11-1" class="td11">
Platelet activation, signalling and aggregation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-11-2" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-11-3" class="td11">
0<span class="cmmi-12">.</span>53358
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-12-1" class="td11">
Phase 1 - Functionalisation of compounds
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-12-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-12-3" class="td11">
0<span class="cmmi-12">.</span>24836
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-13-1" class="td11">
Amine ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-13-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-13-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3195
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-14-1" class="td11">
Acyl chain remodelling of PE
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-14-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-14-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7307
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-15-1" class="td11">
Signalling by GPCR
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-15-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-15-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9939
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-16-1" class="td11">
<strong><span class="cmbx-12">Molecules associated with elastic fibres</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-16-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-16-3" class="td11">
0<span class="cmmi-12">.</span>0072929
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-17-1" class="td11">
DAP12 interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-17-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-17-3" class="td11">
0<span class="cmmi-12">.</span>78273
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-18-1" class="td11">
Cytochrome P<sub><span class="cmr-8">450</span></sub> - arranged by substrate type
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-18-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-18-3" class="td11">
0<span class="cmmi-12">.</span>87019
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-19-1" class="td11">
GPCR ligand binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-19-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-19-3" class="td11">
0<span class="cmmi-12">.</span>99417
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-20-1" class="td11">
Acyl chain remodelling of PC
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-20-2" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-20-3" class="td11">
0<span class="cmmi-12">.</span>65415
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-21-1" class="td11">
Response to elevated platelet cytosolic Ca<sup><span class="cmr-8">2+</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-21-2" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-21-3" class="td11">
0<span class="cmmi-12">.</span>55461
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-22-1" class="td11">
<span class="cmti-12">Arachidonic acid metabolism </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-22-2" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-22-3" class="td11">
0<span class="cmmi-12">.</span>060298
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-23-1" class="td11">
Defective B4GALT7 causes EDS, progeroid type
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-23-2" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-23-3" class="td11">
0<span class="cmmi-12">.</span>15497
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-24-1" class="td11">
Defective B3GAT3 causes JDSSDHD
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-24-2" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-24-3" class="td11">
0<span class="cmmi-12">.</span>15497
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-25-1" class="td11">
<strong><span class="cmbx-12">Elastic fibre formation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-25-2" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-25-3" class="td11">
0<span class="cmmi-12">.</span>0019227
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-26-1" class="td11">
<strong><span class="cmbx-12">HS-GAG degradation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-26-2" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-26-3" class="td11">
0<span class="cmmi-12">.</span>017747
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-27-1" class="td11">
Bile acid and bile salt metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-27-2" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-27-3" class="td11">
0<span class="cmmi-12">.</span>15497
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-28-1" class="td11">
Netrin-1 signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-28-2" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-28-3" class="td11">
0<span class="cmmi-12">.</span>95056
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-29-1" class="td11">
<strong><span class="cmbx-12">Integration of energy metabolism</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-29-2" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-29-3" class="td11">
0<span class="cmmi-12">.</span>0019287
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-30-1" class="td11">
DAP12 signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-30-2" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-30-3" class="td11">
0<span class="cmmi-12">.</span>67835
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-31-1" class="td11">
GPCR downstream signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-31-2" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-31-3" class="td11">
0<span class="cmmi-12">.</span>88678
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-32-1" class="td11">
<strong><span class="cmbx-12">Diseases associated with glycosaminoglycan metabolism</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-32-2" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-32-3" class="td11">
0<span class="cmmi-12">.</span>017747
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-33-1" class="td11">
<strong><span class="cmbx-12">Diseases of glycosylation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-33-2" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-33-3" class="td11">
0<span class="cmmi-12">.</span>017747
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-34-1" class="td11">
Signalling by Retinoic Acid
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-34-2" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-34-3" class="td11">
0<span class="cmmi-12">.</span>13592
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-35-1" class="td11">
Signalling by Leptin
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-35-2" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-35-3" class="td11">
0<span class="cmmi-12">.</span>15497
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-36-1" class="td11">
Signalling by SCF-KIT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-36-2" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-36-3" class="td11">
0<span class="cmmi-12">.</span>73399
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-37-1" class="td11">
Opioid Signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-37-2" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-37-3" class="td11">
0<span class="cmmi-12">.</span>99417
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-38-1" class="td11">
Signalling by NOTCH
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-38-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0001
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-38-3" class="td11">
0<span class="cmmi-12">.</span>26453
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-39-1" class="td11">
Platelet homeostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-39-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0001
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-39-3" class="td11">
0<span class="cmmi-12">.</span>55912
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-40-1" class="td11">
Signalling by NOTCH1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-40-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00011
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-40-3" class="td11">
0<span class="cmmi-12">.</span>13797
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-41-1" class="td11">
Class B/2 (Secretin family receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-41-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00011
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-41-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4659
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-42-1" class="td11">
Diseases of Immune System
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-42-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-42-3" class="td11">
0<span class="cmmi-12">.</span>15497
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-43-1" class="td11">
Diseases associated with the TLR signalling cascade
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-43-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-43-3" class="td11">
0<span class="cmmi-12">.</span>15497
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-44-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-44-1" class="td11">
A tetrasaccharide linker sequence is required for GAG synthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-44-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-44-3" class="td11">
0<span class="cmmi-12">.</span>33566
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-45-1" class="td11">
Nuclear Receptor transcription pathway
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-45-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00016
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-45-3" class="td11">
0<span class="cmmi-12">.</span>22735
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-46-1" class="td11">
<strong><span class="cmbx-12">Formation of Fibrin Clot (Clotting Cascade)</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-46-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00016
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-46-3" class="td11">
0<span class="cmmi-12">.</span>0054639
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-47-1" class="td11">
Syndecan interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-47-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00016
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-47-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3974
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-48-1" class="td11">
Class A/1 (Rhodopsin-like receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-48-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00016
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-48-3" class="td11">
0<span class="cmmi-12">.</span>99454
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-49-1" class="td11">
HS-GAG biosynthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-49-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0002
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-49-3" class="td11">
0<span class="cmmi-12">.</span>37199
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-50-1" class="td11">
Platelet degranulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-50-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0002
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-50-3" class="td11">
0<span class="cmmi-12">.</span>39003
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-63-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-63-51-1" class="td11">
EPH-ephrin mediated repulsion of cells
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-51-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00021
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-63-51-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6193
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-63-52-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-63-52-1"  
class="td11">
<pre><code>                                                   &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 963-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Over-representation (hypergeometric test) and Permutation p-values adjusted for multiple tests across pathways</span> <span class="cmr-10x-x-109">(FDR). Significant pathways are marked in bold (FDR </span><span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05) and italics (FDR </span><span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">1).</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 966-->
<p class="indent">
Many of the pathways supported in the intersection by permutation analysis were also replicated in the mtSLIPT anlaysis of partners tested with <span class="cmti-12">CDH1 </span>mutation (in Table <a 
href="#x1-208002r6">D.6<!--tex4ht:ref: tab:pathway_perm_overlap_mtSL --></a>), including G<sub><span class="cmmi-8">αs</span></sub>, elastic fibres, HS-GAG, and energy metabolism. While there were differences between the pathways Identified by over-representation anlaysis, those replicated by permutation were highly concordant supported the combined use of these pathways approaches to identify synthetic lethal gene functions and targets.
</p>
<!--l. 968-->
<p class="indent">
While this indicates that G<sub><span class="cmmi-8">αs</span></sub> and GPCR class A/1 signalling events were significantly detected by both approaches, GPCR signalling pathways overall were not. It is likely that GPCRs were primarily over-represented in the intersection with the experimental candidates due to strong over-representation of these pathways in experimental candidates, rather than detection by <a 
href="#glo:SLIPT">SLIPT</a>, which may be driven by these more specific constituent pathways.
</p>
<!--l. 971-->
<p class="indent">
However, several pathways, including some immune functions and neurotransmitters, were supported by the resampling analysis (in Tables <a 
href="#x1-128003r6">4.6<!--tex4ht:ref: tab:pathway_perm_overlap --></a> and <a 
href="#x1-208002r6">D.6<!--tex4ht:ref: tab:pathway_perm_overlap_mtSL --></a>) when the initial pathway over-representation test was not significant. These functions appear to have been detected by both approaches more than expected by chance but must be interpreted with caution since they were still not common enough to be detected in pathway over-representation analysis.
</p>
<!--l. 973-->
<p class="indent">
Therefore computational and experimental approaches to synthetic lethal screening for <span class="cmti-12">CDH1 </span>lead to a broader functional characterisation and many candidate partners, when combined, despite different strengths and limitations. Compared to candidate gene approaches, experimental genome-wide screens are an appealing unbiased strategy for identifying synthetic lethal interactions. Since these screens are costly, laborious, and subject to genetic background, computational analysis can augment candidate triage to either reduce the initial panel of screened genes or prioritise validation.
</p>
<!--l. 975-->
<p class="indent">
<p>GPCR pathways were detected among both computational and experimental synthetic lethal candidates, with more support in the experimental screen (Table <a 
href="#x1-128003r6">4.6<!--tex4ht:ref: tab:pathway_perm_overlap --></a>). The homogeneous cell line model may be more likely to detect particular pathways. For instance, <a 
href="#glo:SLIPT">SLIPT</a> identified immune pathways, not expected to be detected in isolated cell culture. GPCR signalling was supported in experimental models <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>) with some of these pathways replicated in varied genetic</p>
backgrounds of patient samples. These pathways require further investigation such as identification of more specific pathways, higher order interactions, and modes of resistance.
</p>
<!--l. 977-->
<p class="indent">
The pathway composition across computational and experimental synthetic lethal candidates was informative with over-representation (Table <a 
href="#x1-127001r4">4.4<!--tex4ht:ref: tab:Venn_over-representation --></a>) and supported by resampling analysis (Table <a 
href="#x1-128003r6">4.6<!--tex4ht:ref: tab:pathway_perm_overlap --></a>), despite a modest intersection of genes between them (Figure <a 
href="#x1-123001r2">4.2<!--tex4ht:ref: fig:Venn_allgenes --></a>). Either approach may be significant for a pathway in this intersection without being supported by the other: resampling analysis may support pathways that were not over-represented due to small effect sizes, thus both tests are required for a candidate pathway. The pathways detected by both over-representation and resampling are the strongest candidates for further investigation, such as G<sub><span class="cmmi-8">αs</span></sub> signalling, a strong candidate in prior analyses with a role in the regulation of translation in cancer <a 
href="#XGao2015">Gao and Roux</a> (<a 
href="#XGao2015">2015</a>), another function supported by <a 
href="#glo:SLIPT">SLIPT</a> analysis.
</p>
<!--l. 982-->
<p class="indent">
The predicted synthetic lethal partners occurred across functionally distinct pathways, including characterised functions of <span class="cmti-12">CDH1</span>. This diversity is consistent with the wide ranging role of <span class="cmti-12">CDH1 </span>in cell-cell adhesion, cell signalling, and the cytoskeletal structure of epithelial tissues. Pathway structure may be relevant to identifying potential drug targets from gene expression signatures, indicating downstream effector genes and mechanisms leading to cell inviability. These distinct synthetic lethal gene clusters and pathways may further lead to the elucidation of drug resistance mechanisms. <a 
 id="x1-128004r166"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">4.3 </span> <a 
 id="x1-1290003"></a>Metagene Analysis
</h3>
<!--l. 1013-->
<p class="noindent">
<p>Metagenes serve as a consistent signal of pathway activity. The direction of metagenes is generally arbitrary but care has been taken to ensure that these occur in a direction which reflect overall activation of the pathway (as described in Section <a 
href="#x1-730003">2.2.3<!--tex4ht:ref: methods:metagene --></a>). This will be supported by examining the pathway expression of gene signatures in breast cancer to ensure they behave as expected in TCGA expression data. These metagenes were also compared to somatic mutation to show the limitations of mutation as a measure of gene activity. Having established that metagenes generated with this procedure reflect gene activity, these were then applied to the Reactome pathways for synthetic lethal analysis of pathways directly to provide an alternative approach to identifying synthetic lethal pathways with <span class="cmti-12">CDH1</span>. <a 
 id="x1-129001r167"></a></p>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.3.1 </span> <a 
 id="x1-1300001"></a>Pathway expression
</h4>
<!--l. 1017-->
<p class="noindent">
<p>Pathway metagenes (generated as described in Section <a 
href="#x1-730003">2.2.3<!--tex4ht:ref: methods:metagene --></a>) for gene signatures of key processes in breast cancer (<a 
href="#XGatza2011">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2011">2011</a>) were used to check that metagenes were generated in the correct direction to indicate pathway activation. These gene signatures were plotting in Figure <a 
href="#x1-130001r9">4.9<!--tex4ht:ref: fig:metagene_expr_Gatza2011 --></a> for comparison with clinical factors and somatic mutations. The “intrinsic subtype” was computed by performing the PAM50 procedure <a 
href="#XParker2009">Parker <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XParker2009">2009</a>) for RNASeq data which was highly concordant with the subtypes provided by UCSC for TCGA samples previously analysed by microarrays (<a 
href="#XTCGA2012">TCGA</a>, <a 
href="#XTCGA2012">2012</a>). Somatic mutations were reported for recurrently mutated genes in breast cancer, as reported by TCGA (<a 
href="#XTCGA2012">TCGA</a>, <a 
href="#XTCGA2012">2012</a>), related genes, and those previously discussed to be important in hereditary breast cancers (<span class="cmti-12">BRCA1</span>, <span class="cmti-12">BRCA2</span>, and <span class="cmti-12">CDH1 </span>).</p>
</p>
<!--l. 1019-->
<p class="indent">
<a 
 id="x1-130001r9"></a>
</p>
<hr class="float" />
<div class="float">
<span class="fbox">
<div class="minipage">
<div class="center">
<!--l. 1032-->
<p class="noindent">
</p>
<!--l. 1032-->
<p class="noindent">
<img 
src="CDH1_Heatmaps_Gatza2011(ranked)_Full_split_by_Subtype_and_CDH1_Stat_corr.png" alt="PIC"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 4.9: </span><span class="content"><strong><span class="cmbx-10x-x-109">Pathway metagene expression profiles.</span></strong> <span class="cmr-10x-x-109">Expression profiles for metagene</span> <span class="cmr-10x-x-109">signatures from </span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">Gatza </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">2011</span></a><span class="cmr-10x-x-109">) in TCGA breast data, annotated for clinical factors and</span> <span class="cmr-10x-x-109">cancer gene mutations. Samples were clustered independently for each intrinsic subtype and</span> <span class="cmr-10x-x-109">by </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">expression status. </span></span></p>
</div>
<!--tex4ht:label?: x1-130001r9 -->
</div>
<p></span></p>
</div>
<hr class="endfloat" />
<!--l. 1035-->
<p class="indent">
These gene signatures reflect intrinsic subtypes as expected. In particular, the estrogen and progesterone receptor signatures are low in the ER<span class="cmsy-10x-x-120">- </span>and PR<span class="cmsy-10x-x-120">- </span>basal subtype tumours. These tumours also had the highest frequency of <span class="cmti-12">TP53 </span>mutations and a corresponding reduction of p53 metagene activity, as expected for loss of a tumour suppressor. The luminal A and luminal B tumour subtypes are the most similar, which is reflected in these metagenes signatures, although they are distinguishable molecular subtypes as shown by elevated PI3K, AKT, RAS, and <span class="cmmi-12">β</span>-catenin signalling in luminal B tumours. Although, these pathways were also elevated in Basal and HER2-enriched subtypes and lowly expressed in the “normal-like” subtype (which contained the normal samples). These intrinsic subtype specific gene signature profiles were further supported with metagenes for an extended set of signatures (<a 
href="#XGatza2014">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2014">2014</a>), as shown in Figure <a 
href="#x1-214001r1">E.1<!--tex4ht:ref: fig:metagene_expr_Gatza2014 --></a>.
</p>
<!--l. 1037-->
<p class="indent">
<span class="cmti-12">TP53 </span>mutations were the most frequent and more common in basal subtype. Similarly, <span class="cmti-12">GATA3 </span>mutations were more common in luminal subtype tumours. PI3K mutations were more frequent across breast tumours, although these were less common in the basal subtype despite an elevated metagene (this discrepancy will the discussed further in Section <a 
href="#x1-1310002">4.3.2<!--tex4ht:ref: chapt3:metagene_mut --></a>). <span class="cmti-12">CDH1 </span>mutations similarly occurred across molecular subtypes with the exception of the basal subtype (as observed in gene expression with Figure <a 
href="#x1-120001r1">4.1<!--tex4ht:ref: fig:slipt_expr --></a>). <span class="cmti-12">CDH1 </span>low samples occurred in all subtypes but were predominantly lobular subtype. Apart from these genes, mutations did not show clear specficity to a particular subtype and the variation between samples was reflects the range of molecular cascades that can result in tumours with similar molecular profiles, supporting the use of expression for cancer diagnostics and identification of molecular targets.
</p>
<!--l. 1039-->
<p class="indent">
<p>The direction of the metagenes were also consistent with the clincal characteristics and formed a consensus of gene activity as shown in Figures <a 
href="#x1-214002r2">E.2<!--tex4ht:ref: fig:metagene_expr_Gatza2011_PI3K --></a>–<a 
href="#x1-214005r5">E.5<!--tex4ht:ref: fig:metagene_expr_Gatza2014_BRCA --></a>. In each of the examples for gene signatures for PI3K (Figure <a 
href="#x1-214002r2">E.2<!--tex4ht:ref: fig:metagene_expr_Gatza2011_PI3K --></a>), p53 (Figure <a 
href="#x1-214003r3">E.3<!--tex4ht:ref: fig:metagene_expr_Gatza2011_P53 --></a>), estrogen receptor (Figure <a 
href="#x1-214004r4">E.4<!--tex4ht:ref: fig:metagene_expr_Gatza2011_ER --></a>), and BRCA (Figure <a 
href="#x1-214005r5">E.5<!--tex4ht:ref: fig:metagene_expr_Gatza2014_BRCA --></a>) genes (<a 
href="#XGatza2011">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2011">2011</a>, <a 
href="#XGatza2014">2014</a>), the expression of the majority of the genes were highly concordant with the metagene, being either positively or negatively correlated. These were generally consistent with established clinical and molecular subtypes of breast cancer and the recurrent mutations shown. However, the <span class="cmti-12">PIK3CA </span>and <span class="cmti-12">PIK3R1</span> mutant samples did not necessarily have elevated PI3K pathway metagene activity (as shown in Figure <a 
href="#x1-214002r2">E.2<!--tex4ht:ref: fig:metagene_expr_Gatza2011_PI3K --></a>). <a 
 id="x1-130002r184"></a></p>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.3.2 </span> <a 
 id="x1-1310002"></a>Somatic mutation
</h4>
<!--l. 1045-->
<p class="indent">
<a 
 id="x1-131005r10"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 1047-->
<p class="noindent">
<a 
 id="x1-131001r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="metagene(ranked)_vioplotx_Mutation_PI3K2011_PIK3CA.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmti-10">PIK3CA</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-131002r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="metagene(ranked)_vioplotx_Mutation_PI3K2011_PIK3CA_PIK3R1.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmti-10">PIK3CA</span> <span class="cmr-10">or</span> <span class="cmti-10">PIK3R1</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-131003r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="metagene(ranked)_vioplotx_Mutation_PI3K2011_CDH1.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmti-10">CDH1</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-131004r4"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="metagene(ranked)_vioplotx_Mutation_PI3K2011_TP53.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(d)</span> <span class="cmti-10">TP53</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 4.10: </span><span class="content"><strong><span class="cmbx-10x-x-109">Somatic mutation against PI3K metagene.</span></strong> <span class="cmr-10x-x-109">Mutations in </span><span class="cmti-10x-x-109">PIK3CA</span><span class="cmr-10x-x-109">,</span> <span class="cmti-10x-x-109">PIK3R1</span><span class="cmr-10x-x-109">, </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">, and </span><span class="cmti-10x-x-109">TP53 </span><span class="cmr-10x-x-109">were examined in TCGA breast cancer for their effect on</span> <span class="cmr-10x-x-109">the PI3K (</span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">Gatza </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109">,</span><span class="cmr-10x-x-109"> </span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">2011</span></a><span class="cmr-10x-x-109">) pathway metagene. The tumour suppressors </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">and</span> <span class="cmti-10x-x-109">TP53 </span><span class="cmr-10x-x-109">showed an increase and decrease in the metagene respectively, whereas </span><span class="cmti-10x-x-109">PIK3CA</span> <span class="cmr-10x-x-109">and </span><span class="cmti-10x-x-109">PIK3R1 </span><span class="cmr-10x-x-109">mutations had little effect on the metagene levels. </span></span></p>
</div>
<!--tex4ht:label?: x1-131005r10 -->
</div>
<hr class="endfloat" />
<!--l. 1069-->
<p class="noindent">
It should be noted that metagenes, while consistent with the consensus of constituent expressed genes, were not neccessarily reflecting the somatic mutation status. The PI3K (<a 
href="#XGatza2011">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2011">2011</a>) metagene levels in particular, were not statistically significant in between mutant and wildtype <span class="cmti-12">PIK3CA </span>samples (shown in Figure <a 
href="#x1-131005r10">4.10<!--tex4ht:ref: fig:mutation_expr_mg --></a>). Although the PI3K metagene differed across <span class="cmti-12">CDH1 </span>and <span class="cmti-12">TP53 </span>mutations, remarkably in opposite directions considering that PI3K is an oncogenic growth pathway and these are both most frequently tumour suppressors inactivated in cancers. This shows that <span class="cmti-12">CDH1</span> and <span class="cmti-12">TP53 </span>deficient tumours have distinct molecular growth pathways and that synthetic lethal inactivations against <span class="cmti-12">CDH1 </span>inactivation may not be applicable to other cancers with driver mutations such as <span class="cmti-12">TP53</span>, although these were kept in the analysis for comparison. These differences may be related to these mutations being more frequent in tumours with difference clinical characteristics (as observed in Section <a 
href="#x1-1300001">4.3.1<!--tex4ht:ref: chapt3:metagene_expression --></a>). Thus mutations do not neccessarily have corresponding changes in pathway expression, particularly for oncogenes which may change in function rather than being upregulated.
</p>
<!--l. 1072-->
<p class="indent">
While the more specific <span class="cmti-12">PIK3CA </span>(<a 
href="#XGatza2014">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2014">2014</a>) metagene showed significant differences with <span class="cmti-12">PIK3CA </span>and <span class="cmti-12">PIK3R1 </span>mutations (as shown in Figure <a 
href="#x1-213005r7">D.7<!--tex4ht:ref: fig:mutation_expr_mg2 --></a>), this metagene replicated stronger differences for <span class="cmti-12">CDH1 </span>and <span class="cmti-12">TP53</span>. These differences were less pronounced in the protein levels of p110<span class="cmmi-12">α </span>(enocded by <span class="cmti-12">PIK3CA</span>) and the downstream AKT gene (shown in Figures <a 
href="#x1-213010r8">D.8<!--tex4ht:ref: fig:mutation_expr_prot --></a> and <a 
href="#x1-213015r9">D.9<!--tex4ht:ref: fig:mutation_expr_prot2 --></a> respectively). Although this may be due to this regulatory cascade (kinases) being transmitted as a change in protein state (phosphorylation) rather than changes in expression levels. Another consideration is that mutations at different loci have different effects on protein function, particularly for oncogenes. <a 
 id="x1-131006r186"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.3.3 </span> <a 
 id="x1-1320003"></a>Mutation locus
</h4>
<!--l. 1078-->
<p class="noindent">
<p>The gene locus distribution of <span class="cmti-12">PIK3CA </span>and it’s receptor <span class="cmti-12">PIK3R1 </span>were consistent with oncogenic and tumour suppressor mutations, as shown in Figure <a 
href="#x1-212005r6">D.6<!--tex4ht:ref: fig:mutation_locus --></a>. <span class="cmti-12">PIK3CA </span>is an has recurrent mutations in 2 hotspots, centered around the E545K and H1047R (shown in Figure <a 
href="#x1-212001r1">D.6(a)<!--tex4ht:ref: fig:mutation_locus:PIK3CA --></a>), as expected for an oncogene. This contrasts with the tumour suppressors, <span class="cmti-12">PIK3R1</span>, and <span class="cmti-12">CDH1 </span>(shown in Figures <a 
href="#x1-212002r2">D.6(b)<!--tex4ht:ref: fig:mutation_locus:PIK3R1 --></a> and <a 
href="#x1-212003r3">D.6(c)<!--tex4ht:ref: fig:mutation_locus:CDH1 --></a> respectively), which have low frequency inactivating mutations spread across them. A notable exception is <span class="cmti-12">TP53 </span>(shown in Figure <a 
href="#x1-212004r4">D.6(d)<!--tex4ht:ref: fig:mutation_locus:TP53 --></a>) which displays both inactivating mutations throughout and</p>
<p>recurrent (oncogenic) mutations at high frequency, consistent with the complex role of <span class="cmti-12">TP53 </span>in cancer biology which is outside of the scope of this thesis and shown for comparison.</p>
</p>
<!--l. 1081-->
<p class="indent">
<a 
 id="x1-132008r11"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 1083-->
<p class="noindent">
<a 
 id="x1-132001r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="geneexpr_vioplotx_Mutation_locus_PIK3CA.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmti-10">PI3KCA</span> <span class="cmr-10">gene</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-132002r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="metagene_vioplotx_Mutation_locus_PIK3CA.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmti-10">PI3KCA</span> <span class="cmr-10">metagene</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-132003r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="geneexpr_vioplotx_Mutation_locus_PIK3R1.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmti-10">PIK3R1</span> <span class="cmr-10">gene</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-132004r4"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="protein_vioplotx_Mutation_locus_PIK3CA.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(d)</span> <span class="cmti-10">PIK3CA</span> <span class="cmr-10">protein</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-132005r5"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="protein_vioplotx_Mutation_locus_AKT.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(e)</span> <span class="cmti-10">AKT</span> <span class="cmr-10">protein</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-132006r6"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="protein_vioplotx_Mutation_locus_CDH1.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(f)</span> <span class="cmti-10">CDH1</span> <span class="cmr-10">protein</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-132007r7"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="protein_vioplotx_Mutation_locus_TP53.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(g)</span> <span class="cmti-10">TP53</span> <span class="cmr-10">protein</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 4.11: </span><span class="content"><strong><span class="cmbx-10x-x-109">Somatic mutation locus against expression.</span></strong> <span class="cmr-10x-x-109">The recurrent E545K</span> <span class="cmr-10x-x-109">and H1047R oncogene mutations in </span><span class="cmti-10x-x-109">PIK3CA </span><span class="cmr-10x-x-109">were examined in TCGA breast cancer to</span> <span class="cmr-10x-x-109">show the effect of mutation locus on gene, pathway, and protein expression. While neither</span> <span class="cmr-10x-x-109">of these mutations had an impact of </span><span class="cmti-10x-x-109">PIK3CA </span><span class="cmr-10x-x-109">mRNA expression, E545K had specifically</span> <span class="cmr-10x-x-109">lower PI3K (</span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">Gatza </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109">,</span><span class="cmr-10x-x-109"> </span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">2011</span></a><span class="cmr-10x-x-109">) metagene levels and both mutations had higher </span><span class="cmti-10x-x-109">PIK3R1</span> <span class="cmr-10x-x-109">mRNA expression. However, these differences were not reflected in the protein expression</span> <span class="cmr-10x-x-109">levels. </span></span></p>
</div>
<!--tex4ht:label?: x1-132008r11 -->
</div>
<hr class="endfloat" />
<!--l. 1117-->
<p class="indent">
These differences in gene locus may explain why mutations do not necessarily have corresponding changes in gene or metagene expression. Specfically, the recurrent E545K and H1047R oncogene mutations in <span class="cmti-12">PIK3CA </span>did not affect <span class="cmti-12">PIK3CA </span>mRNA expression but E545K had specifically lower PI3K (<a 
href="#XGatza2011">Gatza <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XGatza2011">2011</a>) metagene levels. Both mutations had higher <span class="cmti-12">PIK3R1 </span>mRNA expression but these differences differences were not reflected in the protein expression levels of p110<span class="cmmi-12">α </span>protein (encoded by <span class="cmti-12">PIK3CA</span>), it’s downstream target AKT, E-cadherin (encoded by CDH1), or p53 (as ashown in Figure <a 
href="#x1-132008r11">4.11<!--tex4ht:ref: fig:mutation_expr --></a>).
</p>
<!--l. 1119-->
<p class="indent">
While the complex effects of mutation in oncogenes such as <span class="cmti-12">PIK3CA </span>are not neccessarily detected in a pathway metagene, these do capture the consensus of patheay gene expression and account for other potential means of pathway activation. Thus metagenes are sufficient as a measure of gene activity for the purposes of synthetic lethal detection with <a 
href="#glo:SLIPT">SLIPT</a>. This approach is more applicable to tumour suppressor genes with a relationship between gene expression and activity (rather than activation at the protein level) but this is not a major concern since synthetic lethality is more clinically relevant for targeting tumour suppressor mutations than oncogenes. <a 
 id="x1-132009r188"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.3.4 </span> <a 
 id="x1-1330004"></a>Synthetic lethal metagenes
</h4>
<!--l. 1125-->
<p class="noindent">
<p>Pathway metagenes for Reactome pathways (generated as described in Section <a 
href="#x1-730003">2.2.3<!--tex4ht:ref: methods:metagene --></a>) were also used for testing synthetic lethal partner pathways with <span class="cmti-12">CDH1 </span>by <a 
href="#glo:SLIPT">SLIPT</a>. Since the metagenes have are higher when the pathway as a whole is activated, they are amenable to <a 
href="#glo:SLIPT">SLIPT</a> analysis using low metagene levels for inactivated pathways. These synthetic lethal metagenes differed to the over-represented pathways among synthetic lethal gene candidates. However, there were some similarities to previous findings, as shown in Tables <a 
href="#x1-133001r7">4.7<!--tex4ht:ref: tab:metagene_SL --></a>. In particular, translational pathways were replicated as observed in Table <a 
href="#x1-119001r2">4.2<!--tex4ht:ref: tab:pathway_exprSL --></a>. While the specific pathways differ, immune pathways (such as NF-<span class="cmmi-12">κ</span>B) were also supported by metagene synthetic lethal analysis.</p>
</p>
<!--l. 1127-->
<p class="indent">
<a 
 id="x1-133001r7"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 4.7: </span><span class="content">Candidate synthetic lethal metagenes against <span class="cmti-12">CDH1 </span>from SLIPT</span></p>
</div>
<!--tex4ht:label?: x1-133001r7 -->
<div class="tabular">
<table id="TBL-64" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-64-1g">
<col 
id="TBL-64-1" />
<col 
id="TBL-64-2" />
<col 
id="TBL-64-3" />
<col 
id="TBL-64-4" />
<col 
id="TBL-64-5" />
<col 
id="TBL-64-6" />
<col 
id="TBL-64-7" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-64-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-1-1" class="td11">
<span class="cmbx-12">Pathway </span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-1-2" class="td11">
ID
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-1-3" class="td11">
Observed
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-1-4" class="td11">
Expected
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-1-5" class="td11">
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-1-6" class="td11">
p-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-1-7" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-2-1" class="td11">
Glycogen storage diseases
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-2-2" class="td11">
3229121
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-2-3" class="td11">
68
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-2-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-2-5" class="td11">
176
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-2-6" class="td11">
6<span class="cmmi-12">.</span>62 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">37</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-2-7" class="td11">
1<span class="cmmi-12">.</span>82 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">34</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-3-1" class="td11">
Myoclonic epilepsy of Lafora
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-3-2" class="td11">
3785653
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-3-3" class="td11">
68
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-3-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-3-5" class="td11">
176
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-3-6" class="td11">
6<span class="cmmi-12">.</span>62 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">37</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-3-7" class="td11">
1<span class="cmmi-12">.</span>82 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">34</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-4-1" class="td11">
Diseases of carbohydrate metabolism
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-4-2" class="td11">
5663084
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-4-3" class="td11">
68
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-4-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-4-5" class="td11">
176
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-4-6" class="td11">
6<span class="cmmi-12">.</span>62 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">37</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-4-7" class="td11">
1<span class="cmmi-12">.</span>82 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">34</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-5-1" class="td11">
Arachidonic acid metabolism
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-5-2" class="td11">
2142753
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-5-3" class="td11">
81
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-5-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-5-5" class="td11">
157
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-5-6" class="td11">
8<span class="cmmi-12">.</span>13 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">33</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-5-7" class="td11">
1<span class="cmmi-12">.</span>49 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-6-1" class="td11">
Translation initiation complex formation
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-6-2" class="td11">
72649
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-6-3" class="td11">
70
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-6-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-6-5" class="td11">
152
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-6-6" class="td11">
7<span class="cmmi-12">.</span>08 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">32</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-6-7" class="td11">
1<span class="cmmi-12">.</span>17 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-7-1" class="td11">
Synthesis of 5-eicosatetraenoic acids
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-7-2" class="td11">
2142688
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-7-3" class="td11">
68
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-7-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-7-5" class="td11">
151
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-7-6" class="td11">
1<span class="cmmi-12">.</span>25 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-7-7" class="td11">
1<span class="cmmi-12">.</span>88 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-8-1" class="td11">
SRP-dependent cotranslational protein targeting to membrane
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-8-2" class="td11">
1799339
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-8-3" class="td11">
69
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-8-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-8-5" class="td11">
150
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-8-6" class="td11">
2<span class="cmmi-12">.</span>01 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-8-7" class="td11">
2<span class="cmmi-12">.</span>76 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-9-1" class="td11">
L13a-mediated translational silencing of Ceruloplasmin expression
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-9-2" class="td11">
156827
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-9-3" class="td11">
72
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-9-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-9-5" class="td11">
148
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-9-6" class="td11">
5<span class="cmmi-12">.</span>91 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-9-7" class="td11">
6<span class="cmmi-12">.</span>44 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-10-1" class="td11">
3’ -UTR-mediated translational regulation
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-10-2" class="td11">
157279
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-10-3" class="td11">
72
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-10-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-10-5" class="td11">
148
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-10-6" class="td11">
5<span class="cmmi-12">.</span>91 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-10-7" class="td11">
6<span class="cmmi-12">.</span>44 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-64-11-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-64-11-1"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-65" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-65-1g">
<col 
id="TBL-65-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-65-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-65-1-1" class="td00">
Activation of the mRNA upon binding of the cap-binding complex and eIFs,
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-65-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-65-2-1" class="td00">
and subsequent binding to 43S
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-11-2" class="td11">
72662
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-11-3" class="td11">
70
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-11-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-11-5" class="td11">
147
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-11-6" class="td11">
1<span class="cmmi-12">.</span>14 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-11-7" class="td11">
9<span class="cmmi-12">.</span>28 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-12-1" class="td11">
Formation of the ternary complex, and subsequently, the 43S complex
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-12-2" class="td11">
72695
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-12-3" class="td11">
70
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-12-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-12-5" class="td11">
147
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-12-6" class="td11">
1<span class="cmmi-12">.</span>14 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-12-7" class="td11">
9<span class="cmmi-12">.</span>28 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-13-1" class="td11">
Ribosomal scanning and start codon recognition
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-13-2" class="td11">
72702
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-13-3" class="td11">
70
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-13-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-13-5" class="td11">
147
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-13-6" class="td11">
1<span class="cmmi-12">.</span>14 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-13-7" class="td11">
9<span class="cmmi-12">.</span>28 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-14-1" class="td11">
Eukaryotic Translation Elongation
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-14-2" class="td11">
156842
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-14-3" class="td11">
72
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-14-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-14-5" class="td11">
146
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-14-6" class="td11">
1<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-14-7" class="td11">
9<span class="cmmi-12">.</span>28 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-15-1" class="td11">
Nonsense Mediated Decay independent of the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-15-2" class="td11">
975956
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-15-3" class="td11">
71
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-15-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-15-5" class="td11">
146
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-15-6" class="td11">
1<span class="cmmi-12">.</span>24 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-15-7" class="td11">
9<span class="cmmi-12">.</span>28 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">29</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-16-1" class="td11">
Viral mRNA Translation
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-16-2" class="td11">
192823
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-16-3" class="td11">
70
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-16-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-16-5" class="td11">
146
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-16-6" class="td11">
1<span class="cmmi-12">.</span>51 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-16-7" class="td11">
1<span class="cmmi-12">.</span>04 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-17-1" class="td11">
Eukaryotic Translation Termination
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-17-2" class="td11">
72764
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-17-3" class="td11">
70
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-17-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-17-5" class="td11">
146
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-17-6" class="td11">
1<span class="cmmi-12">.</span>51 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-17-7" class="td11">
1<span class="cmmi-12">.</span>04 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-18-1" class="td11">
NF-kB is activated and signals survival
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-18-2" class="td11">
209560
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-18-3" class="td11">
71
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-18-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-18-5" class="td11">
145
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-18-6" class="td11">
1<span class="cmmi-12">.</span>90 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-18-7" class="td11">
1<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-19-1" class="td11">
Peptide chain elongation
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-19-2" class="td11">
156902
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-19-3" class="td11">
72
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-19-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-19-5" class="td11">
145
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-19-6" class="td11">
1<span class="cmmi-12">.</span>91 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-19-7" class="td11">
1<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-20-1" class="td11">
Influenza Life Cycle
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-20-2" class="td11">
168255
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-20-3" class="td11">
70
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-20-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-20-5" class="td11">
145
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-20-6" class="td11">
1<span class="cmmi-12">.</span>95 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-20-7" class="td11">
1<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-21-1" class="td11">
Formation of a pool of free 40S subunits
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-21-2" class="td11">
72689
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-21-3" class="td11">
73
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-21-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-21-5" class="td11">
145
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-21-6" class="td11">
2<span class="cmmi-12">.</span>01 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-21-7" class="td11">
1<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-22-1" class="td11">
Nonsense-Mediated Decay
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-22-2" class="td11">
927802
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-22-3" class="td11">
71
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-22-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-22-5" class="td11">
145
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-22-6" class="td11">
2<span class="cmmi-12">.</span>44 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-22-7" class="td11">
1<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-23-1" class="td11">
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-23-2" class="td11">
975957
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-23-3" class="td11">
71
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-23-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-23-5" class="td11">
145
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-23-6" class="td11">
2<span class="cmmi-12">.</span>44 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-23-7" class="td11">
1<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-24-1" class="td11">
GTP hydrolysis and joining of the 60S ribosomal subunit
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-24-2" class="td11">
72706
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-24-3" class="td11">
72
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-24-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-24-5" class="td11">
145
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-24-6" class="td11">
2<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-24-7" class="td11">
1<span class="cmmi-12">.</span>37 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-25-1" class="td11">
Influenza Viral RNA Transcription and Replication
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-25-2" class="td11">
168273
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-25-3" class="td11">
72
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-25-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-25-5" class="td11">
144
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-25-6" class="td11">
4<span class="cmmi-12">.</span>01 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-25-7" class="td11">
2<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-64-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-64-26-1" class="td11">
Signaling by NOTCH1 HD Domain Mutants in Cancer
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-64-26-2" class="td11">
2691230
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-26-3" class="td11">
79
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-26-4" class="td11">
130
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-26-5" class="td11">
143
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-26-6" class="td11">
5<span class="cmmi-12">.</span>99 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-64-26-7" class="td11">
2<span class="cmmi-12">.</span>82 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-64-27-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-64-27-1"  
class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 1194-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Strongest candidate SL partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">with observed and expected samples with low expression of both genes</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 1197-->
<p class="indent">
Signalling pathways were more strongly supported by mtSLIPT analysis of metagene pathway expression against <span class="cmti-12">CDH1 </span>mutation, as shown in Table <a 
href="#x1-210001r7">D.7<!--tex4ht:ref: tab:metagene_mtSL --></a>. Although these results were generally less statistically significant than expression analyses. Signalling pathways detected as synthetic lethal metagenes include G˙<span class="cmmi-12">αz</span>, insulin-related growth factor (IGF), GABA receptor, G˙<span class="cmmi-12">αs</span>, S6K1 and various toxin responses mediated by GPCRs. Metabolic processes including processing of carbohydrates and fatty acids were also implicated across these analyses.
</p>
<!--l. 1199-->
<p class="indent">
The metagene analyses differ more between expresssion and <span class="cmti-12">CDH1 </span>mutation than previous analyses, with more specific signalling pathways identified in the mutation analysis. This supports the usage of a complete null mutant model in experimental testing for synthetic lethality of signalling pathways against CDH1 inactivation rather than a knockdown in expression. However, low expression of partners has been used in either case to be applicable to dose-dependent pharmacological inhibition and across genes where mutations have different functional consequences, including variants of unknown significance.
</p>
<!--l. 1201-->
<p class="indent">
These results show an independent pathway approach to detecting synthetic lethal gene functions interacting with <span class="cmti-12">CDH1</span>. Synthetic lethal metagenes, replicates support for these pathways independent of pathway size (as genes are weighted equally). The synthetic lethal analysis against low <span class="cmti-12">CDH1 </span>expression support prior findings in translational and immune pathways even if they were not able to detected in an experimental screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). Together these findings support the role of <span class="cmti-12">CDH1</span> loss in cancer disrupting cell signalling with wider effects on protein translation and metabolism necessary for the proliferation of cancer cells. This is consistent with the GPCR pathways such as G˙<span class="cmmi-12">αs</span> signalling being supported by <a 
href="#glo:SLIPT">SLIPT</a> gene candidates and the experimental primary siRNA screen, as shown by resampling in Section <a 
href="#x1-1280001">4.2.1.4.1<!--tex4ht:ref: chapt3:compare_pathway_perm --></a>. <a 
 id="x1-133002r183"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">4.4 </span> <a 
 id="x1-1340004"></a>Replication in stomach cancer
</h3>
<!--l. 1226-->
<p class="noindent">
<p>The synthetic lethal analysis of genes and pathways (previously described for TCGA breast cancer data) was replicated in TCGA stomach cancer. The accompanying data for <a 
href="#glo:SLIPT">SLIPT</a> and mtSLIPT analyses against <span class="cmti-12">CDH1 </span>expression and mutation are in Appendices <a 
href="#x1-215000F">F<!--tex4ht:ref: appendix:stad_exprSL --></a> and <a 
href="#x1-220000G">G<!--tex4ht:ref: appendix:stad_mtSL --></a> respectively.</p>
</p>
<!--l. 1228-->
<p class="indent">
The experimental screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) was conducted in MCF10A breast cells so it may not be as comparable to stomach cancer. Nevertheless, <span class="cmti-12">CDH1 </span>is also important in stomach cancer biology as a driver tumour suppressor gene and as a germline mutation in many cases of hereditary diffuse gastric cancer.
</p>
<!--l. 1230-->
<p class="indent">
While the sample size was lower for TCGA stomach cancer (particularly for mutations), these results serve to support the findings in breast cancer in an independent patient cohort and tissue samples. The molecular profiling, including RNA-Seq expression, were performed by TCGA using the sample procedures as for breast cancer and the findings reported here were performed used data analysis techniques identical to those presented previously. These procedures should ensure as close comparison as feasible across cancer types for those relevant to HDGC and recurrent <span class="cmti-12">CDH1</span> mutations. <a 
 id="x1-134001r190"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.4.1 </span> <a 
 id="x1-1350001"></a>Synthetic Lethal Genes and Pathways
</h4>
<!--l. 1234-->
<p class="noindent">
The strongest <a 
href="#glo:SLIPT">SLIPT</a> genes for stomach cancer (shown in Table <a 
href="#x1-216001r1">F.1<!--tex4ht:ref: tab:gene_stad_SL --></a>) did not neccessarily directly correspond to those observed in breast cancer (shown in Table <a 
href="#x1-118001r1">4.1<!--tex4ht:ref: tab:gene_SL --></a>). However, several gene functions were replicated in stomach cancer. Cell membrane genes including <span class="cmti-12">EMP3</span>, <span class="cmti-12">GYPC</span>, <span class="cmti-12">LGALS1</span>, <span class="cmti-12">PRR24</span>, and <span class="cmti-12">FUNCD2 </span>were among the strongest SL candidates. Similarly, cell signalling genes including <span class="cmti-12">PLEKHO1</span>, <span class="cmti-12">RARRES2</span>, <span class="cmti-12">VEGFB</span>, <span class="cmti-12">HSPB2</span>, and <span class="cmti-12">CREM </span>were detected in stomach cancers. It is notable that several of these genes (<span class="cmti-12">EMP3</span>, <span class="cmti-12">PLEKHO1</span>, and <span class="cmti-12">FUNCD2</span>) have a known role in cancer. Together these genes support the roles of <span class="cmti-12">CDH1 </span>in cell membrane and signalling functions (of epithelial tissues) which are perturbed in both breast and stomach cancers.
</p>
<!--l. 1236-->
<p class="indent">
The strongest mtSLIPT genes tested against <span class="cmti-12">CDH1 </span>mutatoin for stomach cancer (shown in Table <a 
href="#x1-221001r1">G.1<!--tex4ht:ref: tab:gene_stad_mtSL --></a>) supported similar gene functions. Membrane and cell-adhesion genes including <span class="cmti-12">KFBP6</span>,<span class="cmti-12">THY1</span>,<span class="cmti-12">CLELC2B</span>, <span class="cmti-12">NISCH</span>, <span class="cmti-12">TSPAN1</span>,and <span class="cmti-12">KCTD12 </span>and signalling genes including <span class="cmti-12">ZEB2</span>, <span class="cmti-12">CCND2</span>, <span class="cmti-12">NEURL1B</span>, <span class="cmti-12">KFBP6</span>, and <span class="cmti-12">OGN </span>were detected. Similarly, these include cancer genes such as <span class="cmti-12">VIM</span>,<span class="cmti-12">ZEB2</span>,<span class="cmti-12">BCL2</span>,<span class="cmti-12">THY1</span>, and <span class="cmti-12">RUNX1T1</span>. The mtSLIPT procedure also replicated several of the strongest candidates in breast cancer (shown in Table <a 
href="#x1-205001r1">D.1<!--tex4ht:ref: tab:gene_mtSL --></a>) such as <span class="cmti-12">NRIP2 </span>and <span class="cmti-12">NISCH</span>.
</p>
<!--l. 1238-->
<p class="indent">
<p>Together, these gene candidates indicate widespread functions of <span class="cmti-12">CDH1 </span>and strongly detectable synthetic lethality with many genes from a strategy that can be applied across cancer types. More specifically, the signalling genes included GPCR signalling genes (such as</p>
<span class="cmti-12">GNG11</span>, <span class="cmti-12">GNAI1</span>, <span class="cmti-12">DZIP1</span>, <span class="cmti-12">PTGFR</span>, and <span class="cmti-12">KCTD12</span>), a growth signalling pathway which was one of the most supported synthetic lethal pathways in breast cancer analysis, the experimental screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), and has many actionable drug targets which have been applied to other diseases.
</p>
<!--l. 1240-->
<p class="indent">
<p>These findings were further supported by the pathways over-represented in <a 
href="#glo:SLIPT">SLIPT</a> candidates from TCGA stomach cancer (shown in Table <a 
href="#x1-135001r8">4.8<!--tex4ht:ref: tab:pathway_stad_exprSL --></a>) which were replicated the translational and immune pathways observed in TCGA breast cancer (shown in Tabel <a 
href="#x1-119001r2">4.2<!--tex4ht:ref: tab:pathway_exprSL --></a>). Further support for GCPR signalling pathways including the class A/1 receptors. The extracellular matrix was also detected at the pathway level in stomach cancer <a 
href="#glo:SLIPT">SLIPT</a> candidates and replicated in mtSLIPT analysis for <span class="cmti-12">CDH1 </span>mutation (shown in Table <a 
href="#x1-221002r2">G.2<!--tex4ht:ref: tab:pathway_stad_mtSL --></a>), including elastic fibres, glycosylation, collagen, and integrin cell-surface interactions. Thus there was strong evidence for the role of extracellular matrix pathways and the tumour microenvironment in <span class="cmti-12">CDH1 </span>deficient stomach cancers, in addition to cell signalling and translation pathways important in tumour growth across breast and stomach cancer.</p>
</p>
<!--l. 1243-->
<p class="indent">
<a 
 id="x1-135001r8"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 4.8: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in stomach cancer</span></p>
</div>
<!--tex4ht:label?: x1-135001r8 -->
<div class="tabular">
<table id="TBL-66" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-66-1g">
<col 
id="TBL-66-1" />
<col 
id="TBL-66-2" />
<col 
id="TBL-66-3" />
<col 
id="TBL-66-4" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-66-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-1-1" class="td11">
<strong><span class="cmbx-12">Pathways Over-represented</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-1-2" class="td11">
<strong><span class="cmbx-12">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-1-3" class="td11">
<strong><span class="cmbx-12">SL Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-1-4" class="td11">
<strong><span class="cmbx-12">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-2-1" class="td11">
Extracellular matrix organization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-2-2" class="td11">
<pre><code>  241         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-2-3&quot;  </code></pre>
class=“td11”&gt; 104
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-2-4" class="td11">
7<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">140</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-3-1" class="td11">
Hemostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-3-2" class="td11">
<pre><code>  445         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-3-3&quot;  </code></pre>
class=“td11”&gt; 138
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-3-4" class="td11">
1<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">121</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-4-1" class="td11">
Developmental Biology
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-4-2" class="td11">
<pre><code>  432         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-4-3&quot;  </code></pre>
class=“td11”&gt; 125
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-4-4" class="td11">
9<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">107</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-5-1" class="td11">
Axon guidance
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-5-2" class="td11">
<pre><code>  289         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-5-3&quot;  </code></pre>
class=“td11”&gt; 94
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-5-4" class="td11">
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">102</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-6-1" class="td11">
Eukaryotic Translation Termination
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-6-2" class="td11">
<pre><code>  84          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-6-3&quot;  </code></pre>
class=“td11”&gt; 49
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-6-4" class="td11">
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">99</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-7-1" class="td11">
GPCR ligand binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-7-2" class="td11">
<pre><code>  373         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-7-3&quot;  </code></pre>
class=“td11”&gt; 108
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-7-4" class="td11">
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">99</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-8-1" class="td11">
Viral mRNA Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-8-2" class="td11">
<pre><code>  82          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-8-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-8-4" class="td11">
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">98</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-9-1" class="td11">
Formation of a pool of free 40S subunits
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-9-2" class="td11">
<pre><code>  94          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-9-3&quot;  </code></pre>
class=“td11”&gt; 51
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-9-4" class="td11">
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">98</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-10-1" class="td11">
Eukaryotic Translation Elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-10-2" class="td11">
<pre><code>  87          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-10-3&quot;  </code></pre>
class=“td11”&gt; 49
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-10-4" class="td11">
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-11-1" class="td11">
Peptide chain elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-11-2" class="td11">
<pre><code>  84          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-11-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-11-4" class="td11">
7<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-12-1" class="td11">
Class A/1 (Rhodopsin-like receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-12-2" class="td11">
<pre><code>  289         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-12-3&quot;  </code></pre>
class=“td11”&gt; 90
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-12-4" class="td11">
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">96</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-13-1" class="td11">
Nonsense Mediated Decay independent of the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-13-2" class="td11">
<pre><code>  89          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-13-3&quot;  </code></pre>
class=“td11”&gt; 49
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-13-4" class="td11">
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">96</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-14-1" class="td11">
Infectious disease
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-14-2" class="td11">
<pre><code>  349         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-14-3&quot;  </code></pre>
class=“td11”&gt; 100
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-14-4" class="td11">
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">94</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-15-1" class="td11">
GTP hydrolysis and joining of the 60S ribosomal subunit
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-15-2" class="td11">
<pre><code>  105         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-15-3&quot;  </code></pre>
class=“td11”&gt; 52
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-15-4" class="td11">
3<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">94</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-16-1" class="td11">
L13a-mediated translational silencing of Ceruloplasmin expression
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-16-2" class="td11">
<pre><code>  104         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-16-3&quot;  </code></pre>
class=“td11”&gt; 51
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-16-4" class="td11">
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-17-1" class="td11">
3’ -UTR-mediated translational regulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-17-2" class="td11">
<pre><code>  104         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-17-3&quot;  </code></pre>
class=“td11”&gt; 51
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-17-4" class="td11">
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-18-1" class="td11">
Neuronal System
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-18-2" class="td11">
<pre><code>  272         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-18-3&quot;  </code></pre>
class=“td11”&gt; 84
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-18-4" class="td11">
8<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-19-1" class="td11">
SRP-dependent cotranslational protein targeting to membrane
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-19-2" class="td11">
<pre><code>  105         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-19-3&quot;  </code></pre>
class=“td11”&gt; 51
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-19-4" class="td11">
9<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-20-1" class="td11">
Eukaryotic Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-20-2" class="td11">
<pre><code>  112         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-20-3&quot;  </code></pre>
class=“td11”&gt; 52
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-20-4" class="td11">
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-66-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-66-21-1" class="td11">
Cap-dependent Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-21-2" class="td11">
<pre><code>  112         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-66-21-3&quot;  </code></pre>
class=“td11”&gt; 52
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-66-21-4" class="td11">
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-66-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-66-22-1"  
class="td11">
<pre><code>                                                         &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 1299-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Gene set over-representation analysis (hypergeometric test) for Reactome pathways in </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">partners for </span><span class="cmti-10x-x-109">CDH1</span></p>
</p>
</div>
<hr class="endfloat" />
<a 
 id="x1-135002r193"></a>
<h4 class="subsectionHead">
<span class="titlemark">4.4.2 </span> <a 
 id="x1-1360002"></a>Synthetic Lethal Expression Profiles
</h4>
<!--l. 1309-->
<p class="noindent">
<p>The expression profiles of candidate synthetic lethal partners dtected by <a 
href="#glo:SLIPT">SLIPT</a> and mtSLIPT in stomach cancer were plotted against clinical characteristics as described in section for breast cancer data in Section <a 
href="#x1-1200002">4.1.2<!--tex4ht:ref: chapt3:exprSL_clusters --></a> (shown in Figures <a 
href="#x1-136001r12">4.12<!--tex4ht:ref: fig:slipt_expr_stad --></a> and <a 
href="#x1-222001r1">G.1<!--tex4ht:ref: fig:slipt_expr_stad_mtSL --></a> respectively). As expected the majority of <span class="cmti-12">CDH1 </span>mutant samples had low expression of <span class="cmti-12">CDH1 </span>and were the diffuse type of stomach cancer.</p>
</p>
<!--l. 1312-->
<p class="indent">
<a 
 id="x1-136001r12"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________      <!--l. 1315-->
<p class="noindent">
<img 
src="CDH1_Heatmaps_Genes_Split_By_CDH1_z-trans_exprSL_cordistx_Pub_stad.png" alt="PIC"  
 /> <br />
</p>
<div class="caption">
<p><span class="id">Figure 4.12: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethal expression profiles of analysed samples.</span></strong> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">expression profile heatmap (correlation distance) of all samples (separated by the</span> <span class="cmr-8">1</span> <span class="cmmi-10x-x-109">∕</span> <span class="cmr-8">3</span> <span class="cmr-10x-x-109">quantile of </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">expression) analysed in TCGA stomach cancer dataset for gene</span> <span class="cmr-10x-x-109">expression of 4,365 candidate partners of E-cadherin (</span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">) from </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">prediction</span> <span class="cmr-10x-x-109">(with significant FDR adjusted </span><span class="cmmi-10x-x-109">p &lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05). Deeply clustered, inter-correlated genes form</span> <span class="cmr-10x-x-109">several main groups, each containing genes that were SL candidates or toxic in an siRNA</span> <span class="cmr-10x-x-109">screen </span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">Telford </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">2015</span></a><span class="cmr-10x-x-109">). Clusters had different sample groups highly expressing the</span> <span class="cmr-10x-x-109">synthetic lethal candidates in </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">low samples, notably diffuse and </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutant</span> <span class="cmr-10x-x-109">samples have elevated expression in one or more distinct clusters, although there was less</span> <span class="cmr-10x-x-109">complexity and variation among candidate synthetic lethal partners than in breast data.</span> <span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">low samples also contained most of samples with </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutations. </span></span></p>
</div>
<!--tex4ht:label?: x1-136001r12 -->
</div>
<hr class="endfloat" />
<!--l. 1325-->
<p class="indent">
The <a 
href="#glo:SLIPT">SLIPT</a> partners of <span class="cmti-12">CDH1 </span>exhibited similar clustering in staomch cancer to breast cancer, replicating the diverse roles of elevated partner genes in different clinical samples. Specifically (in Figure <a 
href="#x1-136001r12">4.12<!--tex4ht:ref: fig:slipt_expr_stad --></a>), the diffuse type stomach cancers had higher expression of the candidate synthetic lethal partners (where <span class="cmti-12">CDH1 </span>has a role as a driver mutation), despite an unbiased clustering. This is consistent with compensating expression of synthetic lethal partners under loss of <span class="cmti-12">CDH1</span>, as suggested by <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2015">2015</a>). The pathway composition of gene clusters for stomach cancer (shown in Table <a 
href="#x1-216002r2">F.2<!--tex4ht:ref: tab:pathway_clusters_stad --></a>) was also highly concordant with breast cancer findings (shown in Table <a 
href="#x1-121001r3">4.3<!--tex4ht:ref: tab:pathway_clusters --></a>). These included replicated of translation (Cluster 1), immune functions (Cluster 2), G˙<span class="cmmi-12">αs</span> signalling (Cluster 3), and further support for the roles of GPCRs and the extracellular matrix (Cluster 4) in the synthetic lethal partners and functions of <span class="cmti-12">CDH1</span>, replicated across stomach and breast cancers. Clusters 1 and 4, which had particularly high expression of <a 
href="#glo:SLIPT">SLIPT</a> candidate partner genes in the diffuse subtype, also had the most significant over-representation of pathways.
</p>
<!--l. 1327-->
<p class="indent">
There was less variation between the expression profiles of mtSLIPT partners of <span class="cmti-12">CDH1 </span>in stomach cancer, although clusters were still detectable (as shown in Figure <a 
href="#x1-222001r1">G.1<!--tex4ht:ref: fig:slipt_expr_stad_mtSL --></a>). While the genes and pathways detected was lewss significant (due to lower sample size), the composition of clusters was further indicative for the roles of extracellular matrix (including elastic fibres), immune functions, and the cell signalling. <a 
 id="x1-136002r195"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.4.3 </span> <a 
 id="x1-1370003"></a>Comparison to Primary Screen
</h4>
<!--l. 1333-->
<p class="noindent">
The number of genes detected by both <a 
href="#glo:SLIPT">SLIPT</a> in TCGA stomach cancer data and siRNA in breast cell lines (shown in Figure <a 
href="#x1-217001r1">F.1<!--tex4ht:ref: fig:Venn_allgenes_stad --></a>) was also not a significant overlap (as observed for breast cancer in Figure <a 
href="#x1-123001r2">4.2<!--tex4ht:ref: fig:Venn_allgenes --></a>). This was particularly the case of mtSLIPT against <span class="cmti-12">CDH1</span> mutation in stomach cancer which detected very few genes (as shown in Figure <a 
href="#x1-223001r2">G.2<!--tex4ht:ref: fig:Venn_allgenes_stad_mtSL --></a>) due to low sample size and mutation frequency.
</p>
<!--l. 1335-->
<p class="indent">
<p>This smaller overlap can also be attributed to the tissue-specific differences between the stomach cancers and the breast cells used for the experimental model (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>). Nevertheless, many genes were detected across <a 
href="#glo:SLIPT">SLIPT</a> in stomach cancers and the experimental screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) and the pathways detected were consistent with prior observations in breast cancer. Despite differences in the specific</p>
genes detected, the functions of <span class="cmti-12">CDH1 </span>were conserved across epithetial cancers in different tissues and synthetic lethal inhibition of interacting pathways may be effective against molecular targets such as <span class="cmti-12">CDH1 </span>inactivation across tissue types.
</p>
<!--l. 1337-->
<p class="indent">
However, the pathway composition of <a 
href="#glo:SLIPT">SLIPT</a>-specific genes and those replicated with the siRNA primary screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) were highly concordant between the pathways identified by <a 
href="#glo:SLIPT">SLIPT</a> in TCGA stomach cancer (shown in Table <a 
href="#x1-217002r3">F.3<!--tex4ht:ref: tab:Venn_over-representation_stad --></a>) and pathways previously identified in TCGA breast cancer (shown in Table <a 
href="#x1-127001r4">4.4<!--tex4ht:ref: tab:Venn_over-representation --></a>). In both cases, translation and immune pathways were highly over-represented in <a 
href="#glo:SLIPT">SLIPT</a>-specific genes, which we would not expect to be detected by siRNA screening in cell lines, as discussed in Section <a 
href="#x1-1270004">4.2.1.4<!--tex4ht:ref: chapt3:compare_pathway --></a>. In addition, the extracellular matrix was supported by in stomach cancer. While the pathways identified by specifically by <a 
href="#glo:SLIPT">SLIPT</a> in stomach cancer or siRNA screening were similar to those observed for breast cancer (in Table <a 
href="#x1-127001r4">4.4<!--tex4ht:ref: tab:Venn_over-representation --></a>), the pathways over-represented in the intersection for stomach cancer <a 
href="#glo:SLIPT">SLIPT</a> candidates and the siRNA primary screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) also had a clear over-representation of signalling pathways, although they differed from those observed in breast cancer <a 
href="#glo:SLIPT">SLIPT</a> candidates. GPCR signalling was supported in genes detected in both TCGA stomach cancer and screening, including G˙<span class="cmmi-12">αq</span>, G˙<span class="cmmi-12">αs</span>, serotonin receptors, and class A signalling (shown in more detail in Table <a 
href="#x1-218002r5">F.5<!--tex4ht:ref: tab:pathway_perm_overlap_stad --></a>). In addition MAPK and NOTCH signalling pathways were detected. These replicate the findings in breast cancer and show consistent detection of signalling pathways in stomach cancer despite less genes being detected by <a 
href="#glo:SLIPT">SLIPT</a> and patient samples differing from the tissue in which the experiments were conducted.
</p>
<!--l. 1339-->
<p class="indent">
Similarly, the <a 
href="#glo:SLIPT">SLIPT</a>-specific gene candidates against <span class="cmti-12">CDH1 </span>mutation (shown in Table <a 
href="#x1-223002r4">G.4<!--tex4ht:ref: tab:Venn_over-representation_stad_mtSL --></a>) replicated pathways observed in breast cancer (shown in Table <a 
href="#x1-207002r4">D.4<!--tex4ht:ref: tab:Venn_over-representation_mtSL --></a>), despite a lower number of genes detected. In particular, the extracellular matrix and elastic fibres were over-represented. While the number of genes overlapping with the siRNA was too low to be amenable to pathway analysis, there is further indication that members of these genes replicated across mutation <a 
href="#glo:SLIPT">SLIPT</a> analyses include cell-membrane, elastic fibre, and GPCR signalling genes. <a 
 id="x1-137001r177"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">4.4.3.1 </span> <a 
 id="x1-1380001"></a>Resampling Analysis
</h5>
<!--l. 1345-->
<p class="noindent">
<p>Similarly, resampling for <a 
href="#glo:SLIPT">SLIPT</a> specific candidates (shown in Tables <a 
href="#x1-218001r4">F.4<!--tex4ht:ref: tab:pathway_perm_stad --></a> and <a 
href="#x1-224001r5">G.5<!--tex4ht:ref: tab:pathway_perm_stad_mtSL --></a>) replicated many of the most highly over-represented pathways in stomach cancer. These</p>
include translational, immune, GPCR signalling, and elastic fibres, consistent with previous analyses in breast cancer (shown in Tables <a 
href="#x1-128002r5">4.5<!--tex4ht:ref: tab:pathway_perm --></a> and <a 
href="#x1-208001r5">D.5<!--tex4ht:ref: tab:pathway_perm_mtSL --></a>).
</p>
<!--l. 1347-->
<p class="indent">
While fewer pathways were supported by resampling for the intersection of <a 
href="#glo:SLIPT">SLIPT</a> and experimental screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) candidate partners in stomach cancer than breast cancer, many of those detected (shown in Table <a 
href="#x1-218002r5">F.5<!--tex4ht:ref: tab:pathway_perm_overlap_stad --></a>) replicate those detected in breast cancer (shown in Tables <a 
href="#x1-128003r6">4.6<!--tex4ht:ref: tab:pathway_perm_overlap --></a> and <a 
href="#x1-208002r6">D.6<!--tex4ht:ref: tab:pathway_perm_overlap_mtSL --></a>). The pathways detected by both permutation and over-representation were more likely to be replicated across stomach and breast cancer than those detected by over-representation alone, supporting the use of this procedure to detect synthetic lethal pathways applicable across cancer types. The include G˙<span class="cmmi-12">αs</span> signalling and elastic fibre formation as discussed for breast cancer (in Section <a 
href="#x1-1280001">4.2.1.4.1<!--tex4ht:ref: chapt3:compare_pathway_perm --></a>)
</p>
<!--l. 1349-->
<p class="indent">
While many pathways were detected by resampling for mtSLIPT against <span class="cmti-12">CDH1 </span>mutation in stomach cancer (shown in Table <a 
href="#x1-224002r6">G.6<!--tex4ht:ref: tab:pathway_perm_overlap_stad_mtSL --></a>), there were not enough genes detected by both mtSLIPT and the siRNA primary screen to determine over-represented pathways. Therefore this may be due to small numbers of genes which does not constitute support for pathway composition. However, this under-powered analysis does not preclude the replicated synthetic lethal pathways detected across <a 
href="#glo:SLIPT">SLIPT</a> expression analyses in TCGA breast and stomach cancer data with an accompanying siRNA primary screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). Rather this further supports the use of <a 
href="#glo:SLIPT">SLIPT</a> to test against low expression of query genes as measure of gene inactivation to avoid this issue, despite mutation (which often produces similar results) being more indicative of complete gene inactivation. <a 
 id="x1-138001r197"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.4.4 </span> <a 
 id="x1-1390004"></a>Metagene Analysis
</h4>
<!--l. 1355-->
<p class="noindent">
Metagene analysis (as conducted in Section <a 
href="#x1-1330004">4.3.4<!--tex4ht:ref: chapt3:metagene_SL --></a>) was also performed for TCGA stomach cancer expression data, using Reactome pathways. These results (as shown in Table <a 
href="#x1-219001r6">F.6<!--tex4ht:ref: tab:metagene_stad_SL --></a>) provided further support for signalling and extracellular processes as synthetic lethal pathways across stomach and breast cancer. Namely, cell-cell communication, VEGF signalling, and various GPCR pathways were detected.
</p>
<!--l. 1357-->
<p class="indent">
<p>Signalling and immune pathways were also supported by mtSLIPT analysis of metagene pathway expression against <span class="cmti-12">CDH1 </span>mutation, as shown in Table <a 
href="#x1-225001r7">G.7<!--tex4ht:ref: tab:metagene_stad_mtSL --></a>. Although these results were generally less statistically significant than expression analyses. Signalling pathways detected as synthetic lethal metagenes include prostacyclin, SCF-KIT, ERK, MAPK, NGF, VEGF, and PI3K/AKT. The innate immune response, the inflammasome, and integrin</p>
signaling were also implicated to be synthetic lethal with <span class="cmti-12">CDH1 mutations</span>. Cell surface interactions, cholesterol biosynthesis, and platelet homeostasis also support the role of extracellular processes in proliferation of <span class="cmti-12">CDH1 </span>deficient cancers and interactions of <span class="cmti-12">CDH1</span> with the extracellular environment that was not tested in the cell line experimental screen. <a 
 id="x1-139001r192"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">4.5 </span> <a 
 id="x1-1400005"></a>Global Synthetic Lethality
</h3>
<!--l. 1367-->
<p class="noindent">
Global levels of synthetic lethality were analysed to address concerns raised by the high numbers of synthetic lethal candidates for <span class="cmti-12">CDH1</span>. The <a 
href="#glo:SLIPT">SLIPT</a> procedure (as described in Section <a 
href="#x1-930001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) was performed with each possible query gene from the TCGA breast cancer RNA-Seq dataset. Due to the computational demands of this procedure, it was performed on the New Zealand eScience Infrastructure Intel Pan supercomputer (as described in Section <a 
href="#x1-910003">2.5.3<!--tex4ht:ref: methods:HPC --></a>).
</p>
<!--l. 1369-->
<p class="indent">
<p>The observed number of <a 
href="#glo:SLIPT">SLIPT</a> appears to be typical for most genes in the TCGA breast RNA- Seq dataset as shown in Figure <a 
href="#x1-140001r13">4.13<!--tex4ht:ref: fig:global_SL --></a>. This figure was actually lower than the majority (95%) of genes tested, although <span class="cmti-12">CDH1 </span>was ranked higher for a similar in <a 
href="#glo:SLIPT">SLIPT</a> analysis of TCGA stomach cancer data, shown in Figure <a 
href="#x1-226001r1">H.1<!--tex4ht:ref: fig:global_SL_stad --></a>. The differences in sample size make these analyses difficult to compare but (in either case), the number of partners detected for <span class="cmti-12">CDH1</span> is not unexpected, eeven when adjust for multiple comparisons across candidate partners.</p>
</p>
<!--l. 1371-->
<p class="indent">
<a 
 id="x1-140001r13"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 1373-->
<p class="noindent">
</p>
<!--l. 1374-->
<p class="noindent">
<img 
src="MostSL_Summary_CDH1_FDR.png" alt="pict"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 4.13: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethal partners across query genes.</span></strong> <span class="cmr-10x-x-109">Global synthetic lethal</span> <span class="cmr-10x-x-109">pairs were examined across the genome in TCGA breast expression data by applying</span> <a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">across query genes. The high number of predicted partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">was typical</span> <span class="cmr-10x-x-109">for a human gene and lower than many other genes. </span></span></p>
</div>
<!--tex4ht:label?: x1-140001r13 -->
</div>
<hr class="endfloat" />
<!--l. 1384-->
<p class="indent">
The number of detected candidates reported here is higher than in Figures <a 
href="#x1-123001r2">4.2<!--tex4ht:ref: fig:Venn_allgenes --></a> and <a 
href="#x1-217001r1">F.1<!--tex4ht:ref: fig:Venn_allgenes_stad --></a> because these exlcuded genes not tested by the siRNA primary screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) for comparison with it. For an statistically rigorous measure of global synthetic lethality, multiple comparison procedures would need to be performed for all pairs of genes tested. However, only partner genes for each query <a 
href="#glo:SLIPT">SLIPT</a> analysis were performed for the purposes of comparing the number of partners predicted with those observed for <span class="cmti-12">CDH1 </span>throughout this thesis. <a 
 id="x1-140002r199"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.5.1 </span> <a 
 id="x1-1410001"></a>Hub Genes
</h4>
<!--l. 1391-->
<p class="noindent">
<p>The genes with the most synthetic lethal interactions by this <a 
href="#glo:SLIPT">SLIPT</a> analysis are the “hub” genes of a synthetic lethal network. These genes with the highest number of candidate partners detected by <a 
href="#glo:SLIPT">SLIPT</a> in TCGA breast cancer expression data are summarised in Table <a 
href="#x1-141001r9">4.9<!--tex4ht:ref: tab:gene_mostSL --></a>. These include several genes involved in cellular signalling such as <span class="cmti-12">TGFBR2</span>, <span class="cmti-12">PDGFRA</span>, <span class="cmti-12">FAM126A</span>, <span class="cmti-12">KCTD12</span>, <span class="cmti-12">MAML2</span>, and <span class="cmti-12">CAV1</span>. Gene regulation including chromatin, DNA, and RNA bindings genes were also observed as hub genes such as <span class="cmti-12">CELF2</span>, <span class="cmti-12">PLAGL1</span>, <span class="cmti-12">TSHZ2</span>, <span class="cmti-12">FOXO1</span>, and <span class="cmti-12">SVEP1</span>. Genes involved in the cellular membrane such as <span class="cmti-12">ANXA1</span> and <span class="cmti-12">FAM171A1 </span>were also observed in addition to genes specifically implicated in cell adhesion and tight junctions such as <span class="cmti-12">TNS1</span>, <span class="cmti-12">BOC</span>, <span class="cmti-12">AMOTL1</span>, <span class="cmti-12">FAT4</span>, and <span class="cmti-12">EPB41L2</span>.</p>
</p>
<!--l. 1393-->
<p class="indent">
<a 
 id="x1-141001r9"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 4.9: </span><span class="content">Query synthetic lethal genes with the most SLIPT partners</span></p>
</div>
<!--tex4ht:label?: x1-141001r9 -->
<div class="tabular">
<table id="TBL-67" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-67-1g">
<col 
id="TBL-67-1" />
<col 
id="TBL-67-2" />
<col 
id="TBL-67-3" />
<col 
id="TBL-67-4" />
<col 
id="TBL-67-5" />
<col 
id="TBL-67-6" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-67-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-1-1" class="td11">
<span class="cmbx-12">Gene </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-1-2" class="td11">
Direction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-1-3" class="td11">
raw p-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-1-4" class="td11">
p-value (FDR)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-1-5" class="td11">
<a 
href="#glo:SLIPT">SLIPT</a> raw p-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-1-6" class="td11">
<a 
href="#glo:SLIPT">SLIPT</a> (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-2-1" class="td11">
<span class="cmti-12">TGFBR2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-2-2" class="td11">
8134
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-2-3" class="td11">
17982
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-2-4" class="td11">
<pre><code> 17973       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-2-5&quot;  </code></pre>
class=“td11”&gt; 8007
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-2-6"  
class="td11">
<pre><code> 8006        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-3-1" class="td11">
<span class="cmti-12">A2M </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-3-2" class="td11">
8571
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-3-3" class="td11">
17605
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-3-4" class="td11">
<pre><code> 17583       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-3-5&quot;  </code></pre>
class=“td11”&gt; 8345
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-3-6"  
class="td11">
<pre><code> 8339        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-4-1" class="td11">
<span class="cmti-12">TNS1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-4-2" class="td11">
8019
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-4-3" class="td11">
17949
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-4-4" class="td11">
<pre><code> 17934       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-4-5&quot;  </code></pre>
class=“td11”&gt; 7874
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-4-6"  
class="td11">
<pre><code> 7873        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-5-1" class="td11">
<span class="cmti-12">PROS1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-5-2" class="td11">
8539
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-5-3" class="td11">
17668
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-5-4" class="td11">
<pre><code> 17642       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-5-5&quot;  </code></pre>
class=“td11”&gt; 8317
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-5-6"  
class="td11">
<pre><code> 8310        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-6-1" class="td11">
<span class="cmti-12">ANXA1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-6-2" class="td11">
9085
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-6-3" class="td11">
17330
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-6-4" class="td11">
<pre><code> 17302       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-6-5&quot;  </code></pre>
class=“td11”&gt; 8689
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-6-6"  
class="td11">
<pre><code> 8682        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-7-1" class="td11">
<span class="cmti-12">CELF2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-7-2" class="td11">
8665
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-7-3" class="td11">
17406
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-7-4" class="td11">
<pre><code> 17368       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-7-5&quot;  </code></pre>
class=“td11”&gt; 8370
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-7-6"  
class="td11">
<pre><code> 8355        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-8-1" class="td11">
<span class="cmti-12">BOC </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-8-2" class="td11">
8694
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-8-3" class="td11">
17371
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-8-4" class="td11">
<pre><code> 17348       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-8-5&quot;  </code></pre>
class=“td11”&gt; 8384
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-8-6"  
class="td11">
<pre><code> 8381        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-9-1" class="td11">
<span class="cmti-12">PLAGL1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-9-2" class="td11">
8792
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-9-3" class="td11">
17361
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-9-4" class="td11">
<pre><code> 17327       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-9-5&quot;  </code></pre>
class=“td11”&gt; 8448
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-9-6"  
class="td11">
<pre><code> 8436        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-10-1" class="td11">
<span class="cmti-12">PDGFRA </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-10-2" class="td11">
8296
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-10-3" class="td11">
17650
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-10-4" class="td11">
<pre><code> 17621       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-10-5&quot;  </code></pre>
class=“td11”&gt; 8095
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-10-6"  
class="td11">
<pre><code> 8087        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-11-1" class="td11">
<span class="cmti-12">FAM171A1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-11-2" class="td11">
8874
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-11-3" class="td11">
17560
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-11-4" class="td11">
<pre><code> 17533       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-11-5&quot;  </code></pre>
class=“td11”&gt; 8567
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-11-6"  
class="td11">
<pre><code> 8562        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-12-1" class="td11">
<span class="cmti-12">FAM126A </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-12-2" class="td11">
8510
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-12-3" class="td11">
17383
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-12-4" class="td11">
<pre><code> 17356       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-12-5&quot;  </code></pre>
class=“td11”&gt; 8184
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-12-6"  
class="td11">
<pre><code> 8178        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-13-1" class="td11">
<span class="cmti-12">TSHZ2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-13-2" class="td11">
7942
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-13-3" class="td11">
17983
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-13-4" class="td11">
<pre><code> 17976       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-13-5&quot;  </code></pre>
class=“td11”&gt; 7787
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-13-6"  
class="td11">
<pre><code> 7786        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-14-1" class="td11">
<span class="cmti-12">KCTD12 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-14-2" class="td11">
8366
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-14-3" class="td11">
17651
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-14-4" class="td11">
<pre><code> 17621       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-14-5&quot;  </code></pre>
class=“td11”&gt; 8115
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-14-6"  
class="td11">
<pre><code> 8108        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-15-1" class="td11">
<span class="cmti-12">MAML2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-15-2" class="td11">
8336
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-15-3" class="td11">
17537
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-15-4" class="td11">
<pre><code> 17503       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-15-5&quot;  </code></pre>
class=“td11”&gt; 8069
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-15-6"  
class="td11">
<pre><code> 8061        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-16-1" class="td11">
<span class="cmti-12">FOXO1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-16-2" class="td11">
8027
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-16-3" class="td11">
17753
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-16-4" class="td11">
<pre><code> 17737       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-16-5&quot;  </code></pre>
class=“td11”&gt; 7840
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-16-6"  
class="td11">
<pre><code> 7836        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-17-1" class="td11">
<span class="cmti-12">AMOTL1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-17-2" class="td11">
8425
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-17-3" class="td11">
17388
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-17-4" class="td11">
<pre><code> 17347       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-17-5&quot;  </code></pre>
class=“td11”&gt; 8147
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-17-6"  
class="td11">
<pre><code> 8139        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-18-1" class="td11">
<span class="cmti-12">FAT4 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-18-2" class="td11">
8111
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-18-3" class="td11">
17750
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-18-4" class="td11">
<pre><code> 17732       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-18-5&quot;  </code></pre>
class=“td11”&gt; 7925
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-18-6"  
class="td11">
<pre><code> 7919        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-19-1" class="td11">
<span class="cmti-12">CAV1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-19-2" class="td11">
8645
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-19-3" class="td11">
17491
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-19-4" class="td11">
<pre><code> 17464       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-19-5&quot;  </code></pre>
class=“td11”&gt; 8342
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-19-6"  
class="td11">
<pre><code> 8331        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-20-1" class="td11">
<span class="cmti-12">SVEP1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-20-2" class="td11">
7945
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-20-3" class="td11">
17859
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-20-4" class="td11">
<pre><code> 17842       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-20-5&quot;  </code></pre>
class=“td11”&gt; 7791
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-20-6"  
class="td11">
<pre><code> 7784        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-67-21-1" class="td11">
<span class="cmti-12">EPB41L2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-21-2" class="td11">
8415
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-21-3" class="td11">
17327
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-67-21-4" class="td11">
<pre><code> 17296       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-67-21-5&quot;  </code></pre>
class=“td11”&gt; 8097
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-67-21-6"  
class="td11">
<pre><code> 8092        &lt;/td&gt;</code></pre>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-67-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-67-22-1"  
class="td11">
<pre><code>       &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 1450-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Genes with the most candidate SL partners </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">in TCGA breast expression data with the number of partner</span> <span class="cmr-10x-x-109">genes predicted by direction criteria and </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">testing separately and combined as a </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">analysis. Where specified,</span> <span class="cmr-10x-x-109">the p-values for the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test were adjusted for multiple tests (FDR).</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 1453-->
<p class="indent">
Genes involved in adhesion and tight junctions were also hub genes in stomach cancer (shown in Table <a 
href="#x1-227001r1">H.1<!--tex4ht:ref: tab:gene_mostSL_stad --></a>) such as <span class="cmti-12">HEG1</span>, <span class="cmti-12">FAT4</span>, <span class="cmti-12">NFASC</span>, <span class="cmti-12">LAMA4</span>, <span class="cmti-12">LAMC1</span>, <span class="cmti-12">TNS1</span>, and <span class="cmti-12">AMOTL1</span>. These also included cytoskeletal genes such as <span class="cmti-12">ANK2</span>, <span class="cmti-12">TTC28</span>, and <span class="cmti-12">MACF1</span>. Cancer genes were also among hub genes across breast and stomach cancer such as <span class="cmti-12">BOC</span>, <span class="cmti-12">FAT4</span>, and <span class="cmti-12">MRVI1</span>.
</p>
<!--l. 1455-->
<p class="indent">
It is therefore unsurprising that signalling and regulatory genes have been detected throughout this thesis. Not only are they suitable targets for anti-cancer therapy, they are also highly interacting genes themselves and so it is plausible that their interactions would be detectable by <a 
href="#glo:SLIPT">SLIPT</a>. This is consistent with the established role of abberant signalling and gene regulation in proliferation and survival of tumours and the importance of these pathways in development with highly redundant functions across many genes under complex regulation. These are also highly amenable to detection by <a 
href="#glo:SLIPT">SLIPT</a> analysis of expression data since their functions are dynamically regulated with corresponding changes in expression.
</p>
<!--l. 1457-->
<p class="indent">
Cytoskeletal, membrane bound, and extracellular matrix genes are also among highly interacting synthetic lethal hubs, including focal adhesion, tight junctions, microtubules, and fibronectin. These support the use of synthetic lethal interactions to target <span class="cmti-12">CDH1</span>, as a tumour suppressor gene involved in these functions. Cellular structure and cell-cell interactions are thus important functions with highly redundant genes for which there are many feasible synthetic lethal interactions by which to understand regulation of cellular functions. These functions may also be exploited as vulnerabilities in cancer as they are frequently disruped in cancers, including HDGC where loss of <span class="cmti-12">CDH1 </span>is a driver of cancer proliferation and malignancy. <a 
 id="x1-141002r202"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.5.2 </span> <a 
 id="x1-1420002"></a>Hub Pathways
</h4>
<!--l. 1464-->
<p class="noindent">
<p>Pathways over-represented among TCGA breast cancer hub genes (as shown in Table <a 
href="#x1-142001r10">4.10<!--tex4ht:ref: tab:pathway_mostSL --></a>) particularly support the importance of signalling pathways, such as the PI3K/AKT pathway, as synthetic lethal hubs. The highly redundant natures of cell-cell interaction and the extracellular matrix functions was also further supported.</p>
</p>
<!--l. 1467-->
<p class="indent">
<a 
 id="x1-142001r10"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 4.10: </span><span class="content">Pathways for genes with the most SLIPT partners</span></p>
</div>
<!--tex4ht:label?: x1-142001r10 -->
<div class="tabular">
<table id="TBL-68" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-68-1g">
<col 
id="TBL-68-1" />
<col 
id="TBL-68-2" />
<col 
id="TBL-68-3" />
<col 
id="TBL-68-4" />
<col 
id="TBL-68-5" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-68-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-1-1" class="td11">
<strong><span class="cmbx-12">Pathways Over-represented</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-1-2" class="td11">
<strong><span class="cmbx-12">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-1-3" class="td11">
<strong><span class="cmbx-12">SL Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-1-4" class="td11">
<strong><span class="cmbx-12">p-value</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-1-5" class="td11">
<strong><span class="cmbx-12">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-2-1" class="td11">
Constitutive Signaling by Aberrant PI3K in Cancer
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-2-2" class="td11">
<pre><code>  56          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-2-3&quot;  </code></pre>
class=“td11”&gt; 10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-2-4" class="td11">
8<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-2-5" class="td11">
8<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-3-1" class="td11">
PI3K/AKT Signaling in Cancer
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-3-2" class="td11">
<pre><code>  78          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-3-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-3-4" class="td11">
2<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-3-5" class="td11">
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-4-1" class="td11">
Role of LAT2/NTAL/LAB on calcium mobilization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-4-2" class="td11">
<pre><code>  96          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-4-3&quot;  </code></pre>
class=“td11”&gt; 12
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-4-4" class="td11">
7<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-4-5" class="td11">
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-5-1" class="td11">
Complement cascade
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-5-2" class="td11">
<pre><code>  33          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-5-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-5-4" class="td11">
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-5-5" class="td11">
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-6-1" class="td11">
Cell surface interactions at the vascular wall
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-6-2" class="td11">
<pre><code>  99          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-6-3&quot;  </code></pre>
class=“td11”&gt; 12
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-6-4" class="td11">
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-6-5" class="td11">
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-7-1" class="td11">
PI3K events in ERBB4 signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-7-2" class="td11">
<pre><code>  87          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-7-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-7-4" class="td11">
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-7-5" class="td11">
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-8-1" class="td11">
PIP3 activates AKT signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-8-2" class="td11">
<pre><code>  87          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-8-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-8-4" class="td11">
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-8-5" class="td11">
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-9-1" class="td11">
PI3K events in ERBB2 signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-9-2" class="td11">
<pre><code>  87          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-9-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-9-4" class="td11">
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-9-5" class="td11">
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-10-1" class="td11">
PI-3K cascade:FGFR1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-10-2" class="td11">
<pre><code>  87          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-10-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-10-4" class="td11">
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-10-5" class="td11">
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-11-1" class="td11">
PI-3K cascade:FGFR2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-11-2" class="td11">
<pre><code>  87          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-11-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-11-4" class="td11">
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-11-5" class="td11">
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-12-1" class="td11">
PI-3K cascade:FGFR3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-12-2" class="td11">
<pre><code>  87          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-12-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-12-4" class="td11">
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-12-5" class="td11">
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-13-1" class="td11">
PI-3K cascade:FGFR4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-13-2" class="td11">
<pre><code>  87          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-13-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-13-4" class="td11">
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-13-5" class="td11">
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-14-1" class="td11">
Extracellular matrix organization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-14-2" class="td11">
<pre><code>  238         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-14-3&quot;  </code></pre>
class=“td11”&gt; 22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-14-4" class="td11">
4<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-14-5" class="td11">
3<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-15-1" class="td11">
Muscle contraction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-15-2" class="td11">
<pre><code>  62          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-15-3&quot;  </code></pre>
class=“td11”&gt; 9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-15-4" class="td11">
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-15-5" class="td11">
3<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-16-1" class="td11">
PI3K/AKT activation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-16-2" class="td11">
<pre><code>  90          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-16-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-16-4" class="td11">
5<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-16-5" class="td11">
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-17-1" class="td11">
GAB1 signalosome
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-17-2" class="td11">
<pre><code>  91          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-17-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-17-4" class="td11">
7<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-17-5" class="td11">
4<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-18-1" class="td11">
Smooth Muscle Contraction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-18-2" class="td11">
<pre><code>  28          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-18-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-18-4" class="td11">
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-18-5" class="td11">
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-19-1" class="td11">
Response to elevated platelet cytosolic Ca<sup><span class="cmr-8">2+</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-19-2" class="td11">
<pre><code>  82          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-19-3&quot;  </code></pre>
class=“td11”&gt; 10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-19-4" class="td11">
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-19-5" class="td11">
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-20-1" class="td11">
Signaling by SCF-KIT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-20-2" class="td11">
<pre><code>  126         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-20-3&quot;  </code></pre>
class=“td11”&gt; 13
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-20-4" class="td11">
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-20-5" class="td11">
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-68-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-68-21-1" class="td11">
Signaling by FGFR
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-21-2" class="td11">
<pre><code>  143         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-68-21-3&quot;  </code></pre>
class=“td11”&gt; 14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-21-4" class="td11">
5<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-68-21-5" class="td11">
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-68-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-68-22-1"  
class="td11">
<pre><code>                                         &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 1523-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Gene set over-representation analysis (hypergeometric test) for Reactome pathways in the top 500 “hub” genes with the most candidate</span> <span class="cmr-10x-x-109">synthetic lethal partners by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">analysis of TCGA breast expression data</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 1526-->
<p class="indent">
Pathway over-representation for synthetic lethal hub genes was replicated in TCGA stomach cancer expression data. However, these pathways differ considerably from breast cancer, as shown in Table <a 
href="#x1-228001r2">H.2<!--tex4ht:ref: tab:pathway_mostSL_stad --></a>. Cell-cell interactions and extracellular matrix pathways, including elastic fibres, were also among the hub genes for stomach cancer. The signalling pathways differ as expected in a different tissue type, although BMP and PAK signalling were detected as hub gene functions. <a 
 id="x1-142002r200"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">4.6 </span> <a 
 id="x1-1430006"></a>Replication in cell line encyclopaedia
</h3>
<!--l. 1534-->
<p class="indent">
<a 
 id="x1-143001r11"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 4.11: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in CCLE</span></p>
</div>
<!--tex4ht:label?: x1-143001r11 -->
<div class="tabular">
<table id="TBL-69" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-69-1g">
<col 
id="TBL-69-1" />
<col 
id="TBL-69-2" />
<col 
id="TBL-69-3" />
<col 
id="TBL-69-4" />
</colgroup>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-1-1" class="td11">
<strong><span class="cmbx-12">Pathways Over-represented</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-1-2" class="td11">
<strong><span class="cmbx-12">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-1-3" class="td11">
<strong><span class="cmbx-12">SL Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-1-4" class="td11">
<strong><span class="cmbx-12">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-2-1" class="td11">
Cell Cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-2-2" class="td11">
<pre><code>  442         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-2-3&quot;  </code></pre>
class=“td11”&gt; 207
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-2-4" class="td11">
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">215</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-3-1" class="td11">
Cell Cycle, Mitotic
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-3-2" class="td11">
<pre><code>  365         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-3-3&quot;  </code></pre>
class=“td11”&gt; 180
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-3-4" class="td11">
2<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">209</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-4-1" class="td11">
Signaling by Rho GTPases
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-4-2" class="td11">
<pre><code>  311         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-4-3&quot;  </code></pre>
class=“td11”&gt; 136
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-4-4" class="td11">
9<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">156</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-5-1" class="td11">
M Phase
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-5-2" class="td11">
<pre><code>  212         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-5-3&quot;  </code></pre>
class=“td11”&gt; 104
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-5-4" class="td11">
8<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">145</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-6-1" class="td11">
Infectious disease
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-6-2" class="td11">
<pre><code>  289         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-6-3&quot;  </code></pre>
class=“td11”&gt; 123
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-6-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">142</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-7-1" class="td11">
RHO GTPase Effectors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-7-2" class="td11">
<pre><code>  207         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-7-3&quot;  </code></pre>
class=“td11”&gt; 98
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-7-4" class="td11">
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">135</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-8-1" class="td11">
HIV Infection
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-8-2" class="td11">
<pre><code>  200         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-8-3&quot;  </code></pre>
class=“td11”&gt; 94
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-8-4" class="td11">
<pre><code>2 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">130</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-9-1" class="td11">
Separation of Sister Chromatids
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-9-2" class="td11">
<pre><code>  140         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-9-3&quot;  </code></pre>
class=“td11”&gt; 77
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-9-4" class="td11">
5<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">128</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-10-1" class="td11">
Organelle biogenesis and maintenance
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-10-2" class="td11">
<pre><code>  258         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-10-3&quot;  </code></pre>
class=“td11”&gt; 107
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-10-4" class="td11">
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">127</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-11-1" class="td11">
Chromatin modifying enzymes
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-11-2" class="td11">
<pre><code>  181         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-11-3&quot;  </code></pre>
class=“td11”&gt; 87
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-11-4" class="td11">
4<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">126</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-12-1" class="td11">
Chromatin organization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-12-2" class="td11">
<pre><code>  181         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-12-3&quot;  </code></pre>
class=“td11”&gt; 87
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-12-4" class="td11">
4<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">126</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-13-1" class="td11">
Mitotic Metaphase and Anaphase
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-13-2" class="td11">
<pre><code>  149         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-13-3&quot;  </code></pre>
class=“td11”&gt; 78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-13-4" class="td11">
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">124</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-14-1" class="td11">
Mitotic Anaphase
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-14-2" class="td11">
<pre><code>  148         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-14-3&quot;  </code></pre>
class=“td11”&gt; 77
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-14-4" class="td11">
6<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">123</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-15-1" class="td11">
Developmental Biology
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-15-2" class="td11">
<pre><code>  421         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-15-3&quot;  </code></pre>
class=“td11”&gt; 142
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-15-4" class="td11">
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">121</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-16-1" class="td11">
RHO GTPases Activate Formins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-16-2" class="td11">
<pre><code>  94          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-16-3&quot;  </code></pre>
class=“td11”&gt; 60
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-16-4" class="td11">
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">118</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-17-1" class="td11">
Mitotic Prometaphase
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-17-2" class="td11">
<pre><code>  93          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-17-3&quot;  </code></pre>
class=“td11”&gt; 59
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-17-4" class="td11">
5<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">116</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-18-1" class="td11">
Hemostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-18-2" class="td11">
<pre><code>  421         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-18-3&quot;  </code></pre>
class=“td11”&gt; 138
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-18-4" class="td11">
7<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">116</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-19-1" class="td11">
Adaptive Immune System
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-19-2" class="td11">
<pre><code>  397         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-19-3&quot;  </code></pre>
class=“td11”&gt; 132
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-19-4" class="td11">
3<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">115</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-20-1" class="td11">
Assembly of the primary cilium
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-20-2" class="td11">
<pre><code>  143         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-20-3&quot;  </code></pre>
class=“td11”&gt; 72
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-20-4" class="td11">
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">114</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-69-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-69-21-1" class="td11">
Transcription
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-21-2" class="td11">
<pre><code>  133         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-69-21-3&quot;  </code></pre>
class=“td11”&gt; 68
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-69-21-4" class="td11">
6<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">111</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-69-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-69-22-1"  
class="td11">
<pre><code>                              &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 1591-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Gene set over-representation analysis (hypergeometric test) for Reactome pathways in </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">partners for</span> <span class="cmti-10x-x-109">CDH1</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 1594-->
<p class="noindent">
As breast cancer cell lines are the experimental system in which many cancer genetics and drug targets are investigated, these were analysed in addition to patient samples from TCGA. The cancer cell line encyclopaedia (CCLE) is a resource for genomics profiles across a range of cell lines. These have also been used to generate synthetic lethal candidates for comparison to those in experimental screen and predictions from TCGA expression data.
</p>
<!--l. 1596-->
<p class="indent">
The cancer cell line encyclopaedia provides further support for synthetic lethal genes and pathways that may be applicable across cell types and reproducible in experimental systems. In contrast to the homogeneous pooled cell samples of patients, the cell lines provide a genetically homogeneous cell population in which to examine molecular functions and as a preclinical model of cancerous disease. The complete set of 1037 cell lines was tested for synthetic lethality across tissues, in addition to the 59 breast cell lines and 38 stomach cell lines being tested separately for partners of <span class="cmti-12">CDH1</span>. Synthetic lethal genes were detected by <a 
href="#glo:SLIPT">SLIPT</a> (as described in Section <a 
href="#x1-930001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) and over-represented synthetic lethal Reactome pathways (as described in Section <a 
href="#x1-770002">2.3.2<!--tex4ht:ref: methods:enrichment --></a>).
</p>
<!--l. 1598-->
<p class="indent">
<p>Synthetic lethal gene candidates were detectable by <a 
href="#glo:SLIPT">SLIPT</a> across each of these sample sets of cells lines (as shown in Tables <a 
href="#x1-229001r1">I.1<!--tex4ht:ref: tab:gene_ccle_SL --></a>–<a 
href="#x1-229003r3">I.3<!--tex4ht:ref: tab:gene_ccle_stad_SL --></a>. Although these were most highly significant across the samples in the CCLE expression dataset (as shown in Table <a 
href="#x1-229001r1">I.1<!--tex4ht:ref: tab:gene_ccle_SL --></a>) and included genes detected in prior analyses such as <span class="cmti-12">VIM</span>, <span class="cmti-12">ZEB2</span>, <span class="cmti-12">EMP3</span>. Pathways were also highly over-represented among synthetic lethal candidates for the full CCLE dataset (as shown in Table <a 
href="#x1-143001r11">4.11<!--tex4ht:ref: tab:pathway_ccle_exprSL --></a>) including Rho GTPase (GPCRs), immmune, and gene regulation (chromatin and transcription). This is unexpected since immune pathways would not be expected to be detectable in isolated cell lines, although this could be attributed to cytokine and integrin signalling occuring the cancer cells in addition to interactions with immune cells in the tumour microenvironment (which could not be distinguished in patient samples). Cell cycle and mitosis were among the highest synthetic lethal pathways across cell lines supporting <span class="cmti-12">CDH1 </span>deficient cells having abberant cell signalling and consequences for proliferation such as cancer cells. However, cell cycle genes were not as strongly supported in TCGA patient samples or the siRNA screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) and they may not be applicable to epithelial tissues such as breast or stomach cancer or amenable to selective inhibition in experimental models.</p>
</p>
<!--l. 1600-->
<p class="indent">
<a 
 id="x1-143002r12"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 4.12: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in breast CCLE</span></p>
</div>
<!--tex4ht:label?: x1-143002r12 -->
<div class="tabular">
<table id="TBL-70" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-70-1g">
<col 
id="TBL-70-1" />
<col 
id="TBL-70-2" />
<col 
id="TBL-70-3" />
<col 
id="TBL-70-4" />
</colgroup>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-70-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-1-1" class="td11">
<strong><span class="cmbx-12">Pathways Over-represented</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-1-2" class="td11">
<strong><span class="cmbx-12">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-1-3" class="td11">
<strong><span class="cmbx-12">SL Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-1-4" class="td11">
<strong><span class="cmbx-12">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-2-1" class="td11">
Cell junction organization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-2-2" class="td11">
<pre><code>  71          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-2-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-2-4"  
class="td11">
<pre><code>  0.006         &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-3-1" class="td11">
Adherens junctions interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-3-2" class="td11">
<pre><code>  29          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-3-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-3-4"  
class="td11">
<pre><code>  0.006         &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-4-1" class="td11">
Dermatan sulfate biosynthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-4-2" class="td11">
<pre><code>  11          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-4-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-4-4"  
class="td11">
<pre><code>  0.006         &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-5-1" class="td11">
Non-integrin membrane-ECM interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-5-2" class="td11">
<pre><code>  52          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-5-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-5-4"  
class="td11">
<pre><code>  0.006         &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-6-1" class="td11">
Regulation of pyruvate dehydrogenase (PDH) complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-6-2" class="td11">
<pre><code>  12          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-6-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-6-4"  
class="td11">
<pre><code> 0.0069        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-7-1" class="td11">
Cell-extracellular matrix interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-7-2" class="td11">
<pre><code>  17          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-7-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-7-4"  
class="td11">
<pre><code>  0.021         &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-8-1" class="td11">
Pyruvate metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-8-2" class="td11">
<pre><code>  17          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-8-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-8-4"  
class="td11">
<pre><code>  0.021         &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-9-1" class="td11">
Cell-cell junction organization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-9-2" class="td11">
<pre><code>  46          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-9-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-9-4"  
class="td11">
<pre><code>  0.039         &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-10-1" class="td11">
Synthesis of substrates in N-glycan biosythesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-10-2" class="td11">
<pre><code>  50          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-10-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-10-4"  
class="td11">
<pre><code>  0.057         &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-11-1" class="td11">
Detoxification of Reactive Oxygen Species
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-11-2" class="td11">
<pre><code>  26          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-11-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-11-4"  
class="td11">
<pre><code>  0.082         &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-12-1" class="td11">
Keratan sulfate biosynthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-12-2" class="td11">
<pre><code>  28          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-12-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-12-4"  
class="td11">
<pre><code>  0.092         &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-13-1" class="td11">
Laminin interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-13-2" class="td11">
<pre><code>  28          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-13-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-13-4"  
class="td11">
<pre><code>  0.092         &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-14-1" class="td11">
Cell-Cell communication
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-14-2" class="td11">
<pre><code>  118         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-14-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-14-4"  
class="td11">
<pre><code>  0.12          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-15-1" class="td11">
Keratan sulfate/keratin metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-15-2" class="td11">
<pre><code>  32          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-15-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-15-4"  
class="td11">
<pre><code>  0.12          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-16-1" class="td11">
Opioid Signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-16-2" class="td11">
<pre><code>  63          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-16-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-16-4"  
class="td11">
<pre><code>  0.12          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-17-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-70-17-1"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-71" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-71-1g">
<col 
id="TBL-71-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-71-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-71-1-1" class="td00">
Biosynthesis of the N-glycan precursor (dolichol lipid-linked oligosaccharide)
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-71-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-71-2-1" class="td00">
and transfer to a nascent protein
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-17-2" class="td11">
<pre><code>  63          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-17-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-17-4"  
class="td11">
<pre><code>  0.12          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-18-1" class="td11">
Intraflagellar transport
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-18-2" class="td11">
<pre><code>  34          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-18-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-18-4"  
class="td11">
<pre><code>  0.14          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-19-1" class="td11">
Signaling by Retinoic Acid
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-19-2" class="td11">
<pre><code>  36          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-19-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-19-4"  
class="td11">
<pre><code>  0.16          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-20-1" class="td11">
Pyruvate metabolism and Citric Acid (TCA) cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-20-2" class="td11">
<pre><code>  36          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-20-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-20-4"  
class="td11">
<pre><code>  0.16          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-70-21-1" class="td11">
Nef mediated downregulation of MHC class I complex cell surface expression
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-70-21-2" class="td11">
<pre><code>  10          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-70-21-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-70-21-4"  
class="td11">
<pre><code>  0.22          &lt;/td&gt;</code></pre>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-70-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-70-22-1"  
class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 1657-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Gene set over-representation analysis (hypergeometric test) for Reactome pathways in </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">partners for </span><span class="cmti-10x-x-109">CDH1</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 1661-->
<p class="indent">
Synthetic lethal pathways specfic to <a 
href="#glo:SLIPT">SLIPT</a> candidates from breast cell lines (as shown in Table <a 
href="#x1-143002r12">4.12<!--tex4ht:ref: tab:pathway_ccle_brca_exprSL --></a>) were more consistent with previous obervations, particularly the established role of E-cadherin in cell junctions and the Adherens complex. Although the number of <a 
href="#glo:SLIPT">SLIPT</a> candidate genes detected in stomach cell lines was insufficient to replicate the findings in breast cell lines to TCGA patient samples. However, <a 
href="#glo:SLIPT">SLIPT</a> candidates across breast and stomach CCLE cell lines were over-represented (as shown in Table <a 
href="#x1-229005r5">I.5<!--tex4ht:ref: tab:pathway_ccle_breast_stad_exprSL --></a>) for similar pathways to breast cell lines with additional support for extracellular matrix pathways including elastic fibres which were replicated with resampling across breast and stomach TCGA analyses and the primary siRNA screen <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>). <a 
 id="x1-143003r206"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">4.7 </span> <a 
 id="x1-1440007"></a>Discussion
</h3>
<a 
 id="x1-144001r204"></a>
<h4 class="subsectionHead">
<span class="titlemark">4.7.1 </span> <a 
 id="x1-1450001"></a>Strengths of the SLIPT Methodology
</h4>
<!--l. 1669-->
<p class="noindent">
Synthetic lethal discovery with <a 
href="#glo:SLIPT">SLIPT</a> used established statistical procedures identify putative partner genes from gene expression data. Such use of the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>-value is amenable to pathway or permutation analyses and could feasibly be applied to other disease gene or pair-wise across the genome. Although genome-wide approaches were unable to find informative candidate genes for E-cadherin <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2015">2015</a>). Synthetic lethal discovery in cancer has focused on genes with severe cellular mutant phenotypes, such as essential genes or the oncogenes <span class="cmti-12">TP53 </span>and <span class="cmti-12">AKT </span><a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2015">2015</a>); <a 
href="#XTiong2014">Tiong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTiong2014">2014</a>); <a 
href="#XWang2013">Wang and Simon</a> (<a 
href="#XWang2013">2013</a>), with other cancer genes, such as <span class="cmti-12">CDH1</span>, requiring more focused investigations. Prior computational approaches for synthetic lethal discovery, in cancer, vary widely (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>; <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>; <a 
href="#XTiong2014">Tiong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTiong2014">2014</a>; <a 
href="#XWappett2016">Wappett <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XWappett2016">2016</a>). There is no consensus for which for the approach is more appropriate and they are difficult to compare as they either do not have a released code implementation or do not make predictions solely from normalised expression data.
</p>
<!--l. 1671-->
<p class="indent">
<p>However, the query-based approach demonstrated by <a 
href="#glo:SLIPT">SLIPT</a> analysis is suitable for wider application on expression data and augmenting experimental studies such as high-throughput screens. This approach has identified biologically plausible synthetic lethal</p>
pathways for <span class="cmti-12">CDH1</span>, triaged candidates from experimenal screening (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), and replicates genes and pathways across breast and somtach cancers datasets. In addition, <a 
href="#glo:SLIPT">SLIPT</a> avoids critical assumptions underlying the design of some prior approaches such as coexpression of synthetic candidates or that they will have known (annotated) similarities in function.
</p>
<!--l. 1673-->
<p class="indent">
The DAISY methodology <a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XJerby2014">2014</a>), which took a similar query-based approach with the tumour suppressor <span class="cmti-12">VHL</span>, has been critiqued for being too stringent <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2015">2015</a>) which impedes pathway analysis. Since functional redundancy does not require genes to be expressed at the same time, the <a 
href="#glo:SLIPT">SLIPT</a> approach does not assume co-expression of synthetic lethal genes which may enrich for synthetic lethal genes in established coregulated pathways. Rather, the interpretation of synthetic lethality for <a 
href="#glo:SLIPT">SLIPT</a> was similar to other computational methods based on ‘co-loss under-representation’, ‘compensation’, or ‘simultaneous differential expression’ <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XLu2015">2015</a>); <a 
href="#XTiong2014">Tiong <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTiong2014">2014</a>); <a 
href="#XWang2013">Wang and Simon</a> (<a 
href="#XWang2013">2013</a>).
</p>
<!--l. 1675-->
<p class="indent">
Genomics analyses are prone to false-positives and require statistical caution, particularly where working with gene-pairs scale up the number of multiple tests drastically, at the expense of statistical power. Experimental screens for synthetic lethality are also error-prone, especially with false-positives, raising the need for understanding the expected behaviour and number of functional relationships and genetic interactions in the genome, or in discovery of synthetic lethal partners of a particular query gene. Thus analyses throughout this thesis have focused on querying for partners of a particular gene of interest. Statistical modelling and simulations (in Section <a 
href="#x1-970003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a> and Chapter <a 
href="#x1-1750006">6<!--tex4ht:ref: chap:simulation --></a>) will further support the design decisions underlying <a 
href="#glo:SLIPT">SLIPT</a> analysis and it’s strengths over other approaches. <a 
 id="x1-145001r210"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.7.2 </span> <a 
 id="x1-1460002"></a>Syntheic Lethal Pathways for E-cadherin
</h4>
<!--l. 1679-->
<p class="noindent">
<p>As specific genes were difficult to replicate across experiments, gene expression profiles for synthetic lethal partners must be more complex than originally expected to directly compensate for loss of query gene or completely lack (or clearly under-representation) mutual loss (<a 
href="#XJerby2014">Jerby-Arnon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XJerby2014">2014</a>; <a 
href="#XKelly2013">Kelly</a>, <a 
href="#XKelly2013">2013</a>; <a 
href="#XLu2015">Lu <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XLu2015">2015</a>). The predicted synthetic lethal partners of <span class="cmti-12">CDH1 </span>(with FDR correction) were investigated with gene expression profiles and clinical variables to find relationships in gene expression, gene function, and clinical characteristics. The large number of hits indicate that synthetic lethal detection is error-prone and identifying genes relevant for clinical application will be difficult</p>
without a supporting biological pathway rationale. As such, investigations into the genes identified by <a 
href="#glo:SLIPT">SLIPT</a>, correlation structure between them, and those which were validated by experimental screening (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) focused on the pathway level throughout this Chapter. Similarly, comparisons across analyses were largely made at the pathway level, including comparisons between expression and mutation, breast and stomach TCGA datasets, and patient sample data with cell line expression profiles.
</p>
<!--l. 1681-->
<p class="indent">
Potential synthetic lethal partners of <span class="cmti-12">CDH1 </span>identified by <a 
href="#glo:SLIPT">SLIPT</a> had many distinct functions, with each gene cluster highly expressed in different patient subgroups (Figure <a 
href="#x1-120001r1">4.1<!--tex4ht:ref: fig:slipt_expr --></a>). The expression profiles of the SL partners of <span class="cmti-12">CDH1 </span>predicted from the TCGA breast cancer RNA-Seq data (expected to have compensating high or stable expression) and their corresponding functional enrichment found subgroups of genes with functional organisation particularly among <span class="cmti-12">CDH1 </span>low breast tumours. Ductal breast cancers show higher expression of synthetic lethal partners suggesting treatment would be more effective in this tumour subtype. However, there is consistently low expression of SL partners in estrogen receptor negative tumours, although this is independent of tumour stage and consistent with poor prognosis in these patients and could inform other treatment strategies or prevent ineffective treatment further impacting quality of life in these patients. These results suggest that synthetic lethal partner expression varies between patients; that these different tumour classes would react differently to the same treatment; that treatment of different pathways and combinations in different patients is the most effective approach to target genes compensating for <span class="cmti-12">CDH1</span> gene loss; and the expression of synthetic partners could be a clinically important biomarker.
</p>
<!--l. 1683-->
<p class="indent">
<p>The pathways that synthetic lethal partners of <span class="cmti-12">CDH1 </span>identified by <a 
href="#glo:SLIPT">SLIPT</a> were involved in a diverse range of biological functions and differed to those detected experimentally. This discrepancy may be accounted for by gene expression analyses detecting both synthetic lethal partners (as screened for experimentally <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>)) and their downstream targets (not detected by siRNA), capturing the wider pathways and mechanisms involved in synthetic lethality with <span class="cmti-12">CDH1 </span>inactivation. In particular, GPCR phosphorylation cascades (which regulates gene expression and translation in cancers <a 
href="#XGao2015">Gao and Roux</a> (<a 
href="#XGao2015">2015</a>)) were predicted to be synthetic lethal with <span class="cmti-12">CDH1</span>. The predicted synthetic lethal partners occurred across functionally distinct pathways, including characterised functions of <span class="cmti-12">CDH1</span>. The most consistently supported pathways include elastic fibres in the extracelullar matrix, GPCR signalling, and translation presenting vulnerabilities for <span class="cmti-12">CDH1</span></p>
deficient cancer cells from extracellular stimuli to the core growth mechanisms of a cell.
</p>
<!--l. 1685-->
<p class="indent">
This diversity in synthetic lethal functions is consistent with the wide ranging role of <span class="cmti-12">CDH1 </span>in cell-cell adhesion, cell signalling, and the cytoskeletal structure of epithelial tissues. Pathway structure may be relevant to identifying potential drug targets from gene expression signatures, indicating downstream effector genes and mechanisms leading to cell inviability. Identification of distinct synthetic lethal gene clusters may further lead to the elucidation of drug resistance mechanisms. While these pathways are indicative of the main functions of E-cadherin and synthetic lethal partners, it remains to identify the genes within these pathways that are the most actionable or supported across <a 
href="#glo:SLIPT">SLIPT</a> analysis in patient samples and detected by experiments in preclincal models (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>; <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). The specific genes within key pathways will be be discussed in Chapter <a 
href="#x1-1530005">5<!--tex4ht:ref: chap:Pathways --></a>, along with further investigations into their relation to pathway structure. While these are important clinical implications, the synthetic lethal predictions lack enough confidence for direct translation into pre-clinical models or clinical applications leading to a need for statistical modelling and simulation of synthetic lethality in genomics expression data.
</p>
<!--l. 1687-->
<p class="indent">
These sythetic lethal pathways have potential clinical implications, particularly those supported in pre-clinical models and in patient expression data. However, further validation of gene candidates will be neccessary to ensure that these are able to reproduced in further pre-clincal studies, they are applicable to tumours <span class="cmti-12">in</span> <span class="cmti-12">vivo</span>, and that effective inhibitory agents can be repurposed or designed against them. <a 
 id="x1-146001r211"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">4.7.3 </span> <a 
 id="x1-1470003"></a>Replication and Validation
</h4>
<a 
 id="x1-147001r198"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">4.7.3.1 </span> <a 
 id="x1-1480001"></a>Integration with siRNA Screening
</h5>
<!--l. 1693-->
<p class="noindent">
<p>The pathway composition across computational and experimental synthetic lethal candidates was informative with over-representation (Table <a 
href="#x1-127001r4">4.4<!--tex4ht:ref: tab:Venn_over-representation --></a>) and supported by resampling analysis (Table <a 
href="#x1-128003r6">4.6<!--tex4ht:ref: tab:pathway_perm_overlap --></a>), despite a modest intersection of genes between them (Figure <a 
href="#x1-123001r2">4.2<!--tex4ht:ref: fig:Venn_allgenes --></a>). Either approach may be significant for a pathway in this intersection without being supported by the other: resampling analysis may support pathways that were not</p>
over-represented due to small effect sizes, thus both tests are required for a candidate pathway.
</p>
<!--l. 1695-->
<p class="indent">
The pathways detected by both over-representation and resampling are the strongest candidates for further investigation and the pathway structure analyses in Chapter <a 
href="#x1-1530005">5<!--tex4ht:ref: chap:Pathways --></a> will focus on these pathways detected by both over-representation and resampling. Particularly, those replicated across datasets or with pathway metagenes. In addition to GCPR pathways detected across these analyses, the PI3K cascade will also be investigated in Chapter <a 
href="#x1-1530005">5<!--tex4ht:ref: chap:Pathways --></a>, this signalling pathway is a well characterised mediator between GCPR receptors and regulation of translation (<a 
href="#XGao2015">Gao and Roux</a>, <a 
href="#XGao2015">2015</a>) (both detected throughout this Chapter) and exhibited unexpected behaviour with pathway the metagenes (in Section <a 
href="#x1-1290003">4.3<!--tex4ht:ref: chapt3:metagene_results --></a>). This pathway is activated by protein Phosphorylation states and thus inactivatino may not be detectable with expression.
</p>
<!--l. 1697-->
<p class="indent">
However, the <a 
href="#glo:SLIPT">SLIPT</a> approach was shown to be predictive of which siRNA primary screen candidate partners of <span class="cmti-12">CDH1 </span>were validated in a secondary screen (as shown in Appendix <a 
href="#x1-203000C">C<!--tex4ht:ref: appendix:secondary_screen --></a>). These results further support <a 
href="#glo:SLIPT">SLIPT</a> for identifying robust synthetic lethal candidates which can be validated and as a triage approach for interpreting screening experiments. <a 
 id="x1-148001r213"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">4.7.3.2 </span> <a 
 id="x1-1490002"></a>Replication across Tissues and Cell lines
</h5>
<!--l. 1701-->
<p class="noindent">
Furthermore, synthetic lethal partners identified by <a 
href="#glo:SLIPT">SLIPT</a> were replicated across breast and stomach cancer. These were particularly concordant at the pathway level, as expected between tissues since synthetic lethal pathways have higher conservation between species (<a 
href="#XDixon2008">Dixon <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XDixon2008">2008</a>). These findings support gene functions conserved across <span class="cmti-12">CDH1 </span>deficient cancers in breast and stomach tissues, presenting vulnerabilities that could be applied against molecular targets in both cancers. In addition, these analyses serve as a replication across independent patient cohorts from breast and stomach cancers, decreasing the likelihood of the synthetic lethal pathways detected being false positives or artifacts of either dataset.
</p>
<!--l. 1703-->
<p class="indent">
<p>Synthetic lethal pathways were also replicated across expression analyses of TCGA patient samples in heterogeneous tumours and homogeneous cell line isolates. This further supports that the subset of synthetic lethal functions detectable in experimental models (<a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>; <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) would be applicable tumours of patients with <span class="cmti-12">CDH1 </span>deficient cancers.</p>
</p>
<!--l. 1705-->
<p class="indent">
There are many gene functions replicated across breast cancer gene expression analyses. Many of these were also replicated with mutation analysis and with stomach cancer or cell line expression data. These pathways were more consistent across replication analyses than previous investigations with TCGA microarray data <a 
href="#XKelly2013">Kelly</a> (<a 
href="#XKelly2013">2013</a>). <a 
 id="x1-149001r209"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">4.8 </span> <a 
 id="x1-1500008"></a>Summary
</h3>
<!--l. 1709-->
<p class="noindent">
We have developed a simple, interpretable, computational approach to predict synthetic lethal partners from genomics data. The analyses focus on gene expression data as it is widely available for applications in other cancers and other disease genes, particularly those with malignant loss of function.
</p>
<!--l. 1711-->
<p class="indent">
This approach has been applied to robustly detect synthetic lethal pathways for the E-cadherin (<span class="cmti-12">CDH1</span>) in TCGA breast cancer molecular profiles with comparisons to experimenal screening (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) in cell lines, and replication in TCGA stomach cancer molecular profiles and across cell types in the cancer cell line encyclopaedia. The pathway replicated across several analyses included extracellular matrix pathways (such as elastic fibres formation), cell signalling (including GPCRs), and core gene regulation and translation processes crucial for the growth and proliferation of cancer cells. These pathways show evidence of non-oncogene addiction for <span class="cmti-12">CDH1 </span>deficient cells and present vulnerabilities which may be exploited for specific treatment against <span class="cmti-12">CDH1 </span>mutations in HCGC and sporadic cancers. There was also support for synthetic lethal pathways with <span class="cmti-12">CDH1 </span>in cell adhesion and cytoskeletal processes to which <span class="cmti-12">CDH1 </span>belongs, supporting the finding that synthetic lethality occurs within biological pathways (<a 
href="#XBoone2007">Boone <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBoone2007">2007</a>; <a 
href="#XKelley2005">Kelley and Ideker</a>, <a 
href="#XKelley2005">2005</a>).
</p>
<!--l. 1713-->
<p class="indent">
While translational and immune pathways detected by <a 
href="#glo:SLIPT">SLIPT</a> were not supported by primary siRNA screening (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), these were replicated across various analyses. Due to the differences between an experimenal cell line model (<a 
href="#XBarretina2012">Barretina <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XBarretina2012">2012</a>; <a 
href="#XChen2014">Chen <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XChen2014">2014</a>; <a 
href="#XFece2015">Fece de la Cruz <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XFece2015">2015</a>) and patient molecular profiles (<a 
href="#XTCGA2014GC">Bass <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTCGA2014GC">2014</a>; <a 
href="#XTCGA2012">TCGA</a>, <a 
href="#XTCGA2012">2012</a>), these would not be expected to be completely concordant. Furthermore, many pathways are difficult to test in an isolated experimental system. Nevertheless, many of the genes and pathways detected by <a 
href="#glo:SLIPT">SLIPT</a> are suitable to inform further investigations and triage of therapeutic targets against <span class="cmti-12">CDH1 </span>deficient tumours in combination with experimenal screening.
</p>
<!--l. 1715-->
<p class="indent">
<p>A characteristic of gene interaction networks is a scale-free topology leading to highly interacting hub genes, these represent important genes in a functional network. Cell surface</p>
interactions, the extracellular matrix, and cell signalling (particularly PI3K/AKT signalling) were also found to be synthetic lethal hubs with more interactions detected than other genes. This indicates that these pathways are functionally important to survival of cancer cells since they are subject to high functional redundancy, despite frequent disruptions in cancer. These pathways being involved in a disproportionate number of synthetic lethal interactions is also consistent with their detection for <span class="cmti-12">CDH1</span>.
</p>
<!--l. 1717-->
<p class="indent">
<p>Thus synthetic lethal pathways have been identified using TCGA patient molecular profiles, CCLE cancer cell line expression data, and experimenal screening results. Some these were robustly replicated across these datasets and against <span class="cmti-12">CDH1 </span>mutation or expression analysis. However, there remains the need to identify actionable genes within these pathways, relationships with experimental candidates, and how these pathways may affect viability when lost. While the genes identified between these analyses were less concordant the results of the TCGA breast cancer analysis will be used to test pathway structure relationships and further examine the synthetic lethal genes detected in the following Chapter.</p>
</p>
<!--l. 1721-->
<p class="noindent">
<span class="paragraphHead"><a 
 id="x1-1510008"></a><span class="cmbx-12">Aims</span></span>
</p>
<ul class="itemize1">
<li class="itemize">
Pathway Structure of Candidate Synthetic Lethal Genes for <span class="cmti-12">CDH1 </span>from TCGA breast data
</li>
<li class="itemize">
Comparisons to Experimental siRNA Screen Candidates
</li>
<li class="itemize">
Replication of Pathways across in TCGA Stomach data
</li>
</ul>
<!--l. 1735-->
<p class="noindent">
<span class="paragraphHead"><a 
 id="x1-1520008"></a><span class="cmbx-12">Summary</span></span>
</p>
<ul class="itemize1">
<li class="itemize">
We have developed a Synthetic Lethal detection method that generates a high number of synthetic lethal candidates
</li>
<li class="itemize">
Pathways in cell signalling, extracellular matrix, and cytoskeletal functions were supported with experimental candidates and the known functions of E-cadherin
</li>
<li class="itemize">
Several candidate pathways were supported by mutation analysis and replicated across breast and stomach cancer
</li>
<li class="itemize">
<p>Translation and immune functions were uniquely detected by the computational approach which may be explained by differences between patient samples and cell line models</p>
</li>
<li class="itemize">
There remains the need to identify actionable genes within these pathways, relationships with experimental candidates, and how these pathways may affect viability when lost
</li>
</ul>
<h2 class="chapterHead">
<span class="titlemark">Chapter 5</span><br /><a 
 id="x1-1530005"></a>Synthetic Lethal Pathway Structure
</h2>
<!--l. 4-->
<p class="noindent">
Having identified key pathways implicated in synthetic lethal genetic interactions with <span class="cmti-12">CDH1</span>, these were investigated for the underlying synthetic lethal genes within them and their relationships to pathway structure in Reactome pathways. This chapter will focus on the pathway structure of biological pathways detected across analyses in Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a>. The synthetic lethal genes considered here are those candidates detected by SLIPT (as described in Section <a 
href="#x1-930001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) in TCGA breast cancer expression and mutation data (<a 
href="#XTCGA2012">TCGA</a>, <a 
href="#XTCGA2012">2012</a>) in comparison to the candidate gene partners from the siRNA screening in breast cell lines (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>).
</p>
<!--l. 6-->
<p class="indent">
The graph structure for Reactome pathways was obtained from Pathway Commons via <a 
href="#glo:BioPAX">BioPAX</a> (as described in Section <a 
href="#x1-850002">2.4.2<!--tex4ht:ref: methods:graph_data --></a>). The pathways describe the (directional) relationships between biomolecules, including proteins (encoded by genes), in biological pathways. These relationships include cell signalling (such as kinase phosphorylation cascades), gene regulation (such as transcription factors, chromatin modifiers, RNA binding proteins), and metabolism (such as the product of an enzyme being the substrate of another). Together these relationships describe the known functional pathways in a human cell with a reasonable resolution, from a curated database supported by publications documenting pathway relationships. While this functional pathway network encapsulates protein complexes and functional modules, protein binding or text-mining alone are not used to determine relationships between genes. The Reactome network is sufficient to test pathway relationships with directional information, although it is not documented whether these relationships are activating or inhibitory.
</p>
<!--l. 8-->
<p class="indent">
<p>Pathway structures were derived from the Reactome network (as described in Section <a 
href="#x1-860003">2.4.3<!--tex4ht:ref: methods:subgraphs --></a>) for the graph structure of each biological pathway. The synthetic lethal candidate genes for notable pathways discussed in Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a>, including candidate synthetic lethal pathways of <span class="cmti-12">CDH1</span>, were examined to show the SLIPT and siRNA candidates within these pathways. Thus synthetic lethal genes were identified within a biological context and with further investigations into their relationship with pathway structure and between synthetic lethal candidates detect by each approach. Synthetic lethal gene candidates in the context of pathway structures and additional support for belonging to a synthetic lethal pathway are ideal for triage of targets specific to <span class="cmti-12">CDH1 </span>deficient tumours and for further experimental validation in preclinical models.</p>
</p>
<!--l. 10-->
<p class="indent">
Network analysis metrics (as described in Sections <a 
href="#x1-870004">2.4.4<!--tex4ht:ref: methods:network_metrics --></a> and <a 
href="#x1-1110003">3.5.3<!--tex4ht:ref: methods:igraph_extensions --></a>) were applied to test whether gene detected by SLIPT, siRNA, or both approaches varied according to these network analysis metrics (of connectivity and importance in the network) to test whether they differed between synthetic lethal genes or approaches to detect them. Another consideration is the relationships between synthetic lethal candidates detected by either approach: these were tested by both a resampling approach (as described in Sections <a 
href="#x1-1040001">3.4.1<!--tex4ht:ref: methods:pathway_str --></a> and  <a 
href="#x1-1050001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>) and compared across a ranking based on biological context (Section <a 
href="#x1-1060002">3.4.1.2<!--tex4ht:ref: methods:pathway_rank --></a>). Together these approaches serve to test the pathway relationships between SLIPT and siRNA synthetic lethal gene candidate partners for <span class="cmti-12">CDH1 </span>within the biological pathways identified and demonstrate a combination of network biology and statistical investigations into structural relationships between genes identified by a Bioinformatics analysis. <a 
 id="x1-153001r215"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">5.1 </span> <a 
 id="x1-1540001"></a>Synthetic Lethal Genes in Reactome Pathways
</h3>
<a 
 id="x1-154001r212"></a>
<h4 class="subsectionHead">
<span class="titlemark">5.1.1 </span> <a 
 id="x1-1550001"></a>The PI3K/AKT Pathway
</h4>
<!--l. 20-->
<p class="noindent">
The <a 
href="#glo:PI3K">phosphoinositide 3-kinase</a> (<a 
href="#glo:PI3K">PI3K</a>) cascade signalling pathway exhibited unexpected results with metagene analyses (as discussed in Section <a 
href="#x1-1290003">4.3<!--tex4ht:ref: chapt3:metagene_results --></a>). This pathway is also of interest because mediating signals between the <a 
href="#glo:GPCR">G protein coupled receptors</a> and regulation of protein translation which have both been strongly implicated to be synthetic lethal pathways with loss of <span class="cmti-12">CDH1 </span>function. All three of these pathways have are also subject to dysregulation in cancer and other diseases. Thus the PI3K cascade will be examined along with the most supported synthetic lethal pathways (as identified in Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a>).
</p>
<!--l. 22-->
<p class="indent">
<p>The <a 
href="#glo:PI3K">PI3K</a> pathway is also an ideal pathway to test pathway structure since it has an established direction of signal transduction from extracellular stimuli (and membrane bound receptors) to the inner mechanisms of the cell, namely the regulation of protein translation. The production of proteins is neccessary for the growth of the cell so it is reasonable to suggest that these processes may be subject to (non-oncogene) addiction in some cancer cells which rely upon them for sustained protein production and cell growth. This is also supported by the oncogenes <span class="cmti-12">PIK3CA </span>and <span class="cmti-12">AKT1 </span>being involved with the PI3K cascade and related PI3K/AKT pathway which may be subject to oncogene addiction when these proto-oncogenes are activated.</p>
</p>
<!--l. 24-->
<p class="indent">
<a 
 id="x1-155001r1"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 26-->
<p class="noindent">
</p>
<!--l. 27-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_Pi3K_exprSL_2".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 5.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality in the PI3K Cascade.</span></strong> <span class="cmr-10x-x-109">The Reactome PI3K</span> <span class="cmr-10x-x-109">Cascade pathway with synthetic lethal candidates coloured as shown in the Legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-155001r1 -->
</div>
<hr class="endfloat" />
<!--l. 40-->
<p class="indent">
Despite the <a 
href="#glo:PI3K">PI3K</a> cascade not being supported across SLIPT and siRNA analysis by over-representation (in Section <a 
href="#x1-1270004">4.2.1.4<!--tex4ht:ref: chapt3:compare_pathway --></a>) or resampling (in Section <a 
href="#x1-1280001">4.2.1.4.1<!--tex4ht:ref: chapt3:compare_pathway_perm --></a>), numerous genes were detected by either <a 
href="#glo:SLIPT">SLIPT</a> in TCGA breast expression data or the <a 
href="#glo:siRNA">siRNA</a> primary screen (as shown in Figure <a 
href="#x1-155001r1">5.1<!--tex4ht:ref: fig:SL_Pathway_Pi3K --></a>). It is also notable, that of the few genes that were identified by both approaches, these include genes that are highly conencted in the PI3K cascade and are hubs to information transmission such as <span class="cmti-12">FGF9</span>,<span class="cmti-12">PDE3B</span>, and <span class="cmti-12">PDE4A</span>. The key upstream genes <span class="cmti-12">PIK3CA </span>and <span class="cmti-12">PIK3CG </span>were detected by <a 
href="#glo:siRNA">siRNA</a> whereas the downstream <span class="cmti-12">PIK3R1</span> and <span class="cmti-12">AKT2 </span>genes were detected by <a 
href="#glo:SLIPT">SLIPT</a>. Gene detected by either method were also prevalent in the <a 
href="#glo:PI3K">PI3K</a>, <a 
href="#glo:PDE">phosphodiesterase (PDE)</a>, and <a 
href="#glo:AMPK">AMP-activated protein kinase (AMPK)</a> modules, in addition to the downstream translation factors and ribosomal genes (<span class="cmti-12">EIF4B</span>, <span class="cmti-12">EEF2K</span>, and <span class="cmti-12">RPS6</span>). Together these suggest that there may further be structure between the <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidates partners of <span class="cmti-12">CDH1 </span>in pathways such as this example. As such, pathway structure will be tested to detect differences in the upstream and downstream gene candidates of those detected by either method. This may further explain the disparity between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> genes, even in pathways such as PI3K where they did not significantly intersect.
</p>
<!--l. 42-->
<p class="indent">
This disparity between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidate synthetic lethal partners of CDH1, that is a high number of genes detected by either approach with few detected by both, was replicated the related PI3K/AKT pathway and the “PI3K/AKT in cancer” pathway (shown in Figures <a 
href="#x1-230001r1">J.1<!--tex4ht:ref: fig:SL_Pathway_Pi3KAkt --></a> and <a 
href="#x1-230002r2">J.2<!--tex4ht:ref: fig:SL_Pathway_Pi3KAktCancer --></a>). With many synthetic lethal candidates at the upstream core of these pathway networks and the downstream extremities. It is particularly notable that the many genes important in cell signalling and gene regulation were detected by either sytnhetic lethal detection approach. These include <span class="cmti-12">AKT1</span>, <span class="cmti-12">AKT2</span>, and <span class="cmti-12">AKT3</span>, the Calmodulin signalling genes <span class="cmti-12">CALM1 </span>and <span class="cmti-12">CAMK4</span>, and the forkhead family transcription factors <span class="cmti-12">FOXO1 </span>(a tumour suppressor) and <span class="cmti-12">FOXO4 </span>and inhibitor of <a 
href="#glo:EMT">EMT</a>. <a 
 id="x1-155002r220"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">5.1.2 </span> <a 
 id="x1-1560002"></a>The Extracellular Matrix
</h4>
<!--l. 49-->
<p class="noindent">
<p>The extracellular pathways elastic fibre formation and fibrin clot formation (shown in Figures <a 
href="#x1-156001r2">5.2<!--tex4ht:ref: fig:SL_Pathway_ElasticFibre --></a> and <a 
href="#x1-156002r3">5.3<!--tex4ht:ref: fig:SL_Pathway_FibrinFormation --></a> respectively) were both supported across analyses (in Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a>). This includes a significant over-representation and resampling the interaction between <a 
href="#glo:SLIPT">SLIPT</a> (for TCGA breast cancer) and <a 
href="#glo:siRNA">siRNA</a> gene candidates showing that <a 
href="#glo:SLIPT">SLIPT</a> has</p>
<p>identified these pathways in addition to their over-representation in the <a 
href="#glo:siRNA">siRNA</a> screen.</p>
</p>
<!--l. 51-->
<p class="indent">
<a 
 id="x1-156001r2"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 53-->
<p class="noindent">
</p>
<!--l. 54-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_ElasticFibre_exprSL_2".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 5.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality in the Elastic Fibre Formation Pathway.</span></strong> <span class="cmr-10x-x-109">The</span> <span class="cmr-10x-x-109">Reactome Elastic Fibre Formation pathway with synthetic lethal candidates coloured as</span> <span class="cmr-10x-x-109">shown in the Legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-156001r2 -->
</div>
<hr class="endfloat" />
<!--l. 67-->
<p class="indent">
Particularly for elastic fibres (in Figure <a 
href="#x1-156001r2">5.2<!--tex4ht:ref: fig:SL_Pathway_ElasticFibre --></a>), the vast majority of genes were detected by either approach in addition to a significant proportion of genes detected by both approaches (as determined in Section <a 
href="#x1-1270004">4.2.1.4<!--tex4ht:ref: chapt3:compare_pathway --></a>). The genes detected by both approaches also appeared to have a non-random distribution in the network with <span class="cmti-12">TFGB1</span>, <span class="cmti-12">ITGB8</span>, and <span class="cmti-12">MFAP2 </span>exhibiting high connectivity and a cental role in their respective pathway modules. In addition to a structural role in the extracellular matrix and connective tissue (including the tumour microenvironment), these proteins including Furin, <a 
href="#glo:TGFB">transforming growth factor <span class="cmmi-12">β </span>(TGF<span class="cmmi-12">β</span>)</a>, and the <a 
href="#glo:BMP">bone morphogenic proteins (BMPs)</a>, are also involved in responses to endocrine signals and interacting with the cellular receptors for signalling pathways. Therefore it is plausible that <span class="cmti-12">CDH1 </span>deficient tumours will be subject to non-oncogene addiction to the extracellular environment and growth signals arising from this pathway. The pathway structure is also worth further investigation into whether the genes detected by <a 
href="#glo:siRNA">siRNA</a> or both approaches are downstream of those detected by <a 
href="#glo:SLIPT">SLIPT</a> in addition to whether they have higher connectivity or centrality than other genes in the pathway.
</p>
<!--l. 69-->
<p class="indent">
<p>Genes detected as synthetic lethal partners of <span class="cmti-12">CDH1 </span>by <a 
href="#glo:SLIPT">SLIPT</a> or <a 
href="#glo:siRNA">siRNA</a> screening were also common in the Fibrin clot formation pathway (shown in Figure <a 
href="#x1-156002r3">5.3<!--tex4ht:ref: fig:SL_Pathway_FibrinFormation --></a>). This is consistent with the established pleiotropic role of <span class="cmti-12">CDH1 </span>in regulating fibrin clotting. It is also notable that the genes detected by either method appear to be highly connected such as <span class="cmti-12">C1QBP</span> <span class="cmti-12">KNG1</span>, <span class="cmti-12">F8</span>, <span class="cmti-12">F10</span>, <span class="cmti-12">F12</span>, <span class="cmti-12">F13A</span>, and <span class="cmti-12">PROC </span>(including many of the coagulation factors). Synthetic lethal candidates also include <span class="cmti-12">SERPINE2 </span>and <span class="cmti-12">PRCP</span>, which only affect downstream genes, in addition to <span class="cmti-12">PROCR </span>and <span class="cmti-12">VWF</span>, which are only affected by upstream genes.</p>
</p>
<!--l. 71-->
<p class="indent">
<a 
 id="x1-156002r3"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 73-->
<p class="noindent">
</p>
<!--l. 74-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_FibrinFormation_exprSL_2".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 5.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality in the Fibrin Clot Formation.</span></strong> <span class="cmr-10x-x-109">The Reactome</span> <span class="cmr-10x-x-109">Fibrin Clot Formation pathway with synthetic lethal candidates coloured as shown in</span> <span class="cmr-10x-x-109">the Legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-156002r3 -->
</div>
<hr class="endfloat" />
<!--l. 88-->
<p class="indent">
Many of these genes are involved in the larger Extracellular Matrix pathway (shown in Figure <a 
href="#x1-230003r3">J.3<!--tex4ht:ref: fig:SL_Pathway_ExtracellularMatrix --></a>), including many of the synthetic lethal candidates discussed for elastic fibres. The number of <a 
href="#glo:SLIPT">SLIPT</a> candidate genes outnumbers those identified by <a 
href="#glo:siRNA">siRNA</a> as expected from an isolated cell model. However, the endocrine response genes (such as <span class="cmti-12">TGFB1 </span>and <span class="cmti-12">LTBP4 </span>) which are potentially artifacts of the cell line growth process were replicated with <a 
href="#glo:SLIPT">SLIPT</a> analysis in patient tumours (TCGA breast cancer data). There is also additional support for synthetic elthal genes such as <span class="cmti-12">ITGB2</span>, <span class="cmti-12">MFAP2</span>, and <span class="cmti-12">SPARC</span> being highly connected networks hubs of the pathway. Although the complexity of extracellular matrix pathway lends credence to the need for formal network analysis approaches to aid interpretation of the structure and relationships among synthetic lethal candidates in a pathway network, in addition to statistical approaches to determine whether such relationships are unlikely to be observed by sampling error. <a 
 id="x1-156003r222"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">5.1.3 </span> <a 
 id="x1-1570003"></a>G Protein Coupled Receptors
</h4>
<!--l. 94-->
<p class="noindent">
<p>Similarly, <a 
href="#glo:GPCR">G protein coupled receptor</a> (<a 
href="#glo:GPCR">GPCR</a>) pathways are highly complex (as shown in Figures <a 
href="#x1-230004r4">J.4<!--tex4ht:ref: fig:SL_Pathway_GPCR --></a> and <a 
href="#x1-230005r5">J.5<!--tex4ht:ref: fig:SL_Pathway_GPCR_Downstream --></a>). Many of these were synthetic lethal candidates by eith <a 
href="#glo:SLIPT">SLIPT</a> pr <a 
href="#glo:siRNA">siRNA</a> screening with many detected with both approaches, consistent with these pathways being supported by prior analyses (in Sections <a 
href="#x1-1270004">4.2.1.4<!--tex4ht:ref: chapt3:compare_pathway --></a> and <a 
href="#x1-1280001">4.2.1.4.1<!--tex4ht:ref: chapt3:compare_pathway_perm --></a>). Synthetic lethal candidates include the <a 
href="#glo:PDE">PDE</a> and Calmodulin genes (as discussed in Section <a 
href="#x1-1570003">5.1.3<!--tex4ht:ref: chapt4:SL_Genes_GPCR --></a>) in addition to others such as the regulators of <a 
href="#glo:RGS">G-protein signaling (RGS)</a>, <a 
href="#glo:CXCR">chemokine receptors (CXCR)</a>, <a 
href="#glo:JAK">Janus kinase</a> (<a 
href="#glo:JAK">JAK</a>), and the <a 
href="#glo:RHO">Ras homolog family (RHO)</a> genes. These are important regulatory signalling pathways necessary for cellular growth and cancer proliferation. Thus the GPCR pathways (and downstream PI3K/AKT signals) are a potentially actionable vulnerability against <span class="cmti-12">CDH1 </span>deficient cancers, particularly since many existing drug targets exist among these signalling pathways, some of which have been experimentally validated (<a 
href="#XKellyHDGC">Kelly <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XKellyHDGC">2017b</a>; <a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). However, the complexity of GPCR networks containing hundreds of genes requires the relationships between <a 
href="#glo:SLIPT">SLIPT</a> and experimental candidates to be tested with a network based statistical approach, although a statistically significant intersection of these approaches has been established (in Sections <a 
href="#x1-1270004">4.2.1.4<!--tex4ht:ref: chapt3:compare_pathway --></a> and <a 
href="#x1-1280001">4.2.1.4.1<!--tex4ht:ref: chapt3:compare_pathway_perm --></a>). <a 
 id="x1-157001r225"></a></p>
</p>
<h4 class="subsectionHead">
<span class="titlemark">5.1.4 </span> <a 
 id="x1-1580004"></a>Gene Regulation and Translation
</h4>
<!--l. 102-->
<p class="noindent">
While very few synthetic lethal genes were detected in translational pathways in an experimental screen against <span class="cmti-12">CDH1 </span><a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XTelford2015">2015</a>), these were highly over-represented in translational elongation (as shown in Figure <a 
href="#x1-230006r6">J.6<!--tex4ht:ref: fig:SL_Pathway_TranslationElongation --></a>). These <a 
href="#glo:SLIPT">SLIPT</a> genes include many ribosomal proteins and the regulatory “elongation factors” which may be subject to responses in the upstream signalling pathways. This observation lends support the notion of pathway structure among synthetic lethal candidates detected by <a 
href="#glo:SLIPT">SLIPT</a> in comparison with <a 
href="#glo:siRNA">siRNA</a> as the computational approach with <a 
href="#glo:SLIPT">SLIPT</a> has demonstrated the ability to detect downstream genes in the core translational processes which experimental screening did not identify. Although it is possible that the experimental screening may detect upstream regulatory genes less sensitive inactivation, that is genes which are less likely to be indiscriminately lethal to both genotypes at high doses of inactivation.
</p>
<!--l. 104-->
<p class="indent">
Many of these <a 
href="#glo:SLIPT">SLIPT</a> candidate genes are also among the <a 
href="#glo:NMD">nonsense-mediated decay (NMD)</a> pathway (shown in Figure <a 
href="#x1-230007r7">J.7<!--tex4ht:ref: fig:SL_Pathway_NMD --></a>) or 3<sup><span class="cmsy-8">′</span></sup> <a 
href="#glo:UTR">untranslated region (UTR)</a> mediated translational regulation (shown in Figure <a 
href="#x1-230008r8">J.8<!--tex4ht:ref: fig:SL_Pathway_Three_prime_UTR --></a>). While genes in these pathways were also supported by experimental screening with <a 
href="#glo:siRNA">siRNA</a>, there was clear pathway structure. In particular, <span class="cmti-12">UPF1 </span>was detected in the <a 
href="#glo:siRNA">siRNA</a> screen and is the focal downstream gene for the entire <a 
href="#glo:NMD">NMD</a> pathway showing that (in this case) <a 
href="#glo:siRNA">siRNA</a> genes are downstream effectors of those detected by <a 
href="#glo:SLIPT">SLIPT</a>. 3<sup><span class="cmsy-8">′</span></sup> <a 
href="#glo:UTR">UTR</a> mediated translational regulation has a similar structure with two modules connected solely by <span class="cmti-12">RPL13A</span>, giving an example of <a 
href="#glo:SLIPT">SLIPT</a> candidates genes with high connectivity, although there were many ribosomal proteins detected by <a 
href="#glo:SLIPT">SLIPT</a>. However, <span class="cmti-12">EIF3K </span>a regulatory elongation factor (not essential to ribosomal function) that was detected by <a 
href="#glo:SLIPT">SLIPT</a> was replicated with <a 
href="#glo:siRNA">siRNA</a> screening while the majority of the elongation factors were not detected by either approach. Regulatory genes being more amenable to experimental validation also support further investigation into pathway structure as the <a 
href="#glo:SLIPT">SLIPT</a> candidates may support them by structural relationships and the downstream genes not being detectable by experimental screening with high dose inhibitors may explain the greater number of <a 
href="#glo:SLIPT">SLIPT</a> candidate partners of <span class="cmti-12">CDH1 </span>than those experimentally identified. <a 
 id="x1-158001r219"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">5.2 </span> <a 
 id="x1-1590002"></a>Network Analysis of Synthetic Lethal Genes
</h3>
<!--l. 113-->
<p class="noindent">
Genes detected as synthetic lethal partners of <span class="cmti-12">CDH1 </span>with the <a 
href="#glo:SLIPT">SLIPT</a> computational approach and the <a 
href="#glo:siRNA">siRNA</a> screen (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>) were compared across network metrics in the example of the PI3K cascade pathway (where the genes differed considerably between synthetic lethal detection methods). These were used to test whether network metrics differed between groups of genes detected by either or both approaches. These analyses serve to both test whether synthetic lethal gene candidates had higher connectivity or importance in a network and to whether either detection approach is constrained to genes with different network properties. <a 
 id="x1-159001r226"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">5.2.1 </span> <a 
 id="x1-1600001"></a>Gene Connectivity and Vertex Degree
</h4>
<!--l. 119-->
<p class="noindent">
<p>Vertex degree (the number of connections) for each gene is a fundamental property of a network. The vast majority of genes had a relatively modest number of connections each with only a few genes in the PI3K pathway (shown in Figure <a 
href="#x1-160001r4">5.4<!--tex4ht:ref: fig:SL_Pathway_PI3K_Vertex_Degree --></a>) having pathway relationships with a high number of genes, consistent with the scle-free property of biological networks <a 
href="#XBarabasi2004">Barabási and Oltvai</a> (<a 
href="#XBarabasi2004">2004</a>). There were few differences in the number of connections between gene groups (by synthetic lethal detection). Although genes detected by <a 
href="#glo:siRNA">siRNA</a> included those with the fewest connections. The median connectivity of genes detected by both approaches was marginally higher.</p>
</p>
<!--l. 122-->
<p class="indent">
<a 
 id="x1-160001r4"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 124-->
<p class="noindent">
</p>
<!--l. 125-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Centrality_exprSL/Pi3K_network_vertex_degree".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 5.4: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality and Vertex Degree.</span></strong> <span class="cmr-10x-x-109">The number of connected</span> <span class="cmr-10x-x-109">genes (vertex degree) was compared (on a log-scale) across genes deteced by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and</span> <a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screening in the Reactome PI3K cascade pathway. There were very few differences</span> <span class="cmr-10x-x-109">in vertex degree between the groups, although genes detected by </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">included those</span> <span class="cmr-10x-x-109">with the fewest connections. </span></span></p>
</div>
<!--tex4ht:label?: x1-160001r4 -->
</div>
<hr class="endfloat" />
<!--l. 138-->
<p class="indent">
The results for the PI3K pathway were very similar when testing synthetic lethality against <span class="cmti-12">CDH1 </span>mutation (mtSLIPT). In this case, there is also indication that mtSLIPT-specific genes may have higher connectivity than those detected by <a 
href="#glo:siRNA">siRNA</a> screening (shown in Figure <a 
href="#x1-231001r1">K.1<!--tex4ht:ref: fig:mtSL_Pathway_PI3K_Vertex_Degree --></a>).
</p>
<!--l. 140-->
<p class="indent">
<p>However, these apparent differences in vertex degree may be due to fewer genes being detected by either approach. There was no statistically significant effect of either computational or experimental synthetic lethal detection method on vertex degree, as determined by <a 
href="#glo:ANOVA">analysis of variance (ANOVA)</a> (shown by Tables <a 
href="#x1-160002r1">5.1<!--tex4ht:ref: tab:SL_Pathway_PI3K_Vertex_Degree --></a> and  <a 
href="#x1-231004r1">K.1<!--tex4ht:ref: tab:mtSL_Pathway_PI3K_Vertex_Degree --></a>). Thus synthetic lethal detection does not discriminate among genes by their connectivity in a pathway network, nor is either approach constrained by a genes connectivity. Both approaches have been demonstrated to detect genes with many and very few connections.</p>
</p>
<!--l. 142-->
<p class="indent">
<a 
 id="x1-160002r1"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 5.1: </span><span class="content"><a 
href="#glo:ANOVA">ANOVA</a> for Synthetic Lethality and Vertex Degree</span></p>
</div>
<!--tex4ht:label?: x1-160002r1 -->
<div class="tabular">
<table id="TBL-72" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-72-1g">
<col 
id="TBL-72-1" />
<col 
id="TBL-72-2" />
<col 
id="TBL-72-3" />
<col 
id="TBL-72-4" />
<col 
id="TBL-72-5" />
<col 
id="TBL-72-6" />
</colgroup>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-72-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-72-1-1" class="td11">
<pre><code>          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-72-1-2&quot;  </code></pre>
class=“td11”&gt; DF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-1-3" class="td11">
Sum Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-1-4" class="td11">
Mean Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-1-5" class="td11">
F-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-1-6" class="td11">
p-value
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-72-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-72-2-1" class="td11">
siRNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-2-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-2-3" class="td11">
<pre><code> 15        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-72-2-4&quot;  </code></pre>
class=“td11”&gt; 15.50
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-2-5" class="td11">
0.0134
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-2-6" class="td11">
0.9082
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-72-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-72-3-1" class="td11">
SLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-3-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-3-3" class="td11">
<pre><code> 506        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-72-3-4&quot;  </code></pre>
class=“td11”&gt; 506.01
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-3-5" class="td11">
0.4378
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-3-6" class="td11">
0.5105
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-72-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-72-4-1" class="td11">
siRNA<span class="cmsy-10x-x-120">×</span>SLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-4-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-4-3" class="td11">
<pre><code>  0         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-72-4-4&quot;  </code></pre>
class=“td11”&gt; 0.05
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-4-5" class="td11">
0.0000
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-72-4-6" class="td11">
0.9947
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-72-5-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-72-5-1"  
class="td11">
<pre><code>           &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 166-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Analysis of variance for vertex degree against synthetic lethal detection approaches (with an</span> <span class="cmr-10x-x-109">interaction term)</span></p>
</p>
</div>
<hr class="endfloat" />
<a 
 id="x1-160003r228"></a>
<h4 class="subsectionHead">
<span class="titlemark">5.2.2 </span> <a 
 id="x1-1610002"></a>Gene Importance and Centrality
</h4>
<a 
 id="x1-161001r214"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">5.2.2.1 </span> <a 
 id="x1-1620001"></a>Information Centrality
</h5>
<!--l. 175-->
<p class="noindent">
Information centrality is a measure of the importance of nodes in a network by how vital they are to the transmission of information throughout the network. This naturally applies well to biological pathways, partcularly gene regulation and cell signalling. The nodes with the highest information centrality are not necessarily the most connected as they may also include nodes which pass signals between highly connected network hubs. Information centrality therefore provides a distinct metric for the connectivity of a gene in a pathway, which has the added benefit of being directly related to the disruption of pathway function were it to be inactivated or removed.
</p>
<!--l. 177-->
<p class="indent">
Information centrality has also been suggested to indicate essentiality of genes or proteins (<a 
href="#XKranthi2013">Kranthi <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XKranthi2013">2013</a>). The information centrality for was computed across the entire Reacomte network (as discussed in Appendix <a 
href="#x1-232000L">L<!--tex4ht:ref: appendix:infocent_essential --></a>). Reactome contains substrates and cofactors in addition to genes or proteins. In support of centrality as a measure of essentiality or importance to the network, a number nodes with the highest centrality (shown in Table <a 
href="#x1-232001r1">L.1<!--tex4ht:ref: tab:infocent_reactome --></a>) were essential nutrients including Mg<sup class="textsuperscript"><span class="cmr-10x-x-109">2+</span></sup>, Ca<sup class="textsuperscript"><span class="cmr-10x-x-109">2+</span></sup>, Zn<sup class="textsuperscript"><span class="cmr-10x-x-109">2+</span></sup>, and Fe.
</p>
<!--l. 179-->
<p class="indent">
<p>Genes important in development of epithelial tissues and breast cancer were also detected with relatively high information centrality (as shown by the distribution across the Reactome network in Figure <a 
href="#x1-232002r1">L.1<!--tex4ht:ref: fig:infocent_reactome --></a>). Interleukin 8 (encoded by <span class="cmti-12">IL8</span>) is a chemokine important in epithelial cells, the innate immune system, and binding GPCRs. <span class="cmti-12">GATA4 </span>is a embryonic transcription factor involved in heart development, <a 
href="#glo:EMT">EMT</a>, and was reccurently mutated in in breast cancer (<a 
href="#XTCGA2012">TCGA</a>, <a 
href="#XTCGA2012">2012</a>). <span class="cmmi-12">β</span>-catenin (encoded by the proto-oncogene <span class="cmti-12">CTNNB1</span>) is a regulatory protein which binds E-cadherin, being involved in cell-cell adhesion and <a 
href="#glo:WNT">Wingless-related integration site (WNT)</a> signalling. Together these show that information cetrality identifies nodes of importance to biological functions in pathway networks, including those relevant to <span class="cmti-12">CDH1 </span>deficient breast cancers.</p>
</p>
<!--l. 182-->
<p class="indent">
<p>Within the PI3K pathway (shown in Figure <a 
href="#x1-162001r5">5.5<!--tex4ht:ref: fig:SL_Pathway_PI3K_InfoCent --></a>), genes detected by <a 
href="#glo:siRNA">siRNA</a> did not include those with lower centrality, although the median information centrality across gene groups detected by either synthetic lethal approach did not differ. The gene with the highest information centrality (<span class="cmti-12">AKT2</span>) was detected by SLIPT and was markedly higher than the other genes in the pathway which is consistent with the known biological role of AKT in PI3K/AKT signalling and the pathway structure (shown in Figure <a 
href="#x1-155001r1">5.1<!--tex4ht:ref: fig:SL_Pathway_Pi3K --></a>). The information centrality of the PI3K pathway was 1.338433.</p>
</p>
<!--l. 184-->
<p class="indent">
<a 
 id="x1-162001r5"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 186-->
<p class="noindent">
</p>
<!--l. 187-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Centrality_exprSL/Pi3K_network_Info_Centrality(Log)".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 5.5: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality and Centrality.</span></strong> <span class="cmr-10x-x-109">The information centrality was</span> <span class="cmr-10x-x-109">compared (on a log-scale across genes deteced by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screening in the</span> <span class="cmr-10x-x-109">Reactome PI3K cascade pathway. Genes detected by </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">had higher connectivity than</span> <span class="cmr-10x-x-109">many genes not detected by either approach. The gene with the highest centrality was</span> <span class="cmr-10x-x-109">detected by </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a><span class="cmr-10x-x-109">. </span></span></p>
</div>
<!--tex4ht:label?: x1-162001r5 -->
</div>
<hr class="endfloat" />
<!--l. 202-->
<p class="indent">
<a 
 id="x1-162002r2"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 5.2: </span><span class="content"><a 
href="#glo:ANOVA">ANOVA</a> for Synthetic Lethality and Information Centrality</span></p>
</div>
<!--tex4ht:label?: x1-162002r2 -->
<div class="tabular">
<table id="TBL-73" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-73-1g">
<col 
id="TBL-73-1" />
<col 
id="TBL-73-2" />
<col 
id="TBL-73-3" />
<col 
id="TBL-73-4" />
<col 
id="TBL-73-5" />
<col 
id="TBL-73-6" />
</colgroup>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-73-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-73-1-1" class="td11">
<pre><code>          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-73-1-2&quot;  </code></pre>
class=“td11”&gt; DF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-1-3" class="td11">
Sum Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-1-4" class="td11">
Mean Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-1-5" class="td11">
F-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-1-6" class="td11">
p-value
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-73-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-73-2-1" class="td11">
siRNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-2-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-2-3" class="td11">
0.000256
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-2-4" class="td11">
0.0002561
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-2-5" class="td11">
0.1854
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-2-6" class="td11">
0.6682
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-73-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-73-3-1" class="td11">
SLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-3-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-3-3" class="td11">
0.003827
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-3-4" class="td11">
0.0038275
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-3-5" class="td11">
2.7717
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-3-6" class="td11">
0.1008
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-73-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-73-4-1" class="td11">
siRNA<span class="cmsy-10x-x-120">×</span>SLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-4-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-4-3" class="td11">
0.000804
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-4-4" class="td11">
0.0008036
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-4-5" class="td11">
0.5820
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-73-4-6" class="td11">
0.4483
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-73-5-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-73-5-1"  
class="td11">
<pre><code>           &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 226-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Analysis of variance for information centrality against synthetic lethal detection approaches</span> <span class="cmr-10x-x-109">(with an interaction term)</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 229-->
<p class="indent">
These findings were replicated (shown in Figure <a 
href="#x1-231002r2">K.2<!--tex4ht:ref: fig:mtSL_Pathway_PI3K_InfoCent --></a>) when testing synthetic lethality against <span class="cmti-12">CDH1 </span>mutation (mtSLIPT). The differences in network centrality between gene groups detected by either method were not statistically significant as determined by <a 
href="#glo:ANOVA">ANOVA</a> (shown by Tables <a 
href="#x1-162002r2">5.2<!--tex4ht:ref: tab:SL_Pathway_PI3K_InfoCent --></a> and  <a 
href="#x1-231005r2">K.2<!--tex4ht:ref: tab:mtSL_Pathway_PI3K_InfoCent --></a>). Thus neither method was unable to detect synthetic lethal genes with particular centrality constraints, although they were also not detecting genes with higher centrality than expected by chance. <a 
 id="x1-162003r232"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">5.2.2.2 </span> <a 
 id="x1-1630002"></a>PageRank Centrality
</h5>
<!--l. 239-->
<p class="noindent">
<p>PageRank centrality is another network analysis procedure to infer a hierarchy of gene importance from a network using connections and structure (<a 
href="#XBrin1998">Brin and Page</a>, <a 
href="#XBrin1998">1998</a>). In constrast to the information centrality approach of removing nodes, PageRank uses the eigenvalue properties of the adjacency matrix to rank genes according to the number of connections and paths they are involved in.</p>
</p>
<!--l. 241-->
<p class="indent">
<a 
 id="x1-163001r6"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 243-->
<p class="noindent">
</p>
<!--l. 244-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Centrality_exprSL/Pi3K_network_pagerank".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 5.6: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality and PageRank.</span></strong> <span class="cmr-10x-x-109">The PageRank centrality was</span> <span class="cmr-10x-x-109">compared (on a log-scale across genes deteced by mtSLIPT and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screening in</span> <span class="cmr-10x-x-109">the Reactome PI3K cascade pathway. Genes detected by </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">had a more restricted</span> <span class="cmr-10x-x-109">range of centrality values (which may be constrained experimental detection in a cell line</span> <span class="cmr-10x-x-109">model) than other genes not detected by either approach, although these groups also had</span> <span class="cmr-10x-x-109">fewer genes and a higher median. </span></span></p>
</div>
<!--tex4ht:label?: x1-163001r6 -->
</div>
<hr class="endfloat" />
<!--l. 257-->
<p class="indent">
<p>This distinction is immediately clear within the PI3K pathway (shown in Figure <a 
href="#x1-163001r6">5.6<!--tex4ht:ref: fig:SL_Pathway_PI3K_PageRank --></a>), which differs considerably from the information centrality scores. While genes not detected by either method had the highest centrality, genes detected by <a 
href="#glo:SLIPT">SLIPT</a> span the complete range of PageRank centrality values for this pathway. This was replicated (shown in Figure <a 
href="#x1-231003r3">K.3<!--tex4ht:ref: fig:mtSL_Pathway_PI3K_PageRank --></a>) when testing synthetic lethality against <span class="cmti-12">CDH1 </span>mutation (mtSLIPT). Thus <a 
href="#glo:SLIPT">SLIPT</a> is not biased towards genes with more crucial role in the pathway as inferred by pathway connectivity and centrality measures and it is therefore independent of pathway structure. However, the genes detected by <a 
href="#glo:siRNA">siRNA</a> screening have a higher median PageRank centrality, although the differences in PageRank centrality between these methods were not statistically significant as determined by <a 
href="#glo:ANOVA">ANOVA</a> (shown by Tables <a 
href="#x1-162002r2">5.2<!--tex4ht:ref: tab:SL_Pathway_PI3K_InfoCent --></a> and  <a 
href="#x1-231005r2">K.2<!--tex4ht:ref: tab:mtSL_Pathway_PI3K_InfoCent --></a>).</p>
</p>
<!--l. 260-->
<p class="indent">
<a 
 id="x1-163002r3"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 5.3: </span><span class="content"><a 
href="#glo:ANOVA">ANOVA</a> for Synthetic Lethality and PageRank Centrality</span></p>
</div>
<!--tex4ht:label?: x1-163002r3 -->
<div class="tabular">
<table id="TBL-74" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-74-1g">
<col 
id="TBL-74-1" />
<col 
id="TBL-74-2" />
<col 
id="TBL-74-3" />
<col 
id="TBL-74-4" />
<col 
id="TBL-74-5" />
<col 
id="TBL-74-6" />
</colgroup>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-74-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-74-1-1" class="td11">
<pre><code>          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-74-1-2&quot;  </code></pre>
class=“td11”&gt; DF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-1-3" class="td11">
Sum Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-1-4" class="td11">
Mean Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-1-5" class="td11">
F-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-1-6" class="td11">
p-value
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-74-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-74-2-1" class="td11">
siRNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-2-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-2-3" class="td11">
0.0002038
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-2-4" class="td11">
2<span class="cmmi-12">.</span>0385 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-2-5" class="td11">
1.1423
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-2-6" class="td11">
0.2892
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-74-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-74-3-1" class="td11">
SLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-3-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-3-3" class="td11">
0.0000208
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-3-4" class="td11">
2<span class="cmmi-12">.</span>0752 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-3-5" class="td11">
0.1163
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-3-6" class="td11">
0.7342
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-74-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-74-4-1" class="td11">
siRNA<span class="cmsy-10x-x-120">×</span>SLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-4-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-4-3" class="td11">
0.0000137
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-4-4" class="td11">
1<span class="cmmi-12">.</span>3743 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-4-5" class="td11">
0.0770
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-74-4-6" class="td11">
0.7823
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-74-5-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-74-5-1"  
class="td11">
<pre><code>           &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 284-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Analysis of variance for PageRank centrality against synthetic lethal detection approaches</span> <span class="cmr-10x-x-109">(with an interaction term)</span></p>
</p>
</div>
<hr class="endfloat" />
<a 
 id="x1-163003r227"></a>
<h3 class="sectionHead">
<span class="titlemark">5.3 </span> <a 
 id="x1-1640003"></a>Testing Pathway Structure of Synthetic Lethal Genes
</h3>
<a 
 id="x1-164001r231"></a>
<h4 class="subsectionHead">
<span class="titlemark">5.3.1 </span> <a 
 id="x1-1650001"></a>Hierarchical Pathway Structure
</h4>
<a 
 id="x1-165001r235"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">5.3.1.1 </span> <a 
 id="x1-1660001"></a>Contextual Hierarchy of PI3K
</h5>
<!--l. 300-->
<p class="noindent">
<p>A contextual hierarchy of genes in the PI3K pathway was performed (as described in in Secion <a 
href="#x1-1060002">3.4.1.2<!--tex4ht:ref: methods:pathway_rank --></a>) to assign scores for their relative order in the pathway. In the case of PI3K (shown in Figure <a 
href="#x1-166001r7">5.7<!--tex4ht:ref: fig:SL_Pathway_PI3K_Ranking --></a>), this orders genes from the upstream genes which respond to signals from extracellular stimuli to the downstream genes which transmit these to the gene expression (translation) responses of the cell. The directionality of this pathway is evident in transmitting signals from the <a 
href="#glo:PI3K">PI3K</a> complex, via AKT, <a 
href="#glo:PDE">PDE</a>, and mTOR to the ribosomal regulatory proteins. This hierarchical procedure enables testing whether the biological context of a gene in a pathway is relevant to detection as a synthetic lethal candidate by either computational <a 
href="#glo:SLIPT">SLIPT</a> analysis or experimental <a 
href="#glo:siRNA">siRNA</a> screening.</p>
</p>
<!--l. 302-->
<p class="indent">
<a 
 id="x1-166001r7"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 304-->
<p class="noindent">
</p>
<!--l. 305-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Discrete_Pi3k/graph_distance".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 5.7: </span><span class="content"><strong><span class="cmbx-10x-x-109">Structure of PI3K Ranking.</span></strong> <span class="cmr-10x-x-109">Structure of PI3K Ranking. </span></span></p>
</div>
<!--tex4ht:label?: x1-166001r7 -->
</div>
<hr class="endfloat" />
<a 
 id="x1-166002r240"></a>
<h5 class="subsubsectionHead">
<span class="titlemark">5.3.1.2 </span> <a 
 id="x1-1670002"></a>Testing Contextual Hierarchy of Synthetic Lethal Genes
</h5>
<!--l. 329-->
<p class="noindent">
<p>This pathway hierarchy in the <a 
href="#glo:PI3K">PI3K</a> cascade was tested for differences between genes detected across <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> screening. The synthetic lethal candidates for <span class="cmti-12">CDH1 </span>detected by either method (as shown by Figure <a 
href="#x1-167001r8">5.8<!--tex4ht:ref: fig:SL_Pathway_PI3K_Distance_Vioplot_Counts --></a>) did not differ, each being distributed throughout the pathway. The <a 
href="#glo:SLIPT">SLIPT</a> candidate genes were more numerous, there was little indication that they are more frequently upstream or downstream of <a 
href="#glo:siRNA">siRNA</a> candidate genes (as shown by Figure <a 
href="#x1-167002r9">5.9<!--tex4ht:ref: fig:SL_Pathway_PI3K_Distance_Barplot_Counts --></a>). Although <a 
href="#glo:SLIPT">SLIPT</a> genes included more with a lower (upstream) hierarchy. Synthetic lethal candidates from both methods were less frequently detected in the downstream effectors of the pathway (such as the mTOR complex), although core pathway genes (such as <span class="cmti-12">AKT2 </span>and <span class="cmti-12">PDE3B</span>) were detectable as synthetic lethal candidates (as discussed for Figure <a 
href="#x1-155001r1">5.1<!--tex4ht:ref: fig:SL_Pathway_Pi3K --></a>).</p>
</p>
<!--l. 331-->
<p class="indent">
<a 
 id="x1-167001r8"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 333-->
<p class="noindent">
</p>
<!--l. 334-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Discrete_Pi3k/SL_distance_counts_vioplot".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 5.8: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality and Hierarchy Score in PI3K.</span></strong> <span class="cmr-10x-x-109">The hierarchical</span> <span class="cmr-10x-x-109">distance scores were similarly distributed across </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes. </span></span></p>
</div>
<!--tex4ht:label?: x1-167001r8 -->
</div>
<hr class="endfloat" />
<!--l. 346-->
<p class="indent">
<a 
 id="x1-167002r9"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 348-->
<p class="noindent">
</p>
<!--l. 349-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Discrete_Pi3k/SL_distance_counts_barplot".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 5.9: </span><span class="content"><strong><span class="cmbx-10x-x-109">Hierarchy Score in PI3K against Synthetic Lethality in PI3K.</span></strong> <span class="cmr-10x-x-109">The</span> <span class="cmr-10x-x-109">number of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes against the hierarchical distance scores showing no</span> <span class="cmr-10x-x-109">significant tendency for either method to either of the pathway upstream or downstream</span> <span class="cmr-10x-x-109">extremities. </span></span></p>
</div>
<!--tex4ht:label?: x1-167002r9 -->
</div>
<hr class="endfloat" />
<!--l. 361-->
<p class="indent">
Similarly, when testing synthetic lethality against <span class="cmti-12">CDH1 </span>mutation (mtSLIPT), the hierarchical score for the <a 
href="#glo:PI3K">PI3K</a> pathway did not differ between mtSLIPT-specific and siRNA-specific gene candidates (as shown by Figure <a 
href="#x1-233001r1">M.1<!--tex4ht:ref: fig:mtSL_Pathway_PI3K_Distance_Vioplot_Counts --></a>). Although the median among genes detected by both approaches was elevated, that is further downstream in the pathway that other synthetic lethal candidates partners of <span class="cmti-12">CDH1</span>. This distinction is particularly notable since there were fewer genes overall with higher scores (shown in Figure <a 
href="#x1-233003r2">M.2<!--tex4ht:ref: fig:mtSL_Pathway_PI3K_Distance_Barplot_Counts --></a>), while these are more frequently detected by both mtSLIPT and siRNA.
</p>
<!--l. 363-->
<p class="indent">
<p>However, there was no significant effect variation in pathway hierarchy (shown by <a 
href="#glo:ANOVA">ANOVA</a> in Tables <a 
href="#x1-167003r4">5.4<!--tex4ht:ref: tab:SL_Pathway_PI3K_Distance_Counts --></a> and <a 
href="#x1-233002r1">M.1<!--tex4ht:ref: tab:mtSL_Pathway_PI3K_Distance_Counts --></a>) accounted for by SLIPT or siRNA detection in the PI3K pathway (as shown in Figure <a 
href="#x1-155001r1">5.1<!--tex4ht:ref: fig:SL_Pathway_Pi3K --></a>). Thus such differences in hierarchical scores may be observed by sampling variation and there is no indication that SLIPT or siRNA detection differs along the direction of the pathway. Genes detected by either method are no more or less common among upstream or downstream of the pathway.</p>
</p>
<!--l. 371-->
<p class="indent">
<a 
 id="x1-167003r4"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 5.4: </span><span class="content"><a 
href="#glo:ANOVA">ANOVA</a> for Synthetic Lethality and PI3K Hierarchy</span></p>
</div>
<!--tex4ht:label?: x1-167003r4 -->
<div class="tabular">
<table id="TBL-75" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-75-1g">
<col 
id="TBL-75-1" />
<col 
id="TBL-75-2" />
<col 
id="TBL-75-3" />
<col 
id="TBL-75-4" />
<col 
id="TBL-75-5" />
<col 
id="TBL-75-6" />
</colgroup>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-75-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-75-1-1" class="td11">
<pre><code>          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-75-1-2&quot;  </code></pre>
class=“td11”&gt; DF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-1-3" class="td11">
Sum Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-1-4" class="td11">
Mean Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-1-5" class="td11">
F-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-1-6" class="td11">
p-value
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-75-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-75-2-1" class="td11">
siRNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-2-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-2-3" class="td11">
<pre><code>0.001       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-75-2-4&quot;  </code></pre>
class=“td11”&gt; 0.00066
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-2-5" class="td11">
0.0004
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-2-6" class="td11">
0.9842
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-75-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-75-3-1" class="td11">
SLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-3-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-3-3" class="td11">
<pre><code>0.456       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-75-3-4&quot;  </code></pre>
class=“td11”&gt; 0.45605
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-3-5" class="td11">
0.2740
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-3-6" class="td11">
0.6016
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-75-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-75-4-1" class="td11">
siRNA<span class="cmsy-10x-x-120">×</span>SLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-4-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-4-3" class="td11">
<pre><code>0.019       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-75-4-4&quot;  </code></pre>
class=“td11”&gt; 0.01878
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-4-5" class="td11">
0.0113
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-75-4-6" class="td11">
0.9156
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-75-5-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-75-5-1"  
class="td11">
<pre><code>           &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 395-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Analysis of variance for PI3K hierarchy score against synthetic lethal detection approaches</span> <span class="cmr-10x-x-109">(with an interaction term)</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 397-->
<p class="indent">
<p>The pathway hierarchy may be applied here. A <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>-test was performed for the <a 
href="#glo:SLIPT">SLIPT</a> or <a 
href="#glo:siRNA">siRNA</a> candidate genes upstream or downstream of each gene. It is unsurprising that these <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> tests were more significant when the gene used as a threshold was in the middle of the pathway (as shown in Figure <a 
href="#x1-167004r10">5.10<!--tex4ht:ref: fig:SL_Pathway_PI3K_Distance_Vioplot --></a>). However, there was no statistically significant support for pathway structure by this approach as none of the <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> values were high enough to detect pathway structure between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates. Nor was structure detectable for mtSLIPT testing synthetic lethality against <span class="cmti-12">CDH1 </span>mutation (as shown in Figure <a 
href="#x1-233004r3">M.3<!--tex4ht:ref: fig:mtSL_Pathway_PI3K_Distance_Vioplot --></a>).</p>
</p>
<!--l. 401-->
<p class="indent">
<a 
 id="x1-167004r10"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 403-->
<p class="noindent">
</p>
<!--l. 404-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Discrete_Pi3k/SL_distance_vioplot".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 5.10: </span><span class="content"><strong><span class="cmbx-10x-x-109">Structure of Synthetic Lethality in PI3K.</span></strong> <span class="cmr-10x-x-109">The number of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and</span> <a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes upstream or downstream of each gene in the Reactome PI3K pathway were</span> <span class="cmr-10x-x-109">tested (by the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup><span class="cmr-10x-x-109">-test). These are plotted as a split violin plot against the hierarchical</span> <span class="cmr-10x-x-109">distance scores showing no significant tendency for either method to either of the pathway</span> <span class="cmr-10x-x-109">upstream or downstream extremities. </span></span></p>
</div>
<!--tex4ht:label?: x1-167004r10 -->
</div>
<hr class="endfloat" />
<a 
 id="x1-167005r239"></a>
<h4 class="subsectionHead">
<span class="titlemark">5.3.2 </span> <a 
 id="x1-1680002"></a>Upstream or Downstream Synthetic Lethality
</h4>
<!--l. 421-->
<p class="noindent">
However, this does not ascertain whether <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidate partners of <span class="cmti-12">CDH1 </span>are upstream or downstream of one and other within a pathway such as the <a 
href="#glo:PI3K">PI3K</a> cascade. The hierarchical approach is designed to detected differences in pathway location between gene groups. An alternative pathway structure method has been devised to use network structures to identify directional relationships between individual <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> genes. This pathway structure methodology will be applied (as described in Section <a 
href="#x1-1040001">3.4.1<!--tex4ht:ref: methods:pathway_str --></a>) to detect the direction of shortest paths between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates. This will be used to demonstrate the methodology on the <a 
href="#glo:PI3K">PI3K</a> pathway, to develop a statistical test for pathway structure between between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidate using resampling (as described in Section <a 
href="#x1-1050001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>, and to apply this test for pathway structure among synthetic lethal gene candidates to the pathways identified in Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a> and discussed in Section <a 
href="#x1-1540001">5.1<!--tex4ht:ref: chapt4:SL_Genes --></a>. <a 
 id="x1-168001r242"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">5.3.2.1 </span> <a 
 id="x1-1690001"></a>Measuring Structure of Candidates within PI3K
</h5>
<!--l. 427-->
<p class="noindent">
<p>Shortest paths in a pathway network were used to devise a strategy to detect pathway structure between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidate partners of <span class="cmti-12">CDH1 </span>(asdescribed in Section <a 
href="#x1-1040001">3.4.1<!--tex4ht:ref: methods:pathway_str --></a>). Thus we can determine whether individual <a 
href="#glo:SLIPT">SLIPT</a> genes have upstream or downstream <a 
href="#glo:siRNA">siRNA</a> candidates (scored as “up” or “down” events respectively). This procedure enables the detection of directional relationships between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates (in contrast to the hierarchical approach).</p>
</p>
<!--l. 431-->
<p class="indent">
<a 
 id="x1-169001r11"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 433-->
<p class="noindent">
</p>
<!--l. 434-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/test_PI3K_exprSL".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 5.11: </span><span class="content"><strong><span class="cmbx-10x-x-109">Structure of Synthetic Lethality Resampling in PI3K.</span></strong> <span class="cmr-10x-x-109">A</span> <span class="cmr-10x-x-109">null distribution with 10,000 iterations of the number of </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes upstream or</span> <span class="cmr-10x-x-109">downstream of </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">genes (depicted as the difference of these) in the PI3K pathway.</span> <span class="cmr-10x-x-109">To assess significance, the observed events (with shortest paths) were compared to the</span> <span class="cmr-10x-x-109">90% and 95% intervals for the null distribution (shown in violet). Genes detected by both</span> <span class="cmr-10x-x-109">methods were fixed to the same number as observed for the alternative null distribution</span> <span class="cmr-10x-x-109">(shown in blue), although the observed number of events (red) was not significant in</span> <span class="cmr-10x-x-109">either case. In both cases, these genes detected by both approaches were included in</span> <span class="cmr-10x-x-109">computing the number of shortest paths (in either direction) between </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes. </span></span></p>
</div>
<!--tex4ht:label?: x1-169001r11 -->
</div>
<hr class="endfloat" />
<!--l. 447-->
<p class="indent">
The total number of gene candidate pairs in either direction can be compared within a pathway network to assess the overall directional relationships in a pathway. This directionality is detectable by the difference between the number <a 
href="#glo:SLIPT">SLIPT</a> candidate genes with upstream and downstream <a 
href="#glo:siRNA">siRNA</a> gene partners. However, this measure alone is not sufficient to determine whether there is evidence of pathway structure between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidate partners of <span class="cmti-12">CDH1 </span>in a pathway network. Although it does serve to measure the magnitude (and direction) of the consensus of directional relationships (upstream and downstream) between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidate partners. This measure of pathway structure can be used for testing for statistical significance of pathway structure by resampling, using a permutation procedure to test whether these relationships are detectable among randomly selected gene groups rather than the detected <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidate partners (as described in Sections <a 
href="#x1-820006">2.3.6<!--tex4ht:ref: methods:permutation --></a> and <a 
href="#x1-1050001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>).
</p>
<!--l. 449-->
<p class="indent">
This resampling procedure was performed for the PI3K network (as shown in Figure <a 
href="#x1-155001r1">5.1<!--tex4ht:ref: fig:SL_Pathway_Pi3K --></a>) which generated a null distribution for the difference in the number of “up events” and “down events” for this Pathway. This provides a distribution to test whether more genes detected by <a 
href="#glo:SLIPT">SLIPT</a> had upstream or downstream <a 
href="#glo:siRNA">siRNA</a> candidates. While there was modest indication that glssiRNA genes were downstream of <a 
href="#glo:SLIPT">SLIPT</a> candidate genes, resampling for the PI3K pathway (as shown in Figure <a 
href="#x1-169001r11">5.11<!--tex4ht:ref: fig:SL_Pathway_PI3K_Perm --></a>) did not detect a significant number of <a 
href="#glo:siRNA">siRNA</a> genes upstream or downstream.
</p>
<!--l. 451-->
<p class="indent">
<p>In contrast, when testing synthetic lethality against <span class="cmti-12">CDH1 </span>mutation (mtSLIPT) there was modest indication that glssiRNA genes were upstream of <a 
href="#glo:SLIPT">SLIPT</a> candidate genes. However, resampling (as shown in Figure <a 
href="#x1-233005r4">M.4<!--tex4ht:ref: fig:mtSL_Pathway_PI3K_Perm --></a>) was also unable to detect a significant number of <a 
href="#glo:siRNA">siRNA</a> genes upstream or downstream of mtSLIPT candidates. Fixing the number of genes detected by both approaches (as shown by the blue line in Figures <a 
href="#x1-169001r11">5.11<!--tex4ht:ref: fig:SL_Pathway_PI3K_Perm --></a> and <a 
href="#x1-233005r4">M.4<!--tex4ht:ref: fig:mtSL_Pathway_PI3K_Perm --></a>) did not alter the findings of this approach. Nor did excluding these jointly detected genes, although these were included in the analysis since they can disproportionately count towards <a 
href="#glo:siRNA">siRNA</a> genes being upstream (or downstream) of <a 
href="#glo:SLIPT">SLIPT</a> genes since they have different proportions of gene detected by either approach upstream (or downstream) of them. Furthermore, expanding the range of shortest paths to consider links in related pathways (using the “metapathways” constructed in Section <a 
href="#x1-860003">2.4.3<!--tex4ht:ref: methods:subgraphs --></a>) also had little effect on the null distribution generated, despite increasing the computational demands of the procedure.</p>
<a 
 id="x1-169002r248"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">5.3.2.2 </span> <a 
 id="x1-1700002"></a>Resampling for Synthetic Lethal Pathway Structure
</h5>
<!--l. 458-->
<p class="noindent">
The permutation procedure (as described in Section <a 
href="#x1-1050001">3.4.1.1<!--tex4ht:ref: methods:network_permutation --></a>) that was performed in Section <a 
href="#x1-1690001">5.3.2.1<!--tex4ht:ref: chapt4:Structure_PI3K --></a> for the <a 
href="#glo:PI3K">PI3K</a> cascade was also applied to other pathways identified in Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a> and discussed in Section <a 
href="#x1-1540001">5.1<!--tex4ht:ref: chapt4:SL_Genes --></a>. These include extracellular matrix (with constituent elastic fibre and fibrin pathways), cell signalling (by PI3K/AKT and GCPRs), and translational pathways (with <a 
href="#glo:NMD">NMD</a> and 3<sup><span class="cmsy-8">′</span></sup><a 
href="#glo:UTR">UTR</a> regulation). The resampling results across these pathways (as shown in Table <a 
href="#x1-170001r5">5.5<!--tex4ht:ref: tab:pathway_str_exprSL --></a>) had limited support for pathway structure, with the majority of these being non-significant as shown for PI3K (in Figure <a 
href="#x1-233005r4">M.4<!--tex4ht:ref: fig:mtSL_Pathway_PI3K_Perm --></a>). However, the distribution for these pathways will differ depending on their structure, the number of genes they consist of, and the proportion of synthetic lethal candidates among them (including a higher frequency of genes detected by both methods pathways identified in Sections <a 
href="#x1-1280001">4.2.1.4.1<!--tex4ht:ref: chapt3:compare_pathway_perm --></a> and  <a 
href="#x1-1380001">4.4.3.1<!--tex4ht:ref: chapt3:compare_pathway_perm_stad_SL --></a>). This resampling is an appropriate procedure to use to detect structural relationships across pathways as it does not assume an underlying test statistic distribution.
</p>
<!--l. 460-->
<p class="indent">
<p>Pathway structure was supported for the <a 
href="#glo:NMD">NMD</a> pathway (which is consistent with <a 
href="#glo:siRNA">siRNA</a> being downstream in Figure <a 
href="#x1-230007r7">J.7<!--tex4ht:ref: fig:SL_Pathway_NMD --></a>). However, this observation rest upon a single gene and was not replicated when testing synthetic lethality (mtSLIPT) against <span class="cmti-12">CDH1 </span>mutation (as shown in Table <a 
href="#x1-233006r2">M.2<!--tex4ht:ref: tab:pathway_str_mtSL --></a>) or supported by the related 3<sup><span class="cmsy-8">′</span></sup><a 
href="#glo:UTR">UTR</a> regulation and translational elongation pathways.</p>
</p>
<!--l. 462-->
<p class="indent">
<a 
 id="x1-170001r5"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table 5.5: </span><span class="content">Resampling for pathway structure of synthetic lethal detection methods</span></p>
</div>
<!--tex4ht:label?: x1-170001r5 -->
<div class="tabular">
<pre><code>                                         &lt;table id=&quot;TBL-76&quot; class=&quot;tabular&quot; </code></pre>
cellspacing=“0” cellpadding=“0” rules=“groups” &gt;
<colgroup id="TBL-76-1g">
<col 
id="TBL-76-1" />
</colgroup>
<colgroup id="TBL-76-2g">
<col 
id="TBL-76-2" />
<col 
id="TBL-76-3" />
</colgroup>
<colgroup id="TBL-76-4g">
<col 
id="TBL-76-4" />
<col 
id="TBL-76-5" />
</colgroup>
<colgroup id="TBL-76-6g">
<col 
id="TBL-76-6" />
<col 
id="TBL-76-7" />
<col 
id="TBL-76-8" />
<col 
id="TBL-76-9" />
</colgroup>
<colgroup id="TBL-76-10g">
<col 
id="TBL-76-10" />
<col 
id="TBL-76-11" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-76-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-76-1-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-76-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-76-2-1" class="td11">
<pre><code>                                 &lt;/td&gt;&lt;td colspan=&quot;2&quot; style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-2-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><strong><span class="cmbx-12">Graph</span></strong></p>
</div>
<pre><code>   &lt;/td&gt;&lt;td colspan=&quot;2&quot; style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-2-4&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><strong><span class="cmbx-12">States</span></strong></p>
</div>
<pre><code>    &lt;/td&gt;&lt;td colspan=&quot;4&quot; style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-2-6&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><strong><span class="cmbx-12">Observed</span></strong></p>
</div>
<pre><code>                &lt;/td&gt;&lt;td colspan=&quot;2&quot; style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-2-10&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><strong><span class="cmbx-12">Permutation p-value</span></strong></p>
</div>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-76-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-76-3-1" class="td11">
<strong><span class="cmbx-12">Pathway</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-3-2" class="td11">
<strong><span class="cmbx-12">Nodes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-3-3" class="td11">
<strong><span class="cmbx-12">Edges</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-3-4" class="td11">
<strong><span class="cmbx-12">SLIPT</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-3-5" class="td11">
<strong><span class="cmbx-12">siRNA</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-3-6" class="td11">
<strong><span class="cmbx-12">Up</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-3-7" class="td11">
<strong><span class="cmbx-12">Down</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-3-8" class="td11">
<strong><span class="cmbx-12">Up</span><span class="cmsy-10x-x-120">-</span><span class="cmbx-12">Down</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-3-9" class="td11">
<strong><span class="cmbx-12">Up</span><span class="cmmi-12">∕</span><span class="cmbx-12">Down</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-3-10" class="td11">
<strong><span class="cmbx-12">Up</span><span class="cmsy-10x-x-120">-</span><span class="cmbx-12">Down</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-3-11" class="td11">
<strong><span class="cmbx-12">Down</span><span class="cmsy-10x-x-120">-</span><span class="cmbx-12">Up</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-76-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-76-4-1" class="td11">
PI3K Cascade
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-4-2" class="td11">
138
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-4-3" class="td11">
1495
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-4-4" class="td11">
38
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-4-5" class="td11">
25
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-4-6" class="td11">
122
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-4-7" class="td11">
128
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-4-8" class="td11">
<pre><code> -6        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-4-9&quot;  </code></pre>
class=“td11”&gt; 0.953
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-4-10" class="td11">
0.5326
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-4-11" class="td11">
0.4606
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-76-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-76-5-1" class="td11">
PI3K/AKT Signaling in Cancer
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-5-2" class="td11">
275
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-5-3" class="td11">
12882
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-5-4" class="td11">
98
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-5-5" class="td11">
44
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-5-6" class="td11">
779
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-5-7" class="td11">
679
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-5-8" class="td11">
<pre><code>100       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-5-9&quot;  </code></pre>
class=“td11”&gt; 1.147
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-5-10" class="td11">
0.3255
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-5-11" class="td11">
0.6734
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-76-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-76-6-1" class="td11">
<strong><span class="cmbx-12">G</span><sub><span class="cmmi-8">αi</span> </sub> <span class="cmbx-12">Signaling</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-6-2" class="td11">
292
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-6-3" class="td11">
22003
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-6-4" class="td11">
95
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-6-5" class="td11">
58
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-6-6" class="td11">
836
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-6-7" class="td11">
1546
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-6-8" class="td11">
<pre><code>-710       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-6-9&quot;  </code></pre>
class=“td11”&gt; 0.541
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-6-10" class="td11">
0.9971
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-6-11" class="td11">
0.0029
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-76-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-76-7-1" class="td11">
GPCR downstream
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-7-2" class="td11">
1270
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-7-3" class="td11">
142071
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-7-4" class="td11">
312
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-7-5" class="td11">
160
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-7-6" class="td11">
9755
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-7-7" class="td11">
9261
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-7-8" class="td11">
<pre><code>494       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-7-9&quot;  </code></pre>
class=“td11”&gt; 1.053
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-7-10" class="td11">
0.3692
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-7-11" class="td11">
0.6305
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-76-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-76-8-1" class="td11">
Elastic fibre formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-8-2" class="td11">
42
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-8-3" class="td11">
175
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-8-4" class="td11">
24
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-8-5" class="td11">
<pre><code>7      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-8-6&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-8-7" class="td11">
2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-8-8" class="td11">
<pre><code> -1        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-8-9&quot;  </code></pre>
class=“td11”&gt; 0.500
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-8-10" class="td11">
0.5461
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-8-11" class="td11">
0.3865
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-76-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-76-9-1" class="td11">
Extracellular matrix
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-9-2" class="td11">
299
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-9-3" class="td11">
3677
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-9-4" class="td11">
127
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-9-5" class="td11">
29
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-9-6" class="td11">
547
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-9-7" class="td11">
455
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-9-8" class="td11">
<pre><code> 92        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-9-9&quot;  </code></pre>
class=“td11”&gt; 1.202
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-9-10" class="td11">
0.3351
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-9-11" class="td11">
0.6636
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-76-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-76-10-1" class="td11">
Formation of Fibrin
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-10-2" class="td11">
52
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-10-3" class="td11">
243
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-10-4" class="td11">
18
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-10-5" class="td11">
<pre><code>5      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-10-6&quot;  </code></pre>
class=“td11”&gt; 12
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-10-7" class="td11">
17
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-10-8" class="td11">
<pre><code> -5        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-10-9&quot;  </code></pre>
class=“td11”&gt; 0.706
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-10-10" class="td11">
0.6198
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-10-11" class="td11">
0.3564
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-76-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-76-11-1" class="td11">
<strong><span class="cmbx-12">Nonsense-Mediated Decay</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-11-2" class="td11">
103
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-11-3" class="td11">
102
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-11-4" class="td11">
74
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-11-5" class="td11">
<pre><code>2      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-11-6&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-11-7" class="td11">
74
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-11-8" class="td11">
<pre><code> -74       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-11-9&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-11-10" class="td11">
1.0000
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-11-11" class="td11">
<span class="cmmi-12">&gt; </span>0<span class="cmmi-12">.</span>0001
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-76-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-76-12-1" class="td11">
3’ -UTR-mediated translational regulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-12-2" class="td11">
107
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-12-3" class="td11">
2860
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-12-4" class="td11">
77
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-12-5" class="td11">
<pre><code>1      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-12-6&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-12-7" class="td11">
0
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-12-8" class="td11">
<pre><code> 0         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-12-9&quot;  </code></pre>
class=“td11”&gt;
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-12-10" class="td11">
0.4902
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-12-11" class="td11">
0.5027
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-76-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-76-13-1" class="td11">
Eukaryotic Translation Elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-13-2" class="td11">
92
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-13-3" class="td11">
3746
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-13-4" class="td11">
76
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-13-5" class="td11">
<pre><code>0      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-13-6&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-13-7" class="td11">
0
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-13-8" class="td11">
<pre><code> 0         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-76-13-9&quot;  </code></pre>
class=“td11”&gt;
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-13-10" class="td11">
0.4943
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-76-13-11" class="td11">
0.4933
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-76-14-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-76-14-1"  
class="td11">
<pre><code>                                 &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 502-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Pathways in the Reactome network tested for structural relationships between </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes by resampling. The raw p-value (computed without adjusting for multiple comparisons</span> <span class="cmr-10x-x-109">over pathways) is given for the difference in upstream and downstream paths from </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">to </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">gene candidate partners of CDH1 with significant pathways highlighted in bold. Sampling was</span> <span class="cmr-10x-x-109">performed only in the target pathway and shortest paths were computed within it. Loops or paths in either direction that could not be resolved were excluded from the analysis. The gene</span> <span class="cmr-10x-x-109">detected by both </span><a 
href="#glo:SLIPT"><span class="cmr-10x-x-109">SLIPT</span></a> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">(or resampling for them) were includued in the analysis and the number of these were fixed to the number observed.</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 505-->
<p class="indent">
There does not appear to be a consensus on the directionality of <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidates across pathways as distinct pathways showed stronger tendency for <a 
href="#glo:siRNA">siRNA</a> genes to be either upstream or downstream. Even related pathways such as <a 
href="#glo:PI3K">PI3K</a> and PI3K/AKT signalling showed directional events in opposite directions. The strongest pathway (among those tested) with support for directional pathways structure is G<sub><span class="cmmi-8">αi</span></sub> signaling which showed significant downstream <a 
href="#glo:siRNA">siRNA</a> genes for both SLIPT and mtSLIPT from a large number of shortest paths (in Tables <a 
href="#x1-170001r5">5.5<!--tex4ht:ref: tab:pathway_str_exprSL --></a> and  <a 
href="#x1-233006r2">M.2<!--tex4ht:ref: tab:pathway_str_mtSL --></a>).However, these results are borderline significant (with raw permutation p-values) and are unlikely to be detected after adjusting for multiple comparisons.
</p>
<!--l. 507-->
<p class="indent">
Therefore, there is insufficent evidence to determine whether there is pathway structure between the <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidates observed in many pathways. In particular, directional structure among synthetic lethal candidates for <span class="cmti-12">CDH1 </span>was not strongly supported in signalling pathways upon which the rationale for pathway structure hypotheses were based on. Despite the design of a robust resampling approach to test relationships between gene groups, this did not detect many structural relationships between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates, although it may apply more broadly to gene networks. Furthermore, the pathway relationships are unlikely to be statistically supported by resampling when testing across the search space of Reactome pathways and adjusting for multiple comparisons. While there is statistically significant over-representation of many of these pathways in gene detected by both <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> (as described in Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a>), these did not show pathway structure, nor does pathway structure account for the discrepancy between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates which did not significantly intersect such as the <a 
href="#glo:PI3K">PI3K</a> cascade. <a 
 id="x1-170002r238"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">5.4 </span> <a 
 id="x1-1710004"></a>Discussion
</h3>
<!--l. 514-->
<p class="noindent">
<p>Synthetic lethal genes and pathways (for <span class="cmti-12">CDH1 </span>loss in cancer) were identified across gene expression and mutation datasets in Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a>. These pathway structure investigations extend those investigations into synthetic lethal gene candidates including exploring the discrepancy between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> candidates genes in a pathway such as <a 
href="#glo:PI3K">PI3K</a> in which they did not significantly intersect. Pathways with replicated synthetic lethal genes across these detection methods, breast and stomach cancer data, and patient and cell line</p>
data were also investigated including pathways from the extracellular microenvironment to core translational pathway and the signalling pathways which mediate between them.
</p>
<!--l. 516-->
<p class="indent">
Many genes were detected by either method and the differences between the computational and experimental screening approaches could feasibly lead to differences in which genes within a synthetic lethal pathway are identified. Genes detected by synthetic lethal detection strategies included those biological importance within synthetic lethal pathways, those which are actionable drug targets, and those with functional implications for the biological growth mechanisms or vulnerabilities of <span class="cmti-12">CDH1 </span>deficient tumours. It appeared that genes detected by both approaches were highly connected (or of importance) in the network structure or some pathways and that there may be some structure with <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> upstream or downstream of each other. However, the complexity of biological pathways meant that they are not reliably interpretable so formal mathematical and computational approaches are needed to analyse large biological networks.
</p>
<!--l. 518-->
<p class="indent">
Network analysis techniques were therefore applied to formalise and quantify the connectivity and importance (centrality) of genes within pathways (using PI3K as an example). However, these network techniques were unable to identify distinct differences in the network properties of genes detected as synthetic lethal candidates by computational or experimental methods. These network metrics support the application of synthetic detection across pathways (and the findings using pathways as gene sets in Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a>) as neither synthetic lethal detection approach was biased towards genes of higher importance or connectivity and neither approach was insensitive to genes of lower importance or connectivity.
</p>
<!--l. 520-->
<p class="indent">
Similarly, a network hierarchy based on biological context (ordered from receiving extracellular stimuli to affecting downstream gene expression and cell growth) was devised to test whether PI3K genes of a particular upstream or downstream level were more frequently detected as synthetic lethal candidates. However, this approach was unable to ascertain whether genes detected by either method were further upstream or downstream in the pathway and there was no statistical evidence that either method differed in which levels of this structure were detected.
</p>
<!--l. 522-->
<p class="indent">
<p>A measure of pathway structure between individual <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> genes within a pathway was also devised using the direction of shortest paths in a directed graph structure. This is amenable to detecting the consensus directionality of the pathway across pairs of genes detected by either method. The pathway structure methodology</p>
developed here is generally applicable to comparison of node groups (allowing overlapping) including genes in biological pathways and their detection by different methodologies. While the pathway structure measure alone is not able to detect structural relationships between gene groups (such <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates), it is amenable to resampling to determine whether these relationships are statistically significant. <a 
 id="x1-171001r252"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">5.5 </span> <a 
 id="x1-1720005"></a>Summary
</h3>
<!--l. 526-->
<p class="noindent">
Together these analyses of biological pathways, network metrics, and statistical procedures devised specifically for purpose were applied to Reactome pathway structures to test whether structural relationships exist between synthetic lethal candidates. Of particular interest was whether these relationships be related to the differences between the computational (<a 
href="#glo:SLIPT">SLIPT</a>) and experimental (<a 
href="#glo:siRNA">siRNA</a>) synthetic lethal candidate partners of <span class="cmti-12">CDH1 </span>(in the pathways discussed in Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a>).
</p>
<!--l. 528-->
<p class="indent">
While biologically relevant relationships were observed in specific pathways, there were not detectable structural relationships between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates. These candidates did not exhibited significant differences in network connectivity or centrality measures. Network analyses were also unable to ascertain whether the candidates detected by either method stratified into upstream and downstream genes on the pathway and they likely do not.
</p>
<!--l. 530-->
<p class="indent">
A statistical resampling procedure was applied to shortest paths to test whether pairs of <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates were more likely to be upstream or downstream of each other. This approach detected very few structural relationships in the synthetic lethal pathways identified in Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a>. Overall, support for pathway structure between <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates is weak and the direction is inconsistent between pathways. Therefore pathway structure does not account for the differences between the <a 
href="#glo:SLIPT">SLIPT</a> and <a 
href="#glo:siRNA">siRNA</a> gene candidates, although this does support the validity of gene set analyses in Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a> and the synthetic lethal pathways identified.
</p>
<!--l. 532-->
<p class="indent">
<p>Furthermore, the resampling procedure demonstrated in this Chapter is more widely applicable to gene states in network structures and may be further utility in the analysis of biological pathway or networks. This approach was able to quantify structural relationships that were otherwise difficult to interpret and to conclusively exclude many potential</p>
<p>relationships. In this respect, the network resampling methdology may also be applicable to triage of experimental validation.</p>
</p>
<!--l. 536-->
<p class="noindent">
<span class="paragraphHead"><a 
 id="x1-1730005"></a><span class="cmbx-12">Aims</span></span>
</p>
<ul class="itemize1">
<li class="itemize">
Synthetic Lethal Genes within a Biological Pathway Structure
</li>
<li class="itemize">
Importance and Connectivity of Synthetic Lethal Genes within Pathway Networks
</li>
<li class="itemize">
Upstream and Downstream Relationships between SLIPT and siRNA Candidates
</li>
</ul>
<!--l. 550-->
<p class="noindent">
<span class="paragraphHead"><a 
 id="x1-1740005"></a><span class="cmbx-12">Summary</span></span>
</p>
<ul class="itemize1">
<li class="itemize">
Synthetic Lethal genes were explored within a graph structures for key pathways identified previously
</li>
<li class="itemize">
In some cases these graph structures appeared to have relationships between synthetic lethal genes
</li>
<li class="itemize">
However, no existing network metrics of importance and connectivity with the networks were elevated significantly for Synthetic Lethal genes
</li>
<li class="itemize">
Nor was there significant evidence of upstream and downstream relationships between SLIPT and siRNA Candidates in a shortest path permutation analysis
</li>
</ul>
<h2 class="chapterHead">
<span class="titlemark">Chapter 6</span><br /><a 
 id="x1-1750006"></a>Simulation and Modeling of Synthetic Lethal Pathways
</h2>
<!--l. 4-->
<p class="noindent">
Simulation and modelling of synthetic lethality in gene expression will be revisited in greater detail in this chapter, building upon the results provided to support the use of <a 
href="#glo:SLIPT">SLIPT</a> in Section <a 
href="#x1-970003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>. A simulation procedure for generating simulated data with underlying (known) synthetic lethal partners of a query gene, such as <span class="cmti-12">CDH1</span>, was developed (as described in Section <a 
href="#x1-960002">3.2.2<!--tex4ht:ref: methods:simulating_SL --></a>) by sampling from a Multivariate normal distribution based on a statistical model of synthetic lethality in expression data (as described in Section <a 
href="#x1-950001">3.2.1<!--tex4ht:ref: methods:SL_Model --></a>). This simulation framework was applied to simulated data (in Section <a 
href="#x1-970003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>), including simple correlation structures to assess the statistical performance of the <a 
href="#glo:SLIPT">SLIPT</a> methodology and support it’s use a computational approach for detecting synthetic lethal candidates from expression data throughout this thesis (in Chapters <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a> and <a 
href="#x1-1530005">5<!--tex4ht:ref: chap:Pathways --></a>).
</p>
<!--l. 6-->
<p class="indent">
While this basic framework was sufficent to support the use of <a 
href="#glo:SLIPT">SLIPT</a> in prior Chapters, further investigations with simulations were conducted to assess the strengths and limitations of the <a 
href="#glo:SLIPT">SLIPT</a> methodology, compare it to alternative statistical approaches to synthetic lethal detection, and assess it’s performance upon more complex correlation structures. Together these simulation investigations assess the performance of the <a 
href="#glo:SLIPT">SLIPT</a> methdology, including on pathway graph structures (such as those discussed in Chapter <a 
href="#x1-1530005">5<!--tex4ht:ref: chap:Pathways --></a>) and determine whether the <a 
href="#glo:SLIPT">SLIPT</a> methdology (or similar refined bioinformatics strategies) are statistically rigourous or suitable for wider genomics applications.
</p>
<!--l. 8-->
<p class="indent">
These simulation investigations continue to utilise the Multivariate Normal procedure (as applied in Section <a 
href="#x1-970003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>) with further refinements. The <a 
href="#glo:SLIPT">SLIPT</a> methodology (and it’s equivalent <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span> </sup> test) were applied across a range of parameters (including altering the quantiles for detecting synthetic lethal direction and compared correlation. This was also applied to with query correlated genes (as performed in Section <a 
href="#x1-970003">3.3<!--tex4ht:ref: chapt2:simulation_2015 --></a>).
</p>
<!--l. 10-->
<p class="indent">
<p>A refined simulation procedure was developed specifically to extend the simulation procedure (described in Section <a 
href="#x1-940002">3.2<!--tex4ht:ref: methods:simulation_SL_expression --></a>) to utilise pathway graph structures for the correlation structures of simulated datasets (as described in Section <a 
href="#x1-1070002">3.4.2<!--tex4ht:ref: methods:graphsim --></a>). This methdology can be applied to simulated correlation structures across simple graph structures to test specfic network modules or use pathway structures based on biological pathways (as discussed in Chapter <a 
href="#x1-1530005">5<!--tex4ht:ref: chap:Pathways --></a>). Thus graph structure and simulation approaches were combined to test whether a gene locus in a pathway affects detection by <a 
href="#glo:SLIPT">SLIPT</a> and whether <a 
href="#glo:SLIPT">SLIPT</a> performance is affected by pathway structure. The simulation procedure based on graph structures were applied in a computational pipeline across many parameters</p>
with high-performance computing (as discussed in Section <a 
href="#x1-910003">2.5.3<!--tex4ht:ref: methods:HPC --></a>) and the core simulation functions have been released as a software package for wider use to test bioinformatics and statistical methods on graph structures (as described in Section <a 
href="#x1-1110003">3.5.3<!--tex4ht:ref: methods:igraph_extensions --></a>). <a 
 id="x1-175001r253"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">6.1 </span> <a 
 id="x1-1760001"></a>Comparing methods
</h3>
<!--l. 22-->
<p class="noindent">
Methods were compared … <a 
 id="x1-176001r247"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.1.1 </span> <a 
 id="x1-1770001"></a>Performance of SLIPT and <span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> across Quantiles
</h4>
<!--l. 26-->
<p class="noindent">
Text <a 
 id="x1-177001r250"></a>
</p>
<h5 class="subsubsectionHead">
<span class="titlemark">6.1.1.1 </span> <a 
 id="x1-1780001"></a>Correlated Query Genes affects Specificity
</h5>
<!--l. 30-->
<p class="noindent">
Text <a 
 id="x1-178001r258"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.1.2 </span> <a 
 id="x1-1790002"></a>Correlation as a Synthetic Lethal Detection Strategy
</h4>
<!--l. 34-->
<p class="noindent">
Text <a 
 id="x1-179001r260"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.1.3 </span> <a 
 id="x1-1800003"></a>Testing for Bimodality with BiSEp
</h4>
<!--l. 38-->
<p class="noindent">
Text <a 
 id="x1-180001r257"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">6.2 </span> <a 
 id="x1-1810002"></a>Simulations with Graph Structures
</h3>
<!--l. 53-->
<p class="noindent">
<p>Text</p>
</p>
<!--l. 55-->
<p class="indent">
<a 
 id="x1-181003r1"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 58-->
<p class="noindent">
<a 
 id="x1-181001r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<span class="fbox"> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph1".png /> </span>
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Activating</span> <span class="cmr-10">pathway</span> <span class="cmr-10">structure</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-181002r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<span class="fbox"> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph1i".png /> </span>
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Pathway</span> <span class="cmr-10">structure</span> <span class="cmr-10">including</span> <span class="cmr-10">inhibitions</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.1: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulated graph structures.</span></strong></strong> <span class="cmr-10x-x-109">A constructed graph structure used as an</span> <span class="cmr-10x-x-109">example to demonstrate the simulation procedure. Activating links are denoted by blue</span> <span class="cmr-10x-x-109">arrows and inhibiting links by red edges.</span></span></p>
</div>
<!--tex4ht:label?: x1-181003r1 -->
</div>
<hr class="endfloat" />
<!--l. 80-->
<p class="indent">
<a 
 id="x1-181006r2"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 83-->
<p class="noindent">
<a 
 id="x1-181004r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<span class="fbox"> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph2".png /> </span>
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Activating</span> <span class="cmr-10">pathway</span> <span class="cmr-10">structure</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-181005r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<span class="fbox"> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph2i".png /> </span>
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Pathway</span> <span class="cmr-10">structure</span> <span class="cmr-10">including</span> <span class="cmr-10">inhibitions</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.2: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulated graph structures.</span></strong></strong> <span class="cmr-10x-x-109">A constructed graph structure used as an</span> <span class="cmr-10x-x-109">example to demonstrate the simulation procedure. Activating links are denoted by blue</span> <span class="cmr-10x-x-109">arrows and inhibiting links by red edges.</span></span></p>
</div>
<!--tex4ht:label?: x1-181006r2 -->
</div>
<hr class="endfloat" />
<!--l. 105-->
<p class="indent">
<a 
 id="x1-181009r3"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 108-->
<p class="noindent">
<a 
 id="x1-181007r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<span class="fbox"> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph3".png /> </span>
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Activating</span> <span class="cmr-10">pathway</span> <span class="cmr-10">structure</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-181008r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<span class="fbox"> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph3i".png /> </span>
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Pathway</span> <span class="cmr-10">structure</span> <span class="cmr-10">including</span> <span class="cmr-10">inhibitions</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.3: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulated graph structures.</span></strong></strong> <span class="cmr-10x-x-109">A constructed graph structure used as an</span> <span class="cmr-10x-x-109">example to demonstrate the simulation procedure. Activating links are denoted by blue</span> <span class="cmr-10x-x-109">arrows and inhibiting links by red edges.</span></span></p>
</div>
<!--tex4ht:label?: x1-181009r3 -->
</div>
<hr class="endfloat" />
<!--l. 130-->
<p class="indent">
<a 
 id="x1-181013r4"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 133-->
<p class="noindent">
<a 
 id="x1-181010r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<span class="fbox"> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4".png /> </span>
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Activating</span> <span class="cmr-10">pathway</span> <span class="cmr-10">structure</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-181011r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<span class="fbox"> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4i".png /> </span>
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Pathway</span> <span class="cmr-10">structure</span> <span class="cmr-10">including</span> <span class="cmr-10">inhibitions</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-181012r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<span class="fbox"> <img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph4i2".png /> </span>
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmr-10">Pathway</span> <span class="cmr-10">structure</span> <span class="cmr-10">including</span> <span class="cmr-10">inhibitions</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.4: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulated graph structures.</span></strong></strong> <span class="cmr-10x-x-109">A constructed graph structure used as an</span> <span class="cmr-10x-x-109">example to demonstrate the simulation procedure. Activating links are denoted by blue</span> <span class="cmr-10x-x-109">arrows and inhibiting links by red edges.</span></span></p>
</div>
<!--tex4ht:label?: x1-181013r4 -->
</div>
<hr class="endfloat" />
<!--l. 162-->
<p class="indent">
<a 
 id="x1-181014r5"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 164-->
<p class="noindent">
</p>
<!--l. 165-->
<p class="noindent">
<span class="fbox"><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph5".png /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.5: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulated graph structures.</span></strong></strong> <span class="cmr-10x-x-109">A constructed graph structure used as an</span> <span class="cmr-10x-x-109">example to demonstrate the simulation procedure. Activating links are denoted by blue</span> <span class="cmr-10x-x-109">arrows and inhibiting links by red edges.</span></span></p>
</div>
<!--tex4ht:label?: x1-181014r5 -->
</div>
<hr class="endfloat" />
<!--l. 174-->
<p class="indent">
<a 
 id="x1-181015r6"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 176-->
<p class="noindent">
</p>
<!--l. 177-->
<p class="noindent">
<span class="fbox"><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph7".png /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.6: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulated graph structures.</span></strong></strong> <span class="cmr-10x-x-109">A constructed graph structure used as an</span> <span class="cmr-10x-x-109">example to demonstrate the simulation procedure. Activating links are denoted by blue</span> <span class="cmr-10x-x-109">arrows and inhibiting links by red edges.</span></span></p>
</div>
<!--tex4ht:label?: x1-181015r6 -->
</div>
<hr class="endfloat" />
<!--l. 186-->
<p class="indent">
<a 
 id="x1-181016r7"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 188-->
<p class="noindent">
</p>
<!--l. 189-->
<p class="noindent">
<span class="fbox"><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph7i".png /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.7: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulated graph structures.</span></strong></strong> <span class="cmr-10x-x-109">A constructed graph structure used as an</span> <span class="cmr-10x-x-109">example to demonstrate the simulation procedure. Activating links are denoted by blue</span> <span class="cmr-10x-x-109">arrows and inhibiting links by red edges.</span></span></p>
</div>
<!--tex4ht:label?: x1-181016r7 -->
</div>
<hr class="endfloat" />
<!--l. 198-->
<p class="indent">
<a 
 id="x1-181017r8"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 200-->
<p class="noindent">
</p>
<!--l. 201-->
<p class="noindent">
<span class="fbox"><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph7i2".png /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.8: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulated graph structures.</span></strong></strong> <span class="cmr-10x-x-109">A constructed graph structure used as an</span> <span class="cmr-10x-x-109">example to demonstrate the simulation procedure. Activating links are denoted by blue</span> <span class="cmr-10x-x-109">arrows and inhibiting links by red edges.</span></span></p>
</div>
<!--tex4ht:label?: x1-181017r8 -->
</div>
<hr class="endfloat" />
<!--l. 210-->
<p class="indent">
<a 
 id="x1-181018r9"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 212-->
<p class="noindent">
</p>
<!--l. 213-->
<p class="noindent">
<span class="fbox"><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph_pi3k".png /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.9: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulated graph structures.</span></strong></strong> <span class="cmr-10x-x-109">A constructed graph structure used as an</span> <span class="cmr-10x-x-109">example to demonstrate the simulation procedure. Activating links are denoted by blue</span> <span class="cmr-10x-x-109">arrows and inhibiting links by red edges.</span></span></p>
</div>
<!--tex4ht:label?: x1-181018r9 -->
</div>
<hr class="endfloat" />
<!--l. 222-->
<p class="indent">
<a 
 id="x1-181019r10"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 224-->
<p class="noindent">
</p>
<!--l. 225-->
<p class="noindent">
<span class="fbox"><img 
src=""/home/tomkelly/Documents/PhD.png" alt="pict"  
Otago Uni/SL_Model/Graph_Gai".png /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure 6.10: </span><span class="content"><strong><strong><span class="cmbx-10x-x-109">Simulated graph structures.</span></strong></strong> <span class="cmr-10x-x-109">A constructed graph structure used as</span> <span class="cmr-10x-x-109">an example to demonstrate the simulation procedure. Activating links are denoted by</span> <span class="cmr-10x-x-109">blue arrows and inhibiting links by red edges.</span></span></p>
</div>
<!--tex4ht:label?: x1-181019r10 -->
</div>
<hr class="endfloat" />
<a 
 id="x1-181020r261"></a>
<h4 class="subsectionHead">
<span class="titlemark">6.2.1 </span> <a 
 id="x1-1820001"></a>Performance over a Graph Structure
</h4>
<!--l. 239-->
<p class="noindent">
Text <a 
 id="x1-182001r273"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.2.2 </span> <a 
 id="x1-1830002"></a>Synthetic Lethality across Graph Stuctures
</h4>
<!--l. 245-->
<p class="noindent">
Text <a 
 id="x1-183001r274"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.2.3 </span> <a 
 id="x1-1840003"></a>Performance with inhibition links
</h4>
<!--l. 251-->
<p class="noindent">
Text <a 
 id="x1-184001r275"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">6.2.4 </span> <a 
 id="x1-1850004"></a>Performance with 20,000 genes
</h4>
<!--l. 257-->
<p class="noindent">
Text <a 
 id="x1-185001r262"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">6.3 </span> <a 
 id="x1-1860003"></a>Simulations over pathway-based graphs
</h3>
<!--l. 263-->
<p class="noindent">
Text <a 
 id="x1-186001r277"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">6.4 </span> <a 
 id="x1-1870004"></a>Discussion
</h3>
<!--l. 268-->
<p class="noindent">
Text <a 
 id="x1-187001r278"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">6.5 </span> <a 
 id="x1-1880005"></a>Summary
</h3>
<!--l. 273-->
<p class="noindent">
<p>Text</p>
</p>
<!--l. 278-->
<p class="noindent">
<span class="paragraphHead"><a 
 id="x1-1890005"></a><span class="cmbx-12">Aims</span></span>
</p>
<ul class="itemize1">
<li class="itemize">
A Model of Synthetic Lethal Genes in Gene Expression Data
</li>
<li class="itemize">
Comparison of SLIPT to Alternative Approaches
</li>
<li class="itemize">
Simulations of Known Synthetic Lethal Genes within Pathway Networks
</li>
</ul>
<!--l. 293-->
<p class="noindent">
<span class="paragraphHead"><a 
 id="x1-1900005"></a><span class="cmbx-12">Summary</span></span>
</p>
<ul class="itemize1">
<li class="itemize">
We have designed a straight-forward rational query-based synthetic lethal detection method with the example of application to <span class="cmti-12">CDH1 </span>in cancer gene expression
</li>
<li class="itemize">
I have developed a simulation pipeline to generate continuous gene expression with pathway structure including a procedure to simulate synthetic lethality
</li>
<li class="itemize">
The simulation procedure shows that SLIPT is robust across pathway structures and has desirable performance compared to other statistical techniques
</li>
</ul>
<h2 class="chapterHead">
<span class="titlemark">Chapter 7</span><br /><a 
 id="x1-1910007"></a>Discussion
</h2>
<!--l. 4-->
<p class="noindent">
<span class="paragraphHead"><a 
 id="x1-1920007"></a><span class="cmbx-12">Aims</span></span>
</p>
<ul class="itemize1">
<li class="itemize">
To develop a statistical approach to detect synthetic lethal gene pairs in cancer from expression data
</li>
<li class="itemize">
To apply this methodology to public cancer gene expression data against <span class="cmti-12">CDH1</span> and analyse pathway structure with comparisons to experimental screen data
</li>
<li class="itemize">
To construct a statistical model of synthetic lethality in multivariate normal expression data
</li>
<li class="itemize">
To develop a simulation pipeline of expression with pathway structure on a high-performance computing cluster
</li>
<li class="itemize">
To examine the statistical performance of the methodology with simulated expression including pathways and compare it to other approaches
</li>
<li class="itemize">
To release the synthetic lethal detection methodology and pathway simulation procedure as R software packages
</li>
</ul>
<!--l. 31-->
<p class="noindent">
<span class="paragraphHead"><a 
 id="x1-1930007"></a><span class="cmbx-12">Summary</span></span>
</p>
<ul class="itemize1">
<li class="itemize">
We have developed a Synthetic Lethal detection method that generates a high number of synthetic lethal candidates
</li>
<li class="itemize">
Pathways in cell signalling, extracellular matrix, and cytoskeletal functions were supported with experimental candidates and the known functions of E-cadherin
</li>
<li class="itemize">
Several candidate pathways were supported by mutation analysis and replicated across breast and stomach cancer
</li>
<li class="itemize">
Translation and immune functions were uniquely detected by the computational approach which may be explained by differences between patient samples and cell line models
</li>
<li class="itemize">
There remains the need to identify actionable genes within these pathways, relationships with experimental candidates, and how these pathways may affect viability when lost
</li>
</ul>
<ul class="itemize1">
<li class="itemize">
Synthetic Lethal genes were explored within a graph structures for key pathways identified previously
</li>
<li class="itemize">
In some cases these graph structures appeared to have relationships between synthetic lethal genes
</li>
<li class="itemize">
<p>However, no existing network metrics of importance and connectivity with the networks were elevated significantly for Synthetic Lethal genes</p>
</li>
<li class="itemize">
Nor was there significant evidence of upstream and downstream relationships between SLIPT and siRNA Candidates in a shortest path permutation analysis
</li>
</ul>
<ul class="itemize1">
<li class="itemize">
We have designed a straight-forward rational query-based synthetic lethal detection method with the example of application to <span class="cmti-12">CDH1 </span>in cancer gene expression
</li>
<li class="itemize">
We have developed a simulation pipeline to generate continuous gene expression with pathway structure including a procedure to simulate synthetic lethality
</li>
<li class="itemize">
Our simulation procedure is robust across pathway structures and has desirable performance compared to other statistical techniques
</li>
</ul>
<a 
 id="x1-193001r279"></a>
<h3 class="sectionHead">
<span class="titlemark">7.1 </span> <a 
 id="x1-1940001"></a>Significance
</h3>
<p>Development of an effective synthetic lethal discovery tool for bioinformatics analysis has a wide range of applications in genetics research including functional genomics, medical and agricultural applications. Of particular interest is a complementary approach to discovery of synthetic lethal drug targets for cancer therapy to aid the cancer research community which currently relies on cell line and mouse models for screening and validation experiments (Fece de la Cruz et al. 2015). The potential for synthetic lethal drug design against cancer mutations including gene loss or overexpression could lead to a revolution in cancer therapy and chemoprevention with personalised treatment of cancers and high risk individuals. Examples of the synthetic lethal strategy to cancer treatment have been shown to be clinically effective with many large-scale RNAi screens underway to discover more cancer gene function and drug targets for similar application. However, there are limitations to both experimental screens and computational approaches, both known to be prone to false-positives. Modelling and simulation of synthetic</p>
<p>lethal discovery in genomic data has been explored to address these concerns and ensure the validity of candidate synthetic lethal interactions, particularly given the recent emergence of a number of conflicting synthetic lethal screening and prediction approaches. Exploring synthetic lethality in simulated data will ensure the optimal performance of our prediction method with comparison to the distribution of test statistic distribution in empirical gene expression data, informed selection of thresholds for prediction, and estimated error rates. The model of gene expression with known synthetic lethal genes is limited by the assumption that it represents the distribution of gene expression when it may not. Having shown synthetic lethality is detectable in simple models and added correlation structure, the model still needs to be developed to better represent real data. However, the behaviour of synthetic lethal genes and effects of parameters explored so far remains important to inform future model design and interpretation of empirical data analysis. The synthetic lethal discovery strategy could be adapted to any form of gene inactivation or disruption such as such as changes to gene expression, regulation, epigenetics, DNA sequence, or copy number which could plausibly induce cell death due to SL interactions. Further applications of synthetic lethal interactions such as analysis of gene networks, tissue specificity, evolutionary conservation, or drug target feasibility are possible with synthetic lethal candidates predicted with confidence on a large scale. Network analysis enables properties of the network and its connectivity to be measured and compared across datasets (Barabsi &amp; Oltvai 2004). Tissue specificity is an important consideration, largely unexplored with synthetic lethal studies, since it has clinical importance to ensure targeted drug treatments are effective, predict adverse effects in other tissues, determine whether targeted treatments could be repurposed for other cancer types or diseases, and whether drug resistance mechanisms could emerge. Comparison of tissues, populations, and species can all ensure that synthetic lethal predictions are robust, that experimental candidates are clinically relevant, and treatments designed to exploit them would be specific to the disease in large patient cohort (with known biomarkers). Drug targets must be feasible to have effective anti-cancer interventions designed against them, which raises the need for targets with existing drugs in the clinic, trials, or feasible to development with structural analysis or screening. Druggable targets could be selected by gene functions known to be amendable to drugs, with a structure amenable with development, with conserved specific sites without homology to other genes, or with</p>
known approval or developing drugs which could be repurposed from other disease applications. <a 
 id="x1-194001r285"></a>
<h3 class="sectionHead">
<span class="titlemark">7.2 </span> <a 
 id="x1-1950002"></a>Future Directions
</h3>
<!--l. 98-->
<p class="noindent">
Such a bioinformatically-informed synthetic lethal screening and validation strategy could be integrated into existing and future screens for synthetic lethality in cancer.
</p>
<!--l. 100-->
<p class="indent">
Possible improvements to the SLIPT method include developing a Bayesian inference method or simulations and modelling to account for pathway structure among synthetic lethal genes. Another extension would be to test for higher order synthetic lethal interactions, where 3 or more genes perform a redundant function.
</p>
<!--l. 105-->
<p class="indent">
Further development of the synthetic lethal model and simulation is needed to explore the parameters, ensure relevance to empirical data analysis, and understanding the implications of findings so far. An example of more complex correlation structure is shown in supplementary Figures S1 and S2 with genes correlated to the Query genes (showing need for directional synthetic lethal condition) and correlated with other non-synthetic lethal genes (showing the predictions are robust to other correlation structure). The impact of these modifications on model performance in a large number of genes or simulation replicates is yet to be seen or whether such correlation structure reflects the correlation structure of empirical data (as shown in Figure 3 with the row dendrogram for correlation distance between genes), known biological pathways, or known synthetic lethal interactions. Correlation between synthetic lethal genes could also be considered.
</p>
<!--l. 107-->
<p class="indent">
Comparing the findings of modelling and simulation with public gene expression analysis and experimental screen targets is still needed to identify putative synthetic lethal interactions. This application will be tested with the example of CDH1 as a query gene in breast cancer for follow up to earlier results, relevance to ongoing research in the Cancer genetics Laboratory, and comparison to the experimental screen data of MCF10A cells by Telford et al. (2015). While this methodology is intended to be widely applicable, particularly to other cancer genes and will be made available to the research community (manuscript and code release in preparation).
</p>
<!--l. 109-->
<p class="indent">
<p>There are several avenues for further research on synthetic lethality in breast cancer. The main alternative themes are network analysis with a focus on tissue specificity or drug feasibility with an emphasis on pharmacogenomics, biological pathways, and whether candidate targets could be inactivated by compounds with favourable</p>
pharmacokinetic properties. Either approach remains within the scope of the project, although each will require adoption of new computational tools, which is important topic for consideration in the meeting and changes to the project direction later in the year. <a 
 id="x1-195001r286"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">7.3 </span> <a 
 id="x1-1960003"></a>Conclusion
</h3>
<!--l. 113-->
<p class="noindent">
<p>Synthetic lethal interactions are important for understanding gene function and development of targeted anti-cancer treatments. Synthetic lethal discovery with experimental screening is error prone and limited by the model systems in which it is performed. A bioinformatics tool to predict synthetic lethal interactions from genomics data would greatly benefit the cancer research community (and wider genetics research community). Several such tools exist, including one we have developed, but they have conflicting design and results are often inconsistent with experimental screen data. Therefore, modelling and simulation of synthetic lethality in gene expression data is needed to ensure the statistical validity of predictions. We have developed a model with correlation structure based on a Multivariate Normal distribution for which simulations detect synthetic lethality with high performance in simple cases and which has the potential to be developed to model complex correlation structure, biological pathways, or patterns observed in empirical gene expression data. The modelling, public data analysis, and experimental screen data approaches will be combined to further examine the example of CDH1 in breast cancer. Analysis of gene networks, tissue specificity, biological pathways, or drug targets remain options to explore tool development and implications for synthetic lethal cancer research in the future.</p>
</p>
<h2 class="chapterHead">
<span class="titlemark">Chapter 8</span><br /><a 
 id="x1-1970008"></a>Conclusion
</h2>
<p><a 
 id="Q1-1-289"></a></p>
<h2 class="likechapterHead">
<a 
 id="x1-1980008"></a>References
</h2>
<div class="thebibliography">
<p class="bibitem">
<span class="biblabel"> <a 
 id="XAarts2015"></a><span class="bibsp">   </span></span>Aarts, M., Bajrami, I., Herrera-Abreu, M.T., Elliott, R., Brough, R., Ashworth, A., Lord, C.J., and Turner, N.C. (2015) Functional genetic screen identifies increased sensitivity to wee1 inhibition in cells with defects in fanconi anemia and hr pathways. <em><span class="cmti-12">Mol Cancer Ther</span></em>, <strong><span class="cmbx-12">14</span></strong>(4): 865–76.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2015PR"></a><span class="bibsp">   </span></span>Abeshouse, A., Ahn, J., Akbani, R., Ally, A., Amin, S., Andry, C.D., Annala, M., Aprikian, A., Armenia, J., Arora, A., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) The Molecular Taxonomy of Primary Prostate Cancer. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">163</span></strong>(4): 1011–1025.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XAdamski2014"></a><span class="bibsp">   </span></span>Adamski, M.G., Gumann, P., and Baird, A.E. (2014) A method for quantitative analysis of standard and high-throughput qPCR expression data based on input sample quantity. <em><span class="cmti-12">PLoS ONE</span></em>, <strong><span class="cmbx-12">9</span></strong>(8): e103917.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="Xvioplot"></a><span class="bibsp">   </span></span>Adler, D. (2005) <em><span class="cmti-12">vioplot: Violin plot</span></em>. R package version 0.2.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XAgarwal2010"></a><span class="bibsp">   </span></span>Agarwal, S., Deane, C.M., Porter, M.A., and Jones, N.S. (2010) Revisiting date and party hubs: Novel approaches to role assignment in protein interaction networks. <em><span class="cmti-12">PLoS Comput Biol</span></em>, <strong><span class="cmbx-12">6</span></strong>(6): e1000817.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2014TH"></a><span class="bibsp">   </span></span>Agrawal, N., Akbani, R., Aksoy, B.A., Ally, A., Arachchi, H., Asa, S.L., Auman, J.T., Balasundaram, M., Balu, S., Baylin, S.B., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Integrated genomic characterization of papillary thyroid carcinoma. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">159</span></strong>(3): 676–690.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2015SK"></a><span class="bibsp">   </span></span>Akbani, R., Akdemir, K.C., Aksoy, B.A., Albert, M., Ally, A., Amin, S.B., Arachchi, H., Arora, A., Auman, J.T., Ayala, B., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) Genomic Classification of Cutaneous Melanoma. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">161</span></strong>(7): 1681–1696.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XAkobeng2007"></a><span class="bibsp">   </span></span>Akobeng, A.K. (2007) Understanding diagnostic tests 3: receiver operating characteristic curves. <em><span class="cmti-12">Acta Pdiatrica</span></em>, <strong><span class="cmbx-12">96</span></strong>(5): 644–647.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XACS2017"></a><span class="bibsp">   </span></span>American Cancer Society (2017) Genetics and cancer. <a 
href="https://www.cancer.org/cancer/cancer-causes/genetics.html" class="url" ><span class="cmtt-12">https://www.cancer.org/cancer/cancer-_causes/genetics.html</span></a>. Accessed: 22/03/2017.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XASCO2017"></a><span class="bibsp">   </span></span>American Society for Clinical Oncology (ASCO) (2017) The genetics of cancer. <a 
href="http://www.cancer.net/navigating-cancer-care/cancer-basics/genetics/genetics-cancer" class="url" ><span class="cmtt-12">http://www.cancer.net/navigating-_cancer-_care/cancer-_basics/genetics/genetics-_cancer</span></a>. Accessed: 22/03/2017.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XAnjomshoaa2008"></a><span class="bibsp">   </span></span>Anjomshoaa, A., Lin, Y.H., Black, M.A., McCall, J.L., Humar, B., Song, S., Fukuzawa, R., Yoon, H.S., Holzmann, B., Friederichs, J., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2008) Reduced expression of a gene proliferation signature is associated with enhanced malignancy in colon cancer. <em><span class="cmti-12">Br J Cancer</span></em>, <strong><span class="cmbx-12">99</span></strong>(6): 966–973.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="Xgenesetdb"></a><span class="bibsp">   </span></span>Araki, H., Knapp, C., Tsai, P., and Print, C. (2012) GeneSetDB: A comprehensive</p>
meta-database, statistical and visualisation framework for gene set analysis. <em><span class="cmti-12">FEBS</span> <span class="cmti-12">Open Bio</span></em>, <strong><span class="cmbx-12">2</span></strong>: 76–82.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XAshburner2000"></a><span class="bibsp">   </span></span>Ashburner, M., Ball, C.A., Blake, J.A., Botstein, D., Butler, H., Cherry, J.M., Davis, A.P., Dolinski, K., Dwight, S.S., Eppig, J.T., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2000) Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">25</span></strong>(1): 25–29.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XAshworth2008"></a><span class="bibsp">   </span></span>Ashworth, A. (2008) A synthetic lethal therapeutic approach: poly(adp) ribose polymerase inhibitors for the treatment of cancers deficient in dna double-strand break repair. <em><span class="cmti-12">J Clin Oncol</span></em>, <strong><span class="cmbx-12">26</span></strong>(22): 3785–90.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XAudeh2010"></a><span class="bibsp">   </span></span>Audeh, M.W., Carmichael, J., Penson, R.T., Friedlander, M., Powell, B., Bell-McGuinn, K.M., Scott, C., Weitzel, J.N., Oaknin, A., Loman, N., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2010) Oral poly(adp-ribose) polymerase inhibitor olaparib in patients with <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2 </span>mutations and recurrent ovarian cancer: a proof-of-concept trial. <em><span class="cmti-12">Lancet</span></em>, <strong><span class="cmbx-12">376</span></strong>(9737): 245–51.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBabyak2004"></a><span class="bibsp">   </span></span>Babyak, M.A. (2004) What you see may not be what you get: a brief, nontechnical introduction to overfitting in regression-type models. <em><span class="cmti-12">Psychosom Med</span></em>, <strong><span class="cmbx-12">66</span></strong>(3): 411–21.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBamford2004"></a><span class="bibsp">   </span></span>Bamford, S., Dawson, E., Forbes, S., Clements, J., Pettett, R., Dogan, A., Flanagan, A., Teague, J., Futreal, P.A., Stratton, M.R., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2004) The COSMIC (Catalogue of Somatic Mutations in Cancer) database and website. <em><span class="cmti-12">Br J Cancer</span></em>, <strong><span class="cmbx-12">91</span></strong>(2): 355–358.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBarabasi1999"></a><span class="bibsp">   </span></span>Barabási, A.L. and Albert, R. (1999) Emergence of scaling in random networks. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">286</span></strong>(5439): 509–12.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBarabasi2004"></a><span class="bibsp">   </span></span>Barabási, A.L. and Oltvai, Z.N. (2004) Network biology: understanding the cell’s functional organization. <em><span class="cmti-12">Nat Rev Genet</span></em>, <strong><span class="cmbx-12">5</span></strong>(2): 101–13.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBarrat2000"></a><span class="bibsp">   </span></span>Barrat, A. and Weigt, M. (2000) On the properties of small-world network models. <em><span class="cmti-12">The European Physical Journal B - Condensed Matter and Complex Systems</span></em>, <strong><span class="cmbx-12">13</span></strong>(3): 547–560.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBarretina2012"></a><span class="bibsp">   </span></span>Barretina, J., Caponigro, G., Stransky, N., Venkatesan, K., Margolin, A.A., Kim, S., Wilson, C.J., Lehar, J., Kryukov, G.V., Sonkin, D., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2012) The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">483</span></strong>(7391): 603–607.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="Xsafe"></a><span class="bibsp">   </span></span>Barry, W.T. (2016) <em><span class="cmti-12">safe: Significance Analysis of Function and Expression</span></em>. R package version 3.14.0.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBaryshnikova2010a"></a><span class="bibsp">   </span></span>Baryshnikova, A., Costanzo, M., Dixon, S., Vizeacoumar, F.J., Myers, C.L., Andrews, B., and Boone, C. (2010a) Synthetic genetic array (sga) analysis in saccharomyces cerevisiae and schizosaccharomyces pombe. <em><span class="cmti-12">Methods Enzymol</span></em>, <strong><span class="cmbx-12">470</span></strong>: 145–79.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XBaryshnikova2010b"></a><span class="bibsp">   </span></span>Baryshnikova, A., Costanzo, M., Kim, Y., Ding, H., Koh, J., Toufighi, K., Youn, J.Y., Ou, J., San Luis, B.J., Bandyopadhyay, S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2010b) Quantitative analysis</p>
of fitness and genetic interactions in yeast on a genome scale. <em><span class="cmti-12">Nat Meth</span></em>, <strong><span class="cmbx-12">7</span></strong>(12): 1017–1024.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2014GC"></a><span class="bibsp">   </span></span>Bass, A.J., Thorsson, V., Shmulevich, I., Reynolds, S.M., Miller, M., Bernard, B., Hinoue, T., Laird, P.W., Curtis, C., Shen, H., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Comprehensive molecular characterization of gastric adenocarcinoma. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">513</span></strong>(7517): 202–209.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XMatrix"></a><span class="bibsp">   </span></span>Bates, D. and Maechler, M. (2016) <em><span class="cmti-12">Matrix: Sparse and Dense Matrix Classes and</span> <span class="cmti-12">Methods</span></em>. R package version 1.2-7.1.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBateson1909"></a><span class="bibsp">   </span></span>Bateson, W. and Mendel, G. (1909) <em><span class="cmti-12">Mendel’s principles of heredity, by W. Bateson</span></em>. University Press, Cambridge [Eng.].
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBeck2016"></a><span class="bibsp">   </span></span>Beck, T.F., Mullikin, J.C., and Biesecker, L.G. (2016) Systematic Evaluation of Sanger Validation of Next-Generation Sequencing Variants. <em><span class="cmti-12">Clin Chem</span></em>, <strong><span class="cmbx-12">62</span></strong>(4): 647–654.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBecker1994"></a><span class="bibsp">   </span></span>Becker, K.F., Atkinson, M.J., Reich, U., Becker, I., Nekarda, H., Siewert, J.R., and Hfler, H. (1994) E-cadherin gene mutations provide clues to diffuse type gastric carcinomas. <em><span class="cmti-12">Cancer Research</span></em>, <strong><span class="cmbx-12">54</span></strong>(14): 3845–3852.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2011OV"></a><span class="bibsp">   </span></span>Bell, D., Berchuck, A., Birrer, M., Chien, J., Cramer, D., Dao, F., Dhir, R., DiSaia, P., Gabra, H., Glenn, P., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2011) Integrated genomic analyses of ovarian carcinoma. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">474</span></strong>(7353): 609–615.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="Xfdr1995"></a><span class="bibsp">   </span></span>Benjamini, Y. and Hochberg, Y. (1995) Controlling the false discovery rate: A practical and powerful approach to multiple testing. <em><span class="cmti-12">Journal of the Royal Statistical</span> <span class="cmti-12">Society Series B (Methodological)</span></em>, <strong><span class="cmbx-12">57</span></strong>(1): 289–300.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBerx1995"></a><span class="bibsp">   </span></span>Berx, G., Cleton-Jansen, A.M., Nollet, F., de Leeuw, W.J., van de Vijver, M., Cornelisse, C., and van Roy, F. (1995) E-cadherin is a tumour/invasion suppressor gene mutated in human lobular breast cancers. <em><span class="cmti-12">EMBO J</span></em>, <strong><span class="cmbx-12">14</span></strong>(24): 6107–15.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBerx1996"></a><span class="bibsp">   </span></span>Berx, G., Cleton-Jansen, A.M., Strumane, K., de Leeuw, W.J., Nollet, F., van Roy, F., and Cornelisse, C. (1996) E-cadherin is inactivated in a majority of invasive human lobular breast cancers by truncation mutations throughout its extracellular domain. <em><span class="cmti-12">Oncogene</span></em>, <strong><span class="cmbx-12">13</span></strong>(9): 1919–25.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBerx2009"></a><span class="bibsp">   </span></span>Berx, G. and van Roy, F. (2009) Involvement of members of the cadherin superfamily in cancer. <em><span class="cmti-12">Cold Spring Harb Perspect Biol</span></em>, <strong><span class="cmbx-12">1</span></strong>: a003129.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBitler2015"></a><span class="bibsp">   </span></span>Bitler, B.G., Aird, K.M., Garipov, A., Li, H., Amatangelo, M., Kossenkov, A.V., Schultz, D.C., Liu, Q., Shih Ie, M., Conejo-Garcia, J.R., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) Synthetic lethality by targeting ezh2 methyltransferase activity in arid1a-mutated cancers. <em><span class="cmti-12">Nat</span> <span class="cmti-12">Med</span></em>, <strong><span class="cmbx-12">21</span></strong>(3): 231–8.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBlake2015"></a><span class="bibsp">   </span></span>Blake, J.A., Christie, K.R., Dolan, M.E., Drabkin, H.J., Hill, D.P., Ni, L., Sitnikov, D., Burgess, S., Buza, T., Gresham, C., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) Gene Ontology Consortium: going forward. <em><span class="cmti-12">Nucleic Acids Res</span></em>, <strong><span class="cmbx-12">43</span></strong>(Database issue): D1049–1056.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBoettcher2014"></a><span class="bibsp">   </span></span>Boettcher, M., Lawson, A., Ladenburger, V., Fredebohm, J., Wolf, J., Hoheisel, J.D., Frezza, C., and Shlomi, T. (2014) High throughput synthetic lethality screen reveals a tumorigenic role of adenylate cyclase in fumarate hydratase-deficient cancer cells. <em><span class="cmti-12">BMC Genomics</span></em>, <strong><span class="cmbx-12">15</span></strong>: 158.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBoone2007"></a><span class="bibsp">   </span></span>Boone, C., Bussey, H., and Andrews, B.J. (2007) Exploring genetic interactions and networks with yeast. <em><span class="cmti-12">Nat Rev Genet</span></em>, <strong><span class="cmbx-12">8</span></strong>(6): 437–49.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBorgatti2005"></a><span class="bibsp">   </span></span>Borgatti, S.P. (2005) Centrality and network flow. <em><span class="cmti-12">Social Networks</span></em>, <strong><span class="cmbx-12">27</span></strong>(1): 55 – 71.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBoucher2013"></a><span class="bibsp">   </span></span>Boucher, B. and Jenna, S. (2013) Genetic interaction networks: better understand to better predict. <em><span class="cmti-12">Front Genet</span></em>, <strong><span class="cmbx-12">4</span></strong>: 290.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBreiman2001"></a><span class="bibsp">   </span></span>Breiman, L. (2001) Random forests. <em><span class="cmti-12">Machine Learning</span></em>, <strong><span class="cmbx-12">45</span></strong>(1): 5–32.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBrin1998"></a><span class="bibsp">   </span></span>Brin, S. and Page, L. (1998) The anatomy of a large-scale hypertextual web search engine. <em><span class="cmti-12">Computer Networks and ISDN Systems</span></em>, <strong><span class="cmbx-12">30</span></strong>(1): 107 – 117.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBryant2005"></a><span class="bibsp">   </span></span>Bryant, H.E., Schultz, N., Thomas, H.D., Parker, K.M., Flower, D., Lopez, E., Kyle, S., Meuth, M., Curtin, N.J., and Helleday, T. (2005) Specific killing of <span class="cmti-12">BRCA2 </span>deficient tumours with inhibitors of poly<img 
src="thesis14x.png" alt="adpribose  "  class="math"  /> polymerase. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">434</span></strong>(7035): 913–7.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2017CERV"></a><span class="bibsp">   </span></span>Burk, R.D., Chen, Z., Saller, C., Tarvin, K., Carvalho, A.L., Scapulatempo-Neto, C., Silveira, H.C., Fregnani, J.H., Creighton, C.J., Anderson, M.L., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2017) Integrated genomic and molecular characterization of cervical cancer. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">543</span></strong>(7645): 378–384.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XBussey2006"></a><span class="bibsp">   </span></span>Bussey, H., Andrews, B., and Boone, C. (2006) From worm genetic networks to complex human diseases. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">38</span></strong>(8): 862–3.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XButland2008"></a><span class="bibsp">   </span></span>Butland, G., Babu, M., Diaz-Mejia, J.J., Bohdana, F., Phanse, S., Gold, B., Yang, W., Li, J., Gagarinova, A.G., Pogoutse, O., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2008) esga: E. coli synthetic genetic array analysis. <em><span class="cmti-12">Nat Methods</span></em>, <strong><span class="cmbx-12">5</span></strong>(9): 789–95.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XCancerResearchUK2017"></a><span class="bibsp">   </span></span>Cancer Research UK (2017) Family history and cancer genes. <a 
href="http://www.cancerresearchuk.org/about-cancer/causes-of-cancer/inherited-cancer-genes-and-increased-cancer-risk/family-history-and-inherited-cancer-genes" class="url" ><span class="cmtt-12">http://www.cancerresearchuk.org/about-_cancer/causes-_of-_cancer/inherited-_cancer-_genes-_and-_increased-_cancer-_risk/family-_history-_and-_inherited-_cancer-_genes</span></a>. Accessed: 22/03/2017.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XCCLE"></a><span class="bibsp">   </span></span>Cancer Cell Line Encyclopedia (CCLE) (2014) Broad-Novartis Cancer Cell Line Encyclopedia. <a 
href="http://www.broadinstitute.org/ccle" class="url" ><span class="cmtt-12">http://www.broadinstitute.org/ccle</span></a>. Accessed: 07/11/2014.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XcBioPortal"></a><span class="bibsp">   </span></span>cBioPortal for Cancer Genomics (cBioPortal) (2017) cBioPortal for Cancer Genomics. <a 
href="http://www.cbioportal.org/" class="url" ><span class="cmtt-12">http://www.cbioportal.org/</span></a>. Accessed: 26/03/2017.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XPathwayCommons"></a><span class="bibsp">   </span></span>Cerami, E.G., Gross, B.E., Demir, E., Rodchenkov, I., Babur, O., Anwar, N., Schultz, N., Bader, G.D., and Sander, C. (2011) Pathway Commons, a web resource for biological pathway data. <em><span class="cmti-12">Nucleic Acids Res</span></em>, <strong><span class="cmbx-12">39</span></strong>(Database issue): D685–690.</p>
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XChen2014"></a><span class="bibsp">   </span></span>Chen, A., Beetham, H., Black, M.A., Priya, R., Telford, B.J., Guest, J., Wiggins, G.A.R., Godwin, T.D., Yap, A.S., and Guilford, P.J. (2014) E-cadherin loss alters cytoskeletal organization and adhesion in non-malignant breast cells but is insufficient to induce an epithelial-mesenchymal transition. <em><span class="cmti-12">BMC Cancer</span></em>, <strong><span class="cmbx-12">14</span></strong>(1): 552.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XChen2015"></a><span class="bibsp">   </span></span>Chen, K., Yang, D., Li, X., Sun, B., Song, F., Cao, W., Brat, D.J., Gao, Z., Li, H., Liang, H., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) Mutational landscape of gastric adenocarcinoma in Chinese: implications for prognosis and therapy. <em><span class="cmti-12">Proc Natl Acad Sci USA</span></em>, <strong><span class="cmbx-12">112</span></strong>(4): 1107–1112.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XChen2007"></a><span class="bibsp">   </span></span>Chen, S. and Parmigiani, G. (2007) Meta-analysis of BRCA1 and BRCA2 penetrance. <em><span class="cmti-12">J Clin Oncol</span></em>, <strong><span class="cmbx-12">25</span></strong>(11): 1329–1333.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XChen2010"></a><span class="bibsp">   </span></span>Chen, X. and Tompa, M. (2010) Comparative assessment of methods for aligning multiple genome sequences. <em><span class="cmti-12">Nat Biotechnol</span></em>, <strong><span class="cmbx-12">28</span></strong>(6): 567–572.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2017UT"></a><span class="bibsp">   </span></span>Cherniack, A.D., Shen, H., Walter, V., Stewart, C., Murray, B.A., Bowlby, R., Hu, X., Ling, S., Soslow, R.A., Broaddus, R.R., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2017) Integrated Molecular Characterization of Uterine Carcinosarcoma. <em><span class="cmti-12">Cancer Cell</span></em>, <strong><span class="cmbx-12">31</span></strong>(3): 411–423.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XChipman2009"></a><span class="bibsp">   </span></span>Chipman, K. and Singh, A. (2009) Predicting genetic interactions with random walks on biological networks. <em><span class="cmti-12">BMC Bioinformatics</span></em>, <strong><span class="cmbx-12">10</span></strong>(1): 17.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XChristofori1999"></a><span class="bibsp">   </span></span>Christofori, G. and Semb, H. (1999) The role of the cell-adhesion molecule</p>
E-cadherin as a tumour-suppressor gene. <em><span class="cmti-12">Trends in Biochemical Sciences</span></em>, <strong><span class="cmbx-12">24</span></strong>(2): 73 – 76.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2015LBC"></a><span class="bibsp">   </span></span>Ciriello, G., Gatza, M.L., Beck, A.H., Wilkerson, M.D., Rhie, S.K., Pastore, A., Zhang, H., McLellan, M., Yau, C., Kandoth, C., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) Comprehensive Molecular Portraits of Invasive Lobular Breast Cancer. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">163</span></strong>(2): 506–519.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XClark2004"></a><span class="bibsp">   </span></span>Clark, M.J. (2004) Endogenous Regulator of G Protein Signaling Proteins Suppress G o-Dependent  -Opioid Agonist-Mediated Adenylyl Cyclase Supersensitization. <em><span class="cmti-12">Journal of Pharmacology and Experimental Therapeutics</span></em>, <strong><span class="cmbx-12">310</span></strong>(1): 215–222.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XGEO2016"></a><span class="bibsp">   </span></span>Clough, E. and Barrett, T. (2016) The Gene Expression Omnibus Database. <em><span class="cmti-12">Methods Mol Biol</span></em>, <strong><span class="cmbx-12">1418</span></strong>: 93–110.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XCollingridge2013"></a><span class="bibsp">   </span></span>Collingridge, D.S. (2013) A primer on quantitized data analysis and permutation testing. <em><span class="cmti-12">Journal of Mixed Methods Research</span></em>, <strong><span class="cmbx-12">7</span></strong>(1): 81–97.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XCollins2007"></a><span class="bibsp">   </span></span>Collins, F.S. and Barker, A.D. (2007) Mapping the cancer genome. Pinpointing the genes involved in cancer will help chart a new course across the complex landscape of human malignancies. <em><span class="cmti-12">Sci Am</span></em>, <strong><span class="cmbx-12">296</span></strong>(3): 50–57.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XCollins2003"></a><span class="bibsp">   </span></span>Collins, F.S., Morgan, M., and Patrinos, A. (2003) The Human Genome Project: lessons from large-scale biology. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">300</span></strong>(5617): 286–290.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XTCGA2014LU"></a><span class="bibsp">   </span></span>Collisson, E., Campbell, J., Brooks, A., Berger, A., Lee, W., Chmielecki, J., Beer,</p>
D., Cope, L., Creighton, C., Danilova, L., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Comprehensive molecular profiling of lung adenocarcinoma. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">511</span></strong>(7511): 543–550.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XCorcoran2012"></a><span class="bibsp">   </span></span>Corcoran, R.B., Ebi, H., Turke, A.B., Coffee, E.M., Nishino, M., Cogdill, A.P., Brown, R.D., Della Pelle, P., Dias-Santagata, D., Hung, K.E., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2012) Egfr-mediated reactivation of mapk signaling contributes to insensitivity of <span class="cmti-12">BRAF </span>-mutant colorectal cancers to raf inhibition with vemurafenib. <em><span class="cmti-12">Cancer</span> <span class="cmti-12">Discovery</span></em>, <strong><span class="cmbx-12">2</span></strong>(3): 227–235.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XCostanzo2010"></a><span class="bibsp">   </span></span>Costanzo, M., Baryshnikova, A., Bellay, J., Kim, Y., Spear, E.D., Sevier, C.S., Ding, H., Koh, J.L., Toufighi, K., Mostafavi, S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2010) The genetic landscape of a cell. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">327</span></strong>(5964): 425–31.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XCostanzo2011"></a><span class="bibsp">   </span></span>Costanzo, M., Baryshnikova, A., Myers, C.L., Andrews, B., and Boone, C. (2011) Charting the genetic interaction map of a cell. <em><span class="cmti-12">Curr Opin Biotechnol</span></em>, <strong><span class="cmbx-12">22</span></strong>(1): 66–74.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2013RCC"></a><span class="bibsp">   </span></span>Creighton, C.J., Morgan, M., Gunaratne, P.H., Wheeler, D.A., Gibbs, R.A., Robertson, A., Chu, A., Beroukhim, R., Cibulskis, K., Signoretti, S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2013) Comprehensive molecular characterization of clear cell renal cell carcinoma. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">499</span></strong>(7456): 43–49.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XReactome"></a><span class="bibsp">   </span></span>Croft, D., Mundo, A.F., Haw, R., Milacic, M., Weiser, J., Wu, G., Caudy, M., Garapati, P., Gillespie, M., Kamdar, M.R., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) The Reactome pathway knowledgebase. <em><span class="cmti-12">Nucleic Acids Res</span></em>, <strong><span class="cmbx-12">42</span></strong>(database issue): D472D477.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XCrunkhorn2014"></a><span class="bibsp">   </span></span>Crunkhorn, S. (2014) Cancer: Predicting synthetic lethal interactions. <em><span class="cmti-12">Nat Rev</span> <span class="cmti-12">Drug Discov</span></em>, <strong><span class="cmbx-12">13</span></strong>(11): 812.</p>
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="Xigraph"></a><span class="bibsp">   </span></span>Csardi, G. and Nepusz, T. (2006) The igraph software package for complex network research. <em><span class="cmti-12">InterJournal</span></em>, <strong><span class="cmbx-12">Complex Systems</span></strong>: 1695.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XMETABRIC2012"></a><span class="bibsp">   </span></span>Curtis, C., Shah, S.P., Chin, S.F., Turashvili, G., Rueda, O.M., Dunning, M.J., Speed, D., Lynch, A.G., Samarajiwa, S., Yuan, Y., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2012) The genomic and transcriptomic architecture of 2,000 breast tumours reveals novel subgroups. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">486</span></strong>(7403): 346–352.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XDai2015"></a><span class="bibsp">   </span></span>Dai, X., Li, T., Bai, Z., Yang, Y., Liu, X., Zhan, J., and Shi, B. (2015) Breast cancer intrinsic subtype classification, clinical use and future trends. <em><span class="cmti-12">Am J Cancer</span> <span class="cmti-12">Res</span></em>, <strong><span class="cmbx-12">5</span></strong>(10): 2929–2943.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XDavierwala2005"></a><span class="bibsp">   </span></span>Davierwala, A.P., Haynes, J., Li, Z., Brost, R.L., Robinson, M.D., Yu, L., Mnaimneh, S., Ding, H., Zhu, H., Chen, Y., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2005) The synthetic genetic interaction spectrum of essential genes. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">37</span></strong>(10): 1147–1152.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XDeLeeuw1997"></a><span class="bibsp">   </span></span>De Leeuw, W.J., Berx, G., Vos, C.B., Peterse, J.L., Van de Vijver, M.J., Litvinov, S., Van Roy, F., Cornelisse, C.J., and Cleton-Jansen, A.M. (1997) Simultaneous loss of E-cadherin and catenins in invasive lobular breast cancer and lobular carcinoma in situ. <em><span class="cmti-12">J Pathol</span></em>, <strong><span class="cmbx-12">183</span></strong>(4): 404–11.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="Xpaxtools"></a><span class="bibsp">   </span></span>Demir, E., Babur, O., Rodchenkov, I., Aksoy, B.A., Fukuda, K.I., Gross, B., Sumer, O.S., Bader, G.D., and Sander, C. (2013) Using biological pathway data with Paxtools. <em><span class="cmti-12">PLoS Comput Biol</span></em>, <strong><span class="cmbx-12">9</span></strong>(9): e1003194.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XDeshpande2013"></a><span class="bibsp">   </span></span>Deshpande, R., Asiedu, M.K., Klebig, M., Sutor, S., Kuzmin, E., Nelson, J., Piotrowski, J., Shin, S.H., Yoshida, M., Costanzo, M., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2013) A comparative genomic approach for identifying synthetic lethal interactions in human cancer. <em><span class="cmti-12">Cancer Res</span></em>, <strong><span class="cmbx-12">73</span></strong>(20): 6128–36.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XDickson1999"></a><span class="bibsp">   </span></span>Dickson, D. (1999) Wellcome funds cancer database. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">401</span></strong>(6755): 729.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XDienstmann2011"></a><span class="bibsp">   </span></span>Dienstmann, R. and Tabernero, J. (2011) <span class="cmti-12">BRAF </span>as a target for cancer therapy. <em><span class="cmti-12">Anticancer Agents Med Chem</span></em>, <strong><span class="cmbx-12">11</span></strong>(3): 285–95.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XDijkstra1959"></a><span class="bibsp">   </span></span>Dijkstra, E.W. (1959) A note on two problems in connexion with graphs. <em><span class="cmti-12">Numerische Mathematik</span></em>, <strong><span class="cmbx-12">1</span></strong>(1): 269–271.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XDixon2009a"></a><span class="bibsp">   </span></span>Dixon, S.J., Andrews, B.J., and Boone, C. (2009) Exploring the conservation of synthetic lethal genetic interaction networks. <em><span class="cmti-12">Commun Integr Biol</span></em>, <strong><span class="cmbx-12">2</span></strong>(2): 78–81.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XDixon2008"></a><span class="bibsp">   </span></span>Dixon, S.J., Fedyshyn, Y., Koh, J.L., Prasad, T.S., Chahwan, C., Chua, G., Toufighi, K., Baryshnikova, A., Hayles, J., Hoe, K.L., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2008) Significant conservation of synthetic lethal genetic interaction networks between distantly related eukaryotes. <em><span class="cmti-12">Proc Natl Acad Sci U S A</span></em>, <strong><span class="cmbx-12">105</span></strong>(43): 16653–8.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XDorogovtsev2003"></a><span class="bibsp">   </span></span>Dorogovtsev, S.N. and Mendes, J.F. (2003) <em><span class="cmti-12">Evolution of networks: From biological</span> <span class="cmti-12">nets to the Internet and WWW </span></em>. Oxford University Press, USA.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XErdos1959"></a><span class="bibsp">   </span></span>Erd<img 
src="thesis15x.png" alt="&#x02DD;o "  class="Huml"  />s, P. and Rényi, A. (1959) On random graphs I. <em><span class="cmti-12">Publ Math Debrecen</span></em>, <strong><span class="cmbx-12">6</span></strong>: 290–297.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XErdos1960"></a><span class="bibsp">   </span></span>Erd<img 
src="thesis16x.png" alt="&#x02DD;o "  class="Huml"  />s, P. and Rényi, A. (1960) On the evolution of random graphs. In <em><span class="cmti-12">Publ.</span> <span class="cmti-12">Math. Inst. Hung. Acad. Sci</span></em>, volume 5, 17–61.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XEroles2012"></a><span class="bibsp">   </span></span>Eroles, P., Bosch, A., Perez-Fidalgo, J.A., and Lluch, A. (2012) Molecular biology in breast cancer: intrinsic subtypes and signaling pathways. <em><span class="cmti-12">Cancer Treat Rev</span></em>, <strong><span class="cmbx-12">38</span></strong>(6): 698–707.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XEzkurdia2014"></a><span class="bibsp">   </span></span>Ezkurdia, I., Juan, D., Rodriguez, J.M., Frankish, A., Diekhans, M., Harrow, J., Vazquez, J., Valencia, A., and Tress, M.L. (2014) Multiple evidence strands suggest that there may be as few as 19 000 human protein-coding genes. <em><span class="cmti-12">Human Molecular</span> <span class="cmti-12">Genetics</span></em>, <strong><span class="cmbx-12">23</span></strong>(22): 5866.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XFarmer2005"></a><span class="bibsp">   </span></span>Farmer, H., McCabe, N., Lord, C.J., Tutt, A.N., Johnson, D.A., Richardson, T.B., Santarosa, M., Dillon, K.J., Hickson, I., Knights, C., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2005) Targeting the dna repair defect in BRCA mutant cells as a therapeutic strategy. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">434</span></strong>(7035): 917–21.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XFawcett2006"></a><span class="bibsp">   </span></span>Fawcett, T. (2006) An introduction to ROC analysis. <em><span class="cmti-12">Pattern Recognition Letters</span></em>, <strong><span class="cmbx-12">27</span></strong>(8): 861 – 874. <span class="cmsy-10x-x-120">{</span>ROC<span class="cmsy-10x-x-120">} </span>Analysis in Pattern Recognition.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XFece2015"></a><span class="bibsp">   </span></span>Fece de la Cruz, F., Gapp, B.V., and Nijman, S.M. (2015) Synthetic lethal vulnerabilities of cancer. <em><span class="cmti-12">Annu Rev Pharmacol Toxicol</span></em>, <strong><span class="cmbx-12">55</span></strong>: 513–531.</p>
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XFerlay2015"></a><span class="bibsp">   </span></span>Ferlay, J., Soerjomataram, I., Dikshit, R., Eser, S., Mathers, C., Rebelo, M., Parkin, D.M., Forman, D., and Bray, F. (2015) Cancer incidence and mortality worldwide: sources, methods and major patterns in GLOBOCAN 2012. <em><span class="cmti-12">Int J Cancer</span></em>, <strong><span class="cmbx-12">136</span></strong>(5): E359–386.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XFisher1919"></a><span class="bibsp">   </span></span>Fisher, R.A. (1919) Xv.the correlation between relatives on the supposition of mendelian inheritance. <em><span class="cmti-12">Earth and Environmental Science Transactions of the Royal</span> <span class="cmti-12">Society of Edinburgh</span></em>, <strong><span class="cmbx-12">52</span></strong>(02): 399–433.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XFong2009"></a><span class="bibsp">   </span></span>Fong, P.C., Boss, D.S., Yap, T.A., Tutt, A., Wu, P., Mergui-Roelvink, M., Mortimer, P., Swaisland, H., Lau, A., O’Connor, M.J., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2009) Inhibition of poly(adp-ribose) polymerase in tumors from BRCA mutation carriers. <em><span class="cmti-12">N Engl J</span> <span class="cmti-12">Med</span></em>, <strong><span class="cmbx-12">361</span></strong>(2): 123–34.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XFong2010"></a><span class="bibsp">   </span></span>Fong, P.C., Yap, T.A., Boss, D.S., Carden, C.P., Mergui-Roelvink, M., Gourley, C., De Greve, J., Lubinski, J., Shanley, S., Messiou, C., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2010) Poly(adp)-ribose polymerase inhibition: frequent durable responses in BRCA carrier ovarian cancer correlating with platinum-free interval. <em><span class="cmti-12">J Clin Oncol</span></em>, <strong><span class="cmbx-12">28</span></strong>(15): 2512–9.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XForbes2015"></a><span class="bibsp">   </span></span>Forbes, S.A., Beare, D., Gunasekaran, P., Leung, K., Bindal, N., Boutselakis, H., Ding, M., Bamford, S., Cole, C., Ward, S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) COSMIC: exploring the world’s knowledge of somatic mutations in human cancer. <em><span class="cmti-12">Nucleic Acids Res</span></em>, <strong><span class="cmbx-12">43</span></strong>(Database issue): D805–811.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XFraser2004"></a><span class="bibsp">   </span></span>Fraser, A. (2004) Towards full employment: using RNAi to find roles for the redundant. <em><span class="cmti-12">Oncogene</span></em>, <strong><span class="cmbx-12">23</span></strong>(51): 8346–52.</p>
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XFutreal2004"></a><span class="bibsp">   </span></span>Futreal, P.A., Coin, L., Marshall, M., Down, T., Hubbard, T., Wooster, R., Rahman, N., and Stratton, M.R. (2004) A census of human cancer genes. <em><span class="cmti-12">Nat Rev Cancer</span></em>, <strong><span class="cmbx-12">4</span></strong>(3): 177–183.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XFutreal2001"></a><span class="bibsp">   </span></span>Futreal, P.A., Kasprzyk, A., Birney, E., Mullikin, J.C., Wooster, R., and Stratton, M.R. (2001) Cancer and genomics. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">409</span></strong>(6822): 850–852.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XGao2015"></a><span class="bibsp">   </span></span>Gao, B. and Roux, P.P. (2015) Translational control by oncogenic signaling pathways. <em><span class="cmti-12">Biochimica et Biophysica Acta</span></em>, <strong><span class="cmbx-12">1849</span></strong>(7): 753–65.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XGatza2011"></a><span class="bibsp">   </span></span>Gatza, M.L., Kung, H.N., Blackwell, K.L., Dewhirst, M.W., Marks, J.R., and Chi, J.T. (2011) Analysis of tumor environmental response and oncogenic pathway activation identifies distinct basal and luminal features in HER2-related breast tumor subtypes. <em><span class="cmti-12">Breast Cancer Res</span></em>, <strong><span class="cmbx-12">13</span></strong>(3): R62.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XGatza2010"></a><span class="bibsp">   </span></span>Gatza, M.L., Lucas, J.E., Barry, W.T., Kim, J.W., Wang, Q., Crawford, M.D., Datto, M.B., Kelley, M., Mathey-Prevot, B., Potti, A., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2010) A pathway-based classification of human breast cancer. <em><span class="cmti-12">Proc Natl Acad Sci USA</span></em>, <strong><span class="cmbx-12">107</span></strong>(15): 6994–6999.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XGatza2014"></a><span class="bibsp">   </span></span>Gatza, M.L., Silva, G.O., Parker, J.S., Fan, C., and Perou, C.M. (2014) An integrated genomics approach identifies drivers of proliferation in luminal-subtype human breast cancer. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">46</span></strong>(10): 1051–1059.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XGentleman2004"></a><span class="bibsp">   </span></span>Gentleman, R.C., Carey, V.J., Bates, D.M., Bolstad, B., Dettling, M., Dudoit, S.,</p>
Ellis, B., Gautier, L., Ge, Y., Gentry, J., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2004) Bioconductor: open software development for computational biology and bioinformatics. <em><span class="cmti-12">Genome Biol</span></em>, <strong><span class="cmbx-12">5</span></strong>(10): R80.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XGenz2009"></a><span class="bibsp">   </span></span>Genz, A. and Bretz, F. (2009) Computation of multivariate normal and t probabilities. In <em><span class="cmti-12">Lecture Notes in Statistics</span></em>, volume 195. Springer-Verlag, Heidelberg.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="Xmvtnorm"></a><span class="bibsp">   </span></span>Genz, A., Bretz, F., Miwa, T., Mi, X., Leisch, F., Scheipl, F., and Hothorn, T. (2016) <em><span class="cmti-12">mvtnorm: Multivariate Normal and t Distributions</span></em>. R package version 1.0-5. URL.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XGilbert1973"></a><span class="bibsp">   </span></span>Gilbert, W. and Maxam, A. (1973) The nucleotide sequence of the lac operator. <em><span class="cmti-12">Proceedings of the National Academy of Sciences</span></em>, <strong><span class="cmbx-12">70</span></strong>(12): 3581–3584.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XGit2010"></a><span class="bibsp">   </span></span>Git, A., Dvinge, H., Salmon-Divon, M., Osborne, M., Kutter, C., Hadfield, J., Bertone, P., and Caldas, C. (2010) Systematic comparison of microarray profiling, real-time PCR, and next-generation sequencing technologies for measuring differential microRNA expression. <em><span class="cmti-12">RNA</span></em>, <strong><span class="cmbx-12">16</span></strong>(5): 991–1006.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XGlobus"></a><span class="bibsp">   </span></span>Globus (Globus) (2017) Research data management simplified. <a 
href="https://www.globus.org/" class="url" ><span class="cmtt-12">https://www.globus.org/</span></a>. Accessed: 25/03/2017.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XGraziano2003"></a><span class="bibsp">   </span></span>Graziano, F., Humar, B., and Guilford, P. (2003) The role of the E-cadherin gene (<span class="cmti-12">CDH1</span>) in diffuse gastric cancer susceptibility: from the laboratory to clinical practice. <em><span class="cmti-12">Annals of Oncology</span></em>, <strong><span class="cmbx-12">14</span></strong>(12): 1705–1713.</p>
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XGuell2014"></a><span class="bibsp">   </span></span>Güell, O., Sagus, F., and Serrano, M. (2014) Essential plasticity and redundancy of metabolism unveiled by synthetic lethality analysis. <em><span class="cmti-12">PLoS Comput Biol</span></em>, <strong><span class="cmbx-12">10</span></strong>(5): e1003637.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XGuilford1999"></a><span class="bibsp">   </span></span>Guilford, P. (1999) E-cadherin downregulation in cancer: fuel on the fire? <em><span class="cmti-12">Molecular</span> <span class="cmti-12">Medicine Today</span></em>, <strong><span class="cmbx-12">5</span></strong>(4): 172 – 177.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XGuilford1998"></a><span class="bibsp">   </span></span>Guilford, P., Hopkins, J., Harraway, J., McLeod, M., McLeod, N., Harawira, P., Taite, H., Scoular, R., Miller, A., and Reeve, A.E. (1998) E-cadherin germline mutations in familial gastric cancer. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">392</span></strong>(6674): 402–5.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XGuilford2010"></a><span class="bibsp">   </span></span>Guilford, P., Humar, B., and Blair, V. (2010) Hereditary diffuse gastric cancer: translation of <span class="cmti-12">CDH1 </span>germline mutations into clinical practice. <em><span class="cmti-12">Gastric Cancer</span></em>, <strong><span class="cmbx-12">13</span></strong>(1): 1–10.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHDGC"></a><span class="bibsp">   </span></span>Guilford, P.J., Hopkins, J.B., Grady, W.M., Markowitz, S.D., Willis, J., Lynch, H., Rajput, A., Wiesner, G.L., Lindor, N.M., Burgart, L.J., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (1999) E-cadherin germline mutations define an inherited cancer syndrome dominated by diffuse gastric cancer. <em><span class="cmti-12">Hum Mutat</span></em>, <strong><span class="cmbx-12">14</span></strong>(3): 249–55.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XGuo2016"></a><span class="bibsp">   </span></span>Guo, J., Liu, H., and Zheng, J. (2016) SynLethDB: synthetic lethality database toward discovery of selective and sensitive anticancer drug targets. <em><span class="cmti-12">Nucleic Acids</span> <span class="cmti-12">Res</span></em>, <strong><span class="cmbx-12">44</span></strong>(D1): D1011–1017.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHajian-Tilaki2013"></a><span class="bibsp">   </span></span>Hajian-Tilaki, K. (2013) Receiver Operating Characteristic (ROC) Curve Analysis for Medical Diagnostic Test Evaluation. <em><span class="cmti-12">Caspian J Intern Med</span></em>, <strong><span class="cmbx-12">4</span></strong>(2): 627–635.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHall2009"></a><span class="bibsp">   </span></span>Hall, M., Frank, E., Holmes, G., Pfahringer, B., Reutemann, P., and Witten, I.H. (2009) The weka data mining software: an update. <em><span class="cmti-12">SIGKDD Explor Newsl</span></em>, <strong><span class="cmbx-12">11</span></strong>(1): 10–18.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2012LUSC"></a><span class="bibsp">   </span></span>Hammerman, P.S., Lawrence, M.S., Voet, D., Jing, R., Cibulskis, K., Sivachenko, A., Stojanov, P., McKenna, A., Lander, E.S., Gabriel, S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2012) Comprehensive genomic characterization of squamous cell lung cancers. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">489</span></strong>(7417): 519–525.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHan2004"></a><span class="bibsp">   </span></span>Han, J.D.J., Bertin, N., Hao, T., Goldberg, D.S., Berriz, G.F., Zhang, L.V., Dupuy, D., Walhout, A.J.M., Cusick, M.E., Roth, F.P., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2004) Evidence for dynamically organized modularity in the yeast protein-protein interaction network. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">430</span></strong>(6995): 88–93.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHanahan2000"></a><span class="bibsp">   </span></span>Hanahan, D. and Weinberg, R.A. (2000) The hallmarks of cancer. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">100</span></strong>(1): 57–70.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHanahan2011"></a><span class="bibsp">   </span></span>Hanahan, D. and Weinberg, R.A. (2011) Hallmarks of cancer: the next generation. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">144</span></strong>(5): 646–674.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XCIX2013"></a><span class="bibsp">   </span></span>Hanna, S. (2003) Cancer incidence in new zealand (2003-2007). In D. Forman, D. Bray F Brewster, C. Gombe Mbalawa, B. Kohler, M. Piñeros,</p>
E. Steliarova-Foucher, R. Swaminathan, and J. Ferlay (editors), <em><span class="cmti-12">Cancer Incidence</span> <span class="cmti-12">in Five Continents</span></em>, volume X, 902–907. International Agency for Research on Cancer, Lyon, France. Electronic version <a 
href="http://ci5.iarc.fr" class="url" ><span class="cmtt-12">http://ci5.iarc.fr</span></a> Accessed 22/03/2017.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XcaArray2014"></a><span class="bibsp">   </span></span>Heiskanen, M., Bian, X., Swan, D., and Basu, A. (2014) caArray microarray database in the cancer biomedical informatics grid<sup class="textsuperscript"><span class="cmr-10x-x-109">TM</span></sup> (caBIG<sup class="textsuperscript"><span class="cmr-10x-x-109">TM</span></sup>). <em><span class="cmti-12">Cancer Research</span></em>, <strong><span class="cmbx-12">67</span></strong>(9 Supplement): 3712–3712.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHeiskanen2012"></a><span class="bibsp">   </span></span>Heiskanen, M.A. and Aittokallio, T. (2012) Mining high-throughput screens for cancer drug targets-lessons from yeast chemical-genomic profiling and synthetic lethality. <em><span class="cmti-12">Wiley Interdisciplinary Reviews: Data Mining and Knowledge Discovery</span></em>, <strong><span class="cmbx-12">2</span></strong>(3): 263–272.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHell1976"></a><span class="bibsp">   </span></span>Hell, P. (1976) Graphs with given neighbourhoods i. problémes combinatorics at theorie des graphes. <em><span class="cmti-12">Proc Coil Int CNRS, Orsay</span></em>, <strong><span class="cmbx-12">260</span></strong>: 219–223.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHerschkowitz2007"></a><span class="bibsp">   </span></span>Herschkowitz, J.I., Simin, K., Weigman, V.J., Mikaelian, I., Usary, J., Hu, Z., Rasmussen, K.E., Jones, L.P., Assefnia, S., Chandrasekharan, S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2007) Identification of conserved gene expression features between murine mammary carcinoma models and human breast tumors. <em><span class="cmti-12">Genome Biol</span></em>, <strong><span class="cmbx-12">8</span></strong>(5): R76.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHillenmeyer2008"></a><span class="bibsp">   </span></span>Hillenmeyer, M.E. (2008) The chemical genomic portrait of yeast: uncovering a phenotype for all genes. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">320</span></strong>: 362–365.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XTCGA2014PAN"></a><span class="bibsp">   </span></span>Hoadley, K.A., Yau, C., Wolf, D.M., Cherniack, A.D., Tamborero, D., Ng,</p>
S., Leiserson, M.D., Niu, B., McLellan, M.D., Uzunangelov, V., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Multiplatform analysis of 12 cancer types reveals molecular classification within and across tissues of origin. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">158</span></strong>(4): 929–944.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHoehndorf2013"></a><span class="bibsp">   </span></span>Hoehndorf, R., Hardy, N.W., Osumi-Sutherland, D., Tweedie, S., Schofield, P.N., and Gkoutos, G.V. (2013) Systematic analysis of experimental phenotype data reveals gene functions. <em><span class="cmti-12">PLoS ONE</span></em>, <strong><span class="cmbx-12">8</span></strong>(4): e60847.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHolm1979"></a><span class="bibsp">   </span></span>Holm, S. (1979) A simple sequentially rejective multiple test procedure. <em><span class="cmti-12">Scandinavian Journal of Statistics</span></em>, <strong><span class="cmbx-12">6</span></strong>(2): 65–70.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHolme2002"></a><span class="bibsp">   </span></span>Holme, P. and Kim, B.J. (2002) Growing scale-free networks with tunable clustering. <em><span class="cmti-12">Physical Review E</span></em>, <strong><span class="cmbx-12">65</span></strong>(2): 026107.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHopkins2008"></a><span class="bibsp">   </span></span>Hopkins, A.L. (2008) Network pharmacology: the next paradigm in drug discovery. <em><span class="cmti-12">Nat Chem Biol</span></em>, <strong><span class="cmbx-12">4</span></strong>(11): 682–690.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHu2006"></a><span class="bibsp">   </span></span>Hu, Z., Fan, C., Oh, D.S., Marron, J.S., He, X., Qaqish, B.F., Livasy, C., Carey, L.A., Reynolds, E., Dressler, L., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2006) The molecular portraits of breast tumors are conserved across microarray platforms. <em><span class="cmti-12">BMC Genomics</span></em>, <strong><span class="cmbx-12">7</span></strong>: 96.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHuang2003"></a><span class="bibsp">   </span></span>Huang, E., Cheng, S., Dressman, H., Pittman, J., Tsou, M., Horng, C., Bild, A., Iversen, E., Liao, M., Chen, C., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2003) Gene expression predictors of breast cancer outcomes. <em><span class="cmti-12">Lancet</span></em>, <strong><span class="cmbx-12">361</span></strong>: 1590–1596.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XIllumina"></a><span class="bibsp">   </span></span>Illumina, Inc (Illumina) (2017) Sequencing and array-based solutions for genetic research. <a 
href="https://www.illumina.com/" class="url" ><span class="cmtt-12">https://www.illumina.com/</span></a>. Accessed: 26/03/2017.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XHapMap2003"></a><span class="bibsp">   </span></span>International HapMap 3 Consortium (HapMap) (2003) The International HapMap Project. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">426</span></strong>(6968): 789–796.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XIHGSC2004"></a><span class="bibsp">   </span></span>Internationl Human Genome Sequencing Consortium (IHGSC) (2004) Finishing the euchromatic sequence of the human genome. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">431</span></strong>(7011): 931–945.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XJerby2014"></a><span class="bibsp">   </span></span>Jerby-Arnon, L., Pfetzer, N., Waldman, Y., McGarry, L., James, D., Shanks, E., Seashore-Ludlow, B., Weinstock, A., Geiger, T., Clemons, P., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Predicting cancer-specific vulnerability via data-driven detection of synthetic lethality. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">158</span></strong>(5): 1199–1209.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XJoachims1999"></a><span class="bibsp">   </span></span>Joachims, T. (1999) Making large-scale support vector machine learning practical. In S. Bernhard, lkopf, J.C.B. Christopher, and J.S. Alexander (editors), <em><span class="cmti-12">Advances</span> <span class="cmti-12">in kernel methods</span></em>, 169–184. MIT Press.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XJu2015"></a><span class="bibsp">   </span></span>Ju, Z., Liu, W., Roebuck, P.L., Siwak, D.R., Zhang, N., Lu, Y., Davies, M.A., Akbani, R., Weinstein, J.N., Mills, G.B., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) Development of a robust classifier for quality control of reverse-phase protein arrays. <em><span class="cmti-12">Bioinformatics</span></em>, <strong><span class="cmbx-12">31</span></strong>(6): 912.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XKaelin2005"></a><span class="bibsp">   </span></span>Kaelin, Jr, W. (2005) The concept of synthetic lethality in the context of anticancer therapy. <em><span class="cmti-12">Nat Rev Cancer</span></em>, <strong><span class="cmbx-12">5</span></strong>(9): 689–98.</p>
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XKaelin2009"></a><span class="bibsp">   </span></span>Kaelin, Jr, W. (2009) Synthetic lethality: a framework for the development of wiser cancer therapeutics. <em><span class="cmti-12">Genome Med</span></em>, <strong><span class="cmbx-12">1</span></strong>: 99.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XKakiuchi2014"></a><span class="bibsp">   </span></span>Kakiuchi, M., Nishizawa, T., Ueda, H., Gotoh, K., Tanaka, A., Hayashi, A., Yamamoto, S., Tatsuno, K., Katoh, H., Watanabe, Y., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Recurrent gain-of-function mutations of RHOA in diffuse-type gastric carcinoma. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">46</span></strong>(6): 583–587.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XKamada1989"></a><span class="bibsp">   </span></span>Kamada, T. and Kawai, S. (1989) An algorithm for drawing general undirected graphs. <em><span class="cmti-12">Information Processing Letters</span></em>, <strong><span class="cmbx-12">31</span></strong>(1): 7–15.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2013ENDO"></a><span class="bibsp">   </span></span>Kandoth, C., Schultz, N., Cherniack, A.D., Akbani, R., Liu, Y., Shen, H., Robertson, A.G., Pashtan, I., Shen, R., Benz, C.C., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2013) Integrated genomic characterization of endometrial carcinoma. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">497</span></strong>(7447): 67–73.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XFANTOM2001"></a><span class="bibsp">   </span></span>Kawai, J., Shinagawa, A., Shibata, K., Yoshino, M., Itoh, M., Ishii, Y., Arakawa, T., Hara, A., Fukunishi, Y., Konno, H., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2001) Functional annotation of a full-length mouse cDNA collection. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">409</span></strong>(6821): 685–690.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XKelley2005"></a><span class="bibsp">   </span></span>Kelley, R. and Ideker, T. (2005) Systematic interpretation of genetic interactions using protein networks. <em><span class="cmti-12">Nat Biotech</span></em>, <strong><span class="cmbx-12">23</span></strong>(5): 561–566.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XKellyBMC"></a><span class="bibsp">   </span></span>Kelly, S., Chen, A., Guilford, P., and Black, M. (2017a) Synthetic lethal interaction prediction of target pathways in E-cadherin deficient breast cancers. Submitted to <span class="cmti-12">BMC Genomics</span>.</p>
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XKelly2013"></a><span class="bibsp">   </span></span>Kelly, S.T. (2013) <em><span class="cmti-12">Statistical Predictions of Synthetic Lethal Interactions in Cancer</span></em>. Dissertation, University of Otago.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XKellyHDGC"></a><span class="bibsp">   </span></span>Kelly, S.T., Single, A.B., Telford, B.J., Beetham, H.G., Godwin, T.D., Chen, A., Black, M.A., and Guilford, P.J. (2017b) Towards HDGC chemoprevention: vulnerabilities in E-cadherin-negative cells identified by genome-wide interrogation of isogenic cell lines and whole tumors. Submitted to <span class="cmti-12">Cancer Prev Res</span>.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XKozlov2015"></a><span class="bibsp">   </span></span>Kozlov, K.N., Gursky, V.V., Kulakovskiy, I.V., and Samsonova, M.G. (2015) Sequence-based model of gap gene regulation network. <em><span class="cmti-12">BMC Genomics</span></em>, <strong><span class="cmbx-12">15</span></strong>(Suppl 12): S6.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XKranthi2013"></a><span class="bibsp">   </span></span>Kranthi, S., Rao, S., and Manimaran, P. (2013) Identification of synthetic lethal pairs in biological systems through network information centrality. <em><span class="cmti-12">Mol BioSyst</span></em>, <strong><span class="cmbx-12">9</span></strong>(8): 2163–2167.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLander2011"></a><span class="bibsp">   </span></span>Lander, E.S. (2011) Initial impact of the sequencing of the human genome. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">470</span></strong>(7333): 187–197.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLander2001"></a><span class="bibsp">   </span></span>Lander, E.S., Linton, L.M., Birren, B., Nusbaum, C., Zody, M.C., Baldwin, J., Devon, K., Dewar, K., Doyle, M., FitzHugh, W., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2001) Initial sequencing and analysis of the human genome. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">409</span></strong>(6822): 860–921.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="Xbowtie"></a><span class="bibsp">   </span></span>Langmead, B., Trapnell, C., Pop, M., and Salzberg, S.L. (2009) Ultrafast and</p>
memory-efficient alignment of short DNA sequences to the human genome. <em><span class="cmti-12">Genome</span> <span class="cmti-12">Biol</span></em>, <strong><span class="cmbx-12">10</span></strong>(3): R25.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLatora2001"></a><span class="bibsp">   </span></span>Latora, V. and Marchiori, M. (2001) Efficient behavior of small-world networks. <em><span class="cmti-12">Phys Rev Lett</span></em>, <strong><span class="cmbx-12">87</span></strong>: 198701.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLaufer2013"></a><span class="bibsp">   </span></span>Laufer, C., Fischer, B., Billmann, M., Huber, W., and Boutros, M. (2013) Mapping genetic interactions in human cancer cells with RNAi and multiparametric phenotyping. <em><span class="cmti-12">Nat Methods</span></em>, <strong><span class="cmbx-12">10</span></strong>(5): 427–31.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLaw2014"></a><span class="bibsp">   </span></span>Law, C.W., Chen, Y., Shi, W., and Smyth, G.K. (2014) voom: precision weights unlock linear model analysis tools for RNA-seq read counts. <em><span class="cmti-12">Genome Biol</span></em>, <strong><span class="cmbx-12">15</span></strong>(2): R29.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2015HNSC"></a><span class="bibsp">   </span></span>Lawrence, M.S., Sougnez, C., Lichtenstein, L., Cibulskis, K., Lander, E., Gabriel, S.B., Getz, G., Ally, A., Balasundaram, M., Birol, I., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) Comprehensive genomic characterization of head and neck squamous cell carcinomas. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">517</span></strong>(7536): 576–582.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLeMeur2008"></a><span class="bibsp">   </span></span>Le Meur, N. and Gentleman, R. (2008) Modeling synthetic lethality. <em><span class="cmti-12">Genome Biol</span></em>, <strong><span class="cmbx-12">9</span></strong>(9): R135.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLeMeur2014"></a><span class="bibsp">   </span></span>Le Meur, N., Jiang, Z., Liu, T., Mar, J., and Gentleman, R.C. (2014) Slgi: Synthetic lethal genetic interaction. r package version 1.26.0.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLee2010a"></a><span class="bibsp">   </span></span>Lee, A.Y., Perreault, R., Harel, S., Boulier, E.L., Suderman, M., Hallett, M., and Jenna, S. (2010a) Searching for signaling balance through the identification of genetic interactors of the rab guanine-nucleotide dissociation inhibitor gdi-1. <em><span class="cmti-12">PLoS ONE</span></em>, <strong><span class="cmbx-12">5</span></strong>(5): e10624.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLee2010b"></a><span class="bibsp">   </span></span>Lee, I., Lehner, B., Vavouri, T., Shin, J., Fraser, A.G., and Marcotte, E.M. (2010b) Predicting genetic modifier loci using functional gene networks. <em><span class="cmti-12">Genome Research</span></em>, <strong><span class="cmbx-12">20</span></strong>(8): 1143–1153.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLee2009"></a><span class="bibsp">   </span></span>Lee, I. and Marcotte, E.M. (2009) Effects of functional bias on supervised learning of a gene network model. <em><span class="cmti-12">Methods Mol Biol</span></em>, <strong><span class="cmbx-12">541</span></strong>: 463–75.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLee2012"></a><span class="bibsp">   </span></span>Lee, M.J., Ye, A.S., Gardino, A.K., Heijink, A.M., Sorger, P.K., MacBeath, G., and Yaffe, M.B. (2012) Sequential application of anticancer drugs enhances cell death by rewiring apoptotic signaling networks. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">149</span></strong>(4): 780–94.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLehner2006"></a><span class="bibsp">   </span></span>Lehner, B., Crombie, C., Tischler, J., Fortunato, A., and Fraser, A.G. (2006) Systematic mapping of genetic interactions in caenorhabditis elegans identifies common modifiers of diverse signaling pathways. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">38</span></strong>(8): 896–903.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLi2014"></a><span class="bibsp">   </span></span>Li, X.J., Mishra, S.K., Wu, M., Zhang, F., and Zheng, J. (2014) Syn-lethality: An integrative knowledge base of synthetic lethality towards discovery of selective anticancer therapies. <em><span class="cmti-12">Biomed Res Int</span></em>, <strong><span class="cmbx-12">2014</span></strong>: 196034.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XTCGA2016PRC"></a><span class="bibsp">   </span></span>Linehan, W.M., Spellman, P.T., Ricketts, C.J., Creighton, C.J., Fei, S.S., Davis, C.,</p>
Wheeler, D.A., Murray, B.A., Schmidt, L., Vocke, C.D., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2016) Comprehensive Molecular Characterization of Papillary Renal-Cell Carcinoma. <em><span class="cmti-12">N Engl J Med</span></em>, <strong><span class="cmbx-12">374</span></strong>(2): 135–145.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLokody2014"></a><span class="bibsp">   </span></span>Lokody, I. (2014) Computational modelling: A computational crystal ball. <em><span class="cmti-12">Nature</span> <span class="cmti-12">Reviews Cancer</span></em>, <strong><span class="cmbx-12">14</span></strong>(10): 649–649.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLord2014"></a><span class="bibsp">   </span></span>Lord, C.J., Tutt, A.N., and Ashworth, A. (2015) Synthetic lethality and cancer therapy: lessons learned from the development of PARP inhibitors. <em><span class="cmti-12">Annu Rev Med</span></em>, <strong><span class="cmbx-12">66</span></strong>: 455–470.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLu2013"></a><span class="bibsp">   </span></span>Lu, X., Kensche, P.R., Huynen, M.A., and Notebaart, R.A. (2013) Genome evolution predicts genetic interactions in protein complexes and reveals cancer drug targets. <em><span class="cmti-12">Nat Commun</span></em>, <strong><span class="cmbx-12">4</span></strong>: 2124.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLu2015"></a><span class="bibsp">   </span></span>Lu, X., Megchelenbrink, W., Notebaart, R.A., and Huynen, M.A. (2015) Predicting human genetic interactions from cancer genome evolution. <em><span class="cmti-12">PLoS One</span></em>, <strong><span class="cmbx-12">10</span></strong>(5): e0125795.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XLum2004"></a><span class="bibsp">   </span></span>Lum, P.Y., Armour, C.D., Stepaniants, S.B., Cavet, G., Wolf, M.K., Butler, J.S., Hinshaw, J.C., Garnier, P., Prestwich, G.D., Leonardson, A., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2004) Discovering modes of action for therapeutic compounds using a genome-wide screen of yeast heterozygotes. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">116</span></strong>(1): 121–137.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XLuo2009"></a><span class="bibsp">   </span></span>Luo, J., Solimini, N.L., and Elledge, S.J. (2009) Principles of Cancer Therapy: Oncogene and Non-oncogene Addiction. <em><span class="cmti-12">Cell</span></em>, <strong><span class="cmbx-12">136</span></strong>(5): 823–837.</p>
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XMachado2001"></a><span class="bibsp">   </span></span>Machado, J., Olivera, C., Carvalh, R., Soares, P., Berx, G., Caldas, C., Sercuca, R., Carneiro, F., and Sorbrinho-Simoes, M. (2001) E-cadherin gene (<span class="cmti-12">CDH1</span>) promoter methylation as the second hit in sporadic diffuse gastric carcinoma. <em><span class="cmti-12">Oncogene</span></em>, <strong><span class="cmbx-12">20</span></strong>: 1525–1528.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XMasciari2007"></a><span class="bibsp">   </span></span>Masciari, S., Larsson, N., Senz, J., Boyd, N., Kaurah, P., Kandel, M.J., Harris, L.N., Pinheiro, H.C., Troussard, A., Miron, P., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2007) Germline E-cadherin mutations in familial lobular breast cancer. <em><span class="cmti-12">J Med Genet</span></em>, <strong><span class="cmbx-12">44</span></strong>(11): 726–31.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XMattison2009"></a><span class="bibsp">   </span></span>Mattison, J., van der Weyden, L., Hubbard, T., and Adams, D.J. (2009) Cancer gene discovery in mouse and man. <em><span class="cmti-12">Biochim Biophys Acta</span></em>, <strong><span class="cmbx-12">1796</span></strong>(2): 140–161.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XMaxam1977"></a><span class="bibsp">   </span></span>Maxam, A.M. and Gilbert, W. (1977) A new method for sequencing DNA. <em><span class="cmti-12">Proceedings of the National Academy of Science</span></em>, <strong><span class="cmbx-12">74</span></strong>(2): 560–564.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XMcCourt2013"></a><span class="bibsp">   </span></span>McCourt, C.M., McArt, D.G., Mills, K., Catherwood, M.A., Maxwell, P., Waugh, D.J., Hamilton, P., O’Sullivan, J.M., and Salto-Tellez, M. (2013) Validation of next generation sequencing technologies in comparison to current diagnostic gold standards for BRAF, EGFR and KRAS mutational analysis. <em><span class="cmti-12">PLoS ONE</span></em>, <strong><span class="cmbx-12">8</span></strong>(7): e69604.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XMcLachlan2016"></a><span class="bibsp">   </span></span>McLachlan, J., George, A., and Banerjee, S. (2016) The current status of parp inhibitors in ovarian cancer. <em><span class="cmti-12">Tumori</span></em>, <strong><span class="cmbx-12">102</span></strong>(5): 433–440.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2008GBM"></a><span class="bibsp">   </span></span>McLendon, R., Friedman, A., Bigner, D., Van Meir, E.G., Brat, D.J., Mastrogianakis, G.M., Olson, J.J., Mikkelsen, T., Lehman, N., Aldape, K., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2008) Comprehensive genomic characterization defines human glioblastoma genes and core pathways. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">455</span></strong>(7216): 1061–1068.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XMiles2001"></a><span class="bibsp">   </span></span>Miles, D.W. (2001) Update on HER-2 as a target for cancer therapy: herceptin in the clinical setting. <em><span class="cmti-12">Breast Cancer Res</span></em>, <strong><span class="cmbx-12">3</span></strong>(6): 380–384.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XMortazavi2008"></a><span class="bibsp">   </span></span>Mortazavi, A., Williams, B.A., McCue, K., Schaeffer, L., and Wold, B. (2008) Mapping and quantifying mammalian transcriptomes by RNA-Seq. <em><span class="cmti-12">Nat Methods</span></em>, <strong><span class="cmbx-12">5</span></strong>(7): 621–628.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2012CRC"></a><span class="bibsp">   </span></span>Muzny, D.M., Bainbridge, M.N., Chang, K., Dinh, H.H., Drummond, J.A., Fowler, G., Kovar, C.L., Lewis, L.R., Morgan, M.B., Newsham, I.F., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2012) Comprehensive molecular characterization of human colon and rectal cancer. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">487</span></strong>(7407): 330–337.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XNagalla2013"></a><span class="bibsp">   </span></span>Nagalla, S., Chou, J.W., Willingham, M.C., Ruiz, J., Vaughn, J.P., Dubey, P., Lash, T.L., Hamilton-Dutoit, S.J., Bergh, J., Sotiriou, C., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2013) Interactions between immunity, proliferation and molecular subtype in breast cancer prognosis. <em><span class="cmti-12">Genome Biol</span></em>, <strong><span class="cmbx-12">14</span></strong>(4): R34.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XNeeley2009"></a><span class="bibsp">   </span></span>Neeley, E.S., Kornblau, S.M., Coombes, K.R., and Baggerly, K.A. (2009) Variable slope normalization of reverse phase protein arrays. <em><span class="cmti-12">Bioinformatics</span></em>, <strong><span class="cmbx-12">25</span></strong>(11): 1384.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="Xmatrixcalc"></a><span class="bibsp">   </span></span>Novomestky, F. (2012) <em><span class="cmti-12">matrixcalc: Collection of functions for matrix calculations</span></em>. R package version 1.0-3.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XOliveira2009"></a><span class="bibsp">   </span></span>Oliveira, C., Senz, J., Kaurah, P., Pinheiro, H., Sanges, R., Haegert, A., Corso, G., Schouten, J., Fitzgerald, R., Vogelsang, H., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2009) Germline <span class="cmti-12">CDH1 </span>deletions in hereditary diffuse gastric cancer families. <em><span class="cmti-12">Human Molecular Genetics</span></em>, <strong><span class="cmbx-12">18</span></strong>(9): 1545–1555.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XOliveira2013"></a><span class="bibsp">   </span></span>Oliveira, C., Seruca, R., Hoogerbrugge, N., Ligtenberg, M., and Carneiro, F. (2013) Clinical utility gene card for: Hereditary diffuse gastric cancer (HDGC). <em><span class="cmti-12">Eur J Hum</span> <span class="cmti-12">Genet</span></em>, <strong><span class="cmbx-12">21</span></strong>(8).
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XPandey2010"></a><span class="bibsp">   </span></span>Pandey, G., Zhang, B., Chang, A.N., Myers, C.L., Zhu, J., Kumar, V., and Schadt, E.E. (2010) An integrative multi-network and multi-classifier approach to predict genetic interactions. <em><span class="cmti-12">PLoS Comput Biol</span></em>, <strong><span class="cmbx-12">6</span></strong>(9).
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XParker2009"></a><span class="bibsp">   </span></span>Parker, J., Mullins, M., Cheung, M., Leung, S., Voduc, D., Vickery, T., Davies, S., Fauron, C., He, X., Hu, Z., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2009) Supervised risk predictor of breast cancer based on intrinsic subtypes. <em><span class="cmti-12">Journal of Clinical Oncology</span></em>, <strong><span class="cmbx-12">27</span></strong>(8): 1160–1167.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XPeltonen2001"></a><span class="bibsp">   </span></span>Peltonen, L. and McKusick, V.A. (2001) Genomics and medicine. Dissecting human disease in the postgenomic era. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">291</span></strong>(5507): 1224–1229.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XMETABRIC2016"></a><span class="bibsp">   </span></span>Pereira, B., Chin, S.F., Rueda, O.M., Vollan, H.K., Provenzano, E., Bardwell, H.A., Pugh, M., Jones, L., Russell, R., Sammut, S.J., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2016) Erratum: The somatic</p>
mutation profiles of 2,433 breast cancers refine their genomic and transcriptomic landscapes. <em><span class="cmti-12">Nat Commun</span></em>, <strong><span class="cmbx-12">7</span></strong>: 11908.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XPerou2000"></a><span class="bibsp">   </span></span>Perou, C.M., Sørlie, T., Eisen, M.B., van de Rijn, M., Jeffrey, S.S., Rees, C.A., Pollack, J.R., Ross, D.T., Johnsen, H., Akslen, L.A., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2000) Molecular portraits of human breast tumours. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">406</span></strong>(6797): 747–752.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XPleasance2010"></a><span class="bibsp">   </span></span>Pleasance, E.D., Cheetham, R.K., Stephens, P.J., McBride, D.J., Humphray, S.J., Greenman, C.D., Varela, I., Lin, M.L., Ordonez, G.R., Bignell, G.R., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2010) A comprehensive catalogue of somatic mutations from a human cancer genome. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">463</span></strong>(7278): 191–196.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XPolyak2009"></a><span class="bibsp">   </span></span>Polyak, K. and Weinberg, R.A. (2009) Transitions between epithelial and mesenchymal states: acquisition of malignant and stem cell traits. <em><span class="cmti-12">Nat Rev Cancer</span></em>, <strong><span class="cmbx-12">9</span></strong>(4): 265–73.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XPrahallad2012"></a><span class="bibsp">   </span></span>Prahallad, A., Sun, C., Huang, S., Di Nicolantonio, F., Salazar, R., Zecchin, D., Beijersbergen, R.L., Bardelli, A., and Bernards, R. (2012) Unresponsiveness of colon cancer to <span class="cmti-12">BRAF</span>(v600e) inhibition through feedback activation of egfr. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">483</span></strong>(7387): 100–3.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XR_core"></a><span class="bibsp">   </span></span>R Core Team (2016) <em><span class="cmti-12">R: A Language and Environment for Statistical Computing</span></em>. R Foundation for Statistical Computing, Vienna, Austria. R version 3.3.2.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XRavnan2012"></a><span class="bibsp">   </span></span>Ravnan, M.C. and Matalka, M.S. (2012) Vemurafenib in patients with <span class="cmti-12">BRAF </span>v600e mutation-positive advanced melanoma. <em><span class="cmti-12">Clin Ther</span></em>, <strong><span class="cmbx-12">34</span></strong>(7): 1474–86.</p>
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="Xlimma"></a><span class="bibsp">   </span></span>Ritchie, M.E., Phipson, B., Wu, D., Hu, Y., Law, C.W., Shi, W., and Smyth, G.K. (2015) limma powers differential expression analyses for RNA-sequencing and microarray studies. <em><span class="cmti-12">Nucleic Acids Research</span></em>, <strong><span class="cmbx-12">43</span></strong>(7): e47.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XRobin2016"></a><span class="bibsp">   </span></span>Robin, J.D., Ludlow, A.T., LaRanger, R., Wright, W.E., and Shay, J.W. (2016) Comparison of DNA Quantification Methods for Next Generation Sequencing. <em><span class="cmti-12">Sci</span> <span class="cmti-12">Rep</span></em>, <strong><span class="cmbx-12">6</span></strong>: 24067.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XRobinson2010"></a><span class="bibsp">   </span></span>Robinson, M.D. and Oshlack, A. (2010) A scaling normalization method for differential expression analysis of RNA-seq data. <em><span class="cmti-12">Genome Biol</span></em>, <strong><span class="cmbx-12">11</span></strong>(3): R25.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XRoguev2008"></a><span class="bibsp">   </span></span>Roguev, A., Bandyopadhyay, S., Zofall, M., Zhang, K., Fischer, T., Collins, S.R., Qu, H., Shales, M., Park, H.O., Hayles, J., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2008) Conservation and rewiring of functional modules revealed by an epistasis map in fission yeast. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">322</span></strong>(5900): 405–10.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XRung2013"></a><span class="bibsp">   </span></span>Rung, J. and Brazma, A. (2013) Reuse of public genome-wide gene expression data. <em><span class="cmti-12">Nat Rev Genet</span></em>, <strong><span class="cmbx-12">14</span></strong>(2): 89–99.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XArrayExpress2013"></a><span class="bibsp">   </span></span>Rustici, G., Kolesnikov, N., Brandizi, M., Burdett, T., Dylag, M., Emam, I., Farne, A., Hastings, E., Ison, J., Keays, M., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2013) ArrayExpress update–trends in database growth and links to data analysis tools. <em><span class="cmti-12">Nucleic Acids Res</span></em>, <strong><span class="cmbx-12">41</span></strong>(Database issue): D987–990.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XRyan2014"></a><span class="bibsp">   </span></span>Ryan, C., Lord, C., and Ashworth, A. (2014) Daisy: Picking synthetic lethals from cancer genomes. <em><span class="cmti-12">Cancer Cell</span></em>, <strong><span class="cmbx-12">26</span></strong>(3): 306–308.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XSander2014"></a><span class="bibsp">   </span></span>Sander, J.D. and Joung, J.K. (2014) Crispr-cas systems for editing, regulating and targeting genomes. <em><span class="cmti-12">Nat Biotechnol</span></em>, <strong><span class="cmbx-12">32</span></strong>(4): 347–55.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XSanger1975"></a><span class="bibsp">   </span></span>Sanger, F. and Coulson, A. (1975) A rapid method for determining sequences in dna by primed synthesis with dna polymerase. <em><span class="cmti-12">Journal of Molecular Biology</span></em>, <strong><span class="cmbx-12">94</span></strong>(3): 441 – 448.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XScheuer2002"></a><span class="bibsp">   </span></span>Scheuer, L., Kauff, N., Robson, M., Kelly, B., Barakat, R., Satagopan, J., Ellis, N., Hensley, M., Boyd, J., Borgen, P., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2002) Outcome of preventive surgery and screening for breast and ovarian cancer in BRCA mutation carriers. <em><span class="cmti-12">J Clin Oncol</span></em>, <strong><span class="cmbx-12">20</span></strong>(5): 1260–1268.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XSemb1998"></a><span class="bibsp">   </span></span>Semb, H. and Christofori, G. (1998) The tumor-suppressor function of E-cadherin. <em><span class="cmti-12">Am J Hum Genet</span></em>, <strong><span class="cmbx-12">63</span></strong>(6): 1588–93.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XROCR"></a><span class="bibsp">   </span></span>Sing, T., Sander, O., Beerenwinkel, N., and Lengauer, T. (2005) Rocr: visualizing classifier performance in r. <em><span class="cmti-12">Bioinformatics</span></em>, <strong><span class="cmbx-12">21</span></strong>(20): 7881.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="Xslurm"></a><span class="bibsp">   </span></span>Slurm development team (Slurm) (2017) Slurm workload manager. <a 
href="https://slurm.schedmd.com/" class="url" ><span class="cmtt-12">https://slurm.schedmd.com/</span></a>. Accessed: 25/03/2017.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XSorlie2001"></a><span class="bibsp">   </span></span>Sørlie, T., Perou, C.M., Tibshirani, R., Aas, T., Geisler, S., Johnsen, H., Hastie, T., Eisen, M.B., van de Rijn, M., Jeffrey, S.S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2001) Gene expression patterns of breast carcinomas distinguish tumor subclasses with clinical implications. <em><span class="cmti-12">Proc Natl</span> <span class="cmti-12">Acad Sci USA</span></em>, <strong><span class="cmbx-12">98</span></strong>(19): 10869–10874.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XStajich2006"></a><span class="bibsp">   </span></span>Stajich, J.E. and Lapp, H. (2006) Open source tools and toolkits for bioinformatics: significance, and where are we? <em><span class="cmti-12">Brief Bioinformatics</span></em>, <strong><span class="cmbx-12">7</span></strong>(3): 287–296.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XStratton2009"></a><span class="bibsp">   </span></span>Stratton, M.R., Campbell, P.J., and Futreal, P.A. (2009) The cancer genome. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">458</span></strong>(7239): 719–724.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XStrom2012"></a><span class="bibsp">   </span></span>Ström, C. and Helleday, T. (2012) Strategies for the use of poly(adenosine diphosphate ribose) polymerase (parp) inhibitors in cancer therapy. <em><span class="cmti-12">Biomolecules</span></em>, <strong><span class="cmbx-12">2</span></strong>(4): 635–649.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XSun2014"></a><span class="bibsp">   </span></span>Sun, C., Wang, L., Huang, S., Heynen, G.J.J.E., Prahallad, A., Robert, C., Haanen, J., Blank, C., Wesseling, J., Willems, S.M., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Reversible and adaptive resistance to <span class="cmti-12">BRAF</span>(v600e) inhibition in melanoma. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">508</span></strong>(7494): 118–122.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTaylor2009"></a><span class="bibsp">   </span></span>Taylor, I.W., Linding, R., Warde-Farley, D., Liu, Y., Pesquita, C., Faria, D., Bull, S., Pawson, T., Morris, Q., and Wrana, J.L. (2009) Dynamic modularity in protein interaction networks predicts breast cancer outcome. <em><span class="cmti-12">Nat Biotechnol</span></em>, <strong><span class="cmbx-12">27</span></strong>(2): 199–204.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XTelford2015"></a><span class="bibsp">   </span></span>Telford, B.J., Chen, A., Beetham, H., Frick, J., Brew, T.P., Gould, C.M., Single, A.,</p>
Godwin, T., Simpson, K.J., and Guilford, P. (2015) Synthetic lethal screens identify vulnerabilities in gpcr signalling and cytoskeletal organization in E-cadherin-deficient cells. <em><span class="cmti-12">Mol Cancer Ther</span></em>, <strong><span class="cmbx-12">14</span></strong>(5): 1213–1223.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="X1000Genomes2010"></a><span class="bibsp">   </span></span>The 1000 Genomes Project Consortium (1000 Genomes) (2010) A map of human genome variation from population-scale sequencing. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">467</span></strong>(7319): 1061–1073.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2012"></a><span class="bibsp">   </span></span>The Cancer Genome Atlas Research Network (TCGA) (2012) Comprehensive molecular portraits of human breast tumours. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">490</span></strong>(7418): 61–70.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2017web"></a><span class="bibsp">   </span></span>The Cancer Genome Atlas Research Network (TCGA) (2017a) The Cancer Genome Atlas Project. <a 
href="https://cancergenome.nih.gov/" class="url" ><span class="cmtt-12">https://cancergenome.nih.gov/</span></a>. Accessed: 26/03/2017.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2017prov"></a><span class="bibsp">   </span></span>The Cancer Genome Atlas Research Network (TCGA) (2017b) The Cancer Genome Atlas Project Data Portal. <a 
href="https://tcga-data.nci.nih.gov/" class="url" ><span class="cmtt-12">https://tcga-_data.nci.nih.gov/</span></a>. Accessed: 06/02/2017 (via cBioPortal.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XCSNZ2017"></a><span class="bibsp">   </span></span>The Cancer Society of New Zealand (Cancer Society of NZ) (2017) What is cancer? <a 
href="https://otago-southland.cancernz.org.nz/en/cancer-information/other-links/what-is-cancer-3/" class="url" ><span class="cmtt-12">https://otago-_southland.cancernz.org.nz/en/cancer-_information/other-_links/what-_is-_cancer-_3/</span></a>. Accessed: 22/03/2017.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XCOSMICdb"></a><span class="bibsp">   </span></span>The Catalogue Of Somatic Mutations In Cancer (COSMIC) (2016) Cosmic: The catalogue of somatic mutations in cancer. <a 
href="http://cancer.sanger.ac.uk/cosmic" class="url" ><span class="cmtt-12">http://cancer.sanger.ac.uk/cosmic</span></a>. Release 79 (23/08/2016), Accessed: 05/02/2017.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XCRAN"></a><span class="bibsp">   </span></span>The Comprehensive R Archive Network (CRAN) (2017) Cran. <a 
href="https://cran.r-project.org/" class="url" ><span class="cmtt-12">https://cran.r-_project.org/</span></a>. Accessed: 24/03/2017.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XENCODE2004"></a><span class="bibsp">   </span></span>The ENCODE Project Consortium (ENCODE) (2004) The ENCODE (ENCyclopedia Of DNA Elements) Project. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">306</span></strong>(5696): 636–640.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XICGC2017web"></a><span class="bibsp">   </span></span>The Internation Cancer Genome Consortium (ICGC) (2017) ICGC Data Portal. <a 
href="https://dcc.icgc.org/" class="url" ><span class="cmtt-12">https://dcc.icgc.org/</span></a>. Accessed: 06/02/2017.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XNCI2015"></a><span class="bibsp">   </span></span>The National Cancer Institute (NCI) (2015) The genetics of cancer. <a 
href="https://www.cancer.gov/about-cancer/causes-prevention/genetics" class="url" ><span class="cmtt-12">https://www.cancer.gov/about-_cancer/causes-_prevention/genetics</span></a>. Published: 22/04/2015, Accessed: 22/03/2017.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XNeSI"></a><span class="bibsp">   </span></span>The New Zealand eScience Infrastructure (NeSI) (2017) NeSI. <a 
href="https://www.nesi.org.nz/" class="url" ><span class="cmtt-12">https://www.nesi.org.nz/</span></a>. Accessed: 25/03/2017.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XPharmac2016"></a><span class="bibsp">   </span></span>The Pharmaceutical Management Agency (PHARMAC) (2016) Approval of multi-product funding proposal with roche.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="Xsnow"></a><span class="bibsp">   </span></span>Tierney, L., Rossini, A.J., Li, N., and Sevcikova, H. (2015) <em><span class="cmti-12">snow: Simple Network</span> <span class="cmti-12">of Workstations</span></em>. R package version 0.4-2.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XTiong2014"></a><span class="bibsp">   </span></span>Tiong, K.L., Chang, K.C., Yeh, K.T., Liu, T.Y., Wu, J.H., Hsieh, P.H., Lin, S.H.,</p>
Lai, W.Y., Hsu, Y.C., Chen, J.Y., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Csnk1e/ctnnb1 are synthetic lethal to tp53 in colorectal cancer and are markers for prognosis. <em><span class="cmti-12">Neoplasia</span></em>, <strong><span class="cmbx-12">16</span></strong>(5): 441–50.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTischler2008"></a><span class="bibsp">   </span></span>Tischler, J., Lehner, B., and Fraser, A.G. (2008) Evolutionary plasticity of genetic interaction networks. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">40</span></strong>(4): 390–391.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTomasetti2015"></a><span class="bibsp">   </span></span>Tomasetti, C. and Vogelstein, B. (2015) Cancer etiology. Variation in cancer risk among tissues can be explained by the number of stem cell divisions. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">347</span></strong>(6217): 78–81.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTong2001"></a><span class="bibsp">   </span></span>Tong, A.H., Evangelista, M., Parsons, A.B., Xu, H., Bader, G.D., Page, N., Robinson, M., Raghibizadeh, S., Hogue, C.W., Bussey, H., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2001) Systematic genetic analysis with ordered arrays of yeast deletion mutants. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">294</span></strong>(5550): 2364–8.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTong2004"></a><span class="bibsp">   </span></span>Tong, A.H., Lesage, G., Bader, G.D., Ding, H., Xu, H., Xin, X., Young, J., Berriz, G.F., Brost, R.L., Chang, M., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2004) Global mapping of the yeast genetic interaction network. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">303</span></strong>(5659): 808–13.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTravers1969"></a><span class="bibsp">   </span></span>Travers, J. and Milgram, S. (1969) An experimental study of the small world problem. <em><span class="cmti-12">Sociometry</span></em>, <strong><span class="cmbx-12">32</span></strong>(4): 425–443.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XTsai2012"></a><span class="bibsp">   </span></span>Tsai, H.C., Li, H., Van Neste, L., Cai, Y., Robert, C., Rassool, F.V., Shin, J.J., Harbom, K.M., Beaty, R., Pappou, E., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2012) Transient low doses of dna-demethylating agents exert durable antitumor effects on hematological and epithelial tumor cells. <em><span class="cmti-12">Cancer Cell</span></em>, <strong><span class="cmbx-12">21</span></strong>(3): 430–46.</p>
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTutt2010"></a><span class="bibsp">   </span></span>Tutt, A., Robson, M., Garber, J.E., Domchek, S.M., Audeh, M.W., Weitzel, J.N., Friedlander, M., Arun, B., Loman, N., Schmutzler, R.K., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2010) Oral poly(adp-ribose) polymerase inhibitor olaparib in patients with <span class="cmti-12">BRCA1 </span>or <span class="cmti-12">BRCA2</span> mutations and advanced breast cancer: a proof-of-concept trial. <em><span class="cmti-12">Lancet</span></em>, <strong><span class="cmbx-12">376</span></strong>(9737): 235–44.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XvanderMeer2014"></a><span class="bibsp">   </span></span>van der Meer, R., Song, H.Y., Park, S.H., Abdulkadir, S.A., and Roh, M. (2014) RNAi screen identifies a synthetic lethal interaction between PIM1 overexpression and PLK1 inhibition. <em><span class="cmti-12">Clinical Cancer Research</span></em>, <strong><span class="cmbx-12">20</span></strong>(12): 3211–3221.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XvanSteen2011"></a><span class="bibsp">   </span></span>van Steen, K. (2012) Travelling the world of genegene interactions. <em><span class="cmti-12">Briefings in</span> <span class="cmti-12">Bioinformatics</span></em>, <strong><span class="cmbx-12">13</span></strong>(1): 1–19.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XvanSteen2010"></a><span class="bibsp">   </span></span>van Steen, M. (2010) <em><span class="cmti-12">Graph Theory and Complex Networks: An Introduction</span></em>. Maarten van Steen, VU Amsterdam.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XVapnik1995"></a><span class="bibsp">   </span></span>Vapnik, V.N. (1995) <em><span class="cmti-12">The nature of statistical learning theory</span></em>. Springer-Verlag New York, Inc.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XVargas2004"></a><span class="bibsp">   </span></span>Vargas, J.J., Gusella, G., Najfeld, V., Klotman, M., and Cara, A. (2004) Novel integrase-defective lentiviral episomal vectors for gene transfer. <em><span class="cmti-12">Hum Gene Ther</span></em>, <strong><span class="cmbx-12">15</span></strong>: 361–372.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XVizeacoumar2013"></a><span class="bibsp">   </span></span>Vizeacoumar, F.J., Arnold, R., Vizeacoumar, F.S., Chandrashekhar, M., Buzina,</p>
A., Young, J.T., Kwan, J.H., Sayad, A., Mero, P., Lawo, S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2013) A negative genetic interaction map in isogenic cancer cell lines reveals cancer cell vulnerabilities. <em><span class="cmti-12">Mol Syst Biol</span></em>, <strong><span class="cmbx-12">9</span></strong>: 696.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XVogelstein2013"></a><span class="bibsp">   </span></span>Vogelstein, B., Papadopoulos, N., Velculescu, V.E., Zhou, S., Diaz, L.A., and Kinzler, K.W. (2013) Cancer genome landscapes. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">339</span></strong>(6127): 1546–1558.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XVos1997"></a><span class="bibsp">   </span></span>Vos, C.B., Cleton-Jansen, A.M., Berx, G., de Leeuw, W.J., ter Haar, N.T., van Roy, F., Cornelisse, C.J., Peterse, J.L., and van de Vijver, M.J. (1997) E-cadherin inactivation in lobular carcinoma in situ of the breast: an early event in tumorigenesis. <em><span class="cmti-12">Br J Cancer</span></em>, <strong><span class="cmbx-12">76</span></strong>(9): 1131–3.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="Xmapsplice"></a><span class="bibsp">   </span></span>Wang, K., Singh, D., Zeng, Z., Coleman, S.J., Huang, Y., Savich, G.L., He, X., Mieczkowski, P., Grimm, S.A., Perou, C.M., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2010) MapSplice: accurate mapping of RNA-seq reads for splice junction discovery. <em><span class="cmti-12">Nucleic Acids Res</span></em>, <strong><span class="cmbx-12">38</span></strong>(18): e178.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XWang2014"></a><span class="bibsp">   </span></span>Wang, K., Yuen, S.T., Xu, J., Lee, S.P., Yan, H.H., Shi, S.T., Siu, H.C., Deng, S., Chu, K.M., Law, S., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Whole-genome sequencing and comprehensive molecular profiling identify new driver mutations in gastric cancer. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">46</span></strong>(6): 573–582.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XWang2013"></a><span class="bibsp">   </span></span>Wang, X. and Simon, R. (2013) Identification of potential synthetic lethal genes to p53 using a computational biology approach. <em><span class="cmti-12">BMC Medical Genomics</span></em>, <strong><span class="cmbx-12">6</span></strong>(1): 30.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XWappett2014"></a><span class="bibsp">   </span></span>Wappett, M. (2014) Bisep: Toolkit to identify candidate synthetic lethality. r package version 2.0.</p>
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XWappett2016"></a><span class="bibsp">   </span></span>Wappett, M., Dulak, A., Yang, Z.R., Al-Watban, A., Bradford, J.R., and Dry, J.R. (2016) Multi-omic measurement of mutually exclusive loss-of-function enriches for candidate synthetic lethal gene pairs. <em><span class="cmti-12">BMC Genomics</span></em>, <strong><span class="cmbx-12">17</span></strong>: 65.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="Xgplots"></a><span class="bibsp">   </span></span>Warnes, G.R., Bolker, B., Bonebakker, L., Gentleman, R., Liaw, W.H.A., Lumley, T., Maechler, M., Magnusson, A., Moeller, S., Schwartz, M., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2015) <em><span class="cmti-12">gplots:</span> <span class="cmti-12">Various R Programming Tools for Plotting Data</span></em>. R package version 2.17.0.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XWatts1998"></a><span class="bibsp">   </span></span>Watts, D.J. and Strogatz, S.H. (1998) Collective dynamics of ’small-world’ networks. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">393</span></strong>(6684): 440–2.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XWeinstein2000"></a><span class="bibsp">   </span></span>Weinstein, I.B. (2000) Disorders in cell circuitry during multistage carcinogenesis: the role of homeostasis. <em><span class="cmti-12">Carcinogenesis</span></em>, <strong><span class="cmbx-12">21</span></strong>(5): 857–864.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2014BL"></a><span class="bibsp">   </span></span>Weinstein, J.N., Akbani, R., Broom, B.M., Wang, W., Verhaak, R.G., McConkey, D., Lerner, S., Morgan, M., Creighton, C.J., Smith, C., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2014) Comprehensive molecular characterization of urothelial bladder carcinoma. <em><span class="cmti-12">Nature</span></em>, <strong><span class="cmbx-12">507</span></strong>(7492): 315–322.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XTCGA2013PAN"></a><span class="bibsp">   </span></span>Weinstein, J.N., Collisson, E.A., Mills, G.B., Shaw, K.R., Ozenberger, B.A., Ellrott, K., Shmulevich, I., Sander, C., Stuart, J.M., Chang, K., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2013) The Cancer Genome Atlas Pan-Cancer analysis project. <em><span class="cmti-12">Nat Genet</span></em>, <strong><span class="cmbx-12">45</span></strong>(10): 1113–1120.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="Xdevtools"></a><span class="bibsp">   </span></span>Wickham, H. and Chang, W. (2016) <em><span class="cmti-12">devtools: Tools to Make Developing R Packages</span> <span class="cmti-12">Easier</span></em>. R package version 1.12.0.</p>
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="Xroxygen"></a><span class="bibsp">   </span></span>Wickham, H., Danenberg, P., and Eugster, M. (2017) <em><span class="cmti-12">roxygen2: In-Line</span> <span class="cmti-12">Documentation for R</span></em>. R package version 6.0.1.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XWong2004"></a><span class="bibsp">   </span></span>Wong, S.L., Zhang, L.V., Tong, A.H.Y., Li, Z., Goldberg, D.S., King, O.D., Lesage, G., Vidal, M., Andrews, B., Bussey, H., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2004) Combining biological networks to predict genetic interactions. <em><span class="cmti-12">Proceedings of the National Academy of Sciences of</span> <span class="cmti-12">the United States of America</span></em>, <strong><span class="cmbx-12">101</span></strong>(44): 15682–15687.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XWorldHealthOrg2017"></a><span class="bibsp">   </span></span>World Health Organization (WHO) (2017) Fact sheet: Cancer. <a 
href="http://www.who.int/mediacentre/factsheets/fs297/en/" class="url" ><span class="cmtt-12">http://www.who.int/mediacentre/factsheets/fs297/en/</span></a>. Updated February 2017, Accessed: 22/03/2017.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XWu2014"></a><span class="bibsp">   </span></span>Wu, M., Li, X., Zhang, F., Li, X., Kwoh, C.K., and Zheng, J. (2014) In silico prediction of synthetic lethality by meta-analysis of genetic interactions, functions, and pathways in yeast and human cancer. <em><span class="cmti-12">Cancer Inform</span></em>, <strong><span class="cmbx-12">13</span></strong>(Suppl 3): 71–80.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XRmpi"></a><span class="bibsp">   </span></span>Yu, H. (2002) Rmpi: Parallel statistical computing in r. <em><span class="cmti-12">R News</span></em>, <strong><span class="cmbx-12">2</span></strong>(2): 10–14.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XZhang2015"></a><span class="bibsp">   </span></span>Zhang, F., Wu, M., Li, X.J., Li, X.L., Kwoh, C.K., and Zheng, J. (2015) Predicting essential genes and synthetic lethality via influence propagation in signaling pathways of cancer cell fates. <em><span class="cmti-12">J Bioinform Comput Biol</span></em>, <strong><span class="cmbx-12">13</span></strong>(3): 1541002.
</p>
<p class="bibitem">
<p><span class="biblabel"> <a 
 id="XICGC2011"></a><span class="bibsp">   </span></span>Zhang, J., Baran, J., Cros, A., Guberman, J.M., Haider, S., Hsu, J., Liang, Y., Rivkin, E., Wang, J., Whitty, B., <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em> (2011) International cancer genome</p>
consortium data portala one-stop shop for cancer genomics data. <em><span class="cmti-12">Database: The</span> <span class="cmti-12">Journal of Biological Databases and Curation</span></em>, <strong><span class="cmbx-12">2011</span></strong>: bar026.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XZhong2006"></a><span class="bibsp">   </span></span>Zhong, W. and Sternberg, P.W. (2006) Genome-wide prediction of c. elegans genetic interactions. <em><span class="cmti-12">Science</span></em>, <strong><span class="cmbx-12">311</span></strong>(5766): 1481–1484.
</p>
<p class="bibitem">
<span class="biblabel"> <a 
 id="XZweig1993"></a><span class="bibsp">   </span></span>Zweig, M.H. and Campbell, G. (1993) Receiver-operating characteristic (roc) plots: a fundamental evaluation tool in clinical medicine. <em><span class="cmti-12">Clinical Chemistry</span></em>, <strong><span class="cmbx-12">39</span></strong>(4): 561–577.
</p>
</div>
<a 
 id="x1-198001r288"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix A</span><br /><a 
 id="x1-199000A"></a>Sample Quality
</h2>
<a 
 id="x1-199001r287"></a>
<h3 class="sectionHead">
<span class="titlemark">A.1 </span> <a 
 id="x1-2000001"></a>Sample Correlation
</h3>
<!--l. 22-->
<p class="noindent">
<p>Samples were excluded from expression analysis based on sample correlations and the clustering analysis presented below, as described in Section <a 
href="#x1-720002">2.2.2<!--tex4ht:ref: methods:sample_qc --></a>.</p>
</p>
<!--l. 24-->
<p class="indent">
<a 
 id="x1-200001r1"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 26-->
<p class="noindent">
</p>
<!--l. 27-->
<p class="noindent">
<img 
src="Raw(log)_Corr_eudist_Cut_part_Pub.png" alt="PIC"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure A.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Correlation profiles of removed samples.</span></strong> <span class="cmr-10x-x-109">Correlation matrix heatmap</span> <span class="cmr-10x-x-109">(Euclidean distance) of all samples in TCGA breast cancer dataset (left) clustered for</span> <span class="cmr-10x-x-109">all samples against removed samples (top): tissue source site (TSS), sample type with</span> <span class="cmr-10x-x-109">reds for tumour and greens for normal, patient (A2QH in pink), with varied analyte and</span> <span class="cmr-10x-x-109">plate (corresponding to batch in Table</span><span class="cmr-10x-x-109"> </span><a 
href="#x1-72007r1"><span class="cmr-10x-x-109">2.1</span><!--tex4ht:ref: tab:qc --></a><span class="cmr-10x-x-109">). Excluded samples cluster at the bottom and</span> <span class="cmr-10x-x-109">annotation (left) show shared properties between samples in the dataset. </span></span></p>
</div>
<!--tex4ht:label?: x1-200001r1 -->
</div>
<hr class="endfloat" />
<!--l. 37-->
<p class="indent">
<a 
 id="x1-200002r2"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 39-->
<p class="noindent">
</p>
<!--l. 40-->
<p class="noindent">
<img 
src="Raw(log)_Corr_eudist_Ann_Pub.png" alt="PIC"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure A.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Correlation analysis and sample removal.</span></strong> <span class="cmr-10x-x-109">Correlation matrix</span> <span class="cmr-10x-x-109">heatmap (Euclidean distance) of all samples in TCGA breast cancer dataset against</span> <span class="cmr-10x-x-109">each other annotated for sample clinical data: sample type, tissue type, tumour stage,</span> <span class="cmr-10x-x-109">Estrogen receptor (IHC) and intrinsic subtype (from the PAM50 method). CDH1 somatic</span> <span class="cmr-10x-x-109">mutation, gene expression, and status for SLIPT prediction are also annotated. Discrete</span> <span class="cmr-10x-x-109">variables are coloured as displayed in the legend and continuous variables on a blue-red</span> <span class="cmr-10x-x-109">scale as shown in the colour key. Trimmed samples cluster at the bottom of the heatmap</span> <span class="cmr-10x-x-109">and the colour bars of the left show which were removed for quality concerns.</span></span></p>
</div>
<!--tex4ht:label?: x1-200002r2 -->
</div>
<hr class="endfloat" />
<a 
 id="x1-200003r292"></a>
<h3 class="sectionHead">
<span class="titlemark">A.2 </span> <a 
 id="x1-2010002"></a>Replicate Samples in TCGA Breast
</h3>
<!--l. 54-->
<p class="noindent">
Replicate samples were picked where possible from the TCGA breast cancer gene expression data to examine for sample quality. Independent samples of the same tumour are expected to have very high Pearson’s correlation between their expression profiles unless there were issues with sample collection or preparation and are thus an indicator of sample quality. The log-transformed raw read counts for replicate samples were examined in Figures <a 
href="#x1-201001r3">A.3<!--tex4ht:ref: fig:rep_cutcut --></a>–<a 
href="#x1-201024r6">A.6<!--tex4ht:ref: fig:rep_keepcut --></a>. These were examined before normalisation which would be expected to increase sample concordance.
</p>
<!--l. 56-->
<p class="indent">
<p>Another consideration are the samples which were removed for quality concerns (in Section <a 
href="#x1-720002">2.2.2<!--tex4ht:ref: methods:sample_qc --></a>). While these were selected by unbiased hierarchical clustering (See Figure <a 
href="#x1-200002r2">A.2<!--tex4ht:ref: fig:corr_map --></a>), it is notable that many of the exluded (tumour) samples were performed in replicate despite relatively few replicate samples in the overall dataset. These samples correlate poorly with the rest of the dataset, in addition to with replicate samples.</p>
</p>
<!--l. 58-->
<p class="indent">
<a 
 id="x1-201001r3"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________      <!--l. 61-->
<p class="noindent">
<img 
src="Rep_picking_CutCut2.png" alt="pict"  
 /> <br />
</p>
<div class="caption">
<p><span class="id">Figure A.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">Replicate excluded samples.</span></strong> <span class="cmr-10x-x-109">Both tumour samples of patient A2QH</span> <span class="cmr-10x-x-109">were excluded as they were poorly correlated with other samples, although they are</span> <span class="cmr-10x-x-109">highly similar to each other as shown by Pearson’s correlation of log-raw counts. </span></span></p>
</div>
<!--tex4ht:label?: x1-201001r3 -->
</div>
<hr class="endfloat" />
<!--l. 70-->
<p class="indent">
<a 
 id="x1-201008r4"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 72-->
<p class="noindent">
<a 
 id="x1-201002r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepKeep(1).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Remaining</span> <span class="cmr-10">triplet</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201003r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepKeep(2).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Remaining</span> <span class="cmr-10">triplet</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201004r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepKeep(3).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmr-10">Remaining</span> <span class="cmr-10">triplet</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201005r4"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepKeep(4).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(d)</span> <span class="cmr-10">Remaining</span> <span class="cmr-10">paired</span> <span class="cmr-10">samples</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201006r5"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepKeep(5).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(e)</span> <span class="cmr-10">Remaining</span> <span class="cmr-10">paired</span> <span class="cmr-10">samples</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201007r6"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepKeep(6).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(f)</span> <span class="cmr-10">Remaining</span> <span class="cmr-10">paired</span> <span class="cmr-10">samples</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure A.4: </span><span class="content"><strong><span class="cmbx-10x-x-109">Replicate samples with all remaining.</span></strong> <span class="cmr-10x-x-109">Patient A26J was sampled 3</span> <span class="cmr-10x-x-109">times and compared pairwise. Pairs of samples were also compared for other patients with</span> <span class="cmr-10x-x-109">replicate samples. In all cases, replicate samples remaining in the dataset were highly</span> <span class="cmr-10x-x-109">concordant as shown by Pearson’s correlation of log-raw counts. </span></span></p>
</div>
<!--tex4ht:label?: x1-201008r4 -->
</div>
<hr class="endfloat" />
<!--l. 110-->
<p class="indent">
<a 
 id="x1-201018r5"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 112-->
<p class="noindent">
<a 
 id="x1-201009r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepCut(1).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmr-10">Remaining</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201010r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepCut(2).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmr-10">Compare</span> <span class="cmr-10">with</span> <span class="cmr-10">excluded</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201011r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepCut(3).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmr-10">Compare</span> <span class="cmr-10">with</span> <span class="cmr-10">excluded</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201012r4"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepCut(4).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(d)</span> <span class="cmr-10">Remaining</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201013r5"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepCut(5).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(e)</span> <span class="cmr-10">Compare</span> <span class="cmr-10">with</span> <span class="cmr-10">excluded</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201014r6"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepCut(6).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(f)</span> <span class="cmr-10">Compare</span> <span class="cmr-10">with</span> <span class="cmr-10">excluded</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201015r7"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepCut(7).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(g)</span> <span class="cmr-10">Remaining</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201016r8"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepCut(8).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(h)</span> <span class="cmr-10">Compare</span> <span class="cmr-10">with</span> <span class="cmr-10">excluded</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201017r9"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepCut(9).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(i)</span> <span class="cmr-10">Compare</span> <span class="cmr-10">with</span> <span class="cmr-10">excluded</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure A.5: </span><span class="content"><strong><span class="cmbx-10x-x-109">Replicate samples with some excluded.</span></strong> <span class="cmr-10x-x-109">Patients A0DB, A13D, A13E,</span> <span class="cmr-10x-x-109">and A26E were each sampled 3 times and compared pairwise. Pairs of samples were also</span> <span class="cmr-10x-x-109">compared for other patients with replicate samples. In all cases, the replicate samples</span> <span class="cmr-10x-x-109">remaining in the dataset more were highly concordant (as shown by Pearson’s correlation</span> <span class="cmr-10x-x-109">of log-raw counts) than those excluded from the analysis. </span></span></p>
</div>
<!--tex4ht:label?: x1-201018r5 -->
</div>
<hr class="endfloat" />
<!--l. 162-->
<p class="indent">
<a 
 id="x1-201024r6"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 164-->
<p class="noindent">
<a 
 id="x1-201019r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepCut(10).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(j)</span> <span class="cmr-10">Remaining</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201020r11"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepCut(11).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(k)</span> <span class="cmr-10">Compare</span> <span class="cmr-10">with</span> <span class="cmr-10">excluded</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201021r12"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepCut(12).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(l)</span> <span class="cmr-10">Compare</span> <span class="cmr-10">with</span> <span class="cmr-10">excluded</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201022r13"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepCut(13).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(m)</span> <span class="cmr-10">Compare</span> <span class="cmr-10">with</span> <span class="cmr-10">excluded</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-201023r14"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Rep_picking_KeepCut(14).png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(n)</span> <span class="cmr-10">Compare</span> <span class="cmr-10">with</span> <span class="cmr-10">excluded</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure A.6: </span><span class="content"><strong><span class="cmbx-10x-x-109">Replicate samples with some excluded.</span></strong> <span class="cmr-10x-x-109">Patients A0DB, A13D, A13E,</span> <span class="cmr-10x-x-109">and A26E were each sampled 3 times and compared pairwise. Pairs of samples were also</span> <span class="cmr-10x-x-109">compared for other patients with replicate samples. In all cases, the replicate samples</span> <span class="cmr-10x-x-109">remaining in the dataset more were highly concordant (as shown by Pearson’s correlation</span> <span class="cmr-10x-x-109">of log-raw counts) than those excluded from the analysis. </span></span></p>
</div>
<!--tex4ht:label?: x1-201024r6 -->
</div>
<hr class="endfloat" />
<a 
 id="x1-201025r288"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix B</span><br /><a 
 id="x1-202000B"></a>Software Used for Thesis
</h2>
<a 
 id="x1-202001r1"></a> <!--l. 209-->
<div class="longtable">
<table id="TBL-77" class="longtable" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-77-1g">
<col 
id="TBL-77-1" />
<col 
id="TBL-77-2" />
<col 
id="TBL-77-3" />
<col 
id="TBL-77-4" />
<col 
id="TBL-77-5" />
<col 
id="TBL-77-6" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-77-1-">
<td colspan="6" style="white-space:nowrap; text-align:center;" id="TBL-77-1-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<div class="caption">
<p><span class="id">Table B.1: </span><span class="content">R Packages used during Thesis</span></p>
</div>
<!--tex4ht:label?: x1-202001r1 -->
</div>
<a 
 id="x1-202002"></a>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-2-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Package</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-2-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Repository</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-2-3" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Laptop</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-2-4" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Lab</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-2-5" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">Server</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-2-6" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p><span class="cmbx-12">NeSI</span></p>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-3-">
<td colspan="6" style="white-space:nowrap; text-align:left;" id="TBL-77-3-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-4-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-5-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-6-">
<td colspan="6" style="white-space:nowrap; text-align:left;" id="TBL-77-6-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-7-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-8-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-9-1" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-9-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-9-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-9-4" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-9-5" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-9-6" class="td11">
3.3.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-10-1" class="td11">
abind
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-10-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-10-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-10-4" class="td11">
1.4-5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-10-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-10-6" class="td11">
1.4-3
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-11-1" class="td11">
acepack
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-11-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-11-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-11-4" class="td11">
1.4.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-11-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-11-6" class="td11">
1.3-3.3
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-12-1" class="td11">
ade4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-12-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-12-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-12-4" class="td11">
1.7-5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-12-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-12-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-13-1" class="td11">
annaffy
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-13-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-13-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-13-4" class="td11">
1.46.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-13-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-13-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-14-1" class="td11">
AnnotationDbi
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-14-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-14-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-14-4" class="td11">
1.36.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-14-5" class="td11">
1.36.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-14-6" class="td11">
1.34.4
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-15-1" class="td11">
apComplex
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-15-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-15-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-15-4" class="td11">
2.40.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-15-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-15-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-16-1" class="td11">
ape
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-16-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-16-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-16-4" class="td11">
4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-16-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-16-6" class="td11">
3.4
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-17-1" class="td11">
arm
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-17-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-17-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-17-4" class="td11">
1.9-3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-17-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-17-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-18-1" class="td11">
assertthat
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-18-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-18-3" class="td11">
0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-18-4" class="td11">
0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-18-5" class="td11">
0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-18-6" class="td11">
0.1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-19-1" class="td11">
backports
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-19-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-19-3" class="td11">
1.0.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-19-4" class="td11">
1.0.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-19-5" class="td11">
1.0.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-19-6" class="td11">
1.0.2
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-20-1" class="td11">
base64
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-20-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-20-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-20-4" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-20-5" class="td11">
2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-20-6" class="td11">
2
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-21-1" class="td11">
base64enc
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-21-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-21-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-21-4" class="td11">
0.1-3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-21-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-21-6" class="td11">
0.1-3
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-22-1" class="td11">
beanplot
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-22-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-22-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-22-4" class="td11">
1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-22-5" class="td11">
1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-22-6" class="td11">
1.2
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-23-1" class="td11">
BH
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-23-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-23-3" class="td11">
1.60.0-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-23-4" class="td11">
1.62.0-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-23-5" class="td11">
1.62.0-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-23-6" class="td11">
1.60.0-2
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-24-1" class="td11">
Biobase
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-24-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-24-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-24-4" class="td11">
2.34.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-24-5" class="td11">
2.34.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-24-6" class="td11">
2.32.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-25-1" class="td11">
BiocGenerics
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-25-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-25-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-25-4" class="td11">
0.20.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-25-5" class="td11">
0.20.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-25-6" class="td11">
0.18.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-26-1" class="td11">
BiocInstaller
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-26-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-26-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-26-4" class="td11">
1.24.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-26-5" class="td11">
1.20.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-26-6" class="td11">
1.22.3
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-27-1" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-27-2" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-27-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-27-4" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-27-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-27-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-28-1" class="td11">
BiocParallel
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-28-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-28-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-28-4" class="td11">
1.8.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-28-5" class="td11">
1.8.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-28-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-29-1" class="td11">
Biostrings
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-29-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-29-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-29-4" class="td11">
2.42.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-29-5" class="td11">
2.42.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-29-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-30-1" class="td11">
BiSEp
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-30-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-30-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-30-4" class="td11">
2.0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-30-5" class="td11">
2.0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-30-6" class="td11">
2.0.1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-31-1" class="td11">
bitops
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-31-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-31-3" class="td11">
1.0-6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-31-4" class="td11">
1.0-6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-31-5" class="td11">
1.0-6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-31-6" class="td11">
1.0-6
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-32-1" class="td11">
boot
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-32-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-32-3" class="td11">
1.3-18
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-32-4" class="td11">
1.3-18
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-32-5" class="td11">
1.3-18
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-32-6" class="td11">
1.3-18
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-33-1" class="td11">
brew
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-33-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-33-3" class="td11">
1.0-6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-33-4" class="td11">
1.0-6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-33-5" class="td11">
1.0-6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-33-6" class="td11">
1.0-6
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-34-1" class="td11">
broom
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-34-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-34-3" class="td11">
0.4.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-34-4" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-34-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-34-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-35-1" class="td11">
caTools
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-35-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-35-3" class="td11">
1.17.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-35-4" class="td11">
1.17.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-35-5" class="td11">
1.17.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-35-6" class="td11">
1.17.1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-36-1" class="td11">
cgdsr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-36-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-36-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-36-4" class="td11">
1.2.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-36-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-36-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-37-1" class="td11">
checkmate
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-37-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-37-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-37-4" class="td11">
1.8.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-37-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-37-6" class="td11">
1.7.4
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-38-1" class="td11">
chron
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-38-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-38-3" class="td11">
2.3-47
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-38-4" class="td11">
2.3-48
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-38-5" class="td11">
2.3-50
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-38-6" class="td11">
2.3-47
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-39-1" class="td11">
class
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-39-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-39-3" class="td11">
7.3-14
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-39-4" class="td11">
7.3-14
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-39-5" class="td11">
7.3-14
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-39-6" class="td11">
7.3-14
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-40-1" class="td11">
cluster
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-40-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-40-3" class="td11">
2.0.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-40-4" class="td11">
2.0.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-40-5" class="td11">
2.0.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-40-6" class="td11">
2.0.4
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-41-1" class="td11">
coda
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-41-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-41-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-41-4" class="td11">
0.19-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-41-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-41-6" class="td11">
0.18-1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-42-1" class="td11">
codetools
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-42-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-42-3" class="td11">
0.2-15
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-42-4" class="td11">
0.2-15
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-42-5" class="td11">
0.2-15
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-42-6" class="td11">
0.2-14
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-43-1" class="td11">
colorRamps
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-43-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-43-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-43-4" class="td11">
2.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-43-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-43-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-44-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-44-1" class="td11">
colorspace
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-44-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-44-3" class="td11">
1.2-6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-44-4" class="td11">
1.3-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-44-5" class="td11">
1.3-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-44-6" class="td11">
1.2-6
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-45-1" class="td11">
commonmark
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-45-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-45-3" class="td11">
1.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-45-4" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-45-5" class="td11">
1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-45-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-46-1" class="td11">
compiler
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-46-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-46-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-46-4" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-46-5" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-46-6" class="td11">
3.3.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-47-1" class="td11">
corpcor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-47-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-47-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-47-4" class="td11">
1.6.8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-47-5" class="td11">
1.6.8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-47-6" class="td11">
1.6.8
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-48-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-49-1" class="td11">
Cprob
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-49-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-49-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-49-4" class="td11">
1.2.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-49-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-49-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-50-1" class="td11">
crayon
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-50-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-50-3" class="td11">
1.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-50-4" class="td11">
1.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-50-5" class="td11">
1.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-50-6" class="td11">
1.3.2
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-51-1" class="td11">
crop
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-51-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-51-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-51-4" class="td11">
0.0-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-51-5" class="td11">
0.0-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-51-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-52-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-52-1" class="td11">
curl
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-52-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-52-3" class="td11">
1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-52-4" class="td11">
2.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-52-5" class="td11">
2.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-52-6" class="td11">
0.9.7
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-53-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-53-1" class="td11">
d3Network
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-53-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-53-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-53-4" class="td11">
0.5.2.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-53-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-53-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-54-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-54-1" class="td11">
data.table
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-54-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-54-3" class="td11">
1.9.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-54-4" class="td11">
1.10.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-54-5" class="td11">
1.10.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-54-6" class="td11">
1.9.6
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-55-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-55-1" class="td11">
data.tree
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-55-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-55-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-55-4" class="td11">
0.7.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-55-5" class="td11">
0.7.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-55-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-56-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-56-1" class="td11">
datasets
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-56-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-56-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-56-4" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-56-5" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-56-6" class="td11">
3.3.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-57-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-57-1" class="td11">
DBI
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-57-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-57-3" class="td11">
0.5-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-57-4" class="td11">
0.5-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-57-5" class="td11">
0.5-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-57-6" class="td11">
0.5-1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-58-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-58-1" class="td11">
dendextend
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-58-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-58-3" class="td11">
1.4.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-58-4" class="td11">
1.4.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-58-5" class="td11">
1.4.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-58-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-59-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-59-1" class="td11">
DEoptimR
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-59-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-59-3" class="td11">
1.0-8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-59-4" class="td11">
1.0-8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-59-5" class="td11">
1.0-8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-59-6" class="td11">
1.0-4
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-60-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-60-1" class="td11">
desc
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-60-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-60-3" class="td11">
1.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-60-4" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-60-5" class="td11">
1.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-60-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-61-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-61-1" class="td11">
devtools
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-61-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-61-3" class="td11">
1.12.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-61-4" class="td11">
1.12.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-61-5" class="td11">
1.12.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-61-6" class="td11">
1.12.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-62-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-62-1" class="td11">
DiagrammeR
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-62-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-62-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-62-4" class="td11">
0.9.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-62-5" class="td11">
0.9.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-62-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-63-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-63-1" class="td11">
dichromat
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-63-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-63-3" class="td11">
2.0-0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-63-4" class="td11">
2.0-0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-63-5" class="td11">
2.0-0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-63-6" class="td11">
2.0-0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-64-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-64-1" class="td11">
digest
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-64-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-64-3" class="td11">
0.6.10
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-64-4" class="td11">
0.6.11
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-64-5" class="td11">
0.6.12
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-64-6" class="td11">
0.6.9
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-65-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-65-1" class="td11">
diptest
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-65-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-65-3" class="td11">
0.75-7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-65-4" class="td11">
0.75-7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-65-5" class="td11">
0.75-7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-65-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-66-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-66-1" class="td11">
doParallel
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-66-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-66-3" class="td11">
1.0.10
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-66-4" class="td11">
1.0.10
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-66-5" class="td11">
1.0.10
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-66-6" class="td11">
1.0.10
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-67-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-67-1" class="td11">
dplyr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-67-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-67-3" class="td11">
0.5.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-67-4" class="td11">
0.5.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-67-5" class="td11">
0.5.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-67-6" class="td11">
0.5.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-68-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-68-1" class="td11">
ellipse
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-68-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-68-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-68-4" class="td11">
0.3-8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-68-5" class="td11">
0.3-8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-68-6" class="td11">
0.3-8
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-69-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-69-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-70-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-70-1" class="td11">
evaluate
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-70-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-70-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-70-4" class="td11">
0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-70-5" class="td11">
0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-70-6" class="td11">
0.9
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-71-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-71-1" class="td11">
fdrtool
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-71-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-71-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-71-4" class="td11">
1.2.15
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-71-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-71-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-72-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-72-1" class="td11">
fields
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-72-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-72-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-72-4" class="td11">
8.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-72-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-72-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-73-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-73-1" class="td11">
flexmix
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-73-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-73-3" class="td11">
2.3-13
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-73-4" class="td11">
2.3-13
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-73-5" class="td11">
2.3-13
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-73-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-74-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-74-1" class="td11">
forcats
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-74-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-74-3" class="td11">
0.2.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-74-4" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-74-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-74-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-75-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-75-1" class="td11">
foreach
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-75-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-75-3" class="td11">
1.4.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-75-4" class="td11">
1.4.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-75-5" class="td11">
1.4.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-75-6" class="td11">
1.4.3
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-76-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-76-1" class="td11">
foreign
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-76-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-76-3" class="td11">
0.8-67
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-76-4" class="td11">
0.8-67
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-76-5" class="td11">
0.8-67
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-76-6" class="td11">
0.8-66
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-77-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-77-1" class="td11">
formatR
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-77-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-77-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-77-4" class="td11">
1.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-77-5" class="td11">
1.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-77-6" class="td11">
1.4
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-78-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-78-1" class="td11">
Formula
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-78-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-78-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-78-4" class="td11">
1.2-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-78-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-78-6" class="td11">
1.2-1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-79-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-79-1" class="td11">
fpc
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-79-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-79-3" class="td11">
2.1-10
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-79-4" class="td11">
2.1-10
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-79-5" class="td11">
2.1-10
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-79-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-80-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-80-1" class="td11">
futile.logger
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-80-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-80-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-80-4" class="td11">
1.4.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-80-5" class="td11">
1.4.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-80-6" class="td11">
1.4.1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-81-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-81-1" class="td11">
futile.options
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-81-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-81-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-81-4" class="td11">
1.0.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-81-5" class="td11">
1.0.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-81-6" class="td11">
1.0.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-82-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-82-1" class="td11">
gdata
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-82-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-82-3" class="td11">
2.17.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-82-4" class="td11">
2.17.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-82-5" class="td11">
2.17.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-82-6" class="td11">
2.17.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-83-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-83-1" class="td11">
geepack
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-83-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-83-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-83-4" class="td11">
1.2-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-83-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-83-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-84-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-84-1" class="td11">
GenomeInfoDb
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-84-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-84-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-84-4" class="td11">
1.10.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-84-5" class="td11">
1.10.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-84-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-85-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-85-1" class="td11">
GenomicAlignments
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-85-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-85-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-85-4" class="td11">
1.10.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-85-5" class="td11">
1.10.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-85-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-86-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-86-1" class="td11">
GenomicRanges
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-86-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-86-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-86-4" class="td11">
1.26.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-86-5" class="td11">
1.26.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-86-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-87-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-87-1" class="td11">
ggm
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-87-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-87-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-87-4" class="td11">
2.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-87-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-87-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-88-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-88-1" class="td11">
ggplot2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-88-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-88-3" class="td11">
2.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-88-4" class="td11">
2.2.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-88-5" class="td11">
2.2.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-88-6" class="td11">
2.1.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-89-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-89-1" class="td11">
git2r
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-89-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-89-3" class="td11">
0.15.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-89-4" class="td11">
0.18.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-89-5" class="td11">
0.16.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-89-6" class="td11">
0.15.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-90-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-90-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-91-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-91-1" class="td11">
glasso
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-91-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-91-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-91-4&quot;  </code></pre>
class=“td11”&gt; 1.8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-91-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-91-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-92-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-92-1" class="td11">
GO.db
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-92-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-92-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-92-4&quot;  </code></pre>
class=“td11”&gt; 3.4.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-92-5" class="td11">
3.2.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-92-6" class="td11">
3.3.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-93-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-93-1" class="td11">
GOSemSim
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-93-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-93-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-93-4&quot;  </code></pre>
class=“td11”&gt; 2.0.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-93-5" class="td11">
1.28.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-93-6" class="td11">
1.30.3
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-94-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-94-1" class="td11">
gplots
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-94-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-94-3" class="td11">
3.0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-94-4" class="td11">
3.0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-94-5" class="td11">
3.0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-94-6" class="td11">
3.0.1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-95-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-95-1" class="td11">
graph
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-95-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-95-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-95-4&quot;  </code></pre>
class=“td11”&gt; 1.52.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-95-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-95-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-96-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-96-1" class="td11">
graphics
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-96-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-96-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-96-4" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-96-5" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-96-6" class="td11">
3.3.0
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-97-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-97-1" class="td11">
graphsim
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-97-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-78" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-78-1g">
<col 
id="TBL-78-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-78-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-78-1-1" class="td00">
GitHub
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-78-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-78-2-1" class="td00">
TomKellyGenetics
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-97-3" class="td11">
0.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-97-4" class="td11">
0.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-97-5" class="td11">
0.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-97-6" class="td11">
0.1.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-98-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-98-1" class="td11">
grDevices
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-98-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-98-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-98-4" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-98-5" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-98-6" class="td11">
3.3.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-99-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-99-1" class="td11">
grid
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-99-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-99-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-99-4" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-99-5" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-99-6" class="td11">
3.3.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-100-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-100-1" class="td11">
gridBase
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-100-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-100-3" class="td11">
0.4-7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-100-4" class="td11">
0.4-7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-100-5" class="td11">
0.4-7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-100-6" class="td11">
0.4-7
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-101-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-101-1" class="td11">
gridExtra
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-101-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-101-3" class="td11">
2.2.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-101-4" class="td11">
2.2.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-101-5" class="td11">
2.2.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-101-6" class="td11">
2.2.1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-102-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-102-1" class="td11">
gridGraphics
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-102-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-102-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-102-4&quot;  </code></pre>
class=“td11”&gt; 0.1-5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-102-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-102-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-103-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-103-1" class="td11">
gtable
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-103-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-103-3" class="td11">
0.2.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-103-4" class="td11">
0.2.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-103-5" class="td11">
0.2.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-103-6" class="td11">
0.2.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-104-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-104-1" class="td11">
gtools
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-104-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-104-3" class="td11">
3.5.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-104-4" class="td11">
3.5.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-104-5" class="td11">
3.5.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-104-6" class="td11">
3.5.0
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-105-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-105-1" class="td11">
haven
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-105-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-105-3" class="td11">
1.0.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-105-4" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-105-5&quot;  </code></pre>
class=“td11”&gt;
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-105-6"  
class="td11">
<pre><code>      &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-106-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-106-1" class="td11">
heatmap.2x
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-106-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-79" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-79-1g">
<col 
id="TBL-79-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-79-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-79-1-1" class="td00">
GitHub
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-79-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-79-2-1" class="td00">
TomKellyGenetics
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-106-3" class="td11">
0.0.0.9000
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-106-4" class="td11">
0.0.0.9000
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-106-5" class="td11">
0.0.0.9000
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-106-6" class="td11">
0.0.0.9000
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-107-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-107-1" class="td11">
hgu133plus2.db
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-107-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-107-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-107-4&quot;  </code></pre>
class=“td11”&gt; 3.2.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-107-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-107-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-108-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-108-1" class="td11">
highr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-108-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-108-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-108-4&quot;  </code></pre>
class=“td11”&gt; 0.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-108-5" class="td11">
0.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-108-6" class="td11">
0.6
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-109-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-109-1" class="td11">
Hmisc
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-109-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-109-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-109-4&quot;  </code></pre>
class=“td11”&gt; 4.0-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-109-5" class="td11">
4.0-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-109-6" class="td11">
3.17-4
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-110-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-110-1" class="td11">
hms
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-110-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-110-3" class="td11">
0.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-110-4" class="td11">
0.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-110-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-110-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-111-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-111-1" class="td11">
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-112-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-112-1" class="td11">
htmlTable
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-112-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-112-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-112-4&quot;  </code></pre>
class=“td11”&gt; 1.8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-112-5" class="td11">
1.9
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-112-6"  
class="td11">
<pre><code>      &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-113-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-113-1" class="td11">
htmltools
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-113-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-113-3" class="td11">
0.3.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-113-4" class="td11">
0.3.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-113-5" class="td11">
0.3.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-113-6" class="td11">
0.3.5
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-114-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-114-1" class="td11">
htmlwidgets
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-114-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-114-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-114-4&quot;  </code></pre>
class=“td11”&gt; 0.8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-114-5" class="td11">
0.8
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-114-6"  
class="td11">
<pre><code>      &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-115-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-115-1" class="td11">
httpuv
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-115-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-115-3" class="td11">
1.3.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-115-4" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-115-5&quot;  </code></pre>
class=“td11”&gt; 1.3.3
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-115-6"  
class="td11">
<pre><code>      &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-116-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-116-1" class="td11">
httr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-116-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-116-3" class="td11">
1.2.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-116-4" class="td11">
1.2.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-116-5" class="td11">
1.2.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-116-6" class="td11">
1.1.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-117-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-117-1" class="td11">
huge
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-117-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-117-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-117-4&quot;  </code></pre>
class=“td11”&gt; 1.2.7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-117-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-117-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-118-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-118-1" class="td11">
hunspell
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-118-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-118-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-118-4&quot;  </code></pre>
class=“td11”&gt; 2.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-118-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-118-6&quot;  </code></pre>
class=“td11”&gt; 2
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-119-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-119-1" class="td11">
hypergraph
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-119-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-119-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-119-4&quot;  </code></pre>
class=“td11”&gt; 1.46.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-119-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-119-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-120-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-120-1" class="td11">
igraph
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-120-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-120-3" class="td11">
1.0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-120-4" class="td11">
1.0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-120-5" class="td11">
1.0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-120-6" class="td11">
1.0.1
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-121-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-121-1" class="td11">
igraph.extensions
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-121-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-80" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-80-1g">
<col 
id="TBL-80-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-80-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-80-1-1" class="td00">
GitHub
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-80-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-80-2-1" class="td00">
TomKellyGenetics
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-121-3" class="td11">
0.1.0.9001
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-121-4" class="td11">
0.1.0.9001
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-121-5" class="td11">
0.1.0.9001
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-121-6" class="td11">
0.1.0.9001
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-122-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-122-1" class="td11">
influenceR
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-122-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-122-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-122-4&quot;  </code></pre>
class=“td11”&gt; 0.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-122-5" class="td11">
0.1.0
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-122-6"  
class="td11">
<pre><code>      &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-123-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-123-1" class="td11">
info.centrality
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-123-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-81" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-81-1g">
<col 
id="TBL-81-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-81-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-81-1-1" class="td00">
GitHub
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-81-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-81-2-1" class="td00">
TomKellyGenetics
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-123-3" class="td11">
0.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-123-4" class="td11">
0.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-123-5" class="td11">
0.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-123-6" class="td11">
0.1.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-124-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-124-1" class="td11">
IRanges
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-124-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-124-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-124-4&quot;  </code></pre>
class=“td11”&gt; 2.8.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-124-5" class="td11">
2.8.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-124-6" class="td11">
2.6.1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-125-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-125-1" class="td11">
irlba
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-125-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-125-3" class="td11">
2.1.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-125-4" class="td11">
2.1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-125-5" class="td11">
2.1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-125-6" class="td11">
2.0.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-126-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-126-1" class="td11">
iterators
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-126-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-126-3" class="td11">
1.0.8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-126-4" class="td11">
1.0.8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-126-5" class="td11">
1.0.8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-126-6" class="td11">
1.0.8
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-127-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-127-1" class="td11">
jpeg
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-127-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-127-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-127-4&quot;  </code></pre>
class=“td11”&gt; 0.1-8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-127-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-127-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-128-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-128-1" class="td11">
jsonlite
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-128-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-128-3" class="td11">
1.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-128-4" class="td11">
1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-128-5" class="td11">
1.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-128-6" class="td11">
0.9.20
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-129-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-129-1" class="td11">
KEGG.db
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-129-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-129-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-129-4&quot;  </code></pre>
class=“td11”&gt; 3.2.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-129-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-129-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-130-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-130-1" class="td11">
kernlab
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-130-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-130-3" class="td11">
0.9-25
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-130-4" class="td11">
0.9-25
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-130-5" class="td11">
0.9-25
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-130-6"  
class="td11">
<pre><code>      &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-131-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-131-1" class="td11">
KernSmooth
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-131-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-131-3" class="td11">
2.23-15
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-131-4" class="td11">
2.23-15
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-131-5" class="td11">
2.23-15
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-131-6" class="td11">
2.23-15
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-132-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-132-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-133-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-133-1" class="td11">
knitr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-133-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-133-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-133-4" class="td11">
1.15.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-133-5" class="td11">
1.15.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-133-6" class="td11">
1.14
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-134-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-134-1" class="td11">
labeling
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-134-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-134-3" class="td11">
0.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-134-4" class="td11">
0.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-134-5" class="td11">
0.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-134-6" class="td11">
0.3
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-135-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-135-1" class="td11">
lambda.r
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-135-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-135-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-135-4" class="td11">
1.1.9
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-135-5" class="td11">
1.1.9
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-135-6" class="td11">
1.1.7
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-136-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-136-1" class="td11">
lattice
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-136-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-136-3" class="td11">
0.20-34
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-136-4" class="td11">
0.20-34
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-136-5" class="td11">
0.20-34
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-136-6" class="td11">
0.20-33
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-137-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-137-1" class="td11">
latticeExtra
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-137-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-137-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-137-4" class="td11">
0.6-28
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-137-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-137-6" class="td11">
0.6-28
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-138-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-138-1" class="td11">
lava
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-138-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-138-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-138-4" class="td11">
1.4.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-138-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-138-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-139-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-139-1" class="td11">
lavaan
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-139-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-139-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-139-4" class="td11">
0.5-22
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-139-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-139-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-140-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-140-1" class="td11">
lazyeval
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-140-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-140-3" class="td11">
0.2.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-140-4" class="td11">
0.2.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-140-5" class="td11">
0.2.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-140-6" class="td11">
0.2.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-141-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-141-1" class="td11">
les
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-141-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-141-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-141-4" class="td11">
1.24.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-141-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-141-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-142-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-142-1" class="td11">
lgtdl
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-142-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-142-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-142-4" class="td11">
1.1.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-142-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-142-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-143-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-143-1" class="td11">
limma
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-143-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-143-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-143-4" class="td11">
3.30.7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-143-5" class="td11">
3.30.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-143-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-144-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-144-1" class="td11">
lme4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-144-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-144-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-144-4" class="td11">
1.1-12
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-144-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-144-6" class="td11">
1.1-12
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-145-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-145-1" class="td11">
lubridate
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-145-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-145-3" class="td11">
1.6.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-145-4" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-145-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-145-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-146-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-146-1" class="td11">
magrittr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-146-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-146-3" class="td11">
1.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-146-4" class="td11">
1.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-146-5" class="td11">
1.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-146-6" class="td11">
1.5
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-147-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-147-1" class="td11">
maps
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-147-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-147-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-147-4" class="td11">
3.1.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-147-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-147-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-148-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-148-1" class="td11">
markdown
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-148-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-148-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-148-4" class="td11">
0.7.7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-148-5" class="td11">
0.7.7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-148-6" class="td11">
0.7.7
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-149-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-149-1" class="td11">
MASS
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-149-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-149-3" class="td11">
7.3-45
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-149-4" class="td11">
7.3-45
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-149-5" class="td11">
7.3-45
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-149-6" class="td11">
7.3-45
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-150-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-150-1" class="td11">
Matrix
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-150-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-150-3" class="td11">
1.2-7.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-150-4" class="td11">
1.2-7.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-150-5" class="td11">
1.2-8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-150-6" class="td11">
1.2-6
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-151-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-151-1" class="td11">
matrixcalc
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-151-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-151-3" class="td11">
1.0-3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-151-4" class="td11">
1.0-3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-151-5" class="td11">
1.0-3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-151-6" class="td11">
1.0-3
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-152-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-152-1" class="td11">
mclust
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-152-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-152-3" class="td11">
5.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-152-4" class="td11">
5.2.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-152-5" class="td11">
5.2.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-152-6" class="td11">
5.2
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-153-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-153-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-154-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-154-1" class="td11">
memoise
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-154-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-154-3" class="td11">
1.0.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-154-4" class="td11">
1.0.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-154-5" class="td11">
1.0.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-154-6" class="td11">
1.0.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-155-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-155-1" class="td11">
methods
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-155-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-155-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-155-4" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-155-5" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-155-6" class="td11">
3.3.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-156-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-156-1" class="td11">
mgcv
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-156-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-156-3" class="td11">
1.8-16
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-156-4" class="td11">
1.8-16
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-156-5" class="td11">
1.8-17
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-156-6" class="td11">
1.8-12
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-157-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-157-1" class="td11">
mi
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-157-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-157-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-157-4" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-157-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-157-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-158-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-158-1" class="td11">
mime
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-158-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-158-3" class="td11">
0.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-158-4" class="td11">
0.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-158-5" class="td11">
0.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-158-6" class="td11">
0.4
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-159-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-159-1" class="td11">
minqa
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-159-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-159-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-159-4" class="td11">
1.2.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-159-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-159-6" class="td11">
1.2.4
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-160-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-160-1" class="td11">
mnormt
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-160-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-160-3" class="td11">
1.5-5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-160-4" class="td11">
1.5-5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-160-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-160-6" class="td11">
1.5-4
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-161-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-161-1" class="td11">
modelr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-161-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-161-3" class="td11">
0.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-161-4" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-161-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-161-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-162-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-162-1" class="td11">
modeltools
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-162-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-162-3" class="td11">
0.2-21
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-162-4" class="td11">
0.2-21
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-162-5" class="td11">
0.2-21
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-162-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-163-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-163-1" class="td11">
multtest
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-163-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-163-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-163-4" class="td11">
2.30.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-163-5" class="td11">
2.30.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-163-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-164-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-164-1" class="td11">
munsell
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-164-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-164-3" class="td11">
0.4.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-164-4" class="td11">
0.4.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-164-5" class="td11">
0.4.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-164-6" class="td11">
0.4.3
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-165-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-165-1" class="td11">
mvtnorm
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-165-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-165-3" class="td11">
1.0-5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-165-4" class="td11">
1.0-5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-165-5" class="td11">
1.0-6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-165-6" class="td11">
1.0-5
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-166-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-166-1" class="td11">
network
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-166-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-166-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-166-4" class="td11">
1.13.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-166-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-166-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-167-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-167-1" class="td11">
nlme
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-167-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-167-3" class="td11">
3.1-128
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-167-4" class="td11">
3.1-128
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-167-5" class="td11">
3.1-131
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-167-6" class="td11">
3.1-128
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-168-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-168-1" class="td11">
nloptr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-168-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-168-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-168-4" class="td11">
1.0.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-168-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-168-6" class="td11">
1.0.4
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-169-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-169-1" class="td11">
NMF
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-169-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-169-3" class="td11">
0.20.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-169-4" class="td11">
0.20.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-169-5" class="td11">
0.20.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-169-6" class="td11">
0.20.6
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-170-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-170-1" class="td11">
nnet
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-170-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-170-3" class="td11">
7.3-12
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-170-4" class="td11">
7.3-12
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-170-5" class="td11">
7.3-12
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-170-6" class="td11">
7.3-12
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-171-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-171-1" class="td11">
numDeriv
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-171-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-171-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-171-4" class="td11">
2016.8-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-171-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-171-6" class="td11">
2014.2-1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-172-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-172-1" class="td11">
openssl
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-172-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-172-3" class="td11">
0.9.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-172-4" class="td11">
0.9.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-172-5" class="td11">
0.9.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-172-6" class="td11">
0.9.4
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-173-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-173-1" class="td11">
org.Hs.eg.db
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-173-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-173-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-173-4" class="td11">
3.1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-173-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-173-6" class="td11">
3.3.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-174-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-174-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-175-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-175-1" class="td11">
org.Sc.sgd.db
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-175-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-175-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-175-4&quot;  </code></pre>
class=“td11”&gt; 3.4.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-175-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-175-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-176-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-176-1" class="td11">
parallel
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-176-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-176-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-176-4" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-176-5" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-176-6" class="td11">
3.3.0
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-177-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-177-1"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-82" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-82-1g">
<col 
id="TBL-82-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-82-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-82-1-1" class="td00">
pathway.structure
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-82-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-82-2-1" class="td00">
.permutation
</td>
</tr>
</table>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-177-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-83" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-83-1g">
<col 
id="TBL-83-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-83-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-83-1-1" class="td00">
GitHub
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-83-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-83-2-1" class="td00">
TomKellyGenetics
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-177-3" class="td11">
0.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-177-4" class="td11">
0.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-177-5" class="td11">
0.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-177-6" class="td11">
0.1.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-178-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-178-1" class="td11">
pbivnorm
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-178-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-178-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-178-4&quot;  </code></pre>
class=“td11”&gt; 0.6.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-178-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-178-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-179-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-179-1" class="td11">
PGSEA
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-179-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-179-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-179-4&quot;  </code></pre>
class=“td11”&gt; 1.48.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-179-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-179-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-180-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-180-1" class="td11">
pkgmaker
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-180-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-180-3" class="td11">
0.22
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-180-4" class="td11">
0.22
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-180-5" class="td11">
0.22
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-180-6" class="td11">
0.22
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-181-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-181-1" class="td11">
PKI
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-181-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-181-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-181-4&quot;  </code></pre>
class=“td11”&gt; 0.1-3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-181-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-181-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-182-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-182-1" class="td11">
plogr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-182-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-182-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-182-4&quot;  </code></pre>
class=“td11”&gt; 0.1-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-182-5" class="td11">
0.1-1
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-182-6"  
class="td11">
<pre><code>      &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-183-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-183-1" class="td11">
plot.igraph
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-183-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-84" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-84-1g">
<col 
id="TBL-84-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-84-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-84-1-1" class="td00">
GitHub
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-84-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-84-2-1" class="td00">
TomKellyGenetics
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-183-3" class="td11">
0.0.0.9001
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-183-4" class="td11">
0.0.0.9001
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-183-5" class="td11">
0.0.0.9001
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-183-6" class="td11">
0.0.0.9001
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-184-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-184-1" class="td11">
plotrix
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-184-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-184-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-184-4&quot;  </code></pre>
class=“td11”&gt; 3.6-4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-184-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-184-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-185-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-185-1" class="td11">
plyr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-185-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-185-3" class="td11">
1.8.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-185-4" class="td11">
1.8.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-185-5" class="td11">
1.8.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-185-6" class="td11">
1.8.3
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-186-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-186-1" class="td11">
png
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-186-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-186-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-186-4&quot;  </code></pre>
class=“td11”&gt; 0.1-7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-186-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-186-6&quot;  </code></pre>
class=“td11”&gt; 0.1-7
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-187-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-187-1" class="td11">
prabclus
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-187-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-187-3" class="td11">
2.2-6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-187-4" class="td11">
2.2-6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-187-5" class="td11">
2.2-6
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-187-6"  
class="td11">
<pre><code>      &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-188-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-188-1" class="td11">
praise
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-188-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-188-3" class="td11">
1.0.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-188-4" class="td11">
1.0.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-188-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-188-6&quot;  </code></pre>
class=“td11”&gt; 1.0.0
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-189-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-189-1" class="td11">
pROC
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-189-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-189-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-189-4&quot;  </code></pre>
class=“td11”&gt; 1.8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-189-5" class="td11">
1.9.1
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-189-6"  
class="td11">
<pre><code>      &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-190-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-190-1" class="td11">
prodlim
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-190-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-190-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-190-4&quot;  </code></pre>
class=“td11”&gt; 1.5.7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-190-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-190-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-191-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-191-1" class="td11">
prof.tree
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-191-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-191-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-191-4&quot;  </code></pre>
class=“td11”&gt; 0.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-191-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-191-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-192-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-192-1" class="td11">
proftools
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-192-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-192-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-192-4&quot;  </code></pre>
class=“td11”&gt; 0.99-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-192-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-192-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-193-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-193-1" class="td11">
progress
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-193-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-193-3" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-193-4&quot;  </code></pre>
class=“td11”&gt;
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-193-5" class="td11">
1.1.2
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-193-6"  
class="td11">
<pre><code>      &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-194-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-194-1" class="td11">
psych
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-194-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-194-3" class="td11">
1.6.12
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-194-4" class="td11">
1.6.12
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-194-5" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-194-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-195-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-195-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-196-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-196-1" class="td11">
purrr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-196-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-196-3" class="td11">
0.2.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-196-4" class="td11">
0.2.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-196-5" class="td11">
0.2.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-196-6" class="td11">
0.2.2
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-197-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-197-1" class="td11">
qgraph
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-197-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-197-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-197-4" class="td11">
1.4.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-197-5" class="td11">
<pre><code>       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-197-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-198-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-198-1" class="td11">
quadprog
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-198-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-198-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-198-4" class="td11">
1.5-5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-198-5" class="td11">
1.5-5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-198-6" class="td11">
1.5-5
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-199-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-199-1" class="td11">
R.methodsS3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-199-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-199-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-199-4" class="td11">
1.7.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-199-5" class="td11">
<pre><code>       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-199-6&quot;  </code></pre>
class=“td11”&gt; 1.7.1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-200-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-200-1" class="td11">
R.oo
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-200-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-200-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-200-4" class="td11">
1.21.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-200-5" class="td11">
<pre><code>       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-200-6&quot;  </code></pre>
class=“td11”&gt; 1.20.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-201-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-201-1" class="td11">
R.utils
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-201-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-201-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-201-4" class="td11">
2.5.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-201-5" class="td11">
<pre><code>       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-201-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-202-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-202-1" class="td11">
R6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-202-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-202-3" class="td11">
2.1.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-202-4" class="td11">
2.2.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-202-5" class="td11">
2.2.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-202-6" class="td11">
2.1.3
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-203-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-203-1" class="td11">
RBGL
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-203-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-203-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-203-4" class="td11">
1.50.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-203-5" class="td11">
<pre><code>       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-203-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-204-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-204-1" class="td11">
RColorBrewer
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-204-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-204-3" class="td11">
1.1-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-204-4" class="td11">
1.1-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-204-5" class="td11">
1.1-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-204-6" class="td11">
1.1-2
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-205-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-205-1" class="td11">
Rcpp
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-205-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-205-3" class="td11">
0.12.7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-205-4" class="td11">
0.12.9
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-205-5" class="td11">
0.12.9
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-205-6" class="td11">
0.12.7
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-206-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-206-1" class="td11">
RcppArmadillo
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-206-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-206-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-206-4" class="td11">
<pre><code>     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-206-5&quot;  </code></pre>
class=“td11”&gt; 0.7.700.0.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-206-6" class="td11">
0.6.700.6.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-207-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-207-1" class="td11">
RcppEigen
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-207-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-207-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-207-4" class="td11">
0.3.2.9.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-207-5" class="td11">
<pre><code>       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-207-6&quot;  </code></pre>
class=“td11”&gt; 0.3.2.8.1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-208-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-208-1" class="td11">
RCurl
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-208-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-208-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-208-4" class="td11">
1.95-4.8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-208-5" class="td11">
1.95-4.8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-208-6" class="td11">
1.95-4.8
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-209-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-209-1" class="td11">
reactome.db
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-209-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-209-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-209-4" class="td11">
1.52.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-209-5" class="td11">
1.52.1
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-209-6"  
class="td11">
<pre><code>       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-210-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-210-1" class="td11">
reactometree
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-210-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-85" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-85-1g">
<col 
id="TBL-85-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-85-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-85-1-1" class="td00">
GitHub
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-85-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-85-2-1" class="td00">
TomKellyGenetics
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-210-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-210-4" class="td11">
0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-210-5" class="td11">
<pre><code>       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-210-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-211-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-211-1" class="td11">
readr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-211-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-211-3" class="td11">
1.0.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-211-4" class="td11">
1.0.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-211-5" class="td11">
<pre><code>       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-211-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-212-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-212-1" class="td11">
readxl
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-212-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-212-3" class="td11">
0.1.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-212-4" class="td11">
<pre><code>     &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-212-5&quot;  </code></pre>
class=“td11”&gt;
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-212-6"  
class="td11">
<pre><code>       &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-213-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-213-1" class="td11">
registry
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-213-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-213-3" class="td11">
0.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-213-4" class="td11">
0.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-213-5" class="td11">
0.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-213-6" class="td11">
0.3
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-214-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-214-1" class="td11">
reshape2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-214-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-214-3" class="td11">
1.4.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-214-4" class="td11">
1.4.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-214-5" class="td11">
1.4.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-214-6" class="td11">
1.4.1
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-215-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-215-1" class="td11">
rgexf
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-215-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-215-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-215-4" class="td11">
0.15.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-215-5" class="td11">
0.15.3
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-215-6"  
class="td11">
<pre><code>       &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-216-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-216-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-217-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-217-1" class="td11">
rgl
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-217-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-217-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-217-4" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-217-5" class="td11">
0.97.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-217-6" class="td11">
0.95.1441
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-218-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-218-1" class="td11">
Rgraphviz
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-218-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-218-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-218-4" class="td11">
2.18.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-218-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-218-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-219-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-219-1" class="td11">
rjson
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-219-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-219-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-219-4" class="td11">
0.2.15
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-219-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-219-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-220-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-220-1" class="td11">
RJSONIO
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-220-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-220-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-220-4" class="td11">
1.3-0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-220-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-220-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-221-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-221-1" class="td11">
rmarkdown
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-221-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-221-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-221-4" class="td11">
1.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-221-5" class="td11">
1.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-221-6" class="td11">
1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-222-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-222-1" class="td11">
Rmpi
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-222-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-222-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-222-4" class="td11">
0.6-6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-222-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-222-6" class="td11">
0.6-5
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-223-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-223-1" class="td11">
rngtools
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-223-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-223-3" class="td11">
1.2.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-223-4" class="td11">
1.2.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-223-5" class="td11">
1.2.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-223-6" class="td11">
1.2.4
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-224-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-224-1" class="td11">
robustbase
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-224-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-224-3" class="td11">
0.92-7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-224-4" class="td11">
0.92-7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-224-5" class="td11">
0.92-7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-224-6" class="td11">
0.92-5
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-225-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-225-1" class="td11">
ROCR
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-225-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-225-3" class="td11">
1.0-7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-225-4" class="td11">
1.0-7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-225-5" class="td11">
1.0-7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-225-6" class="td11">
1.0-7
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-226-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-226-1" class="td11">
Rook
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-226-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-226-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-226-4" class="td11">
1.1-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-226-5" class="td11">
1.1-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-226-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-227-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-227-1" class="td11">
roxygen2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-227-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-227-3" class="td11">
6.0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-227-4" class="td11">
5.0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-227-5" class="td11">
6.0.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-227-6" class="td11">
5.0.1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-228-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-228-1" class="td11">
rpart
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-228-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-228-3" class="td11">
4.1-10
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-228-4" class="td11">
4.1-10
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-228-5" class="td11">
4.1-10
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-228-6" class="td11">
4.1-10
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-229-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-229-1" class="td11">
rprojroot
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-229-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-229-3" class="td11">
1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-229-4" class="td11">
1.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-229-5" class="td11">
1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-229-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-230-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-230-1" class="td11">
Rsamtools
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-230-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-230-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-230-4" class="td11">
1.26.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-230-5" class="td11">
1.26.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-230-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-231-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-231-1" class="td11">
rsconnect
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-231-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-231-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-231-4" class="td11">
0.7
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-231-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-231-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-232-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-232-1" class="td11">
RSQLite
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-232-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-232-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-232-4" class="td11">
1.1-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-232-5" class="td11">
1.1-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-232-6" class="td11">
1.0.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-233-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-233-1" class="td11">
rstudioapi
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-233-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-233-3" class="td11">
0.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-233-4" class="td11">
0.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-233-5" class="td11">
0.6
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-233-6" class="td11">
0.6
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-234-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-234-1" class="td11">
rvest
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-234-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-234-3" class="td11">
0.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-234-4" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-234-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-234-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-235-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-235-1" class="td11">
S4Vectors
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-235-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-235-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-235-4" class="td11">
0.12.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-235-5" class="td11">
0.12.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-235-6" class="td11">
0.10.3
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-236-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-236-1" class="td11">
safe
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-236-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-236-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-236-4" class="td11">
3.14.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-236-5" class="td11">
3.10.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-236-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-237-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-237-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-238-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-238-1" class="td11">
scales
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-238-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-238-3" class="td11">
0.4.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-238-4" class="td11">
0.4.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-238-5" class="td11">
0.4.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-238-6" class="td11">
0.4.0
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-239-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-239-1" class="td11">
selectr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-239-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-239-3" class="td11">
0.3-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-239-4" class="td11">
<pre><code>   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-239-5&quot;  </code></pre>
class=“td11”&gt;
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-239-6"  
class="td11">
<pre><code>   &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-240-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-240-1" class="td11">
sem
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-240-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-240-3" class="td11">
<pre><code>   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-240-4&quot;  </code></pre>
class=“td11”&gt; 3.1-8
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-240-5" class="td11">
<pre><code>   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-240-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-241-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-241-1" class="td11">
shiny
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-241-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-241-3" class="td11">
0.14
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-241-4" class="td11">
<pre><code>   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-241-5&quot;  </code></pre>
class=“td11”&gt; 1.0.0
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-241-6"  
class="td11">
<pre><code>   &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-242-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-242-1" class="td11">
slipt
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-242-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-86" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-86-1g">
<col 
id="TBL-86-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-86-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-86-1-1" class="td00">
GitHub
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-86-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-86-2-1" class="td00">
TomKellyGenetics
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-242-3" class="td11">
0.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-242-4" class="td11">
0.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-242-5" class="td11">
0.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-242-6" class="td11">
0.1.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-243-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-243-1" class="td11">
sm
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-243-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-243-3" class="td11">
2.2-5.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-243-4" class="td11">
2.2-5.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-243-5" class="td11">
<pre><code>   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-243-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-244-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-244-1" class="td11">
sna
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-244-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-244-3" class="td11">
<pre><code>   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-244-4&quot;  </code></pre>
class=“td11”&gt; 2.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-244-5" class="td11">
<pre><code>   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-244-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-245-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-245-1" class="td11">
snow
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-245-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-245-3" class="td11">
0.4-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-245-4" class="td11">
0.4-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-245-5" class="td11">
0.4-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-245-6" class="td11">
0.3-13
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-246-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-246-1" class="td11">
sourcetools
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-246-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-246-3" class="td11">
0.1.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-246-4" class="td11">
<pre><code>   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-246-5&quot;  </code></pre>
class=“td11”&gt; 0.1.5
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-246-6"  
class="td11">
<pre><code>   &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-247-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-247-1" class="td11">
SparseM
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-247-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-247-3" class="td11">
<pre><code>   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-247-4&quot;  </code></pre>
class=“td11”&gt; 1.74
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-247-5" class="td11">
<pre><code>   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-247-6&quot;  </code></pre>
class=“td11”&gt; 1.7
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-248-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-248-1" class="td11">
spatial
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-248-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-248-3" class="td11">
7.3-11
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-248-4" class="td11">
7.3-11
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-248-5" class="td11">
7.3-11
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-248-6" class="td11">
7.3-11
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-249-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-249-1" class="td11">
splines
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-249-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-249-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-249-4" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-249-5" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-249-6" class="td11">
3.3.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-250-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-250-1" class="td11">
statnet.common
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-250-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-250-3" class="td11">
<pre><code>   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-250-4&quot;  </code></pre>
class=“td11”&gt; 3.3.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-250-5" class="td11">
<pre><code>   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-250-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-251-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-251-1" class="td11">
stats
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-251-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-251-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-251-4" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-251-5" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-251-6" class="td11">
3.3.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-252-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-252-1" class="td11">
stats4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-252-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-252-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-252-4" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-252-5" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-252-6" class="td11">
3.3.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-253-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-253-1" class="td11">
stringi
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-253-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-253-3" class="td11">
1.1.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-253-4" class="td11">
1.1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-253-5" class="td11">
1.1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-253-6" class="td11">
1.0-1
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-254-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-254-1" class="td11">
stringr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-254-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-254-3" class="td11">
1.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-254-4" class="td11">
1.1.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-254-5" class="td11">
1.2.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-254-6" class="td11">
1.0.0
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-255-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-255-1"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-87" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-87-1g">
<col 
id="TBL-87-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-87-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-87-1-1" class="td00">
Summarized
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-87-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-87-2-1" class="td00">
Experiment
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-255-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-255-3" class="td11">
<pre><code>   &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-255-4&quot;  </code></pre>
class=“td11”&gt; 1.4.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-255-5" class="td11">
1.4.0
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-255-6"  
class="td11">
<pre><code>   &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-256-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-256-1" class="td11">
survival
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-256-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-256-3" class="td11">
2.39-4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-256-4" class="td11">
2.40-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-256-5" class="td11">
2.40-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-256-6" class="td11">
2.39-4
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-257-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-257-1" class="td11">
tcltk
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-257-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-257-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-257-4" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-257-5" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-257-6" class="td11">
3.3.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-258-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-258-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-259-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-259-1" class="td11">
testthat
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-259-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-259-3" class="td11">
1.0.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-259-4" class="td11">
1.0.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-259-5" class="td11">
<pre><code>    &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-259-6&quot;  </code></pre>
class=“td11”&gt; 1.0.2
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-260-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-260-1" class="td11">
tibble
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-260-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-260-3" class="td11">
1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-260-4" class="td11">
1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-260-5" class="td11">
1.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-260-6" class="td11">
1.2
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-261-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-261-1" class="td11">
tidyr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-261-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-261-3" class="td11">
0.6.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-261-4" class="td11">
0.6.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-261-5" class="td11">
0.6.1
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-261-6"  
class="td11">
<pre><code>    &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-262-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-262-1" class="td11">
tidyverse
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-262-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-88" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-88-1g">
<col 
id="TBL-88-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-88-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-88-1-1" class="td00">
GitHub
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-88-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-88-2-1" class="td00">
hadley
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-262-3" class="td11">
1.1.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-262-4" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-262-5&quot;  </code></pre>
class=“td11”&gt;
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-262-6"  
class="td11">
<pre><code>    &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-263-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-263-1" class="td11">
timeline
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-263-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-263-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-263-4" class="td11">
0.9
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-263-5" class="td11">
<pre><code>    &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-263-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-264-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-264-1" class="td11">
tools
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-264-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-264-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-264-4" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-264-5" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-264-6" class="td11">
3.3.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-265-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-265-1" class="td11">
tpr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-265-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-265-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-265-4" class="td11">
0.3-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-265-5" class="td11">
<pre><code>    &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-265-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-266-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-266-1" class="td11">
trimcluster
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-266-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-266-3" class="td11">
0.1-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-266-4" class="td11">
0.1-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-266-5" class="td11">
0.1-2
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-266-6"  
class="td11">
<pre><code>    &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-267-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-267-1" class="td11">
Unicode
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-267-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-267-3" class="td11">
9.0.0-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-267-4" class="td11">
9.0.0-1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-267-5" class="td11">
9.0.0-1
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-267-6"  
class="td11">
<pre><code>    &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-268-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-268-1" class="td11">
utils
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-268-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-268-3" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-268-4" class="td11">
3.3.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-268-5" class="td11">
3.3.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-268-6" class="td11">
3.3.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-269-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-269-1" class="td11">
vioplot
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-269-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-269-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-269-4" class="td11">
0.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-269-5" class="td11">
<pre><code>    &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-269-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-270-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-270-1" class="td11">
vioplotx
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-270-2"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-89" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-89-1g">
<col 
id="TBL-89-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-89-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-89-1-1" class="td00">
GitHub
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-89-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-89-2-1" class="td00">
TomKellyGenetics
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-270-3" class="td11">
0.0.0.9000
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-270-4" class="td11">
0.0.0.9000
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-270-5" class="td11">
<pre><code>    &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-270-6&quot;  </code></pre>
class=“td11”&gt;
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-271-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-271-1" class="td11">
viridis
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-271-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-271-3" class="td11">
0.3.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-271-4" class="td11">
0.3.4
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-271-5" class="td11">
0.3.4
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-271-6"  
class="td11">
<pre><code>    &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-272-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-272-1" class="td11">
visNetwork
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-272-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-272-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-272-4" class="td11">
1.0.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-272-5" class="td11">
1.0.3
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-272-6"  
class="td11">
<pre><code>    &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-273-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-273-1" class="td11">
whisker
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-273-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-273-3" class="td11">
0.3-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-273-4" class="td11">
0.3-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-273-5" class="td11">
0.3-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-273-6" class="td11">
0.3-2
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-274-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-274-1" class="td11">
withr
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-274-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-274-3" class="td11">
1.0.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-274-4" class="td11">
1.0.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-274-5" class="td11">
1.0.2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-274-6" class="td11">
1.0.2
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-275-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-275-1" class="td11">
XML
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-275-2" class="td11">
base
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-275-3" class="td11">
3.98-1.3
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-275-4" class="td11">
3.98-1.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-275-5" class="td11">
3.98-1.5
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-275-6" class="td11">
3.98-1.4
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-276-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-276-1" class="td11">
xml2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-276-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-276-3" class="td11">
1.1.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-276-4" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:left;&quot; id=&quot;TBL-77-276-5&quot;  </code></pre>
class=“td11”&gt; 1.1.1
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-276-6" class="td11">
1.0.0
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-277-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-277-1" class="td11">
xtable
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-277-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-277-3" class="td11">
1.8-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-277-4" class="td11">
1.8-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-277-5" class="td11">
1.8-2
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-277-6" class="td11">
1.8-2
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-77-278-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-278-1" class="td11">
XVector
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-278-2" class="td11">
Bioconductor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-278-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-278-4" class="td11">
0.14.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-278-5" class="td11">
0.14.0
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-77-278-6"  
class="td11">
<pre><code>    &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-279-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-279-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-280-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-280-1" class="td11">
yaml
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-280-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-280-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-280-4" class="td11">
2.1.14
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-280-5" class="td11">
2.1.14
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-280-6" class="td11">
2.1.13
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-281-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-281-1" class="td11">
zlibbioc
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-281-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-281-3" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-281-4" class="td11">
1.20.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-281-5" class="td11">
1.20.0
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-281-6" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-282-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-282-1" class="td11">
zoo
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-282-2" class="td11">
CRAN
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-282-3" class="td11">
1.7-13
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-282-4" class="td11">
1.7-14
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-282-5" class="td11">
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-77-282-6" class="td11">
1.7-13
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-283-">
<td colspan="6" style="white-space:nowrap; text-align:left;" id="TBL-77-283-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-284-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-284-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-285-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-285-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-286-">
<td colspan="6" style="white-space:nowrap; text-align:left;" id="TBL-77-286-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-287-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-287-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-288-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-288-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-289-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-289-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-290-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-290-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-291-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-291-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-292-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-292-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-77-293-">
<td style="white-space:nowrap; text-align:left;" id="TBL-77-293-1" class="td11">
</td>
</tr>
</table>
</div>
<a 
 id="x1-202003r288"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix C</span><br /><a 
 id="x1-203000C"></a>Secondary Screen Data
</h2>
<!--l. 19-->
<p class="noindent">
<p>A series of experimental genome-wide siRNA screens have been performed on synthetic lethal partners of <span class="cmti-12">CDH1 </span>(<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>). The strongest candidates from a primary screen were subject to a further secondary screen for validation by independent replication with 4 gene knockdowns with different targeting siRNA. As shown in Table <a 
href="#x1-203001r1">C.1<!--tex4ht:ref: tab:secondary_screen --></a>, there is significant (<span class="cmmi-12">p </span>= 7<span class="cmmi-12">.</span>49 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">3</span></sup> by Fisher’s exact test) association between SLIPT candidates and stronger validations of siRNA candidates. Since there were more SLIPT<span class="cmsy-10x-x-120">- </span>genes among those not validated and more SLIPT+ genes among those validated with several siRNAs, this supports the use of SLIPT as a synthetic lethal discovery procedure which may augment such screening experiments.</p>
</p>
<!--l. 22-->
<p class="indent">
<a 
 id="x1-203001r1"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table C.1: </span><span class="content">Comparing SLIPT genes against Secondary siRNA Screen in breast cancer</span></p>
</div>
<!--tex4ht:label?: x1-203001r1 -->
<div class="center" 
>
<!--l. 25-->
<p class="noindent">
</p>
<div class="tabular">
<table id="TBL-90" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-90-1g">
<col 
id="TBL-90-1" />
<col 
id="TBL-90-2" />
<col 
id="TBL-90-3" />
<col 
id="TBL-90-4" />
<col 
id="TBL-90-5" />
<col 
id="TBL-90-6" />
<col 
id="TBL-90-7" />
<col 
id="TBL-90-8" />
</colgroup>
<tr  
 style="vertical-align:baseline;" id="TBL-90-1-">
<td style="white-space:nowrap; text-align:right;" id="TBL-90-1-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:right;&quot; id=&quot;TBL-90-1-2&quot;  </code></pre>
class=“td11”&gt;
</td>
<td colspan="5" style="white-space:nowrap; text-align:center;" id="TBL-90-1-3" class="td11">
<pre><code>    &lt;div class=&quot;multicolumn&quot;  style=&quot;white-space:nowrap; text-align:center;&quot;&gt;&lt;span </code></pre>
class=“cmbx-12”&gt;Secondary Screen</span>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-90-1-8"  
class="td11">
<pre><code>    &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-90-2-">
<td style="white-space:nowrap; text-align:right;" id="TBL-90-2-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-90-3-">
<td style="white-space:nowrap; text-align:right;" id="TBL-90-3-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:right;&quot; id=&quot;TBL-90-3-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:right;">

</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-3-3" class="td11">
0/4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-3-4" class="td11">
1/4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-3-5" class="td11">
2/4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-3-6" class="td11">
3/4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-3-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>4/4</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-90-3-8" class="td11">
<strong><span class="cmbx-12">Total</span></strong>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-90-4-">
<td style="white-space:nowrap; text-align:right;" id="TBL-90-4-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-90-5-">
<td style="white-space:nowrap; text-align:right;" id="TBL-90-5-1" class="td11">
<pre><code>     &lt;div class=&quot;multicolumn&quot;  style=&quot;white-space:nowrap; text-align:right;&quot;&gt;&lt;/div&gt; &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:right;&quot; id=&quot;TBL-90-5-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p>Observed</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-5-3" class="td11">
70
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-5-4" class="td11">
46
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-5-5" class="td11">
31
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-5-6" class="td11">
8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-5-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>2</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-90-5-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">

</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-90-6-">
<td style="white-space:nowrap; text-align:right;" id="TBL-90-6-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<div class="multirow">
<!-- rows=-2 -->
<p><span class="cmbx-12">SLIPT</span>+</p>
</div>
</div>
</td>
<td style="white-space:nowrap; text-align:right;" id="TBL-90-6-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p>Expected</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-6-3" class="td11">
85
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-6-4" class="td11">
44
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-6-5" class="td11">
10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-6-6" class="td11">
4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-6-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>2</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-90-6-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<div class="multirow">
<!-- rows=-2 -->
<p>157</p>
</div>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-90-7-">
<td style="white-space:nowrap; text-align:right;" id="TBL-90-7-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-90-8-">
<td style="white-space:nowrap; text-align:right;" id="TBL-90-8-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:right;">

</div>
<pre><code>           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:right;&quot; id=&quot;TBL-90-8-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p>Observed</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-8-3" class="td11">
190
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-8-4" class="td11">
90
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-8-5" class="td11">
31
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-8-6" class="td11">
10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-8-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>4</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-90-8-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">

</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-90-9-">
<td style="white-space:nowrap; text-align:right;" id="TBL-90-9-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<div class="multirow">
<!-- rows=-2 -->
<p><span class="cmbx-12">SLIPT</span><span class="cmsy-10x-x-120">-</span></p>
</div>
</div>
</td>
<td style="white-space:nowrap; text-align:right;" id="TBL-90-9-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p>Expected</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-9-3" class="td11">
175
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-9-4" class="td11">
91
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-9-5" class="td11">
42
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-9-6" class="td11">
12
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-9-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>4</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-90-9-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<div class="multirow">
<!-- rows=-2 -->
<p>325</p>
</div>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-90-10-">
<td style="white-space:nowrap; text-align:right;" id="TBL-90-10-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-90-11-">
<td style="white-space:nowrap; text-align:right;" id="TBL-90-11-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:right;&quot; id=&quot;TBL-90-11-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p><span class="cmbx-12">Total</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-11-3" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>280</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-11-4" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>136</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-11-5" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>52</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-11-6" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>18</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-90-11-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>6</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-90-11-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>482</p>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-90-12-">
<td style="white-space:nowrap; text-align:right;" id="TBL-90-12-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-90-13-">
<td  style="white-space:nowrap; text-align:right;" id="TBL-90-13-1"  
class="td11">
<pre><code>      &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/div&gt;&lt;/div&gt;
                                                                                
                                                                                </code></pre>
</div>
<hr class="endfloat" />
<!--l. 46-->
<p class="indent">
<p>Similar analysis with mtSLIPT, comparing SLIPT against <span class="cmti-12">CDH1 </span>somatic mutation with siRNA validation results was not significant (<span class="cmmi-12">p </span>= 7<span class="cmmi-12">.</span>02 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">1</span></sup> by Fisher’s exact test). However, as shown in Table <a 
href="#x1-203002r2">C.2<!--tex4ht:ref: tab:secondary_screen_mtSL --></a>, the observed and expected values were in a direction consistent with that observed above for SLIPT against low <span class="cmti-12">CDH1 </span>expression. It is not unexpected that this result does not have comparable statistical support due to the lower sample size for mutation data.</p>
</p>
<!--l. 48-->
<p class="indent">
<a 
 id="x1-203002r2"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table C.2: </span><span class="content">Comparing mtSLIPT genes against Secondary siRNA Screen in breast cancer</span></p>
</div>
<!--tex4ht:label?: x1-203002r2 -->
<div class="center" 
>
<!--l. 51-->
<p class="noindent">
</p>
<div class="tabular">
<table id="TBL-91" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-91-1g">
<col 
id="TBL-91-1" />
<col 
id="TBL-91-2" />
<col 
id="TBL-91-3" />
<col 
id="TBL-91-4" />
<col 
id="TBL-91-5" />
<col 
id="TBL-91-6" />
<col 
id="TBL-91-7" />
<col 
id="TBL-91-8" />
</colgroup>
<tr  
 style="vertical-align:baseline;" id="TBL-91-1-">
<td style="white-space:nowrap; text-align:right;" id="TBL-91-1-1" class="td11">
<pre><code>        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:right;&quot; id=&quot;TBL-91-1-2&quot;  </code></pre>
class=“td11”&gt;
</td>
<td colspan="5" style="white-space:nowrap; text-align:center;" id="TBL-91-1-3" class="td11">
<pre><code>    &lt;div class=&quot;multicolumn&quot;  style=&quot;white-space:nowrap; text-align:center;&quot;&gt;&lt;span </code></pre>
class=“cmbx-12”&gt;Secondary Screen</span>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-91-1-8"  
class="td11">
<pre><code>    &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-91-2-">
<td style="white-space:nowrap; text-align:right;" id="TBL-91-2-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-91-3-">
<td style="white-space:nowrap; text-align:right;" id="TBL-91-3-1" class="td11">
<pre><code>         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:right;&quot; id=&quot;TBL-91-3-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:right;">

</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-3-3" class="td11">
0/4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-3-4" class="td11">
1/4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-3-5" class="td11">
2/4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-3-6" class="td11">
3/4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-3-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>4/4</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-91-3-8" class="td11">
<strong><span class="cmbx-12">Total</span></strong>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-91-4-">
<td style="white-space:nowrap; text-align:right;" id="TBL-91-4-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-91-5-">
<td style="white-space:nowrap; text-align:right;" id="TBL-91-5-1" class="td11">
<pre><code>        &lt;div class=&quot;multicolumn&quot;  style=&quot;white-space:nowrap; text-align:right;&quot;&gt;&lt;/div&gt; &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:right;&quot; id=&quot;TBL-91-5-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p>Observed</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-5-3" class="td11">
54
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-5-4" class="td11">
35
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-5-5" class="td11">
17
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-5-6" class="td11">
4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-5-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>6</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-91-5-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">

</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-91-6-">
<td style="white-space:nowrap; text-align:right;" id="TBL-91-6-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<div class="multirow">
<!-- rows=-2 -->
<p><span class="cmbx-12">mtSLIPT</span>+</p>
</div>
</div>
</td>
<td style="white-space:nowrap; text-align:right;" id="TBL-91-6-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p>Expected</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-6-3" class="td11">
60
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-6-4" class="td11">
31
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-6-5" class="td11">
14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-6-6" class="td11">
4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-6-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>1</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-91-6-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<div class="multirow">
<!-- rows=-2 -->
<p>111</p>
</div>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-91-7-">
<td style="white-space:nowrap; text-align:right;" id="TBL-91-7-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-91-8-">
<td style="white-space:nowrap; text-align:right;" id="TBL-91-8-1" class="td11">
<pre><code>        &lt;div class=&quot;multicolumn&quot;  style=&quot;white-space:nowrap; text-align:right;&quot;&gt;&lt;/div&gt; &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:right;&quot; id=&quot;TBL-91-8-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p>Observed</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-8-3" class="td11">
206
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-8-4" class="td11">
101
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-8-5" class="td11">
45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-8-6" class="td11">
14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-8-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>5</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-91-8-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">

</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-91-9-">
<td style="white-space:nowrap; text-align:right;" id="TBL-91-9-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<div class="multirow">
<!-- rows=-2 -->
<p><span class="cmbx-12">mtSLIPT</span><span class="cmsy-10x-x-120">-</span></p>
</div>
</div>
</td>
<td style="white-space:nowrap; text-align:right;" id="TBL-91-9-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p>Expected</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-9-3" class="td11">
200
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-9-4" class="td11">
105
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-9-5" class="td11">
48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-9-6" class="td11">
14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-9-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>4</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-91-9-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<div class="multirow">
<!-- rows=-2 -->
<p>371</p>
</div>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-91-10-">
<td style="white-space:nowrap; text-align:right;" id="TBL-91-10-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-91-11-">
<td style="white-space:nowrap; text-align:right;" id="TBL-91-11-1" class="td11">
<pre><code>         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:right;&quot; id=&quot;TBL-91-11-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p><span class="cmbx-12">Total</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-11-3" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>269</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-11-4" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>143</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-11-5" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>63</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-11-6" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>19</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-91-11-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>6</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-91-11-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>482</p>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-91-12-">
<td style="white-space:nowrap; text-align:right;" id="TBL-91-12-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-91-13-">
<td  style="white-space:nowrap; text-align:right;" id="TBL-91-13-1"  
class="td11">
<pre><code>         &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/div&gt;&lt;/div&gt;
                                                                                
                                                                                </code></pre>
</div>
<hr class="endfloat" />
<!--l. 72-->
<p class="indent">
<p>This analysis was replicated on a (smaller) stomach cancer dataset but it was less conclusive (<span class="cmmi-12">p </span>= 2<span class="cmmi-12">.</span>36 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">1</span></sup> by Fisher’s exact test). As shown in Table <a 
href="#x1-203003r3">C.3<!--tex4ht:ref: tab:secondary_screen_stad --></a>, fewer SLIPT candidates were validated than expected statistically. However, these results in stomach cancer may not be directly comparable to experiments in a breast cell line. Genes validated by 0 or 1 siRNA behave consistently with the results above.</p>
</p>
<!--l. 74-->
<p class="indent">
<a 
 id="x1-203003r3"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table C.3: </span><span class="content">Comparing SLIPT genes against Secondary siRNA Screen in stomach cancer</span></p>
</div>
<!--tex4ht:label?: x1-203003r3 -->
<div class="center" 
>
<!--l. 77-->
<p class="noindent">
</p>
<div class="tabular">
<table id="TBL-92" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-92-1g">
<col 
id="TBL-92-1" />
<col 
id="TBL-92-2" />
<col 
id="TBL-92-3" />
<col 
id="TBL-92-4" />
<col 
id="TBL-92-5" />
<col 
id="TBL-92-6" />
<col 
id="TBL-92-7" />
<col 
id="TBL-92-8" />
</colgroup>
<tr  
 style="vertical-align:baseline;" id="TBL-92-1-">
<td style="white-space:nowrap; text-align:right;" id="TBL-92-1-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:right;&quot; id=&quot;TBL-92-1-2&quot;  </code></pre>
class=“td11”&gt;
</td>
<td colspan="5" style="white-space:nowrap; text-align:center;" id="TBL-92-1-3" class="td11">
<pre><code>    &lt;div class=&quot;multicolumn&quot;  style=&quot;white-space:nowrap; text-align:center;&quot;&gt;&lt;span </code></pre>
class=“cmbx-12”&gt;Secondary Screen</span>
</div>
</td>
<td  style="white-space:nowrap; text-align:left;" id="TBL-92-1-8"  
class="td11">
<pre><code>    &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-92-2-">
<td style="white-space:nowrap; text-align:right;" id="TBL-92-2-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-92-3-">
<td style="white-space:nowrap; text-align:right;" id="TBL-92-3-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:right;&quot; id=&quot;TBL-92-3-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:right;">

</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-3-3" class="td11">
0/4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-3-4" class="td11">
1/4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-3-5" class="td11">
2/4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-3-6" class="td11">
3/4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-3-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>4/4</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-92-3-8" class="td11">
<strong><span class="cmbx-12">Total</span></strong>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-92-4-">
<td style="white-space:nowrap; text-align:right;" id="TBL-92-4-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-92-5-">
<td style="white-space:nowrap; text-align:right;" id="TBL-92-5-1" class="td11">
<pre><code>     &lt;div class=&quot;multicolumn&quot;  style=&quot;white-space:nowrap; text-align:right;&quot;&gt;&lt;/div&gt; &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:right;&quot; id=&quot;TBL-92-5-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p>Observed</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-5-3" class="td11">
67
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-5-4" class="td11">
47
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-5-5" class="td11">
13
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-5-6" class="td11">
4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-5-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>1</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-92-5-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">

</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-92-6-">
<td style="white-space:nowrap; text-align:right;" id="TBL-92-6-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<div class="multirow">
<!-- rows=-2 -->
<p><span class="cmbx-12">SLIPT</span>+</p>
</div>
</div>
</td>
<td style="white-space:nowrap; text-align:right;" id="TBL-92-6-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p>Expected</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-6-3" class="td11">
71
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-6-4" class="td11">
37
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-6-5" class="td11">
17
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-6-6" class="td11">
5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-6-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>2</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-92-6-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<div class="multirow">
<!-- rows=-2 -->
<p>132</p>
</div>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-92-7-">
<td style="white-space:nowrap; text-align:right;" id="TBL-92-7-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-92-8-">
<td style="white-space:nowrap; text-align:right;" id="TBL-92-8-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:right;">

</div>
<pre><code>           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:right;&quot; id=&quot;TBL-92-8-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p>Observed</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-8-3" class="td11">
195
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-8-4" class="td11">
90
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-8-5" class="td11">
50
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-8-6" class="td11">
14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-8-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>5</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-92-8-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">

</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-92-9-">
<td style="white-space:nowrap; text-align:right;" id="TBL-92-9-1" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<div class="multirow">
<!-- rows=-2 -->
<p><span class="cmbx-12">SLIPT</span><span class="cmsy-10x-x-120">-</span></p>
</div>
</div>
</td>
<td style="white-space:nowrap; text-align:right;" id="TBL-92-9-2" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p>Expected</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-9-3" class="td11">
190
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-9-4" class="td11">
100
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-9-5" class="td11">
46
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-9-6" class="td11">
13
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-9-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>4</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-92-9-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<div class="multirow">
<!-- rows=-2 -->
<p>354</p>
</div>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-92-10-">
<td style="white-space:nowrap; text-align:right;" id="TBL-92-10-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-92-11-">
<td style="white-space:nowrap; text-align:right;" id="TBL-92-11-1" class="td11">
<pre><code>      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:right;&quot; id=&quot;TBL-92-11-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:right;">
<p><span class="cmbx-12">Total</span></p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-11-3" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>262</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-11-4" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>137</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-11-5" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>63</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-11-6" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>19</p>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-92-11-7" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p>6</p>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-92-11-8" class="td11">
<div class="multicolumn" style="white-space:nowrap; text-align:left;">
<p>486</p>
</div>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-92-12-">
<td style="white-space:nowrap; text-align:right;" id="TBL-92-12-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-92-13-">
<td  style="white-space:nowrap; text-align:right;" id="TBL-92-13-1"  
class="td11">
<pre><code>      &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/div&gt;&lt;/div&gt;
                                                                                
                                                                                </code></pre>
</div>
<hr class="endfloat" />
<a 
 id="x1-203004r288"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix D</span><br /><a 
 id="x1-204000D"></a>Mutation Analysis in Breast Cancer
</h2>
<a 
 id="x1-204001r295"></a>
<h3 class="sectionHead">
<span class="titlemark">D.1 </span> <a 
 id="x1-2050001"></a>Synthetic Lethal Genes and Pathways
</h3>
<!--l. 103-->
<p class="noindent">
<p>SLIPT expression analysis (described in Section <a 
href="#x1-930001">3.1<!--tex4ht:ref: methods:SLIPT --></a>) on TCGA breast cancer data (<span class="cmmi-12">n </span>= 969) found the following genes and pathways, described in sections <a 
href="#x1-1180001">4.1<!--tex4ht:ref: chapt3:exprSL_genes --></a> and <a 
href="#x1-1190001">4.1.1<!--tex4ht:ref: chapt3:exprSL_pathways --></a>.</p>
</p>
<!--l. 105-->
<p class="indent">
<a 
 id="x1-205001r1"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table D.1: </span><span class="content">Candidate synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from mtSLIPT</span></p>
</div>
<!--tex4ht:label?: x1-205001r1 -->
<div class="tabular">
<table id="TBL-93" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-93-1g">
<col 
id="TBL-93-1" />
<col 
id="TBL-93-2" />
<col 
id="TBL-93-3" />
<col 
id="TBL-93-4" />
<col 
id="TBL-93-5" />
<col 
id="TBL-93-6" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-93-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-1-1" class="td11">
<span class="cmbx-12">Gene </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-1-2" class="td11">
Observed
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-1-3" class="td11">
Expected
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-1-4" class="td11">
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-1-5" class="td11">
p-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-1-6" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-2-1" class="td11">
<span class="cmti-12">TFAP2B </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-2-2" class="td11">
<pre><code>8       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-2-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-2-4" class="td11">
89.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-2-5" class="td11">
3<span class="cmmi-12">.</span>60 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-2-6" class="td11">
8<span class="cmmi-12">.</span>37 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-3-1" class="td11">
<span class="cmti-12">ZNF423 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-3-2" class="td11">
<pre><code>15      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-3-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-3-4" class="td11">
78.8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-3-5" class="td11">
7<span class="cmmi-12">.</span>89 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-3-6" class="td11">
1<span class="cmmi-12">.</span>22 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-4-1" class="td11">
<span class="cmti-12">CALCOCO1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-4-2" class="td11">
<pre><code>11      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-4-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-4-4" class="td11">
76.8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-4-5" class="td11">
2<span class="cmmi-12">.</span>09 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-4-6" class="td11">
2<span class="cmmi-12">.</span>59 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-5-1" class="td11">
<span class="cmti-12">RBM5 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-5-2" class="td11">
<pre><code>13      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-5-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-5-4" class="td11">
75.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-5-5" class="td11">
3<span class="cmmi-12">.</span>65 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-5-6" class="td11">
4<span class="cmmi-12">.</span>00 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-6-1" class="td11">
<span class="cmti-12">BTG2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-6-2" class="td11">
<pre><code>7       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-6-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-6-4" class="td11">
71.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-6-5" class="td11">
2<span class="cmmi-12">.</span>72 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-6-6" class="td11">
1<span class="cmmi-12">.</span>81 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-7-1" class="td11">
<span class="cmti-12">RXRA </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-7-2" class="td11">
<pre><code>6       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-7-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-7-4" class="td11">
70.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-7-5" class="td11">
5<span class="cmmi-12">.</span>00 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-7-6" class="td11">
2<span class="cmmi-12">.</span>97 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-8-1" class="td11">
<span class="cmti-12">SLC27A1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-8-2" class="td11">
<pre><code>11      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-8-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-8-4" class="td11">
70.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-8-5" class="td11">
5<span class="cmmi-12">.</span>42 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-8-6" class="td11">
2<span class="cmmi-12">.</span>97 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-9-1" class="td11">
<span class="cmti-12">MEF2D </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-9-2" class="td11">
<pre><code>12      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-9-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-9-4" class="td11">
69.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-9-5" class="td11">
7<span class="cmmi-12">.</span>86 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-9-6" class="td11">
3<span class="cmmi-12">.</span>95 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-10-1" class="td11">
<span class="cmti-12">NISCH </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-10-2" class="td11">
<pre><code>12      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-10-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-10-4" class="td11">
69.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-10-5" class="td11">
7<span class="cmmi-12">.</span>86 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-10-6" class="td11">
3<span class="cmmi-12">.</span>95 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-11-1" class="td11">
<span class="cmti-12">AVPR2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-11-2" class="td11">
<pre><code>9       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-11-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-11-4" class="td11">
69.2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-11-5" class="td11">
9<span class="cmmi-12">.</span>36 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-11-6" class="td11">
4<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-12-1" class="td11">
<span class="cmti-12">CRY2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-12-2" class="td11">
<pre><code>13      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-12-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-12-4" class="td11">
68.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-12-5" class="td11">
1<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-12-6" class="td11">
4<span class="cmmi-12">.</span>98 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-13-1" class="td11">
<span class="cmti-12">RAPGEF3 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-13-2" class="td11">
<pre><code>13      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-13-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-13-4" class="td11">
68.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-13-5" class="td11">
1<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-13-6" class="td11">
4<span class="cmmi-12">.</span>98 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-14-1" class="td11">
<span class="cmti-12">NRIP2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-14-2" class="td11">
<pre><code>10      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-14-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-14-4" class="td11">
68.2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-14-5" class="td11">
1<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-14-6" class="td11">
7<span class="cmmi-12">.</span>18 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-15-1" class="td11">
<span class="cmti-12">DARC </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-15-2" class="td11">
<pre><code>12      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-15-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-15-4" class="td11">
66.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-15-5" class="td11">
3<span class="cmmi-12">.</span>76 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-15-6" class="td11">
1<span class="cmmi-12">.</span>54 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-16-1" class="td11">
<span class="cmti-12">SFRS5 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-16-2" class="td11">
<pre><code>12      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-16-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-16-4" class="td11">
66.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-16-5" class="td11">
3<span class="cmmi-12">.</span>76 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-16-6" class="td11">
1<span class="cmmi-12">.</span>54 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-17-1" class="td11">
<span class="cmti-12">NOSTRIN </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-17-2" class="td11">
<pre><code>5       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-17-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-17-4" class="td11">
65.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-17-5" class="td11">
7<span class="cmmi-12">.</span>40 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-17-6" class="td11">
2<span class="cmmi-12">.</span>70 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-18-1" class="td11">
<span class="cmti-12">KIF13B </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-18-2" class="td11">
<pre><code>12      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-18-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-18-4" class="td11">
63.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-18-5" class="td11">
1<span class="cmmi-12">.</span>69 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-18-6" class="td11">
5<span class="cmmi-12">.</span>16 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-19-1" class="td11">
<span class="cmti-12">TENC1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-19-2" class="td11">
<pre><code>10      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-19-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-19-4" class="td11">
62.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-19-5" class="td11">
2<span class="cmmi-12">.</span>67 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-19-6" class="td11">
7<span class="cmmi-12">.</span>40 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-20-1" class="td11">
<span class="cmti-12">MFAP4 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-20-2" class="td11">
<pre><code>12      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-20-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-20-4" class="td11">
60.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-20-5" class="td11">
7<span class="cmmi-12">.</span>17 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-20-6" class="td11">
1<span class="cmmi-12">.</span>67 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-21-1" class="td11">
<span class="cmti-12">ELN </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-21-2" class="td11">
<pre><code>13      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-21-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-21-4" class="td11">
59.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-21-5" class="td11">
1<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-21-6" class="td11">
2<span class="cmmi-12">.</span>32 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-22-1" class="td11">
<span class="cmti-12">SGK223 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-22-2" class="td11">
<pre><code>14      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-22-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-22-4" class="td11">
59
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-22-5" class="td11">
1<span class="cmmi-12">.</span>51 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-22-6" class="td11">
3<span class="cmmi-12">.</span>05 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-23-1" class="td11">
<span class="cmti-12">KIF12 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-23-2" class="td11">
<pre><code>11      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-23-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-23-4" class="td11">
58.8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-23-5" class="td11">
1<span class="cmmi-12">.</span>74 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-23-6" class="td11">
3<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-24-1" class="td11">
<span class="cmti-12">SELP </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-24-2" class="td11">
<pre><code>11      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-24-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-24-4" class="td11">
58.8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-24-5" class="td11">
1<span class="cmmi-12">.</span>74 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-24-6" class="td11">
3<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-25-1" class="td11">
<span class="cmti-12">CIRBP </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-25-2" class="td11">
<pre><code>9       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-25-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-25-4" class="td11">
58.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-25-5" class="td11">
1<span class="cmmi-12">.</span>83 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-25-6" class="td11">
3<span class="cmmi-12">.</span>41 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-93-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-93-26-1" class="td11">
<span class="cmti-12">CTDSP1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-26-2" class="td11">
<pre><code>9       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-93-26-3&quot;  </code></pre>
class=“td11”&gt; 36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-26-4" class="td11">
58.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-26-5" class="td11">
1<span class="cmmi-12">.</span>83 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-93-26-6" class="td11">
3<span class="cmmi-12">.</span>41 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-93-27-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-93-27-1"  
class="td11">
<pre><code>         &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 172-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Strongest candidate SL partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">by mtSLIPT with observed and expected mutant samples</span> <span class="cmr-10x-x-109">with low expression of partner genes</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 176-->
<p class="indent">
<a 
 id="x1-205002r2"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table D.2: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT</span></p>
</div>
<!--tex4ht:label?: x1-205002r2 -->
<div class="tabular">
<table id="TBL-94" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-94-1g">
<col 
id="TBL-94-1" />
<col 
id="TBL-94-2" />
<col 
id="TBL-94-3" />
<col 
id="TBL-94-4" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-94-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-1-1" class="td11">
<strong><span class="cmbx-12">Pathways Over-represented</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-1-2" class="td11">
<strong><span class="cmbx-12">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-1-3" class="td11">
<strong><span class="cmbx-12">SL Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-1-4" class="td11">
<strong><span class="cmbx-12">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-2-1" class="td11">
Eukaryotic Translation Elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-2-2" class="td11">
<pre><code>  86          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-2-3&quot;  </code></pre>
class=“td11”&gt; 60
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-2-4" class="td11">
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">128</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-3-1" class="td11">
Peptide chain elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-3-2" class="td11">
<pre><code>  83          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-3-3&quot;  </code></pre>
class=“td11”&gt; 59
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-3-4" class="td11">
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">128</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-4-1" class="td11">
Eukaryotic Translation Termination
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-4-2" class="td11">
<pre><code>  83          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-4-3&quot;  </code></pre>
class=“td11”&gt; 58
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-4-4" class="td11">
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">125</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-5-1" class="td11">
Viral mRNA Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-5-2" class="td11">
<pre><code>  81          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-5-3&quot;  </code></pre>
class=“td11”&gt; 57
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-5-4" class="td11">
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">124</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-6-1" class="td11">
Nonsense Mediated Decay independent of the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-6-2" class="td11">
<pre><code>  88          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-6-3&quot;  </code></pre>
class=“td11”&gt; 59
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-6-4" class="td11">
8<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">124</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-7-1" class="td11">
Nonsense-Mediated Decay
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-7-2" class="td11">
<pre><code>  103         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-7-3&quot;  </code></pre>
class=“td11”&gt; 61
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-7-4" class="td11">
5<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">117</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-8-1" class="td11">
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-8-2" class="td11">
<pre><code>  103         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-8-3&quot;  </code></pre>
class=“td11”&gt; 61
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-8-4" class="td11">
5<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">117</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-9-1" class="td11">
Formation of a pool of free 40S subunits
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-9-2" class="td11">
<pre><code>  93          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-9-3&quot;  </code></pre>
class=“td11”&gt; 58
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-9-4" class="td11">
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">116</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-10-1" class="td11">
L13a-mediated translational silencing of Ceruloplasmin expression
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-10-2" class="td11">
<pre><code>  103         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-10-3&quot;  </code></pre>
class=“td11”&gt; 59
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-10-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">111</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-11-1" class="td11">
3’ -UTR-mediated translational regulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-11-2" class="td11">
<pre><code>  103         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-11-3&quot;  </code></pre>
class=“td11”&gt; 59
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-11-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">111</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-12-1" class="td11">
GTP hydrolysis and joining of the 60S ribosomal subunit
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-12-2" class="td11">
<pre><code>  104         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-12-3&quot;  </code></pre>
class=“td11”&gt; 59
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-12-4" class="td11">
6<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">111</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-13-1" class="td11">
SRP-dependent cotranslational protein targeting to membrane
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-13-2" class="td11">
<pre><code>  104         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-13-3&quot;  </code></pre>
class=“td11”&gt; 58
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-13-4" class="td11">
2<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">108</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-14-1" class="td11">
Eukaryotic Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-14-2" class="td11">
<pre><code>  111         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-14-3&quot;  </code></pre>
class=“td11”&gt; 59
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-14-4" class="td11">
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">106</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-15-1" class="td11">
Cap-dependent Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-15-2" class="td11">
<pre><code>  111         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-15-3&quot;  </code></pre>
class=“td11”&gt; 59
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-15-4" class="td11">
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">106</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-16-1" class="td11">
Influenza Viral RNA Transcription and Replication
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-16-2" class="td11">
<pre><code>  108         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-16-3&quot;  </code></pre>
class=“td11”&gt; 57
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-16-4" class="td11">
5<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">103</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-17-1" class="td11">
Influenza Infection
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-17-2" class="td11">
<pre><code>  117         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-17-3&quot;  </code></pre>
class=“td11”&gt; 59
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-17-4" class="td11">
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">102</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-18-1" class="td11">
Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-18-2" class="td11">
<pre><code>  141         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-18-3&quot;  </code></pre>
class=“td11”&gt; 64
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-18-4" class="td11">
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">101</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-19-1" class="td11">
Influenza Life Cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-19-2" class="td11">
<pre><code>  112         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-19-3&quot;  </code></pre>
class=“td11”&gt; 57
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-19-4" class="td11">
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">100</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-20-1" class="td11">
GPCR downstream signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-20-2" class="td11">
<pre><code>  472         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-20-3&quot;  </code></pre>
class=“td11”&gt; 116
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-20-4" class="td11">
1<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-94-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-94-21-1" class="td11">
Hemostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-21-2" class="td11">
<pre><code>  422         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-94-21-3&quot;  </code></pre>
class=“td11”&gt; 105
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-94-21-4" class="td11">
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">78</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-94-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-94-22-1"  
class="td11">
<pre><code>                                                         &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 232-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Gene set over-representation analysis (hypergeometric test) for Reactome pathways in mtSLIPT partners for </span><span class="cmti-10x-x-109">CDH1</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 237-->
<p class="indent">
The genes and pathways identified in Tables <a 
href="#x1-205001r1">D.1<!--tex4ht:ref: tab:gene_mtSL --></a> and <a 
href="#x1-205002r2">D.2<!--tex4ht:ref: tab:pathway_mtSL --></a> were derived from comparing the expression profiles of potential partners to the mutation status of <span class="cmti-12">CDH1 </span>(as shown in Figure <a 
href="#x1-93002r2">3.2<!--tex4ht:ref: fig:SLIPT_Method_mtSL --></a>). Thus the following analysis is only limited the samples for which TCGA provides both expression and somatic mutation data. <a 
 id="x1-205003r307"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">D.2 </span> <a 
 id="x1-2060002"></a>Synthetic Lethal Expression Profiles
</h3>
<!--l. 244-->
<p class="noindent">
<p>Similar to the analysis of synthetic lethal partners against low <span class="cmti-12">CDH1 </span>expression in <a 
href="#x1-1200002">4.1.2<!--tex4ht:ref: chapt3:exprSL_clusters --></a>, the partners detected from <span class="cmti-12">CDH1 </span>mutation were also examined for their expression profiles and the pathway composition of gene clusters. Hierachical clustering was performed on mtSLIPT partners for <span class="cmti-12">CDH1 </span>as showing in Figure <a 
href="#x1-206001r1">D.1<!--tex4ht:ref: fig:slipt_expr_mtSL --></a>. Over-representation for Reactome pathways for each of the gene clusters identified is given in Table <a 
href="#x1-206002r3">D.3<!--tex4ht:ref: tab:pathway_clusters_mtSL --></a>.</p>
</p>
<!--l. 246-->
<p class="indent">
<a 
 id="x1-206001r1"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________      <!--l. 249-->
<p class="noindent">
<img 
src="CDH1_Heatmaps_Genes_Split_By_CDH1_z-trans_mtSL_cordistx_Pub.png" alt="PIC"  
 /> <br />
</p>
<div class="caption">
<p><span class="id">Figure D.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethal expression profiles of analysed samples.</span></strong> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">expression profile heatmap (correlation distance) of all samples (separated by </span><span class="cmti-10x-x-109">CDH1</span> <span class="cmr-10x-x-109">somatic mutation status) analysed in TCGA breast cancer dataset for gene expression</span> <span class="cmr-10x-x-109">of 3,743 candidate partners of E-cadherin (</span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">) from mtSLIPT prediction (with</span> <span class="cmr-10x-x-109">significant FDR adjusted </span><span class="cmmi-10x-x-109">p &lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05). Deeply clustered, inter-correlated genes form several</span> <span class="cmr-10x-x-109">main groups, each containing genes that were SL candidates or toxic in an siRNA</span> <span class="cmr-10x-x-109">screen </span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">Telford </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">2015</span></a><span class="cmr-10x-x-109">). Clusters had different sample groups highly expressing the</span> <span class="cmr-10x-x-109">synthetic lethal candidates in </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutant samples and often lowly expressing </span><span class="cmti-10x-x-109">CDH1</span> <span class="cmr-10x-x-109">wildtype samples (which were not tested for), although many of the </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutant</span> <span class="cmr-10x-x-109">samples had among the lowest </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">expression. In contrast to the expression analysis</span> <span class="cmr-10x-x-109">the (predominantly </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">wildtype) basal subtype and estrogen receptor negative</span> <span class="cmr-10x-x-109">samples have depleted expression among most candidate synthetic lethal partners. </span></span></p>
</div>
<!--tex4ht:label?: x1-206001r1 -->
</div>
<hr class="endfloat" />
<!--l. 262-->
<p class="indent">
<a 
 id="x1-206002r3"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table D.3: </span><span class="content">Pathway composition for clusters of <span class="cmti-12">CDH1 </span>partners from mtSLIPT</span></p>
</div>
<!--tex4ht:label?: x1-206002r3 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-95" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-95-1g">
<col 
id="TBL-95-1" />
<col 
id="TBL-95-2" />
<col 
id="TBL-95-3" />
<col 
id="TBL-95-4" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-95-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-1-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 1</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-1-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-1-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-1-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-2-1" class="td11">
Olfactory Signaling Pathway
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-2-2" class="td11">
<pre><code>    57            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-2-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-2-4" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-3-1" class="td11">
Assembly of the primary cilium
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-3-2" class="td11">
<pre><code>   149           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-3-3&quot;  </code></pre>
class=“td11”&gt; 14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-3-4" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-4-1" class="td11">
Sphingolipid metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-4-2" class="td11">
<pre><code>    62            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-4-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-4-4" class="td11">
<pre><code> 9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-5-1" class="td11">
Signaling by ERBB4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-5-2" class="td11">
<pre><code>   133           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-5-3&quot;  </code></pre>
class=“td11”&gt; 12
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-5-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-6-1" class="td11">
PI3K Cascade
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-6-2" class="td11">
<pre><code>    65            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-6-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-6-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-7-1" class="td11">
Circadian Clock
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-7-2" class="td11">
<pre><code>    33            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-7-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-7-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-8-1" class="td11">
Nuclear signaling by ERBB4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-8-2" class="td11">
<pre><code>    34            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-8-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-8-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-9-1" class="td11">
Intraflagellar transport
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-9-2" class="td11">
<pre><code>    35            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-9-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-9-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-10-1" class="td11">
PI3K events in ERBB4 signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-10-2" class="td11">
<pre><code>    87            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-10-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-10-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-11-1" class="td11">
PIP3 activates AKT signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-11-2" class="td11">
<pre><code>    87            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-11-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-11-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-12-1" class="td11">
PI3K events in ERBB2 signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-12-2" class="td11">
<pre><code>    87            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-12-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-12-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-13-1" class="td11">
PI-3K cascade:FGFR1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-13-2" class="td11">
<pre><code>    87            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-13-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-13-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-14-1" class="td11">
PI-3K cascade:FGFR2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-14-2" class="td11">
<pre><code>    87            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-14-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-14-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-15-1" class="td11">
PI-3K cascade:FGFR3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-15-2" class="td11">
<pre><code>    87            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-15-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-15-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-16-1" class="td11">
PI-3K cascade:FGFR4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-16-2" class="td11">
<pre><code>    87            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-16-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-16-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-17-1" class="td11">
Deadenylation of mRNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-17-2" class="td11">
<pre><code>    22            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-17-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-17-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-18-1" class="td11">
PI3K/AKT activation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-18-2" class="td11">
<pre><code>    90            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-18-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-18-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-19-1" class="td11">
Cargo trafficking to the periciliary membrane
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-19-2" class="td11">
<pre><code>    38            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-19-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-19-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-20-1" class="td11">
Signaling by Hedgehog
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-20-2" class="td11">
<pre><code>   108           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-20-3&quot;  </code></pre>
class=“td11”&gt; 9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-20-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-21-1" class="td11">
Downstream signal transduction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-21-2" class="td11">
<pre><code>   143           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-21-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-21-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-95-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-95-22-1"  
class="td11">
<pre><code>                                                              &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-23-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 2</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-23-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-23-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-23-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-24-1" class="td11">
G<sub><span class="cmmi-8">αs</span></sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-24-2" class="td11">
<pre><code>    83            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-24-3&quot;  </code></pre>
class=“td11”&gt; 19
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-24-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-25-1" class="td11">
Extracellular matrix organization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-25-2" class="td11">
<pre><code>   238           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-25-3&quot;  </code></pre>
class=“td11”&gt; 30
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-25-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-26-1" class="td11">
Hemostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-26-2" class="td11">
<pre><code>   422           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-26-3&quot;  </code></pre>
class=“td11”&gt; 46
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-26-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-27-1" class="td11">
Aquaporin-mediated transport
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-27-2" class="td11">
<pre><code>    32            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-27-3&quot;  </code></pre>
class=“td11”&gt; 9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-27-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-28-1" class="td11">
Transcriptional regulation of white adipocyte differentiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-28-2" class="td11">
<pre><code>    56            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-28-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-28-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-29-1" class="td11">
Degradation of the extracellular matrix
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-29-2" class="td11">
<pre><code>   102           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-29-3&quot;  </code></pre>
class=“td11”&gt; 15
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-29-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-30-1" class="td11">
Integration of energy metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-30-2" class="td11">
<pre><code>    84            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-30-3&quot;  </code></pre>
class=“td11”&gt; 13
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-30-4" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-31-1" class="td11">
GPCR downstream signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-31-2" class="td11">
<pre><code>   472           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-31-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-31-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-32-1" class="td11">
G<sub><span class="cmmi-8">αz</span></sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-32-2" class="td11">
<pre><code>    15            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-32-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-32-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-33-1" class="td11">
Molecules associated with elastic fibres
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-33-2" class="td11">
<pre><code>    33            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-33-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-33-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-34-1" class="td11">
Phase 1 - Functionalization of compounds
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-34-2" class="td11">
<pre><code>    67            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-34-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-34-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-35-1" class="td11">
Platelet activation, signaling and aggregation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-35-2" class="td11">
<pre><code>   179           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-35-3&quot;  </code></pre>
class=“td11”&gt; 20
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-35-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-36-1" class="td11">
Vasopressin regulates renal water homeostasis via Aquaporins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-36-2" class="td11">
<pre><code>    24            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-36-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-36-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-37-1" class="td11">
Elastic fibre formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-37-2" class="td11">
<pre><code>    37            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-37-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-37-4" class="td11">
<pre><code> &lt;span </code></pre>
class=“cmmi-12”&gt;.</span>03 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-38-1" class="td11">
Calmodulin induced events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-38-2" class="td11">
<pre><code>    27            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-38-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-38-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-39-1" class="td11">
CaM pathway
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-39-2" class="td11">
<pre><code>    27            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-39-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-39-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-40-1" class="td11">
cGMP effects
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-40-2" class="td11">
<pre><code>    18            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-40-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-40-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-41-1" class="td11">
G<sub><span class="cmmi-8">αi</span></sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-41-2" class="td11">
<pre><code>   167           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-41-3&quot;  </code></pre>
class=“td11”&gt; 18
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-41-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-42-1" class="td11">
Ca-dependent events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-42-2" class="td11">
<pre><code>    29            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-42-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-42-4" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-43-1" class="td11">
Binding and Uptake of Ligands by Scavenger Receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-43-2" class="td11">
<pre><code>    40            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-43-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-43-4" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-95-44-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-95-44-1"  
class="td11">
<pre><code>                                                              &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-45-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 3</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-45-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-45-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-45-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-46-1" class="td11">
Eukaryotic Translation Elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-46-2" class="td11">
<pre><code>    86            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-46-3&quot;  </code></pre>
class=“td11”&gt; 55
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-46-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">112</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-47-1" class="td11">
Peptide chain elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-47-2" class="td11">
<pre><code>    83            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-47-3&quot;  </code></pre>
class=“td11”&gt; 54
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-47-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">112</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-48-1" class="td11">
Viral mRNA Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-48-2" class="td11">
<pre><code>    81            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-48-3&quot;  </code></pre>
class=“td11”&gt; 53
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-48-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">111</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-49-1" class="td11">
Eukaryotic Translation Termination
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-49-2" class="td11">
<pre><code>    83            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-49-3&quot;  </code></pre>
class=“td11”&gt; 53
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-49-4" class="td11">
<pre><code>7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">110</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-50-1" class="td11">
Nonsense Mediated Decay independent of the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-50-2" class="td11">
<pre><code>    88            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-50-3&quot;  </code></pre>
class=“td11”&gt; 54
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-50-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">108</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-51-1" class="td11">
Formation of a pool of free 40S subunits
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-51-2" class="td11">
<pre><code>    93            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-51-3&quot;  </code></pre>
class=“td11”&gt; 53
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-51-4" class="td11">
<pre><code>4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">102</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-52-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-52-1" class="td11">
Nonsense-Mediated Decay
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-52-2" class="td11">
<pre><code>   103           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-52-3&quot;  </code></pre>
class=“td11”&gt; 54
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-52-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">98</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-53-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-53-1" class="td11">
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-53-2" class="td11">
<pre><code>   103           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-53-3&quot;  </code></pre>
class=“td11”&gt; 54
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-53-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">98</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-54-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-54-1" class="td11">
L13a-mediated translational silencing of Ceruloplasmin expression
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-54-2" class="td11">
<pre><code>   103           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-54-3&quot;  </code></pre>
class=“td11”&gt; 53
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-54-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">95</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-55-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-55-1" class="td11">
3’ -UTR-mediated translational regulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-55-2" class="td11">
<pre><code>   103           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-55-3&quot;  </code></pre>
class=“td11”&gt; 53
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-55-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">95</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-56-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-56-1" class="td11">
SRP-dependent cotranslational protein targeting to membrane
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-56-2" class="td11">
<pre><code>   104           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-56-3&quot;  </code></pre>
class=“td11”&gt; 53
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-56-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">95</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-57-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-57-1" class="td11">
GTP hydrolysis and joining of the 60S ribosomal subunit
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-57-2" class="td11">
<pre><code>   104           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-57-3&quot;  </code></pre>
class=“td11”&gt; 53
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-57-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">95</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-58-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-58-1" class="td11">
Influenza Viral RNA Transcription and Replication
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-58-2" class="td11">
<pre><code>   108           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-58-3&quot;  </code></pre>
class=“td11”&gt; 53
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-58-4" class="td11">
<pre><code> 9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">93</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-59-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-59-1" class="td11">
Eukaryotic Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-59-2" class="td11">
<pre><code>   111           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-59-3&quot;  </code></pre>
class=“td11”&gt; 53
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-59-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">91</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-60-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-60-1" class="td11">
Cap-dependent Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-60-2" class="td11">
<pre><code>   111           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-60-3&quot;  </code></pre>
class=“td11”&gt; 53
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-60-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">91</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-61-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-61-1" class="td11">
Influenza Life Cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-61-2" class="td11">
<pre><code>   112           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-61-3&quot;  </code></pre>
class=“td11”&gt; 53
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-61-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-62-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-62-1" class="td11">
Influenza Infection
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-62-2" class="td11">
<pre><code>   117           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-62-3&quot;  </code></pre>
class=“td11”&gt; 53
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-62-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">88</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-63-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-63-1" class="td11">
Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-63-2" class="td11">
<pre><code>   141           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-63-3&quot;  </code></pre>
class=“td11”&gt; 55
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-63-4" class="td11">
<pre><code> 3 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">81</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-64-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-64-1" class="td11">
Formation of the ternary complex, and subsequently, the 43S complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-64-2" class="td11">
<pre><code>    47            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-64-3&quot;  </code></pre>
class=“td11”&gt; 23
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-64-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">48</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-65-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-65-1" class="td11">
Translation initiation complex formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-65-2" class="td11">
<pre><code>    54            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-65-3&quot;  </code></pre>
class=“td11”&gt; 23
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-65-4" class="td11">
<pre><code> 9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">45</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-95-66-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-95-66-1"  
class="td11">
<pre><code>                                                              &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-67-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-67-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 4</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-67-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-67-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-67-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-68-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-68-1" class="td11">
ECM proteoglycans
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-68-2" class="td11">
<pre><code>    66            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-68-3&quot;  </code></pre>
class=“td11”&gt; 10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-68-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-69-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-69-1" class="td11">
deactivation of the beta-catenin transactivating complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-69-2" class="td11">
<pre><code>    38            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-69-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-69-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-70-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-70-1" class="td11">
Arachidonic acid metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-70-2" class="td11">
<pre><code>    41            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-70-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-70-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-71-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-71-1" class="td11">
G˙<span class="cmmi-12">αq</span> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-71-2" class="td11">
<pre><code>   149           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-71-3&quot;  </code></pre>
class=“td11”&gt; 14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-71-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-72-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-72-1" class="td11">
HS-GAG degradation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-72-2" class="td11">
<pre><code>    21            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-72-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-72-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-73-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-73-1" class="td11">
Uptake and actions of bacterial toxins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-73-2" class="td11">
<pre><code>    22            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-73-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-73-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-74-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-74-1" class="td11">
Gastrin-CREB signalling pathway via PKC and MAPK
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-74-2" class="td11">
<pre><code>   170           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-74-3&quot;  </code></pre>
class=“td11”&gt; 15
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-74-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-75-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-75-1" class="td11">
RNA Polymerase I, RNA Polymerase III, and Mitochondrial Transcription <span id="textcolor30">ll</span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-75-2" class="td11">
<pre><code>    64            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-75-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-75-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-76-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-76-1" class="td11">
Non-integrin membrane-ECM interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-76-2" class="td11">
<pre><code>    53            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-76-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-76-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-77-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-77-1" class="td11">
Syndecan interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-77-2" class="td11">
<pre><code>    25            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-77-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-77-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-78-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-78-1" class="td11">
NOTCH1 Intracellular Domain Regulates Transcription
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-78-2" class="td11">
<pre><code>    40            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-78-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-78-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-79-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-79-1" class="td11">
Synthesis of Leukotrienes and Eoxins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-79-2" class="td11">
<pre><code>    15            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-79-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-79-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-80-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-80-1" class="td11">
Signaling by NOTCH1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-80-2" class="td11">
<pre><code>    59            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-80-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-80-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-81-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-81-1" class="td11">
Regulation of insulin secretion
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-81-2" class="td11">
<pre><code>    44            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-81-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-81-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-82-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-82-1" class="td11">
Metabolism of lipids and lipoproteins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-82-2" class="td11">
<pre><code>   471           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-82-3&quot;  </code></pre>
class=“td11”&gt; 37
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-82-4" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-83-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-83-1" class="td11">
Signaling by NOTCH
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-83-2" class="td11">
<pre><code>    80            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-83-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-83-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-84-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-84-1" class="td11">
Platelet activation, signaling and aggregation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-84-2" class="td11">
<pre><code>   179           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-84-3&quot;  </code></pre>
class=“td11”&gt; 14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-84-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-85-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-85-1" class="td11">
Recruitment of mitotic centrosome proteins and complexes
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-85-2" class="td11">
<pre><code>    64            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-85-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-85-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-86-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-86-1" class="td11">
Centrosome maturation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-86-2" class="td11">
<pre><code>    64            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-86-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-86-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-95-87-">
<td style="white-space:nowrap; text-align:left;" id="TBL-95-87-1" class="td11">
Biological oxidations
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-87-2" class="td11">
<pre><code>   133           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-95-87-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-95-87-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-95-88-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-95-88-1"  
class="td11">
<pre><code>                                                              &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                                       &lt;/div&gt;
                                                                  
                                                                                </code></pre>
</div>
<hr class="endfloat" />
<a 
 id="x1-206003r310"></a>
<h3 class="sectionHead">
<span class="titlemark">D.3 </span> <a 
 id="x1-2070003"></a>Comparison to Primary Screen
</h3>
<!--l. 455-->
<p class="noindent">
The mutation synthetic lethal partners with <span class="cmti-12">CDH1 </span>were also compared to siRNA primary screen data (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), as performed in Section <a 
href="#x1-1230001">4.2.1<!--tex4ht:ref: chapt3:primary_screen --></a>. These are expected to be more concordant with the experimental results performed on a null mutant, however this not the case at the gene level: less genes overlapped with experimental candidates in Figure <a 
href="#x1-207001r2">D.2<!--tex4ht:ref: fig:Venn_allgenes_mtSL --></a>. This may be affected by lower sample size for mutations in TCGA data or lower frequency (expected value) of <span class="cmti-12">CDH1 </span>mutations compared to low expression.
</p>
<hr class="figure" />
<div class="figure">
<p><a 
 id="x1-207001r2"></a></p>
<hr />
____________________________________________________________________________________      <!--l. 460-->
<p class="noindent">
<img 
src="Venn_mtSL_siRNA_allgenes_reduced_Pub.png" alt="PIC"  
 /> <br />
</p>
<div class="caption">
<p><span class="id">Figure D.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Comparison of mtSLIPT to siRNA.</span></strong> <span class="cmr-10x-x-109">Testing the overlap of gene</span> <span class="cmr-10x-x-109">candidates for E-cadherin synthetic lethal partners between computational (SLIPT) and</span> <span class="cmr-10x-x-109">experimental screening (siRNA) approaches. The </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test suggests that the overlap is no</span> <span class="cmr-10x-x-109">more than would be expected by chance (</span><span class="cmmi-10x-x-109">p </span><span class="cmr-10x-x-109">= 0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">281). </span></span></p>
</div>
<!--tex4ht:label?: x1-207001r2 -->
</div>
<hr class="endfigure" />
<!--l. 471-->
<p class="indent">
<p>Despite a lower sample size (and low number of a predicted partners) for mutation analysis, the pathway composition (Tables <a 
href="#x1-205002r2">D.2<!--tex4ht:ref: tab:pathway_mtSL --></a> and <a 
href="#x1-207002r4">D.4<!--tex4ht:ref: tab:Venn_over-representation_mtSL --></a>) is simlar to expression analysis, as described in Section <a 
href="#x1-1270004">4.2.1.4<!--tex4ht:ref: chapt3:compare_pathway --></a>. In particular, the resampling analysis (Section <a 
href="#x1-2080001">D.3.1<!--tex4ht:ref: appendix:compare_pathway_perm_mtSL --></a>) supported many of the results of expression analysis (Section <a 
href="#x1-1280001">4.2.1.4.1<!--tex4ht:ref: chapt3:compare_pathway_perm --></a>) with Tables <a 
href="#x1-208001r5">D.5<!--tex4ht:ref: tab:pathway_perm_mtSL --></a> and <a 
href="#x1-208002r6">D.6<!--tex4ht:ref: tab:pathway_perm_overlap_mtSL --></a> detecting many of the same or functionally-related pathways.</p>
</p>
<!--l. 474-->
<p class="indent">
<a 
 id="x1-207002r4"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table D.4: </span><span class="content">Pathway composition for <span class="cmti-12">CDH1 </span>partners from mtSLIPT and siRNA</span></p>
</div>
<!--tex4ht:label?: x1-207002r4 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-96" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-96-1g">
<col 
id="TBL-96-1" />
<col 
id="TBL-96-2" />
<col 
id="TBL-96-3" />
<col 
id="TBL-96-4" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-96-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-1-1" class="td11">
<span class="cmbx-12">Predicted only by SLIPT (2901 genes) </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-1-2" class="td11">
Pathway Size
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-1-3" class="td11">
Genes Identified
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-1-4" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-2-1" class="td11">
Eukaryotic Translation Elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-2-2" class="td11">
<pre><code>  87         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-2-3&quot;  </code></pre>
class=“td11”&gt; 57
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-2-4" class="td11">
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">120</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-3-1" class="td11">
Peptide chain elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-3-2" class="td11">
<pre><code>  84         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-3-3&quot;  </code></pre>
class=“td11”&gt; 56
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-3-4" class="td11">
3<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">120</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-4-1" class="td11">
Eukaryotic Translation Termination
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-4-2" class="td11">
<pre><code>  84         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-4-3&quot;  </code></pre>
class=“td11”&gt; 55
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-4-4" class="td11">
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">117</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-5-1" class="td11">
Viral mRNA Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-5-2" class="td11">
<pre><code>  82         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-5-3&quot;  </code></pre>
class=“td11”&gt; 54
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-5-4" class="td11">
4<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">116</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-6-1" class="td11">
Nonsense Mediated Decay independent of the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-6-2" class="td11">
<pre><code>  89         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-6-3&quot;  </code></pre>
class=“td11”&gt; 55
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-6-4" class="td11">
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">113</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-7-1" class="td11">
Formation of a pool of free 40S subunits
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-7-2" class="td11">
<pre><code>  94         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-7-3&quot;  </code></pre>
class=“td11”&gt; 55
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-7-4" class="td11">
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">109</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-8-1" class="td11">
Nonsense-Mediated Decay
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-8-2" class="td11">
<pre><code> 104        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-8-3&quot;  </code></pre>
class=“td11”&gt; 57
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-8-4" class="td11">
8<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">108</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-9-1" class="td11">
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-9-2" class="td11">
<pre><code> 104        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-9-3&quot;  </code></pre>
class=“td11”&gt; 57
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-9-4" class="td11">
8<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">108</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-10-1" class="td11">
L13a-mediated translational silencing of Ceruloplasmin expression
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-10-2" class="td11">
<pre><code> 104        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-10-3&quot;  </code></pre>
class=“td11”&gt; 56
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-10-4" class="td11">
3<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">105</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-11-1" class="td11">
3’ -UTR-mediated translational regulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-11-2" class="td11">
<pre><code> 104        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-11-3&quot;  </code></pre>
class=“td11”&gt; 56
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-11-4" class="td11">
3<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">105</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-12-1" class="td11">
GTP hydrolysis and joining of the 60S ribosomal subunit
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-12-2" class="td11">
<pre><code> 105        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-12-3&quot;  </code></pre>
class=“td11”&gt; 56
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-12-4" class="td11">
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">104</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-13-1" class="td11">
Eukaryotic Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-13-2" class="td11">
<pre><code> 112        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-13-3&quot;  </code></pre>
class=“td11”&gt; 56
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-13-4" class="td11">
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">100</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-14-1" class="td11">
Cap-dependent Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-14-2" class="td11">
<pre><code> 112        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-14-3&quot;  </code></pre>
class=“td11”&gt; 56
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-14-4" class="td11">
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">100</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-15-1" class="td11">
SRP-dependent cotranslational protein targeting to membrane
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-15-2" class="td11">
<pre><code> 105        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-15-3&quot;  </code></pre>
class=“td11”&gt; 54
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-15-4" class="td11">
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">99</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-16-1" class="td11">
Influenza Viral RNA Transcription and Replication
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-16-2" class="td11">
<pre><code> 109        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-16-3&quot;  </code></pre>
class=“td11”&gt; 54
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-16-4" class="td11">
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-17-1" class="td11">
Influenza Life Cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-17-2" class="td11">
<pre><code> 113        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-17-3&quot;  </code></pre>
class=“td11”&gt; 54
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-17-4" class="td11">
9<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">95</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-18-1" class="td11">
Influenza Infection
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-18-2" class="td11">
<pre><code> 118        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-18-3&quot;  </code></pre>
class=“td11”&gt; 55
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-18-4" class="td11">
1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">94</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-19-1" class="td11">
Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-19-2" class="td11">
<pre><code> 142        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-19-3&quot;  </code></pre>
class=“td11”&gt; 60
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-19-4" class="td11">
3<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">94</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-20-1" class="td11">
Infectious disease
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-20-2" class="td11">
<pre><code> 349        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-20-3&quot;  </code></pre>
class=“td11”&gt; 77
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-20-4" class="td11">
5<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">62</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-21-1" class="td11">
Extracellular matrix organization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-21-2" class="td11">
<pre><code> 241        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-21-3&quot;  </code></pre>
class=“td11”&gt; 54
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-21-4" class="td11">
3<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">52</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-96-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-96-22-1"  
class="td11">
<pre><code>                                                         &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-23-1" class="td11">
<span class="cmbx-12">Detected only by siRNA screen (1752 genes) </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-23-2" class="td11">
Pathway Size
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-23-3" class="td11">
Genes Identified
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-23-4" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-24-1" class="td11">
Class A/1 (Rhodopsin-like receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-24-2" class="td11">
<pre><code> 282        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-24-3&quot;  </code></pre>
class=“td11”&gt; 69
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-24-4" class="td11">
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">59</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-25-1" class="td11">
GPCR ligand binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-25-2" class="td11">
<pre><code> 363        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-25-3&quot;  </code></pre>
class=“td11”&gt; 78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-25-4" class="td11">
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">54</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-26-1" class="td11">
Peptide ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-26-2" class="td11">
<pre><code> 175        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-26-3&quot;  </code></pre>
class=“td11”&gt; 41
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-26-4" class="td11">
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">42</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-27-1" class="td11">
<span class="cmmi-12">G</span><sub><span class="cmmi-8">αi</span></sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-27-2" class="td11">
<pre><code> 184        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-27-3&quot;  </code></pre>
class=“td11”&gt; 41
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-27-4" class="td11">
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">40</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-28-1" class="td11">
Gastrin-CREB signalling pathway via PKC and MAPK
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-28-2" class="td11">
<pre><code> 180        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-28-3&quot;  </code></pre>
class=“td11”&gt; 37
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-28-4" class="td11">
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">35</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-29-1" class="td11">
<span class="cmmi-12">G</span><sub><span class="cmmi-8">αq</span></sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-29-2" class="td11">
<pre><code> 159        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-29-3&quot;  </code></pre>
class=“td11”&gt; 34
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-29-4" class="td11">
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">35</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-30-1" class="td11">
DAP12 interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-30-2" class="td11">
<pre><code> 159        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-30-3&quot;  </code></pre>
class=“td11”&gt; 27
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-30-4" class="td11">
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-31-1" class="td11">
VEGFA-VEGFR2 Pathway
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-31-2" class="td11">
<pre><code>  91         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-31-3&quot;  </code></pre>
class=“td11”&gt; 19
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-31-4" class="td11">
1<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">23</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-32-1" class="td11">
Downstream signal transduction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-32-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-32-3&quot;  </code></pre>
class=“td11”&gt; 24
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-32-4" class="td11">
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-33-1" class="td11">
Signaling by VEGF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-33-2" class="td11">
<pre><code>  99         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-33-3&quot;  </code></pre>
class=“td11”&gt; 19
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-33-4" class="td11">
2<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-34-1" class="td11">
DAP12 signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-34-2" class="td11">
<pre><code> 149        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-34-3&quot;  </code></pre>
class=“td11”&gt; 24
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-34-4" class="td11">
4<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-35-1" class="td11">
Organelle biogenesis and maintenance
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-35-2" class="td11">
<pre><code> 264        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-35-3&quot;  </code></pre>
class=“td11”&gt; 34
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-35-4" class="td11">
4<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-36-1" class="td11">
Downstream signaling of activated FGFR1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-36-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-36-3&quot;  </code></pre>
class=“td11”&gt; 21
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-36-4" class="td11">
4<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-37-1" class="td11">
Downstream signaling of activated FGFR2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-37-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-37-3&quot;  </code></pre>
class=“td11”&gt; 21
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-37-4" class="td11">
4<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-38-1" class="td11">
Downstream signaling of activated FGFR3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-38-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-38-3&quot;  </code></pre>
class=“td11”&gt; 21
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-38-4" class="td11">
4<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-39-1" class="td11">
Downstream signaling of activated FGFR4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-39-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-39-3&quot;  </code></pre>
class=“td11”&gt; 21
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-39-4" class="td11">
4<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-40-1" class="td11">
Signaling by ERBB2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-40-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-40-3&quot;  </code></pre>
class=“td11”&gt; 22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-40-4" class="td11">
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-41-1" class="td11">
Signaling by FGFR
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-41-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-41-3&quot;  </code></pre>
class=“td11”&gt; 22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-41-4" class="td11">
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-42-1" class="td11">
Signaling by FGFR1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-42-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-42-3&quot;  </code></pre>
class=“td11”&gt; 22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-42-4" class="td11">
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-43-1" class="td11">
Signaling by FGFR2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-43-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-43-3&quot;  </code></pre>
class=“td11”&gt; 22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-43-4" class="td11">
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-96-44-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-96-44-1"  
class="td11">
<pre><code>                                                         &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-45-1" class="td11">
<span class="cmbx-12">Intersection of SLIPT and siRNA screen (450 genes) </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-45-2" class="td11">
Pathway Size
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-45-3" class="td11">
Genes Identified
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-45-4" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-46-1" class="td11">
HS-GAG degradation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-46-2" class="td11">
<pre><code>  21         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-46-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-46-4" class="td11">
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-47-1" class="td11">
Retinoid metabolism and transport
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-47-2" class="td11">
<pre><code>  39         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-47-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-47-4" class="td11">
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-48-1" class="td11">
Platelet activation, signaling and aggregation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-48-2" class="td11">
<pre><code> 186        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-48-3&quot;  </code></pre>
class=“td11”&gt; 13
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-48-4" class="td11">
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-49-1" class="td11">
Signaling by NOTCH4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-49-2" class="td11">
<pre><code>  11         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-49-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-49-4" class="td11">
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-50-1" class="td11">
<span class="cmmi-12">G</span><sub><span class="cmmi-8">αs</span> </sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-50-2" class="td11">
<pre><code> 100        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-50-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-50-4" class="td11">
5<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-51-1" class="td11">
Defective EXT2 causes exostoses 2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-51-2" class="td11">
<pre><code>  12         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-51-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-51-4" class="td11">
5<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-52-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-52-1" class="td11">
Defective EXT1 causes exostoses 1, TRPS2 and CHDS
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-52-2" class="td11">
<pre><code>  12         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-52-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-52-4" class="td11">
5<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-53-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-53-1" class="td11">
Class A/1 (Rhodopsin-like receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-53-2" class="td11">
<pre><code> 289        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-53-3&quot;  </code></pre>
class=“td11”&gt; 18
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-53-4" class="td11">
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-54-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-54-1" class="td11">
Signaling by PDGF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-54-2" class="td11">
<pre><code> 173        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-54-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-54-4" class="td11">
2<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-55-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-55-1" class="td11">
Circadian Clock
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-55-2" class="td11">
<pre><code>  34         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-55-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-55-4" class="td11">
2<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-56-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-56-1" class="td11">
Signaling by ERBB4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-56-2" class="td11">
<pre><code> 139        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-56-3&quot;  </code></pre>
class=“td11”&gt; 9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-56-4" class="td11">
4<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-57-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-57-1" class="td11">
Role of LAT2/NTAL/LAB on calcium mobilization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-57-2" class="td11">
<pre><code>  99         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-57-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-57-4" class="td11">
4<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-58-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-58-1" class="td11">
Peptide ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-58-2" class="td11">
<pre><code> 181        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-58-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-58-4" class="td11">
4<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-59-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-59-1" class="td11">
Defective B4GALT7 causes EDS, progeroid type
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-59-2" class="td11">
<pre><code>  19         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-59-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-59-4" class="td11">
4<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-60-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-60-1" class="td11">
Defective B3GAT3 causes JDSSDHD
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-60-2" class="td11">
<pre><code>  19         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-60-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-60-4" class="td11">
4<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-61-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-61-1" class="td11">
Signaling by NOTCH
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-61-2" class="td11">
<pre><code>  80         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-61-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-61-4" class="td11">
4<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-62-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-62-1" class="td11">
<span class="cmmi-12">G</span><sub><span class="cmmi-8">αq</span> </sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-62-2" class="td11">
<pre><code> 164        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-62-3&quot;  </code></pre>
class=“td11”&gt; 10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-62-4" class="td11">
5<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-63-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-63-1" class="td11">
Response to elevated platelet cytosolic Ca<sup>2ˆ+</sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-63-2" class="td11">
<pre><code>  84         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-63-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-63-4" class="td11">
7<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-64-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-64-1" class="td11">
Signaling by ERBB2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-64-2" class="td11">
<pre><code> 148        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-64-3&quot;  </code></pre>
class=“td11”&gt; 9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-64-4" class="td11">
7<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-96-65-">
<td style="white-space:nowrap; text-align:left;" id="TBL-96-65-1" class="td11">
Signaling by SCF-KIT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-65-2" class="td11">
<pre><code> 129        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-96-65-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-96-65-4" class="td11">
8<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-96-66-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-96-66-1"  
class="td11">
<pre><code>                                                         &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                         &lt;/div&gt;
                                                                                
                                                                                </code></pre>
</div>
<hr class="endfloat" />
<a 
 id="x1-207003r276"></a>
<h4 class="subsectionHead">
<span class="titlemark">D.3.1 </span> <a 
 id="x1-2080001"></a>Resampling Analysis
</h4>
<!--l. 622-->
<p class="indent">
<a 
 id="x1-208001r5"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table D.5: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT</span></p>
</div>
<!--tex4ht:label?: x1-208001r5 -->
<div class="tabular">
<table id="TBL-97" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-97-1g">
<col 
id="TBL-97-1" />
<col 
id="TBL-97-2" />
<col 
id="TBL-97-3" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-97-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-1-1" class="td11">
<span class="cmbx-12">Reactome Pathway </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-1-2" class="td11">
Over-representation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-1-3" class="td11">
Permutation
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-2-1" class="td11">
<strong><span class="cmbx-12">Eukaryotic Translation Elongation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-2-2" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">128</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-2-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-3-1" class="td11">
<strong><span class="cmbx-12">Peptide chain elongation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-3-2" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">128</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-3-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-4-1" class="td11">
<strong><span class="cmbx-12">Eukaryotic Translation Termination</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-4-2" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">125</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-4-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-5-1" class="td11">
<strong><span class="cmbx-12">Viral mRNA Translation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-5-2" class="td11">
<pre><code>4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">124</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-5-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-6-1" class="td11">
<strong><span class="cmbx-12">Nonsense Mediated Decay independent of the Exon Junction Complex</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-6-2" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">123</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-6-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-7-1" class="td11">
<strong><span class="cmbx-12">Nonsense-Mediated Decay</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-7-2" class="td11">
<pre><code>8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">117</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-7-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-8-1" class="td11">
<strong><span class="cmbx-12">Nonsense Mediated Decay enhanced by the Exon Junction Complex</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-8-2" class="td11">
<pre><code>8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">117</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-8-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-9-1" class="td11">
<strong><span class="cmbx-12">Formation of a pool of free 40S subunits</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-9-2" class="td11">
<pre><code>2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">116</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-9-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-10-1" class="td11">
<strong><span class="cmbx-12">L13a-mediated translational silencing of Ceruloplasmin expression</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-10-2" class="td11">
<pre><code>2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">111</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-10-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-11-1" class="td11">
<strong><span class="cmbx-12">3’ -UTR-mediated translational regulation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-11-2" class="td11">
<pre><code>2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">111</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-11-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-12-1" class="td11">
<strong><span class="cmbx-12">GTP hydrolysis and joining of the 60S ribosomal subunit</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-12-2" class="td11">
<pre><code>9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">111</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-12-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-13-1" class="td11">
<strong><span class="cmbx-12">SRP-dependent cotranslational protein targeting to membrane</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-13-2" class="td11">
<pre><code>4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">108</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-13-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-14-1" class="td11">
<strong><span class="cmbx-12">Eukaryotic Translation Initiation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-14-2" class="td11">
<pre><code>4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">106</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-14-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-15-1" class="td11">
<strong><span class="cmbx-12">Cap-dependent Translation Initiation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-15-2" class="td11">
<pre><code>4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">106</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-15-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-16-1" class="td11">
<strong><span class="cmbx-12">Influenza Viral RNA Transcription and Replication</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-16-2" class="td11">
<pre><code>8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">103</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-16-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-17-1" class="td11">
<strong><span class="cmbx-12">Influenza Infection</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-17-2" class="td11">
<pre><code>2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">102</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-17-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-18-1" class="td11">
<strong><span class="cmbx-12">Translation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-18-2" class="td11">
<pre><code>6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">101</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-18-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-19-1" class="td11">
<strong><span class="cmbx-12">Influenza Life Cycle</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-19-2" class="td11">
<pre><code>2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">100</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-19-3" class="td11">
<span class="cmmi-12">&lt; </span>7<span class="cmmi-12">.</span>035 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-20-1" class="td11">
<strong><span class="cmbx-12">Disease</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-20-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-20-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>013347
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-21-1" class="td11">
<strong><span class="cmbx-12">GPCR downstream signaling</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-21-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-21-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>095478
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-22-1" class="td11">
Hemostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-22-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">78</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-22-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2671
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-23-1" class="td11">
Signaling by GPCR
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-23-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">73</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-23-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>44939
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-24-1" class="td11">
<span class="cmti-12">Extracellular matrix organization </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-24-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">67</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-24-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>054008
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-25-1" class="td11">
Metabolism of proteins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-25-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">66</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-25-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9607
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-26-1" class="td11">
Signal Transduction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-26-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">66</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-26-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>48184
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-27-1" class="td11">
Developmental Biology
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-27-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">66</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-27-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>54075
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-28-1" class="td11">
Innate Immune System
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-28-2" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">66</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-28-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9589
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-29-1" class="td11">
Infectious disease
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-29-2" class="td11">
<pre><code> 9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">66</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-29-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>21075
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-30-1" class="td11">
Signalling by NGF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-30-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">62</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-30-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>43356
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-97-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-97-31-1" class="td11">
Immune System
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-31-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">62</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-97-31-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>23052
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-97-32-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-97-32-1"  
class="td11">
<pre><code>                                                                  &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 741-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Over-representation (hypergeometric test) and Permutation p-values adjusted for multiple tests across pathways (FDR). Significant</span> <span class="cmr-10x-x-109">pathways are marked in bold (FDR </span><span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05) and italics (FDR </span><span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">1).</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 745-->
<p class="indent">
<a 
 id="x1-208002r6"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table D.6: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT and siRNA primary screen</span></p>
</div>
<!--tex4ht:label?: x1-208002r6 -->
<div class="tabular">
<table id="TBL-98" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-98-1g">
<col 
id="TBL-98-1" />
<col 
id="TBL-98-2" />
<col 
id="TBL-98-3" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-98-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-1-1" class="td11">
<span class="cmbx-12">Reactome Pathway </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-1-2" class="td11">
Over-representation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-1-3" class="td11">
Permutation
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-2-1" class="td11">
Visual phototransduction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-2-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-2-3" class="td11">
0<span class="cmmi-12">.</span>86279
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-3-1" class="td11">
<strong><span class="cmbx-12">G</span><sub><span class="cmmi-8">αs</span> </sub> <span class="cmbx-12">signalling events</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-3-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-3-3" class="td11">
0<span class="cmmi-12">.</span>023066
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-4-1" class="td11">
Retinoid metabolism and transport
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-4-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-4-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>299
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-5-1" class="td11">
Acyl chain remodelling of PS
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-5-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-5-3" class="td11">
0<span class="cmmi-12">.</span>42584
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-6-1" class="td11">
Transcriptional regulation of white adipocyte differentiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-6-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-6-3" class="td11">
0<span class="cmmi-12">.</span>53928
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-7-1" class="td11">
Chemokine receptors bind chemokines
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-7-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-7-3" class="td11">
0<span class="cmmi-12">.</span>95259
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-8-1" class="td11">
<span class="cmti-12">Signaling by NOTCH4 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-8-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-8-3" class="td11">
0<span class="cmmi-12">.</span>079229
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-9-1" class="td11">
Defective EXT2 causes exostoses 2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-9-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-9-3" class="td11">
0<span class="cmmi-12">.</span>22292
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-10-1" class="td11">
Defective EXT1 causes exostoses 1, TRPS2 and CHDS
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-10-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-10-3" class="td11">
0<span class="cmmi-12">.</span>22292
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-11-1" class="td11">
Platelet activation, signaling and aggregation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-11-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-11-3" class="td11">
0<span class="cmmi-12">.</span>48853
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-12-1" class="td11">
Serotonin receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-12-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-12-3" class="td11">
0<span class="cmmi-12">.</span>34596
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-13-1" class="td11">
Nicotinamide salvaging
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-13-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-13-3" class="td11">
0<span class="cmmi-12">.</span>70881
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-14-1" class="td11">
Phase 1 - Functionalization of compounds
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-14-2" class="td11">
<pre><code>  2 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-14-3" class="td11">
0<span class="cmmi-12">.</span>31142
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-15-1" class="td11">
Amine ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-15-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-15-3" class="td11">
0<span class="cmmi-12">.</span>34934
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-16-1" class="td11">
Acyl chain remodelling of PE
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-16-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-16-3" class="td11">
0<span class="cmmi-12">.</span>42615
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-17-1" class="td11">
Signaling by GPCR
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-17-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-17-3" class="td11">
0<span class="cmmi-12">.</span>93888
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-18-1" class="td11">
<strong><span class="cmbx-12">Molecules associated with elastic fibres</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-18-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-18-3" class="td11">
0<span class="cmmi-12">.</span>017982
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-19-1" class="td11">
DAP12 interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-19-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-19-3" class="td11">
0<span class="cmmi-12">.</span>71983
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-20-1" class="td11">
Beta defensins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-20-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-20-3" class="td11">
0<span class="cmmi-12">.</span>91458
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-21-1" class="td11">
Cytochrome P<sub><span class="cmr-8">450</span></sub> - arranged by substrate type
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-21-2" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-21-3" class="td11">
0<span class="cmmi-12">.</span>83493
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-22-1" class="td11">
GPCR ligand binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-22-2" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-22-3" class="td11">
0<span class="cmmi-12">.</span>95258
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-23-1" class="td11">
Acyl chain remodelling of PC
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-23-2" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-23-3" class="td11">
0<span class="cmmi-12">.</span>42584
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-24-1" class="td11">
Response to elevated platelet cytosolic Ca<sup>2ˆ+</sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-24-2" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-24-3" class="td11">
0<span class="cmmi-12">.</span>54046
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-25-1" class="td11">
<strong><span class="cmbx-12">Arachidonic acid metabolism</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-25-2" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-25-3" class="td11">
0<span class="cmmi-12">.</span>026696
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-26-1" class="td11">
Defective B4GALT7 causes EDS, progeroid type
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-26-2" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-26-3" class="td11">
0<span class="cmmi-12">.</span>24921
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-27-1" class="td11">
Defective B3GAT3 causes JDSSDHD
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-27-2" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-27-3" class="td11">
0<span class="cmmi-12">.</span>24921
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-28-1" class="td11">
Hydrolysis of LPC
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-28-2" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-28-3" class="td11">
0<span class="cmmi-12">.</span>80663
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-29-1" class="td11">
<strong><span class="cmbx-12">Elastic fibre formation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-29-2" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-29-3" class="td11">
0<span class="cmmi-12">.</span>0058768
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-30-1" class="td11">
<strong> <span class="cmbx-12">HS-GAG degradation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-30-2" class="td11">
<pre><code> 9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-30-3" class="td11">
0<span class="cmmi-12">.</span>0083179
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-31-1" class="td11">
<span class="cmti-12">Bile acid and bile salt metabolism </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-31-2" class="td11">
<pre><code> 9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-31-3" class="td11">
0<span class="cmmi-12">.</span>079905
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-32-1" class="td11">
Netrin-1 signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-32-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00011
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-32-3" class="td11">
0<span class="cmmi-12">.</span>92216
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-33-1" class="td11">
<strong><span class="cmbx-12">Integration of energy metabolism</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-33-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00011
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-33-3" class="td11">
0<span class="cmmi-12">.</span>011152
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-34-1" class="td11">
Dectin-2 family
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-34-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00012
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-34-3" class="td11">
0<span class="cmmi-12">.</span>10385
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-35-1" class="td11">
Platelet sensitization by LDL
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-35-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00012
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-35-3" class="td11">
0<span class="cmmi-12">.</span>34596
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-36-1" class="td11">
DAP12 signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-36-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00012
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-36-3" class="td11">
0<span class="cmmi-12">.</span>62787
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-37-1" class="td11">
Defensins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-37-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00012
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-37-3" class="td11">
0<span class="cmmi-12">.</span>77542
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-38-1" class="td11">
GPCR downstream signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-38-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00012
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-38-3" class="td11">
0<span class="cmmi-12">.</span>79454
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-39-1" class="td11">
<span class="cmti-12">Diseases associated with glycosaminoglycan metabolism </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-39-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-39-3" class="td11">
0<span class="cmmi-12">.</span>065927
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-40-1" class="td11">
<span class="cmti-12">Diseases of glycosylation </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-40-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-40-3" class="td11">
0<span class="cmmi-12">.</span>065927
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-41-1" class="td11">
Signaling by Retinoic Acid
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-41-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-41-3" class="td11">
0<span class="cmmi-12">.</span>22292
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-42-1" class="td11">
Signaling by Leptin
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-42-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-42-3" class="td11">
0<span class="cmmi-12">.</span>34596
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-43-1" class="td11">
Signaling by SCF-KIT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-43-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-43-3" class="td11">
0<span class="cmmi-12">.</span>70881
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-44-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-44-1" class="td11">
Opioid Signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-44-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-44-3" class="td11">
0<span class="cmmi-12">.</span>96053
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-45-1" class="td11">
Signaling by NOTCH
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-45-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00015
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-45-3" class="td11">
0<span class="cmmi-12">.</span>26884
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-46-1" class="td11">
Platelet homeostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-46-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00015
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-46-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4878
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-47-1" class="td11">
Signaling by NOTCH1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-47-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00016
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-47-3" class="td11">
0<span class="cmmi-12">.</span>13043
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-48-1" class="td11">
Class B/2 (Secretin family receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-48-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00016
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-48-3" class="td11">
0<span class="cmmi-12">.</span>13994
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-49-1" class="td11">
<span class="cmti-12">Diseases of Immune System </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-49-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0002
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-49-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0795
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-50-1" class="td11">
<span class="cmti-12">Diseases associated with the TLR signaling cascade </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-50-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0002
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-50-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0795
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-98-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-98-51-1" class="td11">
A tetrasaccharide linker sequence is required for GAG synthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-51-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0002
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-98-51-3" class="td11">
0<span class="cmmi-12">.</span>42615
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-98-52-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-98-52-1"  
class="td11">
<pre><code>                                                   &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 862-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Over-representation (hypergeometric test) and Permutation p-values adjusted for multiple tests across pathways</span> <span class="cmr-10x-x-109">(FDR). Significant pathways are marked in bold (FDR </span><span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05) and italics (FDR </span><span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">1).</span></p>
</p>
</div>
<hr class="endfloat" />
<a 
 id="x1-208003r313"></a>
<h3 class="sectionHead">
<span class="titlemark">D.4 </span> <a 
 id="x1-2090004"></a>Compare SLIPT genes
</h3>
<!--l. 906-->
<p class="noindent">
<p>The mutation synthetic lethal partners with <span class="cmti-12">CDH1 </span>were also compared to siRNA primary screen data (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), by correlation and siRNA viability as described in sections <a 
href="#x1-1240001">4.2.1.1<!--tex4ht:ref: chapt3:compare_correlation --></a> and <a 
href="#x1-1250002">4.2.1.2<!--tex4ht:ref: chapt3:compare_viability --></a>.</p>
</p>
<!--l. 908-->
<p class="indent">
<a 
 id="x1-209001r3"></a><a 
 id="x1-209002r4"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 910-->
<p class="noindent">
</p>
<!--l. 911-->
<p class="noindent">
<img 
src="mtSLIPT_siRNA_vs_Correlation_with_CDH1.png" alt="PIC"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure D.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">Compare mtSLIPT and siRNA genes with correlation.</span></strong> <span class="cmr-10x-x-109">The</span> <span class="cmr-10x-x-109">mtSLIPT p-values were compared against Pearson’s correlation of expression with </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">.</span> <span class="cmr-10x-x-109">Genes detected by SLIPT or siRNA are coloured according to the legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-209001r3 -->
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 923-->
<p class="noindent">
</p>
<!--l. 924-->
<p class="noindent">
<img 
src="vioplotx_mtSLIPT_siRNA_vs_CDH1_Correlation_with_CDH1.png" alt="pict"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure D.4: </span><span class="content"><strong><span class="cmbx-10x-x-109">Compare mtSLIPT and siRNA genes with correlation.</span></strong> <span class="cmr-10x-x-109">Genes</span> <span class="cmr-10x-x-109">detected by mtSLIPT against </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutation and siRNA screening were compared</span> <span class="cmr-10x-x-109">against Pearson’s correlation of expression with </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">. There were no differences in</span> <span class="cmr-10x-x-109">correlation between the gene groups. </span></span></p>
</div>
<!--tex4ht:label?: x1-209002r4 -->
</div>
<hr class="endfloat" />
<!--l. 934-->
<p class="indent">
<a 
 id="x1-209003r5"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 936-->
<p class="noindent">
</p>
<!--l. 937-->
<p class="noindent">
<img 
src="vioplotx_mtSLIPT_siRNA_vs_Viability_Ratio_with_CDH1.png" alt="pict"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure D.5: </span><span class="content"><strong><span class="cmbx-10x-x-109">Compare mtSLIPT and siRNA genes with siRNA viability.</span></strong> <span class="cmr-10x-x-109">Genes</span> <span class="cmr-10x-x-109">detected as candidate synthetic lethal partners by mtSLIPT (in TCGA breast cancer)</span> <span class="cmr-10x-x-109">expression analysis against </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutation and experimental screening (with siRNA)</span> <span class="cmr-10x-x-109">were compared against the viability ratio of </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutant and wildtype cells in the</span> <span class="cmr-10x-x-109">primary siRNA screen. There were clear no differences in viability between genes detected</span> <span class="cmr-10x-x-109">by mtSLIPT and those not with the differences being primarily due to viability thresholds</span> <span class="cmr-10x-x-109">being used to detect synthetic lethality by </span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">Telford </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">2015</span></a><span class="cmr-10x-x-109">). </span></span></p>
</div>
<!--tex4ht:label?: x1-209003r5 -->
</div>
<hr class="endfloat" />
<a 
 id="x1-209004r319"></a>
<h3 class="sectionHead">
<span class="titlemark">D.5 </span> <a 
 id="x1-2100005"></a>Metagene Analysis
</h3>
<!--l. 951-->
<p class="noindent">
<p>Metagene analysis was also performed for synthetic lethal candidates for <span class="cmti-12">CDH1 </span>mutation. These are described and compared to expression analysis in Section <a 
href="#x1-1330004">4.3.4<!--tex4ht:ref: chapt3:metagene_SL --></a>.</p>
</p>
<!--l. 953-->
<p class="indent">
<a 
 id="x1-210001r7"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table D.7: </span><span class="content">Candidate synthetic lethal metagenes against <span class="cmti-12">CDH1 </span>from mtSLIPT</span></p>
</div>
<!--tex4ht:label?: x1-210001r7 -->
<div class="tabular">
<table id="TBL-99" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-99-1g">
<col 
id="TBL-99-1" />
<col 
id="TBL-99-2" />
<col 
id="TBL-99-3" />
<col 
id="TBL-99-4" />
<col 
id="TBL-99-5" />
<col 
id="TBL-99-6" />
<col 
id="TBL-99-7" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-99-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-1-1" class="td11">
<span class="cmbx-12">Pathway </span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-1-2" class="td11">
ID
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-1-3" class="td11">
Observed
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-1-4" class="td11">
Expected
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-1-5" class="td11">
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-1-6" class="td11">
p-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-1-7" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-2-1" class="td11">
Neurotoxicity of clostridium toxins
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-2-2" class="td11">
168799
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-2-3" class="td11">
8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-2-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-2-5" class="td11">
79.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-2-6" class="td11">
5<span class="cmmi-12">.</span>71 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-2-7" class="td11">
3<span class="cmmi-12">.</span>14 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-3-1" class="td11">
Aquaporin-mediated transport
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-3-2" class="td11">
445717
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-3-3" class="td11">
8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-3-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-3-5" class="td11">
76.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-3-6" class="td11">
2<span class="cmmi-12">.</span>73 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-3-7" class="td11">
9<span class="cmmi-12">.</span>01 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-4-1" class="td11">
Toxicity of botulinum toxin type G (BoNT/G)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-4-2" class="td11">
5250989
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-4-3" class="td11">
8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-4-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-4-5" class="td11">
76.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-4-6" class="td11">
2<span class="cmmi-12">.</span>73 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-4-7" class="td11">
9<span class="cmmi-12">.</span>01 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-5-1" class="td11">
ABC-family proteins mediated transport
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-5-2" class="td11">
382556
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-5-3" class="td11">
10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-5-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-5-5" class="td11">
68.2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-5-6" class="td11">
1<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-5-7" class="td11">
1<span class="cmmi-12">.</span>86 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-6-1" class="td11">
G<sub><span class="cmmi-8">αz</span> </sub> signalling events
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-6-2" class="td11">
418597
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-6-3" class="td11">
10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-6-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-6-5" class="td11">
59.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-6-6" class="td11">
9<span class="cmmi-12">.</span>97 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-6-7" class="td11">
5<span class="cmmi-12">.</span>48 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-7-1" class="td11">
Regulation of IGF transport and uptake by IGFBPs
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-7-2" class="td11">
381426
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-7-3" class="td11">
9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-7-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-7-5" class="td11">
56.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-7-6" class="td11">
5<span class="cmmi-12">.</span>88 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-7-7" class="td11">
2<span class="cmmi-12">.</span>11 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-8-1" class="td11">
GP1b-IX-V activation signalling
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-8-2" class="td11">
430116
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-8-3" class="td11">
8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-8-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-8-5" class="td11">
55.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-8-6" class="td11">
8<span class="cmmi-12">.</span>20 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-8-7" class="td11">
2<span class="cmmi-12">.</span>76 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-9-1" class="td11">
GABA receptor activation
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-9-2" class="td11">
977443
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-9-3" class="td11">
12
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-9-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-9-5" class="td11">
55.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-9-6" class="td11">
1<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-9-7" class="td11">
3<span class="cmmi-12">.</span>26 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-10-1" class="td11">
Vasopressin regulates renal water homeostasis via Aquaporins
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-10-2" class="td11">
432040
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-10-3" class="td11">
9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-10-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-10-5" class="td11">
54.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-10-6" class="td11">
1<span class="cmmi-12">.</span>77 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-10-7" class="td11">
4<span class="cmmi-12">.</span>88 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-11-1" class="td11">
Toxicity of botulinum toxin type D (BoNT/D)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-11-2" class="td11">
5250955
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-11-3" class="td11">
14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-11-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-11-5" class="td11">
53.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-11-6" class="td11">
2<span class="cmmi-12">.</span>54 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-11-7" class="td11">
6<span class="cmmi-12">.</span>64 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-12-1" class="td11">
Toxicity of botulinum toxin type F (BoNT/F)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-12-2" class="td11">
5250981
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-12-3" class="td11">
14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-12-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-12-5" class="td11">
53.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-12-6" class="td11">
2<span class="cmmi-12">.</span>54 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-12-7" class="td11">
6<span class="cmmi-12">.</span>64 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-13-1" class="td11">
STAT6-mediated induction of chemokines
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-13-2" class="td11">
3249367
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-13-3" class="td11">
16
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-13-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-13-5" class="td11">
52.2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-13-6" class="td11">
4<span class="cmmi-12">.</span>72 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-13-7" class="td11">
1<span class="cmmi-12">.</span>13 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-14-1" class="td11">
Toxicity of botulinum toxin type B (BoNT/B)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-14-2" class="td11">
5250958
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-14-3" class="td11">
14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-14-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-14-5" class="td11">
50.8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-14-6" class="td11">
9<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-14-7" class="td11">
1<span class="cmmi-12">.</span>98 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-15-1" class="td11">
S6K1 signalling
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-15-2" class="td11">
165720
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-15-3" class="td11">
12
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-15-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-15-5" class="td11">
50.2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-15-6" class="td11">
1<span class="cmmi-12">.</span>24 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-15-7" class="td11">
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-16-1" class="td11">
G<sub><span class="cmmi-8">αs</span> </sub> signalling events
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-16-2" class="td11">
418555
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-16-3" class="td11">
11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-16-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-16-5" class="td11">
49.2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-16-6" class="td11">
2<span class="cmmi-12">.</span>08 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-16-7" class="td11">
3<span class="cmmi-12">.</span>85 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-17-1" class="td11">
RHO GTPases activate CIT
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-17-2" class="td11">
5625900
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-17-3" class="td11">
14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-17-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-17-5" class="td11">
48.2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-17-6" class="td11">
3<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-17-7" class="td11">
5<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-18-1" class="td11">
NADE modulates death signalling
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-18-2" class="td11">
205025
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-18-3" class="td11">
15
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-18-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-18-5" class="td11">
47.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-18-6" class="td11">
5<span class="cmmi-12">.</span>00 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-18-7" class="td11">
8<span class="cmmi-12">.</span>32 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-19-1" class="td11">
Keratan sulfate degradation
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-19-2" class="td11">
2022857
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-19-3" class="td11">
10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-19-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-19-5" class="td11">
46.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-19-6" class="td11">
7<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-19-7" class="td11">
1<span class="cmmi-12">.</span>15 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-20-1" class="td11">
Signaling by Retinoic Acid
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-20-2" class="td11">
5362517
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-20-3" class="td11">
10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-20-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-20-5" class="td11">
46.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-20-6" class="td11">
7<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-20-7" class="td11">
1<span class="cmmi-12">.</span>15 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-21-1" class="td11">
Adenylate cyclase inhibitory pathway
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-21-2" class="td11">
170670
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-21-3" class="td11">
14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-21-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-21-5" class="td11">
45.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-21-6" class="td11">
1<span class="cmmi-12">.</span>11 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-21-7" class="td11">
1<span class="cmmi-12">.</span>59 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-22-1" class="td11">
Inhibition of adenylate cyclase pathway
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-22-2" class="td11">
997269
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-22-3" class="td11">
14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-22-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-22-5" class="td11">
45.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-22-6" class="td11">
1<span class="cmmi-12">.</span>11 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-22-7" class="td11">
1<span class="cmmi-12">.</span>59 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-23-1" class="td11">
Fatty acids
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-23-2" class="td11">
211935
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-23-3" class="td11">
6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-23-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-23-5" class="td11">
45.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-23-6" class="td11">
1<span class="cmmi-12">.</span>21 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-23-7" class="td11">
1<span class="cmmi-12">.</span>72 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-24-1" class="td11">
Ionotropic activity of Kainate Receptors
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-24-2" class="td11">
451306
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-24-3" class="td11">
13
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-24-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-24-5" class="td11">
44.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-24-6" class="td11">
2<span class="cmmi-12">.</span>03 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-24-7" class="td11">
2<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-25-1" class="td11">
Activation of Ca-permeable Kainate Receptor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-25-2" class="td11">
451308
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-25-3" class="td11">
13
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-25-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-25-5" class="td11">
44.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-25-6" class="td11">
2<span class="cmmi-12">.</span>03 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-25-7" class="td11">
2<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-99-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-99-26-1" class="td11">
RA biosynthesis pathway
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-99-26-2" class="td11">
5365859
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-26-3" class="td11">
13
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-26-4" class="td11">
36.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-26-5" class="td11">
44.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-26-6" class="td11">
2<span class="cmmi-12">.</span>03 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-99-26-7" class="td11">
2<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-99-27-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-99-27-1"  
class="td11">
<pre><code>                                                 &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 1020-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Strongest candidate SL partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">by mtSLIPT with observed and expected mutant samples with low expression of partner metagenes</span></p>
</p>
</div>
<hr class="endfloat" />
<a 
 id="x1-210002r323"></a>
<h3 class="sectionHead">
<span class="titlemark">D.6 </span> <a 
 id="x1-2110006"></a>Mutation Variation
</h3>
<!--l. 1026-->
<p class="noindent">
Mutations have different effects as shown by the following examples in cancer genes. <a 
 id="x1-211001r316"></a>
</p>
<h4 class="subsectionHead">
<span class="titlemark">D.6.1 </span> <a 
 id="x1-2120001"></a>Mutation Frequency
</h4>
<!--l. 1034-->
<p class="indent">
<a 
 id="x1-212005r6"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 1036-->
<p class="noindent">
<a 
 id="x1-212001r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Mutation_locus_PIK3CA.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmti-10">PI3KCA</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-212002r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Mutation_locus_PIK3R1.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmti-10">PI3KR1</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-212003r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Mutation_locus_CDH1.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmti-10">CDH1</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-212004r4"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="Mutation_locus_TP53.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(d)</span> <span class="cmti-10">TP53</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure D.6: </span><span class="content"><strong><span class="cmbx-10x-x-109">Somatic mutation locus.</span></strong> <span class="cmr-10x-x-109">Mutation frequency at each locus in TCGA</span> <span class="cmr-10x-x-109">breast cancer. </span><span class="cmti-10x-x-109">PIK3CA </span><span class="cmr-10x-x-109">shows clear recurrent E545K and H1047R oncogene mutations</span> <span class="cmr-10x-x-109">consistent with it being an oncogene. </span><span class="cmti-10x-x-109">PIK3R1 </span><span class="cmr-10x-x-109">and </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">are tumour suppressors with</span> <span class="cmr-10x-x-109">inactivating mutations distributed throughout the gene, whereas </span><span class="cmti-10x-x-109">TP53 </span><span class="cmr-10x-x-109">exhibits both of</span> <span class="cmr-10x-x-109">these properties and a very high mutation frequency compared to other genes. </span></span></p>
</div>
<!--tex4ht:label?: x1-212005r6 -->
</div>
<hr class="endfloat" />
<a 
 id="x1-212006r326"></a>
<h4 class="subsectionHead">
<span class="titlemark">D.6.2 </span> <a 
 id="x1-2130002"></a>PI3K Mutation Expression
</h4>
<!--l. 1064-->
<p class="indent">
<a 
 id="x1-213005r7"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 1066-->
<p class="noindent">
<a 
 id="x1-213001r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="metagene(ranked)_vioplotx_Mutation_PIK3CA2014_PIK3CA.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmti-10">PIK3CA</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-213002r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="metagene(ranked)_vioplotx_Mutation_PIK3CA2014_PIK3CA_PIK3R1.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmti-10">PIK3CA</span> <span class="cmr-10">or</span> <span class="cmti-10">PIK3R1</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-213003r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="metagene(ranked)_vioplotx_Mutation_PIK3CA2014_CDH1.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmti-10">CDH1</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-213004r4"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="metagene(ranked)_vioplotx_Mutation_PIK3CA2014_TP53.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(d)</span> <span class="cmti-10">TP53</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure D.7: </span><span class="content"><strong><span class="cmbx-10x-x-109">Somatic mutation against PIK3CA metagene.</span></strong> <span class="cmr-10x-x-109">Mutations in</span> <span class="cmti-10x-x-109">PIK3CA</span><span class="cmr-10x-x-109">, </span><span class="cmti-10x-x-109">PIK3R1</span><span class="cmr-10x-x-109">, </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">, and </span><span class="cmti-10x-x-109">TP53 </span><span class="cmr-10x-x-109">were examined in TCGA breast cancer for their</span> <span class="cmr-10x-x-109">effect on the PIK3CA (</span><a 
href="#XGatza2014"><span class="cmr-10x-x-109">Gatza </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109">,</span><span class="cmr-10x-x-109"> </span><a 
href="#XGatza2014"><span class="cmr-10x-x-109">2014</span></a><span class="cmr-10x-x-109">) pathway metagene. The tumour suppressors</span> <span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">and </span><span class="cmti-10x-x-109">TP53 </span><span class="cmr-10x-x-109">showed an increase and decrease in the metagene respectively, whereas</span> <span class="cmti-10x-x-109">PIK3CA </span><span class="cmr-10x-x-109">and </span><span class="cmti-10x-x-109">PIK3R1 </span><span class="cmr-10x-x-109">mutations weaker evidence of decrease in metagene levels. </span></span></p>
</div>
<!--tex4ht:label?: x1-213005r7 -->
</div>
<hr class="endfloat" />
<!--l. 1089-->
<p class="indent">
<a 
 id="x1-213010r8"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 1091-->
<p class="noindent">
<a 
 id="x1-213006r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="protein_vioplotx_Mutation_PI3K_PIK3CA.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmti-10">PIK3CA</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-213007r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="protein_vioplotx_Mutation_PI3K_PIK3CA_PIK3R1.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmti-10">PIK3CA</span> <span class="cmr-10">or</span> <span class="cmti-10">PIK3R1</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-213008r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="protein_vioplotx_Mutation_PI3K_CDH1.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmti-10">CDH1</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-213009r4"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="protein_vioplotx_Mutation_PI3K_TP53.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(d)</span> <span class="cmti-10">TP53</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure D.8: </span><span class="content"><strong><span class="cmbx-10x-x-109">Somatic mutation against PI3K protein.</span></strong> <span class="cmr-10x-x-109">Mutations in </span><span class="cmti-10x-x-109">PIK3CA</span><span class="cmr-10x-x-109">,</span> <span class="cmti-10x-x-109">PIK3R1</span><span class="cmr-10x-x-109">, </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">, and </span><span class="cmti-10x-x-109">TP53 </span><span class="cmr-10x-x-109">were examined in TCGA breast cancer for their effect on the</span> <span class="cmr-10x-x-109">expression of the p110</span><span class="cmmi-10x-x-109">α </span><span class="cmr-10x-x-109">protein (encoded by </span><span class="cmti-10x-x-109">PIK3CA</span><span class="cmr-10x-x-109">). Protein levels were significantly</span> <span class="cmr-10x-x-109">elevated in samples with </span><span class="cmti-10x-x-109">PIK3CA </span><span class="cmr-10x-x-109">or </span><span class="cmti-10x-x-109">PIK3R1 </span><span class="cmr-10x-x-109">mutations and lower in samples with </span><span class="cmti-10x-x-109">TP53</span> <span class="cmr-10x-x-109">mutations. </span></span></p>
</div>
<!--tex4ht:label?: x1-213010r8 -->
</div>
<hr class="endfloat" />
<!--l. 1113-->
<p class="indent">
<a 
 id="x1-213015r9"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 1115-->
<p class="noindent">
<a 
 id="x1-213011r1"></a>
</p>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="protein_vioplotx_Mutation_AKT_PIK3CA.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(a)</span> <span class="cmti-10">PIK3CA</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-213012r2"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="protein_vioplotx_Mutation_AKT_PIK3CA_PIK3R1.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(b)</span> <span class="cmti-10">PIK3CA</span> <span class="cmr-10">or</span> <span class="cmti-10">PIK3R1</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-213013r3"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="protein_vioplotx_Mutation_AKT_CDH1.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(c)</span> <span class="cmti-10">CDH1</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
<a 
 id="x1-213014r4"></a>
<div class="subfigure">
<table>
<tr>
<td style="text-align:left">
<img 
src="protein_vioplotx_Mutation_AKT_TP53.png" alt="pict"  
 />
<div class="subfigcaption">
<p><span class="cmr-10">(d)</span> <span class="cmti-10">TP53</span> <span class="cmr-10">mutation</span></p>
</div>
</td>
</tr>
</table>
</div>
</div>
<br />
<div class="caption">
<p><span class="id">Figure D.9: </span><span class="content"><strong><span class="cmbx-10x-x-109">Somatic mutation against AKT protein.</span></strong> <span class="cmr-10x-x-109">Mutations in </span><span class="cmti-10x-x-109">PIK3CA</span><span class="cmr-10x-x-109">,</span> <span class="cmti-10x-x-109">PIK3R1</span><span class="cmr-10x-x-109">, </span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">, and </span><span class="cmti-10x-x-109">TP53 </span><span class="cmr-10x-x-109">were examined in TCGA breast cancer for their effect on the</span> <span class="cmr-10x-x-109">expression of the AKT protein (a downstream target of </span><span class="cmti-10x-x-109">PIK3CA</span><span class="cmr-10x-x-109">). Protein levels were</span> <span class="cmr-10x-x-109">not significantly different in samples mutations in any of these cancer genes. </span></span></p>
</div>
<!--tex4ht:label?: x1-213015r9 -->
</div>
<hr class="endfloat" />
<a 
 id="x1-213016r288"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix E</span><br /><a 
 id="x1-214000E"></a>Metagene Expression Profiles
</h2>
<!--l. 1142-->
<p class="indent">
<a 
 id="x1-214001r1"></a>
</p>
<hr class="float" />
<div class="float">
<span class="fbox">
<div class="minipage">
<div class="center">
<!--l. 1155-->
<p class="noindent">
</p>
<!--l. 1155-->
<p class="noindent">
<img 
src="CDH1_Heatmaps_Gatza2014(ranked)_Full_split_by_Subtype_and_CDH1_Stat_corr.png" alt="PIC"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure E.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Pathway metagene expression profiles.</span></strong> <span class="cmr-10x-x-109">Expression profiles for metagene signatures from</span> <a 
href="#XGatza2014"><span class="cmr-10x-x-109">Gatza </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XGatza2014"><span class="cmr-10x-x-109">2014</span></a><span class="cmr-10x-x-109">) in TCGA breast data, annotated for clinical factors and cancer gene mutations. </span></span></p>
</div>
<!--tex4ht:label?: x1-214001r1 -->
</div>
<p></span></p>
</div>
<hr class="endfloat" />
<!--l. 1158-->
<p class="indent">
<a 
 id="x1-214002r2"></a>
</p>
<hr class="float" />
<div class="float">
<span class="fbox">
<div class="minipage">
<div class="center">
<!--l. 1171-->
<p class="noindent">
</p>
<!--l. 1171-->
<p class="noindent">
<img 
src="CDH1_Heatmaps_Gatza2011(PI3K)_Full_Metagene_mgorder.png" alt="PIC"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure E.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Expression profiles for constituent genes of PI3K.</span></strong> <span class="cmr-10x-x-109">Expression profiles the genes</span> <span class="cmr-10x-x-109">contained in the PI3K gene signature from </span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">Gatza </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">2011</span></a><span class="cmr-10x-x-109">) in TCGA breast data, annotated for clinical</span> <span class="cmr-10x-x-109">factors and cancer gene mutations. Samples are separated by </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">expression status and sorted by the</span> <span class="cmr-10x-x-109">metagene. In both cases, the majority of genes were consistent with the direction of the PI3K metagene,</span> <span class="cmr-10x-x-109">although considerable proportion were inversely correlated with the metagene. Normal samples had low</span> <span class="cmr-10x-x-109">PI3K metagene expression and </span><span class="cmti-10x-x-109">TP53 </span><span class="cmr-10x-x-109">mutant samples had high PI3K expression. Although, oncogenic</span> <span class="cmti-10x-x-109">PIK3CA </span><span class="cmr-10x-x-109">and tumour suppressor </span><span class="cmti-10x-x-109">PIK3R1 </span><span class="cmr-10x-x-109">mutations across samples including those with low metagene</span> <span class="cmr-10x-x-109">response. </span></span></p>
</div>
<!--tex4ht:label?: x1-214002r2 -->
</div>
<p></span></p>
</div>
<hr class="endfloat" />
<!--l. 1174-->
<p class="indent">
<a 
 id="x1-214003r3"></a>
</p>
<hr class="float" />
<div class="float">
<span class="fbox">
<div class="minipage">
<div class="center">
<!--l. 1187-->
<p class="noindent">
</p>
<!--l. 1187-->
<p class="noindent">
<img 
src="CDH1_Heatmaps_Gatza2011(P53)_Full_Metagene_mgorder.png" alt="PIC"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure E.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">Expression profiles for p53 related genes.</span></strong> <span class="cmr-10x-x-109">Expression profiles the genes contained in</span> <span class="cmr-10x-x-109">the </span><span class="cmti-10x-x-109">TP53 </span><span class="cmr-10x-x-109">gene signature from </span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">Gatza </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">2011</span></a><span class="cmr-10x-x-109">) in TCGA breast data, annotated for clinical factors and</span> <span class="cmr-10x-x-109">cancer gene mutations. Samples are separated by </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">expression status and sorted by the metagene.</span> <span class="cmr-10x-x-109">In both cases, the majority of genes were consistent with the direction of the metagene, with few very</span> <span class="cmr-10x-x-109">exceptions. </span><span class="cmti-10x-x-109">TP53 </span><span class="cmr-10x-x-109">mutant samples had low metagene expression, consistent with loss of tumour suppressor</span> <span class="cmr-10x-x-109">functions, and were less likely to have </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">or </span><span class="cmti-10x-x-109">PIK3CA </span><span class="cmr-10x-x-109">mutations. </span></span></p>
</div>
<!--tex4ht:label?: x1-214003r3 -->
</div>
<p></span></p>
</div>
<hr class="endfloat" />
<!--l. 1190-->
<p class="indent">
<a 
 id="x1-214004r4"></a>
</p>
<hr class="float" />
<div class="float">
<span class="fbox">
<div class="minipage">
<div class="center">
<!--l. 1203-->
<p class="noindent">
</p>
<!--l. 1203-->
<p class="noindent">
<img 
src="CDH1_Heatmaps_Gatza2011(ER)_Full_Metagene_mgorder.png" alt="PIC"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure E.4: </span><span class="content"><strong><span class="cmbx-10x-x-109">Expression profiles for estrogen receptor related genes.</span></strong> <span class="cmr-10x-x-109">Expression profiles the genes</span> <span class="cmr-10x-x-109">contained in the estrogen receptor (ER) gene signature from </span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">Gatza </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XGatza2011"><span class="cmr-10x-x-109">2011</span></a><span class="cmr-10x-x-109">) in TCGA breast data,</span> <span class="cmr-10x-x-109">annotated for clinical factors and cancer gene mutations. Samples are separated by </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">expression status</span> <span class="cmr-10x-x-109">and sorted by the metagene. In both cases, the majority of genes were consistent with the direction of the</span> <span class="cmr-10x-x-109">metagene, with very few exceptions being inversely correlated. Estrogen receptor (by antibody staining)</span> <span class="cmr-10x-x-109">negative samples had low metagene expression, as expected. These were more likely to be ductal and basal</span> <span class="cmr-10x-x-109">subtypes, lacking </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">or </span><span class="cmti-10x-x-109">PIK3CA </span><span class="cmr-10x-x-109">mutations. </span></span></p>
</div>
<!--tex4ht:label?: x1-214004r4 -->
</div>
<p></span></p>
</div>
<hr class="endfloat" />
<!--l. 1207-->
<p class="indent">
<a 
 id="x1-214005r5"></a>
</p>
<hr class="float" />
<div class="float">
<span class="fbox">
<div class="minipage">
<div class="center">
<!--l. 1220-->
<p class="noindent">
</p>
<!--l. 1220-->
<p class="noindent">
<img 
src="CDH1_Heatmaps_Gatza2014(BRCA1)_Full_Metagene_mgorder.png" alt="PIC"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure E.5: </span><span class="content"><strong><span class="cmbx-10x-x-109">Expression profiles for BRCA related genes.</span></strong> <span class="cmr-10x-x-109">Expression profiles the genes contained</span> <span class="cmr-10x-x-109">in the gene signature related to </span><span class="cmti-10x-x-109">BRCA1 </span><span class="cmr-10x-x-109">and </span><span class="cmti-10x-x-109">BRCA2 </span><span class="cmr-10x-x-109">functions from </span><a 
href="#XGatza2014"><span class="cmr-10x-x-109">Gatza </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109"> (</span><a 
href="#XGatza2014"><span class="cmr-10x-x-109">2014</span></a><span class="cmr-10x-x-109">) in TCGA breast</span> <span class="cmr-10x-x-109">data, annotated for clinical factors and cancer gene mutations. Samples are separated by </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">expression</span> <span class="cmr-10x-x-109">status and sorted by the metagene. In both cases, the majority of genes were consistent with the direction</span> <span class="cmr-10x-x-109">of the metagene, with few very exceptions. </span><span class="cmti-10x-x-109">BRCA1 </span><span class="cmr-10x-x-109">and </span><span class="cmti-10x-x-109">BRCA2 </span><span class="cmr-10x-x-109">mutant samples had higher metagene</span> <span class="cmr-10x-x-109">expression than most samples for the ductal subtype, although this was not the case (for the lobular samples</span> <span class="cmr-10x-x-109">for which the metagene was lower). However, the metagene was higher for basal subtype and estrogen</span> <span class="cmr-10x-x-109">receptor negative samples. </span></span></p>
</div>
<!--tex4ht:label?: x1-214005r5 -->
</div>
<p></span></p>
</div>
<hr class="endfloat" />
<a 
 id="x1-214006r288"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix F</span><br /><a 
 id="x1-215000F"></a>Stomach Expression Analysis
</h2>
<!--l. 1228-->
<p class="noindent">
The following results are a replication of the TCGA results (in Chapter <a 
href="#x1-1170004">4<!--tex4ht:ref: chap:SLIPT --></a>) with stomach cancer data, using synthetic lethality (SLIPT) against <span class="cmti-12">CDH1 </span>mutation. <a 
 id="x1-215001r325"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">F.1 </span> <a 
 id="x1-2160001"></a>Synthetic Lethal Genes and Pathways
</h3>
<!--l. 1232-->
<p class="indent">
<a 
 id="x1-216001r1"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table F.1: </span><span class="content">Synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from SLIPT in stomach cancer</span></p>
</div>
<!--tex4ht:label?: x1-216001r1 -->
<div class="tabular">
<table id="TBL-100" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-100-1g">
<col 
id="TBL-100-1" />
<col 
id="TBL-100-2" />
<col 
id="TBL-100-3" />
<col 
id="TBL-100-4" />
<col 
id="TBL-100-5" />
<col 
id="TBL-100-6" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-100-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-1-1" class="td11">
<span class="cmbx-12">Gene </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-1-2" class="td11">
Observed
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-1-3" class="td11">
Expected
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-1-4" class="td11">
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-1-5" class="td11">
p-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-1-6" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-2-1" class="td11">
<span class="cmti-12">PRAF2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-2-2" class="td11">
<pre><code>17      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-2-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-2-4" class="td11">
121
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-2-5" class="td11">
3<span class="cmmi-12">.</span>54 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-2-6" class="td11">
1<span class="cmmi-12">.</span>45 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-3-1" class="td11">
<span class="cmti-12">EMP3 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-3-2" class="td11">
<pre><code>17      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-3-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-3-4" class="td11">
115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-3-5" class="td11">
5<span class="cmmi-12">.</span>06 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-3-6" class="td11">
1<span class="cmmi-12">.</span>48 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-4-1" class="td11">
<span class="cmti-12">PLEKHO1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-4-2" class="td11">
<pre><code>22      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-4-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-4-4" class="td11">
112
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-4-5" class="td11">
2<span class="cmmi-12">.</span>14 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">23</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-4-6" class="td11">
4<span class="cmmi-12">.</span>75 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-5-1" class="td11">
<span class="cmti-12">SELM </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-5-2" class="td11">
<pre><code>20      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-5-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-5-4" class="td11">
111
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-5-5" class="td11">
5<span class="cmmi-12">.</span>13 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">23</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-5-6" class="td11">
8<span class="cmmi-12">.</span>09 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-6-1" class="td11">
<span class="cmti-12">GYPC </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-6-2" class="td11">
<pre><code>20      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-6-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-6-4" class="td11">
110
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-6-5" class="td11">
5<span class="cmmi-12">.</span>77 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">23</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-6-6" class="td11">
8<span class="cmmi-12">.</span>45 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-7-1" class="td11">
<span class="cmti-12">COX7A1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-7-2" class="td11">
<pre><code>18      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-7-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-7-4" class="td11">
109
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-7-5" class="td11">
1<span class="cmmi-12">.</span>15 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-7-6" class="td11">
1<span class="cmmi-12">.</span>39 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-8-1" class="td11">
<span class="cmti-12">TNFSF12 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-8-2" class="td11">
<pre><code>20      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-8-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-8-4" class="td11">
106
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-8-5" class="td11">
4<span class="cmmi-12">.</span>06 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-8-6" class="td11">
4<span class="cmmi-12">.</span>38 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-9-1" class="td11">
<span class="cmti-12">SEPT4 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-9-2" class="td11">
<pre><code>17      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-9-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-9-4" class="td11">
106
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-9-5" class="td11">
6<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-9-6" class="td11">
5<span class="cmmi-12">.</span>91 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-10-1" class="td11">
<span class="cmti-12">LGALS1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-10-2" class="td11">
<pre><code>19      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-10-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-10-4" class="td11">
105
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-10-5" class="td11">
6<span class="cmmi-12">.</span>64 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-10-6" class="td11">
5<span class="cmmi-12">.</span>91 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-11-1" class="td11">
<span class="cmti-12">RARRES2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-11-2" class="td11">
<pre><code>27      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-11-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-11-4" class="td11">
105
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-11-5" class="td11">
8<span class="cmmi-12">.</span>02 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-11-6" class="td11">
6<span class="cmmi-12">.</span>85 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-12-1" class="td11">
<span class="cmti-12">VEGFB </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-12-2" class="td11">
<pre><code>16      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-12-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-12-4" class="td11">
104
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-12-5" class="td11">
1<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-12-6" class="td11">
9<span class="cmmi-12">.</span>74 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-13-1" class="td11">
<span class="cmti-12">PRR24 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-13-2" class="td11">
<pre><code>22      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-13-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-13-4" class="td11">
102
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-13-5" class="td11">
2<span class="cmmi-12">.</span>96 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-13-6" class="td11">
2<span class="cmmi-12">.</span>02 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-14-1" class="td11">
<span class="cmti-12">SYNC </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-14-2" class="td11">
<pre><code>19      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-14-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-14-4" class="td11">
102
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-14-5" class="td11">
3<span class="cmmi-12">.</span>73 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-14-6" class="td11">
2<span class="cmmi-12">.</span>39 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-15-1" class="td11">
<span class="cmti-12">MAGEH1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-15-2" class="td11">
<pre><code>17      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-15-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-15-4" class="td11">
100
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-15-5" class="td11">
9<span class="cmmi-12">.</span>52 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-15-6" class="td11">
5<span class="cmmi-12">.</span>01 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-16-1" class="td11">
<span class="cmti-12">HSPB2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-16-2" class="td11">
<pre><code>23      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-16-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-16-4" class="td11">
99.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-16-5" class="td11">
1<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-16-6" class="td11">
5<span class="cmmi-12">.</span>82 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-17-1" class="td11">
<span class="cmti-12">SMARCD3 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-17-2" class="td11">
<pre><code>19      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-17-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-17-4" class="td11">
99
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-17-5" class="td11">
1<span class="cmmi-12">.</span>59 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-17-6" class="td11">
7<span class="cmmi-12">.</span>57 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-18-1" class="td11">
<span class="cmti-12">CREM </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-18-2" class="td11">
<pre><code>13      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-18-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-18-4" class="td11">
98.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-18-5" class="td11">
2<span class="cmmi-12">.</span>48 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-18-6" class="td11">
1<span class="cmmi-12">.</span>13 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-19-1" class="td11">
<span class="cmti-12">GNG11 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-19-2" class="td11">
<pre><code>20      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-19-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-19-4" class="td11">
97.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-19-5" class="td11">
3<span class="cmmi-12">.</span>68 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-19-6" class="td11">
1<span class="cmmi-12">.</span>59 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-20-1" class="td11">
<span class="cmti-12">GNAI2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-20-2" class="td11">
<pre><code>17      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-20-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-20-4" class="td11">
96.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-20-5" class="td11">
5<span class="cmmi-12">.</span>75 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-20-6" class="td11">
2<span class="cmmi-12">.</span>36 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-21-1" class="td11">
<span class="cmti-12">FUNDC2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-21-2" class="td11">
<pre><code>22      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-21-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-21-4" class="td11">
95.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-21-5" class="td11">
7<span class="cmmi-12">.</span>39 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-21-6" class="td11">
2<span class="cmmi-12">.</span>91 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-22-1" class="td11">
<span class="cmti-12">CNRIP1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-22-2" class="td11">
<pre><code>21      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-22-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-22-4" class="td11">
95.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-22-5" class="td11">
1<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-22-6" class="td11">
3<span class="cmmi-12">.</span>66 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-23-1" class="td11">
<span class="cmti-12">CALHM2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-23-2" class="td11">
<pre><code>22      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-23-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-23-4" class="td11">
93.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-23-5" class="td11">
2<span class="cmmi-12">.</span>94 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-23-6" class="td11">
1<span class="cmmi-12">.</span>06 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-24-1" class="td11">
<span class="cmti-12">ARID5A </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-24-2" class="td11">
<pre><code>18      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-24-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-24-4" class="td11">
92.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-24-5" class="td11">
3<span class="cmmi-12">.</span>47 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-24-6" class="td11">
1<span class="cmmi-12">.</span>22 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-25-1" class="td11">
<span class="cmti-12">ST3GAL3 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-25-2" class="td11">
<pre><code>27      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-25-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-25-4" class="td11">
92.2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-25-5" class="td11">
4<span class="cmmi-12">.</span>49 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-25-6" class="td11">
1<span class="cmmi-12">.</span>56 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-100-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-100-26-1" class="td11">
<span class="cmti-12">LOC339524 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-26-2" class="td11">
<pre><code>21      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-100-26-3&quot;  </code></pre>
class=“td11”&gt; 50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-26-4" class="td11">
92.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-26-5" class="td11">
4<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-100-26-6" class="td11">
1<span class="cmmi-12">.</span>59 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-100-27-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-100-27-1"  
class="td11">
<pre><code>        &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 1299-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">SLIPT partners of </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">with observed and expected mutant samples of both genes</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 1324-->
<p class="indent">
<a 
 id="x1-216002r2"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table F.2: </span><span class="content">Pathway composition for clusters of <span class="cmti-12">CDH1 </span>partners in stomach SLIPT</span></p>
</div>
<!--tex4ht:label?: x1-216002r2 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-101" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-101-1g">
<col 
id="TBL-101-1" />
<col 
id="TBL-101-2" />
<col 
id="TBL-101-3" />
<col 
id="TBL-101-4" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-101-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-1-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 1</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-1-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-1-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-1-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-2-1" class="td11">
Viral mRNA Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-2-2" class="td11">
<pre><code>    82            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-2-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-2-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-3-1" class="td11">
Formation of a pool of free 40S subunits
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-3-2" class="td11">
<pre><code>    94            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-3-3&quot;  </code></pre>
class=“td11”&gt; 51
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-3-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-4-1" class="td11">
Eukaryotic Translation Elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-4-2" class="td11">
<pre><code>    87            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-4-3&quot;  </code></pre>
class=“td11”&gt; 49
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-4-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-5-1" class="td11">
Peptide chain elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-5-2" class="td11">
<pre><code>    84            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-5-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-5-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">96</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-6-1" class="td11">
Eukaryotic Translation Termination
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-6-2" class="td11">
<pre><code>    84            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-6-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-6-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">96</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-7-1" class="td11">
GTP hydrolysis and joining of the 60S ribosomal subunit
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-7-2" class="td11">
<pre><code>   105           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-7-3&quot;  </code></pre>
class=“td11”&gt; 52
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-7-4" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">94</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-8-1" class="td11">
Nonsense Mediated Decay independent of the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-8-2" class="td11">
<pre><code>    89            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-8-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-8-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">93</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-9-1" class="td11">
L13a-mediated translational silencing of Ceruloplasmin expression
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-9-2" class="td11">
<pre><code>   104           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-9-3&quot;  </code></pre>
class=“td11”&gt; 51
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-9-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-10-1" class="td11">
3’ -UTR-mediated translational regulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-10-2" class="td11">
<pre><code>   104           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-10-3&quot;  </code></pre>
class=“td11”&gt; 51
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-10-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-11-1" class="td11">
SRP-dependent cotranslational protein targeting to membrane
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-11-2" class="td11">
<pre><code>   105           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-11-3&quot;  </code></pre>
class=“td11”&gt; 51
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-11-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">91</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-12-1" class="td11">
Eukaryotic Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-12-2" class="td11">
<pre><code>   112           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-12-3&quot;  </code></pre>
class=“td11”&gt; 52
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-12-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-13-1" class="td11">
Cap-dependent Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-13-2" class="td11">
<pre><code>   112           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-13-3&quot;  </code></pre>
class=“td11”&gt; 52
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-13-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-14-1" class="td11">
Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-14-2" class="td11">
<pre><code>   142           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-14-3&quot;  </code></pre>
class=“td11”&gt; 56
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-14-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">85</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-15-1" class="td11">
Nonsense-Mediated Decay
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-15-2" class="td11">
<pre><code>   104           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-15-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-15-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">84</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-16-1" class="td11">
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-16-2" class="td11">
<pre><code>   104           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-16-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-16-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">84</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-17-1" class="td11">
Influenza Viral RNA Transcription and Replication
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-17-2" class="td11">
<pre><code>   109           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-17-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-17-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">82</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-18-1" class="td11">
Influenza Life Cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-18-2" class="td11">
<pre><code>   113           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-18-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-18-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-19-1" class="td11">
Influenza Infection
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-19-2" class="td11">
<pre><code>   118           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-19-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-19-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">78</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-20-1" class="td11">
Infectious disease
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-20-2" class="td11">
<pre><code>   349           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-20-3&quot;  </code></pre>
class=“td11”&gt; 68
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-20-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">50</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-21-1" class="td11">
Formation of the ternary complex, and subsequently, the 43S complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-21-2" class="td11">
<pre><code>    48            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-21-3&quot;  </code></pre>
class=“td11”&gt; 21
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-21-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">43</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-101-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-101-22-1"  
class="td11">
<pre><code>                                                               &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-23-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 2</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-23-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-23-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-23-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-24-1" class="td11">
Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-24-2" class="td11">
<pre><code>    65            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-24-3&quot;  </code></pre>
class=“td11”&gt; 12
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-24-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-25-1" class="td11">
Phosphorylation of CD3 and TCR zeta chains
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-25-2" class="td11">
<pre><code>    18            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-25-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-25-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-26-1" class="td11">
Generation of second messenger molecules
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-26-2" class="td11">
<pre><code>    29            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-26-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-26-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-27-1" class="td11">
PD-1 signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-27-2" class="td11">
<pre><code>    21            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-27-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-27-4" class="td11">
<pre><code> 7.4 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-28-1" class="td11">
TCR signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-28-2" class="td11">
<pre><code>    62            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-28-3&quot;  </code></pre>
class=“td11”&gt; 9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-28-4" class="td11">
<pre><code> 4.3 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-29-1" class="td11">
Translocation of ZAP-70 to Immunological synapse
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-29-2" class="td11">
<pre><code>    16            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-29-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-29-4" class="td11">
<pre><code> 1.1 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-30-1" class="td11">
Interferon alpha/beta signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-30-2" class="td11">
<pre><code>    68            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-30-3&quot;  </code></pre>
class=“td11”&gt; 9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-30-4" class="td11">
<pre><code> 1.6 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-31-1" class="td11">
Initial triggering of complement
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-31-2" class="td11">
<pre><code>    17            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-31-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-31-4" class="td11">
<pre><code> 1.6 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-32-1" class="td11">
IKK complex recruitment mediated by RIP1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-32-2" class="td11">
<pre><code>    19            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-32-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-32-4" class="td11">
<pre><code> 5.1 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-33-1" class="td11">
TRIF-mediated programmed cell death
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-33-2" class="td11">
<pre><code>    10            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-33-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-33-4" class="td11">
<pre><code> 6.2 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-34-1" class="td11">
Creation of C4 and C2 activators
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-34-2" class="td11">
<pre><code>    11            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-34-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-34-4" class="td11">
<pre><code> 1.3 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-35-1" class="td11">
RHO GTPases Activate NADPH Oxidases
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-35-2" class="td11">
<pre><code>    11            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-35-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-35-4" class="td11">
<pre><code> 1.3 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;×</span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-36-1" class="td11">
Interferon Signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-36-2" class="td11">
<pre><code>   175           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-36-3&quot;  </code></pre>
class=“td11”&gt; 15
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-36-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-37-1" class="td11">
Chemokine receptors bind chemokines
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-37-2" class="td11">
<pre><code>    52            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-37-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-37-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-38-1" class="td11">
Interferon gamma signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-38-2" class="td11">
<pre><code>    74            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-38-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-38-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-39-1" class="td11">
TRAF6 mediated induction of TAK1 complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-39-2" class="td11">
<pre><code>    15            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-39-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-39-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-40-1" class="td11">
Activation of IRF3/IRF7 mediated by TBK1/IKK epsilon
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-40-2" class="td11">
<pre><code>    16            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-40-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-40-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-41-1" class="td11">
Downstream TCR signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-41-2" class="td11">
<pre><code>    45            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-41-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-41-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-42-1" class="td11">
Ligand-dependent caspase activation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-42-2" class="td11">
<pre><code>    17            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-42-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-42-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-43-1" class="td11">
Complement cascade
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-43-2" class="td11">
<pre><code>    34            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-43-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-43-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-101-44-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-101-44-1"  
class="td11">
<pre><code>                                                               &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-45-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 3</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-45-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-45-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-45-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-46-1" class="td11">
Uptake and actions of bacterial toxins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-46-2" class="td11">
<pre><code>    22            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-46-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-46-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-47-1" class="td11">
Neurotoxicity of clostridium toxins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-47-2" class="td11">
<pre><code>    10            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-47-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-47-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-48-1" class="td11">
Activation of PPARGC1A (PGC-1alpha) by phosphorylation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-48-2" class="td11">
<pre><code>    10            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-48-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-48-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-49-1" class="td11">
SMAD2/SMAD3:SMAD4 heterotrimer regulates transcription
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-49-2" class="td11">
<pre><code>    28            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-49-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-49-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-50-1" class="td11">
Assembly of the primary cilium
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-50-2" class="td11">
<pre><code>   149           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-50-3&quot;  </code></pre>
class=“td11”&gt; 10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-50-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-51-1" class="td11">
Serotonin Neurotransmitter Release Cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-51-2" class="td11">
<pre><code>    15            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-51-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-51-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-52-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-52-1" class="td11">
Glycosaminoglycan metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-52-2" class="td11">
<pre><code>   114           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-52-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-52-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-53-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-53-1" class="td11">
Platelet homeostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-53-2" class="td11">
<pre><code>    54            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-53-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-53-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-54-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-54-1" class="td11">
Norepinephrine Neurotransmitter Release Cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-54-2" class="td11">
<pre><code>    17            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-54-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-54-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-55-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-55-1" class="td11">
Acetylcholine Neurotransmitter Release Cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-55-2" class="td11">
<pre><code>    17            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-55-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-55-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-56-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-56-1" class="td11">
G˙<span class="cmmi-12">αs</span> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-56-2" class="td11">
<pre><code>   100           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-56-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-56-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-57-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-57-1" class="td11">
GABA synthesis, release, reuptake and degradation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-57-2" class="td11">
<pre><code>    19            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-57-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-57-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-58-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-58-1" class="td11">
deactivation of the beta-catenin transactivating complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-58-2" class="td11">
<pre><code>    39            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-58-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-58-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-59-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-59-1" class="td11">
Dopamine Neurotransmitter Release Cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-59-2" class="td11">
<pre><code>    20            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-59-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-59-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-60-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-60-1" class="td11">
IRS-related events triggered by IGF1R
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-60-2" class="td11">
<pre><code>    83            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-60-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-60-4" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-61-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-61-1" class="td11">
Generic Transcription Pathway
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-61-2" class="td11">
<pre><code>   186           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-61-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-61-4" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-62-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-62-1" class="td11">
Termination of O-glycan biosynthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-62-2" class="td11">
<pre><code>    21            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-62-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-62-4" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-63-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-63-1" class="td11">
Kinesins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-63-2" class="td11">
<pre><code>    22            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-63-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-63-4" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-64-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-64-1" class="td11">
Signaling by Type 1 Insulin-like Growth Factor 1 Receptor (IGF1R)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-64-2" class="td11">
<pre><code>    86            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-64-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-64-4" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-65-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-65-1" class="td11">
IGF1R signaling cascade
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-65-2" class="td11">
<pre><code>    86            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-65-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-65-4" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-101-66-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-101-66-1"  
class="td11">
<pre><code>                                                               &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-67-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-67-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 4</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-67-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-67-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-67-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-68-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-68-1" class="td11">
Extracellular matrix organization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-68-2" class="td11">
<pre><code>   241           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-68-3&quot;  </code></pre>
class=“td11”&gt; 97
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-68-4" class="td11">
<pre><code>8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">126</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-69-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-69-1" class="td11">
Axon guidance
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-69-2" class="td11">
<pre><code>   289           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-69-3&quot;  </code></pre>
class=“td11”&gt; 75
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-69-4" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">72</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-70-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-70-1" class="td11">
Hemostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-70-2" class="td11">
<pre><code>   445           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-70-3&quot;  </code></pre>
class=“td11”&gt; 101
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-70-4" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">72</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-71-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-71-1" class="td11">
Developmental Biology
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-71-2" class="td11">
<pre><code>   432           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-71-3&quot;  </code></pre>
class=“td11”&gt; 95
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-71-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">67</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-72-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-72-1" class="td11">
Response to elevated platelet cytosolic Ca<sup>2ˆ+</sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-72-2" class="td11">
<pre><code>    84            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-72-3&quot;  </code></pre>
class=“td11”&gt; 37
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-72-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">67</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-73-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-73-1" class="td11">
Platelet degranulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-73-2" class="td11">
<pre><code>    79            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-73-3&quot;  </code></pre>
class=“td11”&gt; 36
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-73-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">67</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-74-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-74-1" class="td11">
Degradation of the extracellular matrix
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-74-2" class="td11">
<pre><code>   104           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-74-3&quot;  </code></pre>
class=“td11”&gt; 39
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-74-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">63</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-75-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-75-1" class="td11">
Platelet activation, signaling and aggregation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-75-2" class="td11">
<pre><code>   186           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-75-3&quot;  </code></pre>
class=“td11”&gt; 52
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-75-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">62</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-76-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-76-1" class="td11">
ECM proteoglycans
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-76-2" class="td11">
<pre><code>    66            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-76-3&quot;  </code></pre>
class=“td11”&gt; 31
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-76-4" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">61</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-77-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-77-1" class="td11">
Neuronal System
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-77-2" class="td11">
<pre><code>   272           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-77-3&quot;  </code></pre>
class=“td11”&gt; 64
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-77-4" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">60</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-78-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-78-1" class="td11">
Signaling by PDGF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-78-2" class="td11">
<pre><code>   173           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-78-3&quot;  </code></pre>
class=“td11”&gt; 47
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-78-4" class="td11">
<pre><code> 9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">57</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-79-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-79-1" class="td11">
Integrin cell surface interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-79-2" class="td11">
<pre><code>    82            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-79-3&quot;  </code></pre>
class=“td11”&gt; 31
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-79-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">53</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-80-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-80-1" class="td11">
Collagen biosynthesis and modifying enzymes
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-80-2" class="td11">
<pre><code>    56            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-80-3&quot;  </code></pre>
class=“td11”&gt; 26
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-80-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">52</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-81-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-81-1" class="td11">
Collagen formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-81-2" class="td11">
<pre><code>    67            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-81-3&quot;  </code></pre>
class=“td11”&gt; 28
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-81-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">52</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-82-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-82-1" class="td11">
Class A/1 (Rhodopsin-like receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-82-2" class="td11">
<pre><code>   289           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-82-3&quot;  </code></pre>
class=“td11”&gt; 61
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-82-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">52</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-83-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-83-1" class="td11">
GPCR ligand binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-83-2" class="td11">
<pre><code>   373           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-83-3&quot;  </code></pre>
class=“td11”&gt; 73
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-83-4" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">52</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-84-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-84-1" class="td11">
Elastic fibre formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-84-2" class="td11">
<pre><code>    38            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-84-3&quot;  </code></pre>
class=“td11”&gt; 22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-84-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">52</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-85-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-85-1" class="td11">
Non-integrin membrane-ECM interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-85-2" class="td11">
<pre><code>    53            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-85-3&quot;  </code></pre>
class=“td11”&gt; 24
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-85-4" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">49</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-86-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-86-1" class="td11">
Glycosaminoglycan metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-86-2" class="td11">
<pre><code>   114           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-86-3&quot;  </code></pre>
class=“td11”&gt; 33
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-86-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">47</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-101-87-">
<td style="white-space:nowrap; text-align:left;" id="TBL-101-87-1" class="td11">
Platelet homeostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-87-2" class="td11">
<pre><code>    54            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-101-87-3&quot;  </code></pre>
class=“td11”&gt; 23
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-101-87-4" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">45</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-101-88-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-101-88-1"  
class="td11">
<pre><code>                                                               &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                                       &lt;/div&gt;
                                                                  
                                                                                </code></pre>
</div>
<hr class="endfloat" />
<a 
 id="x1-216003r339"></a>
<h3 class="sectionHead">
<span class="titlemark">F.2 </span> <a 
 id="x1-2170002"></a>Comparison to Primary Screen
</h3>
<!--l. 1520-->
<p class="noindent">
The synthetic lethal partners with <span class="cmti-12">CDH1 </span>expression in stoamch cancers were also compared to siRNA primary screen data (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), as performed in Section <a 
href="#x1-1230001">4.2.1<!--tex4ht:ref: chapt3:primary_screen --></a>. These are expected to be more concordant with the experimental results performed on a null mutant, however this not the case at the gene level: less genes overlapped with experimental candidates in Figure <a 
href="#x1-217001r1">F.1<!--tex4ht:ref: fig:Venn_allgenes_stad --></a>. This may be affected by lower sample size for mutations in TCGA data or lower frequency (expected value) of <span class="cmti-12">CDH1 </span>mutations compared to low expression.
</p>
<hr class="figure" />
<div class="figure">
<p><a 
 id="x1-217001r1"></a></p>
<hr />
____________________________________________________________________________________      <!--l. 1527-->
<p class="noindent">
<img 
src="Venn_exprSL_siRNA_allgenes_reduced_Pub_stad.png" alt="PIC"  
 /> <br />
</p>
<div class="caption">
<p><span class="id">Figure F.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Comparison of SLIPT in stomach to siRNA.</span></strong> <span class="cmr-10x-x-109">Testing the overlap of</span> <span class="cmr-10x-x-109">gene candidates for E-cadherin synthetic lethal partners between computational (SLIPT)</span> <span class="cmr-10x-x-109">and experimental screening (siRNA) approaches. The </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test suggests that the overlap</span> <span class="cmr-10x-x-109">is no more than would be expected by chance (</span><span class="cmmi-10x-x-109">p </span><span class="cmr-10x-x-109">= 0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">281). </span></span></p>
</div>
<!--tex4ht:label?: x1-217001r1 -->
</div>
<hr class="endfigure" />
<!--l. 1541-->
<p class="indent">
<a 
 id="x1-217002r3"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table F.3: </span><span class="content">Pathway composition for <span class="cmti-12">CDH1 </span>partners from SLIPT and siRNA screening</span></p>
</div>
<!--tex4ht:label?: x1-217002r3 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-102" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-102-1g">
<col 
id="TBL-102-1" />
<col 
id="TBL-102-2" />
<col 
id="TBL-102-3" />
<col 
id="TBL-102-4" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-102-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-1-1" class="td11">
<span class="cmbx-12">Predicted only by SLIPT (3392 genes) </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-1-2" class="td11">
Pathway Size
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-1-3" class="td11">
Genes Identified
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-1-4" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-2-1" class="td11">
Extracellular matrix organization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-2-2" class="td11">
<pre><code> 238        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-2-3&quot;  </code></pre>
class=“td11”&gt; 90
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-2-4" class="td11">
3<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">107</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-3-1" class="td11">
Eukaryotic Translation Termination
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-3-2" class="td11">
<pre><code>  79         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-3-3&quot;  </code></pre>
class=“td11”&gt; 46
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-3-4" class="td11">
7<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">91</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-4-1" class="td11">
Viral mRNA Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-4-2" class="td11">
<pre><code>  77         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-4-3&quot;  </code></pre>
class=“td11”&gt; 45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-4-4" class="td11">
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">89</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-5-1" class="td11">
Eukaryotic Translation Elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-5-2" class="td11">
<pre><code>  82         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-5-3&quot;  </code></pre>
class=“td11”&gt; 46
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-5-4" class="td11">
5<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">89</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-6-1" class="td11">
Peptide chain elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-6-2" class="td11">
<pre><code>  79         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-6-3&quot;  </code></pre>
class=“td11”&gt; 45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-6-4" class="td11">
2<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">88</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-7-1" class="td11">
Nonsense Mediated Decay independent of the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-7-2" class="td11">
<pre><code>  84         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-7-3&quot;  </code></pre>
class=“td11”&gt; 46
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-7-4" class="td11">
9<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">88</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-8-1" class="td11">
Formation of a pool of free 40S subunits
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-8-2" class="td11">
<pre><code>  89         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-8-3&quot;  </code></pre>
class=“td11”&gt; 47
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-8-4" class="td11">
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">87</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-9-1" class="td11">
GTP hydrolysis and joining of the 60S ribosomal subunit
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-9-2" class="td11">
<pre><code> 100        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-9-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-9-4" class="td11">
3<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">83</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-10-1" class="td11">
Axon guidance
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-10-2" class="td11">
<pre><code> 284        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-10-3&quot;  </code></pre>
class=“td11”&gt; 84
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-10-4" class="td11">
3<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">82</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-11-1" class="td11">
Developmental Biology
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-11-2" class="td11">
<pre><code> 426        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-11-3&quot;  </code></pre>
class=“td11”&gt; 111
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-11-4" class="td11">
4<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">82</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-12-1" class="td11">
L13a-mediated translational silencing of Ceruloplasmin expression
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-12-2" class="td11">
<pre><code>  99         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-12-3&quot;  </code></pre>
class=“td11”&gt; 47
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-12-4" class="td11">
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">81</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-13-1" class="td11">
3’ -UTR-mediated translational regulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-13-2" class="td11">
<pre><code>  99         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-13-3&quot;  </code></pre>
class=“td11”&gt; 47
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-13-4" class="td11">
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">81</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-14-1" class="td11">
SRP-dependent cotranslational protein targeting to membrane
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-14-2" class="td11">
<pre><code>  99         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-14-3&quot;  </code></pre>
class=“td11”&gt; 47
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-14-4" class="td11">
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">81</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-15-1" class="td11">
Nonsense-Mediated Decay
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-15-2" class="td11">
<pre><code>  99         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-15-3&quot;  </code></pre>
class=“td11”&gt; 47
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-15-4" class="td11">
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">81</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-16-1" class="td11">
Nonsense Mediated Decay enhanced by the Exon Junction Complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-16-2" class="td11">
<pre><code>  99         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-16-3&quot;  </code></pre>
class=“td11”&gt; 47
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-16-4" class="td11">
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">81</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-17-1" class="td11">
Hemostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-17-2" class="td11">
<pre><code> 438        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-17-3&quot;  </code></pre>
class=“td11”&gt; 112
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-17-4" class="td11">
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-18-1" class="td11">
Eukaryotic Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-18-2" class="td11">
<pre><code> 107        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-18-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-18-4" class="td11">
8<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-19-1" class="td11">
Cap-dependent Translation Initiation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-19-2" class="td11">
<pre><code> 107        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-19-3&quot;  </code></pre>
class=“td11”&gt; 48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-19-4" class="td11">
8<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-20-1" class="td11">
Infectious disease
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-20-2" class="td11">
<pre><code> 338        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-20-3&quot;  </code></pre>
class=“td11”&gt; 90
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-20-4" class="td11">
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">76</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-21-1" class="td11">
Neuronal System
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-21-2" class="td11">
<pre><code> 267        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-21-3&quot;  </code></pre>
class=“td11”&gt; 77
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-21-4" class="td11">
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">76</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-102-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-102-22-1"  
class="td11">
<pre><code>                                                         &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-23-1" class="td11">
<span class="cmbx-12">Detected only by siRNA screen (1803 genes) </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-23-2" class="td11">
Pathway Size
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-23-3" class="td11">
Genes Identified
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-23-4" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-24-1" class="td11">
Class A/1 (Rhodopsin-like receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-24-2" class="td11">
<pre><code> 282        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-24-3&quot;  </code></pre>
class=“td11”&gt; 62
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-24-4" class="td11">
8<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">50</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-25-1" class="td11">
GPCR ligand binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-25-2" class="td11">
<pre><code> 363        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-25-3&quot;  </code></pre>
class=“td11”&gt; 71
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-25-4" class="td11">
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">46</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-26-1" class="td11">
Peptide ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-26-2" class="td11">
<pre><code> 175        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-26-3&quot;  </code></pre>
class=“td11”&gt; 38
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-26-4" class="td11">
7<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">38</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-27-1" class="td11">
G˙<span class="cmmi-12">αi</span> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-27-2" class="td11">
<pre><code> 184        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-27-3&quot;  </code></pre>
class=“td11”&gt; 37
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-27-4" class="td11">
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">34</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-28-1" class="td11">
Gastrin-CREB signalling pathway via PKC and MAPK
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-28-2" class="td11">
<pre><code> 180        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-28-3&quot;  </code></pre>
class=“td11”&gt; 35
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-28-4" class="td11">
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">32</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-29-1" class="td11">
G˙<span class="cmmi-12">αq</span> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-29-2" class="td11">
<pre><code> 159        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-29-3&quot;  </code></pre>
class=“td11”&gt; 32
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-29-4" class="td11">
4<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">32</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-30-1" class="td11">
DAP12 interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-30-2" class="td11">
<pre><code> 159        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-30-3&quot;  </code></pre>
class=“td11”&gt; 29
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-30-4" class="td11">
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">27</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-31-1" class="td11">
Downstream signal transduction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-31-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-31-3&quot;  </code></pre>
class=“td11”&gt; 26
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-31-4" class="td11">
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-32-1" class="td11">
DAP12 signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-32-2" class="td11">
<pre><code> 149        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-32-3&quot;  </code></pre>
class=“td11”&gt; 26
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-32-4" class="td11">
6<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-33-1" class="td11">
VEGFA-VEGFR2 Pathway
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-33-2" class="td11">
<pre><code>  91         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-33-3&quot;  </code></pre>
class=“td11”&gt; 19
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-33-4" class="td11">
8<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">24</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-34-1" class="td11">
Signaling by PDGF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-34-2" class="td11">
<pre><code> 172        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-34-3&quot;  </code></pre>
class=“td11”&gt; 27
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-34-4" class="td11">
5<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">23</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-35-1" class="td11">
Signaling by ERBB2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-35-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-35-3&quot;  </code></pre>
class=“td11”&gt; 24
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-35-4" class="td11">
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-36-1" class="td11">
Signaling by VEGF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-36-2" class="td11">
<pre><code>  99         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-36-3&quot;  </code></pre>
class=“td11”&gt; 19
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-36-4" class="td11">
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-37-1" class="td11">
Visual phototransduction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-37-2" class="td11">
<pre><code>  85         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-37-3&quot;  </code></pre>
class=“td11”&gt; 17
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-37-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-38-1" class="td11">
Downstream signaling of activated FGFR1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-38-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-38-3&quot;  </code></pre>
class=“td11”&gt; 22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-38-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-39-1" class="td11">
Downstream signaling of activated FGFR2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-39-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-39-3&quot;  </code></pre>
class=“td11”&gt; 22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-39-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-40-1" class="td11">
Downstream signaling of activated FGFR3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-40-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-40-3&quot;  </code></pre>
class=“td11”&gt; 22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-40-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-41-1" class="td11">
Downstream signaling of activated FGFR4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-41-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-41-3&quot;  </code></pre>
class=“td11”&gt; 22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-41-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-42-1" class="td11">
Signaling by FGFR
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-42-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-42-3&quot;  </code></pre>
class=“td11”&gt; 23
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-42-4" class="td11">
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-43-1" class="td11">
Signaling by FGFR1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-43-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-43-3&quot;  </code></pre>
class=“td11”&gt; 23
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-43-4" class="td11">
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-102-44-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-102-44-1"  
class="td11">
<pre><code>                                                         &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-45-1" class="td11">
<span class="cmbx-12">Intersection of SLIPT and siRNA screen (547 genes) </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-45-2" class="td11">
Pathway Size
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-45-3" class="td11">
Genes Identified
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-45-4" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-46-1" class="td11">
Class A/1 (Rhodopsin-like receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-46-2" class="td11">
<pre><code> 282        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-46-3&quot;  </code></pre>
class=“td11”&gt; 25
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-46-4" class="td11">
3<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-47-1" class="td11">
Platelet activation, signaling and aggregation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-47-2" class="td11">
<pre><code> 182        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-47-3&quot;  </code></pre>
class=“td11”&gt; 17
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-47-4" class="td11">
3<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-48-1" class="td11">
Response to elevated platelet cytosolic Ca2<sup><span class="cmr-8">+</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-48-2" class="td11">
<pre><code>  82         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-48-3&quot;  </code></pre>
class=“td11”&gt; 9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-48-4" class="td11">
5<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-49-1" class="td11">
Platelet homeostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-49-2" class="td11">
<pre><code>  53         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-49-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-49-4" class="td11">
5<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-50-1" class="td11">
Nucleotide-like (purinergic) receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-50-2" class="td11">
<pre><code>  16         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-50-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-50-4" class="td11">
1<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-51-1" class="td11">
Platelet degranulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-51-2" class="td11">
<pre><code>  77         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-51-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-51-4" class="td11">
2<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-52-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-52-1" class="td11">
Peptide ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-52-2" class="td11">
<pre><code> 175        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-52-3&quot;  </code></pre>
class=“td11”&gt; 14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-52-4" class="td11">
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-53-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-53-1" class="td11">
Molecules associated with elastic fibres
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-53-2" class="td11">
<pre><code>  34         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-53-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-53-4" class="td11">
7<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-54-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-54-1" class="td11">
Amine ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-54-2" class="td11">
<pre><code>  35         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-54-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-54-4" class="td11">
8<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-55-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-55-1" class="td11">
G˙<span class="cmmi-12">αi</span> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-55-2" class="td11">
<pre><code> 184        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-55-3&quot;  </code></pre>
class=“td11”&gt; 14
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-55-4" class="td11">
9<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-56-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-56-1" class="td11">
GPCR ligand binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-56-2" class="td11">
<pre><code> 363        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-56-3&quot;  </code></pre>
class=“td11”&gt; 27
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-56-4" class="td11">
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-57-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-57-1" class="td11">
Elastic fibre formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-57-2" class="td11">
<pre><code>  38         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-57-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-57-4" class="td11">
1<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-58-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-58-1" class="td11">
G˙<span class="cmmi-12">αq</span> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-58-2" class="td11">
<pre><code> 159        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-58-3&quot;  </code></pre>
class=“td11”&gt; 12
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-58-4" class="td11">
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-59-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-59-1" class="td11">
Serotonin receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-59-2" class="td11">
<pre><code>  12         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-59-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-59-4" class="td11">
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-60-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-60-1" class="td11">
P2Y receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-60-2" class="td11">
<pre><code>  12         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-60-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-60-4" class="td11">
3<span class="cmmi-12">.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-61-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-61-1" class="td11">
Signal amplification
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-61-2" class="td11">
<pre><code>  16         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-61-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-61-4" class="td11">
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-62-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-62-1" class="td11">
Gastrin-CREB signalling pathway via PKC and MAPK
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-62-2" class="td11">
<pre><code> 180        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-62-3&quot;  </code></pre>
class=“td11”&gt; 12
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-62-4" class="td11">
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-63-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-63-1" class="td11">
Complement cascade
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-63-2" class="td11">
<pre><code>  33         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-63-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-63-4" class="td11">
2<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-64-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-64-1" class="td11">
Glycosaminoglycan metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-64-2" class="td11">
<pre><code> 110        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-64-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-64-4" class="td11">
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-102-65-">
<td style="white-space:nowrap; text-align:left;" id="TBL-102-65-1" class="td11">
Glycogen breakdown (glycogenolysis)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-65-2" class="td11">
<pre><code>  17         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-102-65-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-102-65-4" class="td11">
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-102-66-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-102-66-1"  
class="td11">
<pre><code>                                                         &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                         &lt;/div&gt;
                                                                                
                                                                                </code></pre>
</div>
<hr class="endfloat" />
<a 
 id="x1-217003r328"></a>
<h4 class="subsectionHead">
<span class="titlemark">F.2.1 </span> <a 
 id="x1-2180001"></a>Resampling Analysis
</h4>
<!--l. 1699-->
<p class="indent">
<a 
 id="x1-218001r4"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table F.4: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in stomach cancer</span></p>
</div>
<!--tex4ht:label?: x1-218001r4 -->
<div class="tabular">
<table id="TBL-103" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-103-1g">
<col 
id="TBL-103-1" />
<col 
id="TBL-103-2" />
<col 
id="TBL-103-3" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-103-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-1-1" class="td11">
<span class="cmbx-12">Reactome Pathway </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-1-2" class="td11">
Over-representation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-1-3" class="td11">
Permutation
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-2-1" class="td11">
<span class="cmti-12">Extracellular matrix organization </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-2-2" class="td11">
<pre><code>7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">140</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-2-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>070215
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-3-1" class="td11">
Hemostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-3-2" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">121</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-3-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>25804
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-4-1" class="td11">
Developmental Biology
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-4-2" class="td11">
<pre><code>9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">107</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-4-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>53032
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-5-1" class="td11">
Axon guidance
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-5-2" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">102</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-5-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6704
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-6-1" class="td11">
<strong><span class="cmbx-12">Eukaryotic Translation Termination</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-6-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">99</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-6-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-7-1" class="td11">
GPCR ligand binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-7-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">99</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-7-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>54914
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-8-1" class="td11">
<strong><span class="cmbx-12">Viral mRNA Translation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-8-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">98</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-8-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-9-1" class="td11">
<strong><span class="cmbx-12">Formation of a pool of free 40S subunits</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-9-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">98</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-9-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-10-1" class="td11">
<strong><span class="cmbx-12">Eukaryotic Translation Elongation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-10-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-10-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-11-1" class="td11">
<strong><span class="cmbx-12">Peptide chain elongation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-11-2" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">97</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-11-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-12-1" class="td11">
Class A/1 (Rhodopsin-like receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-12-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">96</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-12-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>58174
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-13-1" class="td11">
<strong><span class="cmbx-12">Nonsense Mediated Decay independent of the Exon Junction Complex</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-13-2" class="td11">
<pre><code> 3 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">96</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-13-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-14-1" class="td11">
Infectious disease
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-14-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">94</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-14-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>25484
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-15-1" class="td11">
<strong><span class="cmbx-12">GTP hydrolysis and joining of the 60S ribosomal subunit</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-15-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">94</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-15-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-16-1" class="td11">
<strong><span class="cmbx-12">L13a-mediated translational silencing of Ceruloplasmin expression</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-16-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-16-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-17-1" class="td11">
<strong><span class="cmbx-12">3’ -UTR-mediated translational regulation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-17-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-17-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-18-1" class="td11">
Neuronal System
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-18-2" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-18-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>53433
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-19-1" class="td11">
<strong><span class="cmbx-12">SRP-dependent cotranslational protein targeting to membrane</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-19-2" class="td11">
<pre><code> 9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">92</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-19-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-20-1" class="td11">
<strong><span class="cmbx-12">Eukaryotic Translation Initiation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-20-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-20-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-21-1" class="td11">
<strong><span class="cmbx-12">Cap-dependent Translation Initiation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-21-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-21-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-22-1" class="td11">
<strong><span class="cmbx-12">Nonsense-Mediated Decay</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-22-2" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-22-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-23-1" class="td11">
<strong><span class="cmbx-12">Nonsense Mediated Decay enhanced by the Exon Junction Complex</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-23-2" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-23-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-24-1" class="td11">
Adaptive Immune System
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-24-2" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">88</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-24-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>14116
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-25-1" class="td11">
<strong><span class="cmbx-12">Translation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-25-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">87</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-25-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-26-1" class="td11">
Platelet activation, signaling and aggregation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-26-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">86</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-26-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>28959
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-27-1" class="td11">
<strong><span class="cmbx-12">Influenza Infection</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-27-2" class="td11">
<pre><code> 1 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">82</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-27-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-28-1" class="td11">
<strong><span class="cmbx-12">Influenza Viral RNA Transcription and Replication</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-28-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">82</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-28-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-29-1" class="td11">
<strong><span class="cmbx-12">Influenza Life Cycle</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-29-2" class="td11">
<pre><code> 2 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-29-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>031 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-30-1" class="td11">
Response to elevated platelet cytosolic Ca2<sup><span class="cmr-8">+</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-30-2" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">78</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-30-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>50817
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-31-1" class="td11">
Signalling by NGF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-31-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">75</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-31-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>38518
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-32-1" class="td11">
Rho GTPase cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-32-2" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">75</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-32-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>14864
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-33-1" class="td11">
Signaling by PDGF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-33-2" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">74</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-33-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>40493
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-34-1" class="td11">
<span class="cmti-12">Signaling by Rho GTPases </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-34-2" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">73</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-34-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>077217
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-35-1" class="td11">
Glycosaminoglycan metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-35-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">68</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-35-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>52984
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-36-1" class="td11">
G˙<span class="cmmi-12">αi</span> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-36-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">66</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-36-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9254
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-37-1" class="td11">
Metabolism of carbohydrates
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-37-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">65</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-37-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>39501
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-38-1" class="td11">
<strong><span class="cmbx-12">G˙</span><span class="cmmi-12">αs</span> <span class="cmbx-12">signalling events</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-38-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">65</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-38-3" class="td11">
0<span class="cmmi-12">.</span>0050293
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-39-1" class="td11">
Potassium Channels
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-39-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">65</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-39-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>53359
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-40-1" class="td11">
Transmission across Chemical Synapses
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-40-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">64</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-40-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>81833
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-41-1" class="td11">
ECM proteoglycans
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-41-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">64</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-41-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>083482
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-42-1" class="td11">
Peptide ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-42-2" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">64</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-42-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>62817
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-43-1" class="td11">
Degradation of the extracellular matrix
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-43-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">63</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-43-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>80879
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-44-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-44-1" class="td11">
Platelet homeostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-44-2" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">63</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-44-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>53134
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-45-1" class="td11">
NGF signalling via TRKA from the plasma membrane
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-45-2" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">63</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-45-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5717
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-46-1" class="td11">
Integration of energy metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-46-2" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">61</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-46-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>10889
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-47-1" class="td11">
Collagen formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-47-2" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">61</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-47-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>29896
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-48-1" class="td11">
Integrin cell surface interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-48-2" class="td11">
<pre><code> 7 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">59</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-48-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>18167
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-49-1" class="td11">
Collagen biosynthesis and modifying enzymes
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-49-2" class="td11">
<pre><code> 7 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">59</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-49-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>30208
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-103-50-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-103-50-1"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-104" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-104-1g">
<col 
id="TBL-104-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-104-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-104-1-1" class="td00">
Neurotransmitter Receptor Binding And Downstream Transmission
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-104-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-104-2-1" class="td00">
In The Postsynaptic Cell
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-50-2" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">57</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-50-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>82522
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-103-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-103-51-1" class="td11">
Signaling by Wnt
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-51-2" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">57</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-103-51-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>25468
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-103-52-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-103-52-1"  
class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 1816-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Over-representation (hypergeometric test) and Permutation p-values adjusted for multiple tests across pathways (FDR). Significant pathways</span> <span class="cmr-10x-x-109">are marked in bold (FDR </span><span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05) and italics (FDR </span><span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">1).</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 1820-->
<p class="indent">
<a 
 id="x1-218002r5"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table F.5: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in stomach and siRNA screen</span></p>
</div>
<!--tex4ht:label?: x1-218002r5 -->
<div class="tabular">
<table id="TBL-105" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-105-1g">
<col 
id="TBL-105-1" />
<col 
id="TBL-105-2" />
<col 
id="TBL-105-3" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-105-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-1-1" class="td11">
<span class="cmbx-12">Reactome Pathway </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-1-2" class="td11">
Over-representation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-1-3" class="td11">
Permutation
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-2-1" class="td11">
Platelet activation, signaling and aggregation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-2-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-2-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>49557
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-3-1" class="td11">
Class A/1 (Rhodopsin-like receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-3-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-3-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>98432
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-4-1" class="td11">
Response to elevated platelet cytosolic Ca2<sup><span class="cmr-8">+</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-4-2" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-4-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>54349
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-5-1" class="td11">
Platelet homeostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-5-2" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-5-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>45017
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-6-1" class="td11">
Nucleotide-like (purinergic) receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-6-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-6-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>36966
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-7-1" class="td11">
Peptide ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-7-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-7-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>91294
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-8-1" class="td11">
<strong><span class="cmbx-12">Molecules associated with elastic fibres</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-8-2" class="td11">
<pre><code> 7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-8-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0025868
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-9-1" class="td11">
Amine ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-9-2" class="td11">
<pre><code> 8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-9-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>43303
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-10-1" class="td11">
G˙<span class="cmmi-12">αi</span> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-10-2" class="td11">
<pre><code> 9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-10-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>99626
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-11-1" class="td11">
GPCR ligand binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-11-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-11-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>97733
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-12-1" class="td11">
<strong><span class="cmbx-12">Elastic fibre formation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-12-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-12-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0025868
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-13-1" class="td11">
G˙<span class="cmmi-12">αq</span> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-13-2" class="td11">
<pre><code> 1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-13-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>86089
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-14-1" class="td11">
P2Y receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-14-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-14-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>18795
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-15-1" class="td11">
Serotonin receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-15-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-15-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>37853
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-16-1" class="td11">
Signal amplification
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-16-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-16-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>47856
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-17-1" class="td11">
Gastrin-CREB signalling pathway via PKC and MAPK
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-17-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-17-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>98567
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-18-1" class="td11">
<strong><span class="cmbx-12">Complement cascade</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-18-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-18-3" class="td11">
<span class="cmmi-12">&gt; </span>3<span class="cmmi-12">.</span>4628 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-19-1" class="td11">
Glycosaminoglycan metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-19-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-19-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>38953
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-20-1" class="td11">
Glycogen breakdown (glycogenolysis)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-20-2" class="td11">
<pre><code> 2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-20-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>83772
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-21-1" class="td11">
Defective B4GALT7 causes EDS, progeroid type
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-21-2" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-21-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>10792
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-22-1" class="td11">
Defective B3GAT3 causes JDSSDHD
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-22-2" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-22-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>10792
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-23-1" class="td11">
Role of LAT2/NTAL/LAB on calcium mobilization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-23-2" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-23-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>35373
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-24-1" class="td11">
Cell surface interactions at the vascular wall
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-24-2" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-24-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>47642
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-25-1" class="td11">
<strong><span class="cmbx-12">G˙</span><span class="cmmi-12">αs</span> <span class="cmbx-12">signalling events</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-25-2" class="td11">
<pre><code>  6 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-25-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>019858
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-26-1" class="td11">
Signaling by NOTCH
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-26-2" class="td11">
<pre><code>  6 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-26-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>19008
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-27-1" class="td11">
A tetrasaccharide linker sequence is required for GAG synthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-27-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00017
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-27-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>47642
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-28-1" class="td11">
<strong><span class="cmbx-12">Extracellular matrix organization</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-28-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00018
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-28-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0047308
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-29-1" class="td11">
Collagen formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-29-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00018
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-29-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>19245
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-30-1" class="td11">
Effects of PIP2 hydrolysis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-30-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0002
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-30-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>37779
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-31-1" class="td11">
Syndecan interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-31-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0002
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-31-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>37779
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-32-1" class="td11">
<strong><span class="cmbx-12">Diseases associated with glycosaminoglycan metabolism</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-32-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00023
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-32-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>01028
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-33-1" class="td11">
<strong><span class="cmbx-12">Diseases of glycosylation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-33-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00023
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-33-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>01028
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-34-1" class="td11">
<span class="cmti-12">Chondroitin sulfate/dermatan sulfate metabolism </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-34-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00023
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-34-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>085541
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-35-1" class="td11">
Integrin alphaIIb beta3 signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-35-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00028
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-35-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>76936
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-36-1" class="td11">
Keratan sulfate biosynthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-36-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00034
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-36-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>68744
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-37-1" class="td11">
Rho GTPase cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-37-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00034
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-37-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>15675
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-38-1" class="td11">
Creation of C4 and C2 activators
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-38-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00035
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-38-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>12275
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-39-1" class="td11">
Abacavir transport and metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-39-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00035
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-39-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>12443
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-40-1" class="td11">
Amine compound SLC transporters
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-40-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00037
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-40-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>69773
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-41-1" class="td11">
FCERI mediated NF-kB activation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-41-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00037
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-41-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>69846
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-42-1" class="td11">
Fc epsilon receptor (FCERI) signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-42-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00056
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-42-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>43303
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-43-1" class="td11">
Defective EXT2 causes exostoses 2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-43-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00067
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-43-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>16053
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-44-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-44-1" class="td11">
Defective EXT1 causes exostoses 1, TRPS2 and CHDS
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-44-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00067
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-44-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>16053
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-45-1" class="td11">
<span class="cmti-12">Collagen biosynthesis and modifying enzymes </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-45-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00071
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-45-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>052911
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-46-1" class="td11">
Keratan sulfate/keratin metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-46-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00073
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-46-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>46533
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-47-1" class="td11">
G alpha (12/13) signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-47-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00078
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-47-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>59164
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-48-1" class="td11">
<strong><span class="cmbx-12">SEMA3A-Plexin repulsion signaling by inhibiting Integrin adhesion</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-48-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00084
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-48-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>038504
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-49-1" class="td11">
Signal attenuation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-49-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00084
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-49-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>37779
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-50-1" class="td11">
Eicosanoid ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-50-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0011
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-50-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>11117
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-105-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-105-51-1" class="td11">
SOS-mediated signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-51-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0011
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-105-51-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>25387
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-105-52-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-105-52-1"  
class="td11">
<pre><code>                                                               &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 1937-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Over-representation (hypergeometric test) and Permutation p-values adjusted for multiple tests across pathways (FDR). Significant</span> <span class="cmr-10x-x-109">pathways are marked in bold (FDR </span><span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05) and italics (FDR </span><span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">1).</span></p>
</p>
</div>
<hr class="endfloat" />
<a 
 id="x1-218003r342"></a>
<h3 class="sectionHead">
<span class="titlemark">F.3 </span> <a 
 id="x1-2190003"></a>Metagene Analysis
</h3>
<!--l. 1949-->
<p class="noindent">
<p>Metagene analysis was also performed for synthetic lethal candidates for <span class="cmti-12">CDH1 </span>expression in stomach cancer. These are described and compared to mutation analysis in Section <a 
href="#x1-2250004">G.4<!--tex4ht:ref: appendix:metagene_stad_mtSL --></a>.</p>
</p>
<!--l. 1951-->
<p class="indent">
<a 
 id="x1-219001r6"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table F.6: </span><span class="content">Candidate synthetic lethal metagenes against <span class="cmti-12">CDH1 </span>from SLIPT in stomach cancer</span></p>
</div>
<!--tex4ht:label?: x1-219001r6 -->
<div class="tabular">
<table id="TBL-106" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-106-1g">
<col 
id="TBL-106-1" />
<col 
id="TBL-106-2" />
<col 
id="TBL-106-3" />
<col 
id="TBL-106-4" />
<col 
id="TBL-106-5" />
<col 
id="TBL-106-6" />
<col 
id="TBL-106-7" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-106-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-1-1" class="td11">
<span class="cmbx-12">Pathway </span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-1-2" class="td11">
ID
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-1-3" class="td11">
Observed
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-1-4" class="td11">
Expected
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-1-5" class="td11">
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-1-6" class="td11">
p-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-1-7" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-2-1" class="td11">
Cell-Cell communication
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-2-2" class="td11">
1500931
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-2-3" class="td11">
18
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-2-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-2-5" class="td11">
110
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-2-6" class="td11">
7<span class="cmmi-12">.</span>43 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">23</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-2-7" class="td11">
1<span class="cmmi-12">.</span>53 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-3-1" class="td11">
VEGFR2 mediated vascular permeability
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-3-2" class="td11">
5218920
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-3-3" class="td11">
19
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-3-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-3-5" class="td11">
109
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-3-6" class="td11">
1<span class="cmmi-12">.</span>36 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">22</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-3-7" class="td11">
2<span class="cmmi-12">.</span>49 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-4-1" class="td11">
Sema4D in semaphorin signaling
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-4-2" class="td11">
400685
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-4-3" class="td11">
20
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-4-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-4-5" class="td11">
104
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-4-6" class="td11">
1<span class="cmmi-12">.</span>62 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-4-7" class="td11">
2<span class="cmmi-12">.</span>12 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-5-1" class="td11">
Ion transport by P-type ATPases
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-5-2" class="td11">
936837
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-5-3" class="td11">
17
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-5-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-5-5" class="td11">
100
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-5-6" class="td11">
8<span class="cmmi-12">.</span>29 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-5-7" class="td11">
8<span class="cmmi-12">.</span>06 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-6-1" class="td11">
Sialic acid metabolism
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-6-2" class="td11">
4085001
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-6-3" class="td11">
19
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-6-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-6-5" class="td11">
95.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-6-6" class="td11">
9<span class="cmmi-12">.</span>95 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-6-7" class="td11">
7<span class="cmmi-12">.</span>82 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-7-1" class="td11">
Synthesis of pyrophosphates in the cytosol
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-7-2" class="td11">
1855167
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-7-3" class="td11">
26
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-7-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-7-5" class="td11">
94
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-7-6" class="td11">
1<span class="cmmi-12">.</span>86 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-7-7" class="td11">
1<span class="cmmi-12">.</span>23 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-8-1" class="td11">
Keratan sulfate/keratin metabolism
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-8-2" class="td11">
1638074
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-8-3" class="td11">
25
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-8-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-8-5" class="td11">
93.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-8-6" class="td11">
2<span class="cmmi-12">.</span>36 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-8-7" class="td11">
1<span class="cmmi-12">.</span>44 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-9-1" class="td11">
Ion channel transport
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-9-2" class="td11">
983712
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-9-3" class="td11">
19
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-9-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-9-5" class="td11">
92.8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-9-6" class="td11">
3<span class="cmmi-12">.</span>37 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-9-7" class="td11">
1<span class="cmmi-12">.</span>99 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-10-1" class="td11">
Keratan sulfate biosynthesis
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-10-2" class="td11">
2022854
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-10-3" class="td11">
26
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-10-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-10-5" class="td11">
91.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-10-6" class="td11">
6<span class="cmmi-12">.</span>79 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-10-7" class="td11">
3<span class="cmmi-12">.</span>62 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-11-1" class="td11">
Arachidonic acid metabolism
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-11-2" class="td11">
2142753
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-11-3" class="td11">
22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-11-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-11-5" class="td11">
90.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-11-6" class="td11">
9<span class="cmmi-12">.</span>81 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-11-7" class="td11">
5<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-12-1" class="td11">
RHO GTPases activate CIT
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-12-2" class="td11">
5625900
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-12-3" class="td11">
22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-12-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-12-5" class="td11">
87
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-12-6" class="td11">
5<span class="cmmi-12">.</span>80 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-12-7" class="td11">
2<span class="cmmi-12">.</span>66 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-13-1" class="td11">
Stimuli-sensing channels
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-13-2" class="td11">
2672351
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-13-3" class="td11">
25
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-13-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-13-5" class="td11">
85.8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-13-6" class="td11">
1<span class="cmmi-12">.</span>03 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-13-7" class="td11">
4<span class="cmmi-12">.</span>58 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-14-1" class="td11">
Synthesis of PI
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-14-2" class="td11">
1483226
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-14-3" class="td11">
19
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-14-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-14-5" class="td11">
85.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-14-6" class="td11">
1<span class="cmmi-12">.</span>15 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-14-7" class="td11">
4<span class="cmmi-12">.</span>89 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-15-1" class="td11">
G-protein activation
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-15-2" class="td11">
202040
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-15-3" class="td11">
19
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-15-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-15-5" class="td11">
85.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-15-6" class="td11">
1<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-15-7" class="td11">
5<span class="cmmi-12">.</span>53 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-16-1" class="td11">
NrCAM interactions
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-16-2" class="td11">
447038
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-16-3" class="td11">
22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-16-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-16-5" class="td11">
84.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-16-6" class="td11">
2<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-16-7" class="td11">
8<span class="cmmi-12">.</span>27 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-17-1" class="td11">
Inwardly rectifying <span class="cmmi-12">K</span><sup><span class="cmr-8">+</span></sup> channels
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-17-2" class="td11">
1296065
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-17-3" class="td11">
24
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-17-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-17-5" class="td11">
83.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-17-6" class="td11">
3<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-17-7" class="td11">
1<span class="cmmi-12">.</span>22 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-18-1" class="td11">
Calcitonin-like ligand receptors
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-18-2" class="td11">
419812
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-18-3" class="td11">
20
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-18-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-18-5" class="td11">
82.2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-18-6" class="td11">
6<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-18-7" class="td11">
2<span class="cmmi-12">.</span>13 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-19-1" class="td11">
Prostacyclin signalling through prostacyclin receptor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-19-2" class="td11">
392851
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-19-3" class="td11">
24
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-19-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-19-5" class="td11">
81.8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-19-6" class="td11">
7<span class="cmmi-12">.</span>27 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-19-7" class="td11">
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-20-1" class="td11">
Presynaptic function of Kainate receptors
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-20-2" class="td11">
500657
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-20-3" class="td11">
26
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-20-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-20-5" class="td11">
79.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-20-6" class="td11">
2<span class="cmmi-12">.</span>00 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-20-7" class="td11">
6<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-21-1" class="td11">
ADP signalling through P2Y purinoceptor 12
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-21-2" class="td11">
392170
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-21-3" class="td11">
23
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-21-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-21-5" class="td11">
79.2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-21-6" class="td11">
2<span class="cmmi-12">.</span>57 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-21-7" class="td11">
7<span class="cmmi-12">.</span>71 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-22-1" class="td11">
regulation of FZD by ubiquitination
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-22-2" class="td11">
4641263
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-22-3" class="td11">
22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-22-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-22-5" class="td11">
78.8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-22-6" class="td11">
3<span class="cmmi-12">.</span>15 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-22-7" class="td11">
9<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-23-1" class="td11">
Toxicity of tetanus toxin (TeNT)
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-23-2" class="td11">
5250982
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-23-3" class="td11">
27
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-23-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-23-5" class="td11">
78.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-23-6" class="td11">
3<span class="cmmi-12">.</span>36 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-23-7" class="td11">
9<span class="cmmi-12">.</span>75 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-24-1" class="td11">
Gap junction degradation
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-24-2" class="td11">
190873
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-24-3" class="td11">
21
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-24-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-24-5" class="td11">
78.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-24-6" class="td11">
3<span class="cmmi-12">.</span>66 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-24-7" class="td11">
1<span class="cmmi-12">.</span>04 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-25-1" class="td11">
Nephrin interactions
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-25-2" class="td11">
373753
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-25-3" class="td11">
25
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-25-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-25-5" class="td11">
78.2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-25-6" class="td11">
4<span class="cmmi-12">.</span>21 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-25-7" class="td11">
1<span class="cmmi-12">.</span>14 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-106-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-106-26-1" class="td11">
GABA synthesis, release, reuptake and degradation
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-106-26-2" class="td11">
888590
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-26-3" class="td11">
26
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-26-4" class="td11">
50.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-26-5" class="td11">
77
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-26-6" class="td11">
7<span class="cmmi-12">.</span>69 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-106-26-7" class="td11">
1<span class="cmmi-12">.</span>95 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-106-27-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-106-27-1"  
class="td11">
<pre><code>                                          &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 2018-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Strongest candidate SL partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">by SLIPT with observed and expected samples with low expression of both genes</span></p>
</p>
</div>
<hr class="endfloat" />
<a 
 id="x1-219002r288"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix G</span><br /><a 
 id="x1-220000G"></a>Stomach Mutation Analysis
</h2>
<!--l. 2027-->
<p class="noindent">
The following results are a replication of the TCGA results (in Appendix <a 
href="#x1-204000D">D<!--tex4ht:ref: appendix:mtSL --></a>) with stomach cancer data, using synthetic lethality (mtSLIPT) against <span class="cmti-12">CDH1 </span>mutation. <a 
 id="x1-220001r348"></a>
</p>
<h3 class="sectionHead">
<span class="titlemark">G.1 </span> <a 
 id="x1-2210001"></a>Synthetic Lethal Genes and Pathways
</h3>
<!--l. 2031-->
<p class="indent">
<a 
 id="x1-221001r1"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table G.1: </span><span class="content">Synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from mtSLIPT in stomach cancer</span></p>
</div>
<!--tex4ht:label?: x1-221001r1 -->
<div class="tabular">
<table id="TBL-107" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-107-1g">
<col 
id="TBL-107-1" />
<col 
id="TBL-107-2" />
<col 
id="TBL-107-3" />
<col 
id="TBL-107-4" />
<col 
id="TBL-107-5" />
<col 
id="TBL-107-6" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-107-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-1-1" class="td11">
<span class="cmbx-12">Gene </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-1-2" class="td11">
Observed
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-1-3" class="td11">
Expected
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-1-4" class="td11">
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-1-5" class="td11">
p-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-1-6" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-2-1" class="td11">
<span class="cmti-12">OLFML1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-2-2" class="td11">
<pre><code>5       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-2-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-2-4" class="td11">
29.2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-2-5" class="td11">
4<span class="cmmi-12">.</span>53 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-2-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0031
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-3-1" class="td11">
<span class="cmti-12">NRIP2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-3-2" class="td11">
<pre><code>6       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-3-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-3-4" class="td11">
25.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-3-5" class="td11">
3<span class="cmmi-12">.</span>11 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-3-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00706
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-4-1" class="td11">
<span class="cmti-12">VIM </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-4-2" class="td11">
<pre><code>3       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-4-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-4-4" class="td11">
24.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-4-5" class="td11">
4<span class="cmmi-12">.</span>29 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-4-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00706
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-5-1" class="td11">
<span class="cmti-12">TCF4 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-5-2" class="td11">
<pre><code>5       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-5-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-5-4" class="td11">
24.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-5-5" class="td11">
4<span class="cmmi-12">.</span>33 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-5-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00706
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-6-1" class="td11">
<span class="cmti-12">ZEB2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-6-2" class="td11">
<pre><code>5       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-6-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-6-4" class="td11">
24.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-6-5" class="td11">
4<span class="cmmi-12">.</span>33 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-6-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00706
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-7-1" class="td11">
<span class="cmti-12">BCL2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-7-2" class="td11">
<pre><code>2       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-7-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-7-4" class="td11">
22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-7-5" class="td11">
1<span class="cmmi-12">.</span>66 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-7-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-8-1" class="td11">
<span class="cmti-12">SMARCA2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-8-2" class="td11">
<pre><code>2       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-8-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-8-4" class="td11">
22
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-8-5" class="td11">
1<span class="cmmi-12">.</span>66 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-8-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-9-1" class="td11">
<span class="cmti-12">CCND2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-9-2" class="td11">
<pre><code>3       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-9-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-9-4" class="td11">
21.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-9-5" class="td11">
2<span class="cmmi-12">.</span>61 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-9-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-10-1" class="td11">
<span class="cmti-12">MMP19 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-10-2" class="td11">
<pre><code>3       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-10-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-10-4" class="td11">
21.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-10-5" class="td11">
2<span class="cmmi-12">.</span>61 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-10-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-11-1" class="td11">
<span class="cmti-12">NEURL1B </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-11-2" class="td11">
<pre><code>3       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-11-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-11-4" class="td11">
21.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-11-5" class="td11">
2<span class="cmmi-12">.</span>61 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-11-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-12-1" class="td11">
<span class="cmti-12">IGFBP6 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-12-2" class="td11">
<pre><code>6       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-12-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-12-4" class="td11">
21.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-12-5" class="td11">
2<span class="cmmi-12">.</span>65 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-12-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-13-1" class="td11">
<span class="cmti-12">OGN </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-13-2" class="td11">
<pre><code>6       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-13-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-13-4" class="td11">
21.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-13-5" class="td11">
2<span class="cmmi-12">.</span>65 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-13-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-14-1" class="td11">
<span class="cmti-12">THY1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-14-2" class="td11">
<pre><code>6       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-14-3&quot;  </code></pre>
class=“td11”&gt; 10.2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-14-4" class="td11">
21
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-14-5" class="td11">
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-14-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-15-1" class="td11">
<span class="cmti-12">DZIP1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-15-2" class="td11">
<pre><code>4       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-15-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-15-4" class="td11">
20.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-15-5" class="td11">
3<span class="cmmi-12">.</span>29 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-15-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-16-1" class="td11">
<span class="cmti-12">LOC650368 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-16-2" class="td11">
<pre><code>4       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-16-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-16-4" class="td11">
20.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-16-5" class="td11">
3<span class="cmmi-12">.</span>29 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-16-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-17-1" class="td11">
<span class="cmti-12">PCOLCE </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-17-2" class="td11">
<pre><code>4       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-17-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-17-4" class="td11">
20.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-17-5" class="td11">
3<span class="cmmi-12">.</span>29 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-17-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-18-1" class="td11">
<span class="cmti-12">PTGFR </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-18-2" class="td11">
<pre><code>4       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-18-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-18-4" class="td11">
20.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-18-5" class="td11">
3<span class="cmmi-12">.</span>29 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-18-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-19-1" class="td11">
<span class="cmti-12">RUNX1T1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-19-2" class="td11">
<pre><code>4       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-19-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-19-4" class="td11">
20.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-19-5" class="td11">
3<span class="cmmi-12">.</span>29 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-19-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-20-1" class="td11">
<span class="cmti-12">CLEC2B </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-20-2" class="td11">
<pre><code>5       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-20-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-20-4" class="td11">
20.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-20-5" class="td11">
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-20-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-21-1" class="td11">
<span class="cmti-12">MSC </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-21-2" class="td11">
<pre><code>5       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-21-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-21-4" class="td11">
20.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-21-5" class="td11">
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-21-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-22-1" class="td11">
<span class="cmti-12">NISCH </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-22-2" class="td11">
<pre><code>5       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-22-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-22-4" class="td11">
20.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-22-5" class="td11">
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-22-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-23-1" class="td11">
<span class="cmti-12">TSPAN11 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-23-2" class="td11">
<pre><code>5       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-23-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-23-4" class="td11">
20.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-23-5" class="td11">
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-23-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0155
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-24-1" class="td11">
<span class="cmti-12">KCTD12 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-24-2" class="td11">
<pre><code>2       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-24-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-24-4" class="td11">
19.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-24-5" class="td11">
7<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-24-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0246
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-25-1" class="td11">
<span class="cmti-12">LRRC55 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-25-2" class="td11">
<pre><code>2       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-25-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-25-4" class="td11">
19.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-25-5" class="td11">
7<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-25-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0246
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-107-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-107-26-1" class="td11">
<span class="cmti-12">PCBP3 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-26-2" class="td11">
<pre><code>2       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-107-26-3&quot;  </code></pre>
class=“td11”&gt; 10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-26-4" class="td11">
19.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-26-5" class="td11">
7<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-107-26-6" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0246
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-107-27-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-107-27-1"  
class="td11">
<pre><code>        &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 2098-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">mtSLIPT partners with observed and expected </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutant samples with low expression</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 2103-->
<p class="indent">
<a 
 id="x1-221002r2"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table G.2: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT in stomach cancer</span></p>
</div>
<!--tex4ht:label?: x1-221002r2 -->
<div class="tabular">
<table id="TBL-108" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-108-1g">
<col 
id="TBL-108-1" />
<col 
id="TBL-108-2" />
<col 
id="TBL-108-3" />
<col 
id="TBL-108-4" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-108-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-1-1" class="td11">
<strong><span class="cmbx-12">Pathways Over-represented</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-1-2" class="td11">
<strong><span class="cmbx-12">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-1-3" class="td11">
<strong><span class="cmbx-12">SL Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-1-4" class="td11">
<strong><span class="cmbx-12">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-108-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-2-1" class="td11">
Extracellular matrix organization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-2-2" class="td11">
<pre><code>  241         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-2-3&quot;  </code></pre>
class=“td11”&gt; 20
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-2-4" class="td11">
<pre><code>9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-108-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-3-1" class="td11">
Elastic fibre formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-3-2" class="td11">
<pre><code>  38          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-3-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-3-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-108-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-4-1" class="td11">
Diseases associated with glycosaminoglycan metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-4-2" class="td11">
<pre><code>  26          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-4-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-4-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-108-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-5-1" class="td11">
Diseases of glycosylation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-5-2" class="td11">
<pre><code>  26          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-5-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-5-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-108-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-6-1" class="td11">
Nitric oxide stimulates guanylate cyclase
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-6-2" class="td11">
<pre><code>  24          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-6-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-6-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-108-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-7-1" class="td11">
Molecules associated with elastic fibres
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-7-2" class="td11">
<pre><code>  34          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-7-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-7-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-108-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-8-1" class="td11">
Platelet homeostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-8-2" class="td11">
<pre><code>  54          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-8-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-8-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-108-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-9-1" class="td11">
Initial triggering of complement
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-9-2" class="td11">
<pre><code>  17          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-9-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-9-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-108-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-10-1" class="td11">
Regulation of IGF transport and uptake by IGFBPs
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-10-2" class="td11">
<pre><code>  17          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-10-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-10-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-108-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-11-1" class="td11">
Collagen degradation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-11-2" class="td11">
<pre><code>  58          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-11-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-11-4" class="td11">
<pre><code>5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-108-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-12-1" class="td11">
Defective B4GALT7 causes EDS, progeroid type
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-12-2" class="td11">
<pre><code>  19          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-12-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-12-4" class="td11">
<pre><code>5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-108-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-13-1" class="td11">
Defective B3GAT3 causes JDSSDHD
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-13-2" class="td11">
<pre><code>  19          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-13-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-13-4" class="td11">
<pre><code>5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-108-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-14-1" class="td11">
Degradation of the extracellular matrix
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-14-2" class="td11">
<pre><code>  104         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-14-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-14-4" class="td11">
<pre><code>8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-108-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-15-1" class="td11">
ECM proteoglycans
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-15-2" class="td11">
<pre><code>  66          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-15-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-108-15-4"  
class="td11">
<pre><code> 0.00017        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-108-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-16-1" class="td11">
A tetrasaccharide linker sequence is required for GAG synthesis <span id="textcolor31">x</span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-16-2" class="td11">
<pre><code>  25          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-16-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-108-16-4"  
class="td11">
<pre><code> 0.00025        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-108-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-17-1" class="td11">
RHO GTPases Activate WASPs and WAVEs
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-17-2" class="td11">
<pre><code>  29          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-17-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-108-17-4"  
class="td11">
<pre><code> 0.00059        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-108-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-18-1" class="td11">
Non-integrin membrane-ECM interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-18-2" class="td11">
<pre><code>  53          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-18-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-108-18-4"  
class="td11">
<pre><code> 0.00065        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-108-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-19-1" class="td11">
Creation of C4 and C2 activators
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-19-2" class="td11">
<pre><code>  11          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-19-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-108-19-4"  
class="td11">
<pre><code> 0.00079        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-108-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-20-1" class="td11">
Dermatan sulfate biosynthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-20-2" class="td11">
<pre><code>  11          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-20-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-108-20-4"  
class="td11">
<pre><code> 0.00079        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-108-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-108-21-1" class="td11">
Integrin cell surface interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-108-21-2" class="td11">
<pre><code>  82          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-108-21-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-108-21-4"  
class="td11">
<pre><code> 0.00098        &lt;/td&gt;</code></pre>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-108-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-108-22-1"  
class="td11">
<pre><code>                                                     &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 2160-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Gene set over-representation analysis (hypergeometric test) for Reactome pathways in mtSLIPT partners for </span><span class="cmti-10x-x-109">CDH1</span></p>
</p>
</div>
<hr class="endfloat" />
<a 
 id="x1-221003r351"></a>
<h3 class="sectionHead">
<span class="titlemark">G.2 </span> <a 
 id="x1-2220002"></a>Synthetic Lethal Expression Profiles
</h3>
<!--l. 2169-->
<p class="noindent">
<p>Similar to the analysis of synthetic lethal partners against low <span class="cmti-12">CDH1 </span>expression in <a 
href="#x1-2160001">F.1<!--tex4ht:ref: appendix:stad_exprSL_genes --></a>, the partners detected from <span class="cmti-12">CDH1 </span>mutation were also examined for their expression profiles and the pathway composition of gene clusters. Hierachical clustering was performed on mtSLIPT partners for <span class="cmti-12">CDH1 </span>as showing in Figure <a 
href="#x1-222001r1">G.1<!--tex4ht:ref: fig:slipt_expr_stad_mtSL --></a>. Over-representation for Reactome pathways for each of the gene clusters identified is given in Table <a 
href="#x1-222002r3">G.3<!--tex4ht:ref: tab:pathway_clusters_stad_mtSL --></a>.</p>
</p>
<!--l. 2172-->
<p class="indent">
<a 
 id="x1-222001r1"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________      <!--l. 2175-->
<p class="noindent">
<img 
src="CDH1_Heatmaps_Genes_Split_By_CDH1_z-trans_mtSL_cordistx_Pub_stad.png" alt="PIC"  
 /> <br />
</p>
<div class="caption">
<p><span class="id">Figure G.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethal expression profiles of analysed samples.</span></strong> <span class="cmr-10x-x-109">Gene</span> <span class="cmr-10x-x-109">expression profile heatmap (correlation distance) of all samples (separated by the</span> <span class="cmr-8">1</span> <span class="cmmi-10x-x-109">∕</span> <span class="cmr-8">3</span> <span class="cmr-10x-x-109">quantile of </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">expression) analysed in TCGA stomach cancer dataset for gene</span> <span class="cmr-10x-x-109">expression of 257 candidate partners of E-cadherin (</span><span class="cmti-10x-x-109">CDH1</span><span class="cmr-10x-x-109">) from SLIPT prediction</span> <span class="cmr-10x-x-109">(with significant FDR adjusted </span><span class="cmmi-10x-x-109">p &lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05). Deeply clustered, inter-correlated genes form</span> <span class="cmr-10x-x-109">several main groups, each containing genes that were SL candidates or toxic in an siRNA</span> <span class="cmr-10x-x-109">screen (</span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">Telford </span><em><span class="cmti-10x-x-109">et</span><span class="cmti-10x-x-109"> al.</span></em></a><span class="cmr-10x-x-109">,</span><span class="cmr-10x-x-109"> </span><a 
href="#XTelford2015"><span class="cmr-10x-x-109">2015</span></a><span class="cmr-10x-x-109">). Clusters had different sample groups highly expressing the</span> <span class="cmr-10x-x-109">synthetic lethal candidates in </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">low samples, notably diffuse and </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutant</span> <span class="cmr-10x-x-109">samples have elevated expression in one or more distinct clusters, although there was less</span> <span class="cmr-10x-x-109">complexity and variation among candidate synthetic lethal partners than in breast data.</span> <span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">low samples also contained most of samples with </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">mutations. </span></span></p>
</div>
<!--tex4ht:label?: x1-222001r1 -->
</div>
<hr class="endfloat" />
<!--l. 2188-->
<p class="indent">
<a 
 id="x1-222002r3"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table G.3: </span><span class="content">Pathway composition for clusters of <span class="cmti-12">CDH1 </span>partners in stomach mtSLIPT</span></p>
</div>
<!--tex4ht:label?: x1-222002r3 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-109" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-109-1g">
<col 
id="TBL-109-1" />
<col 
id="TBL-109-2" />
<col 
id="TBL-109-3" />
<col 
id="TBL-109-4" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-109-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-1-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 1</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-1-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-1-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-1-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-2-1" class="td11">
CD28 dependent PI3K/Akt signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-2-2" class="td11">
<pre><code>    15            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-2-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-2-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-3-1" class="td11">
Hormone-sensitive lipase (HSL)-mediated triacylglycerol hydrolysis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-3-2" class="td11">
<pre><code>    19            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-3-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-3-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-4-1" class="td11">
CD28 co-stimulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-4-2" class="td11">
<pre><code>    26            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-4-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-4-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;&lt;/tr&gt;&lt;tr  </code></pre>
style=“vertical-align:baseline;” id=“TBL-109-5-”&gt;
<td style="white-space:nowrap; text-align:left;" id="TBL-109-5-1" class="td11">
Lipid digestion, mobilization, and transport
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-5-2" class="td11">
48
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-5-3" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-5-4" class="td11">
1
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-6-1" class="td11">
Costimulation by the CD28 family
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-6-2" class="td11">
<pre><code>    51            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-6-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-6-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-7-1" class="td11">
Dectin-1 mediated noncanonical NF-kB signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-7-2" class="td11">
<pre><code>    58            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-7-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-7-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-8-1" class="td11">
CLEC7A (Dectin-1) signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-8-2" class="td11">
<pre><code>    99            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-8-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-8-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-9-1" class="td11">
C-type lectin receptors (CLRs)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-9-2" class="td11">
<pre><code>   123           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-9-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-9-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-10-1" class="td11">
Adaptive Immune System
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-10-2" class="td11">
<pre><code>   418           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-10-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-10-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-11-1" class="td11">
Metabolism of lipids and lipoproteins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-11-2" class="td11">
<pre><code>   494           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-11-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-11-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-12-1" class="td11">
Interleukin-6 signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-12-2" class="td11">
<pre><code>    10            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-12-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-12-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-13-1" class="td11">
Apoptosis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-13-2" class="td11">
<pre><code>   150           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-13-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-13-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-14-1" class="td11">
Hemostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-14-2" class="td11">
<pre><code>   445           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-14-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-14-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;&lt;/tr&gt;&lt;tr  </code></pre>
style=“vertical-align:baseline;” id=“TBL-109-15-”&gt;
<td style="white-space:nowrap; text-align:left;" id="TBL-109-15-1" class="td11">
Intrinsic Pathway for Apoptosis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-15-2" class="td11">
36
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-15-3" class="td11">
0
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-15-4" class="td11">
1
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-16-1" class="td11">
Cleavage of Growing Transcript in the Termination Region
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-16-2" class="td11">
<pre><code>    33            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-16-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-16-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-17-1" class="td11">
PKB-mediated events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-17-2" class="td11">
<pre><code>    28            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-17-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-17-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-18-1" class="td11">
PI3K Cascade
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-18-2" class="td11">
<pre><code>    68            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-18-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-18-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-19-1" class="td11">
RAF/MAP kinase cascade
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-19-2" class="td11">
<pre><code>    10            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-19-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-19-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-20-1" class="td11">
Global Genomic NER (GG-NER)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-20-2" class="td11">
<pre><code>    35            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-20-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-20-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-21-1" class="td11">
Repair synthesis for gap-filling by DNA polymerase in TC-NER
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-21-2" class="td11">
<pre><code>    15            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-21-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-21-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-109-22-1"  
class="td11">
<pre><code>                                                                     &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-109-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-23-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 2</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-23-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-23-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-23-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-24-1" class="td11">
Kinesins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-24-2" class="td11">
<pre><code>    22            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-24-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-24-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;&lt;/tr&gt;&lt;tr  </code></pre>
style=“vertical-align:baseline;” id=“TBL-109-25-”&gt;
<td style="white-space:nowrap; text-align:left;" id="TBL-109-25-1" class="td11">
O-linked glycosylation of mucins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-25-2" class="td11">
49
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-25-3" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-25-4" class="td11">
1
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-26-1" class="td11">
O-linked glycosylation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-26-2" class="td11">
<pre><code>    59            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-26-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-26-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-27-1" class="td11">
MHC class II antigen presentation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-27-2" class="td11">
<pre><code>    85            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-27-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-27-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-28-1" class="td11">
Factors involved in megakaryocyte development and platelet production <span id="textcolor32">cellll</span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-28-2" class="td11">
<pre><code>   120           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-28-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-28-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-29-1" class="td11">
Post-translational protein modification
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-29-2" class="td11">
<pre><code>   303           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-29-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-29-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-30-1" class="td11">
Adaptive Immune System
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-30-2" class="td11">
<pre><code>   418           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-30-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-30-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-31-1" class="td11">
Hemostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-31-2" class="td11">
<pre><code>   445           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-31-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-31-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-32-1" class="td11">
Interleukin-6 signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-32-2" class="td11">
<pre><code>    10            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-32-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-32-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-33-1" class="td11">
Apoptosis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-33-2" class="td11">
<pre><code>   150           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-33-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-33-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-34-1" class="td11">
Intrinsic Pathway for Apoptosis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-34-2" class="td11">
<pre><code>    36            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-34-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-34-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-35-1" class="td11">
Cleavage of Growing Transcript in the Termination Region
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-35-2" class="td11">
<pre><code>    33            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-35-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-35-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-36-1" class="td11">
PKB-mediated events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-36-2" class="td11">
<pre><code>    28            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-36-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-36-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;&lt;/tr&gt;&lt;tr  </code></pre>
style=“vertical-align:baseline;” id=“TBL-109-37-”&gt;
<td style="white-space:nowrap; text-align:left;" id="TBL-109-37-1" class="td11">
PI3K Cascade
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-37-2" class="td11">
68
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-37-3" class="td11">
0
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-37-4" class="td11">
1
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-38-1" class="td11">
RAF/MAP kinase cascade
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-38-2" class="td11">
<pre><code>    10            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-38-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-38-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-39-1" class="td11">
Global Genomic NER (GG-NER)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-39-2" class="td11">
<pre><code>    35            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-39-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-39-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-40-1" class="td11">
Repair synthesis for gap-filling by DNA polymerase in TC-NER
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-40-2" class="td11">
<pre><code>    15            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-40-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-40-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-41-1" class="td11">
Gap-filling DNA repair synthesis and ligation in TC-NER
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-41-2" class="td11">
<pre><code>    17            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-41-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-41-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-42-1" class="td11">
Formation of transcription-coupled NER (TC-NER) repair complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-42-2" class="td11">
<pre><code>    29            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-42-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-42-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-43-1" class="td11">
Dual incision reaction in TC-NER
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-43-2" class="td11">
<pre><code>    29            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-43-3&quot;  </code></pre>
class=“td11”&gt; 0
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-43-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;&lt;/tr&gt;&lt;tr </code></pre>
class=“hline”&gt;
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-109-44-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-44-1" class="td11">
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-109-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-45-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 3</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-45-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-45-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-45-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-109-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-46-1" class="td11">
Extracellular matrix organization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-46-2" class="td11">
<pre><code>   241           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-46-3&quot;  </code></pre>
class=“td11”&gt; 20
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-46-4" class="td11">
<pre><code> 9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-109-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-47-1" class="td11">
Elastic fibre formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-47-2" class="td11">
<pre><code>    38            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-47-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-47-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-109-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-48-1" class="td11">
Diseases associated with glycosaminoglycan metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-48-2" class="td11">
<pre><code>    26            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-48-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-48-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-109-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-49-1" class="td11">
Diseases of glycosylation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-49-2" class="td11">
<pre><code>    26            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-49-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-49-4" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-109-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-50-1" class="td11">
Molecules associated with elastic fibres
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-50-2" class="td11">
<pre><code>    34            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-50-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-50-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-109-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-51-1" class="td11">
Initial triggering of complement
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-51-2" class="td11">
<pre><code>    17            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-51-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-51-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-52-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-109-52-1"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-110" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-110-1g">
<col 
id="TBL-110-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-110-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-110-1-1" class="td00">
Regulation of IGF transport and uptake by IGFBPs
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-52-2" class="td11">
<pre><code>    17            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-52-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-52-4" class="td11">
<pre><code> 4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-109-53-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-53-1" class="td11">
Collagen degradation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-53-2" class="td11">
<pre><code>    58            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-53-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-53-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-109-54-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-54-1" class="td11">
Defective B4GALT7 causes EDS, progeroid type
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-54-2" class="td11">
<pre><code>    19            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-54-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-54-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-109-55-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-55-1" class="td11">
Defective B3GAT3 causes JDSSDHD
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-55-2" class="td11">
<pre><code>    19            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-55-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-55-4" class="td11">
<pre><code> 6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-109-56-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-56-1" class="td11">
Degradation of the extracellular matrix
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-56-2" class="td11">
<pre><code>   104           &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-56-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-56-4" class="td11">
<pre><code> 9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-57-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-57-1" class="td11">
ECM proteoglycans
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-57-2" class="td11">
<pre><code>    66            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-57-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-57-4"  
class="td11">
<pre><code>   0.0002           &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-58-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-58-1" class="td11">
A tetrasaccharide linker sequence is required for GAG synthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-58-2" class="td11">
<pre><code>    25            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-58-3&quot;  </code></pre>
class=“td11”&gt; 5 3
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-58-4"  
class="td11">
<pre><code>  0.00029          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-59-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-59-1" class="td11">
Non-integrin membrane-ECM interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-59-2" class="td11">
<pre><code>    53            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-59-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-59-4"  
class="td11">
<pre><code>  0.00079          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-60-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-60-1" class="td11">
Creation of C4 and C2 activators
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-60-2" class="td11">
<pre><code>    11            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-60-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-60-4"  
class="td11">
<pre><code>  0.00093          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-61-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-61-1" class="td11">
Dermatan sulfate biosynthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-61-2" class="td11">
<pre><code>    11            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-61-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-61-4"  
class="td11">
<pre><code>  0.00093          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-62-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-62-1" class="td11">
Integrin cell surface interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-62-2" class="td11">
<pre><code>    82            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-62-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-62-4"  
class="td11">
<pre><code>   0.0012           &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-63-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-63-1" class="td11">
Keratan sulfate degradation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-63-2" class="td11">
<pre><code>    12            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-63-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-63-4"  
class="td11">
<pre><code>   0.0012           &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-64-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-64-1" class="td11">
Complement cascade
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-64-2" class="td11">
<pre><code>    34            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-64-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-64-4"  
class="td11">
<pre><code>   0.0013           &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-65-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-65-1" class="td11">
CS/DS degradation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-65-2" class="td11">
<pre><code>    13            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-65-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-65-4"  
class="td11">
<pre><code>   0.0015           &lt;/td&gt;</code></pre>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-66-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-109-66-1"  
class="td11">
<pre><code>                                                                     &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-109-67-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-67-1" class="td11">
<strong><span class="cmbx-12x-x-120">Pathways Over-represented in Cluster 4</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-67-2" class="td11">
<strong><span class="cmbx-12x-x-120">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-67-3" class="td11">
<strong><span class="cmbx-12x-x-120">Cluster Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-67-4" class="td11">
<strong><span class="cmbx-12x-x-120">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-68-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-68-1" class="td11">
cGMP effects
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-68-2" class="td11">
<pre><code>    18            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-68-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-68-4"  
class="td11">
<pre><code>    0.11            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-69-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-69-1" class="td11">
Nitric oxide stimulates guanylate cyclase
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-69-2" class="td11">
<pre><code>    24            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-69-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-69-4"  
class="td11">
<pre><code>    0.19            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-70-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-70-1" class="td11">
Neurotoxicity of clostridium toxins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-70-2" class="td11">
<pre><code>    10            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-70-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-70-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-71-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-71-1" class="td11">
Platelet homeostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-71-2" class="td11">
<pre><code>    54            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-71-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-71-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-72-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-72-1" class="td11">
Eicosanoid ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-72-2" class="td11">
<pre><code>    14            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-72-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-72-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-73-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-73-1" class="td11">
Prolactin receptor signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-73-2" class="td11">
<pre><code>    15            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-73-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-73-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-74-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-74-1" class="td11">
Acyl chain remodelling of PI
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-74-2" class="td11">
<pre><code>    15            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-74-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-74-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-75-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-75-1" class="td11">
Signaling by FGFR1 fusion mutants
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-75-2" class="td11">
<pre><code>    15            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-75-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-75-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-76-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-76-1" class="td11">
PKA activation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-76-2" class="td11">
<pre><code>    16            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-76-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-76-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-77-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-77-1" class="td11">
PKA-mediated phosphorylation of CREB
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-77-2" class="td11">
<pre><code>    17            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-77-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-77-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-78-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-78-1" class="td11">
Synthesis of glycosylphosphatidylinositol (GPI)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-78-2" class="td11">
<pre><code>    17            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-78-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-78-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-79-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-79-1" class="td11">
PKA activation in glucagon signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-79-2" class="td11">
<pre><code>    17            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-79-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-79-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-80-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-80-1" class="td11">
Butyrate Response Factor 1 (BRF1) destabilizes mRNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-80-2" class="td11">
<pre><code>    17            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-80-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-80-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-81-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-81-1" class="td11">
Other semaphorin interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-81-2" class="td11">
<pre><code>    19            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-81-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-81-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-82-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-82-1" class="td11">
Acyl chain remodelling of PE
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-82-2" class="td11">
<pre><code>    21            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-82-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-82-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-83-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-83-1" class="td11">
Signaling by Leptin
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-83-2" class="td11">
<pre><code>    21            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-83-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-83-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-84-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-84-1" class="td11">
DARPP-32 events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-84-2" class="td11">
<pre><code>    22            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-84-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-84-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-85-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-85-1" class="td11">
Glucagon-like Peptide-1 (GLP1) regulates insulin secretion
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-85-2" class="td11">
<pre><code>    22            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-85-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-85-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-86-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-86-1" class="td11">
Uptake and actions of bacterial toxins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-86-2" class="td11">
<pre><code>    22            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-86-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-86-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-87-">
<td style="white-space:nowrap; text-align:left;" id="TBL-109-87-1" class="td11">
Acyl chain remodelling of PC
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-109-87-2" class="td11">
<pre><code>    23            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-109-87-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-109-87-4"  
class="td11">
<pre><code>     1              &lt;/td&gt;</code></pre>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-88-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-109-88-1"  
class="td11">
<pre><code>                                                                     &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-109-89-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-109-89-1"  
class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                                       &lt;/div&gt;
                                                                                
                                                                                </code></pre>
</div>
<hr class="endfloat" />
<a 
 id="x1-222003r354"></a>
<h3 class="sectionHead">
<span class="titlemark">G.3 </span> <a 
 id="x1-2230003"></a>Comparison to Primary Screen
</h3>
<!--l. 2383-->
<p class="noindent">
The mutation synthetic lethal partners with <span class="cmti-12">CDH1 </span>were also compared to siRNA primary screen data (<a 
href="#XTelford2015">Telford <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XTelford2015">2015</a>), as performed in Section <a 
href="#x1-1230001">4.2.1<!--tex4ht:ref: chapt3:primary_screen --></a>. These are expected to be more concordant with the experimental results performed on a null mutant, however this not the case at the gene level: less genes overlapped with experimental candidates in Figure <a 
href="#x1-223001r2">G.2<!--tex4ht:ref: fig:Venn_allgenes_stad_mtSL --></a>. This may be affected by lower sample size for mutations in TCGA data or lower frequency (expected value) of <span class="cmti-12">CDH1 </span>mutations compared to low expression.
</p>
<hr class="figure" />
<div class="figure">
<p><a 
 id="x1-223001r2"></a></p>
<hr />
____________________________________________________________________________________      <!--l. 2388-->
<p class="noindent">
<img 
src="Venn_mtSL_siRNA_allgenes_reduced_Pub_stad.png" alt="PIC"  
 /> <br />
</p>
<div class="caption">
<p><span class="id">Figure G.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Comparison of mtSLIPT in stomach to siRNA.</span></strong> <span class="cmr-10x-x-109">Testing the overlap</span> <span class="cmr-10x-x-109">of gene candidates for E-cadherin synthetic lethal partners between computational</span> <span class="cmr-10x-x-109">(mtSLIPT) and experimental screening (siRNA) approaches. The </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test suggests that</span> <span class="cmr-10x-x-109">the overlap is no more than would be expected by chance (</span><span class="cmmi-10x-x-109">p </span><span class="cmr-10x-x-109">= 0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">872). </span></span></p>
</div>
<!--tex4ht:label?: x1-223001r2 -->
</div>
<hr class="endfigure" />
<!--l. 2398-->
<p class="indent">
<a 
 id="x1-223002r4"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table G.4: </span><span class="content">Pathway composition for <span class="cmti-12">CDH1 </span>partners from mtSLIPT and siRNA</span></p>
</div>
<!--tex4ht:label?: x1-223002r4 -->
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-111" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-111-1g">
<col 
id="TBL-111-1" />
<col 
id="TBL-111-2" />
<col 
id="TBL-111-3" />
<col 
id="TBL-111-4" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-111-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-1-1" class="td11">
<span class="cmbx-12">Predicted only by SLIPT (217 genes) </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-1-2" class="td11">
Pathway Size
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-1-3" class="td11">
Genes Identified
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-1-4" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-2-1" class="td11">
Diseases associated with glycosaminoglycan metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-2-2" class="td11">
<pre><code>  26         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-2-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-2-4" class="td11">
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-3-1" class="td11">
Diseases of glycosylation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-3-2" class="td11">
<pre><code>  26         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-3-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-3-4" class="td11">
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-4-1" class="td11">
Extracellular matrix organization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-4-2" class="td11">
<pre><code> 238        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-4-3&quot;  </code></pre>
class=“td11”&gt; 18
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-4-4" class="td11">
1<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-5-1" class="td11">
Elastic fibre formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-5-2" class="td11">
<pre><code>  38         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-5-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-5-4" class="td11">
4<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-6-1" class="td11">
Initial triggering of complement
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-6-2" class="td11">
<pre><code>  16         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-6-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-6-4" class="td11">
7<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-7-1" class="td11">
Regulation of IGF transport and uptake by IGFBPs
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-7-2" class="td11">
<pre><code>  17         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-7-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-7-4" class="td11">
8<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-8-1" class="td11">
Defective B4GALT7 causes EDS, progeroid type
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-8-2" class="td11">
<pre><code>  19         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-8-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-8-4" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00013
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-9-1" class="td11">
Defective B3GAT3 causes JDSSDHD
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-9-2" class="td11">
<pre><code>  19         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-9-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-9-4" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00013
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-10-1" class="td11">
Collagen degradation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-10-2" class="td11">
57
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-10-3" class="td11">
5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-10-4" class="td11">
0<span class="cmmi-12">.</span>00013
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-11-1" class="td11">
ECM proteoglycans
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-11-2" class="td11">
<pre><code>  65         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-11-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-11-4" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00039
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-12-1" class="td11">
A tetrasaccharide linker sequence is required for GAG synthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-12-2" class="td11">
<pre><code>  24         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-12-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-12-4" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00039
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-13-1" class="td11">
Nitric oxide stimulates guanylate cyclase
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-13-2" class="td11">
<pre><code>  24         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-13-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-13-4" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00039
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-14-1" class="td11">
RHO GTPases Activate WASPs and WAVEs
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-14-2" class="td11">
<pre><code>  28         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-14-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-14-4" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00094
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-15-1" class="td11">
Creation of C4 and C2 activators
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-15-2" class="td11">
<pre><code>  10         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-15-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-15-4" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00098
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-16-1" class="td11">
Non-integrin membrane-ECM interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-16-2" class="td11">
52
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-16-3" class="td11">
4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-16-4" class="td11">
0<span class="cmmi-12">.</span>0012
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-17-1" class="td11">
Dermatan sulfate biosynthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-17-2" class="td11">
<pre><code>  11         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-17-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-17-4" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0013
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-18-1" class="td11">
Degradation of the extracellular matrix
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-18-2" class="td11">
<pre><code> 101        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-18-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-18-4" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0016
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-19-1" class="td11">
Keratan sulfate degradation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-19-2" class="td11">
<pre><code>  12         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-19-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-19-4" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0016
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-20-1" class="td11">
Complement cascade
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-20-2" class="td11">
<pre><code>  33         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-20-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-20-4" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0018
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-21-1" class="td11">
Molecules associated with elastic fibres
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-21-2" class="td11">
<pre><code>  34         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-21-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-21-4" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>002
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-111-22-1"  
class="td11">
<pre><code>                                                        &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-23-1" class="td11">
<span class="cmbx-12">Detected only by siRNA screen (2323 genes) </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-23-2" class="td11">
Pathway Size
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-23-3" class="td11">
Genes Identified
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-23-4" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-24-1" class="td11">
Class A/1 (Rhodopsin-like receptors)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-24-2" class="td11">
<pre><code> 282        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-24-3&quot;  </code></pre>
class=“td11”&gt; 86
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-24-4" class="td11">
6<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">85</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-25-1" class="td11">
GPCR ligand binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-25-2" class="td11">
<pre><code> 363        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-25-3&quot;  </code></pre>
class=“td11”&gt; 97
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-25-4" class="td11">
9<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">79</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-26-1" class="td11">
Peptide ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-26-2" class="td11">
<pre><code> 175        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-26-3&quot;  </code></pre>
class=“td11”&gt; 52
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-26-4" class="td11">
4<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">61</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-27-1" class="td11">
G<sub><span class="cmmi-8">αi</span></sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-27-2" class="td11">
<pre><code> 184        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-27-3&quot;  </code></pre>
class=“td11”&gt; 49
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-27-4" class="td11">
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">53</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-28-1" class="td11">
G<sub><span class="cmmi-8">αq</span></sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-28-2" class="td11">
<pre><code> 159        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-28-3&quot;  </code></pre>
class=“td11”&gt; 43
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-28-4" class="td11">
5<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">50</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-29-1" class="td11">
Gastrin-CREB signalling pathway via PKC and MAPK
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-29-2" class="td11">
<pre><code> 180        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-29-3&quot;  </code></pre>
class=“td11”&gt; 46
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-29-4" class="td11">
9<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">50</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-30-1" class="td11">
DAP12 interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-30-2" class="td11">
<pre><code> 159        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-30-3&quot;  </code></pre>
class=“td11”&gt; 35
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-30-4" class="td11">
8<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">37</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-31-1" class="td11">
Platelet activation, signaling and aggregation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-31-2" class="td11">
<pre><code> 182        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-31-3&quot;  </code></pre>
class=“td11”&gt; 37
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-31-4" class="td11">
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">35</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-32-1" class="td11">
Hemostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-32-2" class="td11">
<pre><code> 438        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-32-3&quot;  </code></pre>
class=“td11”&gt; 71
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-32-4" class="td11">
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">35</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-33-1" class="td11">
Downstream signal transduction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-33-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-33-3&quot;  </code></pre>
class=“td11”&gt; 32
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-33-4" class="td11">
7<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">35</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-34-1" class="td11">
Signaling by PDGF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-34-2" class="td11">
<pre><code> 172        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-34-3&quot;  </code></pre>
class=“td11”&gt; 35
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-34-4" class="td11">
2<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">34</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-35-1" class="td11">
DAP12 signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-35-2" class="td11">
<pre><code> 149        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-35-3&quot;  </code></pre>
class=“td11”&gt; 32
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-35-4" class="td11">
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">34</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-36-1" class="td11">
Signaling by ERBB2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-36-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-36-3&quot;  </code></pre>
class=“td11”&gt; 31
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-36-4" class="td11">
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">33</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-37-1" class="td11">
Signalling by NGF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-37-2" class="td11">
<pre><code> 266        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-37-3&quot;  </code></pre>
class=“td11”&gt; 44
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-37-4" class="td11">
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-38-1" class="td11">
Downstream signaling of activated FGFR1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-38-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-38-3&quot;  </code></pre>
class=“td11”&gt; 28
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-38-4" class="td11">
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-39-1" class="td11">
Downstream signaling of activated FGFR2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-39-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-39-3&quot;  </code></pre>
class=“td11”&gt; 28
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-39-4" class="td11">
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-40-1" class="td11">
Downstream signaling of activated FGFR3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-40-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-40-3&quot;  </code></pre>
class=“td11”&gt; 28
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-40-4" class="td11">
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-41-1" class="td11">
Downstream signaling of activated FGFR4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-41-2" class="td11">
<pre><code> 134        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-41-3&quot;  </code></pre>
class=“td11”&gt; 28
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-41-4" class="td11">
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">31</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-42-1" class="td11">
Signaling by FGFR
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-42-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-42-3&quot;  </code></pre>
class=“td11”&gt; 29
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-42-4" class="td11">
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-43-1" class="td11">
Signaling by FGFR1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-43-2" class="td11">
<pre><code> 146        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-43-3&quot;  </code></pre>
class=“td11”&gt; 29
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-43-4" class="td11">
2<span class="cmmi-12">.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">30</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-44-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-111-44-1"  
class="td11">
<pre><code>                                                        &lt;/td&gt;</code></pre>
</tr>
<tr style="vertical-align:baseline;" id="TBL-111-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-45-1" class="td11">
<span class="cmbx-12">Intersection of SLIPT and siRNA screen (23 genes) </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-45-2" class="td11">
Pathway Size
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-45-3" class="td11">
Genes Identified
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-45-4" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-46-1" class="td11">
ADP signalling through P2Y purinoceptor 1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-46-2" class="td11">
<pre><code>  10         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-46-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-46-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-47-1" class="td11">
G-protein beta:gamma signalling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-47-2" class="td11">
<pre><code>  11         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-47-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-47-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-48-1" class="td11">
G-protein activation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-48-2" class="td11">
<pre><code>  12         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-48-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-48-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-49-1" class="td11">
Eicosanoid ligand-binding receptors
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-49-2" class="td11">
<pre><code>  14         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-49-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-49-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-50-1" class="td11">
Platelet homeostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-50-2" class="td11">
<pre><code>  53         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-50-3&quot;  </code></pre>
class=“td11”&gt; 2
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-50-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-51-1" class="td11">
G<sub><span class="cmmi-8">αz</span> </sub> signalling events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-51-2" class="td11">
<pre><code>  15         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-51-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-51-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-52-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-52-1" class="td11">
Signal amplification
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-52-2" class="td11">
<pre><code>  16         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-52-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-52-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-53-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-53-1" class="td11">
Activation of Kainate Receptors upon glutamate binding
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-53-2" class="td11">
<pre><code>  17         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-53-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-53-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-54-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-54-1" class="td11">
Thrombin signalling through proteinase activated receptors (PARs)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-54-2" class="td11">
<pre><code>  17         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-54-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-54-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-55-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-55-1" class="td11">
Nitric oxide stimulates guanylate cyclase
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-55-2" class="td11">
<pre><code>  24         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-55-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-55-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-56-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-56-1" class="td11">
Activation of G protein gated Potassium channels
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-56-2" class="td11">
<pre><code>  25         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-56-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-56-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-57-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-57-1" class="td11">
G protein gated Potassium channels
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-57-2" class="td11">
<pre><code>  25         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-57-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-57-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-58-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-58-1" class="td11">
Inhibition of voltage gated Ca2<sup><span class="cmr-8">+</span></sup> channels via Gbeta/gamma subunits
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-58-2" class="td11">
<pre><code>  25         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-58-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-58-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-59-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-59-1" class="td11">
Laminin interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-59-2" class="td11">
<pre><code>  29         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-59-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-59-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-60-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-60-1" class="td11">
Inwardly rectifying K<sup><span class="cmr-8">+</span></sup> channels
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-60-2" class="td11">
<pre><code>  31         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-60-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-60-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-61-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-61-1" class="td11">
Glucagon signaling in metabolic regulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-61-2" class="td11">
<pre><code>  33         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-61-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-61-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-62-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-62-1" class="td11">
Molecules associated with elastic fibres
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-62-2" class="td11">
<pre><code>  34         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-62-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-62-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-63-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-63-1" class="td11">
Ca2<sup><span class="cmr-8">+</span> </sup> pathway
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-63-2" class="td11">
<pre><code>  36         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-63-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-63-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-64-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-64-1" class="td11">
Elastic fibre formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-64-2" class="td11">
<pre><code>  38         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-64-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-64-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-65-">
<td style="white-space:nowrap; text-align:left;" id="TBL-111-65-1" class="td11">
GABA B receptor activation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-111-65-2" class="td11">
<pre><code>  38         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-111-65-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-111-65-4"  
class="td11">
<pre><code>   1          &lt;/td&gt;</code></pre>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-111-66-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-111-66-1"  
class="td11">
<pre><code>                                                        &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;                                                         &lt;/div&gt;
                                                                                
                                                                                </code></pre>
</div>
<hr class="endfloat" />
<a 
 id="x1-223003r345"></a>
<h4 class="subsectionHead">
<span class="titlemark">G.3.1 </span> <a 
 id="x1-2240001"></a>Resampling Analysis
</h4>
<!--l. 2550-->
<p class="indent">
<a 
 id="x1-224001r5"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table G.5: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT in stomach cancer</span></p>
</div>
<!--tex4ht:label?: x1-224001r5 -->
<div class="tabular">
<table id="TBL-112" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-112-1g">
<col 
id="TBL-112-1" />
<col 
id="TBL-112-2" />
<col 
id="TBL-112-3" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-112-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-1-1" class="td11">
<span class="cmbx-12">Reactome Pathway </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-1-2" class="td11">
Over-representation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-1-3" class="td11">
Permutation
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-2-1" class="td11">
<span class="cmti-12">Extracellular matrix organization </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-2-2" class="td11">
<pre><code> 9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-2-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>057678
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-3-1" class="td11">
<strong><span class="cmbx-12">Elastic fibre formation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-3-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-3-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>033817
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-4-1" class="td11">
<strong><span class="cmbx-12">Diseases associated with glycosaminoglycan metabolism</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-4-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-4-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>049336
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-5-1" class="td11">
<strong><span class="cmbx-12">Diseases of glycosylation</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-5-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-5-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>049336
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-6-1" class="td11">
<strong><span class="cmbx-12">Nitric oxide stimulates guanylate cyclase</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-6-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-6-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>037904
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-7-1" class="td11">
<strong><span class="cmbx-12">Initial triggering of complement</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-7-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-7-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>020828
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-8-1" class="td11">
<strong><span class="cmbx-12">Molecules associated with elastic fibres</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-8-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-8-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>027865
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-9-1" class="td11">
<span class="cmti-12">Regulation of IGF transport and uptake by IGFBPs </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-9-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-9-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>069102
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-10-1" class="td11">
<span class="cmti-12">Platelet homeostasis </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-10-2" class="td11">
<pre><code> 3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-10-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>097294
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-11-1" class="td11">
<span class="cmti-12">Defective B4GALT7 causes EDS, progeroid type </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-11-2" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-11-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>081505
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-12-1" class="td11">
<span class="cmti-12">Defective B3GAT3 causes JDSSDHD </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-12-2" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-12-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>081505
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-13-1" class="td11">
Collagen degradation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-13-2" class="td11">
<pre><code> 5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-13-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1104
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-14-1" class="td11">
Degradation of the extracellular matrix
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-14-2" class="td11">
<pre><code>  8 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-14-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>43477
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-15-1" class="td11">
ECM proteoglycans
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-15-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00017
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-15-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>06469
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-16-1" class="td11">
A tetrasaccharide linker sequence is required for GAG synthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-16-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00025
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-16-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>10536
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-17-1" class="td11">
<span class="cmti-12">RHO GTPases Activate WASPs and WAVEs </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-17-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00059
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-17-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>053929
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-18-1" class="td11">
Non-integrin membrane-ECM interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-18-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00065
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-18-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>10424
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-19-1" class="td11">
<span class="cmti-12">Creation of C4 and C2 activators </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-19-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00079
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-19-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>05461
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-20-1" class="td11">
Dermatan sulfate biosynthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-20-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00079
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-20-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>21163
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-21-1" class="td11">
<span class="cmti-12">Integrin cell surface interactions </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-21-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00098
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-21-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>092405
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-22-1" class="td11">
Glucagon signaling in metabolic regulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-22-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00098
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-22-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>13425
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-23-1" class="td11">
Keratan sulfate degradation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-23-2" class="td11">
<pre><code>  0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00098
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-23-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>22137
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-24-1" class="td11">
<strong><span class="cmbx-12">Complement cascade</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-24-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0011
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-24-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>01552
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-25-1" class="td11">
<span class="cmti-12">CS/DS degradation </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-25-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0012
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-25-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>065012
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-26-1" class="td11">
<span class="cmti-12">Eicosanoid ligand-binding receptors </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-26-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0016
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-26-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>066128
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-27-1" class="td11">
Nuclear signaling by ERBB4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-27-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0016
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-27-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>15511
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-28-1" class="td11">
Collagen formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-28-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0026
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-28-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>13447
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-29-1" class="td11">
<strong><span class="cmbx-12">cGMP effects</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-29-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0041
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-29-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>020195
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-30-1" class="td11">
<span class="cmti-12">Voltage gated Potassium channels </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-30-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0041
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-30-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>068923
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-31-1" class="td11">
<strong><span class="cmbx-12">Chondroitin sulfate biosynthesis</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-31-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0059
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-31-3" class="td11">
<span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>5862 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-32-1" class="td11">
<span class="cmti-12">Chondroitin sulfate/dermatan sulfate metabolism </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-32-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0065
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-32-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>087745
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-33-1" class="td11">
<span class="cmti-12">Heparan sulfate/heparin (HS-GAG) metabolism </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-33-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0071
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-33-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>085622
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-34-1" class="td11">
<span class="cmti-12">Synthesis of substrates in N-glycan biosythesis </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-34-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0085
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-34-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>09456
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-35-1" class="td11">
<span class="cmti-12">Regulation of actin dynamics for phagocytic cup formation </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-35-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0085
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-35-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>096227
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-36-1" class="td11">
CDO in myogenesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-36-2" class="td11">
<pre><code>    0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>01
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-36-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>32599
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-37-1" class="td11">
Myogenesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-37-2" class="td11">
<pre><code>    0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>01
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-37-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>32599
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-38-1" class="td11">
Syndecan interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-38-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>012
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-38-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>10975
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-39-1" class="td11">
Activation of Matrix Metalloproteinases
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-39-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>012
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-39-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>33499
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-40-1" class="td11">
Glycosaminoglycan metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-40-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>012
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-40-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>29716
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-41-1" class="td11">
Collagen biosynthesis and modifying enzymes
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-41-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-41-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>10774
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-42-1" class="td11">
Keratan sulfate biosynthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-42-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>016
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-42-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>12644
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-43-1" class="td11">
O-linked glycosylation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-43-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>016
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-43-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>65101
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-44-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-44-1" class="td11">
Laminin interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-44-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>021
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-44-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>12766
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-112-45-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-112-45-1"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-113" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-113-1g">
<col 
id="TBL-113-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-113-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-113-1-1" class="td00">
<span class="cmti-12">Biosynthesis of the N-glycan precursor (dolichol lipid-linked oligosaccharide)</span>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-113-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-113-2-1" class="td00">
<span class="cmti-12">and transfer to a nascent protein</span>
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-45-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>027
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-45-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>065782
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-46-1" class="td11">
Sialic acid metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-46-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>027
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-46-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>13413
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-47-1" class="td11">
Keratan sulfate/keratin metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-47-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>029
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-47-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>15708
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-48-1" class="td11">
Potassium Channels
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-48-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>032
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-48-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>43477
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-49-1" class="td11">
Fcgamma receptor (FCGR) dependent phagocytosis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-49-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>042
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-49-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>15851
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-50-1" class="td11">
Ion transport by P-type ATPases
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-50-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>048
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-50-3" class="td11">
<pre><code> 0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>66686
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-112-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-112-51-1" class="td11">
<span class="cmti-12">Retinoid metabolism and transport </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-51-2" class="td11">
<pre><code>   0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>051
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-112-51-3" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>058715
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-112-52-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-112-52-1"  
class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 2668-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Over-representation (hypergeometric test) and Permutation p-values adjusted for multiple tests across pathways (FDR). Significant pathways are</span> <span class="cmr-10x-x-109">marked in bold (FDR </span><span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">05) and italics (FDR </span><span class="cmmi-10x-x-109">&lt; </span><span class="cmr-10x-x-109">0</span><span class="cmmi-10x-x-109">.</span><span class="cmr-10x-x-109">1).</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 2671-->
<p class="indent">
<a 
 id="x1-224002r6"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table G.6: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from mtSLIPT in stomach and siRNA screen</span></p>
</div>
<!--tex4ht:label?: x1-224002r6 -->
<div class="tabular">
<table id="TBL-114" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-114-1g">
<col 
id="TBL-114-1" />
<col 
id="TBL-114-2" />
<col 
id="TBL-114-3" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-114-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-1-1" class="td11">
<span class="cmbx-12">Reactome Pathway </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-1-2" class="td11">
Over-representation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-1-3" class="td11">
Permutation
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-2-1" class="td11">
SLBP independent Processing of Histone Pre-mRNAs
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-2-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-2-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-3-1" class="td11">
Mitochondrial protein import
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-3-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-3-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-4-1" class="td11">
Voltage gated Potassium channels
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-4-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-4-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-5-1" class="td11">
Tandem pore domain potassium channels
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-5-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-5-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-6-1" class="td11">
L13a-mediated translational silencing of Ceruloplasmin expression
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-6-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-6-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-7-1" class="td11">
Eukaryotic Translation Elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-7-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-7-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-8-1" class="td11">
Peptide chain elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-8-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-8-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-9-1" class="td11">
3’ -UTR-mediated translational regulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-9-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-9-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-10-1" class="td11">
Activation of Matrix Metalloproteinases
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-10-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-10-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-11-1" class="td11">
HIV Infection
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-11-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-11-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-12-1" class="td11">
Cell Cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-12-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-12-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-13-1" class="td11">
Influenza Infection
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-13-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-13-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-14-1" class="td11">
Influenza Life Cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-14-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-14-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-15-1" class="td11">
Influenza Viral RNA Transcription and Replication
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-15-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-15-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-16-1" class="td11">
Neurotoxicity of clostridium toxins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-16-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-16-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-17-1" class="td11">
p38MAPK events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-17-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-17-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-18-1" class="td11">
SCF-beta-TrCP mediated degradation of Emi1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-18-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-18-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-19-1" class="td11">
SRP-dependent cotranslational protein targeting to membrane
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-19-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-19-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-20-1" class="td11">
Vpu mediated degradation of CD4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-20-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-20-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-21-1" class="td11">
Serotonin Neurotransmitter Release Cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-21-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-21-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-22-1" class="td11">
Acetylcholine Binding And Downstream Events
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-22-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-22-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-23-1" class="td11">
Viral mRNA Translation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-23-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-23-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-24-1" class="td11">
Cobalamin (Cbl, vitamin B12) transport and metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-24-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-24-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-25-1" class="td11">
ERK/MAPK targets
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-25-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-25-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-26-1" class="td11">
Vitamin B5 (pantothenate) metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-26-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-26-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-27-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-27-1" class="td11">
Signaling by BMP
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-27-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-27-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-28-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-28-1" class="td11">
Synthesis of Leukotrienes (LT) and Eoxins (EX)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-28-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-28-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-29-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-29-1" class="td11">
Separation of Sister Chromatids
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-29-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-29-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-30-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-30-1" class="td11">
Mitotic Metaphase and Anaphase
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-30-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-30-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-31-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-31-1" class="td11">
TRP channels
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-31-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-31-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-32-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-32-1" class="td11">
Defects in cobalamin (B12) metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-32-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-32-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-33-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-33-1" class="td11">
Regulation by c-FLIP
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-33-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-33-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-34-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-34-1" class="td11">
Attenuation phase
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-34-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-34-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-35-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-35-1" class="td11">
Autodegradation of the E3 ubiquitin ligase COP1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-35-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-35-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-36-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-36-1" class="td11">
Apoptotic cleavage of cell adhesion proteins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-36-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-36-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-37-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-37-1" class="td11">
Negative regulation of TCF-dependent signaling by WNT ligand antagonists
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-37-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-37-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-38-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-38-1" class="td11">
PERK regulates gene expression
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-38-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-38-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-39-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-39-1" class="td11">
Regulation of the Fanconi anemia pathway
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-39-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-39-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-40-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-40-1" class="td11">
Passive transport by Aquaporins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-40-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-40-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-41-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-41-1" class="td11">
Lysosome Vesicle Biogenesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-41-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-41-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-42-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-42-1" class="td11">
Zinc transporters
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-42-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-42-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-43-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-43-1" class="td11">
Zinc influx into cells by the SLC39 gene family
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-43-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-43-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-44-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-44-1" class="td11">
Asparagine N-linked glycosylation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-44-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-44-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-45-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-45-1" class="td11">
AUF1 (hnRNP D0) destabilizes mRNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-45-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-45-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-46-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-46-1" class="td11">
Asymmetric localization of PCP proteins
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-46-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-46-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-47-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-47-1" class="td11">
degradation of DVL
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-47-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-47-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-48-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-48-1" class="td11">
CASP8 activity is inhibited
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-48-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-48-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-49-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-49-1" class="td11">
Degradation of GLI1 by the proteasome
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-49-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-49-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-50-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-50-1" class="td11">
BBSome-mediated cargo-targeting to cilium
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-50-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-50-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-114-51-">
<td style="white-space:nowrap; text-align:left;" id="TBL-114-51-1" class="td11">
Regulation of necroptotic cell death
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-114-51-2" class="td11">
<pre><code>     1              &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-114-51-3&quot;  </code></pre>
class=“td11”&gt; <span class="cmmi-12">&gt; </span>1<span class="cmmi-12">.</span>2349 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-114-52-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-114-52-1"  
class="td11">
<pre><code>                                                              &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 2788-->
<p class="noindent">
</div>
</p>
</div>
<hr class="endfloat" />
<a 
 id="x1-224003r357"></a>
<h3 class="sectionHead">
<span class="titlemark">G.4 </span> <a 
 id="x1-2250004"></a>Metagene Analysis
</h3>
<!--l. 2795-->
<p class="noindent">
<p>Metagene analysis was also performed for synthetic lethal candidates for <span class="cmti-12">CDH1 </span>mutation in stomach cancer. These are described and compared to expression analysis in Section <a 
href="#x1-2190003">F.3<!--tex4ht:ref: appendix:metagene_stad_exprSL --></a>.</p>
</p>
<!--l. 2797-->
<p class="indent">
<a 
 id="x1-225001r7"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table G.7: </span><span class="content">Candidate synthetic lethal metagenes against <span class="cmti-12">CDH1 </span>from mtSLIPT in stomach cancer</span></p>
</div>
<!--tex4ht:label?: x1-225001r7 -->
<div class="tabular">
<table id="TBL-115" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-115-1g">
<col 
id="TBL-115-1" />
<col 
id="TBL-115-2" />
<col 
id="TBL-115-3" />
<col 
id="TBL-115-4" />
<col 
id="TBL-115-5" />
<col 
id="TBL-115-6" />
<col 
id="TBL-115-7" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-115-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-1-1" class="td11">
<span class="cmbx-12">Pathway </span>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-1-2" class="td11">
ID
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-1-3" class="td11">
Observed
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-1-4" class="td11">
Expected
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-1-5" class="td11">
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup>value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-1-6" class="td11">
p-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-1-7" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-2-1" class="td11">
Prostacyclin signalling through prostacyclin receptor
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-2-2" class="td11">
392851
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-2-3" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-2-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-2-5" class="td11">
26.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-2-6" class="td11">
1<span class="cmmi-12">.</span>73 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-2-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00286
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-3-1" class="td11">
Cell surface interactions at the vascular wall
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-3-2" class="td11">
202733
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-3-3" class="td11">
3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-3-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-3-5" class="td11">
21.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-3-6" class="td11">
2<span class="cmmi-12">.</span>61 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-3-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00642
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-4-1" class="td11">
The NLRP1 inflammasome
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-4-2" class="td11">
844455
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-4-3" class="td11">
3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-4-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-4-5" class="td11">
21.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-4-6" class="td11">
2<span class="cmmi-12">.</span>61 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-4-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00642
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-5-1" class="td11">
Innate Immune System
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-5-2" class="td11">
168249
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-5-3" class="td11">
6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-5-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-5-5" class="td11">
21.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-5-6" class="td11">
2<span class="cmmi-12">.</span>65 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-5-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00642
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-6-1" class="td11">
Keratan sulfate/keratin metabolism
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-6-2" class="td11">
1638074
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-6-3" class="td11">
4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-6-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-6-5" class="td11">
20.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-6-6" class="td11">
3<span class="cmmi-12">.</span>29 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-6-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00642
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-7-1" class="td11">
Keratan sulfate biosynthesis
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-7-2" class="td11">
2022854
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-7-3" class="td11">
4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-7-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-7-5" class="td11">
20.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-7-6" class="td11">
3<span class="cmmi-12">.</span>29 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-7-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00642
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-8-1" class="td11">
Signaling by SCF-KIT
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-8-2" class="td11">
1433557
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-8-3" class="td11">
5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-8-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-8-5" class="td11">
20.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-8-6" class="td11">
3<span class="cmmi-12">.</span>30 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-8-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00642
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-9-1" class="td11">
VEGFA-VEGFR2 Pathway
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-9-2" class="td11">
4420097
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-9-3" class="td11">
5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-9-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-9-5" class="td11">
20.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-9-6" class="td11">
3<span class="cmmi-12">.</span>30 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-9-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00642
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-10-1" class="td11">
p130Cas linkage to MAPK signaling for integrins
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-10-2" class="td11">
372708
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-10-3" class="td11">
2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-10-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-10-5" class="td11">
19.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-10-6" class="td11">
7<span class="cmmi-12">.</span>19 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-10-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-11-1" class="td11">
cGMP effects
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-11-2" class="td11">
418457
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-11-3" class="td11">
8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-11-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-11-5" class="td11">
19
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-11-6" class="td11">
7<span class="cmmi-12">.</span>46 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-11-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-115-12-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-115-12-1"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-116" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-116-1g">
<col 
id="TBL-116-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-116-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-116-1-1" class="td00">
Regulation of cytoskeletal remodeling and cell spreading by IPP
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-116-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-116-2-1" class="td00">
complex components
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-12-2" class="td11">
446388
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-12-3" class="td11">
8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-12-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-12-5" class="td11">
19
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-12-6" class="td11">
7<span class="cmmi-12">.</span>46 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-12-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-13-1" class="td11">
Fcgamma receptor (FCGR) dependent phagocytosis
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-13-2" class="td11">
2029480
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-13-3" class="td11">
3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-13-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-13-5" class="td11">
17.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-13-6" class="td11">
0<span class="cmmi-12">.</span>000127
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-13-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-14-1" class="td11">
A third proteolytic cleavage releases NICD
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-14-2" class="td11">
157212
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-14-3" class="td11">
7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-14-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-14-5" class="td11">
17.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-14-6" class="td11">
0<span class="cmmi-12">.</span>00013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-14-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-15-1" class="td11">
Signalling by NGF
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-15-2" class="td11">
166520
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-15-3" class="td11">
7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-15-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-15-5" class="td11">
17.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-15-6" class="td11">
0<span class="cmmi-12">.</span>00013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-15-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-16-1" class="td11">
Signaling by VEGF
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-16-2" class="td11">
194138
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-16-3" class="td11">
7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-16-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-16-5" class="td11">
17.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-16-6" class="td11">
0<span class="cmmi-12">.</span>00013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-16-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-17-1" class="td11">
Regulation of thyroid hormone activity
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-17-2" class="td11">
350864
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-17-3" class="td11">
7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-17-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-17-5" class="td11">
17.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-17-6" class="td11">
0<span class="cmmi-12">.</span>00013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-17-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-18-1" class="td11">
Nitric oxide stimulates guanylate cyclase
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-18-2" class="td11">
392154
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-18-3" class="td11">
7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-18-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-18-5" class="td11">
17.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-18-6" class="td11">
0<span class="cmmi-12">.</span>00013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-18-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-19-1" class="td11">
Platelet homeostasis
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-19-2" class="td11">
418346
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-19-3" class="td11">
7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-19-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-19-5" class="td11">
17.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-19-6" class="td11">
0<span class="cmmi-12">.</span>00013
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-19-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-20-1" class="td11">
PI3K events in ERBB4 signaling
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-20-2" class="td11">
1250342
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-20-3" class="td11">
4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-20-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-20-5" class="td11">
17.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-20-6" class="td11">
0<span class="cmmi-12">.</span>000179
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-20-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-21-1" class="td11">
PIP3 activates AKT signaling
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-21-2" class="td11">
1257604
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-21-3" class="td11">
4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-21-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-21-5" class="td11">
17.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-21-6" class="td11">
0<span class="cmmi-12">.</span>000179
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-21-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-22-1" class="td11">
GAB1 signalosome
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-22-2" class="td11">
180292
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-22-3" class="td11">
4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-22-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-22-5" class="td11">
17.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-22-6" class="td11">
0<span class="cmmi-12">.</span>000179
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-22-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-23-1" class="td11">
PI3K events in ERBB2 signaling
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-23-2" class="td11">
1963642
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-23-3" class="td11">
4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-23-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-23-5" class="td11">
17.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-23-6" class="td11">
0<span class="cmmi-12">.</span>000179
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-23-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-24-1" class="td11">
PI3K/AKT Signaling in Cancer
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-24-2" class="td11">
2219528
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-24-3" class="td11">
4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-24-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-24-5" class="td11">
17.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-24-6" class="td11">
0<span class="cmmi-12">.</span>000179
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-24-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-25-1" class="td11">
Rap1 signalling
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-25-2" class="td11">
392517
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-25-3" class="td11">
4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-25-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-25-5" class="td11">
17.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-25-6" class="td11">
0<span class="cmmi-12">.</span>000179
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-25-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-115-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-115-26-1" class="td11">
Lysosphingolipid and LPA receptors
</td>
<td style="white-space:nowrap; text-align:left;" id="TBL-115-26-2" class="td11">
419408
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-26-3" class="td11">
4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-26-4" class="td11">
10.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-26-5" class="td11">
17.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-26-6" class="td11">
0<span class="cmmi-12">.</span>000179
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-115-26-7" class="td11">
<pre><code>0&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>00651
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-115-27-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-115-27-1"  
class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 2864-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Strongest candidate SL partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">by mtSLIPT with observed and expected mutant samples with low expression of partner metagenes</span></p>
</p>
</div>
<hr class="endfloat" />
<a 
 id="x1-225002r288"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix H</span><br /><a 
 id="x1-226000H"></a>Global Synthetic Lethality in Stomach Cancer
</h2>
<!--l. 2871-->
<p class="indent">
<a 
 id="x1-226001r1"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 2873-->
<p class="noindent">
</p>
<!--l. 2874-->
<p class="noindent">
<img 
src="MostSL_Summary_CDH1_FDR_stad.png" alt="pict"  
 />
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure H.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic lethal partners across query genes.</span></strong> <span class="cmr-10x-x-109">Global synthetic lethal</span> <span class="cmr-10x-x-109">pairs were examined across the genome in TCGA stomach expression data by applying</span> <span class="cmr-10x-x-109">SLIPT across query genes. The high number of predicted partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">was typical</span> <span class="cmr-10x-x-109">for a human gene and lower than many other genes. </span></span></p>
</div>
<!--tex4ht:label?: x1-226001r1 -->
</div>
<hr class="endfloat" />
<a 
 id="x1-226002r363"></a>
<h3 class="sectionHead">
<span class="titlemark">H.1 </span> <a 
 id="x1-2270001"></a>Hub Genes
</h3>
<!--l. 2892-->
<p class="indent">
<a 
 id="x1-227001r1"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table H.1: </span><span class="content">Query synthetic lethal genes with the most SLIPT partners</span></p>
</div>
<!--tex4ht:label?: x1-227001r1 -->
<div class="tabular">
<table id="TBL-117" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-117-1g">
<col 
id="TBL-117-1" />
<col 
id="TBL-117-2" />
<col 
id="TBL-117-3" />
<col 
id="TBL-117-4" />
<col 
id="TBL-117-5" />
<col 
id="TBL-117-6" />
</colgroup>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-117-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-1-1" class="td11">
<span class="cmbx-12">Gene </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-1-2" class="td11">
Direction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-1-3" class="td11">
raw p-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-1-4" class="td11">
p-value (FDR)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-1-5" class="td11">
SLIPT raw p-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-1-6" class="td11">
SLIPT (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-2-1" class="td11">
<span class="cmti-12">HEG1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-2-2" class="td11">
10719
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-2-3" class="td11">
16956
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-2-4" class="td11">
<pre><code> 16724       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-2-5&quot;  </code></pre>
class=“td11”&gt; 9616
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-2-6"  
class="td11">
<pre><code> 9532        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-3-1" class="td11">
<span class="cmti-12">SYNE1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-3-2" class="td11">
10755
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-3-3" class="td11">
17210
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-3-4" class="td11">
<pre><code> 16984       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-3-5&quot;  </code></pre>
class=“td11”&gt; 9749
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-3-6"  
class="td11">
<pre><code> 9676        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-4-1" class="td11">
<span class="cmti-12">A2M </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-4-2" class="td11">
10743
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-4-3" class="td11">
16650
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-4-4" class="td11">
<pre><code> 16378       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-4-5&quot;  </code></pre>
class=“td11”&gt; 9529
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-4-6"  
class="td11">
<pre><code> 9433        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-5-1" class="td11">
<span class="cmti-12">ANK2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-5-2" class="td11">
11008
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-5-3" class="td11">
16616
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-5-4" class="td11">
<pre><code> 16355       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-5-5&quot;  </code></pre>
class=“td11”&gt; 9764
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-5-6"  
class="td11">
<pre><code> 9653        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-6-1" class="td11">
<span class="cmti-12">TTC28 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-6-2" class="td11">
10757
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-6-3" class="td11">
16523
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-6-4" class="td11">
<pre><code> 16248       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-6-5&quot;  </code></pre>
class=“td11”&gt; 9530
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-6-6"  
class="td11">
<pre><code> 9429        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-7-1" class="td11">
<span class="cmti-12">FAT4 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-7-2" class="td11">
10451
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-7-3" class="td11">
16286
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-7-4" class="td11">
<pre><code> 15978       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-7-5&quot;  </code></pre>
class=“td11”&gt; 9225
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-7-6"  
class="td11">
<pre><code> 9115        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-8-1" class="td11">
<span class="cmti-12">MRVI1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-8-2" class="td11">
10904
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-8-3" class="td11">
16967
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-8-4" class="td11">
<pre><code> 16718       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-8-5&quot;  </code></pre>
class=“td11”&gt; 9775
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-8-6"  
class="td11">
<pre><code> 9686        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-9-1" class="td11">
<span class="cmti-12">PAPLN </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-9-2" class="td11">
10483
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-9-3" class="td11">
16405
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-9-4" class="td11">
<pre><code> 16104       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-9-5&quot;  </code></pre>
class=“td11”&gt; 9305
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-9-6"  
class="td11">
<pre><code> 9193        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-10-1" class="td11">
<span class="cmti-12">NFASC </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-10-2" class="td11">
10773
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-10-3" class="td11">
16575
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-10-4" class="td11">
<pre><code> 16307       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-10-5&quot;  </code></pre>
class=“td11”&gt; 9578
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-10-6"  
class="td11">
<pre><code> 9475        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-11-1" class="td11">
<span class="cmti-12">MACF1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-11-2" class="td11">
9697
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-11-3" class="td11">
16378
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-11-4" class="td11">
<pre><code> 16058       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-11-5&quot;  </code></pre>
class=“td11”&gt; 8620
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-11-6"  
class="td11">
<pre><code> 8540        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-12-1" class="td11">
<span class="cmti-12">HMCN1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-12-2" class="td11">
10475
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-12-3" class="td11">
16101
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-12-4" class="td11">
<pre><code> 15733       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-12-5&quot;  </code></pre>
class=“td11”&gt; 9156
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-12-6"  
class="td11">
<pre><code> 9008        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-13-1" class="td11">
<span class="cmti-12">MPDZ </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-13-2" class="td11">
10878
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-13-3" class="td11">
16550
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-13-4" class="td11">
<pre><code> 16299       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-13-5&quot;  </code></pre>
class=“td11”&gt; 9599
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-13-6"  
class="td11">
<pre><code> 9491        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-14-1" class="td11">
<span class="cmti-12">FLRT2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-14-2" class="td11">
10776
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-14-3" class="td11">
16760
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-14-4" class="td11">
<pre><code> 16473       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-14-5&quot;  </code></pre>
class=“td11”&gt; 9590
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-14-6"  
class="td11">
<pre><code> 9464        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-15-1" class="td11">
<span class="cmti-12">SETBP1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-15-2" class="td11">
10869
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-15-3" class="td11">
16632
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-15-4" class="td11">
<pre><code> 16349       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-15-5&quot;  </code></pre>
class=“td11”&gt; 9615
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-15-6"  
class="td11">
<pre><code> 9489        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-16-1" class="td11">
<span class="cmti-12">LAMA4 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-16-2" class="td11">
10463
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-16-3" class="td11">
16447
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-16-4" class="td11">
<pre><code> 16121       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-16-5&quot;  </code></pre>
class=“td11”&gt; 9273
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-16-6"  
class="td11">
<pre><code> 9151        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-17-1" class="td11">
<span class="cmti-12">IL1R1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-17-2" class="td11">
10611
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-17-3" class="td11">
16185
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-17-4" class="td11">
<pre><code> 15803       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-17-5&quot;  </code></pre>
class=“td11”&gt; 9299
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-17-6"  
class="td11">
<pre><code> 9174        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-18-1" class="td11">
<span class="cmti-12">ABCA6 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-18-2" class="td11">
10499
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-18-3" class="td11">
16573
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-18-4" class="td11">
<pre><code> 16318       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-18-5&quot;  </code></pre>
class=“td11”&gt; 9260
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-18-6"  
class="td11">
<pre><code> 9158        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-19-1" class="td11">
<span class="cmti-12">LAMC1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-19-2" class="td11">
10238
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-19-3" class="td11">
15777
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-19-4" class="td11">
<pre><code> 15392       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-19-5&quot;  </code></pre>
class=“td11”&gt; 8837
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-19-6"  
class="td11">
<pre><code> 8691        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-20-1" class="td11">
<span class="cmti-12">TNS1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-20-2" class="td11">
10920
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-20-3" class="td11">
17038
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-20-4" class="td11">
<pre><code> 16806       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-20-5&quot;  </code></pre>
class=“td11”&gt; 9836
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-20-6"  
class="td11">
<pre><code> 9751        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-117-21-1" class="td11">
<span class="cmti-12">AMOTL1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-21-2" class="td11">
10612
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-21-3" class="td11">
16458
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-117-21-4" class="td11">
<pre><code> 16178       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-117-21-5&quot;  </code></pre>
class=“td11”&gt; 9367
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-117-21-6"  
class="td11">
<pre><code> 9250        &lt;/td&gt;</code></pre>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-117-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-117-22-1"  
class="td11">
<pre><code>      &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 2950-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Genes with the most candidate SL partners SLIPT in TCGA stomach expression data with the number of partner</span> <span class="cmr-10x-x-109">genes predicted by direction criteria and </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">testing separately and combined as a SLIPT analysis. Where specified,</span> <span class="cmr-10x-x-109">the p-values for the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup> <span class="cmr-10x-x-109">test were adjusted for multiple tests (FDR).</span></p>
</p>
</div>
<hr class="endfloat" />
<a 
 id="x1-227002r367"></a>
<h3 class="sectionHead">
<span class="titlemark">H.2 </span> <a 
 id="x1-2280002"></a>Hub Pathways
</h3>
<!--l. 2960-->
<p class="indent">
<a 
 id="x1-228001r2"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table H.2: </span><span class="content">Pathways for genes with the most SLIPT partners</span></p>
</div>
<!--tex4ht:label?: x1-228001r2 -->
<div class="tabular">
<table id="TBL-118" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-118-1g">
<col 
id="TBL-118-1" />
<col 
id="TBL-118-2" />
<col 
id="TBL-118-3" />
<col 
id="TBL-118-4" />
<col 
id="TBL-118-5" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-118-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-1-1" class="td11">
<strong><span class="cmbx-12">Pathways Over-represented</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-1-2" class="td11">
<strong><span class="cmbx-12">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-1-3" class="td11">
<strong><span class="cmbx-12">SL Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-1-4" class="td11">
<strong><span class="cmbx-12">p-value</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-1-5" class="td11">
<strong><span class="cmbx-12">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-2-1" class="td11">
Molecules associated with elastic fibres
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-2-2" class="td11">
<pre><code>  34          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-2-3&quot;  </code></pre>
class=“td11”&gt; 10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-2-4" class="td11">
4<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-2-5" class="td11">
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-3-1" class="td11">
Extracellular matrix organization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-3-2" class="td11">
<pre><code>  241         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-3-3&quot;  </code></pre>
class=“td11”&gt; 29
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-3-4" class="td11">
5<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">21</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-3-5" class="td11">
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-4-1" class="td11">
Smooth Muscle Contraction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-4-2" class="td11">
<pre><code>  29          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-4-3&quot;  </code></pre>
class=“td11”&gt; 9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-4-4" class="td11">
5<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-4-5" class="td11">
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-5-1" class="td11">
Elastic fibre formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-5-2" class="td11">
<pre><code>  38          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-5-3&quot;  </code></pre>
class=“td11”&gt; 10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-5-4" class="td11">
6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">20</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-5-5" class="td11">
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">17</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-6-1" class="td11">
Nitric oxide stimulates guanylate cyclase
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-6-2" class="td11">
<pre><code>  24          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-6-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-6-4" class="td11">
6<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-6-5" class="td11">
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-7-1" class="td11">
Muscle contraction
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-7-2" class="td11">
<pre><code>  64          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-7-3&quot;  </code></pre>
class=“td11”&gt; 12
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-7-4" class="td11">
8<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">19</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-7-5" class="td11">
1<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-8-1" class="td11">
Platelet homeostasis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-8-2" class="td11">
<pre><code>  54          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-8-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-8-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-8-5" class="td11">
1<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-9-1" class="td11">
cGMP effects
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-9-2" class="td11">
<pre><code>  18          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-9-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-9-4" class="td11">
3<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">15</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-9-5" class="td11">
4<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-10-1" class="td11">
Laminin interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-10-2" class="td11">
<pre><code>  30          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-10-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-10-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">14</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-10-5" class="td11">
1<span class="cmmi-12">.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">12</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-11-1" class="td11">
Axon guidance
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-11-2" class="td11">
<pre><code>  289         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-11-3&quot;  </code></pre>
class=“td11”&gt; 25
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-11-4" class="td11">
5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-11-5" class="td11">
5<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-12-1" class="td11">
Signaling by BMP
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-12-2" class="td11">
<pre><code>  23          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-12-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-12-4" class="td11">
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-12-5" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-13-1" class="td11">
RHO GTPases activate PAKs
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-13-2" class="td11">
<pre><code>  23          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-13-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-13-4" class="td11">
3<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-13-5" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-14-1" class="td11">
Non-integrin membrane-ECM interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-14-2" class="td11">
<pre><code>  53          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-14-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-14-4" class="td11">
7<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">11</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-14-5" class="td11">
<pre><code>5&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-15-1" class="td11">
Rho GTPase cycle
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-15-2" class="td11">
<pre><code>  120         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-15-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-15-4" class="td11">
1<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-15-5" class="td11">
<pre><code>8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-16-1" class="td11">
Degradation of the extracellular matrix
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-16-2" class="td11">
<pre><code>  104         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-16-3&quot;  </code></pre>
class=“td11”&gt; 10
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-16-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-16-5" class="td11">
<pre><code>8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-17-1" class="td11">
Netrin-1 signaling
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-17-2" class="td11">
<pre><code>  42          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-17-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-17-4" class="td11">
2<span class="cmmi-12">.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-17-5" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-18-1" class="td11">
Developmental Biology
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-18-2" class="td11">
<pre><code>  432         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-18-3&quot;  </code></pre>
class=“td11”&gt; 32
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-18-4" class="td11">
8<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-18-5" class="td11">
<pre><code>5 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-19-1" class="td11">
L1CAM interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-19-2" class="td11">
<pre><code>  80          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-19-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-19-4" class="td11">
8<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">10</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-19-5" class="td11">
<pre><code>5 &lt;span </code></pre>
class=“cmsy-10x-x-120”&gt;× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-20-1" class="td11">
Semaphorin interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-20-2" class="td11">
<pre><code>  64          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-20-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-20-4" class="td11">
1<span class="cmmi-12">.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-20-5" class="td11">
<pre><code>6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-118-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-118-21-1" class="td11">
Cell-extracellular matrix interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-21-2" class="td11">
<pre><code>  18          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-118-21-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-21-4" class="td11">
1<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">9</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-118-21-5" class="td11">
<pre><code>6&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">8</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-118-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-118-22-1"  
class="td11">
<pre><code>                                 &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 3016-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Gene set over-representation analysis (hypergeometric test) for Reactome pathways in the top 500 “hub” genes with the most</span> <span class="cmr-10x-x-109">candidate synthetic lethal partners by SLIPT analysis of TCGA stomach expression data</span></p>
</p>
</div>
<hr class="endfloat" />
<a 
 id="x1-228002r288"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix I</span><br /><a 
 id="x1-229000I"></a>Replication in cell line encyclopaedia
</h2>
<!--l. 3024-->
<p class="indent">
<a 
 id="x1-229001r1"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table I.1: </span><span class="content">Candidate synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from SLIPT in CCLE</span></p>
</div>
<!--tex4ht:label?: x1-229001r1 -->
<div class="tabular">
<table id="TBL-119" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-119-1g">
<col 
id="TBL-119-1" />
<col 
id="TBL-119-2" />
<col 
id="TBL-119-3" />
<col 
id="TBL-119-4" />
<col 
id="TBL-119-5" />
<col 
id="TBL-119-6" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-119-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-1-1" class="td11">
<span class="cmbx-12">Gene </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-1-2" class="td11">
Observed
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-1-3" class="td11">
Expected
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-1-4" class="td11">
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-1-5" class="td11">
p-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-1-6" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-2-1" class="td11">
<span class="cmti-12">ZEB1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-2-2" class="td11">
<pre><code>24      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-2-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-2-4" class="td11">
555
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-2-5" class="td11">
7<span class="cmmi-12">.</span>84 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">119</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-2-6" class="td11">
3<span class="cmmi-12">.</span>62 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">116</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-3-1" class="td11">
<span class="cmti-12">RP11-620J15.3 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-3-2" class="td11">
<pre><code>17      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-3-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-3-4" class="td11">
471
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-3-5" class="td11">
1<span class="cmmi-12">.</span>54 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">100</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-3-6" class="td11">
3<span class="cmmi-12">.</span>68 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">98</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-4-1" class="td11">
<span class="cmti-12">AP1S2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-4-2" class="td11">
<pre><code>20      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-4-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-4-4" class="td11">
462
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-4-5" class="td11">
1<span class="cmmi-12">.</span>38 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">98</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-4-6" class="td11">
3<span class="cmmi-12">.</span>07 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">96</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-5-1" class="td11">
<span class="cmti-12">VIM </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-5-2" class="td11">
<pre><code>24      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-5-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-5-4" class="td11">
424
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-5-5" class="td11">
1<span class="cmmi-12">.</span>73 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">90</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-5-6" class="td11">
3<span class="cmmi-12">.</span>06 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">88</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-6-1" class="td11">
<span class="cmti-12">CCDC88A </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-6-2" class="td11">
<pre><code>24      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-6-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-6-4" class="td11">
418
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-6-5" class="td11">
3<span class="cmmi-12">.</span>94 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">89</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-6-6" class="td11">
6<span class="cmmi-12">.</span>86 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">87</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-7-1" class="td11">
<span class="cmti-12">RECK </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-7-2" class="td11">
<pre><code>28      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-7-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-7-4" class="td11">
416
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-7-5" class="td11">
8<span class="cmmi-12">.</span>23 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">89</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-7-6" class="td11">
1<span class="cmmi-12">.</span>42 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">86</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-8-1" class="td11">
<span class="cmti-12">AP1M1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-8-2" class="td11">
<pre><code>16      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-8-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-8-4" class="td11">
414
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-8-5" class="td11">
2<span class="cmmi-12">.</span>42 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">88</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-8-6" class="td11">
4<span class="cmmi-12">.</span>06 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">86</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-9-1" class="td11">
<span class="cmti-12">ZEB2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-9-2" class="td11">
<pre><code>23      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-9-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-9-4" class="td11">
396
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-9-5" class="td11">
2<span class="cmmi-12">.</span>32 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">84</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-9-6" class="td11">
3<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">82</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-10-1" class="td11">
<span class="cmti-12">WIPF1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-10-2" class="td11">
<pre><code>25      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-10-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-10-4" class="td11">
390
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-10-5" class="td11">
4<span class="cmmi-12">.</span>9 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">83</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-10-6" class="td11">
6<span class="cmmi-12">.</span>74 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">81</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-11-1" class="td11">
<span class="cmti-12">SLC35B4 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-11-2" class="td11">
<pre><code>29      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-11-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-11-4" class="td11">
386
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-11-5" class="td11">
3<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">82</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-11-6" class="td11">
4<span class="cmmi-12">.</span>38 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">80</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-12-1" class="td11">
<span class="cmti-12">SACS </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-12-2" class="td11">
<pre><code>28      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-12-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-12-4" class="td11">
373
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-12-5" class="td11">
2<span class="cmmi-12">.</span>13 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">79</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-12-6" class="td11">
2<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">77</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-13-1" class="td11">
<span class="cmti-12">ST3GAL2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-13-2" class="td11">
<pre><code>25      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-13-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-13-4" class="td11">
351
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-13-5" class="td11">
9<span class="cmmi-12">.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">75</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-13-6" class="td11">
1<span class="cmmi-12">.</span>08 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">72</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-14-1" class="td11">
<span class="cmti-12">ATP8B2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-14-2" class="td11">
<pre><code>38      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-14-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-14-4" class="td11">
341
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-14-5" class="td11">
1<span class="cmmi-12">.</span>53 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">72</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-14-6" class="td11">
1<span class="cmmi-12">.</span>61 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">70</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-15-1" class="td11">
<span class="cmti-12">IFFO1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-15-2" class="td11">
<pre><code>39      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-15-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-15-4" class="td11">
332
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-15-5" class="td11">
1<span class="cmmi-12">.</span>66 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">70</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-15-6" class="td11">
1<span class="cmmi-12">.</span>65 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">68</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-16-1" class="td11">
<span class="cmti-12">EMP3 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-16-2" class="td11">
<pre><code>38      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-16-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-16-4" class="td11">
329
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-16-5" class="td11">
5<span class="cmmi-12">.</span>04 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">70</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-16-6" class="td11">
4<span class="cmmi-12">.</span>95 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">68</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-17-1" class="td11">
<span class="cmti-12">LEPRE1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-17-2" class="td11">
<pre><code>40      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-17-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-17-4" class="td11">
325
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-17-5" class="td11">
5<span class="cmmi-12">.</span>4 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">69</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-17-6" class="td11">
5<span class="cmmi-12">.</span>22 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">67</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-18-1" class="td11">
<span class="cmti-12">STARD9 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-18-2" class="td11">
<pre><code>39      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-18-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-18-4" class="td11">
311
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-18-5" class="td11">
4<span class="cmmi-12">.</span>52 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">66</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-18-6" class="td11">
3<span class="cmmi-12">.</span>96 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">64</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-19-1" class="td11">
<span class="cmti-12">DENND5A </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-19-2" class="td11">
<pre><code>48      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-19-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-19-4" class="td11">
304
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-19-5" class="td11">
1<span class="cmmi-12">.</span>89 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">64</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-19-6" class="td11">
1<span class="cmmi-12">.</span>59 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">62</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-20-1" class="td11">
<span class="cmti-12">SYT11 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-20-2" class="td11">
<pre><code>38      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-20-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-20-4" class="td11">
300
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-20-5" class="td11">
1<span class="cmmi-12">.</span>21 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">63</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-20-6" class="td11">
9<span class="cmmi-12">.</span>89 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">62</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-21-1" class="td11">
<span class="cmti-12">EID2B </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-21-2" class="td11">
<pre><code>38      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-21-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-21-4" class="td11">
299
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-21-5" class="td11">
1<span class="cmmi-12">.</span>99 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">63</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-21-6" class="td11">
1<span class="cmmi-12">.</span>61 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">61</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-22-1" class="td11">
<span class="cmti-12">NXPE3 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-22-2" class="td11">
<pre><code>35      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-22-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-22-4" class="td11">
294
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-22-5" class="td11">
1<span class="cmmi-12">.</span>71 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">62</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-22-6" class="td11">
1<span class="cmmi-12">.</span>35 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">60</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-23-1" class="td11">
<span class="cmti-12">STX2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-23-2" class="td11">
<pre><code>49      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-23-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-23-4" class="td11">
293
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-23-5" class="td11">
3<span class="cmmi-12">.</span>83 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">62</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-23-6" class="td11">
3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">60</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-24-1" class="td11">
<span class="cmti-12">ARHGEF6 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-24-2" class="td11">
<pre><code>43      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-24-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-24-4" class="td11">
289
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-24-5" class="td11">
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">61</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-24-6" class="td11">
1<span class="cmmi-12">.</span>71 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">59</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-25-1" class="td11">
<span class="cmti-12">KATNAL1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-25-2" class="td11">
<pre><code>50      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-25-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-25-4" class="td11">
283
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-25-5" class="td11">
4<span class="cmmi-12">.</span>45 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">60</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-25-6" class="td11">
3<span class="cmmi-12">.</span>38 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">58</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-119-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-119-26-1" class="td11">
<span class="cmti-12">ANXA6 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-26-2" class="td11">
<pre><code>37      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-119-26-3&quot;  </code></pre>
class=“td11”&gt; 115
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-26-4" class="td11">
282
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-26-5" class="td11">
8<span class="cmmi-12">.</span>92 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">60</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-119-26-6" class="td11">
6<span class="cmmi-12">.</span>67 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">58</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-119-27-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-119-27-1"  
class="td11">
<pre><code>           &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 3091-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Strongest candidate SL partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">by SLIPT with observed and expected samples with low</span> <span class="cmr-10x-x-109">expression of both genes</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 3095-->
<p class="indent">
<a 
 id="x1-229002r2"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table I.2: </span><span class="content">Candidate synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from SLIPT in breast CCLE</span></p>
</div>
<!--tex4ht:label?: x1-229002r2 -->
<div class="tabular">
<table id="TBL-120" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-120-1g">
<col 
id="TBL-120-1" />
<col 
id="TBL-120-2" />
<col 
id="TBL-120-3" />
<col 
id="TBL-120-4" />
<col 
id="TBL-120-5" />
<col 
id="TBL-120-6" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-120-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-1-1" class="td11">
<span class="cmbx-12">Gene </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-1-2" class="td11">
Observed
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-1-3" class="td11">
Expected
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-1-4" class="td11">
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-1-5" class="td11">
p-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-1-6" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-2-1" class="td11">
<span class="cmti-12">MIR155HG </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-2-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-2-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-2-4" class="td11">
31.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-2-5" class="td11">
2<span class="cmmi-12">.</span>41 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-2-6"  
class="td11">
<pre><code>0.00371      &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-3-1" class="td11">
<span class="cmti-12">ENPP2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-3-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-3-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-3-4" class="td11">
30.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-3-5" class="td11">
3<span class="cmmi-12">.</span>47 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-3-6"  
class="td11">
<pre><code>0.00383      &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-4-1" class="td11">
<span class="cmti-12">DCLK2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-4-2" class="td11">
<pre><code>3       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-4-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-4-4" class="td11">
28.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-4-5" class="td11">
1<span class="cmmi-12">.</span>08 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-4-6"  
class="td11">
<pre><code>0.0071       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-5-1" class="td11">
<span class="cmti-12">PID1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-5-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-5-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-5-4" class="td11">
27.8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-5-5" class="td11">
1<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-5-6"  
class="td11">
<pre><code>0.00791      &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-6-1" class="td11">
<span class="cmti-12">SCFD2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-6-2" class="td11">
<pre><code>5       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-6-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-6-4" class="td11">
27.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-6-5" class="td11">
1<span class="cmmi-12">.</span>42 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-6-6"  
class="td11">
<pre><code>0.00791      &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-7-1" class="td11">
<span class="cmti-12">FAT4 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-7-2" class="td11">
<pre><code>4       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-7-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-7-4" class="td11">
27.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-7-5" class="td11">
1<span class="cmmi-12">.</span>69 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-7-6"  
class="td11">
<pre><code>0.00865      &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-8-1" class="td11">
<span class="cmti-12">ILK </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-8-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-8-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-8-4" class="td11">
26.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-8-5" class="td11">
2<span class="cmmi-12">.</span>04 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-8-6"  
class="td11">
<pre><code>0.00884      &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-9-1" class="td11">
<span class="cmti-12">RWDD1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-9-2" class="td11">
<pre><code>0       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-9-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-9-4" class="td11">
26.8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-9-5" class="td11">
2<span class="cmmi-12">.</span>15 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-9-6"  
class="td11">
<pre><code>0.00884      &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-10-1" class="td11">
<span class="cmti-12">RIC8A </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-10-2" class="td11">
<pre><code>2       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-10-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-10-4" class="td11">
26.8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-10-5" class="td11">
2<span class="cmmi-12">.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-10-6"  
class="td11">
<pre><code>0.00884      &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-11-1" class="td11">
<span class="cmti-12">F2RL2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-11-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-11-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-11-4" class="td11">
26.6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-11-5" class="td11">
2<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-11-6"  
class="td11">
<pre><code>0.00901      &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-12-1" class="td11">
<span class="cmti-12">SDCBP </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-12-2" class="td11">
<pre><code>5       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-12-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-12-4" class="td11">
25.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-12-5" class="td11">
3<span class="cmmi-12">.</span>26 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-12-6"  
class="td11">
<pre><code>0.0108       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-13-1" class="td11">
<span class="cmti-12">PPM1F </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-13-2" class="td11">
<pre><code>4       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-13-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-13-4" class="td11">
25.8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-13-5" class="td11">
3<span class="cmmi-12">.</span>41 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-13-6"  
class="td11">
<pre><code>0.0108       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-14-1" class="td11">
<span class="cmti-12">IKBIP </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-14-2" class="td11">
<pre><code>5       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-14-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-14-4" class="td11">
25.8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-14-5" class="td11">
3<span class="cmmi-12">.</span>49 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-14-6"  
class="td11">
<pre><code>0.0108       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-15-1" class="td11">
<span class="cmti-12">SPRED1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-15-2" class="td11">
<pre><code>3       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-15-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-15-4" class="td11">
25.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-15-5" class="td11">
3<span class="cmmi-12">.</span>97 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-15-6"  
class="td11">
<pre><code>0.0108       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-16-1" class="td11">
<span class="cmti-12">RNH1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-16-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-16-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-16-4" class="td11">
25.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-16-5" class="td11">
4<span class="cmmi-12">.</span>22 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-16-6"  
class="td11">
<pre><code>0.0108       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-17-1" class="td11">
<span class="cmti-12">SYDE1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-17-2" class="td11">
<pre><code>3       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-17-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-17-4" class="td11">
25.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-17-5" class="td11">
4<span class="cmmi-12">.</span>22 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-17-6"  
class="td11">
<pre><code>0.0108       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-18-1" class="td11">
<span class="cmti-12">LINC00968 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-18-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-18-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-18-4" class="td11">
25.2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-18-5" class="td11">
4<span class="cmmi-12">.</span>63 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-18-6"  
class="td11">
<pre><code>0.0109       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-19-1" class="td11">
<span class="cmti-12">ARHGEF10 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-19-2" class="td11">
<pre><code>5       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-19-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-19-4" class="td11">
24.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-19-5" class="td11">
6<span class="cmmi-12">.</span>22 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-19-6"  
class="td11">
<pre><code>0.0116       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-20-1" class="td11">
<span class="cmti-12">P4HA1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-20-2" class="td11">
<pre><code>0       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-20-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-20-4" class="td11">
24.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-20-5" class="td11">
6<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-20-6"  
class="td11">
<pre><code>0.0116       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-21-1" class="td11">
<span class="cmti-12">AZI2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-21-2" class="td11">
<pre><code>2       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-21-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-21-4" class="td11">
24.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-21-5" class="td11">
6<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-21-6"  
class="td11">
<pre><code>0.0116       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-22-1" class="td11">
<span class="cmti-12">TNFAIP6 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-22-2" class="td11">
<pre><code>2       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-22-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-22-4" class="td11">
24.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-22-5" class="td11">
6<span class="cmmi-12">.</span>34 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-22-6"  
class="td11">
<pre><code>0.0116       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-23-1" class="td11">
<span class="cmti-12">CD200 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-23-2" class="td11">
<pre><code>4       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-23-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-23-4" class="td11">
24.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-23-5" class="td11">
6<span class="cmmi-12">.</span>37 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-23-6"  
class="td11">
<pre><code>0.0116       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-24-1" class="td11">
<span class="cmti-12">SMPD1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-24-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-24-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-24-4" class="td11">
24.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-24-5" class="td11">
6<span class="cmmi-12">.</span>67 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-24-6"  
class="td11">
<pre><code>0.0116       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-25-1" class="td11">
<span class="cmti-12">ATP6V1G2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-25-2" class="td11">
<pre><code>3       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-25-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-25-4" class="td11">
24.2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-25-5" class="td11">
7<span class="cmmi-12">.</span>33 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-25-6"  
class="td11">
<pre><code>0.0123       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-120-26-1" class="td11">
<span class="cmti-12">FGF2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-26-2" class="td11">
<pre><code>4       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-120-26-3&quot;  </code></pre>
class=“td11”&gt; 6.78
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-26-4" class="td11">
24.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-120-26-5" class="td11">
7<span class="cmmi-12">.</span>49 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-120-26-6"  
class="td11">
<pre><code>0.0123       &lt;/td&gt;</code></pre>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-120-27-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-120-27-1"  
class="td11">
<pre><code>        &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 3162-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Strongest candidate SL partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">by SLIPT with observed and expected samples with low</span> <span class="cmr-10x-x-109">expression of both genes</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 3165-->
<p class="indent">
<a 
 id="x1-229003r3"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table I.3: </span><span class="content">Candidate synthetic lethal gene partners of <span class="cmti-12">CDH1 </span>from SLIPT in stomach CCLE</span></p>
</div>
<!--tex4ht:label?: x1-229003r3 -->
<div class="tabular">
<table id="TBL-121" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-121-1g">
<col 
id="TBL-121-1" />
<col 
id="TBL-121-2" />
<col 
id="TBL-121-3" />
<col 
id="TBL-121-4" />
<col 
id="TBL-121-5" />
<col 
id="TBL-121-6" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-121-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-1-1" class="td11">
<span class="cmbx-12">Gene </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-1-2" class="td11">
Observed
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-1-3" class="td11">
Expected
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-1-4" class="td11">
<span class="cmmi-12">χ</span><sup><span class="cmr-8">2</span></sup> value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-1-5" class="td11">
<span class="cmmi-12">p </span><span class="cmsy-10x-x-120">- </span><span class="cmmi-12">value </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-1-6" class="td11">
p-value (FDR)
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-2-1" class="td11">
<span class="cmti-12">ZEB1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-2-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-2-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-2-4" class="td11">
36
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-2-5" class="td11">
2<span class="cmmi-12">.</span>84 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-2-6"  
class="td11">
<pre><code>0.00175      &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-3-1" class="td11">
<span class="cmti-12">WDR47 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-3-2" class="td11">
<pre><code>0       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-3-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-3-4" class="td11">
26.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-3-5" class="td11">
2<span class="cmmi-12">.</span>3 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-3-6"  
class="td11">
<pre><code> 0.013        &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-4-1" class="td11">
<span class="cmti-12">KANK2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-4-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-4-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-4-4" class="td11">
25.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-4-5" class="td11">
4<span class="cmmi-12">.</span>81 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-4-6"  
class="td11">
<pre><code>0.0222       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-5-1" class="td11">
<span class="cmti-12">LEPRE1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-5-2" class="td11">
<pre><code>0       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-5-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-5-4" class="td11">
24.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-5-5" class="td11">
6<span class="cmmi-12">.</span>26 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-5-6"  
class="td11">
<pre><code>0.0228       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-6-1" class="td11">
<span class="cmti-12">KATNAL1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-6-2" class="td11">
<pre><code>0       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-6-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-6-4" class="td11">
24.3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-6-5" class="td11">
6<span class="cmmi-12">.</span>88 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-6-6"  
class="td11">
<pre><code>0.0231       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-7-1" class="td11">
<span class="cmti-12">TET1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-7-2" class="td11">
<pre><code>0       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-7-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-7-4" class="td11">
23.9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-7-5" class="td11">
8<span class="cmmi-12">.</span>23 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-7-6"  
class="td11">
<pre><code>0.0249       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-8-1" class="td11">
<span class="cmti-12">AP1S2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-8-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-8-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-8-4" class="td11">
23.1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-8-5" class="td11">
0<span class="cmmi-12">.</span>00012
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-8-6"  
class="td11">
<pre><code>0.0273       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-9-1" class="td11">
<span class="cmti-12">CDKN2C </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-9-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-9-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-9-4" class="td11">
22.8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-9-5" class="td11">
0<span class="cmmi-12">.</span>000136
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-9-6"  
class="td11">
<pre><code>0.0292       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-10-1" class="td11">
<span class="cmti-12">ARMC4 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-10-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-10-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-10-4" class="td11">
22.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-10-5" class="td11">
0<span class="cmmi-12">.</span>000164
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-10-6"  
class="td11">
<pre><code>0.0315       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-11-1" class="td11">
<span class="cmti-12">CSTF3 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-11-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-11-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-11-4" class="td11">
22.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-11-5" class="td11">
0<span class="cmmi-12">.</span>000166
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-11-6"  
class="td11">
<pre><code>0.0315       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-12-1" class="td11">
<span class="cmti-12">FAM216A </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-12-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-12-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-12-4" class="td11">
22.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-12-5" class="td11">
0<span class="cmmi-12">.</span>000166
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-12-6"  
class="td11">
<pre><code>0.0315       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-13-1" class="td11">
<span class="cmti-12">ANKRD32 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-13-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-13-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-13-4" class="td11">
22.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-13-5" class="td11">
0<span class="cmmi-12">.</span>000166
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-13-6"  
class="td11">
<pre><code>0.0315       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-14-1" class="td11">
<span class="cmti-12">WDR35 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-14-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-14-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-14-4" class="td11">
22.4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-14-5" class="td11">
0<span class="cmmi-12">.</span>000169
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-14-6"  
class="td11">
<pre><code>0.0315       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-15-1" class="td11">
<span class="cmti-12">ECI2 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-15-2" class="td11">
<pre><code>0       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-15-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-15-4" class="td11">
21.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-15-5" class="td11">
0<span class="cmmi-12">.</span>000232
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-15-6"  
class="td11">
<pre><code>0.0378       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-16-1" class="td11">
<span class="cmti-12">SAMD8 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-16-2" class="td11">
<pre><code>0       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-16-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-16-4" class="td11">
21.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-16-5" class="td11">
0<span class="cmmi-12">.</span>000232
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-16-6"  
class="td11">
<pre><code>0.0378       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-17-1" class="td11">
<span class="cmti-12">CHST12 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-17-2" class="td11">
<pre><code>0       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-17-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-17-4" class="td11">
21.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-17-5" class="td11">
0<span class="cmmi-12">.</span>000232
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-17-6"  
class="td11">
<pre><code>0.0378       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-18-1" class="td11">
<span class="cmti-12">RPL23AP32 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-18-2" class="td11">
<pre><code>0       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-18-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-18-4" class="td11">
21.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-18-5" class="td11">
0<span class="cmmi-12">.</span>000232
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-18-6"  
class="td11">
<pre><code>0.0378       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-19-1" class="td11">
<span class="cmti-12">STARD9 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-19-2" class="td11">
<pre><code>1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-19-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-19-4" class="td11">
21.7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-19-5" class="td11">
0<span class="cmmi-12">.</span>000232
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-19-6"  
class="td11">
<pre><code>0.0378       &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-121-20-1" class="td11">
<span class="cmti-12">MCM8 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-20-2" class="td11">
<pre><code>0       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-121-20-3&quot;  </code></pre>
class=“td11”&gt; 4.45
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-20-4" class="td11">
21.5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-121-20-5" class="td11">
0<span class="cmmi-12">.</span>000255
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-121-20-6"  
class="td11">
<pre><code>0.0379       &lt;/td&gt;</code></pre>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-121-21-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-121-21-1"  
class="td11">
<pre><code>        &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 3220-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Strongest candidate SL partners for </span><span class="cmti-10x-x-109">CDH1 </span><span class="cmr-10x-x-109">by SLIPT with observed and expected samples with low</span> <span class="cmr-10x-x-109">expression of both genes</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 3225-->
<p class="indent">
<a 
 id="x1-229004r4"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table I.4: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in stomach CCLE</span></p>
</div>
<!--tex4ht:label?: x1-229004r4 -->
<div class="tabular">
<table id="TBL-122" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-122-1g">
<col 
id="TBL-122-1" />
<col 
id="TBL-122-2" />
<col 
id="TBL-122-3" />
<col 
id="TBL-122-4" />
</colgroup>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-122-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-1-1" class="td11">
<strong><span class="cmbx-12">Pathways Over-represented</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-1-2" class="td11">
<strong><span class="cmbx-12">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-1-3" class="td11">
<strong><span class="cmbx-12">SL Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-1-4" class="td11">
<strong><span class="cmbx-12">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-2-1" class="td11">
Nef mediated downregulation of MHC class I complex cell surface expression
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-2-2" class="td11">
<pre><code>  10          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-2-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-2-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-3-1" class="td11">
Unwinding of DNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-3-2" class="td11">
<pre><code>  11          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-3-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-3-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-4-1" class="td11">
Processing of Intronless Pre-mRNAs
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-4-2" class="td11">
<pre><code>  13          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-4-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-4-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-5-1" class="td11">
E2F mediated regulation of DNA replication
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-5-2" class="td11">
<pre><code>  20          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-5-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-5-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-6-1" class="td11">
Chondroitin sulfate biosynthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-6-2" class="td11">
<pre><code>  20          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-6-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-6-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-7-1" class="td11">
Post-Elongation Processing of Intronless pre-mRNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-7-2" class="td11">
<pre><code>  21          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-7-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-7-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-8-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-122-8-1"  
class="td11">
<!--tex4ht:inline-->
<div class="tabular">
<table id="TBL-123" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-123-1g">
<col 
id="TBL-123-1" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-123-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-123-1-1" class="td00">
Nef-mediates down modulation of cell surface receptors by recruiting them
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-123-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-123-2-1" class="td00">
to clathrin adapters
</td>
</tr>
</table>
</div>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-8-2" class="td11">
<pre><code>  21          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-8-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-8-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-9-1" class="td11">
Processing of Capped Intronless Pre-mRNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-9-2" class="td11">
<pre><code>  21          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-9-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-9-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-10-1" class="td11">
Post-Elongation Processing of Intron-Containing pre-mRNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-10-2" class="td11">
<pre><code>  23          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-10-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-10-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-11-1" class="td11">
Activation of the pre-replicative complex
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-11-2" class="td11">
<pre><code>  23          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-11-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-11-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-12-1" class="td11">
mRNA 3’-end processing
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-12-2" class="td11">
<pre><code>  23          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-12-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-12-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-13-1" class="td11">
Golgi Associated Vesicle Biogenesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-13-2" class="td11">
<pre><code>  24          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-13-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-13-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-14-1" class="td11">
Lysosome Vesicle Biogenesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-14-2" class="td11">
<pre><code>  25          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-14-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-14-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-15-1" class="td11">
Oncogene Induced Senescence
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-15-2" class="td11">
<pre><code>  27          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-15-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-15-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-16-1" class="td11">
The role of Nef in HIV-1 replication and disease pathogenesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-16-2" class="td11">
<pre><code>  28          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-16-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-16-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-17-1" class="td11">
Cyclin D associated events in G1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-17-2" class="td11">
<pre><code>  29          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-17-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-17-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-18-1" class="td11">
G1 Phase
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-18-2" class="td11">
<pre><code>  29          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-18-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-18-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-19-1" class="td11">
Cleavage of Growing Transcript in the Termination Region
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-19-2" class="td11">
<pre><code>  31          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-19-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-19-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-20-1" class="td11">
Activation of ATR in response to replication stress
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-20-2" class="td11">
<pre><code>  31          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-20-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-20-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-122-21-1" class="td11">
DNA strand elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-122-21-2" class="td11">
<pre><code>  31          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-122-21-3&quot;  </code></pre>
class=“td11”&gt; 1
</td>
<td  style="white-space:nowrap; text-align:center;" id="TBL-122-21-4"  
class="td11">
<pre><code>   1            &lt;/td&gt;</code></pre>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-122-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-122-22-1"  
class="td11">
<pre><code>                                                                     &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 3283-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Gene set over-representation analysis (hypergeometric test) for Reactome pathways in SLIPT partners for </span><span class="cmti-10x-x-109">CDH1</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 3287-->
<p class="indent">
<a 
 id="x1-229005r5"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table I.5: </span><span class="content">Pathways for <span class="cmti-12">CDH1 </span>partners from SLIPT in breast and stomach CCLE</span></p>
</div>
<!--tex4ht:label?: x1-229005r5 -->
<div class="tabular">
<table id="TBL-124" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-124-1g">
<col 
id="TBL-124-1" />
<col 
id="TBL-124-2" />
<col 
id="TBL-124-3" />
<col 
id="TBL-124-4" />
</colgroup>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-1-1" class="td11">
<strong><span class="cmbx-12">Pathways Over-represented</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-1-2" class="td11">
<strong><span class="cmbx-12">Pathway Size</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-1-3" class="td11">
<strong><span class="cmbx-12">SL Genes</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-1-4" class="td11">
<strong><span class="cmbx-12">p-value (FDR)</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-2-1" class="td11">
Collagen formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-2-2" class="td11">
<pre><code>  66          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-2-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-2-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-3-1" class="td11">
Glycosaminoglycan metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-3-2" class="td11">
<pre><code>  111         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-3-3&quot;  </code></pre>
class=“td11”&gt; 11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-3-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-4-1" class="td11">
Extracellular matrix organization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-4-2" class="td11">
<pre><code>  236         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-4-3&quot;  </code></pre>
class=“td11”&gt; 20
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-4-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-5-1" class="td11">
Collagen biosynthesis and modifying enzymes
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-5-2" class="td11">
<pre><code>  55          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-5-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-5-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-6-1" class="td11">
Keratan sulfate biosynthesis
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-6-2" class="td11">
<pre><code>  28          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-6-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-6-4" class="td11">
<pre><code>2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>2 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-7-1" class="td11">
Keratan sulfate/keratin metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-7-2" class="td11">
<pre><code>  32          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-7-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-7-4" class="td11">
<pre><code>7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">7</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-8-1" class="td11">
ECM proteoglycans
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-8-2" class="td11">
<pre><code>  65          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-8-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-8-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-9-1" class="td11">
Non-integrin membrane-ECM interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-9-2" class="td11">
<pre><code>  52          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-9-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-9-4" class="td11">
<pre><code>2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-10-1" class="td11">
Cell junction organization
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-10-2" class="td11">
<pre><code>  71          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-10-3&quot;  </code></pre>
class=“td11”&gt; 7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-10-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>0 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-11-1" class="td11">
Assembly of collagen fibrils and other multimeric structures
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-11-2" class="td11">
<pre><code>  39          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-11-3&quot;  </code></pre>
class=“td11”&gt; 5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-11-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>6 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-12-1" class="td11">
Post-chaperonin tubulin folding pathway
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-12-2" class="td11">
<pre><code>  14          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-12-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-12-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-13-1" class="td11">
Adherens junctions interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-13-2" class="td11">
<pre><code>  29          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-13-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-13-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-14-1" class="td11">
Cell-Cell communication
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-14-2" class="td11">
<pre><code>  118         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-14-3&quot;  </code></pre>
class=“td11”&gt; 9
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-14-4" class="td11">
<pre><code>1&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>7 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-15-1" class="td11">
Sialic acid metabolism
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-15-2" class="td11">
<pre><code>  31          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-15-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-15-4" class="td11">
<pre><code>2&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-16-1" class="td11">
Synthesis and interconversion of nucleotide di- and triphosphates
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-16-2" class="td11">
<pre><code>  16          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-16-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-16-4" class="td11">
<pre><code>3&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>1 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-17-1" class="td11">
Transport to the Golgi and subsequent modification
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-17-2" class="td11">
<pre><code>  34          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-17-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-17-4" class="td11">
<pre><code>4&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-18-1" class="td11">
Asparagine N-linked glycosylation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-18-2" class="td11">
<pre><code>  113         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-18-3&quot;  </code></pre>
class=“td11”&gt; 8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-18-4" class="td11">
<pre><code>7&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>8 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-19-1" class="td11">
Elastic fibre formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-19-2" class="td11">
<pre><code>  37          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-19-3&quot;  </code></pre>
class=“td11”&gt; 4
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-19-4" class="td11">
<pre><code>8&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-20-1" class="td11">
L1CAM interactions
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-20-2" class="td11">
<pre><code>  77          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-20-3&quot;  </code></pre>
class=“td11”&gt; 6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-20-4" class="td11">
<pre><code>9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-124-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-124-21-1" class="td11">
Signal transduction by L1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-21-2" class="td11">
<pre><code>  20          &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-124-21-3&quot;  </code></pre>
class=“td11”&gt; 3
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-124-21-4" class="td11">
<pre><code>9&lt;span </code></pre>
class=“cmmi-12”&gt;.</span>5 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-124-22-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-124-22-1"  
class="td11">
<pre><code>                                                    &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 3344-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Gene set over-representation analysis (hypergeometric test) for Reactome pathways in SLIPT partners for </span><span class="cmti-10x-x-109">CDH1</span></p>
</p>
</div>
<hr class="endfloat" />
<a 
 id="x1-229006r288"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix J</span><br /><a 
 id="x1-230000J"></a>Synthetic Lethal Genes in Pathways
</h2>
<!--l. 41-->
<p class="indent">
<a 
 id="x1-230001r1"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 43-->
<p class="noindent">
</p>
<!--l. 44-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_Pi3kAkt_exprSL_2".png" alt="PIC"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure J.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality in the PI3K/AKT Pathway.</span></strong> <span class="cmr-10x-x-109">The Reactome</span> <span class="cmr-10x-x-109">PI3K/AKT pathway with synthetic lethal candidates coloured as shown in the Legend.</span> </span></p>
</div>
<!--tex4ht:label?: x1-230001r1 -->
</div>
<hr class="endfloat" />
<!--l. 57-->
<p class="indent">
<a 
 id="x1-230002r2"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 59-->
<p class="noindent">
</p>
<!--l. 60-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_Pi3kAktCancer_exprSL_2".png" alt="PIC"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure J.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality in the PI3K/AKT Pathway in Cancer.</span></strong> <span class="cmr-10x-x-109">The</span> <span class="cmr-10x-x-109">Reactome PI3K/AKT Pathway in Cancer pathway with synthetic lethal candidates</span> <span class="cmr-10x-x-109">coloured as shown in the Legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-230002r2 -->
</div>
<hr class="endfloat" />
<!--l. 110-->
<p class="indent">
<a 
 id="x1-230003r3"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 112-->
<p class="noindent">
</p>
<!--l. 113-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_ExtracellularMatrix_exprSL_2".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure J.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality in the Extracellular Matrix.</span></strong> <span class="cmr-10x-x-109">The Reactome</span> <span class="cmr-10x-x-109">Extracellular Matrix pathway with synthetic lethal candidates coloured as shown in the</span> <span class="cmr-10x-x-109">Legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-230003r3 -->
</div>
<hr class="endfloat" />
<!--l. 127-->
<p class="indent">
<a 
 id="x1-230004r4"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 129-->
<p class="noindent">
</p>
<!--l. 130-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_GPCR_exprSL_2".png" alt="PIC"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure J.4: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality in the GPCRs.</span></strong> <span class="cmr-10x-x-109">The Reactome G</span><sub><span class="cmmi-8">αi</span></sub> <span class="cmr-10x-x-109">pathway with</span> <span class="cmr-10x-x-109">synthetic lethal candidates coloured as shown in the Legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-230004r4 -->
</div>
<hr class="endfloat" />
<!--l. 143-->
<p class="indent">
<a 
 id="x1-230005r5"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 145-->
<p class="noindent">
</p>
<!--l. 146-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_GPCR_Downstream_exprSL_2".png" alt="PIC"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure J.5: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality in the GPCR Downstream.</span></strong> <span class="cmr-10x-x-109">The Reactome</span> <span class="cmr-10x-x-109">GPCR Downstream pathway with synthetic lethal candidates coloured as shown in the</span> <span class="cmr-10x-x-109">Legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-230005r5 -->
</div>
<hr class="endfloat" />
<!--l. 159-->
<p class="indent">
<a 
 id="x1-230006r6"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 161-->
<p class="noindent">
</p>
<!--l. 162-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_TranslationElongation_exprSL_2".png" alt="PIC"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure J.6: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality in the Translation Elongation.</span></strong> <span class="cmr-10x-x-109">The Reactome</span> <span class="cmr-10x-x-109">Translation Elongation pathway with synthetic lethal candidates coloured as shown in</span> <span class="cmr-10x-x-109">the Legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-230006r6 -->
</div>
<hr class="endfloat" />
<!--l. 176-->
<p class="indent">
<a 
 id="x1-230007r7"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 178-->
<p class="noindent">
</p>
<!--l. 179-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_NMD_exprSL_2".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure J.7: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality in the Nonsense-mediated Decay.</span></strong> <span class="cmr-10x-x-109">The Reactome</span> <span class="cmr-10x-x-109">Nonsense-mediated Decay pathway with synthetic lethal candidates coloured as shown</span> <span class="cmr-10x-x-109">in the Legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-230007r7 -->
</div>
<hr class="endfloat" />
<!--l. 192-->
<p class="indent">
<a 
 id="x1-230008r8"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 194-->
<p class="noindent">
</p>
<!--l. 195-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/graph_plot_Three_prime_UTR_exprSL_2".png" alt="PIC"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure J.8: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality in the 3</span><sup><span class="cmsy-8">′</span></sup> <span class="cmbx-10x-x-109">UTR.</span></strong> <span class="cmr-10x-x-109">The Reactome 3</span><sup><span class="cmsy-8">′</span></sup> <span class="cmr-10x-x-109">UTR pathway</span> <span class="cmr-10x-x-109">with synthetic lethal candidates coloured as shown in the Legend. </span></span></p>
</div>
<!--tex4ht:label?: x1-230008r8 -->
</div>
<hr class="endfloat" />
<a 
 id="x1-230009r288"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix K</span><br /><a 
 id="x1-231000K"></a>Pathway Connectivity for Mutation SLIPT
</h2>
<!--l. 213-->
<p class="indent">
<a 
 id="x1-231001r1"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 215-->
<p class="noindent">
</p>
<!--l. 216-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Centrality_mtSL/Pi3K_network_vertex_degree".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure K.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality and Vertex Degree.</span></strong> <span class="cmr-10x-x-109">The number of connected</span> <span class="cmr-10x-x-109">genes (vertex degree) was compared (on a log-scale across genes deteced by mtSLIPT and</span> <a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screening in the Reactome PI3K cascade pathway. There were very few differences</span> <span class="cmr-10x-x-109">in vertex degree between the groups, although genes detected by </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">included those</span> <span class="cmr-10x-x-109">with the fewest connections. </span></span></p>
</div>
<!--tex4ht:label?: x1-231001r1 -->
</div>
<hr class="endfloat" />
<!--l. 231-->
<p class="indent">
<a 
 id="x1-231002r2"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 233-->
<p class="noindent">
</p>
<!--l. 234-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Centrality_mtSL/Pi3K_network_Info_Centrality(Log)".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure K.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality and Centrality.</span></strong> <span class="cmr-10x-x-109">The information centrality was</span> <span class="cmr-10x-x-109">compared (on a log-scale across genes deteced by mtSLIPT and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screening in the</span> <span class="cmr-10x-x-109">Reactome PI3K cascade pathway. Genes detected by </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">had higher connectivity than</span> <span class="cmr-10x-x-109">many genes not detected by either approach. The gene with the highest centrality was</span> <span class="cmr-10x-x-109">detected by mtSLIPT. </span></span></p>
</div>
<!--tex4ht:label?: x1-231002r2 -->
</div>
<hr class="endfloat" />
<!--l. 252-->
<p class="indent">
<a 
 id="x1-231003r3"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 254-->
<p class="noindent">
</p>
<!--l. 255-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Centrality_mtSL/Pi3K_network_pagerank".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure K.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality and PageRank.</span></strong> <span class="cmr-10x-x-109">The PageRank centrality was</span> <span class="cmr-10x-x-109">compared (on a log-scale across genes deteced by mtSLIPT and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">screening in the</span> <span class="cmr-10x-x-109">Reactome PI3K cascade pathway. Genes detected by </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">had a more restricted range</span> <span class="cmr-10x-x-109">of centrality values than other genes not detected by either approach, although these</span> <span class="cmr-10x-x-109">groups also had fewer genes. </span></span></p>
</div>
<!--tex4ht:label?: x1-231003r3 -->
</div>
<hr class="endfloat" />
<!--l. 269-->
<p class="indent">
<a 
 id="x1-231004r1"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table K.1: </span><span class="content">ANOVA for Synthetic Lethality and Vertex Degree</span></p>
</div>
<!--tex4ht:label?: x1-231004r1 -->
<div class="tabular">
<table id="TBL-125" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-125-1g">
<col 
id="TBL-125-1" />
<col 
id="TBL-125-2" />
<col 
id="TBL-125-3" />
<col 
id="TBL-125-4" />
<col 
id="TBL-125-5" />
<col 
id="TBL-125-6" />
</colgroup>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-125-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-125-1-1" class="td11">
<pre><code>            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-125-1-2&quot;  </code></pre>
class=“td11”&gt; DF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-1-3" class="td11">
Sum Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-1-4" class="td11">
Mean Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-1-5" class="td11">
F-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-1-6" class="td11">
p-value
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-125-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-125-2-1" class="td11">
siRNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-2-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-2-3" class="td11">
<pre><code> 15        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-125-2-4&quot;  </code></pre>
class=“td11”&gt; 15.50
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-2-5" class="td11">
0.0134
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-2-6" class="td11">
0.9084
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-125-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-125-3-1" class="td11">
mtSLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-3-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-3-3" class="td11">
<pre><code> 196        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-125-3-4&quot;  </code></pre>
class=“td11”&gt; 195.94
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-3-5" class="td11">
0.1689
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-3-6" class="td11">
0.6825
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-125-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-125-4-1" class="td11">
siRNA<span class="cmsy-10x-x-120">×</span>mtSLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-4-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-4-3" class="td11">
<pre><code>  9         &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-125-4-4&quot;  </code></pre>
class=“td11”&gt; 9.17
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-4-5" class="td11">
0.0079
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-125-4-6" class="td11">
0.9294
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-125-5-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-125-5-1"  
class="td11">
<pre><code>             &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 293-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Analysis of variance for vertex degree against synthetic lethal detection approaches (with an</span> <span class="cmr-10x-x-109">interaction term)</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 297-->
<p class="indent">
<a 
 id="x1-231005r2"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table K.2: </span><span class="content">ANOVA for Synthetic Lethality and Information Centrality</span></p>
</div>
<!--tex4ht:label?: x1-231005r2 -->
<div class="tabular">
<table id="TBL-126" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-126-1g">
<col 
id="TBL-126-1" />
<col 
id="TBL-126-2" />
<col 
id="TBL-126-3" />
<col 
id="TBL-126-4" />
<col 
id="TBL-126-5" />
<col 
id="TBL-126-6" />
</colgroup>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-126-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-126-1-1" class="td11">
<pre><code>            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-126-1-2&quot;  </code></pre>
class=“td11”&gt; DF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-1-3" class="td11">
Sum Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-1-4" class="td11">
Mean Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-1-5" class="td11">
F-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-1-6" class="td11">
p-value
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-126-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-126-2-1" class="td11">
siRNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-2-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-2-3" class="td11">
0.000256
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-2-4" class="td11">
0.0002561
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-2-5" class="td11">
0.1851
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-2-6" class="td11">
0.6685
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-126-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-126-3-1" class="td11">
mtSLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-3-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-3-3" class="td11">
0.003225
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-3-4" class="td11">
0.0032247
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-3-5" class="td11">
2.3308
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-3-6" class="td11">
0.1318
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-126-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-126-4-1" class="td11">
siRNA<span class="cmsy-10x-x-120">×</span>mtSLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-4-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-4-3" class="td11">
0.001238
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-4-4" class="td11">
0.0012385
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-4-5" class="td11">
0.8952
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-126-4-6" class="td11">
0.3476
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-126-5-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-126-5-1"  
class="td11">
<pre><code>             &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 321-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Analysis of variance for information centrality against synthetic lethal detection approaches (with</span> <span class="cmr-10x-x-109">an interaction term)</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 326-->
<p class="indent">
<a 
 id="x1-231006r3"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table K.3: </span><span class="content">ANOVA for Synthetic Lethality and PageRank Centrality</span></p>
</div>
<!--tex4ht:label?: x1-231006r3 -->
<div class="tabular">
<table id="TBL-127" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-127-1g">
<col 
id="TBL-127-1" />
<col 
id="TBL-127-2" />
<col 
id="TBL-127-3" />
<col 
id="TBL-127-4" />
<col 
id="TBL-127-5" />
<col 
id="TBL-127-6" />
</colgroup>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-127-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-127-1-1" class="td11">
<pre><code>            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-127-1-2&quot;  </code></pre>
class=“td11”&gt; DF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-1-3" class="td11">
Sum Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-1-4" class="td11">
Mean Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-1-5" class="td11">
F-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-1-6" class="td11">
p-value
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-127-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-127-2-1" class="td11">
siRNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-2-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-2-3" class="td11">
0.0002038
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-2-4" class="td11">
2<span class="cmmi-12">.</span>0385 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-2-5" class="td11">
1.1423
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-2-6" class="td11">
0.2892
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-127-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-127-3-1" class="td11">
mtSLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-3-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-3-3" class="td11">
0.0000208
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-3-4" class="td11">
2<span class="cmmi-12">.</span>0752 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-3-5" class="td11">
0.1163
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-3-6" class="td11">
0.7342
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-127-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-127-4-1" class="td11">
siRNA<span class="cmsy-10x-x-120">×</span>mtSLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-4-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-4-3" class="td11">
0.0000137
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-4-4" class="td11">
1<span class="cmmi-12">.</span>3743 <span class="cmsy-10x-x-120">× </span>10<sup><span class="cmsy-8">-</span><span class="cmr-8">5</span></sup>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-4-5" class="td11">
0.0770
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-127-4-6" class="td11">
0.7823
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-127-5-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-127-5-1"  
class="td11">
<pre><code>             &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 350-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Analysis of variance for PageRank centrality against synthetic lethal detection approaches (with</span> <span class="cmr-10x-x-109">an interaction term)</span></p>
</p>
</div>
<hr class="endfloat" />
<a 
 id="x1-231007r288"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix L</span><br /><a 
 id="x1-232000L"></a>Information Centrality for Gene Essentiality
</h2>
<!--l. 362-->
<p class="noindent">
<p>Network structure is another useful strategy to analyse gene function and this has been used to investigate network properties of a network constructed from of Reactome pathways imported via Pathway Commons with Paxtools (<a 
href="#XPathwayCommons">Cerami <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#XPathwayCommons">2011</a>; <a 
href="#Xpaxtools">Demir <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a>, <a 
href="#Xpaxtools">2013</a>). Most notably, information centrality which has been proposed as a measure of gene essentiality was calculated as performed by <a 
href="#XKranthi2013">Kranthi <em><span class="cmti-12">et</span><span class="cmti-12"> al.</span></em></a> (<a 
href="#XKranthi2013">2013</a>) using the efficiency and shortest path between each pair or nodes in the network before and after a node of interest is removed to test the importance of a node to network connectivity. Reactome contains substrates and cofactors in addition to genes or proteins. In support of centrality as a measure of essentiality, a number nodes with the highest centrality (shown in Table <a 
href="#x1-232001r1">L.1<!--tex4ht:ref: tab:infocent_reactome --></a>) were essential nutrients including Mg<sup class="textsuperscript"><span class="cmr-10x-x-109">2+</span></sup>, Ca<sup class="textsuperscript"><span class="cmr-10x-x-109">2+</span></sup>, Zn<sup class="textsuperscript"><span class="cmr-10x-x-109">2+</span></sup>, and Fe. In addition, there were genes important in development of epithelial tissues and breast cancer such as <span class="cmti-12">IL8</span>, <span class="cmti-12">GATA3</span>, and <span class="cmti-12">CTNNB1 </span>detected with relatively high information centrality.</p>
</p>
<!--l. 365-->
<p class="indent">
<a 
 id="x1-232001r1"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table L.1: </span><span class="content">Information centrality for genes and molecules in the Reactome network</span></p>
</div>
<!--tex4ht:label?: x1-232001r1 -->
<div class="tabular">
<table id="TBL-128" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-128-1g">
<col 
id="TBL-128-1" />
<col 
id="TBL-128-2" />
</colgroup>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-1-1" class="td11">
<strong><span class="cmbx-12">Node</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-1-2" class="td11">
<strong><span class="cmbx-12">Centrality</span></strong>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-2-1" class="td11">
<span class="cmti-12">ZNF473 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-2-2" class="td11">
0.0510
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-3-1" class="td11">
magnesium(2+)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-3-2" class="td11">
0.0082
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-4-1" class="td11">
<span class="cmti-12">XBP1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-4-2" class="td11">
0.0053
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-5-1" class="td11">
calcium(2+)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-5-2" class="td11">
0.0050
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-6-1" class="td11">
zinc(2+)
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-6-2" class="td11">
0.0048
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-7-1" class="td11">
iron atom
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-7-2" class="td11">
0.0041
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-8-1" class="td11">
<span class="cmti-12">FMN </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-8-2" class="td11">
0.0040
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-9-1" class="td11">
<span class="cmti-12">AGT </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-9-2" class="td11">
0.0037
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-10-1" class="td11">
<span class="cmti-12">HSP90AA1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-10-2" class="td11">
0.0029
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-11-1" class="td11">
phosphatidyl-L-serine
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-11-2" class="td11">
0.0029
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-12-1" class="td11">
<span class="cmti-12">P2RX7 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-12-2" class="td11">
0.0026
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-13-1" class="td11">
<span class="cmti-12">PANX1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-13-2" class="td11">
0.0024
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-14-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-14-1" class="td11">
<span class="cmti-12">NCAM1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-14-2" class="td11">
0.0022
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-15-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-15-1" class="td11">
<span class="cmti-12">NUDT1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-15-2" class="td11">
0.0021
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-16-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-16-1" class="td11">
<span class="cmti-12">PLAUR </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-16-2" class="td11">
0.0020
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-17-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-17-1" class="td11">
<span class="cmti-12">IL8 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-17-2" class="td11">
0.0020
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-18-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-18-1" class="td11">
<span class="cmti-12">HSPA8 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-18-2" class="td11">
0.0019
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-19-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-19-1" class="td11">
<span class="cmti-12">TYROBP </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-19-2" class="td11">
0.0019
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-20-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-20-1" class="td11">
<span class="cmti-12">CASP3 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-20-2" class="td11">
0.0017
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-21-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-21-1" class="td11">
<span class="cmti-12">GNAL </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-21-2" class="td11">
0.0015
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-22-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-22-1" class="td11">
<span class="cmti-12">CBLB </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-22-2" class="td11">
0.0015
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-23-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-23-1" class="td11">
<span class="cmti-12">HBB </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-23-2" class="td11">
0.0014
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-24-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-24-1" class="td11">
<span class="cmti-12">GATA4 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-24-2" class="td11">
0.0013
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-25-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-25-1" class="td11">
<span class="cmti-12">TGS1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-25-2" class="td11">
0.0013
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-128-26-">
<td style="white-space:nowrap; text-align:left;" id="TBL-128-26-1" class="td11">
<span class="cmti-12">CTNNB1 </span>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-128-26-2" class="td11">
0.0012
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-128-27-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-128-27-1"  
class="td11">
<pre><code>                &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 432-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Highest information centrality for genes (proteins), cofactors, and minerals in the Reactome</span> <span class="cmr-10x-x-109">network</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 436-->
<p class="indent">
<a 
 id="x1-232002r1"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 438-->
<p class="noindent">
</p>
<!--l. 439-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Networks/infocent_genes".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure L.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Information centrality distribution.</span></strong> <span class="cmr-10x-x-109">Information centrality in the</span> <span class="cmr-10x-x-109">Reactome network for nodes, including genes/proteins and other biomolecules. </span></span></p>
</div>
<!--tex4ht:label?: x1-232002r1 -->
</div>
<hr class="endfloat" />
<a 
 id="x1-232003r288"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix M</span><br /><a 
 id="x1-233000M"></a>Pathway Structure for Mutation SLIPT
</h2>
<!--l. 458-->
<p class="indent">
<a 
 id="x1-233001r1"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 460-->
<p class="noindent">
</p>
<!--l. 461-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Discrete_Pi3k/SL_distance_counts_vioplot_mtSL".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure M.1: </span><span class="content"><strong><span class="cmbx-10x-x-109">Synthetic Lethality and Heirarchy Score in PI3K.</span></strong> <span class="cmr-10x-x-109">The hierarchical</span> <span class="cmr-10x-x-109">distance scores were similarly distributed across mtSLIPT and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes. Genes</span> <span class="cmr-10x-x-109">detected by both methods had a higher (downstream) median than either group. </span></span></p>
</div>
<!--tex4ht:label?: x1-233001r1 -->
</div>
<hr class="endfloat" />
<!--l. 474-->
<p class="indent">
<a 
 id="x1-233002r1"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table M.1: </span><span class="content">ANOVA for Synthetic Lethality and PI3K Hierarchy</span></p>
</div>
<!--tex4ht:label?: x1-233002r1 -->
<div class="tabular">
<table id="TBL-129" class="tabular" 
cellspacing="0" cellpadding="0"  
>
<colgroup id="TBL-129-1g">
<col 
id="TBL-129-1" />
<col 
id="TBL-129-2" />
<col 
id="TBL-129-3" />
<col 
id="TBL-129-4" />
<col 
id="TBL-129-5" />
<col 
id="TBL-129-6" />
</colgroup>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-129-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-129-1-1" class="td11">
<pre><code>            &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-129-1-2&quot;  </code></pre>
class=“td11”&gt; DF
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-1-3" class="td11">
Sum Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-1-4" class="td11">
Mean Squares
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-1-5" class="td11">
F-value
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-1-6" class="td11">
p-value
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-129-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-129-2-1" class="td11">
siRNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-2-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-2-3" class="td11">
<pre><code>0.001       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-129-2-4&quot;  </code></pre>
class=“td11”&gt; 0.00070
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-2-5" class="td11">
0.0004
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-2-6" class="td11">
0.9841
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-129-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-129-3-1" class="td11">
mtSLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-3-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-3-3" class="td11">
<pre><code>0.007       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-129-3-4&quot;  </code></pre>
class=“td11”&gt; 0.0066
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-3-5" class="td11">
0.0040
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-3-6" class="td11">
0.9496
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-129-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-129-4-1" class="td11">
siRNA<span class="cmsy-10x-x-120">×</span>mtSLIPT
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-4-2" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-4-3" class="td11">
<pre><code>3.906       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-129-4-4&quot;  </code></pre>
class=“td11”&gt; 3.9056
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-4-5" class="td11">
2.3829
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-129-4-6" class="td11">
0.1250
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-129-5-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-129-5-1"  
class="td11">
<pre><code>             &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 498-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Analysis of variance for PI3K hierarchy score against synthetic lethal detection approaches (with</span> <span class="cmr-10x-x-109">an interaction term)</span></p>
</p>
</div>
<hr class="endfloat" />
<!--l. 501-->
<p class="indent">
<a 
 id="x1-233003r2"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 503-->
<p class="noindent">
</p>
<!--l. 504-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Discrete_Pi3k/SL_distance_counts_barplot_mtSL".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure M.2: </span><span class="content"><strong><span class="cmbx-10x-x-109">Heirarchy Score in PI3K against Synthetic Lethality in PI3K.</span></strong> <span class="cmr-10x-x-109">The</span> <span class="cmr-10x-x-109">number of mtSLIPT and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes against the hierarchical distance scores showing no</span> <span class="cmr-10x-x-109">significant tendency for either method to either of the pathway upstream or downstream</span> <span class="cmr-10x-x-109">extremities. </span></span></p>
</div>
<!--tex4ht:label?: x1-233003r2 -->
</div>
<hr class="endfloat" />
<!--l. 517-->
<p class="indent">
<a 
 id="x1-233004r3"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 519-->
<p class="noindent">
</p>
<!--l. 520-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/Discrete_Pi3k/SL_distance_vioplot_mtSL".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure M.3: </span><span class="content"><strong><span class="cmbx-10x-x-109">Structure of Synthetic Lethality in PI3K.</span></strong> <span class="cmr-10x-x-109">The number of mtSLIPT</span> <span class="cmr-10x-x-109">and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes upstream or downstream of each gene in the Reactome PI3K pathway</span> <span class="cmr-10x-x-109">were tested (by the </span><span class="cmmi-10x-x-109">χ</span><sup><span class="cmr-8">2</span></sup><span class="cmr-10x-x-109">-test). These are plotted as a split violin plot against the</span> <span class="cmr-10x-x-109">hierarchical distance scores showing no significant tendency for either method to either</span> <span class="cmr-10x-x-109">of the pathway upstream or downstream extremities. </span></span></p>
</div>
<!--tex4ht:label?: x1-233004r3 -->
</div>
<hr class="endfloat" />
<!--l. 534-->
<p class="indent">
<a 
 id="x1-233005r4"></a>
</p>
<hr class="float" />
<div class="float">
<hr />
____________________________________________________________________________________     
<div class="center">
<!--l. 536-->
<p class="noindent">
</p>
<!--l. 537-->
<p class="noindent">
<span class="fbox"> <img 
src=""/home/tomkelly/Downloads/Pathway_Structure/test_PI3K_mtSL".png" alt="pict"  
 /> </span>
</p>
</div>
<br />
<div class="caption">
<p><span class="id">Figure M.4: </span><span class="content"><strong><span class="cmbx-10x-x-109">Structure of Synthetic Lethality Resampling.</span></strong> <span class="cmr-10x-x-109">A null distribution</span> <span class="cmr-10x-x-109">(10,000 iterations) of the </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes upstream or downstream of mtSLIPT genes (shown</span> <span class="cmr-10x-x-109">by the difference) in the PI3K pathway. The observed events (red) were compared to the</span> <span class="cmr-10x-x-109">the distribution (violet) and were not significant. Genes detected by both methods were</span> <span class="cmr-10x-x-109">fixed for the distribution (blue). The genes detected by both approaches were used. </span></span></p>
</div>
<!--tex4ht:label?: x1-233005r4 -->
</div>
<hr class="endfloat" />
<!--l. 552-->
<p class="indent">
<a 
 id="x1-233006r2"></a>
</p>
<hr class="float" />
<div class="float" 
>


<div class="caption">
<p><span class="id">Table M.2: </span><span class="content">Resampling for pathway structure of synthetic lethal detection methods</span></p>
</div>
<!--tex4ht:label?: x1-233006r2 -->
<div class="tabular">
<pre><code>                                        &lt;table id=&quot;TBL-130&quot; class=&quot;tabular&quot; </code></pre>
cellspacing=“0” cellpadding=“0” rules=“groups” &gt;
<colgroup id="TBL-130-1g">
<col 
id="TBL-130-1" />
</colgroup>
<colgroup id="TBL-130-2g">
<col 
id="TBL-130-2" />
<col 
id="TBL-130-3" />
</colgroup>
<colgroup id="TBL-130-4g">
<col 
id="TBL-130-4" />
<col 
id="TBL-130-5" />
</colgroup>
<colgroup id="TBL-130-6g">
<col 
id="TBL-130-6" />
<col 
id="TBL-130-7" />
<col 
id="TBL-130-8" />
<col 
id="TBL-130-9" />
</colgroup>
<colgroup id="TBL-130-10g">
<col 
id="TBL-130-10" />
<col 
id="TBL-130-11" />
</colgroup>
<tr style="vertical-align:baseline;" id="TBL-130-1-">
<td style="white-space:nowrap; text-align:left;" id="TBL-130-1-1" class="td11">
</td>
</tr>
<tr 
class="cline">
<td>
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-130-2-">
<td style="white-space:nowrap; text-align:left;" id="TBL-130-2-1" class="td11">
<pre><code>                                 &lt;/td&gt;&lt;td colspan=&quot;2&quot; style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-130-2-2&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><strong><span class="cmbx-12">Graph</span></strong></p>
</div>
<pre><code>  &lt;/td&gt;&lt;td colspan=&quot;2&quot; style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-130-2-4&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><strong><span class="cmbx-12">States</span></strong></p>
</div>
<pre><code>  &lt;/td&gt;&lt;td colspan=&quot;4&quot; style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-130-2-6&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><strong><span class="cmbx-12">Observed</span></strong></p>
</div>
<pre><code>              &lt;/td&gt;&lt;td colspan=&quot;2&quot; style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-130-2-10&quot;  </code></pre>
class=“td11”&gt;
<div class="multicolumn" style="white-space:nowrap; text-align:center;">
<p><strong><span class="cmbx-12">Permutation p-value</span></strong></p>
</div>
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-130-3-">
<td style="white-space:nowrap; text-align:left;" id="TBL-130-3-1" class="td11">
Pathway
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-3-2" class="td11">
Nodes
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-3-3" class="td11">
Edges
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-3-4" class="td11">
mtSL
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-3-5" class="td11">
siRNA
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-3-6" class="td11">
Up
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-3-7" class="td11">
Down
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-3-8" class="td11">
Up<span class="cmsy-10x-x-120">-</span>Down
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-3-9" class="td11">
Up<span class="cmmi-12">∕</span>Down
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-3-10" class="td11">
Up<span class="cmsy-10x-x-120">-</span>Down
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-3-11" class="td11">
Down<span class="cmsy-10x-x-120">-</span>Up
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-130-4-">
<td style="white-space:nowrap; text-align:left;" id="TBL-130-4-1" class="td11">
PI3K Cascade
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-4-2" class="td11">
138
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-4-3" class="td11">
1495
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-4-4" class="td11">
42
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-4-5" class="td11">
25
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-4-6" class="td11">
131
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-4-7" class="td11">
123
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-4-8" class="td11">
<pre><code> 8        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-130-4-9&quot;  </code></pre>
class=“td11”&gt; 1.065
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-4-10" class="td11">
0.4473
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-4-11" class="td11">
0.5466
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-130-5-">
<td style="white-space:nowrap; text-align:left;" id="TBL-130-5-1" class="td11">
PI3K/AKT Signaling in Cancer
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-5-2" class="td11">
275
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-5-3" class="td11">
12882
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-5-4" class="td11">
56
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-5-5" class="td11">
44
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-5-6" class="td11">
478
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-5-7" class="td11">
440
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-5-8" class="td11">
<pre><code>38       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-130-5-9&quot;  </code></pre>
class=“td11”&gt; 1.086
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-5-10" class="td11">
0.4163
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-5-11" class="td11">
0.5810
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-130-6-">
<td style="white-space:nowrap; text-align:left;" id="TBL-130-6-1" class="td11">
<strong><span class="cmbx-12">G</span><sub><span class="cmmi-8">αi</span> </sub> <span class="cmbx-12">Signaling</span></strong>
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-6-2" class="td11">
292
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-6-3" class="td11">
22003
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-6-4" class="td11">
57
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-6-5" class="td11">
58
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-6-6" class="td11">
543
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-6-7" class="td11">
866
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-6-8" class="td11">
<pre><code>-323      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-130-6-9&quot;  </code></pre>
class=“td11”&gt; 0.627
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-6-10" class="td11">
0.9507
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-6-11" class="td11">
0.0488
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-130-7-">
<td style="white-space:nowrap; text-align:left;" id="TBL-130-7-1" class="td11">
GPCR downstream
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-7-2" class="td11">
1270
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-7-3" class="td11">
142071
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-7-4" class="td11">
218
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-7-5" class="td11">
160
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-7-6" class="td11">
7632
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-7-7" class="td11">
6500
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-7-8" class="td11">
1132
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-7-9" class="td11">
1.174
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-7-10" class="td11">
0.1707
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-7-11" class="td11">
0.8291
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-130-8-">
<td style="white-space:nowrap; text-align:left;" id="TBL-130-8-1" class="td11">
Elastic fibre formation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-8-2" class="td11">
42
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-8-3" class="td11">
175
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-8-4" class="td11">
16
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-8-5" class="td11">
7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-8-6" class="td11">
6
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-8-7" class="td11">
7
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-8-8" class="td11">
<pre><code> -1       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-130-8-9&quot;  </code></pre>
class=“td11”&gt; 0.857
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-8-10" class="td11">
0.5512
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-8-11" class="td11">
0.3681
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-130-9-">
<td style="white-space:nowrap; text-align:left;" id="TBL-130-9-1" class="td11">
Extracellular matrix
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-9-2" class="td11">
299
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-9-3" class="td11">
3677
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-9-4" class="td11">
81
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-9-5" class="td11">
29
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-9-6" class="td11">
313
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-9-7" class="td11">
347
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-9-8" class="td11">
<pre><code>-34      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-130-9-9&quot;  </code></pre>
class=“td11”&gt; 0.902
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-9-10" class="td11">
0.5762
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-9-11" class="td11">
0.4215
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-130-10-">
<td style="white-space:nowrap; text-align:left;" id="TBL-130-10-1" class="td11">
Formation of Fibrin
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-10-2" class="td11">
52
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-10-3" class="td11">
243
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-10-4" class="td11">
11
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-10-5" class="td11">
5
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-10-6" class="td11">
8
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-10-7" class="td11">
19
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-10-8" class="td11">
<pre><code>-11      &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-130-10-9&quot;  </code></pre>
class=“td11”&gt; 0.421
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-10-10" class="td11">
0.7993
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-10-11" class="td11">
0.1800
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-130-11-">
<td style="white-space:nowrap; text-align:left;" id="TBL-130-11-1" class="td11">
Nonsense-Mediated Decay
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-11-2" class="td11">
103
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-11-3" class="td11">
102
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-11-4" class="td11">
56
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-11-5" class="td11">
2
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-11-6" class="td11">
0
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-11-7" class="td11">
0
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-11-8" class="td11">
<pre><code> 0        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-130-11-9&quot;  </code></pre>
class=“td11”&gt;
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-11-10" class="td11">
0.197
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-11-11" class="td11">
0.1373
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-130-12-">
<td style="white-space:nowrap; text-align:left;" id="TBL-130-12-1" class="td11">
3<sup><span class="cmsy-8">′</span> </sup> -UTR-mediated translational regulation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-12-2" class="td11">
107
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-12-3" class="td11">
2860
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-12-4" class="td11">
56
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-12-5" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-12-6" class="td11">
52
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-12-7" class="td11">
1
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-12-8" class="td11">
<pre><code>51       &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-130-12-9&quot;  </code></pre>
class=“td11”&gt; 52
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-12-10" class="td11">
0.1210
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-12-11" class="td11">
0.8751
</td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-130-13-">
<td style="white-space:nowrap; text-align:left;" id="TBL-130-13-1" class="td11">
Eukaryotic Translation Elongation
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-13-2" class="td11">
92
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-13-3" class="td11">
3746
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-13-4" class="td11">
57
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-13-5" class="td11">
0
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-13-6" class="td11">
0
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-13-7" class="td11">
0
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-13-8" class="td11">
<pre><code> 0        &lt;/td&gt;&lt;td  style=&quot;white-space:nowrap; text-align:center;&quot; id=&quot;TBL-130-13-9&quot;  </code></pre>
class=“td11”&gt;
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-13-10" class="td11">
0.4952
</td>
<td style="white-space:nowrap; text-align:center;" id="TBL-130-13-11" class="td11">
0.4892
</td>
</tr>
<tr 
class="hline">
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
<td>
<hr />
</td>
</tr>
<tr  
 style="vertical-align:baseline;" id="TBL-130-14-">
<td  style="white-space:nowrap; text-align:left;" id="TBL-130-14-1"  
class="td11">
<pre><code>                                 &lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;</code></pre>
<!--l. 592-->
<p class="noindent">
</div>
<p><span class="cmr-10x-x-109">Pathways in the Reactome network tested for structural relationships between mtSLIPT and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">genes by resampling. The raw p-value (computed without adjusting for multiple</span> <span class="cmr-10x-x-109">comparisons over pathways) is given for the difference in upstream and downstream paths from mtSLIPT to </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">gene candidate partners of CDH1 with significant pathways</span> <span class="cmr-10x-x-109">highlighted in bold. Sampling was performed only in the target pathway and shortest paths were computed within it. Loops or paths in either direction that could not be resolved</span> <span class="cmr-10x-x-109">were excluded from the analysis. The gene detected by both mtSLIPT and </span><a 
href="#glo:siRNA"><span class="cmr-10x-x-109">siRNA</span></a> <span class="cmr-10x-x-109">(or resampling for them) were includued in the analysis and the number of these were fixed to the</span> <span class="cmr-10x-x-109">number observed.</span></p>
</p>
</div>
<hr class="endfloat" />
<a 
 id="x1-233007r288"></a>
<h2 class="appendixHead">
<span class="titlemark">Appendix N</span><br /><a 
 id="x1-234000N"></a>Simulation
</h2>
</body>
</html>
</body>
</html>
